<html lang="en"><head><!-- Shady DOM styles for custom-style --><!-- Shady DOM styles for dom-template --><!-- Shady DOM styles for dom-repeat --><!-- Shady DOM styles for array-selector --><!-- Shady DOM styles for dom-if --><!-- Shady DOM styles for state-modifier --><style scope="state-modifier">a[inherit].state-modifier {
  color: inherit;
}</style><!-- Shady DOM styles for browser-ui --><!-- Shady DOM styles for outlined-text-field --><style scope="outlined-text-field">outlined-text-field {
  display: flex;
  font-family: "Google Sans", Roboto, Arial, sans-serif;
  font-weight: 500;
  flex-direction: column;
}

outlined-text-field span.outlined-text-field {
  display: flex;
  flex-direction: row;
  align-items: center;
}

outlined-text-field input.outlined-text-field {
  height: 40px;
  padding: 0 33px 0 16px;
  
  border-color: #b0b3b6;
  border-width: 1px;
  border-style: solid;
  
  border-radius: 8px;
  caret-color: #3c78e0;
  color: #6a6c71;
  font-size: 14px;
  flex-grow: 1;
}

outlined-text-field[orig] input.outlined-text-field {
  border-color: #e5e5e5;
  
  border-radius: 0;
}

outlined-text-field[dense] input.outlined-text-field {
  height: 32px;
}

outlined-text-field[uppercase] input.outlined-text-field {
  text-transform: uppercase;
}

outlined-text-field:not([wizard-focused]):hover input.outlined-text-field {
  border-color: #737375;
  color: #414447;
}

outlined-text-field:not([wizard-focused]):focus-within input.outlined-text-field,outlined-text-field[wizard-focused] input.outlined-text-field {
  border-color: #3574e0;
  caret-color: #80a5f1;
}

outlined-text-field[wizard-focused] input.outlined-text-field {
  border-width: 2px;
}

outlined-text-field input.outlined-text-field::placeholder {
  color: #65686c;
}

outlined-text-field[uppercase] input.outlined-text-field::placeholder {
  text-transform: none;
}</style><!-- Shady DOM styles for mat-keyword-editor --><!-- Shady DOM styles for mat-chips-input --><!-- Shady DOM styles for mat-input-chip --><!-- Shady DOM styles for pat-dialog --><style scope="pat-dialog">pat-dialog {
  background: white;
  position: absolute;
  border-radius: 8px;
  
  box-shadow: 0 24px 38px 3px rgb(0 0 0 / 14%),
              0 9px 46px 8px rgb(0 0 0 / 12%),
              0 11px 15px -7px rgb(0 0 0 / 20%);
  z-index: 1;
  display: flex;
  flex-direction: column;
}

pat-dialog:not([open]) {
  display: none;
}</style><!-- Shady DOM styles for mat-tabs --><!-- Shady DOM styles for filter-chip --><style scope="filter-chip">filter-chip {
  align-items: center;
  background-color: #fff;
  border-color: #dadce0;
  border-radius: 8px;
  border-style: solid;
  border-width: 1px;
  color: #606368;
  cursor: pointer;
  font-family: "Google Sans", Roboto, Arial, sans-serif;
  font-size: 14px;
  font-weight: 500;
  height: 32px;
  letter-spacing: 0.2px;
  padding: 0 16px;
  user-select: none;
  display: inline-flex;
  white-space: nowrap;
}

filter-chip[disabled] {
  background-color: #e8e8e8;
  border-color: #e8e8e8;
  color: #a9abad;
  cursor: auto;
}

filter-chip svg.filter-chip {
  visibility: collapse;
  font-size: 18px;
  width: 0;
  height: 18px;
}

filter-chip[selected] svg.filter-chip {
  visibility: visible;
  margin: 0 8px;
  width: 18px;
}

filter-chip[selected] svg.filter-chip path.filter-chip {
  fill: #4577d1;
}

filter-chip:not([disabled]):hover {
  background-color: #f7f8f8;
  border-color: #d7d9dc;
  color: #454649;
}

filter-chip:not([disabled]):hover:active {
  background-color: #eff0f0;
  border-color: #d2d5d8;
  color: #222326;
}

filter-chip:not([disabled]):focus {
  background-color: #e7e8e8;
  border-color: #232528;
  color: #202124;
}

filter-chip:not([disabled])[selected] {
  background-color: #e9f0fd;
  border-color: #e9f0fd;
  color: #4577d1;
  padding-left: 0;
}

filter-chip:not([disabled]):focus[selected] {
  background-color: #d5e1fb;
  border-color: #d5e1fb;
  color: #2550a0;
}

filter-chip:not([disabled]):hover[selected] {
  box-shadow: rgba(0, 0, 0, 0.2) 0px 3px 1px -2px,
    rgba(0, 0, 0, 0.14) 0px 2px 2px 0px, rgba(0, 0, 0, 0.12) 0px 1px 5px 0px;
}

filter-chip:not([disabled]):hover:active[selected] {
  background-color: #abc8f2;
  border-color: #abc8f2;
  color: #2a54a3;
}</style><!-- Shady DOM styles for figure-callout --><!-- Shady DOM styles for mat-button --><style scope="mat-button-0">.mat-button-0 {
  ;
  align-items: center;
  border-radius: 4px;
  color: hsl(217 4% 39%); 
  cursor: pointer;
  font-family: "Google Sans", Roboto, Arial, sans-serif;
  font-size: 14px;
  font-weight: 500;
  height: 36px;
  letter-spacing: 0.2px;
  padding: 0 8px;
  user-select: none;
  display: inline-flex;
  white-space: nowrap;
}

.mat-button-0[filled] {
  background-color: hsl(217 73% 54%); 
  color: white;
}

.mat-button-0[filled]:hover {
  background-color: hsl(217 60% 47%);
}

.mat-button-0[filled]:focus {
  background-color: hsl(217 59% 44%);
}

.mat-button-0[filled]:active {
  background-color: hsl(217 59% 45%);
}

.mat-button-0[filled][disabled] {
  background-color: hsl(217 2% 90%); 
  color: hsl(217 1% 65%);
}</style><!-- Shady DOM styles for search-term-wizard --><style scope="search-term-wizard">search-term-wizard:not([open]) {
  display: none;
}

search-term-wizard .header.search-term-wizard {
  border-bottom: 1px solid #e5e5e5;
  padding: 24px;
  padding-bottom: 0;
}

search-term-wizard .title.search-term-wizard {
  color: #3c4043;
  font-family: Google Sans,Roboto,Arial,sans-serif;
  font-size: 22px;
  user-select: none;
  margin: 0 48px;
}

search-term-wizard #panels.search-term-wizard > *.search-term-wizard:not([selected]) {
  display: none;
}</style><!-- Shady DOM styles for outlined-textarea --><style scope="outlined-textarea">outlined-textarea {
  display: flex;
  font-family: "Google Sans", Roboto, Arial, sans-serif;
  font-weight: 500;
  flex-direction: row;
  
  position: relative;
}

outlined-textarea span.outlined-textarea {
  display: flex;
  flex-direction: column;
  flex-grow: 1;
  justify-content: center;
}

outlined-textarea span.right-button-container.outlined-textarea {
  position: absolute;
  right: 0;
  height: 100%;
}

outlined-textarea textarea.outlined-textarea {
  padding: 16px 16px 0 16px;
  border-color: #b0b3b6;
  border-width: 1px;
  border-style: solid;
  border-radius: 8px;
  caret-color: #3c78e0;
  color: #6a6c71;
  font-size: 14px;
  font-family: "Google Sans", Roboto, Arial, sans-serif;
  font-weight: 500;
  resize: none;
}

outlined-textarea[dense] textarea.outlined-textarea {
  padding: 8px;
}

outlined-textarea[has-icon] textarea.outlined-textarea {
  padding-right: 38px;
}

outlined-textarea[orig] textarea.outlined-textarea {
  border-color: #e5e5e5;
  
  border-radius: 0;
}

outlined-textarea:not([rows]) textarea.outlined-textarea {
  height: 40px;
}

outlined-textarea textarea.resizable.outlined-textarea {
  resize: vertical;
}

outlined-textarea textarea.outlined-textarea:focus,outlined-textarea[wizard-focused] textarea.outlined-textarea,outlined-textarea textarea.outlined-textarea:focus:hover {
  border-color: #3574e0 !important;
  
  outline: none !important;
}

outlined-textarea textarea.outlined-textarea:hover {
  border-color: #202124;
  color: #414447;
}

outlined-textarea.hasValue textarea.outlined-textarea {
  border-color: #888d91;
  color: #45474a;
}

outlined-textarea #helperText.outlined-textarea {
  color: #6a6c71;
  font-size: 12px;
  letter-spacing: 0.3px;
  margin-left: 17px;
  margin-top: 6px;
  user-select: none;
}

outlined-textarea:not([helper-text]) #helperText.outlined-textarea {
  display: none;
}

outlined-textarea[error] textarea.outlined-textarea {
  border-color: #cb4b3d;
  caret-color: #ca493a;
  color: #3d4043;
}

outlined-textarea textarea.outlined-textarea::placeholder {
  font-size: 14px;
  font-weight: 400;
}

outlined-textarea:not([error]):not([disabled]) textarea.outlined-textarea::placeholder {
  color: #6a6c71;
}

outlined-textarea[error] textarea.outlined-textarea::placeholder {
  color: #c84132;
}

outlined-textarea[error] #helperText.outlined-textarea {
  color: #cd5648;
}

outlined-textarea[error] textarea.outlined-textarea:focus {
  border-color: #c84031;
  border-width: 2px;
  outline: none !important;
}

outlined-textarea[error]:not([disabled]) textarea.outlined-textarea:hover {
  border-color: #97231b;
}

outlined-textarea[error]:not([disabled]) textarea.outlined-textarea:hover::placeholder,outlined-textarea[error]:not([disabled]) textarea.outlined-textarea:hover + #helperText.outlined-textarea {
  color: #a94945;
}

outlined-textarea[disabled] textarea.outlined-textarea {
  border-color: #e9e9e9;
  color: #b5b6b8;
}

outlined-textarea[disabled] #helperText.outlined-textarea {
  color: #c4c6c7;
}

outlined-textarea[error][disabled] {
  opacity: 50%;
}

outlined-textarea[error][disabled] textarea.outlined-textarea {
  border-color: #cb4b3d;
  caret-color: #ca493a;
}

outlined-textarea[error][disabled] #helperText.outlined-textarea,outlined-textarea[error][disabled] textarea.outlined-textarea {
  color: #a94945;
}

outlined-textarea[error][disabled] textarea.outlined-textarea::placeholder {
  color: #c84132;
}</style><!-- Shady DOM styles for chemistry-search-term-wizard --><style scope="chemistry-search-term-wizard">chemistry-search-term-wizard {
  display: flex;
  flex-direction: column;
  padding: 24px;
  padding-top: 0;
}

chemistry-search-term-wizard > span.chemistry-search-term-wizard {
  display: flex;
  flex-direction: row;
  align-items: center;
  flex-wrap: wrap;
  margin-top: 16px;
}

chemistry-search-term-wizard .suggestions.chemistry-search-term-wizard {
  margin-top: 4px;
}

chemistry-search-term-wizard > span.chemistry-search-term-wizard svg.chemistry-search-term-wizard {
  margin-right: 24px;
}

chemistry-search-term-wizard path.chemistry-search-term-wizard {
  fill: #5f6368;
}

chemistry-search-term-wizard .help.chemistry-search-term-wizard {
  margin-left: 12px;
  margin-right: 0;
  cursor: pointer;
}

chemistry-search-term-wizard outlined-text-field.chemistry-search-term-wizard {
  flex-grow: 1;
}

chemistry-search-term-wizard mat-button.chemistry-search-term-wizard {
  margin-left: 48px;
}

chemistry-search-term-wizard filter-chip.chemistry-search-term-wizard {
  margin-right: 4px;
}

chemistry-search-term-wizard outlined-textarea.chemistry-search-term-wizard {
  flex-grow: 1;
}

chemistry-search-term-wizard outlined-text-field[batchMode].chemistry-search-term-wizard {
  display: none;
}

chemistry-search-term-wizard outlined-textarea.chemistry-search-term-wizard:not([batchMode]) {
  display: none;
}</style><!-- Shady DOM styles for text-search-term-wizard --><style scope="text-search-term-wizard">text-search-term-wizard {
  display: flex;
  flex-direction: column;
  padding: 24px;
  padding-top: 0;
}

text-search-term-wizard > span.text-search-term-wizard {
  display: flex;
  flex-direction: row;
  align-items: center;
  flex-wrap: wrap;
  margin-top: 16px;
}

text-search-term-wizard > span.text-search-term-wizard svg.text-search-term-wizard {
  margin-right: 24px;
}

text-search-term-wizard path.text-search-term-wizard {
  fill: #5f6368;
}

text-search-term-wizard outlined-text-field.text-search-term-wizard {
  flex-grow: 1;
}

text-search-term-wizard mat-button.text-search-term-wizard {
  margin-left: 48px;
}

text-search-term-wizard filter-chip.text-search-term-wizard {
  margin-right: 4px;
}</style><!-- Shady DOM styles for classification-wizard --><style scope="classification-wizard">classification-wizard {
  display: flex;
  flex-direction: column;
  padding: 24px;
  padding-top: 0;
  max-width: 674px;
}

classification-wizard > span.classification-wizard {
  display: flex;
  flex-direction: row;
  align-items: center;
  flex-wrap: wrap;
  margin-top: 16px;
}

classification-wizard .suggestions.classification-wizard {
  margin-top: 4px;
}

classification-wizard > span.classification-wizard svg.classification-wizard {
  margin-right: 24px;
}

classification-wizard path.classification-wizard {
  fill: #5f6368;
}

classification-wizard outlined-text-field.classification-wizard {
  flex-grow: 1;
}

classification-wizard mat-button.classification-wizard {
  margin-left: 48px;
}

classification-wizard filter-chip.classification-wizard {
  margin-right: 4px;
}

classification-wizard .no-matches.classification-wizard {
  margin-left: 64px;
}

classification-wizard .table-container.classification-wizard {
  max-height: 300px;
  overflow: scroll;
  margin-top: 8px;
}

classification-wizard .table-container.classification-wizard::-webkit-scrollbar {
  -webkit-appearance: none;
  width: 10px;
}

classification-wizard .table-container.classification-wizard::-webkit-scrollbar-thumb {
  border-radius: 5px;
  background-color: #5f636880;
  -webkit-box-shadow: 0 0 1px rgba(255,255,255,.5);
}

classification-wizard table.classification-wizard {
  margin-left: 51px;
  max-width: 575px;
}

classification-wizard tr.classification-wizard {
  margin-top: 4px;
}

classification-wizard tr.classification-wizard suggestive-chip.classification-wizard {
  margin-right: 8px;
}

classification-wizard #spinner.classification-wizard {
  justify-content: space-around;
  margin-left: 48px;
  transform: scale(0.5);
}</style><!-- Shady DOM styles for suggestive-chip --><!-- Shady DOM styles for mat-checkbox --><!-- Shady DOM styles for measure-search-term-wizard --><style scope="measure-search-term-wizard">measure-search-term-wizard {
  display: flex;
  flex-direction: column;
  padding: 24px;
  padding-top: 0;
  max-width: 674px;
}

measure-search-term-wizard > span.measure-search-term-wizard {
  display: flex;
  flex-direction: row;
  align-items: center;
  flex-wrap: wrap;
  margin-top: 16px;
}

measure-search-term-wizard .suggestions.measure-search-term-wizard {
  margin-top: 4px;
}

measure-search-term-wizard > span.measure-search-term-wizard svg.measure-search-term-wizard {
  margin-right: 24px;
}

measure-search-term-wizard path.measure-search-term-wizard {
  fill: #5f6368;
}

measure-search-term-wizard outlined-text-field.measure-search-term-wizard {
  flex-grow: 1;
}

measure-search-term-wizard mat-button.measure-search-term-wizard {
  margin-left: 48px;
}

measure-search-term-wizard filter-chip.measure-search-term-wizard {
  margin-right: 4px;
}</style><!-- Shady DOM styles for numerical-search-term-wizard --><style scope="numerical-search-term-wizard">numerical-search-term-wizard {
  display: flex;
  flex-direction: column;
  padding: 24px;
  padding-top: 0;
  max-width: 674px;
}

numerical-search-term-wizard > span.numerical-search-term-wizard {
  display: flex;
  flex-direction: row;
  align-items: center;
  flex-wrap: wrap;
  margin-top: 16px;
}

numerical-search-term-wizard .suggestions.numerical-search-term-wizard {
  margin-top: 4px;
}

numerical-search-term-wizard > span.numerical-search-term-wizard svg.numerical-search-term-wizard {
  margin-right: 24px;
}

numerical-search-term-wizard path.numerical-search-term-wizard {
  fill: #5f6368;
}

numerical-search-term-wizard outlined-text-field.numerical-search-term-wizard,numerical-search-term-wizard outlined-textarea.numerical-search-term-wizard {
  flex-grow: 1;
}

numerical-search-term-wizard mat-button.numerical-search-term-wizard {
  margin-left: 48px;
}

numerical-search-term-wizard filter-chip.numerical-search-term-wizard {
  margin-right: 4px;
}

numerical-search-term-wizard #spinner.numerical-search-term-wizard {
  justify-content: space-around;
  margin-left: 48px;
  transform: scale(0.5);
}

numerical-search-term-wizard #suggestions.numerical-search-term-wizard {
  margin-left: 58px;
  display: flex;
  flex-direction: column;
  align-items: flex-start;
}

numerical-search-term-wizard .suggestion.numerical-search-term-wizard {
  display: flex;
  flex-direction: row;
  align-items: center;
}

numerical-search-term-wizard .suggestion.numerical-search-term-wizard mat-checkbox.numerical-search-term-wizard {
  margin-right: 8px;
}</style><!-- Shady DOM styles for text-button --><style scope="text-button">text-button {
  align-items: center;
  border-radius: 4px;
  cursor: pointer;
  font-family: "Google Sans", Roboto, Arial, sans-serif;
  font-size: 14px;
  font-weight: 500;
  height: 36px;
  letter-spacing: .25px;
  margin: 6px 0;
  padding: 0 8px;
  user-select: none;
  display: inline-flex;
  white-space: nowrap;
  line-height: 36px;
  text-decoration: none;
  text-transform: none;
  color: #5f6368;
}

text-button:hover {
  background-color: #f8f9fa;
}

text-button:active {
  background-color: #d5e2fa;
}

text-button[selected] {
  background-color: #e8f0fe;
  color: #1967d2;
}

text-button[selected]:active {
  background-color: #d2e3fc;
}</style><!-- Shady DOM styles for mat-spinner --><!-- Shady DOM styles for requery-tile --><style scope="requery-tile">requery-tile {
  display: flex;
  justify-content: space-between;
  align-items: center;
  box-sizing: border-box;
  min-height: 40px;
  padding: 8px 12px 8px 8px;
  gap: 8px;

  border-radius: 16px;
  background: #f7f8f9;
  color: #1f1f1f;
}

requery-tile:hover {
  background: #d7d8d9;
}

requery-tile > [slot="image"] {
  flex-shrink: 0;

  border-radius: 8px;

  background: #fff;
}

.label.requery-tile {
  flex-grow: 999;

  font: 16px/20px "Google Sans", Roboto, Arial, sans-serif;
  font-weight: 500;
}

.search-icon.requery-tile {
  flex-shrink: 0;

  width: 24px;
  height: 24px;
  background-image: url('https://fonts.gstatic.com/s/i/short-term/release/googlesymbols/search/default/24px.svg');
}</style><!-- Shady DOM styles for s-more-inline --><style scope="s-more-inline">s-more-inline {
  display: block;
  margin-top: 24px;
  padding: 16px;
  padding-top: 0;
  position: relative;

  line-height: 18px;
  text-overflow: ellipsis;
  white-space: nowrap;
  cursor: pointer;
}

hr.s-more-inline {
  background-color: #dadce0;
  margin-top: 18px;
  position: absolute;
  border: 0;
  height: 1px;
  left: 0;
  width: 100%;
}

.button.s-more-inline {
  display: flex;
  align-items: center;
  justify-content: center;

  position: relative; 
  margin: 0 auto;

  font: 400 14px Roboto, arial, sans-serif;
  height: 36px;
  width: 300px;
  border-radius: 18px;
  background-color: #f1f3f4;
}

.button.s-more-inline:hover {
  background-color: #d8d7dc;
}

.button.s-more-inline > [slot] {
  padding-right: 8px;
}

.expand.s-more-inline {
  display: inline-block;
  width: 20px;
  height: 20px;
}

.expand.less.s-more-inline {
  background-image: url('https://fonts.gstatic.com/s/i/short-term/release/googlesymbols/expand_less/default/20px.svg');
}

.expand.more.s-more-inline {
  background-image: url('https://fonts.gstatic.com/s/i/short-term/release/googlesymbols/expand_more/default/20px.svg');
}</style><!-- Shady DOM styles for workspace-ui-search --><!-- Shady DOM styles for keyword-wizard --><style scope="keyword-wizard">keyword-wizard {
  display: inline-block;
}

iron-icon.keyword-wizard {
  height: 20px;
    width: 20px;
    color: #999;
    cursor: pointer;
}

iron-icon[active].keyword-wizard {
  color: #4285f4 !important;
}

iron-icon.keyword-wizard:hover {
  color: #ccc;
}

#wizardIcon.keyword-wizard {
  padding-left: 1px;
    height: 16px;
    width: 16px;
    margin: 4px 1px 1px 1px;
}

#wizard.keyword-wizard {
  padding: 0;
    left: 0;
    top: 37px;
    position: absolute;
    background-color: white;
    z-index: 2;
    overflow: hidden;
}

.wizard-header.keyword-wizard {
  display: block;
    width: 100%;
    color: #737373;
    text-transform: uppercase;
    font-size: 13px;
    margin: 16px 0 6px 12px;
}

.filter-chips.keyword-wizard {
  margin-left: 20px;
}

.chip.keyword-wizard {
  background-color: #ebebeb;
    color: #6a6a6a;
    display: inline-block;
    font-size: 13px;
    font-family: inherit;
    border-radius: 12px;
    padding: 6px 12px;
}

.chip.keyword-wizard:hover {
  cursor: pointer;
}

.chip[selected].keyword-wizard {
  background-color: #4285f4;
    color: white;
}

.input-label.keyword-wizard {
  display: inline-block;
    font-size: 13px;
    font-family: inherit;
}

.keyword-input.keyword-wizard {
  display: inline-block;
    border-top: 0;
    border-left: 0;
    border-right: 0;
    border-bottom: 1px solid #dfe1e5;
    padding: 3px;
}

.keyword-input.keyword-wizard::placeholder {
  font-style: italic;
}

.keyword-input.keyword-wizard:hover,.keyword-input.keyword-wizard:focus {
  border-color: #4284f4;
    border-width: 1px;
    outline: none;
}

.cpc-field.keyword-wizard {
  display: inline-block;
    margin: 6px 0 0 24px;
}

#checkboxCPC.keyword-wizard:hover {
  cursor: pointer;
}

.checkbox-label.keyword-wizard {
  font-size: 13px;
    font-style: italic;
    color: #737373;
}

.chemistry-field.keyword-wizard {
  display: block;
}

#inputChem.input-batch.keyword-wizard {
  height: 150px;
    border: 1px solid #dfe1e5;
    padding: 6px;
}

#inputChem.keyword-wizard {
  resize: none;
    height: 15px;
    vertical-align: text-top;
    font-size: 13px;
    font-family: inherit;
    padding: 2px;
}

.chem-type-label.keyword-wizard {
  display: inline;
    font-size: 13px;
    margin-left: 12px;
    margin-right: 4px;
}

#chemType.keyword-wizard {
  border-top: 0;
    border-left: 0;
    border-right: 0;
    border-bottom: 1px solid #dfe1e5;
    margin: 6px;
    padding: 2px;
    outline: none;
    font-family: inherit;
    font-size: 13px;
}

.import.keyword-wizard {
  font-size: 13px;
    font-family: inherit;
}

.import-text.keyword-wizard {
  vertical-align: middle;
    display: inline-block;
}

.import-button.keyword-wizard {
  margin: 6px 0 0 24px;
}

.import-button.keyword-wizard button.keyword-wizard {
  border: 0;
    border-radius: 4px;
    padding: 6px 12px;
    font-family: inherit;
    background-color: #4284f4;
    color: white;
}

.import-button.keyword-wizard button.keyword-wizard:hover {
  cursor: pointer;
    background-color: #4284f4;
    color: white;
}

.wizard-buttons.keyword-wizard {
  float: right;
    padding: 12px;
}

.wizard-buttons.keyword-wizard button.keyword-wizard {
  border: 0;
    border-radius: 4px;
    padding: 6px 12px;
    margin: 2px;
    font-family: inherit;
    background-color: #4284f4;
    color: white;
}

.wizard-buttons.keyword-wizard button.keyword-wizard:hover {
  cursor: pointer;
}

@media screen and (max-width: 700px) {
#wizard.keyword-wizard {
  width: 275px;
}

.chip.keyword-wizard {
  margin: 4px 2px;
}

.keyword-input.keyword-wizard {
  width: 82%;
      margin: 4px 12px 12px 24px;
}

.search-within.keyword-wizard .input-label.keyword-wizard {
  display: none;
}

.cpc-field.keyword-wizard .keyword-input.keyword-wizard {
  width: 225px;
      margin: 12px 0 6px 0;
}

.find-patents.keyword-wizard .input-label.keyword-wizard {
  display: none;
}

.search-by.keyword-wizard .input-label.keyword-wizard {
  width: 55px;
      margin-left: 24px;
}

.search-by.keyword-wizard .keyword-input.keyword-wizard {
  margin-top: 6px;
}

#inputChem.keyword-wizard {
  width: 82%;
      padding: 2px;
      margin: 8px 12px 12px 24px;
}

#inputChem.input-batch.keyword-wizard {
  width: 220px;
      padding: 4px;
}

#chemType.keyword-wizard {
  width: 150px;
}

.import-text.keyword-wizard {
  margin: 6px 12px 6px 24px;
      width: 220px;
}

iron-icon.help-header.keyword-wizard {
  display: none;
}

}

@media screen and (min-width: 700px) {
#wizard.keyword-wizard {
  width: 540px;
}

.chip.keyword-wizard {
  margin: 0 2px;
}

.cpc-field.keyword-wizard .keyword-input.keyword-wizard {
  width: 240px;
      margin: 12px 16px 6px 6px;
}

.cpc-inputs.keyword-wizard {
  display: inline;
}

.find-patents.keyword-wizard .keyword-input.keyword-wizard {
  width: 53%;
      margin: 4px 10px;
}

.cpc-field.keyword-wizard .keyword-input.keyword-wizard::placeholder,#inputAnd.keyword-wizard::placeholder,#inputExact.keyword-wizard::placeholder,#inputOr.keyword-wizard::placeholder,#inputMinus.keyword-wizard::placeholder {
  color: white;
}

.find-patents.keyword-wizard .input-label.keyword-wizard {
  width: 35%;
      margin-left: 24px;
}

.search-by.keyword-wizard .keyword-input.keyword-wizard {
  width: 42%;
      margin: 6px 4px;
}

.search-by.keyword-wizard .input-label.keyword-wizard {
  width: 55px;
      margin-left: 24px;
}

#inputChem.keyword-wizard {
  width: 290px;
      margin: 2px 12px 2px 6px;
}

#inputChem.input-batch.keyword-wizard {
  width: 280px;
      padding: 6px;
}

.import-text.keyword-wizard {
  margin: 6px 12px;
      width: 415px;
}

#chemType.keyword-wizard {
  width: 105px;
}

iron-icon.keyword-wizard:hover.help-header {
  cursor: default;
}

iron-icon.help-header.keyword-wizard {
  height: 16px;
      padding: 0 2px 2px 2px;
      color: #a3a3a3;
}

.help.keyword-wizard p.keyword-wizard {
  margin: 0 0 8px 0
}

.help.keyword-wizard .q.keyword-wizard {
  font-family: monospace;
      color: #0b8043;
      white-space: nowrap;
}

.help.keyword-wizard {
  z-index: 4;
      position: absolute;
      right: 10px;
      padding: 12px;
      border-radius: 2px;
      border: 1px solid #e0e0e0;
      box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.2), 0 6px 10px 0 rgba(0, 0, 0, 0.3);
      background: white;
      font-size: 13px;
      line-height: 20px;
      width: 450px;
      max-height: 400px;
      overflow: auto;
}

#withinHelp.keyword-wizard {
  top: 75px;
}

#chemHelp.keyword-wizard {
  top: 0;
}

#searchBooleanHelp.keyword-wizard {
  top: 140px;
}

}</style><!-- Shady DOM styles for bottom-bar --><style scope="bottom-bar">.fixed.bottom-bar {
  display: block;
  position: fixed;
  height: 46px;
  bottom: 0;
}

.fixed[medium].bottom-bar {
  height: 280px;
}

.fixed[large].bottom-bar {
  height: 480px;
}

.shadow.bottom-bar {
  background: white;
  padding: 16px;
  border-radius: 8px 8px 0px 0px;
  box-shadow: 0px 8px 10px 1px rgba(0,0,0,.14), 0px 3px 14px 2px rgba(0,0,0,.12), 0px 5px 5px -3px rgba(0,0,0,.2);
}

.oneline.bottom-bar {
  white-space: nowrap;
  overflow: hidden;
  text-overflow: ellipsis;
  cursor: pointer;
}

.button.bottom-bar {
  cursor: pointer;
}

.buttonAlign.bottom-bar {
  margin-top: -4px;
}

.header.bottom-bar {
  height: 30px;
  font-family: 'Google Sans';
  font-size: 18px;
  padding-bottom: 8px;
}

.list.bottom-bar {
  overflow-y: auto;
  line-height: 20px;
}

.definition.bottom-bar {
  padding: 0px 0px 4px 0px;
  cursor: pointer;
}

[medium].bottom-bar .content.bottom-bar {
  height: 230px;
}

[large].bottom-bar .content.bottom-bar {
  height: 430px;
}

[medium].bottom-bar .content.bottom-bar .list.bottom-bar {
  height: 200px;
}

[large].bottom-bar .content.bottom-bar .list.bottom-bar {
  height: 400px;
}

[hidden].bottom-bar {
  display: none;
}

.label.bottom-bar {
  font-family: 'Google Sans';
  font-size: 12px;
  line-height: 16px;
  font-weight: 400;
  padding: 6px 0px 4px 0px;
  color: #1f1f1f;
}

[pad].bottom-bar {
  margin-left: 16px;
}

.layout.vertical.bottom-bar {
  display: flex;
  flex-direction: column;
}

.layout.horizontal.bottom-bar {
  display: flex;
  flex-direction: row;
}

.flex.bottom-bar {
  flex: 1;
  flex-basis: 0.000000001px;
}

.layout.start.bottom-bar {
  align-items: flex-start;
}

.layout.center.bottom-bar {
  align-items: center;
}

.layout.end.bottom-bar {
  align-items: flex-end;
}

.flex-2.bottom-bar {
  flex: 2;
}

.flex-3.bottom-bar {
  flex: 3;
}

.flex-4.bottom-bar {
  flex: 4;
}

.flex-5.bottom-bar {
  flex: 5;
}

.flex-6.bottom-bar {
  flex: 6;
}

.flex-7.bottom-bar {
  flex: 7;
}

.flex-8.bottom-bar {
  flex: 8;
}

.flex-9.bottom-bar {
  flex: 9;
}

.flex-10.bottom-bar {
  flex: 10;
}

.flex-11.bottom-bar {
  flex: 11;
}

.flex-12.bottom-bar {
  flex: 12;
}</style><!-- Shady DOM styles for landscape-link --><!-- Shady DOM styles for landscape-summary-box --><!-- Shady DOM styles for search-state --><!-- Shady DOM styles for iron-overlay-backdrop --><!-- Shady DOM styles for opaque-animation --><!-- Shady DOM styles for iron-dropdown --><!-- Shady DOM styles for iron-meta --><!-- Shady DOM styles for iron-icon --><style scope="iron-icon-0">.iron-icon-0 {
  display: -ms-inline-flexbox;
        display: -webkit-inline-flex;
        display: inline-flex;-ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;-ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
        position: relative;

        vertical-align: middle;

        fill: currentcolor;
        stroke: none;

        width: 24px;
        height: 24px;;
}

.iron-icon-0[hidden] {
  display: none;
}</style><!-- Shady DOM styles for iron-iconset-svg --><!-- Shady DOM styles for iron-list --><!-- Shady DOM styles for paper-icon-item --><!-- Shady DOM styles for paper-item-body --><!-- Shady DOM styles for paper-item --><!-- Shady DOM styles for dropdown-menu --><!-- Shady DOM styles for chips-input --><!-- Shady DOM styles for input-chip --><!-- Shady DOM styles for keyword-editor --><!-- Shady DOM styles for paper-ripple --><!-- Shady DOM styles for paper-button --><!-- Shady DOM styles for date-editor --><!-- Shady DOM styles for metadata-editor --><!-- Shady DOM styles for restrict-editor --><!-- Shady DOM styles for overlay-tooltip --><style scope="overlay-tooltip">overlay-tooltip {
  display: none;
        color: #fff;
        background-color: #000;
        padding: 10px 12px;
        border-radius: 3px;
        font-size: 13px;
        font-weight: normal;
        line-height: normal;
        position: fixed;
        z-index: 100;
}

#arrow.overlay-tooltip {
  width: 0;
        height: 0;
        border-style: solid;
        border-width: 0 6px 8px 6px;
        border-color: transparent transparent #000 transparent;
        position: absolute;
        top: -8px;
        left: calc(50% - 6px);
}</style><!-- Shady DOM styles for query-renderer --><!-- Shady DOM styles for result-nav --><style scope="result-nav">result-nav {
  display: block;
        white-space: nowrap;
        padding: 0 30px;
        line-height: 30px;
        height: 30px;
        margin-top: 17px;
        font-weight: 500;
        font-size: 15px;
}

state-modifier.result-nav {
  color: #333;
}

state-modifier[disabled].result-nav {
  pointer-events: none;
        color: #ccc;
}</style><!-- Shady DOM styles for raw-html --><!-- Shady DOM styles for paper-material --><style scope="paper-material-0">.paper-material-0 {
  display: block;
        position: relative;
}

.paper-material-0[elevation="1"] {
  box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.14),
                    0 1px 5px 0 rgba(0, 0, 0, 0.12),
                    0 3px 1px -2px rgba(0, 0, 0, 0.2);
}

.paper-material-0[elevation="2"] {
  box-shadow: 0 4px 5px 0 rgba(0, 0, 0, 0.14),
                    0 1px 10px 0 rgba(0, 0, 0, 0.12),
                    0 2px 4px -1px rgba(0, 0, 0, 0.4);
}

.paper-material-0[elevation="3"] {
  box-shadow: 0 6px 10px 0 rgba(0, 0, 0, 0.14),
                    0 1px 18px 0 rgba(0, 0, 0, 0.12),
                    0 3px 5px -1px rgba(0, 0, 0, 0.4);
}

.paper-material-0[elevation="4"] {
  box-shadow: 0 8px 10px 1px rgba(0, 0, 0, 0.14),
                    0 3px 14px 2px rgba(0, 0, 0, 0.12),
                    0 5px 5px -3px rgba(0, 0, 0, 0.4);
}

.paper-material-0[elevation="5"] {
  box-shadow: 0 16px 24px 2px rgba(0, 0, 0, 0.14),
                    0  6px 30px 5px rgba(0, 0, 0, 0.12),
                    0  8px 10px -5px rgba(0, 0, 0, 0.4);
}

.paper-material-0[animated] {
  transition: box-shadow 0.28s cubic-bezier(0.4, 0, 0.2, 1);
}

.paper-material-0 {
  ;
}</style><!-- Shady DOM styles for search-box --><style scope="search-box">.layout.horizontal.search-box,.layout.vertical.search-box {
  display: -ms-flexbox;
        display: -webkit-flex;
        display: flex;
}

.layout.inline.search-box {
  display: -ms-inline-flexbox;
        display: -webkit-inline-flex;
        display: inline-flex;
}

.layout.horizontal.search-box {
  -ms-flex-direction: row;
        -webkit-flex-direction: row;
        flex-direction: row;
}

.layout.vertical.search-box {
  -ms-flex-direction: column;
        -webkit-flex-direction: column;
        flex-direction: column;
}

.layout.wrap.search-box {
  -ms-flex-wrap: wrap;
        -webkit-flex-wrap: wrap;
        flex-wrap: wrap;
}

.layout.no-wrap.search-box {
  -ms-flex-wrap: nowrap;
        -webkit-flex-wrap: nowrap;
        flex-wrap: nowrap;
}

.layout.center.search-box,.layout.center-center.search-box {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.layout.center-justified.search-box,.layout.center-center.search-box {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.flex.search-box {
  -ms-flex: 1 1 0.000000001px;
        -webkit-flex: 1;
        flex: 1;
        -webkit-flex-basis: 0.000000001px;
        flex-basis: 0.000000001px;
}

.flex-auto.search-box {
  -ms-flex: 1 1 auto;
        -webkit-flex: 1 1 auto;
        flex: 1 1 auto;
}

.flex-none.search-box {
  -ms-flex: none;
        -webkit-flex: none;
        flex: none;
}

.inputbox.search-box {
  border: 1px solid #ccc;
        padding: 0 12px;
        background: white;
}

.inputbox[focused].search-box {
  border: 1px solid #4285f4 !important;
}

.inputbox.search-box:hover {
  border: 1px solid #999;
}

#searchInput.search-box {
  font-family: inherit;
        font-size: 16px;
        padding: 8px 0;
        outline: none;
        border-right: none;
        width: 100%;
        margin-right: 0;
        border: 0;
}

#searchButton.search-box {
  cursor: pointer;
        background-color: #4285f4;
        border: none;
        color: #fff;
        vertical-align: top;
        width: 70px;
        height: 37px;
        outline: none;
}

.suggest-bg.search-box {
  display: inline-block;
        min-width: 0;
        white-space: nowrap;
}

#suggestMenu.search-box {
  position: absolute;
        margin-top: 38px;
        width: 100%;
        background: white;
        z-index: 1;
}

#suggestMenu.search-box .row.search-box {
  font-size: 13px;
        line-height: 28px;
        white-space: nowrap;
        overflow: hidden;
        text-overflow: ellipsis;
        padding: 4px 2px;
        cursor: pointer;
}

.row[selected].search-box {
  background: rgba(0, 0, 0, 0.05);
}

.suggest-icon.search-box {
  height: 20px;
        width: 20px;
        padding: 0 12px 0 6px;
        transition: transform .15s linear;
        transform: scale(1);
}

.suggest-icon[hide].search-box {
  transform: scale(0);
}

#corpora.search-box {
  padding: 6px 0;
        user-select: none;
}

#corpora.search-box iron-icon.search-box {
  height: 20px;
        width: 20px;
        color: #ccc;
        cursor: pointer;
}

#corpora.search-box iron-icon[active].search-box {
  color: #4285f4 !important;
}

#corpora.search-box iron-icon.search-box:hover {
  color: #999;
}</style><!-- Shady DOM styles for iron-media-query --><!-- Shady DOM styles for paper-header-panel --><!-- Shady DOM styles for iron-a11y-announcer --><style scope="iron-a11y-announcer">iron-a11y-announcer {
  display: inline-block;
        position: fixed;
        clip: rect(0px,0px,0px,0px);
}</style><!-- Shady DOM styles for iron-autogrow-textarea --><!-- Shady DOM styles for iron-input --><!-- Shady DOM styles for paper-input-char-counter --><!-- Shady DOM styles for paper-input-container --><!-- Shady DOM styles for paper-input-error --><!-- Shady DOM styles for paper-input --><!-- Shady DOM styles for paper-textarea --><!-- Shady DOM styles for paper-toast --><style scope="paper-toast-0">.paper-toast-0 {
  display: block;
        position: fixed;
        background-color: #323232;
        color: #f1f1f1;
        min-height: 48px;
        min-width: 288px;
        padding: 16px 24px;
        box-sizing: border-box;
        box-shadow: 0 2px 5px 0 rgba(0, 0, 0, 0.26);
        border-radius: 2px;
        margin: 12px;
        font-size: 14px;
        cursor: default;
        -webkit-transition: -webkit-transform 0.3s, opacity 0.3s;
        transition: transform 0.3s, opacity 0.3s;
        opacity: 0;
        -webkit-transform: translateY(100px);
        transform: translateY(100px);font-family: 'Roboto', 'Noto', sans-serif;
        -webkit-font-smoothing: antialiased;
}

.paper-toast-0.capsule {
  border-radius: 24px;
}

.paper-toast-0.fit-bottom {
  width: 100%;
        min-width: 0;
        border-radius: 0;
        margin: 0;
}

.paper-toast-0.paper-toast-open {
  opacity: 1;
        -webkit-transform: translateY(0px);
        transform: translateY(0px);
}</style><!-- Shady DOM styles for paper-toolbar --><!-- Shady DOM styles for search-footer --><style scope="search-footer">#footer.search-footer {
  font-size: 13px;
        margin-top: 10px;
        background: #fafafa;
        padding: 20px 10px;
        align-items: center;
        border-top: 1px solid #ebebeb;
}

#footer.search-footer .links.search-footer > *.search-footer {
  display: inline-block;
        margin: 0 12px;
}

.layout.horizontal.search-footer,.layout.vertical.search-footer {
  display: -ms-flexbox;
        display: -webkit-flex;
        display: flex;
}

.layout.inline.search-footer {
  display: -ms-inline-flexbox;
        display: -webkit-inline-flex;
        display: inline-flex;
}

.layout.horizontal.search-footer {
  -ms-flex-direction: row;
        -webkit-flex-direction: row;
        flex-direction: row;
}

.layout.vertical.search-footer {
  -ms-flex-direction: column;
        -webkit-flex-direction: column;
        flex-direction: column;
}

.layout.wrap.search-footer {
  -ms-flex-wrap: wrap;
        -webkit-flex-wrap: wrap;
        flex-wrap: wrap;
}

.layout.no-wrap.search-footer {
  -ms-flex-wrap: nowrap;
        -webkit-flex-wrap: nowrap;
        flex-wrap: nowrap;
}

.layout.center.search-footer,.layout.center-center.search-footer {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.layout.center-justified.search-footer,.layout.center-center.search-footer {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.flex.search-footer {
  -ms-flex: 1 1 0.000000001px;
        -webkit-flex: 1;
        flex: 1;
        -webkit-flex-basis: 0.000000001px;
        flex-basis: 0.000000001px;
}

.flex-auto.search-footer {
  -ms-flex: 1 1 auto;
        -webkit-flex: 1 1 auto;
        flex: 1 1 auto;
}

.flex-none.search-footer {
  -ms-flex: none;
        -webkit-flex: none;
        flex: none;
}

.layout.start.search-footer {
  -ms-flex-align: start;
        -webkit-align-items: flex-start;
        align-items: flex-start;
}

.layout.center.search-footer,.layout.center-center.search-footer {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.layout.end.search-footer {
  -ms-flex-align: end;
        -webkit-align-items: flex-end;
        align-items: flex-end;
}

.layout.baseline.search-footer {
  -ms-flex-align: baseline;
        -webkit-align-items: baseline;
        align-items: baseline;
}

.layout.start-justified.search-footer {
  -ms-flex-pack: start;
        -webkit-justify-content: flex-start;
        justify-content: flex-start;
}

.layout.center-justified.search-footer,.layout.center-center.search-footer {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.layout.end-justified.search-footer {
  -ms-flex-pack: end;
        -webkit-justify-content: flex-end;
        justify-content: flex-end;
}

.layout.around-justified.search-footer {
  -ms-flex-pack: distribute;
        -webkit-justify-content: space-around;
        justify-content: space-around;
}

.layout.justified.search-footer {
  -ms-flex-pack: justify;
        -webkit-justify-content: space-between;
        justify-content: space-between;
}

.self-start.search-footer {
  -ms-align-self: flex-start;
        -webkit-align-self: flex-start;
        align-self: flex-start;
}

.self-center.search-footer {
  -ms-align-self: center;
        -webkit-align-self: center;
        align-self: center;
}

.self-end.search-footer {
  -ms-align-self: flex-end;
        -webkit-align-self: flex-end;
        align-self: flex-end;
}

.self-stretch.search-footer {
  -ms-align-self: stretch;
        -webkit-align-self: stretch;
        align-self: stretch;
}

.self-baseline.search-footer {
  -ms-align-self: baseline;
        -webkit-align-self: baseline;
        align-self: baseline;
}

.layout.start-aligned.search-footer {
  -ms-flex-line-pack: start;  
        -ms-align-content: flex-start;
        -webkit-align-content: flex-start;
        align-content: flex-start;
}

.layout.end-aligned.search-footer {
  -ms-flex-line-pack: end;  
        -ms-align-content: flex-end;
        -webkit-align-content: flex-end;
        align-content: flex-end;
}

.layout.center-aligned.search-footer {
  -ms-flex-line-pack: center;  
        -ms-align-content: center;
        -webkit-align-content: center;
        align-content: center;
}

.layout.between-aligned.search-footer {
  -ms-flex-line-pack: justify;  
        -ms-align-content: space-between;
        -webkit-align-content: space-between;
        align-content: space-between;
}

.layout.around-aligned.search-footer {
  -ms-flex-line-pack: distribute;  
        -ms-align-content: space-around;
        -webkit-align-content: space-around;
        align-content: space-around;
}</style><!-- Shady DOM styles for search-header --><style scope="search-header">search-header {
  display: block;
}

.lockup-logo.search-header,.lockup-text.search-header {
  display: inline-block;
}

.lockup-logo.search-header {
  width: 74px;
        height: 24px;
        background-image: url('https://www.gstatic.com/images/branding/googlelogo/1x/googlelogo_color_74x24dp.png');
        background-size: 74px 24px;
}

.lockup-text.search-header {
  font-family: "Product Sans", Arial, Helvetica, sans-serif;

        
        font-size: 22.1px;

        
        text-rendering: optimizeLegibility;

        
        position: relative;
        top: -6px;
        left: -5px;

        
        -webkit-font-smoothing: antialiased;
        -moz-osx-font-smoothing: grayscale;
}

.lockup-brand.search-header {
  opacity: 0.54;
}

@media (-webkit-min-device-pixel-ratio: 1.5), (min-resolution: 144dpi) {
.lockup-logo.search-header {
  background-image: url('https://www.gstatic.com/images/branding/googlelogo/2x/googlelogo_color_74x24dp.png');
}

.lockup-text.search-header {
  font-size: 21.3px;
          left: -7px;
}

}

header.search-header {
  background-color: #fafafa;
        border-bottom: 1px solid #ebebeb;
        color: #333;
        height: 60px;
        width: 100%;
        z-index: 1;
        display: block;
}

[latestStyle].search-header header.search-header {
  background-color: transparent;
}

.headerpad.search-header {
  margin-left: 16px;
}

[wide].search-header .headerpad.search-header {
  margin-left: 32px;
}

.leftheader.search-header {
  width: 322px;
}

header.search-header h1.search-header {
  display: none;
}

header.search-header h1.search-header state-modifier.search-header {
  color: #777;
        text-decoration: none;
        display: inline-block;
}

[wide].search-header header.search-header h1.search-header {
  box-sizing: border-box;
        margin: 0;
        padding: 20px 0 10px;
        font-size: 20px;
        font-weight: 400;
        white-space: nowrap;
        display: block;
}

search-box.search-header {
  height: 37px;
        padding: 0 16px;
        width: 900px;
        align-self: center;
}

.layout.horizontal.search-header,.layout.vertical.search-header {
  display: -ms-flexbox;
        display: -webkit-flex;
        display: flex;
}

.layout.inline.search-header {
  display: -ms-inline-flexbox;
        display: -webkit-inline-flex;
        display: inline-flex;
}

.layout.horizontal.search-header {
  -ms-flex-direction: row;
        -webkit-flex-direction: row;
        flex-direction: row;
}

.layout.vertical.search-header {
  -ms-flex-direction: column;
        -webkit-flex-direction: column;
        flex-direction: column;
}

.layout.wrap.search-header {
  -ms-flex-wrap: wrap;
        -webkit-flex-wrap: wrap;
        flex-wrap: wrap;
}

.layout.no-wrap.search-header {
  -ms-flex-wrap: nowrap;
        -webkit-flex-wrap: nowrap;
        flex-wrap: nowrap;
}

.layout.center.search-header,.layout.center-center.search-header {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.layout.center-justified.search-header,.layout.center-center.search-header {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.flex.search-header {
  -ms-flex: 1 1 0.000000001px;
        -webkit-flex: 1;
        flex: 1;
        -webkit-flex-basis: 0.000000001px;
        flex-basis: 0.000000001px;
}

.flex-auto.search-header {
  -ms-flex: 1 1 auto;
        -webkit-flex: 1 1 auto;
        flex: 1 1 auto;
}

.flex-none.search-header {
  -ms-flex: none;
        -webkit-flex: none;
        flex: none;
}

.layout.start.search-header {
  -ms-flex-align: start;
        -webkit-align-items: flex-start;
        align-items: flex-start;
}

.layout.center.search-header,.layout.center-center.search-header {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.layout.end.search-header {
  -ms-flex-align: end;
        -webkit-align-items: flex-end;
        align-items: flex-end;
}

.layout.baseline.search-header {
  -ms-flex-align: baseline;
        -webkit-align-items: baseline;
        align-items: baseline;
}

.layout.start-justified.search-header {
  -ms-flex-pack: start;
        -webkit-justify-content: flex-start;
        justify-content: flex-start;
}

.layout.center-justified.search-header,.layout.center-center.search-header {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.layout.end-justified.search-header {
  -ms-flex-pack: end;
        -webkit-justify-content: flex-end;
        justify-content: flex-end;
}

.layout.around-justified.search-header {
  -ms-flex-pack: distribute;
        -webkit-justify-content: space-around;
        justify-content: space-around;
}

.layout.justified.search-header {
  -ms-flex-pack: justify;
        -webkit-justify-content: space-between;
        justify-content: space-between;
}

.self-start.search-header {
  -ms-align-self: flex-start;
        -webkit-align-self: flex-start;
        align-self: flex-start;
}

.self-center.search-header {
  -ms-align-self: center;
        -webkit-align-self: center;
        align-self: center;
}

.self-end.search-header {
  -ms-align-self: flex-end;
        -webkit-align-self: flex-end;
        align-self: flex-end;
}

.self-stretch.search-header {
  -ms-align-self: stretch;
        -webkit-align-self: stretch;
        align-self: stretch;
}

.self-baseline.search-header {
  -ms-align-self: baseline;
        -webkit-align-self: baseline;
        align-self: baseline;
}

.layout.start-aligned.search-header {
  -ms-flex-line-pack: start;  
        -ms-align-content: flex-start;
        -webkit-align-content: flex-start;
        align-content: flex-start;
}

.layout.end-aligned.search-header {
  -ms-flex-line-pack: end;  
        -ms-align-content: flex-end;
        -webkit-align-content: flex-end;
        align-content: flex-end;
}

.layout.center-aligned.search-header {
  -ms-flex-line-pack: center;  
        -ms-align-content: center;
        -webkit-align-content: center;
        align-content: center;
}

.layout.between-aligned.search-header {
  -ms-flex-line-pack: justify;  
        -ms-align-content: space-between;
        -webkit-align-content: space-between;
        align-content: space-between;
}

.layout.around-aligned.search-header {
  -ms-flex-line-pack: distribute;  
        -ms-align-content: space-around;
        -webkit-align-content: space-around;
        align-content: space-around;
}</style><!-- Shady DOM styles for search-ui --><style scope="search-ui-0">.search-ui-0 paper-button.search-ui {
  display: inline-block;
        text-align: center;
        font-family: inherit;
}

.search-ui-0 .layout.horizontal.search-ui,.search-ui-0 .layout.vertical.search-ui {
  display: -ms-flexbox;
        display: -webkit-flex;
        display: flex;
}

.search-ui-0 .layout.inline.search-ui {
  display: -ms-inline-flexbox;
        display: -webkit-inline-flex;
        display: inline-flex;
}

.search-ui-0 .layout.horizontal.search-ui {
  -ms-flex-direction: row;
        -webkit-flex-direction: row;
        flex-direction: row;
}

.search-ui-0 .layout.vertical.search-ui {
  -ms-flex-direction: column;
        -webkit-flex-direction: column;
        flex-direction: column;
}

.search-ui-0 .layout.wrap.search-ui {
  -ms-flex-wrap: wrap;
        -webkit-flex-wrap: wrap;
        flex-wrap: wrap;
}

.search-ui-0 .layout.no-wrap.search-ui {
  -ms-flex-wrap: nowrap;
        -webkit-flex-wrap: nowrap;
        flex-wrap: nowrap;
}

.search-ui-0 .layout.center.search-ui,.search-ui-0 .layout.center-center.search-ui {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.search-ui-0 .layout.center-justified.search-ui,.search-ui-0 .layout.center-center.search-ui {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.search-ui-0 .flex.search-ui {
  -ms-flex: 1 1 0.000000001px;
        -webkit-flex: 1;
        flex: 1;
        -webkit-flex-basis: 0.000000001px;
        flex-basis: 0.000000001px;
}

.search-ui-0 .flex-auto.search-ui {
  -ms-flex: 1 1 auto;
        -webkit-flex: 1 1 auto;
        flex: 1 1 auto;
}

.search-ui-0 .flex-none.search-ui {
  -ms-flex: none;
        -webkit-flex: none;
        flex: none;
}

.search-ui-0 .layout.start.search-ui {
  -ms-flex-align: start;
        -webkit-align-items: flex-start;
        align-items: flex-start;
}

.search-ui-0 .layout.center.search-ui,.search-ui-0 .layout.center-center.search-ui {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.search-ui-0 .layout.end.search-ui {
  -ms-flex-align: end;
        -webkit-align-items: flex-end;
        align-items: flex-end;
}

.search-ui-0 .layout.baseline.search-ui {
  -ms-flex-align: baseline;
        -webkit-align-items: baseline;
        align-items: baseline;
}

.search-ui-0 .layout.start-justified.search-ui {
  -ms-flex-pack: start;
        -webkit-justify-content: flex-start;
        justify-content: flex-start;
}

.search-ui-0 .layout.center-justified.search-ui,.search-ui-0 .layout.center-center.search-ui {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.search-ui-0 .layout.end-justified.search-ui {
  -ms-flex-pack: end;
        -webkit-justify-content: flex-end;
        justify-content: flex-end;
}

.search-ui-0 .layout.around-justified.search-ui {
  -ms-flex-pack: distribute;
        -webkit-justify-content: space-around;
        justify-content: space-around;
}

.search-ui-0 .layout.justified.search-ui {
  -ms-flex-pack: justify;
        -webkit-justify-content: space-between;
        justify-content: space-between;
}

.search-ui-0 .self-start.search-ui {
  -ms-align-self: flex-start;
        -webkit-align-self: flex-start;
        align-self: flex-start;
}

.search-ui-0 .self-center.search-ui {
  -ms-align-self: center;
        -webkit-align-self: center;
        align-self: center;
}

.search-ui-0 .self-end.search-ui {
  -ms-align-self: flex-end;
        -webkit-align-self: flex-end;
        align-self: flex-end;
}

.search-ui-0 .self-stretch.search-ui {
  -ms-align-self: stretch;
        -webkit-align-self: stretch;
        align-self: stretch;
}

.search-ui-0 .self-baseline.search-ui {
  -ms-align-self: baseline;
        -webkit-align-self: baseline;
        align-self: baseline;
}

.search-ui-0 .layout.start-aligned.search-ui {
  -ms-flex-line-pack: start;  
        -ms-align-content: flex-start;
        -webkit-align-content: flex-start;
        align-content: flex-start;
}

.search-ui-0 .layout.end-aligned.search-ui {
  -ms-flex-line-pack: end;  
        -ms-align-content: flex-end;
        -webkit-align-content: flex-end;
        align-content: flex-end;
}

.search-ui-0 .layout.center-aligned.search-ui {
  -ms-flex-line-pack: center;  
        -ms-align-content: center;
        -webkit-align-content: center;
        align-content: center;
}

.search-ui-0 .layout.between-aligned.search-ui {
  -ms-flex-line-pack: justify;  
        -ms-align-content: space-between;
        -webkit-align-content: space-between;
        align-content: space-between;
}

.search-ui-0 .layout.around-aligned.search-ui {
  -ms-flex-line-pack: distribute;  
        -ms-align-content: space-around;
        -webkit-align-content: space-around;
        align-content: space-around;
}

.search-ui-0 {
  display: block;
        min-width: 350px;
        height: 100%;
}

.search-ui-0 #content.search-ui {
  height: 100%;
}

.search-ui-0 #leftBarOverflow.search-ui {
  padding-right: 16px;
}

.search-ui-0 [wide].search-ui #leftBarOverflow.search-ui {
  overflow: auto;
}

.search-ui-0 #leftBar.search-ui {
  box-sizing: border-box;
        color: #666;
        font-size: 13px;
        padding: 16px 0 16px 16px;
        border-bottom: 1px solid #e5e5e5;
}

.search-ui-0 [wide].search-ui #leftBar.search-ui {
  display: -ms-flexbox;
        display: -webkit-flex;
        display: flex;

        -ms-flex-direction: column;
        -webkit-flex-direction: column;
        flex-direction: column;
        position: relative;
        width: 350px;
        margin-bottom: 0;
        padding-bottom: 0;
        border-bottom: none;
}

@media print {
.search-ui-0 #leftBar.search-ui {
  display: none !important;
}

}

.search-ui-0 #leftBar.search-ui h3.search-ui {
  text-transform: uppercase;
        font-size: 13px;
        font-weight: normal;
        margin-top: 6px;
        margin-bottom: 6px;
        cursor: default;
        display: inline-block;
        color: #a3a3a3;
}

.search-ui-0 #closeButton.search-ui {
  position: absolute;
        right: 0;
}

.search-ui-0 #searchTermsHeader.search-ui {
  position: relative;
}

.search-ui-0 #searchTermsQ.search-ui iron-icon.search-ui {
  height: 16px;
        padding: 0 2px 2px 2px;
        color: #a3a3a3;
}

.search-ui-0 #searchTermsHeader.search-ui:hover iron-icon.search-ui {
  color: #4285f4;
}

.search-ui-0 #closeLeftBar.search-ui {
  height: 16px;
        width: 16px;
        padding: 0 0 8px 0;
        cursor: pointer;
}

.search-ui-0 #compactQuery.search-ui iron-icon.search-ui {
  height: 20px;
}

.search-ui-0 #backToResultsButton.search-ui {
  color: #fff;
        background: #4285f4;
        margin-top: 20px;
        text-align: center;
}

.search-ui-0 #main.search-ui {
  display: block;
        min-height: calc(100vh - 127px);
}

@media print {
.search-ui-0 [wide].search-ui main.search-ui {
  margin-left: 0;
}

}</style><!-- Shady DOM styles for import-list --><!-- Shady DOM styles for iron-request --><!-- Shady DOM styles for iron-ajax --><!-- Shady DOM styles for paper-dialog --><!-- Shady DOM styles for coverage-stats --><!-- Shady DOM styles for iron-image --><!-- Shady DOM styles for iron-selector --><!-- Shady DOM styles for paper-card --><!-- Shady DOM styles for paper-checkbox --><!-- Shady DOM styles for landing-page --><!-- Shady DOM styles for remote-fetch --><!-- Shady DOM styles for paper-spinner --><style scope="paper-spinner-0">.paper-spinner-0 {
  display: inline-block;
        position: relative;
        width: 28px;
        height: 28px;
}

.paper-spinner-0 #spinnerContainer.paper-spinner {
  width: 100%;
        height: 100%;

        
        direction: ltr;
}

.paper-spinner-0 #spinnerContainer.active.paper-spinner {
  -webkit-animation: container-rotate-paper-spinner-0 1568ms linear infinite;
        animation: container-rotate-paper-spinner-0 1568ms linear infinite;
}

@-webkit-keyframes container-rotate-paper-spinner-0 {
to {
  -webkit-transform: rotate(360deg)
}

}

@keyframes container-rotate-paper-spinner-0 {
to {
  transform: rotate(360deg)
}

}

.paper-spinner-0 .spinner-layer.paper-spinner {
  position: absolute;
        width: 100%;
        height: 100%;
        opacity: 0;
        white-space: nowrap;
        color: #4285f4;
}

.paper-spinner-0 .layer-1.paper-spinner {
  color: #4285f4;
}

.paper-spinner-0 .layer-2.paper-spinner {
  color: #db4437;
}

.paper-spinner-0 .layer-3.paper-spinner {
  color: #f4b400;
}

.paper-spinner-0 .layer-4.paper-spinner {
  color: #0f9d58;
}

.paper-spinner-0 .active.paper-spinner .spinner-layer.paper-spinner {
  -webkit-animation-name: fill-unfill-rotate-paper-spinner-0;
        -webkit-animation-duration: 5332ms;
        -webkit-animation-timing-function: cubic-bezier(0.4, 0.0, 0.2, 1);
        -webkit-animation-iteration-count: infinite;
        animation-name: fill-unfill-rotate-paper-spinner-0;
        animation-duration: 5332ms;
        animation-timing-function: cubic-bezier(0.4, 0.0, 0.2, 1);
        animation-iteration-count: infinite;
        opacity: 1;
}

.paper-spinner-0 .active.paper-spinner .spinner-layer.layer-1.paper-spinner {
  -webkit-animation-name: fill-unfill-rotate-paper-spinner-0, layer-1-fade-in-out-paper-spinner-0;
        animation-name: fill-unfill-rotate-paper-spinner-0, layer-1-fade-in-out-paper-spinner-0;
}

.paper-spinner-0 .active.paper-spinner .spinner-layer.layer-2.paper-spinner {
  -webkit-animation-name: fill-unfill-rotate-paper-spinner-0, layer-2-fade-in-out-paper-spinner-0;
        animation-name: fill-unfill-rotate-paper-spinner-0, layer-2-fade-in-out-paper-spinner-0;
}

.paper-spinner-0 .active.paper-spinner .spinner-layer.layer-3.paper-spinner {
  -webkit-animation-name: fill-unfill-rotate-paper-spinner-0, layer-3-fade-in-out-paper-spinner-0;
        animation-name: fill-unfill-rotate-paper-spinner-0, layer-3-fade-in-out-paper-spinner-0;
}

.paper-spinner-0 .active.paper-spinner .spinner-layer.layer-4.paper-spinner {
  -webkit-animation-name: fill-unfill-rotate-paper-spinner-0, layer-4-fade-in-out-paper-spinner-0;
        animation-name: fill-unfill-rotate-paper-spinner-0, layer-4-fade-in-out-paper-spinner-0;
}

@-webkit-keyframes fill-unfill-rotate-paper-spinner-0 {
12.5% {
  -webkit-transform: rotate(135deg)
}

25% {
  -webkit-transform: rotate(270deg)
}

37.5% {
  -webkit-transform: rotate(405deg)
}

50% {
  -webkit-transform: rotate(540deg)
}

62.5% {
  -webkit-transform: rotate(675deg)
}

75% {
  -webkit-transform: rotate(810deg)
}

87.5% {
  -webkit-transform: rotate(945deg)
}

to {
  -webkit-transform: rotate(1080deg)
}

}

@keyframes fill-unfill-rotate-paper-spinner-0 {
12.5% {
  transform: rotate(135deg)
}

25% {
  transform: rotate(270deg)
}

37.5% {
  transform: rotate(405deg)
}

50% {
  transform: rotate(540deg)
}

62.5% {
  transform: rotate(675deg)
}

75% {
  transform: rotate(810deg)
}

87.5% {
  transform: rotate(945deg)
}

to {
  transform: rotate(1080deg)
}

}

@-webkit-keyframes layer-1-fade-in-out-paper-spinner-0 {
0% {
  opacity: 1
}

25% {
  opacity: 1
}

26% {
  opacity: 0
}

89% {
  opacity: 0
}

90% {
  opacity: 1
}

to {
  opacity: 1
}

}

@keyframes layer-1-fade-in-out-paper-spinner-0 {
0% {
  opacity: 1
}

25% {
  opacity: 1
}

26% {
  opacity: 0
}

89% {
  opacity: 0
}

90% {
  opacity: 1
}

to {
  opacity: 1
}

}

@-webkit-keyframes layer-2-fade-in-out-paper-spinner-0 {
0% {
  opacity: 0
}

15% {
  opacity: 0
}

25% {
  opacity: 1
}

50% {
  opacity: 1
}

51% {
  opacity: 0
}

to {
  opacity: 0
}

}

@keyframes layer-2-fade-in-out-paper-spinner-0 {
0% {
  opacity: 0
}

15% {
  opacity: 0
}

25% {
  opacity: 1
}

50% {
  opacity: 1
}

51% {
  opacity: 0
}

to {
  opacity: 0
}

}

@-webkit-keyframes layer-3-fade-in-out-paper-spinner-0 {
0% {
  opacity: 0
}

40% {
  opacity: 0
}

50% {
  opacity: 1
}

75% {
  opacity: 1
}

76% {
  opacity: 0
}

to {
  opacity: 0
}

}

@keyframes layer-3-fade-in-out-paper-spinner-0 {
0% {
  opacity: 0
}

40% {
  opacity: 0
}

50% {
  opacity: 1
}

75% {
  opacity: 1
}

76% {
  opacity: 0
}

to {
  opacity: 0
}

}

@-webkit-keyframes layer-4-fade-in-out-paper-spinner-0 {
0% {
  opacity: 0
}

65% {
  opacity: 0
}

75% {
  opacity: 1
}

90% {
  opacity: 1
}

to {
  opacity: 0
}

}

@keyframes layer-4-fade-in-out-paper-spinner-0 {
0% {
  opacity: 0
}

65% {
  opacity: 0
}

75% {
  opacity: 1
}

90% {
  opacity: 1
}

to {
  opacity: 0
}

}

.paper-spinner-0 .circle-clipper.paper-spinner {
  display: inline-block;
        position: relative;
        width: 50%;
        height: 100%;
        overflow: hidden;
}

.paper-spinner-0 .spinner-layer.paper-spinner::after {
  left: 45%;
        width: 10%;
        border-top-style: solid;
}

.paper-spinner-0 .spinner-layer.paper-spinner::after,.paper-spinner-0 .circle-clipper.paper-spinner::after {
  content: '';
        box-sizing: border-box;
        position: absolute;
        top: 0;
        border-width: 3px;
        border-radius: 50%;
}

.paper-spinner-0 .circle-clipper.paper-spinner::after {
  bottom: 0;
        width: 200%;
        border-style: solid;
        border-bottom-color: transparent !important;
}

.paper-spinner-0 .circle-clipper.left.paper-spinner::after {
  left: 0;
        border-right-color: transparent !important;
        -webkit-transform: rotate(129deg);
        transform: rotate(129deg);
}

.paper-spinner-0 .circle-clipper.right.paper-spinner::after {
  left: -100%;
        border-left-color: transparent !important;
        -webkit-transform: rotate(-129deg);
        transform: rotate(-129deg);
}

.paper-spinner-0 .active.paper-spinner .gap-patch.paper-spinner::after,.paper-spinner-0 .active.paper-spinner .circle-clipper.paper-spinner::after {
  -webkit-animation-duration: 1333ms;
        -webkit-animation-timing-function: cubic-bezier(0.4, 0.0, 0.2, 1);
        -webkit-animation-iteration-count: infinite;
        animation-duration: 1333ms;
        animation-timing-function: cubic-bezier(0.4, 0.0, 0.2, 1);
        animation-iteration-count: infinite;
}

.paper-spinner-0 .active.paper-spinner .circle-clipper.left.paper-spinner::after {
  -webkit-animation-name: left-spin-paper-spinner-0;
        animation-name: left-spin-paper-spinner-0;
}

.paper-spinner-0 .active.paper-spinner .circle-clipper.right.paper-spinner::after {
  -webkit-animation-name: right-spin-paper-spinner-0;
        animation-name: right-spin-paper-spinner-0;
}

@-webkit-keyframes left-spin-paper-spinner-0 {
0% {
  -webkit-transform: rotate(130deg)
}

50% {
  -webkit-transform: rotate(-5deg)
}

to {
  -webkit-transform: rotate(130deg)
}

}

@keyframes left-spin-paper-spinner-0 {
0% {
  transform: rotate(130deg)
}

50% {
  transform: rotate(-5deg)
}

to {
  transform: rotate(130deg)
}

}

@-webkit-keyframes right-spin-paper-spinner-0 {
0% {
  -webkit-transform: rotate(-130deg)
}

50% {
  -webkit-transform: rotate(5deg)
}

to {
  -webkit-transform: rotate(-130deg)
}

}

@keyframes right-spin-paper-spinner-0 {
0% {
  transform: rotate(-130deg)
}

50% {
  transform: rotate(5deg)
}

to {
  transform: rotate(-130deg)
}

}

.paper-spinner-0 #spinnerContainer.cooldown.paper-spinner {
  -webkit-animation: container-rotate-paper-spinner-0 1568ms linear infinite, fade-out-paper-spinner-0 400ms cubic-bezier(0.4, 0.0, 0.2, 1);
        animation: container-rotate-paper-spinner-0 1568ms linear infinite, fade-out-paper-spinner-0 400ms cubic-bezier(0.4, 0.0, 0.2, 1);
}

@-webkit-keyframes fade-out-paper-spinner-0 {
0% {
  opacity: 1
}

to {
  opacity: 0
}

}

@keyframes fade-out-paper-spinner-0 {
0% {
  opacity: 1
}

to {
  opacity: 0
}

}</style><!-- Shady DOM styles for paper-spinner-lite --><!-- Shady DOM styles for concept-card --><!-- Shady DOM styles for concept-mention --><!-- Shady DOM styles for fterm-concept-card --><!-- Shady DOM styles for fterm-concept-mention --><!-- Shady DOM styles for paper-fab --><!-- Shady DOM styles for paper-icon-button --><!-- Shady DOM styles for paper-icon-button-light --><!-- Shady DOM styles for classification-tree --><!-- Shady DOM styles for classification-viewer --><style scope="classification-viewer">.table.classification-viewer {
  border: 1px solid #eaeaea;
        display: block;
}

.more.classification-viewer {
  padding: 6px 12px;
        cursor: pointer;
        font-style: italic;
        color: #aaa;
}

.more.classification-viewer:hover {
  background: #fafafa;
}

.chip.classification-viewer {
  display: inline-block;
        padding: 4px 6px;
        border-radius: 2px;
}

.chip[additional].classification-viewer {
  font-style: italic;
}</style><!-- Shady DOM styles for image-carousel --><!-- Shady DOM styles for image-viewer --><!-- Shady DOM styles for patent-result --><style scope="patent-result">#wrapper.patent-result {
  padding: 20px;
        max-width: 1050px;
        outline: 0;
}

.scroll-target.patent-result:before {
  display: block;
        content: '';
        visibility: hidden;
}

h1.patent-result {
  font-weight: 500;
        font-size: 20px;
        margin: 0 0 20px 0;
}

h3.patent-result {
  font-size: 15px;
        font-weight: 500;
        margin: 28px 0 18px 0;
}

dl.patent-result {
  line-height: 1.5;
        font-size: 0;
}

dt.patent-result,dd.patent-result {
  display: inline;
        font-size: 13px;
        margin: 0;
}

dt.patent-result {
  font-weight: 500;
        color: #000;
}

dt.patent-result::after {
  content: ': ';
}

dd.patent-result {
  color: #aaa;
}

dd.patent-result ~ dd.patent-result::before {
  content: ', ';
}

dt.patent-result + dd.patent-result::before {
  content: '';
}

dd.patent-result ~ dt.patent-result::before {
  display: block;
        content: '';
        margin-top: 8px;
}

dd.patent-result core-icon.patent-result {
  width: 14px;
        height: 14px;
        vertical-align: top;
        color: inherit;
}

.table.patent-result {
  display: table;
        border-collapse: collapse;
        border-spacing: 0;
        border: 1px solid #eaeaea;
        width: 100%;
        box-sizing: border-box;
}

.tr.patent-result {
  display: table-row;
        border-bottom: 1px solid #eaeaea;
}

.th.patent-result,.td.patent-result {
  display: table-cell;
        padding: 10px 12px;
}

.thead.patent-result {
  display: table-header-group;
}

.tbody.patent-result {
  display: table-row-group;
}

.th.patent-result {
  text-align: left;
        white-space: nowrap;
        background-color: #fafafa;
        color: #757575;
}

.td.patent-result {
  vertical-align: top;
}

.td.patent-result > p.patent-result {
  margin-top: 0;
}

.nowrap.patent-result {
  white-space: nowrap;
}

.knowledge-card.patent-result {
  border: 1px solid #e5e5e5;
        border-radius: 2px;
}

.knowledge-card.patent-result > header.patent-result {
  -webkit-font-smoothing: antialiased;
        -moz-osx-font-smoothing: grayscale;
        background: #4285f4;
        color: #fff;
        padding: 20px 12px;
}

.knowledge-card.patent-result > header.patent-result state-modifier.patent-result,.knowledge-card.patent-result > header.patent-result a.patent-result {
  color: inherit;
        margin: 0 12px 0 0;
        display: inline-block;
}

.knowledge-card.patent-result > header.patent-result h2.patent-result,.knowledge-card.patent-result > header.patent-result .tagline.patent-result {
  margin: 0;
        font-weight: normal;
        padding-top: 4px;
}

.knowledge-card.patent-result > header.patent-result h2.patent-result {
  font-size: 20px;
}

.knowledge-card.patent-result .knowledge-card-action-bar.patent-result {
  background: #3367d6;
        padding: 10px 12px;
        margin: 20px -12px -20px;
}

.knowledge-card.patent-result .knowledge-card-action-bar.patent-result core-icon.patent-result {
  margin-right: 12px;
        width: 24px;
        height: 24px;
}

.knowledge-card.patent-result dl.patent-result {
  padding: 16px 12px;
}

.knowledge-card.patent-result hr.patent-result {
  border: none;
        height: 1px;
        background: #e5e5e5;
        margin: 0;
}

.main-link.patent-result {
  margin: 14px;
        font-weight: 500;
}

.footer.patent-result h3.patent-result state-modifier.patent-result,.footer.patent-result h3.patent-result a.patent-result {
  float: right;
        font-weight: 400;
        font-size: 13px;
        line-height: 24px;
}

.footer.patent-result dl.patent-result {
  margin-left: 30px;
}

.footer.patent-result > ul.patent-result > li.patent-result {
  margin-bottom: 8px;
}

h3.patent-result .righthead.patent-result {
  color: #aaa;
        font-weight: 400;
        font-size: 13px;
        cursor: help;
}

.tooltip-hint.patent-result {
  border-bottom: 1px dotted #ccc;
}

[marginRight].patent-result {
  margin-right: 60px;
}

.flex-width.patent-result {
  min-width: 0;
        word-break: break-word;
}

.responsive-table.patent-result {
  width: 100%;
        overflow-x: auto;
        overflow-y: hidden;
}

.layout.horizontal.patent-result,.layout.vertical.patent-result {
  display: -ms-flexbox;
        display: -webkit-flex;
        display: flex;
}

.layout.inline.patent-result {
  display: -ms-inline-flexbox;
        display: -webkit-inline-flex;
        display: inline-flex;
}

.layout.horizontal.patent-result {
  -ms-flex-direction: row;
        -webkit-flex-direction: row;
        flex-direction: row;
}

.layout.vertical.patent-result {
  -ms-flex-direction: column;
        -webkit-flex-direction: column;
        flex-direction: column;
}

.layout.wrap.patent-result {
  -ms-flex-wrap: wrap;
        -webkit-flex-wrap: wrap;
        flex-wrap: wrap;
}

.layout.no-wrap.patent-result {
  -ms-flex-wrap: nowrap;
        -webkit-flex-wrap: nowrap;
        flex-wrap: nowrap;
}

.layout.center.patent-result,.layout.center-center.patent-result {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.layout.center-justified.patent-result,.layout.center-center.patent-result {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.flex.patent-result {
  -ms-flex: 1 1 0.000000001px;
        -webkit-flex: 1;
        flex: 1;
        -webkit-flex-basis: 0.000000001px;
        flex-basis: 0.000000001px;
}

.flex-auto.patent-result {
  -ms-flex: 1 1 auto;
        -webkit-flex: 1 1 auto;
        flex: 1 1 auto;
}

.flex-none.patent-result {
  -ms-flex: none;
        -webkit-flex: none;
        flex: none;
}

.layout.start.patent-result {
  -ms-flex-align: start;
        -webkit-align-items: flex-start;
        align-items: flex-start;
}

.layout.center.patent-result,.layout.center-center.patent-result {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.layout.end.patent-result {
  -ms-flex-align: end;
        -webkit-align-items: flex-end;
        align-items: flex-end;
}

.layout.baseline.patent-result {
  -ms-flex-align: baseline;
        -webkit-align-items: baseline;
        align-items: baseline;
}

.layout.start-justified.patent-result {
  -ms-flex-pack: start;
        -webkit-justify-content: flex-start;
        justify-content: flex-start;
}

.layout.center-justified.patent-result,.layout.center-center.patent-result {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.layout.end-justified.patent-result {
  -ms-flex-pack: end;
        -webkit-justify-content: flex-end;
        justify-content: flex-end;
}

.layout.around-justified.patent-result {
  -ms-flex-pack: distribute;
        -webkit-justify-content: space-around;
        justify-content: space-around;
}

.layout.justified.patent-result {
  -ms-flex-pack: justify;
        -webkit-justify-content: space-between;
        justify-content: space-between;
}

.self-start.patent-result {
  -ms-align-self: flex-start;
        -webkit-align-self: flex-start;
        align-self: flex-start;
}

.self-center.patent-result {
  -ms-align-self: center;
        -webkit-align-self: center;
        align-self: center;
}

.self-end.patent-result {
  -ms-align-self: flex-end;
        -webkit-align-self: flex-end;
        align-self: flex-end;
}

.self-stretch.patent-result {
  -ms-align-self: stretch;
        -webkit-align-self: stretch;
        align-self: stretch;
}

.self-baseline.patent-result {
  -ms-align-self: baseline;
        -webkit-align-self: baseline;
        align-self: baseline;
}

.layout.start-aligned.patent-result {
  -ms-flex-line-pack: start;  
        -ms-align-content: flex-start;
        -webkit-align-content: flex-start;
        align-content: flex-start;
}

.layout.end-aligned.patent-result {
  -ms-flex-line-pack: end;  
        -ms-align-content: flex-end;
        -webkit-align-content: flex-end;
        align-content: flex-end;
}

.layout.center-aligned.patent-result {
  -ms-flex-line-pack: center;  
        -ms-align-content: center;
        -webkit-align-content: center;
        align-content: center;
}

.layout.between-aligned.patent-result {
  -ms-flex-line-pack: justify;  
        -ms-align-content: space-between;
        -webkit-align-content: space-between;
        align-content: space-between;
}

.layout.around-aligned.patent-result {
  -ms-flex-line-pack: distribute;  
        -ms-align-content: space-around;
        -webkit-align-content: space-around;
        align-content: space-around;
}

.flex.patent-result,.flex-1.patent-result {
  -ms-flex: 1 1 0.000000001px;
        -webkit-flex: 1;
        flex: 1;
        -webkit-flex-basis: 0.000000001px;
        flex-basis: 0.000000001px;
}

.flex-2.patent-result {
  -ms-flex: 2;
        -webkit-flex: 2;
        flex: 2;
}

.flex-3.patent-result {
  -ms-flex: 3;
        -webkit-flex: 3;
        flex: 3;
}

.flex-4.patent-result {
  -ms-flex: 4;
        -webkit-flex: 4;
        flex: 4;
}

.flex-5.patent-result {
  -ms-flex: 5;
        -webkit-flex: 5;
        flex: 5;
}

.flex-6.patent-result {
  -ms-flex: 6;
        -webkit-flex: 6;
        flex: 6;
}

.flex-7.patent-result {
  -ms-flex: 7;
        -webkit-flex: 7;
        flex: 7;
}

.flex-8.patent-result {
  -ms-flex: 8;
        -webkit-flex: 8;
        flex: 8;
}

.flex-9.patent-result {
  -ms-flex: 9;
        -webkit-flex: 9;
        flex: 9;
}

.flex-10.patent-result {
  -ms-flex: 10;
        -webkit-flex: 10;
        flex: 10;
}

.flex-11.patent-result {
  -ms-flex: 11;
        -webkit-flex: 11;
        flex: 11;
}

.flex-12.patent-result {
  -ms-flex: 12;
        -webkit-flex: 12;
        flex: 12;
}

@media print {
#text.patent-result {
  flex-direction: column-reverse !important;
}

}

patent-result-ref-list.patent-result {
  width: 100%;
}

patent-result-ref-list.patent-result + patent-result-ref-list.patent-result {
  margin-left: 10px;
}

[narrow].patent-result {
  flex-direction: column-reverse !important;
}

.famheader.patent-result {
  text-align: left;
        white-space: nowrap;
        background-color: #fafafa;
        color: #757575;
}

.collapse.patent-result {
  color: #616161;
        font-size: 13px;
        cursor: pointer;
}

#imageDrag.patent-result {
  position: fixed;
        cursor: col-resize;
}

#imageDrag.patent-result div.patent-result {
  width: 4px;
        height: 100%;
        margin: 0 4px;
        background: #eee;
}

#imageDrag.patent-result:hover {
  background: #eee;
}

#imageDrag.patent-result:hover div.patent-result {
  background: #ccc;
}

.imageWrapper.patent-result {
  position: fixed;
        top: 60px;
        background: white;
        margin-left: 12px;
}

#notices.patent-result {
  padding: 20px;
}

#figures.patent-result {
  background: #efefef;
}

.headerButton.patent-result {
  color: #3c80f6;
        cursor: pointer;
        display: inline-block;
        padding: 0 14px;
        line-height: 27px;
}

.headerButton.patent-result iron-icon.patent-result {
  width: 16px;
        height: 16px;
}

.tr.conceptDomain.patent-result {
  display: table-row;
}

.responsive-grid.patent-result {
  display: grid;
        gap: 8px;
        grid-template-columns: none;
}</style><!-- Shady DOM styles for family-viewer --><!-- Shady DOM styles for app-row --><!-- Shady DOM styles for application-timeline --><style scope="application-timeline">.layout.horizontal.application-timeline,.layout.vertical.application-timeline {
  display: -ms-flexbox;
        display: -webkit-flex;
        display: flex;
}

.layout.inline.application-timeline {
  display: -ms-inline-flexbox;
        display: -webkit-inline-flex;
        display: inline-flex;
}

.layout.horizontal.application-timeline {
  -ms-flex-direction: row;
        -webkit-flex-direction: row;
        flex-direction: row;
}

.layout.vertical.application-timeline {
  -ms-flex-direction: column;
        -webkit-flex-direction: column;
        flex-direction: column;
}

.layout.wrap.application-timeline {
  -ms-flex-wrap: wrap;
        -webkit-flex-wrap: wrap;
        flex-wrap: wrap;
}

.layout.no-wrap.application-timeline {
  -ms-flex-wrap: nowrap;
        -webkit-flex-wrap: nowrap;
        flex-wrap: nowrap;
}

.layout.center.application-timeline,.layout.center-center.application-timeline {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.layout.center-justified.application-timeline,.layout.center-center.application-timeline {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.flex.application-timeline {
  -ms-flex: 1 1 0.000000001px;
        -webkit-flex: 1;
        flex: 1;
        -webkit-flex-basis: 0.000000001px;
        flex-basis: 0.000000001px;
}

.flex-auto.application-timeline {
  -ms-flex: 1 1 auto;
        -webkit-flex: 1 1 auto;
        flex: 1 1 auto;
}

.flex-none.application-timeline {
  -ms-flex: none;
        -webkit-flex: none;
        flex: none;
}

.wrap.application-timeline {
  padding: 16px 12px;
}

.event.application-timeline {
  padding: 3px 0;
}

[date].application-timeline {
  color: #5f6368;
        font-size: 13px;
        padding: 3px 0;
        width: 70px;
        line-height: 16px;
}

[critical].application-timeline [date].application-timeline {
  color: #202124;
        font-weight: 500;
        line-height: 18px;
}

.line.application-timeline {
  margin: 9px 6px;
        border-radius: 4px;
        background: #dadce0;
        width: 4px;
        height: 4px;
}

.dot.application-timeline {
  margin: 3px 0;
        border-radius: 4px;
        background: #dadce0;
        width: 4px;
        height: 4px;
        display: inline-block;
}

[critical].application-timeline .line.application-timeline {
  background: #4285f4;
}

[current].application-timeline .line.application-timeline {
  background: #81c995;
}

[current][critical].application-timeline .line.application-timeline {
  background: #81c995;
}

.title.application-timeline {
  font-size: 13px;
        padding: 3px 0;
        line-height: 16px;
}

.title-text.application-timeline {
  color: #5f6368;
}

.description-item.application-timeline {
  font-size: 12px;
        padding: 6px 0;
        max-width: 300px;
}

#legal.application-timeline {
  height: 14px;
        width: 14px;
        padding: 0 3px 1px 3px;
        vertical-align: text-bottom;
}

#icon.application-timeline {
  height: 12px;
        width: 12px;
        padding: 0 3px 3px 3px;
        vertical-align: text-bottom;
}

.unknown.application-timeline {
  font-style: italic;
}

.not_active.application-timeline {
  text-decoration: line-through;
}

.active.application-timeline {
  font-weight: 500;
}

#cc[this].application-timeline {
  color: #202124 !important;
        text-decoration: underline;
}

#cc[this].not_active.application-timeline {
  text-decoration: line-through underline;
}

hr.application-timeline {
  border: none;
        height: 1px;
        background: #e5e5e5;
        margin: 10px 0 16px 0;
}

.header.application-timeline {
  font-size: 13px;
        line-height: 20px;
        font-weight: 500;
        color: #202124;
        padding-bottom: 8px;
}

.block.application-timeline {
  padding: 0 6px 0 0;
}

.year.application-timeline {
  color: #5f6368;
        font-size: 13px;
        font-weight: 500;
        padding: 3px 1px;
        width: 70px;
}

.expand.application-timeline {
  cursor: pointer;
        padding: 6px 0 3px 0;
}</style><!-- Shady DOM styles for patent-text --><style scope="patent-text">.layout.horizontal.patent-text,.layout.vertical.patent-text {
  display: -ms-flexbox;
        display: -webkit-flex;
        display: flex;
}

.layout.inline.patent-text {
  display: -ms-inline-flexbox;
        display: -webkit-inline-flex;
        display: inline-flex;
}

.layout.horizontal.patent-text {
  -ms-flex-direction: row;
        -webkit-flex-direction: row;
        flex-direction: row;
}

.layout.vertical.patent-text {
  -ms-flex-direction: column;
        -webkit-flex-direction: column;
        flex-direction: column;
}

.layout.wrap.patent-text {
  -ms-flex-wrap: wrap;
        -webkit-flex-wrap: wrap;
        flex-wrap: wrap;
}

.layout.no-wrap.patent-text {
  -ms-flex-wrap: nowrap;
        -webkit-flex-wrap: nowrap;
        flex-wrap: nowrap;
}

.layout.center.patent-text,.layout.center-center.patent-text {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.layout.center-justified.patent-text,.layout.center-center.patent-text {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.flex.patent-text {
  -ms-flex: 1 1 0.000000001px;
        -webkit-flex: 1;
        flex: 1;
        -webkit-flex-basis: 0.000000001px;
        flex-basis: 0.000000001px;
}

.flex-auto.patent-text {
  -ms-flex: 1 1 auto;
        -webkit-flex: 1 1 auto;
        flex: 1 1 auto;
}

.flex-none.patent-text {
  -ms-flex: none;
        -webkit-flex: none;
        flex: none;
}

.layout.start.patent-text {
  -ms-flex-align: start;
        -webkit-align-items: flex-start;
        align-items: flex-start;
}

.layout.center.patent-text,.layout.center-center.patent-text {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.layout.end.patent-text {
  -ms-flex-align: end;
        -webkit-align-items: flex-end;
        align-items: flex-end;
}

.layout.baseline.patent-text {
  -ms-flex-align: baseline;
        -webkit-align-items: baseline;
        align-items: baseline;
}

.layout.start-justified.patent-text {
  -ms-flex-pack: start;
        -webkit-justify-content: flex-start;
        justify-content: flex-start;
}

.layout.center-justified.patent-text,.layout.center-center.patent-text {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.layout.end-justified.patent-text {
  -ms-flex-pack: end;
        -webkit-justify-content: flex-end;
        justify-content: flex-end;
}

.layout.around-justified.patent-text {
  -ms-flex-pack: distribute;
        -webkit-justify-content: space-around;
        justify-content: space-around;
}

.layout.justified.patent-text {
  -ms-flex-pack: justify;
        -webkit-justify-content: space-between;
        justify-content: space-between;
}

.self-start.patent-text {
  -ms-align-self: flex-start;
        -webkit-align-self: flex-start;
        align-self: flex-start;
}

.self-center.patent-text {
  -ms-align-self: center;
        -webkit-align-self: center;
        align-self: center;
}

.self-end.patent-text {
  -ms-align-self: flex-end;
        -webkit-align-self: flex-end;
        align-self: flex-end;
}

.self-stretch.patent-text {
  -ms-align-self: stretch;
        -webkit-align-self: stretch;
        align-self: stretch;
}

.self-baseline.patent-text {
  -ms-align-self: baseline;
        -webkit-align-self: baseline;
        align-self: baseline;
}

.layout.start-aligned.patent-text {
  -ms-flex-line-pack: start;  
        -ms-align-content: flex-start;
        -webkit-align-content: flex-start;
        align-content: flex-start;
}

.layout.end-aligned.patent-text {
  -ms-flex-line-pack: end;  
        -ms-align-content: flex-end;
        -webkit-align-content: flex-end;
        align-content: flex-end;
}

.layout.center-aligned.patent-text {
  -ms-flex-line-pack: center;  
        -ms-align-content: center;
        -webkit-align-content: center;
        align-content: center;
}

.layout.between-aligned.patent-text {
  -ms-flex-line-pack: justify;  
        -ms-align-content: space-between;
        -webkit-align-content: space-between;
        align-content: space-between;
}

.layout.around-aligned.patent-text {
  -ms-flex-line-pack: distribute;  
        -ms-align-content: space-around;
        -webkit-align-content: space-around;
        align-content: space-around;
}

patent-text {
  display: block;
}

#text.patent-text {
  line-height: 20px;
        min-width: 0;
}

#margin.patent-text {
  width: 16px;
}

#createHighlight.patent-text {
  position: relative;
        width: 16px;
        height: 16px;
        cursor: pointer;
}

.notranslate.patent-text > .google-src-text.patent-text {
  display: none;
}

.notranslate.patent-text:hover,mark.patent-text {
  background-color: #fbfcae;
}

.description-paragraph.patent-text {
  margin-block-start: 1em;
        margin-block-end: 1em;
}

.claim-text.patent-text {
  margin: 14px 0 0 20px;
}

.claim-dependent.patent-text {
  margin-left: 40px;
        color: #888;
}

[hide-claims].patent-text .claim-dependent.patent-text .claim-text.patent-text {
  height: 0;
        width: 0;
        visibility: hidden;
}

[hide-claims].patent-text .claim-dependent.patent-text .claim.patent-text:after {
  content: "…"
}

.patent-full-image.patent-text {
  max-width: 100%;
        height: auto;
}

.small-patent-image.patent-text {
  display: inline;
}

ul.patent-text {
  padding: 0 0 0 50px;
}

li.patent-text {
  list-style-type: none;
        position: relative;
}

ol.claims.patent-text li.patent-text {
  list-style-type: decimal;
        font-weight: bold;
}

ol.claims.patent-text li.patent-text > *.patent-text {
  font-weight: normal;
}

li.patent-text > para-num.patent-text {
  position: absolute;
        left: -45px;
        color: #888;
}

mark.patent-text {
  background: #fff59d;
}

mark.searchText.patent-text {
  background: #80deea;
}

.searchTextHeader.patent-text {
  font-weight: 500;
}

pre.patent-text {
  white-space: pre-wrap;
}

heading.patent-text {
  display: block;
        font-weight: 500;
        font-size: 13px;
}

#comment.patent-text {
  width: 16px;
        height: 16px;
}

[has-search][no-results].patent-text {
  display: none;
}

.highlight.patent-text {
  background-color: #fef7e0;
}</style><!-- Shady DOM styles for para-num --><!-- Shady DOM styles for result-container --><style scope="result-container">paper-spinner.result-container {
  position: absolute;
        
        top: calc(50%  + 56px - 16px);
        left: calc(50% + 175px - 16px);
}</style><!-- Shady DOM styles for scholar-result --><!-- Shady DOM styles for search-result --><style scope="search-result">.layout.horizontal.search-result,.layout.vertical.search-result {
  display: -ms-flexbox;
        display: -webkit-flex;
        display: flex;
}

.layout.inline.search-result {
  display: -ms-inline-flexbox;
        display: -webkit-inline-flex;
        display: inline-flex;
}

.layout.horizontal.search-result {
  -ms-flex-direction: row;
        -webkit-flex-direction: row;
        flex-direction: row;
}

.layout.vertical.search-result {
  -ms-flex-direction: column;
        -webkit-flex-direction: column;
        flex-direction: column;
}

.layout.wrap.search-result {
  -ms-flex-wrap: wrap;
        -webkit-flex-wrap: wrap;
        flex-wrap: wrap;
}

.layout.no-wrap.search-result {
  -ms-flex-wrap: nowrap;
        -webkit-flex-wrap: nowrap;
        flex-wrap: nowrap;
}

.layout.center.search-result,.layout.center-center.search-result {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.layout.center-justified.search-result,.layout.center-center.search-result {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.flex.search-result {
  -ms-flex: 1 1 0.000000001px;
        -webkit-flex: 1;
        flex: 1;
        -webkit-flex-basis: 0.000000001px;
        flex-basis: 0.000000001px;
}

.flex-auto.search-result {
  -ms-flex: 1 1 auto;
        -webkit-flex: 1 1 auto;
        flex: 1 1 auto;
}

.flex-none.search-result {
  -ms-flex: none;
        -webkit-flex: none;
        flex: none;
}

.layout.start.search-result {
  -ms-flex-align: start;
        -webkit-align-items: flex-start;
        align-items: flex-start;
}

.layout.center.search-result,.layout.center-center.search-result {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.layout.end.search-result {
  -ms-flex-align: end;
        -webkit-align-items: flex-end;
        align-items: flex-end;
}

.layout.baseline.search-result {
  -ms-flex-align: baseline;
        -webkit-align-items: baseline;
        align-items: baseline;
}

.layout.start-justified.search-result {
  -ms-flex-pack: start;
        -webkit-justify-content: flex-start;
        justify-content: flex-start;
}

.layout.center-justified.search-result,.layout.center-center.search-result {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.layout.end-justified.search-result {
  -ms-flex-pack: end;
        -webkit-justify-content: flex-end;
        justify-content: flex-end;
}

.layout.around-justified.search-result {
  -ms-flex-pack: distribute;
        -webkit-justify-content: space-around;
        justify-content: space-around;
}

.layout.justified.search-result {
  -ms-flex-pack: justify;
        -webkit-justify-content: space-between;
        justify-content: space-between;
}

.self-start.search-result {
  -ms-align-self: flex-start;
        -webkit-align-self: flex-start;
        align-self: flex-start;
}

.self-center.search-result {
  -ms-align-self: center;
        -webkit-align-self: center;
        align-self: center;
}

.self-end.search-result {
  -ms-align-self: flex-end;
        -webkit-align-self: flex-end;
        align-self: flex-end;
}

.self-stretch.search-result {
  -ms-align-self: stretch;
        -webkit-align-self: stretch;
        align-self: stretch;
}

.self-baseline.search-result {
  -ms-align-self: baseline;
        -webkit-align-self: baseline;
        align-self: baseline;
}

.layout.start-aligned.search-result {
  -ms-flex-line-pack: start;  
        -ms-align-content: flex-start;
        -webkit-align-content: flex-start;
        align-content: flex-start;
}

.layout.end-aligned.search-result {
  -ms-flex-line-pack: end;  
        -ms-align-content: flex-end;
        -webkit-align-content: flex-end;
        align-content: flex-end;
}

.layout.center-aligned.search-result {
  -ms-flex-line-pack: center;  
        -ms-align-content: center;
        -webkit-align-content: center;
        align-content: center;
}

.layout.between-aligned.search-result {
  -ms-flex-line-pack: justify;  
        -ms-align-content: space-between;
        -webkit-align-content: space-between;
        align-content: space-between;
}

.layout.around-aligned.search-result {
  -ms-flex-line-pack: distribute;  
        -ms-align-content: space-around;
        -webkit-align-content: space-around;
        align-content: space-around;
}

search-result {
  display: block;
        font-size: 13px;
}

#mainContainer.search-result {
  min-height: calc(100vh - 127px);
        color: #333;
}

#compactQuery.search-result {
  padding: 18px 20px 10px 20px;
        cursor: pointer;
}

#compactQuery.search-result > span.search-result {
  padding: 0 14px 0 0;
}

#compactQuery.search-result iron-icon.search-result {
  padding-right: 6px;
        height: 16px;
        width: 16px;
}</style><!-- Shady DOM styles for webdoc-result --><!-- Shady DOM styles for iron-pages --><!-- Shady DOM styles for paper-tab --><!-- Shady DOM styles for paper-tabs --><!-- Shady DOM styles for debug-info --><!-- Shady DOM styles for search-paging --><!-- Shady DOM styles for search-result-item --><!-- Shady DOM styles for search-result-image --><!-- Shady DOM styles for search-result-entities --><!-- Shady DOM styles for search-result-measures --><!-- Shady DOM styles for search-results --><!-- Shady DOM styles for summary-box --><!-- Shady DOM styles for timeline-page --><!-- Shady DOM styles for iron-localstorage --><!-- Shady DOM styles for search-app --><style scope="search-app">search-app {
  display: block;
        height: 100%;
}

#toast.search-app {
  min-height: 64px;
        min-width: 742px;
}</style><style>body {transition: opacity ease-in 0.2s; } 
body[unresolved] {opacity: 0; display: block; overflow: hidden; position: relative; } 
</style>
    <title>AU2013204668A1 - Chimeric antigens for breaking host tolerance to foreign antigens 
      - Google Patents</title>

    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta charset="UTF-8">
    <meta name="referrer" content="origin-when-crossorigin">
    <link rel="canonical" href="https://patents.google.com/patent/AU2013204668A1/en">
    <meta name="description" content="
     Disclosed herein are compositions and methods for eliciting immune responses against antigens. In particular, the compounds and methods elicit immune responses against foreign antigens that are otherwise recognized by the host as &quot;self' antigens, thus breaking host tolerance to those antigens. Presenting the host immune system with a chimeric antigen comprising an immune response domain and a target binding domain, wherein the target binding domain comprises an antibody fragment, enhances the immune response against the foreign or tolerated antigen. Antigen presenting cells take up, process, and present the chimeric antigen, eliciting both a humoral and cellular immune response against the desired antigen. 
   
   ">
    <meta name="DC.type" content="patent">
    <meta name="DC.title" content="Chimeric antigens for breaking host tolerance to foreign antigens 
     ">
    <meta name="DC.date" content="2013-04-12" scheme="dateSubmitted">
    <meta name="DC.description" content="
     Disclosed herein are compositions and methods for eliciting immune responses against antigens. In particular, the compounds and methods elicit immune responses against foreign antigens that are otherwise recognized by the host as &quot;self' antigens, thus breaking host tolerance to those antigens. Presenting the host immune system with a chimeric antigen comprising an immune response domain and a target binding domain, wherein the target binding domain comprises an antibody fragment, enhances the immune response against the foreign or tolerated antigen. Antigen presenting cells take up, process, and present the chimeric antigen, eliciting both a humoral and cellular immune response against the desired antigen. 
   
   ">
    <meta name="citation_patent_application_number" content="AU:2013204668A">
    <meta name="citation_pdf_url" content="https://patentimages.storage.googleapis.com/30/37/1b/05561c7a60a77e/AU2013204668A1.pdf">
    <meta name="citation_patent_publication_number" content="AU:2013204668:A1">
    <meta name="DC.date" content="2013-05-09">
    <meta name="DC.contributor" content="Rajan George" scheme="inventor">
    <meta name="DC.contributor" content="Allan Ma" scheme="inventor">
    <meta name="DC.contributor" content="Antoine Noujaim" scheme="inventor">
    <meta name="DC.contributor" content="Lorne Tyrrell" scheme="inventor">
    <meta name="DC.contributor" content="Dakun Wang" scheme="inventor">
    <meta name="DC.contributor" content="Akshaya Bio Inc" scheme="assignee">
    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:400,400italic,500,500italic,700">
    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Product+Sans">
    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Google+Sans:wght@400;500;700">

    <style>
      body { transition: none; }
    </style>

    <script src="https://apis.google.com/_/scs/abc-static/_/js/k=gapi.lb.en.IKZeRvoAYNY.O/m=client/rt=j/sv=1/d=1/ed=1/am=AAAQ/rs=AHpOoo-SMWTzMRJrTty6iE5dL_aWGYOnuw/cb=gapi.loaded_0?le=scs" async=""></script><script>
      window.version = 'patent-search.search_20240531_RC00';

      function sendFeedback() {
        userfeedback.api.startFeedback({
          'productId': '713680',
          'bucket': 'patent-search-web',
          'productVersion': window.version,
        });
      }

      window.experiments = {};
      window.experiments.patentCountries = "ae,ag,al,am,ao,ap,ar,at,au,aw,az,ba,bb,bd,be,bf,bg,bh,bj,bn,bo,br,bw,bx,by,bz,ca,cf,cg,ch,ci,cl,cm,cn,co,cr,cs,cu,cy,cz,dd,de,dj,dk,dm,do,dz,ea,ec,ee,eg,em,ep,es,fi,fr,ga,gb,gc,gd,ge,gh,gm,gn,gq,gr,gt,gw,hk,hn,hr,hu,ib,id,ie,il,in,ir,is,it,jo,jp,ke,kg,kh,km,kn,kp,kr,kw,kz,la,lc,li,lk,lr,ls,lt,lu,lv,ly,ma,mc,md,me,mg,mk,ml,mn,mo,mr,mt,mw,mx,my,mz,na,ne,ng,ni,nl,no,nz,oa,om,pa,pe,pg,ph,pl,pt,py,qa,ro,rs,ru,rw,sa,sc,sd,se,sg,si,sk,sl,sm,sn,st,su,sv,sy,sz,td,tg,th,tj,tm,tn,tr,tt,tw,tz,ua,ug,us,uy,uz,vc,ve,vn,wo,yu,za,zm,zw";
      
      
      window.experiments.keywordWizard = true;
      
      
      
      window.experiments.definitions = true;

      window.Polymer = {
        dom: 'shady',
        lazyRegister: true,
      };
    </script>

    <script src="//www.gstatic.com/patent-search/frontend/patent-search.search_20240531_RC00/scs/compiled_dir/webcomponentsjs/webcomponents-lite.min.js"></script>
    <style>[hidden]:not([style-scope]):not(.style-scope) {
  display: none !important;
}

</style><style></style><style></style><style></style><style></style><link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:400,300,300italic,400italic,500,500italic,700,700italic"><link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto+Mono:400,700"><style></style><style></style><style>
  html, body {
    min-width: 100%;
    width: 100%;
    min-height: 100%;
    height: 100%;
  }

  body {
    background-color: #fff;
    font-family: 'Roboto', sans-serif;
    color: #333;
    margin: 0;
    padding: 0;
    /*
     * Turn off https://github.com/WICG/ScrollAnchoring/blob/master/explainer.md
     * https://memegen.googleplex.com/5230756293246976
     */
    overflow-anchor: none;
  }

  body h1, body h2, body h3 {
    -webkit-font-smoothing: antialiased;
    -moz-osx-font-smoothing: grayscale;
  }

  body a:link {
    color: #3c80f6;
    text-decoration: none;
  }

  body a:visited {
    color: #3c80f6;
  }

  body a[open-result]:visited {
    color: #7e57c2;
  }

  /* For Firefox */
  a:link {
    color: #3c80f6;
    text-decoration: none;
  }

  a:visited {
    color: #3c80f6;
  }

  a[open-result]:visited {
    color: #7e57c2;
  }

  ::selection , body ::selection {
    background-color: #c6dafc;
    color: #333;
  }
</style><link rel="import" href="//www.gstatic.com/patent-search/frontend/patent-search.search_20240531_RC00/scs/compiled_dir/search-app-vulcanized.html">
  <style>[hidden] { display: none !important; }</style><script type="text/javascript" charset="UTF-8" src="https://apis.google.com/js/client.js" gapi_processed="true"></script></head>
  <body>
    <script src="//www.gstatic.com/patent-search/frontend/patent-search.search_20240531_RC00/scs/compiled_dir/search-app-vulcanized.js"></script><span style="font-size: 13px; white-space: pre-wrap; display: inline-block; position: absolute; left: -99999px; top: -99999px; max-width: 217px;"></span><span style="font-size: 13px; white-space: pre-wrap; display: inline-block; position: absolute; left: -99999px; top: -99999px; max-width: 217px;"></span>
    <search-app>
    
    <remote-fetch class="style-scope search-app"></remote-fetch>
    <search-state class="style-scope search-app"></search-state>
    <browser-ui class="style-scope search-app"></browser-ui>
    <template is="dom-if" restamp="" class="style-scope search-app"></template>
    <template is="dom-if" restamp="" class="style-scope search-app"></template>
    
      <search-result id="result" class="style-scope search-app">
    
    
    <search-ui mode="result" class="style-scope search-result x-scope search-ui-0">
    
    
    
    <div id="content" class="style-scope search-ui" wide="">
      <search-header show-search="" class="style-scope search-ui">
    
    
    <div class="style-scope search-header" wide="">
      <header class="style-scope search-header">
        <div class="layout horizontal headerpad style-scope search-header">
          <div class="layout horizontal leftheader style-scope search-header">
            <h1 class="style-scope search-header">
              <state-modifier inherit="" act="{&quot;type&quot;: &quot;HOME&quot;}" class="style-scope search-header"><a id="link" href="#" class="style-scope state-modifier" inherit="">
                <div class="lockup style-scope search-header">
                  <span class="lockup-logo style-scope search-header"></span>
                  <span class="lockup-text style-scope search-header">
                    <span class="lockup-brand style-scope search-header">&nbsp;Patents</span>
                  </span>
                </div>
              </a></state-modifier>
            </h1>
          </div>
          
            <search-box class="style-scope search-header">
    
    <div class="layout horizontal style-scope search-box" style="position: relative;">
      <div class="inputbox flex layout horizontal style-scope search-box">
        <div class="suggest-bg flex style-scope search-box">
          <input name="q" id="searchInput" data-proto="SEARCH_INPUT" autocomplete="off" class="style-scope search-box">
        </div>
        <div id="corpora" class="style-scope search-box">
          <iron-icon icon="icons:lightbulb-outline" id="patentIcon" class="style-scope search-box x-scope iron-icon-0" active=""><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M9 21c0 .55.45 1 1 1h4c.55 0 1-.45 1-1v-1H9v1zm3-19C8.14 2 5 5.14 5 9c0 2.38 1.19 4.47 3 5.74V17c0 .55.45 1 1 1h6c.55 0 1-.45 1-1v-2.26c1.81-1.27 3-3.36 3-5.74 0-3.86-3.14-7-7-7zm2.85 11.1l-.85.6V16h-4v-2.3l-.85-.6C7.8 12.16 7 10.63 7 9c0-2.76 2.24-5 5-5s5 2.24 5 5c0 1.63-.8 3.16-2.15 4.1z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
          <overlay-tooltip positioning-target="#patentIcon" direction="bottom" class="style-scope search-box" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      Include patents
    </div>
  </overlay-tooltip>
          <iron-icon icon="social:school" id="nplIcon" class="style-scope search-box x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M5 13.18v4L12 21l7-3.82v-4L12 17l-7-3.82zM12 3L1 9l11 6 9-4.91V17h2V9L12 3z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
          <overlay-tooltip positioning-target="#nplIcon" direction="bottom" class="style-scope search-box" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      Include non-patent literature
    </div>
  </overlay-tooltip>
          <keyword-wizard id="wizardIcon" class="style-scope search-box">


<div id="wizardContainer" class="style-scope keyword-wizard">
  <iron-icon icon="image:tune" id="wizardIcon" class="style-scope keyword-wizard x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M3 17v2h6v-2H3zM3 5v2h10V5H3zm10 16v-2h8v-2h-8v-2h-2v6h2zM7 9v2H3v2h4v2h2V9H7zm14 4v-2H11v2h10zm-6-4h2V7h4V5h-4V3h-2v6z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
  <paper-material elevation="2" id="wizard" class="style-scope keyword-wizard x-scope paper-material-0" hidden="">
    
    

    
    
    <div class="search-within style-scope keyword-wizard">
      <span class="wizard-header style-scope keyword-wizard">
        Search within
        <iron-icon icon="icons:help-outline" class="help-header style-scope keyword-wizard x-scope iron-icon-0" id="within"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M11 18h2v-2h-2v2zm1-16C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8zm0-14c-2.21 0-4 1.79-4 4h2c0-1.1.9-2 2-2s2 .9 2 2c0 2-3 1.75-3 5h2c0-2.25 3-2.5 3-5 0-2.21-1.79-4-4-4z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      </span>
      
      <div class="help style-scope keyword-wizard" id="withinHelp" hidden="">
        <p class="style-scope keyword-wizard"><b class="style-scope keyword-wizard">Search within the title, abstract, claims, or full patent document</b>: You can
          restrict your search to a specific field using field names.</p>
        <p class="style-scope keyword-wizard">Use <span class="q style-scope keyword-wizard">TI=</span> to search in the title, <span class="q style-scope keyword-wizard">AB=</span> for the
          abstract, <span class="q style-scope keyword-wizard">CL=</span> for the claims, or <span class="q style-scope keyword-wizard">TAC=</span> for
          all three. For example, <span class="q style-scope keyword-wizard">TI=(safety belt)</span>.
        </p>
        <p class="style-scope keyword-wizard"><b class="style-scope keyword-wizard">Search by Cooperative Patent Classifications (CPCs)</b>: These are commonly used to
          represent ideas in place of keywords, and can also be entered in a search term box. If
          you're searching for<span class="q style-scope keyword-wizard">seat belts</span>, you could also search for
          <span class="q style-scope keyword-wizard">B60R22/00</span> to retrieve documents that mention safety belts or body
          harnesses. <span class="q style-scope keyword-wizard">CPC=B60R22</span> will match documents with exactly this CPC,
          <span class="q style-scope keyword-wizard">CPC=B60R22/low</span> matches documents with this CPC or a child
          classification of this CPC.
        </p>
          <a href="https://support.google.com/faqs/answer/7049475" target="_blank" class="style-scope keyword-wizard">Learn More</a>
      </div>
      <div class="filter-chips style-scope keyword-wizard">
        <span class="chip style-scope keyword-wizard" id="title" selected="">Title</span>
        <span class="chip style-scope keyword-wizard" id="abstract" selected="">Abstract</span>
        <span class="chip style-scope keyword-wizard" id="claims" selected="">Claims</span>
        <span class="chip style-scope keyword-wizard" id="fullDoc" selected="">Full Document</span>
      </div>
      
      <div class="cpc-field style-scope keyword-wizard">
        <span class="input-label style-scope keyword-wizard"><label for="inputCPC" class="style-scope keyword-wizard">or CPC</label></span>
        <div class="cpc-inputs style-scope keyword-wizard">
          <input class="keyword-input style-scope keyword-wizard" id="inputCPC" placeholder="Or search with CPC">
          <input type="checkbox" id="childCPC" checked="" class="style-scope keyword-wizard">
          <label class="checkbox-label style-scope keyword-wizard" for="childCPC">
            Include child classifications
          </label>
        </div>
      </div>
    </div>
    
    <div class="find-patents style-scope keyword-wizard">
      <span class="wizard-header style-scope keyword-wizard">
        Find patents
        <iron-icon class="help-header style-scope keyword-wizard x-scope iron-icon-0" icon="icons:help-outline" id="searchBoolean"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M11 18h2v-2h-2v2zm1-16C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8zm0-14c-2.21 0-4 1.79-4 4h2c0-1.1.9-2 2-2s2 .9 2 2c0 2-3 1.75-3 5h2c0-2.25 3-2.5 3-5 0-2.21-1.79-4-4-4z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      </span>
      
      <div class="help style-scope keyword-wizard" id="searchBooleanHelp" hidden="">
        <p class="style-scope keyword-wizard"><b class="style-scope keyword-wizard">Keywords and boolean syntax (USPTO or EPO format)</b>:
          <span class="q style-scope keyword-wizard">seat belt</span> searches these two words, or their plurals and close
          synonyms. <span class="q style-scope keyword-wizard">"seat belt"</span> searches this exact phrase, in order.
          <span class="q style-scope keyword-wizard">-seat -belt</span> searches for documents not containing either word.
        </p>
        <p class="style-scope keyword-wizard"><b class="style-scope keyword-wizard">For searches using boolean logic</b>, the default operator is
          <span class="q style-scope keyword-wizard">AND</span> with left associativity. <i class="style-scope keyword-wizard">Note:</i> this means
          <span class="q style-scope keyword-wizard">safety OR seat belt</span> is searched as <span class="q style-scope keyword-wizard">(safety OR seat)
          AND belt</span>. Each word automatically includes plurals and close synonyms. Adjacent
          words that are implicitly ANDed together, such as <span class="q style-scope keyword-wizard">(safety belt)</span>,
          are treated as a phrase when generating synonyms.
        </p>
        <a href="https://support.google.com/faqs/answer/7049475" target="_blank" class="style-scope keyword-wizard">Learn More</a>
      </div>
      <div class="style-scope keyword-wizard">
        <span class="input-label style-scope keyword-wizard"><label for="inputAnd" class="style-scope keyword-wizard">with ALL of the words</label></span>
        <input class="keyword-input style-scope keyword-wizard" id="inputAnd" placeholder="with ALL of the words">
      </div>
      <div class="style-scope keyword-wizard">
        <span class="input-label style-scope keyword-wizard"><label for="inputExact" class="style-scope keyword-wizard">with the EXACT PHRASE</label></span>
        <input class="keyword-input style-scope keyword-wizard" id="inputExact" placeholder="with the EXACT PHRASE"></div>
      <div class="style-scope keyword-wizard">
        <span class="input-label style-scope keyword-wizard"><label for="inputOr" class="style-scope keyword-wizard">with AT LEAST ONE of the words</label></span>
        <input class="keyword-input style-scope keyword-wizard" id="inputOr" placeholder="with AT LEAST ONE of the words"></div>
      <div class="style-scope keyword-wizard">
        <span class="input-label style-scope keyword-wizard"><label for="inputMinus" class="style-scope keyword-wizard">WITHOUT the words</label></span>
        <input class="keyword-input style-scope keyword-wizard" id="inputMinus" placeholder="WITHOUT the words"></div>
    </div>
    <div class="search-by style-scope keyword-wizard">
      <div class="wizard-header style-scope keyword-wizard">
          Search by
          <iron-icon icon="icons:help-outline" class="help-header style-scope keyword-wizard x-scope iron-icon-0" id="chem"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M11 18h2v-2h-2v2zm1-16C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8zm0-14c-2.21 0-4 1.79-4 4h2c0-1.1.9-2 2-2s2 .9 2 2c0 2-3 1.75-3 5h2c0-2.25 3-2.5 3-5 0-2.21-1.79-4-4-4z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      </div>
      
      <div class="help style-scope keyword-wizard" id="chemHelp" hidden="">
        <p class="style-scope keyword-wizard"><b class="style-scope keyword-wizard">Chemistry searches</b> match terms (trade names, IUPAC names, etc.
          extracted from the entire document, and processed from .MOL files.)
        </p>
        <p class="style-scope keyword-wizard">Substructure (use <span class="q style-scope keyword-wizard">SSS=</span>) and similarity (use
          <span class="q style-scope keyword-wizard">~</span>) searches are limited to one per search at the top-level
          <span class="q style-scope keyword-wizard">AND</span> condition. Exact searches can be used multiple times throughout
          the search query.
        </p>
        <p class="style-scope keyword-wizard">Searching by SMILES or InChi key requires no special syntax. To search by SMARTS, use
          <span class="q style-scope keyword-wizard">SMARTS=</span>.
        </p>
        <p class="style-scope keyword-wizard">To search for multiple molecules, select "Batch" in the "Type" menu. Enter multiple
          molecules separated by whitespace or by comma.
        </p>
        <a href="https://support.google.com/faqs/answer/7049475" target="_blank" class="style-scope keyword-wizard">Learn More</a>
      </div>
      
      <div class="chemistry-field style-scope keyword-wizard">
          <span class="input-label style-scope keyword-wizard"><label for="inputChem" class="style-scope keyword-wizard">Chemistry</label></span>
          <span class="chem-type-label style-scope keyword-wizard"><label for="chemType" hidden="" class="style-scope keyword-wizard">Type</label></span>
          <select id="chemType" class="style-scope keyword-wizard">
            <option value="identical" class="style-scope keyword-wizard">Identical</option>
            <option value="substructure" class="style-scope keyword-wizard">Substructure</option>
            <option value="smarts" class="style-scope keyword-wizard">SMARTS</option>
            <option value="similarity" class="style-scope keyword-wizard">Similarity</option>
            <option value="batch" class="style-scope keyword-wizard">Batch</option>
          </select>
          <textarea class="keyword-input style-scope keyword-wizard" id="inputChem" placeholder="Trade name, SMILES, InChiKey"></textarea>
      </div>
      <div class="import style-scope keyword-wizard">
        <div class="wizard-header style-scope keyword-wizard">
          Patent numbers
        </div>
        <span class="import-button style-scope keyword-wizard">
          <a href="https://patents.google.com/import" class="style-scope keyword-wizard"><button class="style-scope keyword-wizard">Import</button></a>
        </span>
        <p class="import-text style-scope keyword-wizard">
          Search specific patents by importing a CSV or list of patent publication or application numbers.
        </p>
      </div>
    </div>
    
    <div class="wizard-buttons style-scope keyword-wizard">
      <button type="reset" class="style-scope keyword-wizard">Close</button>
      <button type="button" autofocus="" class="style-scope keyword-wizard">Add</button>
    </div>
  
  </paper-material>
</div>
</keyword-wizard>
          <overlay-tooltip positioning-target="#wizardIcon" direction="bottom" class="style-scope search-box" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      Display advanced search options
    </div>
  </overlay-tooltip>
        </div>
      </div>
      <paper-material id="suggestMenu" class="style-scope search-box x-scope paper-material-0" elevation="1" hidden="">
    
    

    
        <template is="dom-repeat" class="style-scope search-box"></template>
        <div class="row style-scope search-box" hidden="">
          <iron-icon class="suggest-icon style-scope search-box x-scope iron-icon-0" icon="error"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm1 15h-2v-2h2v2zm0-4h-2V7h2v6z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon> Sorry, we couldn't find this patent number.
        </div>
        <div class="row style-scope search-box" hidden="">
          <iron-icon class="suggest-icon style-scope search-box x-scope iron-icon-0" icon="error"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm1 15h-2v-2h2v2zm0-4h-2V7h2v6z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon> 
        </div>
      
  </paper-material>
      <button id="searchButton" class="style-scope search-box"><iron-icon class="style-scope search-box x-scope iron-icon-0" icon="search"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon></button>
    </div>
  </search-box>
          <template is="dom-if" class="style-scope search-header"></template>
          <div class="flex style-scope search-header"></div>
          
            <div id="header-container" class="style-scope search-header">
              <result-nav class="style-scope search-header">
    
    <div class="style-scope result-nav" hidden="">
      <state-modifier id="current" act="{&quot;type&quot;: &quot;BACK_TO_RESULTS&quot;}" inherit="" class="style-scope result-nav"><a id="link" href="#" class="style-scope state-modifier" inherit=""><span class="style-scope result-nav"></span> of
      <template is="dom-if" class="style-scope result-nav"></template>
      0<template is="dom-if" class="style-scope result-nav"></template></a></state-modifier>
      <state-modifier id="prevResult" inherit="" act="{&quot;type&quot;: &quot;NAV_RESULT&quot;, &quot;result&quot;: &quot;$id&quot;, &quot;page&quot;: $page}" class="style-scope result-nav" disabled=""><a id="link" href="#" class="style-scope state-modifier" inherit="">
        <iron-icon icon="chevron-left" class="style-scope result-nav x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M15.41 7.41L14 6l-6 6 6 6 1.41-1.41L10.83 12z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      </a></state-modifier><state-modifier id="nextResult" inherit="" act="{&quot;type&quot;: &quot;NAV_RESULT&quot;, &quot;result&quot;: &quot;$id&quot;, &quot;page&quot;: $page}" class="style-scope result-nav" disabled=""><a id="link" href="#" class="style-scope state-modifier" inherit="">
        <iron-icon icon="chevron-right" class="style-scope result-nav x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M10 6L8.59 7.41 13.17 12l-4.58 4.59L10 18l6-6z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      </a></state-modifier>
      <overlay-tooltip positioning-target="#prevResult" direction="bottom" arrow="" class="style-scope result-nav" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip"></div>
    <div class="style-scope overlay-tooltip">
      Previous result
    </div>
  </overlay-tooltip>
      <overlay-tooltip positioning-target="#nextResult" direction="bottom" arrow="" class="style-scope result-nav" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip"></div>
    <div class="style-scope overlay-tooltip">
      Next result
    </div>
  </overlay-tooltip>
    </div>
  </result-nav>
            </div>
          <template is="dom-if" class="style-scope search-header"></template>
        </div>
      </header>
    </div>
  </search-header>

      <div class="layout horizontal style-scope search-ui">
        <template is="dom-if" class="style-scope search-ui"></template>

        <div id="main" class="style-scope search-ui">
          <search-term-wizard class="style-scope search-ui"><pat-dialog open="" class="style-scope search-term-wizard">
  <span class="header style-scope search-term-wizard">
    <span class="title style-scope search-term-wizard">Search tools</span>
    <span id="language" class="style-scope search-term-wizard">
      <text-button id="textButton" selected="" class="style-scope search-term-wizard">Text
</text-button>
      <text-button id="classificationButton" class="style-scope search-term-wizard">Classification
</text-button>
      <text-button id="chemistryButton" class="style-scope search-term-wizard">Chemistry
</text-button>
      <text-button id="measureButton" class="style-scope search-term-wizard">Measure
</text-button>
      <text-button id="numbersButton" class="style-scope search-term-wizard">Numbers
</text-button>
    </span>
  </span>
  <span id="panels" class="style-scope search-term-wizard">
    <text-search-term-wizard id="textPanel" class="style-scope search-term-wizard" selected=""><span class="style-scope text-search-term-wizard">
  <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49" class="style-scope text-search-term-wizard">
    <path d="M3.85 15.275V13h10.3v2.275Zm0-4.275V8.725h16.3V11Z" class="style-scope text-search-term-wizard"></path>
  </svg>
  <outlined-text-field id="textField" placeholder="Type a concept you want to search for here" class="style-scope text-search-term-wizard" value=""><span class="style-scope outlined-text-field">
  <input id="input" spellcheck="false" autocomplete="off" class="style-scope outlined-text-field" aria-disabled="false" placeholder="Type a concept you want to search for here">
  
</span>
</outlined-text-field>
</span>
<span class="style-scope text-search-term-wizard">
  <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49" class="style-scope text-search-term-wizard">
    <path d="M8 18.025h8v-2.1H8Zm0-4h8v-2.1H8ZM6.075 22.2q-.95 0-1.612-.662-.663-.663-.663-1.613V4.075q0-.95.663-1.613.662-.662 1.612-.662h8L20.2 7.925v12q0 .95-.662 1.613-.663.662-1.613.662Zm6.85-13.125v-5h-6.85v15.85h11.85V9.075Zm-6.85-5v5-5 15.85-15.85Z" class="style-scope text-search-term-wizard"></path>
  </svg>
  <filter-chip id="fullDocuments" selected="" class="style-scope text-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Full documents</filter-chip>
  <filter-chip id="title" class="style-scope text-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Title</filter-chip>
  <filter-chip id="abstract" class="style-scope text-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Abstract</filter-chip>
  <filter-chip id="claims" class="style-scope text-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Claims</filter-chip>
</span>
<span class="style-scope text-search-term-wizard">
  <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49" class="style-scope text-search-term-wizard">
    <path d="M16 16.925q2.05 0 3.488-1.437Q20.925 14.05 20.925 12t-1.437-3.488Q18.05 7.075 16 7.075q-.625 0-1.2.15t-1.075.425q.7.925 1.087 2.025.388 1.1.388 2.325t-.388 2.325q-.387 1.1-1.087 2.025.5.275 1.075.425.575.15 1.2.15Zm-4-2.075q.45-.575.688-1.313.237-.737.237-1.537 0-.8-.237-1.538Q12.45 9.725 12 9.15q-.45.575-.688 1.312-.237.738-.237 1.538t.237 1.537q.238.738.688 1.313Zm-4 2.075q.625 0 1.2-.15t1.075-.425q-.7-.925-1.087-2.025Q8.8 13.225 8.8 12t.388-2.325q.387-1.1 1.087-2.025-.5-.275-1.075-.425-.575-.15-1.2-.15-2.05 0-3.487 1.437Q3.075 9.95 3.075 12t1.438 3.488Q5.95 16.925 8 16.925ZM8 19.2q-3 0-5.1-2.1Q.8 15 .8 12q0-3 2.1-5.1Q5 4.8 8 4.8q1.125 0 2.137.312 1.013.313 1.863.913.85-.6 1.863-.913Q14.875 4.8 16 4.8q3 0 5.1 2.1Q23.2 9 23.2 12q0 3-2.1 5.1-2.1 2.1-5.1 2.1-1.125 0-2.137-.313-1.013-.312-1.863-.912-.85.6-1.863.912Q9.125 19.2 8 19.2Z" class="style-scope text-search-term-wizard"></path>
  </svg>
  <filter-chip id="all" selected="" class="style-scope text-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
All</filter-chip>
  <filter-chip id="any" class="style-scope text-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Any</filter-chip>
  <filter-chip id="exact" class="style-scope text-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Exact</filter-chip>
  <filter-chip id="not" class="style-scope text-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Not</filter-chip>
</span>
<span class="style-scope text-search-term-wizard">
  <mat-button id="button" filled="" class="style-scope text-search-term-wizard x-scope mat-button-0" disabled=""><span slot="label" class="style-scope text-search-term-wizard">Add AND condition</span></mat-button></span></text-search-term-wizard>
    <classification-wizard id="classificationPanel" class="style-scope search-term-wizard"><span class="style-scope classification-wizard">
  <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49" class="style-scope classification-wizard">
    <path d="M3.85 15.275V13h10.3v2.275Zm0-4.275V8.725h16.3V11Z" class="style-scope classification-wizard"></path>
  </svg>
  <outlined-text-field id="textField" placeholder="Enter a CPC here" uppercase="" class="style-scope classification-wizard" value=""><span class="style-scope outlined-text-field">
  <input id="input" spellcheck="false" autocomplete="off" class="style-scope outlined-text-field" aria-disabled="false" placeholder="Enter a CPC here">
  
</span>
</outlined-text-field>
</span>
<template is="dom-if" class="style-scope classification-wizard"></template>
<template is="dom-if" class="style-scope classification-wizard"></template>
<template is="dom-if" class="style-scope classification-wizard"></template>
<span class="style-scope classification-wizard">
  <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49" class="style-scope classification-wizard">
    <path d="M16 16.925q2.05 0 3.488-1.437Q20.925 14.05 20.925 12t-1.437-3.488Q18.05 7.075 16 7.075q-.625 0-1.2.15t-1.075.425q.7.925 1.087 2.025.388 1.1.388 2.325t-.388 2.325q-.387 1.1-1.087 2.025.5.275 1.075.425.575.15 1.2.15Zm-4-2.075q.45-.575.688-1.313.237-.737.237-1.537 0-.8-.237-1.538Q12.45 9.725 12 9.15q-.45.575-.688 1.312-.237.738-.237 1.538t.237 1.537q.238.738.688 1.313Zm-4 2.075q.625 0 1.2-.15t1.075-.425q-.7-.925-1.087-2.025Q8.8 13.225 8.8 12t.388-2.325q.387-1.1 1.087-2.025-.5-.275-1.075-.425-.575-.15-1.2-.15-2.05 0-3.487 1.437Q3.075 9.95 3.075 12t1.438 3.488Q5.95 16.925 8 16.925ZM8 19.2q-3 0-5.1-2.1Q.8 15 .8 12q0-3 2.1-5.1Q5 4.8 8 4.8q1.125 0 2.137.312 1.013.313 1.863.913.85-.6 1.863-.913Q14.875 4.8 16 4.8q3 0 5.1 2.1Q23.2 9 23.2 12q0 3-2.1 5.1-2.1 2.1-5.1 2.1-1.125 0-2.137-.313-1.013-.312-1.863-.912-.85.6-1.863.912Q9.125 19.2 8 19.2Z" class="style-scope classification-wizard"></path>
  </svg>
  <filter-chip id="notExact" class="style-scope classification-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>

    These CPCs and their children
  </filter-chip>
  <filter-chip selected="" id="exact" class="style-scope classification-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>

    These exact CPCs
  </filter-chip>
</span>
<span class="style-scope classification-wizard">
  <mat-button id="button" filled="" class="style-scope classification-wizard x-scope mat-button-0" disabled=""><span slot="label" class="style-scope classification-wizard">Add AND condition</span></mat-button>
</span></classification-wizard>
    <chemistry-search-term-wizard id="chemistryPanel" class="style-scope search-term-wizard"><span class="style-scope chemistry-search-term-wizard">
  <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49" class="style-scope chemistry-search-term-wizard">
    <path d="M3.85 15.275V13h10.3v2.275Zm0-4.275V8.725h16.3V11Z" class="style-scope chemistry-search-term-wizard"></path>
  </svg>
  <outlined-text-field id="textField" placeholder="Trade name, SMILES, InChI Key" class="style-scope chemistry-search-term-wizard" value=""><span class="style-scope outlined-text-field">
  <input id="input" spellcheck="false" autocomplete="off" class="style-scope outlined-text-field" aria-disabled="false" placeholder="Trade name, SMILES, InChI Key">
  
</span>
</outlined-text-field>
  <outlined-textarea id="textArea" placeholder="Enter multiple SMILES or InChI Keys, one per line" rows="4" class="style-scope chemistry-search-term-wizard"><span class="style-scope outlined-textarea">
  <textarea id="textarea" autocomplete="off" autocorrect="off" spellcheck="false" tabindex="0" aria-disabled="false" wrap="soft" class="style-scope outlined-textarea" rows="4" placeholder="Enter multiple SMILES or InChI Keys, one per line">  </textarea>
  <div id="helperText" class="style-scope outlined-textarea"> </div>
</span>
<span class="right-button-container style-scope outlined-textarea">
  
</span>
</outlined-textarea>
  <a href="https://support.google.com/faqs/answer/7049475#zippy=%2Csearching-by-chemistry" target="_blank" class="style-scope chemistry-search-term-wizard">
    <svg class="help style-scope chemistry-search-term-wizard" xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49">
      <path d="M11.95 18q.525 0 .888-.363.362-.362.362-.887t-.362-.887q-.363-.363-.888-.363t-.888.363q-.362.362-.362.887t.362.887q.363.363.888.363Zm-.9-3.85h1.85q0-.825.188-1.3.187-.475 1.062-1.3.65-.65 1.025-1.238.375-.587.375-1.412 0-1.4-1.025-2.15T12.1 6q-1.425 0-2.312.75-.888.75-1.238 1.8l1.65.65q.125-.45.563-.975Q11.2 7.7 12.1 7.7q.8 0 1.2.437.4.438.4.963 0 .5-.3.937-.3.438-.75.813-1.1.975-1.35 1.475-.25.5-.25 1.825ZM12 22q-2.075 0-3.9-.788-1.825-.787-3.175-2.137-1.35-1.35-2.137-3.175Q2 14.075 2 12t.788-3.9q.787-1.825 2.137-3.175 1.35-1.35 3.175-2.138Q9.925 2 12 2t3.9.787q1.825.788 3.175 2.138 1.35 1.35 2.137 3.175Q22 9.925 22 12t-.788 3.9q-.787 1.825-2.137 3.175-1.35 1.35-3.175 2.137Q14.075 22 12 22Zm0-2q3.35 0 5.675-2.325Q20 15.35 20 12q0-3.35-2.325-5.675Q15.35 4 12 4 8.65 4 6.325 6.325 4 8.65 4 12q0 3.35 2.325 5.675Q8.65 20 12 20Zm0-8Z" class="style-scope chemistry-search-term-wizard"></path>
    </svg>
  </a>
</span>
<span id="chemQuantifiers" class="style-scope chemistry-search-term-wizard">
  <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49" class="style-scope chemistry-search-term-wizard">
    <path d="M5 21.2q-1.4 0-2-1.25-.6-1.25.3-2.325l5.5-6.7V5.15h-.825q-.475 0-.8-.338Q6.85 4.475 6.85 4q0-.475.325-.813.325-.337.8-.337h8.05q.475 0 .8.337.325.338.325.813 0 .475-.325.812-.325.338-.8.338H15.2v5.775l5.5 6.7q.9 1.075.3 2.325-.6 1.25-2 1.25Zm2.15-3.275h9.7l-3.275-3.85h-3.15Zm-2 1h13.7l-5.925-7.2V5.15h-1.85v6.575Zm6.85-6.9Z" class="style-scope chemistry-search-term-wizard"></path>
  </svg>
  <filter-chip id="exact" selected="" class="style-scope chemistry-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Exact</filter-chip>
  <filter-chip id="exactBatch" class="style-scope chemistry-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Exact Batch</filter-chip>
  <filter-chip id="similar" class="style-scope chemistry-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Similar</filter-chip>
  <filter-chip id="substructure" class="style-scope chemistry-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Substructure</filter-chip>
  <filter-chip id="smarts" class="style-scope chemistry-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Substructure (SMARTS)</filter-chip>
</span>
<span id="location" class="style-scope chemistry-search-term-wizard">
  <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49" class="style-scope chemistry-search-term-wizard">
    <path d="M8 18.025h8v-2.1H8Zm0-4h8v-2.1H8ZM6.075 22.2q-.95 0-1.612-.662-.663-.663-.663-1.613V4.075q0-.95.663-1.613.662-.662 1.612-.662h8L20.2 7.925v12q0 .95-.662 1.613-.663.662-1.613.662Zm6.85-13.125v-5h-6.85v15.85h11.85V9.075Zm-6.85-5v5-5 15.85-15.85Z" class="style-scope chemistry-search-term-wizard"></path>
  </svg>
  <filter-chip id="fullDocuments" selected="" class="style-scope chemistry-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Full documents</filter-chip>
  <filter-chip id="claims" class="style-scope chemistry-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Claims only</filter-chip>
</span>
<span class="style-scope chemistry-search-term-wizard">
  <mat-button id="button" filled="" class="style-scope chemistry-search-term-wizard x-scope mat-button-0" disabled=""><span slot="label" class="style-scope chemistry-search-term-wizard">Add AND condition</span></mat-button>
</span>
</chemistry-search-term-wizard>
    <measure-search-term-wizard id="measurePanel" class="style-scope search-term-wizard"><span class="style-scope measure-search-term-wizard">
  <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49" class="style-scope measure-search-term-wizard">
    <path d="M4.1 18.15q-.95 0-1.612-.663-.663-.662-.663-1.612v-7.75q0-.95.663-1.613Q3.15 5.85 4.1 5.85h15.8q.95 0 1.613.662.662.663.662 1.613v7.75q0 .95-.662 1.612-.663.663-1.613.663Zm0-2.275h15.8v-7.75H17V12h-2V8.125h-2V12h-2V8.125H9V12H7V8.125H4.1v7.75ZM7 12h2Zm4 0h2Zm4 0h2Zm-3 0Z" class="style-scope measure-search-term-wizard"></path>
  </svg>
  <outlined-text-field id="measureExpression" placeholder="1.5 mm, 20 ft, 400-500 fahrenheit, 800 MHz, 0.01-100 mol" class="style-scope measure-search-term-wizard" value=""><span class="style-scope outlined-text-field">
  <input id="input" spellcheck="false" autocomplete="off" class="style-scope outlined-text-field" aria-disabled="false" placeholder="1.5 mm, 20 ft, 400-500 fahrenheit, 800 MHz, 0.01-100 mol">
  
</span>
</outlined-text-field>
</span>
<span class="style-scope measure-search-term-wizard">
  <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49" class="style-scope measure-search-term-wizard">
    <path d="M3.875 20.15V8.85h4.25v11.3Zm6 0V3.85h4.25v16.3Zm6 0v-7.3h4.25v7.3Z" class="style-scope measure-search-term-wizard"></path>
  </svg>
  <outlined-text-field id="context" placeholder="activity, wavelength, embossing depth, absorption" class="style-scope measure-search-term-wizard" value=""><span class="style-scope outlined-text-field">
  <input id="input" spellcheck="false" autocomplete="off" class="style-scope outlined-text-field" aria-disabled="false" placeholder="activity, wavelength, embossing depth, absorption">
  
</span>
</outlined-text-field>
</span>
<span class="style-scope measure-search-term-wizard">
  <mat-button id="button" filled="" class="style-scope measure-search-term-wizard x-scope mat-button-0" disabled=""><span slot="label" class="style-scope measure-search-term-wizard">Add AND condition</span></mat-button>
</span>
</measure-search-term-wizard>
    <numerical-search-term-wizard id="numericalPanel" class="style-scope search-term-wizard"><span class="style-scope numerical-search-term-wizard">
  <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49" class="style-scope numerical-search-term-wizard">
    <path d="M3.85 15.275V13h10.3v2.275Zm0-4.275V8.725h16.3V11Z" class="style-scope numerical-search-term-wizard"></path>
  </svg>
  <outlined-textarea id="batch" placeholder="Enter multiple Patent Application and Publication Numbers, one per line" rows="4" class="style-scope numerical-search-term-wizard"><span class="style-scope outlined-textarea">
  <textarea id="textarea" autocomplete="off" autocorrect="off" spellcheck="false" tabindex="0" aria-disabled="false" wrap="soft" class="style-scope outlined-textarea" rows="4" placeholder="Enter multiple Patent Application and Publication Numbers, one per line">  </textarea>
  <div id="helperText" class="style-scope outlined-textarea"> </div>
</span>
<span class="right-button-container style-scope outlined-textarea">
  
</span>
</outlined-textarea>
</span>
<template is="dom-if" class="style-scope numerical-search-term-wizard"></template>
<template is="dom-if" class="style-scope numerical-search-term-wizard"></template>
<span class="style-scope numerical-search-term-wizard">
  <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49" class="style-scope numerical-search-term-wizard">
    <path d="m10.825 23.45-1.6-1.625L10.9 20.15H8q-.35.9-1.162 1.488-.813.587-1.838.587-1.35 0-2.287-.937-.938-.938-.938-2.288 0-1.025.588-1.837Q2.95 16.35 3.85 16V8q-.9-.35-1.487-1.163Q1.775 6.025 1.775 5q0-1.35.938-2.288Q3.65 1.775 5 1.775q1.025 0 1.838.587Q7.65 2.95 8 3.85h2.9L9.225 2.175l1.6-1.625L15.275 5l-4.45 4.45-1.6-1.625L10.9 6.15H8q-.25.65-.725 1.125T6.15 8v8q.65.25 1.125.725T8 17.85h2.9l-1.675-1.675 1.6-1.625 4.45 4.45ZM19 22.225q-1.35 0-2.287-.937-.938-.938-.938-2.288 0-1.05.588-1.863.587-.812 1.487-1.137V8q-.9-.325-1.487-1.137-.588-.813-.588-1.863 0-1.35.938-2.288.937-.937 2.287-.937 1.35 0 2.288.937.937.938.937 2.288 0 1.05-.587 1.863Q21.05 7.675 20.15 8v8q.9.35 1.488 1.163.587.812.587 1.837 0 1.35-.937 2.288-.938.937-2.288.937ZM5 19.95q.4 0 .675-.275T5.95 19q0-.4-.275-.675T5 18.05q-.4 0-.675.275T4.05 19q0 .4.275.675T5 19.95Zm14 0q.4 0 .675-.275T19.95 19q0-.4-.275-.675T19 18.05q-.4 0-.675.275T18.05 19q0 .4.275.675t.675.275Zm-14-14q.4 0 .675-.275T5.95 5q0-.4-.275-.675T5 4.05q-.4 0-.675.275T4.05 5q0 .4.275.675T5 5.95Zm14 0q.4 0 .675-.275T19.95 5q0-.4-.275-.675T19 4.05q-.4 0-.675.275T18.05 5q0 .4.275.675T19 5.95ZM5 19Zm14 0ZM5 5Zm14 0Z" class="style-scope numerical-search-term-wizard"></path>
  </svg>
  <filter-chip id="applicationNumbers" class="style-scope numerical-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Application Numbers</filter-chip>
  <filter-chip id="publicationNumbers" class="style-scope numerical-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Publication Numbers</filter-chip>
  <filter-chip id="either" selected="" class="style-scope numerical-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Either</filter-chip>
</span>
<span class="style-scope numerical-search-term-wizard">
  <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49" class="style-scope numerical-search-term-wizard">
    <path d="M12 22.2q-2.1 0-3.962-.8-1.863-.8-3.25-2.175Q3.4 17.85 2.6 15.988 1.8 14.125 1.8 12t.8-3.988q.8-1.862 2.188-3.237Q6.175 3.4 8.038 2.6 9.9 1.8 12 1.8q2.125 0 3.988.8 1.862.8 3.237 2.175Q20.6 6.15 21.4 8.012q.8 1.863.8 3.988t-.8 3.988q-.8 1.862-2.175 3.237Q17.85 20.6 15.988 21.4q-1.863.8-3.988.8Zm-1-2.325v-1.95q-.8 0-1.387-.575-.588-.575-.588-1.4v-.975L4.275 10.2q-.075.45-.137.9-.063.45-.063.9 0 3 1.975 5.25T11 19.875Zm6.85-2.525q.5-.55.888-1.175.387-.625.65-1.313.262-.687.4-1.412.137-.725.137-1.45 0-2.425-1.338-4.438-1.337-2.012-3.612-2.887v.375q0 .825-.587 1.4-.588.575-1.388.575h-2v2q0 .4-.287.687-.288.288-.688.288h-2v2h5.95q.425 0 .713.287.287.288.287.688v2.975h1q.65 0 1.163.387.512.388.712 1.013Z" class="style-scope numerical-search-term-wizard"></path>
  </svg>
  <outlined-text-field id="countryFilter" placeholder="Restrict search to these countries" class="style-scope numerical-search-term-wizard" value=""><span class="style-scope outlined-text-field">
  <input id="input" spellcheck="false" autocomplete="off" class="style-scope outlined-text-field" aria-disabled="false" placeholder="Restrict search to these countries">
  
</span>
</outlined-text-field>
</span>
<span class="style-scope numerical-search-term-wizard">
  <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49" class="style-scope numerical-search-term-wizard">
    <path d="M2.85 18.225V15.95h6.3v2.275Zm0-5.075v-2.3h12.3v2.3Zm0-5.1V5.775h18.3V8.05Z" class="style-scope numerical-search-term-wizard"></path>
  </svg>
  <outlined-text-field id="countryOrder" placeholder="Prefer results from these countries in this order" class="style-scope numerical-search-term-wizard" value="US, EP, WO, JP, CN"><span class="style-scope outlined-text-field">
  <input id="input" spellcheck="false" autocomplete="off" class="style-scope outlined-text-field" aria-disabled="false" placeholder="Prefer results from these countries in this order">
  
</span>
</outlined-text-field>
</span>
<span class="style-scope numerical-search-term-wizard">
  <mat-button id="button" filled="" class="style-scope numerical-search-term-wizard x-scope mat-button-0" disabled=""><span slot="label" class="style-scope numerical-search-term-wizard">Add AND condition</span></mat-button>
</span>
</numerical-search-term-wizard>
  </span>

</pat-dialog>
</search-term-wizard>
          
      <div content="" class="style-scope search-result">
        <div id="mainContainer" class="vertical layout style-scope search-result">
          <template is="dom-if" class="style-scope search-result"></template>
          <result-container class="style-scope search-result">
    

    <paper-spinner class="style-scope result-container x-scope paper-spinner-0" aria-hidden="true" hidden=""><div id="spinnerContainer" class="  style-scope paper-spinner"><div class="spinner-layer layer-1 style-scope paper-spinner"><div class="circle-clipper left style-scope paper-spinner"></div><div class="circle-clipper right style-scope paper-spinner"></div></div><div class="spinner-layer layer-2 style-scope paper-spinner"><div class="circle-clipper left style-scope paper-spinner"></div><div class="circle-clipper right style-scope paper-spinner"></div></div><div class="spinner-layer layer-3 style-scope paper-spinner"><div class="circle-clipper left style-scope paper-spinner"></div><div class="circle-clipper right style-scope paper-spinner"></div></div><div class="spinner-layer layer-4 style-scope paper-spinner"><div class="circle-clipper left style-scope paper-spinner"></div><div class="circle-clipper right style-scope paper-spinner"></div></div></div></paper-spinner>
    
      <patent-result class="style-scope result-container">
    
    
    <div class="layout horizontal style-scope patent-result">
      <div style="width: 1200px; height: auto; overflow: auto;" class="style-scope patent-result">
        <div id="wrapper" tabindex="0" class="style-scope patent-result">
          <result-tags class="style-scope patent-result"></result-tags>
          <h1 id="title" class="scroll-target style-scope patent-result">
            Chimeric antigens for breaking host tolerance to foreign antigens 
     </h1>
          <div class="horizontal layout style-scope patent-result">
            
            <div class="flex-3 style-scope patent-result" style="margin-top:-20px;overflow:hidden;" marginright="">
              <section id="abstract" class="style-scope patent-result">
                <h3 class="layout horizontal style-scope patent-result">
                  <div class="flex style-scope patent-result">Abstract</div>
                  <div class="righthead style-scope patent-result" hidden="">translated from </div>
                </h3>
                <patent-text name="abstract" class="style-scope patent-result" style="padding-top: 0px;">
    
    
    <template is="dom-if" restamp="" class="style-scope patent-text"></template>
    <overlay-tooltip id="tooltip" class="style-scope patent-text" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      
    </div>
  </overlay-tooltip>
    <div class="layout horizontal style-scope patent-text">
      <section id="text" class="flex style-scope patent-text"><abstract mxw-id="PA111188033" lang="EN" source="national office" load-source="docdb" class="style-scope patent-text">
    <div class="abstract style-scope patent-text">Disclosed herein are compositions and methods for eliciting immune responses against antigens. In particular, the compounds and methods elicit immune responses against foreign antigens that are otherwise recognized by the host as "self' antigens, thus breaking host tolerance to those antigens. Presenting the host immune system with a chimeric antigen comprising an immune response domain and a target binding domain, wherein the target binding domain comprises an antibody fragment, enhances the immune response against the foreign or tolerated antigen. Antigen presenting cells take up, process, and present the chimeric antigen, eliciting both a humoral and cellular immune response against the desired antigen.</div>
  </abstract>
  </section>
      <div id="margin" class="style-scope patent-text">
        <div id="createHighlight" style="top: 0px;" class="style-scope patent-text" hidden="">
          <iron-icon id="comment" icon="editor:insert-comment" class="style-scope patent-text x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M20 2H4c-1.1 0-2 .9-2 2v12c0 1.1.9 2 2 2h14l4 4V4c0-1.1-.9-2-2-2zm-2 12H6v-2h12v2zm0-3H6V9h12v2zm0-3H6V6h12v2z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
        </div>
      </div>
    </div>
  </patent-text>
              </section>
              
              <template is="dom-if" class="style-scope patent-result"></template>
              
              <section id="classifications" class="style-scope patent-result" hidden="">
                <h3 class="layout horizontal style-scope patent-result">
                  <div class="flex style-scope patent-result">Classifications</div>
                  <div id="machineClassifiedWarning" class="righthead tooltip-hint style-scope patent-result" hidden="">machine-classified</div>
                  <div id="machineClassifiedCPCWarning" class="righthead tooltip-hint style-scope patent-result" hidden="">cpc-machine-classified</div>
                  <div id="machineClassifiedFtermWarning" class="righthead tooltip-hint style-scope patent-result" hidden="">fterm-machine-classified</div>
                  <div class="style-scope patent-result" hidden="">&nbsp;</div>
                  <div id="familyClassifiedFtermWarning" class="righthead tooltip-hint style-scope patent-result" hidden="">fterm-family-classified</div>
                </h3>
                <overlay-tooltip positioning-target="#machineClassifiedWarning" direction="bottom" class="style-scope patent-result" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      The classifications are assigned by a computer and are not a legal conclusion. <br class="style-scope patent-result"> Google has not performed a legal analysis and makes no representation as to the accuracy of the classifications listed.
    </div>
  </overlay-tooltip>
                <overlay-tooltip positioning-target="#machineClassifiedCPCWarning" direction="bottom" class="style-scope patent-result" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      The CPC classifications are assigned by a computer and are not a legal conclusion. <br class="style-scope patent-result"> Google has not performed a legal analysis and makes no representation as to the accuracy of the classifications listed.
    </div>
  </overlay-tooltip>
                <overlay-tooltip positioning-target="#familyClassifiedFtermWarning" direction="bottom" class="style-scope patent-result" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      The F-term classifications are assigned based on a patent family member containing these classification codes.
    </div>
  </overlay-tooltip>
                <overlay-tooltip positioning-target="#machineClassifiedFtermWarning" direction="bottom" class="style-scope patent-result" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      The F-term classifications are assigned by a computer and are not a legal conclusion. <br class="style-scope patent-result"> Google has not performed a legal analysis and makes no representation as to the accuracy of the classifications listed.
    </div>
  </overlay-tooltip>
                <classification-viewer class="style-scope patent-result">
    
    <div class="table style-scope classification-viewer">
      <template is="dom-repeat" class="style-scope classification-viewer"></template>
      <template is="dom-if" class="style-scope classification-viewer"></template>
    </div>
  </classification-viewer>
              </section>
              
              <template is="dom-if" class="style-scope patent-result"></template>
              
              <template is="dom-if" class="style-scope patent-result"></template>
              
              
                <section class="style-scope patent-result">
                  <h3 class="layout horizontal style-scope patent-result">
                    <div class="flex style-scope patent-result">Landscapes</div>
                  </h3>
                  <div class="responsive-grid style-scope patent-result">
                    
                      <landscape-link class="style-scope patent-result"><state-modifier act="{&quot;type&quot;:&quot;QUERY_SET_TERM&quot;,&quot;term&quot;:&quot;$term&quot;,&quot;origin&quot;:&quot;LANDSCAPE_LINK&quot;}" class="style-scope landscape-link" data-term="Medicines Containing Antibodies Or Antigens For Use As Internal Diagnostic Agents"><a id="link" href="#" class="style-scope state-modifier">
  <requery-tile class="area style-scope landscape-link">
<span class="label style-scope requery-tile">
    Medicines Containing Antibodies Or Antigens For Use As Internal Diagnostic Agents
  </span>
<span class="search-icon style-scope requery-tile"></span>
</requery-tile>
</a></state-modifier>
</landscape-link>
                    
                      <landscape-link class="style-scope patent-result"><state-modifier act="{&quot;type&quot;:&quot;QUERY_SET_TERM&quot;,&quot;term&quot;:&quot;$term&quot;,&quot;origin&quot;:&quot;LANDSCAPE_LINK&quot;}" class="style-scope landscape-link" data-term="Medicines That Contain Protein Lipid Enzymes And Other Medicines"><a id="link" href="#" class="style-scope state-modifier">
  <requery-tile class="area style-scope landscape-link">
<span class="label style-scope requery-tile">
    Medicines That Contain Protein Lipid Enzymes And Other Medicines
  </span>
<span class="search-icon style-scope requery-tile"></span>
</requery-tile>
</a></state-modifier>
</landscape-link>
                    <template is="dom-repeat" class="style-scope patent-result"></template>
                  </div>
                  <s-more-inline class="style-scope patent-result"><hr aria-hidden="true" class="style-scope s-more-inline">

<div class="button style-scope s-more-inline" style="" hidden="">
  <span slot="expand_text" class="style-scope patent-result">Show more</span>
  <span class="expand more style-scope s-more-inline"></span>
</div>
<div class="button style-scope s-more-inline">
  <span slot="collapse_text" class="style-scope patent-result">Show less</span>
  <span class="expand less style-scope s-more-inline"></span>
</div>
</s-more-inline>
                </section>
              <template is="dom-if" class="style-scope patent-result"></template>
            </div>
            <div class="flex-2 style-scope patent-result">
              <section class="knowledge-card style-scope patent-result">
                <header class="style-scope patent-result">
                  <h2 id="pubnum" class="style-scope patent-result">AU2013204668A1</h2>
                  <p class="tagline style-scope patent-result">Australia</p>
                  <overlay-tooltip positioning-target="#pubnum" direction="bottom" class="style-scope patent-result" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      
                    <div style="max-width: 300px;" class="style-scope patent-result">Open to public inspection</div>
                  
    </div>
  </overlay-tooltip>

                  <div class="knowledge-card-action-bar style-scope patent-result">
                    <a target="_blank" class="style-scope patent-result" href="https://patentimages.storage.googleapis.com/30/37/1b/05561c7a60a77e/AU2013204668A1.pdf"><iron-icon icon="description" class="style-scope patent-result x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M14 2H6c-1.1 0-1.99.9-1.99 2L4 20c0 1.1.89 2 1.99 2H18c1.1 0 2-.9 2-2V8l-6-6zm2 16H8v-2h8v2zm0-4H8v-2h8v2zm-3-5V3.5L18.5 9H13z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon> Download PDF</a>
                    <state-modifier inherit="" act="{&quot;type&quot;: &quot;QUERY_FIND_PRIOR_ART&quot;, &quot;keywords&quot;: $keywords, &quot;before&quot;: &quot;$before&quot;}" class="style-scope patent-result" data-keywords="[&quot;antigen&quot;,&quot;chimeric antigen&quot;,&quot;immune response&quot;,&quot;protein&quot;,&quot;cells&quot;]" data-before="20030808"><a id="link" href="#" class="style-scope state-modifier" inherit="">
                      <iron-icon icon="find-in-page" class="style-scope patent-result x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M20 19.59V8l-6-6H6c-1.1 0-1.99.9-1.99 2L4 20c0 1.1.89 2 1.99 2H18c.45 0 .85-.15 1.19-.4l-4.43-4.43c-.8.52-1.74.83-2.76.83-2.76 0-5-2.24-5-5s2.24-5 5-5 5 2.24 5 5c0 1.02-.31 1.96-.83 2.75L20 19.59zM9 13c0 1.66 1.34 3 3 3s3-1.34 3-3-1.34-3-3-3-3 1.34-3 3z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon> Find Prior Art
                    </a></state-modifier>
                    <state-modifier inherit="" act="{&quot;type&quot;: &quot;QUERY_SIMILAR_DOCUMENTS&quot;, &quot;id&quot;: &quot;$id&quot;}" class="style-scope patent-result" data-id="patent/AU2013204668A1"><a id="link" href="#" class="style-scope state-modifier" inherit="">
                      <iron-icon icon="editor:functions" class="style-scope patent-result x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M18 4H6v2l6.5 6L6 18v2h12v-3h-7l5-5-5-5h7z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon> Similar
                    </a></state-modifier>
                  </div>
                </header>

                <dl class="important-people style-scope patent-result">
                  <template is="dom-if" class="style-scope patent-result"></template>
                  
                  <dt class="style-scope patent-result">Inventor</dt>
                  <dd class="style-scope patent-result"><state-modifier act="{&quot;type&quot;: &quot;QUERY_SEARCH_INVENTOR&quot;, &quot;inventor&quot;: &quot;$inventor&quot;}" class="style-scope patent-result" data-inventor="Rajan George"><a id="link" href="#" class="style-scope state-modifier">Rajan George</a></state-modifier></dd><dd class="style-scope patent-result"><state-modifier act="{&quot;type&quot;: &quot;QUERY_SEARCH_INVENTOR&quot;, &quot;inventor&quot;: &quot;$inventor&quot;}" class="style-scope patent-result" data-inventor="Allan Ma"><a id="link" href="#" class="style-scope state-modifier">Allan Ma</a></state-modifier></dd><dd class="style-scope patent-result"><state-modifier act="{&quot;type&quot;: &quot;QUERY_SEARCH_INVENTOR&quot;, &quot;inventor&quot;: &quot;$inventor&quot;}" class="style-scope patent-result" data-inventor="Antoine Noujaim"><a id="link" href="#" class="style-scope state-modifier">Antoine Noujaim</a></state-modifier></dd><dd class="style-scope patent-result"><state-modifier act="{&quot;type&quot;: &quot;QUERY_SEARCH_INVENTOR&quot;, &quot;inventor&quot;: &quot;$inventor&quot;}" class="style-scope patent-result" data-inventor="Lorne Tyrrell"><a id="link" href="#" class="style-scope state-modifier">Lorne Tyrrell</a></state-modifier></dd><dd class="style-scope patent-result"><state-modifier act="{&quot;type&quot;: &quot;QUERY_SEARCH_INVENTOR&quot;, &quot;inventor&quot;: &quot;$inventor&quot;}" class="style-scope patent-result" data-inventor="Dakun Wang"><a id="link" href="#" class="style-scope state-modifier">Dakun Wang</a></state-modifier></dd><template is="dom-repeat" strip-whitespace="" class="style-scope patent-result"></template>
                  <template is="dom-if" class="style-scope patent-result"></template>

                  
                  <dt class="style-scope patent-result">
                    <span class="tooltip-hint style-scope patent-result" id="assigneeWarning">Current Assignee</span>
                    <overlay-tooltip positioning-target="#assigneeWarning" direction="bottom" class="style-scope patent-result" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      
                      The listed assignees may be inaccurate.<br class="style-scope patent-result">
                      Google has not performed a legal analysis and makes no representation or warranty as to the accuracy of the list.
                    
    </div>
  </overlay-tooltip>
                  </dt>
                  <dd class="style-scope patent-result">
                    
    Akshaya Bio Inc
  
                  </dd><template is="dom-repeat" strip-whitespace="" class="style-scope patent-result"></template>
                  <template is="dom-if" class="style-scope patent-result"></template>
                </dl>

                <hr class="style-scope patent-result">

                <application-timeline class="style-scope patent-result">
    
    
    <div class="wrap style-scope application-timeline">
      <div class="header style-scope application-timeline">Worldwide applications</div>
      <div class="event style-scope application-timeline">
        
          <span class="block style-scope application-timeline">
            <span class="year style-scope application-timeline">2013</span>
            <span class="dot style-scope application-timeline"></span>
            
              <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope application-timeline" data-result="patent/AU2013204668A1/en"><a id="link" href="#" class="style-scope state-modifier">
                <span id="cc" class="not_active style-scope application-timeline" this="">AU</span>
              </a></state-modifier>
              <overlay-tooltip positioning-target="#cc" direction="bottom" class="style-scope application-timeline" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      
                <div class="style-scope application-timeline">Application number: AU2013204668A</div>
                <div class="style-scope application-timeline">Filing date: 2013-04-12</div>
                <div class="style-scope application-timeline">Legal status: Abandoned</div>
              
    </div>
  </overlay-tooltip>
            <template is="dom-repeat" class="style-scope application-timeline"></template>
          </span>
        <template is="dom-repeat" class="style-scope application-timeline"></template>
      </div>
      <hr class="style-scope application-timeline">
      <div class="header style-scope application-timeline">Application AU2013204668A events <iron-icon id="legal" icon="icons:help-outline" class="style-scope application-timeline x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M11 18h2v-2h-2v2zm1-16C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8zm0-14c-2.21 0-4 1.79-4 4h2c0-1.1.9-2 2-2s2 .9 2 2c0 2-3 1.75-3 5h2c0-2.25 3-2.5 3-5 0-2.21-1.79-4-4-4z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
        <overlay-tooltip positioning-target="#legal" direction="right" class="style-scope application-timeline" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      
          <div style="max-width: 300px;" class="style-scope application-timeline">
            A timeline of key events for this patent application, including priority claims, publications, legal status, reassignments, and litigation.<br class="style-scope application-timeline"><br class="style-scope application-timeline">
            Google has not performed a legal analysis and makes no representation as to the accuracy or completeness of the events listed.
          </div>
        
    </div>
  </overlay-tooltip>
      </div>
      <div class="event layout horizontal style-scope application-timeline"><div date="" class="external-priority style-scope application-timeline">2012-02-21</div><div class="line style-scope application-timeline"></div><div class="flex title style-scope application-timeline"><template is="dom-if" class="style-scope application-timeline"></template><state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope application-timeline" data-result="patent/AU2012200998B2/en"><a id="link" href="#" class="style-scope state-modifier"><span class="title-text style-scope application-timeline">Priority claimed from AU2012200998A</span></a></state-modifier><template is="dom-if" class="style-scope application-timeline"></template><template is="dom-if" class="style-scope application-timeline"></template><template is="dom-if" class="style-scope application-timeline"></template><template is="dom-if" class="style-scope application-timeline"></template></div></div><div class="event layout horizontal style-scope application-timeline" critical=""><div date="" class="filed style-scope application-timeline">2013-04-12</div><div class="line style-scope application-timeline"></div><div class="flex title style-scope application-timeline"><template is="dom-if" class="style-scope application-timeline"></template><template is="dom-if" class="style-scope application-timeline"></template><state-modifier act="{&quot;type&quot;: &quot;QUERY_SEARCH_ASSIGNEE&quot;, &quot;assignee&quot;: &quot;$assignee&quot;}" class="style-scope application-timeline" data-assignee="Akshaya Bio Inc"><a id="link" href="#" class="style-scope state-modifier"><span class="title-text style-scope application-timeline">Application filed by Akshaya Bio Inc</span></a></state-modifier><template is="dom-if" class="style-scope application-timeline"></template><template is="dom-if" class="style-scope application-timeline"></template><template is="dom-if" class="style-scope application-timeline"></template></div></div><div class="event layout horizontal style-scope application-timeline" critical=""><div date="" class="publication style-scope application-timeline">2013-05-09</div><div class="line style-scope application-timeline"></div><div class="flex title style-scope application-timeline"><template is="dom-if" class="style-scope application-timeline"></template><state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope application-timeline" data-result="patent/AU2013204668A1/en"><a id="link" href="#" class="style-scope state-modifier"><span class="title-text style-scope application-timeline">Publication of AU2013204668A1</span></a></state-modifier><template is="dom-if" class="style-scope application-timeline"></template><template is="dom-if" class="style-scope application-timeline"></template><template is="dom-if" class="style-scope application-timeline"></template><template is="dom-if" class="style-scope application-timeline"></template></div></div><div class="event layout horizontal style-scope application-timeline" current="" critical=""><div date="" class="legal-status style-scope application-timeline">Status</div><div class="line style-scope application-timeline"></div><div class="flex title style-scope application-timeline"><template is="dom-if" class="style-scope application-timeline"></template><template is="dom-if" class="style-scope application-timeline"></template><template is="dom-if" class="style-scope application-timeline"></template><span class="title-text style-scope application-timeline">Abandoned</span><template is="dom-if" class="style-scope application-timeline"></template><template is="dom-if" class="style-scope application-timeline"></template></div></div><template is="dom-repeat" strip-whitespace="" class="style-scope application-timeline"></template>
      <template is="dom-if" class="style-scope application-timeline"></template>
    </div>
  </application-timeline>
                <hr class="style-scope patent-result">

                <dl class="links style-scope patent-result">
                  <dt class="style-scope patent-result">Info</dt>

                  <template is="dom-if" restamp="" class="style-scope patent-result"></template>

                  <template is="dom-if" restamp="" class="style-scope patent-result"></template>

                  <template is="dom-if" restamp="" class="style-scope patent-result"></template>

                  
                  <dd class="style-scope patent-result"><a href="#legalEvents" class="style-scope patent-result">Legal events</a></dd>
                  <template is="dom-if" restamp="" class="style-scope patent-result"></template>

                  
                  <dd class="style-scope patent-result"><a href="#similarDocuments" class="style-scope patent-result">Similar documents</a></dd>
                  <template is="dom-if" restamp="" class="style-scope patent-result"></template>

                  
                  <dd class="style-scope patent-result"><a href="#relatedApplications" class="style-scope patent-result">Priority and Related Applications</a></dd>
                  <template is="dom-if" restamp="" class="style-scope patent-result"></template>

                  <dt class="style-scope patent-result">External links</dt>

                  <dd class="style-scope patent-result"><a target="_blank" class="style-scope patent-result" href="https://worldwide.espacenet.com/publicationDetails/biblio?CC=AU&amp;NR=2013204668A1&amp;KC=A1&amp;FT=D">Espacenet</a></dd><dd class="style-scope patent-result"><a target="_blank" class="style-scope patent-result" href="https://globaldossier.uspto.gov/#/result/publication/AU/2013204668/1">Global Dossier</a></dd><dd class="style-scope patent-result"><a target="_blank" class="style-scope patent-result" href="https://patents.stackexchange.com/questions/tagged/AU2013204668A1">Discuss</a></dd><template is="dom-repeat" strip-whitespace="" class="style-scope patent-result"></template>
                </dl>
              </section>
            </div>
          </div>

          <div id="text" class="horizontal layout style-scope patent-result">
            <div class="flex flex-width style-scope patent-result" marginright="">
              <section id="description" class="style-scope patent-result">
                <h3 class="layout horizontal style-scope patent-result">
                  <div class="flex style-scope patent-result">Description</div>
                  <div class="righthead style-scope patent-result" hidden="">translated from </div>
                </h3>
                <patent-text name="description" id="descriptionText" class="style-scope patent-result" style="padding-top: 0px;">
    
    
    <template is="dom-if" restamp="" class="style-scope patent-text"></template>
    <overlay-tooltip id="tooltip" class="style-scope patent-text" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      
    </div>
  </overlay-tooltip>
    <div class="layout horizontal style-scope patent-text">
      <section id="text" class="flex style-scope patent-text"><div mxw-id="PDES121556940" lang="EN" load-source="patent-office" class="description style-scope patent-text">
    <div class="description-paragraph style-scope patent-text">AUSTRALIA Regulation 3.2 Patents Act 1990 Complete Specification Standard Patent APPLICANT: Akshaya Bio Inc. Invention Title: CHIMERIC ANTIGENS FOR BREAKING HOST TOLERANCE The following statement is a full description of this invention, including the best method of performing it known to me: CHIMERIC ANTIGENS FOR BREAKING HOST TOLERANCE TO FOREIGN ANTIGENS I. INTRODUCTION A. Related Applications This application is a divisional of Australian Patent Application No. 2012200998, which is a divisional of Australian Patent Application No. 2004263561, which claims the 5 benefit of U.S. Provisional Application No. 60/493,449, filed August 8, 2003, which is herein incorporated by reference. B. Technical Field The present invention relates to methods and compositions for eliciting or enhancing an immune response and for breaking host tolerance to foreign antigens. 10 The present invention relates to chimeric antigens (e.g., fusion proteins) for targeting and activating antigen presenting cells (APCs) to elicit cellular and humoral immune responses. In particular, the invention describes compositions and methods that contain or use one or more chimeric antigens that contain one or more pre-selected Hepatitis C Virus (HCV) antigen(s), and an immunoglobulin fragment, wherein the 15 chimeric antigen is capable of binding and activating APCs, especially dendritic cells, which process amd perform antigen presentation to elicit cellular and humoral immune responses. C. Background 20 Any discussion of the prior art throughout the specification should in no way be considered as an admission that such prior art is widely known or forms part of common general knowledge in the field. When a healthy host (human or animal) encounters a foreign antigen, such as a protein derived from a bacterium, virus and/or parasite, the host normally initiates an 25 immune response. This immune response can be a humoral response and/or a cellular response. In the humoral response, antibodies are produced by B cells and are secreted into the blood and/or lymph in response to an antigenic stimulus. The antibody then 1 neutralizes the antigen, e.g. a virus, by binding specifically to antigens on its surface, marking it for destruction by phagocytotic cells and or complement-mediated mechanisms, or by blocking binding or by enhancing clearance of free antigen from circulation. The cellular response is characterized by the selection and expansion of 5 specific helper and cytotoxic T-lymphocytes capable of directly or indirectly eliminating the cells that contain the antigen. In some individuals, the immune system fails to respond to certain foreign antigens. When an antigen does not stimulate the production of specific antibodies and/or killer T cells, the immune system is unable to prevent the resultant disease. Thus, the 10 infectious agent, e.g. a viras, can establish a chronic infection and the host immune system becomes tolerant to the antigens produced by the infectious agent. While the mechanism by which the infectious agent evades the host immune machinery is not clearly established, the lack of proper presentation of foreign antigens to the host immune system may be a contributing factor to development of chronic 15 infections. Antigen presenting cells (APCs) process the encountered antigens differently depending on the localization of the antigen. Exogenous antigens are endocytosed and subsequently processed within the endosomes of the antigen presenting cell. The peptide fragments generated from exogenous antigens are presented on the surface of the cell complexed with Major Histocompatibility Complex (MHC) Class II. The presentation of 20 this complex to CD4+ T cells stimulates the CD4+ T helper cells to secrete cytokines that stimulate B cells to produce antibodies against the exogenous antigen (humoral response). Intracellular antigens, on the other hand, are processed and presented as complexes with MHC Class I on the surface of antigen presenting cells. Antigen presentation to CD8+ T cells results in a cytotoxic T cell (CTL) immune response against 25 host cells that carry the antigen. In subjects with chronic viral or parasitic infections (where the organism is resident inside a host cell at some point during its life cycle), antigens will be produced by and expressed in the host cell and secreted antigens will be present in the circulation. As an example, in the case of a chronic human hepatitis B virus (HBN) carrier, virions 30 and the HBV surface antigens and a surrogate of core antigens (in the form of the e antigen) can be detected in the blood. 2 An effective therapy for a chronic infection requires a strong CTL response against antigens associated with the infectious agent. This can be achieved either by producing the antigen within the host cell, or delivering the antigen to the appropriate cellular compartment so that it gets processed and presented so as to elicit a cellular 5 response. Several approaches have been documented in the literature to deliver an antigen intracellularly. Among these, viral vectors (Lorenz et ab, Hum. Gen. Ther. 10:623-631 (1999)), the use of cDNA-transfected cells (Donnelly et ab, Ann. Rev. Immunol. 15:617(1997)) as well as the expression of the antigen through injected cDNA vectors (Lai et ab, Crit. Rev. Immunol. 18:449-484 (1988); and US Patent No. 5,589,466), have 10 been documented. Further, DNA vaccines expressing antigens targeted to dendritic cells have been described (You, et ab, Cancer Res 61:3704-3711 (2001)). Delivery vehicles capable of carrying the antigens to the cytosolic compartment of the cell for MHC Class I pathway processing have also been used. Hilgers, et al. (Vaccine 17: 219-228 (1999)) have described in detail the use of adjuvants to achieve the 15 same goal. Another approach is the use of biodegradable microspheres for cytoplasmic delivery of antigens, exemplified by the generation of a Thl immune response against ovalbumin peptide (Newman, et ab, J Control Release 54:49-59 (1998); and Newman, et ab, J Biomed Mater Res 50:591-597 (2000)). Additionally, antigen presenting cells, e.g., dendritic cells, take up PLGA nanospheres (Newman, et ab, J Biomed Mater Res 60:480 20 486 (2002)). The ability of dendritic cells to capture, process, and present the antigen and to stimulate nalve T cells has made them very important tools for therapeutic vaccine development (Laupeze, et ab, Hum Immunol 60:591-597 (1999)). Targeting of the antigen to the dendritic cells is a crucial step in antigen presentation and the presence of 25 several receptors on dendritic cells for the Fc region of antibodies have been exploited for this purpose (Regnault, et ab, J Exp Med 189:371-380 (1999)). Additional examples of this approach include ovarian cancer Mab-B43.13, Anti-PSA antibody as well as Anti HBV antibody antigen complexes (Wen, et ab, Int Rev Immunol 18:251-258 (1999)). Cancer immunotherapy using dendritic cells loaded with tumor associated antigens has 30 been shown to produce tumor-specific immune responses and anti-tumor activity (Fong and Engleman, Ann Rev Immunol 96:1865-1972 (2000); and Campton, et al. J Invest 3 Dermatol 115:57-61 (2000)). Promising results were obtained in clinical trials in vivo using tumor-antigen-pulsed dendritic cells (Tarte and Klein, Leukemia 13:653-663 (1999)). These studies clearly demonstrate the efficacy of using dendritic cells to generate immune responses against cancer antigens. 5 Antigen presentation can also be affected via mannose receptors, in place of, or in addition to, utilizing the Fc receptor on antigen presenting cells. The macrophage mannose receptor (MMR), also known as CD206, is expressed on antigen presenting cells such as dendritic cells (DC). This molecule is a member of the C-type lectin family of endocytic receptors. Mannosylated antigens can be bound and internalized by CD206. 10 In general, exogenous antigen is thought to be processed and presented primarily through the MHC class II pathway. However, in the case of targeting through CD206, there is evidence that both the MHC class I and class II pathways are involved (Apostolopoulos et ab, Eur. J. Immunol. 30:1714 (200Q); Apostolopoulos and McKenzie, Curr. Mol. Med. 1:469 (2001); Ramakrishna et ab, J. Immunol. 172:2845-2852 (2004)). 15 Infectious disease and cancer are major public healthcare issues. For example, World Health Organization statistics show that more than 2 billion people have been infected by HBN. Among these, 370 million are chronically infected and, as a result, have a high probability of developing cirrhosis of the liver and hepatocellular carcinoma. Approximately 170 million people worldwide are chronic carriers of HCN, for which 20 there is no effective prophylactic or therapeutic vaccine. The World Health Organization reports that 10 million people are diagnosed with cancer every year. Cancer causes 6 million deaths every year, 12% of deaths worldwide. Thus a need exists for new, therapeutically effective compositions and methods for the eliciting immune responses against infections and cancer, as well as new methods for producing such compositions. 25 More than 170 million people worldwide are chronic carriers of HCV [Delwaide et al. (2000) Rev. Med. Liege 55:337-340] . There is neither a prophylactic nor a therapeutic vaccine currently available for HCV. The route of infection is via blood and other body fluids and over 70% of patients become chronic carriers of the virus. Persistent infection results in chronic active hepatitis which may lead to progressive liver 30 disease [Alter et al. (1999) N. Engl. J. Med. 341:556-562]. Presently, the only therapy for hepatitis C infection is interferon-I (IFN-I) and Ribavirin. However, this therapy is 4 expensive, has substantial side effects, and is effective in only approximately 50% of a selected group of patients. Therapeutic vaccines that enhance host immune responses to eliminate chronic HCV infection will be a major advancement in the treatment of this disease. 5 The immune system plays a key role in the outcome of an HCV infection. Most individuals that are exposed to HCV mount a broad strong and multi-antigen-specific CD4+ (regulatory) and CD8+ (cytotoxic) T cell response to the virus. These individuals develop only a self-limited infection. However, in some individuals exposed to HCV, a weak or undetectable and narrowly focused immune response results in chronic infection. 10 HCV is a member of the flaviviridae family of RNA viruses. The HCV genome is a positive sense single stranded RNA molecule of approximately 9.5Kb that encodes a single polyprotein which is cleaved into individual proteins catalyzed by host and viral proteases to produce three structural proteins (core, El, E2), p7 protein and 6 non structural proteins (NS2, NS3, NS4A, NS4B, NS5A, NS5B) [Hijikkata et al. (1991) Proc. 15 Natl. Acad. Sci. USA 88:5547-5551]. The NS3 protein is the viral serine protease involved in the proteolytic processing of the non-structural proteins [Bartenschlager et al. (1993) J. Virol. 67: 3835-3844]. The mechanism by which the virus evades the host immune machinery is not clearly established [Shoukry et al. (2004) Ann. Rev. Microbiol. 58:391-424]. Several 20 HCV proteins have been implicated in the immune evasion mechanism. These include: NS5A, suggested to induce the production of IL-8 which inhibits the IFN-induced antiviral response [Polyak at al. (2001) J. Virol. 75: 6095-6106] and to inhibit the cellular IFN-y-induced PKR protein kinase, thus inhibiting antiviral immune responses [Tan et al. (2001) Virol 284: 1-12]; Core and NS3, suggested to inhibit DC differentiation 25 [Dolganiuc et al. (2003) J. Immunol. 170:5615-5624]; and Core and El [Sarobe et al. (2002) J. Virol. 77:10862-10871; and Grakoui et al. (2003) Science 302:659-662] suggested to modulate T cell responses by modulating DC maturation; and finally the lack of memory T cell help [Shoji et al. (1999) Virology 254: 315-323]. Malaria is caused by apicomplexan protozoan parasites of the Plasmodium genus 30 which are endemic in 108 countries; with 3 billion people being at risk of infection(Crawley et al., 2010). There are currently an estimated 200-500 million cases 5 of malaria worldwide with 80% of all cases occurring in sub-saharan Africa(Rogerson et al., 2010). Malaria kills between one and three million people annually with 91% of deaths occurring in sub-saharan Africa (Crawley et al., 2010; Rogerson et al., 2010; Tuteja, 2007). Pregnant women and children under the age of five are at highest risk of 5 succumbing to malaria with 80% of sub-saharan deaths occurring in children of this age group (Crawley et al., 2010; Rogerson et al., 2010; Travassos and Laufer, 2009; Tuteja, 2007). Five species of Plasmodium parasites can infect humans (Pflaciparum, P vivax, P ovale, P malariae and P knowlesi)(Crawley et al., 2010). The most common and 10 deadliest species infecting humans is Plasmodiumfalciparum(Crawley et al., 2010). These five species differ morphologically, immunologically, geographically, in their relapse pattern and drug responses(Tuteja, 2007). The Plasmodium parasite has a complex life cycle, reproducing asexually in humans and sexually in mosquitos of the genus Anopheles(Tuteja, 2007). When an infected mosquito takes a blood meal, 15 infectious malaria sporozoites ride into the bloodstream with the mosquito's saliva and migrate to the liver, where they infect and reproduce asexually inside hepatocytes for 9 16 days(Tuteja, 2007). The receptors on sporozoites responsible for hepatocyte invasion are mainly the thrombospondin domains on the circumsporozoite protein (CSP) and on thrombospondin-related proteins(Tuteja, 2007). One infected, hepatocyte can give rise to 20 tens of thousands of mature merozoites which burst out of the liver cells and migrate back into the blood, where each merozoite can infect a red blood cell(Tuteja, 2007). Inside erythrocytes, merozoites feed on the cytoplasm and evade the immune system while they reproduce asexually once again eventually killing the erythrocyte. Every infected erythrocyte bursts, giving rise to 20 more merozoites which invade fresh 25 erythrocytes, perpetuating and exacerbating the cycle of infection(Tuteja, 2007). Each successive round of erythrocyte lysis takes 48h in Pfalciparum and causes release of TNF and other cytokines which trigger sharp increases in body temperature and other symptoms such as shivering, cough, joint pain, headache, diarrhea, vomiting and convulsions(Tuteja, 2007). Loss of erythrocytes by lysis can lead to severe anemia. As 30 infected erythrocytes adhere to blood vessel walls and obstruct blood flow in various organs, more severe symptoms such as respiratory distress, retinopathy, kidney failure, as 6 well as placental and cerebral malaria can develop(Crawley et al., 2010; Tuteja, 2007). These symptoms can result in death if the patient cannot develop strong immune responses to contain and clear the infection. There is currently no vaccine available for malaria(Tuteja, 2007). Current 5 preventive measures include vector control with insecticide regimens and use of bed netting(Tuteja, 2007), whereas infected patients are given anti malarial drugs to kill the parasite. Historically, Quinine from the bark of the Cinchona tree was widely used as an anti-malarial treatment(Tuteja, 2007). Unfortunately, Plasmodium parasites have become resistant to these drugs and in many parts of the world, Quinine and its derivatives such 10 as chloroquine are no longer effective(Travassos and Laufer, 2009; Tuteja, 2007). To overcome this problem, combination therapy with several drugs such as sulfadoxine pyrimethamine (SP), Fansidar, mefloquine, lumefantrine, artemisinin or its derivative artesunate must be used to effectively kill the parasites(Crawley et al., 2010; Travassos and Laufer, 2009; Tuteja, 2007). In addition to parasite resistance, the cost of production 15 of drugs such as artemisinin is often prohibitive to treatment in endemic areas, highlighting the need for vaccine therapies. Traditional approaches to vaccine development have focused on adjuvanted antigens or attenuated pathogens (Barbosa et al., 2009; Casares et al., 2010; Clyde et al., 1973; Thera et al., 2010). In the case of malaria, these approaches have resulted in very 20 limited success due either to immune evasion by the parasite or ineffective vaccine activation of dendritic cells (DC) and insufficient stimulation of either humoral or cell mediated immunity(Barbosa et al., 2009; Casares et al., 2010; Crawley et al., 2010). Effective protective immune responses against malaria infection involve both the generation of neutralizing antibodies to sporozoite and merozoite forms of the parasite in 25 blood, as well as cell-mediated responses to clear infected liver cells. Interest in the development of multi-antigen vaccines that target both stages of the life cycle are now increasing(Crawley et al., 2010). Several parasite-specific proteins have recently emerged as potential candidates for such multi-antigen vaccines. These include CSP, liver stage antigen-I (LSA-1), apical membrane antigen-I (AMA-1) and merozoite 30 surface protein-I (MSP-1)(Arnot et al., 2008; Bannister et al., 2003; Barale et al., 1999; Barbosa et al., 2009; Bargieri et al., 2010; Casares et al., 2010; Epp et al., 2003; Fidock, 7 1994; Heppner et al., 2005; Hillier et al., 2005; Lau et al., 2010; Pan et al., 2004; Plassmeyer et al., 2009; Thera et al., 2010; Tuteja, 2007). In some of these studies, these proteins were shown to induce strong IgG responses in several studies and two of them, CSP and AMA-1, have been engineered into adjuvanted vaccines by Glaxo Smith Kline 5 in collaboration with PATH MVI, the Howard Hughes Medical Institute and the Walter Reed Army Institute of Research. These two vaccines (GSK RTS,S/ASO1 and FMP2. 1/ASO2) are currently undergoing clinical trials in Africa(Barbosa et al., 2009; Casares et al., 2010; Cohen et al., 2010; Thera et al., 2010). However, early evidence has shown that although these vaccines induce IgG responses against the cognate antigen, 10 they do not induce strong cell-mediated immune responses and only offer partial and transient protection in a limited subset of the population (Barbosa et al., 2009; Casares et al., 2010; Crawley et al., 2010; Thera et al., 2010). A recent report (Olotu et al., 2013) showed that the vaccine has very limited long term outcome in protecting the vaccinated population, and the efficacy of the vaccine disappeared by year 4 after the three dose 15 vaccination with RTS,S vaccine (GSK). When a healthy host (human or animal) encounters a foreign antigen (such as proteins derived from a bacterium, virus and/or parasite), the host normally initiates an immune response. This immune response may be humoral, cellular or both. In the case of the humoral response, antibodies are produced by B cells and are secreted into the blood 20 and/or lymph in response to an antigenic stimulus(Whitton et al., 2004). The antibodies neutralize the antigen, (e.g. a virus) by binding specifically to antigens on its surface, marking it for destruction by phagocytotic cells and/or complement-mediated mechanisms to lyse the infected cells. The cellular response is characterized by the selection and expansion of specific helper and cytotoxic T lymphocytes (CTLs) capable 25 of eliminating the infected cells through direct or indirect pathways. The direct pathway involves target cell apoptosis mediated by granzyme B (GrB), a serine protease released from CTLs. The indirect pathway is mediated by cytokines such as IFN y. In many individuals, the immune system does not respond to certain specific antigens and therefore these antigens do not stimulate the production of a specific 30 antibody and/or CTL response. The immune system becomes tolerant to the antigens produced by the virus and as a consequence, the host's immune system is unable to 8 abrogate the resultant disease process. In this instance, the infectious agent, such as a virus, is able to establish a chronic infectious state. The mechanism(s) by which the virus evades the host immune machinery vary widely. Some examples include: interference with antigen presentation by modulating major histocompatibility complex (MHC) or co 5 stimulatory molecule expression on antigen presenting cells; exhaustion of T-cells due to persistent exposure to viral antigens leading to impairment of functions such as cytokine production, cytotoxicity and proliferation; T-cell deletion; mutation of viral epitopes; expression of immunosuppressive cytokines (IL-10, TGF-); regulatory T-cells (Treg); and over expression of cell surface inhibitory receptor molecules such as PD-i and 10 CTLA-4(Ha et al., 2008). Some of the best-known examples of chronic infections are hepatitis (hepatitis B and hepatitis C viruses), acquired immune deficiency disease or AIDS (human immunodeficiency virus), human papilloma virus infection, cold sores and genital herpes (herpes simplex viruses). In the examples above, the virus escapes attack by the host immune system 15 because the antigen-presenting cells recognize the viral antigens as "self," rather than "non-self' and are subsequently tolerated by the host. The mechanism used by Plasmodium to avoid immune surveillance is not well understood, but is likely to involve similar evasion mechanisms (Wykes et al., 2007). The rapid increase in parasite load and the resulting increase in the concentration of parasite antigens may make the immune 20 system tolerant to the parasite. Many mechanisms may contribute to the development of tolerance and suppression of effector T cell responses by CD4+CD25+ Treg cells play may be an important component of the development of tolerance to Pfalciparum. Recently the role of Tregs in viral infectious diseases has attracted a lot of attention. It has been demonstrated that Tregs not only play a role in the regulation of immune 25 responses but also exert suppressive function of immune responses which may contribute to the persistent infection in both viral and parasitic disease(Belkaid and Rouse, 2005). One study reported that suppressive function of Tregs can be enhanced by virus infection (Suvas et al., 2003). A number of viral studies have shown that Tregs in PBMCs isolated from HBV-infected patients inhibited the HBV-specific immune response (Franzese et 30 al., 2005; Peng et al., 2008; Stoop et al., 2005). 9 The major participant in the antigen presentationi process is the dendritic cell (DC), which captures and processes the antigens (Steinman et al., 1999; Wykes et al., 2007). In addition, DCs express lymphocyte co-stimulatory molecules and migrate to lymphoid organs where they secrete cytokines to initiate immune responses. DCs also 5 control the proliferation of B and T lymphocytes which are the mediators of adaptive immunity (Steinman et al., 1999). The generation of a CTL response is critical in the elimination of parasite-infected hepatocytes to eliminate the infection. Antigen presenting cells (APC) process antigens differently depending on the localization of the antigen (Steinman et al., 1999). Exogenous antigens are taken up by 10 phagocytosis and processed within the endosomes of the APC. This process generates peptide fragments which are loaded onto MHC class II and presented on the surface of the APC. The presentation of this complex to CD4+ T cells stimulates their clonal expansion and mediator release. As a result, cytokines secreted by the CD4+ T cells stimulate B cells to produce antibodies against the exogenous antigen (humoral 15 response). Immunizations using antigens typically generate antibody response through this endosomal antigen processing pathway. However, in addition to MHC class II, some exogenous antigens can gain access to the MHC class I pathway of APC by so called "cross-presentation" through the receptors, such as mannose receptors and Fcy receptors using the endosome-to-cytosol pathway (Amigorena, 2002; Burgdorf et al., 2007; Moron 20 et al., 2003). On the other hand, intracellular antigens are processed in the proteasome and the resulting peptide fragments are presented as complexes with MHC class I on the surface of APCs. Presentation of the antigen loaded MHC class I to CD8+ T cells activates the CTL response which removes the infected host cells by apoptosis in a granzyme B-dependent process. 25 The immunology of Pfalciparum malaria is complex and incompletely understood. It has been shown that people living in malaria endemic countries develop immunity by repeated exposure, but that development of full immunity can take decades (Kappe et al., 2010; Roussilhon et al., 2010). Central to the development of immunity is the DC, which captures, processes and presents antigen to CD4+ and CD8+ T-cells, 30 activating antibody production and cell-mediated immunity respectively (Wykes et al., 2007). In malaria, reduced T cell proliferation and IFN- y responses to parasite antigens 10 have been described (Bejon et al., 2007; Plebanski et al., 1999). Parasitic load also correlates strongly with circulating levels of Treg cells (Bueno et al., 2010). In mouse models of malaria, one study has shown that C57BL/6 mice, which have lower absolute numbers of Treg cells than BALB/c mice, had more vigorous CD4+ and CD8+ immune 5 responses to the parasite and succumbed to cerebral malaria, which did not occur in BALB/c mice (Wu et al., 2010). Taken together, these results suggest that the attenuation of the CD4+CD25+ Treg-dependent tolerogenic response might be a potential therapeutic strategy for treatment of malaria infected patients. The lack of proper presentation of the appropriate Plasmodium antigen to the host immune system may also be a contributing 10 factor. It has been shown that infected erythrocytes adhere to human DC and inhibit their maturation and subsequent T cell activation (Urban et al., 1999; Urban et al., 2001). Furthermore, it has been shown that DC cross-presentation to MHC class I is impaired in a P berghei mouse model, and that DC fail to mature in a Pyoelii mouse model (Ocana Morgner et al., 2003; Wilson et al., 2006). Lack of cross-presentation or delay in DC 15 maturation could be a strategy employed by the parasite to ensure survival through the liver stage and progression to the blood stage of infection (Ocana-Morgner et al., 2003; Wykes and Good, 2008; Zheng et al., 2009). This theory could partially explain the finding that prolonged antigen exposure is required for optimal CD8+ immune responses to malaria (Cockburn et al., 2010). The success in eliminating the parasite will result 20 from the manner in which the antigen is processed and presented by the APC, the involvement of CTL and down-regulation of the Treg. All of these parameters are essential in vaccine development. In order to induce a strong CTL response, a therapeutic vaccine must be processed through the proteasomal pathway and presented via MHC class I (Larsson et al., 2001). 25 This can be achieved either by producing the antigen within the host cell, or it can be delivered to the appropriate cellular compartment so that it is processed and presented in a manner such that it will elicit the desired cellular response. Several approaches have been documented in the literature for the intracellular delivery of the antigen. Among these, viral vectors (Lorenz et al., 1999), the use of DNA-transfected cells (Donnelly et 30 al., 1997) as well as the expression of the antigen through injected DNA vectors (Lai and Bennett, 1998) (US patent No. 5,589,466), have been documented. 11 Delivery vehicles capable of carrying the antigens to the cytosolic compartment of the cell for MHC class I pathway processing have also been used. The use of adjuvants to achieve the same goal has been described in detail (Hilgers et al., 1999). Another approach is the use of biodegradable microspheres in the cytoplasmic delivery of 5 antigens(Newman et al., 2000), exemplified by the generation of a Th1 immune response against ovalbumin peptide (Newman et al., 2000; Newman et al., 1998). It has also been shown that poly(D,L-lactic-co-glycolic acid) (PLGA) nanospheres are taken up by DCs (Newman et al., 2002). By virtue of their capability as professional APC, DCs, which are derived from 10 monocytes, have been shown to have great potential as immune modulators that stimulate primary T cell responses (Banchereau and Steinman, 1998; Steinman et al., 1999). This unique property of the DCs to capture, process, and present the antigen to stimulate naive T cells has made them very important tools for therapeutic vaccine development (Laupeze et al., 1999). 15 Targeting of the antigen to the DCs is the crucial step in antigen presentation and the presence of several receptors on the DCs for the Fc region of monoclonal antibodies have been exploited for this purpose (Regnault et al., 1999). Examples of this approach include ovarian cancer mAb-B43.13(Noujaim et al., 2001), anti-PSA antibody as well as anti-HBV antibody antigen complexes (Wen et al., 1999). Cancer immunotherapy using 20 DCs loaded with tumor-associated antigens have been shown to produce tumor-specific immune responses and anti-tumor activity (Campton et al., 2000; Fong and Engleman, 2000). Promising results were obtained in clinical trials in vivo using tumor-antigen pulsed DCs (Tarte and Klein, 1999). These studies clearly demonstrate the efficacy of using DCs to generate immune responses against cancer antigens. 25 Human immunodeficiency virus (HIV) type 1, the causative agent of acquired immune deficiency syndrome (AIDS), has infected more than 33 million people worldwide and an estimated 2.7 million people are newly infected every year. The current treatment for HIV infection, HAART (Highly Active Antiretroviral Therapy), is effective in prolonging the life-span of chronically infected individuals. However, 30 persistent reservoirs of HIV remain even after effective and continuous HAART. In 12 addition, although millions of people have received HAART, many millions more do not have access to these drug therapies. The latest statistics of the global HIV and AIDS epidemic were published by UNAIDS in November 2010, and refer to the end of 2009. According to this report, there 5 are more than 33 Million people living with HIV/AIDS in the world, among these, more than 30 million are adults, 16 million women, 2.5 million children, and there are more than 16 million orphans (0-17 years) due to AIDS in 2009. In 2009 there were 2.6 million new infections out of which 2.2 million are adults. There were 1.8 million deaths due to HIV/AIDS in 2009. At the end of 2009, women accounted for just over half of all adults 10 living with HIV worldwide. With around 68 percent of all people living with HIV residing in sub-Saharan Africa, the region carries the greatest burden of the epidemic. Epidemics in Asia have remained relatively stable and are still largely concentrated among high-risk groups. Conversely, the number of people living with HIV in Eastern Europe and Central Asia 15 has almost tripled since 2000. According to Public Health Agency of Canada HIV/AIDS Epi Update July 2010, there are more than 65,000 Canadians living with HIV infection at the end of 2008, compared to 57,000 at the end of 2005. It is estimated that approximately 4000 new infections occur in Canada every year. 20 Therefore, the search for an HIV vaccine is among the highest of health priorities and thus a major unmet medical need. At present, there is neither a prophylactic nor a therapeutic vaccine available for HIV infection. This is mainly due to the nature of the infection by the HIV and the destruction of the immune machinery by the virus. The approaches using HAART 25 antiviral agents have only been moderately successful in increasing the lifespan of the infected individuals and in improving the quality of their lives. Due to lack of access and cost, millions do not receive HAART. Competition in this field has been focusing on developing adjuvant -supported vaccine technologies and viral vectored vaccines, but have not produced substantially 30 convincing progress. 13 II. SUMMARY OF THE INVENTION The invention provides chimeric antigens for eliciting an immune response, the chimeric antigens comprising an immune response domain and a target binding domain, wherein the target binding domain comprises an antibody fragment. 5 Another aspect of the invention provides methods of enhancing antigen presentation in an antigen presenting cell comprising contacting the antigen presenting cell with a composition comprising a chimeric antigen of the invention. Yet another aspect of the invention provides methods of activating antigen presenting cells comprising contacting the antigen presenting cells* with a chimeric 10 antigen of the invention. An aspect of the invention provides methods of eliciting an immune response comprising administering to a subject, a composition comprising a chimeric antigen of the invention. Another aspect of the invention provides methods of breaking tolerance 15 comprising administering a chimeric antigen of the invention to a subject. In a preferred embodiment, the subject is chronically infected with a virus or an obligate intracellular parasite. One aspect of the invention provides methods of treating an immune-treatable condition comprising administering, to a subject in need thereob a therapeutically 20 effective amount of a chimeric antigen of the invention. In a preferred embodiment, the immune-treatable condition is an infection, especially a chronic infection, or a cancer. Yet another aspect of the invention provides methods of vaccinating a subject against an infection comprising administering a chimeric antigen of the present invention to the subject. The subject can be prophylactically or therapeutically vaccinated. In a preferred 25 embodiment, the subject develops an immune response to more than one epitope of the chimeric antigen, and more preferably to more than one epitope of the immune response domain Preferably, the infection is a viral infection or an obligate intracellular parasitic infection. Another aspect of the invention provides a pharmaceutical composition 30 comprising a chimeric antigen of the present invention and a pharmaceutically acceptable excipient. 14 An aspect of the invention provides articles of manufacture comprising a chimeric antigen of the invention and instructions for administering the chimeric antigen to. a subject in need thereof Another aspect of the invention provides polynucleotides encoding a chimeric 5 antigen, said polynucleotide comprising a first polynucleotide portion encoding an immune response domain and a second polynucleotide portion encoding a target binding domain, wherein the target binding domain comprises an antibody fragment. The invention also provides microorganisms and cell lines comprising such polynucleotides Yet another aspect of the invention provides methods of producing a chimeric antigen of 10 the invention comprising providing a microorganism or cell line, which comprises a polynucleotide that encodes a chimeric antigen of the invention, and culturing the microorganism or cell line under conditions whereby the chimeric antigen is expressed. The details of one or more embodiments of the invention are set forth in the accompanying drawings and the description below. Other features, objects, and 15 advantages of the invention will be apparent from the description and drawings, and from the claims. The present invention pertains to compositions and methods for targeting and activating APCs, one of the first steps in eliciting an immune response. The compositions of the present invention include a novel class of molecules (hereinafter designated as 20 "chimeric antigens") that include an immune response domain (IRD), for example a recombinant protein, linked to a target binding domain (TBD), for example, an antibody fragment portion. More specifically, the chimeric antigens are molecules that couple viral antigens, such as Hepatitis C Core, envelope proteins such El and E2, or non-structural proteins, to an immunoglobulin fragment, such as a murine immunoglobulin G Fc 25 fragment. In some embodiments, the antibody fragment is a xenotypic antibody fragment. The compositions and methods of the present invention are useful for targeting and activating APCs. The compositions and methods of the present invention are useful for inducing cellular and/or humoral host immune responses against any viral antigen 30 associated with HCV. The invention includes therapeutic vaccines for the treatment of 15 chronic HCV infections as well as prophylactic vaccines for the prevention of HCV infections. One or more embodiments of the present invention include one or more chimeric antigens suitable for initiating an immune response against HCV. In these embodiments 5 of the invention, selected HCV antigens are linked to fragments of antibodies. The resulting chimeric antigens are capable of targeting and activating APCs, such as dendritic cells. The present invention also includes methods for cloning DNA constructs encoding fusion proteins and producing fusion proteins in a heterologous expression 10 system. In preferred embodiments of the invention, the cloning and production methods introduce unique post-translational modifications including, but not limited to glycosylation (e.g., mannosylation) of the expressed fusion proteins. In order to provide efficient presentation of the antigens, the inventors have developed a novel viral antigen-murine monoclonal antibody Fc fragment fusion protein. 15 This molecule, by virtue of the Fc fragment, is recognized at a high efficiency via specific receptors by APCs (e.g., dendritic cells), the fusion protein is processed and peptide epitopes from the viral antigen are presented as complexes with Major Histocompatibility Complex (MHC) Class I. This processing and antigen presentation results in the up-regulation of the response by cytotoxic T-lymphocytes, resulting in the 20 elimination of virus-infected cell population. In addition, due to antigen presentation by MHC Class II molecules and activation of helper T cells, a humoral response can be induced against the viral antigen that will help prevent and/or eliminate viral infection. The chimeric nature of the molecule helps to target the antigen to the proper antigen-presenting cells (e.g., dendritic cells), making it a unique approach in the therapy 25 of chronic infectious diseases by specifically targeting the APC receptors. This is useful for developing therapeutic vaccines to treat chronic Hepatitis C infections. The administration of these chimeric fusion proteins can elicit a broad immune response from the host, including both cellular and humoral responses. Thus, they can be used as therapeutic vaccines to treat subjects that are immune tolerant to a HCV 30 infection. 16 More specifically, the invention features a chimeric antigen for eliciting an immune response, the chimeric antigen containing an immune response domain and a target binding domain, the immune response domain containing a hepatitis C (HCV) antigen and the target binding domain containing an antibody fragment. The antibody fragment can be a xenotypic antibody fragment. The chimeric antigen can elicit a humoral immune response, a cellular immune response, or a both humoral immune response and a cellular immune response. In addition, the chimeric antigen can elicit a Th1 immune response, a Th2 immune response or both a Th1 and a Th2 immune response. The immune response can be an in vivo or an ex vivo immune response. The immune response domain can contain more than one protein; it can, for example, contain one or more immunogenic portions of one or more proteins that include, for example, a HCV Core (1-191) protein, a HCV Core (1-177) protein, a HCV p7 protein, a HCV El protein, a HCV E2 protein, a HCV El -E2 protein, a HCV NS3 protein, a HCV NS4B protein, or a HCV NS5A protein. The target binding domain can be capable of binding to an antigen presenting cell (APC). The antibody fragment can be a Fc fragment. The chimeric antigen can further comprise one or more of a 6xHis tag, a protease cleavage site, and a linker for linking the immune response domain and the target binding domain. The linker can be selected from leucine zippers, biotin bound to avidin, and a covalent peptide linkage. Furthermore, the chimeric antigen can be glycosylated, e.g., mannose glycosylated. The antibody fragment can include an immunoglobulin heavy chain fragment and the immunoglobulin heavy chain fragment can contain a hinge region. In addition, the immunoglobulin heavy chain fragment can contain all or a part of an antibody fragment selected from the group consisting of the CHI, the hinge region, the CH2 domain, and the CH3 domain. Another embodiment of the invention is a method of delivering an antigen to an antigen presenting cell, the method comprising administering to the antigen presenting cell any of the chimeric antigens disclosed herein. The antigen presenting cell can be a dendritic cell. The invention also provides a method of activating an antigen presenting cell; the method can involve contacting an antigen presenting cell with a any of the chimeric antigens described herein. The contacting can take place ex vivo or in vivo. It can take 17 place, for example, in a human. The method can include administering to a subject a composition comprising any of the chimeric antigens of the invention, the antigen presenting cell being in the subject. The contacting can result in a humoral immune response, a cellular immune response, or both a humoral immune response and a cellular immune response. The cellular immune response can be one or more of a Th1 response, a Th2 response, and a CTL response. The subject can have, or be likely to have, an immune-treatable condition. The immune-treatable condition can be an acute infection (e.g., an acute viral infection) or it can be a chronic infection (e.g., a chronic viral infection). The chronic infection can be a chronic hepatitis C viral infection. The immune-treatable condition can be a hepatitis C viral infection and the immune response domain can contain one or more antigenic portions of one or more proteins selected from the group consisting of a HCV Core (1-191) protein, a HCV Core (1-177) protein, a HCV El protein, a HCV E2 protein, a HCV El -E2 protein, a HCV P7 protein, a HCV NS3 protein, a HCV NS4B protein, and a HCV NS5A protein. Using the method, the subject can be vaccinated against a viral infection, e.g., prophylactically vaccinated against a viral infection or therapeutically vaccinated against an existing viral infection. Another aspect of the invention is a method of producing a chimeric antigen. The method can involve: (a) providing a microorganism or a cell, the microorganism or cell containing a polynucleotide that encodes a chimeric antigen; and (b) culturing the microorganism or cell under conditions whereby the chimeric antigen is expressed. The microorganism or cell can be a eukaryotic microorganism or cell. The cell can be a yeast cell, a plant cell or an insect cell. In addition the chimeric antigen can be post translationally modified to comprise glycosylation, e.g., it can be post-translationally modified to comprise a mannose glycosylation. Yet another embodiment of the invention is a polynucleotide encoding a chimeric antigen, the polynucleotide containing a first polynucleotide portion encoding an immune response domain and a second polynucleotide portion encoding a target binding domain, the target binding domain containing an antibody fragment. The antibody fragment can be a xenotypic antibody fragment. The polynucleotide can contain, for example, a nucleotide sequence selected from the group consisting of the nucleotide sequences set forth in SEQ ID NOs:39X and 41X-51X. Moreover, the polynucleotide can encode a 18 chimeric antigen that is at least 90% identical to an entire amino acid sequence selected from the group consisting of the amino acid sequences set forth in SEQ ID NOs:40X and 52X-62X. The polynucleotide can selectively hybridize under stringent conditions to a polynucleotide having a nucleotide sequence selected from the group consisting of nucleotide sequences set forth in SEQ ID NOs:39X and 41X-51X. The invention also provides a vector containing any of the polynucleotides disclosed herein, e.g., a vector in which the polynucleotide is operably linked to a transcriptional regulatory element (TRE). In addition, the invention embraces a microorganism or cell containing any of the polynucleotides disclosed herein. Another embodiment of the invention is an article of manufacture that can contain any of the chimeric antigens disclosed herein and instructions for administering the chimeric antigen to a subject in need thereof. Yet another aspect of the invention is a pharmaceutical composition containing any of the chimeric antigens disclosed herein and a pharmaceutically acceptable excipient. Moreover, the invention provides another method of producing a chimeric antigen. The method can involve: (a) providing a microorganism or a cell, the microorganism or cell containing a polynucleotide that encodes a target binding domain linker molecule, the target-binding domain-linker molecule containing a target binding domain bound to a linker molecule; (b) culturing the microorganism or cell under conditions whereby the target binding domain-linker molecule is expressed; and (c) contacting the target binding domain-linker molecule and an immune response domain under conditions that allow for the binding of the linker to the immune response domain, the binding resulting in a chimeric antigen. The microorganisms or cells, the polynucleotides, the target binding domains, the linker molecules, and the immune response domains can be any of those disclosed herein. Unless otherwise defined, all technical and scientific terms used herein have the same meaning as commonly understood by one of ordinary skill in the art to which this invention pertains. In case of conflict, the present document, including definitions, will control. Preferred methods and materials are described below, although methods and 5 materials similar or equivalent to those described herein can be used in the practice or 19 testing of the present invention. All publications, patent applications, patents and other references mentioned herein are incorporated by reference in their entirety. The materials, methods, and examples disclosed herein are illustrative only and not intended to be limiting. 5 Other features and advantages of the invention, e.g., chimeric antigens for treating or preventing an immune-treatable or condition, will be apparent from the following description, from the drawings and from the claims. In some embodiments, recombinant "chimeric antigens" can comprise selected antigens and specific regions of a xenotypic antibody, produced as fusion proteins. The 10 bifunctional design of the molecule can be tailored to target the antigen to the proper antigen presenting cells in order to break the tolerance to the antigen and to elicit both humoral and the most desirable cellular immune response to clear the infection. The technology can use the fusion of relevant antigen(s) with a xenotypic antibody fragment (Fc) that instructs DC to stimulate both antibody production and killing of 15 infected cells by T-helper and CTL respectively. The technology can combine the advantages of antigen and antibody-based therapies and possesses the competitive advantage of using the host immune system to generate both strong antibody and cellular immune responses against selected antigens. The chimeric antigens can have two domains: 1) an immune recognition domain (IRD), containing the antigen(s) of interest 20 (in some cases, as well as hepatitis B virus (HBV) core protein) and 2) a target binding domain (TBD), which can comprise a xenotypic Fc fragment from a monoclonal antibody (for example, a murine IgG Fc fragment). The chimeric antigen can be expressed in Sf9 insect cells. The unique features of this platform include adaptability, non-mammalian glycosylation and high immunogenicity without adjuvant. The chimeric 25 antigen can also be cheaper to manufacture in insect cells. In some embodiments, the chimeric antigen can target, bind to, and activate DC in a receptor-dependent fashion giving rise to DC-mediated antigen-specific immune response by both CD4+ and CD8+ T-cells. 20 The design of the molecule can impart several unique properties in its function. The unique chimeric design favors the formation of antibody-like structures that facilitate its uptake through specific receptors and results in appropriate antigen presentation. It can generate a broad immune response, both cellular (MHC class I) and humoral (MHC class 5 II). It can be processed through the proteasomal pathway and the peptides presented as complexes with MHC class I, resulting in a CTL response. The chimeric antigens can also be processed via the endosomal pathway, presented by MHC class II, and produce a humoral response. The TBD mediates the binding of the chimeric antigen DC Fcy receptors which results in class II presentation and class I cross-presentation. The 10 xenotypic nature of the TBD can make the molecule more immunogenic. In addition, the linker peptides of varying lengths, unrelated to the native proteins, incorporated at the N and C- termini of the antigen, and the antibody fragments in the the chimeric antigen, can help the molecule to be recognized as "foreign" by the host immune system. The expression of the recombinant proteins in insect cells can impart non 15 mammalian glycosylation, a feature that can add another dimension to the immunogenicity of the protein. Mannose/pauci mannose glycosylation introduced in insect cells also permits the targeting of mannose receptors on APCs for uptake. The chimeric antigen can be taken up by the APCs through specific Fcy receptors I, II and III (CD64, CD32, CD16), mannose receptor (CD206), other C-type lectin receptors and by 20 phagocytosis (Geijtenbeek et al., 2004). The uptake via these receptors, processing through the endosomal and proteasomal pathways and presentation on both classes of MHCs can result in a broad immune response to eliminate infected cells and remove any circulating antigen. Generation of a CTL response can be critical to clear pathogen-infected cells. In 25 the chimeric antigen technology, antibody/antigen stoichiometry can be maintained in the vaccine molecule which can reduce the challenge of optimizing antibody concentration seen in classical antibody therapy. An added advantage is that the chimeric antigen does not have to rely on circulating antigen for the presentation. A monomeric fusion protein can be schematically represented as: 21 (N-terminus) 6xHis-rTEV Protease Site-HBV Antigen ----- [IRD] ------ Linker Peptide-Part CH1-CH2-CH3-Peptide (C-terminus) [TBD] In some embodiments, the chimeric antigens can comprise Malaria Multi-antigen Vaccines incorporating 4 Plasmodiumfalciparum antigens in conjunction with the 5 hepatitis B virus (HBV) core protein. The chimeric antigens can be cloned and expressed in Sf9 insect cell using the Bac-to-Bac" baculovirus expression system (Invitrogen). The chimeric antigens can be purified and its biochemical and immunological properties can be evaluated, for example, in in vitro human DC/T cell antigen presentation assays using PBMCs from healthy donors. Referring to Figure 1Z, a schematic representation of the 10 Malaria antigen/HBV Core/TBD Vaccine in its dimerized form is depicted. The Pfalciparum antigens chosen for certain embodiments of the chimeric antigen molecule were the best candidates for eliciting both CD4+ and CD8+ responses based on current clinical trials or other studies in the literature. The most promising candidates were the CSP (RTS,S/ASO2 vaccine) and LSA-1 liver stage antigens, the AMA-1 15 (FMP2.1/ASO2 vaccine) and MSP-1 blood stage antigens. In many studies using exogenous peptide vaccines, a conformational epitope is required for recognition by T cells (Xue et al., 2010). This is not the case for this chimeric antigen technology as whole protein antigens can be incorporated into the vaccine so that antigen presenting cells can present all of the relevant T and B cell epitopes thus resulting in broad, multi 20 antigen, multi-epitopic immune responses. In some embodiments, the chimeric antigen can comprise a Malaria multi-antigen (CSP/AMA-1/LSA-1 20 /MSP-1 42 /HBV core/Fc) vaccine that can be used as a prophylactic and/or therapeutic vaccine for Plasmodiumfalciparum infections with its design, cloning, expression, purification, and evaluation being described herein. 25 Embodiments of the technology as described herein can provide a better approach to vaccine development using the dendritic cell receptor-based approach. This approach can be highly effective in inducing a protective action against HIV infection. This technology can also be useful in "early intervention" therapeutics development during the early phase of the infection, while the immune machinery has not been fully compromised by the 30 virus. 22 In some embodiments, the design of the vaccine can incorporate six immune-relevant HIV antigens into the immune response domain (IRD) of the chimeric antigen. The antigens can include: Gag, which comprises the structural proteins, capsid, matrix and nucleocapsid; Env, which comprises the envelope glycoproteins; and several viral 5 accessory proteins including Tat (transactivator of transcription), Rev (exports spliced viral RNA from nucleus), Vpr (regulates nuclear import of pre-integration complex) and Vpu (involved in viral budding). This design can be expected to elicit robust prophylactic antibody (humoral) immune responses against the Env proteins. The therapeutic (CTL) responses will be targeted to 10 HIV-infected cells presenting the viral peptides on MHC class I, derived from either Gag and/or any or all of the above mentioned accessory proteins. The chimeric antigens and the corresponding antigens can be cloned, the respective baculovirus stock can be produced and can be produced in Sf9 insect cells. The chimeric antigens can be purified in laboratory scale quantities under denaturing conditions using 15 Ni-affinity chromatography (AKTA FPLC, GE Healthcare), using previously established protocols with the necessary modifications. Other options to purify the protein in larger scale can also be used. The binding of the chimeric antigens to DCs can be measured by flow cytometry. Immunogenicity of the chimeric antigens can be assessed using an ex vivo Peripheral 20 Blood Mononuclear Cells (PBMC)-based DC/T cell assay. Briefly, PBMC-derived DC can be loaded with the chimeric antigens and matured using Poly I:C. Autologous T cells derived from the same PBMC can then be co-cultured with the mature DC (mDC) and CD4+ and CD8+ T cell responses can be measured. Specific readouts can include T cell proliferation, IFN-y, TNF-a, perforin, granzyme B and TRAIL expression, as well as 25 lactate dehydrogenase (LDH) release as a surrogate of CTL activity. Chimeric antigen-specific T cell recall responses can also be measured by stimulating chimeric antigen-exposed T cells with mDC loaded with Gag/Env/Tat/Rev/Vpr/Vpu overlapping peptide pools. Further analysis of the humoral immune response can be performed by purifying autologous B cells and analyzing 30 maturation of these cells into antibody producing plasma cells following the exposure to chimeric antigen-loaded mDC. 23 This description includes several abbreviations which can be defined (unless otherwise stated) as follows: AMA-I - apical membrane antigen-1, APC - antigen presenting cell, CSP - circumsporozoite protein, CTL - cytotoxic T lymphocyte, DC dendritic cell, GrB - Granzyme B, HBV - hepatitis B virus, HLA - human leukocyte 5 antigen, IFN - interferon, IgG - immunoglobulin G, IRD - immune response domain, mAb - monoclonal antibody, LSA-I - liver stage antigen-1, MHC - major histocompatibility complex, MMR - macrophage mannose receptor, MSP-I - merozoite surface protein-1, PAGE - polyacrylamide gel electrophoresis, PBMC - peripheral blood, mononuclear cell, PBS - phosphate buffered saline, PHA - phytohemagglutinin, PKR 10 protein kinase ds RNA-dependent, Poly I:C - polyriboinosinic polyribocytidylic acid, Treg - Regulatory T Cell, Tresp - Responder T Cell, SDS - sodium dodecyl sulphate, TBD - target binding domain, TLR - toll-like receptor, TNF - tumor necrosis factor, TT tetanus toxoid. It is an object of the present invention to overcome or ameliorate at least one of the 15 disadvantages of the prior art, or to provide a useful alternative. Unless the context clearly requires otherwise, throughout the description and claims, the words "comprise", "comprising", and the like are to be construed in an inclusive sense as opposed to an exclusive or exhaustive sense; that is to say, in the sense of "including, but not limited to". 20 III. DESCRIPTION OF DRAWINGS FIG. I A provides a schematic diagram illustrating the structure of a chimeric antigen of the present invention as a monomer, wherein the chimeric antigen has two portions, namely an immune response domain and a target binding domain. The schematic also illustrates a preferred embodiment, in which a hinge region is present. 25 FIG. iB provides a schematic diagram illustrating the structure of a chimeric antigen of invention in its normal state, assembled as a dimer. This schematic illustrates a particularly preferred embodiment, in which the chimeric antigen comprises a 6xHis tag and peptide linker in addition to the immune response and target binding domains. 24 FIG. 2 demonstrates that stimulation of T cells with HBN S1/S2-TBD generates a cytotoxic T cell (CTL) response specifically against an epitope from the HBN SI protein. FIG. 3 demonstrates that stimulation of T cells with HBN S1/S2-TBD generates a 5 cytotoxic T cell (CTL) response specifically against an epitope from the HBN S2 protein. FIG. 4 shows the comparison of uptake of HBV S1/S2-TBD, IgG1, and IgG2 by maturing dendritic cells as a function of concentration. 10 FIG. 5 shows the correlation of HBV S1/S2-TBD binding to CD32 and CD206 expression on dendritic cells. Fig. IX is schematic depiction of a dimerized form of Chimigen3 vaccine. It contains two subunits, each composed of an immune response domain (IRD) and a target 15 binding domain (TBD). Figs. 2X A and B are depictions of the nucleotide sequence (SEQ ID NO:9X) of the ORF (open reading frame) in plasmid pFastBacHTa-TBD and the amino acid sequence (SEQ ID NO:1OX) encoded by the ORF, respectively. 20 Figs. 3X A and B are depictions of the nucleotide sequence (SEQ ID NO:39X) of the ORF in plasmid pFastBacHTa-NS5A-TBD and the amino acid sequence (SEQ ID NO:40X) encoded by the ORF, respectively. There is a conserved spontaneous mutation (TTC (Phe) to TTT (Phe)) at the underlined and bolded positions in Fig. 3X A and Fig. 25 3B. In the nucleotide sequence: nucleotides 1-3: start codon nucleotides 13-30: 6XHis epitope tag 30 nucleotides 91-143 1: HCV NS5A nucleotides 1432-1461: linker peptide 25 nucleotides 1462-2157: TBD nucleotides 2158-2187: terminal peptide nucleotides 2188-2190: stop codon nucleotides 1639-1641: TBD TTC-TTT conserved mutation 5 In the amino acid sequence: amino acids 5-10: 6xHis epitope tag amino acids 31-477: HCV NS5A amino acids 478-487: linker peptide 10 amino acids 488-719: TBD amino acids 720-729: terminal peptide Figs. 4X A and B are depictions of the nucleotide sequence (SEQ ID NO:41X) of the ORF in plasmid pFastBacHTa-gp64-NS5A-TBD and the amino acid sequence (SEQ 15 ID NO:52X) encoded by the ORF, respectively. There is an artifactual mutation (GAT (Asp) to TAT (Tyr)) and a conserved spontaneous mutation (TTC (Phe) to TTT (Phe)) at the underlined and bolded positions in Fig. 4X A and Fig. 4X B. In the nucleotide sequence: 20 nucleotides 1-3: start codon nucleotides 1-72: gp64 signal peptide nucleotides 97-114: 6XHis epitope tag nucleotides 175-1515: HCV NS5A nucleotides 1516-1545: linker peptide 25 nucleotides 1546-2241: TBD nucleotides 2242-2271: terminal peptide nucleotides 2272-2274: stop codon nucleotides 61-63: signal peptide GAT to TAT artifactual mutation nucleotide 1725: TBD TTC to TTT conserved mutation 30 In the amino acid sequence: 26 amino acids 1-24: gp64 secretion signal amino acids 33-38: 6xHis epitope tag amino acids 59-505: HCV NS5A amino acids 506-515: linker peptide 5 amino acids 516-747: TBD amino acids 748-757: terminal peptide amino acid 21: signal peptide D to Y artifactual mutation 10 Figs. 5X A and B are depictions of the nucleotide sequence (SEQ ID NO:42X) of the ORF in plasmid pPSC12-NS5A-TBD and the amino acid sequence (SEQ ID NO:53X) encoded by the ORF, respectively. Figs. 6X A and B are depictions of the nucleotide sequence (SEQ ID NO:43X) of 15 the ORF in plasmid pFastBacHTa-gp64-NS3-TBD and the amino acid sequence (SEQ ID NO:54X) encoded by the ORF, respectively. There are two mutations shown by the underlined codons in Fig. 6X A and amino acids in Fig. 6X B. Upstream to downstream for Fig. 6X A and N-terminal to C-terminal for Fig.6X B, the mutations are: an engineered CGG (Arg) to GCG (Ala) mutation; and a spontaneous CCA (Pro) to GGA 20 (Gly) mutation. Figs. 7X A and B are depictions of the nucleotide sequence (SEQ ID NO: 44X) of the ORF in plasmid pFastBacHTa NS3mut-TBD and the amino acid sequence (SEQ ID NO:55X) encoded by the ORF, respectively. There are two mutations shown by the 25 underlined and bolded codons in Fig. 7X A and amino acids in Fig. 7X B. Upstream to downstream for Fig. 7X A and N-terminal to C-terminal for Fig.7X B, the mutations are: a spontaneous conserved AGG (Arg) to CGG (Arg) mutation; and an engineered CGG (Arg) to GCG (Ala) mutation. 30 In the nucleotide sequence: nucleotides 1-3: start codon 27 nucleotides 13-30: 6XHis epitope tag nucleotides 91-1965: HCV NS3mut nucleotides 1966-1995: linker peptide nucleotides 1996-2691: TBD 5 nucleotides 2692-2721: terminal peptide nucleotides 2722-2724: stop codon nucleotide 1462-1464: NS3mut AGG to CGG spontaneous mutation nucleotides 1474-1476: NS3mut CGG to GCG engineered mutation 10 In the amino acid sequence: amino acids 5-10: 6xHis epitope tag amino acids 31-655: HCV NS3mut amino acids 656-665: linker peptide 15 amino acids 666-897: TBD amino acids 898-907: terminal peptide amino acid 488: NS3mut R to R spontaneous mutation amino acid 492: NS3mut R to A engineered mutation 20 Figs. 8X A and B are depictions of the nucleotide sequence (SEQ ID NO:45X) of the ORF in plasmid pFastBacHTa-gp64-NS3mut-TBD and the amino acid sequence (SEQ ID NO:56X) encoded by the ORF, respectively. There are three mutations shown by the highlighted codons in Fig. 8X A and amino acids in Fig. 8X B. Upstream to 25 downstream for Fig. 8X A and N-terminal to C-terminal for Fig.8X B, the mutations are: an artifactual mutation (GAT (Asp) to TAT (Tyr)); a spontaneous conserved AGG (Arg) to CGG (Arg) mutation; and an engineered CGG (Arg) to GCG (Ala) mutation. In the nucleotide sequence: 30 nucleotides 1-3: start codon nucleotides 1-72: gp64 signal peptide 28 nucleotides 97-114: 6XHis epitope tag nucleotides 175-2049: HCV NS3mut nucleotides 2050-2079: linker peptide nucleotides 2080-2775: TBD 5 nucleotides 2776-2805: terminal peptide nucleotides 2806-2808: stop codon nucleotides 61-63: signal peptide GAT to TAT artifactual mutation nucleotides 1546-1548: NS3mut AGG to CGG conserved mutation nucleotides 1558-1560: NS3mut CGG-GCG engineered mutation 10 In the amino acid sequence: amino acids 1-24: gp64 secretion signal amino acids 33-38: 6xHis epitope tag amino acids 59-683: HCV NS3mut 15 amino acids 684-693: linker peptide amino acids 694-925: TBD amino acids 926-935: terminal peptide amino acid 21: signal peptide D to Y artifactual mutation amino acid 520: NS3mut R to A engineered mutation 20 Figs. 9X A and B are depictions of the nucleotide sequence (SEQ ID NO:46X) of the ORF in plasmid pFastBacHTa-gp64 NS3-NS4B-NS5A-TBD and the amino acid sequence (SEQ ID NO:57X) encoded by the ORF, respectively. There are four mutations shown by the underlined and bolded codons in Fig. 9X A and amino acids in Fig. 9X B. 25 Upstream to downstream for Fig. 9X A and N-terminal to C-terminal for Fig. 9X B, the mutations are: an artifactual mutation (GAT (Asp) to TAT (Tyr)); an engineered TCG (Ser) to GCG (Ala) mutation; an engineered CGG (Arg) to GCG (Ala) mutation; and a spontaneous CCA (Pro) to GGA (Gly) mutation. 30 Figs. lOX A and B are depictions of the nucleotide sequence (SEQ ID NO:47X) of the ORF in plasmid FastBacHTa-gp64-NS3-NS5A-TBD and the amino acid sequence 29 (SEQ ID NO:58X) encoded by the ORF, respectively. There are four mutations shown by the underlined codons in Fig. lOX A and amino acids in Fig. 1OX B. Upstream to downstream for Fig. lOX A and N-terminal to C-terminal for Fig. lOX B, the mutations are: an artifactual mutation (GAT (Asp) to TAT (Tyr)); an engineered TCG (Ser) to GCG 5 (Ala) mutation; an engineered CGG (Arg) to GCG (Ala) mutation; and a spontaneous CCA (Pro) to GGA (Gly) mutation. In the nucleotide sequence: nucleotides 1-3: start codon 10 nucleotides 1-72: gp64 signal peptide nucleotides 97-114: 6XHis epitope tag nucleotides 175-2049: HCV NS3mut nucleotides 2050-2058: linker peptide nucleotides 2059-3402: HCV NS5A 15 nucleotides 3403-3426: linker peptide nucleotides 3427-4122: TBD nucleotides 4123-4152: terminal peptide nucleotides 4153-4155: stop codon nucleotides 61-63: signal peptide GAT to TAT artifactual mutation 20 nucleotide 589: NS3mut TCG to GCG engineered mutation nucleotides 1558-1559: NS3mut CGG to GCG engineered mutation nucleotides 2050-2052: NS3mut CCA to GGA spontaneous mutation In the amino acid sequence: 25 amino acids 1-24: gp64 secretion signal amino acids 33-38: 6xHis epitope tag amino acids 59-683: HCV NS3 amino acids 684-686: linker peptide amino acids 687-1134: HCV NS5A 30 amino acids 1135-1374: TBD amino acids 1375-1384: terminal peptide 30 amino acid 21: signal peptide D to Y artifactual mutation amino acid 197: NS3mut S to A engineered mutation amino acid 520: NS3mut R to A engineered mutation amino acid 684: NS3mut P to G spontaneous mutation 5 Figs. 1IX A and B are depictions of the nucleotide sequence (SEQ ID NO:48X) of the ORF in plasmid pFastBacHTa HCV core (1-177)-TBD and the amino acid sequence (SEQ ID NO:59X) encoded by the ORF, respectively. 10 Figs. 12X A and B are depictions of the nucleotide sequence (SEQ ID NO:49X) of the ORF in plasmid pFastBacHTa-El-TBD and the amino acid sequence (SEQ ID NO:60X) encoded by the ORF, respectively. Figs. 13X A and B are depictions of the nucleotide sequence (SEQ ID NO:50X) 15 of the ORF in plasmid pFastBacHTa E2-TBD and the amino acid sequence (SEQ ID NO:61X) encoded by the ORF, respectively. Figs. 14X A and B are depictions of the nucleotide sequence (SEQ ID NO:51X) of the ORF in plasmid pFastBacHTa-El-E2-TBD and the amino acid sequence (SEQ ID 20 NO:62X) encoded by the ORF, respectively. Fig. 15X is a series of fluorescence flow cytometry (FFC) profiles showing binding, at three different concentrations, of the NS5A Chimigen3 Protein to immature DCs. 25 Fig. 16X is a pair of bar graphs showing inhibition of binding of the NS5A Chimigen3 Protein to immature DCs by antibodies specific for CD32 and CD206. Fig. 17X is a series of bar graphs showing the expression of the indicated cell 30 surface markers by mature DC produced as described in the Examples. The data were 31 obtained by FFC and are presented as "% percent positive cells" (top graphs) and "mean fluorescence intensity" ("MFI") (bottom graphs). Figs. 18X A - B are three sets of bar graphs showing the proportion of CD69 5 expressing T cells (Fig. 18X A), CD69 expressing CD8+ T cells (Fig. 18X B), and CD69 expressing CD4+ T cells (Fig. 18X C) in day 4 cultures of various concentrations of T cells and NS5A Chimigen3 Protein or tetanus toxoid loaded DC. The data were obtained by FFC. 5ACl and 5AC2: two different preparations ofNS5A Chimigen3 Protein. 10 Figs. 19X A-C are three sets of bar graphs showing the proportion of CFSElo T cells (Fig. 19X A), CFSElo CD8+ T cells (Fig. 19X B), and CFSElo CD4+ T cells (Fig. 19X C) in day 4 cultures of various concentrations of T cells and NS5A Chimigen3 Protein or tetanus toxoid loaded DC. The data were obtained by FFC. 5ACl and 5AC2: two different preparations ofNS5A Chimigen3 Protein. 15 Figs. 20X A-C are three sets of bar graphs showing the proportion of CD69 expressing T cells (Fig. 20X A), CD69 expressing CD8+ T cells (Fig. 20X B), and CD69 expressing CD4+ T cells (Fig. 20X C) in day 7 cultures of various concentrations of T cells and NS5A Chimigen3 Protein or tetanus toxoid loaded DC or phytohemagglutinin 20 (PHA; in Fig. 20X A). The data were obtained by FFC. 5ACl and 5AC2: two different preparations ofNS5A Chimigen3 Protein. Figs. 21X A-C are three sets of bar graphs showing the proportion of CFSElo T cells (Fig. 21X A), CFSElo CD8+ T cells (Fig. 21X B), and CFSElo CD4+ T cells (Fig. 25 21X C) in day 7 cultures of various concentrations of T cells and NS5A Chimigen3 Protein or tetanus toxoid loaded DC or PHA (in Fig. 21X A). The data were obtained by FFC. 5ACl and 5AC2: two different preparations ofNS5A Chimigen3 Protein. Fig. 22X is a series of bar graphs showing the proportion of T cells that are blasts 30 in 7 day cultures of various concentrations of T cells and NS5A Chimigen3 Protein or 32 tetanus toxoid loaded DC or PHA. The data were obtained by FFC. 5ACl and 5AC2: two different preparations ofNS5A Chimigen3 Protein. Fig. 23X is a series of bar graphs showing the expression by matured, antigen 5 loaded DC of the indicated cell surface markers. The data were obtained by FFC. Fig. 24X is a pair of bar graphs showing the proportion of T cells that are blasts after three stimulations with matured, antigen loaded DC made as described in the Examples. The data were obtained by (FFC). 5AC: NS5A Chimigen3 Protein. 10 Fig. 25X is a series of bar graphs showing the proportion of T cells containing intracellular interferon-K (IFN-K) after three stimulations with matured, antigen loaded DC made as described in the Examples. The data were obtained by FFC. 5AC: NS5A Chimigen3 Protein; PMA: phorbol myristic acid; Dulbecco's phosphate buffered saline 15 (DPBS). Fig. 26X is a pair of bar graphs showing the proportion of CD8+ T cells containing intracellular IFN-K after three stimulations with matured, antigen loaded DC made as described in the Examples. The data were obtained by FFC. 5AC: NS5A 20 Chimigen3 Protein. Fig. 27X is a pair of bar graphs showing the proportion of CD4+ T cells containing intracellular IFN-K after three stimulations with matured, antigen loaded DC made as described in the Examples. The data were obtained by FFC. 5AC: NS5A 25 Chimigen3 Protein. Fig. 28X is a pair of bar graphs showing the proportion of T cells containing intracellular tumor necrosis factor-I (TNF-I) after three stimulations with matured, antigen loaded DC made as described in the Examples. The data were obtained by FFC. 30 5AC: NS5A Chimigen3 Protein; PMA: phorbol myristic acid; DPBS. 33 Fig. 29X is a pair of bar graphs showing the proportion of CD8 T cells (left graph) and CD4 m T cells (right graph) containing intracellular TNF-I after three stimulations with matured, antigen loaded DC made as described in the Examples. The 5 data were obtained by FFC. 5AC: NS5A Chimigen3 Protein. Fig. 30X is a pair of bar graphs showing the proportion of CD8 T cells expressing the granular proteins GrB (left graph) and Pfn (right graph) after three stimulations with matured, antigen loaded DC made as described in the Examples. The 10 data were obtained by FFC. 5AC: NS5A Chimigen3 Protein. Fig. 31X is a pair of bar graphs showing the total number of lymphocytes (RI gated cells) and the proportion of blast cells after three stimulations with matured, antigen loaded DC made as described for in the Examples. The cells were analyzed 6 days after 15 the last stimulation. The data were obtained by FFC. 5AC: NS5A Chimigen3 Protein. Fig. 32X is a pair of graphs showing the relative proportions of CD69 expressing CD8 m T cells (left graph) and CD69 expressing CD4 T cells (right graph) after three stimulations with matured, antigen loaded DC made as described in the Examples. The 20 cells were analyzed 6 days after the last stimulation. The data were obtained by FFC. 5AC: NS5A Chimigen3 Protein. Fig. 33X is a pair of bar graphs showing the relative proportions of CD8 m T cells (left graph) and CD4 m T cells (right graph) having antigen specific T cell receptors (TCR) 25 that bound an EBVpeptide/HLA-A2 tetramer (positive control) or a control tetramer (negative tetramer) after three stimulations with matured, antigen loaded DC made as described in the Examples. The cells from three individual culture wells (corresponding to the three bars in the test and control groups) were analyzed 6 days after the last stimulation. The data were obtained by FFC. 30 34 Fig. 34X is a pair of bar graphs showing the relative proportions of CD8+ T cells having TCR that bound NS5A peptide/HLA-A2 pentamer after three stimulations (using different numbers of T cells and DC) with matured, antigen loaded DC made as described in the Examples. The cells from three individual culture wells (corresponding to the 5 three bars in the test and control groups) were analyzed 6 days after the last stimulation. The data were obtained by FFC. 5AC: NS5A Chimigen3 Protein. Fig. 35X is a series of FFC profiles showing binding, at two different concentrations, of the NS3 Chimigen3 Protein to immature DCs. 10 Fig. 36X is a pair of bar graphs showing inhibition of binding of the NS3 Chimigen3 Protein to immature DCs by antibodies specific for CD32 and CD206. Fig. 37X is a series of bar graphs showing the proportion of CD69 expressing 15 CD8+ T cells (left graphs) and CD69 expressing CD4+ T cells (right graphs) in day 4 (top graphs) and day 7 cultures containing NS3 Chimigen3 Protein or tetanus toxoid loaded DC. The data were obtained by FFC. 3C: NS3 Chimigen3 Protein. Fig. 38X is a series of bar graphs showing the proportion of CFSElo CD8+ T cells 20 (left graphs) and CFSElo CD4+ T cells (right graphs) in day 4 (top graphs) and day 7 cultures containing NS3 Chimigen3 Protein or tetanus toxoid loaded DC. The data were obtained by FFC. 3C: NS3 Chimigen3 Protein. Fig. 39X is a pair of bar graphs showing the proportion of T cells that are blasts 25 after three stimulations with matured, antigen loaded DC made as described for the NS5A Chimigen3 Protein in the Examples. The stimulations were performed using two different T cell and two different DC concentrations. The data were obtained by FC. 3C: NS3 Chimigen3 Protein. 30 Fig. 40X is a series of bar graphs showing the proportion of T cells containing intracellular IFN-K after three stimulations with matured, antigen loaded DC made as 35 described in the Examples. The data were obtained by FFC. 3C: NS3 Chimigen3 Protein. Fig. 41X is a pair of bar graphs showing the proportion of CD8+ T cells (left 5 graph) and CD4+ T cells (right graph) containing intracellular IFN-K after three stimulations with matured, antigen loaded DC made as described in the Examples. The data were obtained by FFC. 3C: NS3 Chimigen3 Protein. Fig. 42X is a pair of bar graphs showing the proportion of CD8+ T cells (left 10 graph) and CD4+ T cells (right graph) containing intracellular TNF-I after three stimulations with matured, antigen loaded DC made as described in the Examples. The data were obtained by FFC. 3C: NS3 Chimigen3 Protein. Fig. 43X is a pair of bar graphs showing the proportion of CD8+ T cells 15 expressing the granular proteins GrB (left graph) and Pfn (right graph) after three stimulations with matured, antigen loaded DC made as described in the Examples. The data were obtained by FFC. 3C: NS3 Chimigen3 Protein. Fig. 44X is a pair of graphs showing the relative proportions of CD69 expressing 20 CD8+ T cells (left graph) and CD69 expressing CD4+ T cells (right graph) after three stimulations with matured, antigen loaded DC made as described in the Examples. The cells were analyzed 6 days after the last stimulation. The data were obtained by FFC. 3C: NS3 Chimigen3 Protein. 25 Fig. 45X is a pair of bar graphs showing the total number of lymphocytes (RI gated cells) (left graph) and the proportion of blast cells (right graph) after three stimulations with matured, antigen loaded DC made as described in the Examples. The cells were analyzed 6 days after the last stimulation. The data were obtained by FFC. 3C, AS60-1: NS3 Chimigen3 Protein. 30 36 Fig. 46X is a pair of bar graphs showing the relative proportions of CD8+ T cells having TCR that bound NS3 peptide/HLA-A2 pentamer after three stimulations (using different numbers of T cells and DC) with matured, antigen loaded DC made as described in the Examples. The cells from three individual culture wells (corresponding to the 5 three bars in both the test groups and the control group) were analyzed 5 days after the last stimulation. The data were obtained by FFC. 3C: NS3 Chimigen3 Protein. Fig. 47X is a series of fluorescence flow cytometry (FFC) profiles showing binding, at three different concentrations, of the HCV Core Chimigen3 Protein to 10 immature DCs. Figs. 48X is a pair of bar graphs showing inhibition of binding of the HCV Chimigen3 Core Protein to immature DCs by antibodies specific for CD32 and CD206, mannosylated bovine serum albumin (mBSA), and murine IgG fragments. 15 Fig. 49X is a pair of bar graphs showing the proportion of CD8+ T cells (left graph) and CD4+ T cells (right graph) containing intracellular IFN-K after three stimulations with matured, antigen loaded DC made as described in the Examples. The data were obtained by FFC. HCV Core-TBD: HCV Core Chimigen3 Protein. 20 Fig. 50X is a pair of two-dimensional FFC dot plots showing the proportion of CD8+ T cells having TCR that bound a HCV Core peptide/HLA-B7 tetramer after three stimulations with DC loaded with the HCV Core Chimigen3 Protein (HCV Core-TBD) (right dot plot) or TBD alone (TBD) (left dot plot). 25 Figs. 1Z to 27Z are depictions of embodiments of malarial chimeric antigens, their design, cloning, expression, purification, and evaluation. Figs. 28Z to 48Z are depictions of embodiments of HIV chimeric antigens, their 30 design, cloning, expression, purification, and evaluation. 37 Fig. 49Z are depictions of embodiments of cancer antigen chimeric antigens IV. DETAILED DESCRIPTION A. Overview 5 Disclosed herein are compositions and methods for eliciting immune responses against antigens. In particular, the compounds and methods elicit immune responses against foreign antigens that are otherwise recognized by the host as "self' antigens, thus breaking host tolerance to those antigens. Presenting the host immune system with a chimeric antigen comprising an immune response domain and a target binding domain, 10 wherein the target binding domain comprises an antibody fragment, enhances the immune response against the foreign or tolerated antigen. Antigen presenting cells take up, process, and present the chimeric antigen, eliciting both a humoral and cellular immune response against the desired antigen. Disclosed herein are compositions and methods for eliciting immune responses 15 against antigens. In particular embodiments, the compounds and methods elicit immune responses against antigens that are otherwise recognized by the host as "self' antigens. The immune response is enhanced by presenting the host immune system with a chimeric antigen comprising an immune response domain and a target binding domain, wherein the target binding domain comprises an antibody fragment. By virtue of the target 20 binding domain, APCs internalize, process and present the chimeric antigen, eliciting both humoral and cellular immune responses. HCV is a member of the flaviviridae family which can infect humans, resulting in acute and chronic hepatitis, and may result in hepatocellular carcinoma [Hoofniagle (2002) Hepatology 36:S21-S29]. The HCV genome is a 9.6 Kb uncapped positive 25 polarity single stranded RNA molecule and the replication occurs via a negative-strand intermediate [Lindenbach and Rice (2005) Nature 436:933-938]. The HCV genome encodes a single open reading frame that encodes a polyprotein, which is processed to generate the core or capsid protein (C), two envelope glycoproteins (El &amp; E2), a small hydrophobic protein (p7), and six non-structural proteins (NS2, NS3, NS4A, NS4B, 30 NS5A &amp; NS5B). The processing of the polyprotein into the individual proteins is 38 catalyzed by host and viral proteases [Lohmann et al. (1996) J. Hepatol. 24:11-19, Penin et al. (2004) J. Hepatol. 24:11-19]. When a healthy host (human or animal) encounters a foreign antigen (such as proteins derived from a bacterium, virus and/or parasite), the host normally initiates an 5 immune response. The adaptive immune response may be humoral, cellular or both [Whitton at al. (2004) Adv. Virus Res. 63: 181-238]. The cellular response is characterized by the selection and expansion of specific T helper cells and T lymphocytes (CTLs) capable of directly eliminating the cells which contain the antigen. In the case of the humoral response, antibodies are produced by B cells and are secreted into the blood 10 and/or lymph in response to an antigenic stimulus. The antibodies neutralize the antigen, (e.g. a virus) by binding specifically to epitopes on its surface, marking it for destruction by phagocytic cells and/or complement-mediated mechanisms to lyse the infected cells [Carroll (2005) Nature Immunol. 5:981-986]. Helper cells (largely CD4 T cells) provide the helper activity that is required for both CTL (largely CD8 T cells) and B cell 15 mediated antibody responses. In individuals with chronic viral infections, the immune system does not respond to the incoming pathogen to produce an adaptive immune response to clear the infection and thus the host becomes tolerant to the pathogen. Although the mechanism HCV uses to evade the immune surveillance is not completely understood, several possibilities have 20 been suggested. These include blockage of IRF3-mediated induction of type I IFN by NS3-4A, E2 and NS5A sequences, blocking of PKR (double stranded RNA-activated protein kinase) as well as interference of HCV proteins with the function of NK cells [Rehermann et al. (2005) Nature Rev. Immunol. 5:215-229]. Recent results also show the pivotal role of T cells in the control and eradication of HCV infection [Bowen et al. 25 (2005) Nature 436:946-952; Wieland et al. (2005) J. Virol. 79:9369-9380] . In acute HCV infection, although virus-specific antibodies were detected 7-8 weeks after HCV infection [Pawlotsky (1999) J. Hepatol. 31(suppl):71-79], the role of antibody is not clear, since it has been shown that HCV infection can be resolved in the absence of anti HCV antibodies in chimpanzees [Cooper et al. (1999) Immunity 10:439-449] and 30 without seroconversion in humans [Post et al. (2004) J. Infect. Dis. 189:1846-1855]. In addition, recent evidence suggests that the failure of individuals to produce detectable 39 levels of CD4+ and CD8+ T-cell responses against HCV resulted in chronic infections [Cooper et al. (1999) supra; Thimme et al. (2001) Proc. Natl. Acad. Sci. USA 99:15661 15668 ; Thimme et al. (2002) J. Exp. Med. 194:1395-1406; Shoukry et al. (2003) J. Exp. Med. 197: 1645-1655]. An interplay of immune functions such as transcriptional 5 changes in type I IFN-response and immune response against double stranded RNA produced during virus replication have been suggested to occur, but no direct evidence for this effect in the clearance of the virus infection has been observed [Rehermann et al. (2005) supra]. It has been observed that in patients who resolve an HCV infection, the immune system produces strong, multi-epitope-specific CD4+ and CD8+ T cell 10 responses [Rehermann et al. (2005) supra], whereas in patients with chronic HCV infection, the T cell response was late, transient or narrowly focused [Thimme et al. (2001) supra; Diepolder et al. (1995) Lancet 36: 1006-1007; Lechner et al. (2000) J. Exp. Med. 191:1499-1522]. The absence of vigorous T cell responses against HCV antigens, which result in 15 chronic infections may also be due to the lack of proper presentation of the appropriate viral antigen to the host immune system. The success in eliminating the virus may result from the manner in which the antigen is processed and presented by the APCs and the involvement of regulatory T helper cells and cytotoxic T lymphocytes (CTLs). The major participant in the antigen presenting process is the dendritic cell (DC), 20 which captures and processes the antigens. In addition, DCs express lymphocyte co stimulatory molecules and migrate to lymphoid organs where they secrete cytokines to initiate immune responses. DCs also control the proliferation of B and T lymphocytes, which are the mediators of immunity [Steinman et al. (1999) Hum. Immunol. 60:562 567]. The generation of a CTL response is critical in the elimination of the virus-infected 25 cells and thus in the resolution of infection. The encountered antigens are processed differently by the APCs depending on the localization of the antigen [Steinman et al. (1999) supra]. Exogenous antigens are processed within the endosomes of the APC and the generated peptide fragments are presented on the surface of the cell complexed with major histocompatibility complex 30 (MHC) class II molecules. The presentation of this complex to CD4+ T cells results in 40 their activation. As a result, cytokines secreted by helper T cells provide the required soluble factors for activation of B cells to produce antibodies against the exogenous antigen (humoral response). Conversely, intracellular antigens are processed in the proteasome and the 5 resulting peptide fragments are presented as complexes with MHC class I molecules on the surface of APCs. Following binding of this complex to the T cell receptor (TCR), antigen presentation to CD8+ T cells occurs, which results in a CTL immune response. CTLs can eliminate the virus by killing the infected cells and by the production of factors such as the cytokine interferon-y (IFN-y), which acts to inhibit viral replication. 10 As the virus is actively replicating in individuals with chronic viral infections, viral antigens are produced within host cells and secreted antigens are present in the circulation. In spite of the presence of these antigens there is a lack of an effective immune response against the virus. An effective immune response would involve the production of CTLs, which could recognize a broad array of viral epitopes with high 15 affinity. Thus an appropriate therapeutic vaccine containing viral antigens must be internalized and processed in the appropriate cellular compartment in order for viral peptides to be presented in the groove of MHC class I molecules. The recognition of the viral epitopes in the context of class I presentation would allow the activation, production, and differentiation of CD8+ T cells to functional CTLs that are able to mount 20 an effective response against the viral infection. Thus a therapeutic vaccine containing viral antigens would be effective if it was processed through the proteasomal pathway and presented via MHC class I [Larsson et al. (2001) Trends Immunol. 22:141-148]. This could be achieved either by producing the antigen within the host cell, or by delivery to the appropriate cellular compartment such 25 that the antigen is processed and presented in a manner that will elicit the desired cellular response. Several approaches have been documented in the literature for the intracellular delivery of antigens, including viral vectors [Lorenz et al. (2001) Hum. Gene Ther. 10:1095-1103], the use of DNA-transfected cells [Donnelly at al. (1997) Annu. Rev. Immunol. 15:617-648] and the expression of the antigen through injected DNA vectors 30 [Lai et al. (1998) Crit. Rev. Immunol. 18:449-484]. 41 By virtue of their APC functionality, DCs which are derived from monocytes, have been shown to have great potential as immune modulators that stimulate primary T cell response [Banchereau et al. (1998) Nature 392:245-252]. This unique property of the DCs to capture, process, and effectively present antigen makes them very important tools 5 for therapeutic vaccine development [Laupeze et al. (1999) Hum Immunol. 60:591-597]. Targeting of the antigen to the DCs is a crucial step and the presence of several receptors on the DCs specific to the Fc region of monoclonal antibodies (mAb) has been exploited for this purpose [Regnault et al. (1999) J. Exp. Med. 189:371-380]. Examples of this approach include ovarian cancer mAb-B43.13 [Berlyn et al. (2001) Clin Immunol. 101: 10 276-283] , anti-PSA mAb, and anti-HBV antibody antigen complexes {Wen et al. (1999) Int. Rev. Immunol. 18:251-258]. Cancer immunotherapy using DCs loaded with tumor associated antigens have ben shown to produce tumor-specific immune responses and anti-tumor activity [Campton et al. (2000) J. Invest. Dermatol. 115:57-61; Fong et al. (2000) Annu. Rev. Immunol. 18:245-273]. Promising results were obtained in clinical 15 trials in vivo using tumor antigen-pulsed DCs [Tarte et al. (1999) Leukemia 13:653-663]. These studies clearly demonstrate the efficacy of using DCs to generate immune responses against cancer antigens. A therapeutic vaccine must be able to elicit host immune responses against viral antigens to which the host immune system is tolerant. This involves the delivery of antigens to DCs, appropriate antigen presentation and 20 priming of HCV-specific CD8+ T cells that can result in therapeutic effect in chronic carriers. Chimeric antigen vaccines of the invention are a novel class of recombinant "chimeric antigens" produced as fusion proteins of selected antigens and specific regions of an antibody. The bifunctional design of the molecule is tailored to target the viral 25 antigen to APCs, especially DCs, to elicit both humoral and cellular immune responses against the selected antigen. The HCV ChimigenTM vaccine in its dimerized form is schematically represented in Fig. IX. The vaccine has two domains: an immune response domain (IRD) that contains the recombinant HCV viral antigen, and a target-binding domain (TBD), which contains 30 an Fc fragment of a monoclonal antibody. The design of the vaccine imparts several unique properties to its function. The chimeric design favors the formation of antibody 42 like structures that facilitate its uptake through specific receptors and results in appropriate antigen presentation. It can be processed through the proteasomal pathway and the peptides presented as complexes with MHC class I, resulting in a CTL response. ChimigenTM vaccines can also be processed via the endosomal pathway, presented by 5 MHC class II, to produce a humoral response. The TBD mediates the binding of the Chimigen T M vaccine to specific APC receptors such as Fcy receptors. While the invention is not limited by any particular mechanism of action, it appears that binding of the molecule to Fcy receptors on APC (e.g., immature DCs) results in the processing of the antigen through the MHC class I 10 pathway. In some embodiments, a xenotypic TBD, the recombinant antigen, the linker peptides of varying lengths incorporated at the amino and carboxy termini of the antigen, make the whole molecule "foreign" and allow the host immune system to mount multi epitopic immune responses against the fusion protein, including the HCV antigen. Fusion protein Chimigen3 proteins can also be produced in non-mammalian cells (e.g., 15 yeast or insect cells) so that they are glycosylated in an non-mammalian fashion, thereby enhancing their immunogenicity in mammalian (e.g., human) hosts. Mannose/pauci mannose glycosylation introduced in insect cells also permits the uptake of the vaccine by mannose receptors on APCs for uptake. Therefore, ChimigenTM vaccines can be internalized by the APCs through specific 20 Fcy receptors I, II and III (CD64, CD32, CD16), mannose receptors (CD206), other C type lectin receptors, and by phagocytosis [Geijtenbeek et al. (2004) Annu. Rev. Immunol. 22:33-54]. The uptake via specific receptors, processing through the endosomal and proteasomal pathways, and presentation on both classes of MHC molecules can result in a broad immune response capable of preventing viral infection or 25 eliminating the virus-infected cells. The generation of a CTL response is critical to clear virus-infected cells [Whitton et al. (2004) Adv. Virus Res. 63:181-238]. HepaVaxx B, ViRexx's first ChimigenTM therapeutic vaccine for the treatment of chronic HBV infections, has shown very promising results in preclinical studies [George et al. (2003) A novel class of therapeutic vaccines for the treatment of chronic viral infections: 30 evaluation in ducks chronically infected with duck hepatitis B virus (DHBV), in Hepdart 2003, Frontiers in Drug Development for Viral Hepatitis: December 14-18, Kauai, 43 Hawaii, USA; George et al. (2003) A novel class of therapeutic vaccines for the treatment of chronic viral infections. International Meeting of the Molecular Biology of Hepatitis B Viruses. September 7-10, Centro Congressi Giovanni XXIII, Bergamo, Italy; George et al. (2004) Immunological Evaluation of a Novel Chimeric Therapeutic Vaccine for the 5 Treatment of Chronic Hepatitis B Infections. (2004) International Meeting of the Molecular Biology of Hepatitis B Viruses. Woods Hole, MA, USA, October 24-27, 2004; George et al. (2005) BioProcessing Journal 4:39-45; George et al. (2006) A new class of therapeutic vaccines for the treatment of chronic hepatitis B infections. In "Framing the Knowledge of Viral Hepatitis" Schinazi, R. F. Editor, IHL Press USA] . 10 B. Definitions The terms used in this application have the meanings indicated by the following definitions (unless otherwise indicated). 15 "Antibody" refers to an immunoglobulin molecule produced by B lymphoid cells. These molecules are characterized by having the ability to bind specifically with an antigen, each being defined in terms of the other. "Antibody response" or "humoral response" refers to a type of immune response in which antibodies are produced by B lymphocytes and are secreted into the blood 20 and/or lymph in response to an antigenic stimulus. In a properly functioning immune response, the antibody binds specifically to antigens on the surface of cells (e.g., a pathogen), marking the cell for destruction by phagocytic cells, antibody-dependent cellular cytotoxicity (ADCC) effector cells, and/or complement-mediated mechanisms. Antibodies also circulate systemically and can bind to free virions. This antibody binding 25 can neutralize the virion and prevent it from infecting a cell as well as marking the virion for elimination from host by phagocytosis or filtration in the kidneys. "Antigen" refers to any substance that, as a result of coming in contact with appropriate cells, induces a state of sensitivity and/or immune responsiveness and that reacts in a demonstrable way with antibodies and/or immune cells of the sensitized 30 subject in vivo or in vitro. Thus, antigens can include, for example, cells or viral 44 particles and/or each of their components. In the case of viruses, the components specifically include viral proteins. "Antigen-presenting cell" ("APC") refers to the accessory cells of antigen inductive events that function primarily by internalizing antigens, processing antigens 5 and presenting antigenic epitopes in context of major histocompatibility complex (MHC) class I or II molecules to lymphocytes. The interaction of APCs with antigens is an essential step in immune induction because it enables lymphocytes to encounter and recognize antigenic molecules and to become activated. Exemplary APCs include macrophages, monocytes, Langerhans cells, interdigitating dendritic cells, Follicular 10 dendritic cells, and B cells. "B cell" refers to a type of lymphocyte that produces immunoglobulins (antibodies) that interact with antigens. "CHI region", "CH2 region", "CH3 region" each refer to a different region of the heavy chain constant domain of an antibody. 15 "Cellular response" or "cellular host response" refers to a type of immune response mediated by specific helper and killer T cells capable of directly or indirectly eliminating virally infected or cancerous cells. As used herein, the term "chimeric antigen" refers to a polypeptide comprising an immune response domain (IRD) and a target binding domain (TBD). The immune 20 response domain and target binding domains may be directly or indirectly linked by covalent or non-covalent means. "Complex" or "antigen-antibody complex" refers to the product of the reaction between an antibody and an antigen. Complexes formed with polyvalent antigens tend to be insoluble in aqueous systems. 25 "Cytotoxic T-lymphocyte" is a specialized type of lymphocyte capable of destroying foreign cells and host cells infected with the infectious agents that produce viral antigens. "Epitope" refers to the simplest form of an antigenic determinant, on a complex antigen molecule; this is the specific portion of an antigen that is recognized by an 30 antibody or a T cell receptor. 45 "Fragment" refers to a part of a disunified entity. In the context of this invention it may also be used to refer to that part as part of a corresponding entity. Accordingly, a fusion protein comprising a Fc fragment may refer to a recombinant molecule comprising the same peptide sequence as the native fragment. 5 "Fusion protein" refers to a protein formed by expression of a hybrid gene made by combining two or more coding sequences. "Hinge region" refers to the portion of an antibody that connects the Fab fragment to the Fc fragment; the hinge region contains disulfide bonds that covalently link the two heavy chains together to form a dimeric molecule. 10 The term "homolog" refers to a molecule which exhibits homology to another molecule, by for example, having sequences of chemical residues that are the same or similar at corresponding positions. The phrase "% homologous" or "% homology" refers to the percent of nucleotides or amino acids at the same position of homologous polynucleotides or polypeptides that are identical or similar. For example, if 75 of 80 15 residues in two proteins are identical, the two proteins are 93.75% homologous. Percent homology can be determined using various software programs known to one of skill in the art. "Host" refers to a warm-blooded animal which suffers from an immune-treatable condition, such as an infection or a cancer. As used herein, "host" also refers to a warm 20 blooded animal, including a human, to which a chimeric antigen is administered. In the context of this invention, "hybridization" means the pairing of complementary strands of oligomeric compounds. In the present invention, the preferred mechanism of pairing involves hydrogen bonding, which may be Watson-Crick, Hoogsteen or reversed Hoogsteen hydrogen bonding, between complementary nucleoside 25 or nucleotide bases (nucleobases) of the strands of oligomeric compounds. For example, adenine and thymine are complementary nucleobases that pair through the formation of hydrogen bonds. Hybridization can occur under varying circumstances. The terms "hybridize", "hybridizing", "hybridizes" and the like, used in the context of polynucleotides, are meant to refer to conventional hybridization conditions, preferably 30 such as hybridization in 50% formamide/6X SSC/0.1% SDS/100 pg/mL mDNA, in 46 which temperatures for hybridization are above 370 C and temperatures for washing in O.1X SSC/0.1% SDS are above 55'C. "Immunity" or "immune response" refers to the body's response to an antigen. In particular embodiments, it refers to the ability of the body to resist or protect itself against 5 infectious disease. "Immune Response Domain (IRD)" refers to the variously configured antigenic portion of a chimeric molecule. The IRD comprises one or more antigens or one or more recombinant antigens. Preferred viral antigens include, but are not limited to, HCV Core, HCV El-E2, HCV El, HCV E2, HCV P7, HCV NS3-serine protease, HCV NS4A, HCV 10 NS4B, and HCV NS5A. As used herein, the phrase "immune-treatable condition" refers to a condition or disease that can be prevented, inhibited or relieved by eliciting or modulating an immune response in the subject. "Lymphocyte" refers to a subset of nucleated cells found, for example, in the 15 blood, which mediate specific immune responses. "Monoclonal antibody" or "mAb" refers to an antibody produced from a clone or genetically homogenous population of fused hybrid cells, i.e., a hybridoma cell. Hybrid cells are cloned to establish cells lines producing a specific monoclonal antibody that is chemically and immunologically homogenous, i.e., that recognizes only one type of 20 antigen. As used herein, "operably linked" means incorporated into a genetic construct so that expression control sequences effectively control expression of a coding sequence of interest. "Peptide linkage" or "peptide bond" refers to the covalent chemical linkage 25 between two or more amino acids. It is a substituted amide linkage between the ax-amino group of one amino acid and the ax-carboxyl group of another amino acid. A "pharmaceutical excipient" comprises a material such as an adjuvant, a carrier, a pH-adjusting and buffering agent, a tonicity adjusting agent, a wetting agent, a preservative, and the like. 30 "Pharmaceutically acceptable" refers to a non-toxic composition that is physiologically compatible with humans or other animals. 47 The term "polynucleotide" as used herein refers to a polymeric form of nucleotides of any length, either ribonucleotides or deoxyribonucleotides. This term refers only to the primary structure of the molecule. Thus, the term includes double- and single-stranded DNA and RNA. It also includes known types of modifications, for 5 example, labels which are known in the art, methylation, "caps", substitution of one or more of the naturally occurring nucleotides with an analog, internucleotide modifications such as, for example, those with uncharged linkages (e.g., methyl phosphonates, phosphotriesters, phosphoamidates, carbamates, etc.) and with charged linkages (e.g., phosphorothioates, phosphorodithioates, etc.), those containing pendant moieties, such as, 10 for example proteins (including e.g., nucleases, toxins, antibodies, signal peptides, poly L-lysine, etc.), those with intercalators (e.g., acridine, psoralen, etc.), those containing chelators (e.g., metals, radioactive metals, boron, oxidative metals, etc.), those containing alkylators, those with modified linkages (e.g., alpha anomeric nucleic acids, etc.), as well as unmodified forms of the polynucleotide. 15 "Polypeptide" and "protein" are used interchangeably and mean any peptide linked chain of amino acids, regardless of length or post-translational modification. As used herein, "prophylaxis" means complete prevention of the symptoms of a disease, a delay in onset of the symptoms of a disease, or a lessening in the severity of subsequently developed disease symptoms. 20 "Prevention" of a disease means that symptoms of the disease are essentially absent. "Protease cleavage site" refers to a site at which proteolytic enzymes catalyze the hydrolysis (break) of peptide bonds between amino acids in polypeptide chains. In the present invention, the phrase "stringent hybridization conditions" or 25 "stringent conditions" refers to conditions under which a compound of the invention will hybridize to its target sequence, but to a minimal number of other sequences. The term "subject" refers to any warm-blooded animal, preferably a human. "Tag" refers to a marker or marker sequence used to isolate or purify a molecule containing the tag. An exemplary tag includes a 6xHis (i.e., a sequence of six histidines) 30 tag . 48 "T cell" refers to a type of lymphocyte that can mount an antigen-specific response to an antigen and which plays a role in humoral and cellular immune responses. "Target Binding Domain (TBD)" refers to all or part of an immunoglobulin heavy chain constant region (e.g., CHI (all or part)-CH2-CH3). 5 The phrase "therapeutically effective amount" refers to an amount of an agent (e.g., a chimeric antigen or a polynucleotide encoding a chimeric antigen) sufficient to elicit an effective B cell, cytotoxic T lymphocyte (CTL) and/or helper T lymphocyte (Th) response to the antigen and to block or to cure or at least partially arrest or slow symptoms and/or complications of a disease or disorder. A subset of T cells function as 10 T helper cells by secreting cytokines that help activate B cells to secrete antibodies or help anotherr T cell subset to become effector cytotoxic T lymphocytes (CTLs). The terms "treating" and "treatment" as used herein cover any treatment of a condition treatable by a chimeric antigen in an animal, particularly a human, and include: (i) preventing the condition from occurring in a subject which may be predisposed to the 15 condition but has not yet been diagnosed as having it; (ii) inhibiting the condition, e.g., arresting or slowing its development; or (iii) relieving the condition, e.g., causing regression of the condition or its symptoms. As used herein, an agent that is "therapeutic" is an agent that causes a complete abolishment of the symptoms of a disease or a decrease in the severity of the symptoms 20 of the disease. "Xenotypic" means originating from a species other than the host. For example, a recombinantly expressed antibody cloned from a mouse genome would be xenotypic to a human but not to a mouse, regardless of whether that recombinantly expressed antibody was produced in a bacterial, insect, human, or mouse cell. Thus, in the context of a 25 chimeric antigen of the invention, a xenotypic TBD (e.g., a xenotypic antibody molecule or a xenotypic antibody fragment) is a TBD derived from a species other than the one to which the chimeric antigen. C. Novel chimeric Antigens 30 The invention provides chimeric antigens for eliciting an immune response comprising an immune response domain and a target binding domain, wherein the target 49 binding domain comprises an antibody fragment. In accordance with the present invention, the chimeric antigen, preferably, is capable of binding to a Fc receptor and/or to a macrophage mannose receptor. The antibody fragment can be xenotypic to the host or not xenotypic to the host. 5 In preferred embodiments of the invention, the chimeric antigen is capable of inducing humoral and/or cellular immune responses. The cellular immune response can include a Thl response, a Th2 response, and/or a cytotoxic T lymphocyte (CTL) response. In yet another preferred embodiment, the chimeric antigen elicits a multi-epitopic immune response. The multi-epitopic immune response can include a response to at least 10 one epitope of the immune response domain and/or a response to at least one epitope of the target binding domain. Alternatively, the multi-epitopic response may be limited to a response to more than one epitope of the immune response domain. The chimeric antigen of the present invention comprises two portions, namely an immune response domain containing an antigenic sequence (such as a viral antigen), and 15 a target binding domain containing an antibody fragment (FIG.1). In a preferred embodiment, the immune response domain can be linked to the target binding domain by any method known to those of skill in the art. Linkers for linking the immune response domain to the target binding domain can include, but are not limited to, covalent peptide linkages, chemical conjugation, leucine zippers and biotin/avidin. In a preferred 20 embodiment, the immune response domain and target binding domain are cloned as a single fusion protein. The covalent peptide linkage of the fusion protein may comprise additional peptide sequences, such as SRPQGGGS or VRPQGGGS (SEQ ID NO: 1). In yet another preferred embodiment, various immune response domains are biotinylated and the target binding domain is generated with streptavidin as a fusion protein to 25 facilitate the production of a wide assortment of chimeric antigens. Alternatively, the immune response domain and the target binding domain each can be expressed as a fusion to a leucine zipper moiety, which will cause the two portions of the chimeric antigen to associate upon mixing. Finally, the immune response domain and target binding domains can be expressed separately and then chemically conjugated using 30 methods known to one of skill in the art. Exemplary methods include use of protein cross-linkers, such as dimethyl suberimidate, to covalently attach the two domains. 50 The immune response domain primarily provides the antigenic portion of the chimeric antigen. The immune response domain comprises at least one antigenic portion of the entity to which an immune response is desired. The chimeric antigen, optionally, can comprise more than one immune response domain. In preferred embodiments, the 5 immune response domain comprises at least one antigenic portion of an infectious agent, such as a viras or an obligate intracellular parasite, or of a cancer antigen. More preferably, the immune response domain comprises at least one antigenic portion of an infectious virus. Examples of preferred infectious viruses include: Retroviridae (e.g., human 10 immunodeficiency viruses, such as Human Immunodeficiency Virus- 1 (HIV-1), also referred to as HTLV-IH, LAV or HTLV-III/LAV, or HIV-III; and other isolates, such as HIV-LP; Picornaviridae (e.g., polio viruses, hepatitis A virus; enterovirases, human coxsackie viruses, rhinoviruses, echoviruses); Calciviridae (e.g., strains that cause gastroenteritis); Togaviridae (e.g., equine encephalitis viruses, rubella viruses); Flaviridae 15 (e.g., Hepatitis C virus, dengue viruses, encephalitis viruses, yellow fever viruses); Coronaviridae (e.g., coronaviruses); Rhabdoviridae (e.g., vesicular stomatitis viruses, rabies viruses); Filoviridae (e.g., Ebola viruses); Paramyxoviridae (e.g., parainfluenza viruses, mumps virus, measles virus, respiratory syncytial virus); Orthomyxoviridae (e.g., influenza viruses); Bungaviridae (e.g., Hantaan viruses, bunga viruses, phleboviruses and 20 Nairo viruses); Arena viridae (hemorrhagic fever viras); Reoviridae (e.g., reoviruses, orbiviruses and rotavirases); Birnaviridae; Hepadnaviridae (human Hepatitis B viras (HBV), duck Hepatitis B viras (DHBV)); Parvoviridae (parvo virases); Papovaviridae (papilloma virases, polyoma virases); Adenoviridae (most adenovirases); Herperviridae (herpes simplex virus (HSV) 1 and 2, varicella zoster virus, cytomegalovirus (CMV), 25 Epstein-Barr viruses, herpes virases); Poxviridae (variola virsues, vaccinia virases, pox viruses); and Iridoviridae (e.g., African swine fever viras); and unclassified virases (e.g., the agent of delta hepatitides, the agents of non-A, non-B hepatitis (class 1 -internally transmitted); class 2- arenterally transmitted; Norwalk and related virases, and astrovirases). In some embodiments of the invention, the immune response domain of the 30 chimeric antigen includes at least one antigenic portion of one or more proteins selected from the group consisting of HBV proteins, DHBV proteins, and HCV proteins. 51 Particularly preferred HBV proteins for use in the present invention include, but are not limited to, HBV S 1/S2, HBV S 1/S2/S, HBV Core, HBV Core ctm (C-terminal modified), HBV e-antigen, and HBV polymerase. Particularly preferred DHBV proteins for use in the present invention include, but are not limited to, DHBV PreS/S, DHBV PreS, DHBV 5 Core, and DHBV polymerase. Particularly preferred HCV proteins for use in the present invention include, but are not limited to, HCV Core (1-191), HCV Core (1-177), HCV El-E2, HCV El, HCV E2, HCV NS3, HCV NS5A and NS4A. Other preferred viral antigens for use in the present invention include HIV gp120, HSV alkaline nuclease and human papilloma viras (HPV) capsid proteins LI and L2, and early region proteins HPV 10 El, HPV E2, HPV E4, HPV E5, HPV E6, and HPV E7. Examples of preferred obligate intracellular parasites include: Tetrahymena sp. (e.g. T. pyriformis), Plasmodium sp. (e.g. P. falciparum), Ciyptospiridium sp., Spraguea sp. (e.g. S. lophii), Giardia sp., Toxoplasma sp. (e.g. T. gondii, T. cruzi), Leishmania sp., Rickettsia sp. (e.g. R. prowazekii), Chlamydia sp., Mycobacterium sp. (e.g. M. 15 tuberculosis), Legionella sp., Listeria sp., (e.g. L. monocytogenes), Coxiella sp. (e.g. C. brunette), Shigella sp., Erlichia sp., and Bartonelia sp. Preferred cancer antiens include: prostate specific antigen (PSA), prostate specific membrane antigen (PSMA), MUC 1, CA 125, WT1, Her-2/neu, carcinoembryonic antigen (CEA), MAGE-3, MART-1, gplOO, NY-ESSO-1, CA19.9, TAG72, CA 15.3, CA 27.9, gp 120, prostatic acid phosphatase 20 (PAP), Heatshock proteins, alpha-fetoprotein (AFP), Telomerase, and ras. In yet another embodiment of the invention, the immune response domain of the chimeric antigen includes 6xHis tag fused to the one or more antigenic portions. In accordance with the present invention, the chimeric antigen is a protein capable of binding to an Fc receptor and/or CD206 on an antigen presenting cell, particularly a 25 dendritic cell, and is subsequently transported into the antigen presenting cell by receptor-mediated uptake. In accordance with the present invention, the presence of an antibody fragment augments the uptake of the chimeric antigen through the Fc receptor on antigen-presenting cells, specifically dendritic cells. By virtue of this specific binding and internalization, the viral antigen is processed and presented as foreign. Thus, an 30 immune response can be effectively elicited to a previously tolerated antigen. The target binding domain comprises an antibody fragment that can be xenotypic to the host or not 52 xenotypic to the host. In a preferred embodiment of the invention, the antibody fragment comprises a murine Fc fragment. In a more preferred embodiment of the invention, the target binding domain comprises a Fc fragment, a hinge region, and a portion of the CRI region, and the chimeric antigen comprises a peptide linker suitable for linking the target 5 binding domain to the immune response domain. In another preferred embodiment, the target binding domain comprises an immunoglobulin heavy chain fragment, and optionally, further comprises a hinge region. In a particularly preferred embodiment, the heavy chain fragment comprises amino acids, VDKKI (SEQ ID NO: 2) of the CHI domain and/or part or all of the CH2 and CH3 domains. 10 As discussed above, antigens that are bound and internalized by CD206 can be presented by both MHC Class I and Class II, thus eliciting both a cellular and humoral immune response. Accordingly, in a preferred embodiment, the chimeric antigen is glycosylated. The immune response domain and/or the target binding domain can be glycosylated. In a particularly preferred embodiment, the chimeric antigen is mannose 15 glycosylated by either high mannose glycosylation or by pauci mannose glycosylation (Jarvis, Yirology 310:1-7 (2003)). A composition of the present invention includes a chimeric antigen comprising an immune response domain (IRD) and a target binding domain (TBD). In preferred embodiments of the invention, the IRD portion is capable of inducing humoral and/or T 20 cell responses, and the target binding portion is capable of binding an APC, such as a dendritic cell. The chimeric antigen of the present invention may also include one or more of the following: a hinge region of an immunoglobulin (or a segment thereof), a CHI region of an immunoglobulin (or a segment thereof), a peptide linker, a protease cleavage site, and a tag suitable for use with a purification protocol. A chimeric antigen 25 of the present invention is capable of binding to and activating an APC. Generally, but not necessarily, the IRD is N-terminal of the TBD. In some embodiments of the invention, the IRD of the chimeric antigen includes one or more (e.g., 2, 3, 4, 5, 6, 7, 8, 9, or 10) proteins (antigens) selected from the group comprising: one or more HCV proteins such as those desecribed herein or one or more 30 recombinant HCV proteins. Between such proteins there can optionally be a linker such as any of the linkers disclosed herein. In the chimeric antigen of the invention, 53 immunogenic fragments of these antigens, rather than the full-length antigens, can be uses. Where more than one antigen is present in a chimeric antigen, only full-length, only immunogenic fragments, or mixturs of full-length antigens and full-length proteins can be used. 5 The chimeric antigens of the invention can be monomeric (i.e., they contain a single unit comprising an IRD and a TBD) or they can be multimeric (i.e., they can contain multiple units, each comprising an IRD and a TBD). Multimers can be, for example, dimers, trimers, tetramers, pentamers, hexamers, septamers, or octamers. In such multimers, the individual units can be identical or different or some can be identical 10 and others different. Fig. IX depicts a dimeric chimeric antigen. In yet another embodiment of the invention, the IRD of the chimeric antigen includes a 6xHis-peptide fused to one or more HCV proteins, or one or more recombinant HCV proteins. In some embodiments of the invention, the TBD of the chimeric antigen can be an 15 antibody fragment. The TBD can be of the same species as the host (subject) to which the relevant chimeric antigen is to be administered. On the other hand, in preferred embodiments of the invention, the TBD of the chimeric antigen is an antibody fragment xenotypic to the host. For example, if the host is a human, an exemplary xenotypic antibody fragment is a non-human animal antibody fragment, such as a mouse antibody 20 fragment. In certain embodiments of the invention, the xenotypic antibody fragment comprises a murine Fc fragment. In the most preferred embodiments of the invention, the TBD comprises a xenotypic Fc fragment (or a segment thereof), a hinge region (or a segment thereof), a CHI region (or a segment thereof), and a peptide linkage suitable for linking the target binding domain to the IRD. 25 The present invention also comprises the use of linking molecules to join the IRD to the TBD. Exemplary linker molecules include leucine zippers, and biotin/avidin. Other linkers that can be used (for example in fusion proteins) are peptide sequences. Such peptide linkers are generally about two to about 40 amino acids (e.g., about 4 -10 amino acids) in length. An exemplary peptide linkers include the amino acid sequence 30 SRPQGGS (SEQ ID NO: 1X). Other linkers are well known in the art and are generally glycine and/or alanine rich to allow for flexibility between the regions they join. 54 Generally, in the chimeric antigens of the invention, the IRD and the TBD are not joined by a physical antigen-antibody interaction between an antigen binding part of the TBD (e.g., an antibody molecule or fragment of an antibody molecule) and an appropriate antigenic epitope on the IRD. 5 In one embodiment, the chimeric antigen of the present invention is a fusion protein having two portions, namely an IRD containing an antigenic sequence (such as a viral antigen(s)), and a TBD containing a xenotypic Fc fragment. The xenotypic murine Fc fragment binds to specific receptors on APC, specifically dendritic cells. The binding region of the chimeric antigen thus targets antigen-presenting cells specifically. The 10 internal machinery of the APC then processes the chimeric antigen and presents specific peptides on MHC class I and class II molecules to contact and activate T cells and generate humoral and cellular immune responses to clear infected cells or other appropriate undesirable cells, e.g., cancer cells. In a further embodiment, the chimeric antigen can be a fusion protein having two 15 portions, namely a modified viral antigen or antigens, antigenic protein fragments or peptides, or any of these with glycosylation at specific sites, and a xenotypic murine Fc fragment, which can also be glycosylated. In yet another embodiment, the invention provides a further modified chimeric antigen, wherein the antigen (IRD) is biotinylated and the TBD (e.g., Fc fragment) is 20 conjugated with avidin (e.g., streptavidin) in, for example, a fusion protein. Such an avidin-conjugated TBD facilitates the production of a wide assortment of IRD-TBD conjugates. Naturally it is appreciated that the IRD can be conjugated with avidin (e.g., in the form of a fusion protein) and the TBD (e.g., Fc fragment) can be biotinylated. In yet another embodiment, the invention provides an association between the 25 IRD (antigen) and the TBD (e.g., antibody Fc fragment) through chemical conjugation. An embodiment of the present invention includes the use of recombinant antigens of HCV fused to an antibody fragment by molecular biological techniques, production of the fusion proteins in a baculovirus expression system and their use as therapeutic vaccines against chronic HCV infections. The present invention provides an efficient 30 method to deliver a HCV antigen to APCs in vivo so as to generate a broad immune response, a Th1 response involving CTLs and a Th2 (antibody) response. The 55 immunogenicity of pre-selected viral antigen (e.g., one unrecognized by a host immune system) can be increased by the presence of a xenotypic antibody fragment as well as by the presence of specific glycosylation introduced in the insect cell expression system. The antigen-antibody fragment fusion protein, due to the presence of the antibody 5 component, will bind to specific receptors present on various cells of the immune system (e.g., APC), including dendritic cells, macrophages, monocytes, B cells, and granulocytes. The fusion proteins administered to either humans or animals will be internalized by APCs, especially DCs, will be hydrolyzed to small peptides and presented on the cell surface, complexed with MHC Class I and/or MHC Class II molecules to T 10 cells have antigen specific T cell receptors (TCR) of the appropriate specificity. In this way the chimeric antigens (fusion proteins) can elicit a broad immune response and clear the viral infection. As used herein, the term "Target Binding Domain (TBD)" refers to all or part of an immunoglobulin heavy chain constant region, which is an antibody fragment capable 15 of binding to an Fc receptor on an APC. In accordance with the present invention, the TBD is a protein capable of binding to an Fc receptor on an APC, particularly a dendritic cell, and is subsequently transported into the APC by receptor-mediated uptake. In accordance with the present invention, the presence of an Fc fragment augments the uptake of the chimeric antigen through the Fc receptor on APCs, specifically DC. By 20 virtue of the specific uptake, the viral antigen is processed and presented as foreign; thus, an immune response is effectively elicited to the viral antigen that, on its own, was tolerated by the host or elicited a very weak immune response in the host. Also, in accordance with the present invention, the chimeric antigen, preferably, is capable of binding to a macrophage mannose receptor/C-type lectin receptors. The 25 macrophage mannose receptor (MMR), also known as CD206, is expressed on APC such as DCs. This molecule is a member of the C-type lectin family of endocytic receptors. Mannosylated chimeric antigen can be bound and internalized by CD206. In general, exogenous antigen is thought to be processed and presented primarily through the MHC class II pathway. However, in the case of targeting through CD206, there is evidence that 30 both the MHC class I and class II pathways are involved [Apostolopoulos et al. (2000) Eur. J. Immunol. 30:1714; Apostolopoulos et al. (2001) Curr. Mol. Med. 1:469; 56 Ramakrishna et al. (2004) J. Immunol. 172:2845-2852]. Thus, monocyte-derived dendritic cells loaded with chimeric antigen that specifically targets CD206 will induce both a potent class I-dependent CD8+ CTL response and a class II-dependent proliferative T helper response [Ramakrishna et al. (2004) J. Immunol. 172(5):2845-52]. 5 An exemplary TBD is derived from Mouse anti-HBVsAg mAb (Hybridoma 2C12) as cloned in pFastBac HTa expression vector, and expressed in an insect cell expression system (Invitrogen, Carlsbad, CA, USA). This TBD consists of part of CHI (having the amino acid sequence VDKKI; SEQ ID NO:2X), and Hinge-CH2-CH3 from N terminal to C-terminal of the mouse anti-HBV sAg mAb. The constant region of the IgGI 10 molecule for the practice of the present invention can contain a linker peptide, part of CHI-hinge and the regions CH2 and CH3. The hinge region portion of the monomeric TBD can form disulphide bonds with a second TBD molecule. The protein can be expressed as an N-terminal fusion protein with a 6xHis tag, a seven amino acid rTEV (recombinant tobacco etch virus) protease cleavage site and the N-terminal fusion of the 15 Target Binding Domain (TBD) of the xenotypic (murine) mAb raised against HBV sAg (Hybridoma 2C12). The exemplary TBD is a fragment of the constant chain of the IgGI mAb from 2C12 with the sequence of amino acids comprising the 8 amino acid peptide linker, five amino acids of the CHI region, the hinge sequences, CH2 and CH3 region sequences and, optionally, a C-terminal peptide of ten additional amino acids encoded by 20 nucleotides derived from the expression vector. The exemplary TBD fragment defined herein forms the parent molecule for the generation of fusion proteins with antigens derived from HCV virus. D. Novel Methods of Utilizing Chimeric Antigens 25 The invention includes methods of eliciting an immune response comprising administering, to a subject, a composition comprising a chimeric antigen of the invention. In order to provide efficient presentation of the antigens, the inventors have developed a novel chimeric antigen comprising an immune response domain and a target binding domain, wherein the target binding domain comprises an antibody fragment. While not 30 being limited to a particular theory of the invention, this molecule, by virtue of the antibody fragment binds to specific receptors on the antigen presenting . cells, and the 57 viral antigen is processed and presented complexed with Major Histocompatibility Complex (MHC) Class I and Class II. Such processing and antigen presentation by MHC Class I elicits an increased response by cytotoxic T-lymphocytes, resulting in the elimination of any infectious agent associated with the antigen of the immune response 5 domain. In addition, antigen presentation by MHC Class II molecules, elicits a humoral response that also aids in clearance of the antigen from the infected cell and/or from circulation. The invention also includes methods of breaking tolerance comprising administering a chimeric antigen of the invention to a subject. In the presentation of 10 antigens to elicit a cellular and/or humoral immune response by the use of chimeric antigens, the antigens, which were treated as "self during a chronic infection, are recognized as "foreign." Accordingly, the host's immune system will mount a CTL response to eliminate infected cells. At the same time, antibodies elicited in response to the chimeric antigen will bind to the infectious agent and remove it from the circulation 15 or block binding of the infectious agent to host cells. Accordingly, the present invention is designed to produce chimeric antigens that can induce a broad immune response in subjects who have chronic infections that are otherwise tolerated by the host immune system. In a preferred embodiment, the chimeric antigen breaks tolerance to an antigen in a subject who is chronically infected with an infectious agent, such as a virus or parasite, 20 or who has a cancer. More preferably, the infectious agent is resident inside a host cell at some point during its life cycle. In a preferred embodiment, the immunogenicity of the pre-selected antigen unrecognized or tolerated by the host immune system is increased due to the presence of the antibody fragment as well as by the presence of glycosylation introduced in an 25 eukaryotic, e.g. insect, cell expression system. Such a chimeric antigen, due to the presence of the antibody component and glycosylation, will bind to specific receptors present on various immune cell types including dendritic cells, macrophages, B cells and granulocytes. Yet another aspect of the invention provides methods of activating antigen presenting 30 cells comprising contacting the antigen presenting cells with a chimeric antigen of the invention. The invention also provides methods of enhancing antigen presentation in an 58 antigen presenting cell comprising contacting the antigen presenting cell with a composition comprising a chimeric antigen of the invention. The chimeric antigen can be contacted with the antigen presenting cells, preferably dendritic cells, in vivo or ex vivo. In a preferred embodiment, contacting the chimeric antigen with antigen presenting cells 5 activates the antigen presenting cells and enhances antigen presentation of more than one epitope. This multi-epitopic response can include presentation of one or more epitopes of the immune response domain and/or presentation of one or more epitopes of the target binding domain. The invention also provides methods of treating an immune-treatable condition 10 comprising administering, to a subject in need thereof, a therapeutically effective amount of a chimeric antigen of the invention. In a preferred embodiment, the immune-treatable condition is an infection or a cancer. The infection can be a viral infection, a parasitic infection or a bacterial infection. Preferably, the infection will have a stage during which the infectious agent is found within a host cell. More preferably, the immune-treatable 15 condition is a chronic viral infection. Most preferably, the immune-treatable condition is a chronic hepatitis B viral (HBV) infection or a chronic hepatitis C viral (HCV) infection. For the treatment of HBV, the immune response domain preferably comprises at least one antigenic portion of a protein selected from the group consisting of a HBV Core protein, a HBV S protein, a HBV S1 protein, a HBV S2 protein, and combinations thereof For the 20 treatment of HCV, the immune response domain preferably comprises at least one antigenic portion of a protein selected from the group consisting of a HCV Core (1-191) protein, a HCV Core (1-177) protein, a HCV El protein, a HCV E2 protein, a HCV El E2 protein, a HCV NS3A protein, a HCV NS5A protein, and combinations thereof. In a preferred embodiment, administration of the chimeric antigen elicits a greater 25 immune response than administration of the immune response domain alone. The amplitude of the immune response can be measured, for example, (i) by the amount of antigen-specific antibody present in the subject; (ii) by the amount of interferon-y secreted by T cells in response to being exposed to antigen presenting cell loaded with the chimeric antigen or immune response domain alone; or (iii) by the amount of antigen 30 specific CD8 T cells elicited in response to being exposed to antigen presenting cell loaded with the chimeric antigen or immune response domain alone. 59 The chimeric antigen can be evaluated for its efficacy in generating an immune response by presenting the chimeric antigen to dendritic cells ex vivo or in vivo. The dendritic cells process and present the chimeric antigen to T-lymphocytes, which are evaluated for proliferation of T cells and for the production of interferon-y as markers of T cell 5 response. Specifically, in the ex vivo situation, naive dendritic cells are isolated from peripheral blood. Activation of the T cells by the dendritic cells is evaluated by measuring markers, e.g. interferon-y levels, by a known procedure. See, e.g., Beriyn, et ab, Clin. Immunol 101(3):276-283 (2001). An increase in the percentage of T cells that secrete interferon-y by at least 50% predicts efficacy in vivo. In the case of the in vivo 10 situation, the chimeric antigen is directly introduced parenterally in the host where available dendritic and other antigen-processing cells have the capacity to interact with antigens and to process them accordingly. Additionally, the invention includes methods of vaccinating a subject against an infection comprising administering a chimeric antigen of the present invention to the 15 subject. The subject can be prophylactically or therapeutically vaccinated. Preferably, the infection is a viral infection. The bifunctional nature of the molecule helps to target the antigen to antigen-presenting cells, e.g. dendritic cells, making it a unique approach in the therapy of chronic infectious diseases by specifically targeting the antigen presenting cells with the most effective stoichiometry of antigen to antibody. This is useful to the 20 development of therapeutic vaccines to cure chronic viral infections such as Hepatitis B, Hepatitis C, Human Immunodeficiency Viras, Human Papilloma Virus and Herpes Simplex Viras, obligate intracellular parasites and may also be applicable to all autologous antigens in diseases such as cancer and autoimmune disorders. The administration of these fusion proteins can elicit a broad immune response from the host, 25 including both cellular and humoral responses. Thus, they can be used as therapeutic vaccines to treat subjects that are immune tolerant to an existing infection, in addition to being useful as prophylactic vaccines to immunize subjects at risk for developing a particular infection. Another aspect of the invention provides methods of enhancing antigen presentation by 30 APCs, said method comprising administering, to the APCs, a chimeric antigen that comprises an immune response domain and a target binding domain, wherein the target 60 binding domain comprises an antibody fragment (e.g., a xenotypic antibody fragment). In a preferred embodiment, the APCs are dendritic cells. An aspect of the invention relates to methods of activating APCs comprising contacting the APC with a chimeric antigen that comprises an immune response domain 5 and a target binding domain, wherein the target binding domain comprises an antibody fragment (e.g., a xenotypic antibody fragment). In a preferred embodiment, the APC is contacted with the chimeric antigen in vivo. In another preferred embodiment, the contacting takes place in a human. Yet another aspect of the invention provides methods of eliciting an immune 10 response, said method comprising administering to an animal a chimeric antigen that comprises an immune response domain and a target binding domain, wherein the target binding domain comprises an antibody fragment (e.g., a xenotypic antibody fragment). The immune response can be a humoral and/or cellular immune response. In a preferred embodiment, the cellular immune response is a Thl, a Th2, and/or a CTL response. 15 Another aspect of the invention provides methods of treating immune-treatable conditions comprising administering, to an animal in need thereof, a chimeric antigen that comprises an immune response domain and a target binding domain, wherein the target binding domain comprises a xenotypic antibody fragment. Preferably, the immune treatable condition is a chronic hepatitis C viral infection. For the treatment of HCV, 20 preferably the immune response domain comprises a protein selected from the group consisting of a HCV Core (1-191) protein, a HCV Core (1-177) protein, a HCV El protein, a HCV E2 protein, a HCV El -E2 protein, a HCV NS2 protein, a HCV NS3 protein, a HCV NS4A protein, a HCV NS4B protein, a HCV NS5A protein, a HCV NS5B protein, a HCV p7 protein, and combinations thereof. 25 Another aspect of the invention provides methods of vaccinating an animal against a viral infection comprising administering to the animal a chimeric antigen that comprises an immune response domain and a target binding domain, wherein the target binding domain comprises an antibody fragment. The method of the invention can prophylactically or therapeutically vaccinate the animal against the viral infection. 30 The present invention also comprises methods of using the compositions of the present invention to bind and activate APCs, such as DCs. The present invention also 61 comprises methods of using the compositions of the present invention to activate T cells. The present invention also comprises a method of delivering an antigen to an immune system cell, such as an APC. The present invention also comprises compositions and methods for activating a humoral and/or cellular immune response in an animal or 5 human, said method comprising administering one or more chimeric antigens of the present invention. Following cloning and expression, the chimeric antigen is evaluated for its efficacy in generating an immune response. Evaluation involves presenting the chimeric antigen to DCs ex vivo or in vivo. The DCs are evaluated for the binding and 10 internalization of the chimeric antigens. The naYve DCs loaded with the chimeric antigen are presented to T-lymphocytes and evaluated for the production of interferon-y as a marker of a T cell response. Specifically, in the ex vivo situation, monocytes are isolated from peripheral blood and differentiated to DCs. DCs bind, internalize, process and present antigen to naive autologous T-lymphocytes. The T cells, which recognize the 15 processed antigens presented by DCs, are activated into effector cells, e.g. helper T cells or cytotoxic T-lymphocytes. Activation of the T cells by the dendritic cells is then evaluated by measuring markers, e.g. interferon-y levels, by a known procedure [e.g., Berlyn, et al. (2001) Clin. Immunol. 101(3):276-283]. An increase in the percentage of T cells that produce interferon-y by at least 50% over background predicts efficacy in vivo. 20 In preferred embodiments, the percentage increase is at least 55%, 60%, 65%, 70%, 75%, 80%, 90% or 100%. In the case of the in vivo situation, the chimeric antigen is directly introduced parenterally in the host, where available dendritic and other antigen processing cells have the capacity to interact with all antigens and process them accordingly. 25 E. Methods of Making Chimeric Antigens One aspect of the invention provides methods for producing a chimeric antigen comprising (a) providing a microorganism or cell line, preferably a eukaryotic, more preferably, a non-mammalian microorganism or cell line that comprises a polynucleotide encoding a chimeric antigen; and (b) culturing said microorganism or cell line under 30 conditions whereby the chimeric antigen is expressed. Preferably, the microorganism or 62 cell line is a yeast, a plant cell line or an insect cell line. More preferably, the cell line is an insect cell line selected from the group consisting of Sf9, Sf21 , Drosophila S2, and High Five</div>
    <div class="description-paragraph style-scope patent-text">TM</div>
    <div class="description-paragraph style-scope patent-text">. One embodiment of the present invention uses established recombinant DNA 5 technology for producing the fusion proteins of selected antigen(s) and the target binding domain that are necessary in the practice of the invention. Fusion protein constructs are generated at the DNA level incorporating specific restriction enzyme sites, which are exploited in incorporating the desired DNA fragment into expression vectors, and used to express the desired fusion proteins in a heterologous expression system. As used herein, 10 the term "vector" denotes plasmids that are capable of carrying the DNA, which encode the desired protein(s). Preferred plasmid vectors for use in the present invention include, but are not limited to, pFastBac HTa and the corresponding recombinant "bacmids" generated in DHlOBac T M E. coli (Invitrogen). A gene encoding a target binding domain can be obtained from any antibody 15 producing cell, for example a hybridoma producing a monoclonal antibody, via polymerase chain reaction (PCR). To facilitate later cloning steps, it is preferable to design the oligonucleotide primers to add unique restriction enzyme recognition sites. Similarly, the antigenic portions of the immune response domain can be obtained from any cell or virus RNA or DNA containing a gene encoding an antigenic portion of the 20 desired target. Preferably, PCR is used to obtain DNA encoding the antigenic portions of the immune response domain and the PCR primers are designed to add unique restriction enzyme recognition sites to facilitate cloning. However, any recombinant DNA method may be used to obtain the DNA encoding the antigenic portions of the immune response domain. The polynucleotides encoding the target binding domain and immune response 25 domain can then be combined in a single construct using standard cloning techniques. Alternatively, the separate domains can be cloned with DNA encoding linkers, such as leucine zippers, streptavidin or biotinylation signals. Preferably, a baculovirus system is used to express the chimeric antigen of the invention not only because large amounts of heterologous proteins are produced, but also 63 because post-translational modifications, such as phosphorylation and glycosylation, of eukaryotic proteins occur within the infected insect cell. Since cloning directly in insect cells can be difficult, it is preferable to generate the polynucleotide encoding the chimeric antigen in a bacterial system and to transfer the final construct into a baculovirus/insect 5 cell expression system. Transfer systems, e.g., the Bac-To-BacTM system (Invitrogen), are known to those of skill in the art. The Bac-to-BacTM system utilizes site-specific transposition with the bacterial transposon Tn7 to transfer the gene of interest into a E. coli-insect cell shuttle vector (bacmid). The resulting recombinant bacmids are transfected into insect cells to generate baculovirases that express recombinant proteins. 10 In order to produce baculovirases, the bacmid is transfected into insect cells, such as Sf9 cells. Following transfection, the cells are incubated for a period of time sufficient to expand the baculoviral population. The medium containing baculovirus is collected and stored at 4' C in the dark. The transfection can be verified by checking for production of baculoviral DNA by subjecting the viral culture to PCR utilizing primers 15 specific for the desired DNA insert. The expression of the heterologous protein in the cells can be verified by any method known in the art, e.g. SDS polyacrylamide gel electrophoresis (SDS-PAGE) or Western blotting. Recombinant bacmids of standardized multiplicity of infection (MOI) are used to infect insect cells. Cells are seeded at a density of approximately 3 x 105 cells/mL and 20 incubated in suspension culture at 27.5'C with shaking until the cell density reached approximately 2-3 x 106 cells/mL. Standardized amounts of the respective recombinant baculovirus are then added to the cells. The incubation temperature is 27.5'C and the appropriate infection period is standardized for individual protein expression. The cells are harvested by centrifugation and used for the purification of the recombinant proteins. 25 Unused portions of cells can be snap frozen in liquid nitrogen and stored at -70'C. Chimeric antigens, preferably, are purified under denaturing conditions. Cells expressing chimeric antigens are lysed in a denaturing buffer, e.g., a buffer containing 6 M guanidinium-HCL Lysis can be increased by mechanical means, such as sonication. The lysate is centrifuged to remove unbroken cells and cell debris. The supernatant is then 30 loaded on to a Ni-NTA Super Flow (Qiagen) bead column pre-equilibrated with lysis 64 buffer. Following loading, the column is washed with a buffered denaturing solution, preferably containing 6 M guanidinium-HCl at approximately pH 8. At this point the denaturant can be exchanged to, e.g., 8 M urea in a buffered solution. The lysis, loading and wash buffers preferably contain a low concentration, e.g., 1-40 mM imidazole. After 5 buffer exchange, the column should be washed with buffer until the OD 28 n drops to, for example, &lt;0.1. The bound protein can be eluted with a buffer containing 8 M urea, and 250 mM imidazole, pH 8 (Elution Buffer). The fractions containing the protein are pooled and dialyzed at 4'C against multiple changes of low (e.g., 100 mM) salt denaturing dialysis buffer, preferably containing 8 M urea. The dialyzed protein is then 10 loaded onto an ion exchange column, such as DEAE (diethylamino ethyl). In a preferred embodiment, dithiothreitol (DTT) or other reducing agent is added to the protein prior to loading onto the ion exchange column. The chimeric antigen will pass through a DEAE column. Therefore, the DEAE flowthrough is collected and dialyzed in a stepwise manner against buffers containing decreasing concentrations of denaturant. In an 15 exemplary method, the protein is then dialyzed against buffered 4 M urea for at least 12 hours, then against buffered 2 M urea for at least 12 hours, then against buffered 1 M urea for at least 12 hours, then against buffered 0.5 M urea for at least 12 hours and finally dialyzed against buffer containing no denaturant for at least 12 hours, preferably followed by two additional periods of 12 hours dialysis against fresh buffer containing no 20 denaturant. Purified, refolded proteins can be concentrated and characterized using standard biochemical techniques including, e.g., SDS gel electrophoresis, isoelectric focusing, or western blot analysis using antibodies against different domains of the expressed protein. The practice of the present invention will employ, unless otherwise indicated, 25 conventional techniques of molecular biology, microbiology, recombinant DNA, and immunology, which are within the skill of the art. Such techniques are explained fully in the literature. See e.g., Sambrook et ab, Molecular Cloning: A Laboratory Manual. 2"d Ed., New York: Cold Spring Harbor Press, 1989; and Ausubel et ab, Current Protocols in Molecular Biology. Wiley Interscience Publishers, (01995, as Supplemented April 2004, 30 Supplement 66). 65 One aspect of the invention provides methods for producing a chimeric antigen comprising (a) providing a microorganism or cell line (or cell), preferably a eukaryotic, more preferably, a non-mammalian microorganism or cell line (or cell), that comprises a polynucleotide encoding a chimeric antigen; and (b) culturing said microorganism or cell 5 line (or cell) under conditions whereby the chimeric antigen is expressed. Preferably, the microorganism or cell line (or cell) is a yeast, a plant cell line (or cell) or an insect cell line (or cell). More preferably, the cell line (or cell) is an insect cell line (or cell) selected from the group consisting of Sf9, Sf21, expresSF®, Drosophila S2, and High FiveTM cell lines or cells. 10 The present invention uses established recombinant DNA technology for producing the fusion proteins of selected antigen(s) and the TBD that are necessary in the practice of the invention. Fusion protein constructs are generated at the DNA level incorporating specific restriction enzyme sites, which are exploited in incorporating the desired DNA fragment into expression vectors, and used to express the desired fusion 15 proteins in a heterologous expression system. As used herein, the term "vector" denotes plasmids that are capable of carrying the DNA, which encode the desired protein(s). The plasmid vectors used in the present invention include, but are not limited to, pFastBac HTa and the corresponding recombinant "BACMIDS" (bacterial artificial chromosomes) generated in DH1OBac T M E. coli (Invitrogen). It is possible to mobilize the ORF of the 20 desired proteins and produce other recombinant plasmids for expression of the proteins in other systems, (bacterial or mammalian), in addition to the Bac-to-Bac@ baculovirus expression system (Invitrogen), employed in the present invention. The term "expression" is used to mean the transcription of the DNA sequence into mRNA, the translation of the mRNA transcript into the fusion protein. 25 This is achieved by the transposition of the gene of interest into bacmids, transfected into Sf9 insect cells and producing recombinant baculovirus. These are used to infect Sf9 or High FiveTM insect cells, which produce the protein of interest. The recombinant proteins produced may have an N-terminal 6xHis tag, which is exploited in the purification of the proteins by using Ni-NTA Agarose (Qiagen, Hilden, Germany). 30 The proteins may also have an N-terminal rTEV protease or other cleavage site cloned in. The Ni-purified protein is subjected to digestion with, for example, rTEV protease 66 (Invitrogen), which also has an N-terminal 6xHis tag. Following the protease digestion, the mixture can be loaded on to a Ni-NTA agarose column and the pure protein can be washed out, while the 6xHis tagged fragments will be bound to the column. This method of purification is standard procedure and one skilled in the art would be able to 5 understand the methodology without further explanation. Cloning and expression of the DNA sequences, which encode the viral antigen and the Fc fragment of the murine monoclonal antibody to generate the chimeric antigen, can be achieved through two approaches. The first approach involves cloning the two proteins as a fusion protein, while the second approach involves incorporating specific 10 "bio-linkers" such as biotin or streptavidin in either of the molecules, purifying them separately and generating the chimeric antigen. In an exemplary embodiment, the hybridoma 2C12, which produces a monoclonal antibody against the Hepatitis B virus surface antigen, was used as a source of the total RNA for the murine immunoglobulin G. Total RNA was isolated and used to clone the 15 murine Fc fragment. Specifically, the total RNA from a hybridoma cell that expresses murine IgG is isolated using Trizol@ reagent (Invitrogen/Gibco BRL, product catalog number 10551-018, 10298-016; a monophasic solution of phenol and guanidine isothiocyante, as described in U.S. Patent No. 5,346,994). The mRNA was purified from total RNA by affinity chromatography on an oligo-dT column (Invitrogen/Gibco BRL, 20 product catalog number 15939-010). A complementary DNA (cDNA) was produced using reverse transcriptase in a polymerase chain reaction. The oligonucleotide primers were designed to add unique restriction enzyme recognition sites to facilitate cloning. This cDNA was cloned using the Bac-to-Bac@ baculovirus expression system (Invitrogen/Gibco BRL, product catalog number 15939-010). 25 The baculovirus system, preferentially, is used because not only are large amounts of heterologous proteins produced, but also because post-translational modifications, such as phosphorylation and glycosylation, of proteins occur within the infected insect cell. In this expression system, the DNA can be cloned into vectors called pFastBac T M (Invitrogen/Gibco BRL, product catalog number 15939-010). In the Bac-to-Bac@ 30 system, the generation of recombinants is based on site-specific transposition with the bacterial transposon Tn7. The gene of interest is cloned into pFastBac@, which has mini 67 Tn7 elements flanking the cloning sites. The plasmid is transformed into Escherichia coli strain DH1OBac TM (Invitrogen/Gibco BRL, product catalog number 10361-012), which has a baculovirus shuttle plasmid (bacmid) containing the attachment site of Tn7 within a LacZ gene. Transposition disrupts the LacZ gene so that only recombinants produce 5 white colonies and are easily selected for. The advantage of using transposition in E. coli is that single colonies contain only recombinants so that plaque purification and screening are not required. The recombinant bacmids are transfected in insect cells to generate baculoviruses that express recombinant proteins. The Bac-to-Bac@ baculovirus expression system is commercially available from 10 Invitrogen and the procedures used are as described in the company protocols, available, for example, at www.invitrogen.com. The gene of interest is cloned into, for example, pFastBac HTa donor plasmid and the production of recombinant proteins is based upon the Bac-To-BacTM baculovirus expression system (Invitrogen). In the next step, the pFastBac HTa donor plasmid containing the gene of interest 15 is used in a site-specific transposition in order to transfer the cloned gene into a baculovirus shuttle vector (bacmid). This is accomplished in E. coli strain DH1OBacTM. The recombinant pFastBac HTa plasmids with the gene of interest are transformed into DH1OBacTM cells for the transposition to generate recombinant bacmids. Recombinant bacmids are isolated by standard protocols (Sambrook, supra); the 20 DNA sample was used for transfections. In order to produce baculoviruses, the bacmid is transfected into Sf9 insect cells. Following transfection, the cells are incubated under appropriate conditions and the medium containing baculovirus is collected and stored. Once production of baculovirus and the expression of protein have been 25 confirmed, the virus stock is amplified to produce a concentrated stock of the baculovirus that carry the gene of interest. It is standard practice in the art to amplify the baculovirus at least two times, and in all protocols described herein this standard practice was adhered to. After the second round of amplification, the concentration of the generated baculovirus can be quantified using a plaque assay according to the protocols described 30 by the manufacturer of the kit (Invitrogen). The most appropriate concentration of the 68 virus to infect insect cells and the optimum time point for the production of the desired protein is generally also established. DNA encoding proteins of interest are generated by PCR with oligonucleotide primers bearing unique restriction enzyme sites from plasmids that contain a copy of the 5 entire viral genome and cloned with the Fc DNA as a fusion protein. This chimeric protein is purified by Ni-NTA, lectin, protein A or protein G affinity chromatography or other standard purification methods known to those skilled in the art. The second approach for linking the IRD and TBD involves incorporating specific "bio-linkers" such as biotin or avidin (e.g., streptavidin) in either of the 10 molecules, purifying them separately and generating the chimeric antigen. The viral antigens of interest are cloned into plasmids that control the expression of proteins by the bacteriophage T7 promoter. The recombinant plasmid is then transformed into an E. coli strain, e.g. BL21 (DE3) Codon PlusTM RIL cells (Stratagene, product catalog number 230245), which has production of T7 RNA polymerase regulated by the lac repressor. 15 The T7 RNA polymerase is highly specific for T7 promoters and is much more processive (~8 fold faster) than the E. coli host's RNA polymerase. When production of T7 RNA polymerase is induced by isopropylthio-p-D-galactoside (IPTG), the specificity and processivity of T7 RNA polymerase results in a high level of transcription of genes under control of the T7 promoter. In order to couple two proteins together, the tight 20 binding between biotin and avidin (e.g., streptavidin) is exploited. In E. coli, the BirA enzyme catalyzes the covalent linkage of biotin to a defined lysine residue in a specific recognition sequence. The murine Fc fragment is expressed in the baculovirus system, as described above, as a fusion protein with avidin. These two proteins can be mixed to form a dimeric protein complex by biotin-streptavidin binding. 25 The practice of the present invention employs, unless otherwise indicated, conventional techniques of molecular biology, microbiology, recombinant DNA, and immunology, which are within the skill of the art. Such techniques are explained fully in the literature. See e.g., Sambrook, supra; and Ausubel, supra. 30 F. Novel Polynucleotides 69 Another aspect of the invention provides polynucleotides encoding a chimeric antigen comprising a first polynucleotide portion encoding an immune response domain and a second polynucleotide portion encoding a target binding domain. The first and second polynucleotide portions may be located on the same or different nucleotide 5 chains. The invention provides polynucleotides corresponding or complementary to genes encoding chimeric antigens, mRNAs, and/or coding sequences, preferably in isolated form, including polynucleotides encoding chimeric antigen variant proteins; DNA, RNA, DNA/RNA hybrids, and related molecules, polynucleotides or oligonucleotides 10 complementary or having at least a 90% homology to the genes encoding a chimeric antigen or mRNA sequences or parts thereob and polynucleotides or oligonucleotides that hybridize to the genes encoding a chimeric antigen, mRNAs, or to chimeric antigen encoding polynucleotides. Additionally, the invention includes analogs of the genes encoding a chimeric 15 antigen specifically disclosed herein. Analogs include, e.g., mutants, that retain the ability to elicit an immune response, and preferably have a homology of at least 80%, more preferably 90%, and most preferably 95% to any of polynucleotides encoding a chimeric antigen, as specifically described by SEQ ID NOs: 26, 28, 30, 32, 34, 36, 38, 40, 42, 44, 46, and 44. Typically, such analogs differ by only 1 to 15 codon changes. Examples 20 include polypeptides with minor amino acid variations from the natural amino acid sequence of a viral antigen or of an antibody fragment, in particular, conservative amino acid replacements. Conservative replacements are those that take place within a family of amino acids that are related in their side chains. Genetically-encoded amino acids are generally divided into four families: (1) acidic=aspartate, glutamate; (2) basic=lysine, 25 arginine, histidine; (3) non-polar=alanine, valine, leucine, isoleucine, proline, phenylalanine, methionine, tryptophan; and (4) uncharged polar=glycine, asparagine, glutamine, cystine, serine, threonine, tyrosine. Phenylalanine, tryptophan, and tyrosine are sometimes classified jointly as aromatic amino acids. For example, it is reasonable to expect that an isolated replacement 70 of a leucine with an isoleucine or valine, an aspartate with a glutamate, a threonine with a seine, or a similar conservative replacement ofan amino acid with a structurally related amino acid will not have a major effect on biological activity. Polypeptide molecules having substantially the same amino acid sequence as any of the polypeptides disclosed 5 in any one of SEQ ID NOs: 27, 29, 31, 33, 35, 37, 39, 41, 43, 45, 47 and 49 but possessing minor amino acid substitutions that do not substantially affect the ability of the chimeric antigens to elicit an immune response, are within the demfintion of a chimeric antigen having the sequence as set forth in SEQ ID NOs: 27, 29, 31, 33, 35, 37, 39, 41, 43, 45, 47 and 49, respectively. Derivatives include aggregative conjugates with other 10 chimeric antigen molecules and covalent conjugates with unrelated chemical moieties. Covalent derivatives are prepared by linkage of finctionalities to groups that are foand in chimeric antigen amino acid chains or at the N- or C-terminal residues by means known in the art. Amino acid abbreviations are provided in Table L is TABLE I Amino Acid Abbreviations Alonine Ala A Asprane As N Asinuitp D Cytiie Cvs C Glutamate Glu E |turamno |Gh Glycine Gly G Histdine His H soleucine |Ile Leucine Lee L Lysine K Methionie Met M Phenvilanae |Pe F| Police Pro P Serire Sir |S Tlveoaine Thr r Trytoplan | VW Tyrosne Y Valun Val V Conservative amino acid substitutions can be made in a protein without altering either the lonfomation orthe function ofthe protein. Proteins ofthe invention can comprise I to 71 15 conservative substitutions. Such changes include substituting any of isoleucine (I), valine (V), and leucine (L) for any other of these hydrophobic amino acids; aspartic acid (D) for glutamic acid (E) and vice versa; glutamine (Q) for asparagine (N) and vice versa; and serine (S) for threonine (T) and vice versa. Other substitutions can also be considered 5 conservative, depending on the environment of the particular amino acid and its role in the three dimensional structure of the protein. For example,' glycine (G) and alanine (A) can frequently be interchangeable, as can alanine (A) and valine (V). Methionine (M), which is relatively hydrophobic, can frequently be interchanged with leucine and isoleucine, and sometimes with valine. Lysine (K) and arginine (R) are frequently 10 interchangeable in locations in which the significant feature of the amino acid residue is its charge and the differing pKs of these two amino acid residues are not significant. Still other changes can be considered "conservative" in particular environments (see, e.g. Biochemistry 4 th Ed., Lubert Stryer ed. (W. H. Freeman and Co.), pages 18-23; Henikoff and Henikoff, Proc Nat 'I Acad Sci USA 89:10915-10919 (1992); Lei et ab, J Biol Chem 15 270(20): 11882-6 (1995)). The inventions also provides polynucleotides that hybridize, preferably under stringent conditions, to a polynucleotide encoding a chimeric antigen, as specifically described by SEQ ID NOs: 26, 28, 30, 32, 34, 36, 38, 40, 42, 44, 46, and 44. Stringency of hybridization reactions is readily determinable by one of ordinary skill in the art and 20 generally is an empirical calculation dependent upon probe length, washing temperature, and salt concentration. In general longer probes require higher temperatures for proper annealing, while shorter probes need lower temperatures. Hybridization generally depends on the ability of denatured nucleic acid sequences to reanneal when complementary strands are present in an environment below their melting temperature. 25 The higher the degree of desired homology between the probe and hybridizable sequence, the higher the relative temperature that can be used. As a result, it follows that higher relative temperatures would tend to make the reaction conditions more stringent, while lower temperatures less so. For additional details and explanation of stringency of hybridization reactions, see, e.g., Ausubel et ab, supra, at pages 2.9.1-2.10.8 and 4.9.1 30 4.9.13. 72 "Stringent conditions" or "high stringency conditions", as defined herein, are identified by, but not limited to, those that (1) employ low ionic strength and high temperature for washing, for example 0.015 M sodium chloride/0.0015 M sodium citrate/0. 1% sodium dodecyl sulfate at 50'C; (2) employ during hybridization a 5 denaturing agent, such as formamide, for example, 50% (v/v) formamide with 0.1% bovine serum albumin/0.1% Ficoll/0.1% polyvinylpyrrolidone/50 mM sodium phosphate buffer at pH 6.5 with 750 mM sodium chloride, 75 mM sodium citrate at 42'C; or (3) employ 50% formamide, 5X SSC (0.75 M NaCb 0.075 M sodium citrate), 50 mM sodium phosphate (pH 6.8), 0.l1% sodium pyrophosphate, 5X Denhardt's solution, 10 sonicated salmon sperm DNA (50 [tg/ml), 0. 1% SDS, and 10% dextran sulfate at 42'C, with washes at 42'C in 0.2X SSC (sodium chloride/sodium citrate) and 50% formamide at 55'C, followed by a high-stringency wash consisting of 0.1X SSC containing EDTA at 55'C "Moderately stringent conditions" are described by, but not limited to, those in Sambrook et ab, supra, and include the use of washing solution and hybridization 15 conditions (e.g., temperature, ionic strength and % SDS) less stringent than those described above. An example of moderately stringent conditions is overnight incubation at 37 0 C in a solution comprising: 20% formamide, 5X SSC (150 mM NaCb 15 mM trisodium citrate), 50 mM sodium phosphate (pH 7.6), 5X Denhardt's solution, 10% dextran sulfate, and 20 mg/mL denatured sheared salmon sperm DNA, followed by 20 washing the filters in IX SSC at about 37-50'C. The skilled artisan will recognize how to adjust the temperature, ionic strength, etc. as necessary to accommodate factors such as probe length and the like. Embodiments of a chimeric antigen-encoding polynucleotide include: a polynucleotide encoding a chimeric antigen having a sequence selected from any of SEQ 25 ID NOs: 27, 29, 31, 33, 35, 37, 39, 41, 43, 45 47 and 49, or a nucleotide sequence of chimeric antigen selected from any of SEQ ID NOs: 26, 28, 30, 32, 34, 36, 38, 40, 42, 44, 46 and 48, wherein T, optionally, can be U; For example, embodiments of chimeric antigen nucleotides comprise, without limitation: (a) a polynucleotide comprising or consisting of a sequence as described by SEQ 30 ID NOs.: nucleotides I to 1326 of SEQ ID NO: 26, nucleotides I to 73 2004 of SEQ ID NO: 28, nucleotides I to 1350 of SEQ ID NO: 30, nucleotides I to 1293 of SEQ ID NO: 32, nucleotides I to 1794 of SEQ ID NO: 34, nucleotides I to 1581 of SEQ ID NO: 36, nucleotides I to 1389 of SEQ ID NO: 38, nucleotides I to 1347 of SEQ ID NO: 40, nucleotides I to 2157 of SEQ ID NO: 42, 5 nucleotides I to 1395 of SEQ ID NO: 44, nucleotides I to 1905 of SEQ ID NO: 46, or nucleotides I to 2484 of SEQ ID NO: 48, wherein T can also be U; (b) a polynucleotide whose sequence is at least 80% homologous to a sequence as described by SEQ ID NOs: 26, 28, 30, 32, 34, 36, 38, 40, 42, 44, 46 or 48; (c) a polynucleotide that encodes a chimeric antigen whose sequence is encoded 10 by a DNA contained in one of the plasmids designated pFastBacHTa HBV SI/S2-TBD, pFastBacHTa HBV core-TBD, pFastBacHTa HCV core(l-177)-TBD, pFastBacHTa HCV NS5A-TBD, and pFastBacHTa HCV E2-TBD deposited with the International Depository Authority of Canada (Bureau of Microbiology at Health Canada) as Accession Nos. 080504-03, 080504-04, 080504-05, 080504-02 and 080504-01 15 respectively; (d) a polynucleotide that encodes a chimeric antigen whose sequence is amino acids I to 442 of SEQ ID NO: 27, amino acids I to 668 of SEQ ID NO: 29, amino acids 1 to 450 of SEQ ID NO: 31, amino acids I to 431 of SEQ ID NO: 33, amino acids I to 598 of SEQ ID NO: 35, amino acids I to 527 of SEQ ID NO: 37, amino acids I to 463 of 20 SEQ ID NO: 39, amino acids I to 449 of SEQ ID NO: 41, amino acids I to 719 of SEQ ID NO: 43, amino acids I to 465 of SEQ ID NO: 45, amino acids I to 635 of SEQ ID NO: 47, or amino acids I to 828 of SEQ ID NO: 49; (e) a polynucleotide that encodes a chimeric antigen-related protein that is at least 25 90% identical to an entire amino acid sequence described by amino acids 1 to 442 of SEQ ID NO: 27, amino acids I to 668 of SEQ ID NO: 29, amino acids I to 450 of SEQ ID NO: 31, amino acids I to 431 of SEQ ID NO: 33, amino acids I to 598 of SEQ ID NO: 74 35, amino acids I to 527 of SEQ ID NO: 37, amino acids I to 463 of SEQ ID NO: 39, amino acids I to 449 of SEQ ID NO: 41, amino acids I to 719 of SEQ ID NO: 43, amino acids I to 465 of SEQ ID NO: 45, amino acids I to 635 of SEQ ID NO: 47, or amino acids I to 828 of SEQ ID NO: 49; 5 (f) a polynucleotide that is fully complementary to a polynucleotide of any one of (a)-(d); and (g) a polynucleotide that selectively hybridizes under stringent conditions to a polynucleotide of (a)-(f). The invention also provides recombinant DNA or RNA molecules containing a chimeric antigen polynucleotide, an analog or homologue thereof, including but not 10 limited to phages, plasmids, phagemids, cosmids, YACs (yeast artificial chromosomes), BACs (bacterial artificial chromosomes), as well as various viral and non- viral vectors well known in the art, and cells transformed or transfected with such recombinant DNA or RNA molecules. Methods for generating such molecules are well known (see, for example, Sambrook et ab, 1989, supra). 15 The invention further provides a host-vector system comprising a recombinant DNA molecule containing a chimeric antigen polynucleotide, analog or homologue thereof within a suitable prokaryotic or eukaryotic host cell. Examples of suitable eukaryotic host cells include a yeast cell, a plant cell, or an animal cell, such as a mammalian cell or an insect cell (e.g., a baculovirus-infectible cell such as a Sf9, Sf21, 20 Drosophila S2 or High Five TM cell). Examples of suitable mammalian cells include various prostate cancer cell lines such as DUl45 and TsuPrl, other transfectable or transducible prostate cancer cell lines, primary cells (PrEC), as well as a number of mammalian cells routinely used for the expression of recombinant proteins (e.g., COS, CHO, 293, 293T cells). More particularly, a polynucleotide comprising the coding 25 sequence of chimeric antigen or a fragment, analog or homolog thereof can be used to generate chimeric antigen thereof using any number of host-vector systems routinely used and widely known in the art. 75 A wide range of host-vector systems suitable for the expression of chimeric antigens thereof are available, see for example, Sambrook et ab, 1989, supra; Ausubeb supra, at pages 1.0.1-1.16.16, 9.01-9.17.3, and 13.4.1-13.6.5). Preferred vectors for insect cell expression include but are not limited to pFastBac HTa (Invitrogen). Using such 5 expression vectors, chimeric antigens can be expressed in several insect cell lines, including for example Sf9, Sf21, Drosophila S2, and High Five. Alternatively, preferred yeast expression systems include Saccharomyces cerevisiae, Schizosaccharomyces pombe, Pichia pastoris, and Pichia august. The host-vector systems of the invention are useful for the production of a chimeric antigen. 10 A chimeric antigen or an analog or homolog thereof can be produced by cells transfected with a construct encoding a chimeric antigen. For example, Sf9 cells can be transfected with an expression plasmid encoding a chimeric antigen or analog or homolog thereof, the chimeric antigen is expressed in the Sf9 cells, and the chimeric antigen is isolated using standard purification methods. Various other expression systems well 15 known in the art can also be employed. Expression constructs encoding a leader peptide joined in frame to the chimeric antigen coding sequence can be used for the generation of a secreted form of chimeric antigen. As discussed herein, redundancy in the genetic code permits variation in chimeric antigen gene sequences. In particular, it is known in the art that specific host species 20 often have specific codon preferences, and thus one can adapt the disclosed sequence as preferred for a desired host. For example, preferred analog codon sequences typically have rare codons (i.e., codons having a usage frequency of less than about 20% in known sequences of the desired host) replaced with higher frequency codons. Codon preferences for a specific species are calculated, for example, by utilizing codon usage tables 25 available on the INTERNET such as at world wide web URL www.kazusa.or.jp/codon. Additional sequence modifications are known to enhance protein expression in a cellular host. These include elimination of sequences encoding spurious polyadenylation signals, exon/intron splice site signals, transposon-like repeats, and/or other such well characterized sequences that are deleterious to gene expression. The GC content of the 76 sequence is adjusted to levels average for a given cellular host, as calculated by reference to known genes expressed in the host cell. Where possible, the sequence is modified to avoid predicted hairpin secondary mRNA structures. Other useful modifications include the addition of a translational 5 initiation consensus sequence at the start of the open reading frame, as described in Kozak, i Mol. Cell Biol, 9:5073-5080 (1989). Skilled artisans understand that the general rule that eukaryotic ribosomes initiate translation exclusively at the 5' proximal AUG codon is abrogated only under rare conditions (see, e.g., Kozak Proc Nat 'I Acad Sci USA 92(7): 2662-2666 (1995) and Kozak Nucl Acids Res 15(20): 8125-8148 (1987)). 10 Another aspect of the invention provides polynucleotides encoding all of the chimeric antigens disclosed herein. The polynucleotides comprise a first polynucleotide portion encoding an immune response domain and a second polynucleotide portion encoding a target binding domain. The first and second polynucleotide portions may be located on the same or different nucleotide chains. 15 In addition to the above described regions of the chimeric antigens of the invention, polynucleotides of the invention generally contain leader sequences encoding leader peptides that facilitate secretion of the chimeric antigen from a cell (e.g., a yeast or insect cell) producing it. The relevant leader sequence is generally cleaved from the chimeric antigen prior to secretion from the cell. Leader sequences can be any of those 20 disclosed herein and others known in the art, for example, AcNPV chitinase signal sequence having the amino acid sequence MPLYKLLNVLWLVAVSNAI (SEQ ID NO:37X) encoded by the nucleotide sequence ATGCCCTTGTACAAATTGTTAAACGTTTTGTGGTTGGTCGCCGTTTCTAACGC GATT (SEQ ID NO:38X) useful for expression in insect cells and the alpha-mating 25 factor leader useful for expression in yeast cells (e.g., Pichia pastoris yeast cells). The invention provides polynucleotides corresponding or complementary to genes encoding chimeric antigens, mRNAs, and/or coding sequences, preferably in isolated form, including polynucleotides encoding chimeric antigen variant proteins; DNA, RNA, DNA/RNA hybrids, and related molecules, polynucleotides or 30 oligonucleotides complementary or having at least a 90% homology to the genes 77 encoding a chimeric antigen or mRNA sequences or parts thereof; and polynucleotides or oligonucleotides that hybridize to the genes encoding a chimeric antigen, mRNAs, or to chimeric antigen-encoding polynucleotides. Additionally, the invention includes analogs of the genes encoding a chimeric 5 antigen specifically disclosed herein. Analogs include, e.g., mutants, that retain the ability to elicit an immune response, and preferably have homology of at least 80%, more preferably 90%, and most preferably 95% to any of polynucleotides encoding a chimeric antigen, as specifically described by the sequences set forth in SEQ ID NOs: 39X and 41X-5 1X. Typically, such analogs differ by only I to 10 codon changes. Examples 10 include polypeptides with minor amino acid variations from the natural amino acid sequence of a viral antigen or of an antibody fragment; in particular, conservative amino acid replacements. Conservative replacements are those that take place within a family of amino acids that are related in their side chains. Genetically-encoded amino acids are generally divided into four families: (1) acidic=aspartate, glutamate; (2) basic=lysine, 15 arginine, histidine; (3) non-polar=alanine, valine, leucine, isoleucine, proline, phenylalanine, methionine, tryptophan; and (4) uncharged polar=glycine, asparagine, glutamine, cystine, serine, threonine, tyrosine. Phenylalanine, tryptophan, and tyrosine are sometimes classified jointly as aromatic amino acids. For example, it is reasonable to expect that an isolated replacement of a leucine with an isoleucine or valine, an aspartate 20 with a glutamate, a threonine with a serine, or a similar conservative replacement of an amino acid with a structurally related amino acid will not have a major effect on biological activity. Polypeptide molecules having substantially the same amino acid sequence as any of the polypeptides disclosed herein but possessing minor amino acid substitutions that do not substantially affect the ability of the chimeric antigens to elicit 25 an immune response, are within the definition of a chimeric antigen. Derivatives include aggregative conjugates with other chimeric antigen molecules and covalent conjugates with unrelated chemical moieties. Covalent derivatives are prepared by linkage of functionalities to groups that are found in chimeric antigen amino acid chains or at the N or C-terminal residues by means known in the art. 30 Amino acid abbreviations are provided in Table 1. 78 TABLE 1: Amino Acid Abbreviations Alanine Ala A Arginine Arg R Asparagine Asn N Aspartate Asp D Cysteine Cys C Glutamate Glu E Glutamine Gln Q Glycine Gly G Histidine His H Isoleucine Ile I Leucine Leu L Lysine Lys K Methionine Met M Phenylalanine Phe F Proline Pro P Serine Ser S Threonine Thr T Tryptophan Trp W Tyrosine Tyr Y Valine Val V Conservative amino acid substitutions can be made in a protein without altering 5 either the conformation or the function of the protein. Proteins of the invention, or useful for the invention, can comprise not more than 15 (e.g., not more than: 14; 13; 12; 11; 10; 9; 8; 7; 6; 5; 4; 3; 2; or 1) conservative substitution(s). Such changes include substituting any of isoleucine (I), valine (V), and leucine (L) for any other of these hydrophobic amino acids; aspartic acid (D) for glutamic acid (E) and vice versa; glutamine (Q) for 10 asparagine (N) and vice versa; and serine (S) for threonine (T) and vice versa. Other substitutions can also be considered conservative, depending on the environment of the particular amino acid and its role in the three dimensional structure of the protein. For example, glycine (G) and alanine (A) can frequently be interchangeable, as can alanine (A) and valine (V). Methionine (M), which is relatively hydrophobic, can frequently be 15 interchanged with leucine and isoleucine, and sometimes with valine. Lysine (K) and arginine (R) are frequently interchangeable in locations in which the significant feature of the amino acid residue is its charge and the differing pK's of these two amino acid residues are not significant Still other changes can be considered "conservative" in 79 particular environments [see, e.g. Biochemistry 4th Ed., Lubert Stryer ed. (W. H. Freeman and Co.), pages 18-23; Henikoff et al. (1992) Proc Natl Acad Sci USA 89:10915-10919; Lei et al. (1995) J. Biol. Chem. 270:11882-11885]. Additional analog polynucleotides include those with one or more (e.g., 2, 3, 4, 5, 5 6, 7, 8, 9, 10, 12, 15, or 20) additions or deletions in any of the TBDs and/or any of the IRDs that serve, for example, to increase the solubility of the relevant chimeric antigen. The additions or deletions can be of one or more (e.g., 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 30, 40, 50, 60, 70, 80, 90, 100, or more) amino acids in the chimeric antigens encoded by the polynucleotides (and the 10 corresponding numbers of nucleotides in the polynucleotides themselves). The invention also includes polynucleotides that selectively hybridize to polynucleotides that encode chimeric antigens. Preferably a polynucleotide of the invention will hybridize under stringent conditions to one or more of the sequences set forth in SEQ ID NOs:39X and 41X-51X. Stringency of hybridization reactions is 15 readily determinable by one of ordinary skill in the art and generally is an empirical calculation dependent upon probe length, washing temperature, and salt concentration. In general longer probes require higher temperatures for proper annealing, while shorter probes need lower temperatures. Hybridization generally depends on the ability of denatured nucleic acid sequences to re-anneal when complementary strands are present in 20 an environment below their melting temperature. The higher the degree of desired homology between the probe and hybridizable sequence, the higher the relative temperature that can be used. As a result, it follows that higher relative temperatures would tend to make the hybridization conditions more stringent, while lower temperatures less so. For additional details and explanation of stringency of hybridization 25 reactions, see, e.g., Ausubel et al., Current Protocols in Molecular Biology, Wiley Interscience Publishers, (@ 1995, as Supplemented April 2004, Supplement 66) at pages 2.9.1-2.10.8 and 4.9.1-4.9.13. "Stringent conditions" or "high stringency conditions", as defined herein, are identified by, but not limited to, those that (1) employ low ionic strength and high 30 temperature for washing, for example 0.015 M sodium chloride/0.0015 M sodium citrate/0. 1% sodium dodecyl sulfate at 50'C; (2) employ, during hybridization, a 80 denaturing agent, such as formamide, for example, 50% (v/v) formamide with 0.1% bovine serum albumin/0.1% Ficoll/0.1% polyvinylpyrrolidone/50 mM sodium phosphate buffer at pH 6.5 with 750 mM sodium chloride, 75 mM sodium citrate at 42'C; or (3) employ 50% formamide, 5X SSC (0.75 M NaCl, 0.075 M sodium citrate), 50 mM 5 sodium phosphate (pH 6.8), 0.1% sodium pyrophosphate, 5X Denhardt's solution, sonicated salmon sperm DNA (50 pg/mL), 0. 1% SDS, and 10% dextran sulfate at 42'C, with washes at 420 C in 0.2X SSC (sodium chloride/sodium citrate) and 50% formamide at 55'C, followed by a high-stringency wash consisting of 0.1X SSC containing EDTA at 55'C. "Moderately stringent conditions" are described by, but not limited to, those in 10 Sambrook et al., Molecular Cloning: A Laboratory Manual, 2 Ed., New York: Cold Spring Harbor Press, 1989, and include the use of washing solution and hybridization conditions (e.g., temperature, ionic strength and % SDS) less stringent than those described above. An example of moderately stringent conditions is overnight incubation at 370 C in a solution comprising: 20% formamide, 5X SSC (150 mM NaCl, 15 mM 15 trisodium citrate), 50 mM sodium phosphate (pH 7.6), 5X Denhardt's solution, 10% dextran sulfate, and 20 mg/mL denatured sheared salmon sperm DNA, followed by washing the filters in IX SSC at about 37-50'C. The skilled artisan will recognize how to adjust the temperature, ionic strength, etc. as necessary to accommodate factors such as probe length and the like. 20 Embodiments of a polynucleotide of the invention include: a polynucleotide encoding a chimeric antigen having a sequence selected from any of the sequences as set forth in SEQ ID NOs: 40X and 52X-62X, a nucleotide sequence of chimeric antigen selected from any of the sequences as set forth in SEQ ID NOs: 39X and 41X-51X but with T nucleotides substituted with U nucleotides. For example, embodiments of 25 chimeric antigen nucleotides comprise, without limitation: (a) a polynucleotide comprising or consisting of a sequence selected from any of the sequences as set forth in SEQ ID NOs: 39X and 41X-5 IX, wherein T can also be U; (b) a polynucleotide whose sequence is at least 80% homologous to a sequence 30 selected from any of the sequences as set forth in SEQ ID NOs: 39X and 41X 51X; 81 (c) a polynucleotide that encodes a chimeric antigen whose sequence is encoded by a DNA contained in any of the plasmids disclosed herein; (d) a polynucleotide that encodes a chimeric antigen whose sequence is a sequence selected from any of the sequences as set forth in SEQ ID NOs: 40X 5 and 52X-62X; (e) a polynucleotide that encodes a chimeric antigen-related protein that is at least 90% identical to an entire amino acid sequence whose sequence is selected from any of the sequences as set forth in SEQ ID NOs: 40X and 52X-62X; (f) a polynucleotide that is fully complementary to a polynucleotide of any one of 10 (a)-(e); (g) a polynucleotide that selectively hybridizes under stringent conditions to a polynucleotide of (a)-(f); and (h) a polynucleotide comprising or consisting of a sequence selected from any of the sequences as set forth in SEQ ID NOs: 39X and 41X-51X but lacking all 15 or some of the sequences other than the IRD (e.g., the HCV proteins listed herein) and the TBD and, optionally containing, for example, one or more alternative linkers and/or an alternative secretory (leader) peptide. In addition, additional sequences (e.g., vector-derived sequences encoding amino acids at the C terminus of the TBD) can be deleted from polynucleotides of 20 the invention. Such additional sequences can be those encoding 1-15 (e.g., 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, or 14) amino acids. The invention also provides recombinant DNA or transcribed RNA molecules containing a chimeric antigen polynucleotide, an analog or homologue thereof, including 25 but not limited to phages, plasmids, phagemids, cosmids, YACs (yeast articifical chromosomes), BACs (bacterial artificial chromosomes), as well as various viral and non-viral vectors well known in the art, and cells transformed or transfected with such recombinant DNA or RNA molecules. Methods for generating such molecules are well known [see, for example, Sambrook et al., 1989, supra]. 30 The invention further provides a host-vector system comprising a recombinant DNA molecule containing a chimeric antigen polynucleotide, analog or homologue 82 thereof within a suitable prokaryotic or eukaryotic host cell. Examples of suitable eukaryotic host cells include a yeast cell, a plant cell, or an animal cell, such as a mammalian cell or an insect cell (e.g., a baculovirus-infectible cell such as an Sf9, Sf21, expresSF®, Drosophila S2 or High FiveTM cell). Examples of suitable mammalian cells 5 include various prostate cancer cell lines such as DU145 and TsuPrl, other transfectable or transducible prostate cancer cell lines, primary cells (PrEC), as well as a number of mammalian cells routinely used for the expression of recombinant proteins (e.g., COS, CHO, 293, 293T cells). More particularly, a polynucleotide comprising the coding sequence of chimeric antigen or a fragment, analog or homolog thereof can be used to 10 generate chimeric antigen thereof using any number of host-vector systems routinely used and widely known in the art. A wide range of host-vector systems suitable for the expression of chimeric antigens thereof are available, see for example, Sambrook et al., 1989, supra; Ausubel, Current Protocols in Molecular Biology, 1995, supra). Preferred vectors for insect cell 15 expression include, but are not limited to, the transfer vector plasmid pFastBac HTa (Invitrogen). Using such transfer vector plasmids, recombinant baculoviruses can be produced in insect cells and these can be used to infect several insect cell lines, including for example Sf9, Sf21, expresSF®, Drosophila S2 or High FiveTM, to express chimeric antigens. An example of this is the Bac to Bac baculovirus expression system 20 (Invitrogen). Alternatively, preferred yeast expression systems include Saccharomyces cerevisiae, Schizosaccharomyces pombe, Pichia pastoris, and Pichia august. The host vector systems of the invention are useful for the production of a chimeric antigen. A chimeric antigen or an analog or homolog thereof can also be produced by the stable transfection of cells (e.g., insect cells) with a plasmid construct containing the an 25 appropriate promoter (e.g., an insect cell promoter) and encoding a chimeric antigen. For example, a recombinant plamid pMIB-V5 (Invitrogen) encoding chimeric antigen or an analog or homolog thereof can be used for stable transfection of Sf9 insect cells. The chimeric antigen or related protein is expressed in the Sf9 cells, and the chimeric antigen is isolated using standard purification methods. Various other expression systems well 30 known in the art can also be employed. Expression constructs encoding a leader peptide 83 joined in frame to the chimeric antigen coding sequence can be used for the generation of a secreted form of chimeric antigen. As discussed herein, redundancy in the genetic code permits variation in chimeric antigen gene sequences. In particular, it is known in the art that specific host species often 5 have specific codon preferences, and thus one can adapt the disclosed sequence as preferred for a desired host. For example, preferred analog codon sequences typically have rare codons (i.e., codons having a usage frequency of less than about 20% in known sequences of the desired host) replaced with higher frequency codons. Codon preferences for a specific species are calculated, for example, by utilizing codon usage tables 10 available on the INTERNET such as at world wide web URL www.kazusa.or.jp/codon. Additional sequence modifications are known to enhance protein expression in a cellular host. These include elimination of sequences encoding spurious polyadenylation signals, exon/intron splice site signals, transposon-like repeats, and/or other such well characterized sequences that are deleterious to gene expression. The GC content of the 15 sequence is adjusted to levels average for a given cellular host, as calculated by reference to known genes expressed in the host cell. Where possible, the sequence is modified to avoid predicted hairpin secondary mRNA structures. Other useful modifications include the addition of a translational initiation consensus sequence at the start of the open reading frame, as described in Kozak [(1989) Mol. Cell Biol. 9:5073-5080]. Skilled 20 artisans understand that the general rule that eukaryotic ribosomes initiate translation exclusively at the 5' proximal AUG codon is abrogated only under rare conditions [see, e.g., Kozak (1995) Proc. Natl. Acad. Sci USA 92:2662-2666; Kozak (1987) Nucl. Acids Res. 15:8125-8148]. Escherichia coli clones, each transformed with one of the plasmids listed below, 25 were deposited on October 11, 2006, under the Budapest Treaty at the International Depository Authority of Canada (IDAC), 1015 Arlington Street Winnipeg, Manitoba, R3E 3R2 Canada (telephone no.: (204) 789-6030; facsimile no.: (204) 789-2018). Each clone is readily identified by the indicated IDAC accession number. 30 Plasmid IDAC accession number pFastBacHTa-gp64 HCV NS3mutS-TBD 111006-01 84 pFastBacHTa-gp64 HCV NS3mut-TBD 111006-02 pFastBacHTa-gp64 NS3-NS5A-TBD 111006-03 pFastBacHTa-gp64 HCV NS5A-TBD 111006-04 pFastBacHTa HCV NS3mut-TBD 111006-05 5 pFastBacHTa HCV NS3-NS4B-NS5A-TBD 111006-06 The samples deposited with the IDAC are taken from the same deposit maintained by the ViRexx Medical Corporation since prior to the filing date of this application. The deposits will be maintained without restriction in the IDAC depository for a period of 30 10 years, or 5 years after the most recent request, or for the effective life of the patent, whichever is longer, and will be replaced if the deposit becomes non-viable during that period. G. Pharmaceutical Compositions of the Invention 15 One aspect of the invention relates to pharmaceutical compositions comprising a pharmaceutically acceptable excipient and a chimeric antigen comprising an immune response domain and a target binding domain, wherein the target binding domain comprises an antibody fragment. In therapeutic applications, the pharmaceutical compositions can be administered to a subject in an amount sufficient to elicit an 20 effective B cell, cytotoxic T lymphocyte (CTL) and/or helper T lymphocyte (Th) response to the antigen and to block infection or to cure or at least partially arrest or slow symptoms and/or complications or a disease or disorder. Amounts effective for this use will depend on, e.g., the particular composition administered, the manner of administration, the stage and severity of the disease being treated, the weight and general 25 state of health of the subject, and the judgment of the prescribing physician. The dosage for an initial therapeutic immunization (with chimeric antigen) generally occurs in a unit dosage range where the lower value is about 1, 5, 50, 500, or 1,000 ng and the higher value is about 10,000; 20,000; 30,000; or 50,000 ug. Dosage values for a human typically range from about 500 ng to about 50,000 ug per 70 kilogram 30 subject. Boosting dosages of between about 1.0 ng to about 50,000 [tg of chimeric 85 antigen pursuant to a boosting regimen over weeks to months may be administered depending upon the subject's response and condition. Administration should continue until at least clinical symptoms or laboratory tests indicate that the condition has been prevented, arrested, slowed or eliminated and for a period thereafter. The dosages, routes 5 of administration, and dose schedules are adjusted in accordance with methodologies known in the art. A human unit dose form of a chimeric antigen is typically included in a pharmaceutical composition that comprises a human unit dose of an acceptable carrier, in one embodiment an aqueous carrier, and is administered in a volume/quantity that is 10 known by those of skill in the art to be useful for administration of such polypeptides to humans (see, e.g., Remington: The Science and Practice of Pharmacy, 20th Edition, A. Gennaro, Editor, Lippincott Williams &amp; Wilkins, Baltimore, Md., 2000). As appreciated by those of skill in the art, various factors can influence the ideal dose in a particular case. Such factors include, for example, half life of the chimeric antigen, the binding 15 affinity of the chimeric antigen, the immunogenicity of the composition, the desired steady-state concentration level, route of administration, frequency of treatment, and the influence of other agents used in combination with the treatment method of the invention, as well as the health status of a particular subject. In certain embodiments, the compositions of the present invention are employed 20 in serious disease states, that is, life-threatening or potentially life- threatening situations. In such cases, as a result of the relative nontoxic nature of the chimeric antigen in preferred compositions of the invention, it is possible and may be felt desirable by the treating physician to administer substantial excesses of these chimeric antigens relative to these stated dosage amounts. 25 The concentration of chimeric antigen of the invention in the pharmaceutical formulations can vary widely, i.e., from less than about 0.1%, usually at or at least about 2% to as much as 20% to 50% or more by weight, and will be selected primarily by fluid volumes, viscosities, etc., in accordance with the particular mode of administration selected. 86 The pharmaceutical compositions can be delivered via any route known in the art, such as parenterally, intrathecally, iritravascularly, intravenously, intramuscularly, transdermally, intradermally, subcutaneously, intranasally, topically, orally, rectally, vaginally, pulmonarily or intraperitoneally. Preferably, the composition is delivered by 5 parenteral routes, such as subcutaneous or intradermal administration. The pharmaceutical compositions can be prepared by mixing the desired chimeric antigens with an appropriate vehicle suitable for the intended route of administration. In making the pharmaceutical compositions of this invention, the chimeric antigen is usually mixed with an excipient, diluted by an excipient or enclosed within a carrier that can be 10 in the form of a capsule, sachet, paper or other container. When the pharmaceutically acceptable excipient serves as a diluent, it can be a solid, semi-solid, or liquid material, which acts as a vehicle, carrier or medium for the therapeutic agent. Thus, the compositions can be in the form of tablets, pills, powders, lozenges, sachets, cachets, elixirs, suspensions, emulsions, solutions, syrups, aerosols (as a solid or in a liquid 15 medium), ointments containing, for example, up to 10% by weight of the chimeric antigen, soft and hard gelatin capsules, suppositories, sterile injectable solutions, and sterile packaged powders. Some examples of suitable excipients include, but are not limited to, dextrose, sucrose, sorbitol, mannitob starches, gum acacia, calcium phosphate, alginates, 20 tragacanth, gelatin, calcium silicate, microcrystalline cellulose, polyvinylpyrrolidone, cellulose, sterile water, syrup, and methyl cellulose. The formulations can additionally include: lubricating agents such as talc, magnesium stearate, and mineral oil; wetting agents; emulsifying and suspending agents; preserving agents such as methyl- and propylhydroxy-benzoates; sweetening agents; and flavoring agents. The compositions of 25 the invention can be formulated so as to provide quick, sustained or delayed release of the chimeric antigen after administration to the subject by employing procedures known in the art. See, e.g., Remington, supra, at pages 903-92 and pages 1015-1050. 87 For preparing solid compositions such as tablets, the chimeric antigen is mixed with a pharmaceutical excipient to form a solid preformulation composition containing a homogeneous mixture of a chimeric antigen of the present invention. When referring to these preformulation compositions as homogeneous, it is meant 5 that the chimeric antigen is dispersed evenly throughout the composition so that the composition may be readily subdivided into equally effective unit dosage fonns such as tablets, pills and capsules. The tablets or pills of the present invention may be coated or otherwise compounded to provide a dosage form affording the advantage of prolonged action. For 10 example, the tablet or pill can comprise an inner dosage and an outer dosage component, the latter being in the form of an envelope over the former. The two components can be separated by an enteric layer, which serves to resist disintegration in the stomach and permit the inner component to pass intact into the duodenum or to be delayed in release. A variety of materials can be used for such enteric layers or coatings, such materials 15 including a number of polymeric acids and mixtures of polymeric acids with such materials as shellac, cetyl alcohol, and cellulose acetate. The liquid forms in which the novel compositions of the present invention may be incorporated for administration orally or by injection include aqueous solutions, suitably flavored syrups, aqueous or oil suspensions, and flavored emulsions with edible oils such 20 as corn oil, cottonseed oil, sesame oil, coconut oil, or peanut oil, as well as elixirs and similar pharmaceutical vehicles. In preparing a composition for parenteral administration strict attention must be paid to tonicity adjustment to reduce irritation. A reconstitutable composition is a sterile solid packaged in a dry form. A reconstitutable composition is prefened because it is 25 more stable when stored as a dry solid rather than in a solution ready for immediate administration. The dry solid is usually packaged in a sterile container with a butyl rubber closure to ensure the solid is kept at an optimal moisture range. A reconstitutable dry solid is formed by dry fill, spray drying, or freeze-drying methods. Descriptions of these 88 methods may be found, e.g., in Remington, supra, at pages 681-685 and 802-803. Compositions for parenteral injection are generally dilute, and the component present in the higher proportion is the vehicle. The vehicle normally has no therapeutic activity and is nontoxic, but presents the chimeric antigen to the body tissues in a form appropriate for 5 absorption. Absorption normally will occur most rapidly and completely when the chimeric antigen is presented as an aqueous solution. However, modification of the vehicle with water-miscible liquids or substitution with water-immiscible liquids can affect the rate of absorption. Preferably, the vehicle of greatest value for this composition is isotonic saline. In preparing the compositions that are suitable for injection, one can 10 use aqueous vehicles, water-miscible vehicles, and nonaqueous vehicles Additional substances may be included in the injectable compositions of this invention to improve or safeguard the quality of the composition. Thus, an added substance may affect solubility, provide for subject comfort, enhance the chemical stability, or protect the preparation against the growth of microorganisms. Thus, the 15 composition may include an appropriate solubilizer, substances to act as antioxidants, and substances that act as a preservative to prevent the growth of microorganisms. These substances will be present in an amount that is appropriate for their function, but will not adversely affect the action of the composition. Examples of appropriate antimicrobial agents include thimerosal, benzethonium chloride, benzalkonium chloride, phenol, 20 methyl p-hydroxybenzoate, and propyl p-hyrodxybenzoate. Appropriate antioxidants maybe found in Remington, supra, at p. 1015-1017. In certain embodiments, liposomes, nanocapsules, microparticles, lipid particles, vesicles, and the like, are used for the administration of the chimeric antigens of the present invention. In particular, the compositions of the present invention may be 25 formulated for delivery either encapsulated in a lipid particle, a liposome, a vesicle, a nanosphere, or a nanoparticle or the like. Alternatively, compositions of the present invention can be bound, either covalently or non-covalently, to the surface of such carrier vehicles. 89 Compositions administered via liposomes may also serve: 1) to target the chimeric antigen to a particular tissue, such as lymphoid tissue; 2) to target selectively to antigen presenting cells; or, 3) to increase the half-life of the peptide composition. Liposomes include emulsions, foams, micelles, insoluble monolayers, liquid crystals, 5 phospholipid dispersions, lamellar layers and the like. In these preparations, the chimeric antigen to be delivered is incorporated as part of a liposome, alone or in conjunction with a molecule that binds to a receptor prevalent among lymphoid cells, such as monoclonal antibodies that bind to the CD45 antigen, or with other therapeutic or immunogenic compositions. Thus, liposomes either filled or decorated with a desired chimeric antigen 10 of the invention can be directed to the site of lymphoid cells, where the liposomes then deliver the chimeric antigens. Liposomes for use in accordance with the invention are formed from standard vesicle-forming lipids, which generally include neutral and negatively charged phospholipids and a sterol, such as cholesterol. The selection of lipids is generally guided by consideration of, e.g., liposome size, acid lability and stability of 15 the liposomes in the blood stream. A variety of methods are available for preparing liposomes, as described in, e.g., Szoka, et ab, Ann. Rev. Biophys. Bioeng. 9:467-508 (1980), and U.S. Pat. Nos. 4,235,871, 4,501,728, 4,837,028, and 5,019,369. A liposome suspension containing a chimeric antigen may be administered intravenously, locally, topically, etc. in a dose which varies according to, inter alia, the manner of 20 administration, the chimeric antigen being delivered, and the stage of the disease being treated. Compositions for inhalation or insufflation include solutions and suspensions in pharmaceutically acceptable, aqueous or organic solvents, or mixtures thereob and powders. The liquid or solid compositions may contain suitable pharmaceutically 25 acceptable excipients as described herein. The compositions can be administered by the oral or nasal respiratory route for local or systemic effect. Compositions in pharmaceutically acceptable solvents may be nebulized by use of inert gases. Nebulized solutions may be inhaled directly from the nebulizing device or the nebulizing device may be attached to a facemask tent, or intermittent positive pressure breathing machine. 30 Solution, suspension, or powder compositions may be administered, preferably orally or 90 nasally, from devices that deliver the formulation in an appropriate manner. Another formulation employed in the methods of the present invention employs transdermal delivery devices ("patches"). Such transdermal patches may be used to provide continuous or discontinuous infusion of the chimeric antigen of the present 5 invention in controlled amounts. The construction and use of transdermal patches for the delivery of pharmaceutical agents is well known in the art. See, for example, U.S. Pat. No. 5,023,252, herein incorporated by reference. Such patches may be constructed for continuous, pulsatile, or on demand delivery of pharmaceutical agents. Additionally, it may be advantageous to include at least one antiviral therapeutic 10 or chemotherapeutic in addition to the chimeric antigen and pharmaceutical excipient. Antiviral therapeutics include, but are not limited to, peptidomimetics (such as amprenavir, indinavir, lopinavir, nelfinavir, ritonavir, and saquinavir), polynucleotides (such as ampligen and fomivirsen), purine/pyrimidinones (such as abacavir, acyclovir, adefovir, cidofovir, cytarabine, didanosine, dideoxyadenosine, dipivoxib edoxudine, 15 emtricitabine, entecovir, famciclovir, ganciclovir, idoxuridine, inosine pranobex, lamivudine, MADU, penciclovir, sorivudine, stavudine, tenofovir, trifluridine, valacyclovir, valganciclovir, vidarabine, zalcitabine, and zidovudine), sialic acid analogs (such as oseltamivir and zanamivir), acemannan, acetylleucine monoethanolamine, amantadine, amidinomycin, ateviridine, capravirine, delavirdine, re-docosanol, efavirenz, 20 foscarnet sodium, interferon-a, interferon-p, interferon-y, kethoxal, lysozyme, methisazone, moroxydine, nevirapine, pentafuside, pleconarib podophyllotoxin, ribavirin, rimantidine, stallimycin, statolon, termacamra, and traomantadine. Other appropriate antiviral agents are discussed in Remington: supra, at Chapter 87: Anti-Infectives, pp. 1507-1561, particularly pp. 1555-1560. Prefened antiviral therapeutics for inclusion in 25 the pharmaceutical compositions of the present invention include adefovir, dipivoxib entecovir, lamivudine and ribavirin. In some embodiments it may be desirable to include in the pharmaceutical compositions of the invention at least one component which primes B-lymphocytes or T lymphocytes. Lipids have been identified as agents capable of priming CTL in vivo. For 30 example, palmitic acid residues can be attached to the c-and a-amino groups of a lysine 91 residue and then linked, e.g. , via one or more linking residues such as Gly, Gly-Gly-, Ser, Ser-Ser, or the like, to an immunogenic peptide. The lipidated peptide can then be administered either directly in a micelle or particle, incorporated into a liposome, or emulsified in an adjuvant, e.g., incomplete Freund's adjuvant. In a prefened embodiment, 5 a particularly effective immunogenic composition comprises palmitic acid attached to c and a-amino groups of Lys, which is attached via linkage, e.g., Ser-Ser, to the amino terminus of the immunogenic peptide. As another example of lipid priming of CTL responses, E. coli lipoproteins, such as tripalmitoyl-S-glycerylcysteinlyseryl-serine (P CSS) can be used to prime viras 10 specific CTL when covalently attached to an appropriate peptide (see, e.g., Deres, et ab, Nature 342:561 (1989)). Chimeric antigens of the invention can be coupled to P 3 CSS, for example, and the lipopeptide administered to an individual to specifically prime an immune response to the target antigen. While the compositions of the present invention should not require the use of 15 adjuvants, adjuvant can be used. Various adjuvants may be used to increase the immunological response, depending on the host species, and including but not limited to Freund's (complete and incomplete), mineral gels such as aluminum hydroxide, surface active substances such as lysolecithin, detergents, pluronic polyols, polyanions, peptides, oil emulsions, keyhole limpet hemocyanins, dinitrophenol, immunostimulatory 20 polynucleotide sequences, and potentially useful human adjuvants such as BCG (bacille Calmette-Guerin) and Corynebacterium parvum. Additional adjuvants are also well known in the art. One aspect of the invention relates to pharmaceutical compositions comprising a pharmaceutically acceptable excipient and a chimeric antigen comprising an immune 25 response domain and a target binding domain, wherein the target binding domain comprises an antibody fragment. In therapeutic applications, the pharmaceutical compositions can be administered to a subject in an amount sufficient to elicit an effective B cell, cytotoxic T lymphocyte (CTL) and/or helper T lymphocyte (Th) response to the antigen and to prevent infection or to cure or at least partially arrest or 92 slow symptoms and/or complications of infection. Amounts effective for this use will depend on, e.g., the particular composition administered, the manner of administration, the stage and severity of the disease being treated, the weight and general state of health of the subject, and the judgment of the prescribing physician. 5 The dosage for an initial therapeutic immunization (with chimeric antigen) generally occurs in a unit dosage range where the lower value is about 1, 5, 50, 500, or 1,000 ng and the higher value is about 10,000; 20,000; 30,000; or 50,000 pg. Dosage values for a human typically range from about 500 ng to about 50,000 pg per 70 kilogram subject. Boosting dosages of between about 1.0 ng to about 50,000 pg of chimeric 10 antigen pursuant to a boosting regimen over days to months may be administered depending upon the subject's response and condition. Administration should continue until at least clinical symptoms or laboratory tests indicate that the condition has been prevented, arrested, slowed or eliminated and for a period thereafter. The dosages, routes of administration, and dose schedules are adjusted in accordance with methodologies 15 known in the art. A human unit dose form of a chimeric antigen is typically included in a pharmaceutical composition that comprises a human unit dose of an acceptable carrier, in one embodiment an aqueous carrier, and is administered in a volume/quantity that is known by those of skill in the art to be useful for administration of such polypeptides to 20 humans (see, e.g., Remington: The Science and Practice of Pharmacy, 20th Edition, A. Gennaro, Editor, Lippincott Williams &amp; Wilkins, Baltimore, Md., 2000). As appreciated by those of skill in the art, various factors can influence the ideal dose in a particular case. Such factors include, for example, half life of the chimeric antigen, the binding affinity of the chimeric antigen, the immunogenicity of the composition, the desired 25 steady-state concentration level, route of administration, frequency of treatment, and the influence of other agents used in combination with the treatment method of the invention, as well as the health status of a particular subject. Generally, sufficient chimeric antigen to elicit an immune response to the chimeric antigen is administered to a subject. The TBD targets the chimeric antigen to 30 specific receptors on APCs, such as DCs. The chimeric antigen is internalized, processed 93 through antigen presentation pathways to elicit both humoral as well as cellular immune responses. In certain embodiments, the compositions of the present invention are employed in serious disease states, that is, life-threatening or potentially life-threatening situations. 5 In such cases, as a result of the relative nontoxic nature of the chimeric antigen in preferred compositions of the invention, it is possible and may be felt desirable by the treating physician to administer substantial excesses of these chimeric antigens relative to these stated dosage amounts. The concentration of chimeric antigen of the invention in the pharmaceutical 10 formulations can vary widely, i.e., from less than about 0.10%, usually at or at least about 2% to as much as 20% to 50% or more by weight, and will be selected primarily by fluid volumes, viscosities, etc., in accordance with the particular mode of administration selected. The pharmaceutical compositions can be delivered via any route known in the art, 15 such as parenterally, intrathecally, intravascularly, intravenously, intramuscularly, transdermally, intradermally, subcutaneously, intranasally, topically, orally, rectally, vaginally, pulmonarily or intraperitoneally. Preferably, the composition is delivered by parenteral routes, such as subcutaneous or intradermal administration. The pharmaceutical compositions can be prepared by mixing the desired chimeric 20 antigens with an appropriate vehicle suitable for the intended route of administration. In making the pharmaceutical compositions of this invention, the chimeric antigen is usually mixed with an excipient, diluted by an excipient or enclosed within a carrier that can be in the form of a capsule, sachet, paper or other container. When the pharmaceutically acceptable excipient serves as a diluent, it can be a solid, semi-solid, or liquid material, 25 which acts as a vehicle, carrier or medium for the therapeutic agent. Thus, the compositions can be in the form of tablets, pills, powders, lozenges, sachets, cachets, elixirs, suspensions, emulsions, solutions, syrups, aerosols (as a solid or in a liquid medium), ointments containing, for example, up to 10% by weight of the chimeric antigen, soft and hard gelatin capsules, suppositories, sterile injectable solutions, and 30 sterile packaged powders. 94 Some examples of suitable excipients include, but are not limited to, dextrose, sucrose, glycerol, sorbitol, mannitol, starches, gum acacia, calcium phosphate, alginates, tragacanth, gelatin, calcium silicate, microcrystalline cellulose, polyvinylpyrrolidone, cellulose, sterile water, syrup, and methyl cellulose. The formulations can additionally 5 include: lubricating agents such as talc, magnesium stearate, and mineral oil; wetting agents; emulsifying and suspending agents; preserving agents such as methyl- and propylhydroxy-benzoates; sweetening agents; and flavoring agents. The compositions of the invention can be formulated so as to provide quick, sustained or delayed release of the chimeric antigen after administration to the subject by employing procedures known in 10 the art. See, e.g., Remington, supra, at pages 903-92 and pages 1015-1050. For preparing solid compositions such as tablets, the chimeric antigen is mixed with a pharmaceutical excipient to form a solid preformulation composition containing a homogeneous mixture of a chimeric antigen of the present invention. When referring to these preformulation compositions as homogeneous, it is meant that the chimeric antigen 15 is dispersed evenly throughout the composition so that the composition may be readily subdivided into equally effective unit dosage forms such as tablets, pills and capsules. The tablets or pills of the present invention may be coated or otherwise compounded to provide a dosage form affording the advantage of prolonged action. For example, the tablet or pill can comprise an inner dosage and an outer dosage component, 20 the latter being in the form of an envelope over the former. The two components can be separated by an enteric layer, which serves to resist disintegration in the stomach and permit the inner component to pass intact into the duodenum or to be delayed in release. A variety of materials can be used for such enteric layers or coatings, such materials including a number of polymeric acids and mixtures of polymeric acids with such 25 materials as shellac, cetyl alcohol, and cellulose acetate. The liquid forms in which the novel compositions of the present invention may be incorporated for administration orally or by injection include aqueous solutions, suitably flavored syrups, aqueous or oil suspensions, and flavored emulsions with edible oils such as corn oil, cottonseed oil, sesame oil, coconut oil, or peanut oil, as well as elixirs and 30 similar pharmaceutical vehicles. 95 In preparing a composition for parenteral administration strict attention must be paid to tonicity adjustment to reduce irritation. A reconstitutable composition is a sterile solid packaged in a dry form. A reconstitutable composition is preferred because it is more stable when stored as a dry solid rather than in a solution ready for immediate 5 administration. The dry solid is usually packaged in a sterile container with a butyl rubber closure to ensure the solid is kept at an optimal moisture range. A reconstitutable dry solid is formed by dry fill, spray drying, or freeze-drying methods. Descriptions of these methods may be found, e.g., in Remington, supra, at pages 681-685 and 802-803. Compositions for parenteral injection are generally dilute, and the component 10 present in the higher proportion is the vehicle. The vehicle normally has no therapeutic activity and is nontoxic, but presents the chimeric antigen to the body tissues in a form appropriate for absorption. Absorption normally will occur most rapidly and completely when the chimeric antigen is presented as an aqueous solution. However, modification of the vehicle with water-miscible liquids or substitution with water-immiscible liquids can 15 affect the rate of absorption. Preferably, the vehicle of greatest value for this composition is isotonic saline. In preparing the compositions that are suitable for injection, one can use aqueous vehicles, water-miscible vehicles, and nonaqueous vehicles Additional substances may be included in the injectable compositions of this invention to improve or safeguard the quality of the composition. Thus, an added 20 substance may affect solubility, provide for subject comfort, enhance the chemical stability, or protect the preparation against the growth of microorganisms. Thus, the composition may include an appropriate solubilizer, substances to act as antioxidants, and substances that act as a preservative to prevent the growth of microorganisms. These substances will be present in an amount that is appropriate for their function, but will not 25 adversely affect the action of the composition. Examples of appropriate antimicrobial agents include thimerosal, benzethonium chloride, benzalkonium chloride, phenol, methyl p-hydroxybenzoate, and propyl p-hyrodxybenzoate. Appropriate antioxidants may be found in Remington, supra, at p. 1015-1017. In certain embodiments, liposomes, nanocapsules, microparticles, lipid particles, 30 vesicles, and the like, are used for the administration of the chimeric antigens of the present invention. In particular, the compositions of the present invention may be 96 formulated for delivery either encapsulated in a lipid particle, a liposome, a vesicle, a nanosphere, or a nanoparticle or the like. Alternatively, compositions of the present invention can be bound, either covalently or non-covalently, to the surface of such carrier vehicles. 5 Compositions administered via liposomes may also serve: 1) to target the chimeric antigen to a particular tissue, such as lymphoid tissue; 2) to target selectively to APCs; 3) to carrier additional stimulatory or regulatory molecules; or 4) to increase the half-life of the chimeric antigen composition. Liposomes include emulsions, foams, micelles, insoluble monolayers, liquid crystals, phospholipid dispersions, lamellar layers 10 and the like. In these preparations, the chimeric antigen to be delivered is incorporated as part of a liposome, alone or in conjunction with a molecule that binds to a receptor prevalent among lymphoid cells, such as monoclonal antibodies that bind to the CD45 antigen, or with other therapeutic or immunogenic compositions. Thus, liposomes either filled or decorated with a desired chimeric antigen of the invention can be directed to the 15 site of lymphoid cells, where the liposomes then deliver the chimeric antigens. Liposomes for use in accordance with the invention are formed from standard vesicle forming lipids, which generally include neutral and negatively charged phospholipids and a sterol, such as cholesterol. The selection of lipids is generally guided by consideration of, e.g., liposome size, acid lability and stability of the liposomes for the desired route of 20 administration, e.g., in the blood stream. A variety of methods are available for preparing liposomes [as described in, e.g., Szoka et al. (1980) Ann. Rev. Biophys. Bioeng. 9:467 508; and U.S. Pat. Nos. 4,235,871, 4,501,728, 4,837,028, and 5,019,369]. A liposome suspension containing a chimeric antigen may be administered intravenously, locally, topically, etc. in a dose which varies according to, inter alia, the manner of 25 administration, the chimeric antigen being delivered, and the stage of the disease being treated. Compositions for inhalation or insufflation include solutions and suspensions in pharmaceutically acceptable, aqueous or organic solvents, or mixtures thereof, and powders. The liquid or solid compositions may contain suitable pharmaceutically 30 acceptable excipients as described herein. The compositions can be administered by the oral or nasal respiratory route for local or systemic effect. Compositions in 97 pharmaceutically acceptable solvents may be nebulized by use of inert gases. Nebulized solutions may be inhaled directly from the nebulizing device or the nebulizing device may be attached to a facemask tent, or intermittent positive pressure breathing machine. Solution, suspension, or powder compositions may be administered, preferably orally or 5 nasally, from devices that deliver the formulation in an appropriate manner. Another formulation employed in the methods of the present invention employs transdermal delivery devices ("patches"). Such transdermal patches may be used to provide continuous or discontinuous infusion of the chimeric antigen of the present invention in controlled amounts. The construction and use of transdermal patches for the 10 delivery of pharmaceutical agents is well known in the art. See, for example, U.S. Pat. No. 5,023,252, herein incorporated by reference. Such patches may be constructed for continuous, pulsatile, or on demand delivery of pharmaceutical agents. Additionally, it may be advantageous to include at least one antiviral therapeutic or chemotherapeutic in addition to the chimeric antigen and pharmaceutical excipient. 15 These include, but are not limited to, interferon-a 2a/b, and antiviral agents such as ribavirin. In some embodiments it may be desirable to include in the pharmaceutical compositions of the invention at least one component which primes B-lymphocytes or T lymphocytes. Lipids have been identified as agents capable of priming CTL in vivo. For 20 example, palmitic acid residues can be attached to the c-and a1-amino groups of a lysine residue and then linked, e.g., via one or more linking residues such as Gly, Gly-Gly-, Ser, Ser-Ser, or the like, to an immunogenic peptide. The lipidated peptide can then be administered either directly in a micelle or particle, incorporated into a liposome, or emulsified in an adjuvant, e.g., incomplete Freund's adjuvant. In a preferred embodiment, 25 a particularly effective immunogenic composition comprises palmitic acid attached to 8 and ax-amino groups of Lys, which is attached via linkage, e.g., Ser-Ser, to the amino terminus of the immunogenic peptide. As another example of lipid priming of CTL responses, E. coli lipoproteins, such as tripalmitoyl-S-glycerylcysteinlyseryl-serine (P 3 CSS) can be used to prime virus 30 specific CTL when covalently attached to an appropriate peptide [see, e.g., Deres et al. (1989) Nature 342:561]. Chimeric antigens of the invention can be coupled to P 3 CSS, 98 for example, and the lipopeptide administered to an individual to specifically prime an immune response to the target antigen. While the compositions of the present invention should not require the use of adjuvants, adjuvant can be used. Various adjuvants may be used to increase the 5 immunological response, depending on the host species, and including but not limited to Freund's (complete and incomplete), mineral gels such as aluminum hydroxide, surface active substances such as lysolecithin, detergents, pluronic polyols, polyanions, peptides, oil emulsions, keyhole limpet hemocyanins, dinitrophenol, immunostimulatory polynucleotide sequences, and potentially useful human adjuvants such as BCG (bacille 10 Calmette-Guerin) and corynebacterium parvum. Additional adjuvants are also well known in the art. H. Article of Manufacture Another aspect of this invention provides an article of manufacture that comprises 15 a container holding a composition, comprising a chimeric antigen, that is suitable for injection or reconstitution for injection in combination with printed labeling instructions providing a discussion of how to administer the composition parenterally, e.g. subcutaneously, intramuscularly, intradermally, nasally or infravascularly. The composition will be contained in any suitable container that will not significantly interact 20 with the composition and will be labeled with the appropriate labeling that indicates it will be for parenteral use. Associated with the container will be the labeling instructions consistent with the method of treatment as described hereinbefore. The container that holds the composition of this invention may be a container having a liquid composition suitable for injection that has an appropriate needle for injection and a syringe so that the 25 patient, doctor, nurse, or other practitioner can administer the chimeric antigen. Alternatively, the composition may be a dry or concentrated composition containing a soluble version of the chimeric antigen, to be combined or diluted with an aqueous or nonaqueous vehicle to dissolve or suspend the composition. Alternatively, the container may have a suspension in a liquid or may be an insoluble version of the salt for 30 combination with a vehicle in which the insoluble version will be suspended. Appropriate 99 containers are discussed in Remington, supra, pages 788-789, 805, 850-851 and 1005 1014 The kit of the invention will typically comprise the container described above and one or more other containers comprising materials desirable from a commercial and user 5 standpoint, including buffers, diluents, filters, needles, syringes, and package inserts with instructions for use. A label can be present on the container to indicate that the composition is used for a specific therapy or non-therapeutic application, and can also indicate directions for either in vivo or ex vivo use, such as those described above. Directions and or other information can also be included on an insert which is included 10 with the kit. I. Combination Therapy Another aspect of the invention provides compositions for treating viral infections comprising a chimeric antigen and an antiviral agent. The invention also provides 15 methods of treating viral infections comprising administering a chimeric antigen and an antiviral agent, either concurrently or sequentially. The use of a chimeric antigen in combination with an antiviral agent, such as a nucleoside analogue, may prove to be highly efficacious in inducing sustained responses in the treatment of subjects suffering from chronic hepatitis C. The mechanisms of action 20 of the two agents used in combination may produce synergistic effects in treatment of hepatitis C subjects. For example, a combination of an HCV antiviral such as ribavirin along with the HCV chimeric antigens described herein will produce antigen-specific cellular as well as humoral immune response and thus clear HCV infection in chronically infected subjects. 25 V. EXAMPLES The following non-limiting examples provide further illustration of the invention. A. Example 1: Construction of TBD Expression Vector 100 Mouse IgGI DNA sequences encoding amino acids of a portion of CHI-Hinge CH2-CH3 region was generated from mRNA isolated from the hybridoma (2C 12), which produces mAb against HBV surface antigen (sAg). Total mRNA was isolated using Trizol@ reagent (Gibco BRL cat. No. 15596-026) and the cDNA of the target binding 5 domain (TBD; mouse immunoglobulins fragment) was generated by RT-PCR using Superscript First-strand Synthesis (Invitrogen Cat. No. 11904-018). The PCR primers contained linker sequences encoding the linker peptide SRPQGGGS- (SEQ ID NO: 1) at the 5' terminus, a unique Not I site at the 5' and a unique Hind III restriction site at the 3' end. The resulting cDNA contains (5' Not I )-linker sequence-CH1(VDKKI) (SEQ ID NO: 10 2). -Hinge Region- CH2- CH 3</div>
    <div class="description-paragraph style-scope patent-text">-(</div>
    <div class="description-paragraph style-scope patent-text">3 * Hind III). Following digestion with the respective enzymes, the fragment is ligated with pFastBac HTa expression vector plasmid (Invitrogen) using the same restriction enzyme sites. The 5' primer used for PCR amplification was (Sense) 5' TGTCATTCTGCGGCCGCAAGGCGGCGGATCCGTGGACAAGAAAATTGTGC 15 CCAGG (SEQ ID NO: 3) and the 3' primer was (antisense) 5' ACGAATCAAGCTTTGCAGCCCAGGAGAGTGGGAGAG (SEQ ID NO: 4), which contained the Not I and Hind III sites, respectively. The following is the protocol used for directional cloning. The generated fragment was digested with the respective enzymes, purified on agarose gel and cloned into the vector plasmid. The DNA sequence and the 20 conectness of the ORF were verified by standard sequencing methods. Following the cloning of the DNA encoding the target binding domain into the pFastBac HTa donor plasmid, the recombinant proteins was expressed using the Bac-to BacTM baculoviras expression system (Invitrogen). The cloned gene was transferred into a baculoviras shuttle vector via site-specific transposition in a strain of E. coli, DHlOBac. 25 The DHlOBac cells contain the shuttle vector, which confers kanamycin resistance and a helper plasmid, which encodes the transposase and confers resistance to tefracycline. A 100 [d aliquot of competent DHlOBac cells was thawed on ice, the pFastBac HTa based plasmids were added and the mixture incubated on ice for 30 minutes. The mixture was heat shocked for 45 seconds at 42'C and then chilled on ice for 2 minutes. The mixture 30 was then added to 900 [L of LB media and incubated for 4 hours at 37'C. The 101 transformed cells were serially diluted with LB to 10"' and 10-2 and 100 l of each dilution was plated on LB agar plates supplemented with 50 [g/ml kanamycin, 7 [g/ml gentamicin, 10 [g/ml tefracycline, 100 [g/ml X-gab and 40 [g/ml IPTG and incubated for at least 36 hours at 37'C. The gentamicin resistance was confened by the pFastBac 5 HTa and the X-gal and IPTG (isopropylthio-p-D-galactoside) were used to differentiate between white colonies (recombinant plasmids) and blue colonies (non-recombinant). The white colonies were picked and inoculated into 2ml of LB supplemented with 50 [g/ml kanamycin, 7 [g/ml gentamicin and 10 [g/ml tefracycline and incubated overnight at 37'C, with shaking. A sterile loop was used to sample a small amount of the overnight 10 culture and the sample was streaked onto a fresh LB agar plate supplemented with 50 [g/ml kanamycin, 7 [g/ml gentamicin, 10 [g/ml tefracycline, 100 [g/ml X-gab and 40 [g/ml IPTG and incubated for at least 36 hours at 37'C to confirm a white phenotype. Recombinant bacmids were isolated by standard protocols (Sambrook, supra), the DNA sample was dissolved in 40 l of TE (lOmM Tris-HCL pH 8, lniM EDTA) and used for 15 transfections. In order to produce baculovirases, the bacmid was transfected into Sf9 insect cells. Sf9 cells (9 x 105) were seeded into each well of a 6-well cell culture dish (35mm wells) in 2 ml of ESF 921 (Expression Systems) and allowed to attach for at least 1 hour at 27'C. Transfections were carried out using Cellfection@ Reagent (Invitrogen, Cat. No. 20 10362-010) as per the protocols provided by the supplier of the Sf9 cells. Following transfection, the cells were incubated at 27'C for 72 hours. The medium containing baculovirus was collected and stored at 4'C in the dark. '- The efficiency of the transfection was verified by checking for production of baculoviral DNA. The isolated baculovirus DNA was subjected to PCR to screen for the inserted gene encoding the 25 TBD. The primers used were (sense) 5' TATTCCGGATTATTCATACCG (SEQ ID NO: 5) and 3' (antisense) 5' CTCTACAAATGTGGTATGGC (SEQ ID NO: 6). Amplified products were ran on an agarose gel (0.8%). The expression of the heterologous protein in the cells was verified by SDS polyacrylamide gel electrophoresis (SDS-PAGE) and Western blots using the 6xHis tag monoclonal antibody (Clonetech) as the probe. 102 Once production of baculovirus and the expression of protein were confunmed, the Virus production was amplified to produce a concentrated stock of the baculoViras that cany the gene encoding the target binding domain. It is standard practice in the art to amplify the baculoviras at least two times, and in all protocols described herein this 5 standard practice was adhered to After the second round ofamplification, the concentration of the generated baculoVirus was quantified using a plaque assay according to the protocols described by the manufacturer ofthe kit (Invitrogen). The most appropriate concentration ofCte virus to infect High Five cells and the optimum time point for the production of the desired protein was established as well. Generally, for the 10 expression of the TB) an MOI of I and a time period of 48 hours was used. B. Example 2: Construction ofChimeric Antigen Expression Vectors The DNA encoding the desired viral antigen was generated from the template using PCR methodology using the 5' sense and 3 anti-sense primers indicated in Table 2. The 5' end of the resulting amplified fragment contained the unique restriction site "5' is enzy," and the 3' end contained the unique restriction site 3' eazy," each of which was used for ligations. TABLE2: ConstructionofChimericAntigenVectors Virian ien Sec primer A ti-sens |Tempai r 5'el, 3 'stI zy BVs1s2 SEQIDNo: 7 SEO]DNt- . RSETB HBV s1/52 .B_ Ned mV1 /2S SEoIDtio:9 SriNO10 |ptBV9 9a I o Noti mBycore EJDNOrl SEO-]2 |palB99 NIil Nlr DIBVPres/S sEQomm5 SEQ]DN:3 |plasBae taPres/ EcoR rN DriBV es SEQoDNo:S SEOtDNOs]4 |pFaaNch S/ EcoU Ner UHBVCoi, SEQ1DNO:h$ SE1IDNO-i pRSETBDBVCor Nio] NOt[ hCVCorel19]) SErIDlO: 17 S No:m |pV77 Eco I R HCVCordii72 SEO:DNO:? S1 $JsNO:]9 pCV-H477c E S iCV NS5A 5EQINO:20 SEQ1DINO:2 | pCV-H77 nE dHCVEI SEOIDlNO:22 SEOIDN'23 |pCV-H77a EcoR1 ]Ii E2 S lDNo:24 SEisDO;:2|C-HT/ EI Sp HCV SEO]DNO:22 SEOIDNO:25 |pCV11c E2 Spl Amplified DNA was digested with appropriate 5' and 3' restriction enzymes and ligated 20 with a pFastBac HTa expression vector to generate the expression plasmid for the viral antigen alone. The same fragment of DNA was also ligated with the plasmid pFastBac 103 HTa-TBD, described in Example 1, following digestion with the respective enzymes to produce the expression plasmid for the viral antigen fused to the target binding domain. The resulting plasmid was used to produce recombinant baeulovirus, as described in Example 1, for subsequent use in expression of the chimeric antigen. The DNA and 5 amino acid sequences of the chimeric antigens are provided in Table . TABLE 3: Chimeric Antigen Sequences Construct DNA sequence| Expssed Protein BVS]/S2-TBD SEsMNO:2c. SE IDNO: 27 BV S/S2/STBD S DNO:2| SEQ IDNO:29 HBV Cor-TBD SEQ IDNO:30 SE IDNO!31 DHBV PrS-ThD SEQ ID NO: 32 SEQ ] NO: 33 DHBV PeS/S-TBD SEQ ]D NO: 34 SEQ ] NO: 35 DHBV Core-TBD SEQ ]DNO: 36 SEQ ]D-NO:37 HiCI1-91TBD SEQMNO38 | SE]NO:39 HCVCore(I-177)-TBD DNO 40 |SEQIDNO: 4] HCVNS5A-TBD) SEQ NO: 42 SEQIDNO: 43 HCVE-TBD . SEQLDNO:44 SEQ]DNO:45 HCV B2-TD sBQMNO:46 SEQ DNO:47 HCV1/E2-TBD SEIDNO:-48 SEQ ID NO: 49 C. Example 3: Expression and Purification of TBD, Viral Antigens and Chimeric Antigens 10 Recombinant baemids ofstandardized multiplicity ofinfection (MOD were used to infect High FiveTM insect cells. For suspension cultures, cells were seeded at a density of 3 x 109cells/meL and incubated at 27. sC with shaking at 138 rpm until the cell density reached 2-3 x 106 cellsmrL. Standardized amounts of the respective recombinant baculoviras was added to the cells. The incubation temperature was 27.5-C and the is appropriate infection period was standardized for individual protein expression. The cells were harvested by cetrifgliation at 2,500 rpm for 10 minutes at 4 "C and used for the purification of the recombinant proteins. Unused portions of cells were snap frozen in liquid nitrogen and stored at -70C. Recombinant proteins were purified under denaturing conditions. The cells were 20 lysed in a buffer containing 6 M Guanidinium-HCI in 100 M NaHzPO , 10 u Tris, 300 mM Nad, 10 mM Imidazole, pH 8.0 -lysis buffer). The suspension was sonicated on ]04 ice with 5 pulses of 1 minute per pulse at a power setting of 60 watts, and was mixed at room temperature for 1 hour. The lysate was centrifuged at 10,000 x g for 10 min to remove unbroken cells and cell debris. The supernatant was loaded on to a Ni-NTA agarose (Qiagen) bead column (1 x 5 cm/ 100 mL cell lysate), pre-equilibrated with lysis 5 buffer. Following loading, the column was washed with 20 column volumes of 6 M guanidinium-HCl in 100 mM NaH 2</div>
    <div class="description-paragraph style-scope patent-text">PO</div>
    <div class="description-paragraph style-scope patent-text">4 , 10 mM Tris, 300 mM Nad, 40 mM Imidazole, pH 8.0 (wash buffer 1), followed by washes with 20 column volumes of 8 M urea in 100 mM NaH 2 PO , 10 mM Tris, 300 mM NaCb 40 mM imidazole, pH 8.0 (wash buffer 2). The bound protein was eluted with a buffer containing 8 M urea, 100 mM NaH 2</div>
    <div class="description-paragraph style-scope patent-text">PO</div>
    <div class="description-paragraph style-scope patent-text">4 , 10 10 mM Tris, 300 mM Nad, 250 mM imidazole, pH 8 (Elution Buffer). The fractions containing the protein were pooled and dialyzed at 4'C against multiple changes of dialysis buffer (lOmM NaH 2</div>
    <div class="description-paragraph style-scope patent-text">PO</div>
    <div class="description-paragraph style-scope patent-text">4 , 300 mM NaCI). Purified proteins were characterized using standard biochemical techniques including SDS gel electrophoresis, isoelectric focusing, and western blot analysis using antibodies against different domains of the 15 expressed protein. D. Example 4: Breaking Tolerance to a " Self " Protein Using a Chimeric Antigen Fusion Protein. In order to evaluate the immune response to chimeric antigen fusion proteins, mice were immunized with purified HBV S1/S2, TBD or S 1/S2-TBD proteins and the 20 antibodies produced against the individual proteins were quantified. Proliferation of splenic T cells harvested from immunized mice was evaluated following challenge with the respective proteins. BALB/c mice at 15 weeks of age were used for the immunizations. Mice were injected subcutaneously with S1/S2-TBD (4.15 [tg),. S1/S2 (4.15 [tg) or TBD (4.15 [tg) 25 four times at two week intervals. Blood samples were collected before the start of immunizations and a week following each of the immunizations. Serum was prepared from clotted blood samples and used for the estimation of antibody levels produced by the host animal against the respective antigens injected. 105 1. ELISA for detection of antibodies against HBV S/S2, TBD or Sl/S2-TBD A 96 well plate was coated with antigens HBV S I /S2. TBD or SI/S2-TBD ala concentration of I.d pg/rmL overnight at 4'C. The plate was washed with PBS contaimng 2% BSA. Diluted scram from the respective animal was added to each of the wells at various 5 dilutions (1 :10 - 1:500) and incubated at 37"C fit lhr. The plate was washed with PBS containing 0.05% Tween 20 (wash buffer). Goat anti-mouse JgG Fab-horse radish peroxidase (HRP) (5000) dilution was added to the wells and incubated at 37*C for hr. The plate was washed with wash buffer and color was developed using 2-2' azino-di-(3 ethyl benzylthiazoline-6-sulfonate) (KPL, Guildford, UK). The optical density of 10 resulting color in the samples was measured using an ELISA plate reader (Molecular Devices, USA) at a wavelength of 405 nm. Negative control for the experiment was pre immune secram from the same animal, which was subtracted from all the experimental values. The results for mice immunized with HBV SI/S2-TD are presented in Table 4. The chimeric antigen elicits a strong antibody response against the chimeric antigen is (SI/S2-TBD). TABLE 4: Humoral response to S U/S2-TBD Levelofantibndybindingto: HBV3SNTBD) 14BVSU1S2 [ TBD Mice jflBV SI/S2-TBD- 0.192 0.059 | 0.04 amnuized HBV S/SZ | 0073 | .05 | .025 wih TBD_ 0.076 0.017 0.036 The antibody response here is ofa multivalent (or nulti-epitopic) nature. The results presented in Table 4 show that antibodies produced by mice immunized with HBV 20 SI /S2-TBD bind to the chimeric antigen and to the SI /S2 protein target coated on the plate. Therefore, antibodies are produced against the S I /S2 component of the chimeric antigen. Likewise, antibodies produced by mice immunized with HBV SI/S2-TBD bind to the target binding domain protein (Table 4). Chimeric antigen that contains protein of mouse origin can generate a hunmoral immune response in a mouse, evidence that the 25 chimeric antigen can convert a "self antigen into "foreign." Accordingly, it is possible to break tolerance to a protein otherwise treated as a "self protein. 106 2. T cell proliferation assay Animals were sacrificed a week, after the fourth immunization, the spleen was removed, and a single cell suspension was produced. Cells were seeded in triplicate at a cell density of 4x1O 5 cells/well in a 96-well plate. They were loaded with the respective antigens, 5 HBV SI/S2,TBD or SI[S2-TBD, at concentrations of0.-1 fg/tL, L.0 kg/iL and 10 pg/miL Negative control cells received media alone and the positive control for T cell proliferation was Phytohemagglutinin (PHA) at 1.0 - 5.0 gmL. The cell cultures were incubated for 4 days at 37"C under an atmosphere of 7% CO, Each of the wells of cells was pulsed with 1.0 mCi of[H}-thymidine and incubated for an additional 18 hours. The 10 cells were harvested using TOMTEC MACH 3 cell harvester (Hamden, CT, USA) and the radioactivity bound to the glass fibre filter (Wallac Oy, Turku, Finland) was quantified using a Wallac Trilux 1450 Microbeta liquid scintillation and luminescence counter (Wallac, USA). The results are shown in Table 5. TABLE 5; Cellular respoime to UBVS/S2-TB) Mean counts per minute (C PM) 0L |10gmL HMys TBD 36.7 I7.0 327 50 213,7.0 HBVSl/S2 10.7 2.1 26.7 9.6 |25.7 10.3 TBD |32.7 +19.1 [17.0 +2.6 135.7 27.2 | e123 12.5 L.omgmlPA 33 lS.6 15 T cell proliferation was seen when challenged with HBV S1/S2-TBD, SI/S2 or TBD. Immunization with the chimeric antigen induced a multivalent T cell response, i.e., a response against different parts of the same protein. Chimeric antigen that contain protein of mouse origin can generate a cellular immune response in a mouse, evidence that the chimeric antigen can convert a "self antigen into "foreign." Therefore, it is possible to 20 break tolerance to a protein otherwise created as a "self protein. E. Example 5: Antigen Presentation Assays The ability of HBV SI/S2-TBD to elicit an immune response was measured using an ex vivo antigen presentation assay. The generation of an effective T cell response following 107 multiple stimulations of naYve T cells with antigen-loaded antigen presenting cells (APCs) such as dendritic cells (DCs) was assessed by quantitating the increase in the number of antigen-specific T cells as well as the ability of the T cells to produce the Thl cytokine IFN-y. 5 1. Selection of Monocytes by Adhesion Peripheral blood mononuclear cells (PBMCs) were thawed by the addition of AIM-V (ratio of 9 ml of AIM-V added to 1 ml of frozen cells). The cells were then centrifuged at 200x g for 5 min, the supernatant removed, and the cells resuspended in AIM-V/1% matched serum and added to either a 100 mm culture dish or a T-25 culture flask. The 10 PBMCs were incubated for 1 hr at 37 0 C in a humidified incubator under 7% CO 2 . To remove non-adherent cells, the culture was triturated several times, the supernatant discarded, and the cells washed once with AIM-V medium. Monocytes were harvested with a cell scraper and centrifuged at 300x g for 5 min. The cell pellet was re-suspended in AIM-V/2.5% 0 matched seram at 2 x 106 cells/ml and seeded into a 24-well dish. The 15 cytokines IL-4 and GM-CSF (1000 IU/ml each) were added to drive the differentiation of monocytes into immature DCs. 2. Fast or Slow Antigen Presentation Assay For the Fast Antigen Presentation Assay (APA), antigen was added to immature DCs within 4 to 24 hr of isolation. After a further 24 hr, antigen loaded immature 20 monocytes were induced to mature by culturing with PGE 2 (1 pM), IL-1j (10 ng/ml), and TNF-a (10 ng/ml) for 24 hr. The mature DCs were then co-cultured (first stimulation) with autologous T cells. The T cells were generated from the same PBMCs as the DCs by means of negative selection using a magnetic T cell isolation kit (Dynal) according to the manufacturer's directions. T cells were then re-stimulated 7 days later with antigen 25 loaded mature DCs in the presence of IL-2 (20 IU/ml), IL-7 (10 ng/ml), and IL-15 (5 ng/ml). Following a 7 day incubation, T cells were re-stimulated a third time with antigen loaded mature DCs. The third stimulation lasted for 6 hr whereupon the T cells were harvested and immunostained for CD3, CD8 and IFN-y expression, and analyzed by flow 108 cytometry. For the Slow APA, monocytes were allowed to differentiate into immature DCs in the presence of GM-CSF and IL-4 for 5 to 6 days before addition of antigen. Two hours after antigen addition, immature DCs were matured with TNF-a (10 ng/ml) and IFN-a (50 IU/ml). Seven days post isolation, the mature DCs were co-cultured (first 5 stimulation) with autologous T cells (described above). T cells were then re-stimulated 7 days later with antigen loaded mature DCs in the presence of IL-2, IL-7, and IL-15. Following a 7 day incubation, cells were re-stimulated a third time with antigen loaded mature DCs. After an 18 hr incubation the T cells were harvested and immunostained for CD3, CD8 and IFN-y expression, and analyzed by flow 10 cytometry. 2. PBMC Antigen Presentation Assay In this assay, the initial culture consists of total PBMCs (ie. lymphocytes and monocytes) that are incubated with antigen and IL-2 with the assumption that the system resembles the in vivo immune response since all the cell types are present to participate (Maini, 15 M.K et. al J. Exp. Med. 191 :1269-1280, 2000). PBMCs are thawed, washed and immediately incubated with antigen. Following 4 days of culture to allow for antigen uptake and presentation, IL-2 (20 IU/ml) was added and left for an additional 8 days (i.e. day 12 of the experiment). Two days prior to the second stimulation (i.e. day 10 of the experiment), DCs are isolated by adhesion as described above, and immediately 20 incubated with GM-CSF, IL-4 and antigen for 24 hr. As with the Fast APA, the immature DCs are allowed to differentiate for 24 hr following the addition of PGE 2 , IL-1j, and TNF-a. The loaded mature DCs are then added to the PBMC culture (second stimulation, day 12 of the experiment) in the presence of IL-2, IL-7, and IL-15. The third stimulation occurred on day 21 of the experiment with antigen loaded mature DCs prepared 2 days 25 prior as described. Following a 6 hr incubation, the T cells were harvested and immunostained for CD3, CD8 and IFN-y expression, and analyzed by flow cytometry. 109 For all antigen presentation assays discussed above, a portion of the T cells at the end of the assay were incubated for an additional 3 - 5 days and examined for specific T cells by tetramer analysis (see below). 4. HBV S1/S2 Elicits a T cell Response Against HBV SI and S2 Peptides 5 The PBMC APA was used to generate T cells which were then assessed for their antigen specificity. Thus, PBMCs from healthy HLA-A2 individuals were cultured in AIM-V containing 2.5% matched sera in 96-well plates at 5 x 105 cells/mb Antigen (ie. 10 [g/ml S1/S2-TBD) was added and the cells were cultured for 4 days at 37 0 C. IL-2 was then added at 20 IU/ml and the cells were cultured for an additional 8 days, with media 10 changes (AIM-V/2.5% matched serum and 20 IU/ml IL-2) every 2-3 days. The majority of the cells remaining at the end of the 12 day culture were T cells, and these T cells were restimulated with autologous antigen-loaded mature DCs in the presence of IL-2 (20 IU/ml), IL-7 (10 ng/ml), and IL-15 (5 ng/ml). The antigen-loaded mature DCs for the second and third stimulation of the T cells 15 in the APA were generated over a 48 hr period using the procedure described below. The monocytes were isolated from total PBMC by adherence on plastic tissue culture dishes. The cells, about 85% of which were monocytes as determined by FACS analysis (CD1 lc+, CD14+, CD19-, and CD3-), were cultured in a 96-well plate at 1 x 105 cells/well containing 100 [d of AIM-V/2.5% 0 matched sera with the cytokines IL-4 and GM-CSF at 20 1000 IU/ml, and 4 hr later antigen such as S1/S2-TBD was added. Following a 20 hr incubation, the generated immature DCs were differentiated to mature DCs by culturing for a further 24 hr in the presence of PGE 2 (1 x 10"6 M), IL-1j (10 ng/ml), and TNF-a (10 ng/ml). T cells were cultured for 7 days following the second stimulation, with media 25 changes (AIM V with 2.5% matched serum and 20 IU/ml IL-2) every 1-2 days. The T cells (day 19 of culture) were then stimulated a third time with antigen-loaded mature DC (generated over a 2 day procedure as outlined above) in the presence of IL-2, IL-7, and IL-15 (as above) and either assessed for IFN-y production after a 6 hr culture or cultured 110 for 5 days (with media changes every 1-2 days with AIM-V/2.5% 0 matched serum and 20 IU/ml IL-2) and then assessed for T cell specificity to HBV preS antigen using HBV preS tetramers (day 24 of culture). Tetramer analysis was performed with custom synthesized iTag MHC class I 5 tetramers (Beckman Coulter) according to the manufacturer's protocol. Thus cells were harvested, washed, and transfened to a 96-well v-bottom plate at ~2 x 105 cells/well in 20 [b The cells were labeled at 20'C for 30 min with mAb specific to CD3 (anti-CD3-FITC) and CD8 (anti-CD8-Cy-Chrome) together with either 2 1d PE-conjugated HLA-A*0201 preSl tetramer (GMLTPVSTI, SEQ ID NO: 50) or a preS2 tetramer (NIASHISSI, SEQ 10 ID NO: 51). The cells were then washed, fixed with 2% paraformaldyde in PBS and transfened into 5 ml FACS tubes. The cells were acquired on a FACSCalibur (BD Biosciences) with 80,000-100,000 events per sample. Analysis was performed using CellQuest software (BD Biosciences) with a gate on the viable (based on the FSC/SSC profile) CD3+ population and the percentage of CD8+ cells labeling with tetramer was 15 determined. When PBMCs were cultured with HBV S1 S2-TBD at 10 [g/ml and restimulated twice with HBV S1/S2-TBD-loaded mature DC, a marked percentage of the cells labeled positive with SI tetramer (Fig. 2) and S2 tetramer (Fig. 3). This is in contrast to T cells cultured with antigen-unloaded mature DC, where the number of tetramer positive cells was not significant. Thus, Sl/S2-TBD-loaded mature DC were able to 20 induce the generation of a significant number of T cells with specificity to determinants of HBV SI and HBV S2 antigens. F. Example 6: Breaking Tolerance to DHBV and DHBV Antigens Using Chimeric Antigen Fusion Protein. DHBV has served as a powerful animal model in the development of antiviral 25 therapy for HBV. Pekin ducks, congenitally infected with DHBV have been used to study the mechanism of replication of the viras and for the screening of antiviral compounds. Two kinds of duck models were used in the present invention. First is the congenitally DHBV-infected ducks. This resembles the vertical transmission of the HBV infection in man. The second model is the persistent infection model where newly hatched ducklings 111 are infected with DHBV and these cany the infection. This second model resembles the horizontal transmission of the HBV infection in man. 1. Congenitally DHBV-infected ducks Congenitally DHBV-infected ducks, at four weeks of age, were divided into two . groups. 5 A sample of blood (1.0 mL) was collected for reference of pre-immunization antibody levels and blood samples were collected every week before the vaccinations. The experimental group received DHBV Core-TBD chimeric antigen fusion protein 19.95 mg/dose injected subcutaneously every week on the same day until week 5. During week 6, the dose was doubled and injected once every four weeks until vaccinations were 10 discontinued at week 26. The placebo group received the equivalent volume of the buffer (20mM Sodium Phosphate pH 8.0, 300mM NaCI). A 96-well plate was coated with antigens, DHBV Core, TBD or DHBV Core TBD, at a concentration of 1.0 [g/mL overnight at 4'C. The plate was washed with phosphate buffered saline (PBS) containing 2% BSA. Diluted serum from the respective 15 animal was added to each of the wells at various dilutions (1 :10- 1 :500) and incubated at 37'C for lhr. The plate was washed with PBS containing 0.05% Tween 20 (wash buffer). Goat anti-duck IgG-HRP (1:5000) dilution was added to the wells and incubated at 37'C for lhr. The plate was washed with wash buffer and color was developed using 2-2' azino-di-(3-ethylbenzylthiazoline-6-sulfonate) (KPL, Guildford, UK). The optical density 20 of resulting color in the samples was measured using an ELISA plate reader (Molecular Devices, USA). Antibody titers were calculated relative to the pre-immune serum from the same animal. Anti-core antibody levels in the serum from congenitally DHBV-infected ducks in the control and experimental groups of ducks, at weeks 0, 3 and 6, are shown in Table 6. 25 Although the ducks have a chronic DHBV infection, the antibody levels are low, due to the chronic nature of the infection and the immune system not recognizing the antigen as a foreign molecule. On immunization with DHBV Core- TBD chimeric antigen, the host immune system recognized the viral antigen and mounted a humoral response against the 112 core antigen that is already present in the host, thus breaking the host tolerance to a viral antiugen TABLE 6: Humoral response to DHBV Core-TBD ArtibodybindinOo) anti-DHBV anti-TBD Vaccinated Wesk 0005+0,005 D.005A 0.003 ducks Weok 3 0131 + 0.029 0.092+0.059 Week 6 0.166 + 0.021 0.147 * 0.038 Conbrolatrup WeekD 0.062+0,016 0.003*0.002 Week 0.074+0,0l5 |0.0ISa005 Week 0.0874 0.012 0.035 0.017 5 Similarly, the duck immune system recognized the TBD component ofihe 10 chimeric antigen as a foreign antigen and generated an immune response against this part of the fusion protein as well. Plates were coated with TBD and the serum from individual ducks evaluated for the antibody levels by ELISA The results from this study are presented in Table 6, 10 2. Post-hatch DHBV-infected ducks 15 Normal ducklings were infected with DHiV-containing duck serum a day after the ducklings were hatched. This is standard practice in the field of DHBV research. The presence of persistent viremia was verified using established techniques at week four before the start of the immunizations. DHBV-infected ducks were divided into two is groups. A sample of blood (1.0 mL) was collected from each duck 20 for reference of pre-immunization antibody levels and blood sanpies were collected every week before the vaccinations. The experimental group received DHBV Core- TBD chimeric antigen fusion protein 19.95 mg/dose injected subcutaneously every week on the same day until week S. During week 6, the dose was doubled and injected once every four weeks until 20 vaccinations were discontinued at week 30. 25 Blood samples were collected from the placebo group, which received the equivalent volume ofthe buffer (20mM Sodium Phosphate pH 8.0, 300mM NaCI). 113 Antibody levels insera collected from ducks at weeks 0, 3 and 6 are presented. Anti-core antibody levels in the serum from post-hatch DHBV-infected ducks in the confrol and experimental groups of ducks are shown in Table 7. Since DHBV has established a persistent section, the antibody levels are low, as the immune system does 5 not recognize the viral antigen as a foreign molecule. On immunization with DHBV Core-MD chimeric antigen, the host immune system recognized the viral antigen and mounted a humoeral response against the core antigen that is already present in the host, thus breaking the host tolerance to a viral antigen. The antibody levels against TBD also increased (Table 7). Therefore there is a multivalent (or multi-epitopic) immune response 10 against different parts of the same chimeric antigen. TABLE 7: Humoral response to DHBV Core-TBD Antibodybodeigto: I DHBV Core TlD Verointedducks Week 0 0.066 t 0.01] 0003 0.002 Week 0.145 0.014 0072+ 0.043 Week 6 0.170 + 0009 0163 + 0.052 coaol Group Week 0 0.083 0.016 0.0 r 0.010 Week r02 0.042 0.011 10.007 Week 6 |0.138 0.041 .026 0.013 G. Example 7: Chemically Cross-Liked HBV sAg-Fe (Murine) Solutions of 100 pg sAg (US Biologicals; Cat# H 1910-27) and 100 Og M-ouse is Polyclonal IgG Fe fragment (Harlan Sern-Lab Ltd., Cat# PP-19-01) were dialyzed against 100 mM HEPES pH 8.7 overnight at 4"C. The protein solutions were mixed together. dimethyI suberimidate (DMS: Pierce Cat # 20700) was added inmmediately to a final concentration of 10 mM, and the mixtue was incubated at rom temperature for I hi. The reaction was stopped by the addition off. I M Tris HCI pH 78. The reaction mixture 20 was loaded on a Sephadex G 75 column (0.7 x 12 cm), and fractions were eluted using phosphate buffered saline. 0.5 mi fractions were collected and the fractions containing sAg/Te at a molar ratio of I : I , as estimated by ELISA using the respective antibodies were pooled. 114 The pooled frctions were used for antigen presentation assays. (Berlyn, et ab, Clin. lnnnunoL 101: 276-283. (2001)) Imtrduce dendritic cells were cultured for fotur days with GM-CSF/IL4, incubated with the sAg-PF conjugate and maured in the presence of TNFa and interferon-a. Autologous CD T cells were added to the mature dendritic cells. Following three rounds of exposure to the mature dendritic cells, T cell stimulation was quantitated by measuring the production of intracellular interferon, using flow cytometry. The levels of intracellular interferon-y produced in T cells in the presence of conjugate were substantially higher than in the presence of the sAg or the Fc fragnt alone (Table 8). 10 TABLE 8: T cell response to lbsAg-Fc DMS conjugate N Eoatige: __ne i . iMue (2.5 g/mL) 4A6i HLOsAg(2.5 pg/mL) 0.04 |H~~s AW MconjuagastaVpmL)07 H. Exeample 8: Anctigen Presentation Assays Antigen presentation assays were performed using human PBMC-derived dendritic cells according to established protocols (Berlyn, et ab. supra (2001)). A protocol 15 sumnaty fir the T cell stimulation assay is presented in schematic form. Mature Mature Mature loaded DCs loaded D7s boaded DCs (DC-1) (DC-2) (DC-3) Day 0 Day 7 Day 14 20 T cells Day 17 Brefeldin A (I 8ub)Brefeldin A (I rs) Secreted &amp; &amp; interferon-y 115 Intracellular cytokine Intracellular cytokine assay ' staining assay staining assay 1. Preparation of Mature, Loaded Dendritic Cells Monocytes were generated from leukapheresis samples from healthy donors and 5 were depleted of lymphocytes and granulocytes by incubation with anti-CD2, CD7, CD 16, CD 19, and CD56 antibodies. This was followed by incubation with magnetic bead conjugated anti-mouse IgG and separation on a magnet (Dynal). Negatively selected cells were greater than 95% pure monocytes as characterized by flow cytometry using a broad CD marker panel (CD14+, CD lc, CD 19", CD3", CD4", CD64+, CD32+, CD86+, CD 10 16). Next, monocytes were incubated with IL-4 and GM-CSF (R&amp;D Systems) for 4 days in AIM V plus 2.5% matched human seram to generate immature dendritic cells. Again, an aliquot of the cells was stained with the broad CD marker panel to ensure purity and identity of the cells. The cells then were loaded with HBV S1/S2-TBD (5.0 [tg/ml), HBV S1/S2 (2.5 [tg/ml), or TBD (2.5 [g/ml) for 2-4 hours at 37 0 C, and matured with 15 interferon-a and TNF-a for 3 days. Dendritic cells were checked again using flow cytometry for an anay of CD markers to ensure that cells had undergone proper maturation. The resulting mature, loaded dendritic cells were used for the T cell stimulation assay. 2. T cell Stimulation Assay: Cytokine Analysis 20 T cells were generated from the same sample of PMBCs as the dendritic cells by means of negative selection using a magnetic T cell isolation kit (Dynal) according to the manufacturer's directions. Mature, loaded dendritic cells (DC-1) were washed thoroughly and added to the T cells (Day 0). The T cells and dendritic cells were incubated for 7 days. On Day 7, the T cells were re-stimulated with mature, loaded dendritic cells (DC 25 2). An aliquot of the cells was taken 2 hours later. The aliquot of cells was incubated with Brefeldin A (GolgiPlug</div>
    <div class="description-paragraph style-scope patent-text">TM</div>
    <div class="description-paragraph style-scope patent-text">, R&amp;D Systems) for 18 hours and then assayed for intracellular cytokine staining as described below. 116 The remaining cells were incubated for another 7 days. On Day 14, the remaining cells were stimulated with a third batch of mature, loaded dendritic cells (DC-3). An aliquot of the cells was taken 2 hours later. The aliquot of cells was incubated with Brefeldin A (GolgiPlug</div>
    <div class="description-paragraph style-scope patent-text">TM</div>
    <div class="description-paragraph style-scope patent-text">, R&amp;D Systems) for 18 hours and then assayed for intracellular 5 cytokine staining as described below. For intracellular cytokine staining, cells were stained with anti-CD3-FITC and anti-CD8-Cy-Chrome for 30 minutes, washed, fixed, permeabilized, and then stained with anti-interferon-y-PE for 30 minutes on ice. The cells were washed and analyzed by flow cytometry (FACScan, BD Biosciences). The results are shown in Table 9. 10 TABLE 9: CD3+/IFN-y+ T cells HBV Sl/S2-TBD 6.2 1 4.6 HB3V S I/S2 L9C i 1.7 TBD L6 ± 0.9 Day 21 No antigen 0.58 0.21 After removal of the aliquot at Day 14, the remaining T cells were incubated for an additional three days and the supernatant then was used for measuring the level of secreted interferon-y by ELISA (Opt E1A ELISA kit, BD Biosciences). T cell stimulation 15 was evaluated by measuring intracellular and secreted interferon-y levels. The results are presented in Table 10. The chimeric antigen S1/S2-TBD induced the production of higher interferon-y levels compared to either the immune response domain or the TBD domain of the molecule when tested alone, at equivalent concentrations. It should be pointed out that 5 tg dose of S l/S2-TBD contains roughly 2.5 pg each of the components. 20 TABLE 10: Intracellular and Secreted Interferon-y Levels FNipositiveT ces Secrcted IFN-7 (pg/ml) }B S/S2-TBD3 60 HIV 1/S2 2.1 18.9 -79 iD -___ 2.5 -- 11'.9 No antgen 0.77 4.4 T cellsalone 0.21 117 Various concentrations of S 1/S2-TBD were tested for the T cell response. The effect of S 1/S2-TBD was greater than the tetanus toxoid treatment at similar concentrations. At concentrations lower than 5 tg/mL, the chimeric antigen elicited a concentration dependent increase in the production and secretion of interferon-y. Interferon-y 5 production and secretion by CD3 T cells increased in a concentration dependent manner following S 1/S2-TBD antigen presentation by dendritic cells, as shown in Table 11. The positive response at low concentrations would be beneficial with respect to the dose necessary for vaccination and the cost of manufacturing of a vaccine. TABLE 11: Concentration Dependence of Response to Chimeric Antigen % IFN-,positive T cells Searcted IFN-y (pg/mi) BV S1/S2-T1D (L25 pg/mi) 15 I8 H4BV S1/S2-TBD (2.5 pg/ml) 4.3 40 10 lYBV S1/52-TBD (5 pg/ml) 3.5 60 43V BD (10 p/) 20 Tetanus toxoid _ _3.3 33 No antigen 0.77 4.4 T cells alonc 0.21 1.6 I. Example 9: Binding and Uptake of Chimeric Antigens 1. Preparation of Mature, Loaded Dendritic Cells Peripheral blood mononuclear cells (PBMC) were obtained from 15 Ficoll/listopaque (Sigma) treatment of a leukapheresis cell preparation (Berlyn, et ab, supra (2001)). Monocytes were separated from the PBMC population by negative selection using a monocyte isolation kit (Dynal) following the manufacturer's directions. The monocytes were greater than 95% pure as assessed by antibody analysis and flow cytometry (CD14+, CDl lc, CD 19", CD3", CD4, CD64+, CD32+, CD86+, CD1 6"). 20 Monocytes were washed twice with AIM-V (Invitrogen) media containing L-glutamine, streptomycin sulfate (50 ptg/mL) and gentamicin sulfate (10 ptg/mL) with 1% donor matched sera (isolated as described in Berlyn, et ab, supra (2001)). Next, the monocytes were cultured in AIM-V media containing 2.5% donor matched sera and the cytokines GM-CSF and IL-4 to differentiate the cells toward the dendritic cell (DC) lineage. The 118 cells were incubated in 12-well tissue culture plates at 370 C under a 7% CO 2 atmosphere. The monocyte-derived dendritic cells were harvested on days 1 through 4. The cells were subsequently washed once with AIM-V media with 0.1% BSA (Sigma), and 5 twice with Dulbecco's phosphate buffered saline (Invitrogen) with 0.1% (w/v) BSA (PBSB). The monocyte-derived dendritic cells were used in 4' C labeling or binding assays or in 37' C binding uptake assays. 2. Binding of Chimeric Antigens to Maturing Dendritic Cells The extent of binding of S1/S2-TBD relative to murine IgGl and IgG2a to 10 maturing dendritic cells was compared. Dendritic cells were isolated at various days of ex vivo culture (from day 0 to day 4) and treated with S1/S2-TBD (10 [g/mL) or with murine IgGl (2C12, the parent mAb from which TBD was produced) or IgG2a (G155 178, 90 [g/mL) for 1 hour at 4'C. The cells were treated with F(ab') 2 goat anti-mouse Alexa-488 (10 [g/mL) in PBSB for 20 minutes. The cells were washed twice with PBSB 15 and resuspended in PBSB with 2% paraformaldehyde (PF) and acquired by a Becton Dickinson (BD) FACScan fitted with CellQuest acquisition and analysis software (BD). A gate was made on the viable cell population as determined by the FSC and SSC scatter profile and &gt;10,000 events were acquired. To determine the percentage of positive cells, a gate was set based on negative control treated cells (isotype control labeled or cells 20 labeled with F(ab') 2 goat anti-mouse Alexa-488 alone). The percent of specific positive cells was calculated as: %positive cells test sample -% positive cells control 100-% positive cells of control The relative mean fluorescent intensity (MFI) was determined as the MFI of the test 25 sample minus the MFI of the control sample. The binding of Sl/S2-TBD relative to IgGl and IgG2a on DC after I to 4 days of culture is shown in Table 12. 119 TABLE 12: Binding of Chimeric Antigen or Antibody to Maturing Dendritic Cells %4S cficposiive dendritieccells liNW Sl/S2-TBD 9. 9SDy 432 2.2 9.5 29.3 49.1 I na2a 28.0 | 7.4 | 14.3 | 3.5 SI/S2-TBD binding was clearly much greater than the binding ofeither IgGI or IgC,2a 5 with more Sl/S2-TBD binding evident on day I than on day 4. These experiments demonstrated that SI/S2-TBD was bound with high efficiency to the maturing dendritic cells. 3. Uptake of Chimeric Antigens to Maturing Dendritic Cells To determine the extent of uptake of chimeric antigens (e.g. HBV SI/S2-TBD) 10 compared with IgGI and IgG 2 a, cells were incubated with various concentrations of the antigen ghl (2C12, the parent nAb from which TBD was produced) or lgG2a (Gl55 178) for I hour at 37" C in AIM V media with 0. I% BSA Cells were washed twice in PBSB and fixed with PBS with 2% PF overnight at 4"C. Subequently, the cells were washed twice in PBSB and permeabilized with PBS containing 0,1% (w/v) saponin is (Sigma) for 40 minutes at 20"C. The cells were washed twice with PBSB and incubated with F(ab') goat anti mouse Alexa-488 (10 pg/mL) in PBSB with 0.1% (w/v) saponin for 20 minutes at 4C. After washing twice in PBSB, the cells were resuspended in PBSB. A variant of this assay involved treating the cells as above with chimeric antigen, Ighl, or [gG2a for 10 20 minutes followed by the addition of F(abh goat anti-mouse Alexa-488 (10 pg/mL) for 50 minutes. Subsequently the cells were washed and resuspended in PBS with 2% PF. Cells were acquired by a Becton Dickinson (Bl) FACScan fitted with Celiquest acquisition and analysis software (BD). A gate was made on the viable cell population as determined by the FSC and SSC scatter profile and &gt;10,000 events were acquired. To 25 determine the percentage of positive cells, a gate was set based on negative control 120 treated cells (isotype control labeled or cells labeled with F(abh goat anti-truse Alexa 488 alone). The percent of specific positive cells was calculated as: - % positive cells test sample -% f positive cells control X sQ.0 100-% positive cells of control 5 The relative mean fluorescent intensity (MFI) was determined as the MFI of the test sample minus the MFI of the control sample. The uptake of Sl /S2-TBD in comparison to murine IgGI and ligG2a was estimated as a function of concentration on day 4 of dendritic cells maturation. The uptake was quantified at 37 C for I hour and the results are shown in FG. 4. There was a linear 1o increase in the uptake of Sl /S2-TBD with concentration. IgGl was taken up at a much lower level and there was very little uptake of IgG2a. - Therefore, the chimeric antigen SI /S2-TBD is taken up by the dendritic cells more efficiently than immunoglobuhns . .L Example 10: Expression of Fe-y Receptors and CD206 on Maturing DC There are several receptors on the antigen presenting cells that bind and take up antigens. is The abundance of these receptors on maturing dendritic cells was evaluated using fluorescent labeled receptor-specific antibodies. FACS analysis was used to estimate percentage of specific receptor positive cells in the total population of dendritic cells. The degree ofreceptor expression was assessed by determination of the relative mean fluorescent intensity and as a function ofrelative fluorescent intensity (Table 13). 20 TABLE 13: Expression of Antigen Binding Receptors on Maturing Bendritin Cells % specific posive cells |Reltive MF to t y I D y2|TJy3|pay4|DyOl Day| Dy 2 Dys ttay4 CD16 20.8 263 6.2 |0. 5.9 |4.3 |0. 2. 0.0 L6 CD32 |9.3 97.4 |78.9 48 37.8 163.4 1 70.5 18.0 14.0 | CD64 84.0 71.9 18.2 9.6 1 53 | 2.0 | 12.8 3. i 2.7 | .0 45S |-2.5 953 99.1 99.3 |.| 373.1 1180.J [3173 680A 121 The expression of CD64 (Fcy receptor I) decreased with time in culture and at day 4 was almost negligible. In contrast, CD32 (Fcy receptor II), and to a lesser extent CD 16(Fcy receptor III), continued to be expressed after 4 days of DC culture. On day 0 of culture, there was essentially no CD206 (mannose macrophage receptor) expression. But 5 expression was induced upon culture with IL-4 and GM-CSF, and by day 4 CD206 was expressed at very high levels. Thus at day 4, when antigen was loaded in the antigen presentation assays, the dendritic cells possessed at least two potential receptors for the binding of chimeric antigens: CD32 and CD206. In addition, they had the full complement of the co-stimulatory molecules (data not shown). The expression of HLA 10 DR (Class II) and HLA-ABC (Class I) also increased with time in culture. Co-stimulatory molecules CD86 (B7.2) and CD80 (B7. 1) were expressed throughout the period of the assay. These results indicate that the monocyte-derived dendritic cells were differentiating towards mature dendritic cells and were capable of antigen processing and presentation to T cells. 15 K. Example 11: Correlation of CD32/CD206 Expression and S1/S2-TBD Binding to Maturing DCs There is a direct conelation between the expression of CD32/CD206 receptors and S1/S2 TBD binding to maturing dendritic cells. Since it was known that murine IgG1 binds to human CD32, it was expected that S1/S2-TBD, which contains the murine Fc component 20 of IgG1, would also bind CD32. Furthermore, S1/S2-TBD by virtue of its high mannose glycosylation, would also be expected to bind to dendritic cells through the CD206 receptor. The dot plots in FIG. 5 show S1/S2-TBD binding (10 [g/mL) and CD32 expression as well as S1/S2-TBD binding and CD206 expression. There was a direct conelation 25 between the extent of S1/S2-TBD binding and the degree of CD32 expression, which was relatively heterogeneous, i.e., there was a broad degree of expression. These results demonstrate that S1/S2-TBD binds to CD32, and that the greater the expression of CD32, the greater was the degree of binding of the chimeric antigen S1/S2-TBD. The dot plot of 122 SI /S2-TBID binding and CD206 expression shows that the vast majority of cells expressing CD206 also bound Sl/S2-TBD A small percentage of the cell population was CD206 negative and was consequently negative for SI/S2-TBD binding. Therefore both CD32 and CD206 receptors conelate with the binding of S I /S2-TBD. 5 L Example 12: Binding and Uptake ofSl/S2-TBD is Primarily Via CD32 with CD206 Involved to a Lesser Extent The uptake of Sl/S2-TBD in comparison to murne IgGI and IgG2a was estimated as a function of concentration on day 4 ofDC maturation. The uptake was quantified at 37 C for I hour in the presence media. mannan (2 mg/mb Sigma.t and/or mouse Fey (2 10 mg/mb Jackson ImmunResearch Laboratories). Mannan is a competitive inhibitor of CD206 binding and therefore ofuptake ofantigens via CD206 on dendritic cells. Foy is a competitive inhibitor ofCD32 binding and therefore CD32-meditated antigen uptake. The results are shown in Table 14. TABLE 14: Inhibition ofChneric Antigen Binding by Fe or Mannan RelativeMTFfI Mannan a s iry M a na &amp;F M edia 0.5 pgmi IBV SI/S2 7.6 0. 0.6 3.0 25 pg/mi HBV 5l/s2- 215 2.0 3.3 226 ThB) 1 6/ml HBVSl/S2-TBD 41-6 57 5.0 49.2 There was a progressive increase in the binding of the chimeric antigen with its concentration. Incubation of the cells with a high concentration of mouse Fcy fragment abolished this binding, whereas miannan, an whibitor of CD206 receptor binding, had only a marginal effect. Therefore, CD32 may be the primary receptor involved in the 20 binding and uptake of the chimeric antigen. M. Example 13Glycosylation ofHBV SI/S2 Antigen Imparts Imununogenicity 123 The insect cell pathway of protein glycosylation is different from that of mammalian cells in that proteins synthesized in insect cells undergo glycosylation that results in high mannose content and a lack of terminal sialic acid residues in the secreted protein (Altaian, et ab, Glycoconjug 16:109-123 (1999)). HBV S1/S2, the antigen component of 5 the chimeric antigen was expressed in both E. coli (no glycosylation) and in High FiveTM insect cells (mannose glycosylation). 1. Effect of glycosylation on binding of antigen These antigens were compared for their binding to dendritic cells, as described in Example 9. Maturing dendritic cells were loaded with 10 ptg/ml of HBV Sl/S2 expressed 10 in insect cells or in E. coli. Glycosylated protein showed better binding by dendritic cells (Table 15). TABLE 15: Effect of Glycosylation on Binding of HBV Sl/S2 % Specific positive cells Relativ MFI [isectcells 69,9 40.3 S .. oli ____12.2 ___ 3.9 2. Effect of glycosylation on eliciting immune response 15 Glycosylation of HBV S1/S2 elicits increased immunogenicity and T Cell responses. HBV S l/S2, expressed in both E. coli and High Five TM insect cells, were compared for T cell responses when presented by dendritic cells. Both intracellular and secreted interferon-y levels were measured, as described in Example 8 (using 2.5 tg/ml HBV S1/S2 protein), and the results are presented in Table 16. 20 Table 16: Effect of Glycosylation on Interferon-y Levels Intracellular IFNy Sccretcd 1FNy (FNy positive T cclls) (p'g/m1) Baculovims HBV SI/S2 2.A 18-9_ E, li HBV S1/S2 0J83 4.3 No antigen 0.77 4.4 T cells alone 0.21 .6 124 HBV S 1/S2 expressed in insect cells generated a higher level of both intracellular and secreted interferon, as compared to the unglycosylated protein expressed in E. coli. Example IX. Materials and Methods Materials 5 The TBD used in the Chimigen3 molecules described in these examples (and, for convenience, referred to in the examples as "TBD") is derived from the Hybridoma 2C12, which produces a murine HBsAg-specific mAb and which was licensed from the Tyrrell laboratory through the University of Alberta, Edmonton, Alberta, Canada. The plasmid pCV-H77C containing the DNA encoding the HCV antigens was obtained from the 10 Tyrrell laboratory at the University of Alberta. The pFastBac-HTa cloning vector, insect cell line Sf9, Cellfectin@ reagent, phosphate buffered saline (PBS), Platinum Pfx DNA polymerase, TRizol reagent, Superscript First-Strand Synthesis reverse transcriptase, X-gal, isopropyl-p-D thiogalactopyranoside (IPTG) and fetal bovine serum (FBS) were purchased from 15 Invitrogen (Carlsbad, CA, USA). Insect cell growth and expression medium ESF 921 was purchased from Expression Systems (Woodland, CA, USA). Restriction enzymes EcoR I, Spe I, Hind III, Rsr II, Ava II and Not I were purchased from New England Biolabs (Ipswich, MA, USA). 20 Viral stocks were titered using the Expression Systems Baculovirus Titering Assay. IgG 2 A-PE (BD Biosciences, San Diego, CA, USA) was diluted 1:10 and used as an isotype control. Baculovirus titer was determined using FACS acquisition and analysis. A Becton Dickinson Biosciences FACSCalibur3 (four-color, dual-laser) acquired cells and CELLQuest Pro3 software (BD Biosciences) was used to analyze the 25 data. A Microsoft Excel spreadsheet was provided by Expression Systems to input data and determine the viral titer based on a standard curve. Purifications were performed with Ni-NTA Superflow3 (Qiagen, Hilden, Germany) and Toyopearl Super Q3 650C (Tosoh Biosciences, Grove City, OH, USA). The 30% acrylamide solution for making sodium dodecyl sulfate polyacrylamide 30 gel electrophoresis (SDS PAGE) gels was purchased from Bio-Rad (Hercules, CA, 125 USA). PageBlue3 stain, 5x loading buffer, PageRuler3 pre-stained protein ladder and 20x reducing agent were purchased from Fermentas (Burlington, ON, Canada). Hybond3 ECL nitrocellulose and the ECL Western Detection kit (GE Healthcare) was used for Western Blotting. 5 Tween 20, hexadecyltrimethylammonium bromide (CTAB), anti-mouse IgG (Fc specific) horseradish peroxidase conjugated antibody, anti-mouse (Fab specific) horseradish peroxidase conjugated secondary antibody, goat-anti-rabbit horseradish peroxidase conjugated secondary antibody and antibiotics kanamycin, ampicillin and gentamicin were purchased from Sigma (St. Louis, MO, USA). 10 The rabbit anti-NS5A, goat anti-NS3, and goat anti-NS4 polyclonal antibodies and mouse anti-NS5A monoclonal antibody were obtained from Abcam (Cambridge, MA, USA). The 6xHis horseradish peroxidase conjugated monoclonal antibody was purchased from Clontech (Palo Alto, CA, USA). Slide-a-lyzer3 cassettes and Micro BCA3 assay kit were purchased from Pierce 15 (Rockford, IL, USA). Pro-Q@ Emerald 300 Glycoprotein Gel and Blot Stain Kit was purchased from Molecular Probes (Carlsbad, CA, USA). Leukapheresis samples from healthy donors were purchased from SeraCare Life Sciences (Oceanside, CA, USA). Dynal Dynabeads3 for T cell negative isolation were 20 purchased from Invitrogen (Carlsbad, CA, USA). AIM V® medium containing L glutamine, streptomycin sulfate (50 pg/mL), and gentamycin sulfate (10 pg/mL) was obtained from Invitrogen. Matched donor sera were obtained from the serum fraction after centrifugation of Ficoll-Hypaque blood preparations. Serum, at 50% in AIM V® medium was heat inactivated, aliquoted, and stored at -20'C. Dulbecco's phosphate 25 buffered saline (PBS) was obtained from Invitrogen. Conjugated monoclonal antibodies (mAbs) with the following specificities were obtained from BD Biosciences (San Diego, CA): CD64-fluorescein isothiocyanate (FITC), CD32-R-phycoerythrin (PE), CD16-PE, CD206-PE-Cy5, CD80-PE, CD86 FITC, CD83-PE, CD40-FITC, CD1 Ic-PE, CD14-FITC, CD19-FITC, CD3-FITC, CD3 30 PE, CD3-allophycocyanin (APC), CD8-PE-Cy5, CD4-APC, CD69-FITC, CD69-APC, HLA-ABC-FITC, HLA-DR-PE, IFN-y-PE, TNF-a-PE, grB-FITC, pfn-FITC and mouse 126 IgGI-biotin. Biotinylated anti-6xHis was obtained from Qiagen (Mississauga, Ontario, Canada). Goat anti-rabbit IgG-biotin antibody was from Jackson ImmunoResearch Laboratories (West Grove, PA). Murine isotype mAbs and SA-PE-Cy5 were obtained from BD Biosciences. Mixed isomer 5-(and-6)-carboxyfluorescein diacetate, 5 succinimidyl ester (5(6)-CFDA, SE; CFSE) was obtained from Invitrogen. Specificity of T cells to antigens was measured with the use of specific PE conjugated tetramers (Beckman Coulter, Mississauga, Ontario, Canada) or pentamers (ProImmune, Springfield, VA). Pentamers used included the HCV NS5A peptide VLSDFKTWL (SEQ ID NO:3X)/HLA-A2 and the HCV NS3 peptide CINGVCWTV 10 (SEQ ID NO:4X)/HLA-A2. Tetramers used included the EBV peptide GLCTLVAML (SEQ ID NO:5X) /HLA-A2, the HCV NS3 peptide KLVALGINAV (SEQ ID NO:6X)/HLA-A2 and a negative control tetramer (multiallelic). The following cytokines were purchased from R&amp;D Systems (Minneapolis, MN): interleukin- 1 (IL-1 I3), interleukin-4 (IL-4), interleukin-6 (IL-6), granulocyte 15 macrophage -colony stimulating factor (GM-CSF), tumor necrosis factor-I (TNF-a), interferon-K (IFN-y), and interferon-I (IFN-a). Cytokines were reconstituted according to the manufacturers' directions, aliquoted, and stored at -70'C. Poly IC was obtained from Sigma. The Wave Bioreactor System23/10EH and Cellbag 1OL/O were purchased from 20 Wave Biotech (Somerset, NJ., USA) Methods Expression Plasmid Construction pFastBacHTa-TBD, the parent plasmid construct 25 The mouse IgGI DNA sequences encoding amino acids of CH1-Hinge-CH2-CH3 region were generated from mRNA isolated from the hybridoma 2C12 that produces a mAb against HBV surface antigen (sAg). Total mRNA was isolated using TRizol reagent and the cDNA of the TBD was generated by RT-PCR using Superscript First Strand Synthesis. The PCR primers contained linker sequences encoding the linker 30 peptide -SRPQGGGS- (SEQ ID NO:1X) at the 5' terminus, a unique Not I site at the 5' end and\ a unique Hind III restriction site at the 3' end. The resulting cDNA contains (5' 127 Not I)-linker sequence-part of CHI (VDKKI; SEQ ID NO: 1X)-CH2-CH3 (3' Hind III). Following digestion with the respective enzymes, the fragment was ligated with pFastBac-HTa expression vector plasmid using the same restriction enzyme sites. The 5' primer used for PCR amplification was (Sense) 5' 5 TGTCATTCTGCGGCCGCAAGGCGGCGGGATCCGTGGACAAGAAAATTGTGCC AGG-3' (SEQ ID NO:7X) and the 3' primer was (antisense) 5' ACGAATCAAGCTTTGCAGCCCAGGAGAGTGGGAGAG-3' (SEQ ID NO:8X), which contained the Not I and Hind III sites, respectively. The amplified DNA was digested with Not I and Hind III, the fragment purified by agarose gels and ligated with 10 pFastBac-HTa expression vector plasmid digested with the same restriction enzymes to produce the expression plasmid pFastBacHTa-TBD. This product was used for the expression of the fusion protein 6xHis tag-rTEV protease cleavage site-TBD. The DNA sequence and the accuracy of the open reading frame (ORF) were verified by standard sequencing methods. The nucleotide sequence (SEQ ID NO:9X) of the ORF in 15 pFastBacHTa-TBD and the amino acid sequence (SEQ ID NO:10X) encoded by the ORF are shown in Fig. 2X. Construction of pFastBacHTa-gp64 For secretion, the signal sequence from the Autographa cahfornica nuclear 20 polyhedrosis virus (AcNPV) gp64 protein was cloned into pFastBac-HTa. Two oligonucleotides were synthesized and annealed together. The oligonucleotide sequences are 5' GCATGGTCCATGGTAAGCGCTATTGTTTTATATGTGCTTTTGGCGGCGGCGGC GCATTCTGCCTTTGCGGATCTGCAGGTACGGTCCGATGC-3' (SEQ ID NO:1IX) 25 and 5' GCATCGGACCGTACCTGCAGATCCGCAAAGGCAGAATGCGCCGCCGCCGCCA AAAGCACATATAAAACAATAGCGCTTACCATGGACCATGC-3' (SEQ ID NO: 12X). The oligonucleotides contain a 5' Ava II site and 3' Rsr II site. After digestion with Ava II and Rsr II, the fragment was cloned into the Rsr II digested pFastBac-HTa, 30 which places the gp64 signal sequence immediately upstream of the 6xHis tag , to generate pFastBacHTa-gp64. 128 Construction of pFastBacHTa HCV NS5A Chimigen TM vaccine fusion protein expression vector plasmid DNA encoding the HCV NS5A fragment was generated from the plasmid pCV 5 H77C template using PCR methodology. The 5' primer used for the PCR was (sense) 5' CCGGAATTCTCCGGTTCCTGGCTAAGG-3' (SEQ ID NO:13X) containing the restriction enzyme EcoR I site. The PCR primer for 3' terminus was (antisense) 5' GGACTAGTCCGCACACGACATCTTCCGT-3' (SEQ ID NO:14X) and contains the restriction enzyme Spe I site. Amplified DNA was digested with the respective enzymes 10 and ligated to pFastBacHTa -TBD to generate the expression plasmid pFastBacHTa HCV NS5A Chimigen TM Vaccine (or pFastBacHTa-NS5A-TBD) . The nucleotide sequence (SEQ ID NO:39X) and the amino acid sequence (SEQ ID NO:40X) encoded by the ORF in pFastBacHTa-NS5A-TBD are presented in Fig. 3X. For NS5A alone, the NS5A fragment was ligated into EcoR I/Spe I digested pFastBac-HTa to generate 15 pFastBacHTa-NS5A. Construction of pFastBacHTa-gp64 NS5A Chimigen TM vaccine expression plasmid for secretion In order to clone NS5A Chimigen TM Vaccine into pFastBacHTa-gp64, the 20 plasmid pFastBacHTa HCV NS5A Chimigen TM Vaccine (described above) was digested with Rsr II and Hind III. and the NS5A Chimigen TM Vaccine fragment was purified by agarose gel electrophoresis. The NS5A Chimigen TM Vaccine fragment was ligated to Rsr II and Hind III digested pFastBacHTa-gp64 plasmid to yield the pFastBacHTa-gp64 HCV NS5A Chimigen TM Vaccine (pFastBacHTa-gp64-NS5A-TBD) expression plasmid 25 (IDAC accession no. 111006-04). The nucleotide sequence (SEQ ID NO:41X) of the ORF in pFastBacHTa-gp64-NS5A-TBD and the amino acid sequence (SEQ ID NO:52X) encoded by the ORF are shown in Fig. 4X. Construction of pPSC12-NS5A-TBD Chimigen TM vaccine expression plasmid for 30 secretion 129 In order to facilitate the secretion of NS5A Chimigen TM Vaccine molecule, cloning into the plasmid pPSC12 (Protein Sciences Corporation) was performed. This plasmid has the signal peptide for the chitinase gene from the baculovirus Autographica cahfornica nuclear polyhedrosis virus (AcNPV). Four PCR primers were required to 5 clone a gene of interest into the transfer vector. The gene of interest was amplified using two unique primers (Primers 1 GTTTCTAACGCGTCGTACTACCATCACCATCAC (SEQ ID NO:15X) and 2 CCGGGGTACCTTACAGCCCAGGAGAGTGGGAGAG (SEQ ID NO: 16X)). Two separate primers were required to amplify a polyhedron upstream region, containing the upstream polyhedron promoter and the signal peptide 10 sequence (Primers 3 CTGGTAGTTCTTCGGAGTGTG (SEQ ID NO:17X) and 4 GGTAGTACGACGCGTTAGAAACGGCGACCAAC (SEQ ID NO:18X)). Finally, two outside primers (Primers 3 and 2, sequences above) were used in the critical overlap extension PCR. NS5A-TBD was amplified from pFastBacHTa-NS5A-TBD by PCR using primer 1 that contains a sequence that would anneal to the 5' end of primer 4 and 15 primer 2 that adds a unique 3' Kpn I site for cloning into the vector. The upstream polyhedron region of pPSC12 was amplified with primer 3 and primer 4 which allowed it to anneal to the 5' end of primer 1 during the overlap extension PCR. This upstream region also contains a unique NgoM IV site which is used for cloning into the vector. The upstream polyhedron promoter, the signal peptide sequence, and the desired gene 20 were seamlessly fused by overlap extension PCR using primers 2 and 3. The full length fused product was digested with NgoM IV and Kpn I and the resulting fragment was ligated into an identically digested pPSC12 to generate pPSC12-NS5A-TBD. The nucleotide sequence (SEQ ID NO:42X) of the ORF in pPSC12-NS5A-TBD and the amino acid sequence (SEQ ID NO:53X) encoded by the ORF are shown in Fig 5X. 25 Construction of pFastBacHTa HCV NS3 Chimigen T M Vaccine Plasmid DNA encoding NS3 was generated by PCR from the plasmid pCV-H77C template from amino acids 1027 to 1652 (nt 3420 to 5294) of the HCV polyprotein using the following primers. The final C-terminal 6 amino acids ofNS3 were not included in the construct because those sequences are the target sequence for the serine protease 30 activity ofNS3. The 5' terminus primer used was 5' 130 CCGGAATTCGCGCCCATCACGGCGTA-3' (SEQ ID NO:19X) containing an Eco RI restriction site and the 3' terminus primer was 5'-CCGGACTAGTCC GGCCGACATGCATGTCATGAT-3' (SEQ ID NO:20X) containing Spe I restriction site. A double digestion with Eco RI and Spe I resulted in a product that was ligated with 5 the plasmid pFastBacHTa-TBD to generate pFastBacHTa NS3-TBD. Mutagenesis of pFastBacHTa HCV NS3 Chimigen T M Vaccine Plasmid Vector Internal cleavage of the NS3 protein when expressed in insect cells, presumably mediated by cellular protease(s), has been reported by Shoji et al. [(1999) Virology 254:315-323] to occur at the arginine residue at 1488. Overlap extension (OE) PCR was 10 used to generate a mutation of the amino acid arginine to alanine and thereby avoid such cleavage of the NS3 part of the NS3 Chimigen3 protein. Two NS3 DNA fragments were generated from the parent pFastBacHTa NS3-TBD plasmid. The 5' NS3 fragment was generated with the primer 5'-CCGGAATTCGCGCCCATCACGGCGTA-3' (SEQ ID NO: 19X) containing the Eco RI restriction site and the mutation primer (containing the 15 arginine to alanine mutation) 5'-CTGCCAGTCCTGCCCGCGCGTTGAGTCCTGGAG 3' (SEQ ID NO:21X). The 3' NS3 fragment was generated with the 5' primer 5' GGCAGGACTGGCAGGGGGAAGCCAGGCAT-3' (SEQ ID NO:22X) and the 3' primer 5'-CCGGACTAGTCCGGCCGACATGCATGTCATGAT-3' (SEQ ID NO:20X) containing the Spe I restriction site. OE PCR was done with the 5' and 3' NS3 20 fragments, plus the two outside primers. A double digestion with Eco RI and Spe I resulted in a product that could be ligated with the plasmid pFastBacHTa-TBD to generate pFastBacHTa NS3mut-TBD (IDAC accession no. 111006-05). The nucleotide sequence (SEQ ID NO:44X) of the ORF in pFastBacHTa NS3mut-TBD and the amino acid sequence (SEQ ID NO:55X) encoded by the ORF are presented in Fig. 7X. The 25 predicted molecular weight of the protein is 98.3 KDa. For NS3mut alone, the NS3mut fragment was isolated by digestion with EcoR I and Spe I and cloned into pFastBac-HTa to generate pFastBacHTa-NS3mut Construction of pFastBacHTa-gp64 HCV NS3mut Chimigen T M Vaccine Vector Plasmid The pFastBacHTa HCV NS3mut-TBD plasmid was digested with Rsr II and 30 Hind III restriction enzymes and the NS3mut-TBD fragment was cloned into Rsr II and 131 Hind III digested pFastBacHTa-gp64 to generate pFastBacHTa-gp64-NS3mut-TBD (IDAC accession no. 111006-02). The nucleotide sequence (SEQ ID NO:45X) of the ORF in pFastBacHTa-gp64-NS3mut-TBD and the amino acid sequence (SEQ ID NO:56X) encoded by the ORF are presented in Fig. 8X. The predicted molecular weight 5 of the protein is 101.5 KDa. A clone of the mutated NS3mut-TBD fragment similar to that used to make pFastBacHTa-gp64-NS3mut-TBD (but lacking one spontaneous mutation and having another) was also ligated into pFastBacHTa-gp64 to generate a second clone of pFastBacHTa-gp64-NS3mut-TBD. The nucleotide sequence (SEQ ID NO:43X) of the ORF in the second clone of pFastBacHTa-gp64-NS3-TBD and the amino 10 acid sequence (SEQ ID NO:54X) encoded by the ORF are presented in Fig. 6X. Construction of pFastBacHTa HCV Multi-antigen Chimigen TM fusion protein expression vector plasmid To make the HCV multi-antigen vector plasmid the NS4B-NS5A sequences were 15 first cloned. The DNA sequence encoding NS4B to NS5A was generated by PCR from the plasmid pCV-H77C using the primers 5' GCGCACTAGTGTCTCAGCACTTACCGTACATC-3' (SEQ ID NO: 23X) for the 5' terminus and 5'-CGGCGCGGCCGCCCGCAGCACACGACATCTTCCG-3' (SEQ ID NO:24X) for the 3' terminus. PCR with these primers resulted in a product with unique 20 restriction enzyme sites of a Spe I site at the 5' end and a Not I site at the 3' end. The PCR product was digested with Spe I and Not I and ligated into a Spe I and Not I digested pFastBacHTa-gp64 to generate pFastBacHTa-gp64 NS4B-NS5A. Next, the TBD portion was added to construct. The plasmids pFastBacHTa-TBD and pFastBacHTa-gp64 NS4B-NS5A were digested with Spe I and Hind III. The Spe I/Hind 25 III digested TBD fragment was isolated and ligated to the digested pFastBacHTa-gp64 NS4B-NS5A to generate pFastBacHTa-gp64 NS4B-NS5A-TBD. Mutagenesis ofNS3 Active Site Serine Residue In NS3 the active site serine (ser 165) was mutated to alanine to abrogate the protease activity. Two NS3 fragments were created using four different primers, two 30 nested and two complimentary to the 5' and 3' ends, by OE PCR with pFastBacHTa 132 gp64 NS3mut-TBD as template. The 5' NS3 fragment was generated using the 5' terminus primer (sense) 5' CCGGAATTCGCGCCCATCACGGCGTA-3' (SEQ ID NO: 19X), which contains the restriction enzyme Eco RI site and the mutation primer (containing the ser to ala mutation) (antisense) 5' 5 CAACAGCGGACCCCCCGCGGAGCCTTTCAAGTAG-3' (SEQ ID NO:25X). The 3' NS3 fragment was generated using the 5' terminus primer (sense) 5'GTCCGCTGTTGTGCCCCGCGGGACACG-3' (SEQ ID NO:26X) and the 3' terminus primer (antisense) 5' -CCGGACTAGTCCGGCCGACATGCATGTCA-3' (SEQ ID NO:27X), which contains the restriction enzyme Spe I site. The full length NS3 10 (ser' 16 -- ala) was generated by OE-PCR from the 5' and 3' fragments and the two outside primers. The resulting product with mutations at Arg 1488 to Ala and Ser 1165 to Ala is called NS3mutS. This fragment was cloned into pFastBacHTa-gp64 to generate pFastBacHTa-gp64 NS3mutS-TBD (IDAC accession no. 111006-001). Construction of pFastBacHTa-gp64 HCV NS3-NS4B-NS5A Multi-antigen Fusion 15 Protein Vector Plasmid To make a construct that can be used for expression the fusion protein HCV NS3mutS-NS4B-NS5, the NS3mutS OE-PCR product was digested with the restriction enzymes Eco RI and Spe I. The digested NS3mutS was ligated into the Eco RI and Spe I digested plasmid pFastBacHTa-gp64 NS4B-NS5A-TBD to make pFastBacHTa-gp64 20 NS3-NS4B-NS5A-TBD (IDAC accession no. 111006-06), which is the pFastBacHTa gp64 HCV Multi-antigen plasmid. The nucleotide sequence (SEQ ID NO:46X) of the ORF in pFastBacHTa-gp64 NS3-NS4B-NS5A-TBD and the amino acid sequence (SEQ ID NO:57X) encoded by the ORF are shown in Fig. 9X. Construction of pFastBacHTa-gp64 HCV NS3-NS5A Multi-antigen Fusion Protein 25 Vector Plasmid The DNA for HCV NS5A and TBD was generated by PCR from the template pFastBacHTa-gp64 HCV NS3-NS4B-NS5A-TBD. Chimigen T M Vaccine fusion protein expression vector plasmid. The 5' primer for the PCR was (sense) 5' GAGGGACTAGTGTCCGGTTCCTGGCTAAGGGAC-3' (SEQ ID NO:28X) 133 containing the recognition site for the restriction enzyme Spe I. The PCR primer for the 3' terminus was (antisense) 5' CCGGTCTAGATTATGATCCTCTAGTACTTCTCGAC-3' (SEQ ID NO:29X). The PCR product (NS5A-TBD) was gel purified and subsequently digested with Spe I and 5 Hind III restriction enzymes. The plasmid pFasTBacHTa-gp64 NS3-NS4B-NS5A-TBD was digested with the restriction enzymes Spe I and Hind III, liberating a fragment consisting of the sequences encoding HCV NS4B-NS5A and the TBD. The resulting pFastBacHTa-gp64 HCV NS3 vector backbone was gel purified and ligated to the NS5A TBD fragment to generate the expression plasmid pFastBacHTa-gp64 HCV NS3-NS5A 10 Chimigen T M Vaccine (pFastBacHTa-gp64-NS3-NS5A-TBD) (IDAC accession no. 111006-03). The nucleotide sequence (SEQ ID NO:47X) of the ORF in FastBacHTa gp64-NS3-NS5A-TBD and the amino acid sequence (SEQ ID NO:58X) encoded by the ORF are shown in Fig. lOX. 15 Construction of pFastBacHTa HCV core (1-177)-TBD fusion protein plasmid and pFastBacHTa HCV core(1-177) The HCV core DNA sequences encoding amino acids 1-177 (nt 342-872) of the HCV polyprotein were amplified by PCR from pCV-H77C with 5' primer CGGAATTCATGAGCACGAATCCTAAAC (SEQ ID NO:30X) and 3' primer 20 GGACTAGTCCGAAGATAGAGAAAGAGC (SEQ ID NO:31X). The primers used added unique 5' EcoR I and 3' Spe I sites. The PCR product was digested with EcoR I and Spe I and ligated into pFastBacHTa-TBD and pFastBac-HTa to generate the Chimigen TM vaccine construct pFastBacHTa HCV core (1-177)-TBD and pFastBacHTa HCV core(1-177), respectively. The nucleotide sequence (SEQ ID NO:48X) of the ORF 25 in pFastBacHTa HCV core (1-177)-TBD and the amino acid sequence (SEQ ID NO:59X) encoded by the ORF are shown in Fig. I1X. The HCV core (1-177) was cloned into pFastBacHTa-gp64 and pPSC12, in order to produce the protein in a secreted form. For cloning into pFastBacHTa-gp64, the HCV core (1-177)-TBD fragment was isolated from pFastBacHTa HCV Core(1-177)-TBD by 30 Rsr II and Hind III digestion and cloned identically digested pFastBacHTa-gp64 to generate pFastBacHTa-gp64 HCV core (1-177)-TBD. 134 For cloning into pPSC12, a similar scheme was used, as described for NS5A TBD, except that primer 2 encodes a unique 3' Bgl II site (AGTAAGATCTTTACAGCCCAGGAGAGTGGGAGAG; SEQ ID NO:32X). The resulting construct is pPSC12-HCV core (1-177)-TBD. 5 Construction of pFastBacHTa HCV E l-TBD fusion protein plasmid and pFastBacHTa El The DNA sequence encoding amino acids 192 to 369 (914-1452) of the HCV polyprotein were amplified from pCV-H77C with 5' primer 10 CCGGAATTCTACCAAGTGCGCAATTCCT (SEQ ID NO:33X) and 3' primer GCGCACTAGTCCCTTCGCCCAGTTCCCCACC (SEQ ID NO:34X) that add a unique 5' EcoR I site and a unique 3' Spe I site. The entire El open reading frame ends at amino acid 383 but the area between amino acids 370 and 383 is the signal sequence for E2 and was therefore not amplified. The PCR product was digested with EcoR I and Spe I and 15 ligated into identically digested pFastBacHTa-TBD to generate the HCV El Chimigen TM construct pFastBacHTa-E l-TBD. To express E l alone, the digested PCR product was cloned into EcoR I and Spe I digested pFastBac-HTa to generate pFastBacHTa-E1. The nucleotide sequence (SEQ ID NO:49X) of the ORF in pFastBacHTa-El-TBD and the amino acid sequence (SEQ ID NO:60X) encoded by the ORF are shown in Fig. 12X. 20 Construction of pFastBacHTa E2-TBD fusion protein plasmid and pFastBacHTa-E2 The E2 sequences from amino acid 384 to 718 (nt 1494-2495 of the HCV polyprotein) were amplified by PCR from pCV-H77C with 5' primer GCGCACTAGTCACCCACGTCACCGGGGGAAATG (SEQ ID NO:35X) and 3' 25 primer GCGCGCGGCCGCCCGTACTCCCACTTAATGGC (SEQ ID NO:36X) that add a unique 5' Spe I site and a unique 3' Not I site. The amino acids 719 to 746 are the signal sequence for p 7 so was not included in construct. The PCR product was digested with Spe I and Not I and ligated to an identically digested pFastBacHTa-TBD to generate the HCV E2 ChimigenTM construct pFastBacHTa E2-TBD. The digested E2 was also 30 cloned into pFastBac-HTa to generate pFastBacHTa-E2 for expression of E2 protein alone. The nucleotide sequence (SEQ ID NO:50X) of the ORF in pFastBacHTa E2-TBD 135 and the amino acid sequence (SEQ ID NO:61X) encoded by the ORF are shown in Fig. 13X. Construction of pFastBacHTa-E l-E2-TBD fusion protein plasmid and pFastBacHTa 5 El-E2 A fusion of the El and E2 proteins was generated by subcloning the El sequence in pFastBacHTa-El into pFastBacHTa-E2. The pFastBacHTa-El plasmid was digested with Eco RI and Spe I and the fragment was cloned into Eco RI and Spe I digested pFastBacHTa-E2 to generate pFastBacHTa-El-E2. To make the El-E2 Chimigen TM 10 construct, pFastBacHTa-E 1 -E2 was digested with Eco RI and Not I and cloned into identically digested pFastBacHTa-TBD to generate pFastBacHTa-El-E2-TBD. The nucleotide sequence (SEQ ID NO:5 1X) of the ORF in pFastBacHTa-El-E2-TBD and the amino acid sequence (SEQ ID NO:62X) encoded by the ORF are shown in Fig. 14X. 15 Production of recombinant baculoviruses in the Bac-to-Bac@ expression system Transformation of E. coli Ligated plasmids were used to transform E. coli DH5ax and the plasmids were isolated by standard protocols. Sequence and open reading frames were verified by sequencing and used for the production of recombinant baculoviruses. 20 Transposition The generation of recombinants is based on the Bac-To-Bac@ cloning system (Invitrogen) that uses site-specific transposition with the bacterial transposon Tn7. This is accomplished in E. coli strain DHOBac. The DHOBac cells contain the bacmid 25 pMON14272, which confers kanamycin resistance, and a helper plasmid (pMON7124) that encodes the transposase and confers resistance to tetracycline. The gene of interest is cloned into the pFastBac plasmid that has mini-Tn7 elements flanking the cloning sites. The plasmid is transformed into the E. coli strain DH1OBac, which has a baculovirus shuttle plasmid (bacmid) containing the attachment 30 site of Tn7 within a LacZax gene. Transposition disrupts the LacZax gene so that only recombinants produce white colonies and thus are easily selected. 136 The advantage of using transposition in E. coli is that single colonies contain only the recombinant. The recombinant bacmids are isolated using standard plasmid isolation protocols and are used for transfection in insect cells to generate baculoviruses that express recombinant proteins. 5 Donor plasmids and pFastBacHTa-gp64 Chimigen TM vaccine vectors were used for the site-specific transposition of the cloned gene into a baculovirus shuttle vector (bacmid). The recombinant pFastBacHTa-gp64 plasmid with the gene of interest was transformed into DH10Bac cells for the transposition to generate recombinant bacmids. A 40 [L aliquot of competent DH10Bac cells was thawed on ice, the pFastBacHTa-gp64 10 based plasmids were added and transformation was performed by electroporation. The transformation mixture was added to ImL of SOC media and incubated for 4 hours at 37'C. The transformed cells were serially diluted with LB to 10^1 and 10-2 and 100 [L of each dilution was plated on LB agar plates supplemented with kanamycin (50 [tg/mL), gentamicin (7 [tg/mL), tetracycline (10 jag/mL), X-gal (200 jg/mL), and IPTG (40 15 jag/mL) and incubated for at least 36 hours at 37'C. Gentamicin resistance was conferred by the pFastBacHTa-gp64 plasmid and X gal and IPTG were used to differentiate between white colonies (recombinant bacmids) from blue colonies (non-recombinant). The white colonies were picked and inoculated into 2 mL of LB supplemented with kanamycin (50 jg/mL), gentamicin (7 jag/mL), and 20 tetracycline (10 jg/nL) and incubated overnight at 37'C with shaking. A sterile loop was used to sample a small amount of the overnight culture and the sample was streaked onto a fresh LB agar plate supplemented with kanamycin (50 jg/mL), gentamicin (7 pag/mL), tetracycline (10 jg/mL), X-gal (100 jg/mL), and IPTG (40 jg/nL) and incubated for at least 36 hours at 37'C to confirm a white phenotype. 25 Recombinant bacmids were isolated by standard protocols [Sambrook et al. (2001) In Molecular Cloning, A Laboratory Manual. Cold Spring Harbor Press], the DNA sample was dissolved in 40 jaL of TE (10 mM Tris-HCl pH 8, 1 mM EDTA) and used for transfections. 137 Transfection: Production of recombinant baculovirus In order to produce recombinant baculoviruses, the relevant bacmid was transfected into Sf9 insect cells. Sf9 cells (9 x 105) were seeded into each well of a 6 well 5 cell culture dish (35 mm wells) in 2 mL of ESF 921 and allowed to attach for at least 1 hour at 27C. Transfections were carried out using Cellfectin@ reagent with the protocols provided by the supplier of the Sf9 cells. Following transfection, the cells were incubated at 27'C for 72 hours. The medium containing baculovirus was collected and stored at 4'C in the dark. 10 The efficiency of the transfection was verified by checking for production of baculoviral DNA. The isolated baculovirus DNA was subjected to PCR to screen for the inserted gene of interest. The expression of the heterologous protein in the cells was verified by SDS-PAGE and Western blots using the 6xHis tag-HRP conjugated monoclonal antibody or anti-mouse IgG (Fc specific) horseradish peroxidase conjugated 15 antibody as the probe. Amplification of the recombinant baculovirus stock Once generation of the baculovirus and the expression of the desired protein were confirmed, the virus concentration was amplified to produce a concentrated stock of the 20 baculovirus that carried the gene of interest. In all the protocols described herein, the standard practice of amplifying the baculovirus at least twice was followed. After the second round of amplification, the concentration of the generated baculovirus was quantified using the baculovirus titering assay (Expression Systems). The most appropriate concentration of the virus to infect Sf9 cells and the optimum time for the 25 production of the desired protein were also established. The protocols for the expression for both monolayer as well as suspension culture of Sf9 cells were developed according to standard procedures. Baculovirus titering assay 30 All viral stocks are titered using the Expression Systems baculovirus titering assay. Viral stocks were diluted serially from 10-1 to 10-4. A 100 [L aliquot of each of 138 the diluted samples was added to wells of a Costar Low Attachment3 96 well plate. Then, 100 [L of Sf9 cells at a concentration of 2 x106 cells/mL was added to each well and the plate incubated for 18 hr at 27'C in an orbital shaker incubator at 200-250 rpm. Following incubation, gp64-PE conjugated antibody was diluted 1:200 and the 5 isotype control (IgG 2 A-PE) was diluted 1:10. The plate was centrifuged for 3 minutes at 1800 rpm. The media was removed by inversion of the plate and 50 L of the gp64-PE conjugated antibody or 50 [L of the isotype control was added to the wells. The plate was then incubated for 20 min at 4'C in the dark. The cells were washed by adding 150 [L cold PBS to each well and centrifuging 10 the plate down as described above. Next, 200 [L of cold PBS was added to each well followed by another spin and finally 200 [L of PBS/0.1% BSA was added to each well to re-suspend the cells and transfer to FACS tubes for analysis. The isotype control was used to set gates on the fluorescence flow cytometer. The viral titer was determined by inserting percentage of the cells population that were positive for expression into the 15 provided Excel spreadsheet and producing a standard curve based on the control virus. Optimization of protein expression ChimigenTM Protein expression was optimized over a range of MOIs and times. Four 50 mL cultures of Sf9 cells in ESF 921 were seeded at 2 x 106 cells/mL and infected 20 with MOIs of 0.5, 1, 5 and 10 and imL of culture was harvested after various time points post infection. Sampled cultures were centrifuged at 12000 x g for 1 min and the supernatant and cells separated. Cells and supernatant were immediately prepared for SDS-PAGE analysis. The cells were resuspended in 500 [L of PBS, and 150 [L of the suspension was added to 40 [L 5x loading buffer and 10 [L 20x reducing agent. Also, 25 150 [L of supernatant was mixed with 40 [L 5x loading buffer and 10 [L 20x reducing agent. Samples were boiled for 5 min and loaded onto a 12% SDS-PAGE gel for Western blot analysis. Protein production was assessed to be best for NS5A ChimigenTM protein at 36 hr and at a MOI of 0.5, for the NS3 Chimigenrm protein at 48 hr at a MOI of 2, and for the multi-antigen ChimigenTm protein at 36 hr at a MOI of 1.5. 30 139 Purification of intracellular ChimigenTM Vaccine Large-scale expression ofNS5A ChimigenTM Protein and preparation of cell lysate Five litres of Sf9 cell culture at a density of ~2 x 106 cell/ml in ESF 921 medium were infected with baculovirus at a MOI of 0.5. Cells were harvested ~ 36 hr after the 5 start of infection when the cell viability was ~ 95%. Longer expression times resulted in increasing loss of cell viability and an intense degradation of the NS5A ChimigenTM Protein. Infected cells were collected by centrifugation and stored at -80 C until use. Frozen pellets from IL infected cell culture were resuspended rapidly by vortexing on ice in 200ml lysis buffer containing a high concentration of Tween 20 (6M 10 GuHCl, 50 mM NaH 2</div>
    <div class="description-paragraph style-scope patent-text">PO</div>
    <div class="description-paragraph style-scope patent-text">4 , 0.5 M NaCl, 1% Tween 20, 1OmM -mercaptoethanol, pH 8.0). A high concentration of Tween 20 was necessary for efficient binding of the NS5A ChimigenTM Protein to the Ni-NTA resin. The resuspended suspension was sonicated on ice 3 x 1 min with 2 min intervals in between each sonication pulse. The sonicated lysate was then stirred for 2 hr at room temperature. The stirred lysate was cleared by 15 centrifugation (~27,000 x g for 15 min. at 1 0 0 C) and the supernatant used for affinity chromatography on Ni-NTA superflow. Expression of NS3 ChimigenTM Protein and preparation of cell lysate Recombinant baculovirus encoding for HCV NS3 Chimigen TM Protein of standardized multiplicity of infection (MOI) was used to infect Sf9 insect cells for protein 20 expression. Sf9 cells were seeded at a density of 6 x 105 cells/mL in 500 mL of ESF 921 media in a 2 L Erlenmeyer flask. The cell culture was incubated at 27.5' C with shaking at 120 rpm until the cell density reached 2-3 x 106 cells/mL. For the HCV NS3 Chimigen TM Protein, cells were infected at a MOI of 2 for 48 hr. A Western Blot analysis on cell lysate was carried out for monitoring the expression of the protein of interest. The 25 cells were harvested by centrifugation at 3,000 rpm (1593 x g, JA10, Beckman Coulter Avanti TM J 25) for 10 min at 4' C and fresh cell pellet was used for the purification of the recombinant protein. Alternatively, cell pellets were re-suspended with the conditioned media, distributed into 50 mL Conical tubes (250 mL cell culture for each tube), spun at 2,200 rpm for 8 min at 4' C in the Beckman GS-6R centrifuge. Cell pellets were snap 30 frozen in liquid nitrogen and stored at -80' C. 140 A frozen cell pellet (equivalent 2 x 500mL of cell culture media) was resuspended on ice in 200 mL of ice cold lysis buffer (6M GuHCl, 150mM NaCl, 20mM Tris-HCl, pH 8.00) by sonication for 1 min, 78W (setting 6.5). The mixture was transferred to 250mL glass beaker and sonicated four more times for 1 min, 78W each time, with 5 min 5 cooling intermissions. The mixture was moved to room temperature and CTAB was added to a final concentration of 1% (w/v). The pH was checked and adjusted to pH 8.00 and the lysate was incubated for 2 hrs. The lysate was centrifuged for 30 min at 15,000rpm (27,000xg) at 10 C using JA 25.50 rotor in a Beckman Avanti J-25 centrifuge and the supernatant was subjected to Ni-NTA affinity chromatography. 10 Large-scale expression of the Multi-Antigen ChimigenTM Protein and preparation of cell lysate Four litres of ESF921 medium was transferred into a Cellbag and was warmed to 27.5 C on a Wave Bioreactor System 2/10 EH. One litre of Sf9 cell culture at 6 x 106 15 cells/mL was added to the Cellbag. The bag was then incubated at 27.5 C with injection of air at 0.3L per minute and was rocked at 130 rpm. When the density of the cells reached to 2 x 106 cells/ml, recombinant baculovirus was inoculated at MOI of 1.5. At 36 hour after infection, the bag was chilled on ice and cells were harvested by centrifugation at 4500 x g for 10 minutes at 4'C. The cell pellet was suspended in ice 20 cold PBS and then transferred into a 50 mL conical tube (pellet from 300 mL culture per tube). The cell pellet was recovered by centrifugation at 2800 x g for 15 minutes at 4'C. The pellet was frozen immediately in liquid nitrogen and was stored in -80'C freezer until use. The frozen Sf9 cell pellet from 250 ml culture was suspended in 20ml IX PBS, 25 1% Tween 20, 5OmM DTT, 5 mM EDTA, pH 8.0 and incubated on ice for 30min. The pH of the lysate was adjusted to pH 12.0 with NaOH and stirred at room temperature for 30min. The pH was lowered to 8.0 with HCl and centrifuged for 30min at 39191 x g. The supernatant was removed and the pellet was suspended in 20ml IX PBS, 1% Tween 20, lOmM DTT, 1 mM EDTA, pH 8.0. The pH was raised to 12.0 and reduced to 8.0, as 30 described above. The supernatants were pooled and dialyzed against 20mM Tris, 0.05% 141 Tween 20, 0.1mM EDTA, 1OmM -mercaptoethanol, pH 8.0 for use in size exlusion and hydrophobic interaction chromatography. For Ni-NTA affininty chromatography, frozen Sf9 cell pellet from 500 ml culture was suspended in 50 mL ice-cold Lysis buffer (6M Guanidine-HCl, 50mM Sodium 5 Carbonate, 20% Ethanol, pH 10). The cell lysate was sonicated five times on ice by Sonicator 3000 (Misonic Inc.) at 80W for 1 minute. Tween 20 was added into the lysate (final concentration 1%) and the lysate was stirred for 2 hours at room temperature. Insoluble particulates in the lysate were removed by centrifugation at 39,191 x g for 30 minutes at 4'C and subjected to Ni-NTA affinity chromatography. 10 Expression of HCV Core ChimigenTM Protein and preparation of cell lysate HCV core ChimigenTM was expressed in two systems. Recombinant viruses were generated with co-transfection of pPSC12-HCV core (1-177)-TBD and a linearized baculovirus genome in Sf9 cells, plaque purified and amplified. After optimization, Sf9 15 cultures were infected at MOI of 5 and harvested after 50hrs of incubation at 27.5'C. Recombinant viruses were also generated using the Bac-to-Bac@ system by transfection of E. coli DH1OBac cells with pFastBacHTa-gp64 HCV core (1-177)-TBD. Recombinant bacmids were isolated and used to transfect Sf9 cells to make recombinant baculoviruses. Sf9 cultures were infected at an MOI of 5 for 49hrs at 27.5 C before 20 harvesting by centrifugation. Lysates were prepared in essentially the same manner described above for other Chimigen TM Proteins and subjected to Ni-NTA affinity chromatography. Ni-NTA affinity chromatography 25 The cell lysate was loaded onto a Ni-NTA superflow column (10 ml resin bed volume per 2.5 L cell culture pellet) that had been equilibrated with 10 bed volumes of lysis buffer. The column was washed with a wash buffer containing reduced % Tween 20 (6M GuHCl, 50 mM NaH 2</div>
    <div class="description-paragraph style-scope patent-text">PO</div>
    <div class="description-paragraph style-scope patent-text">4 , 0.5 M NaCl, 0. 1% Tween 20, 1OmM a-mercaptoethanol, pH 8.0) until A 2 8 0 &lt;0.01 and then with the same wash buffer containing 15 mM imidazole. 30 Target protein was then eluted with 10 ml bed volumes of elution buffer (6M GuHCl, 50 mM NaH 2</div>
    <div class="description-paragraph style-scope patent-text">PO</div>
    <div class="description-paragraph style-scope patent-text">4 , 250 mM imidazole, 0.1 % Tween 20, lOmM a-mercaptoethanol, pH 8.0) 142 in 1 bed volume fractions. Fractions containing eluted protein were pooled and dialysized against dialysis buffer 3 x 4L (8M Urea, 20mM Tris, 0.1% Tween 20, 25 mM ethylenediamine, 10mIM -mercaptoethanol, pH 8.5). All urea-containing buffers were made with deionized urea to prevent carbamylation of the protein. Urea was deionized 5 with Amberlite@ MB-I (Supelco, PA, USA) (10 g/L/hr) and the cyanate scavanger ethyenediamine was added (25mM final) to the buffer. Ion Exchange Chromatography The dialyzed NS5A Chimigen3 Protein-containing sample obtained by Ni-NTA 10 affinity chromatography was next passed over a Toyopearl@ Super Q3 resin column (2.5 ml bed volume/2.5 L cell culture pellet) that had been equilibrated in dialysis buffer. The ion exchange column was washed with ion exchange wash buffer (8M Urea, 20mM Tris, 0.05% Tween 20, 25 mM ethylenediamine, 1OmM a-mercaptoethanol, pH 8.5) until A 280 &lt; 0.01. Proteins were then eluted from the column with 10 bed volumes of wash buffer 15 containing increasing concentrations of salt (75 mM, 150mM and 500mM NaCl). One bed volume fractions were collected. The NS5A ChimigenTM Protein eluted off predominantly in the 150mM NaCl fractions. A contaminating protein of slightly lower MW eluted off at 75 mM NaCl. Eluted protein fractions were pooled and dialyzed immediately against final dialysis buffer (150mM NaCl, lOmM NaH 2</div>
    <div class="description-paragraph style-scope patent-text">PO</div>
    <div class="description-paragraph style-scope patent-text">4 , 0.05% Tween 20 20, pH8.5) at 4 0 C. 2L per dialysis, with five changes dialysis buffer. Dialyzed proteins were filtered through a 0.2 [m filter that had been pre-wet to prevent proteins sticking to it. Purified NS5A ChimigenTM Protein was stored at 4 0 C. For further purification of the NS3 Chimigen3 Protein, CM-Sepharose3 Fast Flow matrix was equilibrated with 8M Urea (de-ionized), 25mM NaH 2</div>
    <div class="description-paragraph style-scope patent-text">PO</div>
    <div class="description-paragraph style-scope patent-text">4 , 5mM 25 Ethylenediamine, 0.05% (v/v) Tween 20, lOmM DTT, pH 6.50. Protein was eluted using a linear gradient (0 to 0.6 M) of sodium chloride in the same buffer at a flow rate of 1 mL/min. Fractions containing the protein (25-50mM NaCl) were pooled. The multi-antigen Chimigen3 Protein containing sample captured by Ni-NTA column was further purified by HiTrap3 Q XL 1 ml column using AKTAexplorer3 100 30 FPLC system. The protein in elution buffer from Ni-NTA affinity chromatography was concentrated by an Amicon Ultra-15 (MWCO 30,000 Da) and then the buffer was 143 exchanged to Buffer A (8M Urea, 50mM Sodium Carbonate, 25mM ethylendiamine, 1% Tween 20, pH 10). Protein was loaded onto a HiTrap QTM XL column, equilibrated with 50ml of Buffer A, at flow-rate of 60mL/hour. The column was washed with Buffer A until A 280 of elution is below 0.01. Proteins were eluted by linear gradient elution, from 5 100% Buffer A to 100% Buffer B, (BufferA with IM Sodium Chloride) in 20 column volumes. HCV multi-antigen ChimigenTM Protein was eluted in the flow-through fractions and in fractions eluted between 40 and 50% Buffer B. Size Exclusion Chromatography 10 Superdex3 200 preparative grade was packed in a Tricon3 column 10/300 (1 x 30 cm, Pharmacia Biotech) under the pressure of 3 MPa using AKTAexplorer 100 FPLC system (GE healthcare). The column was washed with 100ml of 6 M Guanidine, 50 mM Sodium Carbonate, pH 10. 0.5 mL of the lysate containing the Chimigen3 multi-antigen protein was loaded onto the Superdex 200 column. Protein was eluted by flow rate at 30 15 mL/hour and 0.5 mL fractions were collected. Protein elution was monitored by the absorbance at 280 nm. Hydrophobic interaction chromatography Phenyl-650C Toyopearl@ ( 0.5mL, TOSOH Corp.) was packed into a Poly-prep 20 column (Bio-Rad). The column was equilibrated with 20 mL HIC binding buffer (0.1 M Tris, 2 M Sodium Chloride, pH 8). Sodium Chloride at final concentration of 2 M was added into the 0.5 mL lysate containing the Chimigen3 Multi-antigen protein and the extract was diluted with 3.5 mL HIC binding buffer. Insoluble particulates were removed by centrifugation at 18,000 rpm (39,191 x g, by JA25.50 rotor, Beckman Coluter 25 Avantirm J-25 centrifuge) for 20 minutes. The supernatant was loaded onto the column at flow rate of 30 mL/hour by gravity flow. The column was then washed with 10 mL HIC binding buffer and protein, bound on the column, was eluted with 5 mL HIC elution buffer (8 M Urea, 50 mM Ethylendiamine, 0.5 % Tween 20, pH 10.5). 144 Biochemical evaluation of purified ChimigenTM Proteins The concentrations of proteins were estimated using the Micro BCA3 protein assay reagent kit in a microplate procedure according to the protocol provided by the manufacturer. 5 For SDS-PAGE analysis, aliquots of purified proteins were denatured by adding 5x protein loading buffer and 20x reducing agent and boiled for 5 mins. Denatured proteins were separated on 12% SDS polyacrylamide gels and the gels were stained with PageBlue3 under the conditions provided by the manufacturer. For Western blot analysis, proteins were separated by 12% SDS-PAGE and 10 electroblotted onto Hybond3 ECL3 nitrocellulose membranes using a buffer containing 48 mM Tris base, 39 mM glycine, 20% methanol and 0.0375% SDS. The membranes were incubated first in blocking buffer (1% skim milk, 0. 1% Tween 20 in PBS) for 1 hr at room temperature. Antibodies for detection were diluted in blocking buffer to the desired concentration. The membranes were incubated with the diluted antibodies for 1 15 hr at room temperature with constant mixing. After incubation with each antibody, the membrane was washed three times with blocking buffer for 10 min per wash at room temperature. Detection of proteins was performed by chemiluminescence with the ECL3 Western blotting detection kit and exposure to Kodak Biomax XAR X-ray film. For the qualitative detection of glycosylation of proteins, the Pro-Q@ Emerald 20 300 Glycoprotein Gel and Blot Stain Kit developed by Molecular Probes were used. This kit can be used for detection of carbohydrates on proteins separated by SDS-PAGE in gels or on blots. The stain is compatible with most total protein stains and if desired, analysis by mass spectrometry. A bright green-fluorescent signal is produced when the stain reacts with periodate-oxidized carbohydrate groups, detecting as little as 0.5ng of 25 glycoprotein per band. The stain is a modification of periodic acid and Schiff methods and the manufacturer claims a 50-fold greater sensitivity level. Included in the kit are CandyCaneTM molecular weight standards. The standards consist of alternating glycosylated and non-glycosylated proteins serving as positive and negative controls respecitvely. Following the SDS-PAGE of the protein sample, the gel was fixed in 50% 30 MeOH and 5% acetic acid overnight. The gel was washed twice for 20 minutes in 3% glacial acetic acid, followed by glycan oxidation in the oxidizing solution periodic acid 145 for 30 minutes. The gel was washed three times for 20 minutes with 3% glacial acetic acid followed by staining in fresh Pro-Q3 Emerald 300 staining solution for a maximum of 120 minutes in the dark. An additional two 20 minutes washes in 3% glacial acetic acid in the dark is required before imaging. The excitation/emission max of the stain is 5 280/530 nm with the most optimal visualization at ~300nm. The gel was visualized and scanned using a GeneGenius (Syngene) transilluminator and corresponding software. Immunological characterization of ChimigenTM Vaccines Human PBMCs (peripheral blood mononuclear cells) 10 PBMCs were obtained by Ficoll-Hypaque gradient centrifugation of a leukapheresis preparation from non-HCV-infected individuals having the HLA-A2 haplotype (Biological Specialty Corporation). PBMCs were stored in liquid nitrogen at 3 X 107 cells/cryovial in freezing media (50% Human AB serum, 40% AIM-V®, and 10% DMSO). 15 Isolation and differentiation of monocytes to immature DC (dendritic cells) PBMCs were cultured on 100 mm tissue culture plates (BD Biosciences) for 1 hr at 37'C in AIM V® media with 2.5% matched serum. Following culture, non-adherent cells were removed and the plate washed with AIM V® media. The adherent cells were 20 then cultured with 2 mL of AIM V@/2.5% matched serum containing IL-4 and GM-CSF (1000 IU/mL of each). Binding of HCV Chimigen TM Proteins to immature DCs Immature DCs were obtained from culturing monocytes in the presence of IL-4 25 and GM-CSF for 24-72 hr. Following culture the cells were harvested, washed once with AIM V® media containing 2.5% matched serum, followed by two washes with Dulbecco's phosphate buffered saline (Invitrogen) containing 0. 1% (w/v) BSA (PBSB). The cells were used to evaluate the binding and internalization of ChimigenTM Protein. The phenotype of the immature DCs was assessed by labeling for various cell surface 30 markers including CD64, CD32, CD 16, CD206, HLA-ABC, HLA-DR, CD 14, CD1 Ic, CD86, CD80, CD40, CD83, CD19, CD3, and CD4. 146 For the binding assay, all steps were performed at 4'C with washes following the incubations. Cells were incubated for 60 min in PBSB with various concentrations of Chimigen T M Protein or the corresponding dialysis buffer (2 x 105 cells/well in 96-well v bottom plates in a volume of 25 pL). Protein binding was detected by incubation of the 5 cells with biotinylated anti-mouse IgGI or anti-6xHis antibody in PBSB for 20 min, followed by SA-PE-Cy5 for 20 min. Cells were resuspended in PBSB containing 2% paraformaldehyde (PF). In experiments using NS5A Chimigen T M Protein, the binding was detected with a rabbit anti-NS5A polyclonal antibody, goat anti-rabbitt IgG-biotin, SA-PE-Cy5 combination. Cells were resuspended in PBSB containing 2% 10 paraformaldehyde (PF) and cell binding assessed by fluorescence flow cytometry (FFC). Characterization of DC receptors for ChimigenTM Vaccines using inhibitors of binding Immature DCs were incubated for 60 min at 4'C in PBSB with anti-CD32 mAb 15 (IgG2b isotype), anti-CD206 (IgGI isotype), or isotype control mouse IgG2b or IgGI mAbs. Subsequently, the cells were incubated with ChimigenTM Vaccines in PBSB for 60 min at 4'C. Following washes, the binding to the cells was detected by FFC analysis using either biotinylated anti-mouse IgGI mAb or biotinylated anti-6xHis mAb followed by SA-PE-Cy5. 20 Fluorescence flow cytometry (FFC) analysis Cells were acquired with a FACSCalibur fitted with CellQuest Pro acquisition and analysis software (BD Biosciences). A gate was set on the viable cell population as determined by the FSC and SSC scatter profile and &gt;20,000 events were acquired. The percentage of specific positive cells was calculated as: (% positive cells test sample - % 25 positive cells control) / (100 - % positive cells of control) x 100. The relative mean fluorescence intensity (MFI) was determined as: MFI of the test sample - MFI of the control sample. Antigen presentation assays (APAs) APAs are used to measure the immune response of T cells to antigen presented by 30 APCs. The assays quantify functional T cell immune responses and the ability of 147 antigen-loaded mature DCs to induce proliferation of antigen-specific T cells. The procedure includes differentiating PBMC-derived monocytes to immature DCs, loading the immature DCs with antigen (ChimigenTM Protein or TT), differentiating the immature DCs to mature DCs, and then culturing the mature antigen-loaded DCs together with 5 autologous naYve T cells. For activation and proliferation assays, T cells were analyzed after 7 days of culture. For analysis of T cell function and specificity, T cells were stimulated two additional times with antigen-loaded mature DCs and the production of IFN-y, TNF-a, granzyme B (grB), and perforin (pfn) assessed. Specificity of T cells to antigens was assessed with specific MHC class I tetramers or pentamers. 10 Generation of antigen-loaded mature DCs Immature DCs were generated as described above and incubated for 8 hr with antigen or buffer (control). The cells were then cultured for 16 hr with the maturing agents poly IC (20 pg/mL), recombinant human (rh) IL-1P (10 ng/mL), rhTNF-a (10 ng/mL), rhIL-6 (10 ng/mL), rhIFN-aA (1000 U/mL), and rhIFN-y (1000 U/mL). The 15 extent of maturation of the DCs was assessed by phenotype analysis. The cells were labeled for various cell surface markers including CD64, CD32, CD16, CD205, CD206, CD209, HLA-ABC, HLA-DR, CD14, CD11c, CD86, CD80, CD83, CD40, CD19, CD3, CD8, and CD4. The matured antigen-loaded DCs were washed and cultured with T cells. Isolation of human PBMC (peripheral blood mononuclear cells)-derived T cells 20 T cells were isolated from PBMCs by negative selection using a Dynal Biotech T cell negative selection kit (Invitrogen) following the manufacturer's procedure. Matched sera were used in place of BSA and FBS. The phenotype of the isolated cells was assessed by phenotype labeling for a variety of cell markers. T cells (CD3+ cells) comprised greater than 98% of the isolated population. The T cells were either labeled 25 with CFSE (see below) or added directly to cell culture with DCs. CFSE labelling of T cells Freshly isolated T cells (1-5 x 107 cells) were suspended in 500 l of PBSB and mixed with 500 l of a freshly prepared 10 [g/ml working stock solution of CFSE. Following an incubation for 10 min at 37 0 C the cells were washed extensively with 148 serum containing media (AIM V@/10% matched serum) to remove unincorporated CFSE. CFSE labeling of T cells was confirmed by FFC. Culture of human PBMC-derived T cells T cells were incubated with antigen-loaded mature DCs at ratios of 1-20 x 104 T 5 cells to 1-5 x 104 DCs per well in AIM V@/2 5% matched serum. For the T cell activation and proliferation APA experiments, T cells were harvested after 4 days and 7 days of culture (see below). For the T cell function and specificity APA experiments, T cells were cultured for 7 days and then restimulated with antigen-loaded mature DCs and cultured for an additional 7 days. The 14-day cultured T cells were then split into two 10 groups (intracellular cytokine (ICC) plate and tetramer plate) and stimulated a third time with antigen-loaded DCs. Brefeldin A (BD Biosciences) at 1 pg/mL was added to the wells of the ICC IFN-y plate and the cells cultured for 6 h. The expression of IFN-y, TNF-a, grB, and pfn was assessed as outlined below. Tetramer analysis was performed five-six days following stimulation as outlined below. 15 T cell activation and proliferation analysis For the activation/proliferation APA, T cells were harvested after 4 or 7 days of culture with antigen-loaded DCs. The T cells were assessed for the expression of CD69 (early activation marker) and CFSE intensity (degree of proliferation). Harvested cells were labeled with anti-CD3-PE, anti-CD8-PE-Cy5, and anti-CD69-APC. Using buffer 20 control samples the population of T cells that had not undergone any doubling was identified. This population labeled with a high degree of fluorescence detected in the FLI hi channel and was designated as CFSE . Cell populations that had undergone one division had half of the MFI of the CFSEhi population. Similarly, populations that had undergone two divisions had approximately 25% (4 times less) MFI of the CFSEhi population. Cells 25 with a CFSE fluorescence lower than the CFSEhi fluorescence were designated as CFSE . Some of cell populations had near background FLI channel fluorescence and could be designated CFSE- (CFSE negative). However for purposes of the experiments outlined here T cells were considered CFSEhi (no cell divisions) or CFSE" (at least one cell division). 149 T cell activation was quantified by assessing the expression of CD69. In some experiments T cell blasts were quantified by gating on the population of high FSC and SSC intensity CD3+ T cells. Thus relative number of blast cells in a cell population was expressed (for these studies) as the proportion of cells with a larger diameter (FSChi ) and 5 with greater cellular complexity (SSChi) compared with the small (GO) resting cells in the population. Detection of Intracellular IFN-y, TNF-a, grB and pfn The production of IFN-y and TNF a and the expression of the serine protease 10 granzyme B (grB) [Lobe et al. (1986) Science 232:858-861] as well as the pore-forming protein perforin (pfn) [Hameed et al. (1992) Am. J. Pathol. 140:1025-1030] were quantified using a standard ICC (intracellular cytokines) protocol (BD Biosciences). In brief, this consisted of labeling the cells with specific fluorochrome conjugated mAbs for detection of CD3 (anti-CD3-APC) and CD8 (anti-CD8-PE-Cy5), followed by fixing and 15 permeablization. The cell samples were then divided into two samples, one of which was incubated with anti-IFN-y-PE antibody and anti-grB-FITC antibody and the other with anti-TNF-a-PE and anti-perforin-FITC. On average between 20,000-100,000 cells per sample were acquired using a BD FACSCalibur. 20 Tetramer and pentamer analysis T cells were labeled with anti-CD8-PE-Cy5, anti-CD4-APC, and anti-CD69-FITC antibodies and one of the following PE-conjugated iTagTM tetramers (Beckman Coulter) or pentamers (ProImmune): HCV NS5A (VLSDFKTWL; SEQ ID NO:3X) HLA A*0201, EBV (GLCTLVAML;SEQ ID NO:5X) HLA-A*0201, HCV NS3 peptide 25 (CINGVCWTV; SEQ ID NO:4X) HLA-A*0201, HCV NS3 peptide (KLVALGINA; SEQ ID NO:6X) HLA-A*0201, and a negative control tetramer (multi-allelic). Approximately 100,000 cells were acquired using the FACSCalibur. 150 Analysis of ChimigenTM Protein binding, internalization and processing by confocal microscopy Binding, internalization and processing of the ChimigenTM Protein by immature DCs was studied using confocal microscopy. Immature DCs used in these studies were 5 obtained by differentiating adherent PBMC derived monocytes for 2 days in the presence of GM-CSF and IL-4 in AIM V® media containing 2.5% donor matched serum. On day 2, immature DCs were transferred to chambered slides and incubated for an additional day before use. Day 3 was chosen as a compromise between cells having the appropriate cell surface receptors and morphology. 10 To study binding of ChimigenTM Protein to DC surfaces, cells were incubated with 5 Tg/mL ChimigenTM Protein or with buffer only as a negative control in PBSB at 4'C for 1 hr. After 1 hr, cells were washed with PBSB and then labeled with biotinylated anti-mouse IgGI antibody followed by streptvidin AlexaFluor@ 546. PBSB washes were performed between each step. After labelling and washing, cells were fixed for 10 min. 15 at 4'C with 4% paraformaldehyde (made in PBSB). Slides were then mounted with SlowFade® Gold antifade reagent with 4',6-diamidino-2-phenylindo le,dihydrochloride (DAPI; Invitrogen) and cover slips were sealed onto the slides with nail polish. Internalization of Chimigen T M protein (5 Tg/mL) by DCs was studied either by directly incubating the cells in media containing Chimigen T M Protein at 37 0 C (7% C0 2 ) 20 or by first labeling the surface receptors at 4'C in PBSB, washing away the unbound protein, and then studying the uptake of the receptor bound protein over time (0 min., 15 min., 60 min. and 240 min.) at 37'C (7% C0 2 ) in AIM V@/2.5% matched serum media. Cells incubated at 37'C were washed with PBSB and then fixed and permeabilized for 10 min. with BD Biosciences Cytofix/CytopermTM solution. Cells were then washed and 25 labeled (1hr) with biotin anti mouse IgGI in BD Biosciences Perm/Wash T M solution followed by labeling with streptavidin Alexa Fluor® 546. Co-labeling with other antibodies was performed as necessary. After the final washing of the cells, the slides were mounted as described above. To confirm that the ChimigenTM Protein was endocytosed, pulse-chase 30 experiments were performed. Immature DCs were pulsed with ChimigenTM Protein (5 Tg/mL) for 30 min. on ice. Cells were washed with PBSB and chased in AIM 151 V@/2.5% matched serum media without ChimigenTM Protein and incubated at 37 0 C (7% C0 2 ) for 15 min. Pulse-chased cells were washed with PBSB, fixed with 4% paraformaldehyde and labeled with MHC Class II antibody to label only the plasma membrane. To determine if the ChimigenTM Protein is present in endosomes, plasma 5 membrane-labeled cells were then fixed and permeabilized for 10 min. with BD Bisosciences Cytofix/CytopermTM solution. After washing with BD Perm/WashTM, ChimigenTM Protein was detected with anti mouse IgGI biotin/streptavidin, as described above. For macropinocytosis studies, FITC Dextran (MW 70,000, anionic, lysine fixable, 10 Invitrogen) was used as a fluid phase marker at 5 mg/ml in AIM V@/2.5% matched serum medium either with or without the ChimigenTM Protein (5 Tg/mL). To study receptor mediated endocytosis Alexa Fluor® 488 transferrin conjugate (Invitrogen) was used at 20 Tg/mL in PBSB containing Chimigen T M Protein (5 Tg/mL). Lactacystin (Sigma) was used both as a cysteine protease inhibitor and as a proteasome 15 inhibitor (final conc. 5 Tg/mL). Evaluation of Immune Responses in in vivo Animal Models These studies use two inbred laboratory (mouse and rat) and one out-bred large animal (piglet) species. In particular, BALB/c mice (6-8 weeks old from Charles River 20 Laboratories), Wistar rats (4-6 weeks old from Charles River Laboratories), and cross bred piglets (4-6 weeks old from Prairie Swine Center, University of Saskatchewan) are used. The study determines immune responses and protective efficacy of ChimigenTM Protein. 25 Safety evaluation HCV ChimigenTM Proteins are administered either subcutaneously (s.c.) or intradermally (i.d.). The following protocol and doses is used for injections. Animals are immunized four times, on day 0, day14, day 28, and day 42, every two weeks either s.c. or i.d. For mice s.c. and i.d injections, a dose of 0.1 Tg, lug or 10 Tg/mouse is used. 30 The doses for the immunization of rats will be 0.15 Tg, 1.5 Tg or 15 Tg/rat and for piglets are 0.2 Tg, 2 Tg or 20 Tg/piglet. 152 Blood samples are collected pre-immunization (day -1) and 7 days after each injection (day 7, 21, 35, 49) for analysis of the quantity of specific antibodies as well as IgG1/IgG2a ratios by ELISA techniques. Animals are sacrificed two weeks after final immunization. The safety profile of 5 HCV ChimigenTM Proteins are evaluated by physical examination of the animals at least three times per week after immunization. This includes body weight and adverse event observation. For systemic toxicology, blood samples collected at regular intervals are used to monitor changes in serum chemistries, including aspartate aminotransferase (AST) and alanine aminotransferase (ALT) levels. In addition, at the end of the 10 experiments, tissues collected from spleen, liver, kidney, heart, lung, muscle, and brain at the time of necropsy are fixed in 10% buffered formalin and embedded in paraffin for future analysis of potential pathological changes. Age-matched animals are used as controls. 15 Immune responses to ChimigenTM Proteins ChimigenTM Proteins are predicted to induce strong cellular and humoral immune responses. Animal trials are performed to determine host immune responses to HCV Chimigen T M Caccines in piglets. Core, NS5A, NS3, and Multi-antigen Chimigen T M proteins are used for these studies. In the first round, immune responses to HCV 20 ChimigenTM Proteins are evaluated. The proteins are given s.c. and i.d., as described above. Splenocytes of mice or rats, and peripheral blood mononuclear cells (PBMCs) from piglets are used to determine the quantity and quality of immune responses to HCV ChimigenTM proteins following s.c. and d routes of administration, as described below. 25 These trials allow us to determine which of the ChimigenTM Vaccines and the routes of administration will induce the strongest immune response. The target-directed delivery of the HCV ChimigenTM proteins could elicit a potent Thl-biased immune response in addition to strong humoral response. Studies from VIDO on HCV vaccines have demonstrated that priming with DNA vaccines followed by protein boosting can induce 30 strong and Thl/Th2-balanced immune responses [Yu et al. (2004) J. Gen. Virol. 85:'533-1543]. 153 Evaluation of immune responses i) Antibody responses. The presence of HCV antigen-specific antibodies is determined by ELISA to test total IgG as well as IgGI and IgG2a antibody levels. The 5 IgG levels demonstrate the quantity of the immune responses, while the relative levels of IgGI and IgG2a demonstrate the quality (Th1 or Th2) of the immune responses. These experiments are performed using established protocols. ii) Lymphocyte proliferation assays. Splenocytes of mice or rats, peripheral blood mononuclear cells (PBMCs) from piglets are stimulated with HCV antigens in vitro. 10 Proliferative responses are measured by [methyl- 3 H] thymidine incorporation into the DNA of dividing cells. iii) Cytokine ELISPOT assays. To further confirm the quality of the immune responses, the number of interferon-y and interleukin-4 secreting cells in splenocytes (mouse and rat) or PBMCs (piglet) are determined in ELISPOT assays after stimulation 15 with HCV antigens as per our established protocol. Protective anti-viral immunity induced by HCV ChimigenTM Proteins HCV has a very narrow host range. It replicates only in humans and chimpanzees. Challenge of vaccinated animals with live HCV is not practical, as 20 chimpanzees are expensive and limited in supply. However, challenging with recombinant vaccinia virus encoding an HCV antigen after vaccination is an alternative model for evaluation of protective ability induced by HCV prophylactic vaccines in animal models. ChimigenTM Proteins, individually or as combinations, are used to immunize the animals. To evaluate the protective immunity induced following the 25 vaccinations, animals are challenged intraperitoneally by recombinant vaccinia viruses encoding the same HCV antigen as relevant Chimigen3 Proteins two weeks after the completion of the scheduled vaccination using a pre-determined strategy. The challenge doses will be 1X10 7 plaque-forming units (PFUs) for mice, 2X10 7 PFUs for rats, and 1X10 9 PFUs for piglets. Five days later, animals will be sacrificed and vaccinia virus 30 titers will be determined by plaque assays as per stablished protocol. 154 Identification of the most suitable candidate(s) for HCV therapeutic and prophylactic vaccines Therapeutic vaccines are based on non-structural proteins of the HCV virus (e.g. NS5A, NS3), whereas the prophylactic vaccines are based on structural proteins (e.g. El, 5 E2) as well as non-structural proteins. A combination of one or more of the ChimigenTM Vaccines is used in both the ex vivo DC/T cell antigen presentation assays and in the animal models and the immunological outcome is evaluated. The immunization protocols as well as the route of administration of the ChimigenTM Vaccines for therapeutic and prophylactic uses are currently being studied. 10 The immune responses are evaluated by measuring antibody levels, lymphocyte proliferation and cytokine production. Protective anti-viral immunity induced by the prophylactic HCV ChimigenTM Vaccine candidates is also be evaluated in challenge experiments, as described above. 15 Example 2X. Results with NS5A Chimigen3 Protein NS5A ChimigenTM Protein has been purified and characterized Purified NS5A Chimigen T M protein migrated by SDS-8%PAGE as a band of~ 105 kDA, although the predicted molecular weight of the protein is ~ 81 KDa. The discrepancy between the observed and predicted molecular weights may result in part due 20 to the high proline content (~11%), glycoyslation and other possible post-tranlational modification of the protein. The purified protein was detected with antibodies against mouse IgG1 Fc, 6xHis tag and NS5A. MS/MS ID (Mass Spectrometry) analysis on the purified protein (band cut from gel) gave significant hits for NS5A, mouse IgG1 heavy chain and HCV polyprotein indicating it was indeed the NS5A Chimigen TM Protein. 25 Purified NS5A ChimigenTM Protein was separated on a 8% SDS gel and stained for glycosylation using the Pro-Q@ Emerald 300 Glycoprotein Gel and Blot Stain Kit and detected by UV illumination. This procedure showed glycosylation of the purified NS5A Chimigen T M Protein. 30 NS5A ChimigenTM Protein binds to immature DCs 155 NS5A ChimigenTM Protein was examined for its ability to bind to immature DCs. The cells were incubated in the presence and absence of various concentrations of NS5A ChimigenTM Protein for 1 hr at 4'C. The bound vaccine was detected with biotinylated anti-mouse IgGI mAb and SA-PE-Cy5. The percentage of cells binding the vaccine (% 5 positive cells) and the relative amount of bound protein (MFI) was determined by flow cytometry (Fig. 15X). With NS5A Chimigen T M Protein at 4-50 pg/mL, most DCs were positive for binding, and there was a dose-dependent increase in the amount of bound protein (Figs. 15X and 16X). Binding of the protein was not much greater at 20 [g/mL compared with 50 [g/mL indicating that the binding to immature DCs was saturable. 10 The high MFI of binding observed suggests that NS5A Chimigen T M Protein binds very effectively and at high levels to immature DCs. That the binding at 4'C was saturable suggests that the process is receptor-mediated. The binding was also very rapid with bound protein detected after 5 min of incubation at 4'C with a MFI of binding approximately half of that observed after 1 hr incubation (data not shown) 15 NS5A ChimigenTM Protein binds to specific receptors on immature DC By virtue of the the Fc fragment it contains, the NS5A ChimigenTM Protein is predicted to bind via its TBD region to CD32 (FcyRII) on immature DCs. In addition, due to its mannose glycosylation, NS5A ChimigenTM Protein is predicted to bind to C 20 type lectin receptors such as CD206 (MMR). To determine the specificity of binding of the vaccine candidate, immature DCs were incubated with NS5A ChimigenTM Protein in the presence of blocking anti-CD32 and/or anti-CD206 mAbs. Immature DCs were incubated with buffer control, or 5 [g/ml of isotype control mAb, anti-CD32 mAb, anti-CD206 mAb, or both anti-CD32 and anti-CD206 for 1 hr at 25 4'C before incubation with NS5A Chimigen T M Protein for 1 hr at 4'C. Bound NS5A Chimigen T M Protein was detected with biotinylated anti-6xHis mAb followed by SA-PE Cy5. Isotype control mAbs (murine IgGI and IgG2b) did not inhibit binding compared with buffer control. However in comparison with buffer control, both anti-CD32 and anti CD206 inhibited binding by approximately 60% and 40%, respectively. 30 The addition of both mAbs further inhibited NS5A Chimigen T M Protein binding resulting in an 80% inhibition. These data indicate a role for both CD32 and CD206 in 156 the binding ofNS5A Chimigen T M Protein. By confocal microscopy, the cells incubated with NS5A ChimigenTM Protein at 4'C showed an intense labeling on their surface compared with buffer only controls indicating the ChimigenTM Protein bound to the cell surface. 5 The internalization of the NS5A ChimigenTM Protein by immature DCs was evaluated at 37'C. Immature DCs incubated with ChimigenTM Protein at 37'C for 1 hr showed a punctuate labeling pattern, often in the vicinity of the nucleus, suggesting the ChimigenTM Protein was internalized and there was very little, if any, surface labeling. DCs are capable of antigen uptake via several routes which include phagocytosis, 10 macropinocytosis, clathrin-mediated endocytosis and non-clathrin/caveolae endocytosis. Macropinocytosis is reported to be a constitutive process in immature DCs (Trombetta and Mellman 2005). The ability of immature DCs to internalize NS5A Chimigen T M Protein by macropinocytosis at 37'C was evaluated using the macropinocytosis marker FITC dextran [Hewlett et al. (1994) J. Cell Biology 124:689-703]. After incubating 15 immature DCs for15 min and 60 min with the ChimigenTM Protein and FITC dextran, vesicle-like structures were observed which contained both protein and FITC dextran, indicating that at 37'C the ChimigenTM Protein may be taken up by macropinocytosis. It should also be noted that FITC Dextran may bind to macrophage mannose receptors (CD206) and thus some of the endosomes containing both ChimigenTM Protein and FITC 20 Dextran may have arisen by receptor-mediated endocytosis. The role of receptor mediated endocytosis in the uptake of ChimigenTM Protein was tudied by pulse-chase experiments. Immature DCs were pulsed with fluorescent labeled ChimigenTM Protein at 4'C, washed, and incubated with for 15 min at 37'C. The cells were fixed, permeabilized and labeled with antibodies to detect the ChimigenTM 25 Protein as well as the transferrin receptor. The transferrin receptor is taken up by receptor mediated endocytosis and then recycled back to the plasma membrane from early endosomes. At 4'C, the NS5A ChimigenTM Protein bound to the surface of the cells while the transferrin receptor was present predominantly within the cells. On switching to 37'C, endosomes were observed to form, some of which contained both the 30 Chimigen ProteinTM and transferrin receptors. The large number of pre-existing intracellular transferrin receptors at the start of the experiment (4'C) is probably 157 responsible for many transferrin containing endosomes not co-localized with the Chimigen TM Protein. The uptake of the NS5A Chimigen T M Protein by DCs and its co-localization with transferrin, by co-labeling with an antibody against the transferrin receptor also was 5 evaluated using confocal microscopy. Transferrin binds to transferrin receptor and is known to be internalized by receptor-mediated processes. This analysis showed co localization of the two molecules, thereby indicating that NS5A ChimigenTM Protein is taken up by receptor-mediated endocytosis. In an attempt to increase the overlap between ChimigenTM Protein and transferrin 10 receptor signals, cells were incubated with Alexa Fluor 488 conjugated to human transferrin so as to indirectly detect only recently endocytosed transferrin receptors rather than all transferrin receptors in the cell. Immature DCs were surface labeled at 4'C with a mixture of ChimigenTM Protein and Alex Fluor 488 conjugated to transferrin. Few Alexa Fluor 488 transferrin positive endosomes were observed, but when present they 15 contained the NS5A ChimigenTM Protein indicating that the ChimigenTM Protein is indeed taken up by receptor mediated endocytosis. These results show that the Chimigen3 Protein is predominantly internalized by receptor-mediated endocytosis. The processing of the NS5A ChimigenTM Protein by immature DCs was studied. Since the vaccine is designed, inter alia, to treat chronic infections, activation of CD8+ 20 cells and antigen cross presentation via MHC Class I receptors is required. Two different processing routes have been proposed for antigen cross presentation [Liz6e et al. (2005) Trends Immunol. 26(3):141-149]. The first involves processing of antigens taken up by phagocytosis, while the second involves processing of antigens taken up be other routes of endocytosis such as receptor mediated endocytosis. In the second route proteins are 25 taken up into early endosomes and the targeted to late endosomes where they are broken down by cathepsins and then loaded onto MHC Class I receptors. Thus, experiments were performed to determine if the NS5A ChimigenTM Protein could be detected in late endosomes and also to determine if it co-localizes with MHC Class I receptors in such structures. Cells that had been pulsed with NS5A ChimigenTM Protein at 4'C and then 30 chased at 37'C were co-labeled to detect both the ChimigenTM Protein and LAMP 1 (a marker of late endosomes/lysosomes). At 4hr and 24 hr, in a few cells, overlap was 158 observed between the NS5A chimigen protein and LAMP 1 indicating that the NS5A chimigen protein was in late endosomes/lysosomes. In another co-labeling experiment, the NS5A ChimigenTM Protein was found to be present in similar structures with MHC Class I molecules, thereby indicating that the ChimigenTM Protein is processed for 5 presentation via MHC Class I molecules. Proteasomes are believed to be involved in the breakdown of antigens for cross presentation via the phagolysosome pathway. Cells were treated with the cell-permeable proteasome inhibitor Lactacystin at a concentration of 5 pg/mL. Lactacystin recently has been shown to be less specific than previously thought at inhibiting the lysosomal 10 protease Cathepsin A [Kozlowski et al. (2001) Tumour Biol. 22(4):211-215]. If processing ofNS5A ChimigenTM Protein involved proteasomes as in the phagolysosome pathway, then one would expect a partial accumulation of incompletely degraded peptides in the cytosol and such an accumulation would not be expected for processing in the late endosome [Liz6e et al. (2005) supra]. Pulsed cells treated with 5ug/mL of 15 lactacystin (pulse and chase) after a 4hr chase showed a larger number of endosomes containing the NS5A ChimigenTM Protein. In addition, the endosomes appeared to contain more of the protein than those of control cells. An increase in cytoplasmic NS5A ChimigenTM protein could not be detected with a monoclonal antibody against mouse IgGI. These data again point to the receptor-mediated endocytosis rather than the 20 phagolysosome pathway being the mechanism whereby Chimigen3 Proteins are internalized in DC. NS5A Chimigen T M Protein presentation by DCs results in both CD8+ and CD4+ T cell activation and proliferation 25 The functional immune response to NS5A ChimigenTM Protein was assessed by ex vivo APAs. This assay can be used to measure various parameters of a functional T cell immune response after stimulation of T cells with antigen-loaded DCs. The assay consists of first generating immature DCs from PBMC-derived monocytes by the addition of IL-4 and GM-CSF. The immature DCs are then incubated with vaccine 30 candidate, carrier buffer (negative control), or tetanus toxoid (TT) (positive control). The DCs are then treated with cytokines to undergo maturation, washed, and incubated with 159 autologous naYve T cells. For measuring cytokine production, the presence of cytotoxic granule components, and the generation ofNS5A-specific T cells, the T cells are stimulated an additional two times allowing for the expansion of ChimigenTM Protein specific T cells. Activation was assessed by measuring the early T cell activation 5 marker CD69, and proliferation was measured by tracking the fluorescence of CFSE labeled T cells. Both CD69 expression and CFSE fluorescence were evaluated after 4 and 7 days of culture with antigen-loaded DCs. A preliminary analysis had indicated that the concentration of DCs and T cells in the culture were important parameters in the determination of T cell immune response. 10 Thus an APA was designed such that six different T cell:DC ratios were assessed. Two sets of DC concentrations were used, a high concentration of 5 x 104 DCs/well and a low concentration of 1 x 104 DCs/well. After 48 hr of culture, the immature DCs were incubated with buffer (negative control), two different preparations ofNS5A ChimigenTM Protein (5AC) at 5 [tg/mL, TT (positive control), or PBS. The DCs were then cultured for 15 8 hr and matured by the addition of poly IC, IL-I, IL-6, TNF-a, IFN-a, and IFN-y. After culture overnight (16 hr) DCs from the PBS control group were washed and examined for the expression of various mature DC markers. Both high and low concentration DCs expressed high levels of HLA-ABC (MHC class I), HLA-DR (MHC class II), CD86, CD80, and CD83 (Fig. 17X). In general, the high concentration DCs expressed slightly 20 higher levels of DC maturation markers. Autologous T cells were isolated by a negative selection procedure and labeled with CFSE for the determination of cell division. To 100 1t/well of DCs in a 96-well plate, 100 1t/well of T cells were added for concentrations per well of: 20 x 104, 5 x 104, or 2 x 104. For the high DC concentration wells (5 x 104 DC/well) the T cell to DC per 25 well ratio combinations were: 20 x 104:5 x 104 (4:1), 5 x 104:5 x 104 (1:1), and 2 x 104:5 X 104 (0.4:1). For the low DC concentration wells (1 x 104 DC/well), the T cell to DC ratio combinations per well were: 20 x 104:1 x 104 (20:1), 5 x 104:1 x 104 (5:1), and 2 x 104:1 x 104 (2:1). T cells were added to the DC in the absence of any exogenous cytokines. As a control, at day 3 of culture PHA at 1 [tg/mL was added to the T cells 30 loaded onto the PBS treated DC group. 160 Following 4 days of culture, half of the cell culture (100 [l) was harvested for analysis of activation and proliferation. To the remaining half of the cell culture, 100 [d of fresh AIM V@/2.5% matched sera was added and the cells cultured for an additional 3 days. The expression of CD69 on the T cells following 4 days of culture is shown in Figs. 5 18X A-C. Fig. 18X A shows the percentage of CD3+ cells expressing CD69 for the different T cell:DC ratios. The majority of PHA treated cells expressed CD69 regardless of the T cell:DC ratio. CD69 was also detected in T cells cultured with the high DC concentration but was barely detected in T cells cultured with the low DC concentration. Compared with buffer control, antigen stimulated T cells expressed a higher level of 10 CD69. NS5A Chimigen T M Protein-loaded DCs induced a higher percentage of CD69 expressing CD8+ T cells than CD4+ T cells at day 4 (Figs. 18X B and X C). The percentage of CD69 expression of CD8+ T cells was equivalent or greater for the Chimigen3 Protein compared with the recall antigen TT. This indicated that the NS5A ChimigenTM Protein is a strong activator of naYve CD8+ T cells 15 The percentage of cells that had undergone at least one division (CFSE ) after four days of culture is shown in Figs. 19X A-C. T cells treated with PHA 24 hr earlier had begun to divide (Fig. 19X A). CD8+ and CD4+ T cells treated with TT-loaded DCs undergo detectable proliferation after 4 days of culture but this was only evident at the high DC concentration (Fig. 19X B and X C). There was little detection of T cell 20 proliferation in the ChimigenTM Protein-treated groups at day 4. Thus naYve T cells were activated by NS5A Chimigen T M Protein-loaded DCs on day 4 of culture as evidenced by expression of CD69 but these T cells have not yet divided or had divided undectably by the assay used. Following 7 days of culture, cells were harvested for analysis of activation and 25 proliferation. The expression of CD69 on the T cells following 7 days of culture is shown in Figs. 20X A-C Fig. 20X A shows the percentage of CD3+ cells expressing CD69 for the different T cell:DC ratios. There was a marked increase in CD69 expression of the T cells treated with PHA. However, the percentage of cells expressing CD69 has decreased from that observed at day 4 consistent with what is expected from a PHA 30 response; rapid induction of CD69 followed by a decrease in expression with time. For Chimigen3 Protein- stimulated T cells, CD69 was detected at levels over 5% in T cells 161 cultured with the high DC concentration but was barely detected in T cells cultured with the low DC concentration. Thus, the low DC concentration (1 x 10 4 DC/well) was not sufficient for antigen-specific T cell activation. Compared with buffer control, a greater number of Chimigen3 Protein- stimulated T cells expressed CD69 for the 5 x 10 4 T cell 5 and 2 x 10 4 T cell : 5 x 10 4 DC ratios. The expression of CD69 was reduced for the recall TT response at day 7. In contrast to d4 T cells, NS5A ChimigenTM Protein-loaded DCs induced a higher percentage of CD69 expressing CD4+ T cells than CD8+ T cells at day 7 (Figs. 20X B and X C). For the higher DC concentration (5 x 10 4 /well) the percentage of CD69 expression of CD8+ and CD4+ T cells was equivalent or greater for 10 the ChimigenTM Protein compared with buffer or the recall antigen TT. Thus the NS5A Chimigen T M activates naYve CD8+ T cells initially, followed by CD4+ T cells The percentage of cells that have undergone at least one division (CFSE ) after seven days of culture are shown in Figs. 21X A-C. The results show that essentially every T cell treated with PHA has divided (Fig. 21X A). DCs loaded with Chimigen3 15 Protein or TT resulted in marked T cell proliferation after 7 days of culture but this was most evident at the high DC concentration. Only the high DC concentration wells induced marked CD8+ T cell proliferation as a result of antigen loading (Fig. 21X B). However, the low concentration of DCs loaded with antigen was sufficient to induce CD4+ T cell proliferation (Fig. 21X C). Notably, at the high DC concentration, the 20 Chimigen3 Protein-loaded DCs induced T cell proliferation to levels comparable to TT loaded DCs Another measure of T cell proliferation is the relative proportion of blast T cells in the T cell population. Blast T cells are defined as those cells possessing a higher FSC (forward light scatter) and SSC (sidelight scatter) then the resting lymphocytes in the 25 lymphocyte gate as assessed by flow cytometry. The percentage of T cell blasts in the cultures is shown in Fig. 22X. These results correlate very well with the percentage of cells undergoing division as shown in Fig. 21X A. Therefore, the assessment of T cell blasts in a population can be used as an alternative to the CFSE assay. Overall, these findings indicate that the NS5A ChimigenTM Protein was quite efficient at inducing a 30 primary T cell response as measured by T cell activation and proliferation. 162 NS5A ChimigenTM Protein presentation by DCs results in the generation of CD8+ and CD4+ T cells producing IFN-y and TNF-a The functional immune response to NS5A ChimigenTM vaccine was assessed by a three stimulation ex vivo APA. Immature DCs at either 4 x 104 DCs/well (high 5 concentration) or at 2 x 104 DCs/well (low concentration) were loaded with control carrier buffer, PBS, TT (positive control), or NS5A ChimigenTM Protein. DCs were then matured and their phenotype evaluated. The DCs maturation was established using the high level expressions of MHC class I, MHC class II, CD86, CD80, and CD83 (Fig. 23X). Autologous T cells were incubated with the matured antigen-loaded DCs at a ratio 10 of 20 x 104 T cells/well : 4 x 104 DCs/well or 4 x 104 T cells/well: 2 x 104 DCs/well. The T cells were stimulated three times and T cell function evaluated 6 hr following the third stimulation by detection of the intracellular levels of the Th1 cytokines IFN-y and TNF-a. In addition the level of blast T cells was also assessed. Fig. 24X shows the percentage of blast T cells at the high and low DC 15 concentrations. The 2:1 T cell:DC ratio resulted in a lower background (buffer) T cell proliferative response compared with the 5:1 ratio. As a result with the 2:1 ratio there was a more marked difference between buffer and antigen-induced T cell proliferative response. The IFN-y response was measured at the 5:1 and 2:1 T cell:DC ratios. The data is 20 shown as the responses of each well of the group and as an average of the three wells with the standard deviation of the mean (Fig. 25X). A comparison of the T cell IFN-y response showed a marked difference between the 5:1 and 2:1 T cell:DC ratios. With the higher DC concentration there was no evidence of a Chimigen 3-induced IFN-y response over that of control buffer. However with the lower DC concentration, very 25 few T cells cultured with control buffer-loaded DCs produced IFN-y whereas a high percentage of T cells cultured with Chimigen3 Protein-loaded DCs produced IFN-y. There were more IFN-y producing cells in the T cells stimulated with DCs that had been loaded with 5 [g/mL compared with 2.5 ug/mL ofNS5A Chimigen T M Protein. The percentage of T cells expressing IFN-y in the CD8+ and CD4+ population was also 30 measured (Figs. 26X and 27X). The low DC concentration groups showed a high percentage of CD8+ T cells expressing IFN-y as a result of stimulation with Chimigen3 163 DCs. The percentage of CD8+ T cells expressing IFN-y was higher for T cells stimulated with Chimigen3 Protein-loaded DCs compared with TT-loaded DCs (Fig. 26X). Similarly, there was also a high percentage of CD4+ T cells that expressed IFN-y upon stimulation with NS5A Chimigen T M Protein compared with control buffer (Fig. 27X). 5 The percentage of CD4+ T cells expressing IFN-K was comparable for T cells stimulated with Chimigen3- Protein-loaded DCs compared with TT-loaded DCs (Fig. 27X). These results indicate that NS5A ChimigenTM Protein induces a marked IFN-y response in both CD8+ and CD4+ T cell populations and suggests that the molecule is processed by the DCs in both the MHC class I and class II pathways. 10 Fig. 28X shows the percentage of T cells that have produced TNF-a as a result of a 6 hr stimulation with antigen-loaded mature DCs. These results are similar to the IFN-y results. Although there was an increase in the percentage of cells producing TNF-a as a result of antigen stimulation of the T cells with the high DC concentration (5:1 ratio), there was an even greater difference with the low DC concentration (2:1 ratio). A higher 15 percentage of T cells produced TNF-a when stimulated by DCs loaded with 5 [g/mL of Chimigen3 Protein compared with 2.5 [g/mL of protein. The TNF-a response was greater for the NS5A Chimigen T M Protein compared with TT. Stimulation with TT loaded DCs resulted in a higher percentage of CD4+ T cells expressing TNF-a compared with CD8+ T cells (Fig. 29X). However, NS5A Chimigen T M Protein-loaded DCs induced 20 a similar degree of TNF-a production in both CD8+ and CD4+ T cell populations (Fig. 29X). NS5A ChimigenTM antigen presentation by DCs results in the generation of CD8+ T cells expressing grB and pfn + and CD4+ T cells 6 hr post 3rd stimulation 25 The ability of T cells to produce the cytotoxic granular proteins grB and pfn was also assessed by ex vivo antigen presentation assays. Immature DCs were loaded with control buffer, with TT (positive control), or varying concentrations of NS5A ChimigenTM Protein and upon maturation were incubated with autologous T cells. The expression of grB can be detected in different ways, including enzymatic assays and by 30 specific antibodies [Ewen et al. (2003) J. Immunol. Meth. 276:89-101; Spaeny-Dekking et al (1998) J. Immunol. 160:3610; Hamann et al. (1997) J. Exp. Med. 186:1407]. GrB 164 and pfn expression were detected by intracellular staining using an anti-grB and anti-pfn mAbs, respectively. Fig. 30X shows the percentage of CD8+ T cells that express grB and pfn following three stimulations with antigen-loaded mature DCs. NS5A ChimigenTM Protein-loaded DCs induced an increase in grB and pfn expression in CD8+ 5 T cells compared to the no antigen control. These results indicate that NS5A ChimigenTM Protein induces the expression of grB and pfn in CD8+ T cells and this suggests that this protein is processed by the DCs in the class I pathways for the effective presentation to T cells which results in their differentiation from naive CD8+ T cells to cytotoxic T lymphocytes (CTLs). 10 NS5A ChimigenTM antigen presentation by mature DCs results in the generation and maintained activation of CD8+ and CD4+ T cells T cells were stimulated with antigen-loaded DCs three times in an APA. After 6 days of culture following the third stimulation the T cells were harvested and investigated 15 by FFC for the percentage of blast cells as a measure of proliferation and for the expression of the activation marker CD69. In addition as a means to estimate absolute numbers of T cells recovered from culture, the number of gated cells falling in the lymphocyte gate (RI gate) based on FSC and SSC flow cytometric analysis was determined. 20 There was a marked difference in the recovery of T cells from TT and Chimigen T M Protein stimulated cells compared with buffer control (Fig. 3 1X). TT stimulated cells gave a higher T cell recovery than the Chimigen3 Protein stimulated cells. However the TT response is a recall response and thus the starting population of T cells specifically responsive to TT would be expected to be higher than that of the 25 starting population of naYve T cells specific for NS5A. Notably, on assessment of the blast cell population, the percentage of blast cells / proliferating cells was actually higher in the NS5A Chimigen3 Protein cultures compared to the TT cultures. There were very few blast cells / proliferating cells in the buffer control cultures. The percentage of activated CD8+ and CD4+ T cells as assessed by CD69 expression is shown in Fig. 32X. 30 There was a high percentage of both CD4+ and CD8+ T cells expressing CD69 in T cells stimulated with Chimigen3 Protein-loaded DCs compared with buffer control. These 165 results show that the stimulation with the Chimigen3 Protein results in marked T cell activation and proliferation that is evident even six days following the third stimulation (day 20 of T cell culture). The Chimigen3 Protein is therefore very effective in the activation and expansion of both CD8+ and CD4+ T cells. 5 NS5A ChimigenTM Protein presentation by mature DCs induces the generation ofNS5A specific CD8+ T cells To evaluate the antigen-specificity of the immune response to NS5A ChimigenTM Protein, the percentage of T cells specific to an immunodominant NS5A epitope in the 10 context of HLA-A2 was quantitated. This was determined by labeling T cells with an NS5A peptide/HLA-A2 pentamer conjugated to PE. Naive T cells were stimulated three times with DCs loaded with different concentrations ofNS5A ChimigenTM Protein and compared to the respective control DCs loaded without antigen (buffer) in an APA. T cells were harvested six days after the third stimulation and NS5A-specific T cells or 15 EBV-specific T cells (control) detected by tetramer labeling and FFC. The percentages of negative tetramer labeling (negative control) and EBV tetramer labeling (positive control) CD8+ and CD4+ T cells are shown in Fig. 33X. One well of the three tested was positive for EBV tetramer labeling (positive tetramer) in the CD8+ T cell population. As the T cells assessed were from the buffer control treated 20 wells, it would be expected that the number of EBV tetramer-labeled T cells would be relatively low. The percentage of CD8+ T cells labeling with an NS5A pentamer following the APA is shown in Fig. 34X. Loading DCs with NS5A Chimigen T M Protein resulted in the generation of T cells with specificity to the NS5A epitope VLSDFKTWL (SEQ ID NO:3X). The marked expansion of CD8+ T cells with this specificity was 25 apparent in two wells of the high DC concentration wells and three wells of the low DC concentration wells. Thus the NS5A Chimigen3 Protein is able to induce the generation of T cells specific to this NS5A immunodominant epitope and it is likely that T cells are present with specificities to other NS5A epitopes. 30 Example 3X. Results with NS3 Chimigen3 Protein 166 NS3 ChimigenTM Protein has been purified and characterized NS3 Chimigen T M Protein expressed in Sf9 cells was purified by Ni-NTA affinity chromatography followed by cation exchange chromatography. Purified samples were analyzed using 10% SDS-PAGE gels. After electrophoresis, gels were transferred to 5 nitrocellulose for Western blotting. The SDS-PAGE gel was stained with PageBlue and Western blots were developed with antibodies specific for different components of the NS3 ChimigenTM Protein. The purified protein appeared as a doublet at approximately 11OKDa and 12OKDa. Both species were detected by antibodies against the N-terminus (anti-6xHis), TBD (anti-Fc) and NS3 (polyclonal anti-NS3), which indicated that the 10 purified protein was intact. A qualitative assessment of glycosylation of purified NS3 ChimigenTM Protein was performed using the Pro-Q@ Emerald 300 Glycoprotein Gel and Blot Stain Kit. After electrophoresis of purified protein on an 8% SDS polyacrylamide gel, the gel was stained using the manufacturer's protocol and scanned under illumination with UV. 15 Since purified protein is a doublet, the difference in molecular weight is presumed to be due to the different levels of glycosylation. NS3 Chimigen T M Protein binds to immature DCs NS3 ChimigenTM Protein was examined for its ability to bind to immature DCs. 20 The cells were incubated in the presence and absence of various concentrations ofNS3 ChimigenTM Protein for 1 hr at 4'C. The bound protein was detected with biotinylated anti-mouse IgG1 mAb and SA-PE-Cy5. The percentage of cells binding the Chimigen3 protein (% positive cells) and the relative amount of bound protein (MFI) was determined by FFC. With NS3 Chimigen T M Protein at 4-55 tg/mL, most DCs were 25 positive for binding, and there was a dose-dependent increase in the amount of bound protein (Figs. 35X and 36X). Binding of the protein was not much greater at 22 [g/mL compared with 55 [g/mL indicating that the binding to immature DCs was saturable . The high MFI of binding observed indicated that NS3 Chimigen T M Protein binds very effectively and at high levels to immature DCs. The binding at 4'C was saturable, 30 indicating that it is receptor-mediated. The binding was also very rapid with bound 167 protein detected after 5 min of incubation at 4'C with a MFI of binding approximately half of that observed after a 60 min incubation (data not shown). NS3 ChimigenTM Protein binds to specific receptors on immature DCs 5 By virtue of the presence of Fc fragment, NS3 ChimigenTM Protein is predicted to bind via its TBD region to CD32 (FcyRII) on immature DCs. In addition, due to its mannose glycosylation, NS3 ChimigenTM Protein is predicted to bind to C-type lectin receptors such as CD206 (MMR). To determine the specificity of binding of the protein, immature DCs were incubated with NS3 ChimigenTM Protein in the presence of blocking 10 anti-CD32 and/or anti-CD206 mAbs. Immature DCs were incubated with buffer control, or 5 [g/mL of isotype control mAb, anti-CD32 mAb, anti-CD206 mAb, or both anti-CD32 and anti-CD206 for 1 hr at 4'C before incubation with NS3 Chimigen T M Protein for 1 hr at 4'C. Bound NS3 Chimigen T M Protein was detected with biotinylated anti-6xHis mAb followed by SA-PE 15 Cy5. Isotype control mAbs (murine IgGI and IgG2b) did not inhibit binding compared with buffer control. However in comparison with buffer control, both anti-CD32 and anti-CD206 inhibited binding by approximately 70% and 60%, respectively (Fig. 36X). The addition of both blocking mAbs further inhibited NS3 Chimigen TM Protein binding, resulting in a 90% inhibition (Fig. 36X). Thus, the data indicates a role for both CD32 20 and CD206 in the binding of the NS3 Chimigen T M Protein binding to immature DCs. The binding ofNS3 Chimigen T M Protein was visualized by confocal microscopy. Immature DCs were incubated with NS3 ChimigenTM Protein at 4oC. Strong labeling on the cell surface compared with buffer only controls indicated that the ChimigenTM Protein bound to the surface of the cells, possibly to receptors. 25 To investigate internalization, immature DCs were incubated with the Chimigen T M Protein at 37'C for 1 hr. The cells showed little if any surface labeling (plasma membrane outlines) but instead showed a punctate labeling pattern often in the vicinity of the nucleus, indicating that the ChimigenTM Protein was internalized. 168 NS3 Chimigen T M Protein presentation by DCs results in both CD8+ and CD4+ T cell activation and proliferation The functional immune response to NS3A ChimigenTM Protein was assessed by 5 ex vivo antigen presentation assays (APAs). This assay can be used to measure various parameters of a functional T cell immune response after stimulation of T cells with antigen-loaded DCs. The assay consists of first generating immature DCs from PBMC derived monocytes by the addition of IL-4 and GM-CSF. The immature DCs are then incubated with vaccine candidate, carrier buffer (negative control), or TT (positive 10 control). Subsequently DCs are treated with cytokines to undergo maturation, washed, and incubated with autologous naYve T cells. For measuring cytokine production, the presence of cytotoxic granule components, and the generation ofNS3-specific T cells, the T cells are stimulated an additional two times allowing for the expansion of ChimigenTM Protein specific T cells. However a single stimulation would be expected to initiate 15 expansion from a naYve T cell population. Activation was assessed by measuring the early T cell activation marker CD69, and proliferation was measured by tracking the fluorescence of CFSE labeled T cells. Both CD69 expression and CFSE fluorescence were evaluated after 4 and 7 days of culture with antigen-loaded DCs. Preliminary analysis had indicated that the concentration of DCs and T cells in the 20 culture were important parameters in the determination of T cell immune response. Thus the APA was designed such that six different T cell: DC concentrations were assessed. Two sets of DC concentrations were used, a high concentration of 5 x 104 DCs/well and a low concentration of 1 x 104 DCs/well. After 48 hr of culture, the immature DCs were incubated with buffer (negative control), NS3 Chimigen T M Protein (3C) at 5 ptg/ml, TT 25 (positive control), or PBS. The DCs were then cultured for 8 hr and matured by the addition of poly IC, IL-I, IL-6, TNF-a, IFN-a, and IFN-y. After culture overnight (16 hr) DCs from the PBS control group were washed and examined for the expression of various mature DC markers. Both high and low concentration DCs expressed high levels of HLA-ABC (MHC class I), HLA-DR (MHC class II), CD86, CD80, and CD83. 30 Autologous T cells were isolated by a negative selection procedure and labeled with CFSE for determination of cell division. To 100 u1well of DCs in a 96-well plate, 169 100 u1well of T cells were added for concentrations per well of: 20 x 104, 5 x 104, or 2 x 104. For the high DC concentration wells (5 x 104 DC/well) the T cell to DC per well ratio combinations were: 20 x 104:5 x 104 (4:1), 5 x 104:5 x 104 (1:1), and 2 x 104:5 x 104 (0.4:1). For the low DC concentration wells (5 x 104 DC/well), the T cell to DC ratio 5 combinations per well were: 20 x 104:1 x 104 (20:1), 5 x 104:1 x 104 (5:1), and 2 x 104:1 x 104 (2:1). T cells were added to the DCs in the absence of any exogenous cytokines. As a control, at day 3 of culture PHA at 1 [g/mL was added to the T cells loaded onto the PBS treated DC group. Following 4 days of culture, half of the cell culture (100 [l) was harvested for 10 analysis of activation and proliferation. To the remaining half of the cell culture, 100 [d of fresh AIM V@/2.5% matched sera was added and the cells cultured for an additional 3 days. The expression of CD69 on the T cells at the 5 x 104 T cells/well:5 x 104 DCs/well ratio (1:1) after 4 and 7 days of culture is shown in Fig. 37X. The majority of PHA treated cells expressed CD69 regardless of the T cell: DC ratio. CD69 was detected in T 15 cells cultured with the high DC concentration but was barely detected in T cells cultured with the low DC concentration (data not shown). Compared with buffer control, antigen stimulated T cells expressed a higher level of CD69. NS3 Chimigen TM Protein-loaded DCs induced a higher percentage of CD69 expressing CD8+ T cells than CD4+ T cells at day 4. The percentage of cells that have undergone at least one division (CFSE ) after 20 four days of culture is shown in Fig. 38X. The results indicate that the T cells treated with PHA 24 hr earlier had begun to divide. CD8+ and CD4+ T cells treated with TT loaded DCs undergo detectable proliferation after 4 days of culture but this was only evident at the high DC concentration (data not shown). There was little detection of T cell proliferation in the vaccine candidate treated groups at day 4. Thus naive T cells are 25 activated by NS3 ChimigenTM Protein-loaded DCs on day 4 of culture as evidenced by expression of CD69 but these T cells have not yet divided. Following 7 days of culture, cells were harvested for analysis of activation and proliferation. The expression of CD69 on the T cells following 7 days of culture is shown in Fig. 37X. For Chimigen3 Protein stimulated T cells, CD69 was detected at 30 levels over 5% in T cells cultured with the high DC concentration, but was barely detected in T cells cultured with the low DC concentration (data not shown). Thus the 170 low DC concentration (1x 104 DC/well) was not sufficient for antigen-specific T cell activation. The expression of CD69 was reduced for the recall TT response at day 7. In contrast to d4 T cells, NS3 ChimigenTM Protein-loaded DCs induced a higher percentage of CD69 expressing CD4+ T cells than CD8+ T cells at day 7. The percentage of CD69 5 expression of CD8+ and CD4+ T cells at day 7 was greater for the Chimigen3 Protein compared with the recall antigen TT. Thus the Chimigen3 Protein initially activates naYve CD8+ T cells, followed by CD4+ T cells. The percentage of cells that have undergone at least one division (CFSE ) after seven days of culture is shown in Fig. 38X. DCs loaded with Chimigen3 Protein or TT resulted in marked CD8+ and CD4+ T cell 10 proliferation after 7 days of culture and this was most evident at the high DC concentration (results not shown). NS3 ChimigenTM Protein presentation by DCs results in the generation of CD8+ and CD4+ T cells producing IFN-y and TNF-a 15 The functional immune response to NS3 ChimigenTM Protein was assessed by a three stimulation ex vivo APA. Immature DCs at either 4 x 104 DCs/well (high concentration) or at 2 x 104 DCs/well (low concentration) were loaded with control carrier buffer, PBS, TT (positive control), or NS3 ChimigenTM Protein. DCs were then matured and their phenotype evaluated. The DCs were assessed as mature as they 20 expressed high levels of MHC class I, MHC class II, CD86, CD80, and CD83. Autologous T cells were incubated with the matured antigen-loaded DCs at a ratio of 20 x 104 T cells/well: 4 x 104 DCs/well or 4 x 104 T cells/well: 2 x 104 DCs/well. The T cells were stimulated three times and T cell function evaluated 6 hr following the third stimulation by detection of the intracellular levels of the Th1 cytokines IFN-y and TNF-a. 25 In addition the extent of blast T cells was also assessed. The measurement of the percentage of blast T cells in a T cell population can be used as a gauge of the extent of T cell proliferation. Blast T cells are defined as those cells possessing a higher FSC and SSC light scatter then the resting lymphocytes in the lymphocyte gate as assessed by flow cytometry. The percentage of T cell blast in the 30 cultures after 14 days of culture is shown in Fig. 39X. NS3 ChimigenTM Protein was efficient at inducing T cell proliferation (blast cell production), with the 2:1 T cell:DC 171 ratio resulting in a lower background (buffer) T cell proliferative response compared with the 5:1 ratio. As a result, at the 2:1 T cell:DC ratio there was a marked difference in T cell proliferation upon stimulation with NS3 ChimigenTM Protein compared to buffer. The IFN-y response was measured at both the 5:1 and 2:1 T cell:DC ratios. The 5 data is shown as the responses of each well of the group and as an average of the three wells with the standard deviation of the mean (Fig. 40X). A comparison of the T cell IFN-y response showed a marked difference between the 5:1 and 2:1 T cell:DC ratios. With the higher DC concentration there was little evidence of a vaccine candidate induced IFN-y response over that of control buffer. However with the lower DC 10 concentration, very few T cells cultured with control buffer-loaded DCs produced IFN-y whereas a high percentage of T cells cultured with vaccine candidate-loaded DCs produced IFN-y . There was no reduction in IFN-y producing cells with the T cells stimulated with DCs that had been loaded with 2.5 [g/mL compared with 5 Tg/mL of NS3 Chimigen T M Protein. The percentage of T cells expressing IFN-y in the CD8+ and 15 CD4+ population was quantified and is shown in Fig. 41X. The percentage of CD8+ T cells expressing IFN-y was comparable for T cells stimulated with 2.5 [g/mL of vaccine candidate-loaded DCs compared with TT-loaded DCs. Likewise, there was also a high percentage of CD4+ T cells that expressed IFN-y upon stimulation with NS3 ChimigenTM Protein compared with control buffer. The percentage of CD4+ T cells expressing IFN-y 20 was comparable for T cells stimulated with vaccine candidate-loaded DCs compared with TT-loaded DCs. These results indicate that NS3 ChimigenTM Protein induces a marked IFN-y response in both CD8+ and CD4+ T cell populations and suggests that the molecule is processed by the DCs in both the MHC class I and class II pathways. Fig. 42X shows the percentage of T cells that have produced TNF-a as a result of 25 a 6 hr stimulation with antigen-loaded mature DCs. These results are similar to the IFN-y results. The TNF-a response was about equivalent or greater for the NS3 ChimigenTM Protein compared with TT. Stimulation with TT or NS3 Chimigen T M Protein-loaded DCs resulted in a higher percentage of CD4+ T cells expressing TNF-a compared with CD8+ T cells. 30 172 NS3 ChimigenTM Protein presentation by DCs results in the generation of CD8+ T cells expressing grB and pfn The ability of T cells to produce the cytotoxic granular proteins grB and pfn was also assessed by ex vivo APAs. Immature DCs were loaded with control buffer, with TT 5 (positive control), or varying concentrations ofNS3 ChimigenTM Protein and upon maturation were incubated with autologous T cells. GrB and pfn expression were detected by intracellular staining using anti-grB and anti-pfn mAbs, respectively. Fig. 43X shows the percentage of CD8+ T cells that expressed grB and pfn following three stimulations with antigen-loaded mature DCs. NS3 ChimigenTM Protein-loaded DCs 10 induced an increase in grB and pfn expression in CD8+ T cells compared to buffer control-treated DCs. These results indicated that NS3 ChimigenTM Vaccine induced the expression of grB and pfn in CD8+ T cells. This finding indicates the vaccine candidate is processed by the DCs in the MHC class I pathway for the effective presentation to T cells to result in their differentiation from naYve CD8+ T cells to cytotoxic T lymphocytes 15 (CTLs). NS3 ChimigenTM Protein presentation by mature DCs results in the generation and maintained activation of CD8+ and CD4+ T cells T cells were stimulated with antigen-loaded DCs three times in an APA. After 6 20 d of culture following the third stimulation the T cells were harvested and investigated by flow cytometry for the percentage of blast cells as a measure of proliferation and for the expression of the activation marker CD69. In addition, as a means to estimate absolute numbers of T cells recovered from culture, the number of gated cells falling in the lymphocyte gate (RI gate) based on FSC and SSC FFC analysis was determined. 25 The percentage of activated CD8+ and CD4+ T cells as assessed by CD69 expression is shown in Fig. 44X. There was an increased percentage of both CD4+ and CD8+ T cells expressing CD69 in T cells stimulated with Chimigen3 Protein-loaded DCs compared with buffer control. There was a marked difference in the recovery of T cells from TT and ChimigenTM Protein stimulated wells compared with buffer control (Fig. 30 45X). TT stimulated wells gave a higher T cell recovery than vaccine candidate stimulated wells. However the TT response is a recall response and thus the starting 173 population of T cells reactive specific for TT would be expected to be higher than that of the starting population of naYve T cells specific for NS3. On examination of the T cell blasts present in the cultures, notably the percentage of blast cells / proliferating cells was higher in the NS3 ChimigenTM Protein-containing cultures compared to the TT cultures. 5 There were very little blast cells / proliferating cells in the buffer control cultures. Thus, stimulation with the Chimigen3 Protein resulted in marked T cell activation and proliferation that is evident even six days following the third stimulation (day 20 of T cell culture). The NS3 ChimigenTM Protein is therefore very effective in the activation and expansion of both CD8+ and CD4+ T cells. 10 NS3 ChimigenTM Protein presentation by mature DCs induces the generation ofNS3 specific CD8+ T cells To evaluate the specificity of the immune response to NS3 ChimigenTM Protein, the percentage of T cells specific to two immunodominant NS3 epitopes in the context of 15 HLA-A2 was quantitated. This was determined by labeling T cells with NS3 peptide/HLA-A2 pentamers conjugated to PE. Naive T cells were stimulated three times with DCs loaded with different concentrations of NS3 ChimigenTM Protein and compared to the respective control DCs loaded without antigen (buffer) in an APA. T cells were harvested six days after the third stimulation and NS3-specific T cells or EBV-specific T 20 cells (control) were detected by tetramer labeling and analyzed by flow cytometry. One well of three of the buffer control group tested positive for EBV tetramer labeling (positive tetramer) in the CD8+ T cell population and no wells were positive for negative tetramer labeling (data not shown). As the T cells assessed were from the buffer control treated wells it would be expected that the number of EBV tetramer labeled T cells would 25 be relatively low. The percentage of CD8+ T cells labeling with an NS3 pentamer following the APA is shown in Fig. 46X. Loading DCs with NS3 Chimigen T M Protein resulted in the generation of T cells with specificity to NS3 epitopes. The marked expansion of CD8+ T cells with this specificity was apparent in four of six wells of the low DC concentration group. Thus the NS3 ChimigenTM Protein was able to induce the 30 generation of T cells specific to NS3 immunodominant epitopes and it is probable that T cells with specificities to other NS3 epitopes were also present. 174 Example 4X. Results with NS3-NS4B-NS5A multiantigen Chimigen3 Protein Expression of HCV HCV NS3-NS4B-NS5A Chimigen T M Protein 5 Time course of the expression of HCV Multi-antigen ChimigenTM Protein in SfO cells was analyzed by Western blot after SDS-PAGE. By considering both factors of expression and degradation of HCV Multi-antigen ChimigenTM Protein, the best condition for protein expression was determined as MOI of 1.5 for 36 hours after infection. 10 Purification of HCV NS3-NS4B-NS5A Chimigen TM Protein from clear lysate of Sf9 cell pellet Ni-NTA affinity chromatography As a first step of purification, HCV NS3-NS4B-NS5A Chimigen T M Protein was 15 captured on a Ni-NTA Superflow3 column. The protein, bound on Ni-NTA Superflow3 column, was analyzed by SDS-PAGE and by Western blot. The Western blot showed a dominant band of the ChimigenTM Protein, however silver staining of the nitrocellulose membrane showed additional bands of non-immunoreactive proteins. 20 HiTRap Q-XL ion exchange chromatography The proteins, captured by Ni-NTA Superflow column, were further separated by HiTrap Q XL 1 mL column chromatography. Protein was eluted in the flow-through fractions and the fractions eluted at salt concentration between 0.4 and 0.5 M NaCl. HCV NS3-NS4B-NS5A multi-antigen Chimigen TM Protein, bound on the column, had 25 less contaminant than the protein in the flow-through fractions. At least 6 non immunoreactive protein bands were seen by silver staining. Superdex 200 chromatography Proteins in the lysate were fractionated by a Superdex 200 column. The HCV 30 NS3-NS4B-NS5A-multi-antigen Chimigen T M Protein was eluted in the first peak. Western blot and silver staining of the fraction were performed. The protein is shown as 175 a dominant band on silver-staining; however numerous bands of contaminants were also visible. The results of western blot suggest the aggregation of the protein during purification 5 Hydrophobic interaction chromatography on Phenyl-650C Toyopearl Cell lysate containing the HCV NS3-NS4B-NS5A multi-antigen Chimigen T M Protein was loaded on Phenyl-650C Toyopearl@ and eluted in both unabsorbed and absorbed fractions. Western blot and silver staining of the fractions were performed. In both fractions, HCV Multi-antigen ChimigenTM Protein was seen as a dominant band on 10 Western blot and silver staining of the nitrocellulose membrane. Example 5X. Results with HCV Core Chimigen3 Protein 15 HCV Core ChimigenTM Protein has been purified and characterized The HCV Core Chimigen3 Protein was purified by Ni chelation chromatography (Ni-NTA superflow) under denaturing conditions followed by cation exchange chromatography (CM sepharose). Purified protein was analyzed on 12% SDS-PAGE gels. The major band was the fusion protein (~ 55 KDa), the second band noticed at 20 28kDa is most likely a degradation product. After separation on a 12% SDS-PAGE gel, the purified proteins were electroblotted to nitrocellulose membranes. Western blotting was performed with anti-6xHis-HRP conjugated antibody, anti-Fc specific-HRP conjugated antibody and anti-HCV core antibody with anti-Fab specific-HRP conjugated antibody as the secondary antibody. Bound antibodies were detected by 25 chemiluminescence. Binding of the antibodies to the blot indicated that the purified HCV core-TBD ha an intact N-terminus, core and TBD portions. In addition, the lower molecular weight band was detected by all 3 antibodies, which indicated that it was a protein derived from the full length HCV core ChimigenTM molecule and was likely the result of degradation. 30 HCV Core Chimigen T M Protein binds to immature DCs 176 HCV Core ChimigenTM vaccine was examined for its ability to bind to immature DCs. The cells were incubated in the presence and absence of various concentrations of HCV Core Chimigen T M Protein for 1 hr at 4'C and binding was detected either by FFC or by confocal microscopy. For FFC analysis, bound HCV Core ChimigenTM Protein was 5 detected with a biotinylated anti-mouse IgGI mAb and SA-PE-Cy5. The percentage of cells binding HCV Core Chimigen T M Protein (% positive cells) and the relative amount of bound Protein (MFI) was determined by FFC. With HCV Core ChimigenTM Protein at 5-40 pg/mL, approximately 100% of the cells were positive for binding, and there was a dose-dependent increase in the amount of bound HCV Core 10 Chimigen T M Protein (Fig. 47X). The high MFI of binding observed suggested that HCV Core ChimigenTM Protein binds very effectively and at high levels to immature DCs. The binding of HCV Core Chimigen T M Protein was studied using confocal microscopy as well. The binding was detected with a FITC conjugated goat anti-mouse IgG. The blue fluorescent dye DAPI was used to image the nucleus. The confocal image 15 and the corresponding light image showed that the protein binds to the membrane of immature DCs after a 1 hr pulse at 4'C. HCV Core ChimigenTM Protein binds to specific receptors on immature DCs By virtue of the presence of Fc fragment, HCV Core ChimigenTM Protein is 20 predicted to be able to bind via its TBD region to CD32 (FcyRII) on immature DCs. In addition, due to its mannose glycosylation, HCV Core ChimigenTM Protein is also predicted to bind to C-type lectin receptors such as CD206 (MMR). To determine the specificity of binding of HCV Core ChimigenTM Protein, immature DCs were incubated with HCV Core Chimigen T M Protein in the presence of blocking mAbs specific to CD32 25 or CD206. The binding was also examined in the presence of competing ligands, murine IgG Fc fragments for Fcy receptors, and mannosylated BSA (mBSA) for C-type lectin receptors. Immature DCs were incubated with PBS (buffer control), murine IgG Fc fragments (500 pg/mL), CD32 mAb (200 pg/mL), mannosylated BSA (500 pg/mL), or 30 anti-CD206 (200 pg/mL) for 1 hr at 4'C before incubation with HCV Core Chimigen T M Protein (30 pg/mL) for 1 hr at 4'C. The bound Protein was detected either with 177 biotinylated anti-mouse IgGI mAb or biotinylated anti-HCV core mAb followed by SA PE-Cy5. The relative amount of bound HCV Core Chimigen T M Protein (MFI) was determined by FFC. The results from the binding and inhibition studies showed that HCV Core ChimigenTM Protein bound to Fcy receptors such as CD32 and C-type lectin 5 receptors such as CD206 (Fig. 48X). HCV Core ChimigenTM Protein presentation by DCs results in increased intracellular IFN-y levels in CD8+ and CD4+ T cells The functional immune response to HCV Core ChimigenTM Protein was assessed 10 by ex vivo antigen presentation assays. Immature DCs were loaded with PBS (buffer control), with tetanus toxoid (positive control), or varying concentrations of HCV Core ChimigenTM Protein. Upon maturation of the DCs, they were incubated with autologous T cells. T cell function was evaluated by detection of the intracellular levels of the Th1 cytokine IFN-y. The CD3 and CD8 phenotype of the cells was also determined by FFC. 15 Figs. 49X A and X B show the percentage of CD8+ and CD4+ T cells, respectively, that express IFN-y 12 hr following the third stimulation with antigen-loaded mature DCs. Tetanus toxoid was used as the positive control for effective antigen presentation. HCV Core ChimigenTM Protein-loaded DCs induced a marked increase in IFN-y expression in CD8+ T cells compared to the no antigen control. There was an increase in the expression 20 of IFN-y in the CD4+ T cell population upon stimulation with HCV Core Chimigen T M Protein-loaded DCs. These results indicate that HCV Core ChimigenTM Protein induces an IFN-y response in both CD8+ and CD4+ T cell populations and suggests that the molecule is processed by the DCs in both the MHC class I and class II pathways. 25 HCV Core ChimigenTM Protein presentation by mature DCs induces the generation of HCV Core-specific CD8+ T cells To evaluate the specificity of the immune response to HCV core, the percentage of T cells specific to an immunodominant HCV core epitope in the context of HLA-B7 was quantitated. This was acheived by labeling T cells with an HCV core peptide/HLA 30 B7 tetramer conjugated to PE. In addition, T cells were labeled with CD4 and CD8 specific mAbs. 178 HCV core naive T cells were stimulated three times with DCs loaded with different concentrations of HCV Core ChimigenrM Protein and compared to the respective control DCs loaded with no antigen, with tetanus toxoid, or with TBD. T cells were harvested 5 days after the third stimulation and HCV core-specific T cells detected 5 by two-dimensional FFC. The two dimensional FFC dot plot in Fig. 50X shows that T cells incubated with DCs loaded with HCV Core Chimigen T M Protein showed a small increase in the core tetramer positive T cells. Example lZ: Plasmodium Materials - Consumables The pFastBac-HTa cloning vector, E coli strain TOPlO', insect cell line Sf9, 10 Cellfectin reagent, phosphate buffered saline (PBS), Pfx50 DNA polymerase, Taq DNA polymerase, FBS, Bluo-gal, isopropyl- [I -D-thiogalactopyranoside (IPTG), fetal bovine serum (FBS) and gentamicin were purchased from Invitrogen T M (Carlsbad, CA, USA). The antibiotics kanamycin, and ampicillin were purchased from Sigma T M (Mississauga, ON). Restriction enzymes AvalI, RsrII, EcoR I, SalI, Spel, NotI, XbaI SphlI, and HindIl 15 were purchased from New England Biolabs TM (Ipswich, MA, USA). Insect cell growth TM and expression media ESF 921 was purchased from Expression SystemsT (Woodland, CA, USA). Viral stocks were titered using the Expression SystemsTM Baculovirus TM Titering Assay. Mouse IgG 2 a-PE (BD Biosciences , San Diego, CA, USA) was diluted 1:10 and used as an isotype control. Baculovirus titer was determined using FACS 20 acquisition and analysis. A BD Biosciences T M FACSCalibur T M (four-color, dual-laser) was used for acquisition and CELLQuestTM Pro software (BD Biosciences</div>
    <div class="description-paragraph style-scope patent-text">TM</div>
    <div class="description-paragraph style-scope patent-text">) was used to analyze the data. A Microsoft T M Excel spreadsheet was provided by Expression SystemsTM to input data and determine the viral titer based on a standard curve. Protein purifications were performed on HisTrap FF columns by FPLC on an 25 AKTA explorer 100 system (GE Healthcare T M , Piscataway, NJ, USA). For protein electrophoresis, TGX SDS PAGE precast gels were purchased from Bio-Rad TM (Hercules, CA, USA). PageBlue stain, 5x loading buffer, PageRuler T M pre-stained protein TTM ladder and 20x reducing agent are from FermentasT (Burlington, ON, Canada). Hybond ECL nitrocellulose and ECL Western Detection kit (GE Healthcare TM) was used for 179 Western blotting.Anti-mouse IgG (Fc specific)-horseradish peroxidase (HRP) conjugated antibody, anti-mouse (Fab specific)-HRP conjugated secondary antibody and goat-anti rabbit-HRP conjugated secondary antibody were purchased from Sigma T M (St. Louis, MO, USA). The anti-HBV core mouse monoclonal antibody is from Abcam Tm 5 (Cambridge, MA, USA). The 6xHis-HRP conjugated monoclonal antibody was purchased from Clontech T M (Palo Alto, CA, USA). The detergent Tween-20 and Con A HRP conjugate were purchased from Sigma T M . Slide-a-lyzer cassettes and Micro BCA assay kit were purchased from Pierce T M (Rockford, IL, USA). Wave Bioreactor System2/10EH and Wavebag TM 1OL were purchased from GE Healthcare T M . 10 Leukapheresis samples from healthy donors were purchased from SeraCare Life SciencesTM (Oceanside, CA). Dynal Dynabeads TM for T cell negative isolation were purchased from Invitrogen T. AIM-V medium containing L-glutamine, streptomycin TM sulfate (50 pg/mL), and gentamicin sulfate (10 mg/mL) was obtained from Invitrogen Matched donor plasma was obtained from the plasma fraction after centrifugation of 15 Ficoll-Hypaque blood preparations. Plasma, at 50% in AIM-V was heat inactivated, aliquoted, and stored at -20'C. Conjugated mAbs with the following specificities were TM obtained from BD Biosciences : CD64-fluorescein isothiocyanate (FITC), CD32-R phycoerythrin (PE), CD16-PE, CD206-FITC, CD206-PE-Cy5, CD205-Alexa Fluor 647, CD209-PE, CD80-PE, CD86-FITC, CD83-PE, CD40-FITC, CD1Ic-PE, CD14-FITC, 20 CD19-FITC, CD3-PE, CD3-allophycocyanin (APC), CD8-PE-Cy5, CD4-FITC, HLA ABC-FITC, HLA-DR-PE, IFN-y-PE, TNF-a-PE, granzyme B-FITC, and mouse IgG1 biotin. Anti-HLA-A2-Alexa 488 (Serotec), anti-CD 1 9-PECy5, anti-CD8-PerCP, anti CD4-APC (BD), anti-CD8-FITC (ProImmune). Fluorotag (ProImmune). Biotinylated anti-6xHis was obtained from Qiagen (Mississauga, Ontario). Murine isotype mAbs TM 25 and SA-PE-Cy5 were obtained from BD Biosciences . The rabbit anti-AMA-i polyclonal antibody was a generous gift from Dr. Michael J. Blackman in the Division of Parasitology, National Institute for Medical Research, London, UK as described previously (Bannister et al., 2003). This antibody is not conjugated and is detected using TM the goat anti-rabbit IgG-HRP (R&amp;D Systems , Minneapolis, MN). 180 Example 2Z: Materials - DNA sequences Plasmodium falciparum DNA P falciparum DNA sequences were obtained from GenBank T M . All nucleotides 5 encoding start/stop codons were removed for expression of Pfalciparum genes as fusion proteins in Bac to Bac* Baculovirus Expression System. Flanking restriction enzyme digest sites were added to each of the four genes for sequential cloning into Bac to Bac* expression system. The resulting DNA sequences were sent to GenScript TM (Piscataway, NJ). Submitted DNA sequences were codon optimized for expression in insect cells, 10 synthesized, sequenced and cloned into pUC57 plasmids at the EcoRV restriction site. GenBankT M accession numbers and exact sequence information for each P falciparum gene is described in detail in the next section. Referring to Figure 2Z, a schematic representation of P falciparum gene 15 engineering as described, is depicted. Native' represents GenBank sequence and 'Chimigen' represents engineered sequence. Circumsporozoite protein (CSP) From the original GenBank T M sequence (accession #: K02194) the DNA encoding 20 a signal peptide (nucleotide 1-75) as well as a GPI anchor (nucleotide 1272-1313) were removed (Figure 2Z). Flanking 5' EcoRI and 3' SalI restriction sites were added to the sequences prior to synthesis. The final DNA sequence sent to GenscriptTM was as follows (CSP origin nucleotides 126-1271). 25 Referring to Figure 3Z, an insect cell codon optimized CSP sequence (blue) is compared to original sequence (black). 5'- EcoRI and 3'- Sail restriction sites are highlighted in yellow. Apical membrane antigen-I (AMA-1) 30 From the original GenBank T M sequence of AMA-I (accession #: XM_001347979) the DNA encoding a signal peptide (nucleotide 1-39) as well as the transmembrane 181 domain (nucleotide 1645-1704) was removed (Figure 2Z). Flanking 5'-SalI and 3'-Spel restriction sites were added prior to synthesis. The final DNA sequence sent to GenScript TM was as follows (AMA-I origin nucleotides 40-1644): 5 Referring to Figure 4Z, an insect cell codon optimized AMA-i sequence (blue) is compared to original sequence (black). 5'- SalI and 3'- Spel restriction sites are highlighted in yellow. Merozoite surface protein-I (MSP-1) and 42 kDa MSP-I fragment (MSP-1 42 ) 10 MSP-I protein is a 190 kDa protein expressed on the membrane of mature merozoites (Koussis et al., 2009; Woehlbier et al., 2006). MSP-I is cleaved by P falciparum subtilisin-like proteases (PfSUBI) into smaller non-covalently associated subunits (Koussis et al., 2009; Woehlbier et al., 2006). These are MSP-1 83 , MSP-1 3 0 , MSP-1 3 8 and MSP-1 4 2 . MSP-1 42 is subsequently cleaved into MSP-1 33 and MSP-1 19 by 15 another protease PfSUB2 (Koussis et al., 2009). Cleavage by PfSUB2 releases the MSP 1 83 /MSP-1 30 /MSP-1 38 /MSP-1 33 complex from the merozoite surface, leaving behind the MSP-119 which remains GPI anchored to the merozoite. From the original GenBank T M sequence of MSP-I (accession #: AB300615), DNA encoding a signal peptide (nucleotides 1-45) as well as a GPI anchor (nucleotides 5041-5085) were removed 20 (Figure 2Z). Flanking 5'- NotI and 3'- XbaI restriction sites were added prior to synthesis. Generation of MSP-1 4 2 was done by touchdown PCR as described herein. Referring to Figure 5Z, an insect cell codon optimized MSP-i sequence (blue) is compared to original GenBank sequence (black). Flanking 5'- NotI and 3'- XbaI 25 restriction sites are highlighted in yellow. DNA encoding MSP-1 42 fragment is underlined. PfSUBI cleavage site generating MSP-1 42 is indicated by a highlighted arrow (j)(Koussis et al., 2009). MSP-1 42 was generated by Touchdown PCR as described herein. 182 Liver stage antigen-I (LSA-1) The full length LSA-1 sequence (accession #: X56203 nucleotides 79-5805) encodes a 230 kDa protein composed of conserved N- and C-teminal sequences which flank 86 repeats of the 17 aa sequence EQQSDLEQERLAKEKLQ . A few minor repeats 5 have an arginine (R) at position 4 rather than a serine (S) or an arginine (R) at position 11 rather than a leucine (L). To reduce the size of the encoded LSA-1, we engineered a truncated version of the protein in which we removed the nucleotides encoding 66 of these repeats (nucleotides 993-4358) from the middle of the sequence of the full length GenBank LSA-1. The resulting gene encodes a 94 kDa LSA-1 protein containing only 10 20 of these 17 aa repeats with all four repeat variants present in the construct. We called this LSA-1 variant LSA-1 20 (Figure 2Z). Flanking 5'- Spel and 3'-NotI restriction sites were added to the sequence prior to synthesis. Referring to Figure 6Z, an insect cell codon optimized LSA-120 sequence (blue) is 15 compared to original GenBank sequence (black). Splice site for LSA-1 20 is indicated by (1). Flanking 5'- Spel and 3'- NotI restriction enzyme sites are highlighted in yellow. pFastBac-HTa-gp64 parent plasmid The pFastBac-HTa/gp64 parent plasmid was contructed by adding the gp64 signal 20 sequence from Autographica caifornica nuclear polyhyedrosis virus (AcNPV) to the pFastBac-HTa cloning vector from the Bac-to-Bac* Baculovirus Expression System TM (Invitrogen , Mississauga, ON). Briefly, two oligonucleotides were synthesized as follows: (5' GCATGGTCCATGGTAAGCGCTATTGTTTTATATGTGCTTTTGGCGGCGGCGGC 25 GCATTCTGCCTTTGCGGATCTGCAGGTACGGTCCGATGC-3') and (5' GCATCGGACCGTACCTGCAGATCCGCAAAGGCAGAATGCGCCGCCGCCGCCA AAAGCACATATAAAACAATAGCGCTTACCATGGACCATGC-3'). These oligonucleotides encode for 5'- AvaIl and 3'- RsrII restriction sites (underlined) and a new start codon (highlighted). Oligonucleotides were mixed and annealed by heating to 30 95"C in annealing buffer (10 mM Tris, pH 8, 50 mM NaCl, 1 mM EDTA) for 5 min, then slow cooling to room temperature over 45 minutes. After annealing, digestion with 183 AvalI and RsrII was performed, and the fragment was gel purified on a 2% agarose gel using the Qiagen gel extraction kit (Qiagen</div>
    <div class="description-paragraph style-scope patent-text">TM</div>
    <div class="description-paragraph style-scope patent-text">, Mississauga, ON). The purified fragment was ligated into the RsrII digested pFastBac-HTa empty plasmid. Insertion of the above oligonucleotide at the RsrII site places the gp64 signal sequence just upstream of the 5 6xHis tag, to generate the pFastBac-HTa/gp64 parent plasmid. Referring to Figure 7Z, pFastBac-HTa-gp64 parent plasmid is depcited. Signal sequence for gp64 is highlighted. New start codon is highlighted. Old start codon is also highlighted. 10 Terminal binding domain (TBD) DNA The TBD was designed from the mouse IgGI DNA sequence encoding amino acids of CH1-Hinge-CH2-CH3 region which was previously generated from mRNA isolated from the hybridoma 2C12. This hybridoma produces murine HBsAg mAb and was 15 licensed from the L.J. Tyrrell laboratory through the University of Alberta. The TBD DNA containing flanking 5' - NotI and 3'- HindIll restriction sites was originally inserted into the pFastBac-HTa/gp64 described herein by digestion with NotI and HindIl to generate pFastBac-HTa/gp64/TBD. This archived construct spliced out NspV, XbaI, PstI, XhoI, Sphl and KpnI restriction sites and thus lacked sufficient restriction sites for P 20 falciparum gene insertion. To circumvent this problem, 1 pL of this archived construct was used as a template to amplify TBD with flanking 5' Sphl and 3' HindIll restriction sites by touchdown PCR in a 50 pL PCR reaction containing the following reagents: 5 pL lOx Pfx50 buffer, 1.5 pL 10 dNTP (10 mM), 1.5 pL TBD sense primer (5' TATATAGCATGCCAAGGCGGCGGATCCG-3'), 1.5 pL TBD antisense primer (5' 25 ATATATAAGCTTTTGCAGCCCAGGAGAG-3') and 1 pL Pfx50 (Invitrogen). Amplification was performed on an MJ mini thermal cycler (Bio-Rad</div>
    <div class="description-paragraph style-scope patent-text">M</div>
    <div class="description-paragraph style-scope patent-text">, Mississauga, ON) using an initial denaturation of 94'C for 2 min followed by 20 cycles of 94'C for 15s, melting temperature (Tm) of 68'C to 58'C (-0.5 0 C/cycle) and extension at 68 'C for 30s. A final extension step at 68'C for 5 min was performed to complete elongation of 30 all products. The resulting 738 bp PCR product (Figure 8Z) was excised from a 1% agarose gel and purified using the gel extraction kit (Qiagen</div>
    <div class="description-paragraph style-scope patent-text">TM</div>
    <div class="description-paragraph style-scope patent-text">). The purified PCR 184 product was digested with Sphl (NEBM) and gel purified again on a 1% agarose gel using the gel extraction kit (Qiagen</div>
    <div class="description-paragraph style-scope patent-text">TM</div>
    <div class="description-paragraph style-scope patent-text">). After the second purification, Sphl digested TBD DNA was stored at -20'C until needed. 5 Referring to Figure 8Z, Terminal binding domain (TBD) DNA sequence is depicted with new flanking 5'- Sphl and 3'- HindIll restriction sites generated by touchdown PCR. Restriction sites are highlighted. Hepatitis B virus (HBV) Core DNA 10 The DNA encoding the HBV core antigen was generated from the plasmid pAltHBV991 template using PCR methodology. The primers used were: (sense) (5' TGCGCTACCATGGACATTGACCCTTATAAAG-3') which contains a restriction enzyme NCo I site, and the 3' primer (antisense) 5' TGTCATTCTGCGGCCGCGAACATTGAGATTCCCGAGATTGAG-3' containing the 15 restriction enzyme site Not I. Amplified DNA was digested with NcoI/Not I and ligated with pFastBac-HTa vector plasmid to generate the donor plasmid pFastBacHTa HBV core for the expression of 6xHis tag-rTEV protease cleavage site-HBV core protein. As for TBD in some embodiments, this construct lacked sufficient restriction sites for insertion of Pfalciparum genes. Thus, as herein, using 1 pL of this archived construct as 20 template, HBV Core DNA with flanking 5' XbaI and 3' Sphl restriction sites was amplified by touchdown PCR in a 50 pL PCR reaction containing the following reagents: 5 pL lOx Pfx50 buffer, 1.5 pL 10 dNTP (10 mM), 1.5 pL HBV Core sense primer (5' TGTCCAGCTCTAGACACATTGACCCTTATAAAG-3'), 1.5 pL HBV Core antisense primer (5'- ATATATGCATGCCGAACATTGAGATTCCCGAG-3') and 1 PL Pfx50 TM TM 25 (Invitrogen ). Amplification was performed on an MJ mini thermal cycler (Bio-Rad Mississauga, ON) using an initial denaturation of 94'C for 2 min followed by 20 cycles of 94'C for 15s, Tm of 68'C to 58'C (-0.5 0 C/cycle) and extension at 68 'C for 30s. A final extension step at 68'C for 5 min was performed to complete elongation of all products. The resulting 581 bp PCR product was excised from a 1% agarose gel and 30 purified using the gel extraction kit (Qiagen</div>
    <div class="description-paragraph style-scope patent-text">TM</div>
    <div class="description-paragraph style-scope patent-text">). The purified PCR product was digested 185 with Sphl (NEBTM) and gel purified again on a 1% agarose gel using the gel extraction kit (Qiagen</div>
    <div class="description-paragraph style-scope patent-text">TM</div>
    <div class="description-paragraph style-scope patent-text">). After the second purification, Sphl digested HBV core DNA was stored at -20'C until needed. 5 Referring to Figure 9Z, DNA sequence encoding HBV Core protein is depicted. Flanking 5'- XbaI and 3'- Sphl restriction sites added by touchdown PCR are highlighted. Example 3Z: Cloning into pFastbac HTa Plasmid Restriction digest and DNA ligation 10 Whenever possible, DNA was double digested in compatible buffer. In some cases, DNA was digested with only one restriction enzyme, or sequentially digested by single digestion with first enzyme in 50 pL volume followed by addition of 0.1 volumes of 3M NaOAC (pH 5.2) and overnight precipitation in 2.5 volumes of absolute ethanol (ETOH). DNA was pelleted at 15 000 x g for 30 min followed by 1 wash in 1 mL 70% ETOH 15 followed by 15 000 x g spin for 5 min. Supernatant was removed and DNA pellet allowed to air dry for 10 min at 22'C. DNA was resuspended in 10 PL H 2 0 and digested with second enzyme in new digestion buffer in a 20 pL reaction volume. Digestion temperatures were as specified by NEB and digestion times ranged from lh to overnight. Restriction enzymes were inactivated as per manufacturer recommendations. 20 DNA ligations were performed using T4 DNA ligase (NEBTM), in 10IPL volume overnight at 16'C. Ligation reactions (2 pL) were transformed into Top10 E. coli and grown for 1 h at 37'C in Luria Bertani broth (LB) at 225 rpm on a SI-600 Lab Companion shaker (Jeiotech T M Inc., Woburn, MA) in the absence of antibiotics. Following outgrowth period, 900 pL LB was added to the cells and 100 pL cells were 25 plated on LB agar plates containing 100 pg/mL ampicillin. Plates were grown overnight at 37 0 C. Resulting colonies were screened by digestion with appropriate restriction enzymes as well as PCR analysis using gene specific primers. DNA sequencing to verify accuracy and direction of insertion was done at the University of Alberta in The Applied Genomics Center (TAGC)</div>
    <div class="description-paragraph style-scope patent-text">TM</div>
    <div class="description-paragraph style-scope patent-text">. 186 pFastBac HTa/gp64/HBV Core/TBD Archived Sphl digested, gel purified TBD (4 pL) and HBV Core (4 pL) PCR products generated herein were ligated in a 10 pL volume using 1 pL T4 DNA ligase 5 (NEB) overnight at 16'C. Of this ligation reaction, 1 pL was used as a template for touchdown PCR in a 50 pL PCR reaction containing the following reagents: 5 PL lOx Pfx50 buffer, 1.5 pL 10 dNTP (10 mM), 1.5 pL HBV Core sense primer (5' TGTCCAGCTCTAGACACATTGACCCTTATAAAG-3'), 1.5 pL TBD antisense primer (5'-ATATATAAGCTTTTGCAGCCCAGGAGAG-3') and 1 pL Pfx50 10 (Invitrogen TM). Amplification was performed on an MJ mini thermal cycler (Bio-Rad T, Mississauga, ON) using an initial denaturation of 94'C for 2 min followed by 20 cycles of 94'C for 15s, Tm of 68'C to 58'C (-0.5 0 C/cycle) and extension at 68 'C for 30s. A final extension step at 68'C for 5 min was performed to complete elongation of all products. The resulting 1301 bp PCR product was excised from a 1% agarose gel and 15 purified using the gel extraction kit (Qiagen</div>
    <div class="description-paragraph style-scope patent-text">TM</div>
    <div class="description-paragraph style-scope patent-text">). Purified PCR product was digested with XbaI and HindlIl in a 50 pL volume and gel purified once again with gel extraction kit (Qiagen</div>
    <div class="description-paragraph style-scope patent-text">TM</div>
    <div class="description-paragraph style-scope patent-text">). HBV Core/TBD purified product (6.5 pL) was ligated into 0.5 PL XbaI and HindlIl digested pFastBac-HTa/gp64 parent plasmid. This reaction gave pFastBac HTa/gp64/HBV core/TBD (Figure 10Z) which is not in frame with the gp64 start codon 20 but has original pFastBac-HTa multiple cloning site (Ehel to XbaI) for downstream P falciparum gene insertion. Referring to Figure 1OZ, a pFastBac HTa/gp64/HBV Core/TBD construct is depicted. HBV Core sequence is highlighted. TBD sequence is highlighted. 25 pFastBac HTa/gp64/MSP-1 42 /HBV Core/TBD Using 1 pL of a 1/10 dilution of pUC57-MSP-1 plasmid (GenScript</div>
    <div class="description-paragraph style-scope patent-text">TM</div>
    <div class="description-paragraph style-scope patent-text">), MSP-1 4 2 PCR product with flanking 5' NotI and 3' XbaI restriction sites was generated by touchdown PCR in a 50 pL PCR reaction containing the following reagents: 5 PL lOx 30 Pfx50 buffer, 1.5 pL 10 dNTP (10 mM), 1.5 pL MSP-1 4 2 sense primer (5' 187 TATATAGCGGCCGCAGCCATCTCTGTGA-3'), 1.5 pL MSP-1 4 2 antisense primer (5'- TATATATCTAGAACCGAGGAAGTTTGAG-3') and 1 pL Pfx50 (Invitrogen). Amplification was performed by touchdown PCR on an MJ mini thermal cycler (Bio RadM, Mississauga, ON) using an initial denaturation of 94'C for 2 min followed by 20 5 cycles of 94'C for 15s, Tm of 68'C to 58'C (-0.5 0 C/cycle) and extension at 68 'C for 30s. A final extension step at 68'C for 5 min was performed to complete elongation of all products. The resulting 1161 bp PCR product was excised from a 1% agarose gel and purified using the gel extraction kit (Qiagen</div>
    <div class="description-paragraph style-scope patent-text">TM</div>
    <div class="description-paragraph style-scope patent-text">). The purified PCR product was double digested with NotI and XbaI (NEBTM) then gel purified again on a 1% agarose gel using 10 the gel extraction kit (Qiagen</div>
    <div class="description-paragraph style-scope patent-text">TM</div>
    <div class="description-paragraph style-scope patent-text">). After the second purification, 10 PL of gel purified NotI/XbaI digested MSP-1 4 2 was ligated into 0.5 pL NotI/XbaI double digested, gel purified pFastBac-HTa-gp64/HBV Core/TBD from (Chimigen®) or pFastBac-HTa/gp64 (control antigen) as described above. These constructs are not in frame with the start codon generated herein. 15 pFastBac HTa/gp64/AMA-1/MSP-1 42 /HBV Core/TBD In separate tubes, 1 pL of plasmid generated herein and pUC57-AMA-1 plasmid (GenscriptTM) were sequentially digested with SalI and Spel as described herein. Linearized pFastBac HTa/gp64/MSP-1 42 /HBV Core/TBD plasmid DNA and 1773 bp 20 AMA-1 DNA fragment were purified on a 1% agarose gel by gel extraction kit (Qiagen</div>
    <div class="description-paragraph style-scope patent-text">TM</div>
    <div class="description-paragraph style-scope patent-text">). Purified Sall/Spel digested AMA-1 (10 pL) was ligated into (0.5 PL) Sall/Spel digested pFastBac-HTa/gp64/MSP-1 42 /HBV Core/TBD (Chimigen*) or pFastBac-HTa/gp64/MSP-1 42 (control protein) as described above. These constructs are not in frame with the start codon generated herein. 25 pFastBac HTa/gp64/CSP/AMA-1/MSP-1 42 /HBV Core/TBD (3 antigen Chimigen®) In separate tubes, 1 pL of plasmids from herein and pUC57-CSP plasmids (GenScriptTM) were double digested with EcoRI and SalI and purified on a 1% agarose gel by gel extraction kit (Qiagen</div>
    <div class="description-paragraph style-scope patent-text">TM</div>
    <div class="description-paragraph style-scope patent-text">). Purified EcoRI/Sall digested CSP fragment (5 PL) 30 was ligated into 0.5 pL EcoRI/Sall digested pFastBac-HTa/gp64/AMA-1/MSP-1 42 /HBV Core/TBD (Chimigen®) or pFastBac-HTa/gp64/AMA-1/MSP-1 42 (control) as described 188 above. The constructs generated in this section are in frame with the gp64 start codon generated herein and were sent for sequencing at TAGC T M (University of Alberta). Once the correct sequence was verified, the constructs were used for generation of the 3 gene Chimigen* and control bacmids. 5 Referring to Figure 1IZ, an embodiment of a three antigen Chimigen* Malaria Multi-antigen Vaccine DNA sequence inserted into pFastBac HTa, is depicted. The 6xHis tag is highlighted. 10 pFastBac HTa-gp64/CSP/AMA-1/LSA-12 0 /MSP-1 42 /HBV Core/TBD (4 antigen Chimigen*) In separate tubes, 1 pL of plasmids from herein and pUC57-LSA-1 2 a plasmids (GenScriptTM) were double digested with Spel/NotI and purified on a 1% agarose by gel extraction kit (Qiagen</div>
    <div class="description-paragraph style-scope patent-text">TM</div>
    <div class="description-paragraph style-scope patent-text">). Purified Spel/NotI digested LSA-1 2 0 DNA fragment (5 PL) 15 was ligated into 0.5 pL Spel/NotI digested pFastBac-HTa/gp64/AMA-1/MSP-1 42 /HBV Core/TBD (Chimigen®) or pFastBac-HTa/gp64/AMA-1/MSP-1 42 (control) as described above. These 3 antigen constructs are in frame with the gp64 start codon generated herein and were sent for sequencing at TAGC T M (University of Alberta). Once sequence and reading frame were verified, the constructs were used for generation or the 4 gene 20 Chimigen® and control bacmids. Referring Figure 12Z, an embodiment of a four antigen Chimigen® Malaria Multi antigen Vaccine DNA sequence inserted into pFastBac HTa is depicted. The 6xHis tag is highlighted. 25 Example 4Z: Transposition and bacmid generation The generation of recombinant bacmids is based on the Bac-To-Bac® cloning system (Invitrogen TM) that uses site-specific transposition with the bacterial transposon Tn7. This is accomplished in E. coli strain DH1OBac. The DH1OBac cells contain the bacmid, which confers Kanamycin resistance and a helper plasmid which encodes the 189 transposase and confers resistance to Tetracycline. The gene of interest is cloned into the donor plasmid pFastBac which has mini-Tn7 elements flanking the cloning sites. The plasmid is used to transform E coli strain DH1OBac, which has a baculovirus shuttle plasmid (bacmid) containing the attachment site of Tn7 within a LacZaX gene. 5 Transposition disrupts the LacZax gene so that only recombinants produce white colonies and are easily selected for. The advantage of using transposition in E coli is that single colonies contain only the recombinant. The recombinant bacmids are isolated using standard alkaline lysis protocols and are used for transfection in Sf9 insect cells to generate baculoviruses that express recombinant proteins. The sequence of events 10 involved in the bacmid recombination is shown in Figure 13Z. Donor plasmids pFastBacHTa-gp64/CSP/AMA-1/MSP-1 42 /HBV Core/TBD, pFastBacHTa-gp64/CSP/AMA-1/MSP-1 42 which encodes the three antigen Chimigen@ Vaccineand pFastBacHTa-gp64/CSP/AMA-1/LSA-12 0 /MSP-1 42 /HBV Core/TBD which encodes the four antigen Chimigen@ Vaccine or their respective control plasmids (no 15 HBV Core, no TBD) were used for the site-specific transposition of the cloned gene into a baculovirus shuttle vector (bacmid). The recombinant pFastBac-HTa plasmids with the gene of interest are used to transform DH1OBac cells for the transposition to generate recombinant bacmids. Bacmids were produced by heat shock transformation of DH1OBac E coli 20 (Invitrogen</div>
    <div class="description-paragraph style-scope patent-text">TM</div>
    <div class="description-paragraph style-scope patent-text">) with 1 pL of pFastBac-HTa construct miniprep DNA diluted 1/200 in water. Cells were grown in LB without antibiotics for 4 h at 37'C and 225 rpm. After outgrowth period, 100 pL of 1/10, 1/100 and 1/1000 cell dilutions in LB were plated onto individual LB agar plates containing 100 pg/mL Bluo-Gal, 40 pg/mL IPTG, 50 pg/mL Kanamycin, 7 pg/mL Gentamycin and 10 pg/mL Tetracyclin for 48h at 37 0 C. From 25 initial plates, 10 white colonies were restreaked onto 1 LB agar plate containing 100 pg/mL Bluo-Gal, 40 pg/mL IPTG, 50 pg/mL Kanamycin, 7 pg/mL Gentamycin and 10 pg/mL Tetracyclin overnight at 37'C to screen for residual blue coloration. For each construct, five colonies were restreaked onto fresh plates for 48h to confirm white phenotype then inoculated into 2 mL LB cotanining 50 pg/mL Kanamycin, 7 pg/mL 30 Gentamycin and 10 pg/mL Tetracyclin and grown overnight at 37'C, 225 rpm. Bacmids 190 were purified by alkaline lysis and neutralized samples were spun at 15 000 x g for 10 min. Supernatant was transferred to fresh 1.5 mL eppendorf tubes and 0.7 volumes of isopropanol was added. Samples were incubated on ice for 10 min and spun at 15 000 x g for 20 min. Supernatant removed and bacmid DNA pellet washed in 1 mL of 70% 5 ETOH. Bacmid DNA was then air dried for 10 min at 22'C and overlaid with 40 PL TE buffer. Bacmid DNA was allowed to dissolve without pipetting and integrity was assessed by PCR analysis using the pUC M13 forward primer in combination with CSP reverse 5'-TCGTTCTCGCCCAAGCTCCTA-3' and pUC M13 reverser primer in combination with either MSP-1 bacmid forward primer 5' 10 GTGCACCAAGCCCGACTCCTA-3' (for control protein) or TBD forward 5' AAAGGCAGACCGAAGGCTCCA-3' (for Chimigen* protein). PCR was performed on 1 pL bacmid template with 19 pL PCR mix (2 pL 1OX PCR buffer, 0.4 pL 10mM dNTP mix, 0.3 pL forward primer, 0.3 pL reverse primer, 0.6 pL MgCl 2 , 15.2 pL H 2 0 and 0.2 pL Taq DNA polymerase). PCR cycling consisted of initial denaturation phase of 93'C 15 for 3 min, followed by 30 cycles of 94'C for 45 s, 55 0 C for 45 s and 72'C for 2 min. Cycling was followed by final extension at 72'C for 7 min and indefinite cooling at 4'C. Bacmid DNA was stored at 4'C for no more than 1 week and used for baculovirus production in Sf9 insect cells. Remaining DNA was ethanol precipitated and stored at 20 0 C. 20 Example 5Z: Baculovirus Transfection and P1 baculovirus stock For transfection, Sf9 insect cells between passage 19 and 50 were plated in 6-well plates (9 x 10 5 /well) in 2 mL ESF 921 insect cell media and allowed to adhere to the plate for lh at 27 0 C. During this incubation, 5 pL purified bacmid as well as 6 pL Cellfectin® 25 (Invitrogen TM) were mixed with 100 pL of ESF 921 in separate tubes. Diluted Cellfectin* was added to diluted bacmid DNA, mixed gently and incubated for 30 min at 22 0 C, followed by addition of 800 pL ESF 921 to DNA:Cellfectin® complexes. Media was removed from adherent cells and DNA:Cellfectin® mixture was immediately added 191 dropwise to cells. Cells incubated for 5h at 27'C, transfection media removed and 2 mL of fresh ESF 921 was immediately added. Cells were then incubated for 72 h at 27 0 C. Baculovirus stock amplification 5 It is standard practice to perform two rounds of viral amplification to generate P3 vaculoviral stocks. In all protocols described herein this standard practice was followed. From P1 stocks, entire supernatant was removed from the cells and added to 100 mm tissue culture treated BD Falcon petri dishes (VWRTM) containing 6 x 106 adhered Sf9 cells in 10 ml ESF 921 and incubated for a further 96 h at 27 0 C. Supernatant from this 10 incubation (P2 stock) was harvested when clear signs of infection were detected and was stored at 4'C following addition of 2% FBS. P2 stocks were probed by Western blot as described herein to probe for protein production. For generation of P3 stocks, 4 mL of P2 baculovirus stock procuding the highest level of protein was added to 1x10 9 Sf9 cells in logarithmic growth phase seeded in 500 ml ESF 921. After 48h, viability was monitored 15 daily and supernatant (P3 stock) was harvested when viability dropped below 80%. Supernatant was clarified by centrifugation at 3000 x g for 10 min and stored in the dark in 2% FBS at 4'C. After the second round of amplification (P3 stock), the concentration of the generated baculovirus was quantified using the baculovirus titering assay according to the protocols described by the manufacturer of the kit (Expression TM 20 Systems , Woodland, CA). Data was acquired using a FACScalibur flow cytometer TM (BD Biosciences , San Jose, CA). Using the P3 baculovirus stock, the most appropriate multiplicity of infection (MOI) to infect Sf9 insect cells (in pfu/cell) and the optimum time point for the production of the desired protein was established. The expression of the heterologous 25 protein in the cells was verified by time course analysis using MOI of 1, 2 and 5 for 24, 48 and 72h respectively. At each MOI and time point, 1 mL cell suspension was harvested. Trypan blue viability was assessed and cells were spun down at 15 000 x g for 5 min. Supernatant was removed and cells were frozen at -20'C until use. For western blot analysis, cells were resuspended in 500 pL PBS and 50 pL was used for western 30 blots as described herein. 192 Optimization of Expression: MOI and time course analysis Three 100 mL PTEG flasks were seeded with 100 x 106 Sf9 insect cells in 50 mL (2 x 10 6 /mL). Titered P3 baculovirus stock was added at MOI of 1, 2 and 5 and flasks incubated for 24, 48 and 72 h to determine best MOI and time course for expression. At 5 each time point, 1 mL of cells was removed from the flask. Cells were counted in trypan blue for cell number and viability. Remaining cells in the 1 mL aliquot were pelleted and resuspended in 500 pL PBS. Resuspended cells were aliquoted into 50 PL aliquots and frozen at -20'C for Western blot analysis of expression. For analysis, 50 PL aliquot was thawed on ice and lysed in 10 pL 5x loading buffer (Fermentas Tm ) and 1.25 PL 20x 10 reducing agent (Fermentas</div>
    <div class="description-paragraph style-scope patent-text">TM</div>
    <div class="description-paragraph style-scope patent-text">). Lysates were boiled for 5 min and loaded onto gel without cooling. Example 6Z - Protein Expression Expression of recombinant proteins in wave bag bioreactors For large-scale expression of the proteins, disposable Wave Bag Bioreactor (Wave 15 BiotechnologiesTM, NJ, USA) and ESF 921 (Expression SystemsTM) were used. Sf9 insect cell stock was seeded at 1 x 106 cells/mL into 500 mL ESF 921 media in a 2 L PTEG flask (Thermo Fisher Scientific</div>
    <div class="description-paragraph style-scope patent-text">TM</div>
    <div class="description-paragraph style-scope patent-text">, Rochester, NY). Cultures were incubated at 27'C with shaking at 130 rpm in a bench-top shaker-incubator until cell density reached 4 x 106 cells/mL (3-4 days). Culture volume was adjusted to 1 L and culture was split into 20 2 separate 2 L Erlenmeyer flasks each with 500 mL insect cell culture at a density of 2 x 106 cells/mL. Cultures were incubated at 27'C with shaking at 130 rpm, in a bench-top shaker-incubator until the cell density reached 8 x 106 cells/mL (3-4 days). A 10 L capacity Wave Bag Bioreactor was used for 5 L cultures with the ESF 921 growth media. Wavebags were filled with 4 L of ESF 921 media and 1 L of the seed 25 culture described herein was inoculated into the medium to make up 5 L final volume at a density of 1.6 x 106 cells/mL. The rocking of the wave bioreactor was set at 32 rpm/5 0 , atmospheric air flow at 0.30 Lpm (litres per minute), temperature at 27.5'C. Rocking of the bag continued until the cell density reached 2 x 106 cells/mL (-24h). The cells were 193 infected with appropriate MOI. The bioreactor was allowed to rock at 32 rpm/5', air flow (30% 02) of 0.30 Lpm and at 27.5'C for the appropriate time point. Culture samples were counted periodically to monitor the progress of infection. Infected cells were harvested by centrifugation using the Beckman Avanti TM J25 centrifuge (JA-10 rotor at 5000 x g) 5 for 10 min, at 4'C. Pellets of Sf9 cells were washed in 25 mL 0.22 Pm filtered PBS, spun at 3000 x g for 5 min. Supernatant was removed and pellets were snap frozen in liquid nitrogen then stored at -80'C. Purification of proteins 10 Purification of the 6xHis-tagged vaccines is based on Ni-chelation affinity chromatography under denaturing conditions using HisTrap FF (GE Healthcare TM) in a single step by FPLC. Purification protocol is described below. Lysis and solubilization 15 The cell pellets from 370 mL culture of infected Sf9 cells were resuspended in 118 mL of lysis buffer (6M GuHCl, 50mM NaH 2</div>
    <div class="description-paragraph style-scope patent-text">PO</div>
    <div class="description-paragraph style-scope patent-text">4 , 0.5M NaCl, pH 7.4) on ice. The cells were sonicated on ice 5 times for 30s per pulse (118 W) with 5 min interval between pulses. After sonication, the pH of the lysate was adjusted to 7.4 and 2% Tween-20, 20 20mM imidazole and 10mM j-mercaptoethanol were added. The lysate was stirred for 2 hrs at room temperature then filtered through a 5 pm Acrodisc syringe filter (Pall Corporation</div>
    <div class="description-paragraph style-scope patent-text">TM</div>
    <div class="description-paragraph style-scope patent-text">, Mississauga, ON). Affinity chromatography 25 The protein was purified using an AKTA Explorer100 FPLC system (GE TM Healthcare , Piscataway, NJ). A 5 mL HisTrap FF column was used. The column was equilibrated with 5 column volumes of equilibration buffer (6M GuHCl, 20mM NaH 2</div>
    <div class="description-paragraph style-scope patent-text">PO</div>
    <div class="description-paragraph style-scope patent-text">4 , 0.5M NaCl, 20mM imidazole, 0.05% Tween-20, pH 7.4). Lysate was loaded onto the column at 1 mL/min and the flow through was collected. The column was 30 washed with equilibration buffer until the absorbance at 280 nm was less then 10 mAU. 194 Next the column was washed with wash buffer (6M GuHCl, 20mM NaH 2</div>
    <div class="description-paragraph style-scope patent-text">PO</div>
    <div class="description-paragraph style-scope patent-text">4 , 20mM imidazole, 10 mM ethylenediamine, 0.05 % Tween 20, pH 7.4) at 1mL/min. Fractions of 5 mL were collected and wash continued until absorbance at 280 nm was less than 10 mAU. Protein was eluted with elution buffer (6M GuHCl, 50mM NaH 2</div>
    <div class="description-paragraph style-scope patent-text">PO</div>
    <div class="description-paragraph style-scope patent-text">4 , 250mM 5 imidazole, 0.05 % Tween-20, pH 7.4) at 1 mL/min and 2 mL fractions were collected. TCA precipitation of the 280 nm elution peak fractions was performed and concentration of protein was determined using the Micro BCA assay. Total protein yield was calculated as follows: 10 Fraction yield (mg/L) = fraction concentration (Ig/mL) x dilution factor x fraction volume (mE) Total volume of lysate Total protein yield was calculated by sum of yields in each fraction of the 280 nm elution 15 peak. Dialysis The fractions containing the highest concentration of eluted protein were placed in Slide-a-lyzer cassettes (Thermo Fischer Scientific T M ) and dialyzed against 500 mL 0.22 20 pm filtered dialysis buffer (l0mM NaH 2</div>
    <div class="description-paragraph style-scope patent-text">PO</div>
    <div class="description-paragraph style-scope patent-text">4 , 150 mM NaCl, 0.05 % Tween-20 pH 7.4) overnight at 4'C with constant stirring. A second dialysis step in 500 mL of fresh buffer was performed for 6h at 4'C with constant stirring. A final dialysis step in 500 mL of fresh buffer was performed overnight at 4'C with constant stirring. Post dialysis concentration of protein was determined using the Micro BCA assay. 25 Western blot To assess baculovirus encoded protein production in Sf9 cells, infected and uninfected cells 2 x 106 cells were resuspended in 500 pL PBS on ice. A 50 pL aliquot was lysed on ice in appropriate volume of 5x loading buffer containing 1x reducing agent TM (Frementas , Burlington, ON). Remaining cells were returned to the freezer. Lysates 30 were boiled for 5 min and 20 pL immediately loaded onto 7.5% TGX precast gels (Bio 195 RadTM, Mississauga, ON) without cooling. Gels were run at 100 V constant voltage for 1.25 h, in Tris-Glycine-SDS buffer followed by wet transfer onto Hybond ECL nitrocellulose membranes (GE Healthcare T M ) at 200 mA constant current for lh. Membranes were washed 1x in PBS and blocked for lh in Tris buffered saline (TBS) (20 5 mM Tris, 150mM NaCl, pH 7.5) containing 0.1% Tween-20 and 1% milk (Bio-Rad T M ). Membranes were then probed using either the 6xHis tag monoclonal antibody (1:10,000) or anti mouse IgG1 (Fc specific) antibody (1:20,000). FPLC purified proteins were TCA precipitated to remove GuHCl and resuspended in an equal volume of 0.1M NaOH. Resuspended purified protein was analyzed by western blot using the same method 10 described above. Detection method for western blot was ECL reagent (GE healthcare T M ), followed by exposure on BioMax TM light film (Sigma-Aldrich T M ). Molecular weight calculations were performed against Pageruler Plus molecular weight standards using Unscan-it gel 6.1 gel digitizing software (Silk Scientific</div>
    <div class="description-paragraph style-scope patent-text">TM</div>
    <div class="description-paragraph style-scope patent-text">, Orem, UT) 15 Glycoprotein stain using Concanavalin A-conjugated HRP (ConA-HRP) Chimigen* Malaria Multi-antigen Vaccines and control proteins were separated by 7.5 % SDS-PAGE and transferred onto Hybond ECL nitrocellulose membranes. The nitrocellulose membrane was incubated in TBS containing 2 % Tween-20 for 10 minutes at room temperature. After rinsing the membrane twice with TBS, the nitrocellulose 20 membrane was incubated with 10 mL of ConA-HRP (concentration at lpg/mL in TBS with 0.05 % Tween 20, 1 mM MgCl 2 , 1 mM MnCl 2 and 1 mM CaCl 2 ) for 30 minutes at room temperature. The membrane was washed three times with TBS containing 0.05% Tween-20. Positive bands were visualized by chemiluminescence using ECL kit (GE TM TM Healthcare</div>
    <div class="description-paragraph style-scope patent-text">TM</div>
    <div class="description-paragraph style-scope patent-text">) and exposed on BioMax light film. 25 Example 7Z: Immunological characterization of Chimigen* Malaria Multi-antigen Vaccine using PBMCs isolated from healthy donors FACS acquisition and analysis Cells were acquired with a FACSCalibur fitted with CellQuest Pro acquisition and analysis software (BD BiosciencesTM). A gate was made on the viable cell population 196 as determined by the forward scatter and side scatter profile and &gt;20,000 events were acquired. The percent of specific positive cells was calculated as: (% positive cells test sample - % positive cells control) / (100 - % positive cells of control) x 100. The relative mean fluorescent intensity (MFI) was determined as: MFI of the test sample - MFI of the 5 control sample. Human PBMC PBMCs from healthy individuals with the HLA-A2 haplotype (Biological Specialty Corporation</div>
    <div class="description-paragraph style-scope patent-text">TM</div>
    <div class="description-paragraph style-scope patent-text">) were isolated from whole blood by Histopaque-1077 density gradient 10 centrifugation (Sigma T). Plasma was heat-inactivated and used for cell culture (2.5% in AIM V) and in the medium for cryopreservation (10% DMSO, 90% plasma) of PBMCs which are not used immediately in the ex vivo assays. PBMCs at 3 x 107 cells/cryovial in freezing media (50% matched donor plasma, 40% AIM-V, and 10% DMSO) were stored in liquid nitrogen. 15 Isolation of human PBMC-derived T cells T cells were isolated from cryopreserved PBMCs generated herein by negative selection using a Dynal Biotech T cell negative selection kit (Invitrogen) following the manufacturer's protocol using matched donor plasma instead of BSA and FBS. The 20 phenotype of the cell population isolated was assessed by labeling for several relevant cell markers. T cells (CD3 cells) comprised greater than 98% of the isolated population. The T cells were used either labeled with CFSE (see below) or left unlabelled and added directly to cell cultures with DCs. 25 Isolation and differentiation of monocytes to immature DC (iDCs) Cryopreserved PBMCs generated herein were used to generate immature DCs (iDCs) in a procedure modified from Whiteside et al. (2004). PBMCs were cultured in AIM V/2.5% autologous plasma (AP) in 60 mm tissue culture dishes for 1 h at 37C. After the 1 h incubation, the non-adherent cells were removed by washing with AIM V. 30 The adherent cells were then cultured for 24 h in 1 mL of AIM V/2.5% AP with 1000 197 IU/mL IL-4 and 1000 IU/mL GM-CSF to generate iDCs. The iDCs were seeded into 96 well plates at 3 x 104 cells/well and cultured for another 24 h (without cytokines) at which point either dialysis buffer, Chimigen* Vaccine, individual vaccine components (protein) or tetanus toxoid (TT) was added to the cells. Six to 8 h later, poly I:C was 5 added (20 pg/mL) to mature the DCs. DCs were incubated with poly I:C for 24 h. Binding of Chimigen* Malaria Multi-antigen Vaccine to immature DCs Immature DCs were obtained as described herein. Following culture, the cells were harvested, washed once with AIM-V media containing 2.5% matched serum, followed by two washes with Dulbecco's phosphate buffered saline (Invitrogen TM) containing 0.1% 10 (w/v) BSA (PBSB). The cells were used to evaluate the binding of Chimigen* Malaria Multi-antigen Vaccines. The phenotype of the iDCs was assessed by labeling for several cell surface markers including CD64, CD32, CD16, CD206, CD205, CD209, HLA-ABC, HLA-DR, CD 14, CD1 Ic, CD86, CD80, CD40, CD83, CD19 and CD3. For the binding assay, all steps were performed at 4'C with washes following the 15 incubations. Cells were incubated for 60 min in PBSB with various concentrations of vaccine or the corresponding dialysis buffer (2 x 105 cells/well in 96-well v-bottom plates in a volume of 25 pL). Vaccine binding was detected by incubation of the cells with anti mouse IgG1-biotin in PBSB for 20 min, followed by SA-PE-Cy5 for 20 min. Cells were resuspended in PBSB containing 2% paraformaldehyde (PF) and the binding assessed by 20 FACS. In some experiments, binding specificity was assessed by pre-incubation with blocking antibodies to CD32 or CD205, CD206 and CD209. Generation of antigen-loaded mature DCs (mDCs) Immature DCs were generated as described herein and incubated for 8 h with 25 antigen or buffer (control). The cells were then cultured for 16 h with poly I:C (20 pg/mL). The extent of maturation of the DCs was assessed by phenotype analysis. The cells were labeled with several surface markers to determine the extent of DC maturation and percentage of purity of the DC preparation. These markers included CD64, CD32, CD16, CD205, CD206, CD209, HLA-ABC, HLA-DR, CD14, CD11c, CD86, CD80, 198 CD83, CD40, CD19 and CD3. In general, greater than 98% of the cells from the preparation were DCs. Most of the DCs were mature (mDCs) as they expressed high levels of the DC maturation markers CD83, CD86 and CD80. The matured antigen loaded DCs were washed and cultured with T cells. 5 Antigen presentation assays An antigen presentation assay was used to measure the immune response of naYve T cells to antigen presented by mDCs. The assay quantifies functional T cell immune responses and the ability of antigen-loaded mDCs to expand antigen-specific T cells. The 10 procedure consists of differentiating PBMC-derived monocytes to iDC, loading the iDCs with antigen, differentiating the iDCs to mDC and then culturing the antigen-loaded mDCs together with autologous naYve T cells. For the analyses of T cell function and specificity, T cells were re-stimulated with fresh preparations of antigen-loaded mDCs and the production of IFN-y, TNF-ax, Perforin (Pfn) and GrB was assessed. 15 Co-Culture of human PBMC-derived T cells with antigen loaded mDC T cells were incubated with antigen-loaded mDCs at T cell to DC ratio of 5:1 (1-2 x 105 T cells : 2-4 x 104 DC) per well in AIM V/2.5% matched serum. T cells were cultured for 10 days and then restimulated with fresh antigen-loaded mDCs. T cell 20 function was assessed 6 h following this second stimulation. Analysis of the expression of IFN-y, TNF-a, Pfn and GrB was performed as outlined below. For determination of intracellular cytokine production, brefeldin A (BD Biosciences Tm ) at 1 pg/mL was added to prevent cytokine release. The protocol for the two stimulation antigen presentation assay is shown in Figure 14Z. 25 CFSE labeling Freshly isolated or previously frozen PBMCs generated herein were washed once with D-PBS/5% AP in a 15 mL conical tube. The cells were resuspended at 4 x 106 199 cells/mL in D-PBS/5% AP and 110 pL/mL of a 50 pM dilution of CFSE (in D-PBS) was added. The cells were mixed and incubated at room temperature for 5 min. After 5 min, the tube of cells was topped up to 15 mL with D-PBS/5% AP and the cells were centrifuged. The cells were washed two more times with 15 mL D-PBS/5% AP. The 5 CFSE-labeled PBMCs were then resuspended at 4 x 106 cells/mL AIM V/2.5% AP. In vitro expansion of Chimigen* Malaria Multi-antigen-specific T cells Freshly isolated or previously frozen PBMCs (CFSE-labeled or unlabeled) were 10 resuspended at 4 x 106 cells/mL in AIM V/2.5% AP and 100 pL aliquots of cells (4 x 105 cells) were seeded into 96 well plates. The cells were rested for 24 h by incubation at 37'C. After 24 h, the PBMCs were stimulated with either dialysis buffer, Chimigen* Vaccine, individual vaccine components or tetanus toxoid (TT). Two days after the stimulation, 50 pL of fresh media was added to the PBMCs for a total of 200 PL of 15 culture media. Immunological assays were performed after 10 days of culture (proliferation assay) or after 3 days of culture (intracellular cytokine staining). T cell Proliferation assay Seven days after in vitro stimulation, the CFSE-labeled PBMCs were washed with 20 D-PBS/2% FBS and stained with fluorescently labeled anti-CD3 (PE), anti-CD4 (Allophycocyanin) and anti-CD8 (PerCP) antibodies for 20 minutes on ice. The cells were then washed twice with D-PBS/2% FBS, fixed in 2% PF and acquired on a flow cytometer. 25 Detection of Intracellular IFN-y and TNF-ax Six h following the second stimulation, the production of IFN-y and TNF-ax were quantified using a standard intracellular cytokine labeling protocol (BD Biosciences). Briefly, cells were labeled with specific fluorochrome conjugated mAbs for detection of CD3 (anti-CD3-Allophycocyanin, APC) and CD8 (anti-CD8-PE-Cy5), followed by 30 fixing and permeabilization. Following this, the cell samples were incubated with anti 200 IFN-y-PE and anti-TNF-ca-FITC. A range of 20,000-100,000 cells per sample were acquired using a BD FACSCalibur. The assay protocol is presented in Figure 14Z. Perforin (Pfn) and granzyme B (GrB) detection 5 Three days after the second stimulation, the expression of Pfn and GrB in T cells was quantified using a standard intracellular staining protocol. T cells were labeled with anti-CD8-PE-Cy5, anti-CD3-APC. The cells were then fixed in 2% PF, permeabilized TM with fixation/permeabilization buffer (eBiosciences , Cat#00-5123 &amp; 00-5223). After washing with permeabilization buffer three times, the T cells were incubated with anti 10 Pfn (eBiosciences T M ) anti-GrB-FITC (BD Biosciences T M ). Approximately 50,000 cells were acquired using the FACSCalibur. The assay protocol is presented in Figure 14Z. IFN-y and TNF-a intracellular staining following stimulation with Chimigen* Malaria Multi-antigen Vaccine 15 PBMCs stimulated for 10 d in vitro were re-stimulated with freshly prepared antigen-loaded mDCs to induce cytokine production. Day 10 PBMCs were co-cultured with antigen-loaded mDCs for 30 min at 37'C, at which time 1 pig/mL Golgi Plug (BD) was added for an additional 6 h of incubation. After 6 h, the cells were washed with D 20 PBS/2% FBS and stained with antibodies to cell surface markers, anti-CD8 (PerCP) and anti-CD3 (Allophycocyanin), for 20 min on ice. The cells were washed twice with D PBS/2% FBS, fixed and left at 4 0 C overnight in the dark. The next day, the cells were permeabilized with permeabilization buffer (eBiosciencesTM) for 30 min on ice and stained with anti-TNF-a (FITC) and anti-IFN-y (PE) for 30 min on ice. The cells were 25 washed twice with permeabilization/wash buffer, resupended in D-PBS/2% FBS/2% paraformaldehyde and acquired on a flow cytometer. IFN-y ELISA Ten days following first stimulation of naive T cells by co-culture with antigen 30 loaded mDC, supernatant was collected to assess T cell secretion of IFN-y using the BD 201 TM OptEIA Human IFN-y ELISA set (BD Bioscience , Mississauga, ON). The assay was completed by following the protocol from BD Biosciences. Detection of GrB, Pfn and lactate dehydrogenase (LDH) 5 At the end of the co-culture, the cells were harvested for intracellular staining of GrB and Pfn production in CD8+ and CD4+ T cells and were evaluated using flow cytometry. In some experiments, the supernatant was collected for LDH release assay. LDH release assay was conducted by LDH based in vitro toxicology assay kit (Sigma Tm ) according to the manufacturer's instructions. 10 Induction of functional B cell responses by the Chimigen* Malaria Multi-antigen Vaccine An assay to measure B cell activation and differentiation into plasma cells, to evaluate the humoral response, is currently being developed. On day 0, PBMC-derived B 15 cells (CD19+) were incubated with vaccine- or vaccine component-loaded mDCs in the presence of autologous T cells to induce B cell differentiation. On day 7 of co-culture, the B cells were added to fresh antigen-loaded mDCs and this process was repeated on day 14. After 21 days in co-culture, expression of plasma cell markers CD27, CD38 and CD138 was assessed. 20 Example 8Z: Results - Expression, purification and characterization of Chimigen* Malaria Multi-antigen Vaccine and control protein The following sections describe the results from the expression optimization, purification, biochemical as well as immunological characterization and the immune responses generated by an embodiment of the Chimigen® Malaria Multi-antigen Vaccine 25 (chimeric antigen). Optimization of Chimigen® and control protein expression (MOI and time of infection) For the 3 antigen Chimigen* Vaccine, the best MOI was 1 and the optimal time for expression was 48 h (viability 86%). For the 3 antigen control protein, the best MOI was 202 5 and the optimal time for expression was 24h (viability 98%). For both the 4 antigen Chimigen@ Vaccineand 4 antigen control protein, the best MOI was 1 and the optimal time for expression was 48 h (83% and 56% viability respectively). These MOI and incubation times gave the most protein expression with the least amount of degradation 5 (Figure 15Z). Figure 15Z depicts: A) Three antigen Chimigen® expression optimization. B) Three antigen control protein expression optimization. C) Four antigen Chimigen® expression optimization; and D) Four antigen control protein expression optimization. 10 Chimigen® Malaria Multi-antigen Vaccine and its control protein (no HBV Core, no TBD) have been purified and characterized Chimigen® HBV Multi-antigen Vaccine was purified under denaturing conditions from baculovirus infected Sf9 cells by Ni-NTA chelation affinity chromatography. 15 Purified proteins were separated on SDS-PAGE gels (4-15%) and stained with PageBlue. A representative gel is shown in Figure 16Z. Figure 16Z depicts: SDS-PAGE of Purified Chimigen® Malaria Multi-antigen Vaccine. Lane 1) PagerRuler Plus prestained markers. Lane 2) 3 antigen Chimigen* Vaccine. Lane 3) 3 antigen control protein. Lane 4) Four antigen Chimigen® Vaccine. Lane 5) 20 Four antigen control protein. The three antigen Chimigen® Malaria Multi-Antigen Vaccine has a theoretical molecular weight of 209 KDa but displayed a calculated molecular weight of 262 KDa (Figure 16Z lane 2), whereas the control protein has a theoretical molecular weight of 162 kDa and displayed a calculated molecular weight of 225 kDa (Figure 16Z lane 3). 25 The four antigen Chimigen® Malaria Multi-Antigen Vaccine has a theoretical molecular weight of 303 kDa and displayed a calculated molecular weight of 324 kDa (Figure 16Z lane 4), whereas the four antigen control protein has a theoretical molecular weight of 256 kDa and displayed a calculated molecular weight of 272 kDa (Figure 16Z lane 5). In the purified product, there were some lower molecular weight bands in both the three and 203 four antigen vaccines as well as in their respective controls. These bands can be attributed to either 1) co-purified insect cell proteins, 2) incomplete vaccine fragments or 3) vaccine degradation. Further analysis is required to determine the nature of these smaller molecular weight bands as described herein. 5 The IRD and TBD of Purified Chimigen® Malaria Multi-Antigen Vaccine are intact Purified protein was separated by 4-15% SDS-PAGE, electroblotted onto nitrocellulose membranes and used for Western detection using 5 different primary antibodies for ih at room temperature: anti-mouse Fc (Sigma T M ), 1:20,000, anti-HBV 10 core 14E11 monoclonal (Abcam m , Cambridge, MA), 1:1,000, anti-6xHis mAb (Clontech T M , Mountain View, CA), 1:10,000, anti-AMA-1 (Michael Blackmanrm, NIMR, UK), 1:10,000, and anti-MSP-1, PEM-1 monoclonal (Abcam), 1:1,000. After 3x5 min washes in TBS-tween, the secondary antibody used was goat anti-mouse IgG (Fab) HRP (Sigma T M ), 1:5,000 for the anti-HBV core and goat anti-rabbit IgG-HRP (Sigma T M ), 15 1:5,000 for the anti-AMA-1 antibodies. After 3x5 min washes in TBS-tween, the binding was detected by chemiluminescence as per manufacturer's protocol (GE Healthcare T M ). Antibodies for CSP and LSA-I are not commercially available, and thus, we were unable to include them in our characterization analysis. The results presented in Figure 17Z show that the purified three antigen Chimigen® Malaria Multi-antigen Vaccine is intact 20 with respect to the N-terminus, the HBV core, Fc domain, and both MSP-1 and AMA-1 antigens. Smaller fragments detected by PAGEblue staining (Figure 16Z) were detected by the anti-6xHis and both AMA-1 and MSP-1 antibodies but were not detected by anti Fc antibodies, suggesting that incompletely synthesized vaccine fragments are co purifying along with the full length vaccine since the 6xHis tag is on the N-terminal end 25 and would be present in all vaccine fragments. The 6xHis tag is currently being relocated to the C-terminal end to eliminate this problem. Figure 17Z depicts characterization of the three and four antigen Chimigen® Malaria Multi-antigen Vaccines by western blot analysis. One pg of Chimigen* or control protein (lane 3, 5, 7, 9) was run on a 4.5-15% gel and transferred to nitrocellulose. 30 Membranes were cut into strips for probing with the antibodies listed below each blot. 204 Blotting was performed in separate trays then, membranes were exposed together on the same piece of film. A) Three antigen vaccine (blue arrow) and control protein (red arrow). B) Four antigen vaccine (blue arrow) and control protein (red arrow). Smaller bands below 200 kDa were detected by anti-6xHis but not anti-Fc indicating that they are 5 incomplete vaccine fragments. Three and four antigen Chimigen* Malaria Multi-Antigen Vaccines and control proteins are glycosylated The expression of recombinant proteins in insect cells results in pauci/high mannose 10 glycosylation of the expressed protein. ConA is a lectin that has affinity for terminal a-D mannosyl residues on glycoproteins. The purified vaccine was separated by 4</div>
    <div class="description-paragraph style-scope patent-text">-</div>
    <div class="description-paragraph style-scope patent-text">15 % SDS PAGE, electrotransferred onto a nitrocellulose membrane and the membrane was blocked with TBS + 2% Tween-20, then stained with ConA-HRP. Purified protein was detected (Figure 18Z) which shows that the Chimgen* Malaria Multi-antigen Vaccine and control 15 proteins are glycosylated when expressed in insect cells. Figure 18Z depicts glycosylation of Chimigen® Malaria Multi-Antigen Vaccines. Lane 1) Pageruler plus MWM, Lanes 2 and 3) Three antigen vaccine and control proteins respectively. Lanes 4 and 5) Four antigen vaccine and control protein 20 respectively. Example 9Z: Results - Immunological evaluation of Chimigen® Malaria Multi-antigen Vaccine Binding of three and four antigen Chimigen® Malaria Multi-antigen Vaccines to 25 immature DCs The three and four antigen Chimigen* Malaria Multi-antigen Vaccines were examined for their ability to bind to iDCs. Immature DCs were incubated with either three or four antigen vaccine for 1 h at 4'C and the binding detected with anti-mouse IgGI biotin conjugated mAbs, together with SA-PE Cy5 which detects the TBD portion 205 of the vaccine. The four antigen control protein was also tested for iDC binding using the anti-mouse IgGI biotin/SA-PE Cy5 antibodies as well as using the anti-AMA-i TM polyclonal rabbit antibody with an anti-rabbit IgG (R&amp;D Systems , Minneapolis, MN) as a secondary antibody. The mean fluorescence intensity (MFI) representing the relative 5 amount of bound protein was determined by flow cytometry (Figure 19Z). The majority of iDCs bound both three and four antigen vaccines at 1-62.5 [tg/mL, as detected by anti mouse IgGI mAb, and binding was not saturated at 50 pg/mL (Figure 19Z A and B). When control protein binding was assessed using anti-Fc antibodies, no binding was observed, but control protein binding was observed when anti-AMA-i antibodies were 10 used instead of anti-Fc (Figure 19Z C). However, when stained with anti-AMA-i antibodies, Vaccine binding was much stronger than control protein binding, suggesting that the vaccine is bound to iDCs through both CD32 (FcyR) as well as CD206 (mannose receptor), whereas the control protein is only bound to iDCs through the CD206 (Figure 19Z C). Blocking experiments using the anti-CD206 antibodies did not reduce binding of 15 vaccine or control protein (data not shown). Figure 19Z depicts binding of three and four antigen Chimigen* Malaria Multi antigen Vaccines to 48 h cultured iDCs. A) Three antigen vaccine binding assessed by anti-Fc antibody staining. B) Four antigen vaccine binding assessed by anti-Fc antibody 20 staining and C) Four antigen vaccine binding assessed by anti-AMA-I staining. Induction of functional T cell responses by the three and four antigen Chimigen* Malaria Multi-antigen Vaccines: CD4+ and CD8+ T cell proliferation 25 The Chimigen® Malaria Multi-antigen Vaccine was assessed for its ability to induce proliferation of naYve T cell induced from PBMC generated herein. CFSE-labeled naYve T cells were generated as described herein and stimulated in vitro with either dialysis buffer, Chimigen® Malaria Multi-antigen Vaccine, malaria proteins (Antigens), mTBD, HBV core (HBV) or tetanus toxoid (TT) for 10 days. For both the three antigen and four 30 antigen vaccines, the percentage of CD3 T cells that had undergone at least one cell division was determined (Figure 20Z A and C). This CD37 T cell proliferation was 206 further broken down into CD4+ and CD8+ subsets to determine if the vaccine activates both humoral and cell mediated T cell responses (Fig. 20Z B and D). At low doses, both the three and four antigen Chimigen® Malaria Multi-antigen Vaccines induced a higher percentage of proliferating CD3 T cells than dialysis buffer or individual vaccine 5 components (antigens, TBD or HBV) in CFSE labeled T cells (Figure 20Z A). In general, the CD4+ T cells underwent a greater number of cell divisions than CD8+ T cells, as determined by the percentage of cells positive for CFSE. For both subsets of T cells, proliferation induced by both vaccines but not their respective controls was comparable to that induced by TT. These results demonstrate that the Chimigen® Malaria Multi 10 antigen vaccine is able to induce robust CD4+ and CD8+ T cell proliferation (Figure 20Z B and D). Figure 20Z depicts proliferation of naYve T cells induced by primary stimulation with mDCs loaded with either Vaccine or individual vaccine components. A and C) 15 CD3 T cells. B and D) CD4+ and CD8+ T cell subsets. Induction of functional T cell responses by the three and four antigen Chimigen® Malaria Multi-antigen Vaccines: IFN-y secretion The ability of the vaccine to induce a functional T cell response was assessed by 20 quantifying the percentage of T cells secreting the Th1 cytokine IFN-y after the first stimulation with antigen -loaded mDCs. NaYve T cells isolated from normal donor PBMC were cultured for 10 days with vaccine-loaded mDCs in the absence of exogenous IL-2 (Figure 14Z). There was a dose dependent increase in the amount of IFN-y secreted in the co-culture supernatant during these 10 days as determined by IFN-y ELISA (BD 25 Biosciences). The IFN-y response induced by the vaccine but not the controls was significantly higher than that induced by TT. There was a marked upregulation of IFN- y secretion following vaccine re-stimulation compared to the other treatments (Fig. 21Z). 207 Figure 21Z depicts ELISA determination of IFN-y secretion in co-culture supernatant by T cells stimulated with mDCs loaded with either three antigen Chimigen* Malaria Multi-antigen Vaccine or individual vaccine components. 5 Induction of functional T cell responses by the three and four antigen Chimigen* Malaria Multi-antigen Vaccines: Intracellular IFN-y and TNF-a expression The ability of the vaccine to induce a functional T cell response was assessed by quantifying the percentage of T cells producing the Thl cytokines IFN-y and TNF-a. NaYve T cells from donors were cultured for 10 days with vaccine-loaded mDCs in the 10 absence of exogenous IL-2 and re-stimulated for 6 h with freshly prepared vaccine loaded mDCs. The percentage of CD8+ and CD4+ T cells expressing TNF-a and IFN-y was determined by ICC labelling and FACS analysis as described herein. There was a marked induction of T cells producing IFN-y following vaccine re-stimulation (Fig. 22Z). The percentage of CD8+ and CD4+ cells from this population that expressed IFN-y is 15 shown in Fig. 22Z B. Compared to controls, both vaccines induced a significantly higher percentage of IFN-y producing CD4+ T and CD8* T cells which was higher than the TT response (Figure 22Z). Compared to CD8+ T cells, a greater percentage of CD4+ T cells expressed IFN-y from vaccine and TT-treated cultures. The production of TNF-a following re-stimulation was also investigated by ICC 20 using a TNF-a mAb with FACS analysis. The vaccine induced a marked increase in the percentage of CD4+ T cells producing TNF-a (Figure 23Z). Both the three antigen and four antigen vaccines increased production of TNF-a in CD4+ and CD8+ T cells. As for the IFN-y response, vaccine-induced TNF-a expression was higher than that induced by TT (Figure 23Z). These results demonstrate that both the three antigen and four antigen 25 Chimigen* Malaria Multi-antigen Vaccines induce the expression of IFN- y and TNF-C in CD4+ and CD8+ T cells, thus activating both humoral and cell-mediated immune responses. Figure 22Z depicts intracellular expression of IFN-y in T cells 6 h following 30 restimulation with Vaccine-loaded mDCs. Three antigen vaccine-induced TNF-a 208 expression in A) CD3 T cells and B) CD4+ and CD8+ T cell subsets. Four antigen vaccine-induced TNF-ax expression in C) CD3 T cells and D) CD4+ and CD8+ T cell subsets. 5 Figure 23Z depicts intracellular expression of TNF-a in T cells 6 h following restimulation with Vaccine-loaded mDCs. Three antigen vaccine-induced TNF-ax expression in A) CD3 T cells and B) CD4+ and CD8+ T cell subsets. Four antigen vaccine-induced TNF-ax expression in C) CD3 T cells and D) CD4+ and CD8+ T cell subsets. 10 Induction of functional T cell responses by the Chimigen* Malaria Multi-antigen Vaccine: GrB and Perforin expression The expression of GrB and Pfn in T cells was quantified as a means to assess the ability of the vaccine to induce a cytotoxic T cell response. T cells were stimulated twice 15 with mDCs loaded with the vaccine, buffer, or TT. Seven days following re-stimulation (day 17 of T cell culture), T cells were labelled for GrB expression using an intracellular staining procedure and analysed by FACS. For both vaccines, there was a marked induction of GrB expression in CD3 T cells from cultures stimulated with vaccine or TT (Fig. 24Z). Both the three and four antigen vaccines but not their respective controls 20 induced a higher percentage of GrB expression in CD3 T cells than TT treatment (Figure 24Z A and C). Both CD4+ and CD8+ T cells expressed GrB in response to vaccine treatment (Figure 24Z B and D). For both vaccines, there was also a marked induction of Pfn expression in CD3 T cells from cultures stimulated with vaccine or TT (Fig. 25). Both the three and four antigen vaccines but not their respective controls induced a higher 25 percentage of Pfn expression in CD3 T cells than TT treatment (Figure 25Z A and C). Both CD4+ and CD8+ T cells expressed Pfn in response to vaccine treatment (Figure 25Z B and D). In summary, the vaccine induced functional T cell responses in antigen presentation assays as assessed by the generation of T cells expressing IFN-y, TNF-ax, GrB and Pfn. 30 209 Figure 24Z depicts intracellular expression of GrB in T cells 3 days following restimulation with Vaccine-loaded mDCs. Three antigen vaccine-induced TNF-ax expression in A) CD3 T cells and B) CD4+ and CD8+ T cell subsets. Four antigen vaccine-induced TNF-ax expression in C) CD3 T cells and D) CD4+ and CD8+ T cell 5 subsets. Figure 25Z depicts intracellular expression of Pfn in T cells 3 days following restimulation with Vaccine-loaded mDCs. Three antigen vaccine-induced TNF-ax expression in A) CD3 T cells and B) CD4+ and CD8+ T cell subsets. Four antigen 10 vaccine-induced TNF-ax expression in C) CD3 T cells and D) CD4+ and CD8+ T cell subsets. Detection of lactate dehydrogenase (LDH) in mDC:T cell co-culture Three days following re-stimulation of T cells with fresh vaccine-loaded mDCs, 15 lactate dehydrogenase (LDH) release was measured in co-culture supernatant. LDH is released from dying or dead cells and is indicative of cytotoxic lymphocyte (CTL) activity and not of T cell death as TT does not induce LDH release (Figure 26Z). When T cells were re-stimulated with vaccine-loaded mDCs, there was a dose-dependent increase in LDH release in culture supernatant, which did not occur when T cells were re 20 stimulated with vaccine component-loaded mDCs. This data suggests that the mDCs expressing vaccine peptides through class I MHC cross presentation are being killed by activated CTL. Figure 26Z depicts Lactate dehydrogenase (LDH) release from dead cells into co 25 culture supernatant 3 days following re-stimulation of T cells with 4 antigen vaccine loaded mDCs. This data is indicative of cytotoxic lymphocyte (CTL) activity in co culture. Induction of functional B cell responses by the Chimigen* Malaria Multi-antigen 30 Vaccine 210 Using only the three antigen Chimigen@ Malaria Multi-antigen Vaccine, we assessed the ability of vaccine-loaded mDC to induce B cell differentiation into plasma cells. The preliminary data for the B cell differentiation assay shows that after three stimulations with antigen-loaded mDCs, B cells differentiate into plasma cells more efficiently in the 5 presence of vaccine-loaded mDCs than in the presence of vaccine component-loaded mDCs or TT treatment alone (Figure 26Z). In the presence of vaccine-loaded mDC, there is a dose-dependent increase in B cell surface expression of IgG and IgM (Figure 26Z A). Vaccine-loaded mDCs also increase cell surface expression of the plasma cell markers CD27, CD38 and CD138 on CD19+ B cells (Figure 26Z B). This data suggests 10 that the Chimigen@ Vaccineis able to induce B cell differentiation into antibody secreting plasma cells. Figure 27Z depicts: A) B cell surface expression of IgG and IgM following stimulation of B cells with vaccine-loaded mDCs. B) Cell surface expression of plasma cell markers 15 CD27, CD38 and CD 138 on CD19+ B cells. Example 1OZ: Malaria Conclusions Malaria Multi-antigen vaccines (3 antigen) and (4 antigen) can be cloned, expressed in Sf9 insect cells and purified by 6xHis affinity chromatography. Both vaccines can bind to immature DCs. In naYve human DC/T cell antigen presentation 20 assays, both 3 antigen and 4 antigen chimeric antigens can: induce activation and proliferation of CD8+ and CD4+ T cells, induce production of IFN-y and TNF-a in CD8+ and CD4+ T cells, induce production of GrB and perforin in CD8+ and CD4+ T cells, stimulate B cell maturation into plasma cells, increase surface CD19/CD277/CD387/CD138+ and IgG expression, and increase LDH release in DC/T 25 cell co-culture. Example 1 1Z: HIV MATERIALS - Candidate Antigens Referring to Figures 28Z and 29Z, the structure of HIV Vaccine Candidate 30 Antigens can be as follows: gag - Nucleocapsid (6 and 7 kDa proteins that tightly bind 211 viral RNA), Capsid (24 kDa protein that self assemble into a particle surrounding viral RNA), Matrix (17 KDa protein surrounding the capsid particle); env - 160 kDa protein post-translationally processed into gp120 and gp41 viral envelope glycoproteins; tat - 16 kDa protein involved in activation of transcription of proviral DNA; rev - 19 kDa protein 5 involved in shuttling of spliced viral RNA from host cell nucleus; Vpr - 14 kDa protein involved in arrest of the cell cycle at G2/M; Vpu - 16 kDa protein involved in budding of new virus from the cell. Example 12Z: HIV METHODS - DESIGN AND CLONING 10 Referring to Figure 30Z, Antigen DNA can be strategically selected to reduce size and optimize expression and immunogenicity of Chimigen* HIV Vaccine. In some embodiments, cloning can be accomplished using the Bac-to-Bac TM system (Invitrogen</div>
    <div class="description-paragraph style-scope patent-text">TM</div>
    <div class="description-paragraph style-scope patent-text">). 15 DNA sequences of the HIV gag, env, tat, rev, vpr and vpu proteins can be obtained from NCBI DNA databanks. A gene that encodes for all six HIV proteins and TBD in a single open reading frame as a fusion protein was synthesized commercially by GenScriptTM (Piscataway, NJ) and was provided cloned into a pUC57 plasmid. The 20 vaccine construct was also codon-optimized for enhanced production in Sf9 insect cells. The gene was designed to have a unique Eco RI site at the 5' end, a unique Xba I site at the junction between the HIV antigens and TBD and a unique Hind III site at the 3' end. For expression in insect cells, the HIV antigen-TBD fragment was subcloned into pFastBac-HTa containing a gp64 signal peptide sequence (pFastBacHTa-gp64). Cloning 25 required 2 steps as the synthetic gene and pFastBac-HTa plasmid are approximately the same size and cannot be separated by agarose gels. First, the pUC57-HIV vaccine plasmid was digested with Eco RI and Xba I restriction enzymes and isolated on an agarose gel. A 4.3 kb fragment of gag-env-tat-rev-vpr-vpu was isolated and extracted from the gel. This was ligated into Eco RI/Xba I digested pFastBacHTa-gp64 to get 30 pFastBacHTa-gp64-gag-vpu. Next, the pUC57-HIV vaccine plasmid was digested with Xba I and Hind III and the 0.7kb fragment of TBD was isolated and extracted from 212 agarose gels. This was cloned into Xba I/Hind III digested pFastBacHTa-gp64-gag-vpu to make the Chimigen@ HIV Vaccine. Transposition of the vaccine expressing cassette into baculoviral bacmid was performed in DH1OBac E. coli and the isolated bacmid will be transfected into Sf9 insect cells to 5 generate a P1 baculoviral stock. Following two successive rounds of amplification, P3 baculoviral stock titers will be measured and multiplicity of infection (MOI) and time of expression will be optimized for subsequent expression of Chimigen@ HIV Vaccine in sf9 insect cells for purification. 10 Example 13Z: HIV METHODS - EXPRESSION Referring to Figures 30Z and 3 1Z, IN some embodiments, expression of the chimeric antigens can be accomplished by sub-cloning DNA construct into pFastBac-HTa vector containing Baculovirus gp64 signal peptide, isolating recombinant plasmid and 15 transforming DH10BAC E.coli, isolating recombinant bacmids (Blue/White Selection), transfecting Sf9 Cells, amplifying to P3 stock, titreing and optimizing expression (MOI and time course), expressing in Wavebags T M (5L). A 10 L capacity Wave Bagim Bioreactor was used for 5 L cultures with the ESF 921 growth media. One litre of Sf9 cells was used to inoculate 4 L of ESF 921 medium to 20 make a 5 L final volume. The rocking of the Wave BagTM Bioreactor was set at 32rpm, 50 rocking angle, atmospheric air flow at 0.30 Lpm (litres per minute), and the temperature at 27.5 C. Rocking of the bag continued until the cell density reached 2.0-2.5 x 106 cells/mL. The cells were infected with the recombinant baculovirus at an MOI of 2. The 25 bioreactor was allowed to rock at 32rpm, 50 rocking angle, air flow (30% 02) of 0.30 Lpm and at 27.5 0 C. The cells were harvested at 48 hours post infection. Infected cells were harvested by centrifugation using the Beckman AvantiTM J-25 centrifuge (JA-10 rotor at 1600 x g) for 10 min, at 4 0 C. Pellets of Sf9 cells were washed with ice cold PBS, harvested by centrifugation as described above, snap frozen in liquid 30 nitrogen and stored at -80 0 C. 213 Example 14Z: HIV METHODS - PURIFICATION AND DIALYSIS Referring to Figures 32Z and 33Z, purification of 6xHis-tagged proteins can be 5 based on a two-step procedure of ammonium sulphate precipitation followed by Ni chelation affinity chromatography under denaturing conditions by FPLC. An example of a purification protocol is described below. The frozen infected Sf9 cell pellet was resuspended in lysis buffer (20 mM Tris HCl pH 9.0). Lysate was sonicated four times at 100 W for 20 seconds. Ammonium 10 sulphate to 30% was added and the lysate was stirred for 1 hour at 4'C and centrifuged. The pellet was re-suspended with 6M GuHCl, 20mM NaH 2</div>
    <div class="description-paragraph style-scope patent-text">PO</div>
    <div class="description-paragraph style-scope patent-text">4 , IM NaCl, 2% Tween 20, lOmM B-mercaptoethanol pH 9.0) and dialyzed against the same buffer to remove the ammonium sulphate. The protein was purified using an AKTA Explorer 100 FPLC system (GE Healthcare</div>
    <div class="description-paragraph style-scope patent-text">TM</div>
    <div class="description-paragraph style-scope patent-text">). Prior to purification, imidazole to 50 mM was added and the 15 pH adjusted to 8.0. Purification was performed on HisTrap FFTM columns (GE healthcare T M ) The column was washed with ten column volumes of 6 M guanidine HCl, 20 mM sodium phosphate, 0.5 M sodium chloride, 10 mM imidazole, 0.05% Tween 20, 10 mM j-mercaoptoethanol, pH 7.4 buffer and subsequently with 6 M guanidine HCl, 20 mM sodium phosphate, 20 mM imidazole, 0.05 % Tween 20, 1OmM j-mercaptoethanol, 20 pH 7.4). Finally, the vaccine was eluted with elution buffer (6 M guanidine HCl, 20 mM sodium phosphate, 60 mM imidazole, 0.05 % Tween 20, pH 7.4) and collected in ImL fractions. The fractions containing the vaccine were pooled and 10 mM j-mercaptoethanol was added. After incubating for 1 hour at room temperature the pooled vaccine was 25 transferred into a Slide-A-Lyzer cassette (Pierce) and dialyzed against IL of Redox shuffling buffer (0.5 M guanidine HCl, 0.5 M Tris, 5 mM GSH, 0.5 mM GSSG, 0.05 % Tween 20, pH 7.4) overnight at 4'C. Subsequently, dialysis against 0.5 M Tris, 0.05 % Tween 20, pH 7.4 for 12 hours was performed. The vaccine was finally dialyzed against 1 L of buffer (10 mM sodium phosphate, 0.15 M NaCl, 0.05 % Tween 20, pH 7.4). The 30 final dialysis buffer was changed three times every 4 hours. The vaccine was sterilized by 214 passing through a 0.2 [m syringe filter. The concentration of protein was determined using the Micro BCA assay. Example 15Z: HIV RESULTS - BIOCHEMICAL CHARACTERIZATION OF THE 5 CHIMERIC ANTIGENS Referring now to Fig. 34Z, blue stained SDS-PAGE gel and Western blots using antibodies against different regions of the Purified vaccine are shown. Chimigen@ HIV Vaccine was prepared for analysis by mixing purified protein 10 with 5X loading buffer and 20X reducing agent (Fermentas) and loaded onto a 7.5% TGX precast polyacrylamide gel (Bio-Rad). The gel was run at 100V constant voltage for 75 min. The gel was washed 3 times for 10 minutes with 100 mL ddH 2 0. PageBlue was added and the gel was stained for 1 hour and destained with several changes of ddH 2 0. Purified protein was separated by 7.5% SDS-PAGE, electroblotted onto Hybond 15 ECL nitrocellulose membranes at 200 mA for 1 hour and used for Western detection using 3 different primary antibodies: anti-6xHis -HRP conjugated mAb (Clontech), anti murine Fc-HRP conjugated (Sigma), and rabbit anti-HIV-1 Gag-HRP conjugated (LSBio). The binding was detected by chemiluminescence (GE Healthcare). The results presented (Fig. 34Z) show that the purified Chimigen@ HIV Vaccine is intact with 20 respect to the N-terminus, antigen and Fc domains. Example 16Z: HIV RESULTS - IMMUNOLOGICAL CHARACTERIZATION 25 Referring to Figures 35Z - 37Z, schematic diagrams are depicted which outline an Antigen Presentation Assay (APA), B-Cell differentiation, and a B-Cell Differentiation assay. Referring now to Fig. 38Z, binding of Chimigen@ HIV Vaccine to 48 hr cultured immature DCs is depicted. Immature DCs (iDCs) were incubated with the vaccine at 30 various concentrations for 1 h at 4'C, and the binding was detected with anti-mouse IgGI 215 biotin conjugated mAbs, followed by SA-PECy5. The relative amount of bound protein was determined by flow cytometry, and the data represent mean fluorescence intensity (MFI). As shown in figure 38Z, the vaccine bound to iDCs in a dose-dependent manner. Referring now to Fig. 39Z and Fig. 44Z, Chimigen@ HIV Vaccine induction of T 5 cell proliferation is depicted. Induction of functional T cell responses by the Chimigen@ HIV Multi-antigen Vaccine was assessed. The Chimigen@ HIV Multi-antigen Vaccine was assessed for its ability to induce T cell proliferation with T cells isolated from PBMCs of HIV uninfected donors. CFSE-labeled naYve T cells were stimulated in vitro with the vaccine-loaded mature DCs for 7 days. Controls used the labeled T cells 10 stimulated with mature DCs pre-treated with buffer, vaccine component (HIV protein or mTBD) or tetanus toxoid (TT). The percentage of T cells that had undergone at least one cell division was determined for the CD4+ and CD8+ populations. The Chimigen@ HIV Multi-antigen Vaccine induced a significant dose-dependent increase in percentage of proliferating CD4+ and CD8+ T cells, compared to groups treated with buffer or vaccine 15 components (Fig. 44ZA &amp; 44ZB). Moreover, the CD4+ T cells underwent a greater number of cell divisions (Fig. 44ZA) than the CD8+ T cells (Fig. 44ZB). These results demonstrate that the vaccine is able to induce T cell proliferation in T cells derived from HIV uninfected donor. Referring now to Fig. 41Z and Fig. 45Z, generation of IFN-y producing CD8+ 20 and CD4+ (CD8-) T cells following re-stimulation with the vaccine is depicted. Induction of functional T cell responses by the Chimigen@ HIV Vaccine: IFN-y and TNF-a expression was assessed. The ability of the vaccine to induce functional T cell responses was assessed by quantifying the percentage of T cells producing the Th1 cytokines IFN-y and TNF-a. Autologous naYve T cells isolated from the donor's PBMCs 25 were co-cultured for 10 days with vaccine-loaded mature DCs in the absence of exogenous IL-2 and re-stimulated for 6 h with vaccine-loaded mature DCs. The percentage of CD8+ and CD4+ T cells expressing TNF-a and IFN-y was determined by ICC labelling with fluorescently tagged antibodies and FACS analysis. There was a marked induction of T cells producing IFN-y following vaccine re-stimulation. The 30 percentage of CD8+ and CD4+ cells from the CD3+ population that expressed IFN- y is shown in Fig. 45Z A &amp; B. Compared to groups treated with buffer or vaccine 216 components, re-stimulation with the vaccine induced a dose-dependent increase in IFN-y producing CD8+ and CD4+ T cells. Compared to CD8+ T cells, a higher percentage of CD4+ T cells expressed IFN- y from vaccine group (Fig. 45Z A &amp; B). Referring now to Fig. 40Z and Fig. 46Z, increases in IFN y release induced by the 5 Chimigen@ HIV Vaccine are depicted. Native T cells were added into vaccine-loaded DC cultures or into buffer or the vaccine component (mTBD or HIV protein)-loaded DC cultures. Supernatants of the co-culture were collected on day 7, and IFN y concentrations were measured using an IFN y ELISA kit. Treatment of the vaccine induced an increase in IFN y release from the activated cells, compared to the controls 10 (Fig. 46Z). Referring now to Fig. 42Z and Fig. 47Z, generation of TNF-a producing CD8+ and CD4+ (CD8-) T cells following re-stimulation with the vaccine is depicted. Following re-stimulation with the vaccine, the production of T cell TNF-a was also investigated using an anti-TNF-a mAb with FACS analysis. The vaccine induced a dose 15 dependent increase in the percentage of TNF-a producing CD4+ T cells (Fig. 47Z A). Production of TNF-a in CD8+ T cells from vaccine-treated groups was also elevated (Fig. 47Z B). These results demonstrate that the Chimigen@ HIV Multi-antigen Vaccine induced increases in expression of IFN- y and TNF-a in both CD8+ and CD4+ T cells. Referring now to Fig. 48Z, generation of GrB and PFN producing CD8+ and 20 CD4+ (CD8-) T cells following re-stimulation with the vaccine is depicted, showing an induction of functional T cell responses by the Chimigen@ HIV Vaccine. The ability of the vaccine to induce T cell granzyme B (GrB) and perforin (PFN) was assessed following re-stimulation of T cells with vaccine-loaded DCs using anti-GrB and anti-PFN mAbs with FACS analysis. The vaccine induced significant increases in the percentage of 25 GrB and PFN producing CD4+ T cells (Fig. 48Z A &amp; C). Production of GrB and PFN in CD8+ T cells from vaccine-treated groups was also elevated (Fig. 48Z B &amp; D). These results show that the Chimigen@ HIV Multi-antigen Vaccine induces increases in GrB and PFN production in both CD8+ and CD4+ T cells. Referring now to Fig. 48Z, anti-HIV IgM antibody in supernatant (day 7,with 30 DCs) tests are depicted. 217 Example 17Z: HIV CONCLUSIONS Chimigen@ HIV Multi-antigen Vaccine (chimeric antigens) can be designed &amp; cloned and can have prophylactic and/or therapeutic applications. The vaccine can be 5 expressed in Sf9 insect cells. The vaccine can be purified. Additional purification methods and scale up methods can be used. Purified protein can be used for immune response studies, ex vivo. In some emboidments, HIV chimeric antigens can binds to immature DCs, induce activation and proliferation of CD8+ and CD4+ T cells, induce production of IFN-y and TNF-a in CD8+ and CD4+ T cells, produce antigen-specific 10 antibodies (IgM), and induce Granzyme B and Perforin in T cells. Example 18Z: CANCER ANTIGENS In some embodiments, chimeric antigens can be designed to comprise one or 15 more cancer antigens. In some embodiments, the one or more cancer antigens can comprise the IRD of the chimeric antigen. In some embodiments, the chimeric antigens have demonstrated ability to target dendritic cells, produce functional CTLs, overcome immune tolerance, overcome immune anergy, overcome antigenemia, and emulate Dendreon's strategy without the 20 individualized vaccine concept. In some emdnbodiments, the cancer antigen can comprise MUC 1 or an antigenic portion thereof MUC 1 can be over-expressed in large number of carcinomas (Breast ,Ovary, Colon, Rectum, Pancreas, Prostate), it is a Type 1 transmembrane Mucin and is expressed low on the apical surface of glandular epithelial cells. MUC 1 can be 25 expressed at very high levels following transformation Additional Cancer Targets can be used as antigen sequences can be used to comprise the IRD and elicit and immune response. See, for example, Fig. 49Z for some embodiments of Chimigen@ Cancer antigens. 30 The disclosure of U.S. Provisional Application No. 60/726,701, including all Attachments, is incorporated herein by reference in its entirety. 218 All publications, patent applications, and patents mentioned in the specification are herein incorporated by reference. Various modifications and variations of the described method and system of the invention will be apparent to those skilled in the art 5 without departing from the scope and spirit of the invention. Although the invention has been described in connection with specific preferred embodiments, it should be understood that the invention as claimed should not be unduly limited to such specific embodiments. Indeed, various modifications of the described modes for carrying out the invention which are obvious to those skilled in the art are intended to be within the scope 10 of the following claims. 219 References Amigorena, S. 2002. Fc gamma receptors and cross-presentation in dendritic cells. JExp Med 195:F1-3. 5 Arnot, D.E., D.R. Cavanagh, E.J. Remarque, A.M. Creasey, M.P. Sowa, W.D. Morgan, A.A. Holder, S. Longacre, and A.W. Thomas. 2008. Comparative testing of six antigen-based malaria vaccine candidates directed toward merozoite-stage Plasmodium falciparum. Clin Vaccine Immunol 15:1345-1355. 10 Banchereau, J., and R.M. Steinman. 1998. Dendritic cells and the control of immunity. Nature 392:245-252. Bannister, L.H., J.M. Hopkins, A.R. Dluzewski, G. Margos, I.T. Williams, M.J. Blackman, C.H.\Kocken, A.W. Thomas, and G.H. Mitchell. 2003. Plasmodium 15 falciparum apical membrane antigen 1 (PfAMA-1) is translocated within micronemes along subpellicular microtubules during merozoite development. J Cell Sci 116:3825-3834. Barale, J.C., T. Blisnick, H. Fujioka, P.M. Alzari, M. Aikawa, C. Braun-Breton, and G. 20 Langsley. 1999. Plasmodium falciparum subtilisin-like protease 2, a merozoite candidate for the merozoite surface protein 1-42 maturase. Proc Natl Acad Sci U SA 96:6445-6450. Barbosa, A., D. Naniche, J.J. Aponte, M.N. Manaca, I. Mandomando, P. Aide, J. 25 Sacarlal, M. Renom, S. Lafuente, W.R. Ballou, and P.L. Alonso. 2009. Plasmodium falciparum-specific cellular immune responses after immunization with the RTS,S/ASO2D candidate malaria vaccine in infants living in an area of high endemicity in Mozambique. Infect Immun 77:4502-4509. 30 Bargieri, D.Y., J.A. Leite, S.C. Lopes, M.E. Sbrogio-Almeida, C.J. Braga, L.C. Ferreira, I.S. Soares, F.T. Costa, and M.M. Rodrigues. 2010. Immunogenic properties of a recombinant fusion protein containing the C-terminal 19 kDa of Plasmodium falciparum merozoite surface protein-i and the innate immunity agonist FliC flagellin of Salmonella typhimurium. Vaccine 28:2818-2826. 35 Bejon, P., J. Mwacharo, 0. Kai, S. Todryk, S. Keating, B. Lowe, T. Lang, T.W. Mwangi, S.C. Gilbert, N. Peshu, K. Marsh, and A.V. Hill. 2007. The induction and persistence of T cell IFN-gamma responses after vaccination or natural exposure is suppressed by Plasmodium falciparum. JImmunol 179:4193-4201. 40 Belkaid, Y., and B.T. Rouse. 2005. Natural regulatory T cells in infectious disease. Nat Immunol 6:353-360. 220 Bergtold, A., D.D. Desai, A. Gavhane, and R. Clynes. 2005. Cell surface recycling of internalized antigen permits dendritic cell priming of B cells. Immunity 23:503 514. 5 Bueno, L.L., C.G. Morais, F.F. Araujo, J.A. Gomes, R. Correa-Oliveira, I.S. Soares, M.V. Lacerda, R.T. Fujiwara, and E.M. Braga. 2010. Plasmodium vivax: induction ofCD4+CD25+FoxP3+ regulatory T cells during infection are directly associated with level of circulating parasites. PLoS One 5:e9623. 10 Burgdorf, S., A. Kautz, V. Bohnert, P.A. Knolle, and C. Kurts. 2007. Distinct pathways of antigen uptake and intracellular routing in CD4 and CD8 T cell activation. Science 316:612-616. 15 Campton, K., W. Ding, Z. Yan, H. Ozawa, K. Seiffert, E. Miranda, A. Lonati, S. Beissert, andR.D. Granstein. 2000. Tumor antigen presentation by dermal antigen presenting cells.J Invest Dermatol 115:57-61. Casares, S., T.D. Brumeanu, and T.L. Richie. 2010. The RTS,S malaria vaccine. Vaccine 20 Clyde, D.F., H. Most, V.C. McCarthy, and J.P. Vanderberg. 1973. Immunization of man againstsporozite-induced falciparum malaria. Am JMed Sci 266:169-177. Cockburn, I.A., Y.C. Chen, M.G. Overstreet, J.R. Lees, N. van Rooijen, D.L. Farber, and F. Zavala. 2010. Prolonged antigen presentation is required for optimal CD8+ T 25 cell responses against malaria liver stage parasites. PLoS Pathog 6:e1000877. Cohen, J., S. Benns, J. Vekemans, and A. Leach. 2010. [The malaria vaccine candidate RTS,S/AS is in phase III clinical trials.]. Ann Pharm Fr 68:370-379. 30 Crawley, J., C. Chu, G. Mtove, and F. Nosten. 2010. Malaria in children. Lancet 375:1468-1481.Donnelly, J.J., J.B. Ulmer, J.W. Shiver, and M.A. Liu. 1997. DNA vaccines. Annu Rev Immunol. 15:617-648. Epp, C., C.W. Kauth, H. Bujard, and R. Lutz. 2003. Expression and purification of 35 Plasmodium falciparum MSP-1(42): A malaria vaccine candidate. J Chromatogr B Analyt Technol Biomed Life Sci 786:61-72. Fidock, D.A., Gras-Masse, H., Lepers, J. P., Brahimi, K., Benmohamed, L., Mellouk, S., Guerin-Marchand, C., Londono, A., Raharimalala, L., Meis, J. F. G. M., Lansley, 40 G., Roussilhon, C., Tartar, A., Druilhe, P. 1994. Plasmodium falciparum Liver Stage Antigen-i is Well Conserved and Contains Potent B and T Cell Determinants. JImmunol 153:190-204. Fong, L., and E.G. Engleman. 2000. Dendritic cells in cancer immunotherapy. Annu Rev 45 Immunol 18:245-273. 221 Franzese, 0., P.T. Kennedy, A.J. Gehring, J. Gotto, R. Williams, M.K. Maini, and A. Bertoletti. 2005. Modulation of the CD8+-T-cell response by CD4+ CD25+ regulatory T cells in patients with hepatitis B virus infection. J Virol 79:3322 3328. 5 Geijtenbeek, T.B., S.J. van Vliet, A. Engering, B.A. t Hart, and Y. van Kooyk. 2004. Self- and nonself-recognition by C-type lectins on dendritic cells. Annu Rev Immunol 22:33-54. 10 George, R., Gutfreund, K., Lacoste, S., Xu, B., Wand, D., Motyka, B., Fischer, K.P., Noujaim, A.A., Tyrrell, D.L.J. 2003a. A novel class of therapeutic vaccines for the treatment of chronic viral infections: Evaluation in ducks chronically infected with duck hepatitis B virus (DHBV). In HepDart 2003, Frontiers in Drug Development for Viral Hepatitis. Kauai, Hawaii, USA. 15 George, R., Ma, A., Lysechko, T., Wang, D., Motyka, B., Gutfreund, K. 2009b. Chimigen@ HCV therapeutic vaccine induces a robust immune response in chronic HCV carrier peripheral blood mononuclear cells ex vivo In 16th International Symposium on Hepatitis C Virus and Related Viruses. Nice, France. 20 George, R., Ma, A., Motyka, B., Xu, B., Polakowski, R., Szpacenko, A., Zabieglo, D., Wang, D., Eng, H., Tyrrell, D.L.J. . 2004. Immunological evaluation of a novel chimeric therapeutic vaccine for the treatment of chronic hepatitis B infections. In International Meeting of the Molecular Biology of Hepatitis B Viruses. Woods 25 Hole, MA, USA. George, R., Ma, A., Wang, D., Gutfreund, K. 2009c. Novel Dendritic cell-receptor targeted HBV and HCV therapeutic vaccines induce robust immune responses in HBV and HCV chronic carrier peripheral blood mononuclear cells ex vivo. In 30 HepDart 2009: Frontiers in Drug Development for Viral Hepatitis. The Fairmont Orchid Kohala Coast, Hawaii, USA. George, R., Ma, A., Xu, B., Wang, D., Motyka, B., Raha, S.K., Noujaim, A.A., Tyrrell, D.L.J. 2003b. Chimigen@ Vaccines: A novel class of therapeutic vaccines for the 35 treatment of chronic viral infections. In International Meeting of the Molecular Biology of Hepatitis B Viruses. Centro Congressin Giovanni XXIII, Bergamo, Italy. George, R., Motyka, B., Ma, A., Wang, D., Eng, H., Xu, B., Polakowski, R., Noujaim, 40 A.A., Tyrrell, D.L.J. 2005. A new class of therapeutic vaccines produced in insect cells for the treatment of chronic viral infections. BioProcessing J 4:39-45. George, R., Tyrrell, D.L.J., Noujaim, A.A., Ma, A., Xu, B., Wang, D., Eng, H., Polakowski, R., Szpacenko, A., Motyka, B. 2006. A new class of therapeutic 45 vaccines for the treatment of chronic hepatitis B infections. In Framing the 222 Knowledge of Therapeutics for Viral Hepatitis. R.F.a.S. Schinazi, E.R., editor IHL Press, USA, George, R., Wang, D., Ma, A., Liu, Q, Griebel, P. 2009a. Chimigen@ HBV therapeutic 5 vaccine induces robust immune responses in sheep. In International Meeting of the Molecular Biology of Hepatitis B Viruses. Tours, Loire Valley, France. Ha, S.J., E.E. West, K. Araki, K.A. Smith, and R. Ahmed. 2008. Manipulating both the inhibitory and stimulatory immune system towards the success of therapeutic 10 vaccination against chronic viral infections. Immunol Rev 223:317-333. Heppner, D.G., Jr., K.E. Kester, C.F. Ockenhouse, N. Tornieporth, 0. Ofori, J.A. Lyon, V.A. Stewart, P. Dubois, D.E. Lanar, U. Krzych, P. Moris, E. Angov, J.F. Cummings, A. Leach, B.T. Hall, S. Dutta, R. Schwenk, C. Hillier, A. Barbosa, 15 L.A. Ware, L. Nair, C.A. Darko, M.R. Withers, B. Ogutu, M.E. Polhemus, M. Fukuda, S. Pichyangkul, M. Gettyacamin, C. Diggs, L. Soisson, J. Milman, M.C. Dubois, N. Garcon, K. Tucker, J. Wittes, C.V. Plowe, M.A. Thera, O.K. Duombo, M.G. Pau, J. Goudsmit, W.R. Ballou, and J. Cohen. 2005. Towards an RTS,S based, multi-stage, multi-antigen vaccine against falciparum malaria: progress at 20 the Walter Reed Army Institute of Research. Vaccine 23:2243-2250. Hilgers, L.A., G. Lejeune, I. Nicolas, M. Fochesato, and B. Boon. 1999. Sulfolipo cyclodextrin in squalane-in-water as a novel and safe vaccine adjuvant. Vaccine 17:219-228. 25 Hillier, C.J., L.A. Ware, A. Barbosa, E. Angov, J.A. Lyon, D.G. Heppner, and D.E. Lanar. 2005. Process development and analysis of liver-stage antigen 1, a preerythrocyte-stage protein-based vaccine for Plasmodium falciparum. Infect Immun 73:2109-2115. 30 Jiang, Y., Tian, C., Ma, A., Wang, D., Petersen, N., George, R. 2009. Internalization and antigen presentation of novel therapeutic hepatitis B vaccine nanoparticles in dendritic cells. In International Conference on Nanoscience and Technology, ChinaNANO. Beijing, China. 35 Kappe, S.H., A.M. Vaughan, J.A. Boddey, and A.F. Cowman. 2010. That was then but this is now: malaria research in the time of an eradication agenda. Science 328:862-866. 40 Koussis, K., C. Withers-Martinez, S. Yeoh, M. Child, F. Hackett, E. Knuepfer, L. Juliano, U. Woehlbier, H. Bujard, and M.J. Blackman. 2009. A multifunctional serine protease primes the malaria parasite for red blood cell invasion. EMBO J 28:725-735. 223 Lai, W.C., and M. Bennett. 1998. DNA vaccines. Crit Rev Immunol 18:449-484. Larsson, M., J.F. Fonteneau, and N. Bhardwaj. 2001. Dendritic cells resurrect antigens from dead cells. Trends Immunol 22:141-148. 5 Lau, O.S., D.W. Ng, W.W. Chan, S.P. Chang, and S.S. Sun. 2010. Production of the 42 kDa fragment of Plasmodium falciparum merozoite surface protein 1, a leading malaria vaccine antigen, in Arabidopsis thaliana seeds. Plant Biotechnol J Laupeze, B., 0. Fardel, M. Onno, N. Bertho, B. Drenou, R. Fauchet, and L. Amiot. 1999. 10 Differential expression of major histocompatibility complex class Ia, Ib, and II molecules on monocytes-derived dendritic and macrophagic cells. Hum Immunol 60:591-597. Lorenz, M.G., J.A. Kantor, J. Schlom, and J.W. Hodge. 1999. Anti-tumor immunity 15 elicited by a recombinant vaccinia virus expressing CD70 (CD27L). Hum Gene Ther 10:1095-1103. Lysechko, T., Motyka, B., Wang, D., Ma, A., Suzuki, S., Banting, G., Gutfreund, K., George, R. 2008. Chimigen@ HBV therapeutic vaccine induces a robust immune 20 response in chronic HBV carrier peripheral blood mononuclear cells ex vivo. In International Meeting, The Molecular Biology of Hepatitis B Viruses. MacPherson, G., N. Kushnir, and M. Wykes. 1999. Dendritic cells, B cells and the regulation of antibody synthesis. Immunol Rev 172:325-334. 25 Moron, V.G., P. Rueda, C. Sedlik, and C. Leclerc. 2003. In vivo, dendritic cells can cross-present virus-like particles using an endosome-to-cytosol pathway. J Immunol 171:2242-2250. 30 Motyka, B., Wang, D., Ma, A., Xu, B., Noujaim, A.A., Tyrrell, D.L.J., George, R. 2004. CD8+ T cell responses to a novel class of therapeutic vaccine for the treatment of chronic hepatitis B infection. In Immunology 2004: Immunodeficiency, Infectious Diseases, Immunomodulation, and Vaccines. . Medimond International Proceedings, Bologna, Italy. 443-448. 35 Newman, K.D., P. Elamanchili, G.S. Kwon, and J. Samuel. 2002. Uptake of poly(D,L lactic-co-glycolic acid) microspheres by antigen-presenting cells in vivo. J Biomed Mater Res 60:480-486. 40 Newman, K.D., G.S. Kwon, G.G. Miller, V. Chlumecky, and J. Samuel. 2000. Cytoplasmic delivery of a macromolecular fluorescent probe by poly(d, 1-lactic co-glycolic acid) microspheres. JBiomed Mater Res 50:591-597. Newman, K.D., J. Samuel, and G. Kwon. 1998. Ovalbumin peptide encapsulated in 45 poly(d,l lactic-co-glycolic acid) microspheres is capable of inducing a T helper type 1 immune response. J Control Release 54:49-59. 224 Noujaim, A.A., B.C. Schultes, R.P. Baum, and R. Madiyalakan. 2001. Induction of CA125-specific B and T cell responses in patients injected with MAb-B43.13evidence for antibody-mediated antigen-processing and presentation of CA125 in 5 vivo. Cancer Biother Radiopharm 16:187-203. Olotu, A, Gregory Fegan, G, Juliana Wambua, J, Nyangweso, G, Awuondo, K.0, Leach, A, Lievens, M, Leboulleux, D, Njuguna, P, Peshu, N, Marsh, K, and Bejon, P. New Eng. J Med. (2013): 368, 1111-1120. 10 Ocana-Morgner, C., M.M. Mota, and A. Rodriguez. 2003. Malaria blood stage suppression of liver stage immunity by dendritic cells. JExp Med 197:143-15 1. Pan, W., D. Huang, Q. Zhang, L. Qu, D. Zhang, X. Zhang, X. Xue, and F. Qian. 2004. 15 Fusion of two malaria vaccine candidate antigens enhances product yield, immunogenicity, and antibody-mediated inhibition of parasite growth in vitro. J Immunol 172:6167-6174. Park, C.S., and Y.S. Choi. 2005. How do follicular dendritic cells interact intimately with 20 B cells in the germinal centre? Immunology 114:2-10. Peng, G., S. Li, W. Wu, Z. Sun, Y. Chen, and Z. Chen. 2008. Circulating CD4+ CD25+ regulatory T cells correlate with chronic hepatitis B infection. Immunology 123:57-65. 25 Plassmeyer, M.L., K. Reiter, R.L. Shimp, Jr., S. Kotova, P.D. Smith, D.E. Hurt, B. House, X. Zou, Y. Zhang, M. Hickman, 0. Uchime, R. Herrera, V. Nguyen, J. Glen, J. Lebowitz, A.J. Jin, L.H. Miller, N.J. MacDonald, Y. Wu, and D.L. Narum. 2009. Structure of the Plasmodium falciparum circumsporozoite protein, 30 a leading malaria vaccine candidate. JBiol Chem 284:26951-26963. Plebanski, M., K.L. Flanagan, E.A. Lee, W.H. Reece, K. Hart, C. Gelder, G. Gillespie, M. Pinder, and A.V. Hill. 1999. Interleukin 10-mediated immunosuppression by a variant CD4 T cell epitope of Plasmodium falciparum. Immunity 10:651-660. 35 Qi, H., J.G. Egen, A.Y. Huang, and R.N. Germain. 2006. Extrafollicular activation of lymph node B cells by antigen-bearing dendritic cells. Science 312:1672-1676. Regnault, A., D. Lankar, V. Lacabanne, A. Rodriguez, C. Thery, M. Rescigno, T. Saito, 40 S. Verbeek, C. Bonnerot, P. Ricciardi-Castagnoli, and S. Amigorena. 1999. Fcgamma receptor-mediated induction of dendritic cell maturation and major histocompatibility complex class I-restricted antigen presentation after immune complex internalization. JExp Med 189:371-380. 225 Rogerson, S.J., R.S. Wijesinghe, and S.R. Meshnick. 2010. Host immunity as a determinant of treatment outcome in Plasmodium falciparum malaria. Lancet Infect Dis 10:51-59. 5 Roussilhon, C., P. Brasseur, P. Agnamey, J.L. Perignon, and P. Druilhe. 2010. Understanding human-Plasmodium falciparum immune interactions uncovers the immunological role of worms. PLoS One 5:e9309. Steinman, R.M., K. Inaba, S. Turley, P. Pierre, and I. Mellman. 1999. Antigen capture, 10 processing, and presentation by dendritic cells: recent cell biological studies. Hum Immunol 60:562-567. Stoop, J.N., R.G. van der Molen, C.C. Baan, L.J. van der Laan, E.J. Kuipers, J.G. Kusters, and H.L. Janssen. 2005. Regulatory T cells contribute to the impaired 15 immune response in patients with chronic hepatitis B virus infection. Hepatology 41:771-778. Suvas, S., U. Kumaraguru, C.D. Pack, S. Lee, and B.T. Rouse. 2003. CD4+CD25+ T cells regulate virus-specific primary and memory CD8+ T cell responses. J Exp 20 Med 198:889-901.Tarte, K., and B. Klein. 1999. Dendritic cell-based vaccine: a promising approach for cancer immunotherapy. Leukemia 13:653-663. Thera, M.A., O.K. Doumbo, D. Coulibaly, M.B. Laurens, A.K. Kone, A.B. Guindo, K. Traore, M. Sissoko, D.A. Diallo, I. Diarra, B. Kouriba, M. Daou, A. Dolo, M. 25 Baby, M.S. Sissoko, I. Sagara, A. Niangaly, I. Traore, A. Olotu, 0. Godeaux, A. Leach, M.C. Dubois, W.R. Ballou, J. Cohen, D. Thompson, T. Dube, L. Soisson, C.L. Diggs, S.L. Takala, K.E. Lyke, B. House, D.E. Lanar, S. Dutta, D.G. Heppner, and C.V. Plowe. 2010. Safety and immunogenicity of an AMAl malaria vaccine in Malian children: results of a phase 1 randomized controlled trial. PLoS 30 One 5:e9041. Travassos, M.A., and M.K. Laufer. 2009. Resistance to antimalarial drugs: molecular, pharmacologic, and clinical considerations. Pediatr Res 65:64R-70R. 35 Trombetta, E.S., and I. Mellman. 2005. Cell biology of antigen processing in vitro and in vivo. Annu Rev Immunol 23:975-1028. Tuteja, R. 2007. Malaria - an overview. FEBS J274:4670-4679. 40 Urban, B.C., D.J. Ferguson, A. Pain, N. Willcox, M. Plebanski, J.M. Austyn, and D.J. Roberts. 1999. Plasmodium falciparum-infected erythrocytes modulate the maturation of dendritic cells. Nature 400:73-77. Urban, B.C., N. Willcox, and D.J. Roberts. 2001. A role for CD36 in the regulation of 45 dendritic cell function. Proc Natl Acad Sci USA 98:8750-8755. 226 Wang, D., Motyka, B., Ma, A., Xu, B., Noujaim, A.A., Tyrrell, D.L.J., George, R. 2004. Characterization of immune responses to a novel therapeutic vaccine for the treatment of chronic hepatitis B virus infections. In Immunology 2004: Immunodeficiency, Infectious Diseases, Immunomodulation, and Vaccines. . Medimond International 5 Proceedings, 511-516. Wen, Y.M., D. Qu, and S.H. Zhou. 1999. Antigen-antibody complex as therapeutic vaccine for viral hepatitis B. Int Rev Immunol 18:251-258. 10 Whiteside, T.L., J. Stanson, M.R. Shurin, and S. Ferrone. 2004. Antigen-processing machinery in human dendritic cells: up-regulation by maturation and down regulation by tumor cells. JImmunol 173:1526-1534. Whitton, J.L., M.K. Slifka, F. Liu, A.K. Nussbaum, and J.K. Whitmire. 2004. The 15 regulation and maturation of antiviral immune responses. Adv Virus Res 63:181 238. Wilson, N.S., G.M. Behrens, R.J. Lundie, C.M. Smith, J. Waithman, L. Young, S.P. Forehan, A. Mount, R.J. Steptoe, K.D. Shortman, T.F. de Koning-Ward, G.T. 20 Belz, F.R. Carbone, B.S. Crabb, W.R. Heath, and J.A. Villadangos. 2006. Systemic activation of dendritic cells by Toll-like receptor ligands or malaria infection impairs cross-presentation and antiviral immunity. Nat Immunol 7:165 172. 25 Woehlbier, U., C. Epp, C.W. Kauth, R. Lutz, C.A. Long, B. Coulibaly, B. Kouyate, M. Arevalo-Herrera, S. Herrera, and H. Bujard. 2006. Analysis of antibodies directed against merozoite surface protein 1 of the human malaria parasite Plasmodium falciparum. Infect Immun 74:1313-1322. 30 Wu, J.J., G. Chen, J. Liu, T. Wang, W. Zheng, and Y.M. Cao. 2010. Natural regulatory T cells mediate the development of cerebral malaria by modifying the pro inflammatory response. Parasitol Int 59:232-241. 35 Wykes, M., C. Keighley, A. Pinzon-Charry, and M.F. Good. 2007. Dendritic cell biology during malaria. Cell Microbiol 9:300-305. Wykes, M., A. Pombo, C. Jenkins, and G.G. MacPherson. 1998. Dendritic cells interact directly with naive B lymphocytes to transfer antigen and initiate class switching 40 in a primary T-dependent response. JImmunol 161:1313-1319. Wykes, M.N., and M.F. Good. 2008. What really happens to dendritic cells during malaria? Nat. Rev. Microbiol. 6:864-870. 227 Xue, X., F. Ding, Q. Zhang, X. Pan, L. Qu, and W. Pan. 2010. Stability and potency of the Plasmodium falciparum MSP1-19/AMA-1(III) chimeric vaccine candidate with Montanide ISA720 adjuvant. Vaccine 28:3152-3158. 5 Zheng, W., Q.H. Wang, H. Feng, J. Liu, H.R. Meng, and Y.M. Cao. 2009. CD4+CD25+Foxp3+ regulatory T cells prevent the development of Th1 immune response by inhibition of dendritic cell function during the early stage of Plasmodium yoelii infection in susceptible BALB/c mice. Folia Parasitol (Praha) 56:242-250. 10 228 INDCATONS RZATTG TO DEPClTh fCRO0CAJS ,n,.Otollnncus8y Augt"t. 200 0805841 F.. SFCTMkAN 9hlN~l~ ICTON II t. i2 229 International Depo.Itay Auhority of Canada Ted: (204) 789-2 070 National Marblologly aboatoy. Health Canado Fx:(204) 789-2007 1015 Aington Sreet Wihnipeq Maniobe Coned. R3E 3R2 I~ergmtlonal Form IDACBP/4 RECSIPT IN THE CASE OF AN OtJ0NAL. DEPOfIN {lsued purist to RuI e 7.1 of the audepest TmayRegualons) ATTACH COPIES OF THE ORIGINAL DEPOSIT CONTRACT AND VIABLTY STATEMENT Tmis Inomaono D epo yAuoily accepts fhe deposif a he fl okr'oorNwaim To (Nane of Deposilor: Irvin Grill, PhD VIeMdical Carp, Address: 1123 DtisfltyPhmay Blk, Uhersy o Albera, Edmonton, AB TBG 2N Idandlolan of DepoSbl Reference assigned by deposit: "pvAo HW, r.o t w Accession Numberfsalgedbythi0lA: 080504.01 Thu dopos iderdlWedabhowe-o--t o ascientilledosotlln (s peiy) O a poposedtIaxonom* desigrallon(spcify): SIgnolmo ofpsonl(s)fliharzsd to mpresent|DQAC: Date: Auguet05.2004 230 -~F--~ 1706DOFCT INDICATIONIULATING TO DEFOSTU1D MICROORGANISM OR OTHEER IOLOGICAL NATERAL (Cr~ulcM 13bir) &amp; The ft &amp; Ito&amp; blw rglpm~ to lhw 6vcit* mkrr3r-"Alim optr bigsi* mffntrial Trk 7 ~nW t in ft drcautton _______________ Wti 8o5 indusive R, ]DINTIECATIO1'N OF DEPOSITwlu cmlmaci~~~ f ~l~l~i lr ANdms of a&amp;od~ M~tkt~ IaiNgcI P.Wrnblol~ Lsborry, Heglth Caniada R3E SR2 Atugust 5,00 00~~ O 50402 C. ADDITIONAL INDICATIONS 6mm wa kyu4W~.L) This iWhfuwmrt is cmdtmueird am I udtbd dre Uk WM16NATE STATES J'OR WFICH INDICATIONS ARIE MAI (gfih rffdiru~ mv jiwfFllikvg~ Siam~) IL SEPAI1ATH FURNISE3UNG OF INDICATIONS 17san &amp;ochk fwri qpfg4.fo TNtm iE lu~n is r!d brlow will b: sjbEtttdl In ~tmniiom1 Bumrau lsi~r (~ ~wf~ r u4 ThLM 3 r4dwl rn1ian1o[~kk~ This *mee was reftivcd by the Intstnoioa Buman ow Autrzod afflucer Amlboimr officer FomFTRUhIM tiuly]99M: rwMT= J1awarY WA)O 231 International Depositary Athorly of Canada Te: (24} Naonal Microb igy Laoralry. Hoafhl Cawda Fax:(204) 789-2097 115Mington Stret Winnipoq, Mantoba Canada M3E 3R2 inleIaionn Form DACIPt4 EC ElPfIN THE CASEOPANMiORIGNNAL DEPOSIT gsed pursuantt IRuloI 7.1ftSedna pO tg 1 Regialgono) ATrACH COPIES OF TEOIGINAL DEPOSIT CONTACT AND VIABUTY STATEME NT Ta Inomalional DeposilutyAidhty cceptsi he dpoar of fe nlrogeriam To (Nme of Veposiftor)- Innillilh. Pb ViRonMedcIl Corp. Address: 1123 Dendry/Pharmac Bldg. Unverlly of AlbertEdoioA. B TG 2N8 Idnfilliatfortof Oeposf Releveneasflgedbydepoeilor: ;^--'c-''': ''t--.'-.</div>
    <div class="description-paragraph style-scope patent-text">Acs nlolNu~mboraossigned by hliDA- (180504-02 Thoedositidenlitled above was aer - - --. O ascieIficdescripln(pecify): f a propasedanrri desrlln (sped); SinaMIre of perna(s)ulized to repreonet IDAC Date: Auguot05, 2004 232 fflo Wol1 o-Pcr 26 8W__ __ __ __ __ B. E l3 tf~o EFB ~~~ &amp;diI~ IitcokhiDea 6lfyo2 M d NAiW rooyLbrt~.H~i Dd W~hn~g Mm233 Internatiol Doslaryutnlt odCoed Tet (204 789-270 NatW Mkrobbdty Ltroor Hoh Fmaa a:(20) "HMocu MonrwbaCnad R 3W~ (sus puan to R 7.1 olth Budaps ITO~gRlO~dstfl) ATTACN COPES OF ThE fiO.ALDePsr CMffCT i .~ 2N8 Idan~fottof - O9pos t RMsno nIrd by dshtC ,rKyXtp t lT Awsku N Mbcn2db isO O&amp;4O The dPO~IS~tIOd no2e 4 ENDECATfON'lSrLAHNGTO DEPOSITED MICROklCANESM OROTrIEliBIOLOGICAL MATERIAL egpg 26 ________ Ii= gStoiSInclus[ve B. EDEl'lliCATIONOFDEPOSn"Flr1e41?1 l iditgsid onmdidll sl'em N imf&amp;peiearyinstilhiin intamanlanal apoltr/Aullieity ofCnaodo Addicasofalmitsr -uiilken (6melgpIe e4 comral Nallanal Milembfalogy Ltobataoy, Healhth Cariada 1O15Adlnglan Strmat Winr~ie, Mariltia oworsioon A mNaiiie AuguetL52004 080504.O4 C. A~DOA O%'ICA1n0Ns (eni-b'kihadgcsM) Tale brifetusil isniludmo etLil s 0. DEIGNATED ITATI!S FOltWHCR [NDTCATIONS AflEMADE W'piid la~ menafiir~rg L |E|PAftATE FURNEEBENG OPIDLCATIONB te~ Mrkrelphii IrOrnnIeAnOlCelsOtl .fPrinkrlsimaueI uteely 6 As1cetasecevewi helirai sapl: l Thedieeericelby ishTerenaednueaun FemPCT]Rd4Mly1998; .spdJtissuay .f114) 235 International Depositry AulhOnty of Canada Tel: (204) 789-2070 National WMoroboloy Labcraloy, 4ealth Canada Fsr.(04) 7894097 1015Minn Stret Winnime. Monfloba Canada R3E Inlnatimo Form lIAC/SPM RECEPT IN THECASE OF M OIGINAL DEPOSIT (Isued pmsuant fWRue? .1 Othe budpestTrayReogulaItins) ATTACHCOPES OF THERIG NALDXE POSIT CONTRACTANID V!ABUTY STATEMENT Ths neanDep osoy A Idho *yaceps The doposfl t f6e aD~moclrgosrm To (Name of Depolor): IrwiNGlI. Pt Vh RelotMedialCop. Adress 1123 Slenfiry/Pharmacy Wdg, UniWa yoflraMc, Edmoien, AB TeG Ideniflaanonof apaft Refernoo assed bydepoltor: t.f-- R " "'-.: o AwssionNur naragned by hls DA: - 080504-04 The depositdidntl ldabovewa e ---- S ascilildeddscptIn (specly: a uprnposedwtxOnoIdelatiIon (specfy): S'gratre of pron(s)uhorized f represent IDAC: Date: Augus 06.2004 Rethsk~maOcassfanilriasos U1 - m'st e() 236 Applicars or agents Iaemaapicl N fiarefortce 17DOQ-pct INMCATIONS RELATING TO DEPOSITED MICROORCAMISM OR OTHER BIOLOGICAL MATERIAL (PCT Rule 13&amp;s) A. The indietions madt below Tle to ft depstuS mitrwgmihm cr tder&amp;biolS rmtrial refnred to in the dewi-rian on pMe 25 lif, 8 bn 15 iicsive s. iDENTFICATION OFDEPOSIT Parher dcpca ts are idemtfid on a&amp; htns] ub1t Name vfdeposhary irntiutlan iernakmal Depvrsitey Autort1y of Conada AdAres ofdeposFry mittioc octr pjaMa end ro p) National Mlrubidiogy Laboratory, Heith Canada 1015 Argion Street WMnnipeg, Man Ioba RE 3R2 Doe dapst Aaietaloo Number C- ADDPONAL INDICAT]ONS Wamw No* Van alpmaW) Tbla iutf mta l cnirmd wn u iAddhiinl alsh pI DFMGNATED STATES FOR WtilCfl IJICAfTONM ARE MADE (YrVA ASdcutouN mmae aoifr oil dinga ) L SEPARATE FURNISHING OF INDCATIONS renmtN iwqpp&amp;aN) The iJcallona lted ialw will be submitid lo he Cteloanal Biwean Inter ep9ydargneindmin &amp; urma reg" w t jigvi ama eic -ly- l!! For ltarudamal, Bu mm we onry This sat was roccirvd with the itmrnaiotipplicadem Tin shest was rmaLvedby thc lterational Borru w AvthotiJ office Autmed officer FrM PCT/RO/i34 (Uy 1998; reprint Jinuay 24) 237 Internatonal DeposltaryAuthorty of Canada Tel: (ZU) 789 (70 Nalicno Micobikgy Laboraloy, HsMh Canada Fase(204) 780-2097 1015 Adington Stot Wkv ,a, Manitoba Canada RSE 3R2 nlemafionalFormIDAC/BP/4 RECEIPTJNTHE CASE OFANOfIGIAtDEPOsIT (issued pursuant o Rue 7.1 dfihem dapes(Tre*Regiatns) ATTACH COPIES OF T ORIGNL DEPOSITT CONTRACT N) VIABIT STATEMNT This l.tmatnjep siotyAulhorky accepts he dpo m Ofthe ncorsner To (am of Depcsitrgrvdn GrIli. t D V i cMd MCorp. AdresB: 112$DsnhisbtryPharmacy BkigUnh.erniyoAtia, Edrnonton.ABTOG 2N8 Idontinaton of Doposit Reere caIslnedbydepcsfor .R. . 1 Accssor Number ssiged bythislDA: 080504-05 The dopostidntrnadaba wal=o----'-''a C ansdenliladescrip~on{specify): fl apro pGod texon Omic0dES~inaOn (spedfy)' Sinalreofperon(s)uthorizedltoresen tDAC: Dale Agust 05. 204 RoetHIpMu s Ofan0rinoLsri Ia,, 238</div>
  </div>
  </section>
      <div id="margin" class="style-scope patent-text">
        <div id="createHighlight" style="top: 0px;" class="style-scope patent-text" hidden="">
          <iron-icon id="comment" icon="editor:insert-comment" class="style-scope patent-text x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M20 2H4c-1.1 0-2 .9-2 2v12c0 1.1.9 2 2 2h14l4 4V4c0-1.1-.9-2-2-2zm-2 12H6v-2h12v2zm0-3H6V9h12v2zm0-3H6V6h12v2z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
        </div>
      </div>
    </div>
  </patent-text>
              </section>
            </div>
            <div class="flex flex-width style-scope patent-result">
              <section id="claims" class="style-scope patent-result">
                <h3 class="style-scope patent-result">
                  <div class="layout horizontal style-scope patent-result">
                    <div class="flex style-scope patent-result">Claims <span class="style-scope patent-result">(44)</span></div>
                    <div class="collapse style-scope patent-result">
                      Hide Dependent <iron-icon style="height: 15px; width: 15px;" icon="expand-less" class="style-scope patent-result x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M12 8l-6 6 1.41 1.41L12 10.83l4.59 4.58L18 14z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon><template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </div>
                  </div>
                  <div class="layout horizontal style-scope patent-result">
                    <div class="flex style-scope patent-result"></div>
                    <div class="righthead style-scope patent-result" hidden="">translated from </div>
                  </div>
                </h3>
                <patent-text name="claims" enable-text-selection="" class="style-scope patent-result" style="padding-top: 0px;">
    
    
    <template is="dom-if" restamp="" class="style-scope patent-text"></template>
    <overlay-tooltip id="tooltip" class="style-scope patent-text" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      
    </div>
  </overlay-tooltip>
    <div class="layout horizontal style-scope patent-text">
      <section id="text" class="flex style-scope patent-text"><div mxw-id="PCLM115500561" lang="EN" load-source="patent-office" class="claims style-scope patent-text">
    <div class="claim style-scope patent-text"> <div num="1" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">1. A chimeric antigen for eliciting an immune response, said chimeric antigen comprising an immune response domain and a target binding domain, wherein the immune response domain comprises one or more antigenic portions 5 of a protein from HCV, HPV, HIV, HSV, an obligate intracellular parasite or a cancer antigen and the target binding domain comprises a xenotypic Fc antibody fragment, wherein the antibody fragment comprises an immunoglobulin heavy chain fragment comprising all or a part of an antibody fragment selected from the group 10 consisting of the CH1, the hinge region, the CH 2 domain, and the CH 3 domain, and wherein the chimeric antigen is glycosylated. </div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="2" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">2. The chimeric antigen of claim 1, wherein the chimeric antigen elicits a humoral immune response, a cellular immune response, or a both humoral immune response and a cellular immune response. 15</div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="3" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">3. The chimeric antigen of either one of claims 1 or 2, wherein the chimeric antigen elicits a Th1 immune response, a Th2 immune response or both a Th1 and a Th2 immune response. </div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="4" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">4. The chimeric antigen of any one of claims 1 to 3, wherein the immune response is an in vivo immune response. 20</div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="5" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">5. The chimeric antigen of any one of claims 1 to 4, wherein the immune response domain comprises more than one protein. </div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="6" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">6. The chimeric antigen of any one of claims I to 5, wherein the immune response domain comprises one or more immunogenic portions of one or more proteins selected from the group consisting of a HCV Core (1-191) protein, a HCV Core 25 (1-177) protein, a HCV p7 protein, a HCV El protein, a HCV E2 protein, a HCV El-E2 protein, a HCV NS3 protein, a HCV NS4B protein, and a HCV NS5A protein. </div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="7" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">7. The chimeric antigen of any one of claims 1 to 5, wherein the immune response domain comprises one or more immunogenic portions of one or more 30 Plasmodium proteins. </div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="8" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">8. The chimeric antigen of any one of claims 1 to 5, wherein the immune response domain comprises one or more immunogenic portions of one or more HIV proteins. 239</div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="9" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">9. The chimeric antigen of any one of claims 1 to 5, wherein the immune response domain comprises one or more immunogenic portions of one or more cancer antigen proteins. </div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="10" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">10. The chimeric antigen of any one of claims 1 to 9, wherein the target binding 5 domain is capable of binding to an antigen presenting cell (APC). </div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="11" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">11. The chimeric antigen of any one of claims 1 to 10, further comprising one or more of a 6xHis tag, a protease cleavage site, and a linker for linking the immune response domain and the target binding domain. </div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="12" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">12. The chimeric antigen of claim 11, wherein the linker is selected from the group 10 consisting of leucine zippers, biotin bound to avidin, and a covalent peptide linkage. </div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="13" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">13. The chimeric antigen of any one of claims 1 to 12, wherein the chimeric antigen is mannose glycosylated. </div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="14" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">14. A method of delivering an antigen to an antigen presenting cell, the method 15 comprising administering to the antigen presenting cell a chimeric antigen of any one of claims I to 13. </div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="15" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">15. The method of claim 14, wherein the antigen presenting cell is a dendritic cell. </div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="16" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">16. A method of activating an antigen presenting cell, the method comprising contacting the antigen presenting cell with a chimeric antigen of any one of 20 claims I to 13. </div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="17" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">17. The method of claim 16, wherein the contacting takes place ex vivo. </div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="18" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">18. The method of claim 16, wherein the contacting takes places in vivo. </div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="19" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">19. The method of claim 18, wherein the contacting takes place in a human. </div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="20" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">20. The method of either one of claims 18 or 19, wherein the method comprises 25 administering to a subject a composition comprising a chimeric antigen of claim 1, and wherein the antigen presenting cell is in the subject. </div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="21" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">21. The method of any one of claims 17 to 20, wherein the contacting results in a humoral immune response, a cellular immune response, or both a humoral immune response and a cellular immune response. 30</div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="22" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">22. The method of claim 21 wherein the cellular immune response is one or more of a Thi response, a Th2 response, and a CTL response. </div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="23" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">23. The method of any one of claims 20 to 22, wherein the subject has, or is likely to have, an immune-treatable condition. 240</div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="24" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">24. The method of claim 23, wherein the immune-treatable condition is an acute infection. </div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="25" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">25. The method of claim 23, wherein the immune-treatable condition is a chronic infection. 5</div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="26" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">26. The method of claim 25, wherein the chronic infection is a chronic hepatitis C viral infection. </div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="27" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">27. The method of any one of claims 23 to 26, wherein the immune-treatable condition is a hepatitis C viral infection and the immune response domain comprises one or more antigenic portions of one or more proteins selected from 10 the group consisting of a HCV Core (1-191) protein, a HCV Core (1-177) protein, a HCV El protein, a HCV E2 protein, a HCV El -E2 protein, a HCV P7 protein, a HCV NS3 protein, a HCV NS4B protein, and a HCV NS5 A protein. </div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="28" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">28. The method of any one of claims 20 to 27, wherein the subject is vaccinated against a viral infection. 15</div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="29" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">29. The method of any one of claims 20 to 28, wherein the subject is prophylactically vaccinated against a viral infection. </div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="30" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">30. The method of either one of claims 28 or 29, wherein the subject is therapeutically vaccinated against an existing viral infection. </div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="31" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">31. A method of producing the chimeric antigen of any one of claims I to 13, 20 comprising: (a) providing a microorganism or a cell, the microorganism or cell comprising a polynucleotide that encodes the chimeric antigen; and (b) culturing said microorganism or cell under conditions whereby the chimeric antigen is expressed. 25</div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="32" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">32. The method of claim 31, wherein the microorganism or cell is a eukaryotic microorganism or cell. </div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="33" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">33. The method of either one of claims 31 or 32, wherein the cell is a yeast cell, a plant cell or an insect cell. </div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="34" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">34. The method of any one of claims 31 to 33, wherein the chimeric antigen is post 30 translationally modified to comprise glycosylation. </div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="35" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">35. The method of any one of claims 31 to 34, wherein the chimeric antigen is post translationally modified to comprise a mannose glycosylation. </div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="36" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">36. A polynucleotide encoding a chimeric antigen of any one of claims 1 to 13, said polynucleotide comprising a first polynucleotide portion encoding an immune 35 response domain and a second polynucleotide portion encoding a target binding domain, wherein the immune response domain comprises one or more antigenic 241 portions of a protein from HCV, HPV, HIV, HSV, an obligate intracellular parasite or a cancer antigen and the target binding domain comprises an antibody fragment. </div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="37" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">37. The polynucleotide of claim 36, wherein the antibody fragment is a xenotypic 5 antibody fragment. </div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="38" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">38. A vector comprising the polynucleotide of either one of claims 36 or 37. </div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="39" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">39. The vector of claim 38, wherein the polynucleotide is operably linked to a transcriptional regulatory element (TRE). </div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="40" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">40. A microorganism or cell comprising the polynucleotide of either one of claims 36 10 or 37. </div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="41" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">41. An article of manufacture comprising a chimeric antigen of any one of claims 1 to 13 and instructions for administering the chimeric antigen to a subject in need thereof </div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="42" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">42. A pharmaceutical composition comprising a chimeric antigen of any one of 15 claims 1 to 13 and a pharmaceutically acceptable excipient. </div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="43" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">43. A method of producing the chimeric antigen of any one of claims 1 to 13, comprising: (a) providing a microorganism or a cell, the microorganism or cell comprising a polynucleotide that encodes a target binding domain-linker molecule, wherein the 20 target-binding domain-linker molecule comprises a target binding domain bound to a linker molecule; (b) culturing said microorganism or cell under conditions whereby the target binding domain-linker molecule is expressed; and (c) contacting the target binding domain-linker molecule and an immune response 25 domain under conditions that allow for the binding of the linker to the immune response domain, the binding resulting in a chimeric antigen. </div>
    </div>
    </div> <div class="claim style-scope patent-text"> <div num="44" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">44. A chimeric antigen for eliciting an immune response; and/or a method of delivering an antigen to an antigen presenting cell; and/or a method of activating an antigen presenting cell; and/or 30 a method of producing a chimeric antigen; and/or a polynucleotide encoding a chimeric antigen; and/or a vector comprising the polynucleotide; and/or a microorganism or cell comprising the polynucleotide; and/or an article of manufacture comprising a chimeric antigen; and/or 35 a pharmaceutical composition comprising a chimeric antigen; substantially as herein described with reference to any one of the embodiments of the invention illustrated in the accompanying drawings and/or examples. 242</div>
    </div>
  </div> </div>
  </section>
      <div id="margin" class="style-scope patent-text">
        <div id="createHighlight" style="top: 0px;" class="style-scope patent-text" hidden="">
          <iron-icon id="comment" icon="editor:insert-comment" class="style-scope patent-text x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M20 2H4c-1.1 0-2 .9-2 2v12c0 1.1.9 2 2 2h14l4 4V4c0-1.1-.9-2-2-2zm-2 12H6v-2h12v2zm0-3H6V9h12v2zm0-3H6V6h12v2z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
        </div>
      </div>
    </div>
  </patent-text>
              </section>
            </div>
          </div>

          <div class="footer style-scope patent-result">

            <template is="dom-if" class="style-scope patent-result"></template>

            <template is="dom-if" class="style-scope patent-result"></template>

            <template is="dom-if" class="style-scope patent-result"></template>

            
            <h3 id="similarDocuments" class="scroll-target style-scope patent-result">
              Similar Documents
              <a href="#title" class="style-scope patent-result"><iron-icon icon="arrow-drop-up" class="style-scope patent-result x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M7 14l5-5 5 5z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon></a>
            </h3>
            <div class="responsive-table style-scope patent-result">
              <div class="table style-scope patent-result">
                <div class="thead style-scope patent-result">
                  <div class="tr style-scope patent-result">
                    <span class="th style-scope patent-result">Publication</span>
                    <span class="th style-scope patent-result">Publication Date</span>
                    <span class="th style-scope patent-result">Title</span>
                  </div>
                </div>
                <div class="tbody style-scope patent-result">
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      
                        <template is="dom-if" class="style-scope patent-result"></template>
                        
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/CA2534911C/en"><a id="link" href="#" class="style-scope state-modifier">CA2534911C</a></state-modifier>
                        <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2018-10-02</span>
                    <span class="td style-scope patent-result">Chimeric antigens for breaking host tolerance to foreign antigens 
     </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      
                        <template is="dom-if" class="style-scope patent-result"></template>
                        
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US8465745B2/en"><a id="link" href="#" class="style-scope state-modifier">US8465745B2</a></state-modifier>
                        <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2013-06-18</span>
                    <span class="td style-scope patent-result">Chimeric antigens for eliciting an immune response 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      
                        <template is="dom-if" class="style-scope patent-result"></template>
                        
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US20110223185A1/en"><a id="link" href="#" class="style-scope state-modifier">US20110223185A1</a></state-modifier>
                        <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2011-09-15</span>
                    <span class="td style-scope patent-result">Chimeric hepatitis c virus antigens for eliciting an immune response 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      
                        <template is="dom-if" class="style-scope patent-result"></template>
                        
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US8029803B2/en"><a id="link" href="#" class="style-scope state-modifier">US8029803B2</a></state-modifier>
                        <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2011-10-04</span>
                    <span class="td style-scope patent-result">Chimeric antigens for eliciting an immune response 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      
                        <template is="dom-if" class="style-scope patent-result"></template>
                        
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US9339535B2/en"><a id="link" href="#" class="style-scope state-modifier">US9339535B2</a></state-modifier>
                        <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2016-05-17</span>
                    <span class="td style-scope patent-result">Vaccines and immunopotentiating compositions and methods for making and using them 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      
                        <template is="dom-if" class="style-scope patent-result"></template>
                        
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/SK285623B6/en"><a id="link" href="#" class="style-scope state-modifier">SK285623B6</a></state-modifier>
                        <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2007-05-03</span>
                    <span class="td style-scope patent-result">Method of preparation hepatitis C virus asialoglycoprotein 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      
                        <template is="dom-if" class="style-scope patent-result"></template>
                        
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/EP1417298A2/en"><a id="link" href="#" class="style-scope state-modifier">EP1417298A2</a></state-modifier>
                        <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2004-05-12</span>
                    <span class="td style-scope patent-result">Core-glycosylated hcv envelope proteins 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="scholar/17914841204159475336"><a id="link" href="#" class="style-scope state-modifier">Crisci et al.</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2009</span>
                    <span class="td style-scope patent-result">Chimeric calicivirus-like particles elicit protective anti-viral cytotoxic responses without adjuvant</span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      
                        <template is="dom-if" class="style-scope patent-result"></template>
                        
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US11291714B2/en"><a id="link" href="#" class="style-scope state-modifier">US11291714B2</a></state-modifier>
                        <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2022-04-05</span>
                    <span class="td style-scope patent-result">Recombinant antigen derived from Zika virus E protein and use thereof 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      
                        <template is="dom-if" class="style-scope patent-result"></template>
                        
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/KR101320702B1/en"><a id="link" href="#" class="style-scope state-modifier">KR101320702B1</a></state-modifier>
                        <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2013-10-22</span>
                    <span class="td style-scope patent-result">Vaccines comprising truncated hbc core protein plus saponin-based adjuvants 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      
                        <template is="dom-if" class="style-scope patent-result"></template>
                        
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/JPH025888A/en"><a id="link" href="#" class="style-scope state-modifier">JPH025888A</a></state-modifier>
                        <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">1990-01-10</span>
                    <span class="td style-scope patent-result">Production of recombinant protein by circumsporozoid gene of hacmosporidium phalciparum 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      
                        <template is="dom-if" class="style-scope patent-result"></template>
                        
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/AU2012200998B2/en"><a id="link" href="#" class="style-scope state-modifier">AU2012200998B2</a></state-modifier>
                        <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2013-11-14</span>
                    <span class="td style-scope patent-result">Chimeric Antigens for Breaking Host Tolerance to Foreign Antigens 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      
                        <template is="dom-if" class="style-scope patent-result"></template>
                        
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/AU2013204668A1/en"><a id="link" href="#" class="style-scope state-modifier">AU2013204668A1</a></state-modifier>
                        <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2013-05-09</span>
                    <span class="td style-scope patent-result">Chimeric antigens for breaking host tolerance to foreign antigens 
     </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      
                        <template is="dom-if" class="style-scope patent-result"></template>
                        
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/AU2013203598A1/en"><a id="link" href="#" class="style-scope state-modifier">AU2013203598A1</a></state-modifier>
                        <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2013-05-02</span>
                    <span class="td style-scope patent-result">Chimeric antigen containing hepatitis c virus polypeptide and fc fragment for eliciting an immune response 
     </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="scholar/4339011989255030364"><a id="link" href="#" class="style-scope state-modifier">Reiser</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2012</span>
                    <span class="td style-scope patent-result">Identification and characterization of immunodominance hierarchies within vaccine-relevant CD8 T cell responses</span>
                  </div>
                  <template is="dom-repeat" class="style-scope patent-result"></template>
                </div>
              </div>
            </div>
            <template is="dom-if" class="style-scope patent-result"></template>

            
            <h3 id="relatedApplications" class="scroll-target style-scope patent-result">Priority And Related Applications</h3>

            
            <h3 id="parentApplications" class="scroll-target style-scope patent-result">
              Parent Applications (1)
              <a href="#title" class="style-scope patent-result"><iron-icon icon="arrow-drop-up" class="style-scope patent-result x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M7 14l5-5 5 5z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon></a>
            </h3>
            <div class="responsive-table style-scope patent-result">
              <div class="table style-scope patent-result">
                <div class="thead style-scope patent-result">
                  <div class="tr style-scope patent-result">
                    <span class="th style-scope patent-result">Application</span>
                    <span class="th style-scope patent-result">Priority date</span>
                    <span class="th style-scope patent-result">Filing date</span>
                    <span class="th style-scope patent-result">Relation</span>
                    <span class="th style-scope patent-result">Title</span>
                  </div>
                </div>
                <div class="tbody style-scope patent-result">
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/AU2012200998B2/en"><a id="link" href="#" class="style-scope state-modifier">AU2012200998A</a></state-modifier><template is="dom-if" class="style-scope patent-result"></template>
                        <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td nowrap style-scope patent-result">2003-08-08</span>
                    <span class="td nowrap style-scope patent-result">2012-02-21</span>
                    <span class="td nowrap style-scope patent-result">
                        Division<template is="dom-if" class="style-scope patent-result"></template>
                        <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">Chimeric Antigens for Breaking Host Tolerance to Foreign Antigens 
       </span>
                  </div>
                  <template is="dom-repeat" class="style-scope patent-result"></template>
                </div>
              </div>
            </div>
            <template is="dom-if" class="style-scope patent-result"></template>

            <template is="dom-if" class="style-scope patent-result"></template>

            <template is="dom-if" class="style-scope patent-result"></template>

            
            <h3 id="appsClaimingPriority" class="scroll-target style-scope patent-result">
              Applications Claiming Priority (2)
              <a href="#title" class="style-scope patent-result"><iron-icon icon="arrow-drop-up" class="style-scope patent-result x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M7 14l5-5 5 5z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon></a>
            </h3>
            <div class="responsive-table style-scope patent-result">
              <div class="table style-scope patent-result">
                <div class="thead style-scope patent-result">
                  <div class="tr style-scope patent-result">
                    <span class="th style-scope patent-result">Application</span>
                    <span class="th style-scope patent-result">Filing date</span>
                    <span class="th style-scope patent-result">Title</span>
                  </div>
                </div>
                <div class="tbody style-scope patent-result">
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                        <template is="dom-if" class="style-scope patent-result"></template>
                        US60/493,449<template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td nowrap style-scope patent-result">2003-08-08</span>
                    <span class="td style-scope patent-result"></span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/AU2012200998B2/en"><a id="link" href="#" class="style-scope state-modifier">AU2012200998A</a></state-modifier><template is="dom-if" class="style-scope patent-result"></template>
                        <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td nowrap style-scope patent-result">2012-02-21</span>
                    <span class="td style-scope patent-result">Chimeric Antigens for Breaking Host Tolerance to Foreign Antigens 
       </span>
                  </div>
                  <template is="dom-repeat" class="style-scope patent-result"></template>
                </div>
              </div>
            </div>
            <template is="dom-if" class="style-scope patent-result"></template>

            <template is="dom-if" class="style-scope patent-result"></template>

            
            <h3 id="legalEvents" class="scroll-target style-scope patent-result">
              Legal Events
              <a href="#title" class="style-scope patent-result"><iron-icon icon="arrow-drop-up" class="style-scope patent-result x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M7 14l5-5 5 5z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon></a>
            </h3>
            <div class="responsive-table style-scope patent-result">
              <div class="table style-scope patent-result">
                <div class="thead style-scope patent-result">
                  <div class="tr style-scope patent-result">
                    <span class="th style-scope patent-result">Date</span>
                    <span class="th style-scope patent-result">Code</span>
                    <span class="th style-scope patent-result">Title</span>
                    <span class="th style-scope patent-result">Description</span>
                  </div>
                </div>
                <div class="tbody style-scope patent-result">
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">2013-05-09</span>
                    <span class="td nowrap style-scope patent-result">TH</span>
                    <span class="td nowrap style-scope patent-result">Corrigenda</span>
                    <span class="td style-scope patent-result">
                      
                      <p class="style-scope patent-result"><strong class="style-scope patent-result">Free format text</strong>: IN VOL 27 , NO 16 , PAGE(S) 2093 UNDER THE HEADING COMPLETE APPLICATIONS FILED - NAME INDEX UNDER THE NAME AKSHAYA BIO INC., APPLICATION NO. 2013204668, UNDER INID (71) CORRECT THE APPLICANT NAME TO AKSHAYA BIO INC.</p>
                      <template is="dom-repeat" as="attr" class="style-scope patent-result"></template>
                    </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">2017-03-02</span>
                    <span class="td nowrap style-scope patent-result">MK4</span>
                    <span class="td nowrap style-scope patent-result">Application lapsed section 142(2)(d) - no continuation fee paid for the application</span>
                    <span class="td style-scope patent-result">
                      <template is="dom-repeat" as="attr" class="style-scope patent-result"></template>
                    </span>
                  </div>
                  <template is="dom-repeat" class="style-scope patent-result"></template>
                </div>
              </div>
            </div>
            <template is="dom-if" class="style-scope patent-result"></template>

            <template is="dom-if" class="style-scope patent-result"></template>

            <div id="notices" class="vertical layout center style-scope patent-result">
              
                <span class="style-scope patent-result">Data provided by IFI CLAIMS Patent Services</span>
              <template is="dom-if" class="style-scope patent-result"></template>
            </div>
          </div>
          <div class="bottombar style-scope patent-result">
            <bottom-bar class="style-scope patent-result"><template is="dom-if" class="style-scope bottom-bar"></template>
</bottom-bar>
          </div>
        </div>
      </div>
      <template is="dom-if" class="style-scope patent-result"></template>
    </div>
  </patent-result>
    <template is="dom-if" restamp="" class="style-scope result-container"></template>
    <template is="dom-if" restamp="" class="style-scope result-container"></template>
    <template is="dom-if" restamp="" class="style-scope result-container"></template>
  </result-container>
        </div>
      </div>
    
        </div>
      </div>
      <search-footer class="style-scope search-ui">
    
    

    <div id="footer" class="vertical layout center style-scope search-footer">
        <span class="links style-scope search-footer">
          <overlay-tooltip positioning-target="#footerAbout" direction="top" class="style-scope search-footer" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      Learn more about data coverage, search syntax and other features.
    </div>
  </overlay-tooltip>
          <a id="footerAbout" data-proto="ABOUT" href="https://support.google.com/faqs/answer/6390996" target="_blank" class="style-scope search-footer">About</a>
          <overlay-tooltip positioning-target="#footerFeedback" direction="top" class="style-scope search-footer" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      Send feedback about technical issues or feature requests for Google Patents.
    </div>
  </overlay-tooltip>
          <a id="footerFeedback" data-proto="SEND_FEEDBACK" href="#" onclick="sendFeedback();" class="style-scope search-footer">Send Feedback</a>
          <a data-proto="PUBLIC_DATASETS" href="https://console.cloud.google.com/marketplace/product/google_patents_public_datasets/google-patents-public-data" target="_blank" class="style-scope search-footer">Public Datasets</a>
          <a data-proto="TERMS" href="https://www.google.com/policies/terms/" target="_blank" class="style-scope search-footer">Terms</a>
          <a data-proto="PRIVACY_POLICY" href="https://www.google.com/privacy/privacy-policy.html" target="blank" class="style-scope search-footer">Privacy Policy</a>
          <a data-proto="SEARCH_HELP" href="https://support.google.com/websearch/" target="_blank" class="style-scope search-footer">Help</a>
        </span>
    </div>
  </search-footer>
    </div>

  </search-ui>
  </search-result>
    <template is="dom-if" restamp="" class="style-scope search-app"></template>
    <template is="dom-if" restamp="" class="style-scope search-app"></template>
    <template is="dom-if" restamp="" class="style-scope search-app"></template>
    <paper-toast id="toast" duration="3000" class="style-scope search-app x-scope paper-toast-0" aria-hidden="true" style="outline: none; display: none;">
    

    <span id="label" class="style-scope paper-toast"></span>
    
      <div id="noticeContent" class="style-scope search-app"></div>
    
  </paper-toast>
  </search-app>
    <script type="text/javascript" src="//www.gstatic.com/feedback/api.js"></script>
    <script type="text/javascript" src="//www.gstatic.com/feedback/js/help/prod/service/lazy.min.js"></script>
    <script type="text/javascript">
      if (window.help && window.help.service) {
        helpApi = window.help.service.Lazy.create(0, {apiKey: 'AIzaSyDTEI_0tLX4varJ7bwK8aT-eOI5qr3BmyI', locale: 'en-US'});
        window.requestedSurveys = new Set();
        window.requestSurvey = function(triggerId) {
          if (window.requestedSurveys.has(triggerId)) {
            return;
          }
          window.requestedSurveys.add(triggerId);
          helpApi.requestSurvey({
            triggerId: triggerId,
            enableTestingMode: false,
            callback: (requestSurveyCallbackParam) => {
              if (!requestSurveyCallbackParam.surveyData) {
                return;
              }
              helpApi.presentSurvey({
                productData: {
                  productVersion: window.version,
                  customData: {
                    "experiments": "",
                  },
                },
                surveyData: requestSurveyCallbackParam.surveyData,
                colorScheme: 1,
                customZIndex: 10000,
              });
            }
          });
        };

        window.requestSurvey('YXTwAsvoW0kedxbuTdH0RArc9VhT');
      }
    </script>
    <script src="/sw/null_loader.js"></script>
  

<iframe id="apiproxyddf475d63cd8173a637a287407211e3c7614fb050.288160019" name="apiproxyddf475d63cd8173a637a287407211e3c7614fb050.288160019" style="width: 1px; height: 1px; position: absolute; top: -100px; display: none;" src="https://scone-pa.clients6.google.com/static/proxy.html?usegapi=1&amp;jsh=m%3B%2F_%2Fscs%2Fabc-static%2F_%2Fjs%2Fk%3Dgapi.lb.en.IKZeRvoAYNY.O%2Fam%3DAAAQ%2Fd%3D1%2Frs%3DAHpOoo-SMWTzMRJrTty6iE5dL_aWGYOnuw%2Fm%3D__features__#parent=https%3A%2F%2Fpatents.google.com&amp;rpctoken=268423046" tabindex="-1" aria-hidden="true"></iframe><iron-a11y-announcer>
    
    <div class="style-scope iron-a11y-announcer" aria-live="polite"></div>
  </iron-a11y-announcer></body></html>