<html lang="en"><head><!-- Shady DOM styles for custom-style --><!-- Shady DOM styles for dom-template --><!-- Shady DOM styles for dom-repeat --><!-- Shady DOM styles for array-selector --><!-- Shady DOM styles for dom-if --><!-- Shady DOM styles for state-modifier --><style scope="state-modifier">a[inherit].state-modifier {
  color: inherit;
}</style><!-- Shady DOM styles for browser-ui --><!-- Shady DOM styles for outlined-text-field --><style scope="outlined-text-field">outlined-text-field {
  display: flex;
  font-family: "Google Sans", Roboto, Arial, sans-serif;
  font-weight: 500;
  flex-direction: column;
}

outlined-text-field span.outlined-text-field {
  display: flex;
  flex-direction: row;
  align-items: center;
}

outlined-text-field input.outlined-text-field {
  height: 40px;
  padding: 0 33px 0 16px;
  
  border-color: #b0b3b6;
  border-width: 1px;
  border-style: solid;
  
  border-radius: 8px;
  caret-color: #3c78e0;
  color: #6a6c71;
  font-size: 14px;
  flex-grow: 1;
}

outlined-text-field[orig] input.outlined-text-field {
  border-color: #e5e5e5;
  
  border-radius: 0;
}

outlined-text-field[dense] input.outlined-text-field {
  height: 32px;
}

outlined-text-field[uppercase] input.outlined-text-field {
  text-transform: uppercase;
}

outlined-text-field:not([wizard-focused]):hover input.outlined-text-field {
  border-color: #737375;
  color: #414447;
}

outlined-text-field:not([wizard-focused]):focus-within input.outlined-text-field,outlined-text-field[wizard-focused] input.outlined-text-field {
  border-color: #3574e0;
  caret-color: #80a5f1;
}

outlined-text-field[wizard-focused] input.outlined-text-field {
  border-width: 2px;
}

outlined-text-field input.outlined-text-field::placeholder {
  color: #65686c;
}

outlined-text-field[uppercase] input.outlined-text-field::placeholder {
  text-transform: none;
}</style><!-- Shady DOM styles for mat-keyword-editor --><!-- Shady DOM styles for mat-chips-input --><!-- Shady DOM styles for mat-input-chip --><!-- Shady DOM styles for pat-dialog --><style scope="pat-dialog">pat-dialog {
  background: white;
  position: absolute;
  border-radius: 8px;
  
  box-shadow: 0 24px 38px 3px rgb(0 0 0 / 14%),
              0 9px 46px 8px rgb(0 0 0 / 12%),
              0 11px 15px -7px rgb(0 0 0 / 20%);
  z-index: 1;
  display: flex;
  flex-direction: column;
}

pat-dialog:not([open]) {
  display: none;
}</style><!-- Shady DOM styles for mat-tabs --><!-- Shady DOM styles for filter-chip --><style scope="filter-chip">filter-chip {
  align-items: center;
  background-color: #fff;
  border-color: #dadce0;
  border-radius: 8px;
  border-style: solid;
  border-width: 1px;
  color: #606368;
  cursor: pointer;
  font-family: "Google Sans", Roboto, Arial, sans-serif;
  font-size: 14px;
  font-weight: 500;
  height: 32px;
  letter-spacing: 0.2px;
  padding: 0 16px;
  user-select: none;
  display: inline-flex;
  white-space: nowrap;
}

filter-chip[disabled] {
  background-color: #e8e8e8;
  border-color: #e8e8e8;
  color: #a9abad;
  cursor: auto;
}

filter-chip svg.filter-chip {
  visibility: collapse;
  font-size: 18px;
  width: 0;
  height: 18px;
}

filter-chip[selected] svg.filter-chip {
  visibility: visible;
  margin: 0 8px;
  width: 18px;
}

filter-chip[selected] svg.filter-chip path.filter-chip {
  fill: #4577d1;
}

filter-chip:not([disabled]):hover {
  background-color: #f7f8f8;
  border-color: #d7d9dc;
  color: #454649;
}

filter-chip:not([disabled]):hover:active {
  background-color: #eff0f0;
  border-color: #d2d5d8;
  color: #222326;
}

filter-chip:not([disabled]):focus {
  background-color: #e7e8e8;
  border-color: #232528;
  color: #202124;
}

filter-chip:not([disabled])[selected] {
  background-color: #e9f0fd;
  border-color: #e9f0fd;
  color: #4577d1;
  padding-left: 0;
}

filter-chip:not([disabled]):focus[selected] {
  background-color: #d5e1fb;
  border-color: #d5e1fb;
  color: #2550a0;
}

filter-chip:not([disabled]):hover[selected] {
  box-shadow: rgba(0, 0, 0, 0.2) 0px 3px 1px -2px,
    rgba(0, 0, 0, 0.14) 0px 2px 2px 0px, rgba(0, 0, 0, 0.12) 0px 1px 5px 0px;
}

filter-chip:not([disabled]):hover:active[selected] {
  background-color: #abc8f2;
  border-color: #abc8f2;
  color: #2a54a3;
}</style><!-- Shady DOM styles for figure-callout --><!-- Shady DOM styles for mat-button --><style scope="mat-button-0">.mat-button-0 {
  ;
  align-items: center;
  border-radius: 4px;
  color: hsl(217 4% 39%); 
  cursor: pointer;
  font-family: "Google Sans", Roboto, Arial, sans-serif;
  font-size: 14px;
  font-weight: 500;
  height: 36px;
  letter-spacing: 0.2px;
  padding: 0 8px;
  user-select: none;
  display: inline-flex;
  white-space: nowrap;
}

.mat-button-0[filled] {
  background-color: hsl(217 73% 54%); 
  color: white;
}

.mat-button-0[filled]:hover {
  background-color: hsl(217 60% 47%);
}

.mat-button-0[filled]:focus {
  background-color: hsl(217 59% 44%);
}

.mat-button-0[filled]:active {
  background-color: hsl(217 59% 45%);
}

.mat-button-0[filled][disabled] {
  background-color: hsl(217 2% 90%); 
  color: hsl(217 1% 65%);
}</style><!-- Shady DOM styles for search-term-wizard --><style scope="search-term-wizard">search-term-wizard:not([open]) {
  display: none;
}

search-term-wizard .header.search-term-wizard {
  border-bottom: 1px solid #e5e5e5;
  padding: 24px;
  padding-bottom: 0;
}

search-term-wizard .title.search-term-wizard {
  color: #3c4043;
  font-family: Google Sans,Roboto,Arial,sans-serif;
  font-size: 22px;
  user-select: none;
  margin: 0 48px;
}

search-term-wizard #panels.search-term-wizard > *.search-term-wizard:not([selected]) {
  display: none;
}</style><!-- Shady DOM styles for outlined-textarea --><style scope="outlined-textarea">outlined-textarea {
  display: flex;
  font-family: "Google Sans", Roboto, Arial, sans-serif;
  font-weight: 500;
  flex-direction: row;
  
  position: relative;
}

outlined-textarea span.outlined-textarea {
  display: flex;
  flex-direction: column;
  flex-grow: 1;
  justify-content: center;
}

outlined-textarea span.right-button-container.outlined-textarea {
  position: absolute;
  right: 0;
  height: 100%;
}

outlined-textarea textarea.outlined-textarea {
  padding: 16px 16px 0 16px;
  border-color: #b0b3b6;
  border-width: 1px;
  border-style: solid;
  border-radius: 8px;
  caret-color: #3c78e0;
  color: #6a6c71;
  font-size: 14px;
  font-family: "Google Sans", Roboto, Arial, sans-serif;
  font-weight: 500;
  resize: none;
}

outlined-textarea[dense] textarea.outlined-textarea {
  padding: 8px;
}

outlined-textarea[has-icon] textarea.outlined-textarea {
  padding-right: 38px;
}

outlined-textarea[orig] textarea.outlined-textarea {
  border-color: #e5e5e5;
  
  border-radius: 0;
}

outlined-textarea:not([rows]) textarea.outlined-textarea {
  height: 40px;
}

outlined-textarea textarea.resizable.outlined-textarea {
  resize: vertical;
}

outlined-textarea textarea.outlined-textarea:focus,outlined-textarea[wizard-focused] textarea.outlined-textarea,outlined-textarea textarea.outlined-textarea:focus:hover {
  border-color: #3574e0 !important;
  
  outline: none !important;
}

outlined-textarea textarea.outlined-textarea:hover {
  border-color: #202124;
  color: #414447;
}

outlined-textarea.hasValue textarea.outlined-textarea {
  border-color: #888d91;
  color: #45474a;
}

outlined-textarea #helperText.outlined-textarea {
  color: #6a6c71;
  font-size: 12px;
  letter-spacing: 0.3px;
  margin-left: 17px;
  margin-top: 6px;
  user-select: none;
}

outlined-textarea:not([helper-text]) #helperText.outlined-textarea {
  display: none;
}

outlined-textarea[error] textarea.outlined-textarea {
  border-color: #cb4b3d;
  caret-color: #ca493a;
  color: #3d4043;
}

outlined-textarea textarea.outlined-textarea::placeholder {
  font-size: 14px;
  font-weight: 400;
}

outlined-textarea:not([error]):not([disabled]) textarea.outlined-textarea::placeholder {
  color: #6a6c71;
}

outlined-textarea[error] textarea.outlined-textarea::placeholder {
  color: #c84132;
}

outlined-textarea[error] #helperText.outlined-textarea {
  color: #cd5648;
}

outlined-textarea[error] textarea.outlined-textarea:focus {
  border-color: #c84031;
  border-width: 2px;
  outline: none !important;
}

outlined-textarea[error]:not([disabled]) textarea.outlined-textarea:hover {
  border-color: #97231b;
}

outlined-textarea[error]:not([disabled]) textarea.outlined-textarea:hover::placeholder,outlined-textarea[error]:not([disabled]) textarea.outlined-textarea:hover + #helperText.outlined-textarea {
  color: #a94945;
}

outlined-textarea[disabled] textarea.outlined-textarea {
  border-color: #e9e9e9;
  color: #b5b6b8;
}

outlined-textarea[disabled] #helperText.outlined-textarea {
  color: #c4c6c7;
}

outlined-textarea[error][disabled] {
  opacity: 50%;
}

outlined-textarea[error][disabled] textarea.outlined-textarea {
  border-color: #cb4b3d;
  caret-color: #ca493a;
}

outlined-textarea[error][disabled] #helperText.outlined-textarea,outlined-textarea[error][disabled] textarea.outlined-textarea {
  color: #a94945;
}

outlined-textarea[error][disabled] textarea.outlined-textarea::placeholder {
  color: #c84132;
}</style><!-- Shady DOM styles for chemistry-search-term-wizard --><style scope="chemistry-search-term-wizard">chemistry-search-term-wizard {
  display: flex;
  flex-direction: column;
  padding: 24px;
  padding-top: 0;
}

chemistry-search-term-wizard > span.chemistry-search-term-wizard {
  display: flex;
  flex-direction: row;
  align-items: center;
  flex-wrap: wrap;
  margin-top: 16px;
}

chemistry-search-term-wizard .suggestions.chemistry-search-term-wizard {
  margin-top: 4px;
}

chemistry-search-term-wizard > span.chemistry-search-term-wizard svg.chemistry-search-term-wizard {
  margin-right: 24px;
}

chemistry-search-term-wizard path.chemistry-search-term-wizard {
  fill: #5f6368;
}

chemistry-search-term-wizard .help.chemistry-search-term-wizard {
  margin-left: 12px;
  margin-right: 0;
  cursor: pointer;
}

chemistry-search-term-wizard outlined-text-field.chemistry-search-term-wizard {
  flex-grow: 1;
}

chemistry-search-term-wizard mat-button.chemistry-search-term-wizard {
  margin-left: 48px;
}

chemistry-search-term-wizard filter-chip.chemistry-search-term-wizard {
  margin-right: 4px;
}

chemistry-search-term-wizard outlined-textarea.chemistry-search-term-wizard {
  flex-grow: 1;
}

chemistry-search-term-wizard outlined-text-field[batchMode].chemistry-search-term-wizard {
  display: none;
}

chemistry-search-term-wizard outlined-textarea.chemistry-search-term-wizard:not([batchMode]) {
  display: none;
}</style><!-- Shady DOM styles for text-search-term-wizard --><style scope="text-search-term-wizard">text-search-term-wizard {
  display: flex;
  flex-direction: column;
  padding: 24px;
  padding-top: 0;
}

text-search-term-wizard > span.text-search-term-wizard {
  display: flex;
  flex-direction: row;
  align-items: center;
  flex-wrap: wrap;
  margin-top: 16px;
}

text-search-term-wizard > span.text-search-term-wizard svg.text-search-term-wizard {
  margin-right: 24px;
}

text-search-term-wizard path.text-search-term-wizard {
  fill: #5f6368;
}

text-search-term-wizard outlined-text-field.text-search-term-wizard {
  flex-grow: 1;
}

text-search-term-wizard mat-button.text-search-term-wizard {
  margin-left: 48px;
}

text-search-term-wizard filter-chip.text-search-term-wizard {
  margin-right: 4px;
}</style><!-- Shady DOM styles for classification-wizard --><style scope="classification-wizard">classification-wizard {
  display: flex;
  flex-direction: column;
  padding: 24px;
  padding-top: 0;
  max-width: 674px;
}

classification-wizard > span.classification-wizard {
  display: flex;
  flex-direction: row;
  align-items: center;
  flex-wrap: wrap;
  margin-top: 16px;
}

classification-wizard .suggestions.classification-wizard {
  margin-top: 4px;
}

classification-wizard > span.classification-wizard svg.classification-wizard {
  margin-right: 24px;
}

classification-wizard path.classification-wizard {
  fill: #5f6368;
}

classification-wizard outlined-text-field.classification-wizard {
  flex-grow: 1;
}

classification-wizard mat-button.classification-wizard {
  margin-left: 48px;
}

classification-wizard filter-chip.classification-wizard {
  margin-right: 4px;
}

classification-wizard .no-matches.classification-wizard {
  margin-left: 64px;
}

classification-wizard .table-container.classification-wizard {
  max-height: 300px;
  overflow: scroll;
  margin-top: 8px;
}

classification-wizard .table-container.classification-wizard::-webkit-scrollbar {
  -webkit-appearance: none;
  width: 10px;
}

classification-wizard .table-container.classification-wizard::-webkit-scrollbar-thumb {
  border-radius: 5px;
  background-color: #5f636880;
  -webkit-box-shadow: 0 0 1px rgba(255,255,255,.5);
}

classification-wizard table.classification-wizard {
  margin-left: 51px;
  max-width: 575px;
}

classification-wizard tr.classification-wizard {
  margin-top: 4px;
}

classification-wizard tr.classification-wizard suggestive-chip.classification-wizard {
  margin-right: 8px;
}

classification-wizard #spinner.classification-wizard {
  justify-content: space-around;
  margin-left: 48px;
  transform: scale(0.5);
}</style><!-- Shady DOM styles for suggestive-chip --><!-- Shady DOM styles for mat-checkbox --><!-- Shady DOM styles for measure-search-term-wizard --><style scope="measure-search-term-wizard">measure-search-term-wizard {
  display: flex;
  flex-direction: column;
  padding: 24px;
  padding-top: 0;
  max-width: 674px;
}

measure-search-term-wizard > span.measure-search-term-wizard {
  display: flex;
  flex-direction: row;
  align-items: center;
  flex-wrap: wrap;
  margin-top: 16px;
}

measure-search-term-wizard .suggestions.measure-search-term-wizard {
  margin-top: 4px;
}

measure-search-term-wizard > span.measure-search-term-wizard svg.measure-search-term-wizard {
  margin-right: 24px;
}

measure-search-term-wizard path.measure-search-term-wizard {
  fill: #5f6368;
}

measure-search-term-wizard outlined-text-field.measure-search-term-wizard {
  flex-grow: 1;
}

measure-search-term-wizard mat-button.measure-search-term-wizard {
  margin-left: 48px;
}

measure-search-term-wizard filter-chip.measure-search-term-wizard {
  margin-right: 4px;
}</style><!-- Shady DOM styles for numerical-search-term-wizard --><style scope="numerical-search-term-wizard">numerical-search-term-wizard {
  display: flex;
  flex-direction: column;
  padding: 24px;
  padding-top: 0;
  max-width: 674px;
}

numerical-search-term-wizard > span.numerical-search-term-wizard {
  display: flex;
  flex-direction: row;
  align-items: center;
  flex-wrap: wrap;
  margin-top: 16px;
}

numerical-search-term-wizard .suggestions.numerical-search-term-wizard {
  margin-top: 4px;
}

numerical-search-term-wizard > span.numerical-search-term-wizard svg.numerical-search-term-wizard {
  margin-right: 24px;
}

numerical-search-term-wizard path.numerical-search-term-wizard {
  fill: #5f6368;
}

numerical-search-term-wizard outlined-text-field.numerical-search-term-wizard,numerical-search-term-wizard outlined-textarea.numerical-search-term-wizard {
  flex-grow: 1;
}

numerical-search-term-wizard mat-button.numerical-search-term-wizard {
  margin-left: 48px;
}

numerical-search-term-wizard filter-chip.numerical-search-term-wizard {
  margin-right: 4px;
}

numerical-search-term-wizard #spinner.numerical-search-term-wizard {
  justify-content: space-around;
  margin-left: 48px;
  transform: scale(0.5);
}

numerical-search-term-wizard #suggestions.numerical-search-term-wizard {
  margin-left: 58px;
  display: flex;
  flex-direction: column;
  align-items: flex-start;
}

numerical-search-term-wizard .suggestion.numerical-search-term-wizard {
  display: flex;
  flex-direction: row;
  align-items: center;
}

numerical-search-term-wizard .suggestion.numerical-search-term-wizard mat-checkbox.numerical-search-term-wizard {
  margin-right: 8px;
}</style><!-- Shady DOM styles for text-button --><style scope="text-button">text-button {
  align-items: center;
  border-radius: 4px;
  cursor: pointer;
  font-family: "Google Sans", Roboto, Arial, sans-serif;
  font-size: 14px;
  font-weight: 500;
  height: 36px;
  letter-spacing: .25px;
  margin: 6px 0;
  padding: 0 8px;
  user-select: none;
  display: inline-flex;
  white-space: nowrap;
  line-height: 36px;
  text-decoration: none;
  text-transform: none;
  color: #5f6368;
}

text-button:hover {
  background-color: #f8f9fa;
}

text-button:active {
  background-color: #d5e2fa;
}

text-button[selected] {
  background-color: #e8f0fe;
  color: #1967d2;
}

text-button[selected]:active {
  background-color: #d2e3fc;
}</style><!-- Shady DOM styles for mat-spinner --><!-- Shady DOM styles for requery-tile --><style scope="requery-tile">requery-tile {
  display: flex;
  justify-content: space-between;
  align-items: center;
  box-sizing: border-box;
  min-height: 40px;
  padding: 8px 12px 8px 8px;
  gap: 8px;

  border-radius: 16px;
  background: #f7f8f9;
  color: #1f1f1f;
}

requery-tile:hover {
  background: #d7d8d9;
}

requery-tile > [slot="image"] {
  flex-shrink: 0;

  border-radius: 8px;

  background: #fff;
}

.label.requery-tile {
  flex-grow: 999;

  font: 16px/20px "Google Sans", Roboto, Arial, sans-serif;
  font-weight: 500;
}

.search-icon.requery-tile {
  flex-shrink: 0;

  width: 24px;
  height: 24px;
  background-image: url('https://fonts.gstatic.com/s/i/short-term/release/googlesymbols/search/default/24px.svg');
}</style><!-- Shady DOM styles for s-more-inline --><style scope="s-more-inline">s-more-inline {
  display: block;
  margin-top: 24px;
  padding: 16px;
  padding-top: 0;
  position: relative;

  line-height: 18px;
  text-overflow: ellipsis;
  white-space: nowrap;
  cursor: pointer;
}

hr.s-more-inline {
  background-color: #dadce0;
  margin-top: 18px;
  position: absolute;
  border: 0;
  height: 1px;
  left: 0;
  width: 100%;
}

.button.s-more-inline {
  display: flex;
  align-items: center;
  justify-content: center;

  position: relative; 
  margin: 0 auto;

  font: 400 14px Roboto, arial, sans-serif;
  height: 36px;
  width: 300px;
  border-radius: 18px;
  background-color: #f1f3f4;
}

.button.s-more-inline:hover {
  background-color: #d8d7dc;
}

.button.s-more-inline > [slot] {
  padding-right: 8px;
}

.expand.s-more-inline {
  display: inline-block;
  width: 20px;
  height: 20px;
}

.expand.less.s-more-inline {
  background-image: url('https://fonts.gstatic.com/s/i/short-term/release/googlesymbols/expand_less/default/20px.svg');
}

.expand.more.s-more-inline {
  background-image: url('https://fonts.gstatic.com/s/i/short-term/release/googlesymbols/expand_more/default/20px.svg');
}</style><!-- Shady DOM styles for workspace-ui-search --><!-- Shady DOM styles for keyword-wizard --><style scope="keyword-wizard">keyword-wizard {
  display: inline-block;
}

iron-icon.keyword-wizard {
  height: 20px;
    width: 20px;
    color: #999;
    cursor: pointer;
}

iron-icon[active].keyword-wizard {
  color: #4285f4 !important;
}

iron-icon.keyword-wizard:hover {
  color: #ccc;
}

#wizardIcon.keyword-wizard {
  padding-left: 1px;
    height: 16px;
    width: 16px;
    margin: 4px 1px 1px 1px;
}

#wizard.keyword-wizard {
  padding: 0;
    left: 0;
    top: 37px;
    position: absolute;
    background-color: white;
    z-index: 2;
    overflow: hidden;
}

.wizard-header.keyword-wizard {
  display: block;
    width: 100%;
    color: #737373;
    text-transform: uppercase;
    font-size: 13px;
    margin: 16px 0 6px 12px;
}

.filter-chips.keyword-wizard {
  margin-left: 20px;
}

.chip.keyword-wizard {
  background-color: #ebebeb;
    color: #6a6a6a;
    display: inline-block;
    font-size: 13px;
    font-family: inherit;
    border-radius: 12px;
    padding: 6px 12px;
}

.chip.keyword-wizard:hover {
  cursor: pointer;
}

.chip[selected].keyword-wizard {
  background-color: #4285f4;
    color: white;
}

.input-label.keyword-wizard {
  display: inline-block;
    font-size: 13px;
    font-family: inherit;
}

.keyword-input.keyword-wizard {
  display: inline-block;
    border-top: 0;
    border-left: 0;
    border-right: 0;
    border-bottom: 1px solid #dfe1e5;
    padding: 3px;
}

.keyword-input.keyword-wizard::placeholder {
  font-style: italic;
}

.keyword-input.keyword-wizard:hover,.keyword-input.keyword-wizard:focus {
  border-color: #4284f4;
    border-width: 1px;
    outline: none;
}

.cpc-field.keyword-wizard {
  display: inline-block;
    margin: 6px 0 0 24px;
}

#checkboxCPC.keyword-wizard:hover {
  cursor: pointer;
}

.checkbox-label.keyword-wizard {
  font-size: 13px;
    font-style: italic;
    color: #737373;
}

.chemistry-field.keyword-wizard {
  display: block;
}

#inputChem.input-batch.keyword-wizard {
  height: 150px;
    border: 1px solid #dfe1e5;
    padding: 6px;
}

#inputChem.keyword-wizard {
  resize: none;
    height: 15px;
    vertical-align: text-top;
    font-size: 13px;
    font-family: inherit;
    padding: 2px;
}

.chem-type-label.keyword-wizard {
  display: inline;
    font-size: 13px;
    margin-left: 12px;
    margin-right: 4px;
}

#chemType.keyword-wizard {
  border-top: 0;
    border-left: 0;
    border-right: 0;
    border-bottom: 1px solid #dfe1e5;
    margin: 6px;
    padding: 2px;
    outline: none;
    font-family: inherit;
    font-size: 13px;
}

.import.keyword-wizard {
  font-size: 13px;
    font-family: inherit;
}

.import-text.keyword-wizard {
  vertical-align: middle;
    display: inline-block;
}

.import-button.keyword-wizard {
  margin: 6px 0 0 24px;
}

.import-button.keyword-wizard button.keyword-wizard {
  border: 0;
    border-radius: 4px;
    padding: 6px 12px;
    font-family: inherit;
    background-color: #4284f4;
    color: white;
}

.import-button.keyword-wizard button.keyword-wizard:hover {
  cursor: pointer;
    background-color: #4284f4;
    color: white;
}

.wizard-buttons.keyword-wizard {
  float: right;
    padding: 12px;
}

.wizard-buttons.keyword-wizard button.keyword-wizard {
  border: 0;
    border-radius: 4px;
    padding: 6px 12px;
    margin: 2px;
    font-family: inherit;
    background-color: #4284f4;
    color: white;
}

.wizard-buttons.keyword-wizard button.keyword-wizard:hover {
  cursor: pointer;
}

@media screen and (max-width: 700px) {
#wizard.keyword-wizard {
  width: 275px;
}

.chip.keyword-wizard {
  margin: 4px 2px;
}

.keyword-input.keyword-wizard {
  width: 82%;
      margin: 4px 12px 12px 24px;
}

.search-within.keyword-wizard .input-label.keyword-wizard {
  display: none;
}

.cpc-field.keyword-wizard .keyword-input.keyword-wizard {
  width: 225px;
      margin: 12px 0 6px 0;
}

.find-patents.keyword-wizard .input-label.keyword-wizard {
  display: none;
}

.search-by.keyword-wizard .input-label.keyword-wizard {
  width: 55px;
      margin-left: 24px;
}

.search-by.keyword-wizard .keyword-input.keyword-wizard {
  margin-top: 6px;
}

#inputChem.keyword-wizard {
  width: 82%;
      padding: 2px;
      margin: 8px 12px 12px 24px;
}

#inputChem.input-batch.keyword-wizard {
  width: 220px;
      padding: 4px;
}

#chemType.keyword-wizard {
  width: 150px;
}

.import-text.keyword-wizard {
  margin: 6px 12px 6px 24px;
      width: 220px;
}

iron-icon.help-header.keyword-wizard {
  display: none;
}

}

@media screen and (min-width: 700px) {
#wizard.keyword-wizard {
  width: 540px;
}

.chip.keyword-wizard {
  margin: 0 2px;
}

.cpc-field.keyword-wizard .keyword-input.keyword-wizard {
  width: 240px;
      margin: 12px 16px 6px 6px;
}

.cpc-inputs.keyword-wizard {
  display: inline;
}

.find-patents.keyword-wizard .keyword-input.keyword-wizard {
  width: 53%;
      margin: 4px 10px;
}

.cpc-field.keyword-wizard .keyword-input.keyword-wizard::placeholder,#inputAnd.keyword-wizard::placeholder,#inputExact.keyword-wizard::placeholder,#inputOr.keyword-wizard::placeholder,#inputMinus.keyword-wizard::placeholder {
  color: white;
}

.find-patents.keyword-wizard .input-label.keyword-wizard {
  width: 35%;
      margin-left: 24px;
}

.search-by.keyword-wizard .keyword-input.keyword-wizard {
  width: 42%;
      margin: 6px 4px;
}

.search-by.keyword-wizard .input-label.keyword-wizard {
  width: 55px;
      margin-left: 24px;
}

#inputChem.keyword-wizard {
  width: 290px;
      margin: 2px 12px 2px 6px;
}

#inputChem.input-batch.keyword-wizard {
  width: 280px;
      padding: 6px;
}

.import-text.keyword-wizard {
  margin: 6px 12px;
      width: 415px;
}

#chemType.keyword-wizard {
  width: 105px;
}

iron-icon.keyword-wizard:hover.help-header {
  cursor: default;
}

iron-icon.help-header.keyword-wizard {
  height: 16px;
      padding: 0 2px 2px 2px;
      color: #a3a3a3;
}

.help.keyword-wizard p.keyword-wizard {
  margin: 0 0 8px 0
}

.help.keyword-wizard .q.keyword-wizard {
  font-family: monospace;
      color: #0b8043;
      white-space: nowrap;
}

.help.keyword-wizard {
  z-index: 4;
      position: absolute;
      right: 10px;
      padding: 12px;
      border-radius: 2px;
      border: 1px solid #e0e0e0;
      box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.2), 0 6px 10px 0 rgba(0, 0, 0, 0.3);
      background: white;
      font-size: 13px;
      line-height: 20px;
      width: 450px;
      max-height: 400px;
      overflow: auto;
}

#withinHelp.keyword-wizard {
  top: 75px;
}

#chemHelp.keyword-wizard {
  top: 0;
}

#searchBooleanHelp.keyword-wizard {
  top: 140px;
}

}</style><!-- Shady DOM styles for bottom-bar --><style scope="bottom-bar">.fixed.bottom-bar {
  display: block;
  position: fixed;
  height: 46px;
  bottom: 0;
}

.fixed[medium].bottom-bar {
  height: 280px;
}

.fixed[large].bottom-bar {
  height: 480px;
}

.shadow.bottom-bar {
  background: white;
  padding: 16px;
  border-radius: 8px 8px 0px 0px;
  box-shadow: 0px 8px 10px 1px rgba(0,0,0,.14), 0px 3px 14px 2px rgba(0,0,0,.12), 0px 5px 5px -3px rgba(0,0,0,.2);
}

.oneline.bottom-bar {
  white-space: nowrap;
  overflow: hidden;
  text-overflow: ellipsis;
  cursor: pointer;
}

.button.bottom-bar {
  cursor: pointer;
}

.buttonAlign.bottom-bar {
  margin-top: -4px;
}

.header.bottom-bar {
  height: 30px;
  font-family: 'Google Sans';
  font-size: 18px;
  padding-bottom: 8px;
}

.list.bottom-bar {
  overflow-y: auto;
  line-height: 20px;
}

.definition.bottom-bar {
  padding: 0px 0px 4px 0px;
  cursor: pointer;
}

[medium].bottom-bar .content.bottom-bar {
  height: 230px;
}

[large].bottom-bar .content.bottom-bar {
  height: 430px;
}

[medium].bottom-bar .content.bottom-bar .list.bottom-bar {
  height: 200px;
}

[large].bottom-bar .content.bottom-bar .list.bottom-bar {
  height: 400px;
}

[hidden].bottom-bar {
  display: none;
}

.label.bottom-bar {
  font-family: 'Google Sans';
  font-size: 12px;
  line-height: 16px;
  font-weight: 400;
  padding: 6px 0px 4px 0px;
  color: #1f1f1f;
}

[pad].bottom-bar {
  margin-left: 16px;
}

.layout.vertical.bottom-bar {
  display: flex;
  flex-direction: column;
}

.layout.horizontal.bottom-bar {
  display: flex;
  flex-direction: row;
}

.flex.bottom-bar {
  flex: 1;
  flex-basis: 0.000000001px;
}

.layout.start.bottom-bar {
  align-items: flex-start;
}

.layout.center.bottom-bar {
  align-items: center;
}

.layout.end.bottom-bar {
  align-items: flex-end;
}

.flex-2.bottom-bar {
  flex: 2;
}

.flex-3.bottom-bar {
  flex: 3;
}

.flex-4.bottom-bar {
  flex: 4;
}

.flex-5.bottom-bar {
  flex: 5;
}

.flex-6.bottom-bar {
  flex: 6;
}

.flex-7.bottom-bar {
  flex: 7;
}

.flex-8.bottom-bar {
  flex: 8;
}

.flex-9.bottom-bar {
  flex: 9;
}

.flex-10.bottom-bar {
  flex: 10;
}

.flex-11.bottom-bar {
  flex: 11;
}

.flex-12.bottom-bar {
  flex: 12;
}</style><!-- Shady DOM styles for landscape-link --><!-- Shady DOM styles for landscape-summary-box --><!-- Shady DOM styles for search-state --><!-- Shady DOM styles for iron-overlay-backdrop --><!-- Shady DOM styles for opaque-animation --><!-- Shady DOM styles for iron-dropdown --><style scope="iron-dropdown">iron-dropdown {
  position: fixed;
}

#contentWrapper.iron-dropdown  > * {
  overflow: auto;
}

#contentWrapper.animating.iron-dropdown  > * {
  overflow: hidden;
}</style><!-- Shady DOM styles for iron-meta --><!-- Shady DOM styles for iron-icon --><style scope="iron-icon-0">.iron-icon-0 {
  display: -ms-inline-flexbox;
        display: -webkit-inline-flex;
        display: inline-flex;-ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;-ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
        position: relative;

        vertical-align: middle;

        fill: currentcolor;
        stroke: none;

        width: 24px;
        height: 24px;;
}

.iron-icon-0[hidden] {
  display: none;
}</style><!-- Shady DOM styles for iron-iconset-svg --><!-- Shady DOM styles for iron-list --><!-- Shady DOM styles for paper-icon-item --><!-- Shady DOM styles for paper-item-body --><!-- Shady DOM styles for paper-item --><!-- Shady DOM styles for dropdown-menu --><style scope="dropdown-menu">dropdown-menu {
  display: inline-block;
        line-height: 24px;
}

.label.dropdown-menu {
  cursor: pointer;
        color: #666;
        align-items: center;
        margin-right: 8px;
        display: inline-block;
        padding: 2px 0;
}

#selected.dropdown-menu {
  color: #212121;
}

.label[open].dropdown-menu {
  border-color: #e0e0e0;
}

iron-dropdown.dropdown-menu {
  margin-top: 26px;
}

iron-icon.dropdown-menu {
  height: 15px;
        width: 15px;
}

#menu.dropdown-menu {
  max-height: 300px;
        padding: 8px 0;
        background: white;
        display: block;
        border-radius: 2px;
        box-shadow: 0 0 2px rgba(0,0,0,0.12), 0 2px 2px rgba(0,0,0,0.24);
}

.item.dropdown-menu {
  cursor: pointer;
        color: rgba(0,0,0,.87);
        font-size: 13px;
        line-height: 18px;
        padding: 6px 18px;
}

.item.dropdown-menu:hover {
  background-color: #e0e0e0;
}

.item[selected].dropdown-menu {
  background-color: #e0e0e0;
}

[multi].dropdown-menu .item.dropdown-menu {
  padding: 6px 18px 6px 34px;
}

[multi].dropdown-menu .item[selected].dropdown-menu {
  background-image: url('https://www.gstatic.com/images/icons/material/system/2x/done_grey600_24dp.png');
        background-position: 8px center;
        background-size: 16px;
        background-repeat: no-repeat;
}</style><!-- Shady DOM styles for chips-input --><!-- Shady DOM styles for input-chip --><!-- Shady DOM styles for keyword-editor --><!-- Shady DOM styles for paper-ripple --><!-- Shady DOM styles for paper-button --><!-- Shady DOM styles for date-editor --><!-- Shady DOM styles for metadata-editor --><!-- Shady DOM styles for restrict-editor --><!-- Shady DOM styles for overlay-tooltip --><style scope="overlay-tooltip">overlay-tooltip {
  display: none;
        color: #fff;
        background-color: #000;
        padding: 10px 12px;
        border-radius: 3px;
        font-size: 13px;
        font-weight: normal;
        line-height: normal;
        position: fixed;
        z-index: 100;
}

#arrow.overlay-tooltip {
  width: 0;
        height: 0;
        border-style: solid;
        border-width: 0 6px 8px 6px;
        border-color: transparent transparent #000 transparent;
        position: absolute;
        top: -8px;
        left: calc(50% - 6px);
}</style><!-- Shady DOM styles for query-renderer --><!-- Shady DOM styles for result-nav --><style scope="result-nav">result-nav {
  display: block;
        white-space: nowrap;
        padding: 0 30px;
        line-height: 30px;
        height: 30px;
        margin-top: 17px;
        font-weight: 500;
        font-size: 15px;
}

state-modifier.result-nav {
  color: #333;
}

state-modifier[disabled].result-nav {
  pointer-events: none;
        color: #ccc;
}</style><!-- Shady DOM styles for raw-html --><!-- Shady DOM styles for paper-material --><style scope="paper-material-0">.paper-material-0 {
  display: block;
        position: relative;
}

.paper-material-0[elevation="1"] {
  box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.14),
                    0 1px 5px 0 rgba(0, 0, 0, 0.12),
                    0 3px 1px -2px rgba(0, 0, 0, 0.2);
}

.paper-material-0[elevation="2"] {
  box-shadow: 0 4px 5px 0 rgba(0, 0, 0, 0.14),
                    0 1px 10px 0 rgba(0, 0, 0, 0.12),
                    0 2px 4px -1px rgba(0, 0, 0, 0.4);
}

.paper-material-0[elevation="3"] {
  box-shadow: 0 6px 10px 0 rgba(0, 0, 0, 0.14),
                    0 1px 18px 0 rgba(0, 0, 0, 0.12),
                    0 3px 5px -1px rgba(0, 0, 0, 0.4);
}

.paper-material-0[elevation="4"] {
  box-shadow: 0 8px 10px 1px rgba(0, 0, 0, 0.14),
                    0 3px 14px 2px rgba(0, 0, 0, 0.12),
                    0 5px 5px -3px rgba(0, 0, 0, 0.4);
}

.paper-material-0[elevation="5"] {
  box-shadow: 0 16px 24px 2px rgba(0, 0, 0, 0.14),
                    0  6px 30px 5px rgba(0, 0, 0, 0.12),
                    0  8px 10px -5px rgba(0, 0, 0, 0.4);
}

.paper-material-0[animated] {
  transition: box-shadow 0.28s cubic-bezier(0.4, 0, 0.2, 1);
}

.paper-material-0 {
  ;
}</style><!-- Shady DOM styles for search-box --><style scope="search-box">.layout.horizontal.search-box,.layout.vertical.search-box {
  display: -ms-flexbox;
        display: -webkit-flex;
        display: flex;
}

.layout.inline.search-box {
  display: -ms-inline-flexbox;
        display: -webkit-inline-flex;
        display: inline-flex;
}

.layout.horizontal.search-box {
  -ms-flex-direction: row;
        -webkit-flex-direction: row;
        flex-direction: row;
}

.layout.vertical.search-box {
  -ms-flex-direction: column;
        -webkit-flex-direction: column;
        flex-direction: column;
}

.layout.wrap.search-box {
  -ms-flex-wrap: wrap;
        -webkit-flex-wrap: wrap;
        flex-wrap: wrap;
}

.layout.no-wrap.search-box {
  -ms-flex-wrap: nowrap;
        -webkit-flex-wrap: nowrap;
        flex-wrap: nowrap;
}

.layout.center.search-box,.layout.center-center.search-box {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.layout.center-justified.search-box,.layout.center-center.search-box {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.flex.search-box {
  -ms-flex: 1 1 0.000000001px;
        -webkit-flex: 1;
        flex: 1;
        -webkit-flex-basis: 0.000000001px;
        flex-basis: 0.000000001px;
}

.flex-auto.search-box {
  -ms-flex: 1 1 auto;
        -webkit-flex: 1 1 auto;
        flex: 1 1 auto;
}

.flex-none.search-box {
  -ms-flex: none;
        -webkit-flex: none;
        flex: none;
}

.inputbox.search-box {
  border: 1px solid #ccc;
        padding: 0 12px;
        background: white;
}

.inputbox[focused].search-box {
  border: 1px solid #4285f4 !important;
}

.inputbox.search-box:hover {
  border: 1px solid #999;
}

#searchInput.search-box {
  font-family: inherit;
        font-size: 16px;
        padding: 8px 0;
        outline: none;
        border-right: none;
        width: 100%;
        margin-right: 0;
        border: 0;
}

#searchButton.search-box {
  cursor: pointer;
        background-color: #4285f4;
        border: none;
        color: #fff;
        vertical-align: top;
        width: 70px;
        height: 37px;
        outline: none;
}

.suggest-bg.search-box {
  display: inline-block;
        min-width: 0;
        white-space: nowrap;
}

#suggestMenu.search-box {
  position: absolute;
        margin-top: 38px;
        width: 100%;
        background: white;
        z-index: 1;
}

#suggestMenu.search-box .row.search-box {
  font-size: 13px;
        line-height: 28px;
        white-space: nowrap;
        overflow: hidden;
        text-overflow: ellipsis;
        padding: 4px 2px;
        cursor: pointer;
}

.row[selected].search-box {
  background: rgba(0, 0, 0, 0.05);
}

.suggest-icon.search-box {
  height: 20px;
        width: 20px;
        padding: 0 12px 0 6px;
        transition: transform .15s linear;
        transform: scale(1);
}

.suggest-icon[hide].search-box {
  transform: scale(0);
}

#corpora.search-box {
  padding: 6px 0;
        user-select: none;
}

#corpora.search-box iron-icon.search-box {
  height: 20px;
        width: 20px;
        color: #ccc;
        cursor: pointer;
}

#corpora.search-box iron-icon[active].search-box {
  color: #4285f4 !important;
}

#corpora.search-box iron-icon.search-box:hover {
  color: #999;
}</style><!-- Shady DOM styles for iron-media-query --><!-- Shady DOM styles for paper-header-panel --><!-- Shady DOM styles for iron-a11y-announcer --><style scope="iron-a11y-announcer">iron-a11y-announcer {
  display: inline-block;
        position: fixed;
        clip: rect(0px,0px,0px,0px);
}</style><!-- Shady DOM styles for iron-autogrow-textarea --><!-- Shady DOM styles for iron-input --><!-- Shady DOM styles for paper-input-char-counter --><!-- Shady DOM styles for paper-input-container --><!-- Shady DOM styles for paper-input-error --><!-- Shady DOM styles for paper-input --><!-- Shady DOM styles for paper-textarea --><!-- Shady DOM styles for paper-toast --><style scope="paper-toast-0">.paper-toast-0 {
  display: block;
        position: fixed;
        background-color: #323232;
        color: #f1f1f1;
        min-height: 48px;
        min-width: 288px;
        padding: 16px 24px;
        box-sizing: border-box;
        box-shadow: 0 2px 5px 0 rgba(0, 0, 0, 0.26);
        border-radius: 2px;
        margin: 12px;
        font-size: 14px;
        cursor: default;
        -webkit-transition: -webkit-transform 0.3s, opacity 0.3s;
        transition: transform 0.3s, opacity 0.3s;
        opacity: 0;
        -webkit-transform: translateY(100px);
        transform: translateY(100px);font-family: 'Roboto', 'Noto', sans-serif;
        -webkit-font-smoothing: antialiased;
}

.paper-toast-0.capsule {
  border-radius: 24px;
}

.paper-toast-0.fit-bottom {
  width: 100%;
        min-width: 0;
        border-radius: 0;
        margin: 0;
}

.paper-toast-0.paper-toast-open {
  opacity: 1;
        -webkit-transform: translateY(0px);
        transform: translateY(0px);
}</style><!-- Shady DOM styles for paper-toolbar --><!-- Shady DOM styles for search-footer --><style scope="search-footer">#footer.search-footer {
  font-size: 13px;
        margin-top: 10px;
        background: #fafafa;
        padding: 20px 10px;
        align-items: center;
        border-top: 1px solid #ebebeb;
}

#footer.search-footer .links.search-footer > *.search-footer {
  display: inline-block;
        margin: 0 12px;
}

.layout.horizontal.search-footer,.layout.vertical.search-footer {
  display: -ms-flexbox;
        display: -webkit-flex;
        display: flex;
}

.layout.inline.search-footer {
  display: -ms-inline-flexbox;
        display: -webkit-inline-flex;
        display: inline-flex;
}

.layout.horizontal.search-footer {
  -ms-flex-direction: row;
        -webkit-flex-direction: row;
        flex-direction: row;
}

.layout.vertical.search-footer {
  -ms-flex-direction: column;
        -webkit-flex-direction: column;
        flex-direction: column;
}

.layout.wrap.search-footer {
  -ms-flex-wrap: wrap;
        -webkit-flex-wrap: wrap;
        flex-wrap: wrap;
}

.layout.no-wrap.search-footer {
  -ms-flex-wrap: nowrap;
        -webkit-flex-wrap: nowrap;
        flex-wrap: nowrap;
}

.layout.center.search-footer,.layout.center-center.search-footer {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.layout.center-justified.search-footer,.layout.center-center.search-footer {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.flex.search-footer {
  -ms-flex: 1 1 0.000000001px;
        -webkit-flex: 1;
        flex: 1;
        -webkit-flex-basis: 0.000000001px;
        flex-basis: 0.000000001px;
}

.flex-auto.search-footer {
  -ms-flex: 1 1 auto;
        -webkit-flex: 1 1 auto;
        flex: 1 1 auto;
}

.flex-none.search-footer {
  -ms-flex: none;
        -webkit-flex: none;
        flex: none;
}

.layout.start.search-footer {
  -ms-flex-align: start;
        -webkit-align-items: flex-start;
        align-items: flex-start;
}

.layout.center.search-footer,.layout.center-center.search-footer {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.layout.end.search-footer {
  -ms-flex-align: end;
        -webkit-align-items: flex-end;
        align-items: flex-end;
}

.layout.baseline.search-footer {
  -ms-flex-align: baseline;
        -webkit-align-items: baseline;
        align-items: baseline;
}

.layout.start-justified.search-footer {
  -ms-flex-pack: start;
        -webkit-justify-content: flex-start;
        justify-content: flex-start;
}

.layout.center-justified.search-footer,.layout.center-center.search-footer {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.layout.end-justified.search-footer {
  -ms-flex-pack: end;
        -webkit-justify-content: flex-end;
        justify-content: flex-end;
}

.layout.around-justified.search-footer {
  -ms-flex-pack: distribute;
        -webkit-justify-content: space-around;
        justify-content: space-around;
}

.layout.justified.search-footer {
  -ms-flex-pack: justify;
        -webkit-justify-content: space-between;
        justify-content: space-between;
}

.self-start.search-footer {
  -ms-align-self: flex-start;
        -webkit-align-self: flex-start;
        align-self: flex-start;
}

.self-center.search-footer {
  -ms-align-self: center;
        -webkit-align-self: center;
        align-self: center;
}

.self-end.search-footer {
  -ms-align-self: flex-end;
        -webkit-align-self: flex-end;
        align-self: flex-end;
}

.self-stretch.search-footer {
  -ms-align-self: stretch;
        -webkit-align-self: stretch;
        align-self: stretch;
}

.self-baseline.search-footer {
  -ms-align-self: baseline;
        -webkit-align-self: baseline;
        align-self: baseline;
}

.layout.start-aligned.search-footer {
  -ms-flex-line-pack: start;  
        -ms-align-content: flex-start;
        -webkit-align-content: flex-start;
        align-content: flex-start;
}

.layout.end-aligned.search-footer {
  -ms-flex-line-pack: end;  
        -ms-align-content: flex-end;
        -webkit-align-content: flex-end;
        align-content: flex-end;
}

.layout.center-aligned.search-footer {
  -ms-flex-line-pack: center;  
        -ms-align-content: center;
        -webkit-align-content: center;
        align-content: center;
}

.layout.between-aligned.search-footer {
  -ms-flex-line-pack: justify;  
        -ms-align-content: space-between;
        -webkit-align-content: space-between;
        align-content: space-between;
}

.layout.around-aligned.search-footer {
  -ms-flex-line-pack: distribute;  
        -ms-align-content: space-around;
        -webkit-align-content: space-around;
        align-content: space-around;
}</style><!-- Shady DOM styles for search-header --><style scope="search-header">search-header {
  display: block;
}

.lockup-logo.search-header,.lockup-text.search-header {
  display: inline-block;
}

.lockup-logo.search-header {
  width: 74px;
        height: 24px;
        background-image: url('https://www.gstatic.com/images/branding/googlelogo/1x/googlelogo_color_74x24dp.png');
        background-size: 74px 24px;
}

.lockup-text.search-header {
  font-family: "Product Sans", Arial, Helvetica, sans-serif;

        
        font-size: 22.1px;

        
        text-rendering: optimizeLegibility;

        
        position: relative;
        top: -6px;
        left: -5px;

        
        -webkit-font-smoothing: antialiased;
        -moz-osx-font-smoothing: grayscale;
}

.lockup-brand.search-header {
  opacity: 0.54;
}

@media (-webkit-min-device-pixel-ratio: 1.5), (min-resolution: 144dpi) {
.lockup-logo.search-header {
  background-image: url('https://www.gstatic.com/images/branding/googlelogo/2x/googlelogo_color_74x24dp.png');
}

.lockup-text.search-header {
  font-size: 21.3px;
          left: -7px;
}

}

header.search-header {
  background-color: #fafafa;
        border-bottom: 1px solid #ebebeb;
        color: #333;
        height: 60px;
        width: 100%;
        z-index: 1;
        display: block;
}

[latestStyle].search-header header.search-header {
  background-color: transparent;
}

.headerpad.search-header {
  margin-left: 16px;
}

[wide].search-header .headerpad.search-header {
  margin-left: 32px;
}

.leftheader.search-header {
  width: 322px;
}

header.search-header h1.search-header {
  display: none;
}

header.search-header h1.search-header state-modifier.search-header {
  color: #777;
        text-decoration: none;
        display: inline-block;
}

[wide].search-header header.search-header h1.search-header {
  box-sizing: border-box;
        margin: 0;
        padding: 20px 0 10px;
        font-size: 20px;
        font-weight: 400;
        white-space: nowrap;
        display: block;
}

search-box.search-header {
  height: 37px;
        padding: 0 16px;
        width: 900px;
        align-self: center;
}

.layout.horizontal.search-header,.layout.vertical.search-header {
  display: -ms-flexbox;
        display: -webkit-flex;
        display: flex;
}

.layout.inline.search-header {
  display: -ms-inline-flexbox;
        display: -webkit-inline-flex;
        display: inline-flex;
}

.layout.horizontal.search-header {
  -ms-flex-direction: row;
        -webkit-flex-direction: row;
        flex-direction: row;
}

.layout.vertical.search-header {
  -ms-flex-direction: column;
        -webkit-flex-direction: column;
        flex-direction: column;
}

.layout.wrap.search-header {
  -ms-flex-wrap: wrap;
        -webkit-flex-wrap: wrap;
        flex-wrap: wrap;
}

.layout.no-wrap.search-header {
  -ms-flex-wrap: nowrap;
        -webkit-flex-wrap: nowrap;
        flex-wrap: nowrap;
}

.layout.center.search-header,.layout.center-center.search-header {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.layout.center-justified.search-header,.layout.center-center.search-header {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.flex.search-header {
  -ms-flex: 1 1 0.000000001px;
        -webkit-flex: 1;
        flex: 1;
        -webkit-flex-basis: 0.000000001px;
        flex-basis: 0.000000001px;
}

.flex-auto.search-header {
  -ms-flex: 1 1 auto;
        -webkit-flex: 1 1 auto;
        flex: 1 1 auto;
}

.flex-none.search-header {
  -ms-flex: none;
        -webkit-flex: none;
        flex: none;
}

.layout.start.search-header {
  -ms-flex-align: start;
        -webkit-align-items: flex-start;
        align-items: flex-start;
}

.layout.center.search-header,.layout.center-center.search-header {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.layout.end.search-header {
  -ms-flex-align: end;
        -webkit-align-items: flex-end;
        align-items: flex-end;
}

.layout.baseline.search-header {
  -ms-flex-align: baseline;
        -webkit-align-items: baseline;
        align-items: baseline;
}

.layout.start-justified.search-header {
  -ms-flex-pack: start;
        -webkit-justify-content: flex-start;
        justify-content: flex-start;
}

.layout.center-justified.search-header,.layout.center-center.search-header {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.layout.end-justified.search-header {
  -ms-flex-pack: end;
        -webkit-justify-content: flex-end;
        justify-content: flex-end;
}

.layout.around-justified.search-header {
  -ms-flex-pack: distribute;
        -webkit-justify-content: space-around;
        justify-content: space-around;
}

.layout.justified.search-header {
  -ms-flex-pack: justify;
        -webkit-justify-content: space-between;
        justify-content: space-between;
}

.self-start.search-header {
  -ms-align-self: flex-start;
        -webkit-align-self: flex-start;
        align-self: flex-start;
}

.self-center.search-header {
  -ms-align-self: center;
        -webkit-align-self: center;
        align-self: center;
}

.self-end.search-header {
  -ms-align-self: flex-end;
        -webkit-align-self: flex-end;
        align-self: flex-end;
}

.self-stretch.search-header {
  -ms-align-self: stretch;
        -webkit-align-self: stretch;
        align-self: stretch;
}

.self-baseline.search-header {
  -ms-align-self: baseline;
        -webkit-align-self: baseline;
        align-self: baseline;
}

.layout.start-aligned.search-header {
  -ms-flex-line-pack: start;  
        -ms-align-content: flex-start;
        -webkit-align-content: flex-start;
        align-content: flex-start;
}

.layout.end-aligned.search-header {
  -ms-flex-line-pack: end;  
        -ms-align-content: flex-end;
        -webkit-align-content: flex-end;
        align-content: flex-end;
}

.layout.center-aligned.search-header {
  -ms-flex-line-pack: center;  
        -ms-align-content: center;
        -webkit-align-content: center;
        align-content: center;
}

.layout.between-aligned.search-header {
  -ms-flex-line-pack: justify;  
        -ms-align-content: space-between;
        -webkit-align-content: space-between;
        align-content: space-between;
}

.layout.around-aligned.search-header {
  -ms-flex-line-pack: distribute;  
        -ms-align-content: space-around;
        -webkit-align-content: space-around;
        align-content: space-around;
}</style><!-- Shady DOM styles for search-ui --><style scope="search-ui-0">.search-ui-0 paper-button.search-ui {
  display: inline-block;
        text-align: center;
        font-family: inherit;
}

.search-ui-0 .layout.horizontal.search-ui,.search-ui-0 .layout.vertical.search-ui {
  display: -ms-flexbox;
        display: -webkit-flex;
        display: flex;
}

.search-ui-0 .layout.inline.search-ui {
  display: -ms-inline-flexbox;
        display: -webkit-inline-flex;
        display: inline-flex;
}

.search-ui-0 .layout.horizontal.search-ui {
  -ms-flex-direction: row;
        -webkit-flex-direction: row;
        flex-direction: row;
}

.search-ui-0 .layout.vertical.search-ui {
  -ms-flex-direction: column;
        -webkit-flex-direction: column;
        flex-direction: column;
}

.search-ui-0 .layout.wrap.search-ui {
  -ms-flex-wrap: wrap;
        -webkit-flex-wrap: wrap;
        flex-wrap: wrap;
}

.search-ui-0 .layout.no-wrap.search-ui {
  -ms-flex-wrap: nowrap;
        -webkit-flex-wrap: nowrap;
        flex-wrap: nowrap;
}

.search-ui-0 .layout.center.search-ui,.search-ui-0 .layout.center-center.search-ui {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.search-ui-0 .layout.center-justified.search-ui,.search-ui-0 .layout.center-center.search-ui {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.search-ui-0 .flex.search-ui {
  -ms-flex: 1 1 0.000000001px;
        -webkit-flex: 1;
        flex: 1;
        -webkit-flex-basis: 0.000000001px;
        flex-basis: 0.000000001px;
}

.search-ui-0 .flex-auto.search-ui {
  -ms-flex: 1 1 auto;
        -webkit-flex: 1 1 auto;
        flex: 1 1 auto;
}

.search-ui-0 .flex-none.search-ui {
  -ms-flex: none;
        -webkit-flex: none;
        flex: none;
}

.search-ui-0 .layout.start.search-ui {
  -ms-flex-align: start;
        -webkit-align-items: flex-start;
        align-items: flex-start;
}

.search-ui-0 .layout.center.search-ui,.search-ui-0 .layout.center-center.search-ui {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.search-ui-0 .layout.end.search-ui {
  -ms-flex-align: end;
        -webkit-align-items: flex-end;
        align-items: flex-end;
}

.search-ui-0 .layout.baseline.search-ui {
  -ms-flex-align: baseline;
        -webkit-align-items: baseline;
        align-items: baseline;
}

.search-ui-0 .layout.start-justified.search-ui {
  -ms-flex-pack: start;
        -webkit-justify-content: flex-start;
        justify-content: flex-start;
}

.search-ui-0 .layout.center-justified.search-ui,.search-ui-0 .layout.center-center.search-ui {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.search-ui-0 .layout.end-justified.search-ui {
  -ms-flex-pack: end;
        -webkit-justify-content: flex-end;
        justify-content: flex-end;
}

.search-ui-0 .layout.around-justified.search-ui {
  -ms-flex-pack: distribute;
        -webkit-justify-content: space-around;
        justify-content: space-around;
}

.search-ui-0 .layout.justified.search-ui {
  -ms-flex-pack: justify;
        -webkit-justify-content: space-between;
        justify-content: space-between;
}

.search-ui-0 .self-start.search-ui {
  -ms-align-self: flex-start;
        -webkit-align-self: flex-start;
        align-self: flex-start;
}

.search-ui-0 .self-center.search-ui {
  -ms-align-self: center;
        -webkit-align-self: center;
        align-self: center;
}

.search-ui-0 .self-end.search-ui {
  -ms-align-self: flex-end;
        -webkit-align-self: flex-end;
        align-self: flex-end;
}

.search-ui-0 .self-stretch.search-ui {
  -ms-align-self: stretch;
        -webkit-align-self: stretch;
        align-self: stretch;
}

.search-ui-0 .self-baseline.search-ui {
  -ms-align-self: baseline;
        -webkit-align-self: baseline;
        align-self: baseline;
}

.search-ui-0 .layout.start-aligned.search-ui {
  -ms-flex-line-pack: start;  
        -ms-align-content: flex-start;
        -webkit-align-content: flex-start;
        align-content: flex-start;
}

.search-ui-0 .layout.end-aligned.search-ui {
  -ms-flex-line-pack: end;  
        -ms-align-content: flex-end;
        -webkit-align-content: flex-end;
        align-content: flex-end;
}

.search-ui-0 .layout.center-aligned.search-ui {
  -ms-flex-line-pack: center;  
        -ms-align-content: center;
        -webkit-align-content: center;
        align-content: center;
}

.search-ui-0 .layout.between-aligned.search-ui {
  -ms-flex-line-pack: justify;  
        -ms-align-content: space-between;
        -webkit-align-content: space-between;
        align-content: space-between;
}

.search-ui-0 .layout.around-aligned.search-ui {
  -ms-flex-line-pack: distribute;  
        -ms-align-content: space-around;
        -webkit-align-content: space-around;
        align-content: space-around;
}

.search-ui-0 {
  display: block;
        min-width: 350px;
        height: 100%;
}

.search-ui-0 #content.search-ui {
  height: 100%;
}

.search-ui-0 #leftBarOverflow.search-ui {
  padding-right: 16px;
}

.search-ui-0 [wide].search-ui #leftBarOverflow.search-ui {
  overflow: auto;
}

.search-ui-0 #leftBar.search-ui {
  box-sizing: border-box;
        color: #666;
        font-size: 13px;
        padding: 16px 0 16px 16px;
        border-bottom: 1px solid #e5e5e5;
}

.search-ui-0 [wide].search-ui #leftBar.search-ui {
  display: -ms-flexbox;
        display: -webkit-flex;
        display: flex;

        -ms-flex-direction: column;
        -webkit-flex-direction: column;
        flex-direction: column;
        position: relative;
        width: 350px;
        margin-bottom: 0;
        padding-bottom: 0;
        border-bottom: none;
}

@media print {
.search-ui-0 #leftBar.search-ui {
  display: none !important;
}

}

.search-ui-0 #leftBar.search-ui h3.search-ui {
  text-transform: uppercase;
        font-size: 13px;
        font-weight: normal;
        margin-top: 6px;
        margin-bottom: 6px;
        cursor: default;
        display: inline-block;
        color: #a3a3a3;
}

.search-ui-0 #closeButton.search-ui {
  position: absolute;
        right: 0;
}

.search-ui-0 #searchTermsHeader.search-ui {
  position: relative;
}

.search-ui-0 #searchTermsQ.search-ui iron-icon.search-ui {
  height: 16px;
        padding: 0 2px 2px 2px;
        color: #a3a3a3;
}

.search-ui-0 #searchTermsHeader.search-ui:hover iron-icon.search-ui {
  color: #4285f4;
}

.search-ui-0 #closeLeftBar.search-ui {
  height: 16px;
        width: 16px;
        padding: 0 0 8px 0;
        cursor: pointer;
}

.search-ui-0 #compactQuery.search-ui iron-icon.search-ui {
  height: 20px;
}

.search-ui-0 #backToResultsButton.search-ui {
  color: #fff;
        background: #4285f4;
        margin-top: 20px;
        text-align: center;
}

.search-ui-0 #main.search-ui {
  display: block;
        min-height: calc(100vh - 127px);
}

@media print {
.search-ui-0 [wide].search-ui main.search-ui {
  margin-left: 0;
}

}</style><!-- Shady DOM styles for import-list --><!-- Shady DOM styles for iron-request --><!-- Shady DOM styles for iron-ajax --><!-- Shady DOM styles for paper-dialog --><!-- Shady DOM styles for coverage-stats --><!-- Shady DOM styles for iron-image --><!-- Shady DOM styles for iron-selector --><!-- Shady DOM styles for paper-card --><!-- Shady DOM styles for paper-checkbox --><!-- Shady DOM styles for landing-page --><!-- Shady DOM styles for remote-fetch --><!-- Shady DOM styles for paper-spinner --><style scope="paper-spinner-0">.paper-spinner-0 {
  display: inline-block;
        position: relative;
        width: 28px;
        height: 28px;
}

.paper-spinner-0 #spinnerContainer.paper-spinner {
  width: 100%;
        height: 100%;

        
        direction: ltr;
}

.paper-spinner-0 #spinnerContainer.active.paper-spinner {
  -webkit-animation: container-rotate-paper-spinner-0 1568ms linear infinite;
        animation: container-rotate-paper-spinner-0 1568ms linear infinite;
}

@-webkit-keyframes container-rotate-paper-spinner-0 {
to {
  -webkit-transform: rotate(360deg)
}

}

@keyframes container-rotate-paper-spinner-0 {
to {
  transform: rotate(360deg)
}

}

.paper-spinner-0 .spinner-layer.paper-spinner {
  position: absolute;
        width: 100%;
        height: 100%;
        opacity: 0;
        white-space: nowrap;
        color: #4285f4;
}

.paper-spinner-0 .layer-1.paper-spinner {
  color: #4285f4;
}

.paper-spinner-0 .layer-2.paper-spinner {
  color: #db4437;
}

.paper-spinner-0 .layer-3.paper-spinner {
  color: #f4b400;
}

.paper-spinner-0 .layer-4.paper-spinner {
  color: #0f9d58;
}

.paper-spinner-0 .active.paper-spinner .spinner-layer.paper-spinner {
  -webkit-animation-name: fill-unfill-rotate-paper-spinner-0;
        -webkit-animation-duration: 5332ms;
        -webkit-animation-timing-function: cubic-bezier(0.4, 0.0, 0.2, 1);
        -webkit-animation-iteration-count: infinite;
        animation-name: fill-unfill-rotate-paper-spinner-0;
        animation-duration: 5332ms;
        animation-timing-function: cubic-bezier(0.4, 0.0, 0.2, 1);
        animation-iteration-count: infinite;
        opacity: 1;
}

.paper-spinner-0 .active.paper-spinner .spinner-layer.layer-1.paper-spinner {
  -webkit-animation-name: fill-unfill-rotate-paper-spinner-0, layer-1-fade-in-out-paper-spinner-0;
        animation-name: fill-unfill-rotate-paper-spinner-0, layer-1-fade-in-out-paper-spinner-0;
}

.paper-spinner-0 .active.paper-spinner .spinner-layer.layer-2.paper-spinner {
  -webkit-animation-name: fill-unfill-rotate-paper-spinner-0, layer-2-fade-in-out-paper-spinner-0;
        animation-name: fill-unfill-rotate-paper-spinner-0, layer-2-fade-in-out-paper-spinner-0;
}

.paper-spinner-0 .active.paper-spinner .spinner-layer.layer-3.paper-spinner {
  -webkit-animation-name: fill-unfill-rotate-paper-spinner-0, layer-3-fade-in-out-paper-spinner-0;
        animation-name: fill-unfill-rotate-paper-spinner-0, layer-3-fade-in-out-paper-spinner-0;
}

.paper-spinner-0 .active.paper-spinner .spinner-layer.layer-4.paper-spinner {
  -webkit-animation-name: fill-unfill-rotate-paper-spinner-0, layer-4-fade-in-out-paper-spinner-0;
        animation-name: fill-unfill-rotate-paper-spinner-0, layer-4-fade-in-out-paper-spinner-0;
}

@-webkit-keyframes fill-unfill-rotate-paper-spinner-0 {
12.5% {
  -webkit-transform: rotate(135deg)
}

25% {
  -webkit-transform: rotate(270deg)
}

37.5% {
  -webkit-transform: rotate(405deg)
}

50% {
  -webkit-transform: rotate(540deg)
}

62.5% {
  -webkit-transform: rotate(675deg)
}

75% {
  -webkit-transform: rotate(810deg)
}

87.5% {
  -webkit-transform: rotate(945deg)
}

to {
  -webkit-transform: rotate(1080deg)
}

}

@keyframes fill-unfill-rotate-paper-spinner-0 {
12.5% {
  transform: rotate(135deg)
}

25% {
  transform: rotate(270deg)
}

37.5% {
  transform: rotate(405deg)
}

50% {
  transform: rotate(540deg)
}

62.5% {
  transform: rotate(675deg)
}

75% {
  transform: rotate(810deg)
}

87.5% {
  transform: rotate(945deg)
}

to {
  transform: rotate(1080deg)
}

}

@-webkit-keyframes layer-1-fade-in-out-paper-spinner-0 {
0% {
  opacity: 1
}

25% {
  opacity: 1
}

26% {
  opacity: 0
}

89% {
  opacity: 0
}

90% {
  opacity: 1
}

to {
  opacity: 1
}

}

@keyframes layer-1-fade-in-out-paper-spinner-0 {
0% {
  opacity: 1
}

25% {
  opacity: 1
}

26% {
  opacity: 0
}

89% {
  opacity: 0
}

90% {
  opacity: 1
}

to {
  opacity: 1
}

}

@-webkit-keyframes layer-2-fade-in-out-paper-spinner-0 {
0% {
  opacity: 0
}

15% {
  opacity: 0
}

25% {
  opacity: 1
}

50% {
  opacity: 1
}

51% {
  opacity: 0
}

to {
  opacity: 0
}

}

@keyframes layer-2-fade-in-out-paper-spinner-0 {
0% {
  opacity: 0
}

15% {
  opacity: 0
}

25% {
  opacity: 1
}

50% {
  opacity: 1
}

51% {
  opacity: 0
}

to {
  opacity: 0
}

}

@-webkit-keyframes layer-3-fade-in-out-paper-spinner-0 {
0% {
  opacity: 0
}

40% {
  opacity: 0
}

50% {
  opacity: 1
}

75% {
  opacity: 1
}

76% {
  opacity: 0
}

to {
  opacity: 0
}

}

@keyframes layer-3-fade-in-out-paper-spinner-0 {
0% {
  opacity: 0
}

40% {
  opacity: 0
}

50% {
  opacity: 1
}

75% {
  opacity: 1
}

76% {
  opacity: 0
}

to {
  opacity: 0
}

}

@-webkit-keyframes layer-4-fade-in-out-paper-spinner-0 {
0% {
  opacity: 0
}

65% {
  opacity: 0
}

75% {
  opacity: 1
}

90% {
  opacity: 1
}

to {
  opacity: 0
}

}

@keyframes layer-4-fade-in-out-paper-spinner-0 {
0% {
  opacity: 0
}

65% {
  opacity: 0
}

75% {
  opacity: 1
}

90% {
  opacity: 1
}

to {
  opacity: 0
}

}

.paper-spinner-0 .circle-clipper.paper-spinner {
  display: inline-block;
        position: relative;
        width: 50%;
        height: 100%;
        overflow: hidden;
}

.paper-spinner-0 .spinner-layer.paper-spinner::after {
  left: 45%;
        width: 10%;
        border-top-style: solid;
}

.paper-spinner-0 .spinner-layer.paper-spinner::after,.paper-spinner-0 .circle-clipper.paper-spinner::after {
  content: '';
        box-sizing: border-box;
        position: absolute;
        top: 0;
        border-width: 3px;
        border-radius: 50%;
}

.paper-spinner-0 .circle-clipper.paper-spinner::after {
  bottom: 0;
        width: 200%;
        border-style: solid;
        border-bottom-color: transparent !important;
}

.paper-spinner-0 .circle-clipper.left.paper-spinner::after {
  left: 0;
        border-right-color: transparent !important;
        -webkit-transform: rotate(129deg);
        transform: rotate(129deg);
}

.paper-spinner-0 .circle-clipper.right.paper-spinner::after {
  left: -100%;
        border-left-color: transparent !important;
        -webkit-transform: rotate(-129deg);
        transform: rotate(-129deg);
}

.paper-spinner-0 .active.paper-spinner .gap-patch.paper-spinner::after,.paper-spinner-0 .active.paper-spinner .circle-clipper.paper-spinner::after {
  -webkit-animation-duration: 1333ms;
        -webkit-animation-timing-function: cubic-bezier(0.4, 0.0, 0.2, 1);
        -webkit-animation-iteration-count: infinite;
        animation-duration: 1333ms;
        animation-timing-function: cubic-bezier(0.4, 0.0, 0.2, 1);
        animation-iteration-count: infinite;
}

.paper-spinner-0 .active.paper-spinner .circle-clipper.left.paper-spinner::after {
  -webkit-animation-name: left-spin-paper-spinner-0;
        animation-name: left-spin-paper-spinner-0;
}

.paper-spinner-0 .active.paper-spinner .circle-clipper.right.paper-spinner::after {
  -webkit-animation-name: right-spin-paper-spinner-0;
        animation-name: right-spin-paper-spinner-0;
}

@-webkit-keyframes left-spin-paper-spinner-0 {
0% {
  -webkit-transform: rotate(130deg)
}

50% {
  -webkit-transform: rotate(-5deg)
}

to {
  -webkit-transform: rotate(130deg)
}

}

@keyframes left-spin-paper-spinner-0 {
0% {
  transform: rotate(130deg)
}

50% {
  transform: rotate(-5deg)
}

to {
  transform: rotate(130deg)
}

}

@-webkit-keyframes right-spin-paper-spinner-0 {
0% {
  -webkit-transform: rotate(-130deg)
}

50% {
  -webkit-transform: rotate(5deg)
}

to {
  -webkit-transform: rotate(-130deg)
}

}

@keyframes right-spin-paper-spinner-0 {
0% {
  transform: rotate(-130deg)
}

50% {
  transform: rotate(5deg)
}

to {
  transform: rotate(-130deg)
}

}

.paper-spinner-0 #spinnerContainer.cooldown.paper-spinner {
  -webkit-animation: container-rotate-paper-spinner-0 1568ms linear infinite, fade-out-paper-spinner-0 400ms cubic-bezier(0.4, 0.0, 0.2, 1);
        animation: container-rotate-paper-spinner-0 1568ms linear infinite, fade-out-paper-spinner-0 400ms cubic-bezier(0.4, 0.0, 0.2, 1);
}

@-webkit-keyframes fade-out-paper-spinner-0 {
0% {
  opacity: 1
}

to {
  opacity: 0
}

}

@keyframes fade-out-paper-spinner-0 {
0% {
  opacity: 1
}

to {
  opacity: 0
}

}</style><!-- Shady DOM styles for paper-spinner-lite --><!-- Shady DOM styles for concept-card --><!-- Shady DOM styles for concept-mention --><style scope="concept-mention">.block.concept-mention {
  display: block;
}

[hidden].concept-mention {
  display: none !important;
}

.invisible.concept-mention {
  visibility: hidden !important;
}

.relative.concept-mention {
  position: relative;
}

.fit.concept-mention {
  position: absolute;
        top: 0;
        right: 0;
        bottom: 0;
        left: 0;
}

body.fullbleed.concept-mention {
  margin: 0;
        height: 100vh;
}

.scroll.concept-mention {
  -webkit-overflow-scrolling: touch;
        overflow: auto;
}

.fixed-bottom.concept-mention,.fixed-left.concept-mention,.fixed-right.concept-mention,.fixed-top.concept-mention {
  position: fixed;
}

.fixed-top.concept-mention {
  top: 0;
        left: 0;
        right: 0;
}

.fixed-right.concept-mention {
  top: 0;
        right: 0;
        bottom: 0;
}

.fixed-bottom.concept-mention {
  right: 0;
        bottom: 0;
        left: 0;
}

.fixed-left.concept-mention {
  top: 0;
        bottom: 0;
        left: 0;
}

#target.concept-mention:hover {
  background-color: #e8f0fe;
}

#target[noicon].concept-mention {
  padding: 1px;
}

#target.concept-mention:hover[noicon] {
  border: 1px solid #e8f0fe;
        padding: 0;
}

#target.concept-mention {
  display: inline-block;
}

.inline-icon.concept-mention {
  cursor: pointer;
        height: 12px;
        width: 12px;
        padding-left: 4px;
        color: #5e6368;
}

concept-card.concept-mention {
  z-index: 1;
        position: absolute;
}</style><!-- Shady DOM styles for fterm-concept-card --><!-- Shady DOM styles for fterm-concept-mention --><!-- Shady DOM styles for paper-fab --><!-- Shady DOM styles for paper-icon-button --><!-- Shady DOM styles for paper-icon-button-light --><!-- Shady DOM styles for classification-tree --><style scope="classification-tree">.layout.horizontal.classification-tree,.layout.vertical.classification-tree {
  display: -ms-flexbox;
        display: -webkit-flex;
        display: flex;
}

.layout.inline.classification-tree {
  display: -ms-inline-flexbox;
        display: -webkit-inline-flex;
        display: inline-flex;
}

.layout.horizontal.classification-tree {
  -ms-flex-direction: row;
        -webkit-flex-direction: row;
        flex-direction: row;
}

.layout.vertical.classification-tree {
  -ms-flex-direction: column;
        -webkit-flex-direction: column;
        flex-direction: column;
}

.layout.wrap.classification-tree {
  -ms-flex-wrap: wrap;
        -webkit-flex-wrap: wrap;
        flex-wrap: wrap;
}

.layout.no-wrap.classification-tree {
  -ms-flex-wrap: nowrap;
        -webkit-flex-wrap: nowrap;
        flex-wrap: nowrap;
}

.layout.center.classification-tree,.layout.center-center.classification-tree {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.layout.center-justified.classification-tree,.layout.center-center.classification-tree {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.flex.classification-tree {
  -ms-flex: 1 1 0.000000001px;
        -webkit-flex: 1;
        flex: 1;
        -webkit-flex-basis: 0.000000001px;
        flex-basis: 0.000000001px;
}

.flex-auto.classification-tree {
  -ms-flex: 1 1 auto;
        -webkit-flex: 1 1 auto;
        flex: 1 1 auto;
}

.flex-none.classification-tree {
  -ms-flex: none;
        -webkit-flex: none;
        flex: none;
}

classification-tree {
  display: block;
        line-height: 20px;
}

.wrap.classification-tree {
  padding: 6px 12px;
}

.wrap.classification-tree:hover {
  background: #fafafa;
}

.description.classification-tree {
  padding-left: 4px;
}

iron-icon.classification-tree {
  color: #eaeaea;
}

.code[first].classification-tree {
  font-weight: 500;
}

.code[additional].classification-tree {
  font-style: italic;
}</style><!-- Shady DOM styles for classification-viewer --><style scope="classification-viewer">.table.classification-viewer {
  border: 1px solid #eaeaea;
        display: block;
}

.more.classification-viewer {
  padding: 6px 12px;
        cursor: pointer;
        font-style: italic;
        color: #aaa;
}

.more.classification-viewer:hover {
  background: #fafafa;
}

.chip.classification-viewer {
  display: inline-block;
        padding: 4px 6px;
        border-radius: 2px;
}

.chip[additional].classification-viewer {
  font-style: italic;
}</style><!-- Shady DOM styles for image-carousel --><!-- Shady DOM styles for image-viewer --><!-- Shady DOM styles for patent-result --><style scope="patent-result">#wrapper.patent-result {
  padding: 20px;
        max-width: 1050px;
        outline: 0;
}

.scroll-target.patent-result:before {
  display: block;
        content: '';
        visibility: hidden;
}

h1.patent-result {
  font-weight: 500;
        font-size: 20px;
        margin: 0 0 20px 0;
}

h3.patent-result {
  font-size: 15px;
        font-weight: 500;
        margin: 28px 0 18px 0;
}

dl.patent-result {
  line-height: 1.5;
        font-size: 0;
}

dt.patent-result,dd.patent-result {
  display: inline;
        font-size: 13px;
        margin: 0;
}

dt.patent-result {
  font-weight: 500;
        color: #000;
}

dt.patent-result::after {
  content: ': ';
}

dd.patent-result {
  color: #aaa;
}

dd.patent-result ~ dd.patent-result::before {
  content: ', ';
}

dt.patent-result + dd.patent-result::before {
  content: '';
}

dd.patent-result ~ dt.patent-result::before {
  display: block;
        content: '';
        margin-top: 8px;
}

dd.patent-result core-icon.patent-result {
  width: 14px;
        height: 14px;
        vertical-align: top;
        color: inherit;
}

.table.patent-result {
  display: table;
        border-collapse: collapse;
        border-spacing: 0;
        border: 1px solid #eaeaea;
        width: 100%;
        box-sizing: border-box;
}

.tr.patent-result {
  display: table-row;
        border-bottom: 1px solid #eaeaea;
}

.th.patent-result,.td.patent-result {
  display: table-cell;
        padding: 10px 12px;
}

.thead.patent-result {
  display: table-header-group;
}

.tbody.patent-result {
  display: table-row-group;
}

.th.patent-result {
  text-align: left;
        white-space: nowrap;
        background-color: #fafafa;
        color: #757575;
}

.td.patent-result {
  vertical-align: top;
}

.td.patent-result > p.patent-result {
  margin-top: 0;
}

.nowrap.patent-result {
  white-space: nowrap;
}

.knowledge-card.patent-result {
  border: 1px solid #e5e5e5;
        border-radius: 2px;
}

.knowledge-card.patent-result > header.patent-result {
  -webkit-font-smoothing: antialiased;
        -moz-osx-font-smoothing: grayscale;
        background: #4285f4;
        color: #fff;
        padding: 20px 12px;
}

.knowledge-card.patent-result > header.patent-result state-modifier.patent-result,.knowledge-card.patent-result > header.patent-result a.patent-result {
  color: inherit;
        margin: 0 12px 0 0;
        display: inline-block;
}

.knowledge-card.patent-result > header.patent-result h2.patent-result,.knowledge-card.patent-result > header.patent-result .tagline.patent-result {
  margin: 0;
        font-weight: normal;
        padding-top: 4px;
}

.knowledge-card.patent-result > header.patent-result h2.patent-result {
  font-size: 20px;
}

.knowledge-card.patent-result .knowledge-card-action-bar.patent-result {
  background: #3367d6;
        padding: 10px 12px;
        margin: 20px -12px -20px;
}

.knowledge-card.patent-result .knowledge-card-action-bar.patent-result core-icon.patent-result {
  margin-right: 12px;
        width: 24px;
        height: 24px;
}

.knowledge-card.patent-result dl.patent-result {
  padding: 16px 12px;
}

.knowledge-card.patent-result hr.patent-result {
  border: none;
        height: 1px;
        background: #e5e5e5;
        margin: 0;
}

.main-link.patent-result {
  margin: 14px;
        font-weight: 500;
}

.footer.patent-result h3.patent-result state-modifier.patent-result,.footer.patent-result h3.patent-result a.patent-result {
  float: right;
        font-weight: 400;
        font-size: 13px;
        line-height: 24px;
}

.footer.patent-result dl.patent-result {
  margin-left: 30px;
}

.footer.patent-result > ul.patent-result > li.patent-result {
  margin-bottom: 8px;
}

h3.patent-result .righthead.patent-result {
  color: #aaa;
        font-weight: 400;
        font-size: 13px;
        cursor: help;
}

.tooltip-hint.patent-result {
  border-bottom: 1px dotted #ccc;
}

[marginRight].patent-result {
  margin-right: 60px;
}

.flex-width.patent-result {
  min-width: 0;
        word-break: break-word;
}

.responsive-table.patent-result {
  width: 100%;
        overflow-x: auto;
        overflow-y: hidden;
}

.layout.horizontal.patent-result,.layout.vertical.patent-result {
  display: -ms-flexbox;
        display: -webkit-flex;
        display: flex;
}

.layout.inline.patent-result {
  display: -ms-inline-flexbox;
        display: -webkit-inline-flex;
        display: inline-flex;
}

.layout.horizontal.patent-result {
  -ms-flex-direction: row;
        -webkit-flex-direction: row;
        flex-direction: row;
}

.layout.vertical.patent-result {
  -ms-flex-direction: column;
        -webkit-flex-direction: column;
        flex-direction: column;
}

.layout.wrap.patent-result {
  -ms-flex-wrap: wrap;
        -webkit-flex-wrap: wrap;
        flex-wrap: wrap;
}

.layout.no-wrap.patent-result {
  -ms-flex-wrap: nowrap;
        -webkit-flex-wrap: nowrap;
        flex-wrap: nowrap;
}

.layout.center.patent-result,.layout.center-center.patent-result {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.layout.center-justified.patent-result,.layout.center-center.patent-result {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.flex.patent-result {
  -ms-flex: 1 1 0.000000001px;
        -webkit-flex: 1;
        flex: 1;
        -webkit-flex-basis: 0.000000001px;
        flex-basis: 0.000000001px;
}

.flex-auto.patent-result {
  -ms-flex: 1 1 auto;
        -webkit-flex: 1 1 auto;
        flex: 1 1 auto;
}

.flex-none.patent-result {
  -ms-flex: none;
        -webkit-flex: none;
        flex: none;
}

.layout.start.patent-result {
  -ms-flex-align: start;
        -webkit-align-items: flex-start;
        align-items: flex-start;
}

.layout.center.patent-result,.layout.center-center.patent-result {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.layout.end.patent-result {
  -ms-flex-align: end;
        -webkit-align-items: flex-end;
        align-items: flex-end;
}

.layout.baseline.patent-result {
  -ms-flex-align: baseline;
        -webkit-align-items: baseline;
        align-items: baseline;
}

.layout.start-justified.patent-result {
  -ms-flex-pack: start;
        -webkit-justify-content: flex-start;
        justify-content: flex-start;
}

.layout.center-justified.patent-result,.layout.center-center.patent-result {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.layout.end-justified.patent-result {
  -ms-flex-pack: end;
        -webkit-justify-content: flex-end;
        justify-content: flex-end;
}

.layout.around-justified.patent-result {
  -ms-flex-pack: distribute;
        -webkit-justify-content: space-around;
        justify-content: space-around;
}

.layout.justified.patent-result {
  -ms-flex-pack: justify;
        -webkit-justify-content: space-between;
        justify-content: space-between;
}

.self-start.patent-result {
  -ms-align-self: flex-start;
        -webkit-align-self: flex-start;
        align-self: flex-start;
}

.self-center.patent-result {
  -ms-align-self: center;
        -webkit-align-self: center;
        align-self: center;
}

.self-end.patent-result {
  -ms-align-self: flex-end;
        -webkit-align-self: flex-end;
        align-self: flex-end;
}

.self-stretch.patent-result {
  -ms-align-self: stretch;
        -webkit-align-self: stretch;
        align-self: stretch;
}

.self-baseline.patent-result {
  -ms-align-self: baseline;
        -webkit-align-self: baseline;
        align-self: baseline;
}

.layout.start-aligned.patent-result {
  -ms-flex-line-pack: start;  
        -ms-align-content: flex-start;
        -webkit-align-content: flex-start;
        align-content: flex-start;
}

.layout.end-aligned.patent-result {
  -ms-flex-line-pack: end;  
        -ms-align-content: flex-end;
        -webkit-align-content: flex-end;
        align-content: flex-end;
}

.layout.center-aligned.patent-result {
  -ms-flex-line-pack: center;  
        -ms-align-content: center;
        -webkit-align-content: center;
        align-content: center;
}

.layout.between-aligned.patent-result {
  -ms-flex-line-pack: justify;  
        -ms-align-content: space-between;
        -webkit-align-content: space-between;
        align-content: space-between;
}

.layout.around-aligned.patent-result {
  -ms-flex-line-pack: distribute;  
        -ms-align-content: space-around;
        -webkit-align-content: space-around;
        align-content: space-around;
}

.flex.patent-result,.flex-1.patent-result {
  -ms-flex: 1 1 0.000000001px;
        -webkit-flex: 1;
        flex: 1;
        -webkit-flex-basis: 0.000000001px;
        flex-basis: 0.000000001px;
}

.flex-2.patent-result {
  -ms-flex: 2;
        -webkit-flex: 2;
        flex: 2;
}

.flex-3.patent-result {
  -ms-flex: 3;
        -webkit-flex: 3;
        flex: 3;
}

.flex-4.patent-result {
  -ms-flex: 4;
        -webkit-flex: 4;
        flex: 4;
}

.flex-5.patent-result {
  -ms-flex: 5;
        -webkit-flex: 5;
        flex: 5;
}

.flex-6.patent-result {
  -ms-flex: 6;
        -webkit-flex: 6;
        flex: 6;
}

.flex-7.patent-result {
  -ms-flex: 7;
        -webkit-flex: 7;
        flex: 7;
}

.flex-8.patent-result {
  -ms-flex: 8;
        -webkit-flex: 8;
        flex: 8;
}

.flex-9.patent-result {
  -ms-flex: 9;
        -webkit-flex: 9;
        flex: 9;
}

.flex-10.patent-result {
  -ms-flex: 10;
        -webkit-flex: 10;
        flex: 10;
}

.flex-11.patent-result {
  -ms-flex: 11;
        -webkit-flex: 11;
        flex: 11;
}

.flex-12.patent-result {
  -ms-flex: 12;
        -webkit-flex: 12;
        flex: 12;
}

@media print {
#text.patent-result {
  flex-direction: column-reverse !important;
}

}

patent-result-ref-list.patent-result {
  width: 100%;
}

patent-result-ref-list.patent-result + patent-result-ref-list.patent-result {
  margin-left: 10px;
}

[narrow].patent-result {
  flex-direction: column-reverse !important;
}

.famheader.patent-result {
  text-align: left;
        white-space: nowrap;
        background-color: #fafafa;
        color: #757575;
}

.collapse.patent-result {
  color: #616161;
        font-size: 13px;
        cursor: pointer;
}

#imageDrag.patent-result {
  position: fixed;
        cursor: col-resize;
}

#imageDrag.patent-result div.patent-result {
  width: 4px;
        height: 100%;
        margin: 0 4px;
        background: #eee;
}

#imageDrag.patent-result:hover {
  background: #eee;
}

#imageDrag.patent-result:hover div.patent-result {
  background: #ccc;
}

.imageWrapper.patent-result {
  position: fixed;
        top: 60px;
        background: white;
        margin-left: 12px;
}

#notices.patent-result {
  padding: 20px;
}

#figures.patent-result {
  background: #efefef;
}

.headerButton.patent-result {
  color: #3c80f6;
        cursor: pointer;
        display: inline-block;
        padding: 0 14px;
        line-height: 27px;
}

.headerButton.patent-result iron-icon.patent-result {
  width: 16px;
        height: 16px;
}

.tr.conceptDomain.patent-result {
  display: table-row;
}

.responsive-grid.patent-result {
  display: grid;
        gap: 8px;
        grid-template-columns: none;
}</style><!-- Shady DOM styles for family-viewer --><!-- Shady DOM styles for app-row --><!-- Shady DOM styles for application-timeline --><style scope="application-timeline">.layout.horizontal.application-timeline,.layout.vertical.application-timeline {
  display: -ms-flexbox;
        display: -webkit-flex;
        display: flex;
}

.layout.inline.application-timeline {
  display: -ms-inline-flexbox;
        display: -webkit-inline-flex;
        display: inline-flex;
}

.layout.horizontal.application-timeline {
  -ms-flex-direction: row;
        -webkit-flex-direction: row;
        flex-direction: row;
}

.layout.vertical.application-timeline {
  -ms-flex-direction: column;
        -webkit-flex-direction: column;
        flex-direction: column;
}

.layout.wrap.application-timeline {
  -ms-flex-wrap: wrap;
        -webkit-flex-wrap: wrap;
        flex-wrap: wrap;
}

.layout.no-wrap.application-timeline {
  -ms-flex-wrap: nowrap;
        -webkit-flex-wrap: nowrap;
        flex-wrap: nowrap;
}

.layout.center.application-timeline,.layout.center-center.application-timeline {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.layout.center-justified.application-timeline,.layout.center-center.application-timeline {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.flex.application-timeline {
  -ms-flex: 1 1 0.000000001px;
        -webkit-flex: 1;
        flex: 1;
        -webkit-flex-basis: 0.000000001px;
        flex-basis: 0.000000001px;
}

.flex-auto.application-timeline {
  -ms-flex: 1 1 auto;
        -webkit-flex: 1 1 auto;
        flex: 1 1 auto;
}

.flex-none.application-timeline {
  -ms-flex: none;
        -webkit-flex: none;
        flex: none;
}

.wrap.application-timeline {
  padding: 16px 12px;
}

.event.application-timeline {
  padding: 3px 0;
}

[date].application-timeline {
  color: #5f6368;
        font-size: 13px;
        padding: 3px 0;
        width: 70px;
        line-height: 16px;
}

[critical].application-timeline [date].application-timeline {
  color: #202124;
        font-weight: 500;
        line-height: 18px;
}

.line.application-timeline {
  margin: 9px 6px;
        border-radius: 4px;
        background: #dadce0;
        width: 4px;
        height: 4px;
}

.dot.application-timeline {
  margin: 3px 0;
        border-radius: 4px;
        background: #dadce0;
        width: 4px;
        height: 4px;
        display: inline-block;
}

[critical].application-timeline .line.application-timeline {
  background: #4285f4;
}

[current].application-timeline .line.application-timeline {
  background: #81c995;
}

[current][critical].application-timeline .line.application-timeline {
  background: #81c995;
}

.title.application-timeline {
  font-size: 13px;
        padding: 3px 0;
        line-height: 16px;
}

.title-text.application-timeline {
  color: #5f6368;
}

.description-item.application-timeline {
  font-size: 12px;
        padding: 6px 0;
        max-width: 300px;
}

#legal.application-timeline {
  height: 14px;
        width: 14px;
        padding: 0 3px 1px 3px;
        vertical-align: text-bottom;
}

#icon.application-timeline {
  height: 12px;
        width: 12px;
        padding: 0 3px 3px 3px;
        vertical-align: text-bottom;
}

.unknown.application-timeline {
  font-style: italic;
}

.not_active.application-timeline {
  text-decoration: line-through;
}

.active.application-timeline {
  font-weight: 500;
}

#cc[this].application-timeline {
  color: #202124 !important;
        text-decoration: underline;
}

#cc[this].not_active.application-timeline {
  text-decoration: line-through underline;
}

hr.application-timeline {
  border: none;
        height: 1px;
        background: #e5e5e5;
        margin: 10px 0 16px 0;
}

.header.application-timeline {
  font-size: 13px;
        line-height: 20px;
        font-weight: 500;
        color: #202124;
        padding-bottom: 8px;
}

.block.application-timeline {
  padding: 0 6px 0 0;
}

.year.application-timeline {
  color: #5f6368;
        font-size: 13px;
        font-weight: 500;
        padding: 3px 1px;
        width: 70px;
}

.expand.application-timeline {
  cursor: pointer;
        padding: 6px 0 3px 0;
}</style><!-- Shady DOM styles for patent-text --><style scope="patent-text">.layout.horizontal.patent-text,.layout.vertical.patent-text {
  display: -ms-flexbox;
        display: -webkit-flex;
        display: flex;
}

.layout.inline.patent-text {
  display: -ms-inline-flexbox;
        display: -webkit-inline-flex;
        display: inline-flex;
}

.layout.horizontal.patent-text {
  -ms-flex-direction: row;
        -webkit-flex-direction: row;
        flex-direction: row;
}

.layout.vertical.patent-text {
  -ms-flex-direction: column;
        -webkit-flex-direction: column;
        flex-direction: column;
}

.layout.wrap.patent-text {
  -ms-flex-wrap: wrap;
        -webkit-flex-wrap: wrap;
        flex-wrap: wrap;
}

.layout.no-wrap.patent-text {
  -ms-flex-wrap: nowrap;
        -webkit-flex-wrap: nowrap;
        flex-wrap: nowrap;
}

.layout.center.patent-text,.layout.center-center.patent-text {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.layout.center-justified.patent-text,.layout.center-center.patent-text {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.flex.patent-text {
  -ms-flex: 1 1 0.000000001px;
        -webkit-flex: 1;
        flex: 1;
        -webkit-flex-basis: 0.000000001px;
        flex-basis: 0.000000001px;
}

.flex-auto.patent-text {
  -ms-flex: 1 1 auto;
        -webkit-flex: 1 1 auto;
        flex: 1 1 auto;
}

.flex-none.patent-text {
  -ms-flex: none;
        -webkit-flex: none;
        flex: none;
}

.layout.start.patent-text {
  -ms-flex-align: start;
        -webkit-align-items: flex-start;
        align-items: flex-start;
}

.layout.center.patent-text,.layout.center-center.patent-text {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.layout.end.patent-text {
  -ms-flex-align: end;
        -webkit-align-items: flex-end;
        align-items: flex-end;
}

.layout.baseline.patent-text {
  -ms-flex-align: baseline;
        -webkit-align-items: baseline;
        align-items: baseline;
}

.layout.start-justified.patent-text {
  -ms-flex-pack: start;
        -webkit-justify-content: flex-start;
        justify-content: flex-start;
}

.layout.center-justified.patent-text,.layout.center-center.patent-text {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.layout.end-justified.patent-text {
  -ms-flex-pack: end;
        -webkit-justify-content: flex-end;
        justify-content: flex-end;
}

.layout.around-justified.patent-text {
  -ms-flex-pack: distribute;
        -webkit-justify-content: space-around;
        justify-content: space-around;
}

.layout.justified.patent-text {
  -ms-flex-pack: justify;
        -webkit-justify-content: space-between;
        justify-content: space-between;
}

.self-start.patent-text {
  -ms-align-self: flex-start;
        -webkit-align-self: flex-start;
        align-self: flex-start;
}

.self-center.patent-text {
  -ms-align-self: center;
        -webkit-align-self: center;
        align-self: center;
}

.self-end.patent-text {
  -ms-align-self: flex-end;
        -webkit-align-self: flex-end;
        align-self: flex-end;
}

.self-stretch.patent-text {
  -ms-align-self: stretch;
        -webkit-align-self: stretch;
        align-self: stretch;
}

.self-baseline.patent-text {
  -ms-align-self: baseline;
        -webkit-align-self: baseline;
        align-self: baseline;
}

.layout.start-aligned.patent-text {
  -ms-flex-line-pack: start;  
        -ms-align-content: flex-start;
        -webkit-align-content: flex-start;
        align-content: flex-start;
}

.layout.end-aligned.patent-text {
  -ms-flex-line-pack: end;  
        -ms-align-content: flex-end;
        -webkit-align-content: flex-end;
        align-content: flex-end;
}

.layout.center-aligned.patent-text {
  -ms-flex-line-pack: center;  
        -ms-align-content: center;
        -webkit-align-content: center;
        align-content: center;
}

.layout.between-aligned.patent-text {
  -ms-flex-line-pack: justify;  
        -ms-align-content: space-between;
        -webkit-align-content: space-between;
        align-content: space-between;
}

.layout.around-aligned.patent-text {
  -ms-flex-line-pack: distribute;  
        -ms-align-content: space-around;
        -webkit-align-content: space-around;
        align-content: space-around;
}

patent-text {
  display: block;
}

#text.patent-text {
  line-height: 20px;
        min-width: 0;
}

#margin.patent-text {
  width: 16px;
}

#createHighlight.patent-text {
  position: relative;
        width: 16px;
        height: 16px;
        cursor: pointer;
}

.notranslate.patent-text > .google-src-text.patent-text {
  display: none;
}

.notranslate.patent-text:hover,mark.patent-text {
  background-color: #fbfcae;
}

.description-paragraph.patent-text {
  margin-block-start: 1em;
        margin-block-end: 1em;
}

.claim-text.patent-text {
  margin: 14px 0 0 20px;
}

.claim-dependent.patent-text {
  margin-left: 40px;
        color: #888;
}

[hide-claims].patent-text .claim-dependent.patent-text .claim-text.patent-text {
  height: 0;
        width: 0;
        visibility: hidden;
}

[hide-claims].patent-text .claim-dependent.patent-text .claim.patent-text:after {
  content: ""
}

.patent-full-image.patent-text {
  max-width: 100%;
        height: auto;
}

.small-patent-image.patent-text {
  display: inline;
}

ul.patent-text {
  padding: 0 0 0 50px;
}

li.patent-text {
  list-style-type: none;
        position: relative;
}

ol.claims.patent-text li.patent-text {
  list-style-type: decimal;
        font-weight: bold;
}

ol.claims.patent-text li.patent-text > *.patent-text {
  font-weight: normal;
}

li.patent-text > para-num.patent-text {
  position: absolute;
        left: -45px;
        color: #888;
}

mark.patent-text {
  background: #fff59d;
}

mark.searchText.patent-text {
  background: #80deea;
}

.searchTextHeader.patent-text {
  font-weight: 500;
}

pre.patent-text {
  white-space: pre-wrap;
}

heading.patent-text {
  display: block;
        font-weight: 500;
        font-size: 13px;
}

#comment.patent-text {
  width: 16px;
        height: 16px;
}

[has-search][no-results].patent-text {
  display: none;
}

.highlight.patent-text {
  background-color: #fef7e0;
}</style><!-- Shady DOM styles for para-num --><!-- Shady DOM styles for result-container --><style scope="result-container">paper-spinner.result-container {
  position: absolute;
        
        top: calc(50%  + 56px - 16px);
        left: calc(50% + 175px - 16px);
}</style><!-- Shady DOM styles for scholar-result --><!-- Shady DOM styles for search-result --><style scope="search-result">.layout.horizontal.search-result,.layout.vertical.search-result {
  display: -ms-flexbox;
        display: -webkit-flex;
        display: flex;
}

.layout.inline.search-result {
  display: -ms-inline-flexbox;
        display: -webkit-inline-flex;
        display: inline-flex;
}

.layout.horizontal.search-result {
  -ms-flex-direction: row;
        -webkit-flex-direction: row;
        flex-direction: row;
}

.layout.vertical.search-result {
  -ms-flex-direction: column;
        -webkit-flex-direction: column;
        flex-direction: column;
}

.layout.wrap.search-result {
  -ms-flex-wrap: wrap;
        -webkit-flex-wrap: wrap;
        flex-wrap: wrap;
}

.layout.no-wrap.search-result {
  -ms-flex-wrap: nowrap;
        -webkit-flex-wrap: nowrap;
        flex-wrap: nowrap;
}

.layout.center.search-result,.layout.center-center.search-result {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.layout.center-justified.search-result,.layout.center-center.search-result {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.flex.search-result {
  -ms-flex: 1 1 0.000000001px;
        -webkit-flex: 1;
        flex: 1;
        -webkit-flex-basis: 0.000000001px;
        flex-basis: 0.000000001px;
}

.flex-auto.search-result {
  -ms-flex: 1 1 auto;
        -webkit-flex: 1 1 auto;
        flex: 1 1 auto;
}

.flex-none.search-result {
  -ms-flex: none;
        -webkit-flex: none;
        flex: none;
}

.layout.start.search-result {
  -ms-flex-align: start;
        -webkit-align-items: flex-start;
        align-items: flex-start;
}

.layout.center.search-result,.layout.center-center.search-result {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.layout.end.search-result {
  -ms-flex-align: end;
        -webkit-align-items: flex-end;
        align-items: flex-end;
}

.layout.baseline.search-result {
  -ms-flex-align: baseline;
        -webkit-align-items: baseline;
        align-items: baseline;
}

.layout.start-justified.search-result {
  -ms-flex-pack: start;
        -webkit-justify-content: flex-start;
        justify-content: flex-start;
}

.layout.center-justified.search-result,.layout.center-center.search-result {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.layout.end-justified.search-result {
  -ms-flex-pack: end;
        -webkit-justify-content: flex-end;
        justify-content: flex-end;
}

.layout.around-justified.search-result {
  -ms-flex-pack: distribute;
        -webkit-justify-content: space-around;
        justify-content: space-around;
}

.layout.justified.search-result {
  -ms-flex-pack: justify;
        -webkit-justify-content: space-between;
        justify-content: space-between;
}

.self-start.search-result {
  -ms-align-self: flex-start;
        -webkit-align-self: flex-start;
        align-self: flex-start;
}

.self-center.search-result {
  -ms-align-self: center;
        -webkit-align-self: center;
        align-self: center;
}

.self-end.search-result {
  -ms-align-self: flex-end;
        -webkit-align-self: flex-end;
        align-self: flex-end;
}

.self-stretch.search-result {
  -ms-align-self: stretch;
        -webkit-align-self: stretch;
        align-self: stretch;
}

.self-baseline.search-result {
  -ms-align-self: baseline;
        -webkit-align-self: baseline;
        align-self: baseline;
}

.layout.start-aligned.search-result {
  -ms-flex-line-pack: start;  
        -ms-align-content: flex-start;
        -webkit-align-content: flex-start;
        align-content: flex-start;
}

.layout.end-aligned.search-result {
  -ms-flex-line-pack: end;  
        -ms-align-content: flex-end;
        -webkit-align-content: flex-end;
        align-content: flex-end;
}

.layout.center-aligned.search-result {
  -ms-flex-line-pack: center;  
        -ms-align-content: center;
        -webkit-align-content: center;
        align-content: center;
}

.layout.between-aligned.search-result {
  -ms-flex-line-pack: justify;  
        -ms-align-content: space-between;
        -webkit-align-content: space-between;
        align-content: space-between;
}

.layout.around-aligned.search-result {
  -ms-flex-line-pack: distribute;  
        -ms-align-content: space-around;
        -webkit-align-content: space-around;
        align-content: space-around;
}

search-result {
  display: block;
        font-size: 13px;
}

#mainContainer.search-result {
  min-height: calc(100vh - 127px);
        color: #333;
}

#compactQuery.search-result {
  padding: 18px 20px 10px 20px;
        cursor: pointer;
}

#compactQuery.search-result > span.search-result {
  padding: 0 14px 0 0;
}

#compactQuery.search-result iron-icon.search-result {
  padding-right: 6px;
        height: 16px;
        width: 16px;
}</style><!-- Shady DOM styles for webdoc-result --><!-- Shady DOM styles for iron-pages --><!-- Shady DOM styles for paper-tab --><!-- Shady DOM styles for paper-tabs --><!-- Shady DOM styles for debug-info --><!-- Shady DOM styles for search-paging --><!-- Shady DOM styles for search-result-item --><!-- Shady DOM styles for search-result-image --><!-- Shady DOM styles for search-result-entities --><!-- Shady DOM styles for search-result-measures --><!-- Shady DOM styles for search-results --><!-- Shady DOM styles for summary-box --><!-- Shady DOM styles for timeline-page --><!-- Shady DOM styles for iron-localstorage --><!-- Shady DOM styles for search-app --><style scope="search-app">search-app {
  display: block;
        height: 100%;
}

#toast.search-app {
  min-height: 64px;
        min-width: 742px;
}</style><style>body {transition: opacity ease-in 0.2s; } 
body[unresolved] {opacity: 0; display: block; overflow: hidden; position: relative; } 
</style>
    <title>HRP20050117A2 - Drosophila g protein coupled receptors, nucleic acids, and methods related to the same 
      - Google Patents</title>

    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta charset="UTF-8">
    <meta name="referrer" content="origin-when-crossorigin">
    <link rel="canonical" href="https://patents.google.com/patent/HRP20050117A2/en">
    
    <meta name="DC.type" content="patent">
    <meta name="DC.title" content="Drosophila g protein coupled receptors, nucleic acids, and methods related to the same 
     ">
    <meta name="DC.date" content="2005-02-04" scheme="dateSubmitted">
    <meta name="DC.description" content="">
    <meta name="citation_patent_application_number" content="HR:20050117A">
    <meta name="citation_pdf_url" content="https://patentimages.storage.googleapis.com/c4/70/ed/9c888f93219a4a/HRP20050117A2.pdf">
    <meta name="citation_patent_publication_number" content="HR:P20050117:A2">
    <meta name="DC.date" content="2005-10-31">
    <meta name="DC.contributor" content="David" scheme="inventor">
    <meta name="DC.contributor" content="Valdin" scheme="inventor">
    <meta name="DC.contributor" content="Teresa" scheme="inventor">
    <meta name="DC.contributor" content="Martha" scheme="inventor">
    <meta name="DC.contributor" content="Pharmacia &amp; Upjohn Company Llc" scheme="assignee">
    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:400,400italic,500,500italic,700">
    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Product+Sans">
    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Google+Sans:wght@400;500;700">

    <style>
      body { transition: none; }
    </style>

    <script src="https://apis.google.com/_/scs/abc-static/_/js/k=gapi.lb.en.IKZeRvoAYNY.O/m=client/rt=j/sv=1/d=1/ed=1/am=AAAQ/rs=AHpOoo-SMWTzMRJrTty6iE5dL_aWGYOnuw/cb=gapi.loaded_0?le=scs" async=""></script><script>
      window.version = 'patent-search.search_20240531_RC00';

      function sendFeedback() {
        userfeedback.api.startFeedback({
          'productId': '713680',
          'bucket': 'patent-search-web',
          'productVersion': window.version,
        });
      }

      window.experiments = {};
      window.experiments.patentCountries = "ae,ag,al,am,ao,ap,ar,at,au,aw,az,ba,bb,bd,be,bf,bg,bh,bj,bn,bo,br,bw,bx,by,bz,ca,cf,cg,ch,ci,cl,cm,cn,co,cr,cs,cu,cy,cz,dd,de,dj,dk,dm,do,dz,ea,ec,ee,eg,em,ep,es,fi,fr,ga,gb,gc,gd,ge,gh,gm,gn,gq,gr,gt,gw,hk,hn,hr,hu,ib,id,ie,il,in,ir,is,it,jo,jp,ke,kg,kh,km,kn,kp,kr,kw,kz,la,lc,li,lk,lr,ls,lt,lu,lv,ly,ma,mc,md,me,mg,mk,ml,mn,mo,mr,mt,mw,mx,my,mz,na,ne,ng,ni,nl,no,nz,oa,om,pa,pe,pg,ph,pl,pt,py,qa,ro,rs,ru,rw,sa,sc,sd,se,sg,si,sk,sl,sm,sn,st,su,sv,sy,sz,td,tg,th,tj,tm,tn,tr,tt,tw,tz,ua,ug,us,uy,uz,vc,ve,vn,wo,yu,za,zm,zw";
      
      
      window.experiments.keywordWizard = true;
      
      
      
      window.experiments.definitions = true;

      window.Polymer = {
        dom: 'shady',
        lazyRegister: true,
      };
    </script>

    <script src="//www.gstatic.com/patent-search/frontend/patent-search.search_20240531_RC00/scs/compiled_dir/webcomponentsjs/webcomponents-lite.min.js"></script>
    <style>[hidden]:not([style-scope]):not(.style-scope) {
  display: none !important;
}

</style><style></style><style></style><style></style><style></style><link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:400,300,300italic,400italic,500,500italic,700,700italic"><link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto+Mono:400,700"><style></style><style></style><style>
  html, body {
    min-width: 100%;
    width: 100%;
    min-height: 100%;
    height: 100%;
  }

  body {
    background-color: #fff;
    font-family: 'Roboto', sans-serif;
    color: #333;
    margin: 0;
    padding: 0;
    /*
     * Turn off https://github.com/WICG/ScrollAnchoring/blob/master/explainer.md
     * https://memegen.googleplex.com/5230756293246976
     */
    overflow-anchor: none;
  }

  body h1, body h2, body h3 {
    -webkit-font-smoothing: antialiased;
    -moz-osx-font-smoothing: grayscale;
  }

  body a:link {
    color: #3c80f6;
    text-decoration: none;
  }

  body a:visited {
    color: #3c80f6;
  }

  body a[open-result]:visited {
    color: #7e57c2;
  }

  /* For Firefox */
  a:link {
    color: #3c80f6;
    text-decoration: none;
  }

  a:visited {
    color: #3c80f6;
  }

  a[open-result]:visited {
    color: #7e57c2;
  }

  ::selection , body ::selection {
    background-color: #c6dafc;
    color: #333;
  }
</style><link rel="import" href="//www.gstatic.com/patent-search/frontend/patent-search.search_20240531_RC00/scs/compiled_dir/search-app-vulcanized.html">
  <style>[hidden] { display: none !important; }</style><script type="text/javascript" charset="UTF-8" src="https://apis.google.com/js/client.js" gapi_processed="true"></script></head>
  <body>
    <script src="//www.gstatic.com/patent-search/frontend/patent-search.search_20240531_RC00/scs/compiled_dir/search-app-vulcanized.js"></script><span style="font-size: 13px; white-space: pre-wrap; display: inline-block; position: absolute; left: -99999px; top: -99999px; max-width: 217px;"></span><span style="font-size: 13px; white-space: pre-wrap; display: inline-block; position: absolute; left: -99999px; top: -99999px; max-width: 217px;"></span>
    <search-app>
    
    <remote-fetch class="style-scope search-app"></remote-fetch>
    <search-state class="style-scope search-app"></search-state>
    <browser-ui class="style-scope search-app"></browser-ui>
    <template is="dom-if" restamp="" class="style-scope search-app"></template>
    <template is="dom-if" restamp="" class="style-scope search-app"></template>
    
      <search-result id="result" class="style-scope search-app">
    
    
    <search-ui mode="result" class="style-scope search-result x-scope search-ui-0">
    
    
    
    <div id="content" class="style-scope search-ui" wide="">
      <search-header show-search="" class="style-scope search-ui">
    
    
    <div class="style-scope search-header" wide="">
      <header class="style-scope search-header">
        <div class="layout horizontal headerpad style-scope search-header">
          <div class="layout horizontal leftheader style-scope search-header">
            <h1 class="style-scope search-header">
              <state-modifier inherit="" act="{&quot;type&quot;: &quot;HOME&quot;}" class="style-scope search-header"><a id="link" href="#" class="style-scope state-modifier" inherit="">
                <div class="lockup style-scope search-header">
                  <span class="lockup-logo style-scope search-header"></span>
                  <span class="lockup-text style-scope search-header">
                    <span class="lockup-brand style-scope search-header">&nbsp;Patents</span>
                  </span>
                </div>
              </a></state-modifier>
            </h1>
          </div>
          
            <search-box class="style-scope search-header">
    
    <div class="layout horizontal style-scope search-box" style="position: relative;">
      <div class="inputbox flex layout horizontal style-scope search-box">
        <div class="suggest-bg flex style-scope search-box">
          <input name="q" id="searchInput" data-proto="SEARCH_INPUT" autocomplete="off" class="style-scope search-box">
        </div>
        <div id="corpora" class="style-scope search-box">
          <iron-icon icon="icons:lightbulb-outline" id="patentIcon" class="style-scope search-box x-scope iron-icon-0" active=""><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M9 21c0 .55.45 1 1 1h4c.55 0 1-.45 1-1v-1H9v1zm3-19C8.14 2 5 5.14 5 9c0 2.38 1.19 4.47 3 5.74V17c0 .55.45 1 1 1h6c.55 0 1-.45 1-1v-2.26c1.81-1.27 3-3.36 3-5.74 0-3.86-3.14-7-7-7zm2.85 11.1l-.85.6V16h-4v-2.3l-.85-.6C7.8 12.16 7 10.63 7 9c0-2.76 2.24-5 5-5s5 2.24 5 5c0 1.63-.8 3.16-2.15 4.1z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
          <overlay-tooltip positioning-target="#patentIcon" direction="bottom" class="style-scope search-box" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      Include patents
    </div>
  </overlay-tooltip>
          <iron-icon icon="social:school" id="nplIcon" class="style-scope search-box x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M5 13.18v4L12 21l7-3.82v-4L12 17l-7-3.82zM12 3L1 9l11 6 9-4.91V17h2V9L12 3z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
          <overlay-tooltip positioning-target="#nplIcon" direction="bottom" class="style-scope search-box" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      Include non-patent literature
    </div>
  </overlay-tooltip>
          <keyword-wizard id="wizardIcon" class="style-scope search-box">


<div id="wizardContainer" class="style-scope keyword-wizard">
  <iron-icon icon="image:tune" id="wizardIcon" class="style-scope keyword-wizard x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M3 17v2h6v-2H3zM3 5v2h10V5H3zm10 16v-2h8v-2h-8v-2h-2v6h2zM7 9v2H3v2h4v2h2V9H7zm14 4v-2H11v2h10zm-6-4h2V7h4V5h-4V3h-2v6z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
  <paper-material elevation="2" id="wizard" class="style-scope keyword-wizard x-scope paper-material-0" hidden="">
    
    

    
    
    <div class="search-within style-scope keyword-wizard">
      <span class="wizard-header style-scope keyword-wizard">
        Search within
        <iron-icon icon="icons:help-outline" class="help-header style-scope keyword-wizard x-scope iron-icon-0" id="within"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M11 18h2v-2h-2v2zm1-16C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8zm0-14c-2.21 0-4 1.79-4 4h2c0-1.1.9-2 2-2s2 .9 2 2c0 2-3 1.75-3 5h2c0-2.25 3-2.5 3-5 0-2.21-1.79-4-4-4z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      </span>
      
      <div class="help style-scope keyword-wizard" id="withinHelp" hidden="">
        <p class="style-scope keyword-wizard"><b class="style-scope keyword-wizard">Search within the title, abstract, claims, or full patent document</b>: You can
          restrict your search to a specific field using field names.</p>
        <p class="style-scope keyword-wizard">Use <span class="q style-scope keyword-wizard">TI=</span> to search in the title, <span class="q style-scope keyword-wizard">AB=</span> for the
          abstract, <span class="q style-scope keyword-wizard">CL=</span> for the claims, or <span class="q style-scope keyword-wizard">TAC=</span> for
          all three. For example, <span class="q style-scope keyword-wizard">TI=(safety belt)</span>.
        </p>
        <p class="style-scope keyword-wizard"><b class="style-scope keyword-wizard">Search by Cooperative Patent Classifications (CPCs)</b>: These are commonly used to
          represent ideas in place of keywords, and can also be entered in a search term box. If
          you're searching for<span class="q style-scope keyword-wizard">seat belts</span>, you could also search for
          <span class="q style-scope keyword-wizard">B60R22/00</span> to retrieve documents that mention safety belts or body
          harnesses. <span class="q style-scope keyword-wizard">CPC=B60R22</span> will match documents with exactly this CPC,
          <span class="q style-scope keyword-wizard">CPC=B60R22/low</span> matches documents with this CPC or a child
          classification of this CPC.
        </p>
          <a href="https://support.google.com/faqs/answer/7049475" target="_blank" class="style-scope keyword-wizard">Learn More</a>
      </div>
      <div class="filter-chips style-scope keyword-wizard">
        <span class="chip style-scope keyword-wizard" id="title" selected="">Title</span>
        <span class="chip style-scope keyword-wizard" id="abstract" selected="">Abstract</span>
        <span class="chip style-scope keyword-wizard" id="claims" selected="">Claims</span>
        <span class="chip style-scope keyword-wizard" id="fullDoc" selected="">Full Document</span>
      </div>
      
      <div class="cpc-field style-scope keyword-wizard">
        <span class="input-label style-scope keyword-wizard"><label for="inputCPC" class="style-scope keyword-wizard">or CPC</label></span>
        <div class="cpc-inputs style-scope keyword-wizard">
          <input class="keyword-input style-scope keyword-wizard" id="inputCPC" placeholder="Or search with CPC">
          <input type="checkbox" id="childCPC" checked="" class="style-scope keyword-wizard">
          <label class="checkbox-label style-scope keyword-wizard" for="childCPC">
            Include child classifications
          </label>
        </div>
      </div>
    </div>
    
    <div class="find-patents style-scope keyword-wizard">
      <span class="wizard-header style-scope keyword-wizard">
        Find patents
        <iron-icon class="help-header style-scope keyword-wizard x-scope iron-icon-0" icon="icons:help-outline" id="searchBoolean"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M11 18h2v-2h-2v2zm1-16C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8zm0-14c-2.21 0-4 1.79-4 4h2c0-1.1.9-2 2-2s2 .9 2 2c0 2-3 1.75-3 5h2c0-2.25 3-2.5 3-5 0-2.21-1.79-4-4-4z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      </span>
      
      <div class="help style-scope keyword-wizard" id="searchBooleanHelp" hidden="">
        <p class="style-scope keyword-wizard"><b class="style-scope keyword-wizard">Keywords and boolean syntax (USPTO or EPO format)</b>:
          <span class="q style-scope keyword-wizard">seat belt</span> searches these two words, or their plurals and close
          synonyms. <span class="q style-scope keyword-wizard">"seat belt"</span> searches this exact phrase, in order.
          <span class="q style-scope keyword-wizard">-seat -belt</span> searches for documents not containing either word.
        </p>
        <p class="style-scope keyword-wizard"><b class="style-scope keyword-wizard">For searches using boolean logic</b>, the default operator is
          <span class="q style-scope keyword-wizard">AND</span> with left associativity. <i class="style-scope keyword-wizard">Note:</i> this means
          <span class="q style-scope keyword-wizard">safety OR seat belt</span> is searched as <span class="q style-scope keyword-wizard">(safety OR seat)
          AND belt</span>. Each word automatically includes plurals and close synonyms. Adjacent
          words that are implicitly ANDed together, such as <span class="q style-scope keyword-wizard">(safety belt)</span>,
          are treated as a phrase when generating synonyms.
        </p>
        <a href="https://support.google.com/faqs/answer/7049475" target="_blank" class="style-scope keyword-wizard">Learn More</a>
      </div>
      <div class="style-scope keyword-wizard">
        <span class="input-label style-scope keyword-wizard"><label for="inputAnd" class="style-scope keyword-wizard">with ALL of the words</label></span>
        <input class="keyword-input style-scope keyword-wizard" id="inputAnd" placeholder="with ALL of the words">
      </div>
      <div class="style-scope keyword-wizard">
        <span class="input-label style-scope keyword-wizard"><label for="inputExact" class="style-scope keyword-wizard">with the EXACT PHRASE</label></span>
        <input class="keyword-input style-scope keyword-wizard" id="inputExact" placeholder="with the EXACT PHRASE"></div>
      <div class="style-scope keyword-wizard">
        <span class="input-label style-scope keyword-wizard"><label for="inputOr" class="style-scope keyword-wizard">with AT LEAST ONE of the words</label></span>
        <input class="keyword-input style-scope keyword-wizard" id="inputOr" placeholder="with AT LEAST ONE of the words"></div>
      <div class="style-scope keyword-wizard">
        <span class="input-label style-scope keyword-wizard"><label for="inputMinus" class="style-scope keyword-wizard">WITHOUT the words</label></span>
        <input class="keyword-input style-scope keyword-wizard" id="inputMinus" placeholder="WITHOUT the words"></div>
    </div>
    <div class="search-by style-scope keyword-wizard">
      <div class="wizard-header style-scope keyword-wizard">
          Search by
          <iron-icon icon="icons:help-outline" class="help-header style-scope keyword-wizard x-scope iron-icon-0" id="chem"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M11 18h2v-2h-2v2zm1-16C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8zm0-14c-2.21 0-4 1.79-4 4h2c0-1.1.9-2 2-2s2 .9 2 2c0 2-3 1.75-3 5h2c0-2.25 3-2.5 3-5 0-2.21-1.79-4-4-4z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      </div>
      
      <div class="help style-scope keyword-wizard" id="chemHelp" hidden="">
        <p class="style-scope keyword-wizard"><b class="style-scope keyword-wizard">Chemistry searches</b> match terms (trade names, IUPAC names, etc.
          extracted from the entire document, and processed from .MOL files.)
        </p>
        <p class="style-scope keyword-wizard">Substructure (use <span class="q style-scope keyword-wizard">SSS=</span>) and similarity (use
          <span class="q style-scope keyword-wizard">~</span>) searches are limited to one per search at the top-level
          <span class="q style-scope keyword-wizard">AND</span> condition. Exact searches can be used multiple times throughout
          the search query.
        </p>
        <p class="style-scope keyword-wizard">Searching by SMILES or InChi key requires no special syntax. To search by SMARTS, use
          <span class="q style-scope keyword-wizard">SMARTS=</span>.
        </p>
        <p class="style-scope keyword-wizard">To search for multiple molecules, select "Batch" in the "Type" menu. Enter multiple
          molecules separated by whitespace or by comma.
        </p>
        <a href="https://support.google.com/faqs/answer/7049475" target="_blank" class="style-scope keyword-wizard">Learn More</a>
      </div>
      
      <div class="chemistry-field style-scope keyword-wizard">
          <span class="input-label style-scope keyword-wizard"><label for="inputChem" class="style-scope keyword-wizard">Chemistry</label></span>
          <span class="chem-type-label style-scope keyword-wizard"><label for="chemType" hidden="" class="style-scope keyword-wizard">Type</label></span>
          <select id="chemType" class="style-scope keyword-wizard">
            <option value="identical" class="style-scope keyword-wizard">Identical</option>
            <option value="substructure" class="style-scope keyword-wizard">Substructure</option>
            <option value="smarts" class="style-scope keyword-wizard">SMARTS</option>
            <option value="similarity" class="style-scope keyword-wizard">Similarity</option>
            <option value="batch" class="style-scope keyword-wizard">Batch</option>
          </select>
          <textarea class="keyword-input style-scope keyword-wizard" id="inputChem" placeholder="Trade name, SMILES, InChiKey"></textarea>
      </div>
      <div class="import style-scope keyword-wizard">
        <div class="wizard-header style-scope keyword-wizard">
          Patent numbers
        </div>
        <span class="import-button style-scope keyword-wizard">
          <a href="https://patents.google.com/import" class="style-scope keyword-wizard"><button class="style-scope keyword-wizard">Import</button></a>
        </span>
        <p class="import-text style-scope keyword-wizard">
          Search specific patents by importing a CSV or list of patent publication or application numbers.
        </p>
      </div>
    </div>
    
    <div class="wizard-buttons style-scope keyword-wizard">
      <button type="reset" class="style-scope keyword-wizard">Close</button>
      <button type="button" autofocus="" class="style-scope keyword-wizard">Add</button>
    </div>
  
  </paper-material>
</div>
</keyword-wizard>
          <overlay-tooltip positioning-target="#wizardIcon" direction="bottom" class="style-scope search-box" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      Display advanced search options
    </div>
  </overlay-tooltip>
        </div>
      </div>
      <paper-material id="suggestMenu" class="style-scope search-box x-scope paper-material-0" elevation="1" hidden="">
    
    

    
        <template is="dom-repeat" class="style-scope search-box"></template>
        <div class="row style-scope search-box" hidden="">
          <iron-icon class="suggest-icon style-scope search-box x-scope iron-icon-0" icon="error"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm1 15h-2v-2h2v2zm0-4h-2V7h2v6z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon> Sorry, we couldn't find this patent number.
        </div>
        <div class="row style-scope search-box" hidden="">
          <iron-icon class="suggest-icon style-scope search-box x-scope iron-icon-0" icon="error"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm1 15h-2v-2h2v2zm0-4h-2V7h2v6z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon> 
        </div>
      
  </paper-material>
      <button id="searchButton" class="style-scope search-box"><iron-icon class="style-scope search-box x-scope iron-icon-0" icon="search"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon></button>
    </div>
  </search-box>
          <template is="dom-if" class="style-scope search-header"></template>
          <div class="flex style-scope search-header"></div>
          
            <div id="header-container" class="style-scope search-header">
              <result-nav class="style-scope search-header">
    
    <div class="style-scope result-nav" hidden="">
      <state-modifier id="current" act="{&quot;type&quot;: &quot;BACK_TO_RESULTS&quot;}" inherit="" class="style-scope result-nav"><a id="link" href="#" class="style-scope state-modifier" inherit=""><span class="style-scope result-nav"></span> of
      <template is="dom-if" class="style-scope result-nav"></template>
      0<template is="dom-if" class="style-scope result-nav"></template></a></state-modifier>
      <state-modifier id="prevResult" inherit="" act="{&quot;type&quot;: &quot;NAV_RESULT&quot;, &quot;result&quot;: &quot;$id&quot;, &quot;page&quot;: $page}" class="style-scope result-nav" disabled=""><a id="link" href="#" class="style-scope state-modifier" inherit="">
        <iron-icon icon="chevron-left" class="style-scope result-nav x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M15.41 7.41L14 6l-6 6 6 6 1.41-1.41L10.83 12z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      </a></state-modifier><state-modifier id="nextResult" inherit="" act="{&quot;type&quot;: &quot;NAV_RESULT&quot;, &quot;result&quot;: &quot;$id&quot;, &quot;page&quot;: $page}" class="style-scope result-nav" disabled=""><a id="link" href="#" class="style-scope state-modifier" inherit="">
        <iron-icon icon="chevron-right" class="style-scope result-nav x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M10 6L8.59 7.41 13.17 12l-4.58 4.59L10 18l6-6z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      </a></state-modifier>
      <overlay-tooltip positioning-target="#prevResult" direction="bottom" arrow="" class="style-scope result-nav" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip"></div>
    <div class="style-scope overlay-tooltip">
      Previous result
    </div>
  </overlay-tooltip>
      <overlay-tooltip positioning-target="#nextResult" direction="bottom" arrow="" class="style-scope result-nav" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip"></div>
    <div class="style-scope overlay-tooltip">
      Next result
    </div>
  </overlay-tooltip>
    </div>
  </result-nav>
            </div>
          <template is="dom-if" class="style-scope search-header"></template>
        </div>
      </header>
    </div>
  </search-header>

      <div class="layout horizontal style-scope search-ui">
        <template is="dom-if" class="style-scope search-ui"></template>

        <div id="main" class="style-scope search-ui">
          <search-term-wizard class="style-scope search-ui"><pat-dialog open="" class="style-scope search-term-wizard">
  <span class="header style-scope search-term-wizard">
    <span class="title style-scope search-term-wizard">Search tools</span>
    <span id="language" class="style-scope search-term-wizard">
      <text-button id="textButton" selected="" class="style-scope search-term-wizard">Text
</text-button>
      <text-button id="classificationButton" class="style-scope search-term-wizard">Classification
</text-button>
      <text-button id="chemistryButton" class="style-scope search-term-wizard">Chemistry
</text-button>
      <text-button id="measureButton" class="style-scope search-term-wizard">Measure
</text-button>
      <text-button id="numbersButton" class="style-scope search-term-wizard">Numbers
</text-button>
    </span>
  </span>
  <span id="panels" class="style-scope search-term-wizard">
    <text-search-term-wizard id="textPanel" class="style-scope search-term-wizard" selected=""><span class="style-scope text-search-term-wizard">
  <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49" class="style-scope text-search-term-wizard">
    <path d="M3.85 15.275V13h10.3v2.275Zm0-4.275V8.725h16.3V11Z" class="style-scope text-search-term-wizard"></path>
  </svg>
  <outlined-text-field id="textField" placeholder="Type a concept you want to search for here" class="style-scope text-search-term-wizard" value=""><span class="style-scope outlined-text-field">
  <input id="input" spellcheck="false" autocomplete="off" class="style-scope outlined-text-field" aria-disabled="false" placeholder="Type a concept you want to search for here">
  
</span>
</outlined-text-field>
</span>
<span class="style-scope text-search-term-wizard">
  <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49" class="style-scope text-search-term-wizard">
    <path d="M8 18.025h8v-2.1H8Zm0-4h8v-2.1H8ZM6.075 22.2q-.95 0-1.612-.662-.663-.663-.663-1.613V4.075q0-.95.663-1.613.662-.662 1.612-.662h8L20.2 7.925v12q0 .95-.662 1.613-.663.662-1.613.662Zm6.85-13.125v-5h-6.85v15.85h11.85V9.075Zm-6.85-5v5-5 15.85-15.85Z" class="style-scope text-search-term-wizard"></path>
  </svg>
  <filter-chip id="fullDocuments" selected="" class="style-scope text-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Full documents</filter-chip>
  <filter-chip id="title" class="style-scope text-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Title</filter-chip>
  <filter-chip id="abstract" class="style-scope text-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Abstract</filter-chip>
  <filter-chip id="claims" class="style-scope text-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Claims</filter-chip>
</span>
<span class="style-scope text-search-term-wizard">
  <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49" class="style-scope text-search-term-wizard">
    <path d="M16 16.925q2.05 0 3.488-1.437Q20.925 14.05 20.925 12t-1.437-3.488Q18.05 7.075 16 7.075q-.625 0-1.2.15t-1.075.425q.7.925 1.087 2.025.388 1.1.388 2.325t-.388 2.325q-.387 1.1-1.087 2.025.5.275 1.075.425.575.15 1.2.15Zm-4-2.075q.45-.575.688-1.313.237-.737.237-1.537 0-.8-.237-1.538Q12.45 9.725 12 9.15q-.45.575-.688 1.312-.237.738-.237 1.538t.237 1.537q.238.738.688 1.313Zm-4 2.075q.625 0 1.2-.15t1.075-.425q-.7-.925-1.087-2.025Q8.8 13.225 8.8 12t.388-2.325q.387-1.1 1.087-2.025-.5-.275-1.075-.425-.575-.15-1.2-.15-2.05 0-3.487 1.437Q3.075 9.95 3.075 12t1.438 3.488Q5.95 16.925 8 16.925ZM8 19.2q-3 0-5.1-2.1Q.8 15 .8 12q0-3 2.1-5.1Q5 4.8 8 4.8q1.125 0 2.137.312 1.013.313 1.863.913.85-.6 1.863-.913Q14.875 4.8 16 4.8q3 0 5.1 2.1Q23.2 9 23.2 12q0 3-2.1 5.1-2.1 2.1-5.1 2.1-1.125 0-2.137-.313-1.013-.312-1.863-.912-.85.6-1.863.912Q9.125 19.2 8 19.2Z" class="style-scope text-search-term-wizard"></path>
  </svg>
  <filter-chip id="all" selected="" class="style-scope text-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
All</filter-chip>
  <filter-chip id="any" class="style-scope text-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Any</filter-chip>
  <filter-chip id="exact" class="style-scope text-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Exact</filter-chip>
  <filter-chip id="not" class="style-scope text-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Not</filter-chip>
</span>
<span class="style-scope text-search-term-wizard">
  <mat-button id="button" filled="" class="style-scope text-search-term-wizard x-scope mat-button-0" disabled=""><span slot="label" class="style-scope text-search-term-wizard">Add AND condition</span></mat-button></span></text-search-term-wizard>
    <classification-wizard id="classificationPanel" class="style-scope search-term-wizard"><span class="style-scope classification-wizard">
  <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49" class="style-scope classification-wizard">
    <path d="M3.85 15.275V13h10.3v2.275Zm0-4.275V8.725h16.3V11Z" class="style-scope classification-wizard"></path>
  </svg>
  <outlined-text-field id="textField" placeholder="Enter a CPC here" uppercase="" class="style-scope classification-wizard" value=""><span class="style-scope outlined-text-field">
  <input id="input" spellcheck="false" autocomplete="off" class="style-scope outlined-text-field" aria-disabled="false" placeholder="Enter a CPC here">
  
</span>
</outlined-text-field>
</span>
<template is="dom-if" class="style-scope classification-wizard"></template>
<template is="dom-if" class="style-scope classification-wizard"></template>
<template is="dom-if" class="style-scope classification-wizard"></template>
<span class="style-scope classification-wizard">
  <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49" class="style-scope classification-wizard">
    <path d="M16 16.925q2.05 0 3.488-1.437Q20.925 14.05 20.925 12t-1.437-3.488Q18.05 7.075 16 7.075q-.625 0-1.2.15t-1.075.425q.7.925 1.087 2.025.388 1.1.388 2.325t-.388 2.325q-.387 1.1-1.087 2.025.5.275 1.075.425.575.15 1.2.15Zm-4-2.075q.45-.575.688-1.313.237-.737.237-1.537 0-.8-.237-1.538Q12.45 9.725 12 9.15q-.45.575-.688 1.312-.237.738-.237 1.538t.237 1.537q.238.738.688 1.313Zm-4 2.075q.625 0 1.2-.15t1.075-.425q-.7-.925-1.087-2.025Q8.8 13.225 8.8 12t.388-2.325q.387-1.1 1.087-2.025-.5-.275-1.075-.425-.575-.15-1.2-.15-2.05 0-3.487 1.437Q3.075 9.95 3.075 12t1.438 3.488Q5.95 16.925 8 16.925ZM8 19.2q-3 0-5.1-2.1Q.8 15 .8 12q0-3 2.1-5.1Q5 4.8 8 4.8q1.125 0 2.137.312 1.013.313 1.863.913.85-.6 1.863-.913Q14.875 4.8 16 4.8q3 0 5.1 2.1Q23.2 9 23.2 12q0 3-2.1 5.1-2.1 2.1-5.1 2.1-1.125 0-2.137-.313-1.013-.312-1.863-.912-.85.6-1.863.912Q9.125 19.2 8 19.2Z" class="style-scope classification-wizard"></path>
  </svg>
  <filter-chip id="notExact" class="style-scope classification-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>

    These CPCs and their children
  </filter-chip>
  <filter-chip selected="" id="exact" class="style-scope classification-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>

    These exact CPCs
  </filter-chip>
</span>
<span class="style-scope classification-wizard">
  <mat-button id="button" filled="" class="style-scope classification-wizard x-scope mat-button-0" disabled=""><span slot="label" class="style-scope classification-wizard">Add AND condition</span></mat-button>
</span></classification-wizard>
    <chemistry-search-term-wizard id="chemistryPanel" class="style-scope search-term-wizard"><span class="style-scope chemistry-search-term-wizard">
  <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49" class="style-scope chemistry-search-term-wizard">
    <path d="M3.85 15.275V13h10.3v2.275Zm0-4.275V8.725h16.3V11Z" class="style-scope chemistry-search-term-wizard"></path>
  </svg>
  <outlined-text-field id="textField" placeholder="Trade name, SMILES, InChI Key" class="style-scope chemistry-search-term-wizard" value=""><span class="style-scope outlined-text-field">
  <input id="input" spellcheck="false" autocomplete="off" class="style-scope outlined-text-field" aria-disabled="false" placeholder="Trade name, SMILES, InChI Key">
  
</span>
</outlined-text-field>
  <outlined-textarea id="textArea" placeholder="Enter multiple SMILES or InChI Keys, one per line" rows="4" class="style-scope chemistry-search-term-wizard"><span class="style-scope outlined-textarea">
  <textarea id="textarea" autocomplete="off" autocorrect="off" spellcheck="false" tabindex="0" aria-disabled="false" wrap="soft" class="style-scope outlined-textarea" rows="4" placeholder="Enter multiple SMILES or InChI Keys, one per line">  </textarea>
  <div id="helperText" class="style-scope outlined-textarea"> </div>
</span>
<span class="right-button-container style-scope outlined-textarea">
  
</span>
</outlined-textarea>
  <a href="https://support.google.com/faqs/answer/7049475#zippy=%2Csearching-by-chemistry" target="_blank" class="style-scope chemistry-search-term-wizard">
    <svg class="help style-scope chemistry-search-term-wizard" xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49">
      <path d="M11.95 18q.525 0 .888-.363.362-.362.362-.887t-.362-.887q-.363-.363-.888-.363t-.888.363q-.362.362-.362.887t.362.887q.363.363.888.363Zm-.9-3.85h1.85q0-.825.188-1.3.187-.475 1.062-1.3.65-.65 1.025-1.238.375-.587.375-1.412 0-1.4-1.025-2.15T12.1 6q-1.425 0-2.312.75-.888.75-1.238 1.8l1.65.65q.125-.45.563-.975Q11.2 7.7 12.1 7.7q.8 0 1.2.437.4.438.4.963 0 .5-.3.937-.3.438-.75.813-1.1.975-1.35 1.475-.25.5-.25 1.825ZM12 22q-2.075 0-3.9-.788-1.825-.787-3.175-2.137-1.35-1.35-2.137-3.175Q2 14.075 2 12t.788-3.9q.787-1.825 2.137-3.175 1.35-1.35 3.175-2.138Q9.925 2 12 2t3.9.787q1.825.788 3.175 2.138 1.35 1.35 2.137 3.175Q22 9.925 22 12t-.788 3.9q-.787 1.825-2.137 3.175-1.35 1.35-3.175 2.137Q14.075 22 12 22Zm0-2q3.35 0 5.675-2.325Q20 15.35 20 12q0-3.35-2.325-5.675Q15.35 4 12 4 8.65 4 6.325 6.325 4 8.65 4 12q0 3.35 2.325 5.675Q8.65 20 12 20Zm0-8Z" class="style-scope chemistry-search-term-wizard"></path>
    </svg>
  </a>
</span>
<span id="chemQuantifiers" class="style-scope chemistry-search-term-wizard">
  <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49" class="style-scope chemistry-search-term-wizard">
    <path d="M5 21.2q-1.4 0-2-1.25-.6-1.25.3-2.325l5.5-6.7V5.15h-.825q-.475 0-.8-.338Q6.85 4.475 6.85 4q0-.475.325-.813.325-.337.8-.337h8.05q.475 0 .8.337.325.338.325.813 0 .475-.325.812-.325.338-.8.338H15.2v5.775l5.5 6.7q.9 1.075.3 2.325-.6 1.25-2 1.25Zm2.15-3.275h9.7l-3.275-3.85h-3.15Zm-2 1h13.7l-5.925-7.2V5.15h-1.85v6.575Zm6.85-6.9Z" class="style-scope chemistry-search-term-wizard"></path>
  </svg>
  <filter-chip id="exact" selected="" class="style-scope chemistry-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Exact</filter-chip>
  <filter-chip id="exactBatch" class="style-scope chemistry-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Exact Batch</filter-chip>
  <filter-chip id="similar" class="style-scope chemistry-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Similar</filter-chip>
  <filter-chip id="substructure" class="style-scope chemistry-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Substructure</filter-chip>
  <filter-chip id="smarts" class="style-scope chemistry-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Substructure (SMARTS)</filter-chip>
</span>
<span id="location" class="style-scope chemistry-search-term-wizard">
  <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49" class="style-scope chemistry-search-term-wizard">
    <path d="M8 18.025h8v-2.1H8Zm0-4h8v-2.1H8ZM6.075 22.2q-.95 0-1.612-.662-.663-.663-.663-1.613V4.075q0-.95.663-1.613.662-.662 1.612-.662h8L20.2 7.925v12q0 .95-.662 1.613-.663.662-1.613.662Zm6.85-13.125v-5h-6.85v15.85h11.85V9.075Zm-6.85-5v5-5 15.85-15.85Z" class="style-scope chemistry-search-term-wizard"></path>
  </svg>
  <filter-chip id="fullDocuments" selected="" class="style-scope chemistry-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Full documents</filter-chip>
  <filter-chip id="claims" class="style-scope chemistry-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Claims only</filter-chip>
</span>
<span class="style-scope chemistry-search-term-wizard">
  <mat-button id="button" filled="" class="style-scope chemistry-search-term-wizard x-scope mat-button-0" disabled=""><span slot="label" class="style-scope chemistry-search-term-wizard">Add AND condition</span></mat-button>
</span>
</chemistry-search-term-wizard>
    <measure-search-term-wizard id="measurePanel" class="style-scope search-term-wizard"><span class="style-scope measure-search-term-wizard">
  <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49" class="style-scope measure-search-term-wizard">
    <path d="M4.1 18.15q-.95 0-1.612-.663-.663-.662-.663-1.612v-7.75q0-.95.663-1.613Q3.15 5.85 4.1 5.85h15.8q.95 0 1.613.662.662.663.662 1.613v7.75q0 .95-.662 1.612-.663.663-1.613.663Zm0-2.275h15.8v-7.75H17V12h-2V8.125h-2V12h-2V8.125H9V12H7V8.125H4.1v7.75ZM7 12h2Zm4 0h2Zm4 0h2Zm-3 0Z" class="style-scope measure-search-term-wizard"></path>
  </svg>
  <outlined-text-field id="measureExpression" placeholder="1.5 mm, 20 ft, 400-500 fahrenheit, 800 MHz, 0.01-100 mol" class="style-scope measure-search-term-wizard" value=""><span class="style-scope outlined-text-field">
  <input id="input" spellcheck="false" autocomplete="off" class="style-scope outlined-text-field" aria-disabled="false" placeholder="1.5 mm, 20 ft, 400-500 fahrenheit, 800 MHz, 0.01-100 mol">
  
</span>
</outlined-text-field>
</span>
<span class="style-scope measure-search-term-wizard">
  <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49" class="style-scope measure-search-term-wizard">
    <path d="M3.875 20.15V8.85h4.25v11.3Zm6 0V3.85h4.25v16.3Zm6 0v-7.3h4.25v7.3Z" class="style-scope measure-search-term-wizard"></path>
  </svg>
  <outlined-text-field id="context" placeholder="activity, wavelength, embossing depth, absorption" class="style-scope measure-search-term-wizard" value=""><span class="style-scope outlined-text-field">
  <input id="input" spellcheck="false" autocomplete="off" class="style-scope outlined-text-field" aria-disabled="false" placeholder="activity, wavelength, embossing depth, absorption">
  
</span>
</outlined-text-field>
</span>
<span class="style-scope measure-search-term-wizard">
  <mat-button id="button" filled="" class="style-scope measure-search-term-wizard x-scope mat-button-0" disabled=""><span slot="label" class="style-scope measure-search-term-wizard">Add AND condition</span></mat-button>
</span>
</measure-search-term-wizard>
    <numerical-search-term-wizard id="numericalPanel" class="style-scope search-term-wizard"><span class="style-scope numerical-search-term-wizard">
  <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49" class="style-scope numerical-search-term-wizard">
    <path d="M3.85 15.275V13h10.3v2.275Zm0-4.275V8.725h16.3V11Z" class="style-scope numerical-search-term-wizard"></path>
  </svg>
  <outlined-textarea id="batch" placeholder="Enter multiple Patent Application and Publication Numbers, one per line" rows="4" class="style-scope numerical-search-term-wizard"><span class="style-scope outlined-textarea">
  <textarea id="textarea" autocomplete="off" autocorrect="off" spellcheck="false" tabindex="0" aria-disabled="false" wrap="soft" class="style-scope outlined-textarea" rows="4" placeholder="Enter multiple Patent Application and Publication Numbers, one per line">  </textarea>
  <div id="helperText" class="style-scope outlined-textarea"> </div>
</span>
<span class="right-button-container style-scope outlined-textarea">
  
</span>
</outlined-textarea>
</span>
<template is="dom-if" class="style-scope numerical-search-term-wizard"></template>
<template is="dom-if" class="style-scope numerical-search-term-wizard"></template>
<span class="style-scope numerical-search-term-wizard">
  <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49" class="style-scope numerical-search-term-wizard">
    <path d="m10.825 23.45-1.6-1.625L10.9 20.15H8q-.35.9-1.162 1.488-.813.587-1.838.587-1.35 0-2.287-.937-.938-.938-.938-2.288 0-1.025.588-1.837Q2.95 16.35 3.85 16V8q-.9-.35-1.487-1.163Q1.775 6.025 1.775 5q0-1.35.938-2.288Q3.65 1.775 5 1.775q1.025 0 1.838.587Q7.65 2.95 8 3.85h2.9L9.225 2.175l1.6-1.625L15.275 5l-4.45 4.45-1.6-1.625L10.9 6.15H8q-.25.65-.725 1.125T6.15 8v8q.65.25 1.125.725T8 17.85h2.9l-1.675-1.675 1.6-1.625 4.45 4.45ZM19 22.225q-1.35 0-2.287-.937-.938-.938-.938-2.288 0-1.05.588-1.863.587-.812 1.487-1.137V8q-.9-.325-1.487-1.137-.588-.813-.588-1.863 0-1.35.938-2.288.937-.937 2.287-.937 1.35 0 2.288.937.937.938.937 2.288 0 1.05-.587 1.863Q21.05 7.675 20.15 8v8q.9.35 1.488 1.163.587.812.587 1.837 0 1.35-.937 2.288-.938.937-2.288.937ZM5 19.95q.4 0 .675-.275T5.95 19q0-.4-.275-.675T5 18.05q-.4 0-.675.275T4.05 19q0 .4.275.675T5 19.95Zm14 0q.4 0 .675-.275T19.95 19q0-.4-.275-.675T19 18.05q-.4 0-.675.275T18.05 19q0 .4.275.675t.675.275Zm-14-14q.4 0 .675-.275T5.95 5q0-.4-.275-.675T5 4.05q-.4 0-.675.275T4.05 5q0 .4.275.675T5 5.95Zm14 0q.4 0 .675-.275T19.95 5q0-.4-.275-.675T19 4.05q-.4 0-.675.275T18.05 5q0 .4.275.675T19 5.95ZM5 19Zm14 0ZM5 5Zm14 0Z" class="style-scope numerical-search-term-wizard"></path>
  </svg>
  <filter-chip id="applicationNumbers" class="style-scope numerical-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Application Numbers</filter-chip>
  <filter-chip id="publicationNumbers" class="style-scope numerical-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Publication Numbers</filter-chip>
  <filter-chip id="either" selected="" class="style-scope numerical-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Either</filter-chip>
</span>
<span class="style-scope numerical-search-term-wizard">
  <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49" class="style-scope numerical-search-term-wizard">
    <path d="M12 22.2q-2.1 0-3.962-.8-1.863-.8-3.25-2.175Q3.4 17.85 2.6 15.988 1.8 14.125 1.8 12t.8-3.988q.8-1.862 2.188-3.237Q6.175 3.4 8.038 2.6 9.9 1.8 12 1.8q2.125 0 3.988.8 1.862.8 3.237 2.175Q20.6 6.15 21.4 8.012q.8 1.863.8 3.988t-.8 3.988q-.8 1.862-2.175 3.237Q17.85 20.6 15.988 21.4q-1.863.8-3.988.8Zm-1-2.325v-1.95q-.8 0-1.387-.575-.588-.575-.588-1.4v-.975L4.275 10.2q-.075.45-.137.9-.063.45-.063.9 0 3 1.975 5.25T11 19.875Zm6.85-2.525q.5-.55.888-1.175.387-.625.65-1.313.262-.687.4-1.412.137-.725.137-1.45 0-2.425-1.338-4.438-1.337-2.012-3.612-2.887v.375q0 .825-.587 1.4-.588.575-1.388.575h-2v2q0 .4-.287.687-.288.288-.688.288h-2v2h5.95q.425 0 .713.287.287.288.287.688v2.975h1q.65 0 1.163.387.512.388.712 1.013Z" class="style-scope numerical-search-term-wizard"></path>
  </svg>
  <outlined-text-field id="countryFilter" placeholder="Restrict search to these countries" class="style-scope numerical-search-term-wizard" value=""><span class="style-scope outlined-text-field">
  <input id="input" spellcheck="false" autocomplete="off" class="style-scope outlined-text-field" aria-disabled="false" placeholder="Restrict search to these countries">
  
</span>
</outlined-text-field>
</span>
<span class="style-scope numerical-search-term-wizard">
  <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49" class="style-scope numerical-search-term-wizard">
    <path d="M2.85 18.225V15.95h6.3v2.275Zm0-5.075v-2.3h12.3v2.3Zm0-5.1V5.775h18.3V8.05Z" class="style-scope numerical-search-term-wizard"></path>
  </svg>
  <outlined-text-field id="countryOrder" placeholder="Prefer results from these countries in this order" class="style-scope numerical-search-term-wizard" value="US, EP, WO, JP, CN"><span class="style-scope outlined-text-field">
  <input id="input" spellcheck="false" autocomplete="off" class="style-scope outlined-text-field" aria-disabled="false" placeholder="Prefer results from these countries in this order">
  
</span>
</outlined-text-field>
</span>
<span class="style-scope numerical-search-term-wizard">
  <mat-button id="button" filled="" class="style-scope numerical-search-term-wizard x-scope mat-button-0" disabled=""><span slot="label" class="style-scope numerical-search-term-wizard">Add AND condition</span></mat-button>
</span>
</numerical-search-term-wizard>
  </span>

</pat-dialog>
</search-term-wizard>
          
      <div content="" class="style-scope search-result">
        <div id="mainContainer" class="vertical layout style-scope search-result">
          <template is="dom-if" class="style-scope search-result"></template>
          <result-container class="style-scope search-result">
    

    <paper-spinner class="style-scope result-container x-scope paper-spinner-0" aria-hidden="true" hidden=""><div id="spinnerContainer" class="  style-scope paper-spinner"><div class="spinner-layer layer-1 style-scope paper-spinner"><div class="circle-clipper left style-scope paper-spinner"></div><div class="circle-clipper right style-scope paper-spinner"></div></div><div class="spinner-layer layer-2 style-scope paper-spinner"><div class="circle-clipper left style-scope paper-spinner"></div><div class="circle-clipper right style-scope paper-spinner"></div></div><div class="spinner-layer layer-3 style-scope paper-spinner"><div class="circle-clipper left style-scope paper-spinner"></div><div class="circle-clipper right style-scope paper-spinner"></div></div><div class="spinner-layer layer-4 style-scope paper-spinner"><div class="circle-clipper left style-scope paper-spinner"></div><div class="circle-clipper right style-scope paper-spinner"></div></div></div></paper-spinner>
    
      <patent-result class="style-scope result-container">
    
    
    <div class="layout horizontal style-scope patent-result">
      <div style="width: 1200px; height: auto; overflow: auto;" class="style-scope patent-result">
        <div id="wrapper" tabindex="0" class="style-scope patent-result">
          <result-tags class="style-scope patent-result"></result-tags>
          <h1 id="title" class="scroll-target style-scope patent-result">
            Drosophila g protein coupled receptors, nucleic acids, and methods related to the same 
     </h1>
          <div class="horizontal layout style-scope patent-result">
            
            <div class="flex-3 style-scope patent-result" style="margin-top:-20px;overflow:hidden;" marginright="">
              <section id="abstract" class="style-scope patent-result" hidden="">
                <h3 class="layout horizontal style-scope patent-result">
                  <div class="flex style-scope patent-result">Abstract</div>
                  <div class="righthead style-scope patent-result" hidden="">translated from </div>
                </h3>
                <patent-text name="abstract" class="style-scope patent-result">
    
    
    <template is="dom-if" restamp="" class="style-scope patent-text"></template>
    <overlay-tooltip id="tooltip" class="style-scope patent-text" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      
    </div>
  </overlay-tooltip>
    <div class="layout horizontal style-scope patent-text">
      <section id="text" class="flex style-scope patent-text"></section>
      <div id="margin" class="style-scope patent-text">
        <div id="createHighlight" style="top: 0px;" class="style-scope patent-text" hidden="">
          <iron-icon id="comment" icon="editor:insert-comment" class="style-scope patent-text x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M20 2H4c-1.1 0-2 .9-2 2v12c0 1.1.9 2 2 2h14l4 4V4c0-1.1-.9-2-2-2zm-2 12H6v-2h12v2zm0-3H6V9h12v2zm0-3H6V6h12v2z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
        </div>
      </div>
    </div>
  </patent-text>
              </section>
              
              <template is="dom-if" class="style-scope patent-result"></template>
              
              <section id="classifications" class="style-scope patent-result">
                <h3 class="layout horizontal style-scope patent-result">
                  <div class="flex style-scope patent-result">Classifications</div>
                  <div id="machineClassifiedWarning" class="righthead tooltip-hint style-scope patent-result" hidden="">machine-classified</div>
                  <div id="machineClassifiedCPCWarning" class="righthead tooltip-hint style-scope patent-result" hidden="">cpc-machine-classified</div>
                  <div id="machineClassifiedFtermWarning" class="righthead tooltip-hint style-scope patent-result" hidden="">fterm-machine-classified</div>
                  <div class="style-scope patent-result" hidden="">&nbsp;</div>
                  <div id="familyClassifiedFtermWarning" class="righthead tooltip-hint style-scope patent-result" hidden="">fterm-family-classified</div>
                </h3>
                <overlay-tooltip positioning-target="#machineClassifiedWarning" direction="bottom" class="style-scope patent-result" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      The classifications are assigned by a computer and are not a legal conclusion. <br class="style-scope patent-result"> Google has not performed a legal analysis and makes no representation as to the accuracy of the classifications listed.
    </div>
  </overlay-tooltip>
                <overlay-tooltip positioning-target="#machineClassifiedCPCWarning" direction="bottom" class="style-scope patent-result" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      The CPC classifications are assigned by a computer and are not a legal conclusion. <br class="style-scope patent-result"> Google has not performed a legal analysis and makes no representation as to the accuracy of the classifications listed.
    </div>
  </overlay-tooltip>
                <overlay-tooltip positioning-target="#familyClassifiedFtermWarning" direction="bottom" class="style-scope patent-result" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      The F-term classifications are assigned based on a patent family member containing these classification codes.
    </div>
  </overlay-tooltip>
                <overlay-tooltip positioning-target="#machineClassifiedFtermWarning" direction="bottom" class="style-scope patent-result" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      The F-term classifications are assigned by a computer and are not a legal conclusion. <br class="style-scope patent-result"> Google has not performed a legal analysis and makes no representation as to the accuracy of the classifications listed.
    </div>
  </overlay-tooltip>
                <classification-viewer class="style-scope patent-result">
    
    <div class="table style-scope classification-viewer">
      
        <classification-tree class="style-scope classification-viewer">
    
    <div class="style-scope classification-tree">
      <div class="layout horizontal wrap style-scope classification-tree">
        <div class="flex style-scope classification-tree">
          
            
              <div class="style-scope classification-tree" hidden="">
                <concept-mention class="style-scope classification-tree">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                  <state-modifier class="code style-scope classification-tree" act="{&quot;type&quot;: &quot;QUERY_ADD_CPC&quot;, &quot;cpc&quot;: &quot;$cpc&quot;}" data-cpc="C"><a id="link" href="#" class="style-scope state-modifier">C</a></state-modifier>
                  <span class="description style-scope classification-tree">CHEMISTRY; METALLURGY</span>
                
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
              </div>
            <template is="dom-if" class="style-scope classification-tree"></template>
            <template is="dom-if" class="style-scope classification-tree"></template>
          
            
              <div class="style-scope classification-tree" hidden="">
                <concept-mention class="style-scope classification-tree">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                  <state-modifier class="code style-scope classification-tree" act="{&quot;type&quot;: &quot;QUERY_ADD_CPC&quot;, &quot;cpc&quot;: &quot;$cpc&quot;}" data-cpc="C12"><a id="link" href="#" class="style-scope state-modifier">C12</a></state-modifier>
                  <span class="description style-scope classification-tree">BIOCHEMISTRY; BEER; SPIRITS; WINE; VINEGAR; MICROBIOLOGY; ENZYMOLOGY; MUTATION OR GENETIC ENGINEERING</span>
                
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
              </div>
            <template is="dom-if" class="style-scope classification-tree"></template>
            <template is="dom-if" class="style-scope classification-tree"></template>
          
            
              <div class="style-scope classification-tree" hidden="">
                <concept-mention class="style-scope classification-tree">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                  <state-modifier class="code style-scope classification-tree" act="{&quot;type&quot;: &quot;QUERY_ADD_CPC&quot;, &quot;cpc&quot;: &quot;$cpc&quot;}" data-cpc="C12N"><a id="link" href="#" class="style-scope state-modifier">C12N</a></state-modifier>
                  <span class="description style-scope classification-tree">MICROORGANISMS OR ENZYMES; COMPOSITIONS THEREOF; PROPAGATING, PRESERVING, OR MAINTAINING MICROORGANISMS; MUTATION OR GENETIC ENGINEERING; CULTURE MEDIA</span>
                
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
              </div>
            <template is="dom-if" class="style-scope classification-tree"></template>
            <template is="dom-if" class="style-scope classification-tree"></template>
          
            
              <div class="style-scope classification-tree" hidden="">
                <concept-mention class="style-scope classification-tree">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                  <state-modifier class="code style-scope classification-tree" act="{&quot;type&quot;: &quot;QUERY_ADD_CPC&quot;, &quot;cpc&quot;: &quot;$cpc&quot;}" data-cpc="C12N15/00"><a id="link" href="#" class="style-scope state-modifier">C12N15/00</a></state-modifier>
                  <span class="description style-scope classification-tree">Mutation or genetic engineering; DNA or RNA concerning genetic engineering, vectors, e.g. plasmids, or their isolation, preparation or purification; Use of hosts therefor</span>
                
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
              </div>
            <template is="dom-if" class="style-scope classification-tree"></template>
            <template is="dom-if" class="style-scope classification-tree"></template>
          
            
              <div class="style-scope classification-tree" hidden="">
                <concept-mention class="style-scope classification-tree">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                  <state-modifier class="code style-scope classification-tree" act="{&quot;type&quot;: &quot;QUERY_ADD_CPC&quot;, &quot;cpc&quot;: &quot;$cpc&quot;}" data-cpc="C12N15/09"><a id="link" href="#" class="style-scope state-modifier">C12N15/09</a></state-modifier>
                  <span class="description style-scope classification-tree">Recombinant DNA-technology</span>
                
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
              </div>
            <template is="dom-if" class="style-scope classification-tree"></template>
            <template is="dom-if" class="style-scope classification-tree"></template>
          
            
              <div class="style-scope classification-tree">
                <concept-mention class="style-scope classification-tree">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                  <state-modifier class="code style-scope classification-tree" act="{&quot;type&quot;: &quot;QUERY_ADD_CPC&quot;, &quot;cpc&quot;: &quot;$cpc&quot;}" first="true" data-cpc="C12N15/10"><a id="link" href="#" class="style-scope state-modifier">C12N15/10</a></state-modifier>
                  <span class="description style-scope classification-tree">Processes for the isolation, preparation or purification of DNA or RNA</span>
                
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
              </div>
            <template is="dom-if" class="style-scope classification-tree"></template>
            <template is="dom-if" class="style-scope classification-tree"></template>
          <template is="dom-repeat" class="style-scope classification-tree"></template>
        </div>
      </div>
    </div>
  </classification-tree>
      <template is="dom-repeat" class="style-scope classification-viewer"></template>
      
      <div class="style-scope classification-viewer">
        
          <classification-tree class="style-scope classification-viewer">
    
    <div class="style-scope classification-tree">
      <div class="layout horizontal wrap style-scope classification-tree">
        <div class="flex style-scope classification-tree">
          
            
              <div class="style-scope classification-tree" hidden="">
                <concept-mention class="style-scope classification-tree">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                  <state-modifier class="code style-scope classification-tree" act="{&quot;type&quot;: &quot;QUERY_ADD_CPC&quot;, &quot;cpc&quot;: &quot;$cpc&quot;}" data-cpc="C"><a id="link" href="#" class="style-scope state-modifier">C</a></state-modifier>
                  <span class="description style-scope classification-tree">CHEMISTRY; METALLURGY</span>
                
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
              </div>
            <template is="dom-if" class="style-scope classification-tree"></template>
            <template is="dom-if" class="style-scope classification-tree"></template>
          
            
              <div class="style-scope classification-tree" hidden="">
                <concept-mention class="style-scope classification-tree">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                  <state-modifier class="code style-scope classification-tree" act="{&quot;type&quot;: &quot;QUERY_ADD_CPC&quot;, &quot;cpc&quot;: &quot;$cpc&quot;}" data-cpc="C07"><a id="link" href="#" class="style-scope state-modifier">C07</a></state-modifier>
                  <span class="description style-scope classification-tree">ORGANIC CHEMISTRY</span>
                
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
              </div>
            <template is="dom-if" class="style-scope classification-tree"></template>
            <template is="dom-if" class="style-scope classification-tree"></template>
          
            
              <div class="style-scope classification-tree" hidden="">
                <concept-mention class="style-scope classification-tree">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                  <state-modifier class="code style-scope classification-tree" act="{&quot;type&quot;: &quot;QUERY_ADD_CPC&quot;, &quot;cpc&quot;: &quot;$cpc&quot;}" data-cpc="C07K"><a id="link" href="#" class="style-scope state-modifier">C07K</a></state-modifier>
                  <span class="description style-scope classification-tree">PEPTIDES</span>
                
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
              </div>
            <template is="dom-if" class="style-scope classification-tree"></template>
            <template is="dom-if" class="style-scope classification-tree"></template>
          
            
              <div class="style-scope classification-tree" hidden="">
                <concept-mention class="style-scope classification-tree">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                  <state-modifier class="code style-scope classification-tree" act="{&quot;type&quot;: &quot;QUERY_ADD_CPC&quot;, &quot;cpc&quot;: &quot;$cpc&quot;}" data-cpc="C07K14/00"><a id="link" href="#" class="style-scope state-modifier">C07K14/00</a></state-modifier>
                  <span class="description style-scope classification-tree">Peptides having more than 20 amino acids; Gastrins; Somatostatins; Melanotropins; Derivatives thereof</span>
                
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
              </div>
            <template is="dom-if" class="style-scope classification-tree"></template>
            <template is="dom-if" class="style-scope classification-tree"></template>
          
            
              <div class="style-scope classification-tree" hidden="">
                <concept-mention class="style-scope classification-tree">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                  <state-modifier class="code style-scope classification-tree" act="{&quot;type&quot;: &quot;QUERY_ADD_CPC&quot;, &quot;cpc&quot;: &quot;$cpc&quot;}" data-cpc="C07K14/435"><a id="link" href="#" class="style-scope state-modifier">C07K14/435</a></state-modifier>
                  <span class="description style-scope classification-tree">Peptides having more than 20 amino acids; Gastrins; Somatostatins; Melanotropins; Derivatives thereof from animals; from humans</span>
                
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
              </div>
            <template is="dom-if" class="style-scope classification-tree"></template>
            <template is="dom-if" class="style-scope classification-tree"></template>
          
            
              <div class="style-scope classification-tree">
                <concept-mention class="style-scope classification-tree">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                  <state-modifier class="code style-scope classification-tree" act="{&quot;type&quot;: &quot;QUERY_ADD_CPC&quot;, &quot;cpc&quot;: &quot;$cpc&quot;}" first="true" data-cpc="C07K14/705"><a id="link" href="#" class="style-scope state-modifier">C07K14/705</a></state-modifier>
                  <span class="description style-scope classification-tree">Receptors; Cell surface antigens; Cell surface determinants</span>
                
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
              </div>
            <template is="dom-if" class="style-scope classification-tree"></template>
            <template is="dom-if" class="style-scope classification-tree"></template>
          <template is="dom-repeat" class="style-scope classification-tree"></template>
        </div>
      </div>
    </div>
  </classification-tree>
        
          <classification-tree class="style-scope classification-viewer">
    
    <div class="style-scope classification-tree">
      <div class="layout horizontal wrap style-scope classification-tree">
        <div class="flex style-scope classification-tree">
          
            
              <div class="style-scope classification-tree" hidden="">
                <concept-mention class="style-scope classification-tree">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                  <state-modifier class="code style-scope classification-tree" act="{&quot;type&quot;: &quot;QUERY_ADD_CPC&quot;, &quot;cpc&quot;: &quot;$cpc&quot;}" data-cpc="A"><a id="link" href="#" class="style-scope state-modifier">A</a></state-modifier>
                  <span class="description style-scope classification-tree">HUMAN NECESSITIES</span>
                
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
              </div>
            <template is="dom-if" class="style-scope classification-tree"></template>
            <template is="dom-if" class="style-scope classification-tree"></template>
          
            
              <div class="style-scope classification-tree" hidden="">
                <concept-mention class="style-scope classification-tree">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                  <state-modifier class="code style-scope classification-tree" act="{&quot;type&quot;: &quot;QUERY_ADD_CPC&quot;, &quot;cpc&quot;: &quot;$cpc&quot;}" data-cpc="A61"><a id="link" href="#" class="style-scope state-modifier">A61</a></state-modifier>
                  <span class="description style-scope classification-tree">MEDICAL OR VETERINARY SCIENCE; HYGIENE</span>
                
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
              </div>
            <template is="dom-if" class="style-scope classification-tree"></template>
            <template is="dom-if" class="style-scope classification-tree"></template>
          
            
              <div class="style-scope classification-tree" hidden="">
                <concept-mention class="style-scope classification-tree">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                  <state-modifier class="code style-scope classification-tree" act="{&quot;type&quot;: &quot;QUERY_ADD_CPC&quot;, &quot;cpc&quot;: &quot;$cpc&quot;}" data-cpc="A61P"><a id="link" href="#" class="style-scope state-modifier">A61P</a></state-modifier>
                  <span class="description style-scope classification-tree">SPECIFIC THERAPEUTIC ACTIVITY OF CHEMICAL COMPOUNDS OR MEDICINAL PREPARATIONS</span>
                
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
              </div>
            <template is="dom-if" class="style-scope classification-tree"></template>
            <template is="dom-if" class="style-scope classification-tree"></template>
          
            
              <div class="style-scope classification-tree" hidden="">
                <concept-mention class="style-scope classification-tree">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                  <state-modifier class="code style-scope classification-tree" act="{&quot;type&quot;: &quot;QUERY_ADD_CPC&quot;, &quot;cpc&quot;: &quot;$cpc&quot;}" data-cpc="A61P33/00"><a id="link" href="#" class="style-scope state-modifier">A61P33/00</a></state-modifier>
                  <span class="description style-scope classification-tree">Antiparasitic agents</span>
                
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
              </div>
            <template is="dom-if" class="style-scope classification-tree"></template>
            <template is="dom-if" class="style-scope classification-tree"></template>
          
            
              <div class="style-scope classification-tree">
                <concept-mention class="style-scope classification-tree">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                  <state-modifier class="code style-scope classification-tree" act="{&quot;type&quot;: &quot;QUERY_ADD_CPC&quot;, &quot;cpc&quot;: &quot;$cpc&quot;}" data-cpc="A61P33/14"><a id="link" href="#" class="style-scope state-modifier">A61P33/14</a></state-modifier>
                  <span class="description style-scope classification-tree">Ectoparasiticides, e.g. scabicides</span>
                
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
              </div>
            <template is="dom-if" class="style-scope classification-tree"></template>
            <template is="dom-if" class="style-scope classification-tree"></template>
          <template is="dom-repeat" class="style-scope classification-tree"></template>
        </div>
      </div>
    </div>
  </classification-tree>
        
          <classification-tree class="style-scope classification-viewer">
    
    <div class="style-scope classification-tree">
      <div class="layout horizontal wrap style-scope classification-tree">
        <div class="flex style-scope classification-tree">
          
            
              <div class="style-scope classification-tree" hidden="">
                <concept-mention class="style-scope classification-tree">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                  <state-modifier class="code style-scope classification-tree" act="{&quot;type&quot;: &quot;QUERY_ADD_CPC&quot;, &quot;cpc&quot;: &quot;$cpc&quot;}" data-cpc="C"><a id="link" href="#" class="style-scope state-modifier">C</a></state-modifier>
                  <span class="description style-scope classification-tree">CHEMISTRY; METALLURGY</span>
                
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
              </div>
            <template is="dom-if" class="style-scope classification-tree"></template>
            <template is="dom-if" class="style-scope classification-tree"></template>
          
            
              <div class="style-scope classification-tree" hidden="">
                <concept-mention class="style-scope classification-tree">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                  <state-modifier class="code style-scope classification-tree" act="{&quot;type&quot;: &quot;QUERY_ADD_CPC&quot;, &quot;cpc&quot;: &quot;$cpc&quot;}" data-cpc="C07"><a id="link" href="#" class="style-scope state-modifier">C07</a></state-modifier>
                  <span class="description style-scope classification-tree">ORGANIC CHEMISTRY</span>
                
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
              </div>
            <template is="dom-if" class="style-scope classification-tree"></template>
            <template is="dom-if" class="style-scope classification-tree"></template>
          
            
              <div class="style-scope classification-tree" hidden="">
                <concept-mention class="style-scope classification-tree">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                  <state-modifier class="code style-scope classification-tree" act="{&quot;type&quot;: &quot;QUERY_ADD_CPC&quot;, &quot;cpc&quot;: &quot;$cpc&quot;}" data-cpc="C07K"><a id="link" href="#" class="style-scope state-modifier">C07K</a></state-modifier>
                  <span class="description style-scope classification-tree">PEPTIDES</span>
                
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
              </div>
            <template is="dom-if" class="style-scope classification-tree"></template>
            <template is="dom-if" class="style-scope classification-tree"></template>
          
            
              <div class="style-scope classification-tree" hidden="">
                <concept-mention class="style-scope classification-tree">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                  <state-modifier class="code style-scope classification-tree" act="{&quot;type&quot;: &quot;QUERY_ADD_CPC&quot;, &quot;cpc&quot;: &quot;$cpc&quot;}" data-cpc="C07K14/00"><a id="link" href="#" class="style-scope state-modifier">C07K14/00</a></state-modifier>
                  <span class="description style-scope classification-tree">Peptides having more than 20 amino acids; Gastrins; Somatostatins; Melanotropins; Derivatives thereof</span>
                
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
              </div>
            <template is="dom-if" class="style-scope classification-tree"></template>
            <template is="dom-if" class="style-scope classification-tree"></template>
          
            
              <div class="style-scope classification-tree" hidden="">
                <concept-mention class="style-scope classification-tree">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                  <state-modifier class="code style-scope classification-tree" act="{&quot;type&quot;: &quot;QUERY_ADD_CPC&quot;, &quot;cpc&quot;: &quot;$cpc&quot;}" data-cpc="C07K14/435"><a id="link" href="#" class="style-scope state-modifier">C07K14/435</a></state-modifier>
                  <span class="description style-scope classification-tree">Peptides having more than 20 amino acids; Gastrins; Somatostatins; Melanotropins; Derivatives thereof from animals; from humans</span>
                
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
              </div>
            <template is="dom-if" class="style-scope classification-tree"></template>
            <template is="dom-if" class="style-scope classification-tree"></template>
          
            
              <div class="style-scope classification-tree" hidden="">
                <concept-mention class="style-scope classification-tree">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                  <state-modifier class="code style-scope classification-tree" act="{&quot;type&quot;: &quot;QUERY_ADD_CPC&quot;, &quot;cpc&quot;: &quot;$cpc&quot;}" data-cpc="C07K14/43504"><a id="link" href="#" class="style-scope state-modifier">C07K14/43504</a></state-modifier>
                  <span class="description style-scope classification-tree">Peptides having more than 20 amino acids; Gastrins; Somatostatins; Melanotropins; Derivatives thereof from animals; from humans from invertebrates</span>
                
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
              </div>
            <template is="dom-if" class="style-scope classification-tree"></template>
            <template is="dom-if" class="style-scope classification-tree"></template>
          
            
              <div class="style-scope classification-tree" hidden="">
                <concept-mention class="style-scope classification-tree">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                  <state-modifier class="code style-scope classification-tree" act="{&quot;type&quot;: &quot;QUERY_ADD_CPC&quot;, &quot;cpc&quot;: &quot;$cpc&quot;}" data-cpc="C07K14/43563"><a id="link" href="#" class="style-scope state-modifier">C07K14/43563</a></state-modifier>
                  <span class="description style-scope classification-tree">Peptides having more than 20 amino acids; Gastrins; Somatostatins; Melanotropins; Derivatives thereof from animals; from humans from invertebrates from insects</span>
                
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
              </div>
            <template is="dom-if" class="style-scope classification-tree"></template>
            <template is="dom-if" class="style-scope classification-tree"></template>
          
            
              <div class="style-scope classification-tree" hidden="">
                <concept-mention class="style-scope classification-tree">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                  <state-modifier class="code style-scope classification-tree" act="{&quot;type&quot;: &quot;QUERY_ADD_CPC&quot;, &quot;cpc&quot;: &quot;$cpc&quot;}" data-cpc="C07K14/43577"><a id="link" href="#" class="style-scope state-modifier">C07K14/43577</a></state-modifier>
                  <span class="description style-scope classification-tree">Peptides having more than 20 amino acids; Gastrins; Somatostatins; Melanotropins; Derivatives thereof from animals; from humans from invertebrates from insects from flies</span>
                
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
              </div>
            <template is="dom-if" class="style-scope classification-tree"></template>
            <template is="dom-if" class="style-scope classification-tree"></template>
          
            
              <div class="style-scope classification-tree">
                <concept-mention class="style-scope classification-tree">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                  <state-modifier class="code style-scope classification-tree" act="{&quot;type&quot;: &quot;QUERY_ADD_CPC&quot;, &quot;cpc&quot;: &quot;$cpc&quot;}" data-cpc="C07K14/43581"><a id="link" href="#" class="style-scope state-modifier">C07K14/43581</a></state-modifier>
                  <span class="description style-scope classification-tree">Peptides having more than 20 amino acids; Gastrins; Somatostatins; Melanotropins; Derivatives thereof from animals; from humans from invertebrates from insects from flies from Drosophila</span>
                
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
              </div>
            <template is="dom-if" class="style-scope classification-tree"></template>
            <template is="dom-if" class="style-scope classification-tree"></template>
          <template is="dom-repeat" class="style-scope classification-tree"></template>
        </div>
      </div>
    </div>
  </classification-tree>
        
          <classification-tree class="style-scope classification-viewer">
    
    <div class="style-scope classification-tree">
      <div class="layout horizontal wrap style-scope classification-tree">
        <div class="flex style-scope classification-tree">
          
            
              <div class="style-scope classification-tree" hidden="">
                <concept-mention class="style-scope classification-tree">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                  <state-modifier class="code style-scope classification-tree" act="{&quot;type&quot;: &quot;QUERY_ADD_CPC&quot;, &quot;cpc&quot;: &quot;$cpc&quot;}" data-cpc="C"><a id="link" href="#" class="style-scope state-modifier">C</a></state-modifier>
                  <span class="description style-scope classification-tree">CHEMISTRY; METALLURGY</span>
                
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
              </div>
            <template is="dom-if" class="style-scope classification-tree"></template>
            <template is="dom-if" class="style-scope classification-tree"></template>
          
            
              <div class="style-scope classification-tree" hidden="">
                <concept-mention class="style-scope classification-tree">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                  <state-modifier class="code style-scope classification-tree" act="{&quot;type&quot;: &quot;QUERY_ADD_CPC&quot;, &quot;cpc&quot;: &quot;$cpc&quot;}" data-cpc="C12"><a id="link" href="#" class="style-scope state-modifier">C12</a></state-modifier>
                  <span class="description style-scope classification-tree">BIOCHEMISTRY; BEER; SPIRITS; WINE; VINEGAR; MICROBIOLOGY; ENZYMOLOGY; MUTATION OR GENETIC ENGINEERING</span>
                
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
              </div>
            <template is="dom-if" class="style-scope classification-tree"></template>
            <template is="dom-if" class="style-scope classification-tree"></template>
          
            
              <div class="style-scope classification-tree" hidden="">
                <concept-mention class="style-scope classification-tree">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                  <state-modifier class="code style-scope classification-tree" act="{&quot;type&quot;: &quot;QUERY_ADD_CPC&quot;, &quot;cpc&quot;: &quot;$cpc&quot;}" data-cpc="C12N"><a id="link" href="#" class="style-scope state-modifier">C12N</a></state-modifier>
                  <span class="description style-scope classification-tree">MICROORGANISMS OR ENZYMES; COMPOSITIONS THEREOF; PROPAGATING, PRESERVING, OR MAINTAINING MICROORGANISMS; MUTATION OR GENETIC ENGINEERING; CULTURE MEDIA</span>
                
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
              </div>
            <template is="dom-if" class="style-scope classification-tree"></template>
            <template is="dom-if" class="style-scope classification-tree"></template>
          
            
              <div class="style-scope classification-tree" hidden="">
                <concept-mention class="style-scope classification-tree">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                  <state-modifier class="code style-scope classification-tree" act="{&quot;type&quot;: &quot;QUERY_ADD_CPC&quot;, &quot;cpc&quot;: &quot;$cpc&quot;}" data-cpc="C12N15/00"><a id="link" href="#" class="style-scope state-modifier">C12N15/00</a></state-modifier>
                  <span class="description style-scope classification-tree">Mutation or genetic engineering; DNA or RNA concerning genetic engineering, vectors, e.g. plasmids, or their isolation, preparation or purification; Use of hosts therefor</span>
                
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
              </div>
            <template is="dom-if" class="style-scope classification-tree"></template>
            <template is="dom-if" class="style-scope classification-tree"></template>
          
            
              <div class="style-scope classification-tree">
                <concept-mention class="style-scope classification-tree">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                  <state-modifier class="code style-scope classification-tree" act="{&quot;type&quot;: &quot;QUERY_ADD_CPC&quot;, &quot;cpc&quot;: &quot;$cpc&quot;}" data-cpc="C12N15/09"><a id="link" href="#" class="style-scope state-modifier">C12N15/09</a></state-modifier>
                  <span class="description style-scope classification-tree">Recombinant DNA-technology</span>
                
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
              </div>
            <template is="dom-if" class="style-scope classification-tree"></template>
            <template is="dom-if" class="style-scope classification-tree"></template>
          <template is="dom-repeat" class="style-scope classification-tree"></template>
        </div>
      </div>
    </div>
  </classification-tree>
        
          <classification-tree class="style-scope classification-viewer">
    
    <div class="style-scope classification-tree">
      <div class="layout horizontal wrap style-scope classification-tree">
        <div class="flex style-scope classification-tree">
          
            
              <div class="style-scope classification-tree" hidden="">
                <concept-mention class="style-scope classification-tree">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                  <state-modifier class="code style-scope classification-tree" act="{&quot;type&quot;: &quot;QUERY_ADD_CPC&quot;, &quot;cpc&quot;: &quot;$cpc&quot;}" data-cpc="C"><a id="link" href="#" class="style-scope state-modifier">C</a></state-modifier>
                  <span class="description style-scope classification-tree">CHEMISTRY; METALLURGY</span>
                
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
              </div>
            <template is="dom-if" class="style-scope classification-tree"></template>
            <template is="dom-if" class="style-scope classification-tree"></template>
          
            
              <div class="style-scope classification-tree" hidden="">
                <concept-mention class="style-scope classification-tree">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                  <state-modifier class="code style-scope classification-tree" act="{&quot;type&quot;: &quot;QUERY_ADD_CPC&quot;, &quot;cpc&quot;: &quot;$cpc&quot;}" data-cpc="C12"><a id="link" href="#" class="style-scope state-modifier">C12</a></state-modifier>
                  <span class="description style-scope classification-tree">BIOCHEMISTRY; BEER; SPIRITS; WINE; VINEGAR; MICROBIOLOGY; ENZYMOLOGY; MUTATION OR GENETIC ENGINEERING</span>
                
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
              </div>
            <template is="dom-if" class="style-scope classification-tree"></template>
            <template is="dom-if" class="style-scope classification-tree"></template>
          
            
              <div class="style-scope classification-tree" hidden="">
                <concept-mention class="style-scope classification-tree">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                  <state-modifier class="code style-scope classification-tree" act="{&quot;type&quot;: &quot;QUERY_ADD_CPC&quot;, &quot;cpc&quot;: &quot;$cpc&quot;}" data-cpc="C12N"><a id="link" href="#" class="style-scope state-modifier">C12N</a></state-modifier>
                  <span class="description style-scope classification-tree">MICROORGANISMS OR ENZYMES; COMPOSITIONS THEREOF; PROPAGATING, PRESERVING, OR MAINTAINING MICROORGANISMS; MUTATION OR GENETIC ENGINEERING; CULTURE MEDIA</span>
                
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
              </div>
            <template is="dom-if" class="style-scope classification-tree"></template>
            <template is="dom-if" class="style-scope classification-tree"></template>
          
            
              <div class="style-scope classification-tree" hidden="">
                <concept-mention class="style-scope classification-tree">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                  <state-modifier class="code style-scope classification-tree" act="{&quot;type&quot;: &quot;QUERY_ADD_CPC&quot;, &quot;cpc&quot;: &quot;$cpc&quot;}" data-cpc="C12N2799/00"><a id="link" href="#" class="style-scope state-modifier">C12N2799/00</a></state-modifier>
                  <span class="description style-scope classification-tree">Uses of viruses</span>
                
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
              </div>
            <template is="dom-if" class="style-scope classification-tree"></template>
            <template is="dom-if" class="style-scope classification-tree"></template>
          
            
              <div class="style-scope classification-tree" hidden="">
                <concept-mention class="style-scope classification-tree">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                  <state-modifier class="code style-scope classification-tree" act="{&quot;type&quot;: &quot;QUERY_ADD_CPC&quot;, &quot;cpc&quot;: &quot;$cpc&quot;}" data-cpc="C12N2799/02"><a id="link" href="#" class="style-scope state-modifier">C12N2799/02</a></state-modifier>
                  <span class="description style-scope classification-tree">Uses of viruses as vector</span>
                
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
              </div>
            <template is="dom-if" class="style-scope classification-tree"></template>
            <template is="dom-if" class="style-scope classification-tree"></template>
          
            
              <div class="style-scope classification-tree">
                <concept-mention class="style-scope classification-tree">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                  <state-modifier class="code style-scope classification-tree" act="{&quot;type&quot;: &quot;QUERY_ADD_CPC&quot;, &quot;cpc&quot;: &quot;$cpc&quot;}" additional="true" data-cpc="C12N2799/021"><a id="link" href="#" class="style-scope state-modifier">C12N2799/021</a></state-modifier>
                  <span class="description style-scope classification-tree">Uses of viruses as vector for the expression of a heterologous nucleic acid</span>
                
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
              </div>
            <template is="dom-if" class="style-scope classification-tree"></template>
            <template is="dom-if" class="style-scope classification-tree"></template>
          <template is="dom-repeat" class="style-scope classification-tree"></template>
        </div>
      </div>
    </div>
  </classification-tree>
        <template is="dom-repeat" class="style-scope classification-viewer"></template>
        <div class="more style-scope classification-viewer" style="" hidden="">View 5 more classifications</div>
        <div class="more style-scope classification-viewer">Hide more classifications</div>
      </div>
      <template is="dom-if" class="style-scope classification-viewer"></template>
    </div>
  </classification-viewer>
              </section>
              
              <template is="dom-if" class="style-scope patent-result"></template>
              
              <template is="dom-if" class="style-scope patent-result"></template>
              
              
                <section class="style-scope patent-result">
                  <h3 class="layout horizontal style-scope patent-result">
                    <div class="flex style-scope patent-result">Landscapes</div>
                  </h3>
                  <div class="responsive-grid style-scope patent-result">
                    
                      <landscape-link class="style-scope patent-result"><state-modifier act="{&quot;type&quot;:&quot;QUERY_SET_TERM&quot;,&quot;term&quot;:&quot;$term&quot;,&quot;origin&quot;:&quot;LANDSCAPE_LINK&quot;}" class="style-scope landscape-link" data-term="Health &amp; Medical Sciences"><a id="link" href="#" class="style-scope state-modifier">
  <requery-tile class="area style-scope landscape-link">
<span class="label style-scope requery-tile">
    Health &amp; Medical Sciences
  </span>
<span class="search-icon style-scope requery-tile"></span>
</requery-tile>
</a></state-modifier>
</landscape-link>
                    
                      <landscape-link class="style-scope patent-result"><state-modifier act="{&quot;type&quot;:&quot;QUERY_SET_TERM&quot;,&quot;term&quot;:&quot;$term&quot;,&quot;origin&quot;:&quot;LANDSCAPE_LINK&quot;}" class="style-scope landscape-link" data-term="Life Sciences &amp; Earth Sciences"><a id="link" href="#" class="style-scope state-modifier">
  <requery-tile class="area style-scope landscape-link">
<span class="label style-scope requery-tile">
    Life Sciences &amp; Earth Sciences
  </span>
<span class="search-icon style-scope requery-tile"></span>
</requery-tile>
</a></state-modifier>
</landscape-link>
                    <template is="dom-repeat" class="style-scope patent-result"></template>
                  </div>
                  <s-more-inline class="style-scope patent-result"><hr aria-hidden="true" class="style-scope s-more-inline">

<div class="button style-scope s-more-inline">
  <span slot="expand_text" class="style-scope patent-result">Show more</span>
  <span class="expand more style-scope s-more-inline"></span>
</div>
<div class="button style-scope s-more-inline" hidden="">
  <span slot="collapse_text" class="style-scope patent-result">Show less</span>
  <span class="expand less style-scope s-more-inline"></span>
</div>
</s-more-inline>
                </section>
              <template is="dom-if" class="style-scope patent-result"></template>
            </div>
            <div class="flex-2 style-scope patent-result">
              <section class="knowledge-card style-scope patent-result">
                <header class="style-scope patent-result">
                  <h2 id="pubnum" class="style-scope patent-result">HRP20050117A2</h2>
                  <p class="tagline style-scope patent-result">Croatia</p>
                  <overlay-tooltip positioning-target="#pubnum" direction="bottom" class="style-scope patent-result" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      
                    <div style="max-width: 300px;" class="style-scope patent-result">Publication of application without search report</div>
                  
    </div>
  </overlay-tooltip>

                  <div class="knowledge-card-action-bar style-scope patent-result">
                    <a target="_blank" class="style-scope patent-result" href="https://patentimages.storage.googleapis.com/c4/70/ed/9c888f93219a4a/HRP20050117A2.pdf"><iron-icon icon="description" class="style-scope patent-result x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M14 2H6c-1.1 0-1.99.9-1.99 2L4 20c0 1.1.89 2 1.99 2H18c1.1 0 2-.9 2-2V8l-6-6zm2 16H8v-2h8v2zm0-4H8v-2h8v2zm-3-5V3.5L18.5 9H13z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon> Download PDF</a>
                    <state-modifier inherit="" act="{&quot;type&quot;: &quot;QUERY_FIND_PRIOR_ART&quot;, &quot;keywords&quot;: $keywords, &quot;before&quot;: &quot;$before&quot;}" class="style-scope patent-result" data-keywords="[&quot;dmgpcr&quot;,&quot;seq&quot;,&quot;sequence&quot;,&quot;binding&quot;,&quot;binding partner&quot;]" data-before="20020806"><a id="link" href="#" class="style-scope state-modifier" inherit="">
                      <iron-icon icon="find-in-page" class="style-scope patent-result x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M20 19.59V8l-6-6H6c-1.1 0-1.99.9-1.99 2L4 20c0 1.1.89 2 1.99 2H18c.45 0 .85-.15 1.19-.4l-4.43-4.43c-.8.52-1.74.83-2.76.83-2.76 0-5-2.24-5-5s2.24-5 5-5 5 2.24 5 5c0 1.02-.31 1.96-.83 2.75L20 19.59zM9 13c0 1.66 1.34 3 3 3s3-1.34 3-3-1.34-3-3-3-3 1.34-3 3z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon> Find Prior Art
                    </a></state-modifier>
                    <state-modifier inherit="" act="{&quot;type&quot;: &quot;QUERY_SIMILAR_DOCUMENTS&quot;, &quot;id&quot;: &quot;$id&quot;}" class="style-scope patent-result" data-id="patent/HRP20050117A2"><a id="link" href="#" class="style-scope state-modifier" inherit="">
                      <iron-icon icon="editor:functions" class="style-scope patent-result x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M18 4H6v2l6.5 6L6 18v2h12v-3h-7l5-5-5-5h7z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon> Similar
                    </a></state-modifier>
                  </div>
                </header>

                <dl class="important-people style-scope patent-result">
                  
                    <dt class="style-scope patent-result">Other languages</dt>
                    <dd class="style-scope patent-result"><state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/HRP20050117A2/hr"><a id="link" href="#" class="style-scope state-modifier">Croatian</a></state-modifier></dd><template is="dom-repeat" strip-whitespace="" class="style-scope patent-result"></template>
                  <template is="dom-if" class="style-scope patent-result"></template>
                  
                  <dt class="style-scope patent-result">Inventor</dt>
                  <dd class="style-scope patent-result"><state-modifier act="{&quot;type&quot;: &quot;QUERY_SEARCH_INVENTOR&quot;, &quot;inventor&quot;: &quot;$inventor&quot;}" class="style-scope patent-result" data-inventor="David"><a id="link" href="#" class="style-scope state-modifier">David</a></state-modifier></dd><dd class="style-scope patent-result"><state-modifier act="{&quot;type&quot;: &quot;QUERY_SEARCH_INVENTOR&quot;, &quot;inventor&quot;: &quot;$inventor&quot;}" class="style-scope patent-result" data-inventor="Valdin"><a id="link" href="#" class="style-scope state-modifier">Valdin</a></state-modifier></dd><dd class="style-scope patent-result"><state-modifier act="{&quot;type&quot;: &quot;QUERY_SEARCH_INVENTOR&quot;, &quot;inventor&quot;: &quot;$inventor&quot;}" class="style-scope patent-result" data-inventor="Teresa"><a id="link" href="#" class="style-scope state-modifier">Teresa</a></state-modifier></dd><dd class="style-scope patent-result"><state-modifier act="{&quot;type&quot;: &quot;QUERY_SEARCH_INVENTOR&quot;, &quot;inventor&quot;: &quot;$inventor&quot;}" class="style-scope patent-result" data-inventor="Martha"><a id="link" href="#" class="style-scope state-modifier">Martha</a></state-modifier></dd><template is="dom-repeat" strip-whitespace="" class="style-scope patent-result"></template>
                  <template is="dom-if" class="style-scope patent-result"></template>

                  <template is="dom-if" class="style-scope patent-result"></template>
                </dl>

                <hr class="style-scope patent-result">

                <application-timeline class="style-scope patent-result">
    
    
    <div class="wrap style-scope application-timeline">
      <div class="header style-scope application-timeline">Worldwide applications</div>
      <div class="event style-scope application-timeline">
        
          <span class="block style-scope application-timeline">
            <span class="year style-scope application-timeline">2002</span>
            <span class="dot style-scope application-timeline"></span>
            
              <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope application-timeline" data-result="patent/US20030162223A1/en"><a id="link" href="#" class="style-scope state-modifier">
                <span id="cc" class="not_active style-scope application-timeline">US</span>
              </a></state-modifier>
              <overlay-tooltip positioning-target="#cc" direction="bottom" class="style-scope application-timeline" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      
                <div class="style-scope application-timeline">Application number: US10/283,423</div>
                <div class="style-scope application-timeline">Filing date: 2002-10-30</div>
                <div class="style-scope application-timeline">Legal status: Abandoned</div>
              
    </div>
  </overlay-tooltip>
            <template is="dom-repeat" class="style-scope application-timeline"></template>
          </span>
        
          <span class="block style-scope application-timeline">
            <span class="year style-scope application-timeline">2003</span>
            <span class="dot style-scope application-timeline"></span>
            
              <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope application-timeline" data-result="patent/AU2003258069A1/en"><a id="link" href="#" class="style-scope state-modifier">
                <span id="cc" class="not_active style-scope application-timeline">AU</span>
              </a></state-modifier>
              <overlay-tooltip positioning-target="#cc" direction="bottom" class="style-scope application-timeline" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      
                <div class="style-scope application-timeline">Application number: AU2003258069A</div>
                <div class="style-scope application-timeline">Filing date: 2003-08-06</div>
                <div class="style-scope application-timeline">Legal status: Abandoned</div>
              
    </div>
  </overlay-tooltip>
            
              <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope application-timeline" data-result="patent/KR20060003326A/en"><a id="link" href="#" class="style-scope state-modifier">
                <span id="cc" class="not_active style-scope application-timeline">KR</span>
              </a></state-modifier>
              <overlay-tooltip positioning-target="#cc" direction="bottom" class="style-scope application-timeline" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      
                <div class="style-scope application-timeline">Application number: KR1020057002333A</div>
                <div class="style-scope application-timeline">Filing date: 2003-08-06</div>
                <div class="style-scope application-timeline">Legal status: Application Discontinuation</div>
              
    </div>
  </overlay-tooltip>
            
              <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope application-timeline" data-result="patent/EP1546726A4/en"><a id="link" href="#" class="style-scope state-modifier">
                <span id="cc" class="not_active style-scope application-timeline">EP</span>
              </a></state-modifier>
              <overlay-tooltip positioning-target="#cc" direction="bottom" class="style-scope application-timeline" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      
                <div class="style-scope application-timeline">Application number: EP03767210A</div>
                <div class="style-scope application-timeline">Filing date: 2003-08-06</div>
                <div class="style-scope application-timeline">Legal status: Withdrawn</div>
              
    </div>
  </overlay-tooltip>
            
              <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope application-timeline" data-result="patent/JP2005537465A/en"><a id="link" href="#" class="style-scope state-modifier">
                <span id="cc" class="active style-scope application-timeline">JP</span>
              </a></state-modifier>
              <overlay-tooltip positioning-target="#cc" direction="bottom" class="style-scope application-timeline" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      
                <div class="style-scope application-timeline">Application number: JP2004526460A</div>
                <div class="style-scope application-timeline">Filing date: 2003-08-06</div>
                <div class="style-scope application-timeline">Legal status: Pending</div>
              
    </div>
  </overlay-tooltip>
            
              <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope application-timeline" data-result="patent/US20060198841A1/en"><a id="link" href="#" class="style-scope state-modifier">
                <span id="cc" class="not_active style-scope application-timeline">US</span>
              </a></state-modifier>
              <overlay-tooltip positioning-target="#cc" direction="bottom" class="style-scope application-timeline" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      
                <div class="style-scope application-timeline">Application number: US10/523,893</div>
                <div class="style-scope application-timeline">Filing date: 2003-08-06</div>
                <div class="style-scope application-timeline">Legal status: Abandoned</div>
              
    </div>
  </overlay-tooltip>
            
              <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope application-timeline" data-result="patent/CA2494311A1/en"><a id="link" href="#" class="style-scope state-modifier">
                <span id="cc" class="not_active style-scope application-timeline">CA</span>
              </a></state-modifier>
              <overlay-tooltip positioning-target="#cc" direction="bottom" class="style-scope application-timeline" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      
                <div class="style-scope application-timeline">Application number: CA002494311A</div>
                <div class="style-scope application-timeline">Filing date: 2003-08-06</div>
                <div class="style-scope application-timeline">Legal status: Abandoned</div>
              
    </div>
  </overlay-tooltip>
            
              <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope application-timeline" data-result="patent/RU2326385C2/en"><a id="link" href="#" class="style-scope state-modifier">
                <span id="cc" class="not_active style-scope application-timeline">RU</span>
              </a></state-modifier>
              <overlay-tooltip positioning-target="#cc" direction="bottom" class="style-scope application-timeline" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      
                <div class="style-scope application-timeline">Application number: RU2005102837/13A</div>
                <div class="style-scope application-timeline">Filing date: 2003-08-06</div>
                <div class="style-scope application-timeline">Legal status: IP Right Cessation</div>
              
    </div>
  </overlay-tooltip>
            
              <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope application-timeline" data-result="patent/WO2004013306A2/en"><a id="link" href="#" class="style-scope state-modifier">
                <span id="cc" class="active style-scope application-timeline">WO</span>
              </a></state-modifier>
              <overlay-tooltip positioning-target="#cc" direction="bottom" class="style-scope application-timeline" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      
                <div class="style-scope application-timeline">Application number: PCT/US2003/024488</div>
                <div class="style-scope application-timeline">Filing date: 2003-08-06</div>
                <div class="style-scope application-timeline">Legal status: Application Filing</div>
              
    </div>
  </overlay-tooltip>
            <template is="dom-repeat" class="style-scope application-timeline"></template>
          </span>
        
          <span class="block style-scope application-timeline">
            <span class="year style-scope application-timeline">2005</span>
            <span class="dot style-scope application-timeline"></span>
            
              <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope application-timeline" data-result="patent/IL166692A0/en"><a id="link" href="#" class="style-scope state-modifier">
                <span id="cc" class="unknown style-scope application-timeline">IL</span>
              </a></state-modifier>
              <overlay-tooltip positioning-target="#cc" direction="bottom" class="style-scope application-timeline" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      
                <div class="style-scope application-timeline">Application number: IL16669205A</div>
                <div class="style-scope application-timeline">Filing date: 2005-02-03</div>
                <div class="style-scope application-timeline">Legal status: </div>
              
    </div>
  </overlay-tooltip>
            
              <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope application-timeline" data-result="patent/HRP20050117A2/en"><a id="link" href="#" class="style-scope state-modifier">
                <span id="cc" class="not_active style-scope application-timeline" this="">HR</span>
              </a></state-modifier>
              <overlay-tooltip positioning-target="#cc" direction="bottom" class="style-scope application-timeline" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      
                <div class="style-scope application-timeline">Application number: HR20050117A</div>
                <div class="style-scope application-timeline">Filing date: 2005-02-04</div>
                <div class="style-scope application-timeline">Legal status: Application Discontinuation</div>
              
    </div>
  </overlay-tooltip>
            
              <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope application-timeline" data-result="patent/NO20050641L/en"><a id="link" href="#" class="style-scope state-modifier">
                <span id="cc" class="not_active style-scope application-timeline">NO</span>
              </a></state-modifier>
              <overlay-tooltip positioning-target="#cc" direction="bottom" class="style-scope application-timeline" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      
                <div class="style-scope application-timeline">Application number: NO20050641A</div>
                <div class="style-scope application-timeline">Filing date: 2005-02-04</div>
                <div class="style-scope application-timeline">Legal status: Application Discontinuation</div>
              
    </div>
  </overlay-tooltip>
            <template is="dom-repeat" class="style-scope application-timeline"></template>
          </span>
        <template is="dom-repeat" class="style-scope application-timeline"></template>
      </div>
      <hr class="style-scope application-timeline">
      <div class="header style-scope application-timeline">Application HR20050117A events <iron-icon id="legal" icon="icons:help-outline" class="style-scope application-timeline x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M11 18h2v-2h-2v2zm1-16C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8zm0-14c-2.21 0-4 1.79-4 4h2c0-1.1.9-2 2-2s2 .9 2 2c0 2-3 1.75-3 5h2c0-2.25 3-2.5 3-5 0-2.21-1.79-4-4-4z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
        <overlay-tooltip positioning-target="#legal" direction="right" class="style-scope application-timeline" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      
          <div style="max-width: 300px;" class="style-scope application-timeline">
            A timeline of key events for this patent application, including priority claims, publications, legal status, reassignments, and litigation.<br class="style-scope application-timeline"><br class="style-scope application-timeline">
            Google has not performed a legal analysis and makes no representation as to the accuracy or completeness of the events listed.
          </div>
        
    </div>
  </overlay-tooltip>
      </div>
      <div class="event layout horizontal style-scope application-timeline"><div date="" class="external-priority style-scope application-timeline">2002-08-06</div><div class="line style-scope application-timeline"></div><div class="flex title style-scope application-timeline"><template is="dom-if" class="style-scope application-timeline"></template><state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope application-timeline" data-result="patent/US7364866B2/en"><a id="link" href="#" class="style-scope state-modifier"><span class="title-text style-scope application-timeline">Priority claimed from US10/213,821</span></a></state-modifier><template is="dom-if" class="style-scope application-timeline"></template><template is="dom-if" class="style-scope application-timeline"></template><template is="dom-if" class="style-scope application-timeline"></template><template is="dom-if" class="style-scope application-timeline"></template></div></div><div class="event layout horizontal style-scope application-timeline" critical=""><div date="" class="filed style-scope application-timeline">2005-02-04</div><div class="line style-scope application-timeline"></div><div class="flex title style-scope application-timeline"><template is="dom-if" class="style-scope application-timeline"></template><template is="dom-if" class="style-scope application-timeline"></template><state-modifier act="{&quot;type&quot;: &quot;QUERY_SEARCH_ASSIGNEE&quot;, &quot;assignee&quot;: &quot;$assignee&quot;}" class="style-scope application-timeline" data-assignee="Pharmacia &amp; Upjohn Company Llc"><a id="link" href="#" class="style-scope state-modifier"><span class="title-text style-scope application-timeline">Application filed by Pharmacia &amp; Upjohn Company Llc</span></a></state-modifier><template is="dom-if" class="style-scope application-timeline"></template><template is="dom-if" class="style-scope application-timeline"></template><template is="dom-if" class="style-scope application-timeline"></template></div></div><div class="event layout horizontal style-scope application-timeline" critical=""><div date="" class="publication style-scope application-timeline">2005-10-31</div><div class="line style-scope application-timeline"></div><div class="flex title style-scope application-timeline"><template is="dom-if" class="style-scope application-timeline"></template><state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope application-timeline" data-result="patent/HRP20050117A2/en"><a id="link" href="#" class="style-scope state-modifier"><span class="title-text style-scope application-timeline">Publication of HRP20050117A2</span></a></state-modifier><template is="dom-if" class="style-scope application-timeline"></template><template is="dom-if" class="style-scope application-timeline"></template><template is="dom-if" class="style-scope application-timeline"></template><template is="dom-if" class="style-scope application-timeline"></template></div></div><template is="dom-repeat" strip-whitespace="" class="style-scope application-timeline"></template>
      <template is="dom-if" class="style-scope application-timeline"></template>
    </div>
  </application-timeline>
                <hr class="style-scope patent-result">

                <dl class="links style-scope patent-result">
                  <dt class="style-scope patent-result">Info</dt>

                  
                  <dd class="style-scope patent-result"><a href="#patentCitations" class="style-scope patent-result">Patent citations (20)</a></dd>
                  <template is="dom-if" restamp="" class="style-scope patent-result"></template>

                  <template is="dom-if" restamp="" class="style-scope patent-result"></template>

                  
                  <dd class="style-scope patent-result"><a href="#citedBy" class="style-scope patent-result">Cited by (5)</a></dd>
                  <template is="dom-if" restamp="" class="style-scope patent-result"></template>

                  
                  <dd class="style-scope patent-result"><a href="#legalEvents" class="style-scope patent-result">Legal events</a></dd>
                  <template is="dom-if" restamp="" class="style-scope patent-result"></template>

                  
                  <dd class="style-scope patent-result"><a href="#similarDocuments" class="style-scope patent-result">Similar documents</a></dd>
                  <template is="dom-if" restamp="" class="style-scope patent-result"></template>

                  
                  <dd class="style-scope patent-result"><a href="#relatedApplications" class="style-scope patent-result">Priority and Related Applications</a></dd>
                  <template is="dom-if" restamp="" class="style-scope patent-result"></template>

                  <dt class="style-scope patent-result">External links</dt>

                  <dd class="style-scope patent-result"><a target="_blank" class="style-scope patent-result" href="https://worldwide.espacenet.com/publicationDetails/biblio?CC=HR&amp;NR=P20050117A2&amp;KC=A2&amp;FT=D">Espacenet</a></dd><dd class="style-scope patent-result"><a target="_blank" class="style-scope patent-result" href="https://globaldossier.uspto.gov/#/result/publication/HR/P20050117/1">Global Dossier</a></dd><dd class="style-scope patent-result"><a target="_blank" class="style-scope patent-result" href="https://patents.stackexchange.com/questions/tagged/HRP20050117A2">Discuss</a></dd><template is="dom-repeat" strip-whitespace="" class="style-scope patent-result"></template>
                </dl>
              </section>
            </div>
          </div>

          <div id="text" class="horizontal layout style-scope patent-result">
            <div class="flex flex-width style-scope patent-result" marginright="">
              <section id="description" class="style-scope patent-result">
                <h3 class="layout horizontal style-scope patent-result">
                  <div class="flex style-scope patent-result">Description</div>
                  <div class="righthead style-scope patent-result">translated from Croatian</div>
                </h3>
                <patent-text name="description" id="descriptionText" class="style-scope patent-result" style="padding-top: 0px;">
    
    
    <template is="dom-if" restamp="" class="style-scope patent-text"></template>
    <overlay-tooltip id="tooltip" class="style-scope patent-text" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      
    </div>
  </overlay-tooltip>
    <div class="layout horizontal style-scope patent-text">
      <section id="text" class="flex style-scope patent-text"><description mxw-id="PDES336231614" lang="EN" load-source="google" class="style-scope patent-text">
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Podruje izuma </span>Field of invention</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Izum je iz podruja farmakologije. </span>The invention is from the field of pharmacology.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Ovaj izum djelomino se odnosi na molekule nukleinske kiseline koje kodiraju nove Drosophila melanogaster G protein vezane receptore (DmGPCR- ovi), nove polipeptide, analize za screening spojeva koji se veu za DmGPCR i/ili moduliraju aktivnost DmGPCR, metode za vezanje DmGPCR, reagense kao to su antitijela na DmGPCR, vodee sekvencije i sonde za detektiranje nukleotidnih sekvencija koje kodiraju DmGPCR, pribore koji ukljuuju antitijela, vodee sekvencije i sonde ovog izuma, smjese koje sadre DmGPCR-ovi-ove, DmGPCR vezujue partnere i DmGPCR modulatore, te metode za kontroliranje populacije insekata koritenjem DmGPCR vezujueg partnera ili modulatora. </span>This invention relates in part to nucleic acid molecules encoding novel Drosophila melanogaster G protein-coupled receptors (DmGPCRs), novel polypeptides, assays for screening compounds that bind to DmGPCR and/or modulate DmGPCR activity, methods for DmGPCR binding, reagents such as which are antibodies to DmGPCR, leader sequences and probes for detecting nucleotide sequences encoding DmGPCR, kits including antibodies, leader sequences and probes of the present invention, mixtures containing DmGPCRs, DmGPCR binding partners and DmGPCR modulators, and methods for controlling insect populations using DmGPCR binding partners or modulators.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Stanje tehnike </span>State of the art</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Ljudi i drugi ivi oblici sastoje se od ivih stanica. Meu mehanizmima putem kojih stanice organizma komuniciraju meusobno i dobivaju informacije te se stimuliraju iz njihove okoline su reeptorske molekule stanine membrane s ekspresijom na povrini stanice. Mnogo takvih receptora je identificirano, karakterizirano i ponekad klasificirano u glavne receptorske superfamilije temeljem strukturnih motive i svojstava prijenosa signala. Takve familije obuhvaaju (ali nisu ograniene na) ligand-usmjerene ionske kanalne receptore, naponski ovisne ionske kanalne reeptore, receptor tirozin kinaze, receptor protein tirozin fosfataze i G protein-vezane receptore. Receptori su prva sutinski bitna veza za prevoenje izvanstaninog signala u stanini fizioloki odgovor. </span>Humans and other living forms are made up of living cells. Among the mechanisms through which the cells of the organism communicate with each other and receive information and are stimulated from their environment are receptor molecules of the cell membrane with expression on the cell surface. Many such receptors have been identified, characterized, and sometimes classified into major receptor superfamilies based on structural motifs and signal transduction properties. Such families include (but are not limited to) ligand-gated ion channel receptors, voltage-gated ion channel receptors, receptor tyrosine kinases, receptor protein tyrosine phosphatases, and G protein-coupled receptors. Receptors are the first essential link for the translation of an extracellular signal into a cellular physiological response.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">G protein-vezani receptori (npr. GPCR-ovi) ine veliku superfamiliju staninih povrinskih receptora koji su karakterizirani amino-terminalnom izvanstaninom domenom, karboksi-terminalnom unutarstaninom domenom i uzvojitom strukturom koja prolazi kroz staninu membranu sedam puta. Dakle, takvi receptori se ponekad nazivaju sedam transmembranskih (7TM) receptora. Ovih sedam transmembranskih domena definira tri izvanstanine petlje i tri unutarstanine petlje, uz amino- i karboksi-terminalne domene. Izvanstanini dijelovi receptora imaju ulogu u prepoznavanju i vezanju jednog ili vie izvanstaninih vezujuih partnera (npr. liganada), pri emu unutarstanini dijelovi imaju ulogu u prepoznavanju i komuniciranju s uzvodnim (downstream) efektorskim molekulama. </span>G protein-coupled receptors (eg, GPCRs) form a large superfamily of cell surface receptors that are characterized by an amino-terminal extracellular domain, a carboxy-terminal intracellular domain, and a coiled-coil structure that crosses the cell membrane seven times. Thus, such receptors are sometimes called seven transmembrane (7TM) receptors. These seven transmembrane domains define three extracellular loops and three intracellular loops, in addition to amino- and carboxy-terminal domains. The extracellular parts of the receptor have a role in recognizing and binding one or more extracellular binding partners (eg ligands), while the intracellular parts have a role in recognizing and communicating with upstream (downstream) effector molecules.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">GPCR-ovi veu cijeli niz liganada ukljuujui kalcijeve ione, hormone, kemokine, neuropeptide, neurotransmitere, nukleotide, lipide, odorante i ak fotone. Ne iznenauje, GPCR-ovi su znaajni u normalnoj (i ponekad aberantnoj) funkciji mnogih staninih tipova. Vidi openito Strosberg, Eur. J. Biochem., 1991,196,1-10; Bohm et al, Biochem J., 1997, 322, 1-18. Kada se specifini ligand vee za svoj odgovarajui receptor, ligand tipino stimulira receptor da aktivira specifini heterotrimerni guanin nukleotid-vezujui regulatorski protein (G protein) koji je vezan za unutarstanini dio ili podruje receptora. G protein, opet, prenosi signal do efektorske molekule unutar stanice bilo stimuliranjem ili inhibiranjem aktivnosti efektorske molekule. Ove efektorske molekule obuhvaaju adenilat ciklazu, fosfolipaze i ionske kanale. Adenilat ciklaza i fosfolipaze su enzimi koji su ukljueni u proizvodnji sekundarne glasnike molekule cAMP, inozitol trifosfata i diaciglicerola. To se postie nizom aktova kojima izvanstanini stimulirajui ligand daje unutarstanine promjene putem G protein-vezanog receptora. Svaki takav receptor ima svoju karakteristinu primarnu strukturu, oblik ekspresije, ligand vezujui profil i unutarstanini efektorski sustav. </span>GPCRs bind a whole range of ligands including calcium ions, hormones, chemokines, neuropeptides, neurotransmitters, nucleotides, lipids, odorants and even photons. Not surprisingly, GPCRs are important in the normal (and sometimes aberrant) function of many cell types. See generally Strosberg, Eur. J. Biochem., 1991,196,1-10; Bohm et al, Biochem J., 1997, 322, 1-18. When a specific ligand binds to its respective receptor, the ligand typically stimulates the receptor to activate a specific heterotrimeric guanine nucleotide-binding regulatory protein (G protein) that is bound to the intracellular portion or region of the receptor. The G protein, in turn, transmits the signal to the effector molecule within the cell by either stimulating or inhibiting the activity of the effector molecule. These effector molecules include adenylate cyclase, phospholipases and ion channels. Adenylate cyclase and phospholipase are enzymes involved in the production of the secondary messenger molecule cAMP, inositol triphosphate and diacylglycerol. This is achieved through a series of actions by which an extracellular stimulating ligand produces intracellular changes via a G protein-coupled receptor. Each such receptor has its own characteristic primary structure, form of expression, ligand binding profile and intracellular effector system.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Zbog vitalne uloge G protein-vezanih receptora u komunikaciji izmeu stanica i njihove okoline, ti receptori su privlane mete za reguliranje, primjerice aktiviranjem ili antagoniziranjem takvih receptora. Za receptore koji imaju poznati ligand, identificiranje agonista ili antagonista moe se vriti specifino da se pojaa ili inhibira djelovanje liganda. Primjerice, neki G protein-vezani receptori imaju ulogu u patogenezi bolesti (npr. neki kemokinski receptori koji djeluju kao HIV koreceptori mogu imati odreenu ulogu u patogenezi AIDS-a) i privlaen su mete za terapijsku intervenciju ak i ako nedostaje znanje o prirodnom ligandu receptora. Drugi receptori su privlane mete za terapijsku intervenciju zbog naina njihove ekspresije u tkivima ili staninim tipovima koji su sami privlane mete za terapijsku intervenciju. Primjeri ove posljednje kategorije receptora obuhvaaju receptore s ekspresijom u imunolokim stanicama, koji mogu biti cilj bilo zbog toga da se inhibiraju autoimunoloki odgovori ili da se pojaaju imunoloki odgovori za borbu s patogenima ili karcinomom, te receptorima s ekspresijom u mozgu ili drugim neurolokim organima ili tkivima, to su vjerojatne mete u tretiranju shizofrenije, depresije, bipolarne bolesti ili drugih neurolokih poremeaja. Ova posljednja kategorija receptora je takoer korisna kao obiljeiva u identificiranju i/ili proiavanju (npr. pomou fluorescencijom aktiviranog staninog sortiranja) staninih podtipova koji vre ekspresiju receptora. </span>Due to the vital role of G protein-coupled receptors in communication between cells and their environment, these receptors are attractive targets for regulation, for example by activating or antagonizing such receptors. For receptors that have a known ligand, identifying agonists or antagonists can be done specifically to enhance or inhibit the action of the ligand. For example, some G protein-coupled receptors play a role in the pathogenesis of disease (eg, some chemokine receptors that act as HIV coreceptors may have a role in the pathogenesis of AIDS) and are attractive targets for therapeutic intervention even if knowledge of the receptor's natural ligand is lacking. . Other receptors are attractive targets for therapeutic intervention due to the way they are expressed in tissues or cell types that are themselves attractive targets for therapeutic intervention. Examples of this latter category of receptors include receptors expressed in immune cells, which can be targeted either to inhibit autoimmune responses or to enhance immune responses to fight pathogens or cancer, and receptors expressed in the brain or other neurological organs or tissues. , which are likely targets in the treatment of schizophrenia, depression, bipolar disease or other neurological disorders. This latter category of receptors is also useful as a marker in identifying and/or purifying (eg, by fluorescence-activated cell sorting) cell subtypes that express the receptor.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Insekti su poznati kao glavni tetnici u zemljoradnji i u kunom domaem okruenju. Insekti su takoer paraziti na domaim ivotinjama i ljudima, pa ih u takvim sluajevima nazivamo ektoparazitima, poveanjem morbiditeta i mortaliteta. Insekti takoer slue kao vektori za prijenos virusnih i parazitnih bolesti na biljke, ivotinje i ljude. Dakle, postoji stalna i prinudna potreba da se otkriju nove metode za kontroliranje populacije insekata te za odbijanje i/ili unitavanje patogenih vrsta. Jedan nain da se kontrolira populacija insekata je da se insekti usmrte ili paraliziraju upotrebom kemijskih tvari, koje se nazivaju insekticidima, koje su selektivno otrovne za insekte i potencijalno za druge beskraljenjake. Trenutano, insekticidi su vrlo vrijedni u kontroli insekata koji su tetni za produkte zemljoradnje, ukljuujui usjeve. Insekticidi se takoer rabe u humanom kunom okruenju, za kontrolu kunih i vrtnih tetoinja te insekata koji su tetni ili razdraljivi za ljude, kao to su insekti koji bodu i ujedaju, muhe i ohari. Insekticidi imaju enormnu vrijednost za tretiranje i prevenciju bolesnih stanja koja su izazvana ektoparazitima, ukljuujui muice, ui, krpelji i grinje, u kunih ljubimaca i domaih ivotinja. Meutim, kemikalije koje se trenutano rabe kao insekticidi nisu optimalne. Neke su pokazale svoju toksinost za sisavce, dok se u nekih ciljanih vrsta pojavila otpornost prema njima. Prema tome, postoji potreba za novim selektivnim insekticidima koji imaju nov mehanizam djelovanja. </span>Insects are known as the main pests in agriculture and in the domestic domestic environment. Insects are also parasites on domestic animals and humans, so in such cases we call them ectoparasites, increasing morbidity and mortality. Insects also serve as vectors for the transmission of viral and parasitic diseases to plants, animals and humans. Thus, there is a constant and compelling need to discover new methods for controlling insect populations and for repelling and/or destroying pathogenic species. One way to control insect populations is to kill or paralyze the insects using chemicals, called insecticides, that are selectively toxic to insects and potentially other invertebrates. Currently, insecticides are very valuable in controlling insects that are harmful to agricultural products, including crops. Insecticides are also used in the humane home environment to control household and garden pests and insects that are harmful or irritating to humans, such as stinging and biting insects, flies and cockroaches. Insecticides are of enormous value for the treatment and prevention of disease conditions caused by ectoparasites, including flies, lice, ticks and mites, in pets and domestic animals. However, the chemicals currently used as insecticides are not optimal. Some have shown their toxicity to mammals, while some target species have developed resistance to them. Therefore, there is a need for new selective insecticides that have a new mechanism of action.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Poznati su primjeri insektnih GPCR-ova koji sadre neuropeptidne ligande (vidi npr. Li, et al., EMBO Journal, 1991, 10, 3221-3229; Li, et al., J. Biol. Chem., 1992, 267, 9-12; Monnier, et al., J. Biol. Chem., 1992, 267, 1298-1302; Vanden Broeck, et al., Int. Rev. Cytology, 1996, 164, 189-268; Guerrero, Peptides, 1997, 18, 1-5; Hauser, et al., J. Biol. Chem., 1997, 272, 1002-1010; Birgul et al., EMBO J., 1999, 18, 5892-5900; Torfs et al., J. Neurochem., 2000, 74, 2182-2189; i Hauser et al., Biochem. Biophys. Res. Comm., 1998, 249, 822-828; Larsen, et al., Biochem. Biophys. Res. Comm., 2001, 286, 895-901; Lenz, et al., Biochem. Biophys. Res. Comm., 2001, 286, 1117-1122; Kubiak et al., Biochem. Biophys. Res. Comm., 2002, 291, 313-320; Staubli et al., Proc. Natl. Acad. Sci. USA, 2002, 9, 3446-3451; Garczynski et al., Peptides, 2002, 23, 773-780; Holmes et al., Insect Molecular Biology, 2000, (5), 457-465; Cazzamali et al., Proc. Natl. Acad. Sci. USA, 2002, 99, 12073-12078; Cazzamali et al., Biochem. Biophys. Res. Comm., 2002, 298, 31-36; Radford et.al., J. Biol. Chem. 2002, 277, 38810-38817; Park et al., Proc. Natl. Acad. Sci. USA, 2002, 99, 11423-11428; Kreienkamp et al., J. Biol. Chem, 10.1074/jbc.M206931200 (publicirano online 6. kolovoza 2002.) i Mertens, et al., Biochem. Biophys. Res. Comm., 2002, 297, 1140-1148. Novije srodne patentne prijave: Ebens, Allen James, Jr.; Torpey, Justin; Keegan, Kevin Patrick, Nucleic acids and and polypeptides of Drosophila melanogaster G protein-coupled receptor and their use as pesticidal and pharmaceutical targets. PCT meunarodna prijava (2001), 43 str. CODEN: PIXXD2 WO 0170981 A2 20010927 CAN 135:268323 AN 2001:713564 CAPLUS. Kravchik, Anibal. Drosophila G protein-coupled receptors, genomic DNA and cDNA molecule encoding GPCR proteins and their uses as insecticidal targets. PCT meunarodna prijava (2001), 392 str. CODEN: PIXXD2 WO 0170980 A2 20010927 CAN 135:269068 AN 2001:713563 CAPLUS. </span>Examples of insect GPCRs containing neuropeptide ligands are known (see, e.g., Li, et al., EMBO Journal, 1991, 10, 3221-3229; Li, et al., J. Biol. Chem., 1992, 267, 9 -12; Monnier, et al., J. Biol. Chem., 1992, 267, 1298-1302; Vanden Broeck, et al., Int. Rev. Cytology, 1996, 164, 189-268; Guerrero, Peptides, 1997 , 18, 1-5; Hauser, et al., J. Biol. Chem., 1997, 272, 1002-1010; Birgul et al., EMBO J., 1999, 18, 5892-5900; Torfs et al., J. Neurochem., 2000, 74, 2182-2189; and Hauser et al., Biochem. Biophys. Res. Comm., 1998, 249, 822-828; Larsen, et al., Biochem. Biophys. Res. Comm. , 2001, 286, 895-901; Lenz, et al., Biochem. Biophys. Res. Comm., 2001, 286, 1117-1122; Kubiak et al., Biochem. Biophys. Res. Comm., 2002, 291, 313-320; Staubli et al., Proc. Natl. Acad. Sci. USA, 2002, 9, 3446-3451; Garczynski et al., Peptides, 2002, 23, 773-780; Holmes et al., Insect Molecular Biology , 2000, (5), 457-465; Cazzamali et al., Proc. Natl. Acad. Sci. USA, 2002, 99, 12073-12078; Cazzamali et al. , Biochem. Biophys. Crisp. Comm., 2002, 298, 31-36; Radford et al., J. Biol. Chem. 2002, 277, 38810-38817; Park et al., Proc. Natl. Acad. Sci. USA, 2002, 99, 11423-11428; Kreienkamp et al., J. Biol. Chem, 10.1074/jbc.M206931200 (published online August 6, 2002) and Mertens, et al., Biochem. Biophys. Crisp. Comm., 2002, 297, 1140-1148. More recent related patent applications: Ebens, Allen James, Jr.; Torpey, Justin; Keegan, Kevin Patrick, Nucleic acids and polypeptides of Drosophila melanogaster G protein-coupled receptor and their use as pesticidal and pharmaceutical targets. PCT International Application (2001), 43 p. CODEN: PIXXD2 WO 0170981 A2 20010927 CAN 135:268323 AN 2001:713564 CAPLUS. Kravchik, Anibal. Drosophila G protein-coupled receptors, genomic DNA and cDNA molecules encoding GPCR proteins and their uses as insecticidal targets. PCT International Application (2001), 392 p. CODEN: PIXXD2 WO 0170980 A2 20010927 CAN 135:269068 AN 2001:713563 CAPLUS.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Velika familija openito duine 4-12 aminokiselina tipino se nalaze u ivotinjama beskraljenjacima (npr. insekti) i ta klasa neuropeptida poznata je ako FMRFamid-srodni peptidi (npr. FaRP-ovi). Prototipini FMRFamidni (FMRFa) peptidi tako su nazvani zbog FMRF koncenzusne aminokiselinske sekvencije na njihovu C-terminalu, koja se sastoji openito od (F,Y)(M,V,I,L)R(F,Y)NH2. Kao neuropeptidi, ove su molekule ukljuene u vitalnim biolokim procesima koji zahtijevaju kontroliranu neuromuskularnu aktivnost. Mada se za neke neurotrasmitere i neuromodulatore (ukljuujui neuropeptide) pokazalo da funkcioniraju kao ligandi za receptore, do sada nisu identificirani FaRP neuropeptidi kao ligand za GPCR. </span>A large family generally 4-12 amino acids in length are typically found in invertebrates (eg, insects) and this class of neuropeptides is known as FMRFamide-related peptides (eg, FaRPs). The prototypic FMRFamide (FMRFa) peptides are so named because of the FMRF consensus amino acid sequence at their C-terminus, which generally consists of (F,Y)(M,V,I,L)R(F,Y)NH2. As neuropeptides, these molecules are involved in vital biological processes that require controlled neuromuscular activity. Although some neurotransmitters and neuromodulators (including neuropeptides) have been shown to function as ligands for receptors, no FaRP neuropeptides have been identified as ligands for GPCRs.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Drosophila peptidi sadre konzervirani FXGXR-amidni motiv i strukturno su srodni ivotinjskim tahikinima te su, prema tome, povezani drotahikinini (Siviter et al., J. Biol. Chem., 2000, 275(30), 23273-23280). Drotahikinini imaju jake stimulirajue uinke na kontrakcije utrobe ivotinje (id.). </span>Drosophila peptides contain a conserved FXGXR-amide motif and are structurally related to animal tachykins and are therefore related drotachykinins (Siviter et al., J. Biol. Chem., 2000, 275(30), 23273-23280). Drotachykinins have strong stimulating effects on the contractions of the animal's womb (id.).</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Leukokinini su skupina rairenih insektnih hormona koji stimuliraju pokretljivost utrobe i brzinu sekrecije tekuine iz tubula. U tubulama, njihovo glavno djelovanje je poveanje permeabilnoti klorida vezanjem receptora na bazolateralnoj membrani. Leukokinin djeluje poveanjem unutarstaninog kalcija samo u zvjezdastim stanicama (ODonnell et al., Am. J. Physiol., 1998, 43, R1039-R1049). </span>Leukokinins are a group of widespread insect hormones that stimulate gut motility and the rate of fluid secretion from the tubules. In tubules, their main action is to increase chloride permeability by binding to receptors on the basolateral membrane. Leukokinin acts by increasing intracellular calcium only in stellate cells (O'Donnell et al., Am. J. Physiol., 1998, 43, R1039-R1049).</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Alatostatini su znaajna skupina insektnih neurohormona koji kontroliraju diverzne funkcije ukljuujui sintezu mladenakih hormona za koje je poznato da imaju sredinju ulogu u preobrazbi i reprodukciji u razliitim vrstama insekata. Prvi Drosophila alatostatin, Ser-Arg-Pro-Tyr-Ser-Phe-Gly-Leu-NH2 (tj. drostatin-3) (sekv. br. 165), izdvojen je iz ekstrakta Drosophila glave (Birgul et al., EMBO J., 1999, 18, 5892-5900). U novije vrijeme, kloniran je Drosophila alatostatin preprophormonski gen koji kodira etiri Drosophila alatostatina: Val-Glu-Arg-Tyr-Ala-Phe-Gly-Leu-NH2 (drostatin-1) (sekv. br. 163), Leu-Pro-Val-Tyr-Asn-Phe-Gly-Leu-NH2 (drostatin-2) (sekv. br. 164), Ser-Arg-Pro-Tyr-Ser-Phe-Gly-Leu-NH2 (drostatin-3) (sekv. br. 165) i Thr-Thr-Arg-Pro-Gln-Pro-Phe-Asn-Phe-Gly-Leu-NH2 (drostatin-4) (sekv. br. 166) (Lenz et al., Biochem. Biophys. Res. Comm., 2000, 273, 1126-1131). Prvi Drosophila alatostatinski receptor klonirao je Birgul sa suradnicima i pokazalo se da je funkcionalno aktiviran pomou drostatin-3 putem Gi/Go puteva (Birgul et al., EMBO J., 1999, 18, 5892-5900). Drugi potencijalni Drosophila alatostatinski receptor (tj. DARII) nedavno je kloniran (Lenz et al., Biochem. Biophys. Res. Comm., 2000, 273, 571-577). DARII receptor cDNA (pristupni br. AF253526) kodira za protein koji je jako srodan s prvim that Drosophila alatostatinskim receptorom. Nedavno, functionalno aktiviranje DARII pomou alatostatina smo pokazali mi (Larsen, et al., Biochem. Biophys. Res. Comm., 2001, 286, 895-901) i drugi (Lenz, et al., Biochem. Biophys. Res. Comm., 2001, 286, 1117-1122). Nedavno, kloniran je Drosophila alatostatin tip C preprophormonski gen koji kodira Drosophila alatostatin-C: Gln-Val-Arg-Tyr-Ghi-Cys-Tyr-Phe-Asn-Pro-Ile-Ser-Cys-Phe-OH (Williamson et al., Biochem. Biophys. Res. Comm., 2001, 282, 124-130). Zreli peptid trebao bi imati pGlu na N-terminalu, koji nastaje kao rezultat N-terminalne Gln ciklizacije da se dobije: pGlu-Val-Axg-Tyr-Gln-Cys-Tyr-Phe-Asn-Pro-Ile-Ser-Cys-Phe-OH (sekv. br. 183) i disulfidni most izmeu Cys6 i Cys13, slino kao za Manduca sexta tip C alatostatina, pGlu-Val-Arg-Phe-Gln-Cys-Tyr-Phe-Asn-Pro-Ile-Ser-Cys-Phe-OH (sekv. br. 182), koji se razlikuje samo na poloaju 4 (Phe4 vs. Tyr4) (Kramer et al., Proc. Natl. Acad. Sci. USA, 1991, 88, 9458-9462). Nichols i suradnici pokazali su snano i produeno inhibiranje miine kontrakcije Drosophila alatostatina-C i nazvali su ga flatline (FLT) peptidom (Nichols et al., Peptides, 2002, 23, 787-794). Sukladno naim spoznajama, do sada nisu identificirani receptori za alatostatin tip-C. </span>Allatostatins are a significant group of insect neurohormones that control diverse functions including the synthesis of juvenile hormones known to play a central role in metamorphosis and reproduction in various insect species. The first Drosophila allatostatin, Ser-Arg-Pro-Tyr-Ser-Phe-Gly-Leu-NH2 (ie, drostatin-3) (SEQ ID NO: 165), was isolated from Drosophila head extract (Birgul et al., EMBO J ., 1999, 18, 5892-5900). Recently, the Drosophila allatostatin preprohormone gene was cloned, encoding four Drosophila allatostatins: Val-Glu-Arg-Tyr-Ala-Phe-Gly-Leu-NH2 (drostatin-1) (SEQ ID NO: 163), Leu-Pro- Val-Tyr-Asn-Phe-Gly-Leu-NH2 (drostatin-2) (SEQ ID NO: 164), Ser-Arg-Pro-Tyr-Ser-Phe-Gly-Leu-NH2 (drostatin-3) (SEQ ID NO: . no. 165) and Thr-Thr-Arg-Pro-Gln-Pro-Phe-Asn-Phe-Gly-Leu-NH2 (drostatin-4) (seq. no. 166) (Lenz et al., Biochem. Biophys Res. Comm., 2000, 273, 1126-1131). The first Drosophila allatostatin receptor was cloned by Birgul et al. and shown to be functionally activated by drostatin-3 via Gi/Go pathways (Birgul et al., EMBO J., 1999, 18, 5892-5900). Another potential Drosophila allatostatin receptor (ie, DARII) has recently been cloned (Lenz et al., Biochem. Biophys. Res. Comm., 2000, 273, 571-577). The DARII receptor cDNA (accession no. AF253526) encodes a protein that is closely related to the first Drosophila allatostatin receptor. Recently, functional activation of DARII by allatostatin has been demonstrated by us (Larsen, et al., Biochem. Biophys. Res. Comm., 2001, 286, 895-901) and others (Lenz, et al., Biochem. Biophys. Res. Comm. ., 2001, 286, 1117-1122). Recently, the Drosophila alatostatin type C preprohormone gene encoding Drosophila alatostatin-C: Gln-Val-Arg-Tyr-Ghi-Cys-Tyr-Phe-Asn-Pro-Ile-Ser-Cys-Phe-OH was cloned (Williamson et al ., Biochem. Biophys. Res. Comm., 2001, 282, 124-130). The mature peptide should have pGlu at the N-terminus, which results from N-terminal Gln cyclization to give: pGlu-Val-Axg-Tyr-Gln-Cys-Tyr-Phe-Asn-Pro-Ile-Ser-Cys- Phe-OH (SEQ ID NO: 183) and a disulfide bridge between Cys6 and Cys13, similar to Manduca sexta type C allatostatin, pGlu-Val-Arg-Phe-Gln-Cys-Tyr-Phe-Asn-Pro-Ile-Ser -Cys-Phe-OH (SEQ ID NO: 182), which differs only at position 4 (Phe4 vs. Tyr4) (Kramer et al., Proc. Natl. Acad. Sci. USA, 1991, 88, 9458-9462 ). Nichols et al. showed a strong and prolonged inhibition of Drosophila allatostatin-C muscle contraction and named it the 'flatline' (FLT) peptide (Nichols et al., Peptides, 2002, 23, 787-794). To our knowledge, no receptors for allatostatin type-C have been identified so far.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Sulfakinini su familija insektnih Tyr-sulfatiranih neuropeptida. Oni pokazuju sekvencijsku i funkcionalnu (miotropni uinci, stimuliranje otputanja digestivnog enzima) slinost s gastrin peptidima kraljenjaka i kolecistokininom. Gen koji kodira dva sulfakinina (takoer nazvani drosulfakinini), DSKI [Phe-Asp-Asp-Tyr(SO3H)-Gly-His-Met-Arg-Phe-amid] (sekv. br. 160) i DSKII [Gly-Gly-Asp-Asp-Gln-Phe-Asp-Asp-Tyr(SO3H)-Gly-His-Met-Arg-Phe-amid] (sekv. br. 161), identificiran je u Drosophila melanogaster (Nichols, Mol. Cell Neuroscience, 1992, 3, 342-347; Nichols et al., J. Biol. Chem., 1988, 263, 12167-12170). C-terminalna heptapeptidna sekvencija, Asp-Tyr(SO3H)-Gly-His-Met-Arg-Phe-amid (sekv. br. 162), identina je u svim sulfakininima koji su dosad identificirani u insektima koji su iroko odijeljeni u evolucijskom smislu. Odravanje heptapeptidne sekvencije, ukljuujui prisutnost sulfatirane Tyr rezidue, u vrlo divergentnoj insektnoj taksi vjerojatno odraava znaaj miotropne aktivne jezgre (Nachman &amp; Holman, u: Insect Neuropeptides: Chemistry, Biology and Action, Menn, Kelly &amp; Massler, Eds., American Chemical Society, Washington, D.C., 1991, str. 194-214). Nedavno, identificirali smo Drosophila usamljeni (orphan) receptor (DmGPCR9) kao drosulfakininski receptor (nazvan DSK-R1) i povezali ga s njegovim aktivirajuim peptidom, Met5Leu modifciiranim drosulfakininom-1, Asp-Tyr(SO3H)-Gly-His-Leu-Arg-Phe-amid (sekv. br. 157) (Kubiak et al., Biochem. Biophys. Res. Comm., 2002, 291, 313-320). Novi neusamljeni Drosophila GPCR-ovi ukljuuju receptore za PRXamidne peptide, CCAP, korazonin i AKH (Park et al., Proc. Natl. Acad. Sci. USA, 2002, 99, 11423-11428; Cazzamali et al., Biochem. Biophys. Res. Comm., 2002, 298, 31-36); leukokinin (Radford et.al, J. Biol. Chem. 2002, 277, 38810-38817); drostatin-C (Kreienkamp et al., J. Biol. Chem, 10.1074/jbc.M206931200 (publicirano online 6. kolovoza 2002.); FMRFamid (Cazzamali et al., Proc. Natl. Acad. Sci. USA, 2002, 39, 12073-12078); te neuoropeptid F (Mertens, et al., Biochem. Biophys. Res. Comm., 2002, 297, 1140-1148). </span>Sulfakinins are a family of insect Tyr-sulfated neuropeptides. They show sequence and functional (myotropic effects, stimulation of digestive enzyme release) similarity to vertebrate gastrin peptides and cholecystokinin. The gene encoding two sulfakinins (also called drosulfakinins), DSKI [Phe-Asp-Asp-Tyr(SO3H)-Gly-His-Met-Arg-Phe-amide] (SEQ ID NO: 160) and DSKII [Gly-Gly- Asp-Asp-Gln-Phe-Asp-Asp-Tyr(SO3H)-Gly-His-Met-Arg-Phe-amide] (SEQ ID NO: 161), was identified in Drosophila melanogaster (Nichols, Mol. Cell Neuroscience, 1992, 3, 342-347; Nichols et al., J. Biol. Chem., 1988, 263, 12167-12170). The C-terminal heptapeptide sequence, Asp-Tyr(SO3H)-Gly-His-Met-Arg-Phe-amide (SEQ ID NO: 162), is identical in all sulfakinins identified so far in widely separated insects in evolutionary terms . The maintenance of a heptapeptide sequence, including the presence of a sulfated Tyr residue, in highly divergent insect taxa likely reflects the importance of a myotropic "active core" (Nachman &amp; Holman, in: Insect Neuropeptides: Chemistry, Biology and Action, Menn, Kelly &amp; Massler, Eds., American Chemical Society, Washington, D.C., 1991, pp. 194-214). Recently, we identified the Drosophila orphan receptor (DmGPCR9) as a drosulfakinin receptor (named DSK-R1) and linked it to its activating peptide, Met5Leu modified drosulfakinin-1, Asp-Tyr(SO3H)-Gly-His-Leu -Arg-Phe-amide (SEQ ID NO: 157) (Kubiak et al., Biochem. Biophys. Res. Comm., 2002, 291, 313-320). Novel non-isolated Drosophila GPCRs include receptors for PRXamide peptides, CCAP, corazonin, and AKH (Park et al., Proc. Natl. Acad. Sci. USA, 2002, 99, 11423-11428; Cazzamali et al., Biochem. Biophys. Res. Comm., 2002, 298, 31-36); leucokinin (Radford et. al, J. Biol. Chem. 2002, 277, 38810-38817); drostatin-C (Kreienkamp et al., J. Biol. Chem, 10.1074/jbc.M206931200 (published online August 6, 2002); FMRFamide (Cazzamali et al., Proc. Natl. Acad. Sci. USA, 2002, 39 , 12073-12078); and neuropeptide F (Mertens, et al., Biochem. Biophys. Res. Comm., 2002, 297, 1140-1148).</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Saetak izuma </span>Summary of the invention</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">U ovom izumu obuhvaeno je iznenaujue otkrie novih polipeptida u Drosophila melanogaster, koji su ovdje nazvani DmGPCR-ovi (Drosophila melanogaster G protein-vezani receptori), te pokazuju razliiti stupanj homologije u odnosu na ostale neuropeptidne GPCR-ove. Ovim izumom dobivaju se geni koji kodiraju dosad nepoznate G protein-vezane receptore, DmGPCR polipeptidi koji su kodirani genima, antitijela za polipeptide, pribori koji koriste polinukleotide i polipeptide te metode dobivanja i koritenja sve navedenog. DmGPCR-ovi mogu imati ulogu kao kljune komponente, primjerice u reguliranju neuropeptidnog vezanja i/ili signaliziranja. DmGPCR-ovi su dakle korisni u potrazi za novim sredstvima koji mogu modificirati i/ili kontrolirati vezanje i/ili signaliziranje pomou neuropeptida i drugih sredstava. Nie su opisani ovi i drugi aspekti ovog izuma. </span>This invention covers the surprising discovery of new polypeptides in Drosophila melanogaster, which are here called DmGPCRs (Drosophila melanogaster G protein-coupled receptors), and which show a different degree of homology in relation to other neuropeptide GPCRs. This invention provides genes that encode hitherto unknown G protein-coupled receptors, DmGPCR polypeptides that are encoded by genes, antibodies for polypeptides, accessories that use polynucleotides and polypeptides, and methods of obtaining and using all of the above. DmGPCRs can play a role as key components, for example in regulating neuropeptide binding and/or signaling. DmGPCRs are therefore useful in the search for new agents that can modify and/or control binding and/or signaling by neuropeptides and other agents. These and other aspects of the present invention are described below.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">U nekim realizacijama, izumom se dobivaju proieni i izolirani DmGPCR polipeptidi koji sadre aminokiselinsku sekvenciju koja se nalazi u bilo kojoj od sekvencija br. 2, 4, 6, 8,10, 12, 14, 16, 18, 20, 22 ili 24 ili njen fragment koji sadri epitop specifian za DmGPCR. Pod epitop specifian za podrazumijeva se dio DmGPCR receptora koji je prepoznat od antitijela koje je specifino za DmGPCR, kao to je u detaljima definirano nie. Jedan realizacija ovog izuma obuhvaa proiene i izdvojene polipeptide koji sadre potpune aminokiselinske sekvencije koje su u aminokiselinskim sekvencijama br.: 2, 4, 6, 8, 10, 12, 14, 16, 18, 20, 22 ili 24, to je navedeni u tablici 4 dolje. Ove aminokiselinske sekvencije su izvedene iz polinukleotidnih sekvencija koje kodiraju DmGPCR (sekv. br. 1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21 ili 23, navedeni u tablici 4 nie). Pojam DmGPCR kako se ovdje rabi u jednini oznauje sekvenciju od deset aminokiselina koja je prikazana nie, koja je kodirana odgovarajuim polinukleotidnim sekvencijama. </span>In some embodiments, the invention provides purified and isolated DmGPCR polypeptides containing the amino acid sequence found in any of the sequences no. 2, 4, 6, 8, 10, 12, 14, 16, 18, 20, 22 or 24 or a fragment thereof containing an epitope specific for DmGPCR. By "epitope specific for" is meant a part of a DmGPCR receptor that is recognized by an antibody that is specific for a DmGPCR, as defined in detail below. One embodiment of the present invention includes purified and isolated polypeptides containing complete amino acid sequences that are in amino acid sequence nos: 2, 4, 6, 8, 10, 12, 14, 16, 18, 20, 22 or 24, as set forth in table 4 below. These amino acid sequences are derived from the polynucleotide sequences encoding DmGPCR (SEQ ID NO: 1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21 or 23, listed in Table 4 below). The term "DmGPCR" as used herein in the singular refers to the ten amino acid sequence shown below, which is encoded by the corresponding polynucleotide sequences.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Premda su definirane sekvencije konkretne Drosophila sekvencije, ovaj izum obuhvaa unutar svoga dosega alelne inaice, oblike DmGPCR kraljenjaka i beskraljenjaka. </span>Although specific Drosophila sequences are defined, this invention includes within its scope allelic versions, vertebrate and invertebrate forms of DmGPCR.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">U nekim realizacijama, ovaj izum definira proiene i izdvojene polinukleotide (npr. cDNA, genomsku DNA, sintetsku DNA, RNA ili njihove kombinacije, i jednostruko i dvostruko uvijene) koji sadre nukleotidnu sekvenciju koja kodira aminokiselinsku sekvenciju polipeptida ovog izuma. Takvi polinukleotidi korisni su za rekombinantnu ekspresiju receptora i takoer za detektiranje ekspresije receptora u stanicama (npr. koritenjem Northern hibridiziranja i in situ hibridizacijskih analiza). Takvi polinukleotidi su korisni u oblikovanju protivsmislenih i drugih molekula za supresiju ili regulaciju ekspresije DmGPCR u uzgojenoj stanici, tkivu ili ivotinji. Specifino izuzeti iz definicije polinukleotida ovog izuma su potpuno izdvojeni, nerekombinantni nativni kromosomi stanica domaina. Polinukleotidi ovog izuma mogu imati sekvenciju bilo koje sekvencije sljedeeg skupa: 1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21 ili 23, to odgovara prirodnim DmGPCR sekvencijama. Valja uoiti da postoje brojne druge polinukleotidne sekvencije koje takoer kodiraju DmGPCR sa sekvencijom iz sljedeeg skupa: 2, 4, 6, 8, 10, 12, 14, 16, 18, 20, 22 ili 24 zbog dobro poznate degeneracije univerzalnog genetskog koda. </span>In some embodiments, the present invention defines purified and isolated polynucleotides (eg, cDNA, genomic DNA, synthetic DNA, RNA, or combinations thereof, both single and double stranded) comprising a nucleotide sequence encoding the amino acid sequence of a polypeptide of the present invention. Such polynucleotides are useful for recombinant expression of receptors and also for detecting receptor expression in cells (eg, using 'Northern' hybridization and in situ hybridization assays). Such polynucleotides are useful in designing antisense and other molecules to suppress or regulate DmGPCR expression in a cultured cell, tissue or animal. Specifically excluded from the definition of polynucleotides of the present invention are fully isolated, non-recombinant native chromosomes of host cells. The polynucleotides of the present invention may have the sequence of any of the following set of sequences: 1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21 or 23, corresponding to native DmGPCR sequences. It should be noted that there are numerous other polynucleotide sequences that also encode a DmGPCR with a sequence from the following set: 2, 4, 6, 8, 10, 12, 14, 16, 18, 20, 22 or 24 due to the well-known degeneracy of the universal genetic code.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Izum takoer definira proiene i izdvojene polinukleotide koji sadre nukleotidnu sekvenciju koja kodira polipeptid sisavaca, pri emu polinukleotid hibridizira prema polinukleotidu sa bilo kojom sekvencijom iz sljedeeg skupa: 1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21 ili 23 ili nekodirajuoj niti koja je komplementarna, u sljedeim vjetima hibridiziranja: </span>The invention also defines purified and isolated polynucleotides comprising a nucleotide sequence encoding a mammalian polypeptide, wherein the polynucleotide hybridizes to a polynucleotide with any sequence from the following set: 1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21 or 23 or to a non-coding strand that is complementary, in the following hybridization sequences:</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">hibridiziranje 16 sati na 42C u otopini za hibridiziranje koja sadri 50% formamid, 1% SDS, 1 M NaCl, 10% dekstran sulfat; i </span>hybridization for 16 hours at 42C in a hybridization solution containing 50% formamide, 1% SDS, 1 M NaCl, 10% dextran sulfate; and</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">ispiranje 2 puta po 30 minuta na 60C u otopini za ispiranje koja sadri 0,1% SSC, 1% SDS. </span>washing 2 times for 30 minutes at 60C in a washing solution containing 0.1% SSC, 1% SDS.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Uvjeti hibridiziranja trebali bi biti takvi da se hibridiziranje zbiva samo s genima u prisutnosti molekula drugih nukleinskih kiselina. U utvrenim uvjetima hibridiziranja hibridiziraju samo sekvencije vrlo komplementarnih nukleinskih kiselina. </span>Hybridization conditions should be such that hybridization occurs only with genes in the presence of other nucleic acid molecules. Under the established hybridization conditions, only highly complementary nucleic acid sequences hybridize.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Takvi uvjeti mogu sprijeiti hibridiziranje nukleinske kiseline koja ima 1 ili 2 pogreke od 20 kontinuiranih nukleotida. </span>Such conditions can prevent nucleic acid having 1 or 2 errors out of 20 contiguous nucleotides from hybridizing.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">U nekim realizacijama, ovaj izum definira vektore koji sadre polinukleotid ovog izuma. Takvi vektori korisni su, npr. za pojaavanje polinukleotida u stanicama domaina da se stvori njegova korisna koliina. U nekim realizacijama, vektor je ekspresijski vektor pri emu je polinukleotid ovog izuma operativno povezan za polinukleotid koji sadri ekspresijsku kontrolnu sekvenciju. Takvi vektori korisni su za rekombinantnu proizvodnju polipeptida ovog izuma. </span>In some embodiments, the present invention defines vectors comprising a polynucleotide of the present invention. Such vectors are useful, eg, for amplifying a polynucleotide in host cells to generate a useful amount thereof. In some embodiments, the vector is an expression vector wherein a polynucleotide of the present invention is operably linked to a polynucleotide comprising an expression control sequence. Such vectors are useful for recombinant production of the polypeptides of the present invention.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">U nekim realizacijama, ovaj izum definira stanice domaina koje su transformirane ili transficirane (postojano ili prolazno) s polinukleotidima ovog izuma ili vektorima ovog izuma. Kao to je prije navedeno, takve stanice domaina korisne su za pojaavanje polinukleotida i takoer za ekspresiju DmGPCR polipeptida ili njegova fragmenta koji je kodiran pomou polinukleotida. </span>In some embodiments, the present invention defines host cells that have been transformed or transfected (permanently or transiently) with polynucleotides of the present invention or vectors of the present invention. As previously noted, such host cells are useful for amplifying the polynucleotide and also for expressing the DmGPCR polypeptide or fragment thereof encoded by the polynucleotide.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">U sljedeoj realizaciji, izumom su definirane metode za dobivanje DmGPCR polipeptida (ili njegova fragmenta) to obuhvaa stupnjeve rasta stanice domaina ovog izuma u hranjivom mediju te izdvajanje polipeptida ili njegove inaice iz stanice ili iz medija. Budui da je DmGPCR sedam transmembranski receptor, uoava se da za neke primjene, kao to su odreene analize aktivnosti, izdvajanje moe ukljuivati izdvajanje staninih membrana koje sadre ugraeni polipeptid, dok za druge primjene moe biti poeljno potpunije izdvajanje. </span>In the next embodiment, the invention defines methods for obtaining a DmGPCR polypeptide (or its fragment), which includes growth stages of the host cell of this invention in a nutrient medium and extracting the polypeptide or its version from the cell or from the medium. Since DmGPCR is a seven transmembrane receptor, it is appreciated that for some applications, such as certain activity assays, isolation may involve separation of cell membranes containing the embedded polypeptide, while for other applications more complete isolation may be desirable.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Uoljivo je da su izvanstanini epitopi osobito korisni za generiranje i screening antitijela i drugih vezujuih spojeva koji se veu za receptore kao to je DmGPCR. Prema tome, u sljedeoj realizaciji, ovaj izum definira proien i izdvojen polipeptid koji sadri bar jednu izvanstaninu domenu (npr. N-terminalnu izvanstaninu domenu ili jednu od triju izvanstaninih petlji) DmGPCR-a kao to je N-terminalna izvanstanina domena DmGPCR-a. Takoer su obuhvaeni ovim izumom proieni polipeptidi koji sadre transmembranske domene DmGPCR-a, izvanstaninu petlju koja povezuje transmembranske domene DmGPCR-a, unutarstaninu petlju koja povezuje transmembranske domene DmGPCR-a, C-terminalno citoplazmatsko podruje DmGPCR i njihove fuzije. Takvi fragmenti mogu biti kontinuirani dijelovi nativnog receptora. Meutim, valja uoiti da poznavanje gena DmGPCR-a i proteinskih sekvencija koje je ovdje definirano omoguuje rekombiniranje razliitih domena koje nisu kontinuirane u nativnom proteinu. </span>It can be seen that extracellular epitopes are particularly useful for the generation and screening of antibodies and other binding compounds that bind to receptors such as DmGPCR. Accordingly, in a further embodiment, the present invention defines a purified and isolated polypeptide comprising at least one extracellular domain (eg, the N-terminal extracellular domain or one of the three extracellular loops) of a DmGPCR such as the N-terminal extracellular domain of a DmGPCR. Also covered by this invention are purified polypeptides containing the transmembrane domains of DmGPCR, the extracellular loop connecting the transmembrane domains of DmGPCR, the intracellular loop connecting the transmembrane domains of DmGPCR, the C-terminal cytoplasmic region of DmGPCR and their fusions. Such fragments may be continuous parts of the native receptor. However, it should be noted that the knowledge of the DmGPCR gene and protein sequences defined here allows the recombination of different domains that are not continuous in the native protein.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">U daljnjoj realizaciji, ovaj izum definira antitijela specifina za DmGPCR ovog izuma. Specifinost antitijela opisana je s vie pojedinosti nie. Meutim, valja naglasiti da se antitijela koja mogu biti generirana iz polipeptida koji su prije opisni u literaturi i koji mogu sluajno interferirati (unakrsna reakcija, cross-reaction) s DmGPCR-om (npr. zbog sluajnog postojanja slinog epitopa u oba polipeptida) smatraju interferencijski reaktivnim (cross-reactive) antitijelima. Takva antitijela nisu antitijela koja se specifina za DmGPCR. Odreivanje je li neko antitijelo specifino DmGPCR ili je interferencijski reaktivno s drugim poznatim receptorom moe se izvriti pomou nekoliko analiza, kao to su Western blotting analize, to je dobro poznato u tehnici. Za identificiranje stanica koje vre ekspresiju DmGPCR i takoer za moduliranje DmGPCR-ligand vezujue aktivnosti, mogu se rabiti antitijela koja se specifino veu za izvanstanini epitop DmGPCR-a. </span>In a further embodiment, the present invention defines antibodies specific for the DmGPCR of the present invention. Antibody specificity is described in more detail below. However, it should be emphasized that antibodies that can be generated from polypeptides previously described in the literature and that can accidentally interfere ("cross-reaction") with DmGPCR (e.g. due to the accidental existence of a similar epitope in both polypeptides) are considered interference-reactive ("cross-reactive") antibodies. Such antibodies are not antibodies that are "specific" for DmGPCR. Determining whether an antibody is DmGPCR specific or cross-reactive with another known receptor can be performed using several assays, such as Western blotting assays, which are well known in the art. To identify cells expressing DmGPCR and also to modulate DmGPCR-ligand binding activity, antibodies that specifically bind to the extracellular epitope of DmGPCR can be used.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">U jednoj inaici, ovaj izum definira monoklonska antitijela. Hibridomi koji proizvode takva antitijela takoer su obuhvaeni kao aspekti ovog izuma. </span>In one embodiment, the present invention defines monoclonal antibodies. Hybridomas producing such antibodies are also encompassed as aspects of the present invention.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">U sljedeoj inaici, ovaj izum definira smjesu bez stanica koja sadri poliklonska antitijela, pri emu bar jedno od antitijela je antitijelo ovog izuma koje je specifino za DmGPCR. Antiserumi koji su izdvojeni iz ivotinje su primjer smjese, kao to je i smjesa koja sadri frakciju antitijela antiseruma koja je resuspendirana u vodi ili u drugom razrjeivau, ekscipijentu ili nosau. </span>In another embodiment, the present invention defines a cell-free mixture comprising polyclonal antibodies, wherein at least one of the antibodies is an antibody of the present invention that is specific for DmGPCR. Antisera that have been isolated from an animal are an example of a mixture, as is a mixture containing an antibody fraction of an antiserum that has been resuspended in water or in another diluent, excipient, or carrier.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">U sljedeoj srodnoj realizaciji, ovaj izum definira anti-idiotipska antitijela za antitijelo koje je specifino za DmGPCR. </span>In another related embodiment, the present invention defines anti-idiotypic antibodies for an antibody that is specific for a DmGPCR.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Dobro je poznato da antitijela sadre relativno malene antigen vezujue domene koje se mogu izdvojiti kemijski ili pomou rekombinantnih tehnika. Takve domene same su korisne su DmGPCR vezujue molekule i takoer mogu biti sjedinjene s toksinima ili drugim polipeptidima. Dakle, u sljedeoj realizaciji, izum definira polipeptid koji sadri fragment DmGPCR-specifinog antitijela, pri emu se fragment i polipeptid veu za DmGPCR. Kao primjer koji ne ograniuje, ovaj izum definira polipeptide koji su jednolanasta antitijela, CDR-usaena antitijela i humanizirana antitijela. </span>It is well known that antibodies contain relatively small antigen-binding domains that can be isolated chemically or by recombinant techniques. Such domains alone are useful DmGPCR binding molecules and can also be fused to toxins or other polypeptides. Thus, in the following embodiment, the invention defines a polypeptide containing a fragment of a DmGPCR-specific antibody, wherein the fragment and the polypeptide bind to the DmGPCR. By way of non-limiting example, the present invention defines polypeptides that are single-chain antibodies, CDR-grafted antibodies, and humanized antibodies.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Takoer su unutar dosega ovog izuma smjese koje sadre polipeptide, polinukleotide ili antitijela ovog izuma koji su formulirani npr. s farmaceutski prihvatljivim nosaem. </span>Also within the scope of this invention are mixtures containing polypeptides, polynucleotides or antibodies of this invention that are formulated, for example, with a pharmaceutically acceptable carrier.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Izum takoer definira metode za koritenje antitijela ovog izuma. Primjerice, izumom su definirane metode za moduliranje ligandnog vezanja DmGPCR-a to ukljuuje stupanj dovoenja u dodir DmGPCR s antitijelom specifinim za DmGPCR, u uvjetima pri emu antitijelo vee receptor. </span>The invention also defines methods for using the antibodies of the invention. For example, the invention defines methods for modulating ligand binding of DmGPCR, which includes the step of bringing the DmGPCR into contact with an antibody specific for DmGPCR, under conditions where the antibody binds the receptor.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Izumom su definirane metode izazivanja imunolokog odgovora u subjekta prema polipeptidu koji sadri sekvenciju iz skupa koji se sastoji od sekvencija s br. 2, 4, 6, 8, 10, 12, 14, 16, 18, 20, 22 i 24 ili homologa ili fragmenta. Metode obuhvaaju primjenu subjektu odreene koliine polipeptida koja je dovoljna da se izazove imunoloki odgovor. </span>The invention defines methods of inducing an immune response in a subject to a polypeptide containing a sequence from the set consisting of sequences with no. 2, 4, 6, 8, 10, 12, 14, 16, 18, 20, 22 and 24 or homologs or fragments thereof. The methods comprise administering to the subject an amount of the polypeptide sufficient to elicit an immune response.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Izum takoer definira analize da se identificiraju spojevi koji veu DmGPCR. Jedna takva analiza ukljuuje sljedee stupnjeve: (a) dovoenje u dodir smjese koja sadri DmGPCR sa spojem za kojega se oekuje da vee DmGPCR; i (b) mjerenje vezanja izmeu spoja i DmGPCR. U jednoj inaici, smjesa sadri stanice s ekspresijom DmGPCR na svojoj povrini. U sljedeoj inaici, koriste se izdvojeni DmGPCR ili stanine membrane koje sadre DmGPCR. Vezanje se moe mjeriti izravno, npr. pomou obiljeenog spoja ili se moe mjeriti posredno pomou nekoliko tehnika, ukljuujui mjerenje unutarstaninog signaliziranja DmGPCR-a koje je izazvano pomou spoja (ili mjerenje promjena u razini signaliziranja DmGPCR-a). </span>The invention also defines assays to identify compounds that bind DmGPCR. One such assay involves the following steps: (a) contacting a mixture containing a DmGPCR with a compound expected to bind the DmGPCR; and (b) measuring binding between the compound and DmGPCR. In one embodiment, the mixture contains cells with DmGPCR expression on its surface. In a further variant, isolated DmGPCRs or cell membranes containing DmGPCRs are used. Binding can be measured directly, eg, using a labeled compound, or can be measured indirectly using several techniques, including measuring compound-induced intracellular DmGPCR signaling (or measuring changes in the level of DmGPCR signaling).</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Izum takoer definira metode vezanja DmGPCR s vezujuim partnerom. Metode ukljuuju sljedee stupnjeve: (a) dovoenje u dodir smjese koja sadri DmGPCR s vezujuim partnerom i (b) omoguavanje da vezujui partner vee DmGPCR. Primjerice, DmGPCR moe biti DmGPCR1 (sekv. br. 1), DmGPCR5 (sekv. br. 9), DmGPCR7 (sekv. br. 17) ili DmGPCR8 (sekv. br. 19). Vezujui partner moe biti, primjerice, drotahikinin, leukokinin ili alatostatin-C. Drotahikinin (DTK) moe biti, primjerice, DTK-1 (sekv. br. 169), Met8-DTK-2 (sekv. br. 170), DTK-2 (sekv. br. 171), DTK-3 (sekv. br. 172), DTK-4 (sekv. br. 173) i DTK-5 (sekv. br. 174). Leukokinin (LK) moe biti, primjerice, LK-I (sekv. br. 175), LK-V (sekv. br. 176), LK-VI (sekv. br. 177) i LK-VIII (sekv. br. 178), kulekinin (sekv. br. 179), mekuac leukokinin-slini peptid, limnokinin (PSFHSWSa) (sekv. br. 180) i Drosophila leukokinin-slini peptidi DLK-1 (NSVVLGKKQRFHSWGa) (sekv. br. 181), DLK-2 (pGlu-RFHSWGa) (sekv. br. 182) i DLK-2 (QRFHSWGa) (sekv. br. 183). Alatostatin (AST) moe biti, primjerice, AST-C (sekv. br. 184) ili DST-C (sekv. br. 185). Drugi vezujui partneri obuhvaaju, bez ogranienja, sekv. br. 186 i sekv. br. 187. </span>The invention also defines methods of binding a DmGPCR to a binding partner. The methods include the following steps: (a) contacting the mixture containing the DmGPCR with the binding partner and (b) allowing the binding partner to bind the DmGPCR. For example, the DmGPCR can be DmGPCR1 (SEQ ID NO: 1), DmGPCR5 (SEQ ID NO: 9), DmGPCR7 (SEQ ID NO: 17) or DmGPCR8 (SEQ ID NO: 19). The binding partner can be, for example, drotachikinin, leucokinin or alatostatin-C. Drotachykinin (DTK) can be, for example, DTK-1 (SEQ ID NO: 169), Met8-DTK-2 (SEQ ID NO: 170), DTK-2 (SEQ ID NO: 171), DTK-3 (SEQ ID NO: 171). no. 172), DTK-4 (seq. no. 173) and DTK-5 (seq. no. 174). Leukokinin (LK) can be, for example, LK-I (SEQ ID NO: 175), LK-V (SEQ ID NO: 176), LK-VI (SEQ ID NO: 177) and LK-VIII (SEQ ID NO: 177). 178), culekinin (SEQ ID NO: 179), mollusc leucokinin-like peptide, limnokinin (PSFHSWSSa) (SEQ ID NO: 180) and Drosophila leucokinin-like peptides DLK-1 (NSVVLGKKQRFHSWGa) (SEQ ID NO: 181), DLK -2 (pGlu-RFHSWGa) (SEQ ID NO: 182) and DLK-2 (QRFHSWGa) (SEQ ID NO: 183). Allatostatin (AST) can be, for example, AST-C (SEQ ID NO: 184) or DST-C (SEQ ID NO: 185). Other binding partners include, without limitation, seq. no. 186 et seq. no. 187.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Izum takoer definira metode za identificiranje modulatora vezanja izmeu DmGPCR i DmGPCR vezujueg partnera, to obuhvaa sljedee stupnjeve: (a) dovoenje u dodir DmGPCR vezujueg partnera i smjese koja sadri DmGPCR u prisutnosti i u odsutnosti potencijalnog modulatorskog spoja; (b) detektiranje vezanja izmeu vezujueg partnera i DmGPCR; i (c) identificiranje modulatorskog spoja ili modulatorskog spoja u svjetlu smanjenog ili poveanog vezanja izmeu vezujueg partnera i DmGPCR u prisutnosti potencijalnog modulatora, u usporedbi s vezanjem u odsutnosti potencijalnog modulatora. Primjerice, DmGPCR moe biti DmGPCR5 (sekv. br. 9), DmGPCR7 (sekv. br. 17) ili DmGPCR8 (sekv. br. 19). Vezujui partner moe biti, primjerice, drotahikinin, leukokinin ili alatostatin. Drotahikinin (DTK) moe biti, primjerice, DTK-1 (sekv. br. 169), Met8-DTK-2 (sekv. br. 170), DTK-2 (sekv. br. 171), DTK-3 (sekv. br. 172), DTK-4 (sekv. br. 173) i DTK-5 (sekv. br. 174). Leukokinin (LK) moe biti, primjerice, LK-I (sekv. br. 175), LK-V (sekv. br. 176), LK-VI (sekv. br. 177) i LK-VIII (sekv. br. 178), kulekinin (sekv. br. 179), mekuac leukokinin-slini peptid, limnokinin (PSFHSWSa) (sekv. br. 180) i Drosophila leukokinin-slini peptidi DLK-1 (NSVVLGKKQRFHSWGa) (sekv. br. 181), DLK-2 (pGlu-RFHSWGa) (sekv. br. 182) i DLK-2A (QRFHSWGa) (sekv. br. 183). Alatostatin (AST) moe biti, primjerice, AST-C (sekv. br. 184) ili DST-C (sekv. br. 185). U jednoj inaici, smjesa sadri stanice koje vre ekspresiju DmGPCR nas svojoj povrini. U drugoj inaici, koriste se izdvojeni DmGPCR ili stanine membrane koje sadre DmGPCR. Vezanje se moe mjeriti izravno, npr. koritenjem obiljeenog spoja ili se moe mjeriti posredno pomou nekoliko tehnika, ukljuujui mjerenje unutarstaninog signaliziranja DmGPCR-a to je izazvano pomou spoja (ili mjerenje promjena razine signaliziranja DmGPCR-a). Primjerice, funkcija se moe mjeriti pomou analize agonistom izazvanog vezanja [35S]GTPS, pomou cAMP analize (induciranje ili inhibiranje proizvodnje cAMP) ili mjerenjem unutarstanine razine kalcija pomou analize fluorimetrijskog itaa ploe (FLIPR). </span>The invention also defines methods for identifying binding modulators between a DmGPCR and a DmGPCR binding partner, comprising the following steps: (a) contacting the DmGPCR binding partner and a mixture containing the DmGPCR in the presence and absence of a potential modulatory compound; (b) detecting binding between the binding partner and the DmGPCR; and (c) identifying the modulatory compound or modulatory compound in light of decreased or increased binding between the binding partner and the DmGPCR in the presence of the potential modulator, compared to binding in the absence of the potential modulator. For example, the DmGPCR can be DmGPCR5 (SEQ ID NO: 9), DmGPCR7 (SEQ ID NO: 17) or DmGPCR8 (SEQ ID NO: 19). The binding partner can be, for example, drotachikinin, leucokinin or allatostatin. Drotachykinin (DTK) can be, for example, DTK-1 (SEQ ID NO: 169), Met8-DTK-2 (SEQ ID NO: 170), DTK-2 (SEQ ID NO: 171), DTK-3 (SEQ ID NO: 171). no. 172), DTK-4 (seq. no. 173) and DTK-5 (seq. no. 174). Leukokinin (LK) can be, for example, LK-I (SEQ ID NO: 175), LK-V (SEQ ID NO: 176), LK-VI (SEQ ID NO: 177) and LK-VIII (SEQ ID NO: 177). 178), culekinin (SEQ ID NO: 179), mollusc leucokinin-like peptide, limnokinin (PSFHSWSSa) (SEQ ID NO: 180) and Drosophila leucokinin-like peptides DLK-1 (NSVVLGKKQRFHSWGa) (SEQ ID NO: 181), DLK -2 (pGlu-RFHSWGa) (SEQ ID NO: 182) and DLK-2A (QRFHSWGa) (SEQ ID NO: 183). Allatostatin (AST) can be, for example, AST-C (SEQ ID NO: 184) or DST-C (SEQ ID NO: 185). In one embodiment, the mixture contains cells that express DmGPCR on their surface. In another variant, isolated DmGPCRs or cell membranes containing DmGPCRs are used. Binding can be measured directly, eg, using a labeled compound, or can be measured indirectly using several techniques, including measuring intracellular DmGPCR signaling induced by the compound (or measuring changes in the level of DmGPCR signaling). For example, function can be measured using an agonist-induced [35S]GTPS binding assay, using a cAMP assay (inducing or inhibiting cAMP production), or measuring intracellular calcium levels using a fluorimetric plate reader (FLIPR) assay.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">DmGPCR vezujui partneri koji stimuliraju aktivnost DmGPCR korisni su kao agonisti za pojaanje ili produljenje signaliziranja DmGPCR i na taj nain interferiraju sa signalnim putovima normalno aktiviranog receptora. DmGPCR vezujui partneri koji blokiraju ligand-upravljano signaliziranje DmGPCR korisni su kao DmGPCR antagonisti u interferenciji s normalnim DmGPCR signaliziranjem te ometaju receptor-upravljane uinke. Nadalje, DmGPCR modulatori, kao i DmGPCR polinukleotidi i polipeptidi, korisni su u dijagnostikim analizama za stanja u kojima je DmGPCR aktivnost poveana ili oteena. </span>DmGPCR binding partners that stimulate DmGPCR activity are useful as agonists to enhance or prolong DmGPCR signaling and thereby interfere with signaling pathways of the normally activated receptor. DmGPCR binding partners that block ligand-directed DmGPCR signaling are useful as DmGPCR antagonists in interfering with normal DmGPCR signaling and interfering with receptor-directed effects. Furthermore, DmGPCR modulators, as well as DmGPCR polynucleotides and polypeptides, are useful in diagnostic assays for conditions in which DmGPCR activity is increased or impaired.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">U sljedeem aspektu, izumom su definirane metode poremeaja ili bolesti koji su izazvani ektoparazitom primjenom subjektu kojemu je to potrebno tvari koja modulira aktivnost ili ekspresiju polipeptida ektoparazita koji je odabran iz skupa kojega sainjavaju sekvencije s br. 2, 4, 6, 8, 10, 12, 14, 16, 18, 20, 22 ili 24. </span>In the next aspect, the invention defines methods of disorders or diseases that are caused by an ectoparasite by administering to a subject in need of it a substance that modulates the activity or expression of an ectoparasite polypeptide selected from the group consisting of sequences with no. 2, 4, 6, 8, 10, 12, 14, 16, 18, 20, 22 or 24.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Tvari koje su korisne za tretiranje poremeaja ili bolesti koji su izazvani ektoparazitom mogu pokazati pozitivne rezultate u jednoj ili vie in vitro analiza na aktivnost koja odgovara tretmanu konkretnog poremeaja ili bolesti. Tvari koje moduliraju aktivnost polipeptida ukljuuju, ali nisu ograniene na protivsmislene oligonukleotide, agoniste i antagonisti te antitijela. </span>Substances useful for treating a disorder or disease caused by an ectoparasite may show positive results in one or more in vitro assays for activity corresponding to the treatment of a particular disorder or disease. Agents that modulate polypeptide activity include, but are not limited to, antisense oligonucleotides, agonists and antagonists, and antibodies.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">U sljedeem aspektu, ovaj izum oblikuje metode za detektiranje polipeptida u uzorku kao dijagnostiko pomagalo za bolesti i poremeaje koji su izazvani ektoparazitima, pri emu metode ukljuuju sljedee stupnjeve: (a) dovoenje u dodir uzorka sa sondom nukleinske kiseline koja hibridizira u hibridizacijskim uvjetima analize ciljano podruje nukleinske kiseline koja kodira polipeptid koji je odabran iz skupa kojega sainjavaju sekvencije s br. 2, 4, 6, 8, 10, 12, 14, 16, 18, 20, 22 ili 24, a navedena sonda sadri sekvenciju nukleinske kiseline koja kodira polipeptid, njegove fragmenta i/ili komplemente sekvencija i fragmente; i (b) detektiranje prisutnosti ili odreene koliine hibrida sonda:ciljano podruje kao indikacije stanja. </span>In a further aspect, the present invention provides methods for detecting polypeptides in a sample as a diagnostic aid for diseases and disorders caused by ectoparasites, the methods comprising the following steps: (a) contacting the sample with a nucleic acid probe that hybridizes under hybridization assay conditions targeted a nucleic acid region encoding a polypeptide selected from the group consisting of sequences with no. 2, 4, 6, 8, 10, 12, 14, 16, 18, 20, 22 or 24, and said probe contains a nucleic acid sequence encoding a polypeptide, its fragments and/or sequence complements and fragments; and (b) detecting the presence or amount of the probe:target hybrid as an indication of the condition.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Testni uzorci koji su odgovarajui za metode sondiranja nukleinske kiseline ovog izuma obuhvaaju primjerice stanice ili ekstrakte stanica nukleinskih kiselina ili bioloke tekuine. Uzorci koji se rabe u gore opisanim metodama e varirati sukladno formatu analiza, metodi detektiranja i prirodi tkiva, stanica ili ekstrakata koji se analiziraju. Metode prireivanja ekstrakata stanica nukleinskih kiselina dobro su poznate u tehnici i mogu se lako prilagoditi da se dobiju uzorci koji su sukladno s koritenom metodom. </span>Test samples suitable for the nucleic acid probing methods of the present invention include, for example, cells or nucleic acid cell extracts or biological fluids. The samples used in the methods described above will vary according to the analysis format, the detection method and the nature of the tissues, cells or extracts being analyzed. Methods for preparing nucleic acid cell extracts are well known in the art and can be readily adapted to obtain samples consistent with the method used.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">U nekim realizacijama ovaj izum definira homologe, kao to su homolozi sisavaca, DmGCPR-ova. Homolozi DmGPCR-a sisavaca mogu biti s ekspresijom u tkivima ukljuujui ali bez ogranienja tkiva ivanog sustava, guterae (te konkretno guterainog otonog tkiva), hipofize, skeletnog miia, adipoznog tkiva, jetre, probavnog (GI)-sustava i titnjae. </span>In some embodiments, the present invention defines homologues, such as mammalian homologues, of DmGCPRs. Mammalian DmGPCR homologues may be expressed in tissues including but not limited to tissues of the nervous system, pancreas (and specifically pancreatic islet tissue), pituitary gland, skeletal muscle, adipose tissue, liver, digestive (GI)-system and thyroid.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">U nekim realizacijama, ovaj izum definira metode identificiranja homologa DmGPCR-a sisavca to obuhvaa stupnjeve screening-a baze nukleinskih kiselina ili biblioteke nukleinskih kiselina sisavaca s molekulom nukleinske kiseline koja je odabrana iz skupa sekvencija s br. 1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21 i 23 ili njenog dijela te odreivanja je li dio baze podataka ili biblioteke homologan sa sekvencijom. </span>In some embodiments, the present invention provides methods of identifying homologues of a mammalian DmGPCR comprising the steps of screening a nucleic acid base or library of mammalian nucleic acids with a nucleic acid molecule selected from the set of sequences no. 1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21 and 23 or part thereof and determining whether part of the database or library is homologous to the sequence.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Sljedeim aspektom izuma su definirane metode kontroliranja populacije insekata primjenom vezujueg partnera ili modulatora DmGPCR polinukleotida ili polipeptida prema insektu da se modificira ekspresija ili aktivnost DmGPCR-a. Primjerice, insekt moe biti odabran iz skupa kojega sainjavaju muha, vona muha, krpelj, buha, ui, grinje i ohar. </span>A further aspect of the invention defines methods of controlling insect populations by applying binding partners or modulators of DmGPCR polynucleotides or polypeptides to the insect to modify the expression or activity of DmGPCR. For example, an insect can be selected from the group consisting of a fly, a fruit fly, a tick, a flea, lice, mites, and a cockroach.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">DmGPCR binding partner moe biti drotahikinin (npr. DTK-1 (sekv. br. 169), Met8-DTK-2 (sekv. br. 170), DTK-2 (sekv. br. 171), DTK-3 (sekv. br. 172), DTK-4 (sekv. br. 173) i DTK-5 (sekv. br. 174)), leukokinin (npr. LK-I (sekv. br. 175), LK-V (sekv. br. 176), LK- VI (sekv. br. 177) i LK-VHI (sekv. br. 178), kulekinin (sekv. br. 179), mekuac leukokinin-slini peptid, limnokinin (PSFHSWSa) (sekv. br. 180), DLK-1 (sekv. br. 181), DLK-2 (sekv. br. 182) i DLK-2A (QRFHSWGa) (sekv. br. 183)) ili alatostatin (AST-C (sekv. br. 184 ili DST-C sekv. br. 185)). Drugi vezujui partneri ukljuuju, bez ogranienja, sekv. br. 186 i sekv. br. 187. DmGPCR modulator moe biti anti- DmGPCR antitijelo ili DmGPCR protivsmisleni polinukleotid. </span>DmGPCR binding partner can be drotachikinin (e.g. DTK-1 (SEQ ID NO: 169), Met8-DTK-2 (SEQ ID NO: 170), DTK-2 (SEQ ID NO: 171), DTK-3 (SEQ ID NO: 171). No. 172), DTK-4 (SEQ ID NO: 173) and DTK-5 (SEQ ID NO: 174)), leucokinin (e.g. LK-I (SEQ ID NO: 175), LK-V (SEQ ID NO: 175) . 176), LK-VI (SEQ ID NO: 177) and LK-VHI (SEQ ID NO: 178), Kulekinin (SEQ ID NO: 179), molluscan leucokinin-like peptide, limnokinin (PSFHSWSa) (SEQ ID NO: 179). 180), DLK-1 (SEQ ID NO: 181), DLK-2 (SEQ ID NO: 182) and DLK-2A (QRFHSWGa) (SEQ ID NO: 183)) or allatostatin (AST-C (SEQ ID NO: 183)). 184 or DST-C seq. no. 185)). Other binding partners include, without limitation, seq. no. 186 et seq. no. 187. The DmGPCR modulator can be an anti-DmGPCR antibody or a DmGPCR antisense polynucleotide.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Sljedeim aspektom izuma su definirane metode prevencije ili tretiranja bolesti ili stanja koje je izazvano ektoparazitom u subjekta-domaina primjenom subjektu vezujueg partnera ili modulatora DmGPCR-ova polinukleotida ili polipeptida da se modificira ekspresija ili aktivnost DmGPCR-a. </span>A further aspect of the invention defines methods of preventing or treating a disease or condition caused by an ectoparasite in a subject-host by applying to the subject a binding partner or modulator of DmGPCR polynucleotides or polypeptides to modify the expression or activity of DmGPCR.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Daljnja svojstva i inaice ovog izuma bit e oigledni onima koji poznaju ovo podruje tehnike iz cjeline ove prijave, ukljuujui detaljni opis i sva takva svojstva koja se smatraju aspektima ovog izuma. Slino, svojstva ovog izuma koja su ovdje opisana mogu se rekombinirati u dodatne realizacije koje se smatraju aspektima ovog izuma, bez obzira smatra li se kombinacija svojstava koja je specifino navedena prije aspektom ili realizacijom ovog izuma. Takoer, samo takva ogranienja koja su ovdje opisana kao kritina treba promatrati kao takva. Inaice ovog izuma koje nemaju ogranienja u ovom opisu koja se smatraju kritinima potpadaju pod aspekte ovog izuma. </span>Further features and variations of the present invention will be apparent to those skilled in the art from the entirety of this application, including the detailed description and all such features which are considered aspects of the present invention. Similarly, the features of this invention described herein may be recombined into additional embodiments that are considered aspects of this invention, regardless of whether the combination of features specifically recited above is considered an aspect or embodiment of this invention. Also, only such limitations described here as critical should be considered as such. Variations of the present invention which are not limited in this description and which are considered critical fall within the aspects of the present invention.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Detaljni opis poeljnih realizacija izuma </span>Detailed description of preferred embodiments of the invention</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Ovaj izum definira, izmeu ostalog, izdvojene ili proiene polinukleotide koji kodiraju D. melanogaster G protein vezani receptor (DmGPCR) ili njegov dio, vektore koji sadre ove polinukleotida, stanice domaina koje su transformirane ovim vektorima, procese dobivanja DmGPCR, metode koritenja gore navedenih polinukleotida i vektora, izdvojene i proiene DmGPCR, metode za screening spojeva koji moduliraju DmGPCR aktivnost i metode identificiranja homologa DmGPCR-a sisavaca, kraljenjaka i beskraljenjaka. </span>This invention defines, among other things, isolated or purified polynucleotides encoding the D. melanogaster G protein-coupled receptor (DmGPCR) or its part, vectors containing these polynucleotides, host cells transformed with these vectors, processes for obtaining DmGPCR, methods of using the aforementioned polynucleotides and vectors, isolated and purified DmGPCR, methods for screening compounds that modulate DmGPCR activity and methods for identifying DmGPCR homologues of mammals, vertebrates and invertebrates.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">U ovom dokumentu postoje razliite definicije. Veina izraza koji se rabe ima znaenje koje im pridjeljuju osobe koje poznaju ovo podruje tehnike. Izrazi koji su specifino deifnirani bilo ni i drugdje u ovom dokumentu imaju znaenje koje je definirano u kontekstu ovog izuma kao cjeline i tipino s epodrazumijeva od strane onih koji poznaju ovo podruje tehnike. </span>There are different definitions in this document. Most of the terms used have a meaning assigned to them by persons familiar with the art. Terms specifically defined either below or elsewhere in this document have the meaning as defined in the context of this invention as a whole and typically understood by those skilled in the art.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Podrazumijeva se da ako je navedena skupina sekvencija, njihove kombinacije i potkombinacije takoer se specifino podrazumijevaju. Primjerice, kada u opisu pie sekvencije s rednim br. 1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21 ili 23, podrazumijeva se da ovaj izum ukljuuje kombinacije i potkombinacije, ali bez ogranienja, kombinacije sekvencija 1 and 3; 1 i 5; 1, 3 i 5; itd. </span>It is understood that if a group of sequences is specified, combinations and subcombinations thereof are also specifically intended. For example, when the description says "sequences with serial no. 1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21 or 23", it is understood that this invention includes combinations and subcombinations, but without limitation, combinations of sequences 1 and 3; 1 and 5; 1, 3 and 5; etc.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Sintetiziran kako se rabi ovdje i u tehnici, odnosi se na polinukleotide koji su dobiveni isto kemijski, za razliku od enzimskih metoda. Potpuno sintetizirane DNA sekvencije su prema tome dobivene iskljuivo kemijskim sredstvima i djelomino sintetizirane DNA obuhvaaju one u kojih je samo dio rezultirajue DNA dobiven kemijskim nainom. </span>"Synthesized" as used herein and in the art refers to polynucleotides that have been obtained purely chemically, as opposed to by enzymatic methods. "Fully" synthesized DNA sequences are thus obtained exclusively by chemical means and "partially" synthesized DNA includes those in which only part of the resulting DNA was obtained by chemical means.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Pojmom podruje oznauje se fiziki kontinuirani dio primarne strukture biomolekule. U sluaju proteina, podruje se definira kao kontinuirani dio aminokiselinske sekvencije toga proteina. </span>The term "region" refers to a physically continuous part of the primary structure of a biomolecule. In the case of a protein, a region is defined as a continuous part of the amino acid sequence of that protein.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Pojam domena koristi se ovdje da oznai strukturni dio biomolekule koji doprinosi poznatoj ili suspektnoj funkciji biomolekule. Domene mogu biti koekstenzivne s njenim dijelovima ili podrujima. Domene mogu takoer sadravati dio molekule koji se razlikuje od konkretnog podruja, uz cjelinu ili dio toga podruja. Primjeri domena GPCR proteina ukljuuju, ali nisu ogranieni na izvanstanine (tj. N-terminal), transmembranske i citoplazmatske (tj. C-terminal) domene, koje su koekstenzivne sa slino nazvanim podrujima GPCR-ova, svakog od sedam transmembranskih segmenata GPCR-a. Svaki od segmenata petlje (i izvanstaninih i unutarstaninih petlji) povezuje susjedne transmembranske segmente. </span>The term "domain" is used herein to denote a structural part of a biomolecule that contributes to a known or suspected function of the biomolecule. Domains can be coextensive with its parts or areas. Domains can also contain a part of the molecule that differs from the specific region, in addition to all or part of that region. Examples of GPCR protein domains include, but are not limited to, extracellular (ie, N-terminal), transmembrane, and cytoplasmic (ie, C-terminal) domains, which are coextensive with similarly named regions of GPCRs, of each of the seven transmembrane segments of GPCRs . Each of the loop segments (both extracellular and intracellular loops) connects adjacent transmembrane segments.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Kako se ovdje rabi, pojam aktivnost odnosi se na cijeli niz mjerljivih indeksa koji ukazuju ili dokazuju vezanje, bilo izravno ili posredno; uz djelovanje na odgovor, tj. postoji mjerljivi uinak kao odgovor na neko izlaganje ili stimuliranje, ukljuujui, primjerice, afinitet spoja za izravno vezanje polipeptida ili polinukleotida ovog izuma, ili, primjerice, mjerenje koliine uzvodnih ili nizvodnih proteina ili drugih funkcija nakon neko stimulansa ili dogaaja. </span>As used herein, the term "activity" refers to a whole range of measurable indices that indicate or prove binding, either directly or indirectly; with a response effect, i.e., there is a measurable effect in response to some exposure or stimulation, including, for example, the affinity of a compound to directly bind a polypeptide or polynucleotide of this invention, or, for example, measuring the amount of 'upstream' or 'downstream' proteins or other functions after some stimulus or event.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Kako se ovdje rabi, pojam antitijelo namijenjen je da oznauje cjelovita, intaktna antitijela i Fab, Fab, F(ab)2, Fv i druge fragmente. Cjelovita, intaktna antitijela ukljuuju monoklonska antitijela kao to su mija monoklonska antitijela, kimerna antitijela, humana antitijela i humanizirana antitijela. </span>As used herein, the term "antibody" is intended to refer to complete, intact antibodies and Fab, Fab', F(ab')2, Fv and other fragments. Complete, intact antibodies include monoclonal antibodies such as murine monoclonal antibodies, chimeric antibodies, human antibodies, and humanized antibodies.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Kako se ovdje rabi, pojam vezanje oznauje fiziku ili kemijsku interakciju izmeu dviju proteina ili spojeva ili povezanih proteina ili spojeva ili njihove kombinacije. Vezanje obuhvaa ionske, neionske, van der Waalsove, hidrofobne interakcije itd. Fizika interakcija, vezanje, moe biti izravno ili posredno. Posredno vezanje moe biti putem ili zbog uinaka drugog proteina ili spoja. Izravno vezanje odnosi se na interakcije koje se ne zbivaju putem ili zbog djelovanja drugog proteina ili spoja ve se zbiva bez stvarnih kemijskih intermedijera. </span>As used herein, the term "binding" refers to a physical or chemical interaction between two proteins or compounds or associated proteins or compounds or a combination thereof. Bonding includes ionic, non-ionic, van der Waals, hydrophobic interactions, etc. Physical interaction, bonding, can be direct or indirect. Indirect binding may be through or due to the effects of another protein or compound. Direct binding refers to interactions that do not occur through or due to the action of another protein or compound, but occur without actual chemical intermediates.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Kako se ovdje rabi, pojam spoj oznauje neku identificirajuu molekulu, ukljuujui, ali bez ogranienja, malenu molekulu, peptid, protein, ugljikohidrat, nukleotid ili nukleinsku kiselinu i takav spoj moe biti prirodni ili umjetni. </span>As used herein, the term "compound" refers to an identifying molecule, including, but not limited to, a small molecule, peptide, protein, carbohydrate, nucleotide, or nucleic acid, and such compound may be natural or artificial.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Kako se ovdje rabi, pojam komplementarni odnosi se na Watson-Crick sparivanje baza izmeu nukleotidnih jedinica molekule nukleinske kiseline. </span>As used herein, the term "complementary" refers to Watson-Crick base pairing between nucleotide units of a nucleic acid molecule.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Kako se ovdje rabi, pojam dovoenje u dodir oznauje meusobno povezivanje, bilo izravno ili posredno, spoja u fizikoj blizini prema polipeptidu ili polinukleotidu ovog izuma. Polipeptid ili polinukleotid moe biti u bilo kojem broju pufera, soli, otopina itd. Dovoenje u dodir ukljuuje, primjerice, stavljanje spoja u posudu, mikrotitarsku splou, tikvicu sa staninom kulturom ili mikroplou, kao to je genski ip ili slino, gdje se nalazi molekula nukleinske kiseline ili polipeptid koji kodira GPCR ili njegov fragment. </span>As used herein, the term "contacting" refers to the mutual linking, either directly or indirectly, of a compound in physical proximity to a polypeptide or polynucleotide of the present invention. The polypeptide or polynucleotide may be in any number of buffers, salts, solutions, etc. Contacting includes, for example, placing the compound in a dish, microtiter plate, cell culture flask, or microplate, such as a gene chip or the like, where the molecule is located. nucleic acids or a polypeptide encoding a GPCR or a fragment thereof.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Kako se ovdje rabi, fraza homologna nukleotidna sekvencija ili homologna aminokiselinska sekvencija, ili njene inaice, odnosi se na sekvencije koje su karakterizirane homologijom na nukleotidnoj razini ili aminokiselinskoj razini, do bar specificiranog postotka. Homologne nukleotidne sekvencije ukljuuju one sekvencije koje kodiraju za izoforme proteina. Takve izoforme mogu imati ekspresiju u razliitim tkivima istog organizma kao rezultat, primjerice, alternativni splicing RNA. Alternativno, izoforme mogu biti kodirane pomou razliitih gena. Homologne nukleotidne sekvencije ukljuuju nukleotidne sekvencije koje kodiraju protein specija koje nisu insekti ukljuujui, ali bez ogranienja, sisavce. Homologne nukleotidne sekvencije takoer ukljuuju, ali nisu ograniene na prirodne alelne inaice i mutacije nukleotidne sekvencije koja je ovdje definirana. Homologna nukleotidna sekvencija ne ukljuuje, meutim, nukleotidnu sekvenciju koja kodira druge poznate GPCR-ove. Homologne aminokiselinske sekvencije ukljuuju one aminokiselinske sekvencije koje kodiraju konzervativne aminokiselinske supstitucije, kao i polipeptide koji imaju aktivnost neuropeptidnog vezanja i/ili signaliziranja. Homologna aminokiselinska sekvencija ne ukljuuje, meutim, aminokiselinsku sekvenciju koja kodira druge poznate GPCR-ove. Udio homologije moe se odrediti pomou, primjerice, Gap programa (Wisconsin Sequence Analysis Package, Version 8 for Unix, Genetics Computer Group, University Research Park, Madison, WI), koritenjem podrazumijevanih postavki, koje rabe algoritam Smiths i Watermana (Adv. Appl. Math., 1981, 2, 482-489, to je ovdje ukljueno kao referenca u cijelosti). </span>As used herein, the phrase "homologous nucleotide sequence" or "homologous amino acid sequence," or variations thereof, refers to sequences that are characterized by homology at the nucleotide level or amino acid level, up to at least a specified percentage. Homologous nucleotide sequences include those sequences that code for protein isoforms. Such isoforms can be expressed in different tissues of the same organism as a result of, for example, alternative RNA splicing. Alternatively, the isoforms may be encoded by different genes. Homologous nucleotide sequences include protein-encoding nucleotide sequences of non-insect species including, but not limited to, mammals. Homologous nucleotide sequences also include, but are not limited to, naturally occurring allelic variants and mutations of a nucleotide sequence as defined herein. A homologous nucleotide sequence does not include, however, a nucleotide sequence encoding other known GPCRs. Homologous amino acid sequences include those amino acid sequences that encode conservative amino acid substitutions, as well as polypeptides that have neuropeptide binding and/or signaling activity. The homologous amino acid sequence does not include, however, the amino acid sequence encoding other known GPCRs. The proportion of homology can be determined using, for example, the Gap program (Wisconsin Sequence Analysis Package, Version 8 for Unix, Genetics Computer Group, University Research Park, Madison, WI), using default settings, which use the algorithm of Smiths and Waterman (Adv. Appl. Math., 1981, 2, 482-489, which is incorporated herein by reference in its entirety).</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Kako se ovdje rabi, pojam izdvojena molekula nukleinske kiseline odnosi se na molekulu nukleinske kiseline (DNA ili RNA) koja je uklonjena iz svoje prirodne okoline. Primjeri izdvojenih molekula nukleinske kiseline ukljuuju, ali nisu ogranieni na rekombinantne DNA molekule koje su sadrane u vektorskoj, rekombinantnoj DNA molekuli koja je zadrana u heterolognoj stanici domaina, djelomino ili potpuno proiene molekule nukleinskih kiselina te sintetske DNA ili RNA molekule. </span>As used herein, the term "isolated" nucleic acid molecule refers to a nucleic acid molecule (DNA or RNA) that has been removed from its natural environment. Examples of isolated nucleic acid molecules include, but are not limited to, recombinant DNA molecules contained in a vector, recombinant DNA molecules retained in a heterologous host cell, partially or fully purified nucleic acid molecules, and synthetic DNA or RNA molecules.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Kako se ovdje rabe, pojmovi regulira, modulira ili modificira oznauju poveanje ili smanjenje koliine, kakvoe ili uinka odreene aktivnosti ili proteina. </span>As used herein, the terms "regulate," "modulate," or "modify" refer to increasing or decreasing the amount, quality, or effect of a particular activity or protein.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Kako se ovdje rabi, pojam pojaana aktivnost oznauje poveanu aktivnost. Pojam oteena aktivnost oznauje smanjenu aktivnost. </span>As used herein, the term "enhanced activity" refers to increased activity. The term "impaired activity" means reduced activity.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Kako se ovdje rabi, pojam oligonukleotid odnosi se na niz povezanih nukleotidnih rezidua koje imaju dovoljan broj baza da se koriste u polimeraznoj lananoj reakciji (PCR). Ova kratka sekvencija temelji se (izvedena je) na genomskoj ili cDNA sekvenciji te se koristi da se pojaa, potvrdi ili pokae prisutnost identine, sline ili komplementarne DNA ili RNA u konkretnoj stanici ili tkivu. Oligonukleotidi sadre dio sekvencije DNA koja ima bar oko 10 nukleotida i bar do oko 50 nukleotida, poeljno oko 15 do 30 nukleotida. Oni se kemijski sintetiziraju i mogu se koristiti kao sonde. </span>As used herein, the term "oligonucleotide" refers to a series of linked nucleotide residues having a sufficient number of bases to be used in polymerase chain reaction (PCR). This short sequence is based on (derived from) a genomic or cDNA sequence and is used to amplify, confirm or demonstrate the presence of identical, similar or complementary DNA or RNA in a particular cell or tissue. Oligonucleotides contain a part of the DNA sequence that has at least about 10 nucleotides and at least up to about 50 nucleotides, preferably about 15 to 30 nucleotides. They are chemically synthesized and can be used as probes.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Kako se ovdje rabi, pojam sonda odnosi se na sekvenciju nukleinskih kiselina promjenjive duljine, poeljno izmeu bar oko 10 i do oko 6000 nukleotida, ovisno o upotrebi. Oni se rabe u detektiranju identinih, slinih ili komplemenatrnih sekvencija nukleinskih kiselina. Sonde vee duljine obino se dobivaju iz prirodnog ili rekombinantnog izvora, visoko su specifine i mnogo sporije hibridiziraju nego oligomeri. One mogu biti jednostruko ili dvostruko uvijene i paljivo obiljeene da bi imale specifinost u PCR, hidridizacijsko membranskim ili ELISA tehnologijama. </span>As used herein, the term "probe" refers to a nucleic acid sequence of variable length, preferably between at least about 10 and up to about 6000 nucleotides, depending on the use. They are used to detect identical, similar or complementary sequences of nucleic acids. Probes of longer length are usually derived from natural or recombinant sources, are highly specific, and hybridize much more slowly than oligomers. They can be single or double stranded and carefully labeled to have specificity in PCR, hydraidization membrane or ELISA technologies.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Dio ili fragment kada se odnosi na polinukleotid ukljuuje polinukleotidnu sekvenciju koja ima bar 14, 16, 18, 20, 25, 50 ili 75 konzekutivnih nukleotida referentnog polinukleotida iz kojega je izveden fragment ili dio. Dio ili fragment kada se odnosi na polipeptid odnosi se na polipeptid koji ima bar 5, 10, 15, 20, 25, 30, 35 ili 40 konzekutivnih aminokiselina referentnog polipeptida iz kojega se izvodi fragment. </span>"Part" or "fragment" when referring to a polynucleotide includes a polynucleotide sequence having at least 14, 16, 18, 20, 25, 50 or 75 consecutive nucleotides of the reference polynucleotide from which the fragment or part is derived. "Part" or "fragment" when referring to a polypeptide refers to a polypeptide having at least 5, 10, 15, 20, 25, 30, 35 or 40 consecutive amino acids of the reference polypeptide from which the fragment is derived.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Pojam prevencija odnosi se na smanjenje mogunosti da organizam iskae ili razvije nenormalno stanje. </span>The term "prevention" refers to the reduction of the possibility of the organism expressing or developing an abnormal condition.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Fraza kontroliranje populacije insekata ili ili njene inaice odnosi se na poveanje ili smanjenje broja insekata u populaciji. Primjerice, metode kontroliranja populacije insekata ukljuuju metode poveanja broja blagotvornih insekata u danoj populaciji insekata te metode smanjenja broja tetnih insekata u danoj populaciji insekata. </span>The phrase "insect population control" or its variants refers to increasing or decreasing the number of insects in a population. For example, methods of controlling insect populations include methods of increasing the number of beneficial insects in a given insect population and methods of reducing the number of harmful insects in a given insect population.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Pojam tretiranje odnosi se na postojanje terapijskog uinka i bar djelomino izbjegavanje ili ublaavanje nenormalnog stanja u organizmu. </span>The term "treatment" refers to the existence of a therapeutic effect and at least partial avoidance or mitigation of an abnormal state in the body.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Pojam subjekt kako se ovdje rabi odnosi se na insekte, kraljenjake, beskraljenjake i sisavce. </span>The term "subject" as used herein refers to insects, vertebrates, invertebrates and mammals.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Pojam terapijski uinak odnosi se na inhibiranje ili aktiviranje imbenika koji izazivaju ili doprinose nenormalnom ili normalnom stanju. Terapijski uinak otklanja do neke mjere jedan ili vie simptoma nenormalnog ili normalnog stanja. Terapijski uinak moe se odnositi na jedno ili vie sljedeeg: (a) poveanje proliferacije, rasta i/ili diferenciranja stanica; (b) inhibiranje (tj. usporavanje ili zaustavljanje) stanine smrti; (c) inhibiranje degeneracije; (d) otklanjanje do neke mjere jednog ili vie simptoma koji su povezani sa stanjem; i (e) pojaavanje funkcije zahvaene populacije stanica. Spojevi koji se pokazuju uinkovitima prema nenormalnim ili normalnim stanjima mogu se identificirati kao to je ovdje opisano. </span>The term "therapeutic effect" refers to the inhibition or activation of factors that cause or contribute to an abnormal or normal condition. The therapeutic effect removes to some extent one or more symptoms of an abnormal or normal condition. The therapeutic effect may relate to one or more of the following: (a) increasing cell proliferation, growth and/or differentiation; (b) inhibiting (ie, slowing or stopping) cell death; (c) inhibiting degeneration; (d) alleviating to some extent one or more symptoms associated with the condition; and (e) enhancing the function of the affected cell population. Compounds shown to be effective against abnormal or normal conditions can be identified as described herein.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Stanje organizma koje treba tretirati moe biti nenormalno ili normalno. Pojam nenormalno stanje odnosi se na funkciju u stanicama ili tkivima organizma koje odstupa od njihove normalne funkcije u tom organizmu. Primjerice, nenormalno stanje moe se odnositi na staninu proliferaciju, stanino diferenciranje, stanino signaliziranje ili stanino preivljavanje. </span>The condition of the organism to be treated can be abnormal or normal. The term "abnormal condition" refers to a function in the cells or tissues of an organism that deviates from their normal function in that organism. For example, the abnormal condition may relate to cell proliferation, cell differentiation, cell signaling, or cell survival.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Fraza normalno stanje odnosi se na normalnu funkciju u stanicama ili tkivu organizma. Primjerice, normalno stanje moe se odnositi na staninu proliferaciju, stanino diferenciranje, stanino signaliziranje ili stanino preivljavanje. </span>The phrase "normal state" refers to normal function in the cells or tissue of an organism. For example, a normal state may refer to cell proliferation, cell differentiation, cell signaling, or cell survival.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Pojam primjena odnosi se na metodu ugradnje spoja u stanice ili tkiva organizma. Stanje se moe sprijeiti, tretirati ili izazvati kada stanice ili tkiva organizma postoje unutar organizma ili izvan organizma. Stanice koje postoje izvan organizma mogu se odravati ili uzgajati u posudama sa staninom kulturom. Za stanice s utoitem unutar organizma, u tehnici postoje brojne tehnike da se primijene spojevi, ukljuujui (ali bez ogranienja) oralnu, parenteralnu, dermalnu, injekcijsku ili aerosolnu primjenu. Za stanice izvan organizma, postoje brojne tehnike za primjenu spojeva, ukljuujui (ali bez ogranienja) tehnike staninog mikroiniciranja, tehnike transformiranja i tehnike nosaa. </span>The term "administration" refers to the method of incorporating the compound into the cells or tissues of the organism. A condition can be prevented, treated or caused when cells or tissues of an organism exist inside or outside the organism. Cells that exist outside the body can be maintained or grown in cell culture dishes. For cells harboring within the organism, there are numerous techniques in the art to administer the compounds, including (but not limited to) oral, parenteral, dermal, injectable, or aerosol administration. For cells outside the organism, there are numerous techniques for administering the compounds, including (but not limited to) cellular microinitiation techniques, transformation techniques, and carrier techniques.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Stanje se moe takoer sprijeiti, tretirati ili izazvati primjenom spoja skupini stanica koje moraju modificirati puteve prijenosa signala organizma koji je subjekt. Uinak primjene spoja na funkciju organizma se zatim moe pratiti. Subjekt moe primjerice biti sisavac, kao to je mi, takor, kuni, svinja, kuna ivotinja (kao to je pas ili maka), domaa ivotinja (kao to je koko, svinja ili krava), koza, konj, majmun ili ovjek, crv ili insekt. </span>The condition can also be prevented, treated, or induced by administering the compound to a group of cells that must modify the signal transmission pathways of the subject organism. The effect of the application of the compound on the function of the organism can then be monitored. The subject can be, for example, a mammal, such as a mouse, rat, rabbit, pig, domestic animal (such as a dog or cat), a domestic animal (such as a chicken, pig or cow), a goat, a horse, a monkey or a human, a worm or an insect.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Pod pojaavanjem podrazumijeva se poveanje broja DNA ili RNA u stanici u usporedbi s normalnim stanicama. Pojaavanje kakvo jest odnosi se na RNA koja se moe detektirati u prisutnosti RNA u stanicama, budui da u nekih normalnih stanica nema bazalne ekspresije RNA. U drugim normalnim stanicama, postoji bazalna razina ekspresije, prema tome u tim sluajevima pojaavanje je detektiranje od bar 1-2-puta i poeljno vie, u usporedbi s bazalnom razinom. </span>"Amplification" means an increase in the number of DNA or RNA in a cell compared to normal cells. "Amplification" as such refers to RNA that can be detected in the presence of RNA in cells, since there is no basal expression of RNA in some normal cells. In other normal cells, there is a basal level of expression, thus in these cases the amplification is detectable at least 1-2-fold and preferably more, compared to the basal level.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Kako se ovdje rabi, fraza strogi uvjeti hibridiziranja  ili strogi uvjeti odnosi se na stanja u kojima e svaka sonda, primjer ili oligonukleotid hibridizirati prema svojoj ciljanoj sekvenciji, ali ne s ostalim sekvencijama. Strogi uvjeti su sekvencijski ovisni i bit e razliiti u razliitim okolnostima. Dulje sekvencije hibridiziraju specifino pri viim temperaturama. Openito, strogi uvjeti odabrani su da budu oko 5C nii nego to je temperatura talita (Tm) za specifinu sekvenciju pri definiranoj ionskoj jakosti i pH. Tm je temperatura (uz definiranu ionsku jakost, pH i koncentraciju nukleinske kiseline) pri kojoj 50% sondi koje su komplementarne prema ciljanoj sekvenciji hibridizira prema ciljanoj sekvenciji u ravnotei. Budui da su ciljane sekvencije obino u suviku, pri Tm, 50% sondi je zauzeto u ravnotei. Tipino, strogi uvjeti su oni u kojima je koncentracija soli manja od oko 1,0 M natrijeva iona, tipino oko 0,01 do 1,0 M natrijeva iona (ili drugih soli) pri pH 7,0 do 8,3 i temperaturi koja je bar oko 30C za kratke sonde, primere i oligonukleotide (npr. 10 do 50 nukleotida) i bar oko 60C za dulje sonde, primere ili oligonukleotide. Strogi uvjeti mogu se postii dodatkom sredstva za destabiliziranje, kao to je formamid. </span>As used herein, the phrase "stringent hybridization conditions" or "stringent conditions" refers to conditions in which each probe, primer, or oligonucleotide will hybridize to its target sequence, but not to other sequences. The strict conditions are sequence dependent and will be different in different circumstances. Longer sequences hybridize specifically at higher temperatures. Generally, the stringency conditions are chosen to be about 5C lower than the melting temperature (Tm) for a specific sequence at a defined ionic strength and pH. Tm is the temperature (with defined ionic strength, pH and nucleic acid concentration) at which 50% of probes complementary to the target sequence hybridize to the target sequence at equilibrium. Since the target sequences are usually in excess, at Tm, 50% of the probes are occupied at equilibrium. Typically, stringent conditions are those in which the salt concentration is less than about 1.0 M sodium ion, typically about 0.01 to 1.0 M sodium ion (or other salts) at pH 7.0 to 8.3 and a temperature that is at least around 30C for short probes, primers and oligonucleotides (eg 10 to 50 nucleotides) and at least around 60C for longer probes, primers or oligonucleotides. Stringent conditions can be achieved by adding a destabilizing agent, such as formamide.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Aminokiselinske sekvencije nalaze se u smjeru od amino prema karboksi, s lijeva na desno. Amino i karboksilne skupine ne nalaze se u sekvenciji. Nukleotidne sekvencije nalaze se samo u jednoj uzvojnici, u smjeru od 5 prema 3, s lijeva na desno. Nukleotidi i aminokiseline predstavljeni su na nain kao to preporuuje IUPAC-IUB Biochemical Nomenclature Commission ili (za aminokiseline) pomou koda s tri slova. </span>Amino acid sequences are arranged from amino to carboxy, from left to right. Amino and carboxyl groups are not in sequence. Nucleotide sequences are found in only one helix, in the direction from 5' to 3', from left to right. Nucleotides and amino acids are represented as recommended by the IUPAC-IUB Biochemical Nomenclature Commission or (for amino acids) using a three-letter code.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Polinukleotidi </span>Polynucleotides</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Genomska DNA ovog izuma sadri protein-kodirajue podruje za polipeptid ovog izuma i takoer je odreeno da ukljuuje njegove alelne inaice. iroko je usvojeno da se, za mnoge gene, DNA transkribira u RNA transkripte koji podlijeu jednom ili vie cijepanja pri emu se introni (nekodirajua podruja) uklanjaju ili izbacuju. RNA transkripti koje se mogu ispreplesti alternativnim mehanizmima i prema tome predmet su uklanjanja razliitih RNA sekvencija ali jo uvijek kodiraju DmGPCR polipeptid, u tehnici se oznauju kao splice inaice koje su obuhvaene ovim izumom. Splice inaice koje su obuhvaene ovim izumom prema tome prema tome su kodirane pomoi identine genomske DNA sekvencije ali koja je iz odvojenih mRNA transkripata. Alelne inaice su modificirani oblici sekvencije gena divljeg tipa, modifikacije koja je rezultat rekombinacije tijekom kromosomske segregacije ili izlaganja uvjetima koji dovode do mutiranja gena. Alelne inaice, poput gena divljeg tipa, su prirodne sekvencije (za razliku od inaica koje se ne nalaze u prirodi i rezultat su in vitro manipuliranja). </span>The genomic DNA of the present invention contains the protein-coding region for the polypeptide of the present invention and is also determined to include allelic variants thereof. It is widely accepted that, for many genes, DNA is transcribed into RNA transcripts that undergo one or more cleavages in which introns (non-coding regions) are removed or "knocked out". RNA transcripts that can be spliced by alternative mechanisms and are therefore subject to the removal of different RNA sequences but still encode the DmGPCR polypeptide, are referred to in the art as "splice versions" that are covered by this invention. The splice variants encompassed by the present invention are therefore encoded by an identical genomic DNA sequence but from separate mRNA transcripts. Allelic variants are modified forms of the wild-type gene sequence, a modification resulting from recombination during chromosomal segregation or exposure to conditions that cause the gene to mutate. Allelic variants, like wild-type genes, are naturally occurring sequences (as opposed to variants that are not found in nature and are the result of in vitro manipulation).</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Ovaj izum takoer obuhvaa cDNA koja se dobiva reverznom transkripcijom RNA polinukleotida koji kodira DmGPCR (standardno slijedi sinteza druge uzvojnice komplementarne uzvojnice da se dobije dvostruko uvijena DNA). </span>The present invention also encompasses cDNA obtained by reverse transcription of an RNA polynucleotide encoding a DmGPCR (standardly followed by synthesis of the second helix of the complementary helix to obtain double-stranded DNA).</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">DNA sekvencija koja kodira DmGPCR polipeptid pripada bilo kojoj sekvenciji iz skupa sekvencija s br. 1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21 ili 23. DNA ovog izuma moe sadravati dvostruko uvijenu molekulu zajedno s komplementarnom molekulom (nekodirajua uzvojnica ili komplement) koja ima sekvenciju koja je nesumnjivo izvedena iz kodirajue uzvojnice sukladno Watson-Crick pravilu sparivanja baza za DNA. Takoer su obuhvaeni ovim izumom drugi polinukleotidi koji kodiraju bilo koji od konkretne DmGPCR polipeptide ovog izuma koji se razlikuju u sekvenciji od konkretnog polinukleotida koji je ovdje opisan zbog dobro poznate degeneracije univerzalnog genetskog koda. </span>The DNA sequence encoding the DmGPCR polypeptide belongs to any sequence from the set of sequences with no. 1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21 or 23. The DNA of the present invention may comprise a double-stranded molecule together with a complementary molecule ("non-coding strand" or "complement") having a sequence that is undoubtedly derived from the coding coil according to the Watson-Crick base pairing rule for DNA. Also encompassed by the present invention are other polynucleotides encoding any of the particular DmGPCR polypeptides of the present invention that differ in sequence from the particular polynucleotide described herein due to the well-known degeneracy of the universal genetic code.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Ovaj izum nadalje obuhvaa specije, kao to su one sisavaca, homologe DmGPCR-a DNA. Specijski homolozi, koji se ponekad nazivaju ortolozi, openito dijele bar 35%, bar 40%, bar 45%, bar 50%, bar 60%, bar 65%, bar 70%, bar 75%, bar 80%, bar 85%, bar 90%, bar 95%, bar 98% ili bar 99% homologije s DNA ovog izuma. Openito, postotoak sekvencije homologije u odnosu na polinukleotide ovog izuma moe se izraunati kao postotak nukleotidnih baza u kandidatskoj sekvenciji koje su identine nukleotidima u DmGPCR sekvenciji koja se nalazi u odreenoj polinukleotidnoj sekvenciji, nakon usklaivanja sekvencija i uvoenja praznina, ako je neophodno, da se postigne maksimalni postotak sekvencijske identinosti. </span>The present invention further encompasses species, such as mammalian, DmGPCR DNA homologues. Species homologs, sometimes called "orthologs," generally share at least 35%, at least 40%, at least 45%, at least 50%, at least 60%, at least 65%, at least 70%, at least 75%, at least 80%, at least 85%, at least 90%, at least 95%, at least 98% or at least 99% homology with the DNA of this invention. In general, the percentage of sequence "homology" to the polynucleotides of this invention can be calculated as the percentage of nucleotide bases in a candidate sequence that are identical to nucleotides in a DmGPCR sequence found in a particular polynucleotide sequence, after aligning the sequences and introducing gaps, if necessary, to the maximum percentage of sequence identity is achieved.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Sljedei aspekt ovog izuma je koritenje nukleotidne sekvencije DmGPCR-a koja je ovdje opisana za identificiranje homologa DmGPCR-a, u drugim ivotinjama, ukljuujui sisavce, kraljenjake i beskraljenjake. Bilo koje nukleotidne sekvencije koje su ovdje opisane ili bilo koji njihov dio, mogu se primjerice koristiti kao sonde za screening baze podataka ili biblioteke nukleinskih kisleina, kao to su primjerice genomske ili cDNA biblioteke, da se identificiraju homolozi, koritenjem screening postupaka koji su dobro poznati onima koji poznaju ovo podruje. </span>A further aspect of the present invention is the use of the DmGPCR nucleotide sequence described herein to identify DmGPCR homologues in other animals, including mammals, vertebrates and invertebrates. Any of the nucleotide sequences described herein, or any portion thereof, can, for example, be used as probes to screen databases or libraries of nucleic acids, such as genomic or cDNA libraries, to identify homologues, using screening procedures well known in the art. to those who know this area.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Informacije o polinukleotidnoj sekvenciji koje se dobivaju ovim izumom omoguuju ekspresiju u irokom opsegu kodiranog polipeptida pomou tehnika koje su dobro poznate i rutinski se primjenjuju u tehnici. Polinukleotidi ovog izuma takoer omoguuju identificiranje i izdvajanje polinukleotida koji kodira srodne DmGPCR polipeptide, kao to su alelne inaice i specijski homolozi, pomou dobro poznatih tehnika ukljuujui Southern i/ili Northern hibiridiziranje i polimeraznu lananu reakciju (PCR). Primjeri srodnih polinukleotida ukljuuju genomske sekvencije, ukljuujui alelne inaice, kao i polinukleotide koji kodiraju polipeptidne homologe za DmGPCR i strukturno srodne polipeptide koji dijele jedna ili vie biolokih, imunolokih i/ili fizikih svojstava DmGPCR-a. Geni koji kodiraju proteine koji su homologni za DmGPCR takoer se mogu identificirati pomou Southern i/ili PCR analize i korisni su ivotinjskim modelima za GPCR poremeaje. Poznavanje sekvencija DNA DmGPCR-a takoer omoguuje upotrebom Southern hibridiziranja ili polimeraza lanane reakcije (PCR) identificiranje genomskih DNA sekvencija koje kodiraju DmGPCR ekspresijske kontrolne regulacijske sekvencije kao to su promotori, operatori, pojaivai, represori i slino. Polinukleotidi ovog izuma su takoer korisni u analizama hibridiziranja da se odredi kapacitet stanica za ekspresiju DmGPCR. Polinukleotidi ovog izuma mogu takoer biti osnovica za dijagnostike metode koje su korisne za identificiranje prisutnosti ektoparazitne ekspresije DmGPCR emu podlijee stanje ili stanja bolesti, a ta informacija je korisna kao za dijagnozi tako i za odabiranje terapijske strategije. </span>The polynucleotide sequence information provided by the present invention enables broad-scale expression of the encoded polypeptide using techniques well known and routinely practiced in the art. The polynucleotides of the present invention also enable the identification and isolation of polynucleotides encoding related DmGPCR polypeptides, such as allelic variants and species homologues, using well-known techniques including 'Southern' and/or 'Northern' hybridization and polymerase chain reaction (PCR). Examples of related polynucleotides include genomic sequences, including allelic variants, as well as polynucleotides encoding polypeptide homologues for DmGPCRs and structurally related polypeptides that share one or more biological, immunological, and/or physical properties of DmGPCRs. Genes encoding proteins that are homologous to DmGPCRs can also be identified by Southern and/or PCR analysis and are useful in animal models for GPCR disorders. Knowing the DNA sequences of DmGPCR also enables the use of 'Southern' hybridization or polymerase chain reaction (PCR) to identify genomic DNA sequences that encode DmGPCR expression control regulatory sequences such as promoters, operators, enhancers, repressors and the like. Polynucleotides of the present invention are also useful in hybridization assays to determine the capacity of cells to express DmGPCR. The polynucleotides of this invention may also be the basis for diagnostic methods useful for identifying the presence of ectoparasitic DmGPCR expression underlying a disease state or states, and this information is useful both for diagnosis and for selecting a therapeutic strategy.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Ovdje dani opis polinukleotida pune duljine koji kodira DmGPCR polipeptid omoguuje dobivanje za onoga koji poznaje ovo podruje tehnike svakog mogueg formata polinukleotida pune duljine. Ovaj izum prema tome definira fragmente DmGPCR-a-koji kodiraju polinukleotide koji se sastoje od bar 14 i poeljno bar 16, 18, 20, 25, 50 ili 75 konzekutivnih nukleotida polinukleotida koji kodiraju DmGPCR. Fragment polinukleotida ovog izuma moe sadravati sekvencije koje su jedinstvene za DmGPCR-kodirajuu polinukleotidnu sekvenciju i prema tome hibridizira u vrlo strogim ili umjereno strogim uvjetima samo (tj. specifino) za polinukleotide koji kodiraju DmGPCR (ili njegove fragmente). Polinukleotidni fragmenti genomskih sekvencija ovog izuma ne sadre samo sekvencije koje su jedinstvene za kodirajue podruje, ve takoer obuhvaaju fragmente sekvencija pune duljine izvedenih iz introna, regulacijskih podruja i/ili drugih neprevedenih sekvencija. Sekvencije koje su jedinstvene za polinukleotide ovog izuma mogu se prepoznati pomou usporeivanja sekvencija prema drugim poznatim nukleotidima i mogu se identificirati pomou programa za usklaivanje koji se rutinski koriste u tehnici, npr. oni koji postoje u javnoj bazi podataka o sekvencijama. Takve sekvencije mogu se takoer prepoznati pomou Southern analiza hibridiziranja da se odredi broj fragmenata genomske DNA za koju e hibridizirati polinukleotid. Polinukleotidi ovog izuma mogu se obiljeiti na nain koji omoguuje njihovo detektiranje, ukljuujui radioaktivno, fluorescentno i enzimsko obiljeavanje. </span>The description given herein of a full-length polynucleotide encoding a DmGPCR polypeptide enables one skilled in the art to obtain any possible full-length polynucleotide format. The present invention therefore defines fragments of DmGPCR-encoding polynucleotides consisting of at least 14 and preferably at least 16, 18, 20, 25, 50 or 75 consecutive nucleotides of a DmGPCR-encoding polynucleotide. A polynucleotide fragment of the present invention may contain sequences that are unique to the DmGPCR-encoding polynucleotide sequence and thus hybridize under highly or moderately stringent conditions only (ie, "specific") to DmGPCR-encoding polynucleotides (or fragments thereof). Polynucleotide fragments of the genomic sequences of the present invention not only contain sequences that are unique to the coding region, but also include fragments of full-length sequences derived from introns, regulatory regions, and/or other untranslated sequences. Sequences that are unique to the polynucleotides of the present invention can be identified by sequence comparison to other known nucleotides and can be identified using alignment programs routinely used in the art, eg, those that exist in public sequence databases. Such sequences can also be identified using 'Southern' hybridization analyzes to determine the number of genomic DNA fragments to which the polynucleotide will hybridize. The polynucleotides of the present invention can be labeled in a manner that allows them to be detected, including radioactive, fluorescent, and enzymatic labeling.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Fragmentni polinukleotidi su osobito korisni kao sonde za detektiranje polinukleotida DmGPCR pune duljine ili fragmenta. Jedan ili vie polinukleotida moe biti sadrano u priborima koji se koriste za detektiranje prisutnosti polinukleotida koji kodira DmGPCR ili se mogu koristiti za detektiranje varijacija u polinukleotidnoj sekvenciji koja kodira DmGPCR. </span>Fragment polynucleotides are particularly useful as probes for detecting full-length or fragmented DmGPCR polynucleotides. One or more polynucleotides may be contained in kits used to detect the presence of a polynucleotide encoding a DmGPCR or may be used to detect variations in a polynucleotide sequence encoding a DmGPCR.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Ovaj izum takoer obuhvaa DNA koje kodiraju DmGPCR polipeptide koje hibridiziraju u umjereno strogim ili osobito strogim uvjetima prema nekodirajuoj zavojnici ili komplementu, polinukleoitida iz skupa sekvencija koji se sastoji od sekvencija s br. 1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21 ili 23. </span>The present invention also encompasses DNAs encoding DmGPCR polypeptides that hybridize under moderately stringent or particularly stringent conditions to the noncoding helix or complement of a polynucleotide from a set of sequences consisting of sequences with no. 1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21 or 23.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Primjeri osobito strogih uvjeta hibridiziranja su sljedei: hibridiziranje na 42C u otopini za hibridiziranje koja sadri 50% formamid, 1% SDS, 1 M NaCl, 10% dekstran sulfat i ispiranje dva puta po 30 minuta na 60C u otopini za ispiranje koja sadri 0,1SSC i 1% SDS. U tehnici se podrazumijeva da se uvjeti identine strogosti mogu postii variranjem temperature i pufera ili koncentracije soli kao to je opisano u: Ausubel et al. (Eds.), Protocols in Molecular Biology, John Wiley &amp; Sons, 1994, str. 6.0.3-6.4.10. Modifikacije i uvjeti hibridiziranja mogu se iskustveno odrediti i tono izraunati temeljem duljine i postotoka guanozin/citozin (GC) sparivanja baza sonde. Uvjeti hibiridiziranja mogu se izraunati sukladno postupku koji je opisan u: Sambrook et al. (Eds.), Molecular Cloning: Laboratory Manual, Cold Spring Harbor Laboratory Press: Cold Spring Harbor, New York, 1989, str. 9.47-9.51. </span>Examples of particularly stringent hybridization conditions are as follows: hybridization at 42C in hybridization solution containing 50% formamide, 1% SDS, 1 M NaCl, 10% dextran sulfate and washing twice for 30 minutes at 60C in wash solution containing 0.1SSC and 1% SDS. It is understood in the art that conditions of identical stringency can be achieved by varying the temperature and buffer or salt concentration as described in: Ausubel et al. (Eds.), Protocols in Molecular Biology, John Wiley &amp; Sons, 1994, p. 6.0.3-6.4.10. Modifications and hybridization conditions can be empirically determined and accurately calculated based on the length and percentage of guanosine/cytosine (GC) pairing of probe bases. Hybridization conditions can be calculated according to the procedure described in: Sambrook et al. (Eds.), Molecular Cloning: Laboratory Manual, Cold Spring Harbor Laboratory Press: Cold Spring Harbor, New York, 1989, p. 9.47-9.51.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Poznavanjem informacije o nukleotidnoj sekvenciji ovog izuma, onaj tko poznaje ovo podruje tehnike moe identificirati i dobiti nukleotidne sekvencije koje kodiraju DmGPCR-ove iz razliitih izvora (tj. razliita tkiva ili razliiti organizmi) na mnogo naina to je poznato onima koji poznaju ovo podruje tehnike i to je primjerice opisano u: Sambrook et al., Molecular cloning: laboratory manual, Second Edition, Cold Spring Harbor Press, Cold Spring Harbor, NY, 1989, to je ovdje ukljueno kao referenca u cijelosti. </span>Knowing the nucleotide sequence information of the present invention, one skilled in the art can identify and obtain nucleotide sequences encoding DmGPCRs from different sources (ie, different tissues or different organisms) in a variety of ways known to those skilled in the art and as described, for example, in: Sambrook et al., Molecular cloning: laboratory manual, Second Edition, Cold Spring Harbor Press, Cold Spring Harbor, NY, 1989, which is incorporated herein by reference in its entirety.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Primjerice, DNA koji kodira DmGPCR moe se dobiti screeningom mRNA, cDNA ili genomske DNA s oligonukleotidnom sondom koja je nastala iz informacije DmGPCR genske sekvencije koja je ovdje definirana. Sonde mogu biti obiljeene s detektirajuom skupinom, kao to je fluorescentna skupina, radioaktivni atom ili kemiluminescentna skupina sukladno s postupcima koji su poznati onima koji poznaju ovo podruje tehnike i koriste se u standardnim analizama hibridizacije, to su primjerice opisali Samrook i suradnici. </span>For example, DNA encoding a DmGPCR can be obtained by screening mRNA, cDNA, or genomic DNA with an oligonucleotide probe generated from the DmGPCR gene sequence information defined herein. Probes can be labeled with a detectable group, such as a fluorescent group, a radioactive atom, or a chemiluminescent group according to procedures known to those skilled in the art and used in standard hybridization assays, as described for example by Samrook et al.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Molekula nukleinske kiseline koja sadri bilo koju od prije opisanih nukleotidnih sekvencija DmGPCR-a moe se alternativno sintetizirati upotrebom postupka polimeraza lanane reakcije (PCR), pri emu su oligonukleotidni primeri proizvedeni iz nukleotidne sekvencije koja je ovdje definirana. Vidi ameriki patent br. 4,683,195 koji pripada Mullis-u i suradnicima te 4,683,202 koji pripada Mullis-u. PCR reakcija definira metodu za selektivno poveanje koncentracije konkretne sekvencije nukleinske kiseline ak i kada ta sekvencija nije prethodno proiena i prisutna je samo u jednom primjerku u konkretnom uzorku. Metoda se moe koristiti za pojaavanje bilo jednostruko ili dvostruko uvijene DNA. Sutina metode obuhvaa koritenje dviju oligonukleotidnih sondi da slue kao primeri za ablonski ovisno, polimeraza-upravljano repliciranje eljene molekule nukleinske kiseline. </span>A nucleic acid molecule comprising any of the previously described DmGPCR nucleotide sequences can alternatively be synthesized using the polymerase chain reaction (PCR) method, wherein oligonucleotide primers are produced from the nucleotide sequence defined herein. See US Pat. No. 4,683,195 belonging to Mullis and associates and 4,683,202 belonging to Mullis. The PCR reaction defines a method for selectively increasing the concentration of a particular nucleic acid sequence even when that sequence has not been previously purified and is present in only one copy in a particular sample. The method can be used to amplify either single or double stranded DNA. The essence of the method involves the use of two oligonucleotide probes to serve as primers for template-dependent, polymerase-directed replication of the desired nucleic acid molecule.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Cijeli niz alternativnih kloniranja i in vitro metodologija pojaavanja je poznato onima koji se bave ovim podrujem tehnike. Primjeri ovih tehnika mogu se nai primjerice u: Berger et al., Guide to Molecular Cloning Techniques, Methods in Enzymology 152 Academic Press, Inc., San Diego, CA (Berger), to je ovdje ukljueno kao referenca u cijelosti. </span>A variety of alternative cloning and in vitro amplification methodologies are known to those skilled in the art. Examples of these techniques can be found, for example, in: Berger et al., Guide to Molecular Cloning Techniques, Methods in Enzymology 152 Academic Press, Inc., San Diego, CA (Berger), which is incorporated herein by reference in its entirety.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Molekule nukleinskih kiselina ovog izuma i fragmenti koji su izvedeni iz njih korisni su za screening restrikcijskih fragmenata dugih polimorfizama (RFLP-ova) i za genetsko mapiranje. </span>The nucleic acid molecules of the present invention and fragments derived therefrom are useful for screening of restriction fragment length polymorphisms (RFLPs) and for genetic mapping.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Metode automatskog sekvenciranja mogu se koristiti da se dobije ili provjeri nukleotidna sekvencija DmGPCR-a. DmGPCR nukleotidne sekvencije ovog izuma vjeruje se da su 100% tone. Meutim, kao to je poznato u tehnici, nukleotidne sekvencije koje su dobivene automatskim metodama mogu sadravati neke pogreke. Nukleotidne sekvencije koje su automatski odreene tipino su bar oko 90%, tipinije bar oko 95% do bar oko 99,9% identine sa stvarnom nukleotidnom sekvencijom dane molekule nukleinske kiseline. Tona sekvencija moe se tonije odrediti koritenjem runih metoda sekvenciranja, koje su dobro poznate u tehnici. Pogreka u sekvenciji koja rezultira ubacivanjem ili brisanjem jednog ili vie nukleotida moe rezultirati pomakom okvira u translaciji tako da e se predviena aminokiselinska sekvencija razlikovati od one koja bi bila predviena prema stvarnoj nukleotidnoj sekvenciji molekule nukleinske kiseline, poevi od toke mutiranja. </span>Automated sequencing methods can be used to obtain or verify the nucleotide sequence of a DmGPCR. The dmGPCR nucleotide sequences of the present invention are believed to be 100% correct. However, as known in the art, nucleotide sequences obtained by automated methods may contain some errors. Nucleotide sequences that are automatically determined are typically at least about 90%, more typically at least about 95% to at least about 99.9% identical to the actual nucleotide sequence of a given nucleic acid molecule. The exact sequence can be more accurately determined using manual sequencing methods, which are well known in the art. An error in the sequence resulting in the insertion or deletion of one or more nucleotides can result in a translational frameshift such that the predicted amino acid sequence will differ from that which would be predicted from the actual nucleotide sequence of the nucleic acid molecule, starting from the point of mutation.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Ekspresijski konstrukti i vektori </span>Expression constructs and vectors</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Autonomno replicirajui rekombinantni ekspresijski konstrukti kao to su plazmidni i virusni DNA vektori koji sadre polinukleotide ovog izuma su takoer definirani. Vektori se ovdje koriste bilo za pojaanje DNA ili RNA koja kodira DmGPCR i/ili za ekspresiju DNA koja kodira DmGPCR. Vektori ovog izuma ukljuuju, ali nisu ogranieni na plazmide, fage, kozmide, episome, virusne estice, viruse i integratabilne DNA fragmente (tj. fragmente koji se mogu integrirati u genom domaina pomou homologne rekombinacije). Virusne estice mogu ukljuivati, ali nisu ogranieni na adenoviruse, bakuloviruse, parvoviruse, herpesviruse, poksviruse, adeno-pridruene viruse, Smeliki Forest viruse, vaccinia viruse i retroviruse. Primjeri ekspresijskih vektora ukljuuju, ali nisu ogranieni na pcDNA3 (Invitrogen) i pSVL (Pharmacia Biotech). Drugi ekspresijski vektori ukljuuju, ali nisu ogranieni na pSPORT vektore, pGEM vektore (Promega), pPROEXvektore (LTI, Bethesda, MD), Bluescript vektore (Stratagene), pQE vektore (Qiagen), pSE420 (Invitrogen) i pYES2 (Invitrogen). </span>Autonomously replicating recombinant expression constructs such as plasmid and viral DNA vectors containing the polynucleotides of the invention are also defined. Vectors are used herein either to amplify DNA or RNA encoding DmGPCR and/or to express DNA encoding DmGPCR. Vectors of the invention include, but are not limited to, plasmids, phages, cosmids, episomes, viral particles, viruses, and integrable DNA fragments (ie, fragments that can be integrated into the host genome by homologous recombination). Viral particles may include, but are not limited to, adenoviruses, baculoviruses, parvoviruses, herpesviruses, poxviruses, adeno-associated viruses, Smeliki Forest viruses, vaccinia viruses, and retroviruses. Examples of expression vectors include, but are not limited to, pcDNA3 (Invitrogen) and pSVL (Pharmacia Biotech). Other expression vectors include, but are not limited to, pSPORT vectors, pGEM vectors (Promega), pPROEXvectors (LTI, Bethesda, MD), Bluescript vectors (Stratagene), pQE vectors (Qiagen), pSE420 (Invitrogen), and pYES2 (Invitrogen).</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Ekspresijski konstrukti pri emu su DmGPCR-kodirajui polinukleotidi operativno povezani su za endogeni ili egzogeni ekspresijski kontrolni DNA sekvencijski i transkripcijski terminator su takoer ovdje definirani. Ekspresijske kontrolne DNA sekvencije openito ukljuuju promotore, pojaivae, operatore i regulacijska elementna mjesta i tipino su odabrane temeljem ekspresijskih sustava u kojima se koristi ekspresijski konstrukt. Promotorske i pojaivake sekvencije openito su odabrane glede sposobnosti poveanja ekspresije gena, dok su operatorske sekvencije openito odabrane glede sposobnosti reguliranja ekspresije gena. Ekspresijski konstrukti ovog izuma mogu takoer ukljuivati sekvencije koje kodiraju jedan ili vie selektibilnih obiljeivaa koji dozvoljavaju identificiranje stanice domaina koja nosu konstrukt. Ekspresijski konstrukti mogu takoer ukljuivati sekvencije koje olakavaju i/ili unaprjeuju homolognu rekombinaciju u stanici domaina. Konstrukti ovog izuma mogu takoer ukljuivati sekvencije koje su neophodne za repliciranje u stanici domaina. </span>Expression constructs wherein the DmGPCR-encoding polynucleotides are operably linked to an endogenous or exogenous expression control DNA sequence and a transcriptional terminator are also defined herein. Expression control DNA sequences generally include promoters, enhancers, operators and regulatory element sites and are typically selected based on the expression systems in which the expression construct is used. Promoter and enhancer sequences are generally selected for their ability to increase gene expression, while operator sequences are generally selected for their ability to regulate gene expression. Expression constructs of the present invention may also include sequences encoding one or more selectable markers that allow identification of the host cell carrying the construct. Expression constructs may also include sequences that facilitate and/or enhance homologous recombination in the host cell. Constructs of the present invention may also include sequences that are necessary for replication in the host cell.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Ekspresijski konstrukti mogu se rabiti za proizvodnju kodiranog proteina, ali se takoer mogu koristiti jednostavno za pojaavanje DmGPCR-kodirajue polinukleotidne sekvencije. U nekim realizacijama, vektor je ekspresijski vektor pri emu je polinukleotid ovog izuma operativno vezan za polinukleotid koji sadri ekspresijsku kontrolnu sekvenciju. Neki ekspresijski vektori su replicirajui DNA konstrukti u kojima je DNA sekvencija koja kodira DmGPCR operativno vezana ili povezana s odgovarajuom kontrolnom sekvencijom koja moe potaknuti ekspresiju DmGPCR-a u prikladnom domainu. DNA podruja su operativno vezana ili povezana kada su meu njima postoji funkcionalna srodnost. Primjerice, promotor je operativno vezan ili povezan s kodirajuom sekvencijom ako ona kontrolira transkripciju sekvencije. Pojaivaki vektori ne zahtijevaju ekspresijske kontrolne domene ve im je potrebna samo sposobnost da repliciraju u domaina, to je obino preneseno izvorom repliciranja i odabirom gena da se olaka prepoznavanje transformanta. Potreba za kontrolnim sekvencijama u ekspresijskom vektoru e varirati ovisno o odabranom domainu i odabranoj metodi transformiranja. Openito, kontrolne sekvencije ukljuuju transkripcijski promotor, proizvoljnu operatorsku sekvenciju za kontrolnu transkripciju, sekvenciju koja kodira ribosomsko vezanje odgovarajuu mRNA i sekvencije koje kontroliraju terminaciju transkripcije i prevoenja. </span>The expression constructs can be used to produce the encoded protein, but can also be used simply to amplify the DmGPCR-encoding polynucleotide sequence. In some embodiments, the vector is an expression vector wherein a polynucleotide of the present invention is operably linked to a polynucleotide containing an expression control sequence. Some expression vectors are replicative DNA constructs in which the DNA sequence encoding the DmGPCR is operably linked or linked to a suitable control sequence capable of driving expression of the DmGPCR in a suitable host. DNA regions are operatively linked or connected when there is a functional affinity between them. For example, a promoter is operably linked or linked to a coding sequence if it controls the transcription of the sequence. Enhancer vectors do not require expression control domains but only need the ability to replicate in the host, which is usually imparted by the source of replication and gene selection to facilitate transformant recognition. The need for control sequences in the expression vector will vary depending on the host chosen and the transformation method chosen. In general, control sequences include a transcriptional promoter, an arbitrary operator sequence for controlling transcription, a sequence that encodes ribosomal binding of the corresponding mRNA, and sequences that control the termination of transcription and translation.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Vektori mogu sadravati promotor koji je prepoznat od organizma domaina. Promotorske sekvencije ovog izuma mogu biti prokariotske, eukariotske ili virusne. Primjeri pogodnih prokariotskih sekvencija ukljuuju PR i PL promotore lambda bakteriofaga (The bacteriophage Lambda, Hershey, A.D., Ed., Cold Spring Harbor Press, Cold Spring Harbor, NY, 1973, to je ovdje ukljueno kao referenca u cijelosti; Lambda II, Hendrix, R.W., Ed., Cold Spring Harbor Press, Cold Spring Harbor, NY, 1980, to je ovdje ukljueno kao referenca u cijelosti); the trp, recA, promotore toplinskog oka i lacZ promotore E. coli i SV40 rani promotor (Benoist et al., Nature, 1981,290, 304-310, to je ovdje ukljueno kao referenca u cijelosti). Daljnji promotori ukljuuju, ali nisu ogranieni na tumorski virus mije dojke, dugo terminalno ponavljanje humanog imunodeficijentnog virusa (HW), maloney virus, trenutani citomegalovirusni rani promotor, Epstein Barr virus, Rous sarkom virus, humani aktin, human miozin, humani hemoglobin, humani miini kreatin i humani metalotionein. </span>Vectors may contain a promoter that is recognized by the host organism. The promoter sequences of the present invention may be prokaryotic, eukaryotic or viral. Examples of suitable prokaryotic sequences include the PR and PL promoters of bacteriophage lambda (The bacteriophage Lambda, Hershey, A.D., Ed., Cold Spring Harbor Press, Cold Spring Harbor, NY, 1973, which is incorporated herein by reference in its entirety; Lambda II, Hendrix, R.W., Ed., Cold Spring Harbor Press, Cold Spring Harbor, NY, 1980, which is hereby incorporated by reference in its entirety); the trp, recA, heat shock and lacZ promoters of E. coli and the SV40 early promoter (Benoist et al., Nature, 1981, 290, 304-310, which is incorporated herein by reference in its entirety). Further promoters include, but are not limited to, murine mammary tumor virus, human immunodeficiency virus (HW) long terminal repeat, Maloney virus, current cytomegalovirus early promoter, Epstein Barr virus, Rous sarcoma virus, human actin, human myosin, human hemoglobin, human muscle creatine and human metallothionein.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Dodatne regulacijske sekvencije mogu takoer biti ukljuene u vektorima ovog izuma. Primjeri pogodnih regulacijskih sekvencija ukljuuju, primjerice, Shine-Dalgarno sekvenciju replikaznog gena faga MS-2 i gen cII bakteriofaga lambda. Nakon Shine-Dalgarno sekvencije moe neposredno slijediti DNA koja kodira DmGPCR, to rezultira ekspresijom zrelog DmGPCR proteina. </span>Additional regulatory sequences may also be included in the vectors of the present invention. Examples of suitable regulatory sequences include, for example, the Shine-Dalgarno sequence of the phage MS-2 replicase gene and the cII gene of bacteriophage lambda. The Shine-Dalgarno sequence can be immediately followed by the DNA encoding the DmGPCR, resulting in the expression of the mature DmGPCR protein.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">tovie, pogodni ekspresijski vektori mogu ukljuivati odgovarajui obiljeiva koji omoguuje screening transformirane stanice domaina. Transformiranje odabranog domaina vri se pomou bilo koje od razliitih tehnika koje su dobro poznate onima koji poznaju ovo podruje i opisao ih je primjerice Sambrook s autorima, supra. </span>Moreover, suitable expression vectors may include a suitable marker that allows screening of the transformed host cell. Transformation of the selected host is performed using any of a variety of techniques well known to those skilled in the art and described, for example, by Sambrook et al., supra.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Izvor replikacije moe se takoer postii bilo konstrukcijom vektora tako da se ukljui egzogeni izvor ili pomou mehanizma kromosomske replikacije stanice domaina. Ako je vektor integriran u kromosom stanice domaina, ovo posljednje moe biti dovoljno. Alternativno, umjesto da koristi vektore koji sadre virusne izvore replikacije, onaj tko poznaje ovo podruje moe transformirati stanica sisavca pomou metode kotransformiranja s odgovarajuim obiljeivaem i DmGPCR DNA. Primjer pogodnog obiljeivaa je dihidrofolat reduktaza (DHFR) ili timidin kinaza (npr. ameriki patent br. 4,399,216). </span>The source of replication can also be achieved either by constructing the vector to include an exogenous source or by the chromosomal replication machinery of the host cell. If the vector is integrated into the chromosome of the host cell, the latter may be sufficient. Alternatively, instead of using vectors containing viral sources of replication, one skilled in the art can transform mammalian cells using the cotransformation method with the appropriate marker and DmGPCR DNA. An example of a suitable label is dihydrofolate reductase (DHFR) or thymidine kinase (eg, US Patent No. 4,399,216).</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Nukleotidne sekvencije koje kodiraju DmGPCR mogu biti rekombinirane s vektorskom DNA sukladno s konvencionalnim tehnikama, ukljuujui blunt ili staggered (tupe i stepenaste) zavretke za ligaciju, razgradnju restrikcijskog enzima da se dobiju odgovarajui zavreci, ispunjavanje kohezivnih zavretaka na odgovarajui nain, tretiranje alkalnom fosfatazom da se izbjegne neeljeno spajanje i ligacija s odgovarajuim ligazama. Tehnike za takvo manipuliranje opisane su u: Sambrook et al., supra i dobro su poznate u tehnici. Metode konstrukcije ekspresijskih vektora sisavaca opisane su u, primjerice, Okayama et al., Mol. Cell. Biol., 1983,3, 280; Cosman et al., Mol. Immunol. 1986,23,935; Cosman et al., Nature, 1984, 312, 768; EP-A-0367566 i WO 91/18982, a svaki je ovdje ukljuen kao referenca u cijelosti. </span>Nucleotide sequences encoding DmGPCRs can be recombined with vector DNA according to conventional techniques, including blunt or staggered end ligation, restriction enzyme digestion to obtain appropriate ends, filling in cohesive ends appropriately, treatment with alkaline phosphatase to avoids unwanted joining and ligation with appropriate ligases. Techniques for such manipulation are described in: Sambrook et al., supra and are well known in the art. Methods for the construction of mammalian expression vectors are described in, for example, Okayama et al., Mol. Cell. Biol., 1983, 3, 280; Cosman et al., Mol. Immunol. 1986, 23,935; Cosman et al., Nature, 1984, 312, 768; EP-A-0367566 and WO 91/18982, each of which is incorporated herein by reference in its entirety.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Stanice domaina </span>Host cells</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Sukladno sljedeem aspektu ovog izuma, definirane su stanice domaina, ukljuujui prokariotske i eukariotske stanice, to obuhvaa polinukleotid ovog izuma (ili vektor ovog izuma) na nain koji omoguuje ekspresiju kodiranog DmGPCR polipeptida. Polinukleotidi ovog izuma mogu biti uneseni u stanicu domaina kao dio cirkularnog plazmida ili kao linearna DNA koja sadri izdvojeno protein-kodirajue podruje ili virusni vektor. Metode za unoenje DNA u stanicu domaima koje su dobro poznate i rutinski se rabe u ukljuuju transformiranje, transfekciju, electroporiranje, injiciranje u jezgu ili fuziju s nosaima kao to su liposomi, micele, ghost stanice domaina i protoplasti. Ekspresijski sustavi ovog izuma ukljuuju bakterijske, kvaeve, gljivine, biljne, insektne, beskraljenike, kraljenike sustave i stanine sustave sisavaca. </span>According to a further aspect of the present invention, host cells, including prokaryotic and eukaryotic cells, are defined which comprise a polynucleotide of the present invention (or a vector of the present invention) in a manner that enables expression of the encoded DmGPCR polypeptide. The polynucleotides of the present invention can be introduced into a host cell as part of a circular plasmid or as linear DNA containing an isolated protein-coding region or a viral vector. Methods for introducing DNA into a host cell that are well known and routinely used include transformation, transfection, electroporation, nuclear injection, or fusion with carriers such as liposomes, micelles, host ghost cells, and protoplasts. Expression systems of the present invention include bacterial, yeast, fungal, plant, insect, invertebrate, vertebrate and mammalian cell systems.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Ovaj izum definira stanice domaina koje su transformirane ili transficirane (postojano ili prijelazno) s polinukleotidima ovog izuma ili vektorima ovog izuma. Kao to je prije navedeno, takve stanice domaina korisne su za pojaavanje polinukleotida i takoer za ekspresiju DmGPCR polipeptida ili njegova fragmenta koji je kodiran polinukleotidom. </span>The present invention defines host cells that have been transformed or transfected (permanently or transiently) with polynucleotides of the present invention or vectors of the present invention. As previously noted, such host cells are useful for amplifying the polynucleotide and also for expressing the DmGPCR polypeptide or fragment thereof encoded by the polynucleotide.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Sukladno nekim aspektima ovog izuma, definirane su transformirane stanice domaina koje imaju ekspresijski vektor koji sadri bilo koju molekulu nukleinske kiseline to je prije opisano. Ekspresija nukleotidne sekvencije zbiva se kada se ekspresijski vektor unese u odgovarajuu stanicu domaina. Pogodne stanice domaina za ekspresiju polipeptida ovog izuma ukljuuju, ali nisu ograniene na prokariote, kvasce i eukariote. Ako se rabi prokariotska ekspresija, onda odgovarajua stanica domaina moe biti neka prokariotska stanica koja moe vriti ekspresiju kloniranih sekvencija. Pogodne prokariotske stanice ukljuuju, ali bez ogranienja, bakterije roda Escherichia, Bacillus, Salmonella, Pseudomonas, Streptomyces i Staphylococcus. </span>In accordance with some aspects of the present invention, transformed host cells having an expression vector comprising any of the nucleic acid molecules described above are defined. Expression of the nucleotide sequence occurs when the expression vector is introduced into the appropriate host cell. Suitable host cells for expressing polypeptides of the present invention include, but are not limited to, prokaryotes, yeasts, and eukaryotes. If prokaryotic expression is used, then the appropriate host cell may be a prokaryotic cell capable of expressing the cloned sequences. Suitable prokaryotic cells include, but are not limited to, bacteria of the genera Escherichia, Bacillus, Salmonella, Pseudomonas, Streptomyces, and Staphylococcus.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Ako se koristi eukariotski ekspresijski vektor, onda bi odgovarajua stanica domaina trebala biti eukariotska stanica koja moe vriti ekspresiju klonirane sekvencije. Eukariotske stanice mogu biti stanice viih eukariota. Pogodne eukariotske stanice ukljuuju, ali nisu ograniene na kulture ne-humanih stanica tkiva dojke i kulture stanica humanog tkiva. Stanice domaina mogu ukljuivati, ali nisu ograniene na stanice insekata, HeLa stanice, stanice jajnika kineskog zamorca (CHO stanice), stanice bubrega afrikog zelenog majmuma (COS cells), humane 293 stanice i mije 3T3 fibroblaste. Rast takvih stanica u staninoj kulturi moe postati rutinski postupak (vidi npr. Tissue Culture, Academic Press, Kruse and Patterson, eds., 1973, to je ovdje ukljueno kao referenca u cijelosti). </span>If a eukaryotic expression vector is used, then the appropriate host cell should be a eukaryotic cell capable of expressing the cloned sequence. Eukaryotic cells can be cells of higher eukaryotes. Suitable eukaryotic cells include, but are not limited to, non-human breast tissue cell cultures and human tissue cell cultures. Host cells may include, but are not limited to, insect cells, HeLa cells, Chinese guinea pig ovary cells (CHO cells), African green monkey kidney cells (COS cells), human 293 cells, and mouse 3T3 fibroblasts. Growth of such cells in cell culture may become a routine procedure (see, e.g., Tissue Culture, Academic Press, Kruse and Patterson, eds., 1973, which is incorporated herein by reference in its entirety).</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Nadalje, kvaev domain moe se koristiti kao stanica domain. Primjeri kvaevih stanica ukljuuju, ali nisu ogranieni na rodove Saccharomyces, Pichia i Kluveromyces. Primjeri kvaenih domaina su S. cerevisiae i P. pastoris. Kvaevi vektori mogu sadravati izvor replikacijske sekvencije iz 2T kvaeva plazmida, autonomne replicirajue sekvencije (ARS), promotorsko podruje, sekvencije za poliadenilaciju, sekvencije za transkripcijsko terminiranje i selektibilni gen obiljeiva. Ovdje su takoer ukljueni shuttle vektori za repliciranje u kvascu i E. coli. </span>Furthermore, a yeast host can be used as a host cell. Examples of yeast cells include, but are not limited to, the genera Saccharomyces, Pichia, and Kluveromyces. Examples of yeast hosts are S. cerevisiae and P. pastoris. Yeast vectors can contain a source of replication sequence from a 2T yeast plasmid, an autonomously replicating sequence (ARS), a promoter region, polyadenylation sequences, transcription termination sequences, and a selectable marker gene. Shuttle vectors for replication in yeast and E. coli are also included here.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Alternativno, insektne stanice mogu se koristiti kao stanice domaina. U jednoj realizaciji, polipeptidi ovog izuma imaju ekspresiju koritenjem bakulovirusnog ekspresijskog sustava (vidi Luckow et al., Bio/Technology, 1988, 6, 47, Baculovirus Expression Vektors: Laboratory Manual, ORielly et al. (Eds.), W.H. Freeman and Company, New York, 1992 i ameriki patent br. 4,879,236, a svi su ovdje ukljueni kao referenca u cijelosti). Nadalje, MAXBAC ekspresijski sustav cjelovitog bakulovirusa (Invitrogen) moe se, primjerice, koristiti za proizvodnju u stanicama insekata. </span>Alternatively, insect cells can be used as host cells. In one embodiment, the polypeptides of the invention are expressed using a baculovirus expression system (see Luckow et al., Bio/Technology, 1988, 6, 47, Baculovirus Expression Vectors: Laboratory Manual, O'Rielly et al. (Eds.), W.H. Freeman and Company, New York, 1992 and US Patent No. 4,879,236, all of which are incorporated herein by reference in their entirety). Furthermore, the MAXBAC whole baculovirus expression system (Invitrogen) can be used, for example, for production in insect cells.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">U sljedeoj srodnoj realizaciji, izumom su definirane metode za dobivanje DmGPCR polipeptida (ili njegova fragmenta) to ukljuuje stupnjeve rasta stanica domaina ovog izuma u hranjivom mediju i izdvajanje polipeptida ili njegove inaice iz stanice ili iz medija. Budui da je DmGPCR sedam transmembranski receptor, razumljivo je da za neke primjene, kao to su odreene analize aktivnosti, izdvajanje moe ukljuivati izdvajanje staninih membrane koje sadre u njima ugraeni polipeptid, dok za druge primjene moe biti poeljno potpunije izdvajanje. </span>In the next related embodiment, the invention defines methods for obtaining a DmGPCR polypeptide (or its fragment) which includes growth stages of host cells of this invention in a nutrient medium and isolation of the polypeptide or its variant from the cell or from the medium. Since DmGPCR is a seven transmembrane receptor, it is understood that for some applications, such as certain activity assays, isolation may involve isolation of cell membranes containing embedded polypeptide, while for other applications more complete isolation may be desired.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Stanice domaina ovog izuma su vrijedan izvor imunogena za razvoj antitijela koja su specifino imunoreaktivna s DmGPCR-om. Stanice domaina ovog izuma su takoer korisne za metode masovne proizvodnje DmGPCR polipeptida pri emu stanice se stanice razvijaju u odgovarajuem mediju kulture i eljeni polipeptidni produkti izdvajaju se iz stanica ili iz medija u kojem rastu stanice, metodama proiavanja koje su poznate u tehnici, npr. standardnim kromatografskim metodama ukljuujui imunoafinitetnu kromatografiju, receptorsku afinitetnu kromatografiju, hidrofobnu interakcijsku kromatografiju, lektin afinitetnu kromatografiju, veliinsko ekskluzivno filtriranje, kationsku ili anionsku izmjenjivaku kromatografiju, visokotlanu tekuinsku kromatografiju (HPLC), reverzno-faznu HPLC i slino. Druge metode proiavanje ukljuuju one metode u kojima se vri ekspresija eljenog proteina i proiavanje kao fuzijskog proteina koji ima specifini privjesak, oznaku ili kelirajuu speciju koja je prepoznata pomou specifinog vezujueg partnera ili sredstva. Proieni protein moe se cijepati da se dobije eljeni protein ili moe ostati kao intaktni fuzijski protein. Cijepanje fuzijske komponente moe proizvesti oblik eljenog proteina koji ima dodatne aminokiselinske rezidue kao rezultat procesa cijepanja. </span>The host cells of the present invention are a valuable source of immunogens for the development of antibodies that are specifically immunoreactive with DmGPCR. The host cells of the present invention are also useful for methods of mass production of DmGPCR polypeptides, wherein the cells are grown in an appropriate culture medium and the desired polypeptide products are isolated from the cells or from the medium in which the cells grow, by purification methods known in the art, e.g., standard chromatographic methods including immunoaffinity chromatography, receptor affinity chromatography, hydrophobic interaction chromatography, lectin affinity chromatography, size exclusive filtration, cation or anion exchange chromatography, high pressure liquid chromatography (HPLC), reverse-phase HPLC and the like. Other purification methods include those methods in which the desired protein is expressed and purified as a fusion protein having a specific pendant, tag, or chelating species that is recognized by a specific binding partner or agent. The purified protein can be cleaved to produce the desired protein or it can remain as an intact fusion protein. Cleavage of the fusion component can produce a form of the desired protein that has additional amino acid residues as a result of the cleavage process.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Poznavanje DNA sekvencija DmGPCR-a omoguuje modificiranje stanica da se omogui ili povea ekspresija endogenog DmGPCR. Stanice mogu biti modificirane (npr. homolognim rekombiniranjem) da se postigne poveana ekspresija zamjenom, djelominom ili potpunom, prirodnog DmGPCR promotora s cijelim ili djelominim heterolognim promotorom tako da stanice vre ekspresiju DmGPCR na viim razinama. Heterologni promotor ubacuje se na takav nain da se operativno vee za kodirajue sekvencije endogenog DmGPCR (vidi npr. PCT meunarodnu publikaciju br. WO 94/12650, PCT meunarodnu publikaciju br. WO 92/20808 i PCT meunarodnu publikaciju br. WO 91/09955.) Takoer se podrazumijeva da se, uz heterolognu promotorsku DNA, pojaivaki obiljeiva DNA (npr. ada, dhfr i viefunkcionalni CAD gen koji kodira karbamoil fosfat sintazu, aspartat transkarbamilazu i dihodroorotazu) i/ili intronska DNA mogu ubaciti zajedno s heterolognom promotorskom DNA. Ako je povezano s DmGPCR kodirajuom sekvencijom, pojaavanje obiljeivake DNA standardnim metodama odabiranja rezultira kopojaavanjem kodirajuih sekvencija DmGPCR-a u stanicama. </span>Knowing the DNA sequences of DmGPCRs allows cells to be modified to enable or increase expression of endogenous DmGPCRs. Cells can be modified (eg, by homologous recombination) to achieve increased expression by replacing, in part or in whole, the native DmGPCR promoter with a whole or in part heterologous promoter so that the cells express the DmGPCR at higher levels. The heterologous promoter is inserted in such a way that it operably binds to the coding sequences of the endogenous DmGPCR (see, e.g., PCT International Publication No. WO 94/12650, PCT International Publication No. WO 92/20808 and PCT International Publication No. WO 91/09955. ) It is also understood that, in addition to the heterologous promoter DNA, enhancer marker DNA (eg, ada, dhfr and the multifunctional CAD gene encoding carbamoyl phosphate synthase, aspartate transcarbamylase and dihydroorotase) and/or intronic DNA can be inserted together with the heterologous promoter DNA. If associated with a DmGPCR coding sequence, amplification of the marker DNA by standard selection methods results in co-amplification of DmGPCR coding sequences in cells.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Izbaeni (knock-outs) </span>Knock-outs</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Informacije o DNA sekvenciji koje daje ovaj izum takoer omoguuju razvoj (npr. homolognim rekombiniranjem ili knock-out strategijama; vidi Capecchi, Science, 1989, 244, 1288-1292) subjekata koji ne mogu vriti ekspresiju funkcionalnog DmGPCR ili koji vre ekspresiju inaice DmGPCR-a. Takvi subjekti (poglavito ukljuujui insekte i crve) korisni su kao modeli za istraivanje in vivo aktivnosti DmGPCR-a i modulatora DmGPCR-a i takoer su korisni za daljnje rasvjetljavanje uloge DmGPCR-ova u insektima i crvima. </span>The DNA sequence information provided by this invention also enables the development (eg, by homologous recombination or knock-out strategies; see Capecchi, Science, 1989, 244, 1288-1292) of subjects that cannot express a functional DmGPCR or that express a variant of DmGPCR. Such subjects (especially including insects and worms) are useful as models for investigating the in vivo activity of DmGPCRs and DmGPCR modulators and are also useful for further elucidating the role of DmGPCRs in insects and worms.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Protivsmisleni </span>Contradictory</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Takoer su postali dostupni ovim izumom protivsmisleni polinukleotidi koji prepoznaju i hibridiziraju polinukleotide koji kodiraju DmGPCR. Dobivaju se protivsmisleni polinukleotidi pune duljine i fragmenti. Fragment protivsmislene molekule ovog izuma ukljuuju one koji specifino prepoznaju i hibridiziraju DmGPCR ekspresijske kontrolne sekvencije ili DmGPCR RNA (kao to se odreuje sekvencijskim usporeivanjem DNA koja kodira DmGPCR prema DNA koja kodira druge poznate molekule). Identifikacijska sekvencija koja je jedinstvena za DmGPCR-kodirajue polinukleotide moe se izvesti upotrebom bilo koje javno dostupne sekvencijske baze i/ili upotrebnom komercijalno dostupnih programa za usporeivanje sekvencija. Nakon identificiranja eljenih sekvencija, moe se izvriti odvajanje pomou restrikcijske razgradnje ili pojaavanja koritenjem neke od razliitih tehnika polimeraza lananih reakcija, koje su dobro poznate u tehnici. Protivsmisleni polinukleotidi su osobito znaajni za reguliranje ekspresije DmGPCR-a pomou onih stanica s ekspresijom DmGPCR mRNA. </span>Also made available by this invention are antisense polynucleotides that recognize and hybridize to polynucleotides encoding DmGPCR. Full-length antisense polynucleotides and fragments are obtained. Fragment antisense molecules of the present invention include those that specifically recognize and hybridize to DmGPCR expression control sequences or DmGPCR RNA (as determined by sequence comparison of DNA encoding DmGPCR to DNA encoding other known molecules). An identification sequence that is unique to DmGPCR-encoding polynucleotides can be derived using any publicly available sequence database and/or using commercially available sequence comparison programs. After identifying the desired sequences, separation can be performed by restriction digestion or amplification using any of the various polymerase chain reaction techniques well known in the art. Antisense polynucleotides are particularly important for regulating DmGPCR expression by those cells expressing DmGPCR mRNA.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Protivsmislene nukleinske kiseline (poeljno 10 do 20 bazinih-par oligonukleotida) koji se mogu specifino vezati za kontrolne ekspresijske sekvencije DmGPCR ili DmGPCR RNA uvode se u stanice (npr. pomou virusnog vektora ili koloidnog disperzijskog sustava kao to je liposom). Protivsmislena nukleinska kiselina vee se za DmGPCR ciljanu nukleotidnu sekvenciju u stanici i sprjeava transkripciju i/ili prevoenje ciljane sekvencije. Fosforotioat ili metilfosfonatni protivsmisleni oligonukleotidi su speciino predvieni za terapijsku upotrebu koritenjem ovog izuma. Protivsmisleni oligonukleotidi mogu biti nadalje modificirani pomou poli-1-lizin, transferin polilizin ili kolesterol specijama na njihovu 5 kraju. Supresija DmGPCR-a ekspresije bilo na razini transkripcije ili prevoenja korisna je za generiranje staninih ili ivotinjskih modela da se istrai bioloka uloga DmGPCR-ova. </span>Antisense nucleic acids (preferably 10 to 20 base-pair oligonucleotides) that can specifically bind to DmGPCR or DmGPCR RNA control expression sequences are introduced into cells (eg, using a viral vector or a colloidal dispersion system such as a liposome). The antisense nucleic acid binds to the DmGPCR target nucleotide sequence in the cell and prevents transcription and/or translation of the target sequence. Phosphorothioate or methylphosphonate antisense oligonucleotides are specifically contemplated for therapeutic use using the present invention. Antisense oligonucleotides can be further modified with poly-1-lysine, transferrin polylysine or cholesterol species at their 5' end. Suppression of DmGPCR expression either at the transcriptional or translational level is useful for generating cell or animal models to investigate the biological role of DmGPCRs.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Protivsmisleni oligonukleotidi ili fragmenti nukleotidnih sekvencija koji su odabrani iz skupa kojega sainjavaju sekvencije s br. 1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21 ili 23 ili sekvencije koje su njima komplementarne ili homologne, izvedeni iz nukleotidne sekvencije ovog izuma koji kodiraju DmGPCR korisni su za sondiranje genske ekspresije u razliitim tkivima. Primjerice, tkivo moe biti sondirano in situ s oligonukleotidnim sondama koje nose detektibilne skupine, pomou standardnih tehnika autoradiografije. Protivsmisleni oligonukleotidi koji su usmjereni prema regulacijskim podrujima nukleotidne sekvencije mogu biti odabrani iz skupa kojega sainjavaju sekvencije s br. 1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21 ili 23 ili odgovarajue mRNA, ukljuujui, ali bez ogranienja, inicijacijski kodon, TATA kutiju, pojaiva sekvencija i slino. </span>Antisense oligonucleotides or fragments of nucleotide sequences that are selected from the set consisting of sequences with no. 1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21 or 23 or sequences complementary or homologous thereto, derived from the nucleotide sequence of the present invention encoding DmGPCR are useful for probing gene expression in various tissues . For example, tissue can be probed in situ with oligonucleotide probes carrying detectable moieties, using standard autoradiography techniques. Antisense oligonucleotides that are directed towards the regulatory regions of the nucleotide sequence can be selected from the group consisting of sequences with no. 1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21 or 23 or corresponding mRNAs, including but not limited to initiation codon, TATA box, enhancer sequences and the like.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Faktori transkripcije </span>Transcription factors</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">DmGPCR sekvencije ovog izuma olakavaju oblikovanje faktora za moduliranje DmGPCR ekspresije u nativnim stanicama i subjektima i stanicama koje su transformirane ili transficirane s DmGPCR polinukleotidima. Primjerice, Cys2-His2 cink finger proteini, koji veu DNA putem svojih cink finger domena, pokazalo se da su podloni strukturnim promjenama koje dovode do prepoznavanja razliitih ciljanih sekvencija. Ovi umjetni cink finger proteini prepoznaju specifina ciljana mjesta s visokim afinitetom i malenim konstantama disocijacije i mogu djelovati kao genski prekidai da se modulira ekspresija gena. Poznavanje konkretne DmGPCR ciljane sekvencije ovog izuma olakava inenjering cink finger proteina koji su specifini za ciljanu sekvenciju pomou poznatih metoda kao to je kombinacija strukturno-zasnovanog modeliranja i screeninga biblioteke faga (Segal et al., Proc. Natl. Acad. Sci. USA, 1999, 96, 2758-2763; Liu et al., Proc. Natl. Acad. Sci. USA, 1997, 94, 5525-5530 (1997); Greisman et al., Science, 1997, 275, 657-661; Choo et al., J. Mol. Biol., 1997, 273, 525-532). Svaka cink finger domena obino prepoznaje tri ili vie parova baza. Budui da je sekvencija za prepoznavanje od 18 parova baza openito dovoljne duljine da se smatra jedinstvenom u bilo kojem poznatom genomu, cink finger protein koji se sastoji od 6 tandem ponavljanja cink finger-a trebalo bi oekivati da jami specifinost za konkretnu sekvenciju (Segal et al.). Umjetna cink finger ponavljanja, koja su oblikovana temeljem DmGPCR sekvencija, sjedinjena su prema aktivacijskim ili represijskim domenama da se pobolja ili suprimira DmGPCR ekspresija (Liu et al.). Alternativno, cink finger domene mogu se sjediniti u TATA kutija-vezujui faktor (TBP) s varirajuim duljinama veznog podruja izmeu cink finger peptida i TBP da se stvore bilo transkripcijski aktivatori ili represori (Kim et al., Proc. Natl. Acad. Sci. USA, 1997, 94, 3616-3620). Takvi proteini i polinukleotidi koji ih kodiraju korisni su za moduliranje DmGPCR ekspresije in vivo. Novi transkripcijski faktor moe se unijeti u ciljane stanice transfekcijom konstrukata koji vre ekspresiju transkripcijskog faktora (genska terapija) ili uvoenjem proteina. Inenjering cink finger proteina moe takoer biti oblikovan da se poveu RNA sekvencije za upotrebu u terapeuticima kao alternative protivsmislenim ili katalitikim RNA metodama (McColl et al., Proc. Natl. Acad. Sci. USA, 1997, 96, 9521-9526; Wu et al, Proc. Natl. Acad. Sci. USA, 1995, 92, 344-348). Ovaj izum podrazumijeva metode oblikovanja takvih transkripcijskih faktora temeljem genske sekvencije ovog izuma, kao i prilagoenih cink finger proteina koji su korisni da se modulira DmGPCR ekspresija u stanicama (prirodnim ili transformiranim) gdje genetski komplement ukljuuje ove sekvencije. </span>The DmGPCR sequences of the present invention facilitate the design of factors to modulate DmGPCR expression in native cells and subjects and cells transformed or transfected with DmGPCR polynucleotides. For example, Cys2-His2 zinc 'finger' proteins, which bind DNA via their zinc 'finger' domains, have been shown to be subject to structural changes that lead to the recognition of different target sequences. These artificial zinc 'finger' proteins recognize specific target sites with high affinity and small dissociation constants and can act as gene switches to modulate gene expression. Knowledge of the specific DmGPCR target sequence of the present invention facilitates the engineering of zinc finger proteins that are specific for the target sequence using known methods such as a combination of structure-based modeling and phage library screening (Segal et al., Proc. Natl. Acad. Sci. USA , 1999, 96, 2758-2763; Liu et al., Proc. Natl. Acad. Sci. USA, 1997, 94, 5525-5530 (1997); Greisman et al., Science, 1997, 275, 657-661; Choo et al., J. Mol. Biol., 1997, 273, 525-532). Each zinc finger domain usually recognizes three or more base pairs. Since a recognition sequence of 18 base pairs is generally of sufficient length to be considered unique in any known genome, a zinc 'finger' protein consisting of 6 tandem zinc 'finger' repeats should be expected to guarantee specificity for a particular sequence (Segal et al.). Artificial zinc 'finger' repeats, which are designed based on DmGPCR sequences, are fused to activation or repression domains to enhance or suppress DmGPCR expression (Liu et al.). Alternatively, zinc 'finger' domains can be fused to a TATA box-binding factor (TBP) with varying lengths of the linker region between the zinc 'finger' peptide and TBP to create either transcriptional activators or repressors (Kim et al., Proc. Natl. Acad. Sci. USA, 1997, 94, 3616-3620). Such proteins and polynucleotides encoding them are useful for modulating DmGPCR expression in vivo. A new transcription factor can be introduced into target cells by transfection of constructs that express the transcription factor (gene therapy) or by introducing proteins. Engineered zinc 'finger' proteins can also be designed to bind RNA sequences for use in therapeutics as an alternative to antisense or catalytic RNA methods (McColl et al., Proc. Natl. Acad. Sci. USA, 1997, 96, 9521-9526; Wu et al, Proc. Natl. Acad. Sci. USA, 1995, 92, 344-348). The present invention encompasses methods of designing such transcription factors based on the gene sequence of the present invention, as well as custom zinc finger proteins useful to modulate DmGPCR expression in cells (native or transformed) where the genetic complement includes these sequences.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Polipeptidi </span>Polypeptides</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Izum takoer definira proiene i izdvojene DmGPCR polipeptide koji su kodirani pomou polinukleotida ovog izuma ukljuujui DmGPCR polipeptid koji sadri aminokiselinsku sekvenciju koja se nalazi u sekvencijama s br. 2, 4, 6, 8, 10, 12, 14, 16, 18, 20, 22 ili 24. </span>The invention also defines purified and isolated DmGPCR polypeptides that are encoded by the polynucleotides of the present invention including a DmGPCR polypeptide comprising the amino acid sequence found in SEQ ID NOs. 2, 4, 6, 8, 10, 12, 14, 16, 18, 20, 22 or 24.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Uoava se da su izvanstanini epitopi osobito korisni za generiranje i screening antitijela i drugih vezujuih spojeva koji se veu za receptore kao to je DmGPCR. Prema tome, u sljedeoj realizaciji, ovaj izum definira proiene i izdvojene polipeptide koji sadre bar jednu izvanstaninu domenu (npr. N-terminalnu izvanstaninu domenu ili jednu od triju izvanstaninih petlji) DmGPCR-a, kao to je N-terminalna izvanstanina domena DmGPCR-a. Takoer su obuhvaeni dosegom ovog izuma proieni i izdvojeni polipeptidi koji sadre DmGPCR fragment koji je odabran iz skupa koji se sastoji od transmembranskih domena DmGPCR-a, izvanstanine petlje koja povezuje transmembranske domene DmGPCR-a, unutarstanine petlje koja povezuje transmembranske domene DmGPCR-a, C-terminalnog citplazmatskog podruja DmGPCR-a i njihovih fuzija. Takvi fragmenti mogu biti kontinuirani dijelovi prirodnog receptora. Meutim, lako je uoiti da poznavanje DmGPCR-a gena i proteinskih sekvencija koje su ovdje definirane omoguuje rekombiniranje razliitih domena koje nisu kontinuirane u nativnom proteinu. Koritenjem FORTRAN kompjuterskog program nazvanog tmtrestall (Parodi et al., Comput. Appl. Biosci., 1994, 5, 527-535), pokazalo se da DmGPCR sadri transmembranski-premotenih domena. </span>Extracellular epitopes are found to be particularly useful for generating and screening antibodies and other binding compounds that bind to receptors such as DmGPCR. Accordingly, in a further embodiment, the present invention defines purified and isolated polypeptides comprising at least one extracellular domain (eg, the N-terminal extracellular domain or one of the three extracellular loops) of a DmGPCR, such as the N-terminal extracellular domain of a DmGPCR . Also included within the scope of this invention are purified and isolated polypeptides containing a DmGPCR fragment that is selected from the group consisting of transmembrane domains of DmGPCR, an extracellular loop that connects transmembrane domains of DmGPCR, an intracellular loop that connects transmembrane domains of DmGPCR, C -terminal cytoplasmic region of DmGPCR and their fusions. Such fragments may be continuous parts of the native receptor. However, it is easy to see that knowledge of the DmGPCR gene and protein sequences defined here allows the recombination of different domains that are not continuous in the native protein. Using a FORTRAN computer program called "tmtrestall" (Parodi et al., Comput. Appl. Biosci., 1994, 5, 527-535), DmGPCR was shown to contain transmembrane-spanning domains.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Ovaj izum takoer obuhvaa polipeptide koji imaju bar 99%, bar 95%, bar 90%, bar 85%, bar 80%, bar 75%, bar 70%, bar 65%, bar 60%, bar 55% ili bar 50% identinosti i/ili homologije s referentnim polipeptidom ovog izuma. Postotak identinosti aminokiselinske sekvencije u odnosu na referentni polipeptid ovog izuma je ovdje definiran kao postotak aminokiselinskih rezidua u kandidatskoj sekvenciji koji je identian s reziduama u DmGPCR sekvenciji nakon usklaivanja obiju sekvencija i uvoenja praznina, ako je potrebno, da se postigne maksimalni postotak sekvencijske identinosti i ne razmatraju bilo kakve konzervativne supstitucije kao dio sekvencijske identinosti. Postotak sekvencijske homologije u odnosu na referentni polipeptid ovog izuma definiran je ovdje kao postotak aminokiselinskih rezidua u kandidatskoj sekvenciji koje su identine s reziduama u DmGPCR sekvenciji nakon usklaivanja sekvencija i uvoenja praznina, ako je potrebno, da se postigne maksimalni postotak sekvencijske identinosti i da se takoer razmotre bilo koje konzervativne supstitucije kao dio sekvencijske identinosti. </span>The present invention also encompasses polypeptides having at least 99%, at least 95%, at least 90%, at least 85%, at least 80%, at least 75%, at least 70%, at least 65%, at least 60%, at least 55% or at least 50% of identity and/or homology with the reference polypeptide of this invention. Percent amino acid sequence "identity" to a reference polypeptide of the present invention is defined herein as the percentage of amino acid residues in the candidate sequence that are identical to residues in the DmGPCR sequence after alignment of both sequences and introduction of gaps, if necessary, to achieve maximum percent sequence identity and do not consider any conservative substitutions as part of sequence identity. Percent sequence "homology" to a reference polypeptide of the present invention is defined herein as the percentage of amino acid residues in the candidate sequence that are identical to residues in the DmGPCR sequence after aligning the sequences and introducing gaps, if necessary, to achieve the maximum percent sequence identity and to any conservative substitutions are also considered as part of the sequence identity.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">U jednom aspektu, postotak homologije izraunava se kao postotak aminokiselinskih rezidua u manjoj od dvoju sekvencija koja se poklapa s identinom aminokiselinskom reziduom u sekvenciji s kojom se usporeuje, kada se etiri praznine duljine 100 aminokisleina mogu uvesti da se maksimira sukladnost (Dayhoff, u: Atlas of Protein Sequence and Structure, vol. 5, National Biochemical Research Foundation, Washington, D.C., 1972, p. 124, ukljueno ovdje kao referenca). </span>In one aspect, the percentage of homology is calculated as the percentage of amino acid residues in the smaller of the two sequences that match the identical amino acid residue in the sequence being compared, when four gaps of 100 amino acids can be introduced to maximize concordance (Dayhoff, in: Atlas of Protein Sequence and Structure, vol. 5, National Biochemical Research Foundation, Washington, D.C., 1972, p. 124, incorporated herein by reference).</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Polipeptidi ovog izuma mogu biti izdvojeni iz prirodnih izvora stanica ili mogu biti kemijski sintetizirani te mogu biti proizvedeni rekombinantnim postupcima koji ukljuuju stanice domaima ovog izuma. Upotreba stanica domaina sisavaca oekuje se da omoguuje takve post-translacijske modifikacije (npr. glikoziliranje, odbacivanje, lipidiranje i fosforiliranje) to se moe pokazati potrebnim da se postigne optimalna bioloka aktivnost na produktima rekombinantne ekspresije ovog izuma. Glikozilirani i neglikozilirani oblici DmGPCR polipeptida obuhvaeni su ovim izumom. </span>The polypeptides of the present invention may be isolated from natural cell sources or may be chemically synthesized and may be produced by recombinant methods involving the host cells of the present invention. The use of mammalian host cells is expected to allow for such post-translational modifications (eg, glycosylation, shedding, lipidation, and phosphorylation) as may prove necessary to achieve optimal biological activity on the recombinant expression products of the present invention. Glycosylated and non-glycosylated forms of DmGPCR polypeptides are encompassed by this invention.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Ovaj izum takoer obuhvaa inaice (ili analoge) DmGPCR polipeptida. U jednom primjeru, definirane su inaice s umetanjem pri emu jedna ili vie aminokiselinskih rezidua dopunjuje DmGPCR aminokiselinsku sekvenciju. Umetanje moe biti vezano za jedan ili oba kraja ili se moe nalaziti unutar internih podruja DmGPCR aminokiselinske sekvencije. Inaice s umetanjem s dodatnim reziduama na jednom ili oba kraja ukljuuju, primjerice, fuzisjke proteine i proteine ukljuujui aminokiselinske tagove ili oznake. </span>The present invention also encompasses variants (or analogs) of DmGPCR polypeptides. In one example, insertion variants are defined wherein one or more amino acid residues complement the DmGPCR amino acid sequence. The insertion may be attached to one or both ends or may be located within internal regions of the DmGPCR amino acid sequence. Insertion variants with additional residues at one or both ends include, for example, fusion proteins and proteins including amino acid tags or tags.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Inaice s umetanjem DmGPCR polipeptide pri emu je jedna ili vie aminokiselinskih rezidua dodano DmGPCR kiselinskoj sekvenciji ili njenom bioloki aktivnom fragmentu. </span>Insertion variants of the DmGPCR polypeptide wherein one or more amino acid residues are added to the DmGPCR acid sequence or biologically active fragment thereof.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Varijantni produkti ovog izuma takoer ukljuuju zrele DmGPCR produkte, tj. DmGPCR produkte pri emu su uklonjene vodee ili signalne sekvencije, uz dodatne aminokiselinske rezidue. Dodatne aminoterminalne rezidue mogu biti izvedene iz drugog proteina ili mogu ukljuivati jednu ili vie rezidua za koje se ne moe identificirati da su izvedene iz specifinih proteina. Takoer su obuhvaeni DmGPCR produkti s dodatnim metioninom u poloaju -1 (Met-1-DmGPCR), kao i inaice s dodatnim metioninom i lizinom u poloajima -2 i -1 (Met-2-Lys-1-DmGPCR). Inaice DmGPCR-a s dodatnim Met, Met-Lys, Lys reziduama (ili openito jednom ili vie bazinih rezidua) su osobito korisne za pojaanu proizvodnju rekombinantnog proteina u bakterijskoj stanici domaina. </span>Variant products of the present invention also include mature DmGPCR products, i.e., DmGPCR products with leader or signal sequences removed, along with additional amino acid residues. Additional amino-terminal residues may be derived from another protein or may include one or more residues that cannot be identified as being derived from specific proteins. Also included are DmGPCR products with an additional methionine in position -1 (Met-1-DmGPCR), as well as versions with additional methionine and lysine in positions -2 and -1 (Met-2-Lys-1-DmGPCR). Versions of DmGPCRs with additional Met, Met-Lys, Lys residues (or generally one or more basic residues) are particularly useful for enhanced recombinant protein production in a bacterial host cell.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Ovaj izum takoer obuhvaa DmGPCR inaice koje imaju dodatne aminokiselinske rezidue koje su rezultat upotrebe specifinih sustava ekspresije. Primjerice, upotreba komercijalno raspoloivih vektora koji vre ekspresiju polipeptida kao dio glutation-S-transferaza (GST) fuzijskog produkta definira eljeni polipeptid koji ima dodatne glicinske rezidue u poloaju -1 nakon cijepanja GST komponente iz eljenog polipeptida. Takoer su obuhvaene inaice koje su rezultat ekspresije u drugim vektorskim sustavima. </span>The present invention also encompasses DmGPCR variants having additional amino acid residues resulting from the use of specific expression systems. For example, the use of commercially available vectors that express the polypeptide as part of a glutathione-S-transferase (GST) fusion product defines a desired polypeptide that has additional glycine residues in the -1 position after cleavage of the GST component from the desired polypeptide. Variants resulting from expression in other vector systems are also included.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Inaice s umetanjem takoer ukljuuju fuzijske proteine pri emu su aminoterminal i/ili karboksilni terminal DmGPCR-a sjedinjeni s drugim polipeptidom. </span>Insertion variants also include fusion proteins wherein the amino terminus and/or carboxyl terminus of the DmGPCR is fused to another polypeptide.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">U sljedeem aspektu, ovaj izum definira inaice brisanja pri emu je jedna ili vie aminokiselinskih rezidua u DmGPCR polipeptidu uklonjeno. Brisanje se moe izvriti na jednom ili na oba kraja DmGPCR polipeptida ili uz uklanjanje jedne ili vie neterminalnih aminokiselinskih rezidua DmGPCR-a. Inaice brisanja, prema tome, ukljuuju sve fragmente DmGPCR polipeptida. </span>In a further aspect, the present invention defines deletion variants wherein one or more amino acid residues in the DmGPCR polypeptide have been removed. Deletion can be performed at one or both ends of the DmGPCR polypeptide or by removing one or more non-terminal amino acid residues of the DmGPCR. Deletion variants, therefore, include all fragments of the DmGPCR polypeptide.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Ovaj izum takoer obuhvaa polipeptidne fragmente sekvencija koje pripadaju skupu kojega sainjavaju sekvencije s br. 2, 4, 6, 8, 10, 12, 14, 16, 18, 20, 22 ili 24 pri emu fragmenti zadravaju bioloka (npr. ligandno vezanje i/ili unutarstanino signaliziranje) te imunoloka svojstva DmGPCR polipeptida. Fragmenti koji sadre bar 5, 10, 15, 20, 25, 30, 35 ili 40 konzekutivnih aminokiselina bilo kojeg polipeptida on onih koji su ovdje opisani, obuhvaeni su ovim izumom. Polipeptidni fragmenti mogu pokazivati antigena svojstva koja su jedinstvena ili specifina za DmGPCR i njegove homologe alela i specija. Fragmenti ovog izuma koji imaju poeljna bioloka i imunoloka svojstva mogu se prirediti metodama koje su dobro poznate i rutinski se koriste u tehnici. </span>This invention also encompasses polypeptide fragments of sequences belonging to the set consisting of sequences with no. 2, 4, 6, 8, 10, 12, 14, 16, 18, 20, 22 or 24, whereby the fragments retain the biological (eg ligand binding and/or intracellular signaling) and immunological properties of the DmGPCR polypeptide. Fragments containing at least 5, 10, 15, 20, 25, 30, 35 or 40 consecutive amino acids of any of the polypeptides described herein are encompassed by this invention. Polypeptide fragments may exhibit antigenic properties that are unique or specific to the DmGPCR and its allele and species homologues. Fragments of the present invention having desirable biological and immunological properties can be prepared by methods well known and routinely used in the art.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">U daljnjem aspektu, ovaj izum definira supstitucijske inaice DmGPCR polipeptida. Supstitucijske inaice ukljuuju one polipeptide pri emu su jedna ili vie aminokiselinskih rezidua DmGPCR polipeptida uklonjene i zamijenjene alternativnim reziduama. U jednom aspektu, supstitucije su konzervativne prirode. Meutim, ovaj izum ukljuuje supstitucije koje takoer nisu konzervativne. Konzervativne supstitucije za ovu namjenu mogu biti definirane kao to je navedeno u tablicama 1, 2 ili 3 nie. </span>In a further aspect, the present invention defines substitutional versions of DmGPCR polypeptides. Substitution variants include those polypeptides wherein one or more amino acid residues of the DmGPCR polypeptide have been removed and replaced with alternative residues. In one aspect, the substitutions are conservative in nature. However, this invention includes substitutions that are also not conservative. Conservative substitutions for this purpose may be defined as set forth in Tables 1, 2 or 3 below.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Varijantni polipeptidi ukljuuju one pri emu su konzervativne supstitucije uvedene modificiranjem polinukleotida koji kodiraju polipeptide ovog izuma. Aminokiseline se mogu klasificirati sukladno fizikim svojstvima i doprinosu sekundarnoj i tercijarnoj proteinskoj strukturi. Konzervativna supstitucija poznata je u tehnici kao supstitucija jedne aminokiseline drugom pri emu one imaju slina svojstva. Primjeri konzervativnih supstitucija navedeni su u tablici 1 (iz WO 97/09433, str. 10, publicirano 13. oujka 1997. (PCT/GB96/02197, podneseno 9/6/96)), neposredno nie. </span>Variant polypeptides include those wherein conservative substitutions have been introduced by modifying the polynucleotides encoding the polypeptides of the invention. Amino acids can be classified according to their physical properties and contribution to secondary and tertiary protein structure. Conservative substitution is known in the art as the substitution of one amino acid for another where they have similar properties. Examples of conservative substitutions are listed in Table 1 (from WO 97/09433, p. 10, published March 13, 1997 (PCT/GB96/02197, filed 9/6/96)), immediately below.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">[image] </span>[image]</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Alternativno, konzervativne aminokiseline mogu se grupoirati kao to je opisao Lehninger, (Biochemistry, Second Edition; Worth Publishers, Inc. NY, NY, 1975, str.71-77) kao to je navedeno u tablici 2, neposredno nie. </span>Alternatively, conservative amino acids may be grouped as described by Lehninger, (Biochemistry, Second Edition; Worth Publishers, Inc. NY, NY, 1975, pp. 71-77) as listed in Table 2, immediately below.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">[image] </span>[image]</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Kao sljedea mogunost, primjeri konzervativnih supstitucija navedeni su u tablici 3, nie. </span>As a further possibility, examples of conservative substitutions are listed in Table 3, below.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">[image] </span>[image]</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Valja imati na umu da je definicija polipeptida ovog izuma previena da ukljuuje polipeptide koji posjeduju druge modifikacije osim ubacivanja, brisanja ili supstitucije aminokiselinskih rezidua. Kao primjer, modifikacije mogu biti kovalentne prirode i ukljuuju primjerice, kemijsko vezanje s polimerima, lipidima, drugim organskim i anorganskim specijama. Takvi derivati mogu se prirediti da se povea cirkulirajue vrijeme poluivota polipeptida ili mogu ili oblikovani da se povea ciljani kapacitet polipeptida za poeljne stanice, tkiva ili organe. Slino, ovaj izum nadalje obuhvaa DmGPCR polipeptide koji su ili kovalentno modificirani ukljuujui jedan ili vie u vodi topljivih polimernih dodataka kao to su polietilen glikol, polioksietilen glikol ili polipropilen glikol. </span>It should be noted that the definition of a polypeptide of the present invention is intended to include polypeptides possessing modifications other than insertions, deletions or substitutions of amino acid residues. As an example, modifications can be covalent in nature and include, for example, chemical bonding with polymers, lipids, other organic and inorganic species. Such derivatives may be engineered to increase the circulating half-life of the polypeptide or may or may be designed to increase the targeting capacity of the polypeptide to desired cells, tissues or organs. Similarly, the present invention further encompasses DmGPCR polypeptides that are either covalently modified by including one or more water-soluble polymeric additives such as polyethylene glycol, polyoxyethylene glycol, or polypropylene glycol.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Inaice koje pokazuju ligandna vezujua svojstva nativnog DmGPCR i koje imaju ekspresiju na viim razinama, kao i inaice koje daju konstitutivno aktivne receptore su osobito korisne u analizama ovog izuma. Te inaice su takoer korisne u analizama ovog izuma i u definiranju staninih, tkivnih i ivotinjskih modela za istraivanje aberantne aktivnosti DmGPCR. </span>Variants that exhibit the ligand binding properties of the native DmGPCR and are expressed at higher levels, as well as variants that provide constitutively active receptors, are particularly useful in the assays of this invention. These variants are also useful in assays of the present invention and in defining cell, tissue and animal models for investigating aberrant DmGPCR activity.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Antitijela </span>Antibodies</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Takoer su ovim izumom obuhvaena antitijela (npr. monoklonska i poliklonska antitijela, jednolanana antitijela, kimerna antitijela, bifunkcionalna/bispecifina antitijela, humanizirana antitijela, humana antitijela i komplementarna determinirajua podruja (CDR)-usaenih antitijela, ukljuujui spojeve koji sadre CDR sekvencije koje specifino prepoznaju polipeptid ovog izuma) specifina za DmGPCR ili njegove fragmente. Fragmenti antitijela, ukljuujui Fab, Fab, F(ab)2 i Fv, takoer su definirani ovim izumom. Pojam specifian za, kada se rabi za opis antitijela ovog izuma, oznauje da promjenjiva podruja antitijela ovog izuma prepoznaju i veu ekskluzivno DmGPCR polipeptide (tj. mogu razlikovati DmGPCR polipeptide od ostalih poznatih GPCR polipeptida temeljem mjerljivih razlika u vezujuem afinitetu, bez obzira na mogue postojanje lokalizirane sekvencijske identinosti, homologije ili slinosti izmeu DmGPCR i takvog polipeptida). Podrazumijeva se da specifina antitijela mogu takoer biti u interakciji s drugim proteinima (primjerice, S. aureus protein ili druga antitijela u ELISA tehnikama) kroz interakcije sa sekvencijama izvan promjenjivog podruja antitijela, i, konkretno, u nepromjenjivom podruju molekule. Screening analize za odreivanje vezujue specifinosti antitijela ovog izuma dobro su poznate i rutinski se primjenjuju u tehnici. Za iscrpno razmatranje takvih analiza, vidi Harlow et al. (Eds.), Antitijela Laboratory Manual, Cold Spring Harbor Laboratory, Cold Spring Harbor, NY, 1988, poglavlje 6. Antitijela koja prepoznaju i veu fragmente DmGPCR polipeptida ovog izuma takoer s epodrazumijevaju pod uvjetom da su antitijela specifina za DmGPCR polipeptide. Antitijela ovog izuma mogu se proizvesti koritenjem bilo koje metode koja je dobro poznata i rutinski se primjenjuje u tehnici. </span>Also encompassed by this invention are antibodies (eg, monoclonal and polyclonal antibodies, single chain antibodies, chimeric antibodies, bifunctional/bispecific antibodies, humanized antibodies, human antibodies, and complementarity determining region (CDR)-implanted antibodies, including compounds containing CDR sequences that specifically recognize polypeptide of this invention) specific for DmGPCR or fragments thereof. Antibody fragments, including Fab, Fab', F(ab')2 and Fv, are also defined by the present invention. The term "specific for," when used to describe the antibodies of the present invention, means that the variable regions of the antibodies of the present invention recognize and bind exclusively DmGPCR polypeptides (ie, can distinguish DmGPCR polypeptides from other known GPCR polypeptides based on measurable differences in binding affinity, regardless of possible existence of localized sequence identity, homology or similarity between DmGPCR and such polypeptide). It is understood that specific antibodies may also interact with other proteins (eg, S. aureus protein or other antibodies in ELISA techniques) through interactions with sequences outside the variable region of the antibody, and, specifically, in the constant region of the molecule. Screening assays to determine the binding specificity of antibodies of the present invention are well known and routinely used in the art. For an exhaustive discussion of such analyses, see Harlow et al. (Eds.), Antibody Laboratory Manual, Cold Spring Harbor Laboratory, Cold Spring Harbor, NY, 1988, Chapter 6. Antibodies that recognize and bind fragments of the DmGPCR polypeptides of the present invention are also contemplated provided that the antibodies are specific for DmGPCR polypeptides. Antibodies of the present invention can be produced using any method well known and routinely practiced in the art.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Ovaj izum definira antitijela koja su specifina za DmGPCR ovog izuma. Specifinost antitijela opisana je detaljno nie. Meutim, valja naglasiti da se antitijela koja mogu nastati iz polipeptida koji su prethodno opisani u literaturi i koji mogu sliajno interferirati s DmGPCR-om (npr. zbog sluajnog postojanja slinog epitopa u oba polipeptida) smatraju se interferencijskim (cross-reactive) antitijelima. Takva interferencijska antitijela nisu antitijela koja su specifina za DmGPCR. Odreivanje je li neko antitijelo specifino za DmGPCR ili je u interferenciji s drugim poznatim receptorom moe se izvriti pomou nekoliko analiza, kao to su Western blotting analize, to je dobro poznato u tehnici. Za identificiranje stanica s ekspresijom DmGPCR i takoer za moduliranje DmGPCR-ligand vezujue aktivnosti, korisna su antitijela koja se specifino veu za izvanstanini epitop DmGPCR-a. </span>The present invention defines antibodies that are specific for the DmGPCR of the present invention. Antibody specificity is described in detail below. However, it should be emphasized that antibodies that can arise from polypeptides previously described in the literature and that can similarly interfere with DmGPCR (e.g. due to the coincidental existence of a similar epitope in both polypeptides) are considered "interference" (cross-reactive) antibodies. . Such interfering antibodies are not antibodies that are "specific" for DmGPCR. Determining whether an antibody is specific for DmGPCR or interferes with another known receptor can be performed using several assays, such as Western blotting assays, which are well known in the art. To identify cells expressing DmGPCR and also to modulate DmGPCR-ligand binding activity, antibodies that specifically bind to the extracellular epitope of DmGPCR are useful.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">U jednoj inaici, ovaj izum definira monoklonska antitijela. Hibridomi koji proizvode takva antitijela takoer se smatraju aspektima ovog izuma. U daljnjoj inaici, ovaj izum definira humanizirana antitijela. Humanizirana antitijela korisna su za in vivo terapijske indikacije za tretiranje bolesti ili stanja koja su izazvana ektoparazitima. </span>In one embodiment, the present invention defines monoclonal antibodies. Hybridomas producing such antibodies are also considered aspects of the present invention. In a further embodiment, the present invention defines humanized antibodies. Humanized antibodies are useful for in vivo therapeutic indications for treating diseases or conditions caused by ectoparasites.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">U drugoj inaici, ovaj izum definira smjesu bez stanica (cell-free) koja sadri poliklonska antitijela, pri emu je bar jedno antitijelo antitijelo ovog izuma specifino za DmGPCR. Antiseurumi koji su izdvojeni iz ivotinje su primjer takve smjese, kao to je i smjesa koja sadri frakciju antitijela antiseruma koja je resuspendirana u vodi ili u drugom otapalu, ekscipijentu ili nosau. </span>In another version, the present invention defines a cell-free mixture containing polyclonal antibodies, wherein at least one antibody is an antibody of the present invention specific for DmGPCR. Antisera that have been isolated from an animal are an example of such a mixture, as is a mixture containing the antibody fraction of an antiserum that has been resuspended in water or another solvent, excipient, or carrier.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">U sljedeoj srodnoj realizaciji, ovaj izum definira anti-idiotipska antitijela koja su specifina za antitijelo koje je specifino za DmGPCR. </span>In another related embodiment, the present invention defines anti-idiotypic antibodies that are specific for an antibody that is specific for a DmGPCR.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Dobro je poznato antitijela sadre relativno malene antigen vezujue domene koje se mogu izdvojiti kemijski ili rekombinantnim tehnikama. Takve domene korisne su za same DmGPCR vezujue molekule i takoer mogu biti sjedinjene s toksinima ili drugim polipeptidima. Prema tome, u sljedeoj realizaciji, ovaj izum definira polipeptid koji sadri DmGPCR-specifinog antitijela, pri emu se fragment i polipeptid veu za DmGPCR. Kao primjer bez ogranienja, ovaj izum definira polipeptide koji su jednolanana antitijela, CDR-usaena antitijela i humanizirana antitijela. </span>It is well known that antibodies contain relatively small antigen-binding domains that can be separated by chemical or recombinant techniques. Such domains are useful for DmGPCR binding molecules themselves and can also be fused to toxins or other polypeptides. Accordingly, in a further embodiment, the present invention defines a polypeptide comprising a DmGPCR-specific antibody, wherein the fragment and the polypeptide bind to a DmGPCR. By way of example without limitation, the present invention defines polypeptides that are single-chain antibodies, CDR-grafted antibodies, and humanized antibodies.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Nehumana antitijela mogu biti humanizirana bilo kojom metodom koja je poznata u tehnici. U jednoj metodi, nehumani CDR-ovi ubacuju se u humano antitijelo ili koncenzusnu sekvenciju osnove antitijela. Nadalje u osnovu antitijela mogu se unijeti promjene da se modulira afinitet ili imunogenost. </span>Non-human antibodies can be humanized by any method known in the art. In one method, non-human CDRs are inserted into a human antibody or consensus antibody core sequence. Furthermore, changes can be made to the antibody backbone to modulate affinity or immunogenicity.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Antitijela ovog izuma korisna su npr. za terapijske potrebe (moduliranjem aktivnosti ektoparazitnog DmGPCR), dijagnostike potrebe da se detektira ili kvantificira ektoparazitni DmGPCR i proiavanje DmGPCR-a. Pribori koji sadre antitijelo ovog izuma za neku ovdje opisanu svrhu takoer su obuhvaeni izumom. Openito, pribor ovog izuma takoer sadri kontrolni antigen na kojega je antitijelo imunospecifino. </span>Antibodies of the present invention are useful, for example, for therapeutic purposes (by modulating the activity of ectoparasitic DmGPCR), diagnostic purposes to detect or quantify ectoparasitic DmGPCR, and purification of DmGPCR. Kits containing an antibody of the present invention for a purpose described herein are also encompassed by the invention. In general, the kit of the present invention also contains a control antigen for which the antibody is immunospecific.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Izum takoer definira metode za koritenje antitijela ovog izuma. Primjerice, izumom su definirane metode za moduliranje ligandnog vezanja DmGPCR-a to ukljuuje stupanj dovoenja u dodir DmGPCR s antitijelom specifinim za DmGPCR, u uvjetima gdje se antitijelo vee za receptor. Antitijela ovog izuma mogu se koristiti za kontroliranje populacije insekata primjenom anti-DmGPCR antitijela insektu da se modulira ligandno vezanje DmGPCR-a. Primjerice, insekti mogu biti odabrani izmeu sljedeih: muha, vona muha, krpelj, buha, ui, grinje i ohar. </span>The invention also defines methods for using the antibodies of the invention. For example, the invention defines methods for modulating ligand binding of DmGPCR, which includes the step of bringing the DmGPCR into contact with an antibody specific for DmGPCR, under conditions where the antibody binds to the receptor. The antibodies of the present invention can be used to control insect populations by administering an anti-DmGPCR antibody to the insect to modulate ligand binding of the DmGPCR. For example, insects can be selected from the following: fly, fruit fly, tick, flea, lice, mite and cockroach.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Manipuliranje genima </span>Gene manipulation</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Manipuliranje genima koritenjem DmGPCR takoer je korisno u subjekata kao to su insekti. Manipuliranje genima ukljuuje obnavljanje aktivnosti DmGPCR-a aktivnost, preekspresiju DmGPCR i negativno reguliranje DmGPCR-a. Ovaj izum takoer obuhvaa manipuliranje genima da se obnovi DmGPCR aktivnost koja je izgubljena zbog gubitka funkcije mutiranja. Unoenje funkcionalnog DmGPCR gena u odgovarajue stanice vri se ex vivo, in situ ili in vivo koritenjem vektora i konkretnije virusnih vektora (npr. adenovirus, adeno-pridrueni virus ili retrovirus) ili ex vivo koritenjem metoda fizikog DNA prijenosa (npr. liposomi ili kemijski tretmani). Vidi npr. Anderson, Nature, 1998, suppl. 392 (6679), 25-20. Za dodatni pregled tehnologije genske terapije vidi Friedmann, Science, 1989,244,1275-1281; Verma, Scientific American, 1990, 68-84; te Miller, Nature, 1992, 357, 455-460. Takoer se podrazumijeva da se manipuliranje genima, primjerice protivsmisleni tretman, moe primijeniti za negativno reguliranje ekspresije DmGPCR-a. Kao neograniavajui primjer, manipuliranje genima moe biti korisno za kontroliranje populacije insekata izbacivanjem (knocking-out) ili podreguliranjem jednog ili vie DmGPCR gena ili njegovih fragmenata (vidi gore i dolje). </span>Gene manipulation using DmGPCR is also useful in subjects such as insects. Gene manipulation includes restoration of DmGPCR activity, overexpression of DmGPCR and negative regulation of DmGPCR. The present invention also encompasses manipulating genes to restore DmGPCR activity that has been lost due to loss of function mutations. The introduction of a functional DmGPCR gene into the corresponding cells is done ex vivo, in situ or in vivo using vectors and more specifically viral vectors (eg adenovirus, adeno-associated virus or retrovirus) or ex vivo using physical DNA transfer methods (eg liposomes or chemical treatments ). See, eg, Anderson, Nature, 1998, suppl. 392 (6679), 25-20. For an additional review of gene therapy technology, see Friedmann, Science, 1989,244,1275-1281; Verma, Scientific American, 1990, 68-84; and Miller, Nature, 1992, 357, 455-460. It is also understood that gene manipulation, for example antisense treatment, can be applied to negatively regulate DmGPCR expression. As a non-limiting example, gene manipulation may be useful for controlling insect populations by knocking-out or down-regulating one or more DmGPCR genes or fragments thereof (see above and below).</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Smjese </span>Mixtures</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Sljedei aspekt ovog izuma odnosi se na smjese, ukljuujui insekticidne i farmaceutske smjese, koje sadre bilo koju molekulu nukleinske kiseline ili rekombinantne ekspresijske vektore koji su prije opisani te prihvatljivi nosa i razrjeiva. Nosa ili razrjeiva moe biti farmaceutski prihvatljiv. Pogodni nosai su opisani u najnovijem izdanju: Remingtons Pharmaceutical Sciences, A. Osol, to je standardni tekst u ovom podruju, to je ovdje ukljueno kao referenca u cijelosti. Primjeri takvih nosaa ili razrjeivaa ukljuuju, ali nisu ogranieni na vodu, slanu otopinu, Ringerovu otopinu, otopinu dekstroze i 5% humani serum albumin. Liposomi i nevodeni nosai kao to su vrsta ulja mogu se takoer koristiti. Formulacije se steriliziraju uobiajenim tehnikama. </span>A further aspect of this invention relates to compositions, including insecticidal and pharmaceutical compositions, containing any of the nucleic acid molecules or recombinant expression vectors previously described and an acceptable carrier and diluent. The carrier or diluent may be pharmaceutically acceptable. Suitable carriers are described in the most recent edition: Remington's Pharmaceutical Sciences, A. Osol, which is a standard text in the field, which is incorporated herein by reference in its entirety. Examples of such carriers or diluents include, but are not limited to, water, saline, Ringer's solution, dextrose solution, and 5% human serum albumin. Liposomes and non-aqueous carriers such as solid oils may also be used. Formulations are sterilized using conventional techniques.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Takoer su unutar dosega ovog izuma smjese koje sadre polipeptide, polinukleotide ili antitijela ovog izuma koji su formulirani s primjerice farmaceutski prihvatljivim nosaem. </span>Also within the scope of this invention are mixtures containing polypeptides, polynucleotides or antibodies of this invention which are formulated with, for example, a pharmaceutically acceptable carrier.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Ovaj izum definira insekticidne smjese koje sadre DmGPCR polinukleotid, DmGPCR polipeptid, anti-DmGPCR antitijelo, njegove fragmente ili dijelove koji posjeduju DmGPCR-vezujuu aktivnost, DmGPCR vezujui partner ili DmGPCR modulator. </span>This invention defines insecticidal compositions containing a DmGPCR polynucleotide, a DmGPCR polypeptide, an anti-DmGPCR antibody, fragments or parts thereof possessing a DmGPCR-binding activity, a DmGPCR binding partner or a DmGPCR modulator.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Pribori i metode </span>Accessories and methods</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Ovaj izum takoer je usmjeren prema priborima, ukljuujui farmaceutske i insekticidne pribore. Pribori mogu sadravati bilo koju molekulu nukleinske kiseline koje su prije opisane, bilo koji od peptida koji su prije opisani i bilo koje antitijelo koje se vee za polipeptid ovog izuma kao to je prije opisano, kao i negativnu kontrolu. Pribor moe sadravati dodatne komponente, kao to su primjerice upute, vrsta podloga, reagensi koji su korisni za kvantificiranje i slino. </span>This invention is also directed to accessories, including pharmaceutical and insecticidal accessories. Kits may contain any of the nucleic acid molecules described above, any of the peptides described above, and any antibody that binds to a polypeptide of the invention as described above, as well as a negative control. The kit may contain additional components, such as instructions, a solid substrate, reagents useful for quantification, and the like.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Moe se nainiti pribor za detektiranje ekspresije polinukleotida ili kodiranih proteina. Primjerice, mogu se nainiti pribori za oligonukleotidno hibiridiziranje koji sadre spremnik koji ima oligonukleotidnu sondu koja je specifina za DmGPCR-specifian DNA i proizvoljno, spremnik s pozitivnim i negativnim kontrolama i/ili upute. Slino, mogu se nainiti PCR pribori koji sadre spremnik s primerima koji su specifini za DmGPCR-specifine sekvencije, DNA i proizvoljno, spremnike s veliinskim obiljeivaima, pozitivnim i negativnim kontrolama i/ili uputama. </span>A kit can be made to detect the expression of polynucleotides or encoded proteins. For example, oligonucleotide hybridization kits can be made that contain a container having an oligonucleotide probe that is specific for DmGPCR-specific DNA and optionally, a container with positive and negative controls and/or instructions. Similarly, PCR kits can be made that contain primers that are specific for DmGPCR-specific sequences, DNA, and optionally, size labelers, positive and negative controls, and/or instructions.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Uvjeti hibiridziranja moraju biti takvi da do hibridiziranja dolazi samo s genima u prisutnosti drugih molekula nukleinske kiseline. U strogim uvjetima hibridiziranja hibridiziraju samo jako komplementarne sekvencije nukleinske kiseline. Takvi uvjeti mogu sprijeiti hibridiziranje nukleinske kiseline koja ima 1 ili 2 neslaganja na 20 kontinuiranih neukleotida. takvi uvjeti su definirani supra. </span>Hybridization conditions must be such that hybridization occurs only with genes in the presence of other nucleic acid molecules. Under stringent hybridization conditions, only highly complementary nucleic acid sequences hybridize. Such conditions may prevent nucleic acid having 1 or 2 mismatches per 20 contiguous nucleotides from hybridizing. such conditions are defined above.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Testni uzorci koji su pogodni za metode sondiranja nukleinske kiseline ukljuuju, primjerice, stanice ili ekstrakte stanica nukleinskih kisleina ili bioloke tekuine. Uzorci koji se koriste u gore navedenim metodama e varirati ovisno o formatu analize, metodi detektiranja i prirodi tkiva, stanica ili ekstrakata koji se analiziraju. Metode prireivanja ekstrakata stanica nukleinskih kiselina dobro su poznate u tehnici i mogu se lako prilagoditi da se dobije uzorak koji je sukladan s koritenom metodom. </span>Test samples suitable for nucleic acid probing methods include, for example, cells or nucleic acid cell extracts or biological fluids. The samples used in the above methods will vary depending on the assay format, the detection method and the nature of the tissue, cells or extracts being analysed. Methods for preparing nucleic acid cell extracts are well known in the art and can be easily adapted to obtain a sample that is compatible with the method used.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">U sljedeem aspektu, izumom su definirane metode za detektiranje polinukleotida u uzorku kao dijagnostiko pomagalo za bolesti i poremeaje koji su izazvani ektoparazitom, pri emu metode ukljuuju sljedee stupnjeve: (a) dovoenje u dodir uzorka sa sondom nukleinske kiseline koja hibridizira u uvjetima hibridizacijske analize prema ciljanom podruju nukleinske kiseline kodirajueg peptida koji ima sekvenciju koja je odabrana iz skupa to ga sainjavaju sekvencije s br. 2, 4, 6, 8, 10, 12, 14, 16, 18, 20, 22 i 24, a navedena sonda sastoji se od sekvencije nukleinske kiseline koja polipeptid, njenih fragmenata te komplemenata sekvencija i fragmenata; i (b) detektiranje prisutnosti ili koliine hibrida sonda:ciljano podruje kao indikacije bolesti. </span>In the next aspect, the invention defines methods for detecting polynucleotides in a sample as a diagnostic aid for diseases and disorders caused by an ectoparasite, wherein the methods include the following steps: (a) bringing the sample into contact with a nucleic acid probe that hybridizes under the conditions of a hybridization analysis according to to the target region of the nucleic acid encoding the peptide having a sequence selected from the group consisting of sequences with no. 2, 4, 6, 8, 10, 12, 14, 16, 18, 20, 22 and 24, and said probe consists of a nucleic acid sequence that is a polypeptide, its fragments, and complementary sequences and fragments; and (b) detecting the presence or amount of probe:target region hybrids as an indication of disease.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Alternativno, mogu se nainiti pribori za imunoloke analize koji sadre spremnike s antitijelima specifinim za DmGPCR protein i proizvoljno, spremnike s pozitivnim i negativnim kontrolama i/ili uputama. </span>Alternatively, kits for immunoassays can be made that contain containers with antibodies specific for the DmGPCR protein and optionally, containers with positive and negative controls and/or instructions.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Pribori se koriste pod uvjetom da su korisni u identificiranju vezujueg partnera DmGPCR-a, kao to su prirodni ligandi ili modulatori (agonisti ili antagonisti). Tvari koje su korisne za tretiranje poremeaja ili bolesti mogu dati pozitivne rezultate u jednoj ili vie in vitro analiza na aktivnost koja odgovara tretmanu dotine bolesti ili poremeaja. Tvari koje moduliraju aktivnost polipeptida ukljuuju, ali bez ogranienja protivsmislene oligonukleotide, agoniste i antagoniste i antitijela. </span>Kits are used provided they are useful in identifying the binding partner of the DmGPCR, such as natural ligands or modulators (agonists or antagonists). Substances useful for the treatment of a disorder or disease may yield positive results in one or more in vitro assays for activity corresponding to the treatment of the disease or disorder in question. Agents that modulate polypeptide activity include, but are not limited to, antisense oligonucleotides, agonists and antagonists, and antibodies.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Izum takoer definira metode za moduliranje ligandnog vezanja DmGPCR-a to ukljuuje stupanj dovoenja u dodir DmGPCR s antitijelom specifinim za DmGPCR, u uvjetima gdje se vee za receptor. </span>The invention also defines methods for modulating ligand binding of a DmGPCR which includes the step of contacting the DmGPCR with an antibody specific for the DmGPCR, under conditions where it binds to the receptor.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Metode izazivanja imunolokog odgovora </span>Methods of eliciting an immune response</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Sljedei aspekt ovog izuma odnosi se na metode izazivanja imunolokog odgovora u subjekta prema polipeptidu ovog izuma primjenom subjektu odreene koliine polipeptida koja je dovoljna da se izazove imunoloki odgovor. Ta koliina ovisi o vrsti subjekta, veliini subjekta i slino i moe je odrediti osoba koja poznaje ovo podruje tehnike. </span>Another aspect of the present invention relates to methods of inducing an immune response in a subject to a polypeptide of the present invention by administering to the subject an amount of the polypeptide sufficient to elicit an immune response. This amount depends on the type of subject, the size of the subject, and the like, and can be determined by a person skilled in the art.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Metode identificiranja liganda </span>Methods of identifying ligands</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Sljedei aspect ovog izuma odnosi se na metode identificiranja spojeva koji se veu za DmGPCR ili molekula nukleinske kiseline koja kodira DmGPCR, to obuhvaa dovoenje u dodir DmGPCR ili molekule nukleinske kiseline koja kodira isto, sa spojem te odreivanje vee li spoj DmGPCR ili molekulu nukleinske kiseline koja kodira isto. Vezanje se moe odrediti pomou analiza vezanja to je dobro poznato onima koji poznaju ovo podruje, ukljuujui, ali bez ogranienja, gel-shift analize, Western blots, analizu radioobiljeene kompeticije, fag-zasnovano ekspresijsko kloniranje, kofrakcioniranje pomou kromatografije, koprecipitacija, unakrsno vezanje, interakcijska analiza zamka/dvohibrid, southwestern analiza, ELISA i slino, to je primjerice opisano u: Current Protocols in Molecular Biology, John Wiley &amp; Sons, NY, 1999, to je ovdje ukljueno kao referenca u cijelosti. Spojevi koje treba provjeriti ukljuuju (to moe ukljuivati spojeve koji su suspektni na vezanje DmGPCR ili molekule nukleinske kiseline koja ih kodira), ali nisu ogranieni na spojeve izvanstaninog, unutarstaninog, biolokog ili kemijskog podrijetla. </span>Another aspect of the present invention relates to methods of identifying compounds that bind to a DmGPCR or a nucleic acid molecule encoding a DmGPCR, which comprises contacting the DmGPCR or a nucleic acid molecule encoding the same with the compound and determining whether the compound binds the DmGPCR or nucleic acid molecule encoding the same. encodes the same. Binding can be determined using binding assays well known to those skilled in the art, including but not limited to gel-shift assays, Western blots, radiolabeled competition assays, phage-based expression cloning, chromatographic cofractionation, coprecipitation, cross- binding, trap/two-hybrid interaction assay, 'southwestern' assay, ELISA and the like, as described for example in: Current Protocols in Molecular Biology, John Wiley &amp; Sons, NY, 1999, which is incorporated herein by reference in its entirety. Compounds to be screened include (which may include compounds suspected of binding DmGPCR or the nucleic acid molecule encoding them) but are not limited to compounds of extracellular, intracellular, biological or chemical origin.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Izum takoer definira analize za identificiranje spojevi koji veu DmGPCR. Jedna takva analiza ukljuuje dovoenje u dodir DmGPCR sa spojem koji je suspektan na vezanje DmGPCR te mjerenje vezanja izmeu spoja i DmGPCR. U nekim realizacijama, smjesa ukljuuje staninu ekspresiju DmGPCR na njenoj povrini. U sljedeoj inaici, koristi se izdvojen DmGPCR ili stanine membrane koje sadre DmGPCR. Vezanje se moe mjeriti izravno, npr. koritenjem obiljeenog spoja ili se moe izmjeriti posredno pomou nekoliko tehnika, ukljuujui mjerenje unutarstaninog signaliziranja DmGPCR-a koje je izazvano spojem (ili mjerenjem promjena razine signaliziranja DmGPCR-a). </span>The invention also defines assays for identifying compounds that bind DmGPCR. One such assay involves contacting the DmGPCR with a compound suspected of binding to the DmGPCR and measuring the binding between the compound and the DmGPCR. In some embodiments, the composition includes cellular expression of DmGPCR on its surface. In the following version, isolated DmGPCR or cell membranes containing DmGPCR are used. Binding can be measured directly, eg, using a labeled compound, or can be measured indirectly using several techniques, including measuring compound-induced intracellular DmGPCR signaling (or measuring changes in DmGPCR signaling levels).</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Specifine vezujue molekule, ukljuujui prirodne ligande i sintetske spojeve, mogu se identificirati ili razviti koritenjem izdvojenih ili rekombinantnih DmGPCR produkata, DmGPCR inaica ili stanica s ekspresijom takvih produkata. Vezujui partneri korisni su za proiavanje DmGPCR produkata te detektiranje ili kvantificiranje produkata DmGPCR-a u tekuim ili tkivnim uzorcima pomou poznatih imunolokih postupaka. Vezujue molekule su takoer korisne u moduliranju (tj. blokiranju, inhibiranju ili stimuliranju) biolokih aktivnosti DmGPCR-a, posebice aktivnosti koje su ukljuene u prijenos signala. </span>Specific binding molecules, including natural ligands and synthetic compounds, can be identified or developed using isolated or recombinant DmGPCR products, DmGPCR variants, or cells expressing such products. The binding partners are useful for purifying DmGPCR products and detecting or quantifying DmGPCR products in liquid or tissue samples using known immunological procedures. Binding molecules are also useful in modulating (ie, blocking, inhibiting, or stimulating) the biological activities of DmGPCRs, particularly activities involved in signal transduction.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Informacija o DNA i aminokiselinskoj sekvenciji koja se dobije ovim izumom takoer omoguuje identificiranje spojeva vezujuih partnera s kojima je u interakciji DmGPCR polipeptid ili polinukleotid. Metode za identificiranje spojeva vezujueg partnera ukljuuju analize u otopini, in vitro analize gdje su DmGPCR polipeptidi imobilizirani i citoloki zasnovane analize. Identificiranje spojeva vezujuih partnera DmGPCR polipeptida odreuje kandidate za terapijsku ili profilaktiku intervenciju u patologijama koje su povezane s ektoparazitima s ekspresijom DmGPCR i kandidata za insekticide. </span>The information on DNA and amino acid sequence obtained by this invention also enables the identification of binding partner compounds with which the DmGPCR polypeptide or polynucleotide interacts. Methods for identifying binding partner compounds include assays in solution, in vitro assays where DmGPCR polypeptides are immobilized, and cytology-based assays. Identifying the binding partner compounds of DmGPCR polypeptides identifies candidates for therapeutic or prophylactic intervention in pathologies associated with DmGPCR-expressing ectoparasites and candidate insecticides.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Ovaj izum obuhvaa nekoliko sustava analiza za identificiranje DmGPCR vezujuih partnera. Ako se radi o analizi o otopini, metode ovog izuma ukljuuju sljedee stupnjeve: (a) dovoenje u dodir DmGPCR polipeptida s jednim ili vie spojeva kandidatskog vezujueg partnera i (b) identificiranje spojeva koji se veu za DmGPCR polipeptid. Identificiranje spojeva koji veu DmGPCR polipeptid moe se postii izdvajanjem kompleksa DmGPCR polipeptid/vezujui partner i izdvajanjem spoja vezujueg partnera od DmGPCR polipeptida. U drugoj realizaciji ovog izuma obuhvaen je dodatni stupanj karakteriziranja fizikih, biolokih i/ili biokemijskih svojstava spoja vezujueg partnera. U jednom aspektu, kompleks DmGPCR polipeptid/vezujui partner izdvaja se koritenjem antitijela koje je imunospecifino bilo za DmGPCR polipeptid ili spoj kandidatskog vezujueg partnera. </span>The present invention encompasses several assay systems for identifying DmGPCR binding partners. In the case of a solution assay, the methods of the present invention include the following steps: (a) contacting the DmGPCR polypeptide with one or more candidate binding partner compounds and (b) identifying the compounds that bind to the DmGPCR polypeptide. Identification of compounds that bind the DmGPCR polypeptide can be achieved by isolating the DmGPCR polypeptide/binding partner complex and isolating the binding partner compound from the DmGPCR polypeptide. In another embodiment of this invention, an additional step of characterizing the physical, biological and/or biochemical properties of the binding partner compound is included. In one aspect, the DmGPCR polypeptide/binding partner complex is isolated using an antibody that is immunospecific for either the DmGPCR polypeptide or the candidate binding partner compound.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">U drugim realizacijama, DmGPCR polipeptid ili spoj kandidatskog vezujueg partnera sadri oznaku ili tag to olakava njegovo izdvajanje i metode ovog izuma da se identificiraju spojevi vezujueg partnera ukljuuju stupanj izdvajanja kompleksa DmGPCR polipeptid/vezujui partner putem interakcije s oznakom ili tagom. Primjerni tag ovog tipa je poli-histidinska sekvencija, openito oko est histidinskih rezidua, koja omoguuje izdvajanje tako oznaenog spoja koritenjem keliranja niklom. Druge oznake i tagovi, kao to je FLAG tag (Eastman Kodak, Rochester, NY), koji su dobro poznati i rutinski se koriste u tehnici, obuhvaeni su ovim izumom. Oznake ovog izuma takoer ukljuuju ali nisu ograniene na radioaktivne oznake (npr. 125I, 35S, 32P, 33P, 3H), fluorescentne oznake, kemiluminscentne oznake, enzimske oznake i imunogene oznake. </span>In other embodiments, the DmGPCR polypeptide or candidate binding partner compound contains a label or tag that facilitates its isolation and the methods of the present invention to identify binding partner compounds include the step of isolating the DmGPCR polypeptide/binding partner complex via interaction with the label or tag. An exemplary tag of this type is a poly-histidine sequence, generally around six histidine residues, which allows isolation of the so-tagged compound using nickel chelation. Other labels and tags, such as the FLAG tag (Eastman Kodak, Rochester, NY), which are well known and routinely used in the art, are encompassed by the present invention. Labels of the present invention also include but are not limited to radioactive labels (eg, 125I, 35S, 32P, 33P, 3H), fluorescent labels, chemiluminescent labels, enzymatic labels, and immunogenic labels.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">U nekim realizacijama in vitro analiza, izumom su definirane metode koje ukljuuju sljedee stupnjeve: (a) dovoenje u dodir imobiliziranog DmGPCR polipeptida sa spojem kandidatskog vezujueg partnera i (b) detektiranje vezanja kandidatskog spoja za DmGPCR polipeptid. U alternativnoj realizaciji, spoj kandidatskog vezujueg partnera je imobiliziran i detektirano je vezanje DmGPCR-a. Imobiliziranje se vri bilo kojom metodom koja je poznata u tehnici, ukljuujui kovalentno vezanje za podlogu, nosa ili kromatografsku smolu, kao i nekovalentne, visoko afinitetne interakcije kao to je vezanje antitijelom ili koritenje vezanja streptavidin/biotin pri emu imobilizirani spoj ukljuuje abiotin speciju. Detektiranje vezanja moe se izvriti (i) koritenjem radioaktivne oznake na spoju koji nije imobiliziran, (ii) koritenjem fluorescentne oznake na imobiliziranom spoju, (iii) koritenjem antitijela imunospecifinog za neimobilizirani spoj, (iv) koritenjem oznake na imobiliziranom spoju koja ekscitira fluorescentnu podlogu na koju je dodan imobilizirani spoj, kao i druge tehnike koje su dobro poznate u rutinskom radu. </span>In some embodiments of in vitro assays, the invention defines methods that include the following steps: (a) contacting the immobilized DmGPCR polypeptide with a candidate binding partner compound and (b) detecting binding of the candidate compound to the DmGPCR polypeptide. In an alternative embodiment, the candidate binding partner compound is immobilized and DmGPCR binding is detected. Immobilization is performed by any method known in the art, including covalent binding to a substrate, support, or chromatographic resin, as well as non-covalent, high-affinity interactions such as antibody binding or using streptavidin/biotin binding wherein the immobilized compound includes an abiotin species. Detection of binding can be performed (i) using a radioactive label on the compound that is not immobilized, (ii) using a fluorescent label on the immobilized compound, (iii) using an antibody immunospecific for the non-immobilized compound, (iv) using a label on the immobilized compound that excites a fluorescent substrate on to which the immobilized compound has been added, as well as other techniques well known in routine work.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Izum takoer definira citoloki zasnovane analize da se identificiraju spojevi vezujueg partnera DmGPCR polipeptida. U jednoj realizaciji, izumom su definirane metode koje ukljuuju dovoenje u dodir DmGPCR polipeptida s ekspresijom na povrini stanice s kandidatskim spojem vezujueg partnera i detektiranje vezanja kandidatskog spoja vezujueg partnera za DmGPCR polipeptid. U sljedeoj realizaciji, detektiranje ukljuuje detektiranje kalcijeva fluksa ili drugih fiziolokih uinaka u stanici koji su izazvani vezanjem molekule. </span>The invention also defines cytology-based assays to identify binding partner compounds of DmGPCR polypeptides. In one embodiment, the invention defines methods that include contacting a cell surface-expressed DmGPCR polypeptide with a candidate binding partner compound and detecting binding of the candidate binding partner compound to the DmGPCR polypeptide. In another embodiment, detecting includes detecting calcium flux or other physiological effects in the cell that are induced by binding of the molecule.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">U sljedeoj realizaciji ovog izuma, koristi se high throughput screening za spojeve koji imaju odgovarajui vezujui afinitet za DmGPCR. Ukratko, sintetiziran je veliki broj razliitih testnih spojeva koji su maleni peptidi na vrstoj podlozi ili su slobodni spojevi otopljeni u odgovarajuem puferu. Testni peptidni spojevi su u dodiru s DmGPCR-om i ispiru se. Vezani DmGPCR se zatim detektira pomou metoda koje su poznate u tehnici. Proieni polipeptidi ovog izuma mogu se takoer postaviti izravno na ploe koje se rabe u prije navedenim vezujuim analizama. Nadalje, neneutralizirajua antitijela mogu se koristiti za hvatanje proteina i imobiliziranje na vrstoj podlozi. </span>In a further embodiment of the present invention, high throughput screening for compounds having appropriate binding affinity for DmGPCR is used. Briefly, a large number of different test compounds have been synthesized which are small peptides on a solid support or are free compounds dissolved in a suitable buffer. Test peptide compounds are in contact with DmGPCR and are washed away. Bound DmGPCR is then detected using methods known in the art. Purified polypeptides of the present invention can also be plated directly onto the plates used in the aforementioned binding assays. Furthermore, non-neutralizing antibodies can be used to capture proteins and immobilize them on a solid support.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Openito, DmGPCR s ekspresijom moe se koristiti za HTS vezujue analize u sprezi s definiranim ligandom. Identificirani peptid se obiljeava s odgovarajuim radioizotopom, ukljuujui, ali bez ogranienja, 125I, 3H, 35S ili 32P, pomou metoda koje su dobro poznate onima koji poznaju ovo podruje tehnike. Alternativno, peptidi mogu biti obiljeeni pomou dobro poznatih metoda s odgovarajuim fluorescentnim derivatom (Baindur et al., Drug Dev. Res., 1994, 33, 373-398; Rogers, Drug Discovery Today, 1997, 2, 156-160). Radioaktivni ligand koji je specifino vezan za receptor u membranskim pripravcima koji su nainjeni iz stanine linije koja vri ekspresiju rekombinantnog proteina, moe se detektirati u HTS analizama jednom od nekoliko standardnih naina, ukljuujui filtriranje kompleksa receptor-ligand da se odvoji vezani ligand od slobodnog liganda (Williams, Med. Res. Rev., 1991, 11, 147-184; Sweetnam et al., J. Natural Products, 1993, 56, 441-455). Alternativne metode ukljuuju analizu scintilacijske blizine (SPA) ili FlashPlate format kada takvo odvajanje nije neophodno (Nakayama, Curr. Opinion Drug Disc. Dev., 1998, 1, 85-91 Bosse et al., J. Biomolecular Screening, 1998, 3, 285-292). Vezanje fluorescentnih liganada moe se detektirati na razliite naine, ukljuujui fluorescentni prijenos energije (FRET), izravnu spektrofluorometrijsku analizu vezanog liganda ili fluorescentnu polarizaciju (Rogers, Drug Discovery Today, 1997, 2, 156-160; Hill, Curr. Opinion Drug Disc. Dev., 1998, 1, 92-97). </span>In general, an expressed DmGPCR can be used for HTS binding assays in conjunction with a defined ligand. The identified peptide is labeled with an appropriate radioisotope, including but not limited to 125 I, 3 H, 35 S or 32 P, using methods well known to those skilled in the art. Alternatively, peptides can be labeled using well-known methods with an appropriate fluorescent derivative (Baindur et al., Drug Dev. Res., 1994, 33, 373-398; Rogers, Drug Discovery Today, 1997, 2, 156-160). Radioactive ligand that is specifically bound to a receptor in membrane preparations made from a cell line expressing the recombinant protein can be detected in HTS assays by one of several standard methods, including filtering the receptor-ligand complex to separate bound ligand from free ligand ( Williams, Med. Res. Rev., 1991, 11, 147-184; Sweetnam et al., J. Natural Products, 1993, 56, 441-455). Alternative methods include scintillation proximity analysis (SPA) or FlashPlate format when such separation is not necessary (Nakayama, Curr. Opinion Drug Disc. Dev., 1998, 1, 85-91 Bosse et al., J. Biomolecular Screening, 1998, 3, 285-292). Binding of fluorescent ligands can be detected in a variety of ways, including fluorescence energy transfer (FRET), direct spectrofluorometric analysis of the bound ligand, or fluorescence polarization (Rogers, Drug Discovery Today, 1997, 2, 156-160; Hill, Curr. Opinion Drug Disc. Dev ., 1998, 1, 92-97).</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Za identificiranje specifinih liganada DmGPCR-a mogu se koristiti druge analize, ukljuujui analize za identificiranje liganada ciljanog proteina putem mjerenja izravnog vezanja testnih liganada za ciljani protein, kao i analize za identificiranje liganada ciljanih proteina putem afinitetnog ultrafiltriranja s ionskom sprej masenom spektroskopijom/HPLC metodama te drugim fizikim i analitikim metodama. Alternativno, takve vezujue interakcije mogu se procijeniti posredno koritenjem kvaeva dvohibridnog sustava koji je opisan u Fields et al. (Nature, 1989, 340, 245-246) te Fields et al. (Trends in Genetics, 1994, 10, 286-292), a obje te reference su ovdje ukljuene. Dvohibridni sustav je genetska analiza za detektiranje interakcija izmeu dva proteina ili polipeptida. Moe se koristiti za identificiranje proteina koji se veu za poznati protein od interesa ili za domena ili rezidua koje su kritine za interakciju. Razvijene su inaice ove metodologije da se kloniraju geni koji kodiraju DNA vezujue proteine, da se identificiraju peptidi koji se veu za protein i za provjeru lijekova. Dvohibridni sustav koristi sposobnost para interakcijskih proteina da dovedu transkripcijsku aktivacijsku domenu vrlo blizu DNA vezujuoj domeni koja se vee za uzvodnu aktivacijsku sekvenciju (UAS) reporterskog gena i openito se provodi u kvascima. Analiza zahtijeva konstrukciju dviju hibridnih gena koji kodiraju (1) DNA-vezujuu domenu koja je sjedinjena s prvim proteinom i (2) aktivacijsku domenu koja je sjedinjena s drugim proteinom. DNA-vezujua domena pogaa prvi hibridni protein UAS reporterskog gena. Meutim, budui da veini proteina nedostaje aktivacijska domena, ovaj DNA-vezujui hibridni protein ne aktivira transkripciju reporterskog gena. Drugi hibridni protein, koji sadri aktivacijsku domenu, ne moe sam aktivirati ekspresiju reporterskog gena budui da on ne vee UAS. Meutim, ako su prisutna oba hibridna proteina, nekovalentna interakcija prvog i drugog proteina sputava aktivacijsku domenu prema UAS, aktiviranjem transkripcije reporterskog gena. Primjerice, kada je prvi protein produkt DmGPCR gena ili njegov fragment, za kojega je poznato da je u interakciji s drugim proteinom ili nukleinskom kisleinom, ova analiza moe se koristiti za detektiranje sredstava koja intereferiraju s vezujuom interakcijom. Prati se ekspresija reporterskog gena kako se dodaju u sustav razliita sredstva koja se ispituju. Prisutnost inhibitorskog sredstva rezultira odsutnou reporterskog signala. </span>Other assays can be used to identify specific DmGPCR ligands, including assays to identify target protein ligands by measuring direct binding of test ligands to the target protein, as well as assays to identify target protein ligands by affinity ultrafiltration with ion spray mass spectroscopy/HPLC methods, and other physical and analytical methods. Alternatively, such binding interactions can be assessed indirectly using the yeast two-hybrid system described in Fields et al. (Nature, 1989, 340, 245-246) and Fields et al. (Trends in Genetics, 1994, 10, 286-292), and both of these references are incorporated herein. The two-hybrid system is a genetic analysis for detecting interactions between two proteins or polypeptides. It can be used to identify proteins that bind to a known protein of interest or to domains or residues that are critical for the interaction. Versions of this methodology have been developed to clone genes encoding DNA-binding proteins, to identify protein-binding peptides, and for drug screening. The two-hybrid system uses the ability of a pair of interacting proteins to bring the transcriptional activation domain very close to the DNA binding domain that binds to the 'upstream' activation sequence (UAS) of the reporter gene and is generally carried out in yeast. The assay requires the construction of two hybrid genes encoding (1) a DNA-binding domain fused to the first protein and (2) an activation domain fused to the second protein. The DNA-binding domain hits the first hybrid protein of the UAS reporter gene. However, since most proteins lack an activation domain, this DNA-binding hybrid protein does not activate transcription of the reporter gene. The second hybrid protein, which contains the activation domain, cannot activate reporter gene expression by itself since it does not bind the UAS. However, if both hybrid proteins are present, the non-covalent interaction of the first and second proteins restrains the activation domain towards the UAS, activating transcription of the reporter gene. For example, when the first protein is a DmGPCR gene product or fragment thereof, which is known to interact with another protein or nucleic acid, this assay can be used to detect agents that interfere with the binding interaction. The expression of the reporter gene is monitored as the various test agents are added to the system. The presence of an inhibitory agent results in the absence of a reporter signal.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Kada je funkcija produkta DmGPCR gena nepoznata i nema poznatih liganada za vezanje s genskim produktom, moe se takoer koristiti kvaeva dvohibridna analiza da se identificiraju proteini koji se veu za genski produkt. U analizi da se identificiraju proteini koji se veu za DmGPCR receptor ili njegov fragment, moe se koristiti fuzijski polinukleotid koji kodira i DmGPCR receptor (ili fragment) i UAS vezujuu domenu (tj. prvi protein). Nadalje, u analizi proizvodi se i provjerava veliki broj hibridnih gena od kojih svaki kodira razliiti sekundarni protein koji je sjedinjen s aktivacijskom domenom. Tipino, drugi protein se kodira s jednim ili vie lanova fuzijske biblioteke ukupne cDNA ili genomske DNA., pri emu je svako kodirajue podruje drugog gena sjedinjeno s aktivacijskom domenom. Ovaj sustav moe se primijeniti za iroki raspon proteina i nije ak nuno poznavati identinost ili funkciju drugog vezujueg proteina. Sustav je vrlo osjetljiv i moe detektirati interakcije koje se ne mogu uoiti drugim metodama. ak i prijelazne interakcije mogu pokrenuti transkripciju da se dobije postojana mRNA koja se moe opetovano prevesti da se dobije reporterski protein. </span>When the function of the DmGPCR gene product is unknown and there are no known ligands to bind to the gene product, yeast two-hybrid analysis can also be used to identify proteins that bind to the gene product. In an assay to identify proteins that bind to a DmGPCR receptor or fragment thereof, a fusion polynucleotide encoding both the DmGPCR receptor (or fragment) and the UAS binding domain (ie, the first protein) can be used. Furthermore, a large number of hybrid genes, each encoding a different secondary protein fused to an activation domain, are produced and screened in the assay. Typically, the second protein is encoded by one or more members of a fusion library of total cDNA or genomic DNA, wherein each coding region of the second gene is fused to an activation domain. This system can be applied to a wide range of proteins and it is not even necessary to know the identity or function of the other binding protein. The system is very sensitive and can detect interactions that cannot be detected by other methods. Even transient interactions can initiate transcription to produce a stable mRNA that can be repeatedly translated to produce a reporter protein.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">U potrazi za sredstvima koja se veu za ciljani protein mogu se koristiti druge analize. Jedna takva screening metoda da se identificira izravno vezanje testnih liganada za ciljani protein opisana je u amerikom patentu br. 5, 585, 277, koji je ovdje ukljuen kao referenca. Ova metoda temelji se na naelu da proteini openito postoje kao smjesa savijenog i nesavijenog stanja i kontinuirano alterniraju izmeu ovih dviju stanja. Kada se testni ligand vee za savijeni oblik ciljanog proteina (tj. kada je testni ligand ligand ciljanog proteina), molekula ciljanog proteina koja je vezana s ligandom ostaje u savijenom stanju. Dakle, savijeni ciljani protein nalazi se u veem dosegu u prisutnosti ciljanog spoja koji savija ciljani protein, nego u odsutnosti liganda. Vezanje liganda za ciljani protein moe se odrediti nekom metodom kojom se mogu razluiti savijeno i nesavijeno stanje ciljanog proteina. Funkcija ciljanog proteina ne mora biti poznata da bi se mogla izvriti ova analiza. Ovom metodom se kao ispitivani ligand moe ispitati bilo koje sredstvo, ukljuujui bez ogranienja metale, polipeptide, proteine, lipide, polisaharide, polinukleotide i malene organske molekule. </span>Other assays can be used to search for agents that bind to the target protein. One such screening method to identify direct binding of test ligands to a target protein is described in US Pat. 5, 585, 277, which is incorporated herein by reference. This method is based on the principle that proteins generally exist as a mixture of folded and unfolded states and continuously alternate between these two states. When the test ligand binds to the folded form of the target protein (ie, when the test ligand is a ligand of the target protein), the target protein molecule that is bound to the ligand remains in the folded state. Thus, the folded target protein is in greater reach in the presence of a targeting compound that folds the target protein, than in the absence of the ligand. Ligand binding to the target protein can be determined by a method that can distinguish between the folded and unfolded states of the target protein. The function of the target protein does not need to be known to perform this analysis. With this method, any agent can be tested as a test ligand, including but not limited to metals, polypeptides, proteins, lipids, polysaccharides, polynucleotides, and small organic molecules.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Druga metoda za identificiranje liganda ciljanog proteina opisana je u: Wieboldt et al. (Anal. Chem., 1997, 69, 1683-1691), to je ovdje ukljueno kao referenca. Ova tehnika koristi kombinatorne biblioteke od 20-30 sredstava istovremeno u tekuoj fazi za vezanje za ciljani protein. Sredstva koja se veu za ciljani protein odvajaju se od ostalih komponenti biblioteke pomou jednostavne tehnike membranskog ispiranja. Specifino odabrane molekule koje su zadrane na filteru kasnije se oslobaaju od ciljanog proteina i analiziraju pomou HPLC i plinski potpomognute elektrosprej (ion sprej) ionizacijske masene spektroskopije. Ovim postupkom odabiru se komponente biblioteke s visokim afinitetom za ciljani protein i posebno je koristan za biblioteke malenih molekula. </span>Another method for identifying a target protein ligand is described in: Wieboldt et al. (Anal. Chem., 1997, 69, 1683-1691), which is incorporated herein by reference. This technique uses combinatorial libraries of 20-30 agents simultaneously in the liquid phase to bind to the target protein. Agents that bind to the target protein are separated from other library components using a simple membrane washing technique. The specifically selected molecules retained on the filter are later released from the target protein and analyzed by HPLC and gas-assisted electrospray (ion spray) ionization mass spectroscopy. This procedure selects library components with high affinity for the target protein and is particularly useful for small molecule libraries.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Druge realizacije ovog izuma obuhvaaju koritenje kompetitivnih screening analiza u kojima neutralizirajua antitijela koja mogu vezati polipeptid ovog izuma specifino kompetiraju s ispitivanim spojem u vezanju za polipeptid. Na taj nain, mogu se koristiti antitijela za detektiranje prisutnosti bilo kakvog peptida koji dijeli jednu ili vie antigenih determinanti s DmGPCR. Istraivanja radioobiljeenog kompetitivnog vezanja opisana su u A.H. Lin et al. (Antimicrobial Agents and Chemotherapy, 1997, 41(10), 2127-2131), a opis je ovdje ukljuen kao referenca u cijelosti. </span>Other embodiments of this invention include the use of competitive screening assays in which neutralizing antibodies capable of binding the polypeptide of this invention specifically compete with the test compound in binding to the polypeptide. In this way, antibodies can be used to detect the presence of any peptide that shares one or more antigenic determinants with the DmGPCR. Radiolabeled competitive binding studies are described in A.H. Lin et al. (Antimicrobial Agents and Chemotherapy, 1997, 41(10), 2127-2131), the disclosure of which is incorporated herein by reference in its entirety.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Metode identificiranja modulirajuih sredstava </span>Methods of identifying modulating agents</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Ovim izumom su takoer definirane metode za identificiranje modulatora vezanja izmeu DmGPCR i DmGPCR vezujueg partnera, to ukljuuje sljedee stupnjeve: (a) dovoenje u dodir DmGPCR vezujueg partnera i smjese koja sadri DmGPCR u prisutnosti i odsutnosti potencijalnog modulatorskog spoja; (b) detektiranje vezanja izmeu vezujueg partnera i DmGPCR; te (c) identificiranje potencijalnog modulatorskog spoja ili modulatorakog spoja u svjetlu smanjenog ili poveanog vezanja izmeu vezujueg partnera i DmGPCR u prisutnosti potencijalnog modulatora, u odnosu na vezanje u odsutnosti potencijalnog modulatora. </span>The present invention also defines methods for identifying binding modulators between a DmGPCR and a DmGPCR binding partner, comprising the following steps: (a) contacting the DmGPCR binding partner and a mixture containing the DmGPCR in the presence and absence of a potential modulatory compound; (b) detecting binding between the binding partner and the DmGPCR; and (c) identifying a potential modulatory compound or modulatory compound in light of decreased or increased binding between the binding partner and the DmGPCR in the presence of the potential modulator, relative to binding in the absence of the potential modulator.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">DmGPCR vezujui partneri koji stimuliraju aktivnost DmGPCR korisni su kao agonisti u uvjetima koji su karakterizirani nedovoljnim DmGPCR signaliziranjem (npr. kao rezultat nedovoljne aktivnosti DmGPCR liganda). DmGPCR vezujui partneri koji blokiraju ligandom upravljano DmGPCR signaliziranje korisni su kao DmGPCR antagonisti u uvjetima kojima je svojstveno prekomjerno DmGPCR signaliziranje. Nadalje, DmGPCR modulatori openito, kao i DmGPCR polinukleotidi i polipeptidi, korisni su za dijagnostike analize bolesti izazvanih ektoparazitima ili stanjima u kojima je DmGPCR aktivnost poveana ili oteena. </span>DmGPCR binding partners that stimulate DmGPCR activity are useful as agonists in conditions characterized by insufficient DmGPCR signaling (eg, as a result of insufficient DmGPCR ligand activity). DmGPCR binding partners that block ligand-gated DmGPCR signaling are useful as DmGPCR antagonists in conditions characterized by excessive DmGPCR signaling. Furthermore, DmGPCR modulators in general, as well as DmGPCR polynucleotides and polypeptides, are useful for diagnostic assays of diseases caused by ectoparasites or conditions in which DmGPCR activity is increased or impaired.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">U sljedeem aspektu, izumom su definirane metode za tretiranje bolesti ili stanja primjenom subjektu kojemu je potreban takav tretman tvari koja modulira aktivnost ili ekspresiju polipeptida koji ima sekvenciju koja je odabrana izmeu onih sa sekvencijskim vrojevima 2, 4, 6, 8, 10, 12, 14, 16, 18, 20, 22 i 24. </span>In a further aspect, the invention defines methods for treating a disease or condition by administering to a subject in need of such treatment a substance that modulates the activity or expression of a polypeptide having a sequence selected from those with sequence numbers 2, 4, 6, 8, 10, 12, 14, 16, 18, 20, 22 and 24.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">U sljedeem aspektu, izumom su definirane metode za kontroliranje populacije insekata primjenom na populaciju insekata vezujueg partnera ili modulatora koji modificira ekspresiju ili aktivnost DmGPCR. </span>In a further aspect, the invention defines methods for controlling an insect population by applying to the insect population a binding partner or modulator that modifies the expression or activity of DmGPCR.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Sredstva koja moduliraju (tj. poveavaju, smanjuju ili blokiraju) aktivnost ili ekspresiju DmGPCR mogu biti identificirana inkubiranjem potencijalnog modulatora sa stanicom koja sadri DmGPCR polipeptid ili polinukleotid te odreivanjem uinka potencijalnog modulatora na aktivnost ili ekspresiju DmGPCR. Selektivnost spoja koji modulira aktivnost DmGPCR moe se procijeniti usporeivanjem njegovih uinaka na DmGPCR prema njegovim uincima na druge GPCR spojeve. Selektivni modulatori mogu primjerice obuhvaati antitijela i druge proteine, peptide i organske molekule koje se specifino veu za DmGPCR polipeptid ili DmGPCR-kodirajuu nukleinsku kiselinu. Modulatori aktivnosti DmGPCR bit e terapijski korisni u tretiranju bolesti i fiziolokih stanja u kojima je ukljuena normalna ili nenormalna aktivnost DmGPCR. </span>Agents that modulate (ie, increase, decrease, or block) DmGPCR activity or expression can be identified by incubating the potential modulator with a cell containing the DmGPCR polypeptide or polynucleotide and determining the effect of the potential modulator on DmGPCR activity or expression. The selectivity of a compound that modulates DmGPCR activity can be assessed by comparing its effects on DmGPCR to its effects on other GPCR compounds. Selective modulators can for example include antibodies and other proteins, peptides and organic molecules that specifically bind to the DmGPCR polypeptide or DmGPCR-encoding nucleic acid. Modulators of DmGPCR activity will be therapeutically useful in treating diseases and physiological conditions in which normal or abnormal DmGPCR activity is involved.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">DmGPCR polinukleotidi i polipeptidi, kao i DmGPCR modulatori, mogu se takoer koristiti u dijagnostikim analizama za bolesti izazvane ektoparazitima ili stanjima koja su karakterizirana poveanom ili oteenom aktivnou DmGPCR. </span>DmGPCR polynucleotides and polypeptides, as well as DmGPCR modulators, can also be used in diagnostic assays for diseases caused by ectoparasites or conditions characterized by increased or impaired DmGPCR activity.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Metode ovog izuma za identificranje modulatora obuhvaaju varijacije bilo koje prije opisane metode da se identificiraju spojevi koji su vezujui partneri, a varijacije obuhvaaju tehnike gdje je identificiran spoj-vezujui partner i vezujua analiza se vri u prisutnosti ili odsutnosti kandidatskog modulatora. Modulator se identificira u onim okolnostima kada se vezanje izmeu DmGPCR polipeptida i vezujueg partnera mijenja u prisutnosti kandidatskog modulatora u odnosu na vezanje u odsutnosti kandidatskog modulatorskog spoja. Modulator koji poveava vezanje izmeu DmGPCR polipeptida i vezujueg partnerskog spoja naziva se pojaiva ili aktivator dok se spoj koji smanjuje vezanje izmeu DmGPCR polipeptida i vezujueg partnerskog spoja naziva inhibitor. </span>The methods of the present invention for identifying modulators include variations of any previously described method to identify compounds that are binding partners, and variations include techniques where a compound-binding partner is identified and binding analysis is performed in the presence or absence of a candidate modulator. A modulator is identified in those circumstances where the binding between the DmGPCR polypeptide and the binding partner is altered in the presence of the candidate modulator relative to binding in the absence of the candidate modulator compound. A modulator that increases binding between a DmGPCR polypeptide and a binding partner compound is called an enhancer or activator, while a compound that decreases binding between a DmGPCR polypeptide and a binding partner compound is called an inhibitor.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Izumom su takoer obuhvaene high-throughput screening (HTS) analize s ciljem da se identificiraju spojevi koji su u interakciji s biolokom aktivnou ili inhibiraju bioloku aktivnost (tj. djeluju na enzimsku aktivnost, aktivnost vezanja itd.) DmGPCR polipeptida. HTS analize omoguuju screening velikog broja spojeva na djelotvoran nain. Stanino zasnovani HTS sustavi namijenjeni su ispitivanju DmGPCR receptor-ligand interakcije. HTS analize oblikovane su da se identificiraju pogoci ili vodei spojevi koji imaju eljeno svojstvo, s kojima se mogu nainiti modifikacije da se pojaa eljeno svojstvo. Kemijsko modificiranje pogotka ili vodeeg spoja esto se zasniva na odnosu identificirajue strukture i aktivnosti izmeu pogotka i DmGPCR polipeptida. </span>The invention also covers 'high-throughput screening' (HTS) analyzes with the aim of identifying compounds that interact with the biological activity or inhibit the biological activity (i.e. affect enzymatic activity, binding activity, etc.) of DmGPCR polypeptides. HTS analyzes enable the screening of a large number of compounds in an efficient manner. Cell-based HTS systems are intended for testing DmGPCR receptor-ligand interaction. HTS assays are designed to identify hits or lead compounds that have a desired property, with which modifications can be made to enhance the desired property. Chemical modification of the "hit" or "lead compound" is often based on the identifying structure-activity relationship between the "hit" and the DmGPCR polypeptide.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Modulatori koji spadaju unutar dosega ovog izuma obuhvaaju, ali nisu ogranieni na nepeptidne molekule kao to su nepeptidni mimetici, nepeptidni alosteriki efektori i peptidi. DmGPCR polipeptid ili polinukleotid koji se koristi u takvom ispitivanju moe biti ili slobodan u otopini, vezan za vrsti nosa, smjeten na staninoj povrini ili unutar stanice ili vezan za dio stanice. Onaj tko poznaje ovo podruje tehnike moe primjerice mjeriti nastajanje kompleksa izmeu DmGPCR i spoja koji se ispituje. Alternativno, onaj tko poznaje ovo podruje tehnike moe ispitati umanjenje nastajanja kompleksa izmeu DmGPCR i njegova supstrata pomou spoja koji se ispituje. </span>Modulators that fall within the scope of this invention include, but are not limited to non-peptide molecules such as non-peptide mimetics, non-peptide allosteric effectors and peptides. The dmGPCR polypeptide or polynucleotide used in such an assay can be either free in solution, attached to a solid support, located on the cell surface or inside the cell, or attached to a part of the cell. One skilled in the art can, for example, measure the formation of a complex between DmGPCR and the compound being tested. Alternatively, one of ordinary skill in the art can examine the reduction of complex formation between the DmGPCR and its substrate by the test compound.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Daljnji aspekt ovog izuma odnosi se na metode identificiranja spojeva koji moduliraju (tj. poveavaju ili smanjuju) aktivnost DmGPCR to obuhvaa dovoenje u dodir DmGPCR sa spojem i odreivanje mijenja li spoj DmGPCR. Aktivnost u prisutnosti ispitivanog spoja usporeuje se s aktivnou u odsutnosti ciljanog spoja. Kada je aktivnost uzorka koji sadri ispitivani spoj vea od aktivnosti uzorka bez ispitivanog spoja, spoje e imati poveanu aktivnost. Slino, ako je aktivnost uzorka koji sadri ispitivani spoj manja od aktivnosti u uzorku u kojem nema ispitivanog spoja, spoj e imati inhibiranu aktivnost. </span>A further aspect of the present invention relates to methods of identifying compounds that modulate (ie, increase or decrease) DmGPCR activity comprising contacting the DmGPCR with the compound and determining whether the compound alters the DmGPCR. The activity in the presence of the test compound is compared to the activity in the absence of the target compound. When the activity of the sample containing the test compound is greater than the activity of the sample without the test compound, the compounds will have increased activity. Similarly, if the activity of a sample containing the test compound is less than the activity of a sample without the test compound, the compound will have inhibited activity.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Ovaj izum je osobito koristan za screening spojeva pomou DmGPCR u cijelom nizu analiza aktivnosti. Spoj koji se podvrgava screening-u obuhvaa (to moe ukljuivati spojeve za koje s eoekuje da moduliraju aktivnost DmGPCR), ali bez ogranienja, spojeve izvanstaninog, unutarstaninog, kemijskog ili biolokog porijekla. DmGPCR polipeptid koji se koristi u takvom testu moe biti u bilo kojem obliku, kao to je slobodan u otopini, vezan za vrsti nosa, smjeten na povrini stanice ili unutar stanice. Onaj tko poznaje ovo podruje tehnike moe primjerice mjeriti nastajanje kompleksa izmeu DmGPCR i spoja koji se ispituje. Alternativno, onaj tko poznaje ovo podruje tehnike moe ispitati umanjenje nastajanja kompleksa izmeu DmGPCR i njegova supstrata to je izazvano spojem koji se ispituje. </span>The present invention is particularly useful for screening compounds using DmGPCR across a range of activity assays. A compound to be screened includes (which may include compounds expected to modulate DmGPCR activity), but is not limited to compounds of extracellular, intracellular, chemical or biological origin. The dmGPCR polypeptide used in such an assay can be in any form, such as free in solution, attached to a solid support, located on the cell surface, or inside the cell. One skilled in the art can, for example, measure the formation of a complex between DmGPCR and the compound being tested. Alternatively, one skilled in the art can examine the reduction of complex formation between the DmGPCR and its substrate induced by the test compound.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Aktivnost DmGPCR polipeptida ovog izuma moe se primjerice odrediti ispitivanjem sposobnosti da se vee ili aktivira pomou kemijski sintetiziranih peptidnih liganada. Alternativno, aktivnost DmGPCR-ova moe se analizirati ispitivanjem njihove sposobnosti da veu kalcijeve ione, hormone, kemokine, neuropeptide, neurotransmitere, nukleotide, lipide, odorante i fotone. Alternativno, aktivnost DmGPCR-ova moe se odrediti ispitvanjem aktivnosti efektorskih molekula to ukljuuje, ali nije ogranieno na adeniat ciklazu, fosfolipaze i ionske kanale. Dakle, modulatori aktivnosti DmGPCR mogu promijeniti funkciju DmGPCR receptora, kao to je svojstvo vezanja receptora ili aktivnost kao to je prijenos signala kojim upravlja G protein ili membranska lokalizacija. U razliitim realizacijama ovog izuma, analiza moe biti u obliku analize ionskog fluksa, analize nehidrolizirajueg GTP analiza kao to je analiza [35S]GTP5, analiza cAMP, analiza inozitol trifosfata, analiza diacilglicerola, analiza ekvorina (Aequorin), analiza luciferase, analiza FLJPR za unutarstaninu koncentraciju Ca2+, analiza mitogeneze, analiza aktivnosti MAP kinaze, analiza otputanja arahidonske kiseline (npr. pomou [3H]-arahidonske kiseline) i analiza brzine izvanstaninog zakiseljavanja, kao i ostale vezujue ili funkcionalno zasnovane analize aktivnosti DmGPCR koje su openito poznate u tehnici. U nekoliko ovih realizacija, izumom su obuhvaena ukljuivanja bilo kojih G proteina koji su poznati u tehnici, kao to su G16, G15 ili kimerni Gqj5, Gqs5, Gqo5, Gqz5 i slino. Aktivnost DmGPCR moe se odrediti metodologijama koje se rabe za analizu aktivnosti FaRP, to je dobro poznato onima koji poznaju ovo podruje tehnike. Bioloke aktivnosti DmGPCR receptora sukladno ovom izumu obuhvaaju, ali bez ogranienja: vezanje prirodnih ili umjetnih liganada, kao i bilo koju od funkcionalnih aktivnosti GPCR-ova poznatih u tehnici. Neograniavajui primjeri GPCR aktivnosti obuhvaaju transmembransko signaliziranje razliitih oblika, to moe ukljuivati povezivanje G proteina i/ili ispoljavanje utjecaja na vezanje G proteina razliitih guanidilatnih nukleotida. Drugi primjer aktivnosti GPCR-ova je vezanje pomonih proteina ili polipeptida koji se razlikuju od poznatih G proteina. </span>The activity of a DmGPCR polypeptide of the present invention can be determined, for example, by testing its ability to bind or activate chemically synthesized peptide ligands. Alternatively, the activity of DmGPCRs can be analyzed by examining their ability to bind calcium ions, hormones, chemokines, neuropeptides, neurotransmitters, nucleotides, lipids, odorants, and photons. Alternatively, the activity of DmGPCRs can be determined by assaying the activity of effector molecules including but not limited to adenate cyclase, phospholipases, and ion channels. Thus, modulators of DmGPCR activity can alter DmGPCR receptor function, such as receptor binding property or activity such as G protein-directed signal transduction or membrane localization. In various embodiments of the present invention, the assay may be in the form of an ion flux assay, a non-hydrolyzable GTP assay such as [35S]GTP5 assay, cAMP assay, inositol triphosphate assay, diacylglycerol assay, Aequorin assay, luciferase assay, FLJPR assay for intracellular Ca2+ concentration, mitogenesis assay, MAP kinase activity assay, arachidonic acid release assay (eg, using [3H]-arachidonic acid) and extracellular acidification rate assay, as well as other binding or functional-based assays of DmGPCR activity generally known in the art. In several of these embodiments, the invention includes the inclusion of any G proteins known in the art, such as G16, G15 or chimeric Gqj5, Gqs5, Gqo5, Gqz5 and the like. DmGPCR activity can be determined by methodologies used to analyze FaRP activity, which are well known to those skilled in the art. Biological activities of DmGPCR receptors according to this invention include, but are not limited to: binding of natural or artificial ligands, as well as any of the functional activities of GPCRs known in the art. Non-limiting examples of GPCR activity include transmembrane signaling of various forms, which may include binding to G proteins and/or exerting an effect on G protein binding of various guanidylate nucleotides. Another example of the activity of GPCRs is the binding of accessory proteins or polypeptides that differ from known G proteins.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Modulatori ovog izuma su razliitih kemijskih struktura, koje se mogu openito grupirati u nepeptidne mimetike prirodnih DmGPCR receptorskih liganada, peptidne i ne-peptidne alosterike efektore DmGPCR receptora te peptide koji mogu djelovati kao aktivatori ili inhibitori (kompetitivni, dekompetitivni i nekompetitivni) (npr. antitijela) DmGPCR receptora. Ovaj izum ne ograniuje izvore odgovarajuih modulatora, koji se mogu dobiti iz prirodnih izvora kao to su biljke, ivotinje i mineralni ekstrakti ili umjetni izvori kao to su biblioteke malenih molekula, ukljuujui produkte kombinatornih kemijskih pristupa izgradnji biblioteka i peptidnim bibliotekama. Primjeri peptidnih modulatora DmGPCR receptora pokazuju sljedee primarne strukture: GLGPRPLRFamid (sekv. br. 49), GNSFLRFamid (sekv. br. 136), GGPQGPLRFamid (sekv. br. 102), GPSGPLRFamid (sekv. br. 103), PDVDHVFLRFamid (sekv. br. 150) i piro-EDVDHVFLRFamid (sekv. br. 167). </span>The modulators of this invention are of different chemical structures, which can generally be grouped into non-peptide mimetics of natural DmGPCR receptor ligands, peptide and non-peptide allosteric effectors of DmGPCR receptors, and peptides that can act as activators or inhibitors (competitive, decompetitive and non-competitive) (e.g. antibodies ) of DmGPCR receptors. The present invention does not limit the sources of suitable modulators, which may be obtained from natural sources such as plant, animal and mineral extracts or artificial sources such as small molecule libraries, including products of combinatorial chemical approaches to library construction and peptide libraries. Examples of DmGPCR receptor peptide modulators show the following primary structures: GLGPRPLRFamide (SEQ ID NO: 49), GNSFLRFamide (SEQ ID NO: 136), GGPQGPLRAFamide (SEQ ID NO: 102), GPSGPLRFamide (SEQ ID NO: 103), PDVDHVFLRFamide (SEQ ID NO: 103). no. 150) and pyro-EDVDHVFLRFamide (seq. no. 167).</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Za ispitivanje enzimske aktivnosti mogu se koristiti druge analize to obuhvaa, ali bez ogranienja fotometrijske, radiometrijske, HPLC, elektrokemijske i sline, to je primjerice opisano u Enzyme Assays: Practical Approach, eds. R. Eisenthal and M. J. Danson, 1992, Oxford University Press, to je ovdje ukljueno kao referenca u cijelosti. </span>Other assays may be used to test enzyme activity, including, but not limited to, photometric, radiometric, HPLC, electrochemical, and the like, as described for example in Enzyme Assays: Practical Approach, eds. R. Eisenthal and M. J. Danson, 1992, Oxford University Press , which is incorporated herein by reference in its entirety.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Koritenje cDNA koje kodiraju GPCR-ove u analizi aktivnosti dobro je poznato. Analize kojima se moe ispitati tisue nepoznatih spojeva dnevno u high-throughput screens analizama (HTS-ovi) su detaljno dokumentirane. U literaturi se mogu nai primjeri upotrebe radioobiljeenih liganada u HTS vezujuim analizama u otkrivanju lijeka (vidi Williams, Medicinal Research Reviews, 1991, 11, 147-184; Sweetnam, et al, J. Natural Products, 1993, 56, 441-455 za pregled). Rekombinantni receptori poeljni su za HTS vezujuu analizu jer omoguuju bolju specifinost (vea relativna istoa), ime se postie sposobnost da se generiraju velike koliine receptorskog materijala koji se moe koristiti u cijelom nizu formata (vidi Hodgson, Bio/Technology, 1992,10, 973-980, ukljueno ovdje kao referenca u cijelosti). </span>The use of cDNAs encoding GPCRs in activity analysis is well known. Analyzes capable of testing thousands of unknown compounds per day in high-throughput screens (HTSs) are documented in detail. Examples of the use of radiolabeled ligands in HTS binding assays in drug discovery can be found in the literature (see Williams, Medicinal Research Reviews, 1991, 11, 147-184; Sweetnam, et al, J. Natural Products, 1993, 56, 441-455 for review). Recombinant receptors are preferred for HTS binding assays because they allow for better specificity (higher relative purity), thereby enabling the ability to generate large quantities of receptor material that can be used in a variety of formats (see Hodgson, Bio/Technology, 1992, 10, 973 -980, incorporated herein by reference in its entirety).</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Postoje brojni heterologni sustavi za funkcionalnu ekspresiju rekombinantnih receptora to je dobro poznato onima koji poznaju ovo podruje. Takvi sustavi obuhvaaju bakterije (Strosberg, et al, Trends in Pharmacological Sciences, 1992, 13, 95-98), kvasce (Pausch, Trends in Biotechnology, 1997, 15, 487-494), nekoliko vrsta insektnih stanica (Vanden Broeck, Int. Rev. Cytology, 1996,164, 189-268), amfibijske stanice (Jayawickreme et al, Curr. Opin. Biotechnol, 1997, 8, 629-634) i nekoliko staninih linija sisavaca (CHO, HEK293, COS, itd., vidi Gerhardt, et al, Eur. J. Pharmacology, 1997, 334, 1-23). Ovi primjeri ne iskljuuju koritenje drugih moguih sustava stanine ekspresije, ukljuujui stanine linije koje su dobivene iz nematoda (PCT prijava WO 98/37177). </span>Numerous heterologous systems exist for the functional expression of recombinant receptors as are well known to those skilled in the art. Such systems include bacteria (Strosberg, et al, Trends in Pharmacological Sciences, 1992, 13, 95-98), yeasts (Pausch, Trends in Biotechnology, 1997, 15, 487-494), several types of insect cells (Vanden Broeck, Int Rev. Cytology, 1996, 164, 189-268), amphibian cells (Jayawickreme et al, Curr. Opin. Biotechnol, 1997, 8, 629-634) and several mammalian cell lines (CHO, HEK293, COS, etc., see Gerhardt, et al, Eur. J. Pharmacology, 1997, 334, 1-23). These examples do not preclude the use of other possible cell expression systems, including cell lines derived from nematodes (PCT application WO 98/37177).</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">U nekim realizacijama ovog izuma, metode za screening spojeva koji moduliraju aktivnost DmGPCR ukljuuju dovoenje u meusoban dodir ispitivanih spojeva s DmGPCR te analizu na prisutnost kompleksa spoja i DmGPCR. U takvoj analizi, ligand je tipino obiljeen. Nakon odgovarajueg inkubiranja, slobodni ligand se odvaja od vezanoga te je koliina slobodnog ili nekompleksiranoga mjera sposobnosti odreenog spoja da se vee za DmGPCR. </span>In some embodiments of the present invention, methods for screening compounds that modulate DmGPCR activity include contacting the test compounds with DmGPCR and analyzing for the presence of a complex of the compound and DmGPCR. In such an analysis, the ligand is typically labeled. After appropriate incubation, the free ligand is separated from the bound one, and the amount of free or uncomplexed is a measure of the ability of a particular compound to bind to DmGPCR.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Poznato je da aktiviranje heterolognih receptora s ekspresijom u rekombinantnim sustavima rezultira cijelim nizom biolokih odgovora, koji su upravljani G proteinima s ekspresijom u stanicama domaina. Zauzee GPCR od strane agonista rezultira izmjenom vezanog GDP za GTP na mjestu vezanja G podjedinice. Moe se koristiti radioaktivni, nehidrolizirajui derivat GTP, [35S]GTP5, da bi se izmjerilo vezanje odreenog agonista za receptor (Sim et al., Neuroreport, 1996, 7, 729-733). Ovo vezanje se moe takoer koristiti da bi se izmjerila sposobnost antagonista da se vee za receptor smanjenjem vezanja [35S]GTP-S u prisutnosti poznatog agonista. Moe se dakle oblikovati HTS analiza zasnovana na vezanju [35S]GTPS. </span>It is known that the activation of heterologous receptors with expression in recombinant systems results in a whole series of biological responses, which are directed by G proteins with expression in host cells. Occupancy of a GPCR by an agonist results in the exchange of bound GDP for GTP at the binding site of the G subunit. A radioactive, non-hydrolyzable derivative of GTP, [35S]GTP5, can be used to measure the binding of a particular agonist to the receptor (Sim et al., Neuroreport, 1996, 7, 729-733). This binding can also be used to measure the ability of an antagonist to bind to a receptor by reducing the binding of [35S]GTP-S in the presence of a known agonist. An HTS assay based on [35S]GTPS binding can therefore be designed.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">G proteini koji su nuni za funkcionalnu ekspresiju heterolognih GPCR-ova mogu biti nativni konstituenti stanice domaina ili mogu biti uneseni dobro poznatom rekombinantnom tehnologijom. G proteini mogu biti intaktni ili kimerni. esto se rabi gotovo univerzalni kompetentni G protein (npr. Gi6) za vezanje bilo kojeg danog receptora za detektibilni put odgovora. Aktiviranje G proteina rezultira stimuliranjem ili inhibiranjem drugih nativnih proteina, efektima koji se mogu povezati za mjerljivi odgovor. </span>G proteins that are necessary for the functional expression of heterologous GPCRs can be native constituents of the host cell or can be introduced by well-known recombinant technology. G proteins can be intact or chimeric. An almost universally competent G protein (eg, Gi6) is often used to bind any given receptor to a detectable response pathway. Activation of G proteins results in stimulation or inhibition of other native proteins, effects that can be linked to a measurable response.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Primjeri takvih biolokih odgovora obuhvaaju, ali nisu ogranieni na sljedee: sposobnost preivljavanja u odsutnosti ograniavajueg nutrienta u posebno inenjerski oblikovanih kvaevih stanica (Pausch, Trends in Biotechnology, 1997, 15, 487-494); promjene unutarstanine koncentracije Ca2+ koja se mjeri pomou fluorescentnih boja (Murphy, et al., Curr. Opm. Drug Disc. Dev., 1998, 1, 192-199). Promjene fluorescencije mogu se takoer koristiti za praenje ligandom izazvanih promjena membranskog potencijala ili unutarstaninog pH. U tu svrhu nainjen je i opisan automatiziran sustav za HTS (Schroeder, et al., J. Biomolecular Screening, 1996, 1, 75-80). Melanofori prireeni iz Xenopus laevis pokazuju promjene pigmentne organizacije kao odgovor na heterologno GPCR aktiviranje. Ovaj odgovor prilagodljiv je na HTS formate (Jayawickreme, et al., Curr. Opin. Biotechnol., 1997, 8, 629-634). Postoje takoer analize za mjerenje standardnih sekundarnih glasnika, ukljuujui cAMP, fosfoinozitide i arahidonsku kiselinu. </span>Examples of such biological responses include, but are not limited to: the ability to survive in the absence of a limiting nutrient in specially engineered yeast cells (Pausch, Trends in Biotechnology, 1997, 15, 487-494); changes in intracellular Ca2+ concentration measured using fluorescent dyes (Murphy, et al., Curr. Opm. Drug Disc. Dev., 1998, 1, 192-199). Fluorescence changes can also be used to monitor ligand-induced changes in membrane potential or intracellular pH. For this purpose, an automated system for HTS was developed and described (Schroeder, et al., J. Biomolecular Screening, 1996, 1, 75-80). Melanophores prepared from Xenopus laevis show changes in pigment organization in response to heterologous GPCR activation. This response is adaptable to HTS formats (Jayawickreme, et al., Curr. Opin. Biotechnol., 1997, 8, 629-634). There are also assays to measure standard second messengers, including cAMP, phosphoinositides, and arachidonic acid.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Metode HTS koje koriste ove receptore ukljuuju permanentno transficirane CHO stanice, u kojima se agonisti i antagonisti mogu identificirati pomou sposobnosti da specifino mijenjaju vezanje [35S]GTPS u membranama koje su prireene iz ovih stanica. U drugoj realizaciji ovog izuma, permanentno transficirane CHO stanice mogu se koristiti za prireivanje membrana koje sadre znaajne koliine rekombinnantnih receptorskih proteina. Ovi membranski pripravci mogu se zatim koristiti u receptorskim vezujuim analizama, koritenjem radioobiljeenog liganda koji je specifian za konkretni receptor. Alternativno, funkcionalna analiza, kao to je fluorescentno praenje ligandom izazvanih promjena unutranje Ca2+ koncentracije ili membranskog potencijala u permanentno transficiranim CHO stanicama koje sadre svaki od ovih receptora pojedinano ili u kombinaciji moe biti korisna za HTS. Jednako koristan moe biti alternativni tip stanice sisavaca, kao to su HEK293 ili COS stanice, u slinim formatima. Permanentno transficirane linije insektnih stanica, kao to su Drosophila S2 stanice i rekombinantne kvaeve stanice koje svre ekspresiju Drosophila melanogaster receptora u HTS formatima dobro su poznate onima koji poznaju ovo podruje tehnike (npr. Pausch, Trends in Biotechnology, 1997, 75, 487-494), to moe takoer biti korisno u ovom izumu. </span>HTS methods using these receptors involve permanently transfected CHO cells, in which agonists and antagonists can be identified by their ability to specifically alter [35S]GTPS binding in membranes prepared from these cells. In another embodiment of the present invention, permanently transfected CHO cells can be used to prepare membranes containing significant amounts of recombinant receptor proteins. These membrane preparations can then be used in receptor binding assays, using a radiolabeled ligand that is specific for that particular receptor. Alternatively, functional analysis such as fluorescent monitoring of ligand-induced changes in internal Ca2+ concentration or membrane potential in permanently transfected CHO cells containing each of these receptors individually or in combination may be useful for HTS. An alternative mammalian cell type, such as HEK293 or COS cells, in similar formats may be equally useful. Permanently transfected insect cell lines, such as Drosophila S2 cells and recombinant yeast cells expressing the Drosophila melanogaster receptor in HTS formats are well known to those skilled in the art (eg, Pausch, Trends in Biotechnology, 1997, 75, 487-494 ), which may also be useful in this invention.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Ovaj izum podrazumijeva vie analiza za screening i identificiranje inhibitora ligandnog vezanja za DmGPCR receptore. U jednom primjeru, DmGPCR receptor je imobiliziran i interakcija s vezujuim partnerom vri se u prisutnosti i odsutnosti kandidatskog modulatora kao to je inhibitorski spoj. U sljedeem primjeru, interakcija izmeu DmGPCR receptora i njegova vezujueg partnera vri se u tekuoj analizi, u prisutnosti i odsutnosti kandidatskog inhibitorskog spoja. U bilo kojoj analizi, inhibitor se identificira kao spoj koji smanjuje vezanje izmeu DmGPCR receptora i njegova vezujueg partnera. Sljedea analiza koja se podrazumijeva ukljuuje inaicu dvohibridne analize pri emu se inhibitor protein/protein interakcija identificira detektiranjem pozitivnog signala u transformiranoj i transficiranoj stanici domaina, kao to je opisano u PCT publikaciji br. WO 95/20652, publicirano 3. kolovoza 1995. </span>The present invention involves multiple assays for screening and identifying inhibitors of ligand binding to DmGPCR receptors. In one example, the DmGPCR receptor is immobilized and the interaction with the binding partner is performed in the presence and absence of a candidate modulator such as an inhibitory compound. In the following example, the interaction between a DmGPCR receptor and its binding partner is performed in a run-on assay, in the presence and absence of a candidate inhibitory compound. In any assay, an inhibitor is identified as a compound that reduces binding between a DmGPCR receptor and its binding partner. The next assay implied involves a version of the two-hybrid assay wherein the protein/protein interaction inhibitor is identified by detecting a positive signal in the transformed and transfected host cell, as described in PCT Publication No. WO 95/20652, published August 3, 1995.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Kandidatski modulatori koji se podrazumijevaju ovim izumom ukljuuju spojevi koji su odabrani iz biblioteka bilo potencijalnih aktivatora ili potencijalnih inhibitora. Postoji odreen broj razliitih biblioteka koje se rabe za identificiranje modulatora malenih molekula, ukljuujui: (1) kemijske biblioteke, (2) biblioteke prirodnih produkata i (3) kombinacijske biblioteke koje sadre sluajne peptide, oligonukleotide ili organske molekule. Kemijske biblioteke sastoje se od sluajnih kemijskih struktura, a neke od njih su analozi poznatih spojeva ili analozi spojeva koji su identificirani kao hitovi ili vodei u drugom screening-u otkrivanja lijekova, neki od njih su izvedeni iz prirodnih produkata dok se neki javljaju u neusmjerneoj sintetskoj organskoj kemiji. Biblioteke prirodnih produkata su zbirke mikroorganizama, ivotinja, bilja ili morskih organizama koji se koriste da se naine smjese za screening pomou: (1) fermentiranja i ekstrakcije iz tla, biljaka ili morskih mikroorganizama ili (2) ekstrahiranja bilja i morskih organizama. Biblioteke prirodnih produkata ukljuuju poliketide, neribosomske peptide i njihove inaice (kojih nema u prirodi). Za pregled, vidi Science, 1998, 282, 63-68. Kombinacijske biblioteke sastavljene su od velikog broja peptida, oligonukleotida ili organskih spojeva kao smjese. Ove biblioteke je razmjerno lako nainiti metodama tradicionalne automatizirane sinteze, PCR, kloniranjem ili pogodnim sintetskim metodama. Od osobita znaaja su nepeptidne kombinacijske biblioteke. Ostale biblioteke od interesa ukljuuju peptide, protein,e peptidomimetike, vieparalelnu sintetsku zbirku, rekombinatorne i polipeptidne biblioteke. Za prikaz kombinatorne kemije i biblioteka koje su nainjene iz njih vidi Myers, Curr. Opin. Biotechnol., 1997, 8, 701-707. Identificiranje modulatora putem upotrebe razliitih biblioteka koje je ovdje opisano omoguuje modificiranje kandidatskih hitova (ili vodia) da bi se optimizirao kapacitet hita da modulira aktivnost. </span>Candidate modulators contemplated by the present invention include compounds selected from libraries of either potential activators or potential inhibitors. There are a number of different libraries used to identify small molecule modulators, including: (1) chemical libraries, (2) natural product libraries, and (3) combinatorial libraries containing random peptides, oligonucleotides, or organic molecules. Chemical libraries consist of random chemical structures, and some of them are analogs of known compounds or analogs of compounds that have been identified as "hits" or "leads" in other drug discovery screenings, some of them are derived from natural products, while some occur in non-directional synthetic organic chemistry. Natural product libraries are collections of microorganisms, animals, plants, or marine organisms that are used to prepare mixtures for screening by: (1) fermentation and extraction from soil, plants, or marine microorganisms, or (2) extraction of plants and marine organisms. Natural product libraries include polyketides, non-ribosomal peptides and their variants (not found in nature). For review, see Science, 1998, 282, 63-68. Combinatorial libraries are composed of a large number of peptides, oligonucleotides or organic compounds as a mixture. These libraries are relatively easy to make using traditional automated synthesis methods, PCR, cloning, or convenient synthetic methods. Of particular importance are non-peptide combinatorial libraries. Other libraries of interest include peptide, protein, and peptidomimetics, multi-parallel synthetic library, recombinant and polypeptide libraries. For an overview of combinatorial chemistry and libraries made from it, see Myers, Curr. Opin. Biotechnol., 1997, 8, 701-707. Identifying modulators through the use of the various libraries described herein allows candidate hits (or guides) to be modified to optimize the capacity of the hit to modulate activity.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">I drugi kandidatski inhibitori podrazumijevaju se ovim izumom i oni mogu biti oblikovani te ukljuuju topljive oblike vezujuih partnera, kao i takve vezujue partnere kao kimerne ili sjedinjene proteine. Pojam vezujui partner kako se ovdje rabi u irem smislu obuhvaa nepeptidne modulatore, kao i takve peptidne modulatore kao to su neuropeptidi koji nisu prirodni ligandi, antitijela, fragmenti antitijela i modificirani spojevi koji sadre domene antitijela koje su imunospecifine za ekspresijski produkt identificiranog DmGPCR gen. </span>Other candidate inhibitors are contemplated by the present invention and may be designed to include soluble forms of binding partners, as well as such binding partners as chimeric or fusion proteins. The term "binding partner" as used herein broadly includes non-peptide modulators, as well as such peptide modulators as neuropeptides that are not natural ligands, antibodies, antibody fragments, and modified compounds containing antibody domains that are immunospecific for the expression product of the identified DmGPCR gene .</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">U drugim realizacijama ovog izuma, polipeptidi ovog izuma koriste se kao istraivako pomagalo za identificiranje, karakteriziranje i proiavanje interakcijskih, regulacijskih proteina. Odgovarajue oznake ugraene su u polipeptide ovog izuma pomou razliitih metoda koje su poznate u tehnici i polipeptidi se koriste za hvatanje interakcijskih molekula. Primjerice, molekule se inkubiraju s obiljenim polipeptidima, ispiru da se uklone nevezani polipeptidi i kvantificira se polipeptidni kompleks. Podaci koji su dobiveni koritenjem razliitih koncentracija polipeptida koriste se za izraunavanje vrijednosti za broj, afinitet i povezivanje polipeptida s proteinskim kompleksom. </span>In other embodiments of the present invention, the polypeptides of the present invention are used as a research tool to identify, characterize, and purify interacting, regulatory proteins. Appropriate tags are incorporated into the polypeptides of the present invention using various methods known in the art and the polypeptides are used to capture interacting molecules. For example, molecules are incubated with labeled polypeptides, washed to remove unbound polypeptides, and the polypeptide complex is quantified. Data obtained using different concentrations of polypeptides are used to calculate values for the number, affinity, and binding of polypeptides to the protein complex.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Obiljeenoi polipeptidi su takoer korisni kao reagensi za proienje molekula s kojima su polipeptidi u interakciji ukljuujui, ali bez ogranienja, inhibitore. U jednoj realizaciji afinitetnog proiavanja, polipeptid je kovalentno vezan za kromatografsku kolonu. Stanice i njihove membrane su ekstrahirani i razliite stanine podkomponente su proputene kroz kolonu. Molekule se veu za kolonu sukladno njihovu afinitetu za polipeptid. Polipeptidni kompleks se skida s kolone, disocira i obnovljena kolona podvrgava se proteinskom sekvenciranju. Aminokiselinska sekvencija se zatim koristi za identificiranje uhvaene molekule ili da se oblikuju degenerirani oligonukleotidi za kloniranje odgovarajueg gena iz pripadne cDNA biblioteke. </span>Labeled polypeptides are also useful as reagents for the purification of molecules with which the polypeptides interact, including, but not limited to, inhibitors. In one embodiment of affinity purification, the polypeptide is covalently attached to a chromatographic column. The cells and their membranes were extracted and the various cellular subcomponents were passed through the column. Molecules bind to the column according to their affinity for the polypeptide. The polypeptide complex is removed from the column, dissociated and the recovered column is subjected to protein sequencing. The amino acid sequence is then used to identify the captured molecule or to design degenerate oligonucleotides to clone the corresponding gene from the corresponding cDNA library.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Alternativno, mogu biti identificirani spojevi koji pokazuju slina svojstva prema ligandu za DmGPCR ovog izuma, ali koji su manji i imaju dulje vrijeme poluivota nego endogeni ligand u ovjejem ili ivotinjskom tijelu. Kada je oblikovan organski spoj, molekula sukladno ovom izumu koristi se kao vodei spoj. Oblikovanje mimetika za poznate farmaceutski aktivne spojeve dobro je poznati pristup u razvoju spojeva koji se temelje na ovim vodeim spojevima. Mimetiko oblikovanje, sinteza i ispitivanje openito se koriste da se izbjegne sluajni screening velikog broja molekula na ciljno svojstvo. tovie, strukturni podaci koji su izvedeni iz analize izvedenih aminokiselinskih sekvencija koje su kodirane pomou DNA ovog izuma korisni su za oblikovanje novih lijekova koji su specifiniji i koji prema tome imaju jau farmakoloku snagu. </span>Alternatively, compounds may be identified which exhibit similar properties to the DmGPCR ligand of the present invention, but which are smaller and have a longer half-life than the endogenous ligand in the human or animal body. When an organic compound is formed, a molecule according to the present invention is used as a "lead" compound. Designing mimetics for known pharmaceutically active compounds is a well-known approach in the development of compounds based on these "lead" compounds. Mimetic design, synthesis, and screening are generally used to avoid random screening of large numbers of molecules for a target property. Moreover, the structural data derived from analysis of the deduced amino acid sequences encoded by the DNA of the present invention is useful for designing new drugs that are more specific and thus have stronger pharmacological potency.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Usporeivanje sekvencija proteina ovog izuma sa sekvencijama koje se nalaze u dostupnim bazama podataka pokazuju znaajnu homologiju s transmembranskim dijelom G protein vezanih receptora. Prema tome, moe se koristiti kompjutersko modeliranje da se razvije potencijalna tercijarna struktura proteina ovog izuma temeljem dostupnih informacija transmembranske domene drugih proteina. Dakle, mogu se oblikovati novi ligandi temeljem predviene strukture DmGPCR. </span>Comparison of protein sequences of the present invention with sequences found in available databases show significant homology to the transmembrane portion of G protein-coupled receptors. Accordingly, computer modeling can be used to develop the potential tertiary structure of proteins of the present invention based on available transmembrane domain information of other proteins. Thus, new ligands can be designed based on the predicted structure of DmGPCR.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">U konkretnoj realizaciji, nove molekule koje su identificirane pomou screening metoda sukladno ovom izumu su niskomolekularne organske molekule, pri emu se mogu prirediti smjese ili farmaceutske smjese za oralno unoenje, kao to su tablete. Smjese ili farmaceutske smjese, koje sadre molekule organskih kiselina, vektore, polipeptide, antitijela i spojeve identificirane su pomou screening metoda koje su ovdje opisane, koje su prireene za bilo koji nain primjene ukljuujui, ali bez ogranienja, oralni, intravenski, kutani, supkutani, nazalni, intramuskularni ili intraperitonealni. Priroda nosaa ili drugih sastojaka ovisi o specifinom putu primjene i konkretnoj realizaciji izuma koja se primjenjuje. Primjeri tehnika i protokola koji se rabe u ovom kontekstu mogu se izmeu ostalog nai u: Remingtons Pharmaceutical Sciences, Osol, (ed.), 1980, to je ovdje ukljueno kao referenca u cijelosti. </span>In a specific embodiment, the new molecules identified using screening methods according to this invention are low molecular weight organic molecules, whereby mixtures or pharmaceutical mixtures for oral administration, such as tablets, can be prepared. Mixtures or pharmaceutical compositions, containing organic acid molecules, vectors, polypeptides, antibodies and compounds identified using the screening methods described herein, are prepared for any route of administration including, but not limited to, oral, intravenous, cutaneous, subcutaneous, nasal, intramuscular or intraperitoneal. The nature of the carrier or other ingredients depends on the specific route of administration and the particular embodiment of the invention being applied. Examples of techniques and protocols used in this context can be found, inter alia, in: Remington's Pharmaceutical Sciences, Osol, (ed.), 1980, which is incorporated herein by reference in its entirety.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Doziranje ovih niskomolekularnih spojeva ovisi o stanju bolesti ili stanju koje se tretira i drugim klinikim imbenicima, kao to je teina i stanje subjekta koji se tretira i putu primjene spoja. Za tretiranje ivotinja, primjenjuje se priblino izmeu 0,5 mg/kg tjelesne teine to 500 mg/kg tjelesne teine spoja. Terapija se tipino primjenjuje uz nie doze sve dok se ne uoi eljeni terapijski uinak. </span>The dosage of these low molecular weight compounds depends on the disease state or condition being treated and other clinical factors, such as the weight and condition of the subject being treated and the route of administration of the compound. For the treatment of animals, approximately between 0.5 mg/kg body weight and 500 mg/kg body weight of the compound is applied. Therapy is typically administered at lower doses until the desired therapeutic effect is observed.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Metode za odreivanje doziranja spojeva koji se primjenjuju subjektu i naina primjene spojeva nekom organizmu opisane su u amerikoj patentnoj prijavi br. 08/702,282, podnesenoj 23. kolovoza 1996. i meunarodnoj patentnoj publikaciji br. WO 96/22976, tiskanoj 1. kolovoza 1996., a obje su ovdje ukljuene kao referenca u cijelosti, ukljuujui sve crtee, slike i tablice. Oni koji poznaju ovo podruje tehnike znaku da se takvi opisi mogu primijeniti na ovaj izum i lako se mogu prilagoditi. </span>Methods for determining the dosage of compounds to be administered to a subject and the method of administration of compounds to an organism are described in US patent application no. 08/702,282, filed August 23, 1996, and International Patent Publication No. WO 96/22976, printed Aug. 1, 1996, both of which are incorporated herein by reference in their entirety, including all drawings, figures, and tables. Those skilled in the art will appreciate that such descriptions are applicable to the present invention and may be readily adapted.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Pravilno doziranje ovisi o razliitim imbenicima kao to je tip bolesti koja se tretira, konkretna smjesa koja se koristi te veliina i fizioloko stanje subjekta. Terapijski uinkovite doze spojeva ovog izuma mogu se poetno procijeniti iz stanine kulture i ivotinjskih modela. Primjerice, doza se moe formulirati u ivotinjskim modelima da se postigne cirkulirajui koncentracijski raspon koji poetno uzima u obzir IC50 koji je odreen u analizi stanine kulture. </span>The correct dosage depends on various factors such as the type of disease being treated, the specific mixture used, and the size and physiological state of the subject. Therapeutically effective doses of the compounds of this invention can be initially estimated from cell culture and animal models. For example, a dose can be formulated in animal models to achieve a circulating concentration range that initially accounts for the IC50 determined in a cell culture assay.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Vrijeme poluivota plazme i bioloka raspodjela lijeka i metabolita u plazmi, tumorima i glavnim organima mogu se takoer odrediti da se olaka izbor lijeka koji najvie odgovara da se zaustavi poremeaj. Mogu se izvriti potrebna mjerenja. Primjerice, HPLC analiza moe se izvriti s plazmom ivotinja koje su tretirane lijekom i smjetaj radioobiljeenih lijekova moe se odrediti koritenjem metoda detekcije kao to su rentgensko zraenje, CAT skeniranje i MRI. Spojevi koji pokazuju jaku aktivnost inhibiranja u screening analizama, ali imaju slaba farmakokinetika svojstva mogu se optimizirati promjenom kemijske strukture i ponovnim ispitivanjem. U tom smislu, spojevi koji pokazuju dobra farmakokinetika svojstva mogu se rabiti kao model. </span>The plasma half-life and biodistribution of drug and metabolites in plasma, tumors, and major organs can also be determined to facilitate the selection of the most appropriate drug to arrest the disorder. The necessary measurements can be made. For example, HPLC analysis can be performed on plasma from drug-treated animals and the location of radiolabeled drugs can be determined using detection methods such as X-rays, CAT scans, and MRI. Compounds that show strong inhibitory activity in screening assays but have weak pharmacokinetic properties can be optimized by changing the chemical structure and retesting. In this sense, compounds that show good pharmacokinetic properties can be used as a model.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Ispitivanje toksinosti moe se takoer izvriti mjerenjem sastava krvnih stanica. Primjerice, ispitivanje toksinosti moe se izvriti na pogodnom ivotinjskom modelu na sljedei nain: 1) spoj se primijeni mievima (takoer se koristi netretirani kontrolni mi); 2) krvni uzorci se periodiki dobiju iz repne vene jednog mia iz svake tretirane skupine; i 3) uzorci se analiziraju na broj crvenih i bijelih krvnih stanica, sastav krvnih stanica i postotak limfocita u odnosu na polimorfonuklearne stanice. Prikazana je usporedba rezultata za svaki reim doziranja s kontrolama koje pokazuju ako je uoena toksinost. </span>Toxicity testing can also be done by measuring blood cell composition. For example, toxicity testing can be performed on a suitable animal model as follows: 1) the compound is administered to mice (an untreated control mouse is also used); 2) blood samples are periodically obtained from the tail vein of one mouse from each treated group; and 3) samples are analyzed for red and white blood cell counts, blood cell composition, and the percentage of lymphocytes in relation to polymorphonuclear cells. A comparison of the results for each dosing regimen with controls is shown to indicate if toxicity was observed.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Na zavretku svakog istraivanja toksinosti, mogu se izvriti daljnja istraivanja rtvovanjem ivotinja (poeljno u skladu s naputkom drutva American Veterinary Medical Association u izvjeu: Report of the American Veterinary Medical Assoc. Panel on Euthanasia, J. Amer. Vet. Med. Assoc, 1993, 202, 229-249). Reprezentativne ivotinje iz svake tretirane skupine mogu se ispitati pomou opsene nekropsije na pojavu metastaza, neobinih bolesti ili toksinost. Abnormnalnosti u tkivu se biljee i tkivo se histoloki ispituje. </span>At the conclusion of each toxicity study, further studies can be performed by sacrificing animals (preferably in accordance with the American Veterinary Medical Association's guideline in the report: Report of the American Veterinary Medical Assoc. Panel on Euthanasia, J. Amer. Vet. Med. Assoc, 1993 , 202, 229-249). Representative animals from each treatment group can be examined by extensive necropsy for metastases, unusual diseases or toxicity. Abnormalities in the tissue are noted and the tissue is examined histologically.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Ovi spojevi i metode, ukljuujui molekule nukleinske kiseline, polipeptide, antitijela, te spojevi identificirani screening metodama koje su ovdje opisane, nalaze brojne farmaceutske i agrikulturne primjene (npr. insekticidne) te se primjerice mogu koristiti za tretiranje ili sprjeavanje stanja izazvanih ektoparazitima ili za kontrolu populacije insekata. </span>These compounds and methods, including nucleic acid molecules, polypeptides, antibodies, and compounds identified by the screening methods described herein, find numerous pharmaceutical and agricultural applications (e.g., insecticidal) and can, for example, be used to treat or prevent conditions caused by ectoparasites or to control insect populations.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Ovaj izum takoer obuhvaa metode za agoniziranje (stimuliranje) ili antagoniziranje aktivnosti koja je povezana s prirodnim DmGPCR vezujuim partnerom u subjekta to obuhvaa primjenu navedenom subjektu odreenog agonista ili antagonista jedom od prije opisanih polipeptida u koliini koja je dovoljna da se izazove navedeni agonizam ili antagonizam. Prema tome, jedna realizacija ovog izuma je metoda tretiranja bolesti ili stanja subjekta izazvanih ektoparazitom, pomou agonista ili antagonista subjekta u koliini koja je dovoljna da se agonizira ili antagonizira ektoparazitne DmGPCR-pridruene funkcije. </span>This invention also includes methods for agonizing (stimulating) or antagonizing the activity associated with a natural DmGPCR binding partner in a subject, which comprises administering to said subject a certain agonist or antagonist with one of the previously described polypeptides in an amount sufficient to cause said agonism or antagonism. Accordingly, one embodiment of the present invention is a method of treating a disease or condition of a subject caused by an ectoparasite, using an agonist or antagonist of the subject in an amount sufficient to agonize or antagonize ectoparasite DmGPCR-associated functions.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">U sljedeoj tablici 4 navedene su sekvencije polinukleotida i polipeptida ovog izuma. </span>The following Table 4 lists the sequences of the polynucleotides and polypeptides of the present invention.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">[image] [image] [image] [image] [image] [image] [image] [image] [image] [image] [image] </span>[image] [image] [image] [image] [image] [image] [image] [image] [image] [image] [image]</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Sukladno s Budimpetanskim tretmanom, klonovi ovog izuma su pohranjeni u zbirci Agricultural Research Culture Collection (NRRL) International Depository Authority, 1815 N. University Street, Peoria, Illinois 61604, U.S.A. Pristupni brojevi i datumo pohranjivanja su navedeni u tablici 5. </span>Pursuant to Budapest treatment, clones of this invention are deposited in the Agricultural Research Culture Collection (NRRL) International Depository Authority, 1815 N. University Street, Peoria, Illinois 61604, U.S.A. Accession numbers and date of storage are listed in Table 5.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Tablica 5 </span>Table 5</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">[image] </span>[image]</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Ovaj izum je nadalje ilustriran putem primjera koji slijede koji su namijenjeni rasvjetljavanju ovog izuma. Ovi primjeri nemaju za cilj i ne treba ih smatrati ogranienjem dosega ovog izuma. Jasno je da se ovaj izum moe ostvariti na drugi nain osim onoga koji je ovdje prikazan. Brojne modifikacije i varijacije ovog izuma su mogue sukladno injenicama koje su ovdje iznesene, pa su prema tome unutar dosega ovog izuma. </span>This invention is further illustrated by way of the following examples which are intended to elucidate this invention. These examples are not intended and should not be considered as limiting the scope of the present invention. It is clear that this invention can be practiced in other ways than that shown here. Numerous modifications and variations of this invention are possible in accordance with the facts presented herein, and are therefore within the scope of this invention.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Podrazumijeva se da je svaki patent, prijava i tiskana publikacija koja je spomenuta u ovom dokumentu ukljuena kao referenca u cijelosti. </span>It is understood that each patent, application and printed publication mentioned herein is incorporated by reference in its entirety.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Primjeri </span>Examples</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Primjer 1: Identificiranje DmGPCR-ova </span>Example 1: Identifying DmGPCRs</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Celera genomska D. melanogaster baza podataka prevedena je u bazu podatka predvienih proteina i mRNA bazu podataka koritenjem cijelog niza softverskih pomagala za pronalaenje gena da se predvide mRNA koje mogu nastati (PnuFlyPep baza podataka). Postupci za analiziranje genomskih baza podataka koritenjem softverskih pomagala za pronalaenje gena poznato je onima koji se bave ovim podrujem tehnike. </span>The Celera genomic D. melanogaster database was translated into a predicted protein database and an mRNA database using a suite of gene discovery software tools to predict putative mRNAs (the "PnuFlyPep" database). Procedures for analyzing genomic databases using gene finding software tools are known to those skilled in the art.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Nukleotidne sekvencije nekoliko C. elegans FaRP GPCR-ova koritene su kao sekvencije za traenje mRNA baze podataka koja je prije opisana. Ova baza podataka pretraivana je na podruja slinosti koritenjem cijelog niza pomonih alata, ukljuujui FASTA i Gapped BLAST (Altschul et al., Nuc. Acids Res., 1997, 25, 3389, to je ovdje ukljueno kao referenca u cijelosti). </span>The nucleotide sequences of several C. elegans FaRP GPCRs were used as sequences to search the mRNA database described previously. This database was searched for regions of similarity using a variety of utility tools, including FASTA and Gapped BLAST (Altschul et al., Nuc. Acids Res., 1997, 25, 3389, which is incorporated herein by reference in its entirety).</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Ukratko, BLAST algoritam, to oznaava Basic Local Alignment Search Tool pogodan je za odreivanje sekvencije slinosti (Altschul et al, J. Mol. Biol., 1990, 215, 403-410, to je ovdje ukljueno kao referenca u cijelosti). Softver za izvrenje BLAST analiza javno je dostupan putem National Center for Biotechnology Information (http://www.ncbi.nlm.nih.gov/). Ovaj algoritam prvo obuhvaa identificiranje sekvencijskih parova s visokim rangom (HSP-ova) identificiranjem kratkih rijei duljine W u sekvenciji za pretraivanje koja bilo odgovara ili zadovoljava nekim graninim T pozitivnim vrijednostima kada se uskladi s rijei identine duljine u sekvencijskoj bazi podataka. T smo oznaili kao granicu skora susjedne rijei (Altschul et al., supra). Ovaj poetni pogodak susjedne rijei slui kao klica za zapoinjanje potrage da se nau HSP-vi koji ih sadre. Pogoene rijei proiruju se u oba smjera du svake sekvencije sve dok se kumulativni indeks usklaenosti moe poveati. Produetak za pogotke rijei u svakom smjesu zaustavlja se kada: 1) kumulativni skor sukladnosti pada za iznos X od njegove maksimalne dosegnute vrijednosti; 2) kumulativni skor pada na nulu ili nie, zbog akumuliranja jednog ili vie poklapanja rezidua s negativnim skorom; ili 3) dosegnut je zavretak sekvencije. Parametri Blast algoritma W, T i X odreuju osjetljivost i brzinu usklaivanja. Blast program koristi podrazumijevanje duljine rijei (W) od 11, BLOSUM62 matrice skora (vidi Henikoff et al, Proc. Natl Acad. Sci. USA, 1992, 89, 10915-10919, to je ovdje ukljueno kao referenca u cijelosti) usklaivanja (B) od 50, oekivanja (E) od 10, M=5, N=4 i usporeivanjem obiju zavojnica. </span>Briefly, the BLAST algorithm, which stands for Basic Local Alignment Search Tool is suitable for determining sequence similarity (Altschul et al, J. Mol. Biol., 1990, 215, 403-410, which is incorporated herein by reference in its entirety). Software for performing BLAST analyzes is publicly available through the National Center for Biotechnology Information (http://www.ncbi.nlm.nih.gov/). This algorithm first involves identifying high-ranking sequence pairs (HSPs) by identifying short words of length W in the search sequence that either match or satisfy some threshold T positive values when matched against words of identical length in the sequence database. We denoted T as the adjacent word score limit (Altschul et al., supra). This initial adjacent word hit serves as the seed to begin the search to find HSPs that contain them. Hit words are extended in both directions along each sequence as long as the cumulative match index can be increased. The extension for word hits in each mixture stops when: 1) the cumulative conformity score falls by an amount X from its maximum value reached; 2) the cumulative score drops to zero or lower, due to the accumulation of one or more matching residuals with a negative score; or 3) the end of the sequence is reached. The parameters of the Blast algorithm W, T and X determine the sensitivity and speed of matching. The Blast program uses a word length (W) implication of 11, BLOSUM62 score matrix (see Henikoff et al, Proc. Natl Acad. Sci. USA, 1992, 89, 10915-10919, incorporated herein by reference in its entirety) alignments (B ) of 50, expectations (E) of 10, M=5, N=4 and comparing both coils.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">BLAST algoritam (Karlin et al, Proc. Natl Acad. Sci. USA, 1993, 90, 5873-5787, to je ovdje ukljueno kao referenca u cijelosti) i Gapped BLAST vre statistiku analizu slinosti izmeu dviju sekvencija. Jedna mjera slinosti koja se dobiva BLAST algoritmom je najmanja suma (P(N)), koja definira naznaku vjerojatnosti kojom se moe sluajno postii podudarnost dviju nukleotida ili aminokselinskih sekvencija. Primjerice, nukleinske kiseline smatraju se slinim prema DmGPCR genu ili cDNA ako je najmanja suma vjerojatnosti pri usporeivanju testne nukleinske kiseline prema DmGPCR nukleinskoj kiselini manja od oko 1, poeljno manja od oko 0,1, poeljnije manja od oko 0,01 i najpoeljnije manja od oko 0,001. </span>The BLAST algorithm (Karlin et al, Proc. Natl Acad. Sci. USA, 1993, 90, 5873-5787, which is incorporated herein by reference in its entirety) and Gapped BLAST perform a statistical analysis of the similarity between two sequences. One measure of similarity obtained by the BLAST algorithm is the smallest sum (P(N)), which defines an indication of the probability that two nucleotide or amino acid sequences can match by chance. For example, nucleic acids are considered similar to a DmGPCR gene or cDNA if the lowest sum of probabilities when comparing a test nucleic acid to a DmGPCR nucleic acid is less than about 1, preferably less than about 0.1, more preferably less than about 0.01, and most preferably less than about 0.001.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">mRNA koje odgovaraju predvienim proteinima preuzete su iz baze podataka predvienih mRNA da se priredi PnuFlyPep baza podataka. One su identificirane kao sljedee nukleotidne sekvencije: 1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21 i 23, od kojih svaka ima statistiki znaajno preklapanje homologije sa sekvencijom za pretraivanje. Nukleotidne sekvencije sa sljedeim sekvencijom brojevima: 3, 5, 9, 11, 13 i 15 (to odgovara DmGPCR-ovima 2a, 2b, 5a, 5b, 6a i 6b) dobivene su PCR kloniranjem i sekvenciranjem druge identificicirane sekvencije (nije prikazano). Svaka od ovih sekvencija predstavlja splice inaicu gena DmGPCR-a. </span>mRNAs corresponding to predicted proteins were retrieved from the database of predicted mRNAs to prepare the PnuFlyPep database. These were identified as the following nucleotide sequences: 1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21, and 23, each having a statistically significant homology overlap with the search sequence. Nucleotide sequences with the following sequence numbers: 3, 5, 9, 11, 13 and 15 (corresponding to DmGPCRs 2a, 2b, 5a, 5b, 6a and 6b) were obtained by PCR cloning and sequencing of the second identified sequence (not shown). Each of these sequences represents a spliced version of the DmGPCR gene.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Primjer 2: Kloniranje DmGPCR-ova </span>Example 2: Cloning of DmGPCRs</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Prireivanje cDNA </span>Preparation of cDNA</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">cDNA prireena je bilo iz zrele Drosophila melanogaster poli A+ RNA (Clontech Laboratories, Palo Alto, CA) ili zrele Drosophila melanogaster ukupne RNA (dolje). Da se dobije ukupna RNA, matina zaliha Drosophila melanogaster (Biological Supply Company, Burlington, NC) anestezirana je hlaenjem i 5 do 6 odraslih jedinki dodano je u posudu koja sadri 10 ml H2O, 10 ml medija 4-24 Instant Drosophila Medium i 6 do 10 grain-a s aktivnog suhog kvasca (Biological Supply Company). Poliuretanski pjenasti ep stavljen je na kraj svake posude i muice su inkubirani na sobnoj temperaturi tijekom 4 do 6 tjedana. Nakon zrelosti, posude su ohlaene i anestezirane muice su stavljene 50 ml polipropilensku epruvetu koja je drana u tekuem duiku. Zamrznute muice su pohranjene na -70 C dok nisu usitnjene u tarioniku uz tekui duik. Prakasto tkivo zajedno s neto tekueg duika dekantirano je u 50 ml polipropilenske epruvete na suhi led. Nakon uparavanja tekueg duika, prakasto tkivo pohranjeno je na -70 C. </span>cDNA was prepared from either mature Drosophila melanogaster poly A+ RNA (Clontech Laboratories, Palo Alto, CA) or mature Drosophila melanogaster total RNA (bottom). To obtain total RNA, Drosophila melanogaster mother stock (Biological Supply Company, Burlington, NC) was anesthetized by chilling, and 5 to 6 adults were added to a dish containing 10 ml H2O, 10 ml 4-24 Instant Drosophila Medium, and 6 to 10 grains of active dry yeast (Biological Supply Company). A polyurethane foam plug was placed at the end of each container and the flies were incubated at room temperature for 4 to 6 weeks. After maturity, the containers were cooled and the anesthetized flies were placed in a 50 ml polypropylene tube that was kept in liquid nitrogen. The frozen flies were stored at -70 C until they were crushed in a crucible under liquid nitrogen. The powdered tissue together with some liquid nitrogen was decanted into a 50 ml polypropylene test tube on dry ice. After evaporation of liquid nitrogen, the powdered tissue was stored at -70 C.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Da se priredi RNA, 300 mg prakastog tkiva stavljeno je u polipropilensku epruvetu na suhi led i dodano je 5 ml 6 M guanidin hidroklorida u 0,1 M NaOAc, pH 5,2. Sve otopine bile su tretirane s DEPC ili prireene s DEPC-tretiranom dH2O i sav stakleni pribor je peen ili je koriten sasvim novi plastini pribort, da se smanje problemi s RNaza kontaminiranjem. Epruvete su mijeane na vorteks-mjeau pa stavljene na led. Prakasto tkivo je homogenizirano uzastopnim proputanjem kroz igle veliine 20, 21 i 22. Epruvete su centrifugirane (1000g tijekom 10 min), zatim je 2,5 do 3 ml supernatanta stavljeno na vrh 8 ml 5,7 M cezijeva klorida u 0,1 M NaOAc sadranom u 1495 mm Ultra-Clear epruveti za centrifugiranje (Beckman Instruments, Inc., Palo Alto, CA). Uzorci su centrifugirani na 25000 rpm tijekom 18 h na 18C u L8-70 ultracentrifugi (Beckman Instruments, Inc.,). Supernatant je odliven pa je epruveta okrenuta i ostavljena da se ocijedi. Talog RNA suspendiran je u 200 ml RNaza-slobodnoj dH2O (Qiagen Inc., Valencia, CA), zatim ispran dva puta sa 100 ml RNaza-slobodnom dH2O (ukupno, 400 ml). RNA je staloena dodatkom 44 ml 3M NaOAc, pH 5,2 i 1 ml hladnog 100% etanola. Nakon dranja preko noi na -70C, epruveta je centrifugirana na 14000 rpm tijekom 1 h (Eppendorf microfuge 5402), isprana s 75% etanolom (prireen s DEPC-tretiranom dH2O), zatim je talog otopljen u RNaza-slobodnoj dH2O. Apsorbancije izmjerene na 260 ili 280 nm odreene u 10 mM Tris-HCl, pH 7,5 koritene su za procjenu koncentracije i istoe RNA. </span>To prepare RNA, 300 mg of powdered tissue was placed in a polypropylene tube on dry ice and 5 ml of 6 M guanidine hydrochloride in 0.1 M NaOAc, pH 5.2, was added. All solutions were DEPC-treated or prepared with DEPC-treated dH2O and all glassware was baked or brand new plasticware was used, to reduce problems with RNase contamination. The test tubes were mixed on a vortex mixer and placed on ice. Pulverized tissue was homogenized by sequentially passing it through 20-, 21-, and 22-gauge needles. The tubes were centrifuged (1000g for 10 min), then 2.5 to 3 ml of the supernatant was placed on top of 8 ml of 5.7 M cesium chloride in 0, 1 M NaOAc contained in a 1495 mm Ultra-Clear centrifuge tube (Beckman Instruments, Inc., Palo Alto, CA). Samples were centrifuged at 25,000 rpm for 18 h at 18C in an L8-70 ultracentrifuge (Beckman Instruments, Inc.). The supernatant was poured off and the test tube was inverted and left to drain. The RNA pellet was suspended in 200 ml RNase-free dH2O (Qiagen Inc., Valencia, CA), then washed twice with 100 ml RNase-free dH2O (total, 400 ml). RNA was precipitated by adding 44 ml of 3M NaOAc, pH 5.2 and 1 ml of cold 100% ethanol. After keeping overnight at -70C, the tube was centrifuged at 14000 rpm for 1 h (Eppendorf microfuge 5402), washed with 75% ethanol (prepared with DEPC-treated dH2O), then the pellet was dissolved in RNase-free dH2O. Absorbances measured at 260 or 280 nm determined in 10 mM Tris-HCl, pH 7.5 were used to estimate RNA concentration and purity.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Prva zavojnica cDNA prireena je sukladno postupku koji je dostavljen sa Superscript II enzimom (GIBCO BRL, Rockville, MD). Bilo 500 ng (2 ml) poli A+ RNA ili 3 mg (4 ml) ukupne RNA dodano je u microfuge epruvete koje sadre RNaza-slobodnu dH2O i 250 ng (2,5 ml) sluajnih primera. Epruvete (12 ml) su inkubirane na 70 C tijekom 10 min, ohlaene na ledu, pa je dodano 4 ml 5  pufera prve zavojnice, 2 ml 0,1 M DTT i 1 ml mM dNTP smjese. Nakon inkubiranja na 25C tijekom 10 min, pa na 42C tijekom 2 min, dodan je 1 m1 (200 jedinica) Superscript II i nastavljeno je inkubiranjem na at 42C tijekom 50 min. Enzim je deaktiviran inkubiranjem na 70C tijekom 15 min. Da se ukloni RNA koja je komplementarna s cDNA, dodano je 2 ml (2 jedinice) RNaze H (Boehringer Mannheim, Indianapolis, IN), pa je nastavljeno inkubiranjem na 37C tijekom 20 min. cDNA pohranjena je na -20C. </span>First strand cDNA was prepared according to the procedure provided with Superscript II enzyme (GIBCO BRL, Rockville, MD). Either 500 ng (2 ml) of poly A+ RNA or 3 mg (4 ml) of total RNA was added to microfuge tubes containing RNase-free dH2O and 250 ng (2.5 ml) of random primers. Test tubes (12 ml) were incubated at 70 C for 10 min, cooled on ice, then 4 ml of 5  buffer of the first coil, 2 ml of 0.1 M DTT and 1 ml of mM dNTP mixture were added. After incubation at 25C for 10 min, then at 42C for 2 min, 1 m1 (200 units) of Superscript II was added and continued incubation at 42C for 50 min. The enzyme was deactivated by incubating at 70C for 15 min. To remove RNA complementary to the cDNA, 2 ml (2 units) of RNase H (Boehringer Mannheim, Indianapolis, IN) was added, followed by incubation at 37C for 20 min. cDNA was stored at -20C.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">PCR reakcije </span>PCR reactions</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Bilo standardna 50/100 ml PCR reakcija ili Hot Start PCR reakcija, koritenjem Ampliwax kuglica (Perlcin Elmer Cetus, Norwalk, CT) koritena je za pojaavanje Drosophila melanogaster G protein-vezanog receptora (DmGPCR-ovi). Destilirana H2O koritena je za otapanje primera (Genosys Biotechnologies, Inc., The Woodlands, TX): 5- i 3- primeri pri konventraciji 10 M, interni primeri sa 1 mM. Svaka PCR reakcija sadravala je 2 do 4 jedinice rTth XL DNA polimeraze, 1,2 do 1,5 mM Mg(OAc)2, 200 mM svakog dNTP i 200 ili 400 nM svakog primera. Za Hot Start PCR, 32 ili 36 ml nieg koktela (dH2O, 3,3  XL-buffer, dNTP i Mg(OAc)2 dodani su na 2 ili 4 ml svakog primer-a (ukupni volumen, 40 ml). Dodana je Ampliwax kuglica (Perkin Elmer Cetus), epruvete su inkubirane na 75C tijekom 5 min, ohlaene na sobnu temperaturu (RT), zatim je dodano 60 ml vieg koktela (dH2O, 3,3  XL-buffer, rTth i ablona). PCR pojaavanja izvrena su u termikom cikleru Perkin Elmer Series 9600. Tipini program za termiki cikler ukljuuje: 1 min na 94C, zatim 30 ciklus apojaavanja (0,5 min na 94C, 0,5 min na 60C, 2 min na 72C), zatim 6 min na 60C. Da bi se dobili 3 A-privjesci na PCR produktu (repovi), 1 ml Taq polimeraze (Invitrogen, Carlsbad, CA) dodano je na kraju PCR pojaavanja i epruvete su inkubirane na 72C tijekom 10 min. Reakcijske smjese su analizirane na 1% agaroza gelu koji je prireen u TAE puferu (5). PCR produkti su tipino proieni koritenjem QIAquick kolona (QIAGEN). </span>Either a standard 50/100 ml PCR reaction or a Hot Start PCR reaction, using Ampliwax beads (Perlcin Elmer Cetus, Norwalk, CT) was used to amplify Drosophila melanogaster G protein-coupled receptors (DmGPCRs). Distilled H2O was used to dissolve primers (Genosys Biotechnologies, Inc., The Woodlands, TX): 5'- and 3'- primers at a concentration of 10 M, internal primers at 1 mM. Each PCR reaction contained 2 to 4 units of rTth XL DNA polymerase, 1.2 to 1.5 mM Mg(OAc) 2 , 200 mM of each dNTP, and 200 or 400 nM of each primer. For Hot Start PCR, 32 or 36 ml of 'lower' cocktail (dH2O, 3.3  XL-buffer, dNTP and Mg(OAc)2) were added to 2 or 4 ml of each primer (total volume, 40 ml). An Ampliwax bead (Perkin Elmer Cetus) was added, the tubes were incubated at 75C for 5 min, cooled to room temperature (RT), then 60 ml of 'higher' cocktail (dH2O, 3.3  XL-buffer, rTth and template) was added ).PCR amplifications were performed in a Perkin Elmer Series 9600 thermal cycler. A typical thermal cycler program includes: 1 min at 94C, then 30 cycles of amplification (0.5 min at 94C, 0.5 min at 60C, 2 min at 72C) , then 6 min at 60C.To obtain 3' A-hangs on the PCR product ('tails'), 1 ml of Taq polymerase (Invitrogen, Carlsbad, CA) was added at the end of the PCR amplification and the tubes were incubated at 72C for 10 min Reaction mixtures were analyzed on a 1% agarose gel prepared in TAE buffer (5).PCR products were typically purified using QIAquick columns (QIAGEN).</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Ligacija i transformiranje </span>Ligation and transformation</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Ligacija svih PCR produkata u PCR 3.1 vektor (Invitrogen) i transformiranje ligatnih produkata u One Shot TOPI10F competentnu stanicu (Invitrogen) nainjeno je sukladno uputama proizvoaa. Tranformanti koje treba skenirati na inserte su propagirani u LB juhu koji sadri 50 mg ampicilin/ml. Kolonije s umecima identificirane su pomou mini-prep vrenje-lizis plazmid postupka (5) ili pomou kolonija PCR postupka koji izravno pojaava plazmid DNA iz transformirane bakterije (6). </span>Ligation of all PCR products into the PCR 3.1 vector (Invitrogen) and transformation of the ligated products into One Shot TOPI10F competent cells (Invitrogen) was performed in accordance with the manufacturer's instructions. Transformants to be scanned for inserts were propagated in LB broth containing 50 mg ampicillin/ml. Colonies with inserts were identified using the mini-prep boiling-lysis plasmid procedure (5) or using a 'colony PCR' procedure that directly amplifies plasmid DNA from transformed bacteria (6).</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Sekvenciranje DNA </span>DNA sequencing</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">DNA za sekvenciranje prireena je koritenjem Qiagen anionsko-izmjenjivakim plazmid priborima (QIAGEN-tip 20) da se izdvoji DNA iz 5 ml LB kultura koje su rasle na 37C preko noi sukladno naputku proizvoaa. etiri primera (T7, M13 reverzni, smisleni i protivsmisleni) se tipino koriste za sekvenciranje svake DNA (tablica 6). Koristi se kemija boja-terminatorskog sekvenciranja, bilo s BigDye  Terminator reagensima (Applied Biosystems, Foster City, CA) ili DYEnamic ET terminator priborom (Amersham Pharmacia Biotech, Inc., Piscataway, NJ). Za prireivanje sekvencirajuih reakcija slijedili smo naputke proizvoaa. Primeri i neugraeni nukleotidi uklonjeni su koritenjem Centri-Sep kolona (Princeton Separations, Adelphia, NJ). Reakcije sekvenciranja analizirane su na Applied Biosystems 377 automatiziranom DNA sekvenceru. DNA sekvencije su sastavljene i analizirane koritenjem sekvencera (Gene Codes, Ann Arbor, MI), GCG skupine programa za analizu sekvencija (Wisconsin Package Version 10.1, Genetics Computer Group (GCG), Madison, WI) i funkcija koje su raspoloive putem Vektor NTI 5.5 paketa programa (Mormax, Bethesda, MD). </span>DNA for sequencing was prepared using Qiagen anion-exchange plasmid kits (QIAGEN-type 20) to extract DNA from 5 ml LB cultures grown at 37C overnight according to the manufacturer's instructions. Four primers (T7, M13 reverse, 'sense' and 'antisense') are typically used to sequence each DNA (Table 6). Dye-terminator sequencing chemistry is used, either with BigDye Terminator reagents (Applied Biosystems, Foster City, CA) or DYEnamic ET terminator kit (Amersham Pharmacia Biotech, Inc., Piscataway, NJ). To prepare the sequencing reactions, we followed the manufacturer's instructions. Primers and unincorporated nucleotides were removed using Centri-Sep columns (Princeton Separations, Adelphia, NJ). Sequencing reactions were analyzed on an Applied Biosystems 377 automated DNA sequencer. DNA sequences were assembled and analyzed using a sequencer (Gene Codes, Ann Arbor, MI), GCG Group sequence analysis program (Wisconsin Package Version 10.1, Genetics Computer Group (GCG), Madison, WI) and functions available through Vektor NTI 5.5 program package (Mormax, Bethesda, MD).</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Tablica 6. DNA sekvencirajui primeri </span>Table 6. DNA sequencing primers</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">[image] </span>[image]</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Rezultati kloniranja i sekvenciranja DmGPCR-ova ovog izuma su sljedei: </span>The results of cloning and sequencing of the DmGPCRs of this invention are as follows:</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">DmGPCR1 </span>DmGPCR1</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">PCR primeri oblikovani za cDNA koja odgovara PnuFlyPep34651 koriteni su za uspjeno pojaavanje PCR produkta iz cDNA pripravka koji je prireen iz Drosophila polyA+ mRNA. Dobiveni produkt je kloniran i sekvenciran. Eksperimentalno dobivena sekvencija identina je predvienoj sekvenciji. Intaktni klon je dobiven i oznaen DmGPCR1. </span>PCR primers designed for cDNA corresponding to PnuFlyPep34651 were used to successfully amplify the PCR product from a cDNA preparation prepared from Drosophila polyA+ mRNA. The obtained product was cloned and sequenced. The experimentally obtained sequence is identical to the predicted sequence. An intact clone was obtained and designated 'DmGPCR1'.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">DmGPCR2 </span>DmGPCR2</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Poetni pokuaji da se pojaa PCR produkt koritenjem primera koji su oblikovani prema cDNA koja odgovara PnuFlyPep67585 nisu bili uspjeni. Usklaenost previene sekvencije prema postojeim C. elegans receptorima i prema drugim neuropeptidnim receptorima pokazuje da je 5 kraj previene sekvencije neuobiajeno dugaak to ukazuje da moe postojati pogreka u predikciji gena na toj strani. Koritenje genomske sekvencije kao vodia, oblikovan je i ispitan cijeli niz alternativnih 5 PCR primera. Jedna od ovih primer kombinacija, koritenjem cDNA koja je prireena iz ukupne RNA, bila je uspjena u dobivanju produkta prave veliine. Sekvenciranje klonova koji su izvedeni iz PCR reakcije pokazalo je da pojaani product sadri predviene 5 i 3 krajeve i da je identian predvienoj sekvenciji s izuzetkom da je u pretpostavljenoj sekvenciji nedostajao maleni segment od 6 aminokiselina. Usporeivanje klonova je takoer pokazalo da postoje dvije isprepletene izoforme, jedna koja je slina predvienoj sekvenciji (oznaeno DmGPCR2a) i druga kojoj nedostaje segment od 23 aminokiseline koja je smjetena upravo nakon TM VII u unutarstaninom C-terminalu molekule (oznaeno DmGPCR2b). </span>Initial attempts to amplify the PCR product using primers designed to the cDNA corresponding to PnuFlyPep67585 were unsuccessful. Alignment of the overlooked sequence to existing C. elegans receptors and to other neuropeptide receptors shows that the 5' end of the overlooked sequence is unusually long, indicating that there may be an error in gene prediction on that side. Using the genomic sequence as a guide, a series of alternative 5' PCR primers were designed and tested. One of these primer combinations, using cDNA prepared from total RNA, was successful in obtaining a product of the correct size. Sequencing of the clones derived from the PCR reaction showed that the amplified product contained the predicted 5' and 3' ends and was identical to the predicted sequence with the exception that a small segment of 6 amino acids was missing from the predicted sequence. Comparison of the clones also showed that there are two spliced isoforms, one that is similar to the predicted sequence (designated 'DmGPCR2a') and another that lacks a 23 amino acid segment located just after TM VII in the intracellular C-terminus of the molecule (designated DmGPCR2b').</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">DmGPCR3 </span>DmGPCR3</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Gen koji odgovara DmGPCR3 predvienom proteinu ve je poznat iz literature. Ovaj gen (GenBank pristupni broj M77168) opisan je kao NKD, razvojem reguliran tahikininski receptor. Monnier D, et al, J. Biol. Chem. 1992, 267(2), 1298-302. Usporeivanje M77168 i PnuFlyPep68505 sekvencija pokazalo je da se previene sekvencije znaajno razlikuju od cDNA. cDNA imala je dulji 5 kraj, nedostajao je ekson koji kodira 51 aminokiseline i bila je znaajno kraa na 3 kraju. PCR primeri su oblikovani prema puliciranoj sekvenciji i PCR produkt je dobiven koritenjem cDNA koja je prireena iz ukupne RNA. Ovaj produkt bio je identine strukture s publiciranom NKD sekvencijom. </span>The gene corresponding to the DmGPCR3 predicted protein is already known from the literature. This gene (GenBank accession number M77168) is described as NKD, "developmentally regulated tachykinin receptor". Monnier D, et al, J. Biol. Chem. 1992, 267(2), 1298-302. Comparison of the M77168 and PnuFlyPep68505 sequences showed that the overlooked sequences were significantly different from the cDNA. The cDNA had a longer 5' end, lacked an exon encoding 51 amino acids, and was significantly shorter at the 3' end. PCR primers were designed according to the published sequence and the PCR product was obtained using cDNA prepared from total RNA. This product was identical in structure to the published NKD sequence.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">DmGPCR4 </span>DmGPCR4</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">cDNA koja odgovara PnuFlyPep 67393 koritena je za oblikovanje PCR primera za pojaanje DmGPCR4. Koritenjem cDNA biblioteke koja je prireena iz ukupne Drosophila mRNA, prireen je i kloniran PCR produkt. Usporeivanje klonova sa sekvencijom koja je predviena pomou PnuFlyPep pokazalo je da su sekvencije identine s izuzetkom da jedan ekson koji je previen pomou HMMGene nije bio prisutan u bilo kojem od kloniranih PCR produkata. DmGPCR4 je nedavno kloniran: Lenz et al, Biochem. Biophys. Res. Comm., 2000, 273, 571-577 i klasificiran je ako drugi potencijalni alatostatinski receptor. </span>The cDNA corresponding to PnuFlyPep 67393 was used to design PCR primers to amplify DmGPCR4. Using a cDNA library prepared from total Drosophila mRNA, a cloned PCR product was prepared. Comparison of the clones with the sequence predicted by PnuFlyPep showed that the sequences were identical with the exception that one exon overlooked by HMMGene was not present in any of the cloned PCR products. DmGPCR4 was recently cloned: Lenz et al, Biochem. Biophys. Crisp. Comm., 2000, 273, 571-577 and is classified as another potential allatostatin receptor.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">DmGPCR-ovi-ovi </span>DmGPCRs</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">DmGPCR5 (FlyPepCG7887) netono sadri frameshift mutaciju. PnuFlyPep inaica, PnuFlyPep67522, koja je opisana u literaturi kao Drosophila receptor za tahikinin-srodne peptide (M77168) (Li XJ, et al, EMBO Journal, 1991, 10(11), 3221-3229), ispravlja tu pogreku ali netono predvia neke unutranje sekvencije i C-terminal. Na prvi pogled, predviena cDNA koja odgovara PnuFlyPep proteinu bila je identina publiciranoj sekvenciji. PCR primeri koriteni su za uspjeno pojaavanje PCR produkta odgovarajue veliine iz cDNA smjese koja je prireena iz Drosophila melanogaster poli A+ mRNA. Sekvenciranje kloniranih PCR produkata pokazalo je da, premda je ukupna splicing slika identina, postoje dvije pogreke sekvenciranja u PnuFlyPep sekvenciji. Ove pogreke rezultiraju frameshift mutacijom nakon koje slijedi kompenzatorska frameshift mutacija to daje razliku od 13 aminokiselina izmeu eksperimentalno naenih i objelodanjenih sekvencija, polazei od aminokiseline na poloaju 46. Ovaj kloniran gen je oznaen DmGPCR5a. </span>DmGPCR5 (FlyPepCG7887) incorrectly contains a frameshift mutation. A PnuFlyPep variant, PnuFlyPep67522, which is described in the literature as 'Drosophila receptor for tachykinin-related peptides' (M77168) (Li XJ, et al, EMBO Journal, 1991, 10(11), 3221-3229), corrects that error but incorrectly predicts some internal sequences and the C-terminal. At first glance, the predicted cDNA corresponding to the PnuFlyPep protein was identical to the published sequence. PCR primers were used to successfully amplify a PCR product of the appropriate size from a cDNA mixture prepared from Drosophila melanogaster poly A+ mRNA. Sequencing of the cloned PCR products showed that, although the overall splicing pattern was identical, there were two sequencing errors in the PnuFlyPep sequence. These errors result in a frameshift mutation followed by a compensatory frameshift mutation that gives a difference of 13 amino acids between the experimentally found and published sequences, starting at amino acid position 46. This cloned gene is designated 'DmGPCR5a.'</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Nadalje, splicing izoforme naene su za DmGPCR5. Ova inaica kodira ekstra tri aminokiseline u N-terminalnoj izvanstaninoj domeni. Ova inaica je oznaena DmGPCR5b. </span>Furthermore, splicing isoforms were found for DmGPCR5. This variant encodes an extra three amino acids in the N-terminal extracellular domain. This version is designated 'DmGPCR5b'.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">DmGPCR6 </span>DmGPCR6</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">GPCR koji odgovara PnuFlyPep15731 ve je opisan u literature kao neuropeptidni Y receptor (M81490. Li XJ, et al., J. Biol. Chem., 1992, 267(1), 9-12). PnuFlyPep-predviena sekvencija razlikuje se od M81490 na oba kraja molekule. PnuFlyPep15731 sadri ekstra 15 aminokiselinu na N-terminu u usporedbi s M81490. 3 kraj PnuFlyPep 15731 takoer se razlikuje od M81490, bio je krnji i nije sadravao konzervirane TM VI i TM VII rezidue. </span>The GPCR corresponding to PnuFlyPep15731 has already been described in the literature as a 'neuropeptide Y' receptor (M81490. Li XJ, et al., J. Biol. Chem., 1992, 267(1), 9-12). The PnuFlyPep-predicted sequence differs from M81490 at both ends of the molecule. PnuFlyPep15731 contains an extra 15 amino acids at the N-terminus compared to M81490. The 3' end of PnuFlyPep 15731 also differs from M81490, being truncated and lacking the conserved TM VI and TM VII residues.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Poetni PCR primeri su oblikovani koritenjem sekvencije M81490. Koritenjem ovih primera i ablone koja je izvedena iz ukupne mRNA, dobiven je PCR produkt. Ispitivanje kloniranog PCR produkta pokazalo je da on koristi identinu sliku onoj M81490. Ovaj klon je oznaen DmGPCR6a. </span>Initial PCR primers were designed using sequence M81490. Using these primers and a template derived from total mRNA, a PCR product was obtained. Examination of the cloned PCR product showed that it uses an identical image to that of M81490. This clone was designated 'DmGPCR6a'.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Tijekom kloniranja DmGPCR-a6a otkrivena je dodatna splicing izoforma. Ova izoforma generirana je upotrebom alternativnog splice akceptorskog mjesta da se generira alternativni 3 kraj molekule koritenjem gotovo identine sekvencijske 6a forme ali u razliitom okviru za itanje. Nadalje, otvoreni okvir za itanje za ovaj klon proirio je proli izvorni 3 PCR primer. Ispitivanje genomske sekvencije na 3 kraju pokazalo je odreen broj slinih kandidatskih eksona. PCR primeri koji odgovaraju broju moguih eksona su ispitani sve dok nije naen jedan koji bi mogao pojaati PCR produkt. Ovaj produkt je oznaen 6b. Ispitivanje genomske sekvencije takoer je predvidjelo da je inicijator ATG koji je predvien pomou PnuFlyPep 15731 bio in-frame s M81490 inicijacijski kodonom koji sadri ekstra 15 aminokiselina i da je vjerojatno da je PnuFlyPep 15731 poetni kodon bio autentian startni kodon. Oblikovan je novi 5 PCR primer koji ima ugraen PnuFlyPep 15731 startni kodon i koriten je u sprezi s dva 3 PCR primera za pojaavanje i kloniranje DmGPCR6aL i DmGPCR6bL (dugaak).  </span>During the cloning of DmGPCR-a6a, an additional splicing isoform was discovered. This isoform was generated using an alternative splice acceptor site to generate an alternative 3' end of the molecule using an almost identical sequence '6a' form but in a different reading frame. Furthermore, the open reading frame for this clone extended past the original 3' PCR primer. Examination of the genomic sequence at the 3' end revealed a certain number of similar candidate exons. PCR primers corresponding to the number of possible exons were tested until one was found that could amplify the PCR product. This product is marked '6b'. Examination of the genomic sequence also predicted that the initiator ATG predicted by PnuFlyPep 15731 was in-frame with the M81490 initiation codon containing an extra 15 amino acids and that it was likely that the PnuFlyPep 15731 start codon was the authentic start codon. A new 5' PCR primer incorporating the PnuFlyPep 15731 start codon was designed and used in conjunction with two 3' PCR primers to amplify and clone 'DmGPCR6aL' and 'DmGPCR6bL' ('long').</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">DmGPCR7 </span>DmGPCR7</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Poetni pokuaji da se pojaa DmGPCR7 genski product bili su neuspjeni. Poklapanje predviene sekvencije (PnuFlyPep67863) s drugim GPCR-ovima ukazalo je na to da je greka vjerojatno bila u predvianju 3 kraja molekule. Predviena sekvencija imala je 3 kraj koji je bio daleko dulji nego onaj od veine drugih GPCR-ova. Ispitivanje genomske sekvencije ukazalo je da je vjerojatna pogreka u predvianju splicing dogaaja koji je uklonio in-frame stop kodon koji bi rezultirao molekulom odgovarajue veliine. 3 PCR primer oblikovan je unutar tog introna. Nadalje, oblikovan je novi 5 PCR primer da se iskoristi in-frame ATG upravo uzvodno od predvienog startnog kodona. PCR pojaavanje cDNA izvedene iz ukupne mRNA rezultiralo je produktom oekivane veliine. </span>Initial attempts to amplify the DmGPCR7 gene product were unsuccessful. The match of the predicted sequence (PnuFlyPep67863) with other GPCRs indicated that the error was probably in the prediction of the 3' end of the molecule. The predicted sequence had a 3' end that was much longer than that of most other GPCRs. Examination of the genomic sequence indicated a likely error in predicting a splicing event that removed an in-frame stop codon that would have resulted in a molecule of the appropriate size. The 3' PCR primer was designed within that intron. Furthermore, a new 5' PCR primer was designed to utilize the in-frame ATG just upstream of the predicted start codon. PCR amplification of cDNA derived from total mRNA resulted in a product of the expected size.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">PnuFlyPep i WO 01/70980 inaice DmGPCR-a7 obje imaju manjak dviju aminokiselina na N-terminalu. Kao to je prije navedeno, PnuFlyPep i FlyPep CGI0626 inaice su takoer netone na C-terminalu. Netone inaice DmGPCR-a7 genskog produkta predviene su da budu potencijalni Drosophila leukokininski receptori (npr. Hewes &amp; Taghert, Genome Res., 2001,11, 1126-1142; Holmes et al, Insect. Mol. Biol., 2000,9, 457-465). Meutim, prije ovog izuma nije bilo eksperimentalnih dokaza da se potvrdi ovo oekivanje. </span>The PnuFlyPep and WO 01/70980 versions of DmGPCR7 both lack two amino acids at the N-terminal. As previously noted, the PnuFlyPep and FlyPep CGI0626 versions are also incorrect at the C-terminal. Missile versions of the DmGPCR-a7 gene product are predicted to be potential Drosophila leukokinin receptors (eg, Hewes &amp; Taghert, Genome Res., 2001,11, 1126-1142; Holmes et al, Insect. Mol. Biol., 2000,9, 457 -465). However, prior to this invention there was no experimental evidence to confirm this expectation.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">DmGPCR8 </span>DmGPCR8</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">DmGPCR8 je uspjeno pojaan koritenjem PCR primera koji su oblikovani prema PnuFlyPep predvienoj sekvenciji. cDNA izvedena iz poli A+ RNA koritena je kao ablona za PCR reakciju. Svih est sekvenciranih klonova bilo je identine strukture s PnuFlyPep-predvienom sekvencijom. Polimorfizam je zabiljeen u poloaju #68 (DNA sekvencija), pri emu polovina klonova ima C u ovom poloaju i druga polovina A. Ova promjena ne rezultira aminokiselinskom promjenom, Asp, odnosno Glu. Celera sekvencija zabiljeila je A, tako da je A klon (Glu) proizvoljno odabran za daljnje istraivanje. Nijedan A klon nije dobiven u tonoj orijentaciji, pa je dakle stupanj subkloniranja, koji koristi Pme I da se ukloni umetak iz izvornog pCR3.1 klona i Pme I-razgraeni pCR3.1 vektor, koriten da se obrne orijentacija. </span>DmGPCR8 was successfully amplified using PCR primers designed according to the PnuFlyPep predicted sequence. cDNA derived from poly A+ RNA was used as a template for the PCR reaction. All six sequenced clones were identical in structure to the PnuFlyPep-predicted sequence. The polymorphism was noted at position #68 (DNA sequence), with half of the clones having a "C" at this position and the other half having an "A." This change does not result in an amino acid change, Asp or Glu. The Celera sequence noted "A," so the "A" clone (Glu) was arbitrarily selected for further investigation. No "A" clone was obtained in the correct orientation, so a subcloning step, using Pme I to remove the insert from the original pCR3.1 clone and the Pme I-digested pCR3.1 vector, was used to reverse the orientation.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">PnuFlyPep inaica je tona. WO 01/70980 inaici, meutim, nedostaje priblino 17 N-terminalnih aminokiselina i priblino 15 internih aminokiselina. Ovaj receptor je klasificiran kao potencijalni somatostatin-slini receptor (npr. Hewes &amp; Taghert, Genome Res., 2001, 11, 1126-1142). Nijedan eksperimentalni dokaz prije ovog izuma nije potvrdio ovo predvianje. </span>The PnuFlyPep version is correct. The WO 01/70980 version, however, lacks approximately 17 N-terminal amino acids and approximately 15 internal amino acids. This receptor has been classified as a potential somatostatin-like receptor (eg, Hewes &amp; Taghert, Genome Res., 2001, 11, 1126-1142). No experimental evidence prior to this invention confirmed this prediction.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">DmGPCR9 </span>DmGPCR9</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">DmGPCR.9 je kloniran koritenjem PCR primera oblikovanih za PnuFlyPep predvienu sekvenciju i cDNA ablone prep prireene iz poli A+ RNA. Genomske strukture su tono predviene u PnuFlyPep. </span>DmGPCR.9 was cloned using PCR primers designed for the PnuFlyPep predicted sequence and a cDNA template prep prepared from poly A+ RNA. Genomic structures are accurately predicted in PnuFlyPep.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">DmGPCRIO </span>DmGPCRIO</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Poetni pokuaji da se dobije PCR produkt s primerima koji su oblikovani za DmGPCRIO (PnuFlyPep70325) nisu bili uspjeni. Ispitivanje predviene cDNA pokazalo je da je predviena sekvencija neobina i da ne sadri vrlo sauvani WXP motiv u TM VI, niti NPXXF motiv u TM VII, mada je prisutno nekoliko drugih sauvanih rezidua. Ispitivanje genomskih sekvencija do 80 kb uzvodno od posljednjeg eksona nije dalo neke druge potencijalne eksone. Pokuaji da se dobije intaktni klon za DmGPCRIO nisu bili uspjeni. </span>Initial attempts to obtain a PCR product with primers designed for DmGPCRIO (PnuFlyPep70325) were unsuccessful. Examination of the predicted cDNA showed that the predicted sequence is unusual and does not contain the highly conserved WXP motif in TM VI, nor the NPXXF motif in TM VII, although several other conserved residues are present. Examination of genomic sequences up to 80 kb upstream of the last exon did not yield any other potential exons. Attempts to obtain an intact clone for DmGPCRIO were unsuccessful.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">DmGPCR1 1 (alatostatinu slian peptidni receptor) </span>DmGPCR1 1 (allatostatin-like peptide receptor)</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">PCR primeri za alatostatin-slini peptidni receptor su oblikovani koritenjem publiciranih sekvencija. Birgul et at, EMBO Journal, 1999, 75(21), 5892- 5900. PCR produkt je dobiven koritenjem cDNA koja je izvedena iz ukupne mRNA prep te je klonirana i sekvencirana. Konana cDNA kodirana za protein identina je onoj koja je opisana u publikaciji. </span>PCR primers for the allatostatin-like peptide receptor were designed using published sequences. Birgul et at, EMBO Journal, 1999, 75(21), 5892-5900. The PCR product was obtained using cDNA that was derived from total mRNA prep and was cloned and sequenced. The final cDNA encoding the protein is identical to that described in the publication.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Primjer 3: Northern Blot analiza </span>Example 3: Northern Blot analysis</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Northern blots moe se izvriti da se istrai ekspresija mRNA. Oligonukleotidi smislene orijentacije i oligonukleotidi protivsmislene orijentacije, opisani prije, koriste se kao primeri da se pojaa dio sekvencije GPCR cDNA iz nukleotidne sekvencije koja je odabrana iz skupa kojega sainjavaju sekvencije s br. 1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21 i 23. </span>Northern blots can be performed to investigate mRNA expression. The sense-orientation oligonucleotides and the antisense-orientation oligonucleotides described above are used as primers to amplify a portion of the GPCR cDNA sequence from a nucleotide sequence selected from the group consisting of sequences no. 1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21 and 23.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Vie humanih tkiva northern blot od tvrtke Clontech (Human II # 7767-1) hibridizirano je sa sondom. Prehibridiziranje je izvreno na 42C tijekom 4 sata u 5SSC, 1x Denhardtov reagens, 0,1% SDS, 50% formamid, 250 mg/ml DNA sperme lososa. Hibridiziranje je izvreno preko noi na 42C u istoj smjesi s dodatkom oko 1,5106 cpm/ml obiljeene sonde. </span>Multiple human tissue northern blots from Clontech (Human II # 7767-1) were hybridized with the probe. Prehybridization was performed at 42C for 4 hours in 5SSC, 1x Denhardt's reagent, 0.1% SDS, 50% formamide, 250 mg/ml salmon sperm DNA. Hybridization was performed overnight at 42C in the same mixture with the addition of about 1.5106 cpm/ml labeled probe.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Sonda je obiljeena s -32P-dCTP pomou Rediprime DNA sustava za obiljeavanje (Amersham Pharmacia), proiena na Nick koloni (Amersham Pharmacia) i dodana otopini za hibridiziranje. Filteri su isprani nekoliko puta na 42C u 0,2 SSC, 0,1% SDS. Filteri su postavljeni na Kodak XAR film (Eastman Kodak Company, Rochester, N.Y., USA) s intenzivirajuim zaslonom -80.  </span>The probe was labeled with -32P-dCTP using the Rediprime DNA labeling system (Amersham Pharmacia), purified on a Nick column (Amersham Pharmacia), and added to the hybridization solution. Filters were washed several times at 42C in 0.2 SSC, 0.1% SDS. Filters were mounted on Kodak XAR film (Eastman Kodak Company, Rochester, N.Y., USA) with an intensifying screen -80.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Primjer 4: Rekombinantna ekspresija DmGPCR u eukariotskim stanicama </span>Example 4: Recombinant expression of DmGPCR in eukaryotic cells</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Ekspresija DmGPCR u stanicama sisavaca </span>Expression of DmGPCR in mammalian cells</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Da se dobije DmGPCR protein, DmGPCR-kodirajui polinukleotid podvrgnut je ekspresiji u odgovarajuoj stanici domaina koritenjem odgovarajueg ekspresijskog vektora i standardnih tehnika genetskog inenjeringa. Primjerice, DmGPCR-kodirajua sekvencija koja je opisana u primjeru 1 je subklonirana u komercijalni ekspresijski vektor pzeoSV2 (Invitrogen, San Diego, CA) i transficirana u stanicama jajnika kineskog zamorca (CHO) koritenjem transfekcijskog reagensa FuGENE 6 (Boehmger-Mannheim) i protokola transfekcije koji je priloen produktu. Druge linije eukariotskih stanica, ukljuujui bubreg ovjejeg zametka (HEK 293) i COS stanice, primjerice, takoer su odgovarajue. Stanice s postojanom ekspresijom DmGPCR odabrane su rastom u prisutnosti 100 g/ml zeocina (Stratagene, LaJolla, CA). Proizvoljno, DmGPCR moe biti proien iz stanica koritenjem standardnih kromatografskih tehnika. Da se olaka proiavanje, antiserum je povien prema jednoj ili vie sekvencija sintetskog peptida koje odgovaraju dijelovima DmGPCR-a aminokiselinske sekvencije i antiserum je koriten za afinitetno proiavanje DmGPCR. DmGPCR takoer moe biti s ekspresijom in-frame s tag sekvencijom (npr. polihistidin, hemaglutinin, FLAG) da se olaka proiavanje. tovie, valja uoiti da mnoge upotrebe DmGPCR polipeptida, kao to su analize nie opisane, ne zahtijevaju proiavanje DmGPCR-a iz stanice domaina. </span>To obtain the DmGPCR protein, the DmGPCR-encoding polynucleotide is subjected to expression in an appropriate host cell using an appropriate expression vector and standard genetic engineering techniques. For example, the DmGPCR-coding sequence described in Example 1 was subcloned into the commercial expression vector pzeoSV2 (Invitrogen, San Diego, CA) and transfected in Chinese guinea pig ovary (CHO) cells using the FuGENE 6 transfection reagent (Boehmger-Mannheim) and the transfection protocol which is attached to the product. Other eukaryotic cell lines, including human embryonic kidney (HEK 293) and COS cells, for example, are also suitable. Cells stably expressing DmGPCR were selected by growth in the presence of 100 g/ml zeocin (Stratagene, LaJolla, CA). Optionally, DmGPCR can be purified from cells using standard chromatographic techniques. To facilitate purification, antiserum was raised against one or more synthetic peptide sequences corresponding to portions of the DmGPCR amino acid sequence and the antiserum was used to affinity purify the DmGPCR. The dmGPCR can also be expressed in-frame with a tag sequence (eg, polyhistidine, hemagglutinin, FLAG) to facilitate purification. Moreover, it should be noted that many uses of DmGPCR polypeptides, such as the assays described below, do not require purification of the DmGPCR from the host cell.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Ekspresija DmGPCR u 293 stanica </span>Expression of DmGPCR in 293 cells</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Za ekspresiju DmGPCR-a u 293 stanicama, prireen je plazmid koji nosi relevantnu DmGPCR kodirajuu sekvenciju, koritenjem vektora pSecTag2A (Invitrogen). Vektor pSecTag2A sadri miju IgK vodeu sekvenciju lanca za sekreciju, c-myc epitop za detektiranje rekombinantnog proteina s anti-myc antitijelom, C-terminalni polihistidin za proiavanje s nikal kelatnom kromatografijom i Zeocin rezistentni gen za odabiranje postojanih transfektanata. Forward primer za pojaavanje ove GPCR cDNA odreen je rutinskim postupcima i poeljno sadri 5 ekstenziju nukleotida za uvoenje HindIII klonirajueg mjesta i nukleotida koji odgovaraju GPCR sekvenciji. Obrnuti primer je takoer odreen rutinskim postupcima i poeljno sadri 5 ekstenziju nukleotida da se uvede XhoI restrikcijsko mjesto za kloniranje i nukleotidi koji odgovaraju obrnutom komplementu sekvencije DmGPCR-a. PCR uvjeti su 55C kao temperature peenja. PCR produkt je proien na gelu i kloniran u HindIII-XhoI mjestima vektora. </span>For DmGPCR expression in 293 cells, a plasmid carrying the relevant DmGPCR coding sequence was prepared using the pSecTag2A vector (Invitrogen). The pSecTag2A vector contains the mouse IgK leader sequence for secretion, c-myc epitope for detection of recombinant protein with anti-myc antibody, C-terminal polyhistidine for purification with nickel chelate chromatography and Zeocin resistance gene for selection of stable transfectants. The forward primer for amplifying this GPCR cDNA is determined by routine procedures and preferably contains a 5' extension of nucleotides for the introduction of the HindIII cloning site and nucleotides corresponding to the GPCR sequence. The reverse primer is also determined by routine procedures and preferably contains a 5' nucleotide extension to introduce an XhoI restriction site for cloning and nucleotides corresponding to the reverse complement of the DmGPCR sequence. PCR conditions are 55C as baking temperature. The PCR product was gel purified and cloned into the HindIII-XhoI sites of the vector.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">DNA je proiena koritenjem Qiagen kromatografskih kolona i transficirana u 293 stanice koritenjem DOTAP transfekcijskog medija (Boehringer Mannheim, Indianapolis, IN). Prijelazno transficirane stanice ispitane su na nakon 24 sata transfekcije, koritenjem western blots sondiranja s antiHis i anti-DmGPCR peptidnim antitijelima. Stalno transficirane stanice oidabrane su sa Zeocinom i propagirane. Proizvodnja rekombinantnog proteina detektirana je iz stanice i medija pomou western blots sondiranja s anti-His, anti-Myc ili anti-GPCR peptidnim antitijelima. </span>DNA was purified using Qiagen chromatography columns and transfected into 293 cells using DOTAP transfection medium (Boehringer Mannheim, Indianapolis, IN). Transiently transfected cells were examined 24 hours after transfection, using western blots probing with antiHis and anti-DmGPCR peptide antibodies. Permanently transfected cells were selected with Zeocin and propagated. Recombinant protein production was detected from cells and media using western blots probing with anti-His, anti-Myc or anti-GPCR peptide antibodies.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Ekspresija DmGPCR u COS stanicama </span>Expression of DmGPCR in COS cells</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Za ekspresiju DmGPCR-a u COS7 stanicama, polinukleotidna molekula koja ima nukleotidnu sekvenciju koja je odabrana iz skupa kojega sainjavaju sekvencije s br. 1, 3, 5,7, 9, 11, 13, 15, 17, 19, 21 ili 23 moe biti klonirana u vektor p3-CI. Ovaj vektor je pUC18-izvedeni plasmid koji sadri HCMV (humani citomegalovirus) promotor-intron koji je smjeten uzvodno od bGH (volovski hormon rasta) poliadenilatcijske sekvencije i viestrukog klonirajueg mjesta. Nadalje, plazmid sadri dhfr (dihidrofolat reduktaza) gen koji definira odabiranje u prisutnosti lijeka metotreksana (MTX) za odabiranje postojanih transformanata. </span>For expression of DmGPCR in COS7 cells, a polynucleotide molecule having a nucleotide sequence selected from the group consisting of sequences with no. 1, 3, 5,7, 9, 11, 13, 15, 17, 19, 21 or 23 can be cloned into vector p3-CI. This vector is a pUC18-derived plasmid containing the HCMV (human cytomegalovirus) promoter-intron located upstream of the bGH (bovine growth hormone) polyadenylation sequence and a multiple cloning site. Furthermore, the plasmid contains a dhfr (dihydrofolate reductase) gene that defines selection in the presence of the drug methotrexan (MTX) to select stable transformants.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Forward primer odreen je rutinskim postupcima i poeljno sadri 5 ekstenziju koja uvodi XbaI restrikcijsko mjesto za kloniranje, nakon ega slijede nukleotidi koji odgovaraju nukleotidnoj sekvenciji koja je odabrana iz skupa kojega sainjavaju sekvencije s br. 1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21 ili 23. Reverzni primer je takoer odreen rutinskim postupcima i poeljno sadri 5- ekstenziju nukleotida koji uvode SaiI klonirajue mjesto nakon ega slijede nukleotidi koji odgovaraju reverznom komplementu nukleotidne sekvencije koja je odabrana iz skupa sekvencija s br. 1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21 ili 23. </span>The forward primer is determined by routine procedures and preferably contains a 5' extension that introduces an XbaI restriction site for cloning, followed by nucleotides corresponding to a nucleotide sequence selected from the set consisting of sequences with no. 1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21 or 23. The reverse primer is also determined by routine procedures and preferably contains a 5'-extension of nucleotides introducing the SaiI cloning site followed by nucleotides corresponding to to the reverse complement of the nucleotide sequence which is selected from the set of sequences with no. 1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21 or 23.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">PCR se sastoji od poetnog stupnja denaturiranja 5 min na 95C, 30 ciklusa od 30 sekundi denaturiranja na 95C, 30 sekundi peenja na 58C i 30 sekundi ekstenzije na 72C, nakon ega slijedi 5 min ekstenzije na 72C. PCR produkt je proien na gelu i ligiran u XbaI i SaiI mjesta vektora p3-CI. Ovaj konstrukt je transformiran u E. coli stanice za pojaavanje i DNA proiavanje. DNA je proiena pomou Qiagen kromatografije na koloni i tranficirana u COS7 stanice koritenjem Lipofektamin reagensa tvrtke BRL, sukladno protokolima proizvoaa. etrdeset osam i 72 sata nakon transfekcije, mediji i stanice ispitani su na ekspresiju rekombinantnog proteina. </span>The PCR consists of an initial denaturation step of 5 min at 95C, 30 cycles of 30 seconds of denaturation at 95C, 30 seconds of annealing at 58C and 30 seconds of extension at 72C, followed by 5 min of extension at 72C. The PCR product was gel purified and ligated into the XbaI and SaiI sites of the p3-CI vector. This construct was transformed into E. coli cells for amplification and DNA purification. DNA was purified using Qiagen column chromatography and transfected into COS7 cells using Lipofectamine reagent from BRL, according to the manufacturer's protocols. Forty-eight and 72 hours after transfection, media and cells were assayed for recombinant protein expression.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">DmGPCR s ekspresijom iz COS stanine kulture moe se proistiti koncentriranjem medija za stanini rast do oko 10 mg protein/ml i proiavanjem proteina pomou, primjerice, kromatografije. Proieni DmGPCR je koncentriran na 0,5 mg/ml u Amicon koncentratoru koji je opremljen s YM-10 membranom i pohranjen na -80C. </span>DmGPCR expressed from COS cell culture can be purified by concentrating the cell growth medium to about 10 mg protein/ml and purifying the protein using, for example, chromatography. Purified DmGPCR was concentrated to 0.5 mg/ml in an Amicon concentrator equipped with a YM-10 membrane and stored at -80C.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Ekspresija DmGPCR u insektnim stanicama </span>Expression of DmGPCR in insect cells</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Za ekspresiju DmGPCR-a u akulovirusnom sustavu, polinukleotidna molekula koja ima nukleotidnu sekvenciju koja je odabrana iz skupa kojega sainjavaju sekvencije s br. 1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21 ili 23 moe se pojaati pomou PCR. Forward primer odreen je rutinskim postupcima i poeljno sadri 5 ekstenziju koja dodaje NdeI klonirajue mjesto, nakon ega slijede nukleotidi koji odgovaraju nukleotidnoj sekvenciji koja je odabrana iz skupa kojega sainjavaju sekvencije s br. 1, 3, 5,1, 9, 11, 13, 15, 17, 19, 21 ili 23. Reverzni primer je takoer odreen rutinskim postupcima i poeljno sadri 5 ekstenziju koja unosi KpnI klonirajue mjesto, nakon ega slijede nukleotidi koji odgovaraju obrnutom komplementu nukleotidne sekvencije koja je odabrana iz skupa kojega sainjavaju sekvencije s br. 1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21 ili 23. </span>For the expression of DmGPCR in an aculovirus system, a polynucleotide molecule having a nucleotide sequence selected from the group consisting of sequences with no. 1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21 or 23 can be amplified by PCR. The forward primer is determined by routine procedures and preferably contains a 5' extension that adds an NdeI cloning site, followed by nucleotides corresponding to a nucleotide sequence selected from the set consisting of sequences with no. 1, 3, 5, 1, 9, 11, 13, 15, 17, 19, 21 or 23. The reverse primer is also determined by routine procedures and preferably contains a 5' extension introducing a KpnI cloning site, followed by nucleotides corresponding to the reverse to the complement of the nucleotide sequence that is selected from the set consisting of sequences with no. 1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21 or 23.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">PCR produkt je proien gelom, razgraen s NdeI i KpnI i kloniran u odgovarajua mjesta vektora pAcHTL-A (Pharmingen, San Diego, CA). pAcHTL ekspresijski vektor sadri jaki polihedrinski promotor Autographa californica nuclear polyhedrosis virusa (AcMNPV) i 6XHis tag uzvodno od viestrukog klonirajueg mjesta. Protein kinaza mjesto za fosforiliranje i trombinsko mjesto za eksciziju rekombinantnog proteina koje prethodi viestrukom klonirajuem mjestu takoer je uoeno. Naravno, mogu se koristiti drugi bakulovirusni vektori umjesto pAcHTL-A, kao to su pAc373, pVL941 i pAcIMl. Drugi odgovarajui vektori za ekspresiju GPCR polipeptida takoer se mogu koristiti, pod uvjetom da vektorski konstrukt sadri odgovarajue smjetene signale za transkripciju, prevoenje i uvoenje, kao to su in-frame AUG i signalni peptid, po potrebi. Takvi vektori su opisani u Luckow et al., Virology 170:31-39, izmeu ostalog. </span>The PCR product was gel purified, digested with NdeI and KpnI, and cloned into the appropriate sites of the vector pAcHTL-A (Pharmingen, San Diego, CA). The pAcHTL expression vector contains the strong polyhedrin promoter of Autographa californica nuclear polyhedrosis virus (AcMNPV) and a 6XHis tag upstream of the multiple cloning site. A protein kinase phosphorylation site and a thrombin excision site for the recombinant protein preceding the multiple cloning site were also observed. Of course, other baculovirus vectors can be used instead of pAcHTL-A, such as pAc373, pVL941 and pAcIM1. Other suitable vectors for expression of GPCR polypeptides may also be used, provided the vector construct contains appropriately positioned signals for transcription, translation and introduction, such as in-frame AUG and signal peptide, as appropriate. Such vectors are described in Luckow et al., Virology 170:31-39, among others.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Virus je uzgojen i izdvojen koritenjem standardnih metoda bukulovirusne ekspresije, kao to su one opisane u: Summers et al, (A Manual of Methods for Baculovirus Vektors and Insect Cell Culture Procedures, Texas Agricultural Experimental Station Bulletin No. 1555 (1987)). </span>Virus was grown and isolated using standard baculovirus expression methods, such as those described in: Summers et al, (A Manual of Methods for Baculovirus Vectors and Insect Cell Culture Procedures, Texas Agricultural Experimental Station Bulletin No. 1555 (1987)).</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">U jednoj realizaciji, pAcHLT-A koji sadri DmGPCR gen uveden je u bakulovirus koritenjem BaculoGold transfekcijskog pribora (Pharmingen, San Diego, CA) koritenjem metoda koje je utvrdio proizvoa. Pojedini virusni izolati su analizirani na proizvodnju proteina radioobiljeavanjem inficirane stanice s 35S-metioninom 24 sata nakon infekcije. Inficirane stanice su sakupljene 48 sati nakon infekcije i obiljeeni proteini su vizualizirani s SDS-PAGE. Virusi koji pokazuju visoku razinu ekspresije mogu se izdvojiti i koristiti za uveanu ekspresiju. </span>In one embodiment, pAcHLT-A containing the DmGPCR gene was introduced into baculovirus using the BaculoGold transfection kit (Pharmingen, San Diego, CA) using methods established by the manufacturer. Individual virus isolates were analyzed for protein production by radiolabeling the infected cell with 35S-methionine 24 hours after infection. Infected cells were harvested 48 hours post-infection and labeled proteins were visualized by SDS-PAGE. Viruses showing high levels of expression can be isolated and used for scale-up expression.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Za ekspresiju DmGPCR polipeptida u Sf9 stanicama, polinukleotidna molekula koja ima nukleotidnu sekvenciju koja je odabrana iz skupa kojega sainjavaju sekvencije s br. 1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21 ili 23, moe se pojaati pomou PCR koritenjem primera i metoda koje su opisane za bakulovirusnu ekspresiju. DmGPCR cDNA je klonirana u vektor pAcHLT-A (Pharmingen) za ekspresiju u Sf9 insektnim stanicama. Umetak je kloniran u NdeI i KpnI mjestima, nakon eliminiranja internog NdeI mjesta (koritenjem istih primera koji su prije opisani za ekspresiju u bakulovirusu). DNA je proiena s Qiagen kromatografijom na koloni i izvrena je ekspresija u Sf9 stanicama. Preliminarni Western blot eksperimenti iz neproienih plakova su testirani na prisutnost rekombinantniog proteina oekivane veliine koji reagira s GPCR-specifinim antitijelom. Ovi rezultati su potvreni nakon daljnjeg proiavanja i ekspresijske optimizacije u HiG5 stanicama. </span>For the expression of DmGPCR polypeptide in Sf9 cells, a polynucleotide molecule having a nucleotide sequence selected from the group consisting of sequences with no. 1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21 or 23, can be amplified by PCR using the primers and methods described for baculovirus expression. DmGPCR cDNA was cloned into pAcHLT-A vector (Pharmingen) for expression in Sf9 insect cells. The insert was cloned into the NdeI and KpnI sites, after eliminating the internal NdeI site (using the same primers previously described for baculovirus expression). DNA was purified by Qiagen column chromatography and expression was performed in Sf9 cells. Preliminary Western blot experiments from unpurified plaques were tested for the presence of recombinant protein of the expected size that reacts with a GPCR-specific antibody. These results were confirmed after further purification and expression optimization in HiG5 cells.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Primjer 5: Interakcija zamka/dvohibridni sustav </span>Example 5: Trap interaction/two-hybrid system</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Da bi se izvrila analiza DmGPCR-interakcijskih proteina, koritena je screening metoda interakcije zamka/dvohibridna biblioteka. Ova analiza je prvi puta opisana u Fields, et al, Nature, 1989, 340, 245, to je ovdje ukljueno kao referenca u cijelosti. Protokol je publiciran u: Current Protocols in Molecular Biology, John Wiley &amp; Sons, NY, 1999 i Ausubel et al, Short Protocols in Molecular Biology, fourth edition, Greene i Wiley-interscience, NY, 1992, to je ovdje ukljueno kao referenca u cijelosti. Pribori su nabavljeni od tvrtke Clontech, Palo Alto, CA (Matchmaker Two-Hybrid System 3). </span>In order to perform the analysis of DmGPCR-interacting proteins, an interaction trap/two-hybrid library screening method was used. This analysis was first described in Fields, et al, Nature, 1989, 340, 245, which is incorporated herein by reference in its entirety. The protocol is published in: Current Protocols in Molecular Biology, John Wiley &amp; Sons, NY, 1999 and Ausubel et al, Short Protocols in Molecular Biology, fourth edition, Greene and Wiley-interscience, NY, 1992, which is incorporated herein by reference in in its entirety. Accessories were obtained from Clontech, Palo Alto, CA (Matchmaker Two-Hybrid System 3).</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Sjedinjavanje nukleotidne sekvencije koja kodira ili sve ili samo dio DmGPCR i kvaeva transkripcijskog faktora GAL4 DNA-vezujue domene (DNA-BD) izvreno je u odgovarajuem plazmidu (tj. pGBKT7) koritenjem standardnih tehnika subkloniranja. Slino, GAL4 aktivna domena (AD) fuzijske biblioteke nainjena je u drugom plazmidu (tj. pGADT7) iz cDNA potencijalnih GPCR-vezujuih proteina (za protokole stvaranja cDNA biblioteka, vidi Sambrook et al., Molecular cloning: laboratory manual, second edition, Cold Spring Harbor Press, Cold Spring Harbor, NY, 1989), to je ovdje ukljueno kao referenca u cijelosti. DNA-BD/GPCR fuzijski konstrukt provjeren je pomou sekvenciranja i ispitan je na autonomno aktiviranje reporterskog gena i staninu toksinost, to oboje sprjeava uspjenu dvohibridnu analizu. Sline kontrole su izvrene s AD/biblioteka fuzijskog konstrukta da se osigura ekspresija u stanicama domaina i nedostatak transkripcijske aktivnosti. Kvaeve stanice su transformirane (ca. 105 transformanata/mg DNA) s GPCR i bibliotekom fuzijskih plazmida sukladno standardnom postupku (Ausubel et al, Short protocols in molecular biology, fourth edition, Greene i Wiley-interscience, NY, 1992, to je ovdje ukljueno kao referenca u cijelosti). In vivo vezanje DNA-BD/GPCR s AD/biblioteka proteina rezultira transkripcijom specifinih kvaevih plazmid reporterskih gena (tj. lacZ, HIS3, ADE2, LEU2). Kvaeve stanice su nanesene na plou na medij koji je deficitaran hranom da se provjeri na ekspresiju reporterskih gena. Kolonije su dvojno analizirane na aktivnost -galaktosidaza nakon rasta u XgaI (5-brom-4-klor-3-indolil--D-galaktozid) dopunskom mediju (filterska analiza na aktivnost -galaktosidaze opisana je u Breeden et al, Cold Spring Harb. Symp. Quant. Biol., 1985, 50, 643, to je ovdje ukljueno kao referenca u cijelosti). Pozitivni AD-biblioteka plazmidi su izvueni iz transformanata i ponovo su uneseni u izvorni kvaev soj kao i druge sojeve koji sadre nesrodne DNA-BD fuzijske proteine da se potvrde specifine DmGPCR/biblioteka proteina interakcije. Ubaena DNA je sekvencirana da se provjeri prisutnost otvorenog okvira za itanje fuzioniranog na GAL4 AD i da se odredi identinost DmGPCR-vezujueg proteina. </span>Fusion of the nucleotide sequence encoding either all or part of the DmGPCR and the yeast transcription factor GAL4 DNA-binding domain (DNA-BD) was performed in the appropriate plasmid (ie, pGBKT7) using standard subcloning techniques. Similarly, a GAL4 active domain (AD) fusion library was constructed in another plasmid (i.e., pGADT7) from cDNAs of potential GPCR-binding proteins (for cDNA library generation protocols, see Sambrook et al., Molecular cloning: laboratory manual, second edition, Cold Spring Harbor Press, Cold Spring Harbor, NY, 1989), which is incorporated herein by reference in its entirety. The DNA-BD/GPCR fusion construct was verified by sequencing and tested for autonomous activation of the reporter gene and cellular toxicity, both of which prevent successful two-hybrid analysis. Similar controls were performed with the AD/fusion construct library to ensure expression in host cells and lack of transcriptional activity. Yeast cells were transformed (ca. 105 transformants/mg DNA) with a GPCR and fusion plasmid library according to a standard procedure (Ausubel et al, Short protocols in molecular biology, fourth edition, Greene and Wiley-interscience, NY, 1992, which is incorporated herein as a reference in its entirety). In vivo binding of DNA-BD/GPCR to AD/protein library results in transcription of specific yeast plasmid reporter genes (ie lacZ, HIS3, ADE2, LEU2). Yeast cells were plated on nutrient-deficient medium to check for reporter gene expression. Colonies were assayed in duplicate for -galactosidase activity after growth in XgaI (5-bromo-4-chloro-3-indolyl--D-galactoside) supplemented medium (filter assay for -galactosidase activity is described in Breeden et al, Cold Spring Harb. Symp. Quant. Biol., 1985, 50, 643, which is hereby incorporated by reference in its entirety). Positive AD-library plasmids were recovered from transformants and reintroduced into the original yeast strain as well as other strains containing unrelated DNA-BD fusion proteins to confirm specific DmGPCR/library protein interactions. Inserted DNA was sequenced to verify the presence of the open reading frame fused to the GAL4 AD and to determine the identity of the DmGPCR-binding protein.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Primjer 6: Pomak mobilnosti DNA-vezujue analize koritenjem gel elektroforeze </span>Example 6: Mobility shift DNA-binding analysis using gel electrophoresis</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Gel elektroforetska analiza pomaka mobilnosti moe brzo detektirati specifine protein-DNA interakcije. Protokoli su lako dostupni u prirunicima kao to su: Sambrook et al, Molecular cloning: laboratory manual, second edition, Cold Spring Harbor Press, Cold Spring Harbor, NY, 1989 i Ausubel et al, Short Protocols in Molecular Biology, fourth edition, Greene i Wiley-interscience, NY, 1992, a svaki od njih je ovdje ukljuen kao referenca u cijelosti. </span>Gel electrophoretic mobility shift analysis can rapidly detect specific protein-DNA interactions. Protocols are readily available in manuals such as: Sambrook et al, Molecular cloning: laboratory manual, second edition, Cold Spring Harbor Press, Cold Spring Harbor, NY, 1989 and Ausubel et al, Short Protocols in Molecular Biology, fourth edition, Greene and Wiley-interscience, NY, 1992, each of which is incorporated herein by reference in its entirety.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Sonda DNA (&lt;300 bp) dobivena je iz sintetskih oligonukleotida, retrikcijskih fragmenata endonukleaze ili PCR fragmenata te krajnje obiljeenih s 32P. Alikvot proienog DmGPCR (ca. 15 g) ili sirovog DmGPCR ekstrakta (ca. 15 ng) inkubiran je na konstantnoj temperaturi (u rasponu 22-37C) tijekom bar 30 minuta u 10-15 l pufera (tj. TAE ili TBE, pH 8,0-8,5) koji sadri radioobiljeenu sondu DNA, nespecifini nosa DNA (ca. 1 g), BSA (300 g/ml) i 10% (v/v) glicerol. Reakcijska smjesa je zatim nanesena na poliakrilamidni gel i primijenjena je struja 30-35 mA sve dok nije postignuto dobro odvajanje slobodne sonde DNA od kompleksa protein-DNA. Gel je zatim osuen i vrpce koje odgovaraju slobodnoj DNA i protein-DNA kompleksima su detektirane autoradiografski. </span>The DNA probe (&lt;300 bp) was obtained from synthetic oligonucleotides, endonuclease restriction fragments or PCR fragments and end-labeled with 32P. An aliquot of purified DmGPCR (ca. 15 g) or crude DmGPCR extract (ca. 15 ng) was incubated at a constant temperature (in the range 22-37C) for at least 30 minutes in 10-15 l buffer (i.e. TAE or TBE, pH 8 .0-8.5) containing radiolabeled DNA probe, non-specific DNA carrier (ca. 1 g), BSA (300 g/ml) and 10% (v/v) glycerol. The reaction mixture was then applied to a polyacrylamide gel and a current of 30-35 mA was applied until a good separation of the free DNA probe from the protein-DNA complex was achieved. The gel was then dried and bands corresponding to free DNA and protein-DNA complexes were detected autoradiographically.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Primjer 7: Antitijela za DmGPCR </span>Example 7: Antibodies for DmGPCR</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Standard tehnike koriste se za generiranje polikoonskih ili monoklonskih antitijela za DmGPCR i da se generiraju korisni antigen-vezujui fragmenti ili inaice, ukljuujui humanizirane inaice. Ti protokoli mogu se nai, primjerice, u: Sambrook et al. (1989), supra i Harlow et al. (Eds.), Antitijela: Laboratory Manual, Cold Spring Harbor Laboratory, Cold Spring Harbor, NY, 1988. U jednoj realizaciji, rekombinantni DmGPCR polipeptidi (ili stanice ili stanine membrane koje sadre takve polipeptide) koriste se kao antigen za generiranje antitijela. U drugoj realizaciji, jedan ili vie peptida koji imaju aminokiselinske sekvencije koje odgovaraju imunogenom dijelu DmGPCR-a (npr. 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20 ili vie aminokiselina) koristi se kao antigen. Peptidi koji odgovaraju izvanstaninim dijelovima DmGPCR-a, posebice hidrofilni izvanstanini dijelovi, obuhvaeni su ovim izumom. Antigen se moe pomijeati s adjuvantom ili se moe vezati za hapten da se povea proizvodnja antitijela. </span>Standard techniques are used to generate polyclonal or monoclonal antibodies to DmGPCR and to generate useful antigen-binding fragments or variants, including humanized versions. These protocols can be found, for example, in: Sambrook et al. (1989), supra and Harlow et al. (Eds.), Antibodies: A Laboratory Manual, Cold Spring Harbor Laboratory, Cold Spring Harbor, NY, 1988. In one embodiment, recombinant DmGPCR polypeptides (or cells or cell membranes containing such polypeptides) are used as an antigen to generate antibodies. In another embodiment, one or more peptides having amino acid sequences corresponding to the immunogenic portion of a DmGPCR (e.g., 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20 or more amino acids) is used as an antigen. Peptides corresponding to extracellular portions of DmGPCR, particularly hydrophilic extracellular portions, are encompassed by the present invention. The antigen can be mixed with an adjuvant or can be attached to a hapten to increase antibody production.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Poliklonska ili monoklonska antitijela </span>Polyclonal or monoclonal antibodies</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Kao jedan primjerni protokol, rekombinantni DmGPCR ili njegov sintetski fragment koristi se za imuniziranje mieva za generiranje monoklonskih antitijela (ili vei sisavci, kao to je kuni, za poliklonska antitijela). Da se povea antigenost, peptidi su konjugirani za Keyhole Lympet Hemocyanin (Pierce), sukladno preporukama proizvoaa. Za poetnu injekciju, antigen je emulgiran s Freundovim cjelovitim adjuvantom, te potkono injiciran. U intervalima od dva do tri tjedna, dodatni alikvoti DmGPCR antigena s emulgirani s Freundovim nepotpounim adjuvantom i injicirani potkono. Prije konane booster injekcije, uzet je uzorak seruma iz imuniziranog mia i analiziran je western blot analizom da se potvrdi prisutnost antitijela koja su u imunoreakciji s DmGPCR-om. Serum imuniziranih ivotinja moe se koristiti kao poliklonski antiserum ili se koristi za izdvajanje poliklonskih antitijela koja prepoznaju DmGPCR. Alternativno, mievi se mogu rtvovati i njihova slezena uzeti za generiranje monoklonskih antitijela. </span>As one exemplary protocol, recombinant DmGPCR or a synthetic fragment thereof is used to immunize mice to generate monoclonal antibodies (or larger mammals, such as rabbit, for polyclonal antibodies). To increase antigenicity, peptides were conjugated to Keyhole Lympet Hemocyanin (Pierce), according to the manufacturer's recommendations. For the initial injection, the antigen was emulsified with complete Freund's adjuvant and injected subcutaneously. At two to three week intervals, additional aliquots of DmGPCR antigen were emulsified with Freund's incomplete adjuvant and injected subcutaneously. Before the final booster injection, a serum sample was taken from the immunized mouse and analyzed by western blot analysis to confirm the presence of antibodies immunoreacting with DmGPCR. Serum from immunized animals can be used as polyclonal antiserum or used to isolate polyclonal antibodies that recognize DmGPCR. Alternatively, mice can be sacrificed and their spleens harvested to generate monoclonal antibodies.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Da se dobiju monoklonska antitijela, slezena je stavljena u 10 ml serum-slobodnog RPMI1640 i nainjena je jednostanana suspenzija usitnjavanjem slezene u serum-slobodnom RPMI 1640, emu je dodano 2 mM L-glutamina, 1 mM natrijeva piruvata, 100 jedinica/ml penicilina i 100 g/ml streptomicina (RPMI) (Gibco, Canada). Stanine suspenzije su filtrirane i isprane centrifugiranjem te resuspendirane u serum-slobodnom RPMI. Timociti su uzeti iz tri nativna Balb/c mia i prireeni su na slian nain te koriteni kao hranjivi sloj. NS-1 stanice mijeloma, drane su u log fazi u RPMI s 10% seruma volovskog zametka (FBS) (Hyclone Laboratories, Inc., Logan, Utah) tijekom tri dana do fuzije te centrifugirane i isprane. </span>To obtain monoclonal antibodies, the spleen was placed in 10 ml of serum-free RPMI 1640 and a single cell suspension was made by mincing the spleen in serum-free RPMI 1640 supplemented with 2 mM L-glutamine, 1 mM sodium pyruvate, 100 units/ml penicillin and 100 g/ml streptomycin (RPMI) (Gibco, Canada). Cell suspensions were filtered and washed by centrifugation and resuspended in serum-free RPMI. Thymocytes were taken from three native Balb/c mice and prepared in a similar manner and used as a nutrient layer. NS-1 myeloma cells were maintained in log phase in RPMI with 10% fetal bovine serum (FBS) (Hyclone Laboratories, Inc., Logan, Utah) for three days until confluent and centrifuged and washed.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Da se dobiju fuzije hibridoma, stanice slezene imuniziranih mieva su kombinirane s NS-1 stanicama i centrifugirane pa je supernatant usisan.Stanini talog je izbaen suenjem epruvete i 2 ml na 37C PEG 1500 (50% u 75 mM HEPES, pH 8,0) (Boehringer-Mannheim) je mijeano s talogom, nakon ega je dodan serum-slobodni RPMI. Nakon toga, stanice su centrifugirane, resuspendirane u RPMI koji sadri 15% FBS, 100 M sodium hipoksantina, 0,4 M aminopterina, 16 M timidina (HAT) (Gibco), 25 jedinica/ml IL-6 (Boehringer-Mannheim) i 1,5106 timociti/ml te nanesene na plou 10 Corning kulture ploa s 96 jaica (Corning, Corning New York). </span>To obtain hybridoma fusions, spleen cells from immunized mice were combined with NS-1 cells and centrifuged, and the supernatant was aspirated. The cell pellet was removed by tapering the tube and 2 ml at 37C PEG 1500 (50% in 75 mM HEPES, pH 8.0) (Boehringer-Mannheim) was mixed with the precipitate, after which serum-free RPMI was added. After that, the cells were centrifuged, resuspended in RPMI containing 15% FBS, 100 M sodium hypoxanthine, 0.4 M aminopterin, 16 M thymidine (HAT) (Gibco), 25 units/ml IL-6 (Boehringer-Mannheim) and 1.510 6 thymocytes/ml and plated on plate 10 Corning 96-well culture plates (Corning, Corning New York).</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Na dane 2, 4 i 6 nakon sjedinjavanja, 100 l medija je uklonjeno iz jaica na fuzijskim ploama i zamijenjeno svjeim medijem. Na dan 8, ploe su provjerene ELISA analizom, ispitane na prisutnost mijeg IgG koji se vee za DmGPCR. Odabrane fuzijske jaice su dalje klonirane razrjeenjem sve dok nije dobivena monoklonska kultura koja daje anti-DmGPCR antitijela. </span>On days 2, 4, and 6 after fusion, 100 l of medium was removed from the wells of the fusion plates and replaced with fresh medium. On day 8, the plates were checked by ELISA, tested for the presence of mouse IgG binding to DmGPCR. Selected fusion wells were further cloned by dilution until a monoclonal culture yielding anti-DmGPCR antibodies was obtained.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Humaniziranje anti-DmGPCR monoklonskih antitijela </span>Humanization of anti-DmGPCR monoclonal antibodies</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Ekspresijska slika DmGPCR-a kao je ovdje navedena i potvreno biljeenje GPCR-ova kao ciljeva za terapijsku intervenciju ukazuju na terapijske indikacije za DmGPCR inhibitore (antagoniste). DmGPCR-neutralizirajua antitijela koja sadre jednu klasu terapeutika korisni su DmGPCR antagonisti. U nastavku slijede protokoli za humaniziranje monoklonskih antitijela ovog izuma. </span>The expression pattern of DmGPCRs as reported here and the confirmed recording of GPCRs as targets for therapeutic intervention indicate therapeutic indications for DmGPCR inhibitors (antagonists). DmGPCR-neutralizing antibodies comprising one class of therapeutics are useful DmGPCR antagonists. The following are protocols for humanizing the monoclonal antibodies of the present invention.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Naela humaniziranja su opisana u literaturi i humaniziranje je olakano modularnim rasporedom proteina antitijela. Da se minimizira mogunost vezanja komplementa, moe se koristiti humanizirano antitijelo IgG4 izotipa. </span>Principles of humanization are described in the literature and humanization is facilitated by the modular arrangement of antibody proteins. To minimize the possibility of binding of complement, a humanized IgG4 isotype antibody can be used.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Primjerice, razina humaniziranja postie se generiranjem kimernih antitijela koja sadre promjenjive domene proteina nehumanih antitijela od interesa s nepromjenjivim domenama molekule humanog antitijela (vidi npr. Morrison et al, Adv. Immunol., 1989, 44, 65-92). Promjenjive domene neutralizirajuih anti-DmGPCR antitijela DmGPCR-a su klonirane iz genomske DNA B-stanica hibridoma ili iz cDNA koja je generirana iz mRNA izdvojene iz hibridoma od interesa. V podruje fragmenata gena povezano je s eksonima koji kodiraju nepromjenjivu domenu humanog antitijela i rezultantni konstrukt ima ekspresiju u odgovarajuoj stanici domaina sisavca (npr. mijelom ili CHO stanice). </span>For example, a level of humanization is achieved by generating chimeric antibodies that contain the variable domains of the nonhuman antibody protein of interest with the constant domains of the human antibody molecule (see, e.g., Morrison et al, Adv. Immunol., 1989, 44, 65-92). Variable domains of neutralizing anti-DmGPCR antibodies DmGPCRs are cloned from genomic DNA of hybridoma B-cells or from cDNA generated from mRNA isolated from the hybridoma of interest. The V region of the gene fragments is ligated to the exons encoding the constant domain of the human antibody and the resulting construct is expressed in the appropriate mammalian host cell (eg, myeloma or CHO cells).</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Da se postigne jo jaa razina humanizacije, klonirani su samo oni dijelovi promjenjive domene fragmenata gena koji kodiraju antigen-vezujua podruja koja odreuju komplementarnost (CDR) nehumanog monoklonskog antitijela, u sekvencije humanog antitijela (vidi npr. Jones et al, Nature, 1986, 321, 522-525; Riechmann et al, Nature, 1988, 332, 323-327; Verhoeyen et al, Science, 1988, 239, 1534-36; i Tempest et al, Bio/Technology, 1991, 9, 266-71). Ako je potrebno, takoer je modificiran -list kostura humanog antitijela koje okruuje CDR3 podruja da jo vie odraava trodimenzionalnu strukturu antigen-vezujue domene izvornog monoklonskog antitijela (vidi Kettleborough et al, Protein Engin., 1991, 4,773-783; i Foote et al, J. Mol. Biol., 1992, 224,487-499). </span>To achieve an even greater level of humanization, only those portions of the variable domain of the gene fragments encoding the antigen-binding complementarity-determining regions ("CDR") of the nonhuman monoclonal antibody were cloned into the human antibody sequences (see, e.g., Jones et al, Nature, 1986 , 321, 522-525; Riechmann et al, Nature, 1988, 332, 323-327; Verhoeyen et al, Science, 1988, 239, 1534-36; and Tempest et al, Bio/Technology, 1991, 9, 266- 71). If necessary, the -sheet of the human antibody backbone surrounding the CDR3 region is also modified to more closely reflect the three-dimensional structure of the antigen-binding domain of the original monoclonal antibody (see Kettleborough et al, Protein Engin., 1991, 4,773-783; and Foote et al , J. Mol. Biol., 1992, 224, 487-499).</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">U alternativnom pristupu, povrina nehumanog monoklonskog antitijela od interesa je humanizirana promjenom odabranih povrinskih rezidua nehumanog antitijela, npr. pomou mjesno-usmjerene mutageneze, uz zadravanje svih unutranjih i kontaktnih rezidua nehumanog antitijela. Vidi Padlan, Molecular Immunol, 1991, 25(4/5), 489-98. </span>In an alternative approach, the surface of the nonhuman monoclonal antibody of interest is humanized by changing selected surface residues of the nonhuman antibody, eg, using site-directed mutagenesis, while retaining all internal and contact residues of the nonhuman antibody. See Padlan, Molecular Immunol, 1991, 25(4/5), 489-98.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Prije navedeni pristupi rabe se koritenjem DmGPCR-neutralizirajueg anti-DmGPCR monoklonskog antitijela i hibridoma koji ih proizvode da se dobiju humanizirana DmGPCR-neutralizirajua antitijela koja su korisna kao terapeutici da se tretira ili palijativno djeluje na stanja pri emu je DmGPCR ekspresija ili ligand-upravljano signaliziranje DmGPCR tetno. </span>The foregoing approaches employ the use of a DmGPCR-neutralizing anti-DmGPCR monoclonal antibody and the hybridomas that produce them to obtain humanized DmGPCR-neutralizing antibodies that are useful as therapeutics to treat or palliate conditions in which DmGPCR expression or ligand-driven signaling is impaired. DmGPCR harmful.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Primjer 8: Analize za identificiranje modulatora aktivnosti DmGPCR </span>Example 8: Assays to identify modulators of DmGPCR activity</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">U nastavku je navedeno nekoliko neograniavajuih analiza za identificiranje modulatora (agonisti i antagonisti) aktivnosti DmGPCR-a. Meu modulatorima koji se mogu identificirati pomou analiza su prirodni ligandni spojevi receptora; sintetski analozi i derivati prirodnih liganada; antitijela, fragmenti antitijela, i/ili antitijelu slini spojevi koji su izvedeni iz prirodnih antitijela ili iz antitijelu slinih kombinatornih biblioteka; i/ili sintetski spojevi koji su identificirani high-throughput screening bibliotekama i slino. Svi modulatori koji veu DmGPCR korisni su za identificiranje DmGPCR u uzorcima tkiva (npr. za dijagnostike potrebe, patoloke svrhe i slino). Modulatori agonista i antagonista korisni su za nadreguliranje i podreguliranje aktivnosti DmGPCR, odnosno, za tretiranje bolesnih stanja koja su karakterizirana nenormalnim razinama aktivnosti DmGPCR-a. Analize se mogu koritenjem samih potencijalnih modulatora i/ili se mogu izvriti koritenjem poznatih agonista u kombinaciji s kandidatskim antagonistom (ili obrnuto). </span>Several non-limiting assays for identifying modulators (agonists and antagonists) of DmGPCR activity are listed below. Among the modulators that can be identified using the assays are naturally occurring ligand compounds of the receptor; synthetic analogues and derivatives of natural ligands; antibodies, antibody fragments, and/or antibody-like compounds derived from natural antibodies or from antibody-like combinatorial libraries; and/or synthetic compounds identified by high-throughput screening libraries and the like. All DmGPCR-binding modulators are useful for identifying DmGPCRs in tissue samples (eg, for diagnostic purposes, pathological purposes, and the like). Agonist and antagonist modulators are useful for upregulating and downregulating DmGPCR activity, respectively, for treating disease states characterized by abnormal levels of DmGPCR activity. Analyzes can be performed using potential modulators alone and/or can be performed using known agonists in combination with a candidate antagonist (or vice versa).</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Analize cAMP </span>Analyzes of cAMP</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">U jednom tipu analize, razine ciklikog adenozin monofosfata (cAMP) mjere se u DmGPCR-transficiranim stanicama koje su izloene kandidatskim modulatorskim spojevima. Protokoli za cAMP analize opisani su u literaturi (vidi npr. Sutherland et al, Circulation, 1968, 37, 279; Frandsen et al, Life Sciences, 1976,18, 529-541; Dooley et al, J. Pharm. i Exper. Ther., 1997, 283(2), 735-41; i George et al, J. Biomolecular Screening, 1997, 2(4), 235-40). Primjerni protokol za takvu analizu, Adenylyl Cyclase Activation FlashPlate analize iz NEN Life Science Products je nie naveden. </span>In one type of assay, levels of cyclic adenosine monophosphate (cAMP) are measured in DmGPCR-transfected cells exposed to candidate modulatory compounds. Protocols for cAMP assays are described in the literature (see, e.g., Sutherland et al, Circulation, 1968, 37, 279; Frandsen et al, Life Sciences, 1976, 18, 529-541; Dooley et al, J. Pharm. and Exper. Ther., 1997, 283(2), 735-41; and George et al, J. Biomolecular Screening, 1997, 2(4), 235-40). An exemplary protocol for such an assay, the Adenylyl Cyclase Activation FlashPlate assay from NEN Life Science Products is listed below.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Ukratko, DmGPCR kodirajua sekvencija (npr. cDNA ili bezintronska genomska DNA) je subklonirana u komercijalni ekspresijski vektor, kao to je pzeoSV2 (Invitrogen) i prijelazno je transficirana u stanice jajnika kineskog zamorca (CHO) koritenjem poznatih metoda, kao to je protokol transfekcije kojega daje such Boehringer-Mannheim kada se rabi FuGENE 6 reagens za transfekciju. Transficirane CHO stanice zasaene su u mikroplou s 96 jaica iz FlashPlate pribora za analizu, koje su prevuene s vrstim scintilantom na kojega se vee antiserum za cAMP. Kao kontrola, neke jaice su imale divlji tip (netransficirane) CHO stanica. Druge jaice na ploi primile su razliite koliine cAMP standardne otopine za dobivanje standardne krivulje. </span>Briefly, the DmGPCR coding sequence (eg, cDNA or intronless genomic DNA) was subcloned into a commercial expression vector, such as pzeoSV2 (Invitrogen) and transiently transfected into Chinese guinea pig ovary (CHO) cells using known methods, such as the transfection protocol of gives such Boehringer-Mannheim when FuGENE 6 transfection reagent is used. Transfected CHO cells were seeded in a 96-well microplate from the FlashPlate assay kit, which was coated with a solid scintillant to which cAMP antiserum binds. As a control, some wells had wild-type (untransfected) CHO cells. Other wells of the plate received different amounts of cAMP standard solution to obtain a standard curve.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Jedan ili vie testnih spojeva (tj. kandidatskih modulatora) dodano je stanicama u svaku jaicu, s vodom i/ili spoj-slobodnim medijem/razrjeivaem koji slui kao kontrola ili kontrole. Nakon tretiranja, cAMP je ostavljen da se akumulira u stanicama tono 15 minuta na sobnoj temperaturi. Analiza je zavrena dodatkom liznog pufera koji sadri [125I]-obiljeeni cAMP i ploe su izbrojane koritenjem Packard Topcount microplate scintilacijskog brojaa s 96 jaica. Neobiljeeni cAMP iz lizirane stanice (ili standarda) i nepromjenjive koliine [125I]-cAMP kompetiraju za antitijelo koje je vezano za plou. Nainjena je standardna krivulja i cAMP vrijednosti za nepoznate uzorke su dobivene interpoliranjem. Promjene u razini unutarstaninog cAMP kao odgovor na izloenost testnom spoju je indikacija modulirajue aktivnosti DmGPCR-a. Modulatori koji djeluju kao agonisti receptora koji se veu za Gs podtip G proteina e stimulirati proizvodnju cAMP, to dovodi do mjerljivog 3-10-strukog poveanja razine cAMP. Agonisti receptora koji se veu za Gi/0 podtip G proteina e inhibirati forskolin-stimuliranu proizvodnju cAMP, to dovodi do mjerljivog smanjenja razine cAMP od 50-100%. Modulatori koji djeluju kao inverzni agonisti e obrnuti ove uinke na receptorima koji su ili konstitutivno aktivni ili su aktivirani pomou poznatih agonista. </span>One or more test compounds (ie, candidate modulators) were added to the cells in each well, with water and/or compound-free medium/diluent serving as a control or controls. After treatment, cAMP was allowed to accumulate in the cells for exactly 15 minutes at room temperature. The assay was terminated by the addition of lysis buffer containing [125I]-labeled cAMP and plates were counted using a Packard Topcount 96-well microplate scintillation counter. Unlabeled cAMP from the lysed cell (or standard) and unaltered amounts of [125I]-cAMP compete for the antibody bound to the plate. A standard curve was constructed and cAMP values for unknown samples were obtained by interpolation. Changes in the level of intracellular cAMP in response to exposure to the test compound is an indication of the modulating activity of the DmGPCR. Modulators that act as receptor agonists that bind to the Gs subtype of G proteins will stimulate cAMP production, leading to a measurable 3-10-fold increase in cAMP levels. Receptor agonists that bind to the Gi/0 subtype of G protein will inhibit forskolin-stimulated cAMP production, leading to a measurable decrease in cAMP levels of 50-100%. Modulators that act as inverse agonists will reverse these effects at receptors that are either constitutively active or are activated by known agonists.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Analize ekvorina (aequorin) </span>Analyzes of aequorin</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">U sljedeoj analizi, stanice (npr. CHO stanice) prijelazno su kotransficirane s DmGPCR ekspresijskim konstruktom i konstruktom koji kodira fotoprotein apoakvorin. U prisutnosti kofaktora coelenterazina, apoakvorin e emitirati mjerljivu luminescenciju koja je razmjerna koliini unutarstaninog (citoplazmatskog) slobodnog kalcija (vidi openito, Cobbold, et al, Aequorin measurements of cytoplasmic free calcium, u: Cellular Calcium: Practical Approach, McCormack J.G. i Cobbold P.H., eds., Oxford: IRL Press, 1991; Stables et al., Anal. Biochem., 1997, 252,115-26; i Haugland, Handbook of Fluorescent Probes i Research Chemicals, sixth edition, Eugene, OR, Molecular Probes, 1996). </span>In the following analysis, cells (eg, CHO cells) were transiently cotransfected with a DmGPCR expression construct and a construct encoding the photoprotein apoaquorin. In the presence of the cofactor coelenterazine, apoaquorin will emit measurable luminescence that is proportional to the amount of intracellular (cytoplasmic) free calcium (see generally, Cobbold, et al, "Aequorin measurements of cytoplasmic free calcium," in: Cellular Calcium: A Practical Approach, McCormack J.G. and Cobbold P.H., eds., Oxford: IRL Press, 1991; Stables et al., Anal. Biochem., 1997, 252,115-26; and Haugland, Handbook of Fluorescent Probes and Research Chemicals, sixth edition, Eugene, OR, Molecular Probes, 1996 ).</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">U jednoj primjernoj analizi, DmGPCR je subkloniran u komercijalni ekspresijski vektor pzeoSV2 (Invitrogen) i prijelazno kotransficiran zajedno s konstruktom koji kodira fotoprotein apoakvorin (Molecular Probes, Eugene, OR) u CHO stanicama koritenjem transfekcijskog reagensa FuGENE 6 (Boehringer-Mannheim) i protokola za transfekciju koji je opisan uz produkt. </span>In one exemplary assay, the DmGPCR was subcloned into the commercial expression vector pzeoSV2 (Invitrogen) and transiently cotransfected together with a construct encoding the photoprotein apoaquorin (Molecular Probes, Eugene, OR) in CHO cells using the FuGENE 6 transfection reagent (Boehringer-Mannheim) and the protocol for transfection described with the product.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Stanice su uzgajane tijekom 24 sata na 37C u MEM (Gibco/BRL, Gaithersburg, MD) kojemu je dodan 10% serum volovskog zametka, 2 mM glutamin, 10 U/ml penicilin i 10 g/ml streptomicin, nakon ega je medij zamijenjen sa serum-slobodnim MEM koji sadri 5 M koelenterazina (Molecular Probes, Eugene, OR). Uzgajanje je zatim nastavljeno daljnjih dva sata na 37C. Nakon toga, stanice su odvojene od ploe pomou VERSEN (Gibco/BRL), isprane i resuspendirane sa 200000 stanica/ml u serum-slobodnom MEM. </span>Cells were grown for 24 hours at 37C in MEM (Gibco/BRL, Gaithersburg, MD) supplemented with 10% fetal bovine serum, 2 mM glutamine, 10 U/ml penicillin, and 10 g/ml streptomycin, after which the medium was replaced with with serum-free MEM containing 5 M coelenterazine (Molecular Probes, Eugene, OR). Cultivation was then continued for a further two hours at 37C. Subsequently, cells were detached from the plate using VERSEN (Gibco/BRL), washed and resuspended at 200,000 cells/ml in serum-free MEM.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Razrjeenja kandidatskih DmGPCR modulatorskih spojeva prireena su u serum-slobodnom MEM i raspodijeljena su u jaice neprozirne ploe s 96 jaica sa 50 l/jaica. Ploe su zatim stavljene na MLX luminometar mikrotitarske ploe (Dynex Technologies, Inc., Chantilly, VA). Instrument je programiran da dozira 50 l stanine suspenzije u svaku jaicu, jaicu po jaicu i da neposredno oita luminescenciju 15 sekundi. Krivulje doza-odgovor za kandidatske modulatore nainjene su koritenjem povrine pod krivuljom za svaki peak svjetlosnog signala. Podaci su analizirani s Slide Write, koritenjem jednadbe za ligand na jednom mjestu i dobivene su vrijednosti EC50. Promjene luminescencije koje su izazvane spojevima smatraju se indikativnima za modulatorsku aktivnost. Modulatori koji djeluju kao agonisti na receptorima koji se veu zao Gq podtip G proteina daju poveanje luminescencije do 100 puta. Modulatori koji djeluju kao inverzni agonisti e obrnuti ovaj efekt bilo da su konstitutivni aktivni ili su aktivirani poznatim agonistima. </span>Dilutions of candidate DmGPCR modulator compounds were prepared in serum-free MEM and dispensed into wells of an opaque 96-well plate at 50 l/well. Plates were then loaded onto an MLX luminometer microtiter plate (Dynex Technologies, Inc., Chantilly, VA). The instrument is programmed to dispense 50 l of cell suspension into each well, well by well, and to read the luminescence directly for 15 seconds. Dose-response curves for candidate modulators were constructed using the area under the curve for each light signal peak. Data were analyzed with Slide Write, using the single-site ligand equation, and EC50 values were obtained. Luminescence changes induced by compounds are considered indicative of modulatory activity. Modulators that act as agonists on receptors that bind to the evil Gq subtype of G protein give an increase in luminescence up to 100 times. Modulators that act as inverse agonists will reverse this effect whether they are constitutively active or are activated by known agonists.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Analiza luciferaza reporter gena </span>Analysis of luciferase reporter genes</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Fotoprotein luciferaza definira drugo korisno pomagalo za analiziranje na modulatorsku aktivnost DmGPCR-a. Stanice (npr. CHO stanice ili COS7 stanice) su prijelazno kotransficirane s DmGPCR ekspresijskim konstruktom (npr. DmGPCR u pzeoSV2) i reporterskim konstruktom koji sadri gen za ludiferaza protein nizvodno od vezujueg mjesta transkripcijskog faktora, kao to je element cAMP-odgovora (CRE), AP-1 ili NF-kapa B. Agonist koji se vee za receptore koji su vezani za Gs podtip G proteina dovodi do poveanja cAMP, ime se aktivira CRE transkripcijski faktor i rezultat je ekspresija luciferaza gena. Agonistiko vezanje za Gq podtip G proteina dovodi do proizvodnje diacilglicerola koji aktivira protein kinazu C, koja aktivira AP-1 ili NF-kapa B transkripcijske faktore, to opet rezultira ekspresijom luciferaza gena. Ekspresijske razine luciferaze odraavaju aktivacijski status signalnih dogaaja. Vidi openito, George et al, J. Biomolecular Screening, 1997, 2(4), 235-240; i Stratowa et al, Curr. Opin. Biotechnol., 1995, 6, 574-581. Aktivnost luciferaze moe se kvantitativno mjeriti koritenjem, npr. reagensa za luciferaza analizu koji se mogu komercijalno nabaviti od tvrtke Promega (Madison, WI). </span>Photoprotein luciferase defines another useful tool for assaying for DmGPCR modulatory activity. Cells (e.g., CHO cells or COS7 cells) are transiently cotransfected with a DmGPCR expression construct (e.g., DmGPCR in pzeoSV2) and a reporter construct containing a gene for a ludiferase protein downstream of a transcription factor binding site, such as a cAMP-response element (CRE). . Agonist binding to the Gq subtype of G protein leads to the production of diacylglycerol which activates protein kinase C, which activates AP-1 or NF-kappa B transcription factors, which in turn results in the expression of the luciferase gene. Luciferase expression levels reflect the activation status of signaling events. See generally, George et al, J. Biomolecular Screening, 1997, 2(4), 235-240; and Stratow et al., Curr. Opin. Biotechnol., 1995, 6, 574-581. Luciferase activity can be quantitatively measured using, for example, luciferase assay reagents commercially available from Promega (Madison, WI).</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">U jednoj primjernoj analizi, CHO stanice su nanesene na plou s 24 jaice s gustoom 100000 stanica/jaica jedan dan prije transfekcije i uzgajanje su na 37C u MEM (Gibco/BRL) kojemu je dodan 10% serum volovskog zametka, 2 mM glutamin, 10 U/ml penicilin i 10 g/ml streptomicin. Stanice su prijelazno kotransficirane s DmGPCR ekspresijskim konstruktom i reporterskim konstruktom koji sadri luciferaza gen. Reporterski plazmidi CRE-luciferaza, AP-1-luciferaza i NF-kapa B-luciferaza mogu se nabaviti od tvrtke Stratagene (LaJolla, CA). Transfekcije su izvrene koritenjem FuGENE 6 reagensa za transfekciju (Boehringer-Mannheim) sukladno naputku proizvoaa. Stanice koje su transficirane samo s reporterskim genom koritene su kao kontrola. Dvadeset i etiri sata nakon transfekcije, stanice su isprane jednom s PBS koji je prethodno zagrijan na 37C. Serum-slobodni MEM zatim je dodan stanicama bilo sam (kontrola) ili s jednim ili vie kandidatskih modulatora te su stanice inkubirane na 37C tijekom pet sati. Nakon toga, stanice su isprane jednom s ledenohladnim PBS i lizirane dodatkom 100 l liznog pufera po jaici iz pribora za analizu luciferaze to proizvodi tvrtka Nakon inkubiranja 15 minuta na sobnoj temperaturi, 15 l lizata pomijeano je s 50 l otopine supstrata (Promega) na neprozirnoj bijeloj ploi s 96 jaica pa je luminescencija trenutano oitana na Wallace model 1450 MicroBeta scintilacijskom i luminescencijskom brojau (Wallace Instruments, Gaithersburg, MD). </span>In an exemplary assay, CHO cells were plated in a 24-well plate at a density of 100,000 cells/well one day before transfection and grown at 37C in MEM (Gibco/BRL) supplemented with 10% fetal bovine serum, 2 mM glutamine, 10 U/ml penicillin and 10 g/ml streptomycin. Cells were transiently cotransfected with a DmGPCR expression construct and a reporter construct containing the luciferase gene. CRE-luciferase, AP-1-luciferase, and NF-kappa B-luciferase reporter plasmids are available from Stratagene (LaJolla, CA). Transfections were performed using FuGENE 6 transfection reagent (Boehringer-Mannheim) according to the manufacturer's instructions. Cells transfected with the reporter gene alone were used as a control. Twenty-four hours after transfection, cells were washed once with PBS prewarmed to 37C. Serum-free MEM was then added to the cells either alone (control) or with one or more of the candidate modulators, and the cells were incubated at 37C for five hours. After that, the cells were washed once with ice-cold PBS and lysed by adding 100 l of lysis buffer per well from the luciferase assay kit produced by the company. After incubation for 15 minutes at room temperature, 15 l of the lysate was mixed with 50 l of substrate solution (Promega) on an opaque white 96-well plate and luminescence was instantaneously read on a Wallace model 1450 MicroBeta scintillation and luminescence counter (Wallace Instruments, Gaithersburg, MD).</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Razlike u luminescenciji u prisutnosti i one u odsutnosti kandidatskog modulatorskog spoja su indicija modulatorske aktivnosti. Receptori koji su konstitutivno aktivni ili su aktivirani agonistima tipino daju 3-20-struko stimuliranje luminescencije u usporedbi sa stanicama koje su transficirane samo s reporterskim genom. Modulatori koji djeluju kao inverzni agonisti e obrnuti ovaj efekt. </span>Differences in luminescence in the presence and absence of a candidate modulatory compound are indicative of modulatory activity. Receptors that are constitutively active or activated by agonists typically provide a 3-20-fold stimulation of luminescence compared to cells transfected with the reporter gene alone. Modulators that act as inverse agonists will reverse this effect.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Mjerenje unutarstaninog kalcija pomou FLIPR </span>Measurement of intracellular calcium using FLIPR</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Promjene u razini unutarstaninog kalcija je drugi potvreni indikator aktivnosti G protein-vezanog receptora i takve analize mogu se izvriti da se izvri pretraivanje na modulatorsku aktivnost DmGPCR-a. Primjerice, CHO stanice postojano su transficirane s DmGPCR ekspresijskim vektorom i nanesene su na plou 4104 stanica/jaica (Packard crnih stijenki, 96 jaica po ploi) koja je posebno oblikovana da se diskriminiraju fluorescencijski signali koji dolaze iz razliitih jaica na ploi. Stanice su inkubirane tijekom 60 minuta na 37C u modificiranom Dulbecco PBS (D-PBS) koji sadri 36 mg/L piruvata i 1 g/L glukoze i jednu od etiri boje kao kalcijeva indikatora (Fluo-3 AM, Fluo-4 AM, Calcium Green-1 AM ili Oregon Green 488 BAPTA-1 AM), svaka pri koncentraciji 4 M. Ploe su isprane jednom s modificiranim D-PBS i inkubirane 10 minuta na 37C da se ukloni ostatna boja iz stanine membrane. Nadalje, izvren je niz ispiranja s modificiranim D-PBS neposredno prije aktivacije kalcijeva odgovora. </span>Changes in the level of intracellular calcium is another validated indicator of G protein-coupled receptor activity, and such analyzes can be performed to search for modulatory activity of DmGPCRs. For example, CHO cells were stably transfected with a DmGPCR expression vector and plated on a 4x104 cells/well plate (Packard black wall, 96 wells per plate) specially designed to discriminate fluorescence signals coming from different wells of the plate. Cells were incubated for 60 minutes at 37C in modified Dulbecco's PBS (D-PBS) containing 36 mg/L pyruvate and 1 g/L glucose and one of four dyes as calcium indicators (Fluo-3 AM, Fluo-4 AM , Calcium Green-1 AM or Oregon Green 488 BAPTA-1 AM), each at a concentration of 4 M. Plates were washed once with modified D-PBS and incubated for 10 minutes at 37C to remove residual dye from the cell membrane. Furthermore, a series of washes with modified D-PBS was performed immediately before the activation of the calcium response.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Kalcijev odgovor je potaknut dodatkom jednog ili vie kandidatskih receptorskih agonistikih spojeva, kalcijeva ionofora A23187 (10 M; pozitivna kontrola) ili ATP (4 M; pozitivna kontrola). Fluorescencija je mjerena pomou Molecular Devices FLIPR s argonskim laserom (ekscitacija na 488 nm) (vidi, npr. Kuntzweiler et at, Drug Dev. Res., 1998, 44(1), 14-20). F-stop (otvor blende) za detektorsku kameru postavljen je na 2,5 i duina ekspozicije je bila 0,4 milisekunde. Osnovna fluorescencija stanica mjerena je tijekom 20 sekundi prije dodavanja kandidatskog agonista, ATP ili A23187 i razina osnovne fluorescencije oduzeta je od signala odgovora. Signal kalcija mjeren je priblino 200 sekundi, uz oitanja svake 2 sekunde. Kalcij ionofor A23187 i ATP poveavaju signal kalcija 200% iznad osnovne razine. Openito, aktivirani GPCR-ovi poveavaju signal kalcija priblino 10-15% iznad osnovnog signala. </span>The calcium response was induced by the addition of one or more candidate receptor agonist compounds, the calcium ionophore A23187 (10 M; positive control) or ATP (4 M; positive control). Fluorescence was measured using Molecular Device's FLIPR with an argon laser (excitation at 488 nm) (see, e.g., Kuntzweiler et at, Drug Dev. Res., 1998, 44(1), 14-20). The f-stop (aperture) for the detector camera was set to 2.5 and the exposure length was 0.4 milliseconds. Baseline fluorescence of the cells was measured for 20 seconds before addition of the candidate agonist, ATP or A23187 and the baseline fluorescence level was subtracted from the response signal. The calcium signal was measured for approximately 200 seconds, with readings taken every 2 seconds. The calcium ionophore A23187 and ATP increase the calcium signal by 200% above baseline. In general, activated GPCRs increase the calcium signal approximately 10-15% above the baseline signal.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Analiza mitogeneze </span>Analysis of mitogenesis</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">U analizi mitogeneze, odreuje se sposobnost kandidatskog modulatora da inducira ili inhibira DmGPCR-upravljano staninu diobu (vidi, npr. Lajiness et al, J. Pharm. i Exper. r., 1993, 267(3), 1573-1581). Primjerice, CHO stanice s postojanom ekspresijom DmGPCR postavljene su u plou s 96 jaica s gustoom 5000 stanica/jaica i uzgajane su na 37C u MEM s 10% seruma goveeg fetusa tijekom 48 sati. Nakon toga su stanice dva puta isprane sa serum-slobodnim MEM. Nakon ispiranja, dodano je 80 l svjeeg MEM ili MEM koji sadri poznati mitogen, zajedno s 20 l MEM koji sadri promjenjive koncentracije jednog ili vie kandidatskih modulatora ili testnih spojeva koji su razrijeeni u serum-slobodnom mediju. Kao kontrole, neke jaice na svakoj ploi sadravale su samo serum-slobodni medij i neke su imale medij koji sadri 10% serum goveeg fetusa. Netransficirane stanice ili stanice koje su transficirane samo s vektorom mogu sluiti kao kontrole. </span>In a mitogenesis assay, the ability of a candidate modulator to induce or inhibit DmGPCR-driven cell division is determined (see, e.g., Lajiness et al, J. Pharm. and Exper. r., 1993, 267(3), 1573-1581). For example, CHO cells stably expressing DmGPCR were plated in a 96-well plate at a density of 5000 cells/well and grown at 37C in MEM with 10% fetal bovine serum for 48 hours. After that, the cells were washed twice with serum-free MEM. After washing, 80 l of fresh MEM or MEM containing a known mitogen was added, along with 20 l of MEM containing varying concentrations of one or more candidate modulators or test compounds diluted in serum-free medium. As controls, some wells on each plate contained only serum-free medium and some had medium containing 10% fetal bovine serum. Untransfected cells or cells transfected with vector alone can serve as controls.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Nakon uzgajanja tijekom 16-18 sati, dodan je u jaice 1 Ci [3H]-timidina (2 Ci/mmol) i stanice su inkubirane daljnjih 2 sata na 37C. Stanice su obraene tripsinom i sakupljene na filteru sakupljaem stanica (Tomtec); pa su filteri izmjereni u Betaplate brojau. Ugradnja [3H]-timidina u serum-slobodnim testnim jaicama usporeena je s rezultatima koji su dobiveni u stanicama koje su stimulirane serumom (pozitivna kontrola). Upotreba vie koncentracija testnih spojeva omoguuje dobivanje i analizu krivulja doza-odgovor koritenjem nelinearne jednadbe za metodu najmanjih kvadrata: A = B  [C/ (D + C)] + G, gdje je A postotak stimuliranja seruma; B je maksimalni uinak minus osnovna razina; C je EC50; D je koncentracija spoja; i G je maksimalni uinak. Parametri B, C i G odreuju se Simplex metodom optimiziranja. </span>After culturing for 16-18 hours, 1 Ci of [3H]-thymidine (2 Ci/mmol) was added to the wells and the cells were incubated for a further 2 hours at 37C. Cells were treated with trypsin and collected on the filter with a cell harvester (Tomtec); so the filters were measured in a Betaplate counter. Incorporation of [3H]-thymidine in serum-free test wells was compared with the results obtained in serum-stimulated cells (positive control). The use of multiple concentrations of test compounds allows dose-response curves to be obtained and analyzed using a nonlinear least-squares equation: A = B  [C/ (D + C)] + G, where A is the percent serum stimulation; B is the maximum effect minus the base level; C is the EC50; D is the concentration of the compound; and G is the maximum output. Parameters B, C and G are determined by the Simplex optimization method.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Agonisti koji se veu za receptor oekuje se da poveavaju ugradnju [3H]-timidina u stanice, to daje do 80% odgovora seruma. Antagonisti koji se veu za receptor e inhibirati stimuliranje koje je uoeno s poznatim agonistom sve do 100%. </span>Agonists that bind to the receptor are expected to increase [3H]-thymidine incorporation into cells, giving up to 80% of the serum response. Antagonists that bind to the receptor will inhibit the stimulation seen with a known agonist up to 100%.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Analiza vezanja [35S]GTPS </span>Analysis of [35S]GTPS binding</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Budui da G protein-vezani receptori signaliziraju kroz unutarstanine G proteine ija aktivnost ukljuuje GTP vezanje i hidrolizu da se dobije vezani GDP, mjerenje vezanja nehidrolizirajueg GTP analoga [35S]GTPS u prisutnosti i odsutnosti kandidatskog modulatora definira drugu analizu za modulator aktivnosti. Vidi, npr. Kowal et al, Neuropharmacology, 1998, 37,179-187. </span>Because G protein-coupled receptors signal through intracellular G proteins whose activity involves GTP binding and hydrolysis to yield bound GDP, measurement of binding of the non-hydrolyzable GTP analog [35S]GTPS in the presence and absence of a candidate modulator defines another assay for modulator activity. See, eg, Kowal et al, Neuropharmacology, 1998, 37,179-187.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">U jednoj primjernoj analizi, stanice koje su postojano transficirane s DmGPCR-om ekspresijskim vektorom rastu u 10 cm posudama za kulture tkiva da se usklade, isperu jednom s 5 ml ledenohladnog Ca2+/Mg2+-slobodne fosfatno puferirane slane otopine i sastruu u 5 ml istog pufera. Stanice su prele u talog centrifugiranjem (500g, 5 minuta), resuspendirane u TEE puferu (25 mM Tris, pH 7,5, 5 mM EDTA, 5 mM EDTA) i zamrznute u tekuem duiku. Nakon uzimanja, stanice su homogenizirane koritenjem Dounce homogenizatora (1 ml TEE po ploi stanica) i centrifugirane na 1,000g tijekom 5 minuta da se uklone jezgre i neslomljene stanice. </span>In an exemplary assay, cells stably transfected with a DmGPCR expression vector are grown in 10 cm tissue culture dishes to confluence, washed once with 5 ml of ice-cold Ca2+/Mg2+-free phosphate-buffered saline, and scraped into 5 ml of the same buffer. . Cells were pelleted by centrifugation (500g, 5 minutes), resuspended in TEE buffer (25 mM Tris, pH 7.5, 5 mM EDTA, 5 mM EDTA) and frozen in liquid nitrogen. After harvesting, cells were homogenized using a Dounce homogenizer (1 ml TEE per plate of cells) and centrifuged at 1,000g for 5 min to remove nuclei and unbroken cells.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Homogenat supernatanta centrifugiran je na 20,000g tijekom 20 minuta da se izdvoji membranska frakcija to i membranski talog je ispran jednom s TEE i resuspendiran u vezujuem puferu (20 mM HEPES, pH 7,5, 150 mM NaCl, 10 mM MgCl2, 1 mM EDTA). Resuspendirane membrane mogu se zamrznuti u tekuem duiku i pohraniti na -70C do upotrebe. </span>The supernatant homogenate was centrifuged at 20,000g for 20 min to isolate the membrane fraction and the membrane pellet was washed once with TEE and resuspended in binding buffer (20 mM HEPES, pH 7.5, 150 mM NaCl, 10 mM MgCl2, 1 mM EDTA). Resuspended membranes can be frozen in liquid nitrogen and stored at -70C until use.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Alikvoti staninih membrane prireeni su kao to je prije opisano i pohranjeni na -70C pa uzeti, homogenizirani i razrijeeni u puferu koji sadri 20 mM HEPES, 10 mM MgCl2, 1 mM EDTA, 120 mM NaCl, 10 M GDP i 0,2 mM askorbata, pri koncentraciji 10-50 g/ml. U konanom volumenu od 90 l, homogenati su inkubirani s promjenjivom koncentracijom spojeva kandidatskog modulatora ili 100 M GTP tijekom 30 minuta na 30C i zatim stavljeni na led. Svakom dodano je 10 l guanozin 5-O-(3[35S]tio) trifosfata (NEN, 1200 Ci/mmol; [35S]-GTPS), do konane koncentracije od 100-200 pM. Uzorci su inkubirani na 30C tijekom daljnjih 30 minuta, dodan je 1 ml 10 mM HEPES, pH 7,4, 10 mM MgCl2, na 4C, pa je reakcija zaustavljena filtriranjem. </span>Aliquots of cell membranes were prepared as previously described and stored at -70C, then harvested, homogenized, and diluted in a buffer containing 20 mM HEPES, 10 mM MgCl2, 1 mM EDTA, 120 mM NaCl, 10 M GDP, and 0.2 mM ascorbate. , at a concentration of 10-50 g/ml. In a final volume of 90 l, homogenates were incubated with varying concentrations of candidate modulator compounds or 100 M GTP for 30 min at 30C and then placed on ice. 10 l of guanosine 5'-O-(3[35S]thio) triphosphate (NEN, 1200 Ci/mmol; [35S]-GTPS) was added to each, to a final concentration of 100-200 pM. The samples were incubated at 30C for a further 30 minutes, 1 ml of 10 mM HEPES, pH 7.4, 10 mM MgCl2 was added at 4C, and the reaction was stopped by filtration.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Uzorci su filtrirani preko Whatman GF/B filtera i filteri su isprani s 20 ml ledenohladnog 10 mM HEPES, pH 7,4, 10 mM MgCl2. Filteri su Izmjereni LC spektroskopijom. Nespecifino vezanje [35S]-GTPS izmjereno je u prisutnosti 100 M GTP i oduzeto od totala. Odabrani su spojevi koji moduliraju koliinu [35S]GTPS vezanja u stanicama, u usporedbi s netransficiranim kontrolnim stanicama. Aktiviranje receptora pomou agonista daje peterostruko poveanje [35S]GTPS vezanja. Ovaj odgovor je blokiran pomou antagonista. </span>Samples were filtered through Whatman GF/B filters and the filters were washed with 20 ml of ice-cold 10 mM HEPES, pH 7.4, 10 mM MgCl 2 . The filters were measured by LC spectroscopy. Nonspecific binding of [35S]-GTPS was measured in the presence of 100 M GTP and subtracted from the total. Compounds were selected that modulate the amount of [35S]GTPS binding in the cells, compared to non-transfected control cells. Activation of the receptor by an agonist results in a fivefold increase in [35S]GTPS binding. This response is blocked by antagonists.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Analiza aktivnosti MAP kinaze </span>Analysis of MAP kinase activity</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Procjena MAP kinaza aktivnost u stanicama s ekspresijom GPCR definira drugu analizu da se identificiraju modulatori DmGPCR aktivnost. Vidi npr. Lajiness et al, J. Pharm. i Exper. r., 1993, 267(3), 1573-1581 te Boulton et al, Cell, 1991, 65, 663-675. </span>Assessment of MAP kinase activity in cells with GPCR expression defines another analysis to identify modulators of DmGPCR activity. See, eg, Lajiness et al, J. Pharm. and Exper. r., 1993, 267(3), 1573-1581 and Boulton et al, Cell, 1991, 65, 663-675.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">U jednoj realizaciji, CHO stanice postojano transficirane s DmGPCR-om su zasaene na ploe sa 6 jaica s gustoom 70000 stanica/jaica 48 sati prije analize. Tijekom ovog perioda od 48-sati, stanice su uzgojene na 37C u MEM mediju kojem je dodan 10% serum volovskog zametka, 2 mM glutamina, 10 U/ml penicilina i 10 g/ml streptomicina. Stanice su serum izgladnjele tijekom 1-2 sata prije dodavanja stimulansa. </span>In one embodiment, CHO cells stably transfected with DmGPCR were seeded in 6-well plates at a density of 70,000 cells/well 48 hours prior to analysis. During this 48-hour period, cells were grown at 37C in MEM medium supplemented with 10% fetal bovine serum, 2 mM glutamine, 10 U/ml penicillin, and 10 g/ml streptomycin. Cells were serum-starved for 1-2 hours before the addition of stimulants.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Za analizu, stanice su tretirane samo s medijem ili s medijem koji sadri kandidatski agonist ili 200 nM Phorbol ester-miristoil acetata (tj. PMA, pozitivna kontrola) i stanice su inkubirane na 37C tijekom razliitog vremena. Da se zaustavi reakcija, ploe su stavljene na led, medij je usisan i stanice su isprane s 1 ml ledenohaldnog PBS koji sadri 1 mM EDTA. Nakon toga, stanicama je dodano 200 l staninog lizatnog pufera (12,5 mM MOPS, pH 7,3, 12,5 mM glicerofosfat, 7,5 mM MgCl2, 0,5 mM EDTA, 0,5 mM natrijeva vanadata, 1 mM benzamidina, 1 mM ditiotreitola, 10 g/ml leupeptina, 10 g/ml aprotinina, 2 g/ml pepstatina i 1 M okadaine kiseline). Stanice su sastrugane s ploa i homogenizirane s 10 prolaza kroz 23 3/4 G iglu i prireena je citosolna frakcija na 20000g tijekom 15 minuta. </span>For analysis, cells were treated with medium alone or with medium containing the candidate agonist or 200 nM Phorbol ester-myristoyl acetate (ie, PMA, positive control) and cells were incubated at 37C for various times. To stop the reaction, the plates were placed on ice, the medium was aspirated, and the cells were washed with 1 ml of ice-cold PBS containing 1 mM EDTA. After that, 200 l of cell lysate buffer (12.5 mM MOPS, pH 7.3, 12.5 mM glycerophosphate, 7.5 mM MgCl2, 0.5 mM EDTA, 0.5 mM sodium vanadate, 1 mM benzamidine, 1 mM dithiothreitol, 10 g/ml leupeptin, 10 g/ml aprotinin, 2 g/ml pepstatin and 1 M okadaic acid). Cells were scraped from the plates and homogenized with 10 passes through a 23 3/4 G needle, and the cytosolic fraction was prepared at 20,000g for 15 minutes.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Alikvoti (5-10 l koji sadri 1-5 g proteina) citosola pomijeani su s 1 mM MAPK supstratnog peptida (APRTPGGRR (sekv. br. 168), Upstate Biotechnology, Inc., N.Y.) i 50 M [-32P]ATP (NEN, 3000 Ci/mmol), razrijeeni na konanu specifinu aktivnost od ~2000 cpm/pmol, u ukupnom volumenu od 25 l. Uzorci su inkubirani 5 minuta na 30C i reakcije su zaustavljene stavljanjem kapi od 20 l na 2 cm2 kvadrat Whatman P81 fosfoceluloznog papira. Filterski kvadrati su isprani u 4 izmjene 1% H3PO4 i kvadrati su podvrgnuti LC spektroskopiji da se kvantificira vezani obiljeeni dio. Ekvivalentni citosolni ekstrakti su inkubirani bez MAPK supstratnog peptida, te je vezani obiljeeni dio iz obih uzoraka oduzet iz odgovarajuih uzoraka sa supstratnim peptidom. Citosolni ekstrakt iz svake jaice koriten je kao odvojena toka. Koncentracije proteina su odreene pomou proteinske analize vezanja boje (Bio-Rad Laboratories). Agonistiko aktiviranje receptora oekuje se da rezultira do peterostrukog poveanja MAPK enzimske aktivnosti. Ovo poveanje je blokirano pomou antagonista. </span>Aliquots (510 l containing 15 g protein) of cytosol were mixed with 1 mM MAPK substrate peptide (APRTPGGR (SEQ ID NO: 168), Upstate Biotechnology, Inc., N.Y.) and 50 M [-32P]ATP (NEN, 3000 Ci/mmol), diluted to a final specific activity of ~2000 cpm/pmol, in a total volume of 25 l. Samples were incubated for 5 minutes at 30C and reactions were stopped by placing a 20 l drop on a 2 cm2 square of Whatman P81 phosphocellulose paper. The filter squares were washed with 4 changes of 1% H3PO4 and the squares were subjected to LC spectroscopy to quantify the bound labeled moiety. Equivalent cytosolic extracts were incubated without MAPK substrate peptide, and the bound labeled part from both samples was subtracted from the corresponding samples with substrate peptide. Cytosolic extract from each well was used as a separate spot. Protein concentrations were determined using a dye-binding protein assay (Bio-Rad Laboratories). Agonistic activation of the receptor is expected to result in up to a fivefold increase in MAPK enzymatic activity. This increase is blocked by antagonists.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Otputanje [3H]arahidonske kiseline </span>Release of [3H]arachidonic acid</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Aktiviranje GPCR-ova takoer je uoeno da jaa otputanje arahidonske kiseline u stanicama, to omoguuje jo jednu korisnu analizu na modulatore GPCR aktivnosti. Vidi npr. Kanterman et al, Molecular Pharmacology, 1991, 39, 364-369. Primjerice, CHO stanice koje su postojano transficirane s DmGPCR ekspresijskim vektorom stavljene su na ploe s 24 jaice s gustoom 15000 stanica/jaica i rasle su u MEM mediju kojem je dodan 10% serum volovskog zametka, 2 mM glutamin, 10 U/ml penicilin i 10 g/ml streptomicin tijekom 48 sati na 37C prije upotrebe. Stanice svake jaice su obiljeene inkubiranjem s [3H]-arahidonskom kiselinom (Amersham Corp., 210 Ci/mmol) na 0,5 piCi/ml u 1 ml MEM kojem je dodan 10 mM HEPES, pH 7,5 i 0,5% masna kiselina-slobodni volovski serum albumin tijekom 2 sata 37C. Stanice su zatim isprane dva puta s 1 ml istog pufera. </span>Activation of GPCRs has also been observed to enhance the release of arachidonic acid in cells, providing another useful assay for modulators of GPCR activity. See, eg, Kanterman et al, Molecular Pharmacology, 1991, 39, 364-369. For example, CHO cells stably transfected with a DmGPCR expression vector were plated in 24-well plates at a density of 15,000 cells/well and grown in MEM medium supplemented with 10% fetal bovine serum, 2 mM glutamine, 10 U/ml penicillin and 10 g/ml streptomycin for 48 hours at 37C before use. Cells in each well were labeled by incubating with [3H]-arachidonic acid (Amersham Corp., 210 Ci/mmol) at 0.5 pCi/ml in 1 ml MEM supplemented with 10 mM HEPES, pH 7.5, and 0.5% fatty acid-free bovine serum albumin for 2 hours at 37C. The cells were then washed twice with 1 ml of the same buffer.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Kandidatski modulatorski spojevi dodani su u 1 ml istog pufera, bilo sami ili s 10 M ATP i stanice su inkubirane na 37C tijekom 30 minuta. Sam puffer i mock-transfektirane stanice koritene su kao kontrole. Uzorci (0,5 ml) iz svake jaice izbrojani su LC spektroskopijom. Agonisti koji aktiviraju receptor vode do jaanja ATP-stimuliranog otputanja [3H]-arahidonske kiseline. Ovo jaanje je blokirano antagonistima. </span>Candidate modulatory compounds were added to 1 ml of the same buffer either alone or with 10 M ATP and cells were incubated at 37C for 30 minutes. Puffer alone and mock-transfected cells were used as controls. Samples (0.5 ml) from each well were counted by LC spectroscopy. Agonists that activate the receptor lead to enhancement of ATP-stimulated release of [3H]-arachidonic acid. This strengthening is blocked by antagonists.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Brzina izvanstanine acidifikacije </span>The rate of extracellular acidification</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">U sljedeoj analizi, uinci kandidatske modulatorske aktivnosti DmGPCR analizirani su praenjem izvanstaninih promjena pH koje su izazvane testnim spojevima. Vidi npr. Dunlop et al., J. Pharmacological i Toxicological Methods, 1998, 40(1), 47-55. U jednoj realizaciji, CHO stanice transficirane s DmGPCR-om ekspresijskim vektorom su nasaene u 12 mm kapsulne aice (Molecular Devices Corp.) na 4105 stanica/aica in MEM kojemu je dodan 10% serum volovskog zametka, 2 mM L-glutamin, 10 U/ml penicilin i 10 g/ml streptomicin. Stanice su inkubirane u ovom mediju na 37C u 5% CO2 tijekom 24 sata. </span>In the following analysis, the effects of the candidate DmGPCR modulatory activity were analyzed by monitoring the extracellular pH changes induced by the test compounds. See, eg, Dunlop et al., J. Pharmacological and Toxicological Methods, 1998, 40(1), 47-55. In one embodiment, CHO cells transfected with a DmGPCR expression vector were seeded in 12 mm capsule dishes (Molecular Devices Corp.) at 410 5 cells/dish in MEM supplemented with 10% fetal bovine serum, 2 mM L-glutamine, 10 U/ml penicillin and 10 g/ml streptomycin. Cells were incubated in this medium at 37C in 5% CO2 for 24 hours.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Brzina izvanstaninog zakiseljavanja mjerena je pomou Cytosensor mikrofiziometra (Molecular Devices Corp.). Kapsulne aice su unesene u komoru senzora mikrofiziometra i komore su perfudirane s tekuim puferom (bikarbonat-slobodan MEM obogaen s 4 mM L-glutamin, 10 jedinica/ml penicilin, 10 g/ml streptomicin, 26 mM NaCl) brzinom od 100 l/minute. Kandidatski agonisti ili druga sredstva su razrijeena u tekuem puferu i perfudirana kroz dugi fluidni put. Tijekom svakog 60-sekundnog ciklusa pumpanja, pumpa je radila 38 sekundi i bila je iskljuena preostale 22 sekunde. pH tekueg pufera u komori senzora je biljeen tijekom ciklusa izmeu 43-58 sekundi i pumpa je ponovo pokrenuta nakon 60 sekundi da se pokrene sljedei ciklus. Brzina zakiseljavanja tekueg pufera tijekom biljeenog vremena izraunata je pomou Cytosoft programa. Promjene u brzini zakiseljavanja izraunate su oduzimanjem osnovne vrijednosti (prosjek od 4 mjerenja brzine neposredno prije dodavanja modulatorskog kandidata) od mjerenja najvee brzine koja je dobivena nakon dodavanja modulatorskog kandidata. Odabrani instrument detektira 61 mV/pH jedinica. Modulatori koji djeluju kao agonisti receptora rezultiraju poveanjem brzine izvanstaninog zakiseljavanja u usporedbi s brzinom u odsutnosti agonista. Ovaj odgovor je blokiran pomou modulatora koji djeluju kao antagonisti receptora. </span>The rate of extracellular acidification was measured using a Cytosensor microphysiometer (Molecular Devices Corp.). Capsule cups were inserted into the sensor chamber of the microphysiometer and the chambers were perfused with liquid buffer (bicarbonate-free MEM supplemented with 4 mM L-glutamine, 10 units/ml penicillin, 10 g/ml streptomycin, 26 mM NaCl) at a rate of 100 l/minute. . Candidate agonists or other agents are diluted in liquid buffer and perfused through a long fluid path. During each 60-second pumping cycle, the pump was on for 38 seconds and off for the remaining 22 seconds. The pH of the liquid buffer in the sensor chamber was recorded during the cycle between 43-58 seconds and the pump was restarted after 60 seconds to start the next cycle. The rate of acidification of the liquid buffer during the recorded time was calculated using the Cytosoft program. Changes in acidification rate were calculated by subtracting the baseline value (average of 4 rate measurements immediately before addition of the modulator candidate) from the peak rate measurement obtained after addition of the modulator candidate. The selected instrument detects 61 mV/pH units. Modulators that act as receptor agonists result in an increase in the rate of extracellular acidification compared to the rate in the absence of agonist. This response is blocked by modulators that act as receptor antagonists.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Primjer 9: Slaganje DmGPCR-ova s peptidnim ligandima </span>Example 9: Docking of DmGPCRs with Peptide Ligands</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Stanine kulture i transfekcije </span>Cell cultures and transfections</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Stanice divljeg tipa jajnika kineskog zamorca (CHO-K1) (iz: American Type Culture Collection, Rockville, MD) ili CHO-10001A stanice uzgajane su na 37C u vlanoj atmosferi s 5% CO2 u zraku u DMEM mediju kojem je dodano 10% toplinski-neaktiviranog FBS, 10 g/ml gentamicin, 0,1 mM neesencijalnih aminokiselina da se dobije cjeloviti DMEM medij. Stanice su transficirane s osamljenim GPCR DNAovima u pCR3.1 vektoru, koritenjem LipofectAMINE PLUS, sutinski sukladno naputku proizvoaa. Ukratko, CHO stanice stavljene su na plou 10 cm sterilne posude tkivne kulture (Corning Glass Works, Corning, NY) i one su bile oko 50-60% konfluentne na dan transfekcije. U plastinoj epruveti, PLUS (20 l/plate) dodan je na cDNA plasmid (5 g /ploa) koji je ranije razrijeen u 0,75 ml OptiMEM, pomijean i inkubiran na sobnoj temperature tijekom 15 min. Odvojeno, LipofectAMINE (30 l/ploa) pomijean je s 0,75 ml OptiMEM i dodan je pre-kompleksiranoj DNA/PLUS smjesi i inkubirano je na sobnoj temperaturi, tijekom 15 minuta. Medij na stanicama zamijenjen je sa serum-slobodnim transfekcijskim medijem (isti DMEM, 5 ml/plate) i dodan je DNA-PLUS-LipofectAMINE kompleks (1,5 ml po ploi) te je njeno mijeano u mediju uz 3 sata inkubiranje na 37C/5% CO2. Medij je zatim dopunjen s cjelovitim DMEM medijem koji sadri 20% FBS (6,5 ml ml/ploa) i nastavljeno je inkubiranjem na 37C/5% CO2 tijekom 24 do 48 sati. Plazmid za zeleni fluorescentni protein (GFP, 4 g/ploa) koriten je za prijelaznu GFP ekspresiju u CHO stanicama da se procijeni prinos transfekcije u identinim uvjetima koji su takoer koriteni za GPCR-ove. </span>Wild-type Chinese guinea pig ovary cells (CHO-K1) (from: American Type Culture Collection, Rockville, MD) or CHO-10001A cells were grown at 37C in a humidified atmosphere with 5% CO2 in air in DMEM medium supplemented with 10% thermal -inactivated FBS, 10 g/ml gentamicin, 0.1 mM non-essential amino acids to obtain complete DMEM medium. Cells were transfected with isolated GPCR DNAs in the pCR3.1 vector, using LipofectAMINE PLUS, essentially according to the manufacturer's instructions. Briefly, CHO cells were plated on a 10 cm sterile tissue culture dish (Corning Glass Works, Corning, NY) and were approximately 5060% confluent on the day of transfection. In a plastic tube, PLUS (20 l/plate) was added to the cDNA plasmid (5 g/plate) previously diluted in 0.75 ml OptiMEM, mixed and incubated at room temperature for 15 min. Separately, LipofectAMINE (30 l/plate) was mixed with 0.75 ml of OptiMEM and added to the pre-complexed DNA/PLUS mixture and incubated at room temperature for 15 minutes. The medium on the cells was replaced with serum-free transfection medium (pure DMEM, 5 ml/plate) and the DNA-PLUS-LipofectAMINE complex (1.5 ml per plate) was added and gently mixed in the medium with 3 hours incubation at 37C/ 5% CO2. The medium was then supplemented with complete DMEM medium containing 20% FBS (6.5 ml ml/plate) and continued incubation at 37C/5% CO2 for 24 to 48 hours. A plasmid for green fluorescent protein (GFP, 4 g/plate) was used for transient GFP expression in CHO cells to assess transfection yield under identical conditions also used for GPCRs.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Prireivanje membrane </span>Preparing the membrane</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Transficirane stanice isprane su jednom s ledenohladnom Dulbecco fosfatno puferiranom slanom otopinom (PBS), 5 ml po ploi 10 cm i sastrugane u 5 ml istog pufera. Stanine suspenzije iz vie ploa su kombinirane i centrifugirane na 500g tijekom 10 min na 4C. Stanini talog je rekonstituiran na ledenohladnom TEE (25 mM TRIS, 5 mM EGTA, 5 mM EDTA). Pogodni alikvoti su naglo zamrznuti u tekuem duiku i pohranjeni na -70C. Nakon uzimanja, stanice su homogenizirane i centrifugirane na 4C, 500g tijekom 5 minuta da se staloe jezgre i neslomljene stanice. Supernatant je centrifugiran na 47000g tijekom 30 minuta na 4C. Membranski talog je ispran jednom s TEE, resuspendiran u 20 mM HEPES, pH 7,4, 100 mM NaCl, 10 mM MgCl2, 1 mM EDTA (pufer analize), podijeljen u alikvote i zamrznut u tekuem duiku. Membranski alikvoti su pohranjeni na -70C. Koncentracija membranskog proteina odreena je koritenjem BCA reagensa za proteinsku analizu od tvrtke Pierce (Rockford, Illinois) i BSA kao standarda. </span>Transfected cells were washed once with ice-cold Dulbecco's phosphate-buffered saline (PBS), 5 ml per 10 cm plate, and scraped into 5 ml of the same buffer. Cell suspensions from multiple plates were combined and centrifuged at 500g for 10 min at 4C. The cell pellet was reconstituted on ice-cold TEE (25 mM TRIS, 5 mM EGTA, 5 mM EDTA). Suitable aliquots were snap frozen in liquid nitrogen and stored at -70C. After collection, cells were homogenized and centrifuged at 4C, 500g for 5 minutes to pellet nuclei and unbroken cells. The supernatant was centrifuged at 47000g for 30 minutes at 4C. The membrane pellet was washed once with TEE, resuspended in 20 mM HEPES, pH 7.4, 100 mM NaCl, 10 mM MgCl2, 1 mM EDTA (assay buffer), aliquoted, and frozen in liquid nitrogen. Membrane aliquots were stored at -70C. Membrane protein concentration was determined using BCA protein assay reagent from Pierce (Rockford, Illinois) and BSA as a standard.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Analiza [35S]GTPS vezanja </span>Analysis of [35S]GTPS binding</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Uzeti su alikvoti staninih mebrana, homogenizirani su i razrijeeni u puferu koji sadri 20 mM HEPES, pH 7,4, 100 mM NaCl, 10 mM MgCl2, 1 mM EDTA (pufer analize). Na poetku, prireene su reakcijske smjese na polipropilenskim ploama s 96 jaica (Nunc). U svaku jaicu, pomijeani su vodena otopina peptida (20 l, 10 ) ili vodena kontrola (20 l), 18,2 M GDP u puferu analize (0,11 ml, 10 M final) i membrane koje su suspendirane u puferu analize (50 l, 10 g membranskog proteina) te stavljeni na led. Smjese ligand-GDP-membrana inkubirane su tijekom 20 min na sobnoj temperaturi na platformi za mukanje i zatim stavljene na led. Svakom uzorku dodano je 20 l guanozin-5-O-(3-[35S]tio)-trifosfata ([35S]GTPS) (600-1200 Ci/mmol iz New England Nuclear, Boston, MA) na ~ 40,000 cpm/0,2 ml ili konane koncentracije 0,1 nM. Ploe s inkubacijskim smjesama (ukupno 0,2 ml/jaica) inkubirane su na sobnoj temperaturi tijekom 45 minuta. Alikvoti reakcijske smjese, 0,175 ml svaki, preneseni su u puferom za ispiranje pretretirane (100 l/jaica) 96-jaica FB MultiScreen filterske ploe (Millipore) i vakuumski filtrirani koritenjem MultiScreen Vacuum pumpe (Millipore). Membrane su zatim isprane 3 puta s 0,25 ml ledenohladnbog pufera za ispiranje/jaica (10 mM HEPES, 10 mM MgCl2, pH 7,4) i vakuumski filtrirane. Nakon zadnjeg ispiranja, dodana je Supermix Opti-phase scintilacijska tekuina (25 l /jaica, Wallac) i ploe su zatvorene te izmjerene u Trilux 1450 Microbeta brojau (Wallac) tijekom 1 minuta/jaica. Kao pozitivne kontrole, membrane iz CHO stanica sa postojanom ekspresijom dopamin tip 2 (rD2) receptora tretirane su s 1 mM dopaminom u 0,025% askorbinskoj kiselini (100 M konana koncentracija dopamina) ili nosau (0,0025% konana koncentracija askorbinske kiseline). Nespecifino vezanje izmjereno je u prisutnosti 100 M hladnog GTPS i oduzeto od totala. Svako tretiranje izvreno je u triplikatu. </span>Aliquots of cell membranes were taken, homogenized and diluted in a buffer containing 20 mM HEPES, pH 7.4, 100 mM NaCl, 10 mM MgCl2, 1 mM EDTA (analysis buffer). Initially, reaction mixtures were prepared in 96-well polypropylene plates (Nunc). Into each well, aqueous peptide solution (20 l, 10) or aqueous control (20 l), 18.2 M GDP in assay buffer (0.11 ml, 10 M final) and membranes suspended in assay buffer were mixed. (50 l, 10 g membrane protein) and placed on ice. Ligand-GDP-membrane mixtures were incubated for 20 min at room temperature on a shaking platform and then placed on ice. To each sample, 20 l of guanosine-5'-O-(3-[35S]thio)-triphosphate ([35S]GTPS) (600-1200 Ci/mmol from New England Nuclear, Boston, MA) was added at ~ 40,000 cpm/ 0.2 ml or a final concentration of 0.1 nM. Plates with incubation mixtures (total 0.2 ml/well) were incubated at room temperature for 45 minutes. Aliquots of the reaction mixture, 0.175 ml each, were transferred to wash buffer-pretreated (100 l/well) 96-well FB MultiScreen filter plates (Millipore) and vacuum filtered using a MultiScreen Vacuum pump (Millipore). Membranes were then washed 3 times with 0.25 ml of ice-cold wash buffer/well (10 mM HEPES, 10 mM MgCl2, pH 7.4) and vacuum filtered. After the last wash, Supermix Opti-phase scintillation fluid (25 l/well, Wallac) was added and the plates were sealed and counted in a Trilux 1450 Microbeta counter (Wallac) for 1 minute/well. As positive controls, membranes from CHO cells stably expressing dopamine type 2 (rD2) receptors were treated with 1 mM dopamine in 0.025% ascorbic acid (100 M final concentration of dopamine) or vehicle (0.0025% final concentration of ascorbic acid). Nonspecific binding was measured in the presence of 100 M cold GTPS and subtracted from the total. Each treatment was performed in triplicate.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Analiza podataka </span>Data analysis</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Ligand-izazvano stimuliranje [35S]GTPS vezanja izraeno je kao poveanje u odnosu na osnovnu aktivnost kada nije dodan ligand. Svaki tretman izvren je u triplikatu ili, povremeno, u duplikatu i vezanje (cpm) je izraunato kao prosjek +/- standardna devijacija. Krivulje doza-odgovor za sustave receptor/ligand analizirane su koritenjem nelinearnog SAS modela metode najmanjih kvadrata, y= BmaxX/(Kd + X). Druge krivulje doza-odgovor analizirane su koritenjem Prism programa (GraphPad Software, Inc. San Diego, CA) i sukladno jednadbi y = dno + (vrh - dno)/ (1 + 10logEC50-X). </span>Ligand-induced stimulation of [35S]GTPS binding was expressed as an increase over baseline activity when no ligand was added. Each treatment was performed in triplicate or, occasionally, in duplicate and binding (cpm) was calculated as the mean +/- standard deviation. Dose-response curves for receptor/ligand systems were analyzed using a non-linear least-squares SAS model, y= BmaxX/(Kd + X). Other dose-response curves were analyzed using the Prism program (GraphPad Software, Inc. San Diego, CA) and according to the equation y = bottom + (peak - bottom)/ (1 + 10logEC50-X).</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Rezultati </span>the results</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Izvorno, odabrali smo GTPS analizu kao funkcionalnu analizu jer agonist-pokrenuto stimuliranje GTPS odraava rane dogaaje u DmGPCR aktivacijskoj kaskadi, bez obzira na dalje aktivacijske putove razliitih nizvodnih signalizirajuih dogaaja. Ovo se ini osobito korisnim za procjenu mogueg aktiviranja usamljenih DmGPCR-ova s nepoznatim funkcijama i nepoznatim putovima signaliziranja. GTPS analiza provedena je s membranama koje su prireene iz CHO stanica koje su prijelazno transficirane s DNA koja kodira Drosophila GPCR-ove koritenjem filterskih ploa s 96 jaica MultiScreen G/FB i MultiScreen vacuumske pumpe (Millipore) za filtriranje. Budui da je poznato da GTPS analiza slabo prepoznaje GPCR-ove koji su vezani za Gq klasu G-proteina, analiza Ca+2 mobilizacije koja se temelji na FLIPR oitanju koritena je takoer za procjenu Gq-vezanih usamljenih GPCR-ova u CHO stanicama koje su prijelazno transficirane s DNA koja kodira Drosophila GPCR-ove. </span>Originally, we chose the GTPS assay as a functional assay because agonist-triggered stimulation of GTPS reflects early events in the DmGPCR activation cascade, regardless of further activation pathways of various downstream signaling events. This seems particularly useful for assessing the possible activation of single DmGPCRs with unknown functions and unknown signaling pathways. GTPS analysis was performed with membranes prepared from CHO cells transiently transfected with DNA encoding Drosophila GPCRs using MultiScreen G/FB 96-well filter plates and a MultiScreen vacuum pump (Millipore) for filtration. Since the GTPS assay is known to poorly recognize GPCRs that are coupled to the Gq class of G-proteins, a Ca+2 mobilization assay based on FLIPR readout was also used to assess Gq-coupled single GPCRs in CHO cells that were transiently transfected with DNA encoding Drosophila GPCRs.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Koritenjem GTPS analize, DmGPCR1 (PnuFlyPep34651) naeno je da je aktiviran pomou dva Drosophila NPF-slina peptida, AQRSPSLRLRF-NH2 (sekv. br. 186) i PIRSPSLRLRF-NH2 (sekv. br. 187 ) to se odrazilo naenim EC50 vrijednostima od oko 2,5 nM. Aktiviranje s DPKQDFMRF-NH2 (sekv. br. 26) i PDNFMRF-NH2 (sekv. br. 27) rezultiralo je GTPS odgovorima s EC50 u rasponu od 370 nM do 500 nM. Kao to je izvijestio Nambu et al. (Neuron, 1988,1, 55-61), ova dva peptida su kodirana na istom prekursorskom genu zajedno s devet drugih FaRPova. Daljnji FaRPovi i drugi neuropeptidi koji stimuliraju GTPS vezanje, premda su manje uinkoviti (EC50 u rasponu 5 do 10 M), ukljuuju sljedee peptide: TDVDHVFLRF-NH2 (sekv. br. 25), TPAEDFMRF-NH2 (sekv. br. 28), SLKQDFMHF-NH2 (sekv. br. 29), SVKQDFMHF-NH2 (sekv. br. 30), AAMDRY-NH2 (sekv. br. 31) i SVQDNFMHF-NH2 (sekv. br. 32). Nadalje, FLIPR analiza identificirala je peptid Colorado krumpirove zlatice, ARGPQLRLRF-NH2 (sekv. br. 33), to odgovara DmGPCR1 receptoru s EC50 od 100-200 nM. Nai podaci pokazuju da bi Dmgpcrl trebao biti klasificiran kao kratki neuropeptidni F receptor jer je jako aktiviran s dva kratka NPF peptida, sekv. br. 186 i sekv. br. 187. </span>Using the GTPS assay, DmGPCR1 (PnuFlyPep34651) was found to be activated by two Drosophila NPF-like peptides, AQRSPSLRRLRF-NH2 (SEQ ID NO: 186) and PIRSPSLRLRF-NH2 (SEQ ID NO: 187), as reflected by EC50 values of approx. 2.5 nM. Activation with DPKQDFMRF-NH2 (SEQ ID NO: 26) and PDNFMRF-NH2 (SEQ ID NO: 27) resulted in GTPS responses with EC50s ranging from 370 nM to 500 nM. As reported by Nambu et al. (Neuron, 1988,1, 55-61), these two peptides are encoded on the same precursor gene along with nine other FaRPs. Further FaRPs and other neuropeptides that stimulate GTPS binding, albeit less efficiently (EC50 in the 5 to 10 M range), include the following peptides: TDVDHVFLRF-NH2 (SEQ ID NO: 25), TPAEDFMRF-NH2 (SEQ ID NO: 28), SLKQDFMHF-NH2 (SEQ ID NO: 29), SVKQDFMHF-NH2 (SEQ ID NO: 30), AAMDRY-NH2 (SEQ ID NO: 31) and SVQDNFMHF-NH2 (SEQ ID NO: 32). Furthermore, FLIPR analysis identified a Colorado potato beetle peptide, ARGPQLRLRF-NH2 (SEQ ID NO: 33), corresponding to the DmGPCR1 receptor with an EC50 of 100-200 nM. Our data indicate that Dmgpcrl should be classified as a short neuropeptide F receptor because it is strongly activated by two short NPF peptides, seq. no. 186 et seq. no. 187.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Kao to su pokazali GTPS odgovori, DmGPCR4 (PnuFlyPep 67393) je aktiviran pomou Drosophila melanogaster alatostatina, drostatina-3 (SRPYSFGL-NH2 (sekv. br. 165)) s EC50 u niskom nanomolarnom rasponu, kao i pomou razliitih Diplotera punctata (ohar) alatostatina, dotino: GDGRLYAFGL-NH2 (sekv. br. 34), DRLYSFGL-NH2 (sekv. br. 35), APSGAQRLYGFGL-NH2 (sekv. br. 36) i GGSLYSFGL-NH2 (sekv. br. 37) (EC50 u rasponu ca. 20-280 nM). Isti peptidi izazvali su vrlo jak kalcijev signal kada su ispitani na 10 M pomou FLIPR. DmGPCR4 bio je nedavno kloniran od strane Lenza i suradnika, supra i klasificiran je kao drugi potencijalni alatostatinski receptor (DARII). Meutim, do danas nema farmakolokih podataka o receptorskom aktiviranju. Sukladno naem znanju, ovo je prvi eksperimentalni dokaz da razliiti alatostatini aktiviraju ovaj receptor. </span>As shown by GTPS responses, DmGPCR4 (PnuFlyPep 67393) was activated by the Drosophila melanogaster allatostatin, drostatin-3 (SRPYSFGL-NH2 (SEQ ID NO: 165)) with an EC50 in the low nanomolar range, as well as by various Diplotera punctata (cockroach) allatostatin, respectively: GDGRLYAFGL-NH2 (SEQ ID NO: 34), DRLYSFGL-NH2 (SEQ ID NO: 35), APSGAQRLYGFGL-NH2 (SEQ ID NO: 36) and GGSLYSFGL-NH2 (SEQ ID NO: 37) (EC50 in range approx. 20-280 nM). The same peptides elicited a very strong calcium signal when probed at 10 M by FLIPR. DmGPCR4 was recently cloned by Lenz et al., supra and classified as a second potential allatostatin receptor (DARII). However, to date there is no pharmacological data on receptor activation. To our knowledge, this is the first experimental evidence that different allatostatins activate this receptor.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Kao to su pokazali GTP75 odgovori, DmGPCR5 (GenBank pristupni br. AX128628) kada prijelazno imaju ekspresiju u CHO-10001A stanicama, aktivirani su pomou drotahikinina (DTKs), dotino DTK-1 (APTSSFIGMR-NH2) (sekv. br. 169), Met8-DTK-2 (APLAFYGMR-NH2) (sekv. br. 170), DTK-2 (APLAPYGLR-NH2) (sekv. br. 171, DTK-3 (APTGFTGMR-NH2) (sekv. br. 172), DTK-4 (APVNSFVGMR-NH2) (sekv. br. 173) i DTK-5 (APNGFLGMR-NH2) (sekv. br. 174). U eksperimentima doza-odgovor, DTK-1, Met8-DTK-2, DTK-3 i DTK-5 stimulirano GTP-S vezanje je s EC50 u 250-500 nM rasponu i maksimalno stimuliranje je cca 1,5-puta iznad osnovne razine. DTK-2 i DTK-4 su slabije snage sudei prema njihovu EC50 koji je u niskom mikromolarnom rasponu. U analizi kalcijeve mobilizacije (FLIPR), DmGPCR5 pokazao je Ca+2 odgovore na iste DTKove s EC50 u rasponu 1-20 nM. Nadalje, DTK-5, DTK-2 i Met8-DTK-2 ispitani su u cAMP (reporter-gen-zasnovanoj) analizi te je stimulirano cAMP otputanje na nain doza-odgovor s EC50 vrijednostima od 197 nM, 1,06 M, odnosno 583 nM. Ovi podaci pokazuju da se DmGPCR5 vee za oba Gs (cAMP) i Gq (Ca+2)-upravljana signalna puta to je analogno signalnim putovima koji su navedeni za tahikinin receptore kraljenjaka. </span>As shown by GTP75 responses, DmGPCR5 (GenBank accession no. AX128628) when transiently expressed in CHO-10001A cells, was activated by drotachikinins (DTKs), namely DTK-1 (APTSSFIGMR-NH2) (SEQ ID NO: 169), Met8-DTK-2 (APLAFYGMR-NH2) (SEQ ID NO: 170), DTK-2 (APLAPYGLR-NH2) (SEQ ID NO: 171, DTK-3 (APTGFTGMR-NH2) (SEQ ID NO: 172), DTK -4 (APVNSFVGMR-NH2) (SEQ ID NO: 173) and DTK-5 (APNGFLGMR-NH2) (SEQ ID NO: 174). In dose-response experiments, DTK-1, Met8-DTK-2, DTK-3 and DTK-5 stimulated GTP-S binding with an EC50 in the 250-500 nM range and maximal stimulation was approximately 1.5-fold above baseline DTK-2 and DTK-4 are less potent judging by their EC50 which is in the low micromolar range. In a calcium mobilization assay (FLIPR), DmGPCR5 showed Ca+2 responses to the same DTKs with EC50 in the range 1-20 nM. Furthermore, DTK-5, DTK-2 and Met8-DTK-2 were tested in cAMP ( reporter-gene-based) analysis and cAMP release was stimulated in a dose-response manner with EC50 values of 197 nM, 1.06 M, or 583 nM. These data indicate that DmGPCR5 binds to both Gs (cAMP) and Gq (Ca+2)-operated signaling pathways analogous to the signaling pathways reported for vertebrate tachykinin receptors.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">DmGPCR6a (M811490) objelodanjen je kao PYY receptor: Li et al. (J. Biol. Chem., 1992,267, 9-12). Koritenjem GTPS analize, peptidi koji su navedeni u tablici 7 ispitani su pri 5 M, stimuliranim GTPS vezanjem (1,7 do 4 puta poveanje prema osnovnoj razini) za membrane iz CHO stanica koje su transficirane s DNA koja kodira DmGPCR6a. Valja naglasiti da, uz bateriju insektnih i C. elegans peptida koji aktiviraju ovaj receptor, takoer je naeno da je humani NPFF (FLFQPQRF-NH2 (sekv. br. 59)) ligand za DmGPCR6 (4-struko poveanje GTPS vezanja za 5 M NPFF). </span>DmGPCR6a (M811490) is disclosed as a PYY receptor: Li et al. (J. Biol. Chem., 1992, 267, 9-12). Using the GTPS assay, the peptides listed in Table 7 were tested at 5 M, stimulated GTPS binding (1.7- to 4-fold increase over baseline) to membranes from CHO cells transfected with DNA encoding DmGPCR6a. It should be noted that, in addition to a battery of insect and C. elegans peptides that activate this receptor, human NPFF (FLFQPQRF-NH2 (SEQ ID NO: 59)) was also found to be a ligand for DmGPCR6 (4-fold increase in GTPS binding by 5 M NPFF ).</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Dmgpcr6aL i Dmgpcr6bL su dvije splice inaice DmGPCR-a6a (M811490). Ovaj drugi naveden je kao PYY receptor od strane Li et al. (J. Biol. Chem., 1992, 267, 9-12). Mi smo nazvali i DmGPCR6aL i DmGPCR6bL, RF-amidnim receptorima jer oni prepoznaju samo peptide koji imaju Arg-Phe-NH2 (RFa) sekvenciju na C-terminalu. Peptidi koji ne vide ove DmGPCR-ove imaju drukije od RFa sekvencija na C-kraju (npr. SFa, QFa, YFa, RLa, DWa, RPa, HFa, LQa, SNa itd.). U analizi kalcijeve mobilizacije (FLIPR), Dmgpcr6aL i Dmgpcr6bL pokazali su vrlo jake Ca+2 odgovore na bateriju FaRPova koji su testirani na 10 M. Sekvencije koje su prikazane nie u tablici 7 predstavljaju sve identificirane aktivne FaRPove koji pripadaju razliitim specijama ukljuujui Drosophila, C. elegans, A. suum, Mollusca, P. redivivus, Trematoda, jastog, ovjek i pijavica. Jedini izuzetak za C-kraj RFamid pravila bio je peptid pGluDRDYRPLQF-NH2 (sekv. br. 120), iji C-terminal zavrava sekvencijom Gln-Phe-NH2 (QFa). Zanimljivo, i Dmgpcr6aL i Dmgpcr6bL takoer prepoznaju NPFF (FLFQPQRF-NH2 (sekv. br. 152)), peptid sisavaca s RFamid sekvencijom na C-terminalu (pozornost: u gore navedenim rezultatima p-Glu ili pQ odnose se na piroglutaminsku kiselinu). </span>Dmgpcr6aL and Dmgpcr6bL are two splice variants of DmGPCR-a6a (M811490). The latter was listed as a PYY receptor by Li et al. (J. Biol. Chem., 1992, 267, 9-12). We named both DmGPCR6aL and DmGPCR6bL RF-amide receptors because they only recognize peptides that have an Arg-Phe-NH2 (RFa) sequence at the C-terminal. Peptides that do not "see" these DmGPCRs have different than RFa sequences at the C-end (eg SFa, QFa, YFa, RLa, DWa, RPa, HFa, LQa, SNa, etc.). In a calcium mobilization assay (FLIPR), Dmgpcr6aL and Dmgpcr6bL showed very strong Ca+2 responses to a battery of FaRPs tested at 10 M. The sequences shown below in Table 7 represent all identified active FaRPs belonging to different species including Drosophila, C. elegans, A. suum, Mollusca, P. redivivus, Trematoda, lobster, human and leech. The only exception to the C-terminus of the "RFamide rule" was the peptide pGluDRDYRPLQF-NH2 (SEQ ID NO: 120), whose C-terminus ends with the sequence Gln-Phe-NH2 (QFa). Interestingly, both Dmgpcr6aL and Dmgpcr6bL also recognize NPFF (FLFQPQRF-NH2 (SEQ ID NO: 152)), a mammalian peptide with a C-terminal RFamide sequence (caution: in the above results p-Glu or pQ refer to pyroglutamic acid).</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Kao to se pokazalo pomou FLIPR analize, DmGPCR7 (GenBank pristupni br. AX128636) s prijelaznom ekspresijom u CHO-10001A stanicama, aktiviran je pomou leukokinina (LK-ovi) i srodnih peptida, dotino LK-I (DPAFNSWGa) (sekv. br. 175), LK-V (GSGFSSWGa) (sekv. br. 176), LK-VI (pGlu-SSFHSWGa) (sekv. br. 177), LK-VHI (GSAFYSWGa) (sekv. br. 178), kulekinin (NPFHSWGa) (sekv. br. 179), mekuac limnokinin (PSFHSWSa) (sekv. br. NO: 180) i Drosophila leukokinin-slini peptidi DLK-1 (NSVVLGKKQRFHSWGa) (sekv. br. 181), DLK-2 (pGlu-RFHSWGa) (sekv. br. 182) i DLK-2A (QRFHSWGa) (sekv. br. 183). DmGPCR7 bio je najbolje aktiviran pomou LK peptida koji imaju zajedniku C-terminalnu tetrapeptidnu sekvenciju, HSWGa. Tretmani s ovom skupinom peptida, koji obuhvaaju DLK-1, DLK-2, DLK-2a, LK-VI i kulekinin, rezultiraju vrlo snanim unutarstaninim otputanjem kalcija (EC50 je u rasponu od pikomolarnog do subnanomolarnog). Nasuprot tome, lokust LK-ovi s C-terminanim S/NSWGa (LK-I, LK-V) kao to je Lymnaea LK (sekv. br. 180), pokazali su se manje snanim (EC50 15-30 nM) i LK-VIII sa svojom YSWGa C-terminalnom sekvencijom bio je najmanje snaan u nizu (EC50 u rasponu 100-200 nM). Nisu uoeni nikakvi GTPS odgovori u membranama koje su prireene iz DmGPCR7/CHO stanica, to je indikacija Gqm-vezanog receptora. Prema tome, DmGPCR7 identificiran je kao kalcij-signalizirajui leukokininski receptor (najvjerojatnije Gq/n -vezan) i u paru je s drolukokininima kao njihov poznat ligand. </span>As shown by FLIPR analysis, DmGPCR7 (GenBank accession no. AX128636) transiently expressed in CHO-10001A cells was activated by leukokinins (LKs) and related peptides, namely LK-I (DPAFNSWGa) (SEQ ID NO. 175), LK-V (GSGFSSWGa) (SEQ ID NO: 176), LK-VI (pGlu-SSFHSWGa) (SEQ ID NO: 177), LK-VHI (GSAFYSWGa) (SEQ ID NO: 178), Kulekinin (NPFHSWGa ) (SEQ ID NO: 179), mollusc limnokinin (PSFHSWSa) (SEQ ID NO: 180) and Drosophila leucokinin-like peptides DLK-1 (NSVVLGKKQRFHSWGa) (SEQ ID NO: 181), DLK-2 (pGlu-RFHSWGa ) (SEQ ID NO: 182) and DLK-2A (QRFHSWGa) (SEQ ID NO: 183). DmGPCR7 was best activated by LK peptides sharing a common C-terminal tetrapeptide sequence, HSWGa. Treatments with this group of peptides, which include DLK-1, DLK-2, DLK-2a, LK-VI, and culekinin, result in very strong intracellular calcium release (EC50 ranges from picomolar to subnanomolar). In contrast, locust LKs with C-terminal S/NSWGa (LK-I, LK-V) such as Lymnaea LK (SEQ ID NO: 180), were shown to be less potent (EC50 15-30 nM) and LK -VIII with its YSWGa C-terminal sequence was the least potent of the series (EC50 in the range 100-200 nM). No GTPS responses were observed in membranes prepared from DmGPCR7/CHO cells, indicating a Gqm-coupled receptor. Therefore, DmGPCR7 has been identified as a calcium-signaling leucokinin receptor (most likely Gq/n -coupled) and is paired with droleukokinins as their known ligand.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Kao to su pokazali GTPS odgovori, DmGPCR8 (GenBank pristupni br. AX128638) s prijelaznom ekspresijom u CHO-10001A stanicama je aktiviran pomou Manduca sexta alatostatina-C (AST-C ili Manse-AC), (pGlu-VRFRQCYFNPISCF-OH) (sekv. br. 184) ili drostatina-C (DST-C), koji je takoer nazvan flatline peptid (FLT) (pGlu-VRYRQCYFNPISCF-OH) (sekv. br. 185). U eksperimentima s doznim odgovorom GTP-S-vezanja, uoeni su vrlo jaki AST-C i DST-C odgovori (EC50 je u donjem nanomolarnom podruju). Ove aktivnosti su u potpunosti uklonjene prethodnim tretiranjem stanica s pertussin toksinom to ukazuje na ukljuenost Gi/Go u receptorskom aktiviranju. U analizi izravne kalcijeve mobilizacije (FLIPR), DmGPCR8 nije pokazao nikakvu aktivnost kada je izazvan s AST-C ili DST-C. Meutim, jaka aktivnost otputanja kalcija prema DST-C uoena je u CHO-10001A stanicama koje su kotransficirane s DmGPCR-8 i kimernim G-proteinima Gqi5 ili Gqo5 (EC50 oko 30 nM). S druge strane, vezanje za Gqz5 bilo je manje uinkovito (EC50 244 nM) i nikakva mobilizacija kalcija nije uoena u stanicama koje su kotransficirane s DmGPCR-8 i Gqs5. Ovi rezultati pokazuju da je DmGPCR8 inhibitorski receptor u CHO stanicama koji se poeljno vee za Gi/Go tipa G-proteina. Ovi rezultati nesumnjivo identificiraju DmGPCR8 kao DST-C/FLT receptor. </span>As shown by GTPS responses, DmGPCR8 (GenBank accession no. AX128638) transiently expressed in CHO-10001A cells was activated by Manduca sexta alatostatin-C (AST-C or Manse-AC), (pGlu-VRFRQCYFNPISCF-OH) (seq .No. 184) or drostatin-C (DST-C), which is also called flatline peptide (FLT) (pGlu-VRYRQCYFNPISCF-OH) (SEQ ID NO: 185). In GTP-S-binding dose-response experiments, very strong AST-C and DST-C responses were observed (EC50 is in the lower nanomolar range). These activities were completely removed by pretreatment of cells with pertussin toxin, indicating the involvement of Gi/Go in receptor activation. In the direct calcium mobilization (FLIPR) assay, DmGPCR8 showed no activity when challenged with AST-C or DST-C. However, strong calcium-releasing activity towards DST-C was observed in CHO-10001A cells co-transfected with DmGPCR-8 and chimeric G-proteins Gqi5 or Gqo5 (EC50 about 30 nM). On the other hand, binding to Gqz5 was less efficient (EC50 244 nM) and no calcium mobilization was observed in cells cotransfected with DmGPCR-8 and Gqs5. These results indicate that DmGPCR8 is an inhibitory receptor in CHO cells that preferentially binds to G-protein-type Gi/Go. These results unequivocally identify DmGPCR8 as a DST-C/FLT receptor.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">DmGPCR9 sparen je sa FDDY(SO3H)GHLRF-NH2 (sekv. br. 157), temeljem njegova vrlo jakog signala u analizi mobilizacije kalcija (EC50 u niem nanomolarnom rasponu). injenica da nikakvi GTPS odgovori na ovaj peptid nisu uoeni s membranama koje su prireene iz CHO stanica koje su transficirane s DNA koja kodira DmGPCR9, pokazuje da je DmGPCR9 najvjerojatnije vezan sa Gq signalne putove. FDDY(SO3H)GHLRF-NH2 (sekv. br. 157) predstavlja Met7Leu7 analog prirodnog drosulfakinina-1 (DSK-1), FDDY(SO3H)GHMRF-NH2 (sekv. br. 159). Prema tome mi smo odredili DmGPCR9 receptor kao sulfakininski receptor. Ovo sparivanje je vrlo specifino jer je ak FDDYGHLRF-NH2 (sekv. br. 158), to je nesulfatirana protutea FDDY(SO3H)GHLRF-NH2 (sekv. br. 157), pokazao samo vrlo slab signal kalcija kada je ispitan s 10 mM ili nijedan od drugih 117 testiranih FaRP-ova i srodnih peptida nije pokazao nekakvu aktivnost bilo u FLIPR ili u GTPS analiza na DmGPCR9 receptoru. </span>DmGPCR9 was paired with FDDY(SO3H)GHLRF-NH2 (SEQ ID NO: 157), based on its very strong signal in the calcium mobilization assay (EC50 in the lower nanomolar range). The fact that no GTPS responses to this peptide were observed with membranes prepared from CHO cells transfected with DNA encoding DmGPCR9 indicates that DmGPCR9 is most likely linked to Gq signaling pathways. FDDY(SO3H)GHLRF-NH2 (SEQ ID NO: 157) represents the Met7Leu7 analog of native drosulfakinin-1 (DSK-1), FDDY(SO3H)GHMRF-NH2 (SEQ ID NO: 159). Accordingly, we designated the DmGPCR9 receptor as a sulfakinin receptor. This pairing is highly specific because even FDDYGHLRF-NH2 (SEQ ID NO: 158), which is the non-sulfated counterpart of FDDY(SO3H)GHLRF-NH2 (SEQ ID NO: 157), showed only a very weak calcium signal when probed with 10 mM or none of the other 117 FaRPs and related peptides tested showed any activity in either the FLIPR or the GTPS assay at the DmGPCR9 receptor.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Podudarni ligandi i njihovi odgovarajui receptori navedeni su nie u tablici 7. </span>Matched ligands and their corresponding receptors are listed below in Table 7.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Tablica 7 </span>Table 7</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">[image] [image] </span>[image] [image]</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">[image] </span>[image]</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Primjer 10: Kompeticijska analiza </span>Example 10: Competitive analysis</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Prireivanje monojodiranog peptide </span>Preparation of monoiodinated peptide</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Peptid je jodiran tipinim kloramin-T postupkom. U staklenu epruvetu od 2 ml dodano je 10 l 1 mM vodene otopine peptida, 10 l 0,1M (pH 7,99) natrij-fosfatnog pufera, 1,0 mCi [125I] natrijeva jodida i 5 ml 2 mg/ml otopine kloramina-T (u fosfatnom puferu). Smjesa je mijeana 60 sekundi i reakcija je zaustavljena dodatkom 25 ml 5 mg/ml otopine natrijeva metabisulfita u fosfatnom puferu. Smjesa je zatim podvrgnuta HPLC analizi injiciranjem na Vydac C18 (0,4515 cm) kolonu i gradijentnim odvajanjem. Koriteni gradijent je 70% i 30% B u vremenu nula do 20% i 80% B u vremenu 25 minuta (A = 0,1M NH4 acetat u vodi. B = 0,1M NH4 acetat u vodi 40%: CH3CN 60%, v:v.). Brzina protoka je 1,0 ml/minuta. Uzorci su sakupljeni u 0,25 ml pufera (0,1M natrij-fosfatni puffer s 0,5% albumina volovskog seruma, 0,1% Triton X100 i 0,05% Tween 20) u intervalima 30 sekundi od t=8 do t=20 minuta. Monojodirani peptid tipino eluira u vremenu t=11 minuta i prinos je priblino 100 mCi u 0,75 ml. </span>The peptide was iodinated by a typical chloramine-T procedure. 10 l of 1 mM aqueous peptide solution, 10 l of 0.1 M (pH 7.99) sodium phosphate buffer, 1.0 mCi [125I] sodium iodide and 5 ml of 2 mg/ml chloramine solution were added to a 2 ml glass tube. -T (in phosphate buffer). The mixture was stirred for 60 seconds and the reaction was stopped by the addition of 25 ml of a 5 mg/ml solution of sodium metabisulfite in phosphate buffer. The mixture was then subjected to HPLC analysis by injection onto a Vydac C18 (0.4515 cm) column and gradient separation. The gradient used is 70% and 30% B in time zero to 20% and 80% B in time 25 minutes (A = 0.1M NH4 acetate in water. B = 0.1M NH4 acetate in water 40%: CH3CN 60%, v: v.). The flow rate is 1.0 ml/minute. Samples were collected in 0.25 ml of buffer (0.1 M sodium phosphate buffer with 0.5% bovine serum albumin, 0.1% Triton X100 and 0.05% Tween 20) at 30 second intervals from t=8 to t = 20 minutes. The monoiodinated peptide typically elutes at time t=11 minutes and the yield is approximately 100 mCi in 0.75 ml.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Analiza vezanja </span>Binding analysis</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Ploa s 96 jaica koritena za analizu bila je Millipore Multiscreen filtracijska ploa (FB neprozirna 1,0 M stakleno vlakno tipa B, kat. br. MAFBNOB50). Millipore Multiscreen viestruko otporno otapalo (kat. br. MAVMO960R ) koriteno je u sprezi s ploama za filtriranje analize na zavretku. Svaki replikat je jedna jaica i ima volumen od 100 l koji sadri 5 g proteina (prireivanje prije opisano). Svaka ispitivana skupina sadri dva replikata. Za svaki ispitivani spoj, jedna skupina je izvrena samo s [125I]peptidom (za ukupno vezanje) i jedna s 1 M (ili kako je predvieno) koncentracijom ispitivanog spoja i [125I]peptidom (za nespecifino vezanje). Redoslijed dodavanja reagensa za svaki replikat je: pufer analize (20 mM HEPES, 10 mM MgCl2, 1% albumin volovskog seruma, pH 7,4) testni spoj (nainjen u puferu analize), [125I]peptid (u puferu analize) i membranska suspenzija (u puferu analize). Dodatak membranske suspenzije potie reakciju vezanja koja se vri tijekom 30 minuta na sobnoj temperature (22 C). Nakon 30 minuta inkubiranja svaka ploa je stavljena na filtracijsku pumpu i primijenjen je vakuum, proputanjem tekuine kroz filter (baeno) i hvatanjem proteina na filterima u svaku jaicu. Za ispiranje, otputen je vakuum i dodano je 200 l pufera analize u svaku jaicu nakon ega je ponovo primijenjen vakuum. Ovo ispiranje je ponovljeno jo dva puta (ukupno 3 ispiranje za svaki replikat). Nakon ispiranja, uklonjen je plastini pokrov na donjoj strani svake ploe i ploa je stavljena na dno zatvorene Microbeta brojake scintilacijske kasete (kat br. 1450-105). U svaku jaicu dodano je 25 l scintilanta i ploa je postavljena na rotacijsku mukalicu pri brzini 80 rpm tijekom 1 sat i zatim u mirovanju preko noi. Sljedeeg dana ploa je izmjerena na Microbeta scintilacijskom brojau. Prosjeno nespecifino vezanje oduzeto je od prosjenog ukupnog vezanja da se dobije specifino vezanje za standard (peptidamid) i uzorke. </span>The 96-well plate used for the assay was a Millipore Multiscreen filtration plate (FB opaque 1.0 M glass fiber type B, cat. no. MAFBNOB50). A Millipore Multiscreen Multi-Resistance Solvent (Cat. No. MAVMO960R ) was used in conjunction with filter plates for analysis at the end. Each replicate is a single well and has a volume of 100 l containing 5 g of protein (preparation described previously). Each tested group contains two replicates. For each test compound, one group was performed with [125I]peptide alone (for total binding) and one with 1 M (or as predicted) concentration of test compound and [125I]peptide (for nonspecific binding). The order of addition of reagents for each replicate is: assay buffer (20 mM HEPES, 10 mM MgCl2, 1% bovine serum albumin, pH 7.4) test compound (made in assay buffer), [125I]peptide (in assay buffer) and membrane suspension (in assay buffer). The addition of the membrane suspension promotes the binding reaction, which takes place during 30 minutes at room temperature (22 C). After 30 minutes of incubation, each plate was placed on a filtration pump and a vacuum was applied, passing the liquid through the filter (discarded) and trapping the proteins on the filters in each well. For washing, the vacuum was released and 200 l of assay buffer was added to each well, after which the vacuum was applied again. This wash was repeated two more times (a total of 3 washes for each replicate). After washing, the plastic cover on the underside of each plate was removed and the plate was placed on the bottom of a sealed Microbeta counting scintillation cassette (cat. no. 1450-105). 25 l of scintillant was added to each well and the plate was placed on a rotary shaker at 80 rpm for 1 hour and then left overnight. The next day, the plate was measured on a Microbeta scintillation counter. The average non-specific binding was subtracted from the average total binding to obtain the specific binding for the standard (peptidamide) and samples.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Kao to je poznato onima koji poznaju ovo podruje, mogue su brojne izmjene i modifikacije realizacija ovog izuma pri emu se ne udaljavamo od duha ovog izuma. Smatra se da sve takve inaice spadaju u doseg ovog izuma. </span>As is known to those skilled in the art, numerous changes and modifications are possible to the embodiments of this invention without departing from the spirit of this invention. All such variations are considered to be within the scope of the present invention.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Cjelokupni sadraj svake citirane publikacije je u cijelosti obuhvaen kao referenca. </span>The entire content of each cited publication is incorporated by reference in its entirety.</span>
  </p>
</description>
  </section>
      <div id="margin" class="style-scope patent-text">
        <div id="createHighlight" style="top: 0px;" class="style-scope patent-text" hidden="">
          <iron-icon id="comment" icon="editor:insert-comment" class="style-scope patent-text x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M20 2H4c-1.1 0-2 .9-2 2v12c0 1.1.9 2 2 2h14l4 4V4c0-1.1-.9-2-2-2zm-2 12H6v-2h12v2zm0-3H6V9h12v2zm0-3H6V6h12v2z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
        </div>
      </div>
    </div>
  </patent-text>
              </section>
            </div>
            <div class="flex flex-width style-scope patent-result">
              <section id="claims" class="style-scope patent-result">
                <h3 class="style-scope patent-result">
                  <div class="layout horizontal style-scope patent-result">
                    <div class="flex style-scope patent-result">Claims <span class="style-scope patent-result">(48)</span></div>
                    <div class="collapse style-scope patent-result">
                      Hide Dependent <iron-icon style="height: 15px; width: 15px;" icon="expand-less" class="style-scope patent-result x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M12 8l-6 6 1.41 1.41L12 10.83l4.59 4.58L18 14z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon><template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </div>
                  </div>
                  <div class="layout horizontal style-scope patent-result">
                    <div class="flex style-scope patent-result"></div>
                    <div class="righthead style-scope patent-result">translated from Croatian</div>
                  </div>
                </h3>
                <patent-text name="claims" enable-text-selection="" class="style-scope patent-result" style="padding-top: 0px;">
    
    
    <template is="dom-if" restamp="" class="style-scope patent-text"></template>
    <overlay-tooltip id="tooltip" class="style-scope patent-text" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      
    </div>
  </overlay-tooltip>
    <div class="layout horizontal style-scope patent-text">
      <section id="text" class="flex style-scope patent-text"><claims mxw-id="PCLM331734993" lang="EN" load-source="google" class="style-scope patent-text">
  <claim num="1" class="style-scope patent-text">
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">1. Metoda za identificiranje modulatora vezanja i/ili funkcije izmeu DmGPCR1 i DmGPCR1 vezujueg partnera, naznaena time to se sastoji od sljedeih stupnjeva:
(a) dovoenje u dodir DmGPCR1 vezujueg partnera i smjese koja sadri DmGPCR1 u prisutnosti ili u odsutnosti potencijalnog modulatorskog spoja;
(b) detektiranje vezanja izmeu DmGPCR1 vezujueg partnera i DmGPCR1; te
(c) odreivanje jesu li vezanje ili funkcija u prisutnosti navedenog potencijalnog modulatorskog spoja poveani ili smanjeni u usporedbi s vezanjem ili funkcijom u odsutnosti navedenog potencijalnog modulatorskog spoja,
pri emu navedeni DmGPCR1 vezujui partner ima sekvenciju s bar 70% sekvencijske identinosti sa sekvencijom koja je odabrana iz skupa kojega sainjavaju sekv. br. 186 i sekv. br. 187.</span>1. A method for identifying a modulator of binding and/or function between DmGPCR1 and DmGPCR1 binding partner, characterized in that it consists of the following steps:
(a) contacting a DmGPCR1 binding partner and a mixture comprising DmGPCR1 in the presence or absence of a potential modulatory compound;
(b) detecting binding between a DmGPCR1 binding partner and DmGPCR1; you
(c) determining whether binding or function in the presence of said potential modulatory compound is increased or decreased compared to binding or function in the absence of said potential modulatory compound,
wherein said DmGPCR1 binding partner has a sequence with at least 70% sequence identity with a sequence that is selected from the set consisting of seq. no. 186 et seq. no. 187.</span>
    </claim-text>
  </claim>
  <claim num="2" class="style-scope patent-text">
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">2. Metoda prema zahtjevu 1, naznaena time to navedeni DmGPCR1 vezujui partner ima sekvenciju s bar 80% sekvencijske identinosti sa sekvencijom koja je odabrana iz skupa kojega sainjavaju sekv. br. 186 i sekv. br. 187.</span>2. The method according to claim 1, characterized in that said DmGPCR1 binding partner has a sequence with at least 80% sequence identity with a sequence that is selected from the set consisting of seq. no. 186 et seq. no. 187.</span>
    </claim-text>
  </claim>
  <claim num="3" class="style-scope patent-text">
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">3. Metoda prema zahtjevu 1, naznaena time to navedeni DmGPCR1 vezujui partner ima sekvenciju s bar 95% sekvencijske identinosti sa sekvencijom koja je odabrana iz skupa kojega sainjavaju sekv. br. 186 i sekv. br. 187.</span>3. The method according to claim 1, characterized in that said DmGPCR1 binding partner has a sequence with at least 95% sequence identity with a sequence that is selected from the set consisting of seq. no. 186 et seq. no. 187.</span>
    </claim-text>
  </claim>
  <claim num="4" class="style-scope patent-text">
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">4. Metoda prema zahtjevu 1, naznaena time to navedeni DmGPCR1 vezujui partner ima sekvenciju koja je odabrana iz skupa kojega sainjavaju sekv. br. 186 i sekv. br. 187.</span>4. The method according to claim 1, characterized in that said DmGPCR1 binding partner has a sequence that is selected from the set consisting of seq. no. 186 et seq. no. 187.</span>
    </claim-text>
  </claim>
  <claim num="5" class="style-scope patent-text">
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">5. Metoda za kontroliranje populacije insekata, naznaena time to obuhvaa primjenu vezujueg partnera ili modulatora DmGPCR1 polinukleotida ili polipeptida insektu da se modificira ekspresija ili aktivnost DmGPCR1, pri emu navedeni vezujui partner ima sekvenciju s bar 70% sekvencijske identinosti sa sekvencijom koja je odabrana iz skupa kojega sainjavaju sekv. br. 186 i sekv. br. 187.</span>5. A method for controlling an insect population, characterized in that it comprises the administration of a binding partner or modulator DmGPCR1 polynucleotide or polypeptide to an insect to modify the expression or activity of DmGPCR1, wherein said binding partner has a sequence with at least 70% sequence identity to a sequence selected from the set which consists of seq. no. 186 et seq. no. 187.</span>
    </claim-text>
  </claim>
  <claim num="6" class="style-scope patent-text">
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">6. Metoda prema zahtjevu 5, naznaena time to navedeni vezujui partner ima sekvenciju s bar 80% identinosti sa sekvencijom koja je odabrana iz skupa kojega sainjavaju sekv. br. 186 i sekv. br. 187.</span>6. The method according to claim 5, characterized in that said binding partner has a sequence with at least 80% identity with a sequence that is selected from the set consisting of seq. no. 186 et seq. no. 187.</span>
    </claim-text>
  </claim>
  <claim num="7" class="style-scope patent-text">
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">7. Metoda prema zahtjevu 5, naznaena time to navedeni vezujui partner ima sekvenciju s bar 95% sekvencijske identinosti sa sekvencijom koja je odabrana iz skupa kojega sainjavaju sekv. br. 186 i sekv. br. 187.</span>7. The method according to claim 5, characterized in that said binding partner has a sequence with at least 95% sequence identity with a sequence that is selected from the set consisting of seq. no. 186 et seq. no. 187.</span>
    </claim-text>
  </claim>
  <claim num="8" class="style-scope patent-text">
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">8. Metoda prema zahtjevu 5, naznaena time to navedeni vezujui partner ima sekvenciju koja je odabrana iz skupa kojega sainjavaju sekv. br. 186 i sekv. br. 187.</span>8. The method according to claim 5, characterized in that said binding partner has a sequence that is selected from the set consisting of seq. no. 186 et seq. no. 187.</span>
    </claim-text>
  </claim>
  <claim num="9" class="style-scope patent-text">
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">9. Metoda prema zahtjevu 5, naznaena time to je insekt odabran iz skupa kojega sainjavaju muha, vona muha, krpelj, buha, ui, grinje i ohar.</span>9. The method according to claim 5, characterized in that the insect is selected from the group consisting of flies, fruit flies, ticks, fleas, lice, mites and cockroaches.</span>
    </claim-text>
  </claim>
  <claim num="10" class="style-scope patent-text">
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">10. Metoda za tretiranje ili prevenciju bolesti ili stanja koje je izazvano ektoparazitom u subjekta, naznaena time to obuhvaa primjenu navedenom subjektu terapijski uinkovite koliine DmGPCR1 vezujueg partnera, pri emu navedeni DmGPCR1 vezujui partner ima sekvenciju s bar 70% identinosti sa sekvencijom koja je odabrana iz skupa kojega sainjavaju sekv. br. 186 i sekv. br. 187.</span>10. A method for treating or preventing a disease or condition caused by an ectoparasite in a subject, characterized in that it comprises administering to said subject a therapeutically effective amount of a DmGPCR1 binding partner, wherein said DmGPCR1 binding partner has a sequence with at least 70% identity to a sequence selected from set consisting of seq. no. 186 et seq. no. 187.</span>
    </claim-text>
  </claim>
  <claim num="11" class="style-scope patent-text">
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">11. Metoda prema zahtjevu 10, naznaena time to navedeni DmGPCR1 vezujui partner ima sekvenciju s bar 80% sekvencijske identinosti sa sekvencijom koja je odabrana iz skupa kojega sainjavaju sekv. br. 186 i sekv. br. 187.</span>11. The method according to claim 10, characterized in that said DmGPCR1 binding partner has a sequence with at least 80% sequence identity with a sequence that is selected from the set consisting of seq. no. 186 et seq. no. 187.</span>
    </claim-text>
  </claim>
  <claim num="12" class="style-scope patent-text">
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">12. Metoda prema zahtjevu 10, naznaena time to navedeni DmGPCR1 vezujui partner ima sekvenciju s bar 95% sekvencijske identinosti sa sekvencijom koja je odabrana iz skupa kojega sainjavaju sekv. br. 186 i sekv. br. 187.</span>12. The method according to claim 10, characterized in that said DmGPCR1 binding partner has a sequence with at least 95% sequence identity with a sequence that is selected from the set consisting of seq. no. 186 et seq. no. 187.</span>
    </claim-text>
  </claim>
  <claim num="13" class="style-scope patent-text">
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">13. Metoda prema zahtjevu 10, naznaena time to navedeni DmGPCR1 vezujui partner ima sekvenciju koja je odabrana iz skupa kojega sainjavaju sekv. br. 186 i sekv. br. 187.</span>13. The method according to claim 10, characterized in that said DmGPCR1 binding partner has a sequence that is selected from the set consisting of seq. no. 186 et seq. no. 187.</span>
    </claim-text>
  </claim>
  <claim num="14" class="style-scope patent-text">
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">14. Metoda prema zahtjevu 10, naznaena time to je navedeni subjekt  ovjek.</span>14. The method according to claim 10, characterized in that the said subject is a human.</span>
    </claim-text>
  </claim>
  <claim num="15" class="style-scope patent-text">
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">15. Metoda vezanja DmGPCR s DmGPCR vezujuim partnerom, naznaena time to obuhvaa sljedee stupnjeve:
dovoenje u dodir smjese koja sadri DmGPCR s DmGPCR vezujuim partnerom; i
omoguavanje da se navedeni DmGPCR vezujui partner vee za navedeni DmGPCR.</span>15. A method of binding DmGPCR with a DmGPCR binding partner, characterized in that it comprises the following steps:
contacting the mixture containing the DmGPCR with the DmGPCR binding partner; and
allowing said DmGPCR binding partner to bind to said DmGPCR.</span>
    </claim-text>
  </claim>
  <claim num="16" class="style-scope patent-text">
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">16. Metoda prema zahtjevu 15, naznaena time to je navedeni DmGPCR - DmGPCR5 (sekv. br. 9).</span>16. The method according to claim 15, characterized in that said DmGPCR is DmGPCR5 (seq. no. 9).</span>
    </claim-text>
  </claim>
  <claim num="17" class="style-scope patent-text">
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">17. Metoda prema zahtjevu 16, naznaena time to je navedeni DmGPCR vezujui partner drotahikinin (DTK).</span>17. The method according to claim 16, characterized in that said DmGPCR binding partner is drotachikinin (DTK).</span>
    </claim-text>
  </claim>
  <claim num="18" class="style-scope patent-text">
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">18. Metoda prema zahtjevu 17, naznaena time to navedeni drotahikinin ima sekvenciju s bar 80% sekvencijske identinosti sa sekvencijom koja je odabrana iz skupa kojega sainjavaju DTK-1 (sekv. br. 169), Met8-DTK-2 (sekv. br. 170), DTK-2 (sekv. br. 171), DTK-3 (sekv. br. 172), DTK-4 (sekv. br. 173) i DTK-5 (sekv. br. 174).</span>18. The method according to claim 17, characterized in that said drotachikinin has a sequence with at least 80% sequence identity with a sequence selected from the group consisting of DTK-1 (SEQ ID NO: 169), Met8-DTK-2 (SEQ ID NO: 170), DTK-2 (SEQ ID NO: 171), DTK-3 (SEQ ID NO: 172), DTK-4 (SEQ ID NO: 173) and DTK-5 (SEQ ID NO: 174).</span>
    </claim-text>
  </claim>
  <claim num="19" class="style-scope patent-text">
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">19. Metoda prema zahtjevu 15, naznaena time to je navedeni DmGPCR - DmGPCR7 (sekv. br. 17).</span>19. The method according to claim 15, characterized in that said DmGPCR is DmGPCR7 (seq. no. 17).</span>
    </claim-text>
  </claim>
  <claim num="20" class="style-scope patent-text">
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">20. Metoda prema zahtjevu 19, naznaena time to je navedeni DmGPCR vezujui partner leukokinin (LK).</span>20. The method according to claim 19, characterized in that said DmGPCR binding partner is leukokinin (LK).</span>
    </claim-text>
  </claim>
  <claim num="21" class="style-scope patent-text">
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">21. Metoda prema zahtjevu 20, naznaena time to navedeni leukokinin ima sekvenciju s bar 80% sekvencijske identinosti sa sekvencijom koja je odabrana iz skupa kojega sainjavaju LK-I (sekv. br. 175), LK-V (sekv. br. 176), LK-VI (sekv. br. 177) i LK-VIII (sekv. br. 178), kulekinin (sekv. br. 179), Lymnaea limnokinin (sekv. br. 180), DLK-1 (sekv. br. 181), DLK-2 (sekv. br. 182), DLK-2a (sekv. br. 183).</span>21. The method according to claim 20, characterized in that said leucokinin has a sequence with at least 80% sequence identity with a sequence selected from the group consisting of LK-I (SEQ ID NO: 175), LK-V (SEQ ID NO: 176) , LK-VI (SEQ ID NO: 177) and LK-VIII (SEQ ID NO: 178), Kulekinin (SEQ ID NO: 179), Lymnaea limnokinin (SEQ ID NO: 180), DLK-1 (SEQ ID NO: 180). 181), DLK-2 (SEQ ID NO: 182), DLK-2a (SEQ ID NO: 183).</span>
    </claim-text>
  </claim>
  <claim num="22" class="style-scope patent-text">
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">22. Metoda prema zahtjevu 15, naznaena time to je navedeni DmGPCR - DmGPCR8 (sekv. br. 19).</span>22. The method according to claim 15, characterized in that said DmGPCR is DmGPCR8 (seq. no. 19).</span>
    </claim-text>
  </claim>
  <claim num="23" class="style-scope patent-text">
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">23. Metoda prema zahtjevu 22, naznaena time to je navedeni DmGPCR vezujui partner alatostatin.</span>23. The method according to claim 22, characterized in that said DmGPCR binding partner is allatostatin.</span>
    </claim-text>
  </claim>
  <claim num="24" class="style-scope patent-text">
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">24. Metoda prema zahtjevu 23, naznaena time to navedeni alatostatin ima sekvenciju s bar 80% sekvencijske identinosti sa sekvencijom koja je odabrana iz skupa kojega sainjavaju AST-C (sekv. br. 184) ili DST-G (sekv. br. 185).</span>24. The method according to claim 23, characterized in that said allatostatin has a sequence with at least 80% sequence identity with a sequence selected from the group consisting of AST-C (SEQ ID NO: 184) or DST-G (SEQ ID NO: 185) .</span>
    </claim-text>
  </claim>
  <claim num="25" class="style-scope patent-text">
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">25. Metoda za identificiranje modulatora vezanja i/ili funkcije izmeu DmGPCR i DmGPCR vezujueg partnera, naznaena time to obuhvaa sljedee stupnjeve:
dovoenje u dodir DmGPCR vezujueg partnera i smjese koja sadri DmGPCR u prisutnosti i odsutnosti potencijalnog modulatorikog spoja;
detektiranje vezanja izmeu DmGPCR vezujueg partnera i DmGPCR; te
odreivanje je vezanje u prisutnosti navedenog potencijalnog modulatorskog spoja poveano ili smanjeno u usporedbi s vezanjem u odsutnosti navedenog potencijalnog modulatorskog spoja,
odreivanje je li funkcija u prisutnosti navedenog potencijalnog modulatorskog spoja poveana ili smanjena u odnosu na funkciju u odsutnosti navedenog potencijalnog modulatorskog spoja, pri emu je navedeni DmGPCR - DmGPCR5 (sekv. br. 9).</span>25. A method for identifying a modulator of binding and/or function between a DmGPCR and a DmGPCR binding partner, characterized in that it comprises the following steps:
contacting the DmGPCR binding partner and the mixture containing the DmGPCR in the presence and absence of a potential modulatory compound;
detecting binding between the DmGPCR binding partner and the DmGPCR; you
determining whether binding in the presence of said potential modulatory compound is increased or decreased compared to binding in the absence of said potential modulatory compound,
determining whether the function in the presence of said potential modulatory compound is increased or decreased compared to the function in the absence of said potential modulatory compound, wherein said DmGPCR - DmGPCR5 (SEQ ID NO: 9).</span>
    </claim-text>
  </claim>
  <claim num="26" class="style-scope patent-text">
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">26. Metoda prema zahtjevu 25, naznaena time to je navedeni DmGPCR vezujui partner drotahikinin.</span>26. The method according to claim 25, characterized in that said DmGPCR binding partner is drotachikinin.</span>
    </claim-text>
  </claim>
  <claim num="27" class="style-scope patent-text">
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">27. Metoda prema zahtjevu 26, naznaena time to navedeni drotahikinin ima sekvenciju s bar 80% sekvencijske identinosti sa sekvencijom koja je odabrana iz skupa kojega sainjavaju DTK-1 (sekv. br. 169), Met8-DTK-2 (sekv. br. 170), DTK-2 (sekv. br. 171), DTK-3 (sekv. br. 172), DTK-4 (sekv. br. 173) i DTK-5 (sekv. br. 174).</span>27. The method according to claim 26, characterized in that said drotachikinin has a sequence with at least 80% sequence identity with a sequence selected from the group consisting of DTK-1 (SEQ ID No. 169), Met8-DTK-2 (SEQ ID NO. 170), DTK-2 (SEQ ID NO: 171), DTK-3 (SEQ ID NO: 172), DTK-4 (SEQ ID NO: 173) and DTK-5 (SEQ ID NO: 174).</span>
    </claim-text>
  </claim>
  <claim num="28" class="style-scope patent-text">
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">28. Metoda za identificiranje modulatora vezanja i/ili funkcije izmeu DmGPCR i DmGPCR vezujueg partnera, naznaena time to obuhvaa sljedee stupnjeve:
dovoenje u dodir DmGPCR vezujueg partnera i smjese koja sadri DmGPCR u prisutnosti ili odsutnosti potencijalnog modulatorskog spoja;
detektiranje vezanja izmeu DmGPCR vezujueg partnera i DmGPCR; te
odreivanje je li vezanje u prisutnosti navedenog potencijalnog modulatorskog spoja poveano ili smanjeno u usporedbi s vezanjem u odsutnosti navedenog potencijalnog modulatorskog spoja,
odreivanje je li funkcija u prisutnosti navedenog potencijalnog modulatorskog spoja poveana ili smanjena u odnosu na funkciju u odsutnosti navedenog potencijalnog modulatorskog spoja, pri emu je navedeni DmGPCR - DmGPCR7 (sekv. br. 17).</span>28. A method for identifying a modulator of binding and/or function between a DmGPCR and a DmGPCR binding partner, characterized in that it comprises the following steps:
contacting the DmGPCR binding partner and the mixture containing the DmGPCR in the presence or absence of a potential modulatory compound;
detecting binding between the DmGPCR binding partner and the DmGPCR; you
determining whether binding in the presence of said potential modulatory compound is increased or decreased compared to binding in the absence of said potential modulatory compound,
determining whether the function in the presence of said potential modulatory compound is increased or decreased compared to the function in the absence of said potential modulatory compound, wherein said DmGPCR - DmGPCR7 (SEQ ID NO: 17).</span>
    </claim-text>
  </claim>
  <claim num="29" class="style-scope patent-text">
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">29. Metoda prema zahtjevu 28, naznaena time to je navedeni DmGPCR vezujui partner leukokinin.</span>29. The method according to claim 28, characterized in that said DmGPCR binding partner is leucokinin.</span>
    </claim-text>
  </claim>
  <claim num="30" class="style-scope patent-text">
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">30. Metoda prema zahtjevu 29, naznaena time to navedeni leukokinin ima sekvenciju s bar 80% sekvencijske identinosti sa sekvencijom koja je odabrana iz skupa kojega sainjavaju LK-I (sekv. br. 175), LK-V (sekv. br. 176), LK-VI (sekv. br. 177), LK-VIII (sekv. br. 178), kulekinin (sekv. br. 179), Lymnaea limnokinin (sekv. br. 180), DLK-1 (sekv. br. 181), DLK-2 (sekv. br. 182) i DLK-2a (sekv. br. 183).</span>30. The method according to claim 29, characterized in that said leucokinin has a sequence with at least 80% sequence identity with a sequence selected from the group consisting of LK-I (SEQ ID NO: 175), LK-V (SEQ ID NO: 176) , LK-VI (SEQ ID NO: 177), LK-VIII (SEQ ID NO: 178), Kulekinin (SEQ ID NO: 179), Lymnaea limnokinin (SEQ ID NO: 180), DLK-1 (SEQ ID NO: 180). 181), DLK-2 (SEQ ID NO: 182) and DLK-2a (SEQ ID NO: 183).</span>
    </claim-text>
  </claim>
  <claim num="31" class="style-scope patent-text">
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">31. Metoda za identificiranje modulatora vezanja i/ili funkcije izmeu DmGPCR i DmGPCR vezujueg partnera, naznaena time to obuhvaa sljedee stupnjeve:
dovoenje u dodir DmGPCR vezujueg partnera i smjese koja sadri DmGPCR u prisutnosti ili u odsutnosti potencijalnog modulatorskog spoja;
detektiranje vezanja izmeu DmGPCR vezujueg partnera i DmGPCR;
te
odreivanje je li vezanje u prisutnosti navedenog potencijalnog modulatorskog spoja poveano ili smanjeno u odnosu na vezanje u odsutnosti navedenog potencijalnog modulatorskog spoja,
odreivanje je li funkcija u prisutnosti navedenog potencijalnog modulatorskog spoja poveana ili smanjena u odnosu na funkciju u odsutnosti navedenog potencijalnog modulatorskog spoja, pri emu je navedeni DmGPCR - DmGPCR8 (sekv. br. 19).</span>31. A method for identifying a modulator of binding and/or function between a DmGPCR and a DmGPCR binding partner, characterized in that it comprises the following steps:
contacting the DmGPCR binding partner and the mixture containing the DmGPCR in the presence or absence of a potential modulatory compound;
detecting binding between the DmGPCR binding partner and the DmGPCR;
you
determining whether binding in the presence of said potential modulatory compound is increased or decreased relative to binding in the absence of said potential modulatory compound,
determining whether the function in the presence of said potential modulatory compound is increased or decreased relative to the function in the absence of said potential modulatory compound, wherein said DmGPCR - DmGPCR8 (SEQ ID NO: 19).</span>
    </claim-text>
  </claim>
  <claim num="32" class="style-scope patent-text">
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">32. Metoda prema zahtjevu 31, naznaena time to je navedeni DmGPCR vezujui partner alatostatin.</span>32. The method according to claim 31, characterized in that said DmGPCR binding partner is allatostatin.</span>
    </claim-text>
  </claim>
  <claim num="33" class="style-scope patent-text">
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">33. Metoda prema zahtjevu 32, naznaena time to navedeni alatostatin ima sekvenciju s bar 80% sekvencijske identinosti sa sekvencijom koja je odabrana iz skupa kojega sainjavaju AST-C (sekv. br. 184) ili DST-C (sekv. br. 185).</span>33. The method according to claim 32, characterized in that said allatostatin has a sequence with at least 80% sequence identity with a sequence selected from the group consisting of AST-C (SEQ ID NO: 184) or DST-C (SEQ ID NO: 185) .</span>
    </claim-text>
  </claim>
  <claim num="34" class="style-scope patent-text">
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">34. Metoda za kontroliranje populacije insekata, naznaena time to obuhvaa primjenu vezujueg partnera ili modulatora DmGPCR polinukleotida ili polipeptida insektu da se modificira ekspresija ili aktivnost DmGPCR.</span>34. A method for controlling an insect population, characterized in that it comprises administering a binding partner or modulator of a DmGPCR polynucleotide or polypeptide to an insect to modify the expression or activity of a DmGPCR.</span>
    </claim-text>
  </claim>
  <claim num="35" class="style-scope patent-text">
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">35. Metoda prema zahtjevu 34, naznaena time to je insekt odabran iz skupa kojega sainjavaju muha, vona muha, krpelj, buha, ui, grinje i ohar.</span>35. The method according to claim 34, characterized in that the insect is selected from the group consisting of flies, fruit flies, ticks, fleas, lice, mites and cockroaches.</span>
    </claim-text>
  </claim>
  <claim num="36" class="style-scope patent-text">
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">36. Metoda prema zahtjevu 34, naznaena time to je navedeni DmGPCR vezujui partner drotahikinin.</span>36. The method according to claim 34, characterized in that said DmGPCR binding partner is drotachikinin.</span>
    </claim-text>
  </claim>
  <claim num="37" class="style-scope patent-text">
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">37. Metoda prema zahtjevu 36, naznaena time to navedeni drotahikinin ima sekvenciju s bar 80% sekvencijske identinosti sa sekvencijom koja je odabrana iz skupa kojega sainjavaju DTK-1 (sekv. br. 169), Met8-DTK-2 (sekv. br. 170), DTK-2 (sekv. br. 171), DTK-3 (sekv. br. 172), DTK-4 (sekv. br. 173) i DTK-5 (sekv. br. 174).</span>37. The method according to claim 36, characterized in that said drotachikinin has a sequence with at least 80% sequence identity with a sequence selected from the group consisting of DTK-1 (SEQ ID NO: 169), Met8-DTK-2 (SEQ ID NO: 170), DTK-2 (SEQ ID NO: 171), DTK-3 (SEQ ID NO: 172), DTK-4 (SEQ ID NO: 173) and DTK-5 (SEQ ID NO: 174).</span>
    </claim-text>
  </claim>
  <claim num="38" class="style-scope patent-text">
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">38. Metoda prema zahtjevu 34, naznaena time to je navedeni DmGPCR vezujui partner leukokinin.</span>38. The method according to claim 34, characterized in that said DmGPCR binding partner is leucokinin.</span>
    </claim-text>
  </claim>
  <claim num="39" class="style-scope patent-text">
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">39. Metoda prema zahtjevu 38, naznaena time to navedeni leukokinin ima sekvenciju s bar 80% sekvencijske identinosti sa sekvencijom koja je odabrana iz skupa kojega sainjavaju LK-I (sekv. br. 175), LK-V (sekv. br. 176), LK-VI (sekv. br. 177), LK-VIII (sekv. br. 178), kulekinin (sekv. br. 179), Lymnaea limnokinin (sekv. br. 180), DLK-1 (sekv. br. 181), DLK-2 (sekv. br. 182) i DLK-2a (sekv. br. 183).</span>39. The method according to claim 38, characterized in that said leucokinin has a sequence with at least 80% sequence identity with a sequence selected from the group consisting of LK-I (SEQ ID NO: 175), LK-V (SEQ ID NO: 176) , LK-VI (SEQ ID NO: 177), LK-VIII (SEQ ID NO: 178), Kulekinin (SEQ ID NO: 179), Lymnaea limnokinin (SEQ ID NO: 180), DLK-1 (SEQ ID NO: 180). 181), DLK-2 (SEQ ID NO: 182) and DLK-2a (SEQ ID NO: 183).</span>
    </claim-text>
  </claim>
  <claim num="40" class="style-scope patent-text">
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">40. Metoda prema zahtjevu 34, naznaena time to je navedeni DmGPCR vezujui partner alatostatin.</span>40. The method according to claim 34, characterized in that said DmGPCR binding partner is allatostatin.</span>
    </claim-text>
  </claim>
  <claim num="41" class="style-scope patent-text">
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">41. Metoda prema zahtjevu 40, naznaena time to navedeni alatostatin ima sekvenciju s bar 80% sekvencijske identinosti sa sekvencijom koja je odabrana iz skupa kojega sainjavaju AST-C (sekv. br. 184) ili DST-C (sekv. br. 185).</span>41. The method according to claim 40, characterized in that said allatostatin has a sequence with at least 80% sequence identity with a sequence selected from the group consisting of AST-C (SEQ ID NO: 184) or DST-C (SEQ ID NO: 185) .</span>
    </claim-text>
  </claim>
  <claim num="42" class="style-scope patent-text">
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">42. Metoda prema zahtjevu 34, naznaena time to je navedeni DmGPCR modulator neko anti-DmGPCR antitijelo, DmGPCR protivsmisleni polinukleotid ili niskomolekularni nepeptidni mimetik.</span>42. The method according to claim 34, characterized in that said DmGPCR modulator is an anti-DmGPCR antibody, DmGPCR antisense polynucleotide or low molecular weight non-peptide mimetic.</span>
    </claim-text>
  </claim>
  <claim num="43" class="style-scope patent-text">
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">43. Metoda prema zahtjevu 42, naznaena time to je navedeni niskomolekularni nepeptidni mimetik neki agonist ili antagonist.</span>43. The method according to claim 42, characterized in that said low molecular weight non-peptide mimetic is an agonist or antagonist.</span>
    </claim-text>
  </claim>
  <claim num="44" class="style-scope patent-text">
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">44. Metoda za tretiranje ili prevenciju bolesti ili stanja izazvanog pomou ektoparazita u subjekta, naznaena time to obuhvaa primjenu navedenom subjektu terapijski uinkovite koliine DmGPCR vezujueg partnera.</span>44. A method for treating or preventing a disease or condition caused by an ectoparasite in a subject, comprising administering to said subject a therapeutically effective amount of a DmGPCR binding partner.</span>
    </claim-text>
  </claim>
  <claim num="45" class="style-scope patent-text">
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">45. Metoda prema zahtjevu 44, naznaena time to je navedeni subjekt kuna ivotinja, stoka, konj ili ovjek.</span>45. The method according to claim 44, characterized in that said subject is a domestic animal, livestock, horse or human.</span>
    </claim-text>
  </claim>
  <claim num="46" class="style-scope patent-text">
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">46. Metoda prema zahtjevu 44, naznaena time to je navedeni vezujui partner drotahikinin, leukokinin, alatostatin ili neko antitijelo.</span>46. The method according to claim 44, characterized in that said binding partner is drotachykinin, leucokinin, allatostatin or an antibody.</span>
    </claim-text>
  </claim>
  <claim num="47" class="style-scope patent-text">
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">47. Metoda prema zahtjevu 44, naznaena time to je navedeni vezujui partner neko antitijelo.</span>47. The method according to claim 44, characterized in that said binding partner is an antibody.</span>
    </claim-text>
  </claim>
  <claim num="48" class="style-scope patent-text">
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">48. Metoda prema zahtjevu 47, naznaena time to je navedeno antitijelo kimerno antitijelo, CDR-usaeno antitijelo, humano antitijelo ili humanizirano antitijelo.</span>48. The method according to claim 47, characterized in that said antibody is a chimeric antibody, a CDR-grafted antibody, a human antibody or a humanized antibody.</span>
    </claim-text>
  </claim>
</claims>
  </section>
      <div id="margin" class="style-scope patent-text">
        <div id="createHighlight" style="top: 0px;" class="style-scope patent-text" hidden="">
          <iron-icon id="comment" icon="editor:insert-comment" class="style-scope patent-text x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M20 2H4c-1.1 0-2 .9-2 2v12c0 1.1.9 2 2 2h14l4 4V4c0-1.1-.9-2-2-2zm-2 12H6v-2h12v2zm0-3H6V9h12v2zm0-3H6V6h12v2z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
        </div>
      </div>
    </div>
  </patent-text>
              </section>
            </div>
          </div>

          <div class="footer style-scope patent-result">

            
            <h3 id="patentCitations" class="scroll-target style-scope patent-result">
              Patent Citations (20)
              <a href="#title" class="style-scope patent-result"><iron-icon icon="arrow-drop-up" class="style-scope patent-result x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M7 14l5-5 5 5z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon></a>
            </h3>
            <div class="responsive-table style-scope patent-result">
              <div class="table style-scope patent-result">
                <div class="thead style-scope patent-result">
                  <div class="tr style-scope patent-result">
                    <span class="th style-scope patent-result">Publication number</span>
                    <span class="th style-scope patent-result">Priority date</span>
                    <span class="th style-scope patent-result">Publication date</span>
                    <span class="th style-scope patent-result">Assignee</span>
                    <span class="th style-scope patent-result">Title</span>
                  </div>
                </div>
                <div class="tbody style-scope patent-result">
                  <template is="dom-repeat" class="style-scope patent-result"></template>

                  <div class="tr style-scope patent-result">
                    <span class="td famheader style-scope patent-result">Family To Family Citations</span>
                    <span class="td famheader style-scope patent-result"></span>
                    <span class="td famheader style-scope patent-result"></span>
                    <span class="td famheader style-scope patent-result"></span>
                    <span class="td famheader style-scope patent-result"></span>
                  </div>

                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                      <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US4343940A/en"><a id="link" href="#" class="style-scope state-modifier">US4343940A</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">1979-02-13</span>
                    <span class="td style-scope patent-result">1982-08-10</span>
                    <span class="td style-scope patent-result">Mead Johnson &amp; Company</span>
                    <span class="td style-scope patent-result">Anti-tumor quinazoline compounds 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                      <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/EP0031237B1/en"><a id="link" href="#" class="style-scope state-modifier">EP0031237B1</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">1979-12-19</span>
                    <span class="td style-scope patent-result">1984-10-17</span>
                    <span class="td style-scope patent-result">National Research Development Corporation</span>
                    <span class="td style-scope patent-result">Quinazoline derivatives, processes for their preparation, compositions containing them and their use as anti-cancer agents 
     </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                      <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US4399216A/en"><a id="link" href="#" class="style-scope state-modifier">US4399216A</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">1980-02-25</span>
                    <span class="td style-scope patent-result">1983-08-16</span>
                    <span class="td style-scope patent-result">The Trustees Of Columbia University</span>
                    <span class="td style-scope patent-result">Processes for inserting DNA into eucaryotic cells and for producing proteinaceous materials 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                      <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US4879236A/en"><a id="link" href="#" class="style-scope state-modifier">US4879236A</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">1984-05-16</span>
                    <span class="td style-scope patent-result">1989-11-07</span>
                    <span class="td style-scope patent-result">The Texas A&amp;M University System</span>
                    <span class="td style-scope patent-result">Method for producing a recombinant baculovirus expression vector 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                      <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US4683195A/en"><a id="link" href="#" class="style-scope state-modifier">US4683195A</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">1986-01-30</span>
                    <span class="td style-scope patent-result">1987-07-28</span>
                    <span class="td style-scope patent-result">Cetus Corporation</span>
                    <span class="td style-scope patent-result">Process for amplifying, detecting, and/or-cloning nucleic acid sequences 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                      <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US4683202A/en"><a id="link" href="#" class="style-scope state-modifier">US4683202A</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">1985-03-28</span>
                    <span class="td style-scope patent-result">1987-07-28</span>
                    <span class="td style-scope patent-result">Cetus Corporation</span>
                    <span class="td style-scope patent-result">Process for amplifying nucleic acid sequences 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                      <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/DE3611194A1/en"><a id="link" href="#" class="style-scope state-modifier">DE3611194A1</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">1986-04-04</span>
                    <span class="td style-scope patent-result">1987-10-08</span>
                    <span class="td style-scope patent-result">Bayer Ag</span>
                    <span class="td style-scope patent-result">
  CANCEROSTATIC AGENT
 
     </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                      <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US5217999A/en"><a id="link" href="#" class="style-scope state-modifier">US5217999A</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">1987-12-24</span>
                    <span class="td style-scope patent-result">1993-06-08</span>
                    <span class="td style-scope patent-result">Yissum Research Development Company Of The Hebrew University Of Jerusalem</span>
                    <span class="td style-scope patent-result">Styryl compounds which inhibit EGF receptor protein tyrosine kinase 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                      <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/DE3920029C2/en"><a id="link" href="#" class="style-scope state-modifier">DE3920029C2</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">1988-06-30</span>
                    <span class="td style-scope patent-result">1999-05-20</span>
                    <span class="td style-scope patent-result">Clariant Finance Bvi Ltd</span>
                    <span class="td style-scope patent-result">
  Dyes for coloring plastics
 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                      <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US5302606A/en"><a id="link" href="#" class="style-scope state-modifier">US5302606A</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">1990-04-16</span>
                    <span class="td style-scope patent-result">1994-04-12</span>
                    <span class="td style-scope patent-result">Rhone-Poulenc Rorer Pharmaceuticals Inc.</span>
                    <span class="td style-scope patent-result">Styryl-substituted pyridyl compounds which inhibit EGF receptor tyrosine kinase 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                      <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US5344776A/en"><a id="link" href="#" class="style-scope state-modifier">US5344776A</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">1991-03-28</span>
                    <span class="td style-scope patent-result">1994-09-06</span>
                    <span class="td style-scope patent-result">The United States Of America As Represented By The Department Of Health And Human Services</span>
                    <span class="td style-scope patent-result">DNA encoding an insect octopamine receptor 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                      <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US5330992A/en"><a id="link" href="#" class="style-scope state-modifier">US5330992A</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">1992-10-23</span>
                    <span class="td style-scope patent-result">1994-07-19</span>
                    <span class="td style-scope patent-result">Sterling Winthrop Inc.</span>
                    <span class="td style-scope patent-result">1-cyclopropyl-4-pyridyl-quinolinones 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                      <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/GB9305120D0/en"><a id="link" href="#" class="style-scope state-modifier">GB9305120D0</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">1993-03-12</span>
                    <span class="td style-scope patent-result">1993-04-28</span>
                    <span class="td style-scope patent-result">Queen Mary &amp; Westfield College</span>
                    <span class="td style-scope patent-result">Neuropeptides and their use as insecticides 
     </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                      <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US5585277A/en"><a id="link" href="#" class="style-scope state-modifier">US5585277A</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">1993-06-21</span>
                    <span class="td style-scope patent-result">1996-12-17</span>
                    <span class="td style-scope patent-result">Scriptgen Pharmaceuticals, Inc.</span>
                    <span class="td style-scope patent-result">Screening method for identifying ligands for target proteins 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                      <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US5880141A/en"><a id="link" href="#" class="style-scope state-modifier">US5880141A</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">1995-06-07</span>
                    <span class="td style-scope patent-result">1999-03-09</span>
                    <span class="td style-scope patent-result">Sugen, Inc.</span>
                    <span class="td style-scope patent-result">Benzylidene-Z-indoline compounds for the treatment of disease 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                      <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US6703491B1/en"><a id="link" href="#" class="style-scope state-modifier">US6703491B1</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">1999-03-17</span>
                    <span class="td style-scope patent-result">2004-03-09</span>
                    <span class="td style-scope patent-result">Exelixis, Inc.</span>
                    <span class="td style-scope patent-result">Drosophila sequences 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                      <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/AU4463000A/en"><a id="link" href="#" class="style-scope state-modifier">AU4463000A</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">1999-04-21</span>
                    <span class="td style-scope patent-result">2000-11-02</span>
                    <span class="td style-scope patent-result">University Of Florida Research Foundation, Inc.</span>
                    <span class="td style-scope patent-result">Neuropeptides and their use for pest control 
     </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                      <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/CA2388599A1/en"><a id="link" href="#" class="style-scope state-modifier">CA2388599A1</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">1999-10-22</span>
                    <span class="td style-scope patent-result">2001-05-03</span>
                    <span class="td style-scope patent-result">Pharmacia &amp; Upjohn Company</span>
                    <span class="td style-scope patent-result">Drosophila g protein coupled receptors, nucleic acids, and methods related to the same 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                      <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US20030162223A1/en"><a id="link" href="#" class="style-scope state-modifier">US20030162223A1</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">1999-10-22</span>
                    <span class="td style-scope patent-result">2003-08-28</span>
                    <span class="td style-scope patent-result">Lowery David E.</span>
                    <span class="td style-scope patent-result">Drosophila G protein coupled receptors, nucleic acids, and methods related to the same 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                      <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/WO2001071042A2/en"><a id="link" href="#" class="style-scope state-modifier">WO2001071042A2</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2000-03-23</span>
                    <span class="td style-scope patent-result">2001-09-27</span>
                    <span class="td style-scope patent-result">Pe Corporation (Ny)</span>
                    <span class="td style-scope patent-result">Detection kits, such as nucleic acid arrays, for detecting the expression of 10,000 or more drosophila genes and uses thereof 
     </span>
                  </div>
                  <template is="dom-repeat" class="style-scope patent-result"></template>

                </div>
              </div>
            </div>
            <div style="padding: 10px 12px; font-size: 11px;" class="style-scope patent-result">* Cited by examiner,  Cited by third party</div>
            <template is="dom-if" class="style-scope patent-result"></template>

            <template is="dom-if" class="style-scope patent-result"></template>

            
            <h3 id="citedBy" class="scroll-target style-scope patent-result">
              Cited By (5)
              <a href="#title" class="style-scope patent-result"><iron-icon icon="arrow-drop-up" class="style-scope patent-result x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M7 14l5-5 5 5z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon></a>
            </h3>
            <div class="responsive-table style-scope patent-result">
              <div class="table style-scope patent-result">
                <div class="thead style-scope patent-result">
                  <div class="tr style-scope patent-result">
                    <span class="th style-scope patent-result">Publication number</span>
                    <span class="th style-scope patent-result">Priority date</span>
                    <span class="th style-scope patent-result">Publication date</span>
                    <span class="th style-scope patent-result">Assignee</span>
                    <span class="th style-scope patent-result">Title</span>
                  </div>
                </div>
                <div class="tbody style-scope patent-result">
                  <template is="dom-repeat" class="style-scope patent-result"></template>

                  <div class="tr style-scope patent-result">
                    <span class="td famheader style-scope patent-result">Family To Family Citations</span>
                    <span class="td famheader style-scope patent-result"></span>
                    <span class="td famheader style-scope patent-result"></span>
                    <span class="td famheader style-scope patent-result"></span>
                    <span class="td famheader style-scope patent-result"></span>
                  </div>

                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                      <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US20030162223A1/en"><a id="link" href="#" class="style-scope state-modifier">US20030162223A1</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">1999-10-22</span>
                    <span class="td style-scope patent-result">2003-08-28</span>
                    <span class="td style-scope patent-result">Lowery David E.</span>
                    <span class="td style-scope patent-result">Drosophila G protein coupled receptors, nucleic acids, and methods related to the same 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                      <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/CA2388599A1/en"><a id="link" href="#" class="style-scope state-modifier">CA2388599A1</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">1999-10-22</span>
                    <span class="td style-scope patent-result">2001-05-03</span>
                    <span class="td style-scope patent-result">Pharmacia &amp; Upjohn Company</span>
                    <span class="td style-scope patent-result">Drosophila g protein coupled receptors, nucleic acids, and methods related to the same 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                      <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/GB0918579D0/en"><a id="link" href="#" class="style-scope state-modifier">GB0918579D0</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2009-10-22</span>
                    <span class="td style-scope patent-result">2009-12-09</span>
                    <span class="td style-scope patent-result">Imp Innovations Ltd</span>
                    <span class="td style-scope patent-result">Gadd45beta targeting agents 
     </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                      <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/PL218313B1/en"><a id="link" href="#" class="style-scope state-modifier">PL218313B1</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2010-10-28</span>
                    <span class="td style-scope patent-result">2014-11-28</span>
                    <span class="td style-scope patent-result">Inst Immunologii I Terapii Dowiadczalnej Pan</span>
                    <span class="td style-scope patent-result">Competitive method for obtaining bacteriophage formulations 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                      <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/KR101461426B1/en"><a id="link" href="#" class="style-scope state-modifier">KR101461426B1</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2013-08-28</span>
                    <span class="td style-scope patent-result">2014-11-13</span>
                    <span class="td style-scope patent-result"> </span>
                    <span class="td style-scope patent-result">Screening method of pesticidal materials 
       </span>
                  </div>
                  <template is="dom-repeat" class="style-scope patent-result"></template>
                </div>

              </div>
            </div>
            <div style="padding: 10px 12px; font-size: 11px;" class="style-scope patent-result">* Cited by examiner,  Cited by third party,  Family to family citation</div>
            <template is="dom-if" class="style-scope patent-result"></template>

            
            <h3 id="similarDocuments" class="scroll-target style-scope patent-result">
              Similar Documents
              <a href="#title" class="style-scope patent-result"><iron-icon icon="arrow-drop-up" class="style-scope patent-result x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M7 14l5-5 5 5z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon></a>
            </h3>
            <div class="responsive-table style-scope patent-result">
              <div class="table style-scope patent-result">
                <div class="thead style-scope patent-result">
                  <div class="tr style-scope patent-result">
                    <span class="th style-scope patent-result">Publication</span>
                    <span class="th style-scope patent-result">Publication Date</span>
                    <span class="th style-scope patent-result">Title</span>
                  </div>
                </div>
                <div class="tbody style-scope patent-result">
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      
                        <template is="dom-if" class="style-scope patent-result"></template>
                        
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US20050112660A1/en"><a id="link" href="#" class="style-scope state-modifier">US20050112660A1</a></state-modifier>
                        <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2005-05-26</span>
                    <span class="td style-scope patent-result">Novel G protein-coupled receptors 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      
                        <template is="dom-if" class="style-scope patent-result"></template>
                        
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/AU5787501A/en"><a id="link" href="#" class="style-scope state-modifier">AU5787501A</a></state-modifier>
                        <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2001-07-09</span>
                    <span class="td style-scope patent-result">Novel G protein-coupled receptors 
     </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      
                        <template is="dom-if" class="style-scope patent-result"></template>
                        
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/AU2005248981B2/en"><a id="link" href="#" class="style-scope state-modifier">AU2005248981B2</a></state-modifier>
                        <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2008-02-14</span>
                    <span class="td style-scope patent-result">Drosophila G protein coupled receptors, nucleic acids, and methods relating to the same 
     </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      
                        <template is="dom-if" class="style-scope patent-result"></template>
                        
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/HRP20050117A2/en"><a id="link" href="#" class="style-scope state-modifier">HRP20050117A2</a></state-modifier>
                        <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2005-10-31</span>
                    <span class="td style-scope patent-result">Drosophila g protein coupled receptors, nucleic acids, and methods related to the same 
     </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      
                        <template is="dom-if" class="style-scope patent-result"></template>
                        
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US7364866B2/en"><a id="link" href="#" class="style-scope state-modifier">US7364866B2</a></state-modifier>
                        <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2008-04-29</span>
                    <span class="td style-scope patent-result">Drosophila G protein coupled receptors, nucleic acids, and methods related to the same 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      
                        <template is="dom-if" class="style-scope patent-result"></template>
                        
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/EP1303601A2/en"><a id="link" href="#" class="style-scope state-modifier">EP1303601A2</a></state-modifier>
                        <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2003-04-23</span>
                    <span class="td style-scope patent-result">G protein-coupled receptors 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      
                        <template is="dom-if" class="style-scope patent-result"></template>
                        
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/WO2001068858A2/en"><a id="link" href="#" class="style-scope state-modifier">WO2001068858A2</a></state-modifier>
                        <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2001-09-20</span>
                    <span class="td style-scope patent-result">Human g protein-coupled receptors 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      
                        <template is="dom-if" class="style-scope patent-result"></template>
                        
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US20030170779A1/en"><a id="link" href="#" class="style-scope state-modifier">US20030170779A1</a></state-modifier>
                        <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2003-09-11</span>
                    <span class="td style-scope patent-result">Novel G protein-coupled receptors 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      
                        <template is="dom-if" class="style-scope patent-result"></template>
                        
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US20050214790A1/en"><a id="link" href="#" class="style-scope state-modifier">US20050214790A1</a></state-modifier>
                        <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2005-09-29</span>
                    <span class="td style-scope patent-result">Novel G protein-coupled receptors 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      
                        <template is="dom-if" class="style-scope patent-result"></template>
                        
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US20020052021A1/en"><a id="link" href="#" class="style-scope state-modifier">US20020052021A1</a></state-modifier>
                        <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2002-05-02</span>
                    <span class="td style-scope patent-result">Novel G protein-coupled receptors 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      
                        <template is="dom-if" class="style-scope patent-result"></template>
                        
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US20030032019A1/en"><a id="link" href="#" class="style-scope state-modifier">US20030032019A1</a></state-modifier>
                        <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2003-02-13</span>
                    <span class="td style-scope patent-result">Novel G protein-coupled receptors 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      
                        <template is="dom-if" class="style-scope patent-result"></template>
                        
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US20020137132A1/en"><a id="link" href="#" class="style-scope state-modifier">US20020137132A1</a></state-modifier>
                        <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2002-09-26</span>
                    <span class="td style-scope patent-result">Novel G protein-coupled receptors 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      
                        <template is="dom-if" class="style-scope patent-result"></template>
                        
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US20030175857A1/en"><a id="link" href="#" class="style-scope state-modifier">US20030175857A1</a></state-modifier>
                        <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2003-09-18</span>
                    <span class="td style-scope patent-result">Novel G protein-coupled receptor 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      
                        <template is="dom-if" class="style-scope patent-result"></template>
                        
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US20040105846A1/en"><a id="link" href="#" class="style-scope state-modifier">US20040105846A1</a></state-modifier>
                        <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2004-06-03</span>
                    <span class="td style-scope patent-result">Polynucleotides encoding G protein-coupled receptors 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      
                        <template is="dom-if" class="style-scope patent-result"></template>
                        
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US20020198368A1/en"><a id="link" href="#" class="style-scope state-modifier">US20020198368A1</a></state-modifier>
                        <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2002-12-26</span>
                    <span class="td style-scope patent-result">Novel G protein-coupled receptors 
       </span>
                  </div>
                  <template is="dom-repeat" class="style-scope patent-result"></template>
                </div>
              </div>
            </div>
            <template is="dom-if" class="style-scope patent-result"></template>

            
            <h3 id="relatedApplications" class="scroll-target style-scope patent-result">Priority And Related Applications</h3>

            <template is="dom-if" class="style-scope patent-result"></template>

            <template is="dom-if" class="style-scope patent-result"></template>

            <template is="dom-if" class="style-scope patent-result"></template>

            
            <h3 id="appsClaimingPriority" class="scroll-target style-scope patent-result">
              Applications Claiming Priority (3)
              <a href="#title" class="style-scope patent-result"><iron-icon icon="arrow-drop-up" class="style-scope patent-result x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M7 14l5-5 5 5z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon></a>
            </h3>
            <div class="responsive-table style-scope patent-result">
              <div class="table style-scope patent-result">
                <div class="thead style-scope patent-result">
                  <div class="tr style-scope patent-result">
                    <span class="th style-scope patent-result">Application</span>
                    <span class="th style-scope patent-result">Filing date</span>
                    <span class="th style-scope patent-result">Title</span>
                  </div>
                </div>
                <div class="tbody style-scope patent-result">
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US7364866B2/en"><a id="link" href="#" class="style-scope state-modifier">US10/213,821</a></state-modifier><template is="dom-if" class="style-scope patent-result"></template>
                        <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td nowrap style-scope patent-result">2002-08-06</span>
                    <span class="td style-scope patent-result">Drosophila G protein coupled receptors, nucleic acids, and methods related to the same 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US20030162223A1/en"><a id="link" href="#" class="style-scope state-modifier">US10/283,423</a></state-modifier><template is="dom-if" class="style-scope patent-result"></template>
                        <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td nowrap style-scope patent-result">2002-10-30</span>
                    <span class="td style-scope patent-result">Drosophila G protein coupled receptors, nucleic acids, and methods related to the same 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/WO2004013306A2/en"><a id="link" href="#" class="style-scope state-modifier">PCT/US2003/024488</a></state-modifier><template is="dom-if" class="style-scope patent-result"></template>
                        <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td nowrap style-scope patent-result">2003-08-06</span>
                    <span class="td style-scope patent-result">Drosophila g protein coupled receptors, nucleic acids, and methods related to the same 
       </span>
                  </div>
                  <template is="dom-repeat" class="style-scope patent-result"></template>
                </div>
              </div>
            </div>
            <template is="dom-if" class="style-scope patent-result"></template>

            <template is="dom-if" class="style-scope patent-result"></template>

            
            <h3 id="legalEvents" class="scroll-target style-scope patent-result">
              Legal Events
              <a href="#title" class="style-scope patent-result"><iron-icon icon="arrow-drop-up" class="style-scope patent-result x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M7 14l5-5 5 5z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon></a>
            </h3>
            <div class="responsive-table style-scope patent-result">
              <div class="table style-scope patent-result">
                <div class="thead style-scope patent-result">
                  <div class="tr style-scope patent-result">
                    <span class="th style-scope patent-result">Date</span>
                    <span class="th style-scope patent-result">Code</span>
                    <span class="th style-scope patent-result">Title</span>
                    <span class="th style-scope patent-result">Description</span>
                  </div>
                </div>
                <div class="tbody style-scope patent-result">
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">2005-10-31</span>
                    <span class="td nowrap style-scope patent-result">A1OB</span>
                    <span class="td nowrap style-scope patent-result">Publication of a patent application</span>
                    <span class="td style-scope patent-result">
                      <template is="dom-repeat" as="attr" class="style-scope patent-result"></template>
                    </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">2006-04-18</span>
                    <span class="td nowrap style-scope patent-result">AIPI</span>
                    <span class="td nowrap style-scope patent-result">Request for the grant of a patent on the basis of a substantive examination of a patent application</span>
                    <span class="td style-scope patent-result">
                      <template is="dom-repeat" as="attr" class="style-scope patent-result"></template>
                    </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">2008-07-31</span>
                    <span class="td nowrap style-scope patent-result">ODRP</span>
                    <span class="td nowrap style-scope patent-result">Renewal fee for the maintenance of a patent</span>
                    <span class="td style-scope patent-result">
                      
                      <p class="style-scope patent-result"><strong class="style-scope patent-result">Payment date</strong>: 20080731</p>
                      
                      <p class="style-scope patent-result"><strong class="style-scope patent-result">Year of fee payment</strong>: 6</p>
                      <template is="dom-repeat" as="attr" class="style-scope patent-result"></template>
                    </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">2010-03-03</span>
                    <span class="td nowrap style-scope patent-result">OBST</span>
                    <span class="td nowrap style-scope patent-result">Application withdrawn</span>
                    <span class="td style-scope patent-result">
                      <template is="dom-repeat" as="attr" class="style-scope patent-result"></template>
                    </span>
                  </div>
                  <template is="dom-repeat" class="style-scope patent-result"></template>
                </div>
              </div>
            </div>
            <template is="dom-if" class="style-scope patent-result"></template>

            
              <h3 id="concepts" class="scroll-target layout horizontal style-scope patent-result">
                <div class="flex style-scope patent-result">Concepts</div>
                <a href="#title" class="style-scope patent-result"><iron-icon icon="arrow-drop-up" class="style-scope patent-result x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M7 14l5-5 5 5z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon></a>
              </h3>
              <div class="layout horizontal style-scope patent-result">
                <div class="flex style-scope patent-result">
                  <span id="conceptsWarning" class="righthead tooltip-hint style-scope patent-result">machine-extracted</span>
                  <overlay-tooltip positioning-target="#conceptsWarning" direction="bottom" class="style-scope patent-result" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      The concepts are extracted by a computer and may be incomplete or incorrect.
    </div>
  </overlay-tooltip>
                </div>
                <span class="headerButton style-scope patent-result"><a href="#" target="_blank" style="display: inline-block;" class="style-scope patent-result"><iron-icon icon="icons:file-download" class="style-scope patent-result x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M19 9h-4V3H9v6H5l7 7 7-7zM5 18v2h14v-2H5z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon> Download</a></span>
                <span class="style-scope patent-result">
                  <dropdown-menu multi="true" label="Filter table" show-label="true" change-action="RESULT_CONCEPT_MENU" class="style-scope patent-result">
    
    <span class="label style-scope dropdown-menu">
      <span class="style-scope dropdown-menu">Filter table</span>
      <span id="selected" class="style-scope dropdown-menu" hidden="">
        <span style="margin: 0 4px 0 2px" class="style-scope dropdown-menu"></span>
        <span class="style-scope dropdown-menu"></span>
      </span>
      <iron-icon icon="icons:arrow-drop-down" class="style-scope dropdown-menu x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M7 10l5 5 5-5z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon><iron-icon icon="icons:arrow-drop-up" class="style-scope dropdown-menu x-scope iron-icon-0" hidden=""><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M7 14l5-5 5 5z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
    </span>
    <iron-dropdown allow-outside-scroll="" horizontal-align="left" vertical-align="top" class="style-scope dropdown-menu" aria-disabled="false" aria-hidden="true" style="outline: none; display: none;">
    

    <div id="contentWrapper" class="style-scope iron-dropdown">
      <div class="dropdown-content style-scope dropdown-menu" slot="dropdown-content" id="menu" multi="">
        <div class="style-scope dropdown-menu">
          
            <div class="item style-scope dropdown-menu">Anatomy</div>
          
            <div class="item style-scope dropdown-menu">Chemical class</div>
          
            <div class="item style-scope dropdown-menu">Chemical compound</div>
          
            <div class="item style-scope dropdown-menu">Chemical group</div>
          
            <div class="item style-scope dropdown-menu">Diseases</div>
          
            <div class="item style-scope dropdown-menu">Drugs</div>
          
            <div class="item style-scope dropdown-menu">Effects</div>
          
            <div class="item style-scope dropdown-menu">Human genes</div>
          
            <div class="item style-scope dropdown-menu">Inorganic materials</div>
          
            <div class="item style-scope dropdown-menu">Methods</div>
          
            <div class="item style-scope dropdown-menu">Natural products</div>
          
            <div class="item style-scope dropdown-menu">Nutrition</div>
          
            <div class="item style-scope dropdown-menu">Polymers</div>
          
            <div class="item style-scope dropdown-menu">Proteins</div>
          
            <div class="item style-scope dropdown-menu">Species</div>
          
            <div class="item style-scope dropdown-menu">Substances</div>
          
            <div class="item style-scope dropdown-menu">Toxicity</div>
          <template is="dom-repeat" class="style-scope dropdown-menu"></template>
        </div>
      </div>
    </div>
  </iron-dropdown>
  </dropdown-menu>
                </span>
              </div>
              <div class="responsive-table style-scope patent-result">
                <div class="table style-scope patent-result">
                  <div class="thead style-scope patent-result">
                    <div class="tr style-scope patent-result">
                      <span class="th style-scope patent-result">Name</span>
                      <span class="th style-scope patent-result">Image</span>
                      <span class="th style-scope patent-result">Sections</span>
                      <span class="th style-scope patent-result">Count</span>
                      <span class="th style-scope patent-result">Query match</span>
                    </div>
                  </div>
                  <div class="tbody style-scope patent-result">
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Methods style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">method</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">title,claims,description</span>
                        <span class="td nowrap style-scope patent-result">203</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Human genes style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">G-Protein-Coupled Receptors</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">title,description</span>
                        <span class="td nowrap style-scope patent-result">65</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Chemical class style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">nucleic acids</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">title,description</span>
                        <span class="td nowrap style-scope patent-result">50</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Human genes style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">nucleic acids</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">title,description</span>
                        <span class="td nowrap style-scope patent-result">48</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Proteins style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">nucleic acids</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">title,description</span>
                        <span class="td nowrap style-scope patent-result">48</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Proteins style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">Drosophila G</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">title,description</span>
                        <span class="td nowrap style-scope patent-result">2</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Proteins style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">processed proteins &amp; peptides</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">249</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Effects style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">binding</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">234</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Methods style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">binding</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">231</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Human genes style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">processed proteins &amp; peptides</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">206</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Polymers style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">polypeptide</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">176</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Chemical class style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">compounds</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">169</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Effects style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">effects</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">107</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Human genes style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">polynucleotide</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">101</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Proteins style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">polynucleotide</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">101</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Substances style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">polynucleotide</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">101</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Effects style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">gene expression</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">96</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Species style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">Hexapoda</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">60</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Substances style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">mixture</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">43</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Substances style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">agonist</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">34</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Diseases style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">diseases, disorders, signs and symptoms</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">28</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Species style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">Homo sapiens</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">24</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Effects style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">increasing effect</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">23</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Proteins style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">allatostatin</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">22</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Diseases style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">disease</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">21</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Chemical group style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">(2s)-n-[(2s)-1-[[(2s)-1-[[(2s)-1-[[2-[[(2s)-1-[[2-[[(2s)-1-amino-4-methyl-1-oxopentan-2-yl]amino]-2-oxoethyl]amino]-1-oxo-3-phenylpropan-2-yl]amino]-2-oxoethyl]amino]-3-(4-hydroxyphenyl)-1-oxopropan-2-yl]amino]-4-methyl-1-oxopentan-2-yl]amino]-5-(diaminom</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">20</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Species style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">Metazoa</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">20</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Substances style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">antagonist</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">20</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Species style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">ectoparasite</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">17</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Effects style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">anti-sense effect</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">12</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Proteins style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">Leucokinin</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">11</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Effects style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">decreasing effect</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">11</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Human genes style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">Protein delta homolog 1</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">10</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Proteins style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">Protein delta homolog 1</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">8</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Species style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">Acari</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">7</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Species style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">Diptera</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">7</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Human genes style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">Protein delta homolog 2</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">7</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Proteins style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">Protein delta homolog 2</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">7</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Species style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">Blattodea</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">6</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Species style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">Phthiraptera</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">5</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Species style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">Lymnaea</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">4</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Species style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">Tephritidae</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">4</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Species style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">Equus caballus</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">2</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Species style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">Siphonaptera</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims</span>
                        <span class="td nowrap style-scope patent-result">2</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Species style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">livestock</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims</span>
                        <span class="td nowrap style-scope patent-result">1</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    <template is="dom-repeat" initial-count="25" class="style-scope patent-result"></template>
                    
                      <div class$="tr layout horizontal conceptDomain" class="style-scope patent-result">
                        <a href="#" class="td flex style-scope patent-result">
                          Show all concepts from the description section
                        </a>
                      </div>
                    <template is="dom-if" class="style-scope patent-result"></template>
                  </div>
                </div>
              </div>
            <template is="dom-if" class="style-scope patent-result"></template>

            <div id="notices" class="vertical layout center style-scope patent-result">
              
                <span class="style-scope patent-result">Data provided by IFI CLAIMS Patent Services</span>
              <template is="dom-if" class="style-scope patent-result"></template>
            </div>
          </div>
          <div class="bottombar style-scope patent-result">
            <bottom-bar class="style-scope patent-result"><template is="dom-if" class="style-scope bottom-bar"></template>
</bottom-bar>
          </div>
        </div>
      </div>
      <template is="dom-if" class="style-scope patent-result"></template>
    </div>
  </patent-result>
    <template is="dom-if" restamp="" class="style-scope result-container"></template>
    <template is="dom-if" restamp="" class="style-scope result-container"></template>
    <template is="dom-if" restamp="" class="style-scope result-container"></template>
  </result-container>
        </div>
      </div>
    
        </div>
      </div>
      <search-footer class="style-scope search-ui">
    
    

    <div id="footer" class="vertical layout center style-scope search-footer">
        <span class="links style-scope search-footer">
          <overlay-tooltip positioning-target="#footerAbout" direction="top" class="style-scope search-footer" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      Learn more about data coverage, search syntax and other features.
    </div>
  </overlay-tooltip>
          <a id="footerAbout" data-proto="ABOUT" href="https://support.google.com/faqs/answer/6390996" target="_blank" class="style-scope search-footer">About</a>
          <overlay-tooltip positioning-target="#footerFeedback" direction="top" class="style-scope search-footer" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      Send feedback about technical issues or feature requests for Google Patents.
    </div>
  </overlay-tooltip>
          <a id="footerFeedback" data-proto="SEND_FEEDBACK" href="#" onclick="sendFeedback();" class="style-scope search-footer">Send Feedback</a>
          <a data-proto="PUBLIC_DATASETS" href="https://console.cloud.google.com/marketplace/product/google_patents_public_datasets/google-patents-public-data" target="_blank" class="style-scope search-footer">Public Datasets</a>
          <a data-proto="TERMS" href="https://www.google.com/policies/terms/" target="_blank" class="style-scope search-footer">Terms</a>
          <a data-proto="PRIVACY_POLICY" href="https://www.google.com/privacy/privacy-policy.html" target="blank" class="style-scope search-footer">Privacy Policy</a>
          <a data-proto="SEARCH_HELP" href="https://support.google.com/websearch/" target="_blank" class="style-scope search-footer">Help</a>
        </span>
    </div>
  </search-footer>
    </div>

  </search-ui>
  </search-result>
    <template is="dom-if" restamp="" class="style-scope search-app"></template>
    <template is="dom-if" restamp="" class="style-scope search-app"></template>
    <template is="dom-if" restamp="" class="style-scope search-app"></template>
    <paper-toast id="toast" duration="3000" class="style-scope search-app x-scope paper-toast-0" aria-hidden="true" style="outline: none; display: none;">
    

    <span id="label" class="style-scope paper-toast"></span>
    
      <div id="noticeContent" class="style-scope search-app"></div>
    
  </paper-toast>
  </search-app>
    <script type="text/javascript" src="//www.gstatic.com/feedback/api.js"></script>
    <script type="text/javascript" src="//www.gstatic.com/feedback/js/help/prod/service/lazy.min.js"></script>
    <script type="text/javascript">
      if (window.help && window.help.service) {
        helpApi = window.help.service.Lazy.create(0, {apiKey: 'AIzaSyDTEI_0tLX4varJ7bwK8aT-eOI5qr3BmyI', locale: 'en-US'});
        window.requestedSurveys = new Set();
        window.requestSurvey = function(triggerId) {
          if (window.requestedSurveys.has(triggerId)) {
            return;
          }
          window.requestedSurveys.add(triggerId);
          helpApi.requestSurvey({
            triggerId: triggerId,
            enableTestingMode: false,
            callback: (requestSurveyCallbackParam) => {
              if (!requestSurveyCallbackParam.surveyData) {
                return;
              }
              helpApi.presentSurvey({
                productData: {
                  productVersion: window.version,
                  customData: {
                    "experiments": "",
                  },
                },
                surveyData: requestSurveyCallbackParam.surveyData,
                colorScheme: 1,
                customZIndex: 10000,
              });
            }
          });
        };

        window.requestSurvey('YXTwAsvoW0kedxbuTdH0RArc9VhT');
      }
    </script>
    <script src="/sw/null_loader.js"></script>
  

<iframe id="apiproxyddf475d63cd8173a637a287407211e3c7614fb050.2094967346" name="apiproxyddf475d63cd8173a637a287407211e3c7614fb050.2094967346" style="width: 1px; height: 1px; position: absolute; top: -100px; display: none;" src="https://scone-pa.clients6.google.com/static/proxy.html?usegapi=1&amp;jsh=m%3B%2F_%2Fscs%2Fabc-static%2F_%2Fjs%2Fk%3Dgapi.lb.en.IKZeRvoAYNY.O%2Fam%3DAAAQ%2Fd%3D1%2Frs%3DAHpOoo-SMWTzMRJrTty6iE5dL_aWGYOnuw%2Fm%3D__features__#parent=https%3A%2F%2Fpatents.google.com&amp;rpctoken=666265722" tabindex="-1" aria-hidden="true"></iframe><iron-a11y-announcer>
    
    <div class="style-scope iron-a11y-announcer" aria-live="polite"></div>
  </iron-a11y-announcer></body></html>