<html lang="en"><head><!-- Shady DOM styles for custom-style --><!-- Shady DOM styles for dom-template --><!-- Shady DOM styles for dom-repeat --><!-- Shady DOM styles for array-selector --><!-- Shady DOM styles for dom-if --><!-- Shady DOM styles for state-modifier --><style scope="state-modifier">a[inherit].state-modifier {
  color: inherit;
}</style><!-- Shady DOM styles for browser-ui --><!-- Shady DOM styles for outlined-text-field --><style scope="outlined-text-field">outlined-text-field {
  display: flex;
  font-family: "Google Sans", Roboto, Arial, sans-serif;
  font-weight: 500;
  flex-direction: column;
}

outlined-text-field span.outlined-text-field {
  display: flex;
  flex-direction: row;
  align-items: center;
}

outlined-text-field input.outlined-text-field {
  height: 40px;
  padding: 0 33px 0 16px;
  
  border-color: #b0b3b6;
  border-width: 1px;
  border-style: solid;
  
  border-radius: 8px;
  caret-color: #3c78e0;
  color: #6a6c71;
  font-size: 14px;
  flex-grow: 1;
}

outlined-text-field[orig] input.outlined-text-field {
  border-color: #e5e5e5;
  
  border-radius: 0;
}

outlined-text-field[dense] input.outlined-text-field {
  height: 32px;
}

outlined-text-field[uppercase] input.outlined-text-field {
  text-transform: uppercase;
}

outlined-text-field:not([wizard-focused]):hover input.outlined-text-field {
  border-color: #737375;
  color: #414447;
}

outlined-text-field:not([wizard-focused]):focus-within input.outlined-text-field,outlined-text-field[wizard-focused] input.outlined-text-field {
  border-color: #3574e0;
  caret-color: #80a5f1;
}

outlined-text-field[wizard-focused] input.outlined-text-field {
  border-width: 2px;
}

outlined-text-field input.outlined-text-field::placeholder {
  color: #65686c;
}

outlined-text-field[uppercase] input.outlined-text-field::placeholder {
  text-transform: none;
}</style><!-- Shady DOM styles for mat-keyword-editor --><!-- Shady DOM styles for mat-chips-input --><!-- Shady DOM styles for mat-input-chip --><!-- Shady DOM styles for pat-dialog --><style scope="pat-dialog">pat-dialog {
  background: white;
  position: absolute;
  border-radius: 8px;
  
  box-shadow: 0 24px 38px 3px rgb(0 0 0 / 14%),
              0 9px 46px 8px rgb(0 0 0 / 12%),
              0 11px 15px -7px rgb(0 0 0 / 20%);
  z-index: 1;
  display: flex;
  flex-direction: column;
}

pat-dialog:not([open]) {
  display: none;
}</style><!-- Shady DOM styles for mat-tabs --><!-- Shady DOM styles for filter-chip --><style scope="filter-chip">filter-chip {
  align-items: center;
  background-color: #fff;
  border-color: #dadce0;
  border-radius: 8px;
  border-style: solid;
  border-width: 1px;
  color: #606368;
  cursor: pointer;
  font-family: "Google Sans", Roboto, Arial, sans-serif;
  font-size: 14px;
  font-weight: 500;
  height: 32px;
  letter-spacing: 0.2px;
  padding: 0 16px;
  user-select: none;
  display: inline-flex;
  white-space: nowrap;
}

filter-chip[disabled] {
  background-color: #e8e8e8;
  border-color: #e8e8e8;
  color: #a9abad;
  cursor: auto;
}

filter-chip svg.filter-chip {
  visibility: collapse;
  font-size: 18px;
  width: 0;
  height: 18px;
}

filter-chip[selected] svg.filter-chip {
  visibility: visible;
  margin: 0 8px;
  width: 18px;
}

filter-chip[selected] svg.filter-chip path.filter-chip {
  fill: #4577d1;
}

filter-chip:not([disabled]):hover {
  background-color: #f7f8f8;
  border-color: #d7d9dc;
  color: #454649;
}

filter-chip:not([disabled]):hover:active {
  background-color: #eff0f0;
  border-color: #d2d5d8;
  color: #222326;
}

filter-chip:not([disabled]):focus {
  background-color: #e7e8e8;
  border-color: #232528;
  color: #202124;
}

filter-chip:not([disabled])[selected] {
  background-color: #e9f0fd;
  border-color: #e9f0fd;
  color: #4577d1;
  padding-left: 0;
}

filter-chip:not([disabled]):focus[selected] {
  background-color: #d5e1fb;
  border-color: #d5e1fb;
  color: #2550a0;
}

filter-chip:not([disabled]):hover[selected] {
  box-shadow: rgba(0, 0, 0, 0.2) 0px 3px 1px -2px,
    rgba(0, 0, 0, 0.14) 0px 2px 2px 0px, rgba(0, 0, 0, 0.12) 0px 1px 5px 0px;
}

filter-chip:not([disabled]):hover:active[selected] {
  background-color: #abc8f2;
  border-color: #abc8f2;
  color: #2a54a3;
}</style><!-- Shady DOM styles for figure-callout --><!-- Shady DOM styles for mat-button --><style scope="mat-button-0">.mat-button-0 {
  ;
  align-items: center;
  border-radius: 4px;
  color: hsl(217 4% 39%); 
  cursor: pointer;
  font-family: "Google Sans", Roboto, Arial, sans-serif;
  font-size: 14px;
  font-weight: 500;
  height: 36px;
  letter-spacing: 0.2px;
  padding: 0 8px;
  user-select: none;
  display: inline-flex;
  white-space: nowrap;
}

.mat-button-0[filled] {
  background-color: hsl(217 73% 54%); 
  color: white;
}

.mat-button-0[filled]:hover {
  background-color: hsl(217 60% 47%);
}

.mat-button-0[filled]:focus {
  background-color: hsl(217 59% 44%);
}

.mat-button-0[filled]:active {
  background-color: hsl(217 59% 45%);
}

.mat-button-0[filled][disabled] {
  background-color: hsl(217 2% 90%); 
  color: hsl(217 1% 65%);
}</style><!-- Shady DOM styles for search-term-wizard --><style scope="search-term-wizard">search-term-wizard:not([open]) {
  display: none;
}

search-term-wizard .header.search-term-wizard {
  border-bottom: 1px solid #e5e5e5;
  padding: 24px;
  padding-bottom: 0;
}

search-term-wizard .title.search-term-wizard {
  color: #3c4043;
  font-family: Google Sans,Roboto,Arial,sans-serif;
  font-size: 22px;
  user-select: none;
  margin: 0 48px;
}

search-term-wizard #panels.search-term-wizard > *.search-term-wizard:not([selected]) {
  display: none;
}</style><!-- Shady DOM styles for outlined-textarea --><style scope="outlined-textarea">outlined-textarea {
  display: flex;
  font-family: "Google Sans", Roboto, Arial, sans-serif;
  font-weight: 500;
  flex-direction: row;
  
  position: relative;
}

outlined-textarea span.outlined-textarea {
  display: flex;
  flex-direction: column;
  flex-grow: 1;
  justify-content: center;
}

outlined-textarea span.right-button-container.outlined-textarea {
  position: absolute;
  right: 0;
  height: 100%;
}

outlined-textarea textarea.outlined-textarea {
  padding: 16px 16px 0 16px;
  border-color: #b0b3b6;
  border-width: 1px;
  border-style: solid;
  border-radius: 8px;
  caret-color: #3c78e0;
  color: #6a6c71;
  font-size: 14px;
  font-family: "Google Sans", Roboto, Arial, sans-serif;
  font-weight: 500;
  resize: none;
}

outlined-textarea[dense] textarea.outlined-textarea {
  padding: 8px;
}

outlined-textarea[has-icon] textarea.outlined-textarea {
  padding-right: 38px;
}

outlined-textarea[orig] textarea.outlined-textarea {
  border-color: #e5e5e5;
  
  border-radius: 0;
}

outlined-textarea:not([rows]) textarea.outlined-textarea {
  height: 40px;
}

outlined-textarea textarea.resizable.outlined-textarea {
  resize: vertical;
}

outlined-textarea textarea.outlined-textarea:focus,outlined-textarea[wizard-focused] textarea.outlined-textarea,outlined-textarea textarea.outlined-textarea:focus:hover {
  border-color: #3574e0 !important;
  
  outline: none !important;
}

outlined-textarea textarea.outlined-textarea:hover {
  border-color: #202124;
  color: #414447;
}

outlined-textarea.hasValue textarea.outlined-textarea {
  border-color: #888d91;
  color: #45474a;
}

outlined-textarea #helperText.outlined-textarea {
  color: #6a6c71;
  font-size: 12px;
  letter-spacing: 0.3px;
  margin-left: 17px;
  margin-top: 6px;
  user-select: none;
}

outlined-textarea:not([helper-text]) #helperText.outlined-textarea {
  display: none;
}

outlined-textarea[error] textarea.outlined-textarea {
  border-color: #cb4b3d;
  caret-color: #ca493a;
  color: #3d4043;
}

outlined-textarea textarea.outlined-textarea::placeholder {
  font-size: 14px;
  font-weight: 400;
}

outlined-textarea:not([error]):not([disabled]) textarea.outlined-textarea::placeholder {
  color: #6a6c71;
}

outlined-textarea[error] textarea.outlined-textarea::placeholder {
  color: #c84132;
}

outlined-textarea[error] #helperText.outlined-textarea {
  color: #cd5648;
}

outlined-textarea[error] textarea.outlined-textarea:focus {
  border-color: #c84031;
  border-width: 2px;
  outline: none !important;
}

outlined-textarea[error]:not([disabled]) textarea.outlined-textarea:hover {
  border-color: #97231b;
}

outlined-textarea[error]:not([disabled]) textarea.outlined-textarea:hover::placeholder,outlined-textarea[error]:not([disabled]) textarea.outlined-textarea:hover + #helperText.outlined-textarea {
  color: #a94945;
}

outlined-textarea[disabled] textarea.outlined-textarea {
  border-color: #e9e9e9;
  color: #b5b6b8;
}

outlined-textarea[disabled] #helperText.outlined-textarea {
  color: #c4c6c7;
}

outlined-textarea[error][disabled] {
  opacity: 50%;
}

outlined-textarea[error][disabled] textarea.outlined-textarea {
  border-color: #cb4b3d;
  caret-color: #ca493a;
}

outlined-textarea[error][disabled] #helperText.outlined-textarea,outlined-textarea[error][disabled] textarea.outlined-textarea {
  color: #a94945;
}

outlined-textarea[error][disabled] textarea.outlined-textarea::placeholder {
  color: #c84132;
}</style><!-- Shady DOM styles for chemistry-search-term-wizard --><style scope="chemistry-search-term-wizard">chemistry-search-term-wizard {
  display: flex;
  flex-direction: column;
  padding: 24px;
  padding-top: 0;
}

chemistry-search-term-wizard > span.chemistry-search-term-wizard {
  display: flex;
  flex-direction: row;
  align-items: center;
  flex-wrap: wrap;
  margin-top: 16px;
}

chemistry-search-term-wizard .suggestions.chemistry-search-term-wizard {
  margin-top: 4px;
}

chemistry-search-term-wizard > span.chemistry-search-term-wizard svg.chemistry-search-term-wizard {
  margin-right: 24px;
}

chemistry-search-term-wizard path.chemistry-search-term-wizard {
  fill: #5f6368;
}

chemistry-search-term-wizard .help.chemistry-search-term-wizard {
  margin-left: 12px;
  margin-right: 0;
  cursor: pointer;
}

chemistry-search-term-wizard outlined-text-field.chemistry-search-term-wizard {
  flex-grow: 1;
}

chemistry-search-term-wizard mat-button.chemistry-search-term-wizard {
  margin-left: 48px;
}

chemistry-search-term-wizard filter-chip.chemistry-search-term-wizard {
  margin-right: 4px;
}

chemistry-search-term-wizard outlined-textarea.chemistry-search-term-wizard {
  flex-grow: 1;
}

chemistry-search-term-wizard outlined-text-field[batchMode].chemistry-search-term-wizard {
  display: none;
}

chemistry-search-term-wizard outlined-textarea.chemistry-search-term-wizard:not([batchMode]) {
  display: none;
}</style><!-- Shady DOM styles for text-search-term-wizard --><style scope="text-search-term-wizard">text-search-term-wizard {
  display: flex;
  flex-direction: column;
  padding: 24px;
  padding-top: 0;
}

text-search-term-wizard > span.text-search-term-wizard {
  display: flex;
  flex-direction: row;
  align-items: center;
  flex-wrap: wrap;
  margin-top: 16px;
}

text-search-term-wizard > span.text-search-term-wizard svg.text-search-term-wizard {
  margin-right: 24px;
}

text-search-term-wizard path.text-search-term-wizard {
  fill: #5f6368;
}

text-search-term-wizard outlined-text-field.text-search-term-wizard {
  flex-grow: 1;
}

text-search-term-wizard mat-button.text-search-term-wizard {
  margin-left: 48px;
}

text-search-term-wizard filter-chip.text-search-term-wizard {
  margin-right: 4px;
}</style><!-- Shady DOM styles for classification-wizard --><style scope="classification-wizard">classification-wizard {
  display: flex;
  flex-direction: column;
  padding: 24px;
  padding-top: 0;
  max-width: 674px;
}

classification-wizard > span.classification-wizard {
  display: flex;
  flex-direction: row;
  align-items: center;
  flex-wrap: wrap;
  margin-top: 16px;
}

classification-wizard .suggestions.classification-wizard {
  margin-top: 4px;
}

classification-wizard > span.classification-wizard svg.classification-wizard {
  margin-right: 24px;
}

classification-wizard path.classification-wizard {
  fill: #5f6368;
}

classification-wizard outlined-text-field.classification-wizard {
  flex-grow: 1;
}

classification-wizard mat-button.classification-wizard {
  margin-left: 48px;
}

classification-wizard filter-chip.classification-wizard {
  margin-right: 4px;
}

classification-wizard .no-matches.classification-wizard {
  margin-left: 64px;
}

classification-wizard .table-container.classification-wizard {
  max-height: 300px;
  overflow: scroll;
  margin-top: 8px;
}

classification-wizard .table-container.classification-wizard::-webkit-scrollbar {
  -webkit-appearance: none;
  width: 10px;
}

classification-wizard .table-container.classification-wizard::-webkit-scrollbar-thumb {
  border-radius: 5px;
  background-color: #5f636880;
  -webkit-box-shadow: 0 0 1px rgba(255,255,255,.5);
}

classification-wizard table.classification-wizard {
  margin-left: 51px;
  max-width: 575px;
}

classification-wizard tr.classification-wizard {
  margin-top: 4px;
}

classification-wizard tr.classification-wizard suggestive-chip.classification-wizard {
  margin-right: 8px;
}

classification-wizard #spinner.classification-wizard {
  justify-content: space-around;
  margin-left: 48px;
  transform: scale(0.5);
}</style><!-- Shady DOM styles for suggestive-chip --><!-- Shady DOM styles for mat-checkbox --><!-- Shady DOM styles for measure-search-term-wizard --><style scope="measure-search-term-wizard">measure-search-term-wizard {
  display: flex;
  flex-direction: column;
  padding: 24px;
  padding-top: 0;
  max-width: 674px;
}

measure-search-term-wizard > span.measure-search-term-wizard {
  display: flex;
  flex-direction: row;
  align-items: center;
  flex-wrap: wrap;
  margin-top: 16px;
}

measure-search-term-wizard .suggestions.measure-search-term-wizard {
  margin-top: 4px;
}

measure-search-term-wizard > span.measure-search-term-wizard svg.measure-search-term-wizard {
  margin-right: 24px;
}

measure-search-term-wizard path.measure-search-term-wizard {
  fill: #5f6368;
}

measure-search-term-wizard outlined-text-field.measure-search-term-wizard {
  flex-grow: 1;
}

measure-search-term-wizard mat-button.measure-search-term-wizard {
  margin-left: 48px;
}

measure-search-term-wizard filter-chip.measure-search-term-wizard {
  margin-right: 4px;
}</style><!-- Shady DOM styles for numerical-search-term-wizard --><style scope="numerical-search-term-wizard">numerical-search-term-wizard {
  display: flex;
  flex-direction: column;
  padding: 24px;
  padding-top: 0;
  max-width: 674px;
}

numerical-search-term-wizard > span.numerical-search-term-wizard {
  display: flex;
  flex-direction: row;
  align-items: center;
  flex-wrap: wrap;
  margin-top: 16px;
}

numerical-search-term-wizard .suggestions.numerical-search-term-wizard {
  margin-top: 4px;
}

numerical-search-term-wizard > span.numerical-search-term-wizard svg.numerical-search-term-wizard {
  margin-right: 24px;
}

numerical-search-term-wizard path.numerical-search-term-wizard {
  fill: #5f6368;
}

numerical-search-term-wizard outlined-text-field.numerical-search-term-wizard,numerical-search-term-wizard outlined-textarea.numerical-search-term-wizard {
  flex-grow: 1;
}

numerical-search-term-wizard mat-button.numerical-search-term-wizard {
  margin-left: 48px;
}

numerical-search-term-wizard filter-chip.numerical-search-term-wizard {
  margin-right: 4px;
}

numerical-search-term-wizard #spinner.numerical-search-term-wizard {
  justify-content: space-around;
  margin-left: 48px;
  transform: scale(0.5);
}

numerical-search-term-wizard #suggestions.numerical-search-term-wizard {
  margin-left: 58px;
  display: flex;
  flex-direction: column;
  align-items: flex-start;
}

numerical-search-term-wizard .suggestion.numerical-search-term-wizard {
  display: flex;
  flex-direction: row;
  align-items: center;
}

numerical-search-term-wizard .suggestion.numerical-search-term-wizard mat-checkbox.numerical-search-term-wizard {
  margin-right: 8px;
}</style><!-- Shady DOM styles for text-button --><style scope="text-button">text-button {
  align-items: center;
  border-radius: 4px;
  cursor: pointer;
  font-family: "Google Sans", Roboto, Arial, sans-serif;
  font-size: 14px;
  font-weight: 500;
  height: 36px;
  letter-spacing: .25px;
  margin: 6px 0;
  padding: 0 8px;
  user-select: none;
  display: inline-flex;
  white-space: nowrap;
  line-height: 36px;
  text-decoration: none;
  text-transform: none;
  color: #5f6368;
}

text-button:hover {
  background-color: #f8f9fa;
}

text-button:active {
  background-color: #d5e2fa;
}

text-button[selected] {
  background-color: #e8f0fe;
  color: #1967d2;
}

text-button[selected]:active {
  background-color: #d2e3fc;
}</style><!-- Shady DOM styles for mat-spinner --><!-- Shady DOM styles for requery-tile --><style scope="requery-tile">requery-tile {
  display: flex;
  justify-content: space-between;
  align-items: center;
  box-sizing: border-box;
  min-height: 40px;
  padding: 8px 12px 8px 8px;
  gap: 8px;

  border-radius: 16px;
  background: #f7f8f9;
  color: #1f1f1f;
}

requery-tile:hover {
  background: #d7d8d9;
}

requery-tile > [slot="image"] {
  flex-shrink: 0;

  border-radius: 8px;

  background: #fff;
}

.label.requery-tile {
  flex-grow: 999;

  font: 16px/20px "Google Sans", Roboto, Arial, sans-serif;
  font-weight: 500;
}

.search-icon.requery-tile {
  flex-shrink: 0;

  width: 24px;
  height: 24px;
  background-image: url('https://fonts.gstatic.com/s/i/short-term/release/googlesymbols/search/default/24px.svg');
}</style><!-- Shady DOM styles for s-more-inline --><style scope="s-more-inline">s-more-inline {
  display: block;
  margin-top: 24px;
  padding: 16px;
  padding-top: 0;
  position: relative;

  line-height: 18px;
  text-overflow: ellipsis;
  white-space: nowrap;
  cursor: pointer;
}

hr.s-more-inline {
  background-color: #dadce0;
  margin-top: 18px;
  position: absolute;
  border: 0;
  height: 1px;
  left: 0;
  width: 100%;
}

.button.s-more-inline {
  display: flex;
  align-items: center;
  justify-content: center;

  position: relative; 
  margin: 0 auto;

  font: 400 14px Roboto, arial, sans-serif;
  height: 36px;
  width: 300px;
  border-radius: 18px;
  background-color: #f1f3f4;
}

.button.s-more-inline:hover {
  background-color: #d8d7dc;
}

.button.s-more-inline > [slot] {
  padding-right: 8px;
}

.expand.s-more-inline {
  display: inline-block;
  width: 20px;
  height: 20px;
}

.expand.less.s-more-inline {
  background-image: url('https://fonts.gstatic.com/s/i/short-term/release/googlesymbols/expand_less/default/20px.svg');
}

.expand.more.s-more-inline {
  background-image: url('https://fonts.gstatic.com/s/i/short-term/release/googlesymbols/expand_more/default/20px.svg');
}</style><!-- Shady DOM styles for workspace-ui-search --><!-- Shady DOM styles for keyword-wizard --><style scope="keyword-wizard">keyword-wizard {
  display: inline-block;
}

iron-icon.keyword-wizard {
  height: 20px;
    width: 20px;
    color: #999;
    cursor: pointer;
}

iron-icon[active].keyword-wizard {
  color: #4285f4 !important;
}

iron-icon.keyword-wizard:hover {
  color: #ccc;
}

#wizardIcon.keyword-wizard {
  padding-left: 1px;
    height: 16px;
    width: 16px;
    margin: 4px 1px 1px 1px;
}

#wizard.keyword-wizard {
  padding: 0;
    left: 0;
    top: 37px;
    position: absolute;
    background-color: white;
    z-index: 2;
    overflow: hidden;
}

.wizard-header.keyword-wizard {
  display: block;
    width: 100%;
    color: #737373;
    text-transform: uppercase;
    font-size: 13px;
    margin: 16px 0 6px 12px;
}

.filter-chips.keyword-wizard {
  margin-left: 20px;
}

.chip.keyword-wizard {
  background-color: #ebebeb;
    color: #6a6a6a;
    display: inline-block;
    font-size: 13px;
    font-family: inherit;
    border-radius: 12px;
    padding: 6px 12px;
}

.chip.keyword-wizard:hover {
  cursor: pointer;
}

.chip[selected].keyword-wizard {
  background-color: #4285f4;
    color: white;
}

.input-label.keyword-wizard {
  display: inline-block;
    font-size: 13px;
    font-family: inherit;
}

.keyword-input.keyword-wizard {
  display: inline-block;
    border-top: 0;
    border-left: 0;
    border-right: 0;
    border-bottom: 1px solid #dfe1e5;
    padding: 3px;
}

.keyword-input.keyword-wizard::placeholder {
  font-style: italic;
}

.keyword-input.keyword-wizard:hover,.keyword-input.keyword-wizard:focus {
  border-color: #4284f4;
    border-width: 1px;
    outline: none;
}

.cpc-field.keyword-wizard {
  display: inline-block;
    margin: 6px 0 0 24px;
}

#checkboxCPC.keyword-wizard:hover {
  cursor: pointer;
}

.checkbox-label.keyword-wizard {
  font-size: 13px;
    font-style: italic;
    color: #737373;
}

.chemistry-field.keyword-wizard {
  display: block;
}

#inputChem.input-batch.keyword-wizard {
  height: 150px;
    border: 1px solid #dfe1e5;
    padding: 6px;
}

#inputChem.keyword-wizard {
  resize: none;
    height: 15px;
    vertical-align: text-top;
    font-size: 13px;
    font-family: inherit;
    padding: 2px;
}

.chem-type-label.keyword-wizard {
  display: inline;
    font-size: 13px;
    margin-left: 12px;
    margin-right: 4px;
}

#chemType.keyword-wizard {
  border-top: 0;
    border-left: 0;
    border-right: 0;
    border-bottom: 1px solid #dfe1e5;
    margin: 6px;
    padding: 2px;
    outline: none;
    font-family: inherit;
    font-size: 13px;
}

.import.keyword-wizard {
  font-size: 13px;
    font-family: inherit;
}

.import-text.keyword-wizard {
  vertical-align: middle;
    display: inline-block;
}

.import-button.keyword-wizard {
  margin: 6px 0 0 24px;
}

.import-button.keyword-wizard button.keyword-wizard {
  border: 0;
    border-radius: 4px;
    padding: 6px 12px;
    font-family: inherit;
    background-color: #4284f4;
    color: white;
}

.import-button.keyword-wizard button.keyword-wizard:hover {
  cursor: pointer;
    background-color: #4284f4;
    color: white;
}

.wizard-buttons.keyword-wizard {
  float: right;
    padding: 12px;
}

.wizard-buttons.keyword-wizard button.keyword-wizard {
  border: 0;
    border-radius: 4px;
    padding: 6px 12px;
    margin: 2px;
    font-family: inherit;
    background-color: #4284f4;
    color: white;
}

.wizard-buttons.keyword-wizard button.keyword-wizard:hover {
  cursor: pointer;
}

@media screen and (max-width: 700px) {
#wizard.keyword-wizard {
  width: 275px;
}

.chip.keyword-wizard {
  margin: 4px 2px;
}

.keyword-input.keyword-wizard {
  width: 82%;
      margin: 4px 12px 12px 24px;
}

.search-within.keyword-wizard .input-label.keyword-wizard {
  display: none;
}

.cpc-field.keyword-wizard .keyword-input.keyword-wizard {
  width: 225px;
      margin: 12px 0 6px 0;
}

.find-patents.keyword-wizard .input-label.keyword-wizard {
  display: none;
}

.search-by.keyword-wizard .input-label.keyword-wizard {
  width: 55px;
      margin-left: 24px;
}

.search-by.keyword-wizard .keyword-input.keyword-wizard {
  margin-top: 6px;
}

#inputChem.keyword-wizard {
  width: 82%;
      padding: 2px;
      margin: 8px 12px 12px 24px;
}

#inputChem.input-batch.keyword-wizard {
  width: 220px;
      padding: 4px;
}

#chemType.keyword-wizard {
  width: 150px;
}

.import-text.keyword-wizard {
  margin: 6px 12px 6px 24px;
      width: 220px;
}

iron-icon.help-header.keyword-wizard {
  display: none;
}

}

@media screen and (min-width: 700px) {
#wizard.keyword-wizard {
  width: 540px;
}

.chip.keyword-wizard {
  margin: 0 2px;
}

.cpc-field.keyword-wizard .keyword-input.keyword-wizard {
  width: 240px;
      margin: 12px 16px 6px 6px;
}

.cpc-inputs.keyword-wizard {
  display: inline;
}

.find-patents.keyword-wizard .keyword-input.keyword-wizard {
  width: 53%;
      margin: 4px 10px;
}

.cpc-field.keyword-wizard .keyword-input.keyword-wizard::placeholder,#inputAnd.keyword-wizard::placeholder,#inputExact.keyword-wizard::placeholder,#inputOr.keyword-wizard::placeholder,#inputMinus.keyword-wizard::placeholder {
  color: white;
}

.find-patents.keyword-wizard .input-label.keyword-wizard {
  width: 35%;
      margin-left: 24px;
}

.search-by.keyword-wizard .keyword-input.keyword-wizard {
  width: 42%;
      margin: 6px 4px;
}

.search-by.keyword-wizard .input-label.keyword-wizard {
  width: 55px;
      margin-left: 24px;
}

#inputChem.keyword-wizard {
  width: 290px;
      margin: 2px 12px 2px 6px;
}

#inputChem.input-batch.keyword-wizard {
  width: 280px;
      padding: 6px;
}

.import-text.keyword-wizard {
  margin: 6px 12px;
      width: 415px;
}

#chemType.keyword-wizard {
  width: 105px;
}

iron-icon.keyword-wizard:hover.help-header {
  cursor: default;
}

iron-icon.help-header.keyword-wizard {
  height: 16px;
      padding: 0 2px 2px 2px;
      color: #a3a3a3;
}

.help.keyword-wizard p.keyword-wizard {
  margin: 0 0 8px 0
}

.help.keyword-wizard .q.keyword-wizard {
  font-family: monospace;
      color: #0b8043;
      white-space: nowrap;
}

.help.keyword-wizard {
  z-index: 4;
      position: absolute;
      right: 10px;
      padding: 12px;
      border-radius: 2px;
      border: 1px solid #e0e0e0;
      box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.2), 0 6px 10px 0 rgba(0, 0, 0, 0.3);
      background: white;
      font-size: 13px;
      line-height: 20px;
      width: 450px;
      max-height: 400px;
      overflow: auto;
}

#withinHelp.keyword-wizard {
  top: 75px;
}

#chemHelp.keyword-wizard {
  top: 0;
}

#searchBooleanHelp.keyword-wizard {
  top: 140px;
}

}</style><!-- Shady DOM styles for bottom-bar --><style scope="bottom-bar">.fixed.bottom-bar {
  display: block;
  position: fixed;
  height: 46px;
  bottom: 0;
}

.fixed[medium].bottom-bar {
  height: 280px;
}

.fixed[large].bottom-bar {
  height: 480px;
}

.shadow.bottom-bar {
  background: white;
  padding: 16px;
  border-radius: 8px 8px 0px 0px;
  box-shadow: 0px 8px 10px 1px rgba(0,0,0,.14), 0px 3px 14px 2px rgba(0,0,0,.12), 0px 5px 5px -3px rgba(0,0,0,.2);
}

.oneline.bottom-bar {
  white-space: nowrap;
  overflow: hidden;
  text-overflow: ellipsis;
  cursor: pointer;
}

.button.bottom-bar {
  cursor: pointer;
}

.buttonAlign.bottom-bar {
  margin-top: -4px;
}

.header.bottom-bar {
  height: 30px;
  font-family: 'Google Sans';
  font-size: 18px;
  padding-bottom: 8px;
}

.list.bottom-bar {
  overflow-y: auto;
  line-height: 20px;
}

.definition.bottom-bar {
  padding: 0px 0px 4px 0px;
  cursor: pointer;
}

[medium].bottom-bar .content.bottom-bar {
  height: 230px;
}

[large].bottom-bar .content.bottom-bar {
  height: 430px;
}

[medium].bottom-bar .content.bottom-bar .list.bottom-bar {
  height: 200px;
}

[large].bottom-bar .content.bottom-bar .list.bottom-bar {
  height: 400px;
}

[hidden].bottom-bar {
  display: none;
}

.label.bottom-bar {
  font-family: 'Google Sans';
  font-size: 12px;
  line-height: 16px;
  font-weight: 400;
  padding: 6px 0px 4px 0px;
  color: #1f1f1f;
}

[pad].bottom-bar {
  margin-left: 16px;
}

.layout.vertical.bottom-bar {
  display: flex;
  flex-direction: column;
}

.layout.horizontal.bottom-bar {
  display: flex;
  flex-direction: row;
}

.flex.bottom-bar {
  flex: 1;
  flex-basis: 0.000000001px;
}

.layout.start.bottom-bar {
  align-items: flex-start;
}

.layout.center.bottom-bar {
  align-items: center;
}

.layout.end.bottom-bar {
  align-items: flex-end;
}

.flex-2.bottom-bar {
  flex: 2;
}

.flex-3.bottom-bar {
  flex: 3;
}

.flex-4.bottom-bar {
  flex: 4;
}

.flex-5.bottom-bar {
  flex: 5;
}

.flex-6.bottom-bar {
  flex: 6;
}

.flex-7.bottom-bar {
  flex: 7;
}

.flex-8.bottom-bar {
  flex: 8;
}

.flex-9.bottom-bar {
  flex: 9;
}

.flex-10.bottom-bar {
  flex: 10;
}

.flex-11.bottom-bar {
  flex: 11;
}

.flex-12.bottom-bar {
  flex: 12;
}</style><!-- Shady DOM styles for landscape-link --><!-- Shady DOM styles for landscape-summary-box --><!-- Shady DOM styles for search-state --><!-- Shady DOM styles for iron-overlay-backdrop --><!-- Shady DOM styles for opaque-animation --><!-- Shady DOM styles for iron-dropdown --><!-- Shady DOM styles for iron-meta --><!-- Shady DOM styles for iron-icon --><style scope="iron-icon-0">.iron-icon-0 {
  display: -ms-inline-flexbox;
        display: -webkit-inline-flex;
        display: inline-flex;-ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;-ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
        position: relative;

        vertical-align: middle;

        fill: currentcolor;
        stroke: none;

        width: 24px;
        height: 24px;;
}

.iron-icon-0[hidden] {
  display: none;
}</style><!-- Shady DOM styles for iron-iconset-svg --><!-- Shady DOM styles for iron-list --><!-- Shady DOM styles for paper-icon-item --><!-- Shady DOM styles for paper-item-body --><!-- Shady DOM styles for paper-item --><!-- Shady DOM styles for dropdown-menu --><!-- Shady DOM styles for chips-input --><!-- Shady DOM styles for input-chip --><!-- Shady DOM styles for keyword-editor --><!-- Shady DOM styles for paper-ripple --><!-- Shady DOM styles for paper-button --><!-- Shady DOM styles for date-editor --><!-- Shady DOM styles for metadata-editor --><!-- Shady DOM styles for restrict-editor --><!-- Shady DOM styles for overlay-tooltip --><style scope="overlay-tooltip">overlay-tooltip {
  display: none;
        color: #fff;
        background-color: #000;
        padding: 10px 12px;
        border-radius: 3px;
        font-size: 13px;
        font-weight: normal;
        line-height: normal;
        position: fixed;
        z-index: 100;
}

#arrow.overlay-tooltip {
  width: 0;
        height: 0;
        border-style: solid;
        border-width: 0 6px 8px 6px;
        border-color: transparent transparent #000 transparent;
        position: absolute;
        top: -8px;
        left: calc(50% - 6px);
}</style><!-- Shady DOM styles for query-renderer --><!-- Shady DOM styles for result-nav --><style scope="result-nav">result-nav {
  display: block;
        white-space: nowrap;
        padding: 0 30px;
        line-height: 30px;
        height: 30px;
        margin-top: 17px;
        font-weight: 500;
        font-size: 15px;
}

state-modifier.result-nav {
  color: #333;
}

state-modifier[disabled].result-nav {
  pointer-events: none;
        color: #ccc;
}</style><!-- Shady DOM styles for raw-html --><!-- Shady DOM styles for paper-material --><style scope="paper-material-0">.paper-material-0 {
  display: block;
        position: relative;
}

.paper-material-0[elevation="1"] {
  box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.14),
                    0 1px 5px 0 rgba(0, 0, 0, 0.12),
                    0 3px 1px -2px rgba(0, 0, 0, 0.2);
}

.paper-material-0[elevation="2"] {
  box-shadow: 0 4px 5px 0 rgba(0, 0, 0, 0.14),
                    0 1px 10px 0 rgba(0, 0, 0, 0.12),
                    0 2px 4px -1px rgba(0, 0, 0, 0.4);
}

.paper-material-0[elevation="3"] {
  box-shadow: 0 6px 10px 0 rgba(0, 0, 0, 0.14),
                    0 1px 18px 0 rgba(0, 0, 0, 0.12),
                    0 3px 5px -1px rgba(0, 0, 0, 0.4);
}

.paper-material-0[elevation="4"] {
  box-shadow: 0 8px 10px 1px rgba(0, 0, 0, 0.14),
                    0 3px 14px 2px rgba(0, 0, 0, 0.12),
                    0 5px 5px -3px rgba(0, 0, 0, 0.4);
}

.paper-material-0[elevation="5"] {
  box-shadow: 0 16px 24px 2px rgba(0, 0, 0, 0.14),
                    0  6px 30px 5px rgba(0, 0, 0, 0.12),
                    0  8px 10px -5px rgba(0, 0, 0, 0.4);
}

.paper-material-0[animated] {
  transition: box-shadow 0.28s cubic-bezier(0.4, 0, 0.2, 1);
}

.paper-material-0 {
  ;
}</style><!-- Shady DOM styles for search-box --><style scope="search-box">.layout.horizontal.search-box,.layout.vertical.search-box {
  display: -ms-flexbox;
        display: -webkit-flex;
        display: flex;
}

.layout.inline.search-box {
  display: -ms-inline-flexbox;
        display: -webkit-inline-flex;
        display: inline-flex;
}

.layout.horizontal.search-box {
  -ms-flex-direction: row;
        -webkit-flex-direction: row;
        flex-direction: row;
}

.layout.vertical.search-box {
  -ms-flex-direction: column;
        -webkit-flex-direction: column;
        flex-direction: column;
}

.layout.wrap.search-box {
  -ms-flex-wrap: wrap;
        -webkit-flex-wrap: wrap;
        flex-wrap: wrap;
}

.layout.no-wrap.search-box {
  -ms-flex-wrap: nowrap;
        -webkit-flex-wrap: nowrap;
        flex-wrap: nowrap;
}

.layout.center.search-box,.layout.center-center.search-box {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.layout.center-justified.search-box,.layout.center-center.search-box {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.flex.search-box {
  -ms-flex: 1 1 0.000000001px;
        -webkit-flex: 1;
        flex: 1;
        -webkit-flex-basis: 0.000000001px;
        flex-basis: 0.000000001px;
}

.flex-auto.search-box {
  -ms-flex: 1 1 auto;
        -webkit-flex: 1 1 auto;
        flex: 1 1 auto;
}

.flex-none.search-box {
  -ms-flex: none;
        -webkit-flex: none;
        flex: none;
}

.inputbox.search-box {
  border: 1px solid #ccc;
        padding: 0 12px;
        background: white;
}

.inputbox[focused].search-box {
  border: 1px solid #4285f4 !important;
}

.inputbox.search-box:hover {
  border: 1px solid #999;
}

#searchInput.search-box {
  font-family: inherit;
        font-size: 16px;
        padding: 8px 0;
        outline: none;
        border-right: none;
        width: 100%;
        margin-right: 0;
        border: 0;
}

#searchButton.search-box {
  cursor: pointer;
        background-color: #4285f4;
        border: none;
        color: #fff;
        vertical-align: top;
        width: 70px;
        height: 37px;
        outline: none;
}

.suggest-bg.search-box {
  display: inline-block;
        min-width: 0;
        white-space: nowrap;
}

#suggestMenu.search-box {
  position: absolute;
        margin-top: 38px;
        width: 100%;
        background: white;
        z-index: 1;
}

#suggestMenu.search-box .row.search-box {
  font-size: 13px;
        line-height: 28px;
        white-space: nowrap;
        overflow: hidden;
        text-overflow: ellipsis;
        padding: 4px 2px;
        cursor: pointer;
}

.row[selected].search-box {
  background: rgba(0, 0, 0, 0.05);
}

.suggest-icon.search-box {
  height: 20px;
        width: 20px;
        padding: 0 12px 0 6px;
        transition: transform .15s linear;
        transform: scale(1);
}

.suggest-icon[hide].search-box {
  transform: scale(0);
}

#corpora.search-box {
  padding: 6px 0;
        user-select: none;
}

#corpora.search-box iron-icon.search-box {
  height: 20px;
        width: 20px;
        color: #ccc;
        cursor: pointer;
}

#corpora.search-box iron-icon[active].search-box {
  color: #4285f4 !important;
}

#corpora.search-box iron-icon.search-box:hover {
  color: #999;
}</style><!-- Shady DOM styles for iron-media-query --><!-- Shady DOM styles for paper-header-panel --><!-- Shady DOM styles for iron-a11y-announcer --><style scope="iron-a11y-announcer">iron-a11y-announcer {
  display: inline-block;
        position: fixed;
        clip: rect(0px,0px,0px,0px);
}</style><!-- Shady DOM styles for iron-autogrow-textarea --><!-- Shady DOM styles for iron-input --><!-- Shady DOM styles for paper-input-char-counter --><!-- Shady DOM styles for paper-input-container --><!-- Shady DOM styles for paper-input-error --><!-- Shady DOM styles for paper-input --><!-- Shady DOM styles for paper-textarea --><!-- Shady DOM styles for paper-toast --><style scope="paper-toast-0">.paper-toast-0 {
  display: block;
        position: fixed;
        background-color: #323232;
        color: #f1f1f1;
        min-height: 48px;
        min-width: 288px;
        padding: 16px 24px;
        box-sizing: border-box;
        box-shadow: 0 2px 5px 0 rgba(0, 0, 0, 0.26);
        border-radius: 2px;
        margin: 12px;
        font-size: 14px;
        cursor: default;
        -webkit-transition: -webkit-transform 0.3s, opacity 0.3s;
        transition: transform 0.3s, opacity 0.3s;
        opacity: 0;
        -webkit-transform: translateY(100px);
        transform: translateY(100px);font-family: 'Roboto', 'Noto', sans-serif;
        -webkit-font-smoothing: antialiased;
}

.paper-toast-0.capsule {
  border-radius: 24px;
}

.paper-toast-0.fit-bottom {
  width: 100%;
        min-width: 0;
        border-radius: 0;
        margin: 0;
}

.paper-toast-0.paper-toast-open {
  opacity: 1;
        -webkit-transform: translateY(0px);
        transform: translateY(0px);
}</style><!-- Shady DOM styles for paper-toolbar --><!-- Shady DOM styles for search-footer --><style scope="search-footer">#footer.search-footer {
  font-size: 13px;
        margin-top: 10px;
        background: #fafafa;
        padding: 20px 10px;
        align-items: center;
        border-top: 1px solid #ebebeb;
}

#footer.search-footer .links.search-footer > *.search-footer {
  display: inline-block;
        margin: 0 12px;
}

.layout.horizontal.search-footer,.layout.vertical.search-footer {
  display: -ms-flexbox;
        display: -webkit-flex;
        display: flex;
}

.layout.inline.search-footer {
  display: -ms-inline-flexbox;
        display: -webkit-inline-flex;
        display: inline-flex;
}

.layout.horizontal.search-footer {
  -ms-flex-direction: row;
        -webkit-flex-direction: row;
        flex-direction: row;
}

.layout.vertical.search-footer {
  -ms-flex-direction: column;
        -webkit-flex-direction: column;
        flex-direction: column;
}

.layout.wrap.search-footer {
  -ms-flex-wrap: wrap;
        -webkit-flex-wrap: wrap;
        flex-wrap: wrap;
}

.layout.no-wrap.search-footer {
  -ms-flex-wrap: nowrap;
        -webkit-flex-wrap: nowrap;
        flex-wrap: nowrap;
}

.layout.center.search-footer,.layout.center-center.search-footer {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.layout.center-justified.search-footer,.layout.center-center.search-footer {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.flex.search-footer {
  -ms-flex: 1 1 0.000000001px;
        -webkit-flex: 1;
        flex: 1;
        -webkit-flex-basis: 0.000000001px;
        flex-basis: 0.000000001px;
}

.flex-auto.search-footer {
  -ms-flex: 1 1 auto;
        -webkit-flex: 1 1 auto;
        flex: 1 1 auto;
}

.flex-none.search-footer {
  -ms-flex: none;
        -webkit-flex: none;
        flex: none;
}

.layout.start.search-footer {
  -ms-flex-align: start;
        -webkit-align-items: flex-start;
        align-items: flex-start;
}

.layout.center.search-footer,.layout.center-center.search-footer {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.layout.end.search-footer {
  -ms-flex-align: end;
        -webkit-align-items: flex-end;
        align-items: flex-end;
}

.layout.baseline.search-footer {
  -ms-flex-align: baseline;
        -webkit-align-items: baseline;
        align-items: baseline;
}

.layout.start-justified.search-footer {
  -ms-flex-pack: start;
        -webkit-justify-content: flex-start;
        justify-content: flex-start;
}

.layout.center-justified.search-footer,.layout.center-center.search-footer {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.layout.end-justified.search-footer {
  -ms-flex-pack: end;
        -webkit-justify-content: flex-end;
        justify-content: flex-end;
}

.layout.around-justified.search-footer {
  -ms-flex-pack: distribute;
        -webkit-justify-content: space-around;
        justify-content: space-around;
}

.layout.justified.search-footer {
  -ms-flex-pack: justify;
        -webkit-justify-content: space-between;
        justify-content: space-between;
}

.self-start.search-footer {
  -ms-align-self: flex-start;
        -webkit-align-self: flex-start;
        align-self: flex-start;
}

.self-center.search-footer {
  -ms-align-self: center;
        -webkit-align-self: center;
        align-self: center;
}

.self-end.search-footer {
  -ms-align-self: flex-end;
        -webkit-align-self: flex-end;
        align-self: flex-end;
}

.self-stretch.search-footer {
  -ms-align-self: stretch;
        -webkit-align-self: stretch;
        align-self: stretch;
}

.self-baseline.search-footer {
  -ms-align-self: baseline;
        -webkit-align-self: baseline;
        align-self: baseline;
}

.layout.start-aligned.search-footer {
  -ms-flex-line-pack: start;  
        -ms-align-content: flex-start;
        -webkit-align-content: flex-start;
        align-content: flex-start;
}

.layout.end-aligned.search-footer {
  -ms-flex-line-pack: end;  
        -ms-align-content: flex-end;
        -webkit-align-content: flex-end;
        align-content: flex-end;
}

.layout.center-aligned.search-footer {
  -ms-flex-line-pack: center;  
        -ms-align-content: center;
        -webkit-align-content: center;
        align-content: center;
}

.layout.between-aligned.search-footer {
  -ms-flex-line-pack: justify;  
        -ms-align-content: space-between;
        -webkit-align-content: space-between;
        align-content: space-between;
}

.layout.around-aligned.search-footer {
  -ms-flex-line-pack: distribute;  
        -ms-align-content: space-around;
        -webkit-align-content: space-around;
        align-content: space-around;
}</style><!-- Shady DOM styles for search-header --><style scope="search-header">search-header {
  display: block;
}

.lockup-logo.search-header,.lockup-text.search-header {
  display: inline-block;
}

.lockup-logo.search-header {
  width: 74px;
        height: 24px;
        background-image: url('https://www.gstatic.com/images/branding/googlelogo/1x/googlelogo_color_74x24dp.png');
        background-size: 74px 24px;
}

.lockup-text.search-header {
  font-family: "Product Sans", Arial, Helvetica, sans-serif;

        
        font-size: 22.1px;

        
        text-rendering: optimizeLegibility;

        
        position: relative;
        top: -6px;
        left: -5px;

        
        -webkit-font-smoothing: antialiased;
        -moz-osx-font-smoothing: grayscale;
}

.lockup-brand.search-header {
  opacity: 0.54;
}

@media (-webkit-min-device-pixel-ratio: 1.5), (min-resolution: 144dpi) {
.lockup-logo.search-header {
  background-image: url('https://www.gstatic.com/images/branding/googlelogo/2x/googlelogo_color_74x24dp.png');
}

.lockup-text.search-header {
  font-size: 21.3px;
          left: -7px;
}

}

header.search-header {
  background-color: #fafafa;
        border-bottom: 1px solid #ebebeb;
        color: #333;
        height: 60px;
        width: 100%;
        z-index: 1;
        display: block;
}

[latestStyle].search-header header.search-header {
  background-color: transparent;
}

.headerpad.search-header {
  margin-left: 16px;
}

[wide].search-header .headerpad.search-header {
  margin-left: 32px;
}

.leftheader.search-header {
  width: 322px;
}

header.search-header h1.search-header {
  display: none;
}

header.search-header h1.search-header state-modifier.search-header {
  color: #777;
        text-decoration: none;
        display: inline-block;
}

[wide].search-header header.search-header h1.search-header {
  box-sizing: border-box;
        margin: 0;
        padding: 20px 0 10px;
        font-size: 20px;
        font-weight: 400;
        white-space: nowrap;
        display: block;
}

search-box.search-header {
  height: 37px;
        padding: 0 16px;
        width: 900px;
        align-self: center;
}

.layout.horizontal.search-header,.layout.vertical.search-header {
  display: -ms-flexbox;
        display: -webkit-flex;
        display: flex;
}

.layout.inline.search-header {
  display: -ms-inline-flexbox;
        display: -webkit-inline-flex;
        display: inline-flex;
}

.layout.horizontal.search-header {
  -ms-flex-direction: row;
        -webkit-flex-direction: row;
        flex-direction: row;
}

.layout.vertical.search-header {
  -ms-flex-direction: column;
        -webkit-flex-direction: column;
        flex-direction: column;
}

.layout.wrap.search-header {
  -ms-flex-wrap: wrap;
        -webkit-flex-wrap: wrap;
        flex-wrap: wrap;
}

.layout.no-wrap.search-header {
  -ms-flex-wrap: nowrap;
        -webkit-flex-wrap: nowrap;
        flex-wrap: nowrap;
}

.layout.center.search-header,.layout.center-center.search-header {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.layout.center-justified.search-header,.layout.center-center.search-header {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.flex.search-header {
  -ms-flex: 1 1 0.000000001px;
        -webkit-flex: 1;
        flex: 1;
        -webkit-flex-basis: 0.000000001px;
        flex-basis: 0.000000001px;
}

.flex-auto.search-header {
  -ms-flex: 1 1 auto;
        -webkit-flex: 1 1 auto;
        flex: 1 1 auto;
}

.flex-none.search-header {
  -ms-flex: none;
        -webkit-flex: none;
        flex: none;
}

.layout.start.search-header {
  -ms-flex-align: start;
        -webkit-align-items: flex-start;
        align-items: flex-start;
}

.layout.center.search-header,.layout.center-center.search-header {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.layout.end.search-header {
  -ms-flex-align: end;
        -webkit-align-items: flex-end;
        align-items: flex-end;
}

.layout.baseline.search-header {
  -ms-flex-align: baseline;
        -webkit-align-items: baseline;
        align-items: baseline;
}

.layout.start-justified.search-header {
  -ms-flex-pack: start;
        -webkit-justify-content: flex-start;
        justify-content: flex-start;
}

.layout.center-justified.search-header,.layout.center-center.search-header {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.layout.end-justified.search-header {
  -ms-flex-pack: end;
        -webkit-justify-content: flex-end;
        justify-content: flex-end;
}

.layout.around-justified.search-header {
  -ms-flex-pack: distribute;
        -webkit-justify-content: space-around;
        justify-content: space-around;
}

.layout.justified.search-header {
  -ms-flex-pack: justify;
        -webkit-justify-content: space-between;
        justify-content: space-between;
}

.self-start.search-header {
  -ms-align-self: flex-start;
        -webkit-align-self: flex-start;
        align-self: flex-start;
}

.self-center.search-header {
  -ms-align-self: center;
        -webkit-align-self: center;
        align-self: center;
}

.self-end.search-header {
  -ms-align-self: flex-end;
        -webkit-align-self: flex-end;
        align-self: flex-end;
}

.self-stretch.search-header {
  -ms-align-self: stretch;
        -webkit-align-self: stretch;
        align-self: stretch;
}

.self-baseline.search-header {
  -ms-align-self: baseline;
        -webkit-align-self: baseline;
        align-self: baseline;
}

.layout.start-aligned.search-header {
  -ms-flex-line-pack: start;  
        -ms-align-content: flex-start;
        -webkit-align-content: flex-start;
        align-content: flex-start;
}

.layout.end-aligned.search-header {
  -ms-flex-line-pack: end;  
        -ms-align-content: flex-end;
        -webkit-align-content: flex-end;
        align-content: flex-end;
}

.layout.center-aligned.search-header {
  -ms-flex-line-pack: center;  
        -ms-align-content: center;
        -webkit-align-content: center;
        align-content: center;
}

.layout.between-aligned.search-header {
  -ms-flex-line-pack: justify;  
        -ms-align-content: space-between;
        -webkit-align-content: space-between;
        align-content: space-between;
}

.layout.around-aligned.search-header {
  -ms-flex-line-pack: distribute;  
        -ms-align-content: space-around;
        -webkit-align-content: space-around;
        align-content: space-around;
}</style><!-- Shady DOM styles for search-ui --><style scope="search-ui-0">.search-ui-0 paper-button.search-ui {
  display: inline-block;
        text-align: center;
        font-family: inherit;
}

.search-ui-0 .layout.horizontal.search-ui,.search-ui-0 .layout.vertical.search-ui {
  display: -ms-flexbox;
        display: -webkit-flex;
        display: flex;
}

.search-ui-0 .layout.inline.search-ui {
  display: -ms-inline-flexbox;
        display: -webkit-inline-flex;
        display: inline-flex;
}

.search-ui-0 .layout.horizontal.search-ui {
  -ms-flex-direction: row;
        -webkit-flex-direction: row;
        flex-direction: row;
}

.search-ui-0 .layout.vertical.search-ui {
  -ms-flex-direction: column;
        -webkit-flex-direction: column;
        flex-direction: column;
}

.search-ui-0 .layout.wrap.search-ui {
  -ms-flex-wrap: wrap;
        -webkit-flex-wrap: wrap;
        flex-wrap: wrap;
}

.search-ui-0 .layout.no-wrap.search-ui {
  -ms-flex-wrap: nowrap;
        -webkit-flex-wrap: nowrap;
        flex-wrap: nowrap;
}

.search-ui-0 .layout.center.search-ui,.search-ui-0 .layout.center-center.search-ui {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.search-ui-0 .layout.center-justified.search-ui,.search-ui-0 .layout.center-center.search-ui {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.search-ui-0 .flex.search-ui {
  -ms-flex: 1 1 0.000000001px;
        -webkit-flex: 1;
        flex: 1;
        -webkit-flex-basis: 0.000000001px;
        flex-basis: 0.000000001px;
}

.search-ui-0 .flex-auto.search-ui {
  -ms-flex: 1 1 auto;
        -webkit-flex: 1 1 auto;
        flex: 1 1 auto;
}

.search-ui-0 .flex-none.search-ui {
  -ms-flex: none;
        -webkit-flex: none;
        flex: none;
}

.search-ui-0 .layout.start.search-ui {
  -ms-flex-align: start;
        -webkit-align-items: flex-start;
        align-items: flex-start;
}

.search-ui-0 .layout.center.search-ui,.search-ui-0 .layout.center-center.search-ui {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.search-ui-0 .layout.end.search-ui {
  -ms-flex-align: end;
        -webkit-align-items: flex-end;
        align-items: flex-end;
}

.search-ui-0 .layout.baseline.search-ui {
  -ms-flex-align: baseline;
        -webkit-align-items: baseline;
        align-items: baseline;
}

.search-ui-0 .layout.start-justified.search-ui {
  -ms-flex-pack: start;
        -webkit-justify-content: flex-start;
        justify-content: flex-start;
}

.search-ui-0 .layout.center-justified.search-ui,.search-ui-0 .layout.center-center.search-ui {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.search-ui-0 .layout.end-justified.search-ui {
  -ms-flex-pack: end;
        -webkit-justify-content: flex-end;
        justify-content: flex-end;
}

.search-ui-0 .layout.around-justified.search-ui {
  -ms-flex-pack: distribute;
        -webkit-justify-content: space-around;
        justify-content: space-around;
}

.search-ui-0 .layout.justified.search-ui {
  -ms-flex-pack: justify;
        -webkit-justify-content: space-between;
        justify-content: space-between;
}

.search-ui-0 .self-start.search-ui {
  -ms-align-self: flex-start;
        -webkit-align-self: flex-start;
        align-self: flex-start;
}

.search-ui-0 .self-center.search-ui {
  -ms-align-self: center;
        -webkit-align-self: center;
        align-self: center;
}

.search-ui-0 .self-end.search-ui {
  -ms-align-self: flex-end;
        -webkit-align-self: flex-end;
        align-self: flex-end;
}

.search-ui-0 .self-stretch.search-ui {
  -ms-align-self: stretch;
        -webkit-align-self: stretch;
        align-self: stretch;
}

.search-ui-0 .self-baseline.search-ui {
  -ms-align-self: baseline;
        -webkit-align-self: baseline;
        align-self: baseline;
}

.search-ui-0 .layout.start-aligned.search-ui {
  -ms-flex-line-pack: start;  
        -ms-align-content: flex-start;
        -webkit-align-content: flex-start;
        align-content: flex-start;
}

.search-ui-0 .layout.end-aligned.search-ui {
  -ms-flex-line-pack: end;  
        -ms-align-content: flex-end;
        -webkit-align-content: flex-end;
        align-content: flex-end;
}

.search-ui-0 .layout.center-aligned.search-ui {
  -ms-flex-line-pack: center;  
        -ms-align-content: center;
        -webkit-align-content: center;
        align-content: center;
}

.search-ui-0 .layout.between-aligned.search-ui {
  -ms-flex-line-pack: justify;  
        -ms-align-content: space-between;
        -webkit-align-content: space-between;
        align-content: space-between;
}

.search-ui-0 .layout.around-aligned.search-ui {
  -ms-flex-line-pack: distribute;  
        -ms-align-content: space-around;
        -webkit-align-content: space-around;
        align-content: space-around;
}

.search-ui-0 {
  display: block;
        min-width: 350px;
        height: 100%;
}

.search-ui-0 #content.search-ui {
  height: 100%;
}

.search-ui-0 #leftBarOverflow.search-ui {
  padding-right: 16px;
}

.search-ui-0 [wide].search-ui #leftBarOverflow.search-ui {
  overflow: auto;
}

.search-ui-0 #leftBar.search-ui {
  box-sizing: border-box;
        color: #666;
        font-size: 13px;
        padding: 16px 0 16px 16px;
        border-bottom: 1px solid #e5e5e5;
}

.search-ui-0 [wide].search-ui #leftBar.search-ui {
  display: -ms-flexbox;
        display: -webkit-flex;
        display: flex;

        -ms-flex-direction: column;
        -webkit-flex-direction: column;
        flex-direction: column;
        position: relative;
        width: 350px;
        margin-bottom: 0;
        padding-bottom: 0;
        border-bottom: none;
}

@media print {
.search-ui-0 #leftBar.search-ui {
  display: none !important;
}

}

.search-ui-0 #leftBar.search-ui h3.search-ui {
  text-transform: uppercase;
        font-size: 13px;
        font-weight: normal;
        margin-top: 6px;
        margin-bottom: 6px;
        cursor: default;
        display: inline-block;
        color: #a3a3a3;
}

.search-ui-0 #closeButton.search-ui {
  position: absolute;
        right: 0;
}

.search-ui-0 #searchTermsHeader.search-ui {
  position: relative;
}

.search-ui-0 #searchTermsQ.search-ui iron-icon.search-ui {
  height: 16px;
        padding: 0 2px 2px 2px;
        color: #a3a3a3;
}

.search-ui-0 #searchTermsHeader.search-ui:hover iron-icon.search-ui {
  color: #4285f4;
}

.search-ui-0 #closeLeftBar.search-ui {
  height: 16px;
        width: 16px;
        padding: 0 0 8px 0;
        cursor: pointer;
}

.search-ui-0 #compactQuery.search-ui iron-icon.search-ui {
  height: 20px;
}

.search-ui-0 #backToResultsButton.search-ui {
  color: #fff;
        background: #4285f4;
        margin-top: 20px;
        text-align: center;
}

.search-ui-0 #main.search-ui {
  display: block;
        min-height: calc(100vh - 127px);
}

@media print {
.search-ui-0 [wide].search-ui main.search-ui {
  margin-left: 0;
}

}</style><!-- Shady DOM styles for import-list --><!-- Shady DOM styles for iron-request --><!-- Shady DOM styles for iron-ajax --><!-- Shady DOM styles for paper-dialog --><!-- Shady DOM styles for coverage-stats --><!-- Shady DOM styles for iron-image --><!-- Shady DOM styles for iron-selector --><!-- Shady DOM styles for paper-card --><!-- Shady DOM styles for paper-checkbox --><!-- Shady DOM styles for landing-page --><!-- Shady DOM styles for remote-fetch --><!-- Shady DOM styles for paper-spinner --><style scope="paper-spinner-0">.paper-spinner-0 {
  display: inline-block;
        position: relative;
        width: 28px;
        height: 28px;
}

.paper-spinner-0 #spinnerContainer.paper-spinner {
  width: 100%;
        height: 100%;

        
        direction: ltr;
}

.paper-spinner-0 #spinnerContainer.active.paper-spinner {
  -webkit-animation: container-rotate-paper-spinner-0 1568ms linear infinite;
        animation: container-rotate-paper-spinner-0 1568ms linear infinite;
}

@-webkit-keyframes container-rotate-paper-spinner-0 {
to {
  -webkit-transform: rotate(360deg)
}

}

@keyframes container-rotate-paper-spinner-0 {
to {
  transform: rotate(360deg)
}

}

.paper-spinner-0 .spinner-layer.paper-spinner {
  position: absolute;
        width: 100%;
        height: 100%;
        opacity: 0;
        white-space: nowrap;
        color: #4285f4;
}

.paper-spinner-0 .layer-1.paper-spinner {
  color: #4285f4;
}

.paper-spinner-0 .layer-2.paper-spinner {
  color: #db4437;
}

.paper-spinner-0 .layer-3.paper-spinner {
  color: #f4b400;
}

.paper-spinner-0 .layer-4.paper-spinner {
  color: #0f9d58;
}

.paper-spinner-0 .active.paper-spinner .spinner-layer.paper-spinner {
  -webkit-animation-name: fill-unfill-rotate-paper-spinner-0;
        -webkit-animation-duration: 5332ms;
        -webkit-animation-timing-function: cubic-bezier(0.4, 0.0, 0.2, 1);
        -webkit-animation-iteration-count: infinite;
        animation-name: fill-unfill-rotate-paper-spinner-0;
        animation-duration: 5332ms;
        animation-timing-function: cubic-bezier(0.4, 0.0, 0.2, 1);
        animation-iteration-count: infinite;
        opacity: 1;
}

.paper-spinner-0 .active.paper-spinner .spinner-layer.layer-1.paper-spinner {
  -webkit-animation-name: fill-unfill-rotate-paper-spinner-0, layer-1-fade-in-out-paper-spinner-0;
        animation-name: fill-unfill-rotate-paper-spinner-0, layer-1-fade-in-out-paper-spinner-0;
}

.paper-spinner-0 .active.paper-spinner .spinner-layer.layer-2.paper-spinner {
  -webkit-animation-name: fill-unfill-rotate-paper-spinner-0, layer-2-fade-in-out-paper-spinner-0;
        animation-name: fill-unfill-rotate-paper-spinner-0, layer-2-fade-in-out-paper-spinner-0;
}

.paper-spinner-0 .active.paper-spinner .spinner-layer.layer-3.paper-spinner {
  -webkit-animation-name: fill-unfill-rotate-paper-spinner-0, layer-3-fade-in-out-paper-spinner-0;
        animation-name: fill-unfill-rotate-paper-spinner-0, layer-3-fade-in-out-paper-spinner-0;
}

.paper-spinner-0 .active.paper-spinner .spinner-layer.layer-4.paper-spinner {
  -webkit-animation-name: fill-unfill-rotate-paper-spinner-0, layer-4-fade-in-out-paper-spinner-0;
        animation-name: fill-unfill-rotate-paper-spinner-0, layer-4-fade-in-out-paper-spinner-0;
}

@-webkit-keyframes fill-unfill-rotate-paper-spinner-0 {
12.5% {
  -webkit-transform: rotate(135deg)
}

25% {
  -webkit-transform: rotate(270deg)
}

37.5% {
  -webkit-transform: rotate(405deg)
}

50% {
  -webkit-transform: rotate(540deg)
}

62.5% {
  -webkit-transform: rotate(675deg)
}

75% {
  -webkit-transform: rotate(810deg)
}

87.5% {
  -webkit-transform: rotate(945deg)
}

to {
  -webkit-transform: rotate(1080deg)
}

}

@keyframes fill-unfill-rotate-paper-spinner-0 {
12.5% {
  transform: rotate(135deg)
}

25% {
  transform: rotate(270deg)
}

37.5% {
  transform: rotate(405deg)
}

50% {
  transform: rotate(540deg)
}

62.5% {
  transform: rotate(675deg)
}

75% {
  transform: rotate(810deg)
}

87.5% {
  transform: rotate(945deg)
}

to {
  transform: rotate(1080deg)
}

}

@-webkit-keyframes layer-1-fade-in-out-paper-spinner-0 {
0% {
  opacity: 1
}

25% {
  opacity: 1
}

26% {
  opacity: 0
}

89% {
  opacity: 0
}

90% {
  opacity: 1
}

to {
  opacity: 1
}

}

@keyframes layer-1-fade-in-out-paper-spinner-0 {
0% {
  opacity: 1
}

25% {
  opacity: 1
}

26% {
  opacity: 0
}

89% {
  opacity: 0
}

90% {
  opacity: 1
}

to {
  opacity: 1
}

}

@-webkit-keyframes layer-2-fade-in-out-paper-spinner-0 {
0% {
  opacity: 0
}

15% {
  opacity: 0
}

25% {
  opacity: 1
}

50% {
  opacity: 1
}

51% {
  opacity: 0
}

to {
  opacity: 0
}

}

@keyframes layer-2-fade-in-out-paper-spinner-0 {
0% {
  opacity: 0
}

15% {
  opacity: 0
}

25% {
  opacity: 1
}

50% {
  opacity: 1
}

51% {
  opacity: 0
}

to {
  opacity: 0
}

}

@-webkit-keyframes layer-3-fade-in-out-paper-spinner-0 {
0% {
  opacity: 0
}

40% {
  opacity: 0
}

50% {
  opacity: 1
}

75% {
  opacity: 1
}

76% {
  opacity: 0
}

to {
  opacity: 0
}

}

@keyframes layer-3-fade-in-out-paper-spinner-0 {
0% {
  opacity: 0
}

40% {
  opacity: 0
}

50% {
  opacity: 1
}

75% {
  opacity: 1
}

76% {
  opacity: 0
}

to {
  opacity: 0
}

}

@-webkit-keyframes layer-4-fade-in-out-paper-spinner-0 {
0% {
  opacity: 0
}

65% {
  opacity: 0
}

75% {
  opacity: 1
}

90% {
  opacity: 1
}

to {
  opacity: 0
}

}

@keyframes layer-4-fade-in-out-paper-spinner-0 {
0% {
  opacity: 0
}

65% {
  opacity: 0
}

75% {
  opacity: 1
}

90% {
  opacity: 1
}

to {
  opacity: 0
}

}

.paper-spinner-0 .circle-clipper.paper-spinner {
  display: inline-block;
        position: relative;
        width: 50%;
        height: 100%;
        overflow: hidden;
}

.paper-spinner-0 .spinner-layer.paper-spinner::after {
  left: 45%;
        width: 10%;
        border-top-style: solid;
}

.paper-spinner-0 .spinner-layer.paper-spinner::after,.paper-spinner-0 .circle-clipper.paper-spinner::after {
  content: '';
        box-sizing: border-box;
        position: absolute;
        top: 0;
        border-width: 3px;
        border-radius: 50%;
}

.paper-spinner-0 .circle-clipper.paper-spinner::after {
  bottom: 0;
        width: 200%;
        border-style: solid;
        border-bottom-color: transparent !important;
}

.paper-spinner-0 .circle-clipper.left.paper-spinner::after {
  left: 0;
        border-right-color: transparent !important;
        -webkit-transform: rotate(129deg);
        transform: rotate(129deg);
}

.paper-spinner-0 .circle-clipper.right.paper-spinner::after {
  left: -100%;
        border-left-color: transparent !important;
        -webkit-transform: rotate(-129deg);
        transform: rotate(-129deg);
}

.paper-spinner-0 .active.paper-spinner .gap-patch.paper-spinner::after,.paper-spinner-0 .active.paper-spinner .circle-clipper.paper-spinner::after {
  -webkit-animation-duration: 1333ms;
        -webkit-animation-timing-function: cubic-bezier(0.4, 0.0, 0.2, 1);
        -webkit-animation-iteration-count: infinite;
        animation-duration: 1333ms;
        animation-timing-function: cubic-bezier(0.4, 0.0, 0.2, 1);
        animation-iteration-count: infinite;
}

.paper-spinner-0 .active.paper-spinner .circle-clipper.left.paper-spinner::after {
  -webkit-animation-name: left-spin-paper-spinner-0;
        animation-name: left-spin-paper-spinner-0;
}

.paper-spinner-0 .active.paper-spinner .circle-clipper.right.paper-spinner::after {
  -webkit-animation-name: right-spin-paper-spinner-0;
        animation-name: right-spin-paper-spinner-0;
}

@-webkit-keyframes left-spin-paper-spinner-0 {
0% {
  -webkit-transform: rotate(130deg)
}

50% {
  -webkit-transform: rotate(-5deg)
}

to {
  -webkit-transform: rotate(130deg)
}

}

@keyframes left-spin-paper-spinner-0 {
0% {
  transform: rotate(130deg)
}

50% {
  transform: rotate(-5deg)
}

to {
  transform: rotate(130deg)
}

}

@-webkit-keyframes right-spin-paper-spinner-0 {
0% {
  -webkit-transform: rotate(-130deg)
}

50% {
  -webkit-transform: rotate(5deg)
}

to {
  -webkit-transform: rotate(-130deg)
}

}

@keyframes right-spin-paper-spinner-0 {
0% {
  transform: rotate(-130deg)
}

50% {
  transform: rotate(5deg)
}

to {
  transform: rotate(-130deg)
}

}

.paper-spinner-0 #spinnerContainer.cooldown.paper-spinner {
  -webkit-animation: container-rotate-paper-spinner-0 1568ms linear infinite, fade-out-paper-spinner-0 400ms cubic-bezier(0.4, 0.0, 0.2, 1);
        animation: container-rotate-paper-spinner-0 1568ms linear infinite, fade-out-paper-spinner-0 400ms cubic-bezier(0.4, 0.0, 0.2, 1);
}

@-webkit-keyframes fade-out-paper-spinner-0 {
0% {
  opacity: 1
}

to {
  opacity: 0
}

}

@keyframes fade-out-paper-spinner-0 {
0% {
  opacity: 1
}

to {
  opacity: 0
}

}</style><!-- Shady DOM styles for paper-spinner-lite --><!-- Shady DOM styles for concept-card --><!-- Shady DOM styles for concept-mention --><!-- Shady DOM styles for fterm-concept-card --><!-- Shady DOM styles for fterm-concept-mention --><!-- Shady DOM styles for paper-fab --><!-- Shady DOM styles for paper-icon-button --><!-- Shady DOM styles for paper-icon-button-light --><!-- Shady DOM styles for classification-tree --><!-- Shady DOM styles for classification-viewer --><style scope="classification-viewer">.table.classification-viewer {
  border: 1px solid #eaeaea;
        display: block;
}

.more.classification-viewer {
  padding: 6px 12px;
        cursor: pointer;
        font-style: italic;
        color: #aaa;
}

.more.classification-viewer:hover {
  background: #fafafa;
}

.chip.classification-viewer {
  display: inline-block;
        padding: 4px 6px;
        border-radius: 2px;
}

.chip[additional].classification-viewer {
  font-style: italic;
}</style><!-- Shady DOM styles for image-carousel --><!-- Shady DOM styles for image-viewer --><!-- Shady DOM styles for patent-result --><style scope="patent-result">#wrapper.patent-result {
  padding: 20px;
        max-width: 1050px;
        outline: 0;
}

.scroll-target.patent-result:before {
  display: block;
        content: '';
        visibility: hidden;
}

h1.patent-result {
  font-weight: 500;
        font-size: 20px;
        margin: 0 0 20px 0;
}

h3.patent-result {
  font-size: 15px;
        font-weight: 500;
        margin: 28px 0 18px 0;
}

dl.patent-result {
  line-height: 1.5;
        font-size: 0;
}

dt.patent-result,dd.patent-result {
  display: inline;
        font-size: 13px;
        margin: 0;
}

dt.patent-result {
  font-weight: 500;
        color: #000;
}

dt.patent-result::after {
  content: ': ';
}

dd.patent-result {
  color: #aaa;
}

dd.patent-result ~ dd.patent-result::before {
  content: ', ';
}

dt.patent-result + dd.patent-result::before {
  content: '';
}

dd.patent-result ~ dt.patent-result::before {
  display: block;
        content: '';
        margin-top: 8px;
}

dd.patent-result core-icon.patent-result {
  width: 14px;
        height: 14px;
        vertical-align: top;
        color: inherit;
}

.table.patent-result {
  display: table;
        border-collapse: collapse;
        border-spacing: 0;
        border: 1px solid #eaeaea;
        width: 100%;
        box-sizing: border-box;
}

.tr.patent-result {
  display: table-row;
        border-bottom: 1px solid #eaeaea;
}

.th.patent-result,.td.patent-result {
  display: table-cell;
        padding: 10px 12px;
}

.thead.patent-result {
  display: table-header-group;
}

.tbody.patent-result {
  display: table-row-group;
}

.th.patent-result {
  text-align: left;
        white-space: nowrap;
        background-color: #fafafa;
        color: #757575;
}

.td.patent-result {
  vertical-align: top;
}

.td.patent-result > p.patent-result {
  margin-top: 0;
}

.nowrap.patent-result {
  white-space: nowrap;
}

.knowledge-card.patent-result {
  border: 1px solid #e5e5e5;
        border-radius: 2px;
}

.knowledge-card.patent-result > header.patent-result {
  -webkit-font-smoothing: antialiased;
        -moz-osx-font-smoothing: grayscale;
        background: #4285f4;
        color: #fff;
        padding: 20px 12px;
}

.knowledge-card.patent-result > header.patent-result state-modifier.patent-result,.knowledge-card.patent-result > header.patent-result a.patent-result {
  color: inherit;
        margin: 0 12px 0 0;
        display: inline-block;
}

.knowledge-card.patent-result > header.patent-result h2.patent-result,.knowledge-card.patent-result > header.patent-result .tagline.patent-result {
  margin: 0;
        font-weight: normal;
        padding-top: 4px;
}

.knowledge-card.patent-result > header.patent-result h2.patent-result {
  font-size: 20px;
}

.knowledge-card.patent-result .knowledge-card-action-bar.patent-result {
  background: #3367d6;
        padding: 10px 12px;
        margin: 20px -12px -20px;
}

.knowledge-card.patent-result .knowledge-card-action-bar.patent-result core-icon.patent-result {
  margin-right: 12px;
        width: 24px;
        height: 24px;
}

.knowledge-card.patent-result dl.patent-result {
  padding: 16px 12px;
}

.knowledge-card.patent-result hr.patent-result {
  border: none;
        height: 1px;
        background: #e5e5e5;
        margin: 0;
}

.main-link.patent-result {
  margin: 14px;
        font-weight: 500;
}

.footer.patent-result h3.patent-result state-modifier.patent-result,.footer.patent-result h3.patent-result a.patent-result {
  float: right;
        font-weight: 400;
        font-size: 13px;
        line-height: 24px;
}

.footer.patent-result dl.patent-result {
  margin-left: 30px;
}

.footer.patent-result > ul.patent-result > li.patent-result {
  margin-bottom: 8px;
}

h3.patent-result .righthead.patent-result {
  color: #aaa;
        font-weight: 400;
        font-size: 13px;
        cursor: help;
}

.tooltip-hint.patent-result {
  border-bottom: 1px dotted #ccc;
}

[marginRight].patent-result {
  margin-right: 60px;
}

.flex-width.patent-result {
  min-width: 0;
        word-break: break-word;
}

.responsive-table.patent-result {
  width: 100%;
        overflow-x: auto;
        overflow-y: hidden;
}

.layout.horizontal.patent-result,.layout.vertical.patent-result {
  display: -ms-flexbox;
        display: -webkit-flex;
        display: flex;
}

.layout.inline.patent-result {
  display: -ms-inline-flexbox;
        display: -webkit-inline-flex;
        display: inline-flex;
}

.layout.horizontal.patent-result {
  -ms-flex-direction: row;
        -webkit-flex-direction: row;
        flex-direction: row;
}

.layout.vertical.patent-result {
  -ms-flex-direction: column;
        -webkit-flex-direction: column;
        flex-direction: column;
}

.layout.wrap.patent-result {
  -ms-flex-wrap: wrap;
        -webkit-flex-wrap: wrap;
        flex-wrap: wrap;
}

.layout.no-wrap.patent-result {
  -ms-flex-wrap: nowrap;
        -webkit-flex-wrap: nowrap;
        flex-wrap: nowrap;
}

.layout.center.patent-result,.layout.center-center.patent-result {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.layout.center-justified.patent-result,.layout.center-center.patent-result {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.flex.patent-result {
  -ms-flex: 1 1 0.000000001px;
        -webkit-flex: 1;
        flex: 1;
        -webkit-flex-basis: 0.000000001px;
        flex-basis: 0.000000001px;
}

.flex-auto.patent-result {
  -ms-flex: 1 1 auto;
        -webkit-flex: 1 1 auto;
        flex: 1 1 auto;
}

.flex-none.patent-result {
  -ms-flex: none;
        -webkit-flex: none;
        flex: none;
}

.layout.start.patent-result {
  -ms-flex-align: start;
        -webkit-align-items: flex-start;
        align-items: flex-start;
}

.layout.center.patent-result,.layout.center-center.patent-result {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.layout.end.patent-result {
  -ms-flex-align: end;
        -webkit-align-items: flex-end;
        align-items: flex-end;
}

.layout.baseline.patent-result {
  -ms-flex-align: baseline;
        -webkit-align-items: baseline;
        align-items: baseline;
}

.layout.start-justified.patent-result {
  -ms-flex-pack: start;
        -webkit-justify-content: flex-start;
        justify-content: flex-start;
}

.layout.center-justified.patent-result,.layout.center-center.patent-result {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.layout.end-justified.patent-result {
  -ms-flex-pack: end;
        -webkit-justify-content: flex-end;
        justify-content: flex-end;
}

.layout.around-justified.patent-result {
  -ms-flex-pack: distribute;
        -webkit-justify-content: space-around;
        justify-content: space-around;
}

.layout.justified.patent-result {
  -ms-flex-pack: justify;
        -webkit-justify-content: space-between;
        justify-content: space-between;
}

.self-start.patent-result {
  -ms-align-self: flex-start;
        -webkit-align-self: flex-start;
        align-self: flex-start;
}

.self-center.patent-result {
  -ms-align-self: center;
        -webkit-align-self: center;
        align-self: center;
}

.self-end.patent-result {
  -ms-align-self: flex-end;
        -webkit-align-self: flex-end;
        align-self: flex-end;
}

.self-stretch.patent-result {
  -ms-align-self: stretch;
        -webkit-align-self: stretch;
        align-self: stretch;
}

.self-baseline.patent-result {
  -ms-align-self: baseline;
        -webkit-align-self: baseline;
        align-self: baseline;
}

.layout.start-aligned.patent-result {
  -ms-flex-line-pack: start;  
        -ms-align-content: flex-start;
        -webkit-align-content: flex-start;
        align-content: flex-start;
}

.layout.end-aligned.patent-result {
  -ms-flex-line-pack: end;  
        -ms-align-content: flex-end;
        -webkit-align-content: flex-end;
        align-content: flex-end;
}

.layout.center-aligned.patent-result {
  -ms-flex-line-pack: center;  
        -ms-align-content: center;
        -webkit-align-content: center;
        align-content: center;
}

.layout.between-aligned.patent-result {
  -ms-flex-line-pack: justify;  
        -ms-align-content: space-between;
        -webkit-align-content: space-between;
        align-content: space-between;
}

.layout.around-aligned.patent-result {
  -ms-flex-line-pack: distribute;  
        -ms-align-content: space-around;
        -webkit-align-content: space-around;
        align-content: space-around;
}

.flex.patent-result,.flex-1.patent-result {
  -ms-flex: 1 1 0.000000001px;
        -webkit-flex: 1;
        flex: 1;
        -webkit-flex-basis: 0.000000001px;
        flex-basis: 0.000000001px;
}

.flex-2.patent-result {
  -ms-flex: 2;
        -webkit-flex: 2;
        flex: 2;
}

.flex-3.patent-result {
  -ms-flex: 3;
        -webkit-flex: 3;
        flex: 3;
}

.flex-4.patent-result {
  -ms-flex: 4;
        -webkit-flex: 4;
        flex: 4;
}

.flex-5.patent-result {
  -ms-flex: 5;
        -webkit-flex: 5;
        flex: 5;
}

.flex-6.patent-result {
  -ms-flex: 6;
        -webkit-flex: 6;
        flex: 6;
}

.flex-7.patent-result {
  -ms-flex: 7;
        -webkit-flex: 7;
        flex: 7;
}

.flex-8.patent-result {
  -ms-flex: 8;
        -webkit-flex: 8;
        flex: 8;
}

.flex-9.patent-result {
  -ms-flex: 9;
        -webkit-flex: 9;
        flex: 9;
}

.flex-10.patent-result {
  -ms-flex: 10;
        -webkit-flex: 10;
        flex: 10;
}

.flex-11.patent-result {
  -ms-flex: 11;
        -webkit-flex: 11;
        flex: 11;
}

.flex-12.patent-result {
  -ms-flex: 12;
        -webkit-flex: 12;
        flex: 12;
}

@media print {
#text.patent-result {
  flex-direction: column-reverse !important;
}

}

patent-result-ref-list.patent-result {
  width: 100%;
}

patent-result-ref-list.patent-result + patent-result-ref-list.patent-result {
  margin-left: 10px;
}

[narrow].patent-result {
  flex-direction: column-reverse !important;
}

.famheader.patent-result {
  text-align: left;
        white-space: nowrap;
        background-color: #fafafa;
        color: #757575;
}

.collapse.patent-result {
  color: #616161;
        font-size: 13px;
        cursor: pointer;
}

#imageDrag.patent-result {
  position: fixed;
        cursor: col-resize;
}

#imageDrag.patent-result div.patent-result {
  width: 4px;
        height: 100%;
        margin: 0 4px;
        background: #eee;
}

#imageDrag.patent-result:hover {
  background: #eee;
}

#imageDrag.patent-result:hover div.patent-result {
  background: #ccc;
}

.imageWrapper.patent-result {
  position: fixed;
        top: 60px;
        background: white;
        margin-left: 12px;
}

#notices.patent-result {
  padding: 20px;
}

#figures.patent-result {
  background: #efefef;
}

.headerButton.patent-result {
  color: #3c80f6;
        cursor: pointer;
        display: inline-block;
        padding: 0 14px;
        line-height: 27px;
}

.headerButton.patent-result iron-icon.patent-result {
  width: 16px;
        height: 16px;
}

.tr.conceptDomain.patent-result {
  display: table-row;
}

.responsive-grid.patent-result {
  display: grid;
        gap: 8px;
        grid-template-columns: none;
}</style><!-- Shady DOM styles for family-viewer --><!-- Shady DOM styles for app-row --><!-- Shady DOM styles for application-timeline --><style scope="application-timeline">.layout.horizontal.application-timeline,.layout.vertical.application-timeline {
  display: -ms-flexbox;
        display: -webkit-flex;
        display: flex;
}

.layout.inline.application-timeline {
  display: -ms-inline-flexbox;
        display: -webkit-inline-flex;
        display: inline-flex;
}

.layout.horizontal.application-timeline {
  -ms-flex-direction: row;
        -webkit-flex-direction: row;
        flex-direction: row;
}

.layout.vertical.application-timeline {
  -ms-flex-direction: column;
        -webkit-flex-direction: column;
        flex-direction: column;
}

.layout.wrap.application-timeline {
  -ms-flex-wrap: wrap;
        -webkit-flex-wrap: wrap;
        flex-wrap: wrap;
}

.layout.no-wrap.application-timeline {
  -ms-flex-wrap: nowrap;
        -webkit-flex-wrap: nowrap;
        flex-wrap: nowrap;
}

.layout.center.application-timeline,.layout.center-center.application-timeline {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.layout.center-justified.application-timeline,.layout.center-center.application-timeline {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.flex.application-timeline {
  -ms-flex: 1 1 0.000000001px;
        -webkit-flex: 1;
        flex: 1;
        -webkit-flex-basis: 0.000000001px;
        flex-basis: 0.000000001px;
}

.flex-auto.application-timeline {
  -ms-flex: 1 1 auto;
        -webkit-flex: 1 1 auto;
        flex: 1 1 auto;
}

.flex-none.application-timeline {
  -ms-flex: none;
        -webkit-flex: none;
        flex: none;
}

.wrap.application-timeline {
  padding: 16px 12px;
}

.event.application-timeline {
  padding: 3px 0;
}

[date].application-timeline {
  color: #5f6368;
        font-size: 13px;
        padding: 3px 0;
        width: 70px;
        line-height: 16px;
}

[critical].application-timeline [date].application-timeline {
  color: #202124;
        font-weight: 500;
        line-height: 18px;
}

.line.application-timeline {
  margin: 9px 6px;
        border-radius: 4px;
        background: #dadce0;
        width: 4px;
        height: 4px;
}

.dot.application-timeline {
  margin: 3px 0;
        border-radius: 4px;
        background: #dadce0;
        width: 4px;
        height: 4px;
        display: inline-block;
}

[critical].application-timeline .line.application-timeline {
  background: #4285f4;
}

[current].application-timeline .line.application-timeline {
  background: #81c995;
}

[current][critical].application-timeline .line.application-timeline {
  background: #81c995;
}

.title.application-timeline {
  font-size: 13px;
        padding: 3px 0;
        line-height: 16px;
}

.title-text.application-timeline {
  color: #5f6368;
}

.description-item.application-timeline {
  font-size: 12px;
        padding: 6px 0;
        max-width: 300px;
}

#legal.application-timeline {
  height: 14px;
        width: 14px;
        padding: 0 3px 1px 3px;
        vertical-align: text-bottom;
}

#icon.application-timeline {
  height: 12px;
        width: 12px;
        padding: 0 3px 3px 3px;
        vertical-align: text-bottom;
}

.unknown.application-timeline {
  font-style: italic;
}

.not_active.application-timeline {
  text-decoration: line-through;
}

.active.application-timeline {
  font-weight: 500;
}

#cc[this].application-timeline {
  color: #202124 !important;
        text-decoration: underline;
}

#cc[this].not_active.application-timeline {
  text-decoration: line-through underline;
}

hr.application-timeline {
  border: none;
        height: 1px;
        background: #e5e5e5;
        margin: 10px 0 16px 0;
}

.header.application-timeline {
  font-size: 13px;
        line-height: 20px;
        font-weight: 500;
        color: #202124;
        padding-bottom: 8px;
}

.block.application-timeline {
  padding: 0 6px 0 0;
}

.year.application-timeline {
  color: #5f6368;
        font-size: 13px;
        font-weight: 500;
        padding: 3px 1px;
        width: 70px;
}

.expand.application-timeline {
  cursor: pointer;
        padding: 6px 0 3px 0;
}</style><!-- Shady DOM styles for patent-text --><style scope="patent-text">.layout.horizontal.patent-text,.layout.vertical.patent-text {
  display: -ms-flexbox;
        display: -webkit-flex;
        display: flex;
}

.layout.inline.patent-text {
  display: -ms-inline-flexbox;
        display: -webkit-inline-flex;
        display: inline-flex;
}

.layout.horizontal.patent-text {
  -ms-flex-direction: row;
        -webkit-flex-direction: row;
        flex-direction: row;
}

.layout.vertical.patent-text {
  -ms-flex-direction: column;
        -webkit-flex-direction: column;
        flex-direction: column;
}

.layout.wrap.patent-text {
  -ms-flex-wrap: wrap;
        -webkit-flex-wrap: wrap;
        flex-wrap: wrap;
}

.layout.no-wrap.patent-text {
  -ms-flex-wrap: nowrap;
        -webkit-flex-wrap: nowrap;
        flex-wrap: nowrap;
}

.layout.center.patent-text,.layout.center-center.patent-text {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.layout.center-justified.patent-text,.layout.center-center.patent-text {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.flex.patent-text {
  -ms-flex: 1 1 0.000000001px;
        -webkit-flex: 1;
        flex: 1;
        -webkit-flex-basis: 0.000000001px;
        flex-basis: 0.000000001px;
}

.flex-auto.patent-text {
  -ms-flex: 1 1 auto;
        -webkit-flex: 1 1 auto;
        flex: 1 1 auto;
}

.flex-none.patent-text {
  -ms-flex: none;
        -webkit-flex: none;
        flex: none;
}

.layout.start.patent-text {
  -ms-flex-align: start;
        -webkit-align-items: flex-start;
        align-items: flex-start;
}

.layout.center.patent-text,.layout.center-center.patent-text {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.layout.end.patent-text {
  -ms-flex-align: end;
        -webkit-align-items: flex-end;
        align-items: flex-end;
}

.layout.baseline.patent-text {
  -ms-flex-align: baseline;
        -webkit-align-items: baseline;
        align-items: baseline;
}

.layout.start-justified.patent-text {
  -ms-flex-pack: start;
        -webkit-justify-content: flex-start;
        justify-content: flex-start;
}

.layout.center-justified.patent-text,.layout.center-center.patent-text {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.layout.end-justified.patent-text {
  -ms-flex-pack: end;
        -webkit-justify-content: flex-end;
        justify-content: flex-end;
}

.layout.around-justified.patent-text {
  -ms-flex-pack: distribute;
        -webkit-justify-content: space-around;
        justify-content: space-around;
}

.layout.justified.patent-text {
  -ms-flex-pack: justify;
        -webkit-justify-content: space-between;
        justify-content: space-between;
}

.self-start.patent-text {
  -ms-align-self: flex-start;
        -webkit-align-self: flex-start;
        align-self: flex-start;
}

.self-center.patent-text {
  -ms-align-self: center;
        -webkit-align-self: center;
        align-self: center;
}

.self-end.patent-text {
  -ms-align-self: flex-end;
        -webkit-align-self: flex-end;
        align-self: flex-end;
}

.self-stretch.patent-text {
  -ms-align-self: stretch;
        -webkit-align-self: stretch;
        align-self: stretch;
}

.self-baseline.patent-text {
  -ms-align-self: baseline;
        -webkit-align-self: baseline;
        align-self: baseline;
}

.layout.start-aligned.patent-text {
  -ms-flex-line-pack: start;  
        -ms-align-content: flex-start;
        -webkit-align-content: flex-start;
        align-content: flex-start;
}

.layout.end-aligned.patent-text {
  -ms-flex-line-pack: end;  
        -ms-align-content: flex-end;
        -webkit-align-content: flex-end;
        align-content: flex-end;
}

.layout.center-aligned.patent-text {
  -ms-flex-line-pack: center;  
        -ms-align-content: center;
        -webkit-align-content: center;
        align-content: center;
}

.layout.between-aligned.patent-text {
  -ms-flex-line-pack: justify;  
        -ms-align-content: space-between;
        -webkit-align-content: space-between;
        align-content: space-between;
}

.layout.around-aligned.patent-text {
  -ms-flex-line-pack: distribute;  
        -ms-align-content: space-around;
        -webkit-align-content: space-around;
        align-content: space-around;
}

patent-text {
  display: block;
}

#text.patent-text {
  line-height: 20px;
        min-width: 0;
}

#margin.patent-text {
  width: 16px;
}

#createHighlight.patent-text {
  position: relative;
        width: 16px;
        height: 16px;
        cursor: pointer;
}

.notranslate.patent-text > .google-src-text.patent-text {
  display: none;
}

.notranslate.patent-text:hover,mark.patent-text {
  background-color: #fbfcae;
}

.description-paragraph.patent-text {
  margin-block-start: 1em;
        margin-block-end: 1em;
}

.claim-text.patent-text {
  margin: 14px 0 0 20px;
}

.claim-dependent.patent-text {
  margin-left: 40px;
        color: #888;
}

[hide-claims].patent-text .claim-dependent.patent-text .claim-text.patent-text {
  height: 0;
        width: 0;
        visibility: hidden;
}

[hide-claims].patent-text .claim-dependent.patent-text .claim.patent-text:after {
  content: ""
}

.patent-full-image.patent-text {
  max-width: 100%;
        height: auto;
}

.small-patent-image.patent-text {
  display: inline;
}

ul.patent-text {
  padding: 0 0 0 50px;
}

li.patent-text {
  list-style-type: none;
        position: relative;
}

ol.claims.patent-text li.patent-text {
  list-style-type: decimal;
        font-weight: bold;
}

ol.claims.patent-text li.patent-text > *.patent-text {
  font-weight: normal;
}

li.patent-text > para-num.patent-text {
  position: absolute;
        left: -45px;
        color: #888;
}

mark.patent-text {
  background: #fff59d;
}

mark.searchText.patent-text {
  background: #80deea;
}

.searchTextHeader.patent-text {
  font-weight: 500;
}

pre.patent-text {
  white-space: pre-wrap;
}

heading.patent-text {
  display: block;
        font-weight: 500;
        font-size: 13px;
}

#comment.patent-text {
  width: 16px;
        height: 16px;
}

[has-search][no-results].patent-text {
  display: none;
}

.highlight.patent-text {
  background-color: #fef7e0;
}</style><!-- Shady DOM styles for para-num --><!-- Shady DOM styles for result-container --><style scope="result-container">paper-spinner.result-container {
  position: absolute;
        
        top: calc(50%  + 56px - 16px);
        left: calc(50% + 175px - 16px);
}</style><!-- Shady DOM styles for scholar-result --><!-- Shady DOM styles for search-result --><style scope="search-result">.layout.horizontal.search-result,.layout.vertical.search-result {
  display: -ms-flexbox;
        display: -webkit-flex;
        display: flex;
}

.layout.inline.search-result {
  display: -ms-inline-flexbox;
        display: -webkit-inline-flex;
        display: inline-flex;
}

.layout.horizontal.search-result {
  -ms-flex-direction: row;
        -webkit-flex-direction: row;
        flex-direction: row;
}

.layout.vertical.search-result {
  -ms-flex-direction: column;
        -webkit-flex-direction: column;
        flex-direction: column;
}

.layout.wrap.search-result {
  -ms-flex-wrap: wrap;
        -webkit-flex-wrap: wrap;
        flex-wrap: wrap;
}

.layout.no-wrap.search-result {
  -ms-flex-wrap: nowrap;
        -webkit-flex-wrap: nowrap;
        flex-wrap: nowrap;
}

.layout.center.search-result,.layout.center-center.search-result {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.layout.center-justified.search-result,.layout.center-center.search-result {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.flex.search-result {
  -ms-flex: 1 1 0.000000001px;
        -webkit-flex: 1;
        flex: 1;
        -webkit-flex-basis: 0.000000001px;
        flex-basis: 0.000000001px;
}

.flex-auto.search-result {
  -ms-flex: 1 1 auto;
        -webkit-flex: 1 1 auto;
        flex: 1 1 auto;
}

.flex-none.search-result {
  -ms-flex: none;
        -webkit-flex: none;
        flex: none;
}

.layout.start.search-result {
  -ms-flex-align: start;
        -webkit-align-items: flex-start;
        align-items: flex-start;
}

.layout.center.search-result,.layout.center-center.search-result {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.layout.end.search-result {
  -ms-flex-align: end;
        -webkit-align-items: flex-end;
        align-items: flex-end;
}

.layout.baseline.search-result {
  -ms-flex-align: baseline;
        -webkit-align-items: baseline;
        align-items: baseline;
}

.layout.start-justified.search-result {
  -ms-flex-pack: start;
        -webkit-justify-content: flex-start;
        justify-content: flex-start;
}

.layout.center-justified.search-result,.layout.center-center.search-result {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.layout.end-justified.search-result {
  -ms-flex-pack: end;
        -webkit-justify-content: flex-end;
        justify-content: flex-end;
}

.layout.around-justified.search-result {
  -ms-flex-pack: distribute;
        -webkit-justify-content: space-around;
        justify-content: space-around;
}

.layout.justified.search-result {
  -ms-flex-pack: justify;
        -webkit-justify-content: space-between;
        justify-content: space-between;
}

.self-start.search-result {
  -ms-align-self: flex-start;
        -webkit-align-self: flex-start;
        align-self: flex-start;
}

.self-center.search-result {
  -ms-align-self: center;
        -webkit-align-self: center;
        align-self: center;
}

.self-end.search-result {
  -ms-align-self: flex-end;
        -webkit-align-self: flex-end;
        align-self: flex-end;
}

.self-stretch.search-result {
  -ms-align-self: stretch;
        -webkit-align-self: stretch;
        align-self: stretch;
}

.self-baseline.search-result {
  -ms-align-self: baseline;
        -webkit-align-self: baseline;
        align-self: baseline;
}

.layout.start-aligned.search-result {
  -ms-flex-line-pack: start;  
        -ms-align-content: flex-start;
        -webkit-align-content: flex-start;
        align-content: flex-start;
}

.layout.end-aligned.search-result {
  -ms-flex-line-pack: end;  
        -ms-align-content: flex-end;
        -webkit-align-content: flex-end;
        align-content: flex-end;
}

.layout.center-aligned.search-result {
  -ms-flex-line-pack: center;  
        -ms-align-content: center;
        -webkit-align-content: center;
        align-content: center;
}

.layout.between-aligned.search-result {
  -ms-flex-line-pack: justify;  
        -ms-align-content: space-between;
        -webkit-align-content: space-between;
        align-content: space-between;
}

.layout.around-aligned.search-result {
  -ms-flex-line-pack: distribute;  
        -ms-align-content: space-around;
        -webkit-align-content: space-around;
        align-content: space-around;
}

search-result {
  display: block;
        font-size: 13px;
}

#mainContainer.search-result {
  min-height: calc(100vh - 127px);
        color: #333;
}

#compactQuery.search-result {
  padding: 18px 20px 10px 20px;
        cursor: pointer;
}

#compactQuery.search-result > span.search-result {
  padding: 0 14px 0 0;
}

#compactQuery.search-result iron-icon.search-result {
  padding-right: 6px;
        height: 16px;
        width: 16px;
}</style><!-- Shady DOM styles for webdoc-result --><!-- Shady DOM styles for iron-pages --><!-- Shady DOM styles for paper-tab --><!-- Shady DOM styles for paper-tabs --><!-- Shady DOM styles for debug-info --><!-- Shady DOM styles for search-paging --><!-- Shady DOM styles for search-result-item --><!-- Shady DOM styles for search-result-image --><!-- Shady DOM styles for search-result-entities --><!-- Shady DOM styles for search-result-measures --><!-- Shady DOM styles for search-results --><!-- Shady DOM styles for summary-box --><!-- Shady DOM styles for timeline-page --><!-- Shady DOM styles for iron-localstorage --><!-- Shady DOM styles for search-app --><style scope="search-app">search-app {
  display: block;
        height: 100%;
}

#toast.search-app {
  min-height: 64px;
        min-width: 742px;
}</style><style>body {transition: opacity ease-in 0.2s; } 
body[unresolved] {opacity: 0; display: block; overflow: hidden; position: relative; } 
</style>
    <title>TW200418849A - Compositions and methods for hydroxylating epothilones 
      - Google Patents</title>

    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta charset="UTF-8">
    <meta name="referrer" content="origin-when-crossorigin">
    <link rel="canonical" href="https://patents.google.com/patent/TW200418849A/en">
    <meta name="description" content="
     Isolated nucleic acid sequences and polypeptides encoded thereby for epothilone B hydroxylase and mutants and variants thereof and a ferredoxin located downstream from the epothilone B hydroxylase gene are provided. Also provided are vectors and cells containing these vectors. In addition, methods for producing recombinant microorganisms, methods for using these recombinant microorganism to produce hydroxyalkyl-bearing epothilones and an epothilone analog produced by a mutant of epothilone B hydroxylase are provided. 
   
   ">
    <meta name="DC.type" content="patent">
    <meta name="DC.title" content="Compositions and methods for hydroxylating epothilones 
     ">
    <meta name="DC.date" content="2003-03-17" scheme="dateSubmitted">
    <meta name="DC.description" content="
     Isolated nucleic acid sequences and polypeptides encoded thereby for epothilone B hydroxylase and mutants and variants thereof and a ferredoxin located downstream from the epothilone B hydroxylase gene are provided. Also provided are vectors and cells containing these vectors. In addition, methods for producing recombinant microorganisms, methods for using these recombinant microorganism to produce hydroxyalkyl-bearing epothilones and an epothilone analog produced by a mutant of epothilone B hydroxylase are provided. 
   
   ">
    <meta name="citation_patent_application_number" content="TW:092105814A">
    <meta name="citation_pdf_url" content="https://patentimages.storage.googleapis.com/61/12/23/f04976b3fac0fc/TW200418849A.pdf">
    <meta name="citation_patent_publication_number" content="TW:200418849:A">
    <meta name="DC.date" content="2004-10-01">
    <meta name="DC.contributor" content="Jonathan D Basch" scheme="inventor">
    <meta name="DC.contributor" content="Shu-Jen Chiang" scheme="inventor">
    <meta name="DC.contributor" content="Suo W Liu" scheme="inventor">
    <meta name="DC.contributor" content="Akbar Nayeem" scheme="inventor">
    <meta name="DC.contributor" content="Yuhua Sun" scheme="inventor">
    <meta name="DC.contributor" content="Li You" scheme="inventor">
    <meta name="DC.contributor" content="Bristol Myers Squibb Co" scheme="assignee">
    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:400,400italic,500,500italic,700">
    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Product+Sans">
    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Google+Sans:wght@400;500;700">

    <style>
      body { transition: none; }
    </style>

    <script src="https://apis.google.com/_/scs/abc-static/_/js/k=gapi.lb.en.IKZeRvoAYNY.O/m=client/rt=j/sv=1/d=1/ed=1/am=AAAQ/rs=AHpOoo-SMWTzMRJrTty6iE5dL_aWGYOnuw/cb=gapi.loaded_0?le=scs" async=""></script><script>
      window.version = 'patent-search.search_20240531_RC00';

      function sendFeedback() {
        userfeedback.api.startFeedback({
          'productId': '713680',
          'bucket': 'patent-search-web',
          'productVersion': window.version,
        });
      }

      window.experiments = {};
      window.experiments.patentCountries = "ae,ag,al,am,ao,ap,ar,at,au,aw,az,ba,bb,bd,be,bf,bg,bh,bj,bn,bo,br,bw,bx,by,bz,ca,cf,cg,ch,ci,cl,cm,cn,co,cr,cs,cu,cy,cz,dd,de,dj,dk,dm,do,dz,ea,ec,ee,eg,em,ep,es,fi,fr,ga,gb,gc,gd,ge,gh,gm,gn,gq,gr,gt,gw,hk,hn,hr,hu,ib,id,ie,il,in,ir,is,it,jo,jp,ke,kg,kh,km,kn,kp,kr,kw,kz,la,lc,li,lk,lr,ls,lt,lu,lv,ly,ma,mc,md,me,mg,mk,ml,mn,mo,mr,mt,mw,mx,my,mz,na,ne,ng,ni,nl,no,nz,oa,om,pa,pe,pg,ph,pl,pt,py,qa,ro,rs,ru,rw,sa,sc,sd,se,sg,si,sk,sl,sm,sn,st,su,sv,sy,sz,td,tg,th,tj,tm,tn,tr,tt,tw,tz,ua,ug,us,uy,uz,vc,ve,vn,wo,yu,za,zm,zw";
      
      
      window.experiments.keywordWizard = true;
      
      
      
      window.experiments.definitions = true;

      window.Polymer = {
        dom: 'shady',
        lazyRegister: true,
      };
    </script>

    <script src="//www.gstatic.com/patent-search/frontend/patent-search.search_20240531_RC00/scs/compiled_dir/webcomponentsjs/webcomponents-lite.min.js"></script>
    <style>[hidden]:not([style-scope]):not(.style-scope) {
  display: none !important;
}

</style><style></style><style></style><style></style><style></style><link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:400,300,300italic,400italic,500,500italic,700,700italic"><link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto+Mono:400,700"><style></style><style></style><style>
  html, body {
    min-width: 100%;
    width: 100%;
    min-height: 100%;
    height: 100%;
  }

  body {
    background-color: #fff;
    font-family: 'Roboto', sans-serif;
    color: #333;
    margin: 0;
    padding: 0;
    /*
     * Turn off https://github.com/WICG/ScrollAnchoring/blob/master/explainer.md
     * https://memegen.googleplex.com/5230756293246976
     */
    overflow-anchor: none;
  }

  body h1, body h2, body h3 {
    -webkit-font-smoothing: antialiased;
    -moz-osx-font-smoothing: grayscale;
  }

  body a:link {
    color: #3c80f6;
    text-decoration: none;
  }

  body a:visited {
    color: #3c80f6;
  }

  body a[open-result]:visited {
    color: #7e57c2;
  }

  /* For Firefox */
  a:link {
    color: #3c80f6;
    text-decoration: none;
  }

  a:visited {
    color: #3c80f6;
  }

  a[open-result]:visited {
    color: #7e57c2;
  }

  ::selection , body ::selection {
    background-color: #c6dafc;
    color: #333;
  }
</style><link rel="import" href="//www.gstatic.com/patent-search/frontend/patent-search.search_20240531_RC00/scs/compiled_dir/search-app-vulcanized.html">
  <style>[hidden] { display: none !important; }</style><script type="text/javascript" charset="UTF-8" src="https://apis.google.com/js/client.js" gapi_processed="true"></script></head>
  <body>
    <script src="//www.gstatic.com/patent-search/frontend/patent-search.search_20240531_RC00/scs/compiled_dir/search-app-vulcanized.js"></script><span style="font-size: 13px; white-space: pre-wrap; display: inline-block; position: absolute; left: -99999px; top: -99999px; max-width: 217px;"></span><span style="font-size: 13px; white-space: pre-wrap; display: inline-block; position: absolute; left: -99999px; top: -99999px; max-width: 217px;"></span>
    <search-app>
    
    <remote-fetch class="style-scope search-app"></remote-fetch>
    <search-state class="style-scope search-app"></search-state>
    <browser-ui class="style-scope search-app"></browser-ui>
    <template is="dom-if" restamp="" class="style-scope search-app"></template>
    <template is="dom-if" restamp="" class="style-scope search-app"></template>
    
      <search-result id="result" class="style-scope search-app">
    
    
    <search-ui mode="result" class="style-scope search-result x-scope search-ui-0">
    
    
    
    <div id="content" class="style-scope search-ui" wide="">
      <search-header show-search="" class="style-scope search-ui">
    
    
    <div class="style-scope search-header" wide="">
      <header class="style-scope search-header">
        <div class="layout horizontal headerpad style-scope search-header">
          <div class="layout horizontal leftheader style-scope search-header">
            <h1 class="style-scope search-header">
              <state-modifier inherit="" act="{&quot;type&quot;: &quot;HOME&quot;}" class="style-scope search-header"><a id="link" href="#" class="style-scope state-modifier" inherit="">
                <div class="lockup style-scope search-header">
                  <span class="lockup-logo style-scope search-header"></span>
                  <span class="lockup-text style-scope search-header">
                    <span class="lockup-brand style-scope search-header">&nbsp;Patents</span>
                  </span>
                </div>
              </a></state-modifier>
            </h1>
          </div>
          
            <search-box class="style-scope search-header">
    
    <div class="layout horizontal style-scope search-box" style="position: relative;">
      <div class="inputbox flex layout horizontal style-scope search-box">
        <div class="suggest-bg flex style-scope search-box">
          <input name="q" id="searchInput" data-proto="SEARCH_INPUT" autocomplete="off" class="style-scope search-box">
        </div>
        <div id="corpora" class="style-scope search-box">
          <iron-icon icon="icons:lightbulb-outline" id="patentIcon" class="style-scope search-box x-scope iron-icon-0" active=""><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M9 21c0 .55.45 1 1 1h4c.55 0 1-.45 1-1v-1H9v1zm3-19C8.14 2 5 5.14 5 9c0 2.38 1.19 4.47 3 5.74V17c0 .55.45 1 1 1h6c.55 0 1-.45 1-1v-2.26c1.81-1.27 3-3.36 3-5.74 0-3.86-3.14-7-7-7zm2.85 11.1l-.85.6V16h-4v-2.3l-.85-.6C7.8 12.16 7 10.63 7 9c0-2.76 2.24-5 5-5s5 2.24 5 5c0 1.63-.8 3.16-2.15 4.1z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
          <overlay-tooltip positioning-target="#patentIcon" direction="bottom" class="style-scope search-box" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      Include patents
    </div>
  </overlay-tooltip>
          <iron-icon icon="social:school" id="nplIcon" class="style-scope search-box x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M5 13.18v4L12 21l7-3.82v-4L12 17l-7-3.82zM12 3L1 9l11 6 9-4.91V17h2V9L12 3z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
          <overlay-tooltip positioning-target="#nplIcon" direction="bottom" class="style-scope search-box" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      Include non-patent literature
    </div>
  </overlay-tooltip>
          <keyword-wizard id="wizardIcon" class="style-scope search-box">


<div id="wizardContainer" class="style-scope keyword-wizard">
  <iron-icon icon="image:tune" id="wizardIcon" class="style-scope keyword-wizard x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M3 17v2h6v-2H3zM3 5v2h10V5H3zm10 16v-2h8v-2h-8v-2h-2v6h2zM7 9v2H3v2h4v2h2V9H7zm14 4v-2H11v2h10zm-6-4h2V7h4V5h-4V3h-2v6z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
  <paper-material elevation="2" id="wizard" class="style-scope keyword-wizard x-scope paper-material-0" hidden="">
    
    

    
    
    <div class="search-within style-scope keyword-wizard">
      <span class="wizard-header style-scope keyword-wizard">
        Search within
        <iron-icon icon="icons:help-outline" class="help-header style-scope keyword-wizard x-scope iron-icon-0" id="within"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M11 18h2v-2h-2v2zm1-16C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8zm0-14c-2.21 0-4 1.79-4 4h2c0-1.1.9-2 2-2s2 .9 2 2c0 2-3 1.75-3 5h2c0-2.25 3-2.5 3-5 0-2.21-1.79-4-4-4z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      </span>
      
      <div class="help style-scope keyword-wizard" id="withinHelp" hidden="">
        <p class="style-scope keyword-wizard"><b class="style-scope keyword-wizard">Search within the title, abstract, claims, or full patent document</b>: You can
          restrict your search to a specific field using field names.</p>
        <p class="style-scope keyword-wizard">Use <span class="q style-scope keyword-wizard">TI=</span> to search in the title, <span class="q style-scope keyword-wizard">AB=</span> for the
          abstract, <span class="q style-scope keyword-wizard">CL=</span> for the claims, or <span class="q style-scope keyword-wizard">TAC=</span> for
          all three. For example, <span class="q style-scope keyword-wizard">TI=(safety belt)</span>.
        </p>
        <p class="style-scope keyword-wizard"><b class="style-scope keyword-wizard">Search by Cooperative Patent Classifications (CPCs)</b>: These are commonly used to
          represent ideas in place of keywords, and can also be entered in a search term box. If
          you're searching for<span class="q style-scope keyword-wizard">seat belts</span>, you could also search for
          <span class="q style-scope keyword-wizard">B60R22/00</span> to retrieve documents that mention safety belts or body
          harnesses. <span class="q style-scope keyword-wizard">CPC=B60R22</span> will match documents with exactly this CPC,
          <span class="q style-scope keyword-wizard">CPC=B60R22/low</span> matches documents with this CPC or a child
          classification of this CPC.
        </p>
          <a href="https://support.google.com/faqs/answer/7049475" target="_blank" class="style-scope keyword-wizard">Learn More</a>
      </div>
      <div class="filter-chips style-scope keyword-wizard">
        <span class="chip style-scope keyword-wizard" id="title" selected="">Title</span>
        <span class="chip style-scope keyword-wizard" id="abstract" selected="">Abstract</span>
        <span class="chip style-scope keyword-wizard" id="claims" selected="">Claims</span>
        <span class="chip style-scope keyword-wizard" id="fullDoc" selected="">Full Document</span>
      </div>
      
      <div class="cpc-field style-scope keyword-wizard">
        <span class="input-label style-scope keyword-wizard"><label for="inputCPC" class="style-scope keyword-wizard">or CPC</label></span>
        <div class="cpc-inputs style-scope keyword-wizard">
          <input class="keyword-input style-scope keyword-wizard" id="inputCPC" placeholder="Or search with CPC">
          <input type="checkbox" id="childCPC" checked="" class="style-scope keyword-wizard">
          <label class="checkbox-label style-scope keyword-wizard" for="childCPC">
            Include child classifications
          </label>
        </div>
      </div>
    </div>
    
    <div class="find-patents style-scope keyword-wizard">
      <span class="wizard-header style-scope keyword-wizard">
        Find patents
        <iron-icon class="help-header style-scope keyword-wizard x-scope iron-icon-0" icon="icons:help-outline" id="searchBoolean"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M11 18h2v-2h-2v2zm1-16C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8zm0-14c-2.21 0-4 1.79-4 4h2c0-1.1.9-2 2-2s2 .9 2 2c0 2-3 1.75-3 5h2c0-2.25 3-2.5 3-5 0-2.21-1.79-4-4-4z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      </span>
      
      <div class="help style-scope keyword-wizard" id="searchBooleanHelp" hidden="">
        <p class="style-scope keyword-wizard"><b class="style-scope keyword-wizard">Keywords and boolean syntax (USPTO or EPO format)</b>:
          <span class="q style-scope keyword-wizard">seat belt</span> searches these two words, or their plurals and close
          synonyms. <span class="q style-scope keyword-wizard">"seat belt"</span> searches this exact phrase, in order.
          <span class="q style-scope keyword-wizard">-seat -belt</span> searches for documents not containing either word.
        </p>
        <p class="style-scope keyword-wizard"><b class="style-scope keyword-wizard">For searches using boolean logic</b>, the default operator is
          <span class="q style-scope keyword-wizard">AND</span> with left associativity. <i class="style-scope keyword-wizard">Note:</i> this means
          <span class="q style-scope keyword-wizard">safety OR seat belt</span> is searched as <span class="q style-scope keyword-wizard">(safety OR seat)
          AND belt</span>. Each word automatically includes plurals and close synonyms. Adjacent
          words that are implicitly ANDed together, such as <span class="q style-scope keyword-wizard">(safety belt)</span>,
          are treated as a phrase when generating synonyms.
        </p>
        <a href="https://support.google.com/faqs/answer/7049475" target="_blank" class="style-scope keyword-wizard">Learn More</a>
      </div>
      <div class="style-scope keyword-wizard">
        <span class="input-label style-scope keyword-wizard"><label for="inputAnd" class="style-scope keyword-wizard">with ALL of the words</label></span>
        <input class="keyword-input style-scope keyword-wizard" id="inputAnd" placeholder="with ALL of the words">
      </div>
      <div class="style-scope keyword-wizard">
        <span class="input-label style-scope keyword-wizard"><label for="inputExact" class="style-scope keyword-wizard">with the EXACT PHRASE</label></span>
        <input class="keyword-input style-scope keyword-wizard" id="inputExact" placeholder="with the EXACT PHRASE"></div>
      <div class="style-scope keyword-wizard">
        <span class="input-label style-scope keyword-wizard"><label for="inputOr" class="style-scope keyword-wizard">with AT LEAST ONE of the words</label></span>
        <input class="keyword-input style-scope keyword-wizard" id="inputOr" placeholder="with AT LEAST ONE of the words"></div>
      <div class="style-scope keyword-wizard">
        <span class="input-label style-scope keyword-wizard"><label for="inputMinus" class="style-scope keyword-wizard">WITHOUT the words</label></span>
        <input class="keyword-input style-scope keyword-wizard" id="inputMinus" placeholder="WITHOUT the words"></div>
    </div>
    <div class="search-by style-scope keyword-wizard">
      <div class="wizard-header style-scope keyword-wizard">
          Search by
          <iron-icon icon="icons:help-outline" class="help-header style-scope keyword-wizard x-scope iron-icon-0" id="chem"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M11 18h2v-2h-2v2zm1-16C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8zm0-14c-2.21 0-4 1.79-4 4h2c0-1.1.9-2 2-2s2 .9 2 2c0 2-3 1.75-3 5h2c0-2.25 3-2.5 3-5 0-2.21-1.79-4-4-4z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      </div>
      
      <div class="help style-scope keyword-wizard" id="chemHelp" hidden="">
        <p class="style-scope keyword-wizard"><b class="style-scope keyword-wizard">Chemistry searches</b> match terms (trade names, IUPAC names, etc.
          extracted from the entire document, and processed from .MOL files.)
        </p>
        <p class="style-scope keyword-wizard">Substructure (use <span class="q style-scope keyword-wizard">SSS=</span>) and similarity (use
          <span class="q style-scope keyword-wizard">~</span>) searches are limited to one per search at the top-level
          <span class="q style-scope keyword-wizard">AND</span> condition. Exact searches can be used multiple times throughout
          the search query.
        </p>
        <p class="style-scope keyword-wizard">Searching by SMILES or InChi key requires no special syntax. To search by SMARTS, use
          <span class="q style-scope keyword-wizard">SMARTS=</span>.
        </p>
        <p class="style-scope keyword-wizard">To search for multiple molecules, select "Batch" in the "Type" menu. Enter multiple
          molecules separated by whitespace or by comma.
        </p>
        <a href="https://support.google.com/faqs/answer/7049475" target="_blank" class="style-scope keyword-wizard">Learn More</a>
      </div>
      
      <div class="chemistry-field style-scope keyword-wizard">
          <span class="input-label style-scope keyword-wizard"><label for="inputChem" class="style-scope keyword-wizard">Chemistry</label></span>
          <span class="chem-type-label style-scope keyword-wizard"><label for="chemType" hidden="" class="style-scope keyword-wizard">Type</label></span>
          <select id="chemType" class="style-scope keyword-wizard">
            <option value="identical" class="style-scope keyword-wizard">Identical</option>
            <option value="substructure" class="style-scope keyword-wizard">Substructure</option>
            <option value="smarts" class="style-scope keyword-wizard">SMARTS</option>
            <option value="similarity" class="style-scope keyword-wizard">Similarity</option>
            <option value="batch" class="style-scope keyword-wizard">Batch</option>
          </select>
          <textarea class="keyword-input style-scope keyword-wizard" id="inputChem" placeholder="Trade name, SMILES, InChiKey"></textarea>
      </div>
      <div class="import style-scope keyword-wizard">
        <div class="wizard-header style-scope keyword-wizard">
          Patent numbers
        </div>
        <span class="import-button style-scope keyword-wizard">
          <a href="https://patents.google.com/import" class="style-scope keyword-wizard"><button class="style-scope keyword-wizard">Import</button></a>
        </span>
        <p class="import-text style-scope keyword-wizard">
          Search specific patents by importing a CSV or list of patent publication or application numbers.
        </p>
      </div>
    </div>
    
    <div class="wizard-buttons style-scope keyword-wizard">
      <button type="reset" class="style-scope keyword-wizard">Close</button>
      <button type="button" autofocus="" class="style-scope keyword-wizard">Add</button>
    </div>
  
  </paper-material>
</div>
</keyword-wizard>
          <overlay-tooltip positioning-target="#wizardIcon" direction="bottom" class="style-scope search-box" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      Display advanced search options
    </div>
  </overlay-tooltip>
        </div>
      </div>
      <paper-material id="suggestMenu" class="style-scope search-box x-scope paper-material-0" elevation="1" hidden="">
    
    

    
        <template is="dom-repeat" class="style-scope search-box"></template>
        <div class="row style-scope search-box" hidden="">
          <iron-icon class="suggest-icon style-scope search-box x-scope iron-icon-0" icon="error"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm1 15h-2v-2h2v2zm0-4h-2V7h2v6z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon> Sorry, we couldn't find this patent number.
        </div>
        <div class="row style-scope search-box" hidden="">
          <iron-icon class="suggest-icon style-scope search-box x-scope iron-icon-0" icon="error"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm1 15h-2v-2h2v2zm0-4h-2V7h2v6z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon> 
        </div>
      
  </paper-material>
      <button id="searchButton" class="style-scope search-box"><iron-icon class="style-scope search-box x-scope iron-icon-0" icon="search"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon></button>
    </div>
  </search-box>
          <template is="dom-if" class="style-scope search-header"></template>
          <div class="flex style-scope search-header"></div>
          
            <div id="header-container" class="style-scope search-header">
              <result-nav class="style-scope search-header">
    
    <div class="style-scope result-nav" hidden="">
      <state-modifier id="current" act="{&quot;type&quot;: &quot;BACK_TO_RESULTS&quot;}" inherit="" class="style-scope result-nav"><a id="link" href="#" class="style-scope state-modifier" inherit=""><span class="style-scope result-nav"></span> of
      <template is="dom-if" class="style-scope result-nav"></template>
      0<template is="dom-if" class="style-scope result-nav"></template></a></state-modifier>
      <state-modifier id="prevResult" inherit="" act="{&quot;type&quot;: &quot;NAV_RESULT&quot;, &quot;result&quot;: &quot;$id&quot;, &quot;page&quot;: $page}" class="style-scope result-nav" disabled=""><a id="link" href="#" class="style-scope state-modifier" inherit="">
        <iron-icon icon="chevron-left" class="style-scope result-nav x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M15.41 7.41L14 6l-6 6 6 6 1.41-1.41L10.83 12z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      </a></state-modifier><state-modifier id="nextResult" inherit="" act="{&quot;type&quot;: &quot;NAV_RESULT&quot;, &quot;result&quot;: &quot;$id&quot;, &quot;page&quot;: $page}" class="style-scope result-nav" disabled=""><a id="link" href="#" class="style-scope state-modifier" inherit="">
        <iron-icon icon="chevron-right" class="style-scope result-nav x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M10 6L8.59 7.41 13.17 12l-4.58 4.59L10 18l6-6z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      </a></state-modifier>
      <overlay-tooltip positioning-target="#prevResult" direction="bottom" arrow="" class="style-scope result-nav" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip"></div>
    <div class="style-scope overlay-tooltip">
      Previous result
    </div>
  </overlay-tooltip>
      <overlay-tooltip positioning-target="#nextResult" direction="bottom" arrow="" class="style-scope result-nav" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip"></div>
    <div class="style-scope overlay-tooltip">
      Next result
    </div>
  </overlay-tooltip>
    </div>
  </result-nav>
            </div>
          <template is="dom-if" class="style-scope search-header"></template>
        </div>
      </header>
    </div>
  </search-header>

      <div class="layout horizontal style-scope search-ui">
        <template is="dom-if" class="style-scope search-ui"></template>

        <div id="main" class="style-scope search-ui">
          <search-term-wizard class="style-scope search-ui"><pat-dialog open="" class="style-scope search-term-wizard">
  <span class="header style-scope search-term-wizard">
    <span class="title style-scope search-term-wizard">Search tools</span>
    <span id="language" class="style-scope search-term-wizard">
      <text-button id="textButton" selected="" class="style-scope search-term-wizard">Text
</text-button>
      <text-button id="classificationButton" class="style-scope search-term-wizard">Classification
</text-button>
      <text-button id="chemistryButton" class="style-scope search-term-wizard">Chemistry
</text-button>
      <text-button id="measureButton" class="style-scope search-term-wizard">Measure
</text-button>
      <text-button id="numbersButton" class="style-scope search-term-wizard">Numbers
</text-button>
    </span>
  </span>
  <span id="panels" class="style-scope search-term-wizard">
    <text-search-term-wizard id="textPanel" class="style-scope search-term-wizard" selected=""><span class="style-scope text-search-term-wizard">
  <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49" class="style-scope text-search-term-wizard">
    <path d="M3.85 15.275V13h10.3v2.275Zm0-4.275V8.725h16.3V11Z" class="style-scope text-search-term-wizard"></path>
  </svg>
  <outlined-text-field id="textField" placeholder="Type a concept you want to search for here" class="style-scope text-search-term-wizard" value=""><span class="style-scope outlined-text-field">
  <input id="input" spellcheck="false" autocomplete="off" class="style-scope outlined-text-field" aria-disabled="false" placeholder="Type a concept you want to search for here">
  
</span>
</outlined-text-field>
</span>
<span class="style-scope text-search-term-wizard">
  <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49" class="style-scope text-search-term-wizard">
    <path d="M8 18.025h8v-2.1H8Zm0-4h8v-2.1H8ZM6.075 22.2q-.95 0-1.612-.662-.663-.663-.663-1.613V4.075q0-.95.663-1.613.662-.662 1.612-.662h8L20.2 7.925v12q0 .95-.662 1.613-.663.662-1.613.662Zm6.85-13.125v-5h-6.85v15.85h11.85V9.075Zm-6.85-5v5-5 15.85-15.85Z" class="style-scope text-search-term-wizard"></path>
  </svg>
  <filter-chip id="fullDocuments" selected="" class="style-scope text-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Full documents</filter-chip>
  <filter-chip id="title" class="style-scope text-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Title</filter-chip>
  <filter-chip id="abstract" class="style-scope text-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Abstract</filter-chip>
  <filter-chip id="claims" class="style-scope text-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Claims</filter-chip>
</span>
<span class="style-scope text-search-term-wizard">
  <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49" class="style-scope text-search-term-wizard">
    <path d="M16 16.925q2.05 0 3.488-1.437Q20.925 14.05 20.925 12t-1.437-3.488Q18.05 7.075 16 7.075q-.625 0-1.2.15t-1.075.425q.7.925 1.087 2.025.388 1.1.388 2.325t-.388 2.325q-.387 1.1-1.087 2.025.5.275 1.075.425.575.15 1.2.15Zm-4-2.075q.45-.575.688-1.313.237-.737.237-1.537 0-.8-.237-1.538Q12.45 9.725 12 9.15q-.45.575-.688 1.312-.237.738-.237 1.538t.237 1.537q.238.738.688 1.313Zm-4 2.075q.625 0 1.2-.15t1.075-.425q-.7-.925-1.087-2.025Q8.8 13.225 8.8 12t.388-2.325q.387-1.1 1.087-2.025-.5-.275-1.075-.425-.575-.15-1.2-.15-2.05 0-3.487 1.437Q3.075 9.95 3.075 12t1.438 3.488Q5.95 16.925 8 16.925ZM8 19.2q-3 0-5.1-2.1Q.8 15 .8 12q0-3 2.1-5.1Q5 4.8 8 4.8q1.125 0 2.137.312 1.013.313 1.863.913.85-.6 1.863-.913Q14.875 4.8 16 4.8q3 0 5.1 2.1Q23.2 9 23.2 12q0 3-2.1 5.1-2.1 2.1-5.1 2.1-1.125 0-2.137-.313-1.013-.312-1.863-.912-.85.6-1.863.912Q9.125 19.2 8 19.2Z" class="style-scope text-search-term-wizard"></path>
  </svg>
  <filter-chip id="all" selected="" class="style-scope text-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
All</filter-chip>
  <filter-chip id="any" class="style-scope text-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Any</filter-chip>
  <filter-chip id="exact" class="style-scope text-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Exact</filter-chip>
  <filter-chip id="not" class="style-scope text-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Not</filter-chip>
</span>
<span class="style-scope text-search-term-wizard">
  <mat-button id="button" filled="" class="style-scope text-search-term-wizard x-scope mat-button-0" disabled=""><span slot="label" class="style-scope text-search-term-wizard">Add AND condition</span></mat-button></span></text-search-term-wizard>
    <classification-wizard id="classificationPanel" class="style-scope search-term-wizard"><span class="style-scope classification-wizard">
  <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49" class="style-scope classification-wizard">
    <path d="M3.85 15.275V13h10.3v2.275Zm0-4.275V8.725h16.3V11Z" class="style-scope classification-wizard"></path>
  </svg>
  <outlined-text-field id="textField" placeholder="Enter a CPC here" uppercase="" class="style-scope classification-wizard" value=""><span class="style-scope outlined-text-field">
  <input id="input" spellcheck="false" autocomplete="off" class="style-scope outlined-text-field" aria-disabled="false" placeholder="Enter a CPC here">
  
</span>
</outlined-text-field>
</span>
<template is="dom-if" class="style-scope classification-wizard"></template>
<template is="dom-if" class="style-scope classification-wizard"></template>
<template is="dom-if" class="style-scope classification-wizard"></template>
<span class="style-scope classification-wizard">
  <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49" class="style-scope classification-wizard">
    <path d="M16 16.925q2.05 0 3.488-1.437Q20.925 14.05 20.925 12t-1.437-3.488Q18.05 7.075 16 7.075q-.625 0-1.2.15t-1.075.425q.7.925 1.087 2.025.388 1.1.388 2.325t-.388 2.325q-.387 1.1-1.087 2.025.5.275 1.075.425.575.15 1.2.15Zm-4-2.075q.45-.575.688-1.313.237-.737.237-1.537 0-.8-.237-1.538Q12.45 9.725 12 9.15q-.45.575-.688 1.312-.237.738-.237 1.538t.237 1.537q.238.738.688 1.313Zm-4 2.075q.625 0 1.2-.15t1.075-.425q-.7-.925-1.087-2.025Q8.8 13.225 8.8 12t.388-2.325q.387-1.1 1.087-2.025-.5-.275-1.075-.425-.575-.15-1.2-.15-2.05 0-3.487 1.437Q3.075 9.95 3.075 12t1.438 3.488Q5.95 16.925 8 16.925ZM8 19.2q-3 0-5.1-2.1Q.8 15 .8 12q0-3 2.1-5.1Q5 4.8 8 4.8q1.125 0 2.137.312 1.013.313 1.863.913.85-.6 1.863-.913Q14.875 4.8 16 4.8q3 0 5.1 2.1Q23.2 9 23.2 12q0 3-2.1 5.1-2.1 2.1-5.1 2.1-1.125 0-2.137-.313-1.013-.312-1.863-.912-.85.6-1.863.912Q9.125 19.2 8 19.2Z" class="style-scope classification-wizard"></path>
  </svg>
  <filter-chip id="notExact" class="style-scope classification-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>

    These CPCs and their children
  </filter-chip>
  <filter-chip selected="" id="exact" class="style-scope classification-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>

    These exact CPCs
  </filter-chip>
</span>
<span class="style-scope classification-wizard">
  <mat-button id="button" filled="" class="style-scope classification-wizard x-scope mat-button-0" disabled=""><span slot="label" class="style-scope classification-wizard">Add AND condition</span></mat-button>
</span></classification-wizard>
    <chemistry-search-term-wizard id="chemistryPanel" class="style-scope search-term-wizard"><span class="style-scope chemistry-search-term-wizard">
  <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49" class="style-scope chemistry-search-term-wizard">
    <path d="M3.85 15.275V13h10.3v2.275Zm0-4.275V8.725h16.3V11Z" class="style-scope chemistry-search-term-wizard"></path>
  </svg>
  <outlined-text-field id="textField" placeholder="Trade name, SMILES, InChI Key" class="style-scope chemistry-search-term-wizard" value=""><span class="style-scope outlined-text-field">
  <input id="input" spellcheck="false" autocomplete="off" class="style-scope outlined-text-field" aria-disabled="false" placeholder="Trade name, SMILES, InChI Key">
  
</span>
</outlined-text-field>
  <outlined-textarea id="textArea" placeholder="Enter multiple SMILES or InChI Keys, one per line" rows="4" class="style-scope chemistry-search-term-wizard"><span class="style-scope outlined-textarea">
  <textarea id="textarea" autocomplete="off" autocorrect="off" spellcheck="false" tabindex="0" aria-disabled="false" wrap="soft" class="style-scope outlined-textarea" rows="4" placeholder="Enter multiple SMILES or InChI Keys, one per line">  </textarea>
  <div id="helperText" class="style-scope outlined-textarea"> </div>
</span>
<span class="right-button-container style-scope outlined-textarea">
  
</span>
</outlined-textarea>
  <a href="https://support.google.com/faqs/answer/7049475#zippy=%2Csearching-by-chemistry" target="_blank" class="style-scope chemistry-search-term-wizard">
    <svg class="help style-scope chemistry-search-term-wizard" xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49">
      <path d="M11.95 18q.525 0 .888-.363.362-.362.362-.887t-.362-.887q-.363-.363-.888-.363t-.888.363q-.362.362-.362.887t.362.887q.363.363.888.363Zm-.9-3.85h1.85q0-.825.188-1.3.187-.475 1.062-1.3.65-.65 1.025-1.238.375-.587.375-1.412 0-1.4-1.025-2.15T12.1 6q-1.425 0-2.312.75-.888.75-1.238 1.8l1.65.65q.125-.45.563-.975Q11.2 7.7 12.1 7.7q.8 0 1.2.437.4.438.4.963 0 .5-.3.937-.3.438-.75.813-1.1.975-1.35 1.475-.25.5-.25 1.825ZM12 22q-2.075 0-3.9-.788-1.825-.787-3.175-2.137-1.35-1.35-2.137-3.175Q2 14.075 2 12t.788-3.9q.787-1.825 2.137-3.175 1.35-1.35 3.175-2.138Q9.925 2 12 2t3.9.787q1.825.788 3.175 2.138 1.35 1.35 2.137 3.175Q22 9.925 22 12t-.788 3.9q-.787 1.825-2.137 3.175-1.35 1.35-3.175 2.137Q14.075 22 12 22Zm0-2q3.35 0 5.675-2.325Q20 15.35 20 12q0-3.35-2.325-5.675Q15.35 4 12 4 8.65 4 6.325 6.325 4 8.65 4 12q0 3.35 2.325 5.675Q8.65 20 12 20Zm0-8Z" class="style-scope chemistry-search-term-wizard"></path>
    </svg>
  </a>
</span>
<span id="chemQuantifiers" class="style-scope chemistry-search-term-wizard">
  <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49" class="style-scope chemistry-search-term-wizard">
    <path d="M5 21.2q-1.4 0-2-1.25-.6-1.25.3-2.325l5.5-6.7V5.15h-.825q-.475 0-.8-.338Q6.85 4.475 6.85 4q0-.475.325-.813.325-.337.8-.337h8.05q.475 0 .8.337.325.338.325.813 0 .475-.325.812-.325.338-.8.338H15.2v5.775l5.5 6.7q.9 1.075.3 2.325-.6 1.25-2 1.25Zm2.15-3.275h9.7l-3.275-3.85h-3.15Zm-2 1h13.7l-5.925-7.2V5.15h-1.85v6.575Zm6.85-6.9Z" class="style-scope chemistry-search-term-wizard"></path>
  </svg>
  <filter-chip id="exact" selected="" class="style-scope chemistry-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Exact</filter-chip>
  <filter-chip id="exactBatch" class="style-scope chemistry-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Exact Batch</filter-chip>
  <filter-chip id="similar" class="style-scope chemistry-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Similar</filter-chip>
  <filter-chip id="substructure" class="style-scope chemistry-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Substructure</filter-chip>
  <filter-chip id="smarts" class="style-scope chemistry-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Substructure (SMARTS)</filter-chip>
</span>
<span id="location" class="style-scope chemistry-search-term-wizard">
  <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49" class="style-scope chemistry-search-term-wizard">
    <path d="M8 18.025h8v-2.1H8Zm0-4h8v-2.1H8ZM6.075 22.2q-.95 0-1.612-.662-.663-.663-.663-1.613V4.075q0-.95.663-1.613.662-.662 1.612-.662h8L20.2 7.925v12q0 .95-.662 1.613-.663.662-1.613.662Zm6.85-13.125v-5h-6.85v15.85h11.85V9.075Zm-6.85-5v5-5 15.85-15.85Z" class="style-scope chemistry-search-term-wizard"></path>
  </svg>
  <filter-chip id="fullDocuments" selected="" class="style-scope chemistry-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Full documents</filter-chip>
  <filter-chip id="claims" class="style-scope chemistry-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Claims only</filter-chip>
</span>
<span class="style-scope chemistry-search-term-wizard">
  <mat-button id="button" filled="" class="style-scope chemistry-search-term-wizard x-scope mat-button-0" disabled=""><span slot="label" class="style-scope chemistry-search-term-wizard">Add AND condition</span></mat-button>
</span>
</chemistry-search-term-wizard>
    <measure-search-term-wizard id="measurePanel" class="style-scope search-term-wizard"><span class="style-scope measure-search-term-wizard">
  <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49" class="style-scope measure-search-term-wizard">
    <path d="M4.1 18.15q-.95 0-1.612-.663-.663-.662-.663-1.612v-7.75q0-.95.663-1.613Q3.15 5.85 4.1 5.85h15.8q.95 0 1.613.662.662.663.662 1.613v7.75q0 .95-.662 1.612-.663.663-1.613.663Zm0-2.275h15.8v-7.75H17V12h-2V8.125h-2V12h-2V8.125H9V12H7V8.125H4.1v7.75ZM7 12h2Zm4 0h2Zm4 0h2Zm-3 0Z" class="style-scope measure-search-term-wizard"></path>
  </svg>
  <outlined-text-field id="measureExpression" placeholder="1.5 mm, 20 ft, 400-500 fahrenheit, 800 MHz, 0.01-100 mol" class="style-scope measure-search-term-wizard" value=""><span class="style-scope outlined-text-field">
  <input id="input" spellcheck="false" autocomplete="off" class="style-scope outlined-text-field" aria-disabled="false" placeholder="1.5 mm, 20 ft, 400-500 fahrenheit, 800 MHz, 0.01-100 mol">
  
</span>
</outlined-text-field>
</span>
<span class="style-scope measure-search-term-wizard">
  <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49" class="style-scope measure-search-term-wizard">
    <path d="M3.875 20.15V8.85h4.25v11.3Zm6 0V3.85h4.25v16.3Zm6 0v-7.3h4.25v7.3Z" class="style-scope measure-search-term-wizard"></path>
  </svg>
  <outlined-text-field id="context" placeholder="activity, wavelength, embossing depth, absorption" class="style-scope measure-search-term-wizard" value=""><span class="style-scope outlined-text-field">
  <input id="input" spellcheck="false" autocomplete="off" class="style-scope outlined-text-field" aria-disabled="false" placeholder="activity, wavelength, embossing depth, absorption">
  
</span>
</outlined-text-field>
</span>
<span class="style-scope measure-search-term-wizard">
  <mat-button id="button" filled="" class="style-scope measure-search-term-wizard x-scope mat-button-0" disabled=""><span slot="label" class="style-scope measure-search-term-wizard">Add AND condition</span></mat-button>
</span>
</measure-search-term-wizard>
    <numerical-search-term-wizard id="numericalPanel" class="style-scope search-term-wizard"><span class="style-scope numerical-search-term-wizard">
  <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49" class="style-scope numerical-search-term-wizard">
    <path d="M3.85 15.275V13h10.3v2.275Zm0-4.275V8.725h16.3V11Z" class="style-scope numerical-search-term-wizard"></path>
  </svg>
  <outlined-textarea id="batch" placeholder="Enter multiple Patent Application and Publication Numbers, one per line" rows="4" class="style-scope numerical-search-term-wizard"><span class="style-scope outlined-textarea">
  <textarea id="textarea" autocomplete="off" autocorrect="off" spellcheck="false" tabindex="0" aria-disabled="false" wrap="soft" class="style-scope outlined-textarea" rows="4" placeholder="Enter multiple Patent Application and Publication Numbers, one per line">  </textarea>
  <div id="helperText" class="style-scope outlined-textarea"> </div>
</span>
<span class="right-button-container style-scope outlined-textarea">
  
</span>
</outlined-textarea>
</span>
<template is="dom-if" class="style-scope numerical-search-term-wizard"></template>
<template is="dom-if" class="style-scope numerical-search-term-wizard"></template>
<span class="style-scope numerical-search-term-wizard">
  <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49" class="style-scope numerical-search-term-wizard">
    <path d="m10.825 23.45-1.6-1.625L10.9 20.15H8q-.35.9-1.162 1.488-.813.587-1.838.587-1.35 0-2.287-.937-.938-.938-.938-2.288 0-1.025.588-1.837Q2.95 16.35 3.85 16V8q-.9-.35-1.487-1.163Q1.775 6.025 1.775 5q0-1.35.938-2.288Q3.65 1.775 5 1.775q1.025 0 1.838.587Q7.65 2.95 8 3.85h2.9L9.225 2.175l1.6-1.625L15.275 5l-4.45 4.45-1.6-1.625L10.9 6.15H8q-.25.65-.725 1.125T6.15 8v8q.65.25 1.125.725T8 17.85h2.9l-1.675-1.675 1.6-1.625 4.45 4.45ZM19 22.225q-1.35 0-2.287-.937-.938-.938-.938-2.288 0-1.05.588-1.863.587-.812 1.487-1.137V8q-.9-.325-1.487-1.137-.588-.813-.588-1.863 0-1.35.938-2.288.937-.937 2.287-.937 1.35 0 2.288.937.937.938.937 2.288 0 1.05-.587 1.863Q21.05 7.675 20.15 8v8q.9.35 1.488 1.163.587.812.587 1.837 0 1.35-.937 2.288-.938.937-2.288.937ZM5 19.95q.4 0 .675-.275T5.95 19q0-.4-.275-.675T5 18.05q-.4 0-.675.275T4.05 19q0 .4.275.675T5 19.95Zm14 0q.4 0 .675-.275T19.95 19q0-.4-.275-.675T19 18.05q-.4 0-.675.275T18.05 19q0 .4.275.675t.675.275Zm-14-14q.4 0 .675-.275T5.95 5q0-.4-.275-.675T5 4.05q-.4 0-.675.275T4.05 5q0 .4.275.675T5 5.95Zm14 0q.4 0 .675-.275T19.95 5q0-.4-.275-.675T19 4.05q-.4 0-.675.275T18.05 5q0 .4.275.675T19 5.95ZM5 19Zm14 0ZM5 5Zm14 0Z" class="style-scope numerical-search-term-wizard"></path>
  </svg>
  <filter-chip id="applicationNumbers" class="style-scope numerical-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Application Numbers</filter-chip>
  <filter-chip id="publicationNumbers" class="style-scope numerical-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Publication Numbers</filter-chip>
  <filter-chip id="either" selected="" class="style-scope numerical-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Either</filter-chip>
</span>
<span class="style-scope numerical-search-term-wizard">
  <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49" class="style-scope numerical-search-term-wizard">
    <path d="M12 22.2q-2.1 0-3.962-.8-1.863-.8-3.25-2.175Q3.4 17.85 2.6 15.988 1.8 14.125 1.8 12t.8-3.988q.8-1.862 2.188-3.237Q6.175 3.4 8.038 2.6 9.9 1.8 12 1.8q2.125 0 3.988.8 1.862.8 3.237 2.175Q20.6 6.15 21.4 8.012q.8 1.863.8 3.988t-.8 3.988q-.8 1.862-2.175 3.237Q17.85 20.6 15.988 21.4q-1.863.8-3.988.8Zm-1-2.325v-1.95q-.8 0-1.387-.575-.588-.575-.588-1.4v-.975L4.275 10.2q-.075.45-.137.9-.063.45-.063.9 0 3 1.975 5.25T11 19.875Zm6.85-2.525q.5-.55.888-1.175.387-.625.65-1.313.262-.687.4-1.412.137-.725.137-1.45 0-2.425-1.338-4.438-1.337-2.012-3.612-2.887v.375q0 .825-.587 1.4-.588.575-1.388.575h-2v2q0 .4-.287.687-.288.288-.688.288h-2v2h5.95q.425 0 .713.287.287.288.287.688v2.975h1q.65 0 1.163.387.512.388.712 1.013Z" class="style-scope numerical-search-term-wizard"></path>
  </svg>
  <outlined-text-field id="countryFilter" placeholder="Restrict search to these countries" class="style-scope numerical-search-term-wizard" value=""><span class="style-scope outlined-text-field">
  <input id="input" spellcheck="false" autocomplete="off" class="style-scope outlined-text-field" aria-disabled="false" placeholder="Restrict search to these countries">
  
</span>
</outlined-text-field>
</span>
<span class="style-scope numerical-search-term-wizard">
  <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49" class="style-scope numerical-search-term-wizard">
    <path d="M2.85 18.225V15.95h6.3v2.275Zm0-5.075v-2.3h12.3v2.3Zm0-5.1V5.775h18.3V8.05Z" class="style-scope numerical-search-term-wizard"></path>
  </svg>
  <outlined-text-field id="countryOrder" placeholder="Prefer results from these countries in this order" class="style-scope numerical-search-term-wizard" value="US, EP, WO, JP, CN"><span class="style-scope outlined-text-field">
  <input id="input" spellcheck="false" autocomplete="off" class="style-scope outlined-text-field" aria-disabled="false" placeholder="Prefer results from these countries in this order">
  
</span>
</outlined-text-field>
</span>
<span class="style-scope numerical-search-term-wizard">
  <mat-button id="button" filled="" class="style-scope numerical-search-term-wizard x-scope mat-button-0" disabled=""><span slot="label" class="style-scope numerical-search-term-wizard">Add AND condition</span></mat-button>
</span>
</numerical-search-term-wizard>
  </span>

</pat-dialog>
</search-term-wizard>
          
      <div content="" class="style-scope search-result">
        <div id="mainContainer" class="vertical layout style-scope search-result">
          <template is="dom-if" class="style-scope search-result"></template>
          <result-container class="style-scope search-result">
    

    <paper-spinner class="style-scope result-container x-scope paper-spinner-0" aria-hidden="true" hidden=""><div id="spinnerContainer" class="  style-scope paper-spinner"><div class="spinner-layer layer-1 style-scope paper-spinner"><div class="circle-clipper left style-scope paper-spinner"></div><div class="circle-clipper right style-scope paper-spinner"></div></div><div class="spinner-layer layer-2 style-scope paper-spinner"><div class="circle-clipper left style-scope paper-spinner"></div><div class="circle-clipper right style-scope paper-spinner"></div></div><div class="spinner-layer layer-3 style-scope paper-spinner"><div class="circle-clipper left style-scope paper-spinner"></div><div class="circle-clipper right style-scope paper-spinner"></div></div><div class="spinner-layer layer-4 style-scope paper-spinner"><div class="circle-clipper left style-scope paper-spinner"></div><div class="circle-clipper right style-scope paper-spinner"></div></div></div></paper-spinner>
    
      <patent-result class="style-scope result-container">
    
    
    <div class="layout horizontal style-scope patent-result">
      <div style="width: 1200px; height: auto; overflow: auto;" class="style-scope patent-result">
        <div id="wrapper" tabindex="0" class="style-scope patent-result">
          <result-tags class="style-scope patent-result"></result-tags>
          <h1 id="title" class="scroll-target style-scope patent-result">
            Compositions and methods for hydroxylating epothilones 
     </h1>
          <div class="horizontal layout style-scope patent-result">
            
            <div class="flex-3 style-scope patent-result" style="margin-top:-20px;overflow:hidden;" marginright="">
              <section id="abstract" class="style-scope patent-result">
                <h3 class="layout horizontal style-scope patent-result">
                  <div class="flex style-scope patent-result">Abstract</div>
                  <div class="righthead style-scope patent-result" hidden="">translated from </div>
                </h3>
                <patent-text name="abstract" class="style-scope patent-result" style="padding-top: 0px;">
    
    
    <template is="dom-if" restamp="" class="style-scope patent-text"></template>
    <overlay-tooltip id="tooltip" class="style-scope patent-text" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      
    </div>
  </overlay-tooltip>
    <div class="layout horizontal style-scope patent-text">
      <section id="text" class="flex style-scope patent-text"><abstract mxw-id="PA148981332" lang="EN" source="national office" load-source="docdb" class="style-scope patent-text">
    <div class="abstract style-scope patent-text">Isolated nucleic acid sequences and polypeptides encoded thereby for epothilone B hydroxylase and mutants and variants thereof and a ferredoxin located downstream from the epothilone B hydroxylase gene are provided. Also provided are vectors and cells containing these vectors. In addition, methods for producing recombinant microorganisms, methods for using these recombinant microorganism to produce hydroxyalkyl-bearing epothilones and an epothilone analog produced by a mutant of epothilone B hydroxylase are provided.</div>
  </abstract>
  </section>
      <div id="margin" class="style-scope patent-text">
        <div id="createHighlight" style="top: 0px;" class="style-scope patent-text" hidden="">
          <iron-icon id="comment" icon="editor:insert-comment" class="style-scope patent-text x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M20 2H4c-1.1 0-2 .9-2 2v12c0 1.1.9 2 2 2h14l4 4V4c0-1.1-.9-2-2-2zm-2 12H6v-2h12v2zm0-3H6V9h12v2zm0-3H6V6h12v2z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
        </div>
      </div>
    </div>
  </patent-text>
              </section>
              
              <template is="dom-if" class="style-scope patent-result"></template>
              
              <section id="classifications" class="style-scope patent-result" hidden="">
                <h3 class="layout horizontal style-scope patent-result">
                  <div class="flex style-scope patent-result">Classifications</div>
                  <div id="machineClassifiedWarning" class="righthead tooltip-hint style-scope patent-result" hidden="">machine-classified</div>
                  <div id="machineClassifiedCPCWarning" class="righthead tooltip-hint style-scope patent-result" hidden="">cpc-machine-classified</div>
                  <div id="machineClassifiedFtermWarning" class="righthead tooltip-hint style-scope patent-result" hidden="">fterm-machine-classified</div>
                  <div class="style-scope patent-result" hidden="">&nbsp;</div>
                  <div id="familyClassifiedFtermWarning" class="righthead tooltip-hint style-scope patent-result" hidden="">fterm-family-classified</div>
                </h3>
                <overlay-tooltip positioning-target="#machineClassifiedWarning" direction="bottom" class="style-scope patent-result" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      The classifications are assigned by a computer and are not a legal conclusion. <br class="style-scope patent-result"> Google has not performed a legal analysis and makes no representation as to the accuracy of the classifications listed.
    </div>
  </overlay-tooltip>
                <overlay-tooltip positioning-target="#machineClassifiedCPCWarning" direction="bottom" class="style-scope patent-result" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      The CPC classifications are assigned by a computer and are not a legal conclusion. <br class="style-scope patent-result"> Google has not performed a legal analysis and makes no representation as to the accuracy of the classifications listed.
    </div>
  </overlay-tooltip>
                <overlay-tooltip positioning-target="#familyClassifiedFtermWarning" direction="bottom" class="style-scope patent-result" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      The F-term classifications are assigned based on a patent family member containing these classification codes.
    </div>
  </overlay-tooltip>
                <overlay-tooltip positioning-target="#machineClassifiedFtermWarning" direction="bottom" class="style-scope patent-result" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      The F-term classifications are assigned by a computer and are not a legal conclusion. <br class="style-scope patent-result"> Google has not performed a legal analysis and makes no representation as to the accuracy of the classifications listed.
    </div>
  </overlay-tooltip>
                <classification-viewer class="style-scope patent-result">
    
    <div class="table style-scope classification-viewer">
      <template is="dom-repeat" class="style-scope classification-viewer"></template>
      <template is="dom-if" class="style-scope classification-viewer"></template>
    </div>
  </classification-viewer>
              </section>
              
              <template is="dom-if" class="style-scope patent-result"></template>
              
              <template is="dom-if" class="style-scope patent-result"></template>
              
              
                <section class="style-scope patent-result">
                  <h3 class="layout horizontal style-scope patent-result">
                    <div class="flex style-scope patent-result">Landscapes</div>
                  </h3>
                  <div class="responsive-grid style-scope patent-result">
                    
                      <landscape-link class="style-scope patent-result"><state-modifier act="{&quot;type&quot;:&quot;QUERY_SET_TERM&quot;,&quot;term&quot;:&quot;$term&quot;,&quot;origin&quot;:&quot;LANDSCAPE_LINK&quot;}" class="style-scope landscape-link" data-term="Micro-Organisms Or Cultivation Processes Thereof"><a id="link" href="#" class="style-scope state-modifier">
  <requery-tile class="area style-scope landscape-link">
<span class="label style-scope requery-tile">
    Micro-Organisms Or Cultivation Processes Thereof
  </span>
<span class="search-icon style-scope requery-tile"></span>
</requery-tile>
</a></state-modifier>
</landscape-link>
                    
                      <landscape-link class="style-scope patent-result"><state-modifier act="{&quot;type&quot;:&quot;QUERY_SET_TERM&quot;,&quot;term&quot;:&quot;$term&quot;,&quot;origin&quot;:&quot;LANDSCAPE_LINK&quot;}" class="style-scope landscape-link" data-term="Preparation Of Compounds By Using Micro-Organisms"><a id="link" href="#" class="style-scope state-modifier">
  <requery-tile class="area style-scope landscape-link">
<span class="label style-scope requery-tile">
    Preparation Of Compounds By Using Micro-Organisms
  </span>
<span class="search-icon style-scope requery-tile"></span>
</requery-tile>
</a></state-modifier>
</landscape-link>
                    <template is="dom-repeat" class="style-scope patent-result"></template>
                  </div>
                  <s-more-inline class="style-scope patent-result"><hr aria-hidden="true" class="style-scope s-more-inline">

<div class="button style-scope s-more-inline" style="" hidden="">
  <span slot="expand_text" class="style-scope patent-result">Show more</span>
  <span class="expand more style-scope s-more-inline"></span>
</div>
<div class="button style-scope s-more-inline">
  <span slot="collapse_text" class="style-scope patent-result">Show less</span>
  <span class="expand less style-scope s-more-inline"></span>
</div>
</s-more-inline>
                </section>
              <template is="dom-if" class="style-scope patent-result"></template>
            </div>
            <div class="flex-2 style-scope patent-result">
              <section class="knowledge-card style-scope patent-result">
                <header class="style-scope patent-result">
                  <h2 id="pubnum" class="style-scope patent-result">TW200418849A</h2>
                  <p class="tagline style-scope patent-result">Taiwan</p>
                  <overlay-tooltip positioning-target="#pubnum" direction="bottom" class="style-scope patent-result" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      
                    <div style="max-width: 300px;" class="style-scope patent-result">Laid open application for patent or patent of addition</div>
                  
    </div>
  </overlay-tooltip>

                  <div class="knowledge-card-action-bar style-scope patent-result">
                    <a target="_blank" class="style-scope patent-result" href="https://patentimages.storage.googleapis.com/61/12/23/f04976b3fac0fc/TW200418849A.pdf"><iron-icon icon="description" class="style-scope patent-result x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M14 2H6c-1.1 0-1.99.9-1.99 2L4 20c0 1.1.89 2 1.99 2H18c1.1 0 2-.9 2-2V8l-6-6zm2 16H8v-2h8v2zm0-4H8v-2h8v2zm-3-5V3.5L18.5 9H13z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon> Download PDF</a>
                    <state-modifier inherit="" act="{&quot;type&quot;: &quot;QUERY_FIND_PRIOR_ART&quot;, &quot;keywords&quot;: $keywords, &quot;before&quot;: &quot;$before&quot;}" class="style-scope patent-result" data-keywords="[&quot;epothilone&quot;,&quot;seq&quot;,&quot;nucleic acid&quot;,&quot;acid sequence&quot;,&quot;hydroxylase&quot;]" data-before="20011226"><a id="link" href="#" class="style-scope state-modifier" inherit="">
                      <iron-icon icon="find-in-page" class="style-scope patent-result x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M20 19.59V8l-6-6H6c-1.1 0-1.99.9-1.99 2L4 20c0 1.1.89 2 1.99 2H18c.45 0 .85-.15 1.19-.4l-4.43-4.43c-.8.52-1.74.83-2.76.83-2.76 0-5-2.24-5-5s2.24-5 5-5 5 2.24 5 5c0 1.02-.31 1.96-.83 2.75L20 19.59zM9 13c0 1.66 1.34 3 3 3s3-1.34 3-3-1.34-3-3-3-3 1.34-3 3z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon> Find Prior Art
                    </a></state-modifier>
                    <state-modifier inherit="" act="{&quot;type&quot;: &quot;QUERY_SIMILAR_DOCUMENTS&quot;, &quot;id&quot;: &quot;$id&quot;}" class="style-scope patent-result" data-id="patent/TW200418849A"><a id="link" href="#" class="style-scope state-modifier" inherit="">
                      <iron-icon icon="editor:functions" class="style-scope patent-result x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M18 4H6v2l6.5 6L6 18v2h12v-3h-7l5-5-5-5h7z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon> Similar
                    </a></state-modifier>
                  </div>
                </header>

                <dl class="important-people style-scope patent-result">
                  
                    <dt class="style-scope patent-result">Other languages</dt>
                    <dd class="style-scope patent-result"><state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/TW200418849A/zh"><a id="link" href="#" class="style-scope state-modifier">Chinese</a></state-modifier></dd><template is="dom-repeat" strip-whitespace="" class="style-scope patent-result"></template>
                  <template is="dom-if" class="style-scope patent-result"></template>
                  
                  <dt class="style-scope patent-result">Inventor</dt>
                  <dd class="style-scope patent-result"><state-modifier act="{&quot;type&quot;: &quot;QUERY_SEARCH_INVENTOR&quot;, &quot;inventor&quot;: &quot;$inventor&quot;}" class="style-scope patent-result" data-inventor="Jonathan D Basch"><a id="link" href="#" class="style-scope state-modifier">Jonathan D Basch</a></state-modifier></dd><dd class="style-scope patent-result"><state-modifier act="{&quot;type&quot;: &quot;QUERY_SEARCH_INVENTOR&quot;, &quot;inventor&quot;: &quot;$inventor&quot;}" class="style-scope patent-result" data-inventor="Shu-Jen Chiang"><a id="link" href="#" class="style-scope state-modifier">Shu-Jen Chiang</a></state-modifier></dd><dd class="style-scope patent-result"><state-modifier act="{&quot;type&quot;: &quot;QUERY_SEARCH_INVENTOR&quot;, &quot;inventor&quot;: &quot;$inventor&quot;}" class="style-scope patent-result" data-inventor="Suo W Liu"><a id="link" href="#" class="style-scope state-modifier">Suo W Liu</a></state-modifier></dd><dd class="style-scope patent-result"><state-modifier act="{&quot;type&quot;: &quot;QUERY_SEARCH_INVENTOR&quot;, &quot;inventor&quot;: &quot;$inventor&quot;}" class="style-scope patent-result" data-inventor="Akbar Nayeem"><a id="link" href="#" class="style-scope state-modifier">Akbar Nayeem</a></state-modifier></dd><dd class="style-scope patent-result"><state-modifier act="{&quot;type&quot;: &quot;QUERY_SEARCH_INVENTOR&quot;, &quot;inventor&quot;: &quot;$inventor&quot;}" class="style-scope patent-result" data-inventor="Yuhua Sun"><a id="link" href="#" class="style-scope state-modifier">Yuhua Sun</a></state-modifier></dd><dd class="style-scope patent-result"><state-modifier act="{&quot;type&quot;: &quot;QUERY_SEARCH_INVENTOR&quot;, &quot;inventor&quot;: &quot;$inventor&quot;}" class="style-scope patent-result" data-inventor="Li You"><a id="link" href="#" class="style-scope state-modifier">Li You</a></state-modifier></dd><template is="dom-repeat" strip-whitespace="" class="style-scope patent-result"></template>
                  <template is="dom-if" class="style-scope patent-result"></template>

                  <template is="dom-if" class="style-scope patent-result"></template>
                </dl>

                <hr class="style-scope patent-result">

                <application-timeline class="style-scope patent-result">
    
    
    <div class="wrap style-scope application-timeline">
      <div class="header style-scope application-timeline">Worldwide applications</div>
      <div class="event style-scope application-timeline">
        
          <span class="block style-scope application-timeline">
            <span class="year style-scope application-timeline">2003</span>
            <span class="dot style-scope application-timeline"></span>
            
              <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope application-timeline" data-result="patent/TW200418849A/en"><a id="link" href="#" class="style-scope state-modifier">
                <span id="cc" class="unknown style-scope application-timeline" this="">TW</span>
              </a></state-modifier>
              <overlay-tooltip positioning-target="#cc" direction="bottom" class="style-scope application-timeline" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      
                <div class="style-scope application-timeline">Application number: TW092105814A</div>
                <div class="style-scope application-timeline">Filing date: 2003-03-17</div>
                <div class="style-scope application-timeline">Legal status: </div>
              
    </div>
  </overlay-tooltip>
            <template is="dom-repeat" class="style-scope application-timeline"></template>
          </span>
        <template is="dom-repeat" class="style-scope application-timeline"></template>
      </div>
      <hr class="style-scope application-timeline">
      <div class="header style-scope application-timeline">Application TW092105814A events <iron-icon id="legal" icon="icons:help-outline" class="style-scope application-timeline x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M11 18h2v-2h-2v2zm1-16C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8zm0-14c-2.21 0-4 1.79-4 4h2c0-1.1.9-2 2-2s2 .9 2 2c0 2-3 1.75-3 5h2c0-2.25 3-2.5 3-5 0-2.21-1.79-4-4-4z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
        <overlay-tooltip positioning-target="#legal" direction="right" class="style-scope application-timeline" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      
          <div style="max-width: 300px;" class="style-scope application-timeline">
            A timeline of key events for this patent application, including priority claims, publications, legal status, reassignments, and litigation.<br class="style-scope application-timeline"><br class="style-scope application-timeline">
            Google has not performed a legal analysis and makes no representation as to the accuracy or completeness of the events listed.
          </div>
        
    </div>
  </overlay-tooltip>
      </div>
      <div class="event layout horizontal style-scope application-timeline"><div date="" class="external-priority style-scope application-timeline">2002-12-17</div><div class="line style-scope application-timeline"></div><div class="flex title style-scope application-timeline"><template is="dom-if" class="style-scope application-timeline"></template><state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope application-timeline" data-result="patent/US6884608B2/en"><a id="link" href="#" class="style-scope state-modifier"><span class="title-text style-scope application-timeline">Priority claimed from US10/321,188</span></a></state-modifier><template is="dom-if" class="style-scope application-timeline"></template><template is="dom-if" class="style-scope application-timeline"></template><template is="dom-if" class="style-scope application-timeline"></template><template is="dom-if" class="style-scope application-timeline"></template></div></div><div class="event layout horizontal style-scope application-timeline" critical=""><div date="" class="filed style-scope application-timeline">2003-03-17</div><div class="line style-scope application-timeline"></div><div class="flex title style-scope application-timeline"><template is="dom-if" class="style-scope application-timeline"></template><template is="dom-if" class="style-scope application-timeline"></template><state-modifier act="{&quot;type&quot;: &quot;QUERY_SEARCH_ASSIGNEE&quot;, &quot;assignee&quot;: &quot;$assignee&quot;}" class="style-scope application-timeline" data-assignee="Bristol Myers Squibb Co"><a id="link" href="#" class="style-scope state-modifier"><span class="title-text style-scope application-timeline">Application filed by Bristol Myers Squibb Co</span></a></state-modifier><template is="dom-if" class="style-scope application-timeline"></template><template is="dom-if" class="style-scope application-timeline"></template><template is="dom-if" class="style-scope application-timeline"></template></div></div><div class="event layout horizontal style-scope application-timeline" critical=""><div date="" class="publication style-scope application-timeline">2004-10-01</div><div class="line style-scope application-timeline"></div><div class="flex title style-scope application-timeline"><template is="dom-if" class="style-scope application-timeline"></template><state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope application-timeline" data-result="patent/TW200418849A/en"><a id="link" href="#" class="style-scope state-modifier"><span class="title-text style-scope application-timeline">Publication of TW200418849A</span></a></state-modifier><template is="dom-if" class="style-scope application-timeline"></template><template is="dom-if" class="style-scope application-timeline"></template><template is="dom-if" class="style-scope application-timeline"></template><template is="dom-if" class="style-scope application-timeline"></template></div></div><template is="dom-repeat" strip-whitespace="" class="style-scope application-timeline"></template>
      <template is="dom-if" class="style-scope application-timeline"></template>
    </div>
  </application-timeline>
                <hr class="style-scope patent-result">

                <dl class="links style-scope patent-result">
                  <dt class="style-scope patent-result">Info</dt>

                  <template is="dom-if" restamp="" class="style-scope patent-result"></template>

                  <template is="dom-if" restamp="" class="style-scope patent-result"></template>

                  
                  <dd class="style-scope patent-result"><a href="#citedBy" class="style-scope patent-result">Cited by (1)</a></dd>
                  <template is="dom-if" restamp="" class="style-scope patent-result"></template>

                  <template is="dom-if" restamp="" class="style-scope patent-result"></template>

                  
                  <dd class="style-scope patent-result"><a href="#similarDocuments" class="style-scope patent-result">Similar documents</a></dd>
                  <template is="dom-if" restamp="" class="style-scope patent-result"></template>

                  
                  <dd class="style-scope patent-result"><a href="#relatedApplications" class="style-scope patent-result">Priority and Related Applications</a></dd>
                  <template is="dom-if" restamp="" class="style-scope patent-result"></template>

                  <dt class="style-scope patent-result">External links</dt>

                  <dd class="style-scope patent-result"><a target="_blank" class="style-scope patent-result" href="https://worldwide.espacenet.com/publicationDetails/biblio?CC=TW&amp;NR=200418849A&amp;KC=A&amp;FT=D">Espacenet</a></dd><dd class="style-scope patent-result"><a target="_blank" class="style-scope patent-result" href="https://globaldossier.uspto.gov/#/result/publication/TW/200418849/1">Global Dossier</a></dd><dd class="style-scope patent-result"><a target="_blank" class="style-scope patent-result" href="https://patents.stackexchange.com/questions/tagged/TW200418849A">Discuss</a></dd><template is="dom-repeat" strip-whitespace="" class="style-scope patent-result"></template>
                </dl>
              </section>
            </div>
          </div>

          <div id="text" class="horizontal layout style-scope patent-result">
            <div class="flex flex-width style-scope patent-result" marginright="">
              <section id="description" class="style-scope patent-result">
                <h3 class="layout horizontal style-scope patent-result">
                  <div class="flex style-scope patent-result">Description</div>
                  <div class="righthead style-scope patent-result">translated from Chinese</div>
                </h3>
                <patent-text name="description" id="descriptionText" class="style-scope patent-result" style="padding-top: 0px;">
    
    
    <template is="dom-if" restamp="" class="style-scope patent-text"></template>
    <overlay-tooltip id="tooltip" class="style-scope patent-text" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      
    </div>
  </overlay-tooltip>
    <div class="layout horizontal style-scope patent-text">
      <section id="text" class="flex style-scope patent-text"><description mxw-id="PDES113261702" lang="EN" load-source="google" class="style-scope patent-text">
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">I I200418849 (1)   B B  B/       B   AB :</span>I I200418849 (1) (ii) Description of the invention [Technical field to which the invention belongs] The present invention relates to epothilone B hydroxylase and its mutants and variants' and is located downstream of the epothilone B hydroxylase gene The isolated nucleic acid sequence of the ferredoxin and the polypeptide encoded by it. It is also related to recombinant microorganisms that can exhibit epothilone B hydroxylase and its mutants and variants and / or ferredoxin, which can hydroxylate small organic molecular compounds with terminal courtyards, such as Paxomycins produce compounds with terminal hydroxyalkyl groups. A method for the recombinant production of these microorganisms' and a method for synthesizing a compound having a terminal motif based on these recombinant microorganisms are also proposed. The composition and method of the present invention can be used to prepare epothilones which have multiple uses in the medical field. The epothilone analog produced by using the epothilone B hydroxylase mutant of the present invention will be described. [Prior art] Epothilones are macrolide compounds useful in the medical field. For example, epothilone A and B having the following structures:</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">A R =  B R = Me pa C1 it axel) (2) 200418849 (TAXOL)-   Bollag et al.Cancer ResVol. 55No. 112 3 2 5 -2 3 3 3 ( 1 9 9 5 )</span>Epothilone AR =  Epothilone BR = Me It has been found to emit microtubules similar to paclitaxel (pa C1 it axel) (2) 200418849 (TAXOL) and is therefore stable against rapid proliferation Cytotoxic activity of sex cells, such as tumor cells or cells associated with other highly proliferative cell diseases, see Bollag et al., Cancer Res , Vol. 55, No. 11, 2 3 2 5 -2 3 3 3 (1 9 9 5) </span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">ABSorangium cellulosum) Hofle DE 4 1 3 8 042 ; WO 93/10121 ; Angew. Chem. Int. Ed. Engl. Vol. 35 No. 1 3 / 1 4, 1567-1569 ( 1 99 6) J. Antibiot. Vol. 49No. 6 560-563(1996)</span>Epothilone A and B are natural anticancer agents produced by Sorangium cellulosum. They were first isolated and identified by Hofle et al., DE 4 1 3 8 042; WO 93/10121; Angew Chem. Int. Ed. Engl. Vol. 35, No. 1 3/1 4, 1567-1569 (1 99 6); and J. Antibiot., Vol. 49, No. 6, 560-563 (1996) . Subsequently, by</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Balog, et al. Angew. Chem. Int. Ed. Engl., Vol. 35 No.2 3 /24, 2 8 0 1 -2 8 03  Meng et al., J. Am. Chem. Soc.,</span>Balog, et al., Angew. Chem. Int. Ed. Engl., Vol. 35, No. 2 3/24, 2 8 0 1-2 8 03; Meng et al., J. Am. Chem. Soc. ,</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Vol. 119No. 421 0073 - 1 0092 ( 1 997)  Nicolaou et al.? J. Am. Chem. SocVol. 119, No, 347974-7991 (1997)</span>Vol. 119, No. 42, 1 0073-1 0092 (1 997); Nicolaou et al.? J. Am. Chem. Soc , Vol. 119, No, 34, 7974-7991 (1997);</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Schinzer et al., Angew. Chem. Int. Ed. Engl., Vol. 36, No. 5, 523-524 (1997); Yang et al., Angew. Chem. Int. Ed. Engl., Vol. 36No.1/21 6 6 - 1 6 81 9 9 7  A BWO 9 8/25 929A B  WO 98/2 2461 CDE Fd S  6 7 7 31  WO 00/3 92 76  SC15847(ATCC P T - 1 0 4 3 ) (Amycolatopsis orientalis)B F -8- (3) (3)200418849 P-4 5 0 ^heme)fn  45 0nm) P -4 5 0    P-450  Streptomyces tendae) nikkomycin(Bruntner, C. e t al. 1 999Mol. Gen. Genet. 262:1 02- 1 1 4)doxorubicin (Dickens, M.L.Strohl, W.R. 1 996J. Bacteriol1 7 8:3 3 8 9-3 3 9 5 ) JulienB. et al.2000, Gene, 249:153-16 0)P-4 5 0  NADHNADH-  P-450 LewisD.F., HlavicaP.2000, Biochem. Biophys. Acta.1 460:3 5 3 -3 74)  P-450  B  -9- (4) (4)200418849  B    B i B B  vector) B/     B /       -10- (5) (5)200418849      A</span>Schinzer et al., Angew. Chem. Int. Ed. Engl., Vol. 36, No. 5, 523-524 (1997); and Yang et al., Angew. Chem. Int. Ed. Engl., Vol. 36, No. 1/2, 1 6 6-1 6 8, 1 9 9 7 etc. disclose the overall synthesis of epothilones A and B. WO 9 8/25 929 discloses a chemical synthesis method of epothilone A, epothilone B, analogs of epothilone and epothilone analogs. WO 98/2 2461 discloses the structure of epothilones C, D, E, and F and the production from Fibriocystis d S M 6 7 7 3. Figure 1 presents a diagrammatic illustration of epothilone B turning into Escherichia coli in actinomycete strain SC15847 (ATCC PT-1 0 4 3)-later identified as Amycolatopsis orientalis, described in WO 00/3 92 76 Biotransformation of Talimycin F. -8- (3) (3) 200418849 Cytochrome P-4 50 enzymes are present in prokaryotic and eukaryotic cells and collectively have a ^ heme binding function fn 'This functional site is associated with carbon monoxide After recombination, they can be distinguished by the absorption peak at 450 nm. Cytochrome P-4 50 enzymes can perform a wide range of oxidative reactions with alkanes on predominantly hydrophobic substrates, including aromatic and benzoic rings. In prokaryotes, they exist as a detoxification system and serve as the first enzymatic step to metabolize substrates such as toluene, benzene and camphor. Cytochrome P-450 enzymes are also present in secondary metabolite biosynthetic pathways, such as nikkomycin (Bruntner, C. et al., 1 999, Mol. Gen. Genet. 262) in Streptomyces tendae. 1 02- 1 1 4), doxorubicin (Dickens, ML, Strohl, WR, 1 996, J. Bacteriol, 1 7 8: 3 3 8 9-3 3 9 5) and epothilone Biosynthetic clusters (Julien, B. et al., 2000, Gene, 249: 153-16 0). With few exceptions, the cytochrome P-4 50 system is composed of three proteins in prokaryotes ... one is the iron redox protein NADH or NADH-dependent reductase, the other is iron-thioferredoxin and Cytochrome P-450 enzymes (Lewis, DF, Hlavica, P., 2000, Biochem. Biophys. Acta., 1 460: 3 5 3 -3 74). The electron system is transferred from ferredoxin reductase to ferredoxin and finally to the cytochrome P-450 enzyme to promote the division of molecular oxygen. [Summary of the Invention] An object of the present invention is to propose an isolated nucleic acid sequence encoding epothilone B hydroxylase and a variant or mutant thereof, and encode ferredoxin-9- (4) (4) 200418849 or An isolated nucleic acid sequence of a variant or mutant thereof. Another item of the present invention is to propose an isolated polypeptide comprising an amino acid sequence possessed by epothilone B enzyme and its variants or mutants, and an enzyme comprising ferredoxin and its variants or mutants. Isolated polypeptide of amino acid sequence. Another item of the present invention is to propose the structural coordinates of an epothilone B hydroxylase homogeneity model. The structural coordinates are those listed in Appendix i. This model of the present invention provides a tool for designing modulators of the biological functions of epothilone B hydroxylase and other mutants of epothilone B enzyme with altered specificity. Another item of the present invention is to propose a vector comprising an isolated nucleic acid encoding epothilone B hydroxylase or a variant or mutant thereof and / or a ferredoxin or a variant or mutant thereof. sequence. In a preferred embodiment, these vectors further include a nucleic acid sequence encoding a ferredoxin. Another item of the present invention is to propose a host cell comprising a vector containing a vector encoding epothilone B hydroxylase or a variant or mutant thereof and / or a ferredoxin or a variant or mutant thereof. Nucleic acid sequence. Another item of the present invention is to propose a method for producing recombinant microorganisms. These recombinant microorganisms can hydroxylate compounds having terminal alkyl groups, especially epothilones, to produce compounds having terminal hydroxyalkyl groups. Compounds. Another item of the present invention is to propose a recombinantly produced microorganism capable of hydroxylating a compound having a terminal alkyl group, particularly epothilone-10- (5) (5) 200418849 to produce Terminal hydroxyalkyl compounds. Another item of the present invention is to propose a method for hydroxylating a compound in such a recombinant microorganism. In particular, the present invention proposes a method for producing epothilone loaded with a hydroxyl group. These compounds can be used as antitumor agents and as starting materials in the preparation of other epothilone analogs. Another item of the present invention is to propose a compound of formula A:</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">A []      ^    -11 - ^418849 (6</span>Include the composition of the compound of formula A and the method of manufacturing the composition. [Detailed description of the invention] The present invention relates to an isolated nucleic acid sequence and a polypeptide, and a method for obtaining a compound having a desired substituent at a terminal carbon position. In particular, the present invention proposes compositions and methods for preparing epothilones bearing hydroxyalkyl groups. These compounds can be used as antitumor agents and can be used as starting materials in the preparation of other epothilone analogs. . ^ Poramycin ", as used herein, means a compound containing an epothilone core and a side chain group as defined herein. The term" epothilone "as used herein As used in this table, the table contains parts of the following nuclear structure (showing the number of the ring system position used in this paper): -11-^ 418849 (6</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text"> Q</span>The substituents are as follows: Q is selected from the group consisting of:</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">w0NR6 ; X R7 ;   sNR8CR9R10 ; B1B20RM(1112</span>w is 0 or NR6; X is selected from the group consisting of: ,  and 7;  is 0, s, NR8, CR9R10; B1 and B2 are selected from 0RM,  (: 1112 Among the</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Ri-R5R^-R75J  R1R2  R6  R7 fD Ri:   R8 -12- (7) (7)200418849 r13c = or14c = o  r15so2; R9R1Q R i 6 C =  R 1 7  C =    AB GGiG2</span>Ri-R5 and R ^ -R 7 are selected from the group consisting of  5J , , , substituted alkyl, aryl, and heterocyclic group, and wherein R1 and R2 are alkane R6 is selected from the group consisting of: , alkyl, and substituted alkyl; R7 fD Ri: is selected from the group consisting of : Fluorene, alkyl, substituted alkyl, trialkylsilyl, alkyldiarylsilyl, and dialkylarylsalyl; R8 is selected from the group consisting of fluorene, alkyl , Substituted alkan-12- (7) (7) 200418849, r13c = o, r14c = o, and r15so2; and R9 and R1Q are selected from the group consisting of: fluorene, halogen, alkyl , Substituted alkyl, aryl, heterocyclyl, hydroxy, R i 6 C = , and R 1 7 OC =   The term "side chain" refers to epothilone A or Substituent G as defined by B or Gi and G2 shown below.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">GiV HO-CH2-(Ai)n-(Q)m-(A2) (V)</span>Gi has the following formula V HO-CH2- (Ai) n- (Q) m- (A2)  (V)</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">g2VI </span>And g2 has the following formula VI where</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">AiA2 Ci-Cs Q13  m 1m1  1 5 - 1 5 -13- (8) (8)200418849   12 017 14  14    2     S2NH2) CNH2) CONHC0NHcNH       &amp;amp;  -14- 200418849 ) B-     B P 4 5   DNARNACDNAmRNA  DNADNA  60%  80%homology) stringent conditio )  6 0 C  2xSSC65C 3XSSC  S E Q ID N 0 : 13 03 23 43 6 3738394041426062 ' 646668707 2 74SEQ ID NO:316 SEQ ID _ 130323436 3738' 3940' 4  42' 6062646668707 2 74SEQ ID NO:3 .5X SSC55C (10) (10)200418849  SEQ ID :1303234363738 39  40  41  42  60  62  64  66  68  70  72  74 ^ SEQ ID NO:330SEQ ID NO : 130323436373839404 142 60626466687072  74  SEQ ID NO:3  3 0 SEQ ID NO: 1 DNA B SEQ ID NO:3DNA      Amycolata autouophica  SEQ ID NO :3 6-42   SEQ ID :13 SEQ ID NO : 13  -16- (11) (11)200418849  /B 1 13 SEQ ID NO : 1  GLU31ARG67ARG88ILE92ALA93 VAL106  ILE130  ALA140  MET176  PHE190  GLU231SER294PHE237 ILE365  SEQ ID NO: 3 0 &amp;gt; 323460626466687072 74  BSEQ ID :1 P45 0 PCR P4 5 0 ij 1  Locus : STMSUACB M3 223 8 ; OmerC. A.J. Bacteriol. 1 72:3 3 3 5 -3 3 4 5 ( 1 990) []2  Locus  STMSUBCB ;M3 223 9  OmerC.A.J. Bacteriol. 1 72:3 3 3 5 -3 3 4 5 ( 1 990)  ij 3  Locus  ABO 1 80 74( tu STMORFA) AB 0 1 8074 ;Ueda K. J. Antibiot. 48:638-646 (1995)  4 : Locus : SSU65940 ;U65940 ; MotamediH J. Bacteriol. 1 7 8:5243 - 524 8 (1990) (12) (12)200418849 [j 5  Locus  STMOLEP ;L3 7200  Rodriguez A.M. FEM S Microbiol. Lett. 127:117-120 (1995) [j 6: Locus  SERCP4 5 0 A M83110; AndersenJ.F. and Hutchinson C.R.J. Bacteriol. 1 74:72 5 - 73 5 ( 1 992)  Myers, E.W. and W. Miller, 1 9 8 8. CABIOS 4:1, 11-17  Scientific and Education</span>Ai and A2 are independently selected from the group consisting of: optionally substituted Ci-Cs alkyl and alkenyl; Q is optionally substituted ring system containing 1 to 3 rings and in At least one ring contains at least one carbon-to-carbon double bond; and , m, and 0 are independently selected from the group consisting of: 0 and 1, wherein at least one of m, , or 0 is 1 . The term "terminal carbon" or "terminal alkyl" refers to the terminal carbon or terminal methyl group contained in the part which is directly bonded to the 15-position of the epothilone core or to the position 15 The side chain group contains a terminal carbon or a terminal courtyard. To understand -13- (8) (8) 200418849, the term "alkyl" includes alkyl and substituted alkyls as defined herein. The term "alkyl" refers to a linear or branched, optionally substituted A chain-type saturated hydrocarbon group having 1 to 20 carbon atoms, preferably 1 to 7 carbon atoms. The expression "low-carbon alkyl group" refers to an optionally substituted alkyl group having 1 to 4 carbon atoms. The term "substituted alkyl" refers to an alkyl group containing 1 to 4 substituents, for example, trifluorofluorenyl, trifluoromethoxy, hydroxyl, alkoxy, cycloalkoxy, heteroepoxy Group, keto group, alkylamino group, aryloxy group, alkyloxy group, amine group, alkylamino group, arylamino group, aralkylamino group, cycloalkylamino group, heterocyclic amino group; The two substituents on the amine group are selected from the group consisting of alkyl, aryl or aralkyl; alkylamino, arylamido, aromaticamido, substituted alkylamino, substituted aromatic Amine, substituted arylamine, thiol, alkylthio, arylthio, aralkylthio, cycloalkylthio, heterocyclicthio, alkylthioketone, arylthioketone, aralkyl Thione, alkanesulfonyl, arylsulfonyl, aralkylsulfonyl, sulfonamido (eg, S02NH2), substituted fluorenylamino, nitro, cyano, carboxyl, aminoformamyl (eg CONH2), substituted aminomethyl amidino (for example, CONH alkyl, CONH aryl, CONH aralkyl, or nitrogen with two substituents selected from alkyl, aryl, or aralkyl Person), alkoxycarbonyl, aryl, Aryl group, guanidino and heterocyclyl, e.g., indolyl, imidazolyl, furyl, thienyl, thiazolyl, pyrrolyl, pyridinyl, pyrimidinyl, and the like. Where it is mentioned above that the substituent further contains a substituent, it is a halogen, an alkyl group, an alkoxy group, an aryl group or an aralkyl group. &amp;amp; According to one aspect of the present invention, an isolated polynucleotide is proposed, which encodes Ang-14-200418849 ) Polanin B hydroxylase -... a kind of epothilones having terminal alkyl groups To make an isolated polynucleotide with a terminal hydroxyalkyl group of salinomycins). According to another aspect of the present invention, an isolated polynucleotide is proposed which encodes a ferredoxin gene whose gene line is located downstream of the epothilone B hydroxylase gene. Ferredoxin is a protein of the cytochrome P 4 50 system. A "polynucleotide", as used herein, is intended to include any form of DNA or RNA, such as CDNA or mRNA, which encodes an enzyme or an active fragment thereof that is produced by breeding or synthetically produced by well-known chemical techniques . DNA can be double-stranded or single-stranded. Single-stranded DNA can include password or meaningful shares or non-cryptographic or anti-meaning shares. Therefore, the term polynucleotide also includes homology to at least 60% or higher, preferably at least 80% or higher, for the sequences disclosed herein, and can be used under stringent conditio  ) To the above-mentioned polynucleotide. As used herein, "urgent conditions" means hybridization conditions at 60  C in 2xSSC buffer. More preferably, it can hybridize to SEQ ID N 0 under the hybridization conditions of 3XSSC at 65  C: 1, 3 0, 3 2, 3 4, 3, 6, 37, 38, 39, 40, 41, 42, 60, 62 '64, 66, 68, 70, 7 2 or 74 or 16 hours of isolated nucleic acid molecules of the nucleic acid sequence described in SEQ ID NO: 3, or are capable of hybridizing to SEQ ID NO. 1, 30, 32, 34, 36 , 37, 38 '39, 40' 4 and 42 '60, 62, 64, 66, 68, 70, 7 2 or 74 or the complementary sequence of the nucleic acid sequence described in SEQ ID NO: 3, the nucleic acid molecule is isolated, Or can be hybridized to SEQ ID NO: 1, 30, 32, 34, 36, 37, 38, 39, 40, 41, 42 under wash conditions of 0.5X SSC, 55  C (10) (10) 200418849 , 60, 62, 64, 66, 68, 70, 72 or 74 ^ the nucleic acid sequence described in SEQ ID NO: 3 for 30 minutes, or hybridized to SEQ ID NO: 1, 30, 32, 34, 36, 37, 38, 39, 40, 4 1, 42, 60, 62, 64, 66, 68, 70, 72, or 74 or the complementary sequence of the nucleic acid sequence described in SEQ ID NO: 3 The nucleic acid molecule is isolated after 30 minutes. In a specific example, the polynucleotide of the present invention includes the genomic DNA shown in SEQ ID NO: 1 or a homologous sequence or fragment thereof, which encodes a polypeptide having an activity similar to that of epothilone B hydroxylase. In addition, the polynucleotide of the present invention may include the genomic DNA shown in SEQ ID NO: 3 or a homologous sequence or fragment thereof, which encodes a polypeptide having an activity similar to that of the ferredoxin. Because of the simplicity of the genetic code, the polynucleotides of the present invention may also include other nucleic acid sequences, variants or active fragments that encode this enzyme and derivative. The invention also relates to variants of these polynucleotides, which may be naturally occurring 'i.e., present in microorganisms such as Streptomyces orientalis or Amycolata autouophica, or in soil or other sources from which nucleic acids can be isolated Or, it may be a mutant made by a well-known mutation generation technique. Exemplary variant polynucleotides of the invention are those shown in SEQ ID NOs: 3 6-42. The term "mutant", as used herein, is intended to cover deletions or additions of one or more point mutations, or nucleic acids, relative to SEQ ID NO: 1 or 3, but still encoding SEQ ID NO: 1 or 3 The nucleic acid sequence of a polypeptide or a fragment thereof having the activity of the polypeptide encoded by it. In a better specific example of -16- (11) (11) 200418849, mutations were made to alter the substrate specificity and / or yield of the enzyme. A preferred mutation region for the epothilone B hydroxylase gene is a nucleotide sequence region encoding about 113 amino acid residues including the active site of the enzyme. In addition, it is also preferred to be encoded at the amino acid positions GLU31, ARG67, ARG88, ILE92, ALA93, VAL106, ILE130, ALA140, MET176, PHE190, GLU231, SER294, PHE237, or ILE365 in SEQ ID NO: 1 At least one amino acid substituted. Exemplary polynucleotide mutants of the present invention are shown in SEQ ID NO: 3 0 &amp;gt; 32, 34, 60, 62, 64, 66, 68, 70, 72, and 74. The clones of the nucleic acid sequence of SEQ ID NO: 1 encoding epothilone B hydroxylase use PCR primers designed by aligning the nucleic acid sequences of six cytochrome P450 genes obtained from bacteria. The following cytochrome P4 50 genes are ranked by comparison: Sequence  ij 1: Locus: STMSUACB; Accession number: M3 223 8; References: Omer, CA, J. Bacteriol. 1 72: 3 3 3 5 -3 3 4 5 (1 990) Preface [] 2: Locus: STMSUBCB; Registration number: M3 223 9; References: Omer, CA, J. Bacteriol. 1 72: 3 3 3 5 -3 3 4 5 (1 990) Preface ij 3: Locus: ABO 1 80 74 (tu STMORFA); accession number: AB 0 1 8074; reference: Ueda, K., J. Antibiot. 48: 638-646 (1995) sequence 4: Locus: SSU65940; Registration number: U65940; References: Motamedi, H., J. Bacteriol. 1 7 8: 5243-524 8 (1990) (12) (12) 200418849 Preface [j 5: Locus: STMOLEP; Registration number: L3 7200 References: Rodriguez, AM, FEM S Microbiol. Lett. 127: 117-120 (1995) Preface [j 6: Locus: SERCP4 5 0 A; Accession number: M83110; References: Andersen, JF and Hutchinson, CR , J. Bacteriol. 1 74:72 5-73 5 (1 992) The ratio is obtained by using Myers, EW and W. Miller, 1 9 8 8. CABIOS 4: 1, 11-17 from Scientifi c and Education</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Software (DurhamNorth CarolinaUSA) I- K-B-B-bulge)L-  P45 0- 1 +(SEQ ID NO:23) P4 50-la + (SEQ ID NO:24)I P450-2 + (SEQ ID NO:25)B-L P4 5 0 -3_(SEQ ID NO:27)  PCR) PCR  Qiaquick (QiagenSanta Clarita, CaliforniaUSA)!  DNA PCRscript Amp  (Stratagene) PCRscript Stratagene La Jolla, CaliforniaUSA) 30-37 C   230-300 (13) 200418849 r pm16-24 100/LBMo Bio Mo BioSolano BeachCalifornia USA)DNAPCR </span>Software (Durham, North Carolina, USA). Three highly conserved regions were identified, in the I-helix, containing oxygen-binding functional sites, in the K-helix, and across B-bulge and L-spirals, containing conserved blood matrix-binding functional sites. Primers were designed for the three conserved regions identified in the alignment. Primers P45 0- 1 + (SEQ ID NO: 23) and P4 50-la + (SEQ ID NO: 24) were designed from the I helix, and primer P450-2 + (SEQ ID NO: 25) was from B- The designer of the bulge and L. helix region and primer P4 50 -3_ (SEQ ID NO: 27) was designed to be anti-complementary to blood matrix-binding proteins. The genomic fragment is then amplified by polymerase chain reaction (PCR). After PCR amplification, the reaction product was separated by gel electrophoresis and a fragment having a desired size was cut out. DNA was extracted from agarose gel slices using the Qiaquick gel extraction program (Qiagen, Santa Clarita, California, USA). The fragments were then cloned into a PCRscript vector (Stratagene, La Jolla, California, USA) using a PCRscript Amp selection kit (Stratagene). At 30-37  C  at 230-300 (13) 200418849 r pm, 16-24 hours, the colonies containing the inserts were picked up in ml of LB liquid medium containing 100 g / ml of ampicillin. The plastid separation was performed using a Mo Bio mini plastid preparation kit (Mo Bio, Solano Beach, California, USA). Use this plastid DNA as a PCR and sequencing template and for restriction digestion analysis.</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">PCR Applied Biosystems (Foster* City, CaliforniaUSA) Big-Dye   ABI310 Applied BiosystemsFoster City CaliforniaUSA) AltschulS.Fet al.Mol. Biol.215 :403-410 (1990) TblastX P45 0  SCI 5 8479 P450DNA cDNAP450DNAcDNA 50cDNA 20DNA  Big-DyeApplied Biosystems) DNAABI3 10 SEQ ID NO: 1BgluII 40444.7 kD a  SEQ ID NO :2 -19&amp;quot; (14) 200418849</span>The selected PCR products were sequenced using a Big-Dye sequencing kit from Applied Biosystems (Foster * City, California, USA) and performed using an ABI310 sequencer (Applied Biosystems, Foster City, California, USA). analysis. The sequence of the insert was used to perform a TblastX search of a non-abundant protein database via the scheme using Altschul, S.F., et al., Mol. Biol. 215: 403-410 (1990). A unique sequence with significant similarities to known cytochrome P450 proteins is retained. In this way, a total of 9 different P450 sequences were identified from SCI 5 847, of which seven were derived from genomic DNA templates and two were derived from cDNA. Two of these P450 sequences were found to be in common between the two templates of DNA and cDNA. Among the 50 cDNA clones analyzed, there were two sequences as the main ones, each with 20 clones. These two sequences were then selected from the genomic DNA. The nucleic acid sequence of this genomic DNA was determined using a Big-Dye sequencing system (Applied Biosystems) and analyzed using an ABI3 10 sequencer. This sequence is shown in SEQ ID NO: 1. In the selected BgluII fragment, an open editing framework code encoding a protein with 404 amino acids and an expected molecular weight of 44.7 kD a was found. The derived amino acid sequence of this polypeptide is shown in SEQ ID NO: 2. This peptide has an amino acid sequence of -19 &amp;quot; (14) 200418849</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">NikFBruntnerC. et al. 1 999, Mol. Gen. Genet. 262:1 02- 1 1 4) 51% S. carbophilus)Sca-248% P45 0105 P450 3 5 6B ebh64 ATGebh  9 S. griseoulus(0Keefe, D.P., et al, 1991, Biochemistry 3 0:44 7-4 5 5 ) S. noursei)(BrautasetT.et al.2000Chem. Biol. 7: 3 95 -4 03 )50% SEQ ID NO:3 SEQ ID NO :4 </span>Li Jing found that the NikF protein (Bruntner, C. et al., 1 999, Mol. Gen. Genet. 262: 1 02- 1 1 4) shared 51% of the same rate with Streptomyces donde and was similar to Streptomyces carbophilus (S carbophilus) Sca-2 protein has a total identity of 48%. Both enzymes belong to the cytochrome P45 0 family 105. The cysteine system found in all blood matrix-binding functional sites contained in all cytochrome P450 enzymes appears at residues 3 56. This epothilone B hydroxylase gene is named ebh. The ATG start codon line of the putative iron redox protein gene with 64 amino acids appears below the stop codon of ebh. This enzyme was found to be associated with S. griseoulus (0'Keefe, DP, et al., 1991, Biochemistry 3 0:44 7-4 5 5) and S. noursei (Brautaset, T., et al., 2000, Chem. Biol. 7: 3 95-4 03), the iron redox protein gene has a 50% identical rate. The nucleic acid sequence encoding the ferredoxin is shown in SEQ ID NO: 3 and the amino acid sequence of the ferredoxin polypeptide is shown in SEQ ID NO: 4.</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">ebh P4 5 01  ebh43 49 1  ebhl 493 0  ebh53 5 60  ebh5 3 5 5 0  ebh39444ebh43333 ebh35165 SEQ ID NO :3 6-42  -20 - (15) (15)200418849 1   ATCC ID  e b h  43 49 1  ebh43491  1 4 93 0  ebh 1 4 93 0  5 3 5 6 0  ebh53560  5 3 5 5 0  ebh53550  3 9444  ebh39444  4 3 3 3 3  ebh4 3 3 3 3  3 5 165  e b h 3 5 1 6 5  ebh43491  ebhl4930  ebh53560  ebh53550ebh39444ebh43333 ebh3 5 1 6 5  SEQ ID NO :43 -492 </span>The ebh gene sequence was also used to isolate the mutant cytochrome P4 50 gene from other microorganisms. Table 1 below lists exemplary polynucleotides ebh43 49 1, ebhl 493 0, ebh53 5 60, ebh5 3 5 5 0, ebh39444, ebh43333, and ebh35165 and the species used for isolation of the variants of the present invention. The nucleic acid sequences of these variants are shown in SEQ ID NO: 3 6-42. -20-(15) (15) 200418849 Table 1. Variant polynucleotide ATCC ID species ebh gene number 43 49 1 Streptomyces orientalis ebh43491 1 4 93 0 Streptomyces orientalis ebh 1 4 93 0 5 3 5 6 0 Streptomyces orientalis ebh53560 5 3 5 5 0 Streptomyces orientalis ebh53550 3 9444 Streptomyces orientalis ebh39444 4 3 3 3 3 Streptomyces orientalis ebh4 3 3 3 3 3 5 165 Streptomyces orientalis ebh 3 5 1 6 5 Exemplary variants ebh43491, ebhl4930, ebh53560, ebh53550, ebh39444, ebh43333, and ebh3 5 1 6 5 The amino acid sequences encoded are shown in SEQ ID NOs: 43 -49 In. Table 2 shows a summary of the amino acid substitutions of these exemplary variants.</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">-21 - (16) 200418849 2 :     e b h  ebh    100 G1 y S e r e b h 1 4 9 3 0  ebh4 3 3 3 3 ebh53550      ebh43491    10 1 Ly s Arg ebhl4930    13 0 lie Leu ebhl4930    192 S er Gin ebhl4930    224 S e r Thr ebhl4930 ' ebh43 3 3 3 ebh53550      ebh43491    285 lie Val ebhl4930  ebh4 3 3 3 3 ebh53550      ebh43491    69 S e r A s n ebh43333    256 Val Ala ebh43333  ebh 5 3 5 5 0 ebh43 49 1  93 Ala Ser ebh53550    326 Asp G1 u ebh53550  eb h4 3 4 9 1   3 3 3 Thr Ala ebh53550  ebh4349 1   13 3 Leu Met ebh43 49 1    3 98 His Arg ebh39444</span>-21-(16) 200418849 Table 2: Amino acid position ebh substitution ebh variant 100 G1 y Serebh 1 4 9 3 0, ebh4 3 3 3 3, ebh53550, ebh43491 10 1 Ly s Arg ebhl4930 13 0 lie Leu ebhl4930 192 S er Gin ebhl4930 224 S er Thr ebhl4930 'ebh43 3 3 3, ebh53550, ebh43491 285 lie Val ebhl4930, ebh4 3 3 3 3, ebh53550, ebh43491 69 S er A sn ebh43333 256 Val 5 5 ebh 5 ebh , Ebh43 49 1 93 Ala Ser ebh53550 326 Asp G1 u ebh53550, eb h4 3 4 9 1 3 3 3 Thr Ala ebh53550, ebh4349 1 13 3 Leu Met ebh43 49 1 3 98 His Arg ebh39444</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">ebh //  SEQ ID NO:30323460 626466687072  74  SEQ ID NO:30ebh25-l  -22- (17) (17)200418849  pANT 849ebh25-l the Budapest Treaty) The International Depository</span>Mutations were also introduced into the codon of the ebh gene to identify improved substrate yields, and / or biotransformation rates and / or altered substrate specificities. Exemplary mutant nucleic acid sequences of the invention are shown in SEQ ID NOs: 30, 32, 34, 60, 62, 64, 66, 68, 70, 72, and 74. The nucleic acid sequence silicon-encoding mutant ebh25-1 'of SEQ ID NO: 30, whose -22- (17) (17) 200418849 exhibits altered substrate specificity. The pANT 849ebh25-l plastid containing this mutant gene is known under the terms of the Budapest Treaty as The International Depository</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Authority200211___  1 0 8 0 1 University Boulevard, Manassas, Virginia 201 10-2209  the American Type Culture Collection  AT C C________   3 05    25(|  NPB29-mut25f(SEQ ID NO:58) NPB29-mut25r(SEQ ID 59)) S. lividans) BFHPLCLC-MS5 23M.W.) B DNANPB29-6f (SEQ ID NO:28) NPB29-7r (SEQ ID NO:29)PCR 17)Big-Dye ebh25-l 1 95 2942 3 8 (18) (18)200418849 2 SEQ ID 30 SEQ ID NO:3 1 SEQ ID NO :32ebh 10-53   10( NPB29-mutl0f(SEQ ID NO:54) NPB29-mutl0r(SEQ ID   : 5 5 )) F DNA NPB29-6f (SEQ ID NO:28) NPB29-7r (SEQ ID NO: 29) PCR16) Big-DyeebhlO-53  23 119023 1 190 SEQ ID NO: 32 SEQ ID NO :33 SEQ ID NO:34ebh24-16  24(NPB 29-mut24f(SEQ ID N0.56) NPB29_mut24r(SEQ ID NO:57)) ebh24-16 F DNA NPB29-6f (SEQ ID NO:28) NPB29-7r (SEQ ID NO:29) PCR  (19) (19)200418849 Big-Dye ebh24-16 23 7 922 3 792 SEQ ID NO :34SEQ ID NO:35  SEQ ID NO:60ebh24-16d8 59(NPB 29-mut59(SEQ ID NO:77) (S. rimosus)ebh24-16d8 FDNA NPB29-6f(SEQIDNO:2 8)NPB29-7r(SEQID NO: 2 9)PCR Big-Dyeebh24-16d8  6 7 SEQ ID NO: 60 SEQ ID NO:61 SEQ ID NO:62ebh24-16cl 1 59(NPB29 mut59(SEQ ID NO:77) ebh24-16cllF DNA (20) (20)200418849  NPB29-6f (SEQ ID NO:28) NPB29-7r (SEQ ID NO:29) P C RB i - D  e ebh24_16cl 1 9 3 30593 3 6 5 S E Q ID   : 6 2 SEQ ID NO:63 SEQ ID N:64ebh24-16-16 ebh24_ 16 ebh24-16-16F DNANPB29-6f (SEQ ID NO:28) NPB29-7r (SEQ ID NO:29) PCR  Big-Dye ebh24-16-16 16SEQ ID NO :64SEQ ID NO:65  SEQ ID NO :34ebh24-16  24( NPB29-mut24f(SEQ ID NO:56) NPB29-mut24r(SEQ ID NO:57)) ebh2 4-16 F -26- (21) (21)200418849  DNA NPB29-6f (SEQ ID NO:28) NPB29-7r (SEQ ID NO:29) PCR  Big-Dye ebh24-16 2 3 7 9223 792 SEQ ID NO :34SEQ ID NO : 3 5  SEQ ID NO:60ebh24-16d8 59(NPB 29-mut59(SEQ ID NO:77) (S. rimosns)ebh24-16d8 FDNA NPB29-6f(SEQIDNO:28)NPB29-7r(SEQID NO:29)PCR Big-Dyeebh24-16d8  6 7 SEQ ID NO: 6 S E Q ID N 0 : 6 1 SEQ ID NO:62ebh24-16cl 1 59(NPB29-mut59(SEQ ID NO77) (22) 200418849 ebh24-16cl 1 F DNA  NPB29-6f (SEQ ID NO:28) NPB29-7r (SEQ ID NO :29)</span>Authorized and accepted by Authority. The deposit was completed on November ___, 2002 at the American Type Culture Collection at 1 08 0 1 University Boulevard, Manassas, Virginia 201 10-2209. Its AT C C login number is ________. All restrictions on public access to this plastid will be irretrievably removed after this patent application is granted. The deposit shall be kept in the public depository for a period of 30 years after the deposit date or for a period of 5 years after the last sample application or the period of patent enforcement, whichever is longer. If the depository cannot dispense live samples, the deposit must be renewed. This S. lividans transformant identified in mutation 25 (Bow | daughter NPB29-mut25f (SEQ ID NO: 58) and NPB29-mut25r (SEQ ID NO.  59)) was identified It was found to produce products with different HPLC elution times than epothilone B or epothilone F. This unknown sample was analyzed by LC-MS and found to have a molecular weight (M.W.) of 5 to 23, consistent with the monohydroxylation of epothilone B. Plastid DNA was isolated from this Streptomyces lividans culture and used as a template in PCR amplification using primers NPB29-6f (SEQ ID NO: 28) and NPB29-7r (SEQ ID NO: 29) (see Example 17). The expected fragments were taken and sequenced using the Big-Dye sequencing system. This ebh25-1 mutant was found to have two mutations that caused a change in the amino acid sequence of the protein, asparagine 195 became serine and serine 294 became proline. The position of the mutation directed to the codon 2 38 was (18) (18) 200418849. It was found that there was a 2 nucleotide change which did not cause a change in the amino acid sequence of the protein. The amino acid sequence of the mutant polypeptide encoded by SEQ ID NO  30 is shown in SEQ ID NO: 31. SEQ ID NO: 32 nucleic acid sequence encodes mutant ebh 10-53, which exhibits improved biotransformation yields. The Streptomyces lividans transformants identified in the screening of mutation 10 (primer NPB29-mutl0f (SEQ ID NO: 54) and NPB29-mutl0r (SEQ ID NO: 5 5)) were found to produce larger Epothilone F yield. Plastid DNA was isolated from this Streptomyces lividans culture and used as a template in PCR amplification using primers NPB29-6f (SEQ ID NO: 28) and NPB29-7r (SEQ ID NO: 29) (see Examples 16). The expected fragments were taken and sequenced using the Big-Dye sequencing system. This ebhlO-53 mutant was found to have two mutations that caused a change in the amino acid sequence of the protein, the amino acid 23 1 was changed to arginine and the phenylalanine 190 was changed to tyrosine. Position 23 1 is the target of mutation generation, and the change at residue 190 is an unexpected change of the artificial product of the mutation generation program. The amino acid sequence possessed by the mutant polypeptide encoded by SEQ ID NO: 32 is shown in SEQ ID NO: 33. The nucleic acid sequence of SEQ ID NO: 34 encodes a mutant ebh24-16, which also exhibits improved biotransformation yields. The Streptomyces lividans transformant, ebh24-16, identified in the screening of mutation 24 (primer NPB 29-mut24f (SEQ ID N0.56) and NPB29_mut24r (SEQ ID NO: 57)) can also produce relatively Large epothilone F yield. Plastid DNA was isolated from this Streptomyces lividans culture and used as a model in PCR amplification using primers NPB29-6f (SEQ ID NO: 28) and NPB29-7r (SEQ ID NO: 29) (19) ( 19) 200418849 board. The expected fragments were taken and sequenced using the Big-Dye sequencing system. The ebh24-16 mutant was found to have two mutations that caused a change in the amino acid sequence of the protein, phenylalanine 23 7 changed to alanine and isoleucine 92 changed to valine. Position 2 3 7 is the target of mutation generation, and the change at residue 92 is an unexpected change of the artificial product of the mutation generation program. The amino acid sequence of the mutant polypeptide encoded by SEQ ID NO: 34 is shown in SEQ ID NO: 35. The nucleic acid sequence of SEQ ID NO: 60 encodes mutants ebh24-16d8, which also exhibits improved biotransformation yields. This S. rimosus transformant, ebh24-16d8, identified in the screening of mutation 59 (primer NPB 29-mut59 (SEQ ID NO: 77)) can also produce larger epothilones Yield F. Plastid DNA was isolated from this Streptomyces riftum culture and used as a template in PCR amplification using primers NPB29-6f (SEQ ID NO: 2 8) and NPB29-7r (SEQ ID NO: 29). The expected fragment was sequenced using the Big-Dye sequencing system. The ebh24-16d8 mutant was found to have a mutation that caused one of the changes in the amino acid sequence of the protein, and the spermine 6 7 was changed to amidamine. The artificial product of a mutation generating program. The amino acid sequence of the mutant polypeptide encoded by SEQ ID NO: 60 is shown in SEQ ID NO: 61. The nucleic acid sequence of SEQ ID NO: 62 encodes the mutant ebh24- 16cl 1, which also exhibits improved biotransformation yield. This Streptomyces fissionus transformant, ebh24-16cll, identified in the screening of mutation 59 (primer NPB29  mut59 (SEQ ID NO: 77)), can also be Produces a larger yield of epothilone F. Plastid DNA is isolated from this Streptomyces spp. Culture and used (20) (20) 200418 849 daughters NPB29-6f (SEQ ID NO: 28) and NPB29-7r (SEQ ID NO: 29) were used as templates in PCR amplification. The expected fragments were obtained and sequenced using the B i - D  e sequencing system The ebh24_16cl 1 mutant was found to have two mutations that cause changes in the amino acid sequence of the protein, 'alanine 9 3 becomes glycine and isoleucine 305 becomes threonine. Position 93 is the target of mutation generation, and The change of 3 6 5 is an artificial product of a mutation generating program. The amino acid sequence of the mutant polypeptide encoded by SEQ ID NO: 6 2 is shown in SEQ ID NO: 63. SEQ ID NO: The 64 nucleic acid sequence encodes a mutant ebh24-16-16 'which also exhibits improved biotransformation yields. This Streptomyces spp. Transformant, identified in the screening of ebh24_16 random mutants, ebh24-16-16, Larger epothilone F yields can also be produced. Plastid DNA is isolated from this Streptomyces spp. Culture and primers NPB29-6f (SEQ ID NO: 28) and NPB29-7r (SEQ ID NO: 29) as template in PCR amplification. The expected fragments were obtained and sequenced using the Big-Dye sequencing system. The ebh24-16-16 Variant has been found to cause changes in the protein amino acid sequence of one of the additional mutations, valine, alanine 16 becomes. The amino acid sequence of the mutant polypeptide encoded by SEQ ID NO: 64 is shown in SEQ ID NO: 65. SEQ ID NO: 34 nucleic acid sequence encodes mutant ebh24-16, which also exhibits improved biotransformation yields. This mutant of Streptomyces lividans, ebh2 4-16, identified in the screening of mutation 24 (primers NPB29-mut24f (SEQ ID NO: 56) and NPB29-mut24r (SEQ ID NO: 57)), can also be Produces greater epothilone F yield. From this culture of Streptomyces lividans -26- (21) (21) 200418849, plastid DNA was isolated and used in primers NPB29-6f (SEQ ID NO: 28) and NPB29-7r (SEQ ID NO: 29). Used as template in PCR amplification. The expected fragments were taken and sequenced using the Big-Dye sequencing system. The ebh24-16 mutant was found to have two mutations that caused a change in the amino acid sequence of the protein, phenylalanine 2 3 7 was changed to alanine and isoleucine 92 was changed to valine. Position 23 7 is the target of mutation generation, and the change at residue 92 is an unexpected change in the artificial product of the mutation generation program. The amino acid sequence of the mutant polypeptide encoded by SEQ ID NO: 34 is shown in SEQ ID NO: 35. The nucleic acid sequence of SEQ ID NO: 60 encodes mutants ebh24-16d8, which also exhibits improved biotransformation yields. This S. rimosns transformant, ebh24-16d8, identified in the screening of mutation 59 (primer NPB 29-mut59 (SEQ ID NO: 77)) can also produce larger epothilone Yield F. Plastid DNA was isolated from this Streptomyces riftum culture and used as a template in PCR amplification using primers NPB29-6f (SEQ ID NO: 28) and NPB29-7r (SEQ ID NO: 29). The expected results were obtained. The fragment was sequenced using the Big-Dye sequencing system. The ebh24-16d8 mutant was found to have a mutation that caused one of the changes in the amino acid sequence of the protein, and the spermine 6 7 was changed to amidamine. This change was a The artificial product of the mutation generation program. The amino acid sequence of the mutant polypeptide encoded by SEQ ID NO: 60 is shown in SEQ ID NO: 61. SEQ ID NO: 62 nucleic acid sequence encodes mutant ebh24 -16cl 1, which also exhibits improved biotransformation yields. This Streptomyces fissureus (22) 200418849 transformant identified in the screening of mutation 59 (primer NPB29-mut59 (SEQ ID NO.  77)), ebh24-16cl 1 can also produce a larger yield of epothilone F. Plastid DNA was isolated from this Streptomyces spp. culture and used in Primers sub NPB29-6f (SEQ ID NO: 28) and NPB29-7r (SEQ ID NO: 29)</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">PCRBig-Dye ebh24-16cl 1 93 3 6593 3 6 5 SEQ ID NO:62 SEQ ID NO:63 SEQ ID NO :66ebh24- 1 6-74</span>Used as template in PCR amplification. Obtain the expected fragments and sequence them using the Big-Dye sequencing system. This ebh24-16cl1 mutant was found to have two mutations that caused changes in the amino acid sequence of the protein, alanine 93 became glycine and isoleucine 3 65 became threonine. Position 93 is the target of mutation generation, and the change at 3 6 5 is an artifact of a mutation generation program. The amino acid sequence possessed by the mutant polypeptide encoded by SEQ ID NO: 62 is shown in SEQ ID NO: 63. SEQ ID NO: 66 nucleic acid sequence encoding mutant ebh24- 1 6-74, which</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">ebh24-16 ebh24-16-74F DNANPB29-6f (SEQ ID NO:28) NPB29-7r (SEQ ID NO:29) PCR  Big-Dye ebh24- 1 6-74  88SEQ ID NO: 66SEQ ID NO : 67  SEQ ID NO:68ebh24-Ml8 ebh ebh24-M18 -28- (23) 200418849</span>It also exhibits improved biotransformation yields. This Streptomyces spp.transformer, ebh24-16-74, identified in the screening of ebh24-16 random mutants can also produce large epothilone F yields. Plastid DNA was isolated from this Streptomyces riftum culture and used as a template for PCR amplification using primers NPB29-6f (SEQ ID NO: 28) and NPB29-7r (SEQ ID NO: 29). The expected fragments were taken and sequenced using the Big-Dye sequencing system. The ebh24-1 6-74 mutant was found to have one additional mutation that resulted in a change in the amino acid sequence of the protein, and arginine 88 became histidine. The amino acid sequence of the mutant polypeptide encoded by SEQ ID NO: 66 is shown in SEQ ID NO: 67. The nucleic acid sequence of SEQ ID NO: 68 encodes mutant ebh24-M18, which also exhibits improved biotransformation yields. The ebh24-M18 transformant identified in the screening of ebh random mutations can also produce -28- (23) 200418849</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">F  DNA NPB29-6f (SEQ ID NO:28) NPB29-7r (SEQ ID NO:29)PCR Big-Dyeebh24-M 1 8 3 11 76 SEQ ID NO: 68 SEQ ID NO: 69</span>Larger epothilone F yield. Plastid DNA was isolated from this Streptomyces riftum culture and used as a template in PCR amplification using primers NPB29-6f (SEQ ID NO: 28) and NPB29-7r (SEQ ID NO: 29). The expected fragments were taken and sequenced using the Big-Dye sequencing system. The ebh24-M 1 8 mutant was found to have two mutations that caused a change in the amino acid sequence of the protein, the amino acid 31 was changed to lysine and the methionine 1 76 was changed to valine. The amino acid sequence of the mutant polypeptide encoded by SEQ ID NO: 68 is shown in SEQ ID NO: 69.</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">SEQ ID NO:72ebh24-16g8  50(NPB29-mut50(SEQ ID NO:78) ebh24-16g8F DNA NPB29-6f (SEQ ID NO:28) 29-7 (SEQ ID NO:29) PCRBig-Dye ebh24-16g8 1 7 6 1 3 01 76 1 3 0 SEQ ID NO: 72 SEQ ID NO: 73 SEQ ID NO:74ebh24-16b9 50(NPB 29-mut50(SEQ ID .78) -29- (24) (24)200418849 ebh24-16b9 DNA NPB29-6f (SEQ ID NO:28) NPB29-7r (SEQ ID NO:29) PCRBig-Dye ebh24-1 6b9 1 7 6 1 401 76 140 SEQ ID NO 74 SEQ ID NO 75 9PANT849ebh24-16 pANT849ebh24- 1 0-5 3  p ANT849ebh24-1 6d8  pANT849ebh24-16cll  p AN T 8 4 9 e b h 2 4 -1 6 - 1 6  p AN T849ebh24 - 16-74  p AN T 8 4 9 e b h 2 4 - 1 6b9  p ANT849ebh24-Ml 8 p ANT849ebh24 -16g8   The International Depository Authority20021 1_  1 0 8 0 1 University Boulevard, Manassas Virginia 20110-2209  the American Type Culture Collection  A T C C________   3 05  -30- (25) (25)200418849  B   SEQ ID NO:2 B/ SEQ ID NO:4/ /  SEQ ID NO :24 SEQ ID NO:24   AlaValLeulieSerThr AspGluAsnGin SEQ ID   : 4 3 - 4 9 2 SEQ ID NO : 13 SEQ ID   : 13  /B -31 - (26) 200418849 113 SEQ ID :1 GLU31  ARG67  ARG88  ILE92  ALA93  VAL106  ILE130  ALA140  MET 176  PHE190  GLU23 1SER2 94PHE2 3 7 ILE3 65 ebh25- 1  ebhl0-53  ebh24-16 &amp;gt; ebh24-16d8  ebh24-</span>The nucleic acid sequence of SEQ ID NO: 72 encodes mutants ebh24-16g8, which also exhibits improved biotransformation yields. This Streptomyces spp.transformer, ebh24-16g8, identified in the screening of mutation 50 (primer NPB29-mut50 (SEQ ID NO: 78)) can also produce a larger yield of epothilone F. From this The plastid DNA was isolated from the Streptomyces riftum culture and used as a template in PCR amplification using primers NPB29-6f (SEQ ID NO: 28) and NPB29-7 (SEQ ID NO: 29). The expected fragment was obtained and used Sequenced by the Big-Dye sequencing system. The ebh24-16g8 mutant was found to have two additional mutations that caused changes in the amino acid sequence of the protein, methionine 1 7 6 to alanine and isoleucine 1 3 0 becomes threonine. Position 1 76 is the target of mutation generation, and the change at 1 3 0 is the artificial product of a mutation generation program. The amino acid sequence of the mutant polypeptide encoded by SEQ ID NO: 72 is shown In SEQ ID NO: 73. The nucleic acid sequence of SEQ ID NO: 74 encodes mutants ebh24-16b9, which also exhibits improved biotransformation yield. In mutation 50 (primer NPB 29-mut50 (SEQ ID NO.  78 The Streptomyces fissures 29- (24) (24) 200418849 identified in the screening of), also produced ebh24-16b9. Large epothilone p production. Plastid DNA was isolated from this Streptomyces riftum culture and used for PCR with primers NPB29-6f (SEQ ID NO: 28) and NPB29-7r (SEQ ID NO: 29) Amplified as a template. The expected fragments were obtained and sequenced using the Big-Dye sequencing system. The ebh24-1 6b9 mutant was found to have two additional mutations that caused changes in the amino acid sequence of the protein, methionine 1 7 6 becomes serine and alanine 1 40 becomes threonine. Position 1 76 is the target of mutation generation, while change at 140 is an artificial product of the mutation generation program. The mutation encoded by SEQ ID NO. 74 The amino acid sequence of the somatic polypeptide is shown in SEQ ID NO  75. The plastids PANT849ebh24-16, pANT849ebh24- 1 0-5 3, p ANT849ebh24-1 6d8, pANT849ebh24- used in these 9 mutant genes are shown. 16cll, p AN T 8 4 9 ebh 2 4 -1 6-1 6, p AN T849ebh24-16-74, p AN T 8 4 9 ebh 2 4-1 6b9, p ANT849ebh24-Ml 8, and p ANT849ebh24 -16g8 It has been deposited and accepted by The International Depository Authority under the terms of the Budapest Convention. The deposit was completed on January 1st, 2002 at the American Type Culture Collection at University Boulevard, Manassas, Virginia 20110-2209. Its A T C C login number is ________. All restrictions on public access to this plastid will be irretrievably removed after this patent application is granted. The deposit shall be kept in the public depository for a period of 30 years after the deposit date or for a period of 5 years after the last sample application or the period of patent enforcement, whichever is longer. If the depository cannot dispense live -30- (25) (25) 200418849 samples, the deposit must be renewed. Thus, according to another aspect of the present invention, an isolated epothilone B melanase polypeptide and its variants and mutants, and an isolated ferredoxin polypeptide or its variant are proposed. In a specific embodiment of the present invention, the polypeptide is intended to include the amino acid sequence of SEQ ID NO: 2 and a fragment or variant thereof that retains organisms that are substantially the same as this epothilone B hydroxylase Activity and / or function. In another embodiment of the present invention, the polypeptide is intended to include the amino acid sequence of SEQ ID NO: 4, and a fragment and / or variant thereof that retain substantially the same biological activity as this iron redox protein And / or features. A "variant", as used in the present invention, is intended to include its amino acid sequence with conservative amino acid substitutions relative to SEQ ID NO: 2 or 4, and it has been shown to exhibit similarity to SEQ ID NO: 2 Or 4 biologically active or functional polypeptide. "Conservative amino acid substitution" is intended to include aliphatic amino acids such as between Ala, Val, Leu and lie, between hydroxyl residues Ser and Thr, between acidic residues Asp and Glu, and amidine residues Asn Permutation with Gin. Exemplary amino acid sequences of the variants of the present invention are shown in SEQ ID NO: 4 3-4 9 and the amino acid substitutions of these exemplary variants are listed in Table 2 above. The term "mutant" as used herein is intended to include a nucleic acid sequence encoded by one or more point mutations, or deletions or additions of a nucleic acid, but still similar, as compared to SEQ ID NO: 1 or 3 A polypeptide having activity in the polypeptide encoded by SEQ ID NO: 1 or 3. In a preferred embodiment, a mutation in a nucleic acid is one that alters the specificity and / or yield of the polypeptide encoded by it. The -31-(26) 200418849 preferred mutation region for the epothilone B hydroxylase gene is a nucleic acid sequence region encoding approximately 113 amino acid residues including the active site of the enzyme. In addition, a mutant having at least one amino acid substitution at the following amino acid positions of SEQ ID NO: 1 is also preferred: GLU31, ARG67, ARG88, ILE92, ALA93, VAL106, ILE130, ALA140, MET 176 , PHE190, GLU23 1, SER2 94, PHE2 3 7, or ILE3 65. Example mutants are ebh25- 1, ebhl0-53, ebh24-16 &amp;gt; ebh24-16d8, ebh24-</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">1 6c 1 1ebh24-16-16ebh24- 1 6- 74ebh24-16g8ebh24-1 6b 9 SEQ ID NO:31333561636567697173  75 SEQ ID NO:30323460626466 687072  74  f Greer et al.(Comparative modeling of homologous proteins. Methods in Enzymology 20223 9-52 19 9 1)Lesk et al.(Homology Modeling: Inferences</span>1 6c 1 1, ebh24-16-16, ebh24- 1 6- 74, ebh24-16g8, ebh24-1 6b 9 and the nucleic acid sequences encoding these mutants of the invention are shown in SEQ ID NOs: 31, 33, 35, 61, 63, 65, 67, 69, 71, 73, and 75, and SEQ ID NOs: 30, 32, 34, 60, 62, 64, 66, 68, 70, 72, and 74. Now, f, also according to Greer et al. (Comparative modeling of homologous proteins. Methods in Enzymology 20223 9-52, 19 9 1), Lesk et al. (Homology Modeling: Inferences</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">from Tables of Aligned Sequences. Curr. Op. Struc. Biol. (2) 2 4 2-247, 1 992) Cardozo et al. (Homology modeling by the ICM method. Proteins 23, 4 0 3 - 1 4, 1 99 5 ) EryF (PDB Code 1 KIN A)B 3 4%3 BSEQ ID NO:2)EryFPDB Code 1KIN A ; SEQ ID NO:76)4 EryFB -32- (27) (27)200418849 EryF (PDB Code 1KIN) B5 5 1 5 1 50 41 4B 1  Cartesian coordinates)    /   1 fractionalization) inversion)  B   SYBYL  6.7  INSIGHTII(Molecular Simulations Inc., San DiegoCA) 2000  (28) (28)200418849  SYBYL conformation) S YB YL 1 )2) equivalency) ; 3)4)  ( # SYBYL   NC aC0)     I SYBYL) B 1 NC aC0) 4.0  3.0 A2.0 A B -34- (29) (29)200418849 1 NC aC) 2.0  1.  A   / / 1 4  0 A3.0 A2 A      B B  B  1  A  LEU39,  GLN43 AL A45, MET57, LEU58, HIS62, PHE63, SER64,  SER65 ASP66 ARG67, GLN68, SER69, LEU74, MET75,  VAL76, ALA77, ARG78, GLN79, ILE80, ASP84, LYS85,  PR086, PHE87, ARG88, PR089, SER90, LEU91 ILE92, (30) (30)200418849 ALA93, MET94, ASP95, HIS99ARG103PHE110ILE155,   PHE 1 69, GLN 1 70, C YS 1 72, SERI 73, SER174, ARG1 75,  MET 1 76, LEU 1 77, SERI 78 ARG1 79, ARG1 86, PHE190,  LEU1 93, VAL23 3, GLY23 4, LEU23 5, AL A23 6, PHE23 7,  LEU23 8, LEU23 9, LEU240, ILE241 , ALA242, GLY243,  HIS244, GLU24 5, THR246, THR247, AL A248, ASN249,  MET2 5 0, LEU2 8 3, THR2 8 7, ILE2 8 8, ALA2 8 9, GLU290,  THR29 1 , AL A292, THR293, SER294, ARG2 95 PHE296,  AL A297, THR298, GLU3 12, GLY3 1 3, VAL3 14, VAL3 15,  GLY3 16, VAL3 44, ALA3 45, PHE346, GLY3 47, PHE3 48,  VAL3 5 0, HIS35 1 GLN3 5 2, CYS 3 53 LEU3 54, GLY3 5 5,  GLN3 5 6, LEU 3 5 8, ALA3 5 9, GLU3 62, LYS3 8 9, ASP391,  SER3 92 THR3 9 3 ILE394  TYR395  1   /  B 4.0  B   1  - 36- (31) (31)200418849  B   1 BLEU3 9, GLN43, ALA45, MET57LEU58HIS62PHE63SER64, SER65ASP66ARG67GLN68, SER69, LEU74MET75, VAL76, ALA77ARG78GLN79ILE80ASP84LYS85, PR086PHE87, ARG88PR089SER90LEU91ILE92, ALA93, MET94ASP95HIS99ARG103PHE110ILE155, PHE 1 69, GLN170, C YS 1 72, SERI 73, SER174 ARG1 75,  MET 1 76, LEU1 77, SERI 78, ARG1 79, ARG1 86, PHE 190,  LEU 1 93 VAL2 3 3, GLY23 4, LEU23 5, AL A23 6, PHE23 7,  LEU23 8, LEU23 9, LEU240, ILE241 ALA242 GLY243,  HIS244, GLU24 5, THR246, THR247, AL A24 8, ASN249,  MET2 5 0, LEU2 8 3, THR2 8 7, ILE2 8 8, AL A28 9, GLU290,  THR29 1 5 AL A292, THR293, SER294, ARG29 5, PHE296,  AL A297, THR2 9 8, GLU3 12, GLY3 1 3, VAL3 14, VAL3 1 5,  GL Y3 1 6, VAL3 44, ALA3 4 5, PHE3 46, GLY3 47, PHE3 4 8,  V AL3 5 0, HIS3 5 1 GLN3 5 2, CYS 3 5 3, LEU3 5 4 GLY 3 5 5,  GLN3 5 6, LEU3 5 8, ALA3 5 9, GLU3 62, LYS 3 8 9, ASP391  SER3 92, THR3 93, ILE394  TYR3 9 5 B1 0 A -37 - (32) (32)200418849 HEM 1  B    B B B  B screening in silico)in v i t r  )    B B B/ 1  A B   B -38 - (33) (33)200418849 B  B     B  INSIGHTIIQUANTA SYBYL docking)MMFF CHARMMAMBER  GRID(Goodford, 1 9 8 5 ) AUTODOCK (Goodsell1 990) DOCK (Kuntz et al. 1 9 82)  B  CAVEAT (Bartlett et al., 1 989) 3D Database  (Martin 1 992)  de novo) LUDI (Bohm 1 992) LeapFrog (Tripos Inc., St. Louis M0)  -39- (34) 200418849 DOCK (Kuntz et al.1 982)     B /     RNADNA     SV4  cosm id) (phagemids)  E. coli)Bacillus)  pBluescriptpGEX-2TpUC PET ColElpCRlpBR3 22pMB9pCW (35) (35)200418849</span>from Tables of Aligned Sequences. Curr. Op. Struc. Biol. (2) 2 4 2-247, 1 992), and Cardozo et al. (Homology modeling by the ICM method. Proteins 23, 4 0 3-1 4, 1 99 5) and the like are generally described on the basis of the structure of the known homologous protein EryF (PDB Code 1 KIN chain A) to construct a three-dimensional model of epothilone B hydroxylase. The homogeneity between these sequences is 34%. FIG. 3 shows an alignment of an epothilone B hydroxylase sequence (SEQ ID NO: 2) and an EryF sequence (PDB Code 1KIN chain A; SEQ ID NO: 76). Figure 4 depicts a homogeneous model of epothilone B hydroxylase obtained from sequence alignment with EryF. -32- (27) (27) 200418849 In addition, an energy plot of epothilin B hydroxylase relative to EryF (PDB Code 1KIN) was also prepared and shown in FIG. 5. The average window size of 51 residues at a given residue position is used to calculate the average energy  of the 51 residues in a sequence centered at the given residue. As shown in Figure 5, all the energies along the sequence are lower than 0, so it is obvious that the model construction shown in Figure 4 and Appendix 1 is reasonable. The three-dimensional structure shown in Fig. 4 epothilone B hydroxylase homogeneity model is defined by a set of structural coordinates listed in Appendix 1. The term "structural coordinates" refers to Cartesian coordinates derived from the construction of the homogeneous model. However, as known to those skilled in the art, a set of structural coordinates for a protein is a set of relative points of a shape defined within a three-dimensional shape. Therefore, a completely different set of coordinates may define a similar or identical shape. Furthermore, slight variations in individual coordinates, such as those using different alignment templates and / or different methods to generate similar homogeneous models in the generation of homogeneous models, will have a small impact on the overall shape. Variations in coordinates are also caused by mathematical calculations that construct the coordinates. For example, the structural coordinates listed in Appendix 1 may be calculated via fractionalization of the structural coordinates; addition or subtraction of integers from the structural coordinate group, inversion of the structural coordinates, or any combination thereof. Therefore, various computational analyses are required to determine whether a molecule or a part thereof is sufficiently similar to all or a part of the epothilone B hydroxylase described above to be regarded as the same degree. These analyses can be performed in existing software applications, such as SYBYL version 6.7 or INSIGHTII (Molecular Simulations Inc., San Diego, CA) version 2000 and user guides as attached (28) (28) 200418849 As described. For example, the overlap tool in the program SYBYL can be used to compare between different configurations and between different conformations of the same configuration. The procedure for comparing structures in S YB YL is divided into four steps: 1) loading the structures to be compared; 2) defining atomic equivalence in these structures; 3) performing a fitting operation; and 4) Analyze the results. Each construct is identified by a name. One structure was identified as the target (i.e., a fixed structure); the second structure (i.e., a mobile structure) was identified as the #source structure. Since the atomic equivalence in SYBYL is defined by user input, for the purposes of this aspect of the invention, equivalent atoms are defined as the protein backbone contained in all conservative residues between the two constructs to be compared Atoms (N, Ca, C, and 0). In addition, only rigid fit calculations are considered. When using the rigid fitting method, the operated structure is translated and rotated to obtain the best fit with the target structure. This fitting operation uses an algorithm that can calculate the best translation and rotation to be applied to the moving structure, so that the root mean square difference  of the fitted  for the specified equivalent atom pair is an absolute minimum . This case was reported by SYBYL in units of Angstrom (). For the purposes of the present invention, when any homogenous model of epothilone B hydroxylase overlaps with the corresponding backbone atom described in the structural coordinates listed in Appendix 1, its conservative residue backbone atom (N, C a, C , 0) Those with a root mean square difference  less than about 4.0 A are considered to be the same. More preferably, the root mean square difference  is less than about 3.0 A. More preferably, the root mean square difference is less than about 2.0 A. For the purposes of the present invention, any epothilone B hydroxylase homogeneous model-34- (29) (29) 200418849 overlaps with the corresponding backbone atom described in the structural coordinates listed in Appendix 1, its conservative residues The base skeleton atoms (N, C a, C, ) have root mean square differences  less than about 2.0 A are considered to be the same. More preferably, the root mean square difference  is less than about 1.  A. In another specific example of the present invention, when the structural model in which the skeleton atom is replaced by other elements overlaps with the corresponding skeleton atom, those having a low root mean square difference are considered to be the same. For example, the homogeneous model of its original skeleton carbon and / or nitrogen and / or oxygen atoms replaced with other elements has approximately 4.0 A when overlapping with the corresponding skeleton atoms described in the structural coordinates listed in Appendix 1. The better is about 3.0 A, and even the better is about 2 A. The root mean square difference is considered to be the same. The term "root-mean-square difference" means the square root of the arithmetic mean  of the square of the deviation  from the mean . It is a way of expressing deviations or variations from a trend or a goal. For the purpose of the present invention, "root mean square difference" refers to the definition of the relevant part of a protein backbone relative to the backbone of the epothilone B hydroxylase moiety contained in the complex defined by the structural coordinates described herein. The present invention, as embodied in the qualitative model, can contribute to the structure-based design of additional mutants of epothilone B hydroxylase. For example, using the homogeneous model of the present invention, it is now possible to define the Residues within 10 A of the binding site contained in the teomycin B hydroxylase. These residues include LEU39, GLN43, AL A45, MET57, LEU58, HIS62, PHE63, SER64, SER65, ASP66, ARG67, GLN68 , SER69, LEU74, MET75, VAL76, ALA77, ARG78, GLN79, ILE80, ASP84, LYS85, PR086, PHE87, ARG88, PR089, SER90, LEU91, ILE92, (30) (30) (30) 200418849 ALA93, MET94, ASP95, HIS99, ARG103, PHE110, ILE155, PHE 1 69, GLN 1 70, C YS 1 72, SERI 73, SER174, ARG1 75, MET 1 76, LEU 1 77, SERI 78, ARG1 79, ARG1 86, PHE190, LEU1 93, VAL23 3, GLY23 4, LEU23 5, AL A23 6, PHE23 7, LEU23 8, LEU23 9, LEU240, ILE241, ALA242, GLY 243, HIS244, GLU24 5, THR246, THR247, AL A248, ASN249, MET2 5 0, LEU2 8 3, THR2 8 7, ILE2 8 8, ALA2 8 9, GLU290, THR29 1, AL A292, THR293, SER294, ARG2 95 , PHE296, AL A297, THR298, GLU3 12, GLY3 1 3, VAL3 14, VAL3 15, GLY3 16, VAL3 44, ALA3 45, PHE346, GLY3 47, PHE3 48, VAL3 5 0, HIS35 1, GLN3 5 2, CYS 3 53, LEU3 54, GLY3 5 5, GLN3 5 6, LEU 3 5 8, ALA3 5 9, GLU3 62, LYS3 8 9, ASP391, SER3 92, THR3 9 3, ILE394 and TYR395, as listed in Appendix 1 By. Mutants with mutations at one or more of these positions are expected to exhibit altered biological function and / or specificity and thus constitute another specific example of a preferred mutant of the invention. Another specific example of a preferred mutant is a molecule having a root mean square difference of not more than about 4.0 humans relative to the backbone atom of the epothilone B hydroxylase. The epothilone B hydroxylase homogeneous model The structural coordinates or parts thereof are stored in a mechanically readable storage medium. This information can be used for a variety of purposes, such as drug discovery. Therefore, another aspect of the present invention is related to the mechanically readable data storage including the data storage content covered by the structural coordinates listed in Appendix 1-36- (31) (31) 200418849 media. The three-dimensional model construction of epothilone B hydroxylase can also be used to identify potential receptors for biological function modulators and enzymes. There are several methods or combinations thereof that can be used to identify such modulators. For example, a test compound can be spatially fitted to the binding site in epothilone B hydroxylase according to Appendix 1. By amino acids LEU3 9, GLN43, ALA45, MET57, LEU58, HIS62, PHE63, SER64, SER65, ASP66, ARG67, GLN68, SER69, LEU74, MET75, VAL76, ALA77, ARG78, GLN79, ILE80, ASP84, LYS85, PR086 , PHE87, ARG88, PR089, SER90, LEU91, ILE92, ALA93, MET94, ASP95, HIS99, ARG103, PHE110, ILE155, PHE 1 69, GLN170, C YS 1 72, SERI 73, SER174, ARG1 75, MET 1 76, LEU1 77, SERI 78, ARG1 79, ARG1 86, PHE 190, LEU 1 93, VAL2 3 3, GLY23 4, LEU23 5, AL A23 6, PHE23 7, LEU23 8, LEU23 9, LEU240, ILE241, ALA242, GLY243, HIS244, GLU24 5, THR246, THR247, AL A24 8, ASN249, MET2 5 0, LEU2 8 3, THR2 8 7, ILE2 8 8, AL A28 9, GLU290, THR29 1 5 AL A292, THR293, SER294, ARG29 5, PHE296, AL A297, THR2 9 8, GLU3 12, GLY3 1 3, VAL3 14, VAL3 1 5, GL Y3 1 6, VAL3 44, ALA3 4 5, PHE3 46, GLY3 47, PHE3 4 8, V AL3 5 0, HIS3 5 1, GLN3 5 2, CYS 3 5 3, LEU3 5 4, GLY 3 5 5, GLN3 5 6, LEU3 5 8, ALA3 5 9, GLU3 62, LYS 3 8 9, ASP391, SER3 92, THR3 93, Epothilone B as defined by ILE394 and TYR 3 9 5 The amino acid in the enzyme binding site 10 A and the coordination blood matrix -37-(32) (32) 200418849 group, HEM 1, the structural coordinates can also be used to identify the desired structure of these modulators Characteristics and chemical characteristics. The identified structural and chemical characteristics can then be used to design or select compounds that are potential epothilone B hydroxylase ligands. "Tectonic and chemical characteristics" are intended to include, but are not limited to, covalent bonds, Van der Waals interactions, hydrogen bonding interactions, charge interactions, hydrophobic bonding interactions, and dipolar interactions. Compounds identified as epothilone B hydroxylase ligands can then be synthesized and tested with characteristics that bind a test compound to epothilone B hydroxylase, or to identify epothilone B hydroxylase Enzymes are screened for assays for their ability to modulate a protease target in the presence of small molecules. Examples of assays that can be used in the screening of potential epothilone B hydroxylase ligands include, but are not limited to, screening in silico, in vitr screening, and high throughput assays . If those skilled in the art understand this reading, they can also use other design methods based on structure. A variety of computational design methods based on structure have been disclosed in the art. For example, many computer modeling systems are available in which epothilone B hydroxylase sequences and epothilone B hydroxylase constructs (ie, epothilone B hydroxylase as provided in the Appendix) can be entered. The atomic coordinates of the enzyme and / or the atomic coordinates as provided above or within 10 A of the binding region). The computer system can then generate a structural detail of one or more of these areas that a potential epothilone B hydroxylase modulator can bind to so that the complementary structural details of the potential modulator can be determined accordingly. The design within these modeled systems is usually based on compounds capable of physically and structurally associating with epothilone B chemical enzymes. In addition to this -38-(33) (33) 200418849, the compound must be able to work out a conformation that allows it to associate with epothilone B hydroxylase. These modeled systems can estimate the potential inhibitory or binding effects of a potential epothilone B hydroxylase substrate or modulator before actual synthesis and testing. Methods for screening chemicals or fragments for their ability to associate with a given protein target are also well known. These methods often start without visually checking the joints on a computer screen. The selected fragment or chemical is then placed in the binding region of epothilone B hydroxylase. Docking is done using software such as INSIGHTII, QUANTA and SYBYL, followed by energy minimization and molecular dynamics using standard molecular mechanical force fields such as MMFF, CHARMM and AMBER. Examples of computer programs that are useful in the selection of chemical fragments or chemical bodies that can be used in the present invention include, but are not limited to, GRID (Goodford, 1 9 8 5), AUTODOCK (Goodsell, 1 990), and DOCK (Kuntz et al., 1 9 82)  After selecting the better chemistries or fragments, they can be visually checked for their relationship with epothilone B hydroxylase and then assembled into a single potential modulator. Programs used to assemble individual chemicals include, but are not limited to, CAVEAT (Bartlett et al., 1 989) and 3D Database system (Martin 1 992)  In addition, empty active sites can be used or a known inhibitor can be included as needed Some parts of the compound are de novo designed. Design methods of this type include, but are not limited to, LUDI (Bohm 1 992) and LeapFrog (Tripos Inc., St. Louis M0) -39- (34) 200418849. For example, DOCK (Kuntz et al., 1 982) can be used. The formula) identifies potential ligands from a database or virtual database using atomic coordinates from a homogeneous model. 'This ligand can potentially bind the active site binding region and thus can be a suitable candidate for synthesis and testing. The invention also proposes a vector comprising a polynucleotide of the invention and a host cell genetically treated using the vector of the invention to produce epothilone B hydroxylase or an active fragment and variant or mutant of this enzyme and / or Ferredoxin or its active fragment. Generally, any vector suitable for maintaining, delivering, or expressing a polynucleotide to produce such polypeptides in a host cell can be used for such expression. According to this aspect of the invention, the vector may be, for example, a plastid vector, a single-stranded or double-stranded phage vector, or a single-stranded or double-stranded RNA or DNA virus vector. Vectors can be extrachromosomal or designed to integrate into the host chromosome. These vectors include, but are not limited to, chromosome, epigenetic, and virus-derived vectors, such as those derived from the following:  Bacterial plastids, bacteriophages, yeast gene appendages, yeast chromosomal elements, and viruses such as Baculovirus, Papillomavirus, SV40, Vaccinia Virus, Adenovirus, Fowlpox Virus, Pseudorabies Virus and Retroviruses, and Viruses Derived from Their Combinations, such as Plastid and Phage Genetic Elements , Those who adhere to the cosm id and phagemids. Expression vectors useful in prokaryotic hosts include, but are not limited to, bacterial plastids, such as those from E. coli, Bacillus, or Streptomyces, including pBluescript, pGEX-2T, pUC vectors, PET Vector, ColEl, pCRl, pBR3 22, pMB9, pCW, (35) (35) 200418849</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Pbms200pBMS2020PI J1 01PIJ702p ANT849 pOJ260p J446pSET 152pKCl 139pKC 1 2 1 8 pFD666 RP4  DNAs 9 8 9 GT10I GT1 1  1 3DN    Yeast Integrating plasmids)( 5) YRpYCp YLppGPD-22 // Y AC s)  Gietz et al.Gene74:5 27-3 4( 1 9 8 8 )YlpUcYEplac  YCplac)  SV40T-COS 1 C 0 S 7EB V 293 -EBNA EBV EBNA-1E1A)  pSV2pBC12BI p 9 1 023 ; Pcdna  -41 - (36) (36)200418849  pvL941 1mRNA   H    pLtrctrplac T7TACTRC fd snpAmelCermE* araB AD CYC1GAL1 GAL10ADH1 - GPD CMV HSVSV40 LTRs (R S V)-1   transvection) (37) (37)200418849    B   B/  C 1273T3CHOHeLa 293  BHK  COS-7 B   B   B  B  B   Bacillus megaterium) -43- (38) (38)200418849 S o r a n g i u m c e 11 u 1 o s u m) (Rhodococcus erythropolis) Streptomyces virginiae) Streptomyces venezuelae)  (Streptomyces a 1 b u s )S t r e p t o m y c e s coelicolor)Streptomyces g r i s e u s ) o             (compactin) pravastatin) (Actinomadura) 5,942,4236,274,360  -44 - (39) (39)200418849  199912 21 WO 00/392769M68,854 I   - C  2 - ( A 1)  - ( Q ) m - ( A 2 )  - E (I) </span>Pbms200, pBMS2020, PI J1 01, PIJ702, p ANT849, pOJ260, p0J446, pSET 152, pKCl 139, pKC 1 2 1 8, pFD666 and their derivatives, broader host range vectors, such as RP4 'phage DNAs For example, many derivatives of phage lambda such as NM9 89, lambda GT10, and I GT1 1, and other phage derivatives such as M 1 3 and filamentous monozole DNA. The vectors of the invention for yeast typically contain a source of replication suitable for use with yeast and a selectable marker that can function in yeast. Examples of yeast vectors that can be used in the present invention include, but are not limited to, Yeast Integrating plasmids (such as 5) and yeast replicating plastids (YRp and  series plastids) yeast centromere (YCp series plastids), yeast-type plastids, yeast artificial chromosomes (Y AC s) based on YLp, pGPD-2, 2 // plastids and their derivatives, and improved round trips Vectors are described, for example, in Gietz et al., Gene, 74: 5 27-3 4 (19 8 8) (YlpUc, YEplac, and YCplac). Mammalian vectors that can be used for recombinant expression can include viral sources, such as SV40 sources (for replication in cell lines expressing large T-antigens such as COS 1 and C 0 S 7 cells), papilloma virus sources, or EB The V source is used for long-term gene epithelial replication (used in, for example, 293-EBNA cells, which can constitutively express the EBV EBNA-1 gene product and adenovirus E1A). Performance in mammals can be achieved by using a variety of plastids, including, but not limited to, pSV2, pBC12BI, and p 9 1 023; Pcdna vectors and lytic virus vectors such as vaccinia virus, adenovirus, and rod Viruses), genetic epithelial viral vectors (such as bovine papilloma disease-41-(36) (36) 200418849), and retroviruses (such as murine retrovirus). Vectors useful for insect cells include baculovirus vectors and pvL941. 1 The selection of genes that are suitable for initiating mRNA transcription and expression vectors is well known. However, in general, the expression construct includes a site for transcription initiation and termination, and a nuclear H-body binding site for translation in the transcribed region. The codons contained in mature transcripts expressed by isoforms include translation initiation codons appropriately placed at the start and stop codons of the end of the polypeptide to be translated. Examples of promoters that can be used in prokaryotes include, but are not limited to, phage pL promoter, trc promoter, hybrids derived from trp and lac promoters, phage T7 promoter, TAC or TRC system's main manipulation of phage lambda Gene and promoter region, fd envelope protein control region, snpA promoter, melC promoter, ermE * promoter or araB AD operon. Examples of the promoter genes that can be used in yeast include, but are not limited to, a CYC1 promoter gene, a GAL1 promoter gene, a GAL10 promoter gene, an ADH1 promoter gene, a yeast alpha-mating system promoter gene, and a GPD promoter gene. Examples of promoters that can be routinely used in mammalian expression vectors include, but are not limited to, CMV early promoter, HSV chest ribokinase, early and late SV40 promoter, reverse transcription LTRs promoter, such as Roche A sarcoma virus (RSV), and a new metal sulfur starter gene, such as the mouse metal sulfur new starter gene. Polynucleotide-containing vectors can be introduced into host cells using any number of well-known techniques including infection, transduction, transfection, transvection, and transformation (37) (37) 200418849. The polynucleotide may be introduced into the host alone, or with additional coding, e.g., a selectable marker or a ferredoxin or the like. In a preferred embodiment of the present invention, the polynucleotides of epothilone B hydroxylase and ferredoxin are introduced into host cells. The host cells used for various expression constructs are well known, and those skilled in the art can routinely select host cells used for expressing epothilone B hydroxylase and / or ferredoxin according to this aspect of the present invention. . Examples of mammalian expression systems that can be used in the present invention include, but are not limited to, C 127, 3T3, CHO, HeLa, human kidney 293 and BHK cell lines, and COS-7 cell line of monkey kidney fibroblasts. In addition, as exemplified herein, both epothilone B hydroxylase and ferredoxin can be recombinantly expressed in microorganisms. Therefore, another aspect of the present invention relates to a recombinantly produced microorganism that can express epothilone B hydroxylase alone or together with ferredoxin and that the microorganism is capable of converting a compound having a terminal alkyl group, particularly It is epothilone, which is hydroxylated and made into a terminal hydroxyalkyl group. The recombinantly produced microbial silicon is produced by transforming a cell such as a bacterial cell using a plastid including a nucleic acid sequence encoding an epothilone B hydroxylase. In a preferred embodiment, the cell lines use a nucleic acid sequence comprising an epothilone B  ase or a mutant or variant thereof, and a cell line located downstream of the epothilone B hydroxylase gene. The plastids of the nucleic acid sequence encoding the ferredoxin are transformed. Examples of microorganisms that can be transformed using these plastids to produce the recombinant microorganism of the present invention include, but are not limited to, E. coli, Bacillus megaterium, Streptomyces orientalis, -43- (38) (38) 200418849 Sorangiumce 11 u 1 osum, Rhodococcus erythropolis, and streptomyces species such as Streptomyces lividans, Streptomyces virginiae, Streptomyces venezuelae , Streptomyces a 1 bus, Streptomyces coelicolor, Streptomyces griseus and Streptomyces griseus o The recombinantly produced microorganisms of the present invention can be used in microbial programs or methods to produce Compounds with terminal hydroxyalkyl groups, and especially epothilones. Generally, a recombinantly-produced microorganism or a microorganism capable of selectively hydroxylating a terminal carbon or an alkyl group can be cultured in an aqueous nutrient medium containing assimilable carbon and nitrogen sources in the presence of an appropriate substrate under submerged aerobic conditions, or The enzymes derived from it produce hydroxyalkyl-loaded products. Suitable epothilones used as substrates in the method of the present invention may be any of these compounds having a terminal carbon or alkyl group capable of accepting the enzymatic hydroxylation of the present invention. The starting materials, or substrates, can be isolated from natural sources, such as P. cellulosae, or they can be synthetically formed epothilones. Any other substrate capable of accepting enzymatic hydroxylation with a terminal carbon or alkyl group can also be used in the method of the present invention. For example, compactin can be used as a substrate, which after hydroxylation will form the compound pravastatin. The method used to hydroxylate compactin through Actinomadura strain to pravastatin is described in U.S. Patent 5,942,423 and U.S. Patent 6,274,360. For example, when using the recombinant microorganism of the present invention, at least one -44-(39) (39) 200418849 epothilone can be prepared, such as WO 00/39276, US Patent No. 09M68, filed on December 21, 1999, The content of 854 is contained in its entirety. Epothilone can be prepared with the following formula I  -C  2-(A 1) -(Q) m-(A 2) -E (I) where</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">AiA2 Ci-Cs- Q13  m1 m1;  II CH3-(A1)n-(Q)m-(A2)-E (II) AiQA2Enm II   B BSo ce90DE 41 38 (40) (40)200418849 042WO 93/10121Deutsche Sammlung von M i k r  o r g a n i s m e n (German Collection of Microorganisms) (DSM) 6773 HofleG ., e t a 1., Ang e w . Chem. Int. Ed. Engl., Vo 1. 35, No. 13/141 5 6 7- 1 5 69 ( 1 996)B MengD. et al.J. Am. Chem. Soc.Vol. 119, No. 421 0073 - 1 0092 ( 1 996)</span>Ai and A2 are independently selected from the group consisting of: optionally substituted Ci-Cs-alkyl and alkenyl; Q is a optionally substituted ring system comprising 1 to 3 rings and at least At least one carbon-to-carbon double bond in a ring; , m and 0 are integers selected from the group consisting of: 0 and 1, where at least one of m or  or  is 1; and E is An epothilone nucleus. This method includes combining at least one epothilone CH3- (A1) n- (Q) m- (A2) 0-E (II) having the following formula II: Ai, Q, A2, E, n, m and All are as defined above; contact with a recombinantly produced microorganism capable of selectively catalyzing the hydroxylation of Formula II, or an enzyme derived therefrom, and promote the hydroxylation. In a preferred embodiment, the starting material is epothilone B. Epothilone B can be obtained from the fermentation of Fibriocystis soce90, as described in DE 41 38 (40) (40) 200418849 042 and WO 93/10121. The strain was deposited with the Deutsche Sammlung von M i k r  r g a n i s meen (German Collection of Microorganisms) (DSM) number 6773. The fermentation method is also described in Hofle, G., eta 1., Ang ew. Chem. Int. Ed. Engl., Vo 1. 35, No. 13/14, 1 5 6 7- 1 5 69 (1 996) in. Epothilone B can also be obtained using chemical methods such as those disclosed by Meng, D. et al., J. Am. Chem. Soc., Vol. 119, No. 42, 1 0073-1 0092 (1 996);</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Nicolaou, K. e t a 1., J. Am. Chem. Soc.Vol. 119No. 34, 7974-7991 (1997) SchinzerD.et al.Chem. Eur. J.</span>Nicolaou, K. e t a 1., J. Am. Chem. Soc., Vol. 119, No. 34, 7974-7991 (1997) and Schinzer, D., et al., Chem. Eur. J.,</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Vol. 5No. 92483 -249 1 ( 1 999)    T. Nagodawithana and J. . Wasileski Chapter 2: Media Design for Industrial Fermentation Nutritional Requirements of Commercially Important Microorganism edited by T. W. Nagodawithana and G. Reed, Esteekay Associates Inc., Milwaukee, W1, 18-45 ( 1 998)  T.l. Miller and B. W. Churchill, Chapter 10:Substrates for Large-Scale Fermentations Manual of Industrial Microbiology and Biotechnology, edited by A. 1. Demain and N. A. Solomon, American Society for</span>Vol. 5, No. 9, 2483 -249 1 (1 999). The growth of recombinantly produced microorganisms selected for use in this method can be achieved by those skilled in the art using appropriate nutrient media. Appropriate media for recombinantly producing microbial growth include those that provide the nutrients needed for microbial cell growth. See, for example, T. Nagodawithana and J. M. Wasileski, Chapter 2: "Media Design for Industrial Fermentation," Nutritional Requirements of Commercially Important Microorganism, edited by TW Nagodawithana and G. Reed, Esteekay Associates, Inc., Milwaukee, W1, 18-45 (1 998); Tl Miller and BW Churchill, Chapter 10: `` Substrates for Large-Scale Fermentations, '' Manual of Industrial Microbiology and Biotechnology, edited by A. 1. Demain and NA Solomon, American Society for</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">MicrobiologyWashingtonD.C.1 22- 1 3 6 ( 1 9 8 6)  -46- (41) 200418849     B </span>Microbiology, Washington, D.C., 1 22-1 36 (1 9 8 6). A typical growth medium includes the required carbon source, nitrogen source, and trace elements. You can also add -46- (41) 200418849 to the inducer in the medium. "Inducer," as used herein, includes any compound that enhances the formation of a desired enzyme activity in recombinantly produced cells. Typical inducers used in the present invention may include solvents used to dissolve substrates, such as dimethylamine Sodium, dimethylformamide, dioxane, ethanol, and acetone. In addition, certain substrates, such as epothilone B, can also be considered as inducers.</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">     N-ZA tryptone)peptone)  </span>Carbon sources may include sugars such as glucose, fructose, galactose, maltose, sucrose, mannose, sorbose, glycerol, starch, and the like; organic acids such as sodium acetate, sodium limonate, and the like; and alcohols such as ethanol , Propanol and the like. Preferred carbon sources include, but are not limited to, glucose, fructose, sucrose, glycerol, and starch. Nitrogen sources may include NZ amine A, corn infusion, soy flour, beef extract, tryptone, peptone, cottonseed flour, peanut flour, amino acids such as sodium glutamate, and the like Among them, sodium citrate, ammonium sulfate and the like.</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">     / pH58 1 4 C3 7(2 8 C m 11 872 -47- (42) 200418849 HP LC)  1 200 72New Brunswick Scientific Innova 5000) 150  300 rpm  2 5 0 rpm2    pH        1 1 ebh B3 0C3 25%75% 5200 HpLC HPLC/MS12)159 S7T F524 -48- (43) (43)200418849 B19.058   ebhB ebhebh pANT849-ebh) PANT84 9-ebh 10-5 3) pANT849-ebh24_16) TK24 3 0C7 B/100 % EtOH.5 % /pANT8 49-ebh24-16)B48 F)244872 HPLC0 B -49- (44) 200418849 B :   ( TK24 p ANT 8 4 9-eb h p ANT849-ebh 1 0-53 p ANT 849-ebh24- 16  0 1 0 0 % 1 0 0 % 1 0 0 % 1 0 0 %  24 9 9% 7 8% 6 9% 5 6%  48 8 7% 19% 3 9% 0%  72 8 7% 0% 3%   F :   TK24 p ANT 84 9- pANT849-ebhl 0- p ANT849-ebh24-    eb h 53 16  (      0 0% 0% 0% 0%  24 0% 4% 9% 23%  48 0% 2 1% 2 9% 5 2%  72 0% 14% 41% ___ ebh  (pANT 8 4 9)p ant 8 4 9-ebh) 3 0 C7   30C250 rpm - 50- (45) 200418849 24HPLC pANT8 4 9)       (p ANT 849) (p ANT8 4 9-ebh)   3 6% 1 1 %   11% 53%</span>Trace elements may include salts such as magnesium, manganese, calcium, cobalt, nickel, iron, sodium and potassium. Phosphate can also be added in a trace amount, or more preferably in a trace amount. The medium used for fermentation may include more than one carbon or nitrogen source or other elements. When the growth and / or basalization of a recombinantly-producing microorganism according to the method of the present invention is carried out, the pH of the culture medium is preferably about 5 to about 8 and the temperature thereof is about 14  C to about 37 later. (:, Preferably, the temperature is 2 8  C. The duration of the reaction is 1 to 100 hours, preferably 8 to 72 hours. -47- (42) 200418849 The culture medium should be placed at a high temperature for a period of time. Chromatography (HP LC) monitoring to determine the time required for complete biotransformation. Typically, the period required to complete the biotransformation is 12 to 00 hours and preferably about 72 hours after the substrate is added. Media After being placed on a rotary shaker (New Brunswick Scientific Innova 5000), it is operated at 150 to 300 rpm and preferably about 250 rpm, and a 2-inch rocking distance. The hydroxyalkyl-loaded product can be obtained from The fermentation broth is recovered by conventional means commonly used to recover other known biologically active substances. Examples of such recovery means include, but are not limited to, extraction with conventional solvents; pH adjustment; treatment with conventional resins, such as using anions Or cation exchange resin or non-ion adsorption resin treatment; use of conventional, for example, adsorbent treatment; distillation; crystallization; or recrystallization; and the like for separation and purification. The extract obtained from the mixture can be further separated and purified by column chromatography and analytical thin layer chromatography. The recombinant production microorganism of the present invention converts an epothilone having a terminal alkyl group into a terminal hydroxyalkane The ability of epothilone is based on proven. In these experiments, the culture of Streptomyces lividans strains containing plastids containing the ebh gene will be described in more detail as in Example 11. The substance and epothilone B suspension were stirred and warmed at 30  C for 3 days. The warmed solution was extracted with an equal volume of 25% methanol: 75% n-butanol, rotated and allowed to settle for 5 minutes. 200 micrometers The liter of organic phase was transferred to a HpLC vial and analyzed by HPLC / MS (Example 12). An S7T was dissolved at a dissolution time of 15.9 minutes to precipitate an epothilone F product peak with a protonation of 524 The molecular weight is -48- (43) (43) 200418849 The epothilone B substrate elutes in 19.0 minutes and has a protonated molecular weight of 50. Both the peak retention time and molecular weight are determined using known standards. The biotransformation rate of epothilone B obtained from cells expressing ebh is similar. The ebh mutant strains were compared for their bioconversion rates. Cells that can express ebh include frozen spore preparations of Streptomyces lividans (pANT849-ebh). Cells that can express mutants include S. lividans (PANT84 9-ebh 10-5 3) and frozen spore preparation of Streptomyces lividans (pANT849-ebh24_16). A frozen spore preparation of Streptomyces lividans TK24 was used as a control. Each cell was pre-warmed at 30  C After 7 days of incubation, epothilone B / 100% EtOH was added to each culture to a final concentration of 0.05% weight / volume. Except for cultures of Streptomyces lividans (pANT8 49-ebh24-16), in which epothilone B has been completely converted to epothilone F within 48 hours, at 0, 24, 48, and 72 Take samples in hours. The samples were analyzed by HPLC. The results were calculated as a percentage relative to epothilone B at 0. -49- (44) 200418849 epothilone B: time (hour) TK24 p ANT 8 4 9-eb hp ANT849-ebh 1 0-53 p ANT 849-ebh24- 16 0 1 0 0% 1 0 0% 1 0 0% 1 0 0% 24 9 9% 7 8% 6 9% 5 6% 48 8 7% 19% 3 9% 0% 72 8 7% 0% 3% epothilone F: TK24 p ANT 84 9- pANT849-ebhl 0- p ANT849-ebh24- inter eb h 53 16 (hours) 0 0% 0% 0% 0% 24 0% 4% 9% 23% 48 0% 2 1% 2 9% 5 2% 72 0% 14% 41% ___ In addition, 'the ability of cells expressing ebh to bioconvert compactin to pravastatin was also explored. In these experiments, frozen spore preparations of Streptomyces lividans (pANT 8 4 9) or Streptomyces lividans (pant 8 4 9-ebh) were grown at 30  C for 7 days. After the pre-warming period, transfer one aliquot of each cell culture solution to a polypropylene culture tube, add permedin to each culture tube, and place the tubes at 30  C. 250 rpm insulation-50- (45) 200418849 24 hours. One aliquot of the culture liquid was then extracted and the compaction and pravastatin relative to the culture of Streptomyces lividans (pANT8 49) were measured by HPLC. The table shows the initial compaction concentration of compactin and pravastatin tin-lead Streptomyces lividans (p ANT 849) (p ANT8 4 9-ebh) compaction 3 6% 1 1% Pravastatin 11% 53%</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">ebh25-l(SEQ ID NO:30) B BF HPLCLC-MS 523M.W.)b MSNMRb 24-- B : -51 - 26 (46)200418849</span>As discussed above, the mutant ebh25-1 (SEQ ID NO: 30) exhibits altered substrate specificity and the bioconversion of epothilone B by this mutant results in Or epothelin F are products of different HPLC elution times. This unknown sample was analyzed by LC-MS and found to have a molecular weight (M.W.) of 523, consistent with the single hydroxylation of epothilone b. According to MS and NMR data (compared with the data of epothilone b), the structure of the biotransformation product was determined to be 24-epi-epothilone B: -51-26 (46) 200418849</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">24--B A</span>24-hydroxy-epothilone B formula A</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text"> C27H41NO7S  523  ES + (m/z): 524([M + H]+)506 LC/MS/MS: +ESI(m/z): 524506476436320 HRMS  [M + H]+524.2682 ; 524.2701 HPLC(Rt) : 7.3HPLC LC/NMR : </span>Molecular formula: C27H41NO7S Molecular weight: 523 Mass spectrum: ES + (m / z): 524 ([M + H] +), 506 LC / MS / MS: + ESI (m / z): 524,506,476,436,320 HRMS: Calculated for [M + H] + 4.2: 524.2682; Measured : 524.2701 HPLC (Rt): 7.3 points (on analytical HPLC system) LC / NMR: observed chemical shift</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Varian AS-600(5 99.624MHz) d2 O/CD3 CN ( 1.94) 4 / 6    5 7  30 (s1H)5 6.43 (s1)530  (mlH) 4.35 (mlH) 3 .81 (mlH) 3.74  (m, 1H), 3.68 (mlH) 3 .43 (mlH) 2.87  (m1H) 2.66 (s3H) 2. 40 (m2H) 1.58  (blH) 1.48 (blH) I.35 (m3H) 1.18 -52- (47) (47)200418849 (s3H)1 . 1 3 (s3H)0. 87 (m6H) *1 . 8 - 2 . 1 p p m  </span>Varian AS-600 (proton: 5 99.624MHz) Solvent d2 O / CD3 CN ( 1.94) 4/6 Proton: 5 7  30 (s, 1H) 5 6.43 (s, 1), 5.30 (m, lH ), 4.35 (m, lH), 3.81 (m, lH), 3.74 (m, 1H), 3.68 (m, lH), 3.43 (m, lH), 2.87 (m, 1H), 2.66 ( s, 3H), 2. 40 (m, 2H), 1.58 (b, 1H), 1.48 (b, 1H), I.35 (m, 3H), 1.18 -52- (47) (47) 200418849 (s , 3H), 1.1 3 (s, 3H), 0.87 (m, 6H) * Due to solvent suppression, peaks between 1.8-2.1 ppm were not observed. Its proton chemical shift is specified as:</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">-53- 200418849</span>-53- 200418849</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">(48)</span>(48)</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">    1    2 2.40 m  3 4.35 m  4    5    6 3.43 m  7 3.68 m  8 1.58 m  9 1.35 b  10 1.48 b  10 1.3 5 b  11 SSP   12    13 2.87 m  14 SSP   15 5.30 m  16    17 6.43 s  18    19 7.30 s  20    21 2.66 s  22 1.18 s  23 0.87 m  24 3.81 m  24 3.74 m  25 0.87 m  26 1.13 s  27 SSP</span>Position proton pattern 1 2 2.40 m 3 4.35 m 4 5 6 3.43 m 7 3.68 m 8 1.58 m 9 1.35 b 10 1.48 b 10 1.3 5 b 11 SSP 12 13 2.87 m 14 SSP 15 5.30 m 16 17 6.43 s 18 19 7.30 s 20 21 2.66 s 22 1.18 s 23 0.87 m 24 3.81 m 24 3.74 m 25 0.87 m 26 1.13 s 27 SSP</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">*SSP : -54- (49) 200418849  -microtubule-stabilizing agents) -24--B(A)      B-T-   d g k i n s  d g k i n s Burketts  S e m i   m a) (Teratocarcinoma)  s c h w a    m a s);   xenderma pigmentosum kefatoactanthoma    - -55- (50) 200418849        I11  A       Epstein-Barr Sindbis   AIDS-  AIDS ;  SSS  -56- (51) (51)200418849  -       1 : R2 103K2SO4(0.25MgCl2 62(10.12 1 Difco Casamino a c i d s (0 . 18 0 08 0 2002.2 Difco Bacto   1  KH2P4(0.5%) 8  CaCl2  6H20(3.6 8%) 1.5L-20%) 10TES5.73%pH 7.2) 0.2ZnCl2 (40FeCl3 6H20(200 CuCl2 2H2O(10 MnCl2 4H20 (52) (52)200418849 (10 Na2B407  1H20 (10  (NH4)6M724  H2 0.5NaOH (1 N)( 0.550/ 3 7/ 7.5/0.5/ R2YER2  15Difco 10%) P(  103 K2SO4(0.25 </span>* SSP: Not observed due to solvent pressing. -54- (49) 200418849 In summary, the compositions and methods of the present invention can be used to produce microtubule-stabilizing agents and new compounds that are expected to be useful as microtubule-stabilizing agents, including Ipo Mycin analogs such as 24-hydroxy-epothilone B (formula A) and pharmaceutically acceptable salts thereof. Microtubule stabilizers made using these compositions and methods can be used to treat a variety of cancers and other proliferative diseases including, but not limited to, the following: Cancer, including bladder, breast, colon, kidney, liver, lung , Ovary, pancreas, stomach, cervix, thyroid and skin; including squamous cell carcinoma; lymphoid hematopoietic tumors, including leukemia, acute lymphocytic leukemia, acute lymphoblastic leukemia, B-cell lymphoma , T-cell lymphoma, dgkins lymphoma, non-dgkins lymphoma, hair cell lymphoma, and Burketts lymphoma; tumors of mesenchymal origin, including fibrosarcoma and rhabdomyosarcoma, and other tumors, including melanoma, seminoma Semioma, Teratocarcinoma, neuroblastoma and glioma; tumors of the central and peripheral nervous system, including astrocytoma, neuroblastoma, glioma, and Schwannomas (schwa   mas); mesenchymal tumors 'including fibrosarcoma, rhabdomyosarcoma and osteosarcoma; and other tumors' including melanoma , Xenderma pigmentosum, kefatoactanthoma, seminoma, thyroid follicular carcinoma and teratoma. The microtubule stabilizer prepared by using the composition and method of the present invention can also inhibit angiogenesis, thereby affecting tumor growth and providing treatment for tumors and tumor-related disorders. These anti-angiogenic -55- (50) 200418849 properties of these compounds can also be used to treat other conditions that are responsive to anti-angiogenic agents including, but not limited to, a blood blindness associated with retinal angiogenesis, Arthritis, especially inflammatory arthritis, multiple sclerosis, retinitis and psoriasis. The microtubule stabilizer prepared by using the composition and method of the present invention can induce or inhibit apoptosis, a physiological cell death process that is critical for normal development and homeostasis. Changes in the apoptotic pathway can contribute to the pathogenesis of many human diseases. The compounds of the present invention, such as those shown in I and 11 and Formula A, can be used to treat a variety of human diseases with apoptotic abnormalities when used as modulators of apoptosis, including, but not limited to, cancer and precancerous Lesions, immune response-related diseases, viral infections, degenerative diseases of the musculoskeletal system and kidney disease. Without wishing to be bound by any mechanism or morphology, microtubule stabilizers made using the compositions and methods of the invention can also be used to treat conditions other than cancer or other proliferative diseases. These conditions include, but are not limited to, viral infections such as herpes virus, scar virus, Epstein-Barr virus, Sindbis virus, and adenovirus; autoimmune diseases such as systemic lupus erythematosus, immune-mediated glomerulonephritis, rheumatic joints Inflammation, psoriasis, inflammatory bowel disease and autoimmune diabetes; neurodegenerative diseases such as Alzheimer's disease, AIDS-related dementia, Parkinson's disease, amyotrophic lateral sclerosis, pigmented retinitis, Spinal muscular atrophy and cerebellar degeneration; AIDS; Symptoms of Spinal Cord Dysplasia; Aplastic Anemia; Hemostatic Injury Associated with Myocardial Infarction; Stroke and Reinfusion Injury; Narrow SSS; Narrow Rhythm; Atherosclerosis Toxin-induced or alcohol-induced liver disease -56- (51) (51) 200418849 disease; hematological diseases such as chronic anemia and aplastic anemia; degenerative diseases of the musculoskeletal system such as osteoporosis and arthritis; aspirin- Sensitive sinusitis; cystic fibrosis; multiple sclerosis; kidney disease; and cancer pain. The following non-limiting examples are provided to further illustrate the invention. [Embodiment] Examples Example 1: The R2 medium was prepared as follows: Preparation containing sucrose (103 g), K2SO4 (0.25 g), MgCl2  620 (10.12 g), glucose (10 g), Difco Casamino A solution of acids (0.1 g) and distilled water (800 ml). Then 80 ml of this solution was poured into a 200 ml screw cap bottle containing 2.2 g of Difco Bacto agar. Cap the bottle and autoclave. When in use, re-melt the medium and add the following autoclaved solutions in the order listed: 1 ml KH2P04 (0.5%) 8 ml CaCl2 6H20 (3.6 8%) 1.5 ml L-proline %) 10 ml TES buffer (5.73%, adjusted to pH 7.2) 0.2 ml trace element solution contains ZnCl2 (40 mg), FeCl3  6H20 (200 mg), CuCl2  2H2O (10 mg), MnCl2  4H20 (52 ) (52) 200418849 (10 mg), Na2B407  10H20 (10 mg), and (NH4) 6M724  H20 0.5 ml NaOH (1 N) (no sterilization required) 0.5 ml nutritional deficiency Growth factors (histidine (50 g / ml); cysteine (37 g / ml); adenine, guanine, thymidine, and uracil (7.5 g / ml); and vitamins (0.5 g / ml) / Ml). R2YE medium was prepared in the same way as R2 medium. However, 5 ml of Difco yeast extract (10%) was added to each 100 ml flask during use. P (protoplast) buffer system Prepare as follows: Prepare a base solution consisting of: sucrose (103 g) K2SO4 (0.25 g</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">MgCl2  6H2O(2.02  R22 800 8 0  1  KH2P4(0.50/) 1  CaCl2  6 H2(3.68%) TES5 ,75%pH 7.2) T ( -58- (53) (53)200418849 2 51  . 3 % ) 7 5 1R2 1  K 2 S 0 4 ( 2.5 % ) 9.3 0.2  CaCl2 (5M) .5Tris1 M Tris pH 8.0 31 PEG1000 L( 10010.3%) 10TES5.73%pH 7.2) 1  K2S04(2.5%) 1R2 1  KH2P4(0.5%) 0.1  MgCl2 6H20(2.5 ) 1  CaCl2 (25) CRM 1dH2010 103 MgCl2 6H2O(10.12 BBLTM  trypticase soy broth) (15 Becton (54) 200418849</span>MgCl2  6H2O (2.02 g) Trace element solution (2 ml) as described for R2 medium Distilled water to 800 ml Then, each 80 ml of the base solution was dispensed and autoclaved. Prior to use, add the following solutions to each flask in the order listed: 1 ml KH2P04 (0.50 / ) 1 ml CaCl2  6 H2 0 (3.68%) TES buffer (5, 75%, adjusted (To pH 7.2) T (transition) buffer is prepared by mixing the following sterile solutions: -58- (53) (53) 200418849 2 5 ml sucrose (10.3%) 7 5 ml distilled water 1 ml as for R2 medium The trace element solution was 1 ml of K 2 S 0 4 (2.5%) and then the following was added to 9.3 ml of this solution: 0.2 ml of CaCl2 (5M) and 0.5 ml of Tris maleic acid buffer solution. A 1 M Tris solution was prepared by adding maleic acid to pH 8.0. In use, 3 parts by volume of the above solution was added to 1 part by weight of the autoclaved PEG1000. L (lysis) buffer is prepared by mixing the following sterile solutions: 100 ml sucrose (10.3%) 10 ml TES buffer (5.73%, adjusted to pH 7.2) 1 ml K2S04 (2.5%) 1 ml as for R2 medium The trace element solution 1 ml KH2P04 (0.5%) 0.1 ml MgCl2 6H20 (2.5M) 1 ml CaCl2 (0.25M) CRM medium A solution containing the following ingredients was prepared in 1 liter dH20: glucose (10 g ), Sucrose (103 g), MgCl2  6H2O (10.12 g), BBLTM trypticase soy broth (15 g) (Becton (54) 200418849</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Dickinson Microbiology Systems, Sparks, Maryland, USA) BBLTM 5 Becton Dickinson Microbiology Systems) 30 t h i  s t r e p t   ) 1 0/  dH2030%(/PEG 1 00010% 65%0.22 </span>Dickinson Microbiology Systems, Sparks, Maryland, USA), and BBLTM Yeast Extract (5 g) (Becton Dickinson Microbiology Systems). This solution was autoclaved for 30 minutes. For cultures using plastid proliferation, thiostrepton (t h i 0 s tr e p t  ) was added to a concentration of 10 g / ml. Electropenetration buffer A solution containing 30% (w / v) PEG 1,000, 10% glycerol, and 6.5% sucrose was prepared in dH20. This solution was sterilized by vacuum filtration through a 0.22 micron cellulose acetate filter.</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">2 :SCI 5 847DNA</span>Example 2: Extraction of chromosomal DNA from strain SCI 5 847</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">-DNAzolInvitrogen, Carlsbad, CaliforniaUSA)  SC 1 5 8 47 (ATCC PT- 1 043 ) DNA SC 1 5 84728CF72.2% 1.0%1.0%0.1%pH 7 .  )2 42 0  20 DNAzol Backman TJ61010100% 3DNA 1 % 15 008 mM NaOH 301 M HEP ES pH 7.2 -60- (55) 200418849 3 : PCR</span>Genomic DNA was isolated from a Streptomyces orientalis soil isolate strain number SC 1 5 8 47 (ATCC PT-1 043) using a guanidine-detergent lysis method and DNAzol agent (Invitrogen, Carlsbad, California, USA). The SC 1 5 847 culture was grown in F7 medium (glucose 2.2%, yeast extract 1.0%, malt extract 1.0%, gastric protein gland 0.1%, pH 7.0) at 28  C for 24 hours. 20 ml of the culture was collected by centrifugation and resuspended in 20 ml of DNAzol, mixed by suction and centrifuged in a Backman TJ6 centrifuge for 10 minutes. Add 10 ml of 100% ethanol, invert several times, and store at room temperature for 3 minutes. The DNA was wound on a glass pipette, washed in 100% ethanol and allowed to air dry for 10 minutes. Shen Jiu was resuspended in 500 microliters of 8 mM NaOH and after dissolution, it was neutralized with 30 microliters of 1 M HEP ES pH 7.2. -60- (55) 200418849 Example 3: PCR reaction</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">PCR50200- 5 0 0 ng)DNA1.0cDNA P45 0- 1 +(SEQ ID NO:23) P45 0-la+ (SEQ ID NO:24) P45 0-2+ (SEQ ID NO:25)  P45 0-3_(SEQ ID NO:27) P45 0-2_(SEQ ID NO: 26)1.4-2.0 //  PCR  1  Taq 2.5  Stratagene)5Taq42.5 mM dNTPs dH20 50 GeneampPCR95C5 [95C 3037C 1530%72C</span>The PCR reaction was prepared in a 50 microliter volume containing 200-500 nanograms (ng) of genomic DNA or 1.0 microliter of cDNA, a forward and reverse primer, and the forward primer line P45 0- 1 + (SEQ ID NO: 23) or P45 0-la + (SEQ ID NO: 24) or P45 0-2 + (SEQ ID NO: 25) and the reverse primer is P45 0-3_ (SEQ ID NO: 27) or P45 0-2_ (SEQ ID NO: 26). All primers were added to a final concentration of 1.4-2.0 // M. PCR reactions were performed with 1 l of Taq enzyme (2.5 units) (Stratagene), 5 l of Taq buffer and 4 l of 2.5 mM dNTPs, plus dH20 to 50 l. The various cycling reactions were performed on the Geneamp PCR system with the following protocol: 95  C for 5 minutes, 5 cycles of [95  C for 30 seconds, 37  C for 15 seconds (30% ramp up), 72  C</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">30]30[94 C 3065C 1572 C 30]72C 7P45 0-</span>30 seconds], 30 cycles of [94  C for 30 seconds, 65  C for 15 seconds, 72  C for 30 seconds], and 72  C for 7 minutes. Expected size of reactant is P45 0-</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">1+(SEQ ID NO:23) P45 0-la+ (SEQ ID NO:24) P4 5 0-3'(SEQ ID NO:27) 3 40 bp  P45 0-2 + (SEQ ID 1^0:25)450-3_(5(^10 1^0:26)130 6 4 :B  Bglll37C2SC 1 5 8 4 7 DNA 6 30k nanosepGelman ScienceAnn ArborMichiganUSA) DNA  -61 - (56) 200418849 402 TE0.7% 1 21 5 kbDN A QiagenDNA  PWB19N ( 5,5 1 6,679) BamHI  SAP I Roche Molecular B i o c h e m i c a 1 s , Indianapolis, Indiana, catalog#l 7 5 8 25 0)</span>1+ (SEQ ID NO: 23) or 3,40 bp of P45 0-la + (SEQ ID NO: 24) and P4 5 0-3 '(SEQ ID NO: 27) primer pair and P45 0-2 + (SEQ ID 1 ^ 0: 25) and? 450-3_ (5 ugly (^ 10 1 ^ 0: 26) primer pairs of 130 6 mouths. Example 4: Epothilone B hydroxylase and ferredoxin gene cloning using Bglll restriction enzyme at 37  Digest 20 g of SC 1 5 8 4 7 genomic DNA for 6 hours at C. Use a 30k nanosep column (Gelman Science, Ann Arbor, Michigan, USA) to concentrate the DNA and remove -61-(56) 200418849 enzymes and buffers The reaction solution was concentrated to 40 microliters and washed with 200 microliters of TE. Then the digestion products were separated on a 0.7% agarose gel and cut from the gel in the range of 1 to 15 kb. Genomic DNA was purified by Qiagen gel extraction. This genomic DNA was then ligated to plastid PWB19N (US Patent 5,5 1 6,679), which was digested with BamHI and SAP I enzyme (Roche Molecular B iochemica 1) s, Indianapolis, Indiana, catalog # l 7 5 8 25 0)</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">151UT4 DNAInvitrogen)11  1 DH10B (Invitrogen)10030/ LB3 7 C</span>Dephosphorylated. The ligation reaction was performed with 1 U of T4 DNA ligase (Invitrogen) in a volume of 15 l at room temperature for 1 hour. 100 l biochemical student competent DH10B cells (Invitrogen) were transformed with 1 l of the linker and 100 l were taken on five LB agar plates containing 30 g / ml neomycin and plated at 37  C. to cultivate.</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text"> 5 nylon)Roche Molecular Biochemicals, IndianapolisIndiana) 25 Whatman10% SDS5 0.5 N NaOH + 1.5 M NaCl  5 1.5  NaCl + 10 M Tris pH 8.0 5  2 SSC  15  the Southern hybridization) 2TB + 3 0 /37 C miniprepMo Bio)DNA NPB29-1  -62- (57) (57)200418849 5 : DNA Big-Dye Applied BiosystemsFoster CityCaliforniaUSA) PCR ABI Prism 310 Applied Biosystems) 6 :RNA ChomczynskiSacchi TrizolInvitrogen) SC 1 5 847 RNA5SCI 5 847  5 00-100F7 30C23 0 rpm 20600nm)ODmo)9,0 16C23 0 rpm 2 5 5B11   % 1 Bi 6 t2 3 0 r p m3 03 0  50-15 51   5 0-FalcnTJ64C5 21 5 2TJ6 1 02.5 -63- (58) (58)200418849 RN A1 4 C1 070% 1 5 0 RNase dH20RNA 50 7.5  LiCl- 20C30RNA4 1 200 70%150 RNasedH20 _  DNaseI(AmbionAustinTexasUSA) RNA25RNA( 5.3/2.5 DNasel1.0DNase37C25 5DNasel2 1 7 : cDNA  Superscript II Invitrogen) RNA   cDNA1RNA(5.3/9 dH201dNTP10 mM) 6 5 t5 41 st 1RNase2.00.1 M DTT1 S u p e r s c r i p t 111  42 C507(TC151 RNaseH37C2070(15 -64- (59) 200418849 4 C 8 : DNA</span>Orient 5 nylon membrane rings (Roche Molecular Biochemicals, Indianapolis, Indiana) with numbers and signs. The films were placed on the plates for 2 minutes and then allowed to dry for 5 minutes. The membranes were then placed on Whatman filter paper, saturated with 10% SDS for 5 minutes, 0.5 N NaOH + 1.5 M NaCl for 5 minutes, 1.5 mM NaCl + 1.0 M Tris pH 8.0 for 5 minutes, and on 2X SSC for 15 minutes. minute. Filter paper was hybridized as described in the Southern hybridization. The hybrid community was picked on 2 ml TB + 30 g / ml neomycin and grown at 37  C all night. Isolation of plastid DNA using the miniprep column program (Mo Bio). This plastid is called NPB29-1. -62- (57) (57) 200418849 Example 5: DNA sequencing and analysis Cyclic sequencing using a fluorescent dye-labeled terminator, Big-Dye sequencing kit (Applied Biosystems, Foster City, California, USA) The selected PCR products were sequenced and analyzed by laser-induced fluorescence capillary electrophoresis using an ABI Prism 310 sequencer (Applied Biosystems). Example 6: Extraction of total RNA Total RNA was isolated from SC 1 5 847 cultures using a single-phase phenylhydrazone and guanidine isothiocyanate solution, Trizol agent (Invitrogen) using a modified method of the Chomczynski and Sacchi methods. 5 ml of SCI 5 847 frozen stock culture was thawed and used to prepare 100 ml of F7 medium in a 500-ml Erlenmeyer flask. The culture was grown in a shaker incubator at 30  C for 20 hours at 23 0 rpm to achieve an optical density (ODmo) of 600 nanometers (nm) of 9,0. The culture was placed in a shaking incubator at 16  C for 20 minutes at 23 rpm. 55 mg of epothilone B was dissolved in 1 ml of 100% ethanol and added to the culture. Rinse residual epothilone B from the tube with a second 1 ml ethanol and add to the culture. The culture was incubated for 30 hours at 2 6 r p m at i 6 t :. 30 ml of the culture was transferred to a 50-ml tube, 150 mg of lysozyme was added to the culture, and the culture was incubated at room temperature for 5 minutes. 10 ml of the culture was placed in a 50-ml Falcon tube and centrifuged in a TJ6 centrifuge at 4  C for 5 minutes. Add 2 ml of chloroform and mix the tube vigorously for 15 seconds. The tube was left at room temperature for 2 minutes, and then centrifuged at top speed in a TJ6 centrifuge for 10 minutes. The aqueous layer was transferred to a new tube and 2.5 ml of isopropyl-63- (58) (58) 200418849 ethyl alcohol was added to precipitate RN A. The tube was left at room temperature for 10 minutes, and then centrifuged at 4  C for 10 minutes. Remove the supernatant, wash the sink with 70% ethanol, and dry slightly under vacuum. This Shen Jiu was resuspended in 150 microliters of RNase-free dH20. Add 50 l of 7.5 M LiCl to the RNA and incubate at -20  C for 30 minutes. The RNA was allowed to settle by centrifuging the RNA in a microcentrifuge for 10 minutes at 4  C. Shenjiu was washed with 200 microliters of 70% ethanol, dried slightly under vacuum and resuspended in 150 microliters of RNase-free dH20. _ Treat the RNA with DNaseI (Ambion, Austin, Texas, USA). Add 25 l of total RNA (5.3 g / l), 2.5 l of DNasel buffer, 1.0 l of DNase and incubate at 37  C for 25 minutes. Add 5 microliters of DNasel inhibitor buffer, incubate for 2 minutes, centrifuge for 1 minute, and transfer the supernatant to a new tube. Example 7: cDNA Synthesis cDNA was synthesized from total RNA using Superscript II enzyme (Invitrogen). The reaction was prepared using 1 l of total RNA (5.3 g / l), 9 l of dH20, 1 l of dNTP mixture (10 mM), and one l of random hexamer. The reaction was incubated at 65 t for 5 minutes and then placed on ice. The following ingredients were then added: 4 microliters of 1 st stock buffer, 1 microliter of RNase inhibitor, 2.0 microliters of 0.1 M DTT, and 1 microliter of Supper s c r i p t 11 enzyme. The reaction was incubated for 10 minutes at room temperature, 50 minutes at 42  C and 15 minutes at 7  C. Add 1 l of RNaseH and incubate at 37  C for 20 minutes at 70. : 15 minutes below -64- (59) 200418849 minutes and stored at 4  C. Example 8: DNA labeling</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">DNAcDNAP4 5 0  pCRscript-29 pCRscript-29  P4 5 0- 1 +(SEQ ID NO:23) P450-3'(SEQ ID NO:27) SC 1 5 84 7  DNA   340 bp PCR2 25 Qiaquick (Qiagen)DNA 5TE5 00  chem link Roche Molecular Biochemicals Indianapolis Indiana, catalog# 1 836 463)</span>The conditions used to amplify P4 50-specific products from genomic DNA and cDNA were used to amplify inserts of plastid pCRscript-29. Plastid pCRscript-29 contains a 340 bp PCR fragment amplified from SC 1 5 84 7 genomic DNA using primers P4 50- 1 + (SEQ ID NO: 23) and P450-3 '(SEQ ID NO: 27). A total of 25 cycles were performed using 2 microliters of the plastid preparation as a template. Amplification products were gel purified using a Qiaquick gel extraction system (Qiagen). The extracted DNA was precipitated with ethanol and resuspended in 5 microliters of TE, and the yield was estimated to be 500 nanograms. Use chem link markers (Roche Molecular Biochemicals, Indianapolis, Indiana, catalog # 1 836 463)</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">(digoxigenin)5PCR0.5 Dig-chem linkdH220 8 5 C3 05 20/ 9 :DNA</span>(digoxigenin). Mix 5 l of the PCR product with 0.5 l of the Dig-chem link and add dH20 to 20 l. After the reaction was incubated at 85  C for 30 minutes, 5 l of the solution was added. The concentration of the probe was estimated at 20 nanograms / microliter. Example 9: Southern Southern DNA Hybridization</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text"> BamHIBglllEcoRIHindlll  Notl  1  DNA( 0.5/12volt) 1 6. 2 5 N HC 11 0 vacuum b 1 o 11 e r) (B i  - R a d Laboratories, Inc., HerculesCaliforniaUSAcatalog#165-5000) 0.4 N -65- (60) 200418849</span>Use BamHI, Bglll, EcoRI, Hindlll, or Notl to digest 10 l of genomic DNA (0.5 g / l) and isolate at 12 volts for 16 hours. The gel was depurinated in 0.2 5 N HC 1 for 10 minutes and a vacuum b 1 o 11 er was used (Bi-Lab Laboratories, Inc., Hercules, California, USA, catalog # 165-5000) at 0.4 N -65- (60) 200418849</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">NaOH 5 HgRoche Molecular B i o c h e m i c a 1 s) 1 M Stratalinker UV Crosslinker (Stratagene) UV- 2X S SC</span>NaOH 5 Hg was transferred to a nylon membrane (Roche Molecular Biochemica 1 s) under vacuum. The membrane was rinsed in a 1 M acetate lens and UV-crosslinked using a Stratalinker UV Crosslinker (Stratagene). The membrane was dipped Rinse in 2X S SC and store at room temperature.</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text"> 20 Dig Easy HybRoche Molecular Biochemicals) 40 C  1  65 C1 5 20//Dig-Easy Hyb42C 2X SCC + 0.1% SDS .5X SCC + 0.1% SDS65C Genius1(10 mM 15 mM NaCl  pH 7.5  0.3% v/v Tween 20) (Roche Molecular Biochemicals, Indianapolis, Indiana) 2  2 %2 % B 1 o c k i n g r e a g e n tG e n i u s  1 Roche Molecular BiochemicalsIndianapolis Indiana)1-dig 501 : 20,00030 50Genius115 Genius3(111^1[1^-HCl10mMNaCl;pH9.5)2l CSPD (3-(4-{12--32-(5- [3.3.1.1.3,7]4-Roche Molecular B i o c h e m i c a 1 s)G e n i u s31 : 1 0 0 537C15 - 66 _ (61) 200418849  Biomax ML Kodak, RochesterNew York, US A) 1  1  : Invitrogen DH 1 0B 10017 X 100- 1 3 42C45 1-20.9SOCInvitrogen) 3 0- 3 7 C200-240 rpm1 Luria30 /1/</span>The membrane was prehybridized in 20 ml of Dig Easy Hyb buffer (Roche Molecular Biochemicals) at 40  C for 1 hour. The probe was denatured at 65  C for 10 minutes and then placed on ice. Take 5 ml of probe / Dig-Easy Hyb with a concentration of about 20 nanograms / ml together with the membrane and incubate at 42  C overnight. The membrane was washed twice in 2X SCC + 0.1% SDS at room temperature and then washed twice in 0.5X SCC + 0.1% SDS at 65  C. The membrane was placed in Genius buffer 1 (10 mM maleic acid, 15 mM NaCl; pH 7.5; 0.3% v / v Tween 20) (Roche Molecular Biochemicals, Indianapolis, Indiana) for 2 minutes, and then mixed with 2% Blocking solution (2% Bockingreagent in Genius Buffer 1) (Roche Molecular Biochemicals, Indianapolis, Indiana) incubate at room temperature for 1 hour. The membrane was incubated with a 1: 20,000 dilution of anti-dig antibody in 50 ml of blocking solution for 30 minutes. The membrane was washed twice in 50 ml of Genius Buffer 1 for 15 minutes. The membrane was equilibrated in Genius buffer 3 (1011 ^ 1 [1 ^ -HCl, 10mM NaCl; pH 9.5) for 2 minutes. 1 ml of CSPD (3- (4-Methoxyspiro {1,2-dioxetane-3,2 '-(5'-chloro) tricyclo [3.3.1.1.3,7] decane Alkyl 4-yl) disodium phenyl phosphate (Roche Molecular Biochemica 1 s) in Genius Buffer 3 was added to the membrane in a 1: 1 dilution and incubated at room temperature for 5 minutes, then at 37 Incubate at  C for 15 minutes-66 _ (61) 200418849 minutes. The membrane was exposed to a Biomax ML membrane (Kodak, Rochester, New York, US A) for 1 hour. Example 10: E. coli transformation competent cell lines were purchased from Invitrogen. E. coli strain DH 1 0B was used as the host for genetic selection. The chemically competent cells were thawed on ice and a 100 microliter portion was added to a 17 X 100-mm polypropylene tube placed on ice. Take 1 l of the ligation mixture to the cells and incubate on ice for 30 minutes. The cells were incubated at 42  C for 45 seconds, and then placed on ice for 1-2 minutes. Add 0.9 ml of SOC medium (Invitrogen) and incubate the cells at 30-37  C for 1 hour at 200-240 rpm. Cells were plated on selective medium (Luria agar containing neomycin or ampicillin at a concentration of 30 g / ml or 100 g / ml, respectively).</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">1 1 :TK24 HindlllpWB19NSAPI  Hindlll  pANT849(Keiseret al.2000, Practical Streptomyces GeneticsJohn Innes) pWB 1 9N8491UT4 DNA 11 XL-1 BlueStratagene) LB3/37t 230/LB -67- (62) 200418849 3 0CMoBio PWB19NpANT849  0.7% Hi n dill BglllpWB19N849Bglll HindlllpANT8 495.3 kb 5.3 kbQiaquick</span>Example 11 1: Transformation of Streptomyces lividans TK24 via Hindlll digestion pWB19N and SAPI treatment and Hindlll digestion pANT849 (Keiser, et al., 2000, Practical Streptomyces Genetics, John Innes) to construct Body pWB 1 9N849. The two linearized fragments were ligated using 1U of T4 DNA ligase at room temperature for 1 hour. XL-1 Blue competent cells (Stratagene) were transformed with 1 microliter of ligation reaction. The recovered cells were plated on LB neomycin (30 g / ml) at 37 t overnight. The colonies were picked onto 2 ml of LB containing 30 g / ml neomycin and -67- (62) 200418849 was incubated at 30  C overnight. MoBio plastid mini-preparation was performed on all media. The electrophoretic mobility was measured on a 0.7% agarose plastid consisting of PWB19N and pANT849. The plastid pWB19N849 was digested with Hindill and Bglll to cut out a 5.3 kb fragment equivalent to the digestion of pANT8 49 with Bglll and Hindlll. This 5.3 kb fragment was purified on an agarose gel and extracted using a Qiaquick gel extraction system.</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">B 1.469 kb DNAPCR50 PCR5Taq2.51 20/NPB29-10.425 mM dNTPs 1.0  NPB29-6F(SEQ ID NO:28) NPB29-7R(SEQ ID   : 2 9 ) ( 5  / 3 8  1  dH2  5  Taq Stratagene) Perkin Elmer 9 70095C 53096C</span>A 1.469 kb DNA fragment containing the epothilone B hydroxylase gene and the downstream ferredoxin gene was amplified by PCR. 50 l of PCR reaction includes 5 l of Taq buffer, 2.5 l of glycerol, 1 l of 20 ng / l NPB29-1 plastids, 0.4 l of 25 mM dNTPs, and 1.0 l of primers each NPB29-6F (SEQ ID NO: 28) and NPB29-7R (SEQ ID NO: 2 9) (5 picomoles / microliter), 3 8  1 microliter dH20 and 0.5 microliter Taq enzyme ( Stratagene). The reaction was performed on a Perkin Elmer 9 700, 95  C for 5 minutes, and then 30 cycles (96  C</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">3 06 0 C 3 07 2 C 2 7 2 C 7 QiagenPCRPCR BglllHindlll0.7%  1.469 kb Qiagen5PCR 2BglllHindlllPANT 8 491 24 TK24 20YEME 4 125-48 -68- (63) (63)200418849 s Practical Streptomyces Genetics  5 0 05p 2,7 5 0 r p m71   P R2YE 2 8 C2 052 5 0/ LB 0.7%7 50/R2YE28C 5  ATCCPTA-4022  1 2 :  Pigac and S chrempf Appl. Environ Microb ., V o 1. 61, No.1, 3 5 2-3 5 6 ( 1 995 ) R6 59320CRM30 C 250 rpm)244C5,000 r p m54 C 2 0 1 0 %4 C50 0 0 rp m5 4 C 1 01 5 %4 C50 0 0 rpm54C 2100 /1 5 %3 7 C3 0 4 C50 0  r p m54 C 2 1 5 %*1 5 %1 -69- (64) (64)200418849 25 0-200 -8 0 C  dH20100 NaCl0.3 24: 1:1 202 100%- 20CDNA30 DNA1070% speed-vac55 dH2   DNA 5 0/  DNA/12dH20 /DNA2  Bio-Rad Laboratories, Richmond, California USA) Gene PulserTM (Bio-Rad Laboratories) 2 kV (10 kV/cm)25 // F400  0.751.0CRM (0-4 C )15- 3 0C3 10-30/  1 3 :  Waters 2690 Separation Module Waters (65) (65)200418849</span>30 seconds, 60  C, 30 seconds, 7 2  C for 2 minutes), and 7 2  C for 7 minutes. PCR products were purified using a Qiagen mini-lysis system using a PCR protocol. The purified product was digested with Bglll and Hindlll and purified on a 0.7% agarose gel. A 1.469 kb band was cut from the gel and lysed using a Qiagen mini-dissolution column. Five microliters of this PCR product was ligated to 2 microliters of PANT 8 49 vector digested with Bglll and Hindlll in 10 microliters of ligation reaction. After the reaction was incubated at room temperature for 24 hours, the protoplasts of Streptomyces lividans TK24 were deformed. 20 ml of YEME medium was inoculated with frozen Streptomyces lividans TK 4 spore suspension and grown in a 125-ml double-well flask for 48 hours -68- (63) (63) 200418849. The protoplast system was prepared as described in Practical Streptomyces Genetics. The ligation reaction was mixed with the protoplasts, 500 microliters of transformation buffer was added, followed by 5 ml of p buffer. The transformation reaction was spun at 2,750 rpm for 7 minutes, resuspended in 1000 microliters of P buffer and plated on an R2YE plate. After the plate was incubated at 28  C for 20 hours, it was covered with 5 ml of LB 0.7% agar containing 250 micrograms / ml thiostrepton. After 7 days, the colonies were picked up on R2YE grids containing 50 g / ml thiostrepton. Colonies were regrown for 5 days at 28  C and stored below. This recombinant microorganism was deposited with the ATCC and was numbered PTA-4022  Example 12 2: The transformation of Streptomyces rupture adopts Pigac and Schrempf Appl. Environ Microb., V o 1. 61, No. 1, 3 5 2-3 5 6 (1 995) procedures to transform Streptomyces fissures. Streptomyces rupture R6 593 was cultured in a 20 ml CRM medium on a rotary shaker (250 rpm) at 30  C. After 24 hours, cells were harvested by centrifugation at 5,000 rpm for 5 minutes at 4  C and resuspended in 10% sucrose in 4 ml of 20 C and centrifuged at 5,000 rpm for 5 minutes at 4  C . Shen Jiu was resuspended in 10 ml of 15% glycerol at 4  C and centrifuged at 5000 rpm for 5 minutes at 4  C. Shen Jiu was resuspended in 2 ml of 15% glycerol containing 100 g / ml lysozyme at 4  C and incubated at 37  C for 30 minutes. 'Centrifuge at 4  C at 5,000 rpm for 5 minutes. Minutes and resuspended in 2  L of 15% glycerol at 4  C. Repeat one * times washing with 15% glycerol and resuspend Su Jiu in 1 -69- (64) (64) 200418849 to 2 ml of electro-penetration buffer. Store cells at 80-200  C in 50-200 l portions. The ligation reaction was prepared as described for the transformation of S. lividans. After the ligation reaction was inoculated, the volume was adjusted to 100 l using dH20, NaCl was added to 0.3 M, and the reaction was extracted with an equal volume of 24: 1: 1 phenol: chloroform: isoamyl alcohol. Add 20 micrograms of glycogen and precipitate the ligated DNA with 2 volumes of 100% ethanol for 30 minutes at -20  C. The DNA was allowed to stand in a microcentrifuge for 10 minutes, washed once with 70% ethanol, dried in a speed-vac concentrator for 5 minutes, and resuspended in 5 l of dH20. A frozen aliquot of the cells was thawed at room temperature and 50 L / tube of each DNA was dispensed for electrical penetration. Keep cells on ice until use. Add DNA / 1 to 2 l of dH20 and mix. The cell / DNA mixture was transferred to a 2 mm gap vial pre-chilled on ice (Bio-Rad Laboratories, Richmond, California, USA). Gene PulserTM (Bio-Rad Laboratories) was used to penetrate cells electrically at a setting of 2 kV (10 kV / cm), 25 // F, 400 . The cells were diluted with 0.75 to 1.0 ml of CRM (0-4  C), transferred to a 15-mL culture tube and cultured with stirring at 30  C for 3 hours. The cells were plated on trypsin soy liquid culture-based agar plates containing 10-30 micrograms / ml thiostrepton. Example 13: High performance liquid chromatography using Waters 2690 Separation Module system (Waters (65) (65) 200418849</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Corp.Milford, MAUSA) 3.5 Symmetry Shield RP82 4.6 x 150 Waters Corp. MilfordMA, USA)1.0/ A/20 : 1) + 10 mMB/20 : 1) 612%B28%B4 28%B2028%B 1 00%B21 2%B 312%B 1 4 : ZMDMicromass, ManchesterUK) Test Juice Waters CorpMilfordMAUSA) Harvard Apparatus, Holliston,  AUS A)10/ 1 3.21 1.2 m/z 10060010/ ES) 290017  15 :ebhSEQ ID NO: 1) P4 5 0 (66) 200418849</span>Corp., Milford, MA, USA) and liquid chromatography were performed on a Symmetry Shield RP82 4.6 x 150 mm column (Waters Corp., Milford, MA, USA) packed with 3.5 micron particles. A gradient mobile phase program was used at a flow rate of 1.0 ml / min. The eluent A was water / acetonitrile (20: 1) + 10 mM ammonium acetate. The eluent B was water / acetonitrile (20: 1). The mobile phase was a linear gradient from 12% B to 28% B in 6 minutes and then held in isocratic solvent at 28% B in 4 minutes. It was followed by a linear gradient from 28% B to 100% B in 20 minutes and then a linear gradient to 12% B in 2 minutes, and then held at 12% B in 3 minutes. Example 14 4: Mass spectrometry The column eluate was directly introduced into the electrospray ion source of a ZMD mass spectrometer (Micromass, Manchester, UK). This instrument was calibrated using a Test Juice reference standard (Waters Corp., Milford, MA, USA) and delivered from a syringe pump (Harvard Apparatus, Holliston, M A, US A) at a flow rate of 10 l / min. The mass spectrometer operates with a low mass resolution of 13.2 and a high quality resolution of 11.2. Spectra were collected using a scan range of m / z 100 to 600 at an acquisition rate of 10 MS / sec. The ionization technology used is cationic electrospray (ES). The sprayer voltage was maintained at 2900 volts and the ion cone voltage was maintained at a potential of 17 volts. Example 15: Isolation of cytochrome P4 50 gene from other microorganisms using ebh gene sequence (SEQ ID NO: 1) (66) 200418849</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text"> DNAzolATCC4 3 49 1 ATCC 1 4 93 0ATCC 5 3 6 3 0ATCC 5 3 5 5 0ATCC 3 9444 ATCC 4 3 3 3 3AT C C 3 5 1 6 5 ) DNA DNAebhPCR NPB29-6f (SEQ ID NO:28) NPB29-7r (SEQ ID NO:29)NPB29-16f (SEQ ID NO:50) NPB29-17r (SEQ ID NO:51) NPB29-19f (SEQ ID NO:52) 29-20 (SEQ ID NO:53) 20PCR20 0- 5 00 DNA  1.4-2.0 //   0.2 </span>Use DNAzol reagent from a group of cultures (ATCC4 3 49 1, ATCC 1 4 93 0, ATCC 5 3 6 3 0, ATCC 5 3 5 5 0, ATCC 3 9444, ATCC 4 3 3 3 3, AT CC 3 5 1 65) Isolate genomic DNA. This DNA was used as a template for PCR using primers designed for the ebh gene sequence. Three sets of primers were used for amplification: NPB29-6f (SEQ ID NO: 28) and NPB29-7r (SEQ ID NO: 29), NPB29-16f (SEQ ID NO: 50), and NPB29-17r (SEQ ID NO: 51) ), And NPB29-19f (SEQ ID NO: 52) and NPB29-20 (SEQ ID NO: 53). Prepare a PCR reaction in a volume of 20 microliters, containing 200-500 nanograms of genomic DNA and forward and reverse primers. All primers were added to a final concentration of 1.4-2.0 // . Use 0.2 l</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">AdvantageTM 2 Taq BD Biosciences ClontechPalo AltoCalifornia, USA)2  Advantage1 M 2 Taq  0.22.5 mM dNTPs dH2020  PCR Geneamp970 Mastercycler gradient (Eppendorf, Westbury? New York, US A)95C35 [96 C 20 54-69C 30 72C 2 ]72C 7PCRNPB29-6f (SEQ ID NO:28) NPB29-7r (SEQ ID NO:29) 1 469 bp  NPB29- 16f (SEQ ID NO:50) NPB29-17r (SEQ ID NO:51) 1 03 4 bp NPB29-19f (SEQ ID NO:52)  NPB29-20r (SEQ ID NO:53) 1318 bp 0.7% PCR -72- (67) 200418849 PCRQiagen Big-Dye 10  16:pPCRscript-ebh</span>AdvantageTM 2 Taq enzyme (BD Biosciences Clontech, Palo Alto, California, USA), 2 l of Advantage 1 M 2 Taq buffer and 0.2 l of 2.5 mM dNTPs, dH20 to 20 l were used to prepare PCR reactions. The cyclic reaction was performed on a Geneamp 9700 system or Mastercycler gradient (Eppendorf, Westbury? New York, US A) using the following scheme: 95  C for five minutes, 35 cycles of [96  C 20 seconds , 54-69  C for 30 seconds, 72  C for 2 minutes], 72  C for 7 minutes. The expected PCR product size is 1 469 bp for NPB29-6f (SEQ ID NO: 28) and NPB29-7r (SEQ ID NO: 29) primer pairs, for NPB29-16f (SEQ ID NO: 50) and NPB29-17r ( The primer pair of SEQ ID NO: 51) is 1 03 4 bp, and the primer pair of NPB29-19f (SEQ ID NO: 52) and NPB29-20r (SEQ ID NO: 53) is 1318 bp. PCR reactions were analyzed on a 0.7% agarose gel. A PCR product with the expected size of -72- (67) 200418849 was cut from the gel and purified using Qiagen gel extraction. The purified products were analyzed using Big-Dye sequencing kits and sequencing using an 10 sequencer. Example 16: Composition of plastid pPCRscript-ebh</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">PCRB 1.469 bp DNA50PCR 5Taq2.51 20/ NPB29_10.4 25 mM dNTPs 1 .0  NPB29-6f(SEQ ID NO:28) NPB29-7r(SEQ ID N  : 2 9 ) ( 5  / 3 8 . 1  d H2   0.5  Taq Stratagene)</span>A 1.467 bp DNA fragment containing the epothilone B hydroxylase gene and the downstream ferredoxin gene was amplified using PCR. 50 l of PCR reaction includes 5 l of Taq buffer, 2.5 l of glycerol, 1 l of 20 ng / l NPB29_1 plastids, 0.4 l of 25 mM dNTPs, 1.0 l of each primer NPB29 -6f (SEQ ID NO: 28) and NPB29-7r (SEQ ID NO: 2 9) (5 picomoles / microliter), 38.1 microliter d H2 0 and 0.5 microliter Taq enzyme (Stratagene) . in</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Gene amp 9 70095C 5</span>Reactions were performed on a Gene amp 9 700 system using the following conditions: 95  C 5 minutes</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">3096t 3060C 3072C 272C 7Qiagen Qiaquick</span>Clock, then 30 cycles (96t: 30 seconds, 60  C for 30 seconds, 72  C for 2 minutes), and 72  C for 7 minutes. Use Qiagen Qiaquick columns to</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">PCRPCRBglll Hi nd III0.7%  1.469 kbQiagen Qiaquick P C R s c r i p t A m p PCRscript Amp 3 0 - 3 7 C  2 3 0 - 3 0 0 rp m1 6 - 2 4 1 - 2  100/LB(Luria Broth)  Mo BioBig-Dye         -73- (68) (68)200418849  pPCRscript-ebh 1 7 : ebh   Stratagene  Quikchange XL Site-Directed Mutagenesis Kit  Quikchange Multi Site-Directed Mutagenesis Kit ebh 35-45 (SEQ ID NO: 5 4-5 9 fP 77)DNA  PfuTUr*bDNA 5,545,5525,866,3955,948,663  Dpnl Dpnl 13 pPCRscript-ebh  12XL 1-Blue  XL10-GldSUatagene) LA(Luria Agar)100/ 1/3 0 -3 7C24-48 51 00/ LB Mo Bio -74- (69) 200418849</span>The PCR product was used to purify the PCR products. The purified product was digested with Bglll and Hind III and purified on a 0.7% agarose gel. A 1.469 kb band was cut from the gel and eluted using the Qiagen Qiaquick gel extraction procedure. This fragment was then cloned into a PCRscript Amp vector using the P C R s c r i p t A m p selection kit. The colonies containing the inserts were picked up at 30-37  C for 2 3-3 0 0 rp m, 16-24 hours, 1-2 ml of 100 g / ml ampicillin LB (Luria Broth). Mo Bio mini plastid preparation kits were used for plastid separation. Use the Big-Dye sequencing kit to determine the sequence of the insert via cyclic sequencing. The plastid -73- (68) (68) 200418849 is named pPCRscript-ebh. Example 17: Mutation Generation of the ebh Gene to Improve Yield or Change Specificity Using the Quikchange XL Site-Directed Mutagenesis Kit and Quikchange Multi Site-Directed Mutagenesis Kit from Stratagene to introduce mutations in the coding region of the ebh gene . Both methods use a 35-45 base DNA primer containing the desired mutation (SEQ ID NO: 5 4-5 9 fP 77), a methylated circular plastid template, and PfuTUr * b DNA polymerase (U.S. Patent Nos. 5,545,552 and 5,866,395 and 5,948,663) to produce a copy of a plastid template with mutations contained in mutation-generating primers. The reactant was subsequently digested with the endonuclease enzyme Dpnl to selectively digest the methylated plastid template, but left intact unmethylated mutant plastids. All procedures were followed by the manufacturer's instructions, except that the incubation time was increased from 1 hour to 3 hours during the Dpnl digestion step. The pPCRscript-ebh vector was used as a template for mutation generation. Transform 1 to 2 microliters of the reaction into XL 1-Blue electric competent cells or XL10-Gold super competent cells (SUatagene). The cells were plated on a 100 g / ml ampicillin plate of LA (Luria Agar) to a density of more than 100 colonies / plate, and incubated at 30 to 37  C for 24-48 hours. The entire plate was resuspended in 5 ml of LB containing 100 g / ml ampicillin. The plastids were directly separated from the resuspended cells by centrifuging the cells, and the plastids were then purified using a Mo Bio mini plastid preparation procedure -74- (69) 200418849</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">NPB29-6f(SEQ ID NO:28) NPB29-7r(SEQ ID NO:29) PCR  BglllHindlll 1.469 kb PCRBglll HindlllpANT849 20-50100/LB 3 0-3 7 C 18-24 Qiagen midi-preps     DNA    BglllHindlll BglllHindlllpANT 849   Leung et al.Technique-A Journal of</span>body. This plastid was then used as a template for PCR using primers NPB29-6f (SEQ ID NO: 28) and NPB29-7r (SEQ ID NO: 29) to amplify the mutant expression cassette. This fragment was prepared using a 1.469 kb PCR product digested with the restriction enzymes Bglll and Hindlll to ligate to the vector pANT849, which was also digested with Bglll and Hindlll. In addition, resuspended cells were used to inoculate 20-50 ml of LB containing 100 g / ml ampicillin and grow at 30-37  C for 18-24 hours. Cultures were subjected to Qiagen midi-preps to isolate plastid DNA containing the desired mutations. Mutated expression cassettes were taken to limit digestion of the enzymes Bglll and Hindlll to connect to pANT 849, a plastid also digested with Bglll and Hindlll. Screening of mutants was performed as described in Streptomyces lividans or Streptomyces lividans. In addition, Leung et al., Technology-A Journal of</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Methods in Cell and Molecular Biology, Vol. 1, No. 1? 11-15 (1989)ebh/ d A TPTaq NotlpCRscript-ebh  0.166 M (NH4)2S40.67 M Tris-HCl pH 8.861 mM MgC26 7 //M EDTA pH 8.01.7/ PCR 10.0NotlpCRscript-ebh(0.1/ 1 1 . 1 //  -</span>Methods in Cell and Molecular Biology, Vol. 1, No. 1? 11-15 (1989) generate random mutation libraries of the ebh gene. Manganese and / or reduced d A TP concentration were used to control the mutation generation frequency of Taq polymerase, and Notl was used to digest plastid pCRscript-ebh to linearize the plastid. Polymerase buffer was prepared using 0.166 M (NH4) 2S04, 0.67 M Tris-HCl pH 8.8, 61 mM MgC2, 6 7 // M EDTA pH 8.0, 1.7 mg / ml bovine blood albumin. PCR reactions were prepared using the following: 10.0 l NotCR-digested pCRscript-ebh (0.1 ng / l), 10 l polymerase buffer, 1.0 l 1 // M-hydrosulfide Jiyi</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">10.0  DMSO1.0  NPB29-6f(SEQ ID :28) 100/1.0NPB29-7r(SEQ DI NO:29)100/10 5 mM -75- (70) (70)200418849</span>Alcohol, 10.0 l DMSO, 1.0 l NPB29-6f (SEQ ID NO: 28) primer (100 picomoles / l), 1.0 l NPB29-7r (SEQ DI NO: 29) primer (100 picomoles / l), 10 l of 5 mM -75- (70) (70) 200418849</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">MnCU10.0  l mM dGTP10.0  2 mM dATP 100  10 mM dTTP10.0  10 mM dCTP 2.0 TaqdH20100  MnCl2 GeneampPCR9 5 C 1 2 5 -3 094C 155C 3 07 2 C 4 72C 7 Qiagen PCRBglllHindlll Qiagen BglllHindlllP ANT 8 49  _   __ ebh24-16 92     23 7    ebh25- 1 195     294    ebh 1 0-53 1 90     23 1    ebh24-16d8 92   (71)200418849 23 7     67    e b h 2 4 - 1 6 c 1 1 92     93     23 7     3 65    ebh24-16-16 92     106     23 7    ebh24-16-74 88     92     23 7    ebhM- 1 8 3 1     176    ebh24-16g8 92     23 7     67     13 0     176  </span>MnCU, 10.0 l 10 mM dGTP, 10.0 l 2 mM dATP, 10.0 l 10 mM dTTP, 10.0 l 10 mM dCTP, and 2.0 l Taq polymerase. Add dH20 to 100 l. In addition, a reactant without MnCl2 was also prepared as described above. The cycling reactions were performed on the Geneamp PCR system with the following protocol: 95  C for 1 minute, 25-30 cycles (94  C for 1 minute, 55  C for 30 seconds, 7 2  C for 4 minutes) Minutes), 72  C for 7 minutes. PCR reactions were separated on an agarose gel using a Qiagen spin column. The fragments were then digested with Bglll and Hindlll and purified using a Qiagen spin column. The purified fragment was then ligated into P ANT 8 49 plastids digested with Bglll and Hindlll. Screening of mutants in Streptomyces lividans or Streptomyces lividans. Table of identified mutants _ mutant position _ substitution _ wild type ebh24-16 92 valine isoleucine 23 7 alanine phenylalanine ebh25- 1 195 serine asparagine 294 proline Serine ebh 1 0-53 1 90 Tyrosine Phenylalanine 23 1 Arginine Phenylamine ebh24-16d8 92 Valine Isoleucine (71) 200418849 23 7 Alanine Phenylalanine 67 Phenylamine Ethylamine ebh 2 4-1 6 c 1 1 92 Valine isoleucine 93 Glycine alanine 23 7 Alanine phenylalanine 3 65 Threonine isoleucine ebh24-16-16 92 Valine isopropyl Leucine 106 Alanine Valine 23 7 Alanine Phenylalanine ebh24-16-74 88 Histamine Arginine 92 Valine Isoleucine 23 7 Alanine Phenylalanine ebhM- 1 8 3 1 Leucine Phenylamine 176 Valine Methionine ebh24-16g8 92 Valine Isoleucine 23 7 Alanine Phenylalanine 67 Phenamine Amine Arginine 13 0 Threonine Isoleucine 176 Methyl Alanine Amino acid</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">-77- (72) (72)200418849 ebh24-16b9 92    23 7    67    140    176   1 8 :ebh B  1 2 5 -2 0 YE ME200TK24 pANT849-ebh)PANT849-1^10-53)1)1^84941^24-16) 30C23 0 rpm48 pANT849- ebh) (pANT849-ebhl0-53)pANT849-ebh24-16) 1 0/4 1 2 5 -2 0R 5 3 0 C2 3 0 r p m1 8 B/100 % EtOH0.5%/ pANT849-ebh24-l6) B48F) 0244872HPLC 0B -78- (73)200418849  B :  TK24 p ANT 8 49- p ANT849- p ANT 849-   (  ebh eb h10 - 5 3 ebh24-16   0 1 0 0 % 1 0 0 % 1 0 0 % 1 0 0 %   24 9 9% 7 8% 6 9% 5 6%   48 8 7% 19% 3 9% 0%   72 8 7% 0% 3%    F :   -------    TK24 p ANT849- pANT849- - p ANT849-   (  eb h eb h10 - 5 3 ebh24-16   0 0% 0% 0% ---- 0%   24 0% 4% 9% 23%   48 0% 2 1% 29% 5 2%   72 0% 14% 4 1% --- ebh B F1  201 /  CRM3C23_30 rpm 1 6-24B/1 00 %  -79- (74) (74)200418849 0.05%/3C 230-300 rPm20-40HPLC BF n -  __-  ebhM- 1 8 5 5%  ebh24-16d8 7 5%  ebh24-16cl1 7 5%  ebh24-16-16 75%  ebh24-16-74 75%  ebh24-16b9 8 0%  ebh24-16g8 8 5% 1 9 :  1 2 5 -2 01 0/ R2YE200 (PANT849)pANT849-ebh) 28C23 0 rpm724 2R2YE28t230 rpm2415_ 1040 /2 8 C2 5 0 r p m2 45 0 0 1 5 - -80 - (75) 200418849 5 0 05 0 mM3 TJ-6 3 000 rpm1HPLC  (p ANT 8 49)      (p ANT849) (p ANT849-ebh)   3 6% 1 1 %   1 1 % 53%  </span>-77- (72) (72) 200418849 ebh24-16b9 92 Valine isoleucine 23 7 Alanine Phenylalanine 67 Amine Amine Arginine 140 Threonine Alanine 176 Serine Methyl Thiamine Implementation Example 18: Comparison of the conversion of epothilone B in cells expressing ebh to its mutants. In these experiments, 20 ml of YE ME medium was inoculated in a 125-ml two-well flask. 200 l of Streptomyces lividans TK24, Streptomyces lividans (pANT849-ebh), Streptomyces lividans (PANT849-1 ^ 10-53), or Streptomyces lividans (1) Human 1 ^ 84941 ^ 24-16) and incubate at 30  C for 48 hours at 23 0 rpm. 10 g / ml sulfur was added to the medium inoculated with Streptomyces lividans (pANT849-ebh), Streptomyces lividans (pANT849-ebhl0-53), and Streptomyces lividans (pANT849-ebh24-16). Streptomycin. 4 ml of the culture was transferred to 20 ml of R 5 medium in a 125-ml Erlenmeyer flask and incubated at 30  C for 18 hours at 230 r pm. Epothilone B / 100% EtOH was added to each culture to a final concentration of 0.05% weight / volume. Except for the culture of Streptomyces lividans (pANT849-ebh24-16), in which epothilone B has been completely converted to epothilone F within 48 hours, at 0, 24, 48, and 72 hours Take a sample. The samples were analyzed by HPLC. The result was calculated as a percentage of epothilone B at -78- (73) 200418849 ratio at 0. Epothilone B: Time TK24 p ANT 8 49- p ANT849- p ANT 849- (hours) ebh eb h10-5 3 ebh24-16 0 1 0 0% 1 0 0% 1 0 0% 1 0 0% 24 9 9% 7 8% 6 9% 5 6% 48 8 7% 19% 3 9% 0%  72 8 7% 0% 3% epothilone F: ------- time TK24 p ANT849- pANT849- - p ANT849- (time) eb h eb h10-5 3 ebh24-16 0 0% 0% 0% ----- 0% 24 0% 4% 9% 23% 48 0% 2 1% 29% 5 2%  72 0% 14% 4 1% --- In addition, the use of E. pylori in host cells of Streptomyces lividans transformed with a plastid-containing transformant that contains the ebh gene and its variants or mutants Biotransformation of B into Epothilone F. One hundred microliters of a frozen Streptomyces lividans transformant culture was inoculated into 20 ml of CRM medium containing 10 micrograms / ml thiostrepton and at 30  C to 23-30  C. Incubate at 0 rpm for 16-24 hours. Add epothilone B / 1 00% ethanol -79- (74) (74) 200418849 to a final concentration of 0.05% weight / volume in each culture. The reaction is typically incubated at 30  C for 20-40 hours at 230-300 rPm. The concentrations of epothilone B and F were determined by HPLC analysis. In Streptomyces chrysophyllum n-empty body __ into --- ebhM- 1 8 5 5% ebh24-16d8 7 5% ebh24-16cl1 7 5% ebh24-16-16 75% ebh24-16-74 75% ebh24 -16b9 8 0% ebh24-16g8 8 5% Example 19: Bioconversion of compaction into pravastatin 20 ml in a 125-ml flask containing 10 micrograms / ml of Streptomyces The peptide's R2YE medium was inoculated with 200 microliters of Streptomyces lividans (PANT849), frozen spore preparations of Streptomyces lividans (pANT849-ebh) and incubated at 28  C for 72 hours at 23 0 rpm. 4 ml of the culture was inoculated into 20 ml of R2YE medium and grown at 230 rpm for 24 hours at 28 t. Transfer 1 ml of the culture to a 5-ml ml polypropylene culture tube. Add 10 L of vasopressin (40 mg / ml) to each culture and incubate at 28  C at 250 rpm. Incubate for 24 hours. Transfer 500 microliters of the culture fluid into a new 15-ml polypropylene culture tube -80-(75) 200418849. Add 500 microliters of 50 mM sodium hydroxide and swirl. 3 ml of methanol was added and rotated, and the tube was centrifuged in a TJ-6 table centrifuge at 3 000 rpm for 10 minutes. The organic phase was analyzed by HPLC. The slugs of compactin and pravastatin are shown as slugs of Streptomyces lividans (p ANT 8 49) relative to the control. The table shows the initial compaction concentration of compactin and pravastatin: Streptomyces lividans (P ANT849) (p ANT849-ebh) Compaction 3 6% 1 1% Vastat 1 1% 53% D</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">20 : </span>Example 20: High-performance liquid tomography for compaction and pravastatin detection</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Hewlett Packard 1 090 Series Separation (Agilent Technologies Palo Alto, California, US A) 5  Spherisorb OD S2  5 0 x 46 K e y s t  n e S c i e n t i f i cI n c . BellefontePennsylvaniaUSA) 2.0  /  A1 0 mM 0.05 %B4 2 0%B90%B -81 - (76) 200418849 21 :ebh25-l  HPLC  Hewlett Packard 1100 Series Liquid Chromatograph4.6 x 15 YMC Packed ODS-AQA) (B)20%90%B10 9 0%20%21/UV 2 5 4</span>The liquid chromatography separation system uses a Hewlett Packard 1 090 Series Separation system (Agilent Technologies, Palo Alto, California, US A) and a 50 x 46 mm column (K eyst) packed with Spherisorb OD S2 with a particle size of 5 m. ne Scientific, Inc., Bellefonte, Pennsylvania, USA). Use a gradient mobile phase program at a flow rate of 2.0 ml / min. The eluent A was water, 10 mM ammonium acetate and 0.05% phosphoric acid. The eluent B is acetonitrile. The mobile phase is a linear gradient from 20% B to 90% B in 4 minutes. -81-(76) 200418849 Example 21: Structural determination of the biotransformation product of mutant ebh25-1 Analytical HPLC is a YMC Packed ODS-AQ tube using Hewlett Packard 1100 Series Liquid Chromatograph and 4.6 mm ID x 15 cm length Column implemented. Gradient system using water (solvent A) and acetonitrile (solvent B): 20% to 90% B linear gradient, 10 minutes; 90% to 20% linear gradient, 2 minutes. The flow rate was 1 ml / min and UV detection was performed at 254 nm.</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">HPLC ^ * Varian ProStar Solvent Delivery Module (Varian Inc., Palo AltoCaliforniaUSA)Gynkotek UVD3 40 S  YMC ODS-A30 X 100 5 30/</span>Preparative HPLC was performed using the following instruments and conditions: Varian ProStar Solvent Delivery Module (Varian Inc., Palo Alto, California, USA). Detector: Gynkotek UVD3 40 S  Column: YMC ODS-A column (30 mm ID x 100 mm length, 5 micron particle size). Dissolution flow rate: 30 ml / min</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text"> A :B :20%B2 20%60%B1 860%B2 60% 90%B190%B390% 20%B2 UV210 LC/NMR40YMC ?&amp;amp;61003-(34.6 X 15 D20(A)-d3(B)30%B1 30%80%B111/  UV254 -82- (77) (77)200418849  Varian AS-600 NMR F19/H1NMR7.5 NMRNMR  / 1381  1 3 000 r*pm2 0. 1LC/NMR 0.9HPLC (0.2-0.4 A1 41 5 B1 6.51 7.5 HPLCBB (Rt 8.5ARt 7.3 AMS 3 NMRHPLC  </span>Dissolution gradient: (solvent A: water; solvent B: acetonitrile) 20% B, 2 minutes; 20% to 60% B linear gradient, 18 minutes; 60% B, 2 minutes; 60% to 90% B linear gradient , 1 minute; 90% B, 3 minutes; 90% to 20% B linear gradient, 2 minutes. Detection: UV, 210 nm LC / NMR is implemented as follows: 40 microliters of sample is injected into YMC &amp;amp; 0 Jean 61003-person (3 columns (4.6 mm inner diameter X 15 cm length) .D20 (solvent A) and acetonitrile-d3 (solvent B) gradient system: 30% B, 1 minute; 30% to 80% B linear gradient, 11 minutes, the column was eluted at a flow rate of 1 ml / min. The eluate was passed through a UV detection absorber (monitored at 254 -82- (77) (77) 200418849 nm) and then passed through a F19 / H1NMR detector in a Varian AS-600 NMR spectrometer. Biotransformation products It was dissolved at about 7.5 minutes and the flow was manually stopped to keep the lysate in the NMR probe for NMR data collection. Isolation and analysis were performed as follows. The butanol / methanol extraction solution (about 10%) was used. Ml) was evaporated to dryness under a stream of nitrogen. To the residue (38 mg) was added 1 ml of methanol and the insoluble matter was removed by centrifugation (13 000 r * pm, 2 minutes). 0.1 ml of the upper solution was used LC / NMR studies and preparative HPLC with the remaining 0.9 mL (0.2-0.4 mL per injection). Two main chromatographic peaks were observed and collected Peak A was eluted between 14 and 15 minutes, and Peak B was eluted between 1 6.5 and 1 7.5 minutes. Analytical HPLC analysis revealed the peak B micro-mother compound, epothilone B (Rt 8.5 minutes) and peak A is a biotransformation product (Rt 7.3 minutes). Peak A liquid was combined and MS analysis data was obtained with the combined liquid. The combined liquid was evaporated to a small volume, and then freeze-dried. 3 mg of white solid was obtained. NMR and HPLC analysis of the white solid (dissolved in methanol) revealed that the biotransformation product was partially decomposed during the drying process. [Schematic description]</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">11 9991221WO 00/3 92 760 9/468,8 5 4 SC15847 (PTA 1 043 )BF  2B -83- (78) (78)200418849 PCRSEQ ID NO:5SEQ ID NO:22 alignment) 3BSEQ ID NO:2) 1^?(?311&amp;gt;13(511^0:76)  43EryF B 5B  EryF(PDB1JIN ; 5 1 51 </span>Figure 1 filed WO 00/3 92 76, filed on December 21, 999, U.S. Application Serial No. 0 9/468, 8 54, as described in Streptomyces orientalis strain SC15847 (PTA 1 043) Schematic of the bioconversion of mycin B into epothilone F. Figure 2 shows the sequence alignment of SEQ ID NO: 5 to SEQ ID NO: 22 used to design a nuclear-83- (78) (78) 200418849 acid sequence encoding the epothilone B hydroxylase. (Alignment). Figure 3 shows epothilone B hydroxylase (SEQ ID NO: 2) and five 1 ^? (? 03 code 1) 1 &amp;gt; 1 chain of people; 3  (511 ^ 0: 76) Sequence alignment. The asterisk indicates the sequence identity, and the colon () indicates similar residues. Figure 4 proposes the homogeneity of epothilone B hydroxylase based on the sequence alignment of EryF shown in Figure 3 Model. Figure 5 shows rescaling plots of the epothilone B hydroxylase model (indicated by dashed lines) against EryF (PDB code 1JIN; indicated by dashes). An average window of 51 residues was used. The size, that is, the energy at a given residue position, is calculated by averaging the energy of 51 residues in a sequence centered at the given residue.</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">-84- (79) (79)200418849 5 APPENDIX 1</span>-84- (79) (79) 200418849 5 APPENDIX 1</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">Atom No. Residue Atom Name X-coord Y-coord Z-coord  1 ALA9 N -39.918 -4.913 -1.651  2 ALA9 CA -38.454 5.033 -1.537  3 ALA9 C -37.953 -4.886 -0.099  4 ALA9  -38.625 -4.31 0.765  5 ALA9 C -37.809 -3.967 -2.415  6 THR10 N -36.781 -5.447 0.146  7 THR10 CA -36.187 -5.437 1.49  8 THR10 C -34.916 -4.585 1.553  9 THR10  -34.016 -4.735 0.72  10 THR10 CB -35.871 -6.887 1.846  11 THR10 G1 -37.075 -7.631 1.717  12 THR10 CG2 -35.355 -7.053 3.271  13 LEU11 N -34.858 -3.699 2.536  14 LEU11 CA -33.669 -2.853 2.745  15 LEU11 C -32.511 -3.649 3.353  16 LEU11  -32.706 -4.468 4.259  17 LEU11 CB -34.033 -1.707 3.687  18 LEU11 CG -35.079 -0.78 3.078  19 LEU11 CD1 -35.53 0.265 4.091  20 LEU11 CD2 -34.555 -0.111 1.81  21 PR012 N -31.32 -3.422 2.823  22 PR012 CA -30.121 -4.119 3.302  23 PR012 C -29.652 -3.606 4.663  24 PR012  -29.656 -2.397 4.918  25 PR012 CB -29.081 -3:842 2.259  26 PR012 CG -29.597 -2.771 1.309  27 PR012 CD -31.031 2.493 1.729  28 LEU13 N -29.278 -4.522 5.54  29 LEU13 CA 28.676 -4.118 6.819  30 LEU13 C -27.183 -3.88 6.627  31 LEU13  -26.449 -4.806 6.267  32 LEU13 CB -28.898 5.196 7.872  33 LEU13 CG -30.374 -5.354 8.217  34 LEU13 CD1 -30.587 -6.516 9.181  35 LEU13 CD2 -30.945 -4.067 8.802  36 ALA14 N -26.72 -2.741 7.112  37 ALA14 CA 25.355 -2.266 6.825  38 ALA14 C -24.244 -2.941 7.634  39 ALA14  -23.058 -2.719 7.372  40 ALA14 CB -25.311 -0.764 7.075  41 ARG15 N -24.628 -3.792 8.569  42 ARG15 CA -23.664 -4.537 9.379  43 ARG15 C -23.478 -5.983 8.91  44 ARG15  -22.815 -6.767 9.599  45 ARG15 CB -24.174 -4.519 10.81  46 ARG15 CG -25.655 -4.879 10.84  47 ARG15 CD -26.2 -4.843 12.26  48 ARG15 NE -27.657 -5.039 12.256  49 ARG15 CZ -28.358 -5.301 13.36  50 ARG15 NH1 -29.69 -5.376 13.3  51 ARG15 NH2 -27.735 -5.412 14.536  52 LYS16 N -24.096 -6.351 7.798 85- (80) LYS16 CA -24.016 -7.741  LYS16 C -22.639 -8.128  LYS16  -21.959 -7.359  LYS16 C -25.061 -7.977  LYS16 CG -26.466 -7.985  LYS16 CD -26.605 -9.079  LYS16 CE -28.002 -9.092  LYS16 NZ -28.113 -10.128  CYS17 N -22.317 -9.392  CYS17 CA -21.061 -10.004  CYS17 C -20.737 -9.771  CYS17  -19.662 -9.205  CYS17 CB -21.096 -11.501  CYS17 SG -21.33 -11.937  PR018 N -21.635 -10.003  PR018 CA 21.293 -9.756  PR018 C -21.123 -8.291  PR018  -21.013 -8.061  PR018 CB -22.388 -10.383  PR018 CG -23.509 -10.812  PR018 CD 23.002 -10.554  PHE19 N -21.137 -7.33  PHE19 CA -20.792 -5.947  PHE19 C -19.279 -5.777  PHE19  -18.789 -4.92  PHE19 CB -21.36 -5.007  PHE19 CG -22.8 -4.568  PHE19 CD1 -23.051 -3.27  PHE19 CD2 -23.856 -5.444  PHE19 CE1 -24.355 -2.853  PHE19 CE2 -25.159 -5.03  PHE19 CZ &amp;gt;25.409 -3.735  SER20 N  8.573 -6.687  SER20 CA -17.102 -6.717  SER20 C -16.569 -7.839  SER20  -16.632 -7.723  SER20 CB -16.557 -5.371  SER20 G -17.236 -5.019  PR021 N -15.974 -8.867  PR021 CA -15.978 9.134  PR021 C -17.267 -9.836  PR021  -18.026 -10.327  PR021 CB -14.8 -10.047  PR021 CG -14.442 -10.669  PR021 CD -15.306 -9.949  PR022 N -17.551 9.859  PR022 CA -16.897 -9.007  PR022 C -17.4 -7.575  PR022  18.341 -7.371  PR022 C -17.32 -9.591  PR022 CG -18.478 -10.549  PR022 CD -18.669 -10.604  PR023 N -16.687 -6.605  PR023 CA -17.224 -5.241 -86- (81) PR023 C -18.525 -5.21  PR023  -18.524 -5.083  PR023 C &amp;gt;16.159 4.417  PR023 CG -15.004 -5.321  PR023 CD -15.388 -6.725  GLU24 N -19.62 -5.122  GLU24 CA -20.963 -5.192  GLU24 C -21.415 -3.843  GLU24  22.323 -3.794  GLU24 CB -21.934 -5.68  GLU24 CG -23.27 -6.137  GLU24 CD -23.982 -7.017  GLU24 E1 -24.613 -7.981  GLU24 E2 -23.833 -6.745  TYR25 N -20.573 -2.843  TYR25 CA -20.842 -1.47  TYR25 C -20.704 -1.311  TYR25  -21.364 -0.436  TYR25 CB -19.828 -0.568  TYR25 CG -19.616 -0.882  TYR25 CD1 20.662 -0.753  TYR25 CD2 -18.364 -1.298  TYR25 CE1 -20.461 -1.062  TYR25 CE2 -18.163 -1.605  TYR25 CZ -19.213 1.492  TYR25 OH *19.026 -1.859  GLU26 N -20.1 -2.296  GLU26 CA -20.009 -2.293  GLU26 C -21.404 -2.483  GLU26  -21.92 -1.572  GLU26 CB -19.129 -3.454  GLU26 CG -17.813 3.593  GLU26 CD -16.94 -2.342  GLU26 E1 -16.345 -2.12  GLU26 E2 -16.773 -1.731  ARG27 N -22.105 -3.488  ARG27 CA -23.437 -3.805  ARG27 C -24.504 -2.909  ARG27  -25.496 -2.591  ARG27 CB -23.752 -5.26  ARG27 CG -22.7 -6.189  ARG27 CD -23.031 -7.653  ARG27 NE -23.146 -7.926  ARG27 CZ -22.251 -8.648  ARG27 NH1 -21.16 -9.11  ARG27 NH2 -22.428 -8.879  LEU28 N -24.197 -2.331  LEU28 CA -25.11 -1.358  LEU28 C -25.131 -0.079  LEU28  -26.214 0.286  LEU28 CB -24.67 -1.039  LEU28 CG -24.868 -2.224  LEU28 CD1 -24.303 -1.916  LEU28 CD2 -26.34 -2.609 -87- (82) ARG29 N -23.969 0.307  ARG29 CA -23.835 1.502  ARG29 C -24.521 1.334  ARG29  -25.271 2.226  ARG29 C -22.345 1.682  ARG29 CG -21.997 2.947  ARG29 CD -20.519 2.941  ARG29 NE -19.696 2.563  ARG29 CZ -18.945 1.459  ARG29 NH1 -18.872 0.673  ARG29 NH2 -18.265 1.145  ARG30 N -24.494 0.109  ARG30 CA -25.112 -0.208  ARG30 C -26.629 -0.386  ARG30  -27.282 -0.429  ARG30 CB -24.503 -1.512  ARG30 CG -22.992 -1.401  ARG30 CD -22.376 -2.745  ARG30 NE -20.909 -2.659  ARG30 CZ -20.12 -3.648  ARG30 NH1 -20.658 -4.772  ARG30 NH2 -18.793 -3.508  GLU31 N -27.194 -0.493  GLU31 CA -28.653 -0.576  GLU31 C -29.207 0.713  GLU31  -30.393 1.032  GLU31 CB -29.025 -1.746  GLU31 CG -28.381 -3.055  GLU31 CD -28.814 -3.443  GLU31 E1 30.013 -3.448  GLU31 E2 -27.961 -3.944  SER32 N -28.319 1.439  SER32 CA -28.652 2.672  SER32 C -27.386 3.393  SER32  -26.706 2.984  SER32 CB -29.509 2.309  SER32 G -28.842 1.268  PR033 N -27.148 4.543  PR033 CA -26.039 5.408  PR033 C -26.227 5.972  PR033  -25.241 6.254  PR033 CB -26.023 6.511  PR33 CG -27.203 6.364  PR033 CD -27.933 5.107  VAL34 N -27.478 6.094  VAL34 CA -27.83 6.472  VAL34 C -28.828 5.447  VAL34  -30.01 5.467  VAL34 CB -28.483 7.85  VAL34 CG1 -28.789 8.339  VAL34 CG2 -27.616 8.865  SER35 N -28.344 4.546  SER35 CA -29.186 3.438  SER35 C -29.512 3.536 - 88- (83) SER35  -28.615 3.692  SER35 CB -28.456 2.126  SER35 G 27.19 2.169  ARG36 N -30.785 3.413  ARG36 CA -31.168 3.431  ARG36 C -30.894 2.072  ARG36  -31.516 1.059  ARG36 C -32.645 3.779  ARG36 CG -33.016 3.857  ARG36 CD -34.513 4.047  ARG36 NE -34.987 5.35  ARG36 CZ -36.272 5.582  ARG36 NH1 -37.16 4.59  ARG36 NH2 -36.662 6.791  VAL37 N -29.921 2.067  VAL37 CA -29.543 0.855  VAL37 C -29.982 0.926  VAL37  -30.313 1.995  VAL37 CB -28.03 0.681  VAL37 CG1 -27.591 0.391  VAL37 CG2 -27.298 1.898  GLY38 N -30.064 -0.24  GLY38 CA -30.404 -0.332  GLY38 C -29.151 -0.563  GLY38  -28.562 -1.652  LEU39 N -28.764 0.463  LEU39 CA -27.607 0.399  LEU39 C -27.911 -0.554  LEU39  -29.028 -1.085  LEU39 CB -27.353 1.814  LEU39 CG -26.198 2.546  LEU39 CD1 -26.368 2.665  LEU39 CD2 -26.011 3.925  PRO40 N -26.919 -0.869  PRO40 CA 27.183 -1.62  PRO40 C -28.423 -1.116  PRO40  -28.771 0.073  PRO40 C -25.933 -1.51  PRO40 CG -24.84 -0.886  PRO40 CD -25.497 -0.52  SER41 N -29.188 2.109  SER41 CA -30.511 -1.986  SER41 C -31.548 -1.213  SER41  -32.379 -0.492  SER41 C 30.387 -1.382  SER41 G -30.036 -0.008  GLY42 N -31.474 -1.34  GLY42 CA -32.521 -0.831  GLY42 C 32.557 0.686  GLY42  -33.59 1.208  GLN43 N -31.471 1.392  GLN43 CA -31.501 2.847  GLN43 C -31.201 3.16  GLN43  -30.079 2.955 89- (84) GLN43 C -30.507 3.53  GLN43 CG -30.681 5.05  GLN43 CD -29.873 5.699  GLN43 E1 -30.31 6.682  GLN43 NE2 -28.723 5.116  THR44 N -32.227 3.582  THR44 CA -32.096 3.832  THR44 C -31.194 5.02  THR44  -31.231 6.071  THR44 CB -33.475 4.077  THR44 G1 -34.009 5.268  THR44 CG2 -34.428 2.923  ALA45 N -30.35 4.799  ALA45 CA &amp;gt;29.426 5.833  ALA45 C -29.16 5.718  ALA45  -29.105 4.619  ALA45 C -28.115 5.705  TRP46 N -28.989 6.859  TRP46 CA -28.702 6.865  TRP46 C -27.212 6.698  TRP46  -26.408 7.589  TRP46 CB -29.185 8.173  TRP46 CG -30.693 8.309  TRP46 CD1 31.509 9.009  TRP46 CD2 -31.552 7.723  TRP46 NE1 -32.788 8.894  TRP46 CE2 -32.862 8.146  TRP46 CE3 &amp;gt;31.324 6.922  TRP46 CZ2 -33.913 7.774  TRP46 CZ3 -32.389 6.538  TRP46 CH2 -33.68 6.967  ALA47 N -26.863 5.559  ALA47 CA -25.475 5.257  ALA47 C -25.153 5.708  ALA47  -25.772 5.272  ALA47 CB -25.248 3.756  LEU48 N -24.185 6.602  LEU48 CA -23.751 7.129  LEU48 C -22.648 6.252  LEU48  -21.546 6.197  LEU48 CB -23.222 8.543  LEU48 CG -24.27 9.464  LEU48 CD1 -23.707 10.863  LEU48 CD2 -25.524 9.515  THR49 N -22.948 5.601  THR49 CA -22.01 4.636  THR49 C -21.197 5.214  THR49  -20.047 4.803  THR49 CB -22.774 3.391  THR49 G1 -23.783 3.769  THR49 CG2 -23.458 2.703  ARG50 N -21.724 6.2  ARG50 CA -20.899 6.838  ARG50 C -20.007 7.927 90- (85) ARG50  -20.456 8.712  ARG50 C -21.737 7.467  ARG50 CG -22.426 6.441  ARG50 CD -22.852 7.085  ARG50 NE -23.597 8.327  ARG50 CZ -23.779 9.27  ARG50 NH1 -24.462 10.375  ARG50 NH2 -23.274 9.111  LEU51 N -18.92 8.175  LEU51 CA -17.931 9.19  LEU51 C -18.52 10.584  LEU51  -18.42 11.426  LEU51 C -16.726 9.066  LEU51 CG -15.377 9.193  LEU51 CD1 -14.233 9.154  LEU51 CD2 -15.267 10.433  GLU52 N -19.404 10.68  GLU52 CA &amp;gt;20.088 11.93  GLU52 C -21.101 12.314  GLU52  -21.114 13.477  GLU52 CB -20.821 11.759  GLU52 CG -19.897 11.56  GLU52 CD -19.749 10.09  GLU52 E1 -19.796 9.24  GLU52 ' E2 -19.502 9.849  ASP53 N -21.659 11.313  ASP53 CA -22.646 11.572  ASP53 C -21.953 11.905  ASP53  -22.4 12.804  ASP53 CB -23.493 10.322  ASP53 CG -24.263 9.94  ASP53 D1 -24.319 8.749  ASP53 D2 -24.633 10.838  ILE54 N -20.75 11.382  ILE54 CA -19.991 11.62  ILE54 C -19.301 12.976  ILE54  &amp;gt;19.36 13.7  ILE54 CB -18.963 10.509  ILE54 CG1 -19.674 9.167  ILE54 CG2 -18.113 10.671  ILE54 CD1 -18.677 8.03  ARG55 N -18.916 13.43  ARG55 CA -18.346 14.776  ARG55 C -19.44 15.836  ARG55  -19.252 16.893  ARG55 CB -17.551 14.883  ARG55 CG -16.293 14.028  ARG55 CD -15.498 14.133  ARG55 NE -16.277 13.61  ARG55 CZ -15.712 13.028  ARG55 NH1 -14.383 12.947  ARG55 NH2 -16.475 12.553  GLU56 N -20.64 15.438  GLU56 CA -21.795 16.331 -91 - (86)200418849 377 GLU56 C -22.287 16.444 -1.539  378 GLU56  -22.628 17.546 -1.095  379 GLU56 C -22.875 15.722 -3.866  380 GLU56 CG -24.103 16.605 -4.028  381 GLU56 CD &amp;gt;25.112 15.838 -4.874  382 GLU56 E1 -25.906 16.463 -5.56  383 GLU56 OE2 -25.055 14.616 -4.834  384 MET57 N -22.065 15.392 -0.767  385 MET57 CA -22.379 15.386 0.665  386 MET57 C -21.4 16.241 1.459  387 MET57  -21.827 17.091 2.248  388 MET57 C -22.242 13.948 1.141  389 MET57 CG -22.423 13.805 2.646  390 MET 57 SD -21.979 12.184 3.306  391 MET57 CE -20.221 12.196 2.89  392 LEU58 N -20.14 16.197 1.056  393 LEU58 CA -19.089 16.973 1.726  394 LEU58 C -19.07 18.444 1.307  395 LEU58  -18.398 19.263 1.946  396 LEU58 CB -17.751 16.327 1.389  397 LEU58 CG -17.638 14.941 2.013  398 LEU58 CD1 -16.504 14.133 1.394  399 LEU58 CD2 -17.49 15.03 3.528  400 SER59 N -19.807 18.776 0.261  401 SER59 CA -19.959 20.171 -0.144  402 SER59 C -21.305 20.739 0.304  403 SER59  -21.531 21.951 0.204  404 SER59 CB -19.852 20.24 -1.661  405 SER59 G -18.59 19.697 -2.022  406 SER60 N -22.175 19.879 0.807  407 SER60 CA -23.5 20.318 1.246  408 SER60 C -23.505 20.806 2.685  409 SER60  -23.464 19.996 3.62  410 SER60 C -24.477 19.156 1.138  411 SER60 G -25.689 19.581 1.749  412 PR061 N 23.91 22.058 2.835  413 PR061 CA -24.023 22.695 4.154  414 PR061 C -25.231 22.233 4.983  415 PR061  -25.41 22.7 6.113  416 PR061 C -24.145 24.157 3.853  417 PR061 CG -24.401 24.343 2.364  418 PR061 CD -24.301 22.959 1.747  419 HIS62 N -26.044 21.333 4.451  420 HIS62 CA -27.21 20.856 5.18  421 HIS62 C -26.949 19.497 5.813  422 HIS62  -27.863 18.935 6.427  423 HIS62 CB -28.379 20.764 4.214  424 HIS62 CG -28.703 22.084 3.55  425 HIS62 ND1 -28.955 23.252 4.171  426 HIS62 CD2 -28.796 22.32 2.198  427 HIS62 CE1 -29.197 24.205 3.248  428 HIS62 NE2 -29.098 23.627 2.029  429 PHE63 N -25.765 18.945 5.596  430 PHE63 CA -25.385 17.693 6.258 92- (87) PHE63 C -24.492 17.977  PHE63  -23.261 17.885  PHE63 CB -24.686 16.783  PHE63 CG 25.651 16.13  PHE63 CD1 -26.92 15.76  PHE63 CD2 -25.265 15.901  PHE63 CE1 -27.804 15.161  PHE63 CE2 -26.147 15.298  PHE63 CZ -27.415 14.928  SER64 N -25.159 18.211  SER64 CA -24.502 18.656  SER64 C -23.765 17.525  SER64  -24.07 16.34  SER64 C -25.587 19.225  SER64 G -24.96 19.785  SER65 N -22.719 17.898  SER65 CA -22.006 16.938  SER65 C -22.463 17.032  SER65  -22.031 16.247  SER65 CB -20.522 17.234  SER65 G -20.167 17.174  ASP66 N -23.368 17.961  ASP66 CA -23.901 18.186  ASP66 C -25.388 18.496  ASP66  -25.978 18.026  ASP66 CB -23.149 19.393  ASP66 CG -22.904 19.311  ASP66 D1 -21.835 19.724  ASP66 OD2 -23.871 19.048  ARG67 N -25.972 19.246  ARG67 CA -27.32 19.831  ARG67 C -28.423 18.78  ARG67  -28.768 18.296  ARG67 CB 27.384 20.684  ARG67 CG -26.263 21.716  ARG67 CD -26.329 22.778  ARG67 NE -25.137 23.64  ARG67 CZ -25.091 24.799  ARG67 NH1 -26.189 25.28  ARG67 NH2 -23.957 25.503  GLN68 N 28.983 18.45  GLN68 CA -30.127 17.538  GLN68 C -31.414 18.348  GLN68  -31.728 19.187  GLN68 CB -30.116 16.757  GLN68 CG -31.207 15.692  GLN68 CD -31.109 14.852  GLN68 OE1 ' -31.941 14.973  GLN68 NE2 -30.137 13.955  SER69 N -32.129 18.102  SER69 CA -33.37 18.833  SER69 C -34.444 18.558  SER69  -34.447 17.495  SER69 C -33.885 18.387 93- (88)200418849 485 SER69 G -34.261 17.025 14.033  486 PRO70 N -35.332 19.526 16.499  487 PRO70 CA -36.438 19.385 17.447  488 PRO70 C -37.244 18.122 17.171  489 PRO70  -37.547 17.795 16.018  490 PRO70 CB -37.267 20.622 17.291  491 PRO70 CG -36.6 21.547 16.285  492 PRO70 CD -35.348 20.824 15.815  493 SER71 N -37.424 17.369 18.245  494 SER71 CA -38.115 16.065 18.289  495 SER71 C -37.589 15.02 17.298  496 SER71  -38.378 14.228 16.769  497 SER71 CB -39.625 16.244 18.111  498 SER71 G -39.919 16.638 16.777  499 PHE72 N -36.282 14.985 17.081  500 PHE72 CA -35.679 13.876 16.321  501 PHE72 C -34.364 13.43 16.957  502 PHE72  -33.281 13.768 16.458  503 PHE72 CB -35.428 14.283 14.872  504 PHE72 CG -36.682 14.456 14.018  505 PHE72 CD1 -37.097 15.724 13.63  506 PHE72 CD2 -37.402 13.339 13.617  507 PHE72 CE1 -38.238 15.875 12.853  508 . PHE72 CE2 -38.544 13.489 12.84  509 PHE72 CZ -38.962 14.758 12.459  510 PR073 N -34.469 12.59 17.979  511 PR073 CA -33.31 12.19 18.786  512 PR073 C -32.522 11.027 18.18  513 PR073  -32.606 9.895 18.668  514 PR073 CB -33.898 11.776 20.099  515 PR073 CG -35.392 11.555 19.917  516 PR073 CD -35.708 12.004 18.5  517 LEU74 N -31.772 11.304 17.127  518 LEU74 CA -30.933 10.263 16.521  519 LEU74 C -29.707 9.976 17.375  520 LEU74  -29.08 10.892 17.926  521 LEU74 CB -30.474 10.697 15.135  522 LEU74 CG -31.627 10J94 14.146  523 LEU74 CD1 -31.094 11.194 12.776  524 LEU74 CD2 -32.381 9.471 14.05  525 MET75 N -29.359 8.705 17.454  526 MET75 CA -28.167 8.306 18.208  527 MET75 C -27.099 7.808 17.243  528 MET75  -27.166 6.675 16.746  529 MET75 CB 28.539 7.208 19.198  530 MET75 CG -27.367 6.867 20.114  531 MET75 SD -27.678 5.549 21.31  532 MET75 CE -28.002 4.197 20.154  533 VAL76 N -26.117 8.657 16.992  534 VAL76 CA -25.071 8.327 16.017  535 VAL76 C -24.274 7.103 16.455  536 VAL76  -23.953 6.925 17.636  537 VAL76 CB -24.151 9.527 15.809  538 VAL76 CG1 -24.904 10.676 15.149 94- (89) VAL76 CG2 -23.504 9.986  ALA77 N -23.836 6.34  ALA77 CA -23.158 5.062  ALA77 C -21.703 5.203  ALA77  -21.033 4.194  ALA77 CB -23.22 4.212  ARG78 N -21.218 6.431  ARG78 CA -19.868 6.689  ARG78 C -19.868 7.178  ARG78  -18.816 7.163  ARG78 CB -19.274 7.772  ARG78 CG 19.445 7.436  ARG78 CD -19.068 8.629  ARG78 NE -19.848 9.81  ARG78 CZ -19.36 11.053  ARG78 NH1 -18.114 11.278  ARG78 NH2 -20.12 12.072  GLN79 N -21.028 7.577  GLN79 CA -21.128 8.129  GLN79 C -22.484 7.818  GLN79  -23.48 8.503  GLN79 CB -20.937 9.651  GLN79 CG -19.486 10.085  GLN79 CD -19.353 11.607  GLN79 E1 -19.071 12.193  GLN79 NE2 -19.508 12.226  ILE80 N -22.504 6.806  ILE80 CA -23.733 6.44  ILE80 C -23.732 7.034  ILE80  -22.666 7.316  ILE80 CB &amp;gt;23.847 4.919  ILE80 CG1 -22.684 4.305  ILE80 CG2 -23.905 4.35  ILE80 CD1 -22.794 2.788  ARG81 N -24.932 7.278  ARG81 CA -25.15 7.84  ARG81 C -24.657 9.276  ARG81  -23.493 9.571  ARG81 CB -24.51 6.964  ARG81 CG -25.437 5.843  ARG81 CD -25.685 5.92  ARG81 NE -26.269 7.22  ARG81 CZ -25.651 8.095  ARG81 NH1 -24.439 7.82  ARG81 NH2 -26.234 9.257  ARG82 N -25.448 10.076  ARG82 CA -25.192 11.523  ARG82 C -23.872 11.866  ARG82  -23.108 12.684  ARG82 C -26.32 12.122  ARG82 CG -27.683 11.796  ARG82 CD -28.801 12.301  ARG82 NE -28.71 11.659  ARG82 CZ -28.623 12.34 95- (90) ARG82 ARG82 GLU83 GLU83 GLU83 GLU83 GLU83 GLU83 GLU83 GLU83 GLU83 ASP84 ASP84 ASP84 ASP84 ASP84 ASP84 ASP84 ASP84 LYS85 LYS85 LYS85 LYS85 LYS85 LYS85 LYS85 LYS85 LYS85 PR086 PR086 PR086 PR086 PR086 PR086 PR086 PHE87 PHE87 PHE87 PHE87 PHE87 PHE87 PHE87 PHE87 PHE87 PHE87 PHE87 ARG88 ARG88 ARG88 ARG88 ARG88 ARG88 ARG88 ARG88</span>Atom No. Residue Atom Name X-coord Y-coord Z-coord 1 ALA9 N -39.918 -4.913 -1.651 2 ALA9 CA -38.454 5.033 -1.537 3 ALA9 C -37.953 -4.886 -0.099 4 ALA9 -38.625 -4.31 0.765 5 ALA9 C  -37.809 -3.967 -2.415 6 THR10 N -36.781 -5.447 0.146 7 THR10 CA -36.187 -5.437 1.49 8 THR10 C -34.916 -4.585 1.553 9 THR10 -34.016 -4.735 0.72 10 THR10 CB -35.871 -6.887 1.846 11 THR10 G1 -37.075 -7.631 1.717 12 THR10 CG2 -35.355 -7.053 3.271 13 LEU11 N -34.858 -3.699 2.536 14 LEU11 CA -33.669 -2.853 2.745 15 LEU11 C -32.511 -3.649 3.353 16 LEU11 -32.706 -4.468 4.259 17 LEU11 CB -34.033 -1.707 3.687 18 LEU11 CG -35.079 -0.78 3.078 19 LEU11 CD1 -35.53 0.265 4.091 20 LEU11 CD2 -34.555 -0.111 1.81 21 PR012 N -31.32 -3.422 2.823 22 PR012 CA -30.121 -4.119 3.302 23 PR012 C- 29.652 -3.606 4.663 24 PR012 -29.656 -2.397 4.918 25 PR012 CB -29.081 -3: 842 2.259 26 PR012 CG -29.597 -2.771 1.309 27 PR012 CD -31.031 '2.493 1.729 28 LEU13 N -29.278 -4.522 5.54 29 LEU13 CA  28.676 -4.118 6.819 30 LEU13 C -27.183 -3.88 6.627 31 LEU13 -26.449 -4.806 6.267 32 LEU13 CB -28.898  5.196 7.872 33 LEU13 CG -30.374 -5.354 8.217 34 LEU13 CD1 -30.587 -6.516 9.181 35 LEU13 CD2- 30.945 -4.067 8.802 36 ALA14 N -26.72 -2.741 7.112 37 ALA14 CA  25.355 -2.266 6.825 38 ALA14 C -24.244 -2.941 7.634 39 ALA14 -23.058 -2.719 7.372 40 ALA14 CB -25.311 -0.764 7.075 41 ARG15 N -24.628- 3.792 8.569 42 ARG15 CA -23.664 -4.537 9.379 43 ARG15 C -23.478 -5.983 8.91 44 ARG15 -22.815 -6.767 9.599 45 ARG15 CB -24.174 -4.519 10.81 46 ARG15 CG -25.655 -4.879 10.84 47 ARG15 CD -26.2 -4.843 12.26 48 ARG15 NE -27.657 -5.039 12.256 49 ARG15 CZ -28.358 -5.301 13.36 50 ARG15 NH1 -29.69 -5.376 13.3 51 ARG15 NH2 -27.735 -5.412 14.536 52 LYS16 N -24.096 -6.351 7.798 85- (80) LYS16 CA -24.016- 7.741 LYS16 C -22.639 -8.128 LYS16 -21.959 -7.359 LYS16 C  -25.061 -7.977 LYS16 CG -26.466 -7.985 LYS16 CD -26.605 -9.079 LYS16 CE -28.002 -9.092 LYS16 NZ -28.113 -10.128 CYS17 N -22.317 -9.392 CYS17 CA -21.061 -10.004 CYS17 C -20.737 -9.771 CYS17 -19.662 -9.205 CYS17 CB -21.096 -11.501 CYS17 SG -21.33 -11.937 PR018 N -21.635 -10.003 CA018 21.293 -9.756 PR018 C -21.123 -8.291 PR018 -21.013 -8.061 PR018 CB -22.388 -10.383 PR018 CG -23.509 -10.812 PR018 CD  23.002 -10.554 PHE19 N -21.137 -7.33 PHE19 CA -20.792 -5.947 PHE19 C -19.279- 5.777 PHE19 -18.789 -4.92 PHE19 CB -21.36 -5.007 PHE19 CG -22.8 -4.568 PHE19 CD1 -23.051 -3.27 PHE19 CD2 -23.856 -5.444 PHE19 CE1 -24.355 -2.853 PHE19 CE2 -25.159 -5.03 PHE19 CZ &amp;gt; 25.409 -3.735 SER20 N  8.573 -6.687 SER20 CA -17.102 -6.717 SER20 C -16.569 -7.839 SER20 -16.632 -7.723 SER20 CB -16.557 -5.371 SER20 G -17.236 -5.019 PR021 N -15.974 -8.867 PR021 CA -15.978  9.134 PR021 C -17.267 -9.836 PR021 -18.026 -10.327 PR021 CB -14.8 -10.047 PR021 CG -14.442 -10.669 PR021 CD -15.306 -9.949 PR022 N -17.551  9.859 PR022 CA -16.897- 9.007 PR022 C -17.4 -7.575 PR022  18.341 -7.371 PR022 C  -17.32 -9.591 PR022 CG -18.478 -10.549 PR022 CD -18.669 -10.604 PR023 N -16.687 -6.605 PR023 CA -17.224 -5.241 -86- (81) PR023 C -18.525 -5.21 PR023 -18.524 -5.083 PR023 C  &amp;gt; 16.159  4.417 PR023 CG -15.004 -5.321 PR023 CD -15.388 -6.725 GLU24 N -19.62 -5.122 GLU24 CA -20.963 -5.192 GLU24 C -21.415 -3.843 GLU24 22.323 -3.794 GLU24 CB -21.934 -5.68 GLU24 CG -23.27 -6.137 GLU24 CD -23.982 -7.017 GLU24 E1 -24.613 -7.981 GLU24 E2 -23.833 -6.745 TYR25 N -20.573 -2.843 TYR25 CA -20.842 -1.4 TYR25 C -20.704 -1.311 TYR25 -21.364 -0.436 TYR25 CB -19.828 -0.568 TYR25 CG -19.616 -0.882 TYR25 CD1  20.662 -0.753 TYR25 CD2 -18.364 -1.298 TYR25 CE1 -20.461 -1.062 TYR25 CEZ -18 -19.213  1.492 TYR25 OH * 19.026 -1.859 GLU26 N -20.1 -2.296 GLU26 CA -20.009 -2.293 GLU26 C -21.404 -2.483 GLU26 -21.92 -1.572 GLU26 CB -19.129 -3.454 GLU26 CG -17.813  3.593 GLU26 CD -16.94 -2.342 GLU26 E1 -16.345 -2.12 GLU26 E2 -16.773 -1.731 ARG27 N -22.105 -3.488 ARG27 CA -23.437 -3.805 ARG27 C -24.504 -2.909 ARG27 -25.496 -2.591 ARG27 CB -23.752- 5.26 ARG27 CG -22.7 -6.189 ARG27 CD -23.031 -7.653 ARG27 NE -23.146 -7.926 ARG27 CZ -22.251 -8.648 ARG27 NH1 -21.16 -9.11 ARG27 NH2 -22.428 -8.879 LEU28 N -24.197 -2.331 LEU28 CA -25.11 -1.358 LEU28 C -25.131 -0.079 LEU28 -26.214 0.286 LEU28 CB -24.67 -1.039 LEU28 CG -24.868 -2.224 LEU28 CD1 -24.303 -1.916 LEU28 CD2 -26.34 -2.609 -87- (82) ARG29 N -23.969 0.307 ARG29 CA -23.835 1.502 ARG29 C -24.521 1.334 ARG29 -25.271 2.226 ARG29 C  -22.345 1.682 ARG29 CG -21.997 2.947 ARG29 CD -20.519 2.941 ARG29 NE -19.696 2.563 ARG29 CZ -18.945 1.459 ARG29 NH1 -18.872 0.673 ARG29 NH2 -18.265 1.145 A145 0.109 ARG30 CA -25.112 -0.208 ARG30 C -26.629 -0.386 ARG30 -27.282 -0.429 ARG30 CB -24.503 -1.512 ARG30 CG -22.992 -1.401 ARG30 CD -22.376 -2.74 5 ARG30 NE -20.909 -2.659 ARG30 CZ -20.12 -3.648 ARG30 NH1 -20.658 -4.772 ARG30 NH2 -18.793 -3.508 GLU31 N -27.194 -0.493 GLU31 CA -28.653 -0.576 GLU31 C -29.207 0.713 GLU31 -30.393 1.032 GLU31 CB 29.025 -1.746 GLU31 CG -28.381 -3.055 GLU31 CD -28.814 -3.443 GLU31 E1  30.013 -3.448 GLU31 E2 -27.961 -3.944 SER32 N -28.319 1.439 SER32 CA -28.652 2.672 SER32 C -27.386 3.393 SER32 -26.706 2.706 CB -29.509 2.309 SER32 G -28.842 1.268 PR033 N -27.148 4.543 PR033 CA -26.039 5.408 PR033 C -26.227 5.972 PR033 -25.241 6.254 PR033 CB -26.023 6.511 PR33 CG -27.203 6.364 PR033 CD -27.933 5.107 VAL34 N- 27.478 6.094 VAL34 CA -27.83 6.472 VAL34 C -28.828 5.447 VAL34 -30.01 5.467 VAL34 CB -28.483 7.85 VAL34 CG1 -28.789 8.339 VAL34 CG2 -27.616 8.865 SER35 N -28.344 4.546 SER35 CA -29.186 3.438 SER35 C-29-512. (83) SER35 -28.615 3.692 SER35 CB -28.456 2.126 SER35 G  27.19 2.169 ARG36 N -30.785 3.413 ARG36 C A -31.168 3.431 ARG36 C -30.894 2.072 ARG36 -31.516 1.059 ARG36 C  -32.645 3.779 ARG36 CG -33.016 3.857 ARG36 CD -34.513 4.047 ARG36 NE -34.987 5.35 ARG36 CZ -36.272 5.582 ARG36 NH1 -37.16 4.59 ARG36 NH 6.NH VAL37 N -29.921 2.067 VAL37 CA -29.543 0.855 VAL37 C -29.982 0.926 VAL37 -30.313 1.995 VAL37 CB -28.03 0.681 VAL37 CG1 -27.591 0.391 VAL37 CG2 -27.298 1.898 GLY38 N -30.064 -0.24 GLY38 CA -30.404 -32 29.151 -0.563 GLY38 -28.562 -1.652 LEU39 N -28.764 0.463 LEU39 CA -27.607 0.399 LEU39 C -27.911 -0.554 LEU39 -29.028 -1.085 LEU39 CB -27.353 1.814 LEU39 CG -26.198 2.546 LEU39 CD1 -26.368 2.665 LEU39 CD2 3.925 PRO40 N -26.919 -0.869 PRO40 CA  27.183 -1.62 PRO40 C -28.423 -1.116 PRO40 -28.771 0.073 PRO40 C  -25.933 -1.51 PRO40 CG -24.84 -0.886 PRO40 CD -25.497 -0.52 SER41 N -29.188  2.109 SER41 CA -30.511 -1.986 SER41 C -31.548 -1.213 SER41 -32.379 -0.492 SER41 C  30.387 -1.382 SE R41 G -30.036 -0.008 GLY42 N -31.474 -1.34 GLY42 CA -32.521 -0.831 GLY42 C  32.557 0.686 GLY42 -33.59 1.208 GLN43 N -31.471 1.392 GLN43 CA -31.501 2.847 GLN43 C -31.201 3.16 GLN43 -30.0 -(84) GLN43 C  -30.507 3.53 GLN43 CG -30.681 5.05 GLN43 CD -29.873 5.699 GLN43 E1 -30.31 6.682 GLN43 NE2 -28.723 5.116 THR44 N -32.227 3.582 THR44 CA -32.096 3.832 THR44 C -31.194 5.02 THR44 6.071 THR44 CB -33.475 4.077 THR44 G1 -34.009 5.268 THR44 CG2 -34.428 2.923 ALA45 N -30.35 4.799 ALA45 CA &amp;gt; 29.426 5.833 ALA45 C -29.16 5.718 ALA45 -29.105 4.619 ALA45 C  -28.115 5.705 TRP46 N -28.46. CA -28.702 6.865 TRP46 C -27.212 6.698 TRP46 -26.408 7.589 TRP46 CB -29.185 8.173 TRP46 CG -30.693 8.309 TRP46 CD1  31.509 9.009 TRP46 CD2 -31.552 7.723 TRP46 NE1 -32.788 8.894 TRP46 CE2 -32.862 8.922; &amp;gt; TRP46 CE TRP46 CZ2 -33.913 7.774 TRP46 CZ3 -32.389 6.538 TRP46 CH2 -33.68 6.967 ALA47 N -26.86 3 5.559 ALA47 CA -25.475 5.257 ALA47 C -25.153 5.708 ALA47 -25.772 5.272 ALA47 CB -25.248 3.756 LEU48 N -24.185 6.602 LEU48 CA -23.751 7.129 LEU48 C -22.648 6.252 LEU48 -21.546 6.197 LEU48 CB -23.222 -543 LEU 24.27 9.464 LEU48 CD1 -23.707 10.863 LEU48 CD2 -25.524 9.515 THR49 N -22.948 5.601 THR49 CA -22.01 4.636 THR49 C -21.197 5.214 THR49 -20.047 4.803 THR49 CB -22.774 3.391 THR49 G1 -23.783 3.769 THR49 458 A2 CG2 -21.724 6.2 ARG50 CA -20.899 6.838 ARG50 C -20.007 7.927 90- (85) ARG50 -20.456 8.712 ARG50 C  -21.737 7.467 ARG50 CG -22.426 6.441 ARG50 CD -22.852 7.085 ARG50 NE -23.597 8.327 ARG50 CZ -23.779 9.27 ARG50 NH1 -24.462 10.375 ARG50 NH2 -23.274 9.111 LEU51 N -18.92 8.175 LEU51 CA -17.931 9.19 LEU51 C -18.52 10.584 LEU51 -18.42 11.426 LEU51 C  -16.726 9.066 LEU51 CG -15.377 9.193 LEU51 CD1 -14.233 9.154 LEU51 10.15 GLU52 N -19.404 10.68 GLU52 CA &amp;gt; 20.088 11.93 GLU52 C -21.101 12.314 GLU52 -21.114 13.477 GLU52 CB -20.821 11.759 GLU52 CG -19.897 11.56 GLU52 CD -19.749 10.09 GLU52 E1 -19.796 9.24 GLU52 'E2 -19.502 9.849 ASP53 N -21.659 11.313 ASP53 CA -22.646 11.572 ASP53 C -21.953 11.905 -22.4 12.804 ASP53 CB -23.493 10.322 ASP53 CG -24.263 9.94 ASP53 D1 -24.319 8.749 ASP53 D2 -24.633 10.838 ILE54 N -20.75 11.382 ILE54 CA -19.991 11.62 ILE54 C -19.301 12.976 ILE54  &amp;gt; 19.36 13.7 ILE54 CB 18.963 10.509 ILE54 CG1 -19.674 9.167 ILE54 CG2 -18.113 10.671 ILE54 CD1 -18.677 8.03 ARG55 N -18.916 13.43 ARG55 CA -18.346 14.776 ARG55 C -19.44 15.836 ARG55 -19.252 16.893 ARG55 CB -17.551 14.883 ARG55 14. CG RG-16 15.498 14.133 ARG55 NE -16.277 13.61 ARG55 CZ -15.712 13.028 ARG55 NH1 -14.383 12.947 ARG55 NH2 -16.475 12.553 GLU56 N -20.64 15.438 GLU56 CA -21.795 16.331 -91-(86) 200418849 377 GLU56 C -22.287 16.444 -1.539 378 GLU56 -22.628 17.546 -1.095 379 GLU56 C  -22.875 15.72 2 -3.866 380 GLU56 CG -24.103 16.605 -4.028 381 GLU56 CD &amp;gt; 25.112 15.838 -4.874 382 GLU56 E1 -25.906 16.463 -5.56 383 GLU56 OE2 -25.055 14.616 -4.834 384 MET57 N -22.065 15.392 -0.767 385 MET57 CA -22.379 15.386 0.665 386 MET57 C -21.4 16.241 1.459 387 MET57 -21.827 17.091 2.248 388 MET57 C  -22.242 13.948 1.141 389 MET57 CG -22.423 13.805 2.646 390 MET 57 SD -21.979 12.184 3.306 391 MET57 CE -20.221 12.196 2.89 392 LEU58 N- 20.14 16.197 1.056 393 LEU58 CA -19.089 16.973 1.726 394 LEU58 C -19.07 18.444 1.307 395 LEU58 -18.398 19.263 1.946 396 LEU58 CB -17.751 16.327 1.389 397 LEU58 CG -17.638 14.941 2.013 398 LEU58 CD1 -16.504 14.133 1.394 399. LEU58 15.03 3.528 400 SER59 N -19.807 18.776 0.261 401 SER59 CA -19.959 20.171 -0.144 402 SER59 C -21.305 20.739 0.304 403 SER59 -21.531 21.951 0.204 404 SER59 CB -19.852 20.24 -1.661 405 SER59 G -18.59 19.697 -2.022 406 SER60 N -22.175 19.879 0.807 407 SER60 CA -23.5 20.318 1.246 408 SER60 C -23.505 20.806 2.685 409 SER60 -23.464 19.996 3.62 410 SER60 C  -24.477 19.156 1.138 411 SER60 G -25.689 19.581 1.749 412 PR061 N  23.91 22.058 2.835 413 PR061 CA -24.023 22.695 4.154 414 PR061 C -25.231 22.233 4.983 415 PR061 -25.41 22.7 6.113 416 PR061 C  -24.145 24.157 3.853 417 PR061 CG -24.401 24.343 2.364 418 PR061 CD -24.301 22.959 1.747 419 HIS62 N -26.044 21.333 4.451 420 HIS62 CA -27.21 20.856 5.18 421 HIS62 C -26.949 19.497813 422 HIS62 -27.863 18.935 6.427 423 HIS62 CB -28.379 20.764 4.214 424 HIS62 CG -28.703 22.084 3.55 425 HIS62 ND1 -28.955 23.252 4.171 426 HIS62 CD2 -28.796 22.32 2.198 427 HIS62 CE1 -29.197 24.205 3.248 428.29 429 HIS62 NE2 PHE63 N -25.765 18.945 5.596 430 PHE63 CA -25.385 17.693 6.258 92- (87) PHE63 C -24.492 17.977 PHE63 -23.261 17.885 PHE63 CB -24.686 16.783 PHE63 CG  25.651 16.13 PHE63 CD1 -26.92 15.76 PHE63 CD2 -25.265 15.901 PHE -27.804 15.161 PHE63 CE2 -26. 147 15.298 PHE63 CZ -27.415 14.928 SER64 N -25.159 18.211 SER64 CA -24.502 18.656 SER64 C -23.765 17.525 SER64 -24.07 16.34 SER64 C  -25.587 19.225 SER64 G -24.96 19.785 SER65 N -22.719 17.898 SER65 CA -2265.006 16. C -22.463 17.032 SER65 -22.031 16.247 SER65 CB -20.522 17.234 SER65 G -20.167 17.174 ASP66 N -23.368 17.961 ASP66 CA -23.901 18.186 ASP66 C -25.388 18.496 ASP66 -25.978 18.026 ASP66 CB -23.149 19.393 ASP66 CG -22.904 19.311 ASP66 D1 -21.835 19.724 ASP66 OD2 -23.871 19.048 ARG67 N -25.972 19.246 ARG67 CA -27.32 19.831 ARG67 C -28.423 18.78 ARG67 -28.768 18.296 ARG67 CB  27.384 20.684 ARG67 CG -26.263 21.716 ARG67 CD -26.329 22.778 ARG67 23.64 ARG67 CZ -25.091 24.799 ARG67 NH1 -26.189 25.28 ARG67 NH2 -23.957 25.503 GLN68 N  28.983 18.45 GLN68 CA -30.127 17.538 GLN68 C -31.414 18.348 GLN68 -31.728 19.187 GLN68 CB -30.68 116.67 16.757 16.719 14.852 GLN68 OE1 '-31.941 14.973 GLN68 NE2 -30.137 13.955 SER69 N -32.129 18.102 SER69 CA -33.37 18.833 SER69 C -34.444 18.558 SER69 -34.447 17.495 SER69 C  -33.885 18.387 93- (88) 200418849 485 SER69 G -34.261 17.025 14.033 486 PRO70 N -35.332 19.526 16.499 487 PRO70 CA -36.438 19.385 17.447 488 PRO70 C -37.244 18.122 17.171 489 PRO70 -37.547 17.795 16.018 490 PRO70 CB -37.267 20.622 17.291 491 PRO70 CG -36.6 21.547 16.285 492 PRO70 CD -35.348 20.824 15.815 493 SER71 N -37.424 17.369 18.245 494 SER71 CA -38.115 16.065 18.289 495 SER71 C -37.589 15.02 17.298 496 SER71 -38.378 14.228 16.769 497 SER71 CB -39.625 16.244 18.111 498 SER71 G -39.919 16.638 16.777 499 PHE72 N -36.282 14.985081 CA -35.679 13.876 16.321 501 PHE72 C -34.364 13.43 16.957 502 PHE72 -33.281 13.768 16.458 503 PHE72 CB -35.428 14.283 14.872 504 PHE72 CG -36.682 14.456 14.018 505 PHE72 CD1 -37.097 15.724 13.63 506 PHE72 CD2 -37.402 13.617 13.339 13. -38.238 15 .875 12.853 508. PHE72 CE2 -38.544 13.489 12.84 509 PHE72 CZ -38.962 14.758 12.459 510 PR073 N -34.469 12.59 17.979 511 PR073 CA -33.31 12.19 18.786 512 PR073 C -32.522 11.027 18.18 513 PR073 -32.606 9.895 18.668 514 PR073 CB- 33.898 11.776 20.099 515 PR073 CG -35.392 11.555 19.917 516 PR073 CD -35.708 12.004 18.5 517 LEU74 N -31.772 11.304 17.127 518 LEU74 CA -30.933 10.263 16.263 519 LEU74 C -29.707 9.976 17.375 520 LEU74 -29.08 10.892 17.926 521 LEU74 C 10.697 15.135 522 LEU74 CG -31.627 10J94 14.146 523 LEU74 CD1 -31.094 11.194 12.776 524 LEU74 CD2 -32.381 9.471 14.05 525 MET75 N -29.359 8.705 17.454 526 MET75 CA -28.167 8.306 18.208 527 MET75 C -27.099 7.808 17.243 528 MET75 16.746 529 MET75 CB  28.539 7.208 19.198 530 MET75 CG -27.367 6.867 20.114 531 MET75 SD -27.678 5.549 21.31 532 MET75 CE -28.002 4.197 20.154 533 VAL76 N -26.117 8.657 16.992 534 VAL76 CA -25.071 8.327 16.017 535 VAL76 C -24. 53 6 VAL76 -23.953 6.925 17.636 537 VAL76 CB -24.151 9.527 15.809 538 VAL76 CG1 -24.904 10.676 15.149 94- (89) VAL76 CG2 -23.504 9.986 ALA77 N -23.836 6.34 ALA77 CA -23.158 5.062 ALA77 C -21.703 5.203 ALA77 -21. 4.194 ALA77 CB -23.22 4.212 ARG78 N -21.218 6.431 ARG78 CA -19.868 6.689 ARG78 C -19.868 7.178 ARG78 -18.816 7.163 ARG78 CB -19.274 7.772 ARG78 CG  19.445 7.436 ARG78 CD -19.068 8.629 ARG78 NE -19.848 9.81 ARG78 36. 11.053 ARG78 NH1 -18.114 11.278 ARG78 NH2 -20.12 12.072 GLN79 N -21.028 7.577 GLN79 CA -21.128 8.129 GLN79 C -22.484 7.818 GLN79 -23.48 8.503 GLN79 CB -20.937 9.651 GLN79 CG -19.486 10.085 GLN79 CDR7979 GLN79 CD 19.071 12.193 GLN79 NE2 -19.508 12.226 ILE80 N -22.504 6.806 ILE80 CA -23.733 6.44 ILE80 C -23.732 7.034 ILE80 -22.666 7.316 ILE80 CB &amp;gt; 23.847 4.919 ILE80 CG1 -22.684 4.305 ILE80 CG2 -23.905 4.35 ILE80 CD 81. -24.932 7.278 ARG81 CA -25.15 7.84 ARG81 C -24.657 9.276 ARG81 -23.493 9.571 ARG81 CB -24.51 6.964 ARG81 CG -25.437 5.843 ARG81 CD -25.685 5.92 ARG81 NE -26.269 7.22 ARG81 CZ -25.651 8.095 ARG81 NH1 -24.439 7.82 ARG81 NH2 -26.234 9.257 ARG82 N -25.448 10.25 11.523 ARG82 C -23.872 11.866 ARG82 -23.108 12.684 ARG82 C  -26.32 12.122 ARG82 CG -27.683 11.796 ARG82 CD -28.801 12.301 ARG82 NE -28.71 11.659 ARG82 CZ -28.623 12.34 95- (90) ARG82 ARG82 GLU83 GLU83 GLU83 GLU83 GLU83 GLU83 GLU83 GLU83 GLU83 GLU83 GLU83 GLU83 ASP84 ASP84 ASP84 ASP84 ASP84 ASP84 ASP84 ASP84 LYS85 LYS85 LYS85 LYS85 LYS85 LYS85 LYS85 LYS85 LYS85 PR086 PR086 PR086 PR086 PR086 PR88 ARG 88 ARG A87 PHE87 PHE87 PHE87 PHE87 PHE87 PHE87 PHE87 PHE87 PHE87 PHE87 PHE87 PHE87 PHE87</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">H1H2A BGDE1E2A BGDD2A B G D E z A B G D A BGD1D2E1E2Z A B G D E NNNCCCCCNCCCCNCCCCCCNNCCCCCNCCCCCCCCCNCCCCCN -28.477 11.689  -28.606 13.675  -23.495 11.077  -22.237 11.334  -21.03 10.69  -19.894 10.902  -22.361 10.828  23.385 11.651  -23.478 11.172  -23.428 9.967  -23.71 12.011  -21.274 9.941  -20.201 9.327  -20.095 10.012  -19.257 9.646  -20.481 7.841  -20.585 7.191  -19.547 6.906  -21.704 7.048  -20.939 11.017  -20.846 11.928  -19.997 13.228  -20.236 14.084  -22.27 12.347  -23.107 11.173  -24.573 11.567  -25.408 10.408  -26.824 10.785  -19.054 13.455  -18.316 14.731  -17.168 14.8  &amp;gt;16.339 15.713  -17.779 14.922  -17.866 13.6  -18.543 12.646  -17.14 13.906  -16.12 13.963  -16.67 14.817  15.963 15.16  -15.848 12.556  15.724 11.449  -16.447 10.277  -14.904 11.591  16.358 9.254  -14.817 10.567  -15.544 9.399  -17.948 15.143  -18.629 16.037  -18.178 17.519  -18.118 18.064  -20.122 15.915  &amp;gt;20.964 16.678  -22.429 16.294  - 22.593 14.851 96- (91) ARG88 ARG88 ARG88 PR089 PR089 PR089 PR089 PR089 PR089 PR089 SER90 SER90 SER90 SER90 SER90 SER90 LEU91 LEU91 LEU91 LEU91 LEU91 LEU91 LEU91 LEU91 ILE92 ILE92 ILE92 ILE92 ILE92 ILE92 ILE92 ILE92 ALA93 ALA93 ALA93 ALA93 ALA93 MET94 MET94 MET94 MET94 MET94 MET94 MET94 MET94 ASP95 ASP95 ASP95 ASP95 ASP95 ASP95 ASP95 ASP95 PR096</span>H1H2A BGDE1E2A BGDD2A BGDE z ABGDA BGD1D2E1E2Z ABGDE NNNCCCCCCNCCCCNCCCCNNCCCCCNCCCCCCCCCNCCCCCN -28.477 11.689 -28.606 13.675 -23.495 11.077 -22.237 10.385 10.385-19.385 19.385-22.385 11.651 -23.478 11.172 -23.428 9.967 -23.71 12.011 -21.274 9.941 -20.201 9.327 -20.095 10.012 -19.257 9.646 -20.481 7.841 -20.585 7.191 -19.547 6.906 -21.704 7.048 -20.939 11.017 -20.846 11.928 -19.997 13.228 -20.236 14.084 -22.27 12.347- 23.107 11.173 -24.573 11.567 -25.408 10.408 -26.824 10.785 -19.054 13.455 -18.316 14.731 -17.168 14.8 &amp;gt; 16.339 15.713 -17.779 14.922 -17.866 13.6 -18.543 12.646 -17.14 13.906 -16.12 13.963 -16.67 14.817  15.963 15.16 -15.848 12.556 11.449 -16.447 10.277 -14.904 11.591  16.358 9.254 -14.817 10.567 -15.544 9.399 -17.948 15.143 -18.629 16.037 -18.178 17.519 -18.118 18.064 -20.122 15.915 &amp;gt; 20.964 16.678 -22.429 16.294-22.593 14.851 96- ( 91) ARG88 ARG88 ARG88 PR089 PR089 PR089 PR089 PR089 PR089 PR089 SER90 SER90 SER90 SER90 SER90 SER90 LEU91 LEU91 LEU91 LEU91 LEU91 LEU91 LEU91 LEU91 ILE92 ILE92 ILE92 ILE92 ASP94 MET94 MET92 ALA93 MET94 AMET ASP95 ASP95 ASP95 ASP95 ASP95 PR096</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">1 2 12 12 1 1 2   H A BGD A B G A B G D D A BGGD A B A BGDE A  GDD CNNNCCCCCNCCCNCCCCCCNCCCCCCNCCCNCCCCSCNCCCCN -23.307 14.07  -23.373 12.757  -23.922 14.598  -17.919 18.194  -17.186 19.48  -15.737 19.333  -14.786 19.454  -17.206 20.033  -17.798 19.004  -18.208 17.832  -15.606 19.29  -14.334 19.04  -14.43 19.413  -15.534 19.592  -14.069 17.55  -15.095 16.928  -13.326 19.173  -13.154 19.564  -14.042 18.804  -14.491 19.39  -11.702 19.238  -11.325 19.637  -11.253 21.153  -9.989 19.012  -14.422 17.582  -15.199 16.791  -16.712 16.857  -17.487 16.497  -14.682 15.351  15.288 14.488  -14.932 14.719  -14.775 13.055  -17.14 17.407  -18.578 17.414  -19.15 18.822  -20.335 19.048  -18.865 16.593  -18.294 19.776  -18.739 21.168  -18.99 21.683  -18.221 21.436  -17.695 22.042  -17.822 21.982  -16.686 23.058  -17.561 23.095  -20.089 22.398  -20.42 23.024  -19.831 24.427  -19.589 25.001  -21.938 23.082  -22.52 21.677  -22.276 20.87  -23.173 21.412  -19.488 24.936 97- (92) PR096 PR096 PR096 PR096 PR096 PR096 PR097 PR097 PR097 PR097 PR097 PR097 P  097 GLU98 GLU98 GLU98 GLU98 GLU98 GLU98 GLU98 GLU98 GLU98 HIS99 HIS99 HIS99 HIS99 HIS99 HIS99 HIS99 HIS99 HIS99 HIS99 GLY100 GLY100 GLY100 GLY100 LYS101 LYS101 LYS101 LYS101 LYS101 LYS101 LYS101 LYS101 LYS101 ALA102 ALA102 ALA102 ALA102 ALA102 ARG103 ARG103 ARG103 ARG103 A BGD A BGD A B G D E1E2 A BGD1D2E1E2A A BGDEZ A B A CCCCCNCCCCCNCCCCCNCCCCNCCNNCCNCCCCCCNNCCCNCC -19.076 26.343  -20.177 27.263  -21.353 26.892  -18.812 26.567  -19.078 25.278  -19.532 24.256  -19.817 28.38  -18.428 28.85  -17.649 28.271  -16.462 28.595  -18.567 30.329  -20.013 30.649  20.777 29.345  -18.233 27.353  -17.555 26.761  -16.353 25.946  -15.224 26.174  -18.513 25.782  -19.894 26.372  -20.857 25.236  -20.697 24.171  -21.799 25.49  -16.575 25.223  -15.52 24.448  14.43 25.323  -13.249 24.993  -16.19 23.646  -15.238 23.09  -14.522 21.957  -14.946 23.649  -13.779 21.805  -14.042 22.852  -14.792 26.524  -13.832 27.439  -12.859 27.944  -11.648 27.716  -13.419 28.385  -12.626 28.895  -11.711 27.823  -10.485 28.02  -13.608 29.33  -12.893 29.892  -13.829 29.939  -14.189 28.531  -12.986 27.796  -12.251 26.624  -11.474 25.523  -10.381 25.022  -9.243 24.859  -12.425 24.379  -10.622 25.091  -9.63 24.64  -8.492 25.644  -7.325 25.236 98- (93) 200418849</span>1 2 12 12 1 1 2   HA BGD ABGABGDDA BGGD ABA BGDE A  GDD CNNNCCCCCNCCCNCOCCCCCNCCCCCCNCCCNCCCCSCNCCCCN -23.307 14.07 -23.373 12.757 -23.922 14.598 -17.919 18.194 -17.186 19.48 -15.737 19.333 -14.786 19.454 -17.206 20.033 -17.798 19.004 -18.208 17.832 -15.606 19.29 -14.334 19.04 -14.43 19.413 -15.534 19.592 -14.069 17.55 -15.095 16.928 -13.326 19.173 -13.154 19.564 -14.042 18.804 -14.491 19.39 19.238 -11.325 19.637 -11.253 21.153 -9.989 19.012 -14.422 17.582 -15.199 16.791 -16.712 16.857 -17.487 16.497 -14.682 15.351  15.288 14.488 -14.932 14.719 -14.775 13.055 -17.14 17.407 -18.578 17.414 -19.15 18.822 -20.335 19.048 -18.865 16.59 16.59 18.294 19.776 -18.739 21.168 -18.99 21.683 -18.221 21.436 -17.695 22.042 -17.822 21.982 -16.686 23.058 -17.561 23.095 -20.089 22.398 -20.42 23.024 -19.831 24.427 -19.589 25.001 -21.938 23.082 -22.52 21.677 -22.276 20.87 -23.173 24.173 21.41297- (92) PR096 PR096 PR096 PR096 PR096 PR096 PR096 PR097 PR097 PR097 PR097 PR097 PR097 P Latch 097 GLU98 GLU98 GLU98 GLU98 GLU98 GLU98 GLU98 GLU98 GLU98 HIS99 HIS99 HIS99 HIS99 HIS99 HLY99 LY101 GLY100 GLY100 LYS 101 GLY100 GLY100 GLY100 GLY100 GLY100 GLY100 GLY100 GLY100 GLY100 LYS GLY100 GLY100 GLY100 GLY100 GLY100 GLY100 GLY100 GLY100 GLY100 GLY100 GLY100 GLY100 GLY100 GLY100 GLY100 GLY100 GLY100 GLY100 GLY100 GLY100 GLY100 GLY101 LYS101 LYS101 ALA102 ALA102 ALA102 ALA102 ALA102 ARG103 ARG103 ARG103 ARG103 A BGD A BGD ABGD E1E2 A BGD1D2E1E2A A BGDEZ ABA CCCCCNCCCCCNCCCCCCCCCNNCC-20CCCCCCCCCCCCCC -18.812 26.567 -19.078 25.278 -19.532 24.256 -19.817 28.38 -18.428 28.85 -17.649 28.271 -16.462 28.595 -18.567 30.329 -20.013 30.649  20.777 29.345 -18.233 27.353 -17.555 26.761 -16.353 25.946 -15.224 26.736 -15.224 26.672 -25.782 -15.224 26.672 -15.224 26.567 -15.224 26.567 -25.782 -15.224 -25.782 -15.224 26.672 -25.946 -15.224 26.567 -25.782 -15.224 -25.782 -15.224 -25.782 -15.224 -25.782 -15.224 -25.782 -15.224 -25.782 -15.224. 25.236 -20.697 24.171 -21.799 25.49 -16.575 25.223 -15.52 24.448  14.43 25.323 -13.249 24.993 -16.19 23.646 -15.238 23.09 -14.522 21.957 -14.946 23.649 -13.779 21.805 -14.042 22.852 -14. 792 26.524 -13.832 27.439 -12.859 27.944 -11.648 27.716 -13.419 28.385 -12.626 28.895 -11.711 27.823 -10.485 28.02 -13.608 29.33 -12.893 29.892 -13.829 29.939 -14.189 28.531 -12.986 27.796 -12.474 25.624 25.624523. -12.425 24.379 -10.622 25.091 -9.63 24.64 -8.492 25.644 -7.325 25.236 98- (93) 200418849</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">755 ARG103 CB  756 ARG103 CG  757 ARG103 CD  758 ARG103 NE  759 ARG103 CZ  760 ARG103 NH1  761 ARG103 NH2  762 ARG104 N  763 ARG104 CA  764 ARG104 C  765 ARG104   766 ARG104 C  767 ARG104 CG  768 ARG104 CD  769 ARG104 NE  770 ARG104 CZ  771 ARG104 NH1  772 ARG104 NH2  773 ASP105 N  774 ASP105 CA  775 ASP105 C  776 ASP105   777 ASP105 CB  778 ASP105 CG  779 ASP105 D1  780 ASP105 D2  781 VAL106 N  782 VAL106 CA  783 VAL106 C  784 VAL106   785 VAL106 CB  786 VAL106 CG1  787 VAL106 CG2  788 VAL107 N  789 VAL107 CA  790 VAL107 C  791 VAL107   792 VAL107 CB  793 VAL107 CG1  794 VAL107 CG2  795 GLY108 N  796 GLY108 CA  797 GLY108 C  798 GLY108   799 GLU109 N  800 GLU109 CA  801 GLU109 C  802 GLU109   803 GLU109 CB  804 GLU109 CG  805 GLU109 CD  806 GLU109 E1  807 GLU109 E2  808 PHE110 N -10.347 24.476 11.762  -9.496 23785 10.705  -10.366 23.455 9.496  -9.651 22.682 8.467  -9.807 21.367 8.287  -10.493 20.645 9.175  -9.174 20.755 7.285  -8.811 26.923 13.087  -7.775 27.957 13.006  -6.906 27.966 14.256  -5.675 28.083 14.149  -8.442 29.316 12.84  -9.166 29.412 11.502  -9.828 30.772 11.319  -10.874 30.999 12.329  -11.061 32.171 12.941  -10.231 33.188 12.701  -12.048 32.31 13.829  &amp;gt;7.5 27.625 15.388  6.718 27.493 16.616  -5.828 26.253 16.594  -4.602 26.392 16.716  -7.67 27.427 17.806  -8.198 28.814 18.165  7.389 29.588 18.666  -9.411 28.938 18.257  -6.379 25.125 16.173  -5.636 23.86 16.236  -4.51 23.761 15.214  -3.414 23.348 15.612  -6.611 22.703 16.046  -5.886 21.37 15.89  -7.587 22.637 17.212  -4.641 24.427 14.075  -3.565 24.397 13.071  2.362 25.259 13.474  -1.225 24.918 13.123  -4.142 24.869 11.737  -3.06 25.104 10.687  5.175 23.879 11.213  &amp;gt;2.576 26.155 14.426  -1.49 26.985 14.953  -0.511 26.183 15.813  0.685 26.491 15.837  -1.006 25.191 16.537  -0.109 24.388 17.376  0.121 22.976 16.836  1.086 22.311 17.229  -0.677 24.35 18.784  0.577 25.728 19.424  0.886 26.1 19.659  1.612 25.244 20.147  1.22 27.255 19.442  -0.686 22.572 15.873 99- (94)200418849 809 PHE110 CA -0.511 21,269 15.221  810 PHE110 C 0.017 21.468 13.798  811 PHE110  -0.547 20.963 12.819  812 PHE110 C -1.866 20.568 15.206  813 PHE110 CG -1.834 19.088 14.838  814 PHE110 CD1 -0.842 18.264 15.352  815 PHE110 CD2 -2.808 18.564 13.998  816 PHE110 CE1 0.819 16.915 15.019  817 PHE110 CE2 -2.784 17.216 13.665  818 PHE110 CZ -1.79 16.392 14.176  819 THR111 N 1.103 22.217 13.706  820 THR111 CA 1.68 22.585 12.409  821 THR111 C 2.353 21.411 11.71  822 THR111  2.846 20.47 12.346  823 THR111 CB 2.745 23.65 12.628  824 THR111 G1 3.887 23.019 13.193  825 THR111 CG2 2.27 24.756 13.561  826 VAL112 N 2.564 21.605 10.417  827 VAL112 CA 3.302 20.63 9.605  828 VAL112 C 4.802 20.712 9.887  829 VAL112  5.492 19.689 9.86  830 VAL112 CB 3.026 20.929 8.134  831 VAL112 CG1 3.819 20.01 7.21  832 VAL112 CG2 1.535 20.833 7.832  833 LYS113 N 5.227 21.845 10.425  834 LYS113 CA 6.608 22.003 10.884  835 LYS113 C 6.892 21.096 12.082  836 LYS113  7.864 20.332 12.044  837 LYS113 CB 6.795 23.456 11.298  838 LYS113 CG 8.168 23.696 11.914  839 LYS113 CD 8.232 25.064 12.582  840 LYS113 CE 7.189 25.184 13.692  841 LYS113 NZ 7.407 24.178 14.747  842 ARG114 N 5.945 21.013 13.008  843 ARG114 CA 6.085 20.094 14.141  844 ARG114 C 6.098 18.627 13.713  845 ARG114  7.034 17.912 14.09  846 ARG114 CB 4.916 20.313 15.096  847 ARG114 CG 4.939 19.283 16.22  848 ARG114 CD 3.721 19.388 17.131  849 ARG114 NE 3.696 20.666 17.858  850 ARG114 CZ 4.078 20.792 19.131  851 ARG114 NH1 3.903 21.953 19.766  852 ARG114 NH2 4.537 19.73 19.798  853 MET115 N 5.28 18.265 12.737  854 MET115 CA 5.23 16.862 12.311  855 MET115 C 6.438 16.467 11.456  856 MET115  6.99 15.378 11.662  857 MET115 C 3.926 16.642 11.555  858 MET115 CG 2.739 16.915 12.474  859 MET115 SD 1.093 16.637 11.78  860 MET115 CE 1.057 17.94 10.532  861 LYS116 N 7.027 17.445 10.787  862 LYS116 CA 8.25 17.222 10.01 100- (95) LYS116 LYS116 LYS116 LYS116 LYS116 L.YS116 LYS116 ALA117 ALA117 ALA117 ALA117 ALA117 LEU118 LEU118 LEU118 LEU118 LEU118 LEU118 LEU118 LEU118 GLN119 GLN119 GLN119 GLN119 GLN119 GLN119 GLN119 GLN119 GLN119 PRO120 PRO120 PRO120 PRO120 PRO120 PRO120 PRO120 ARG121 ARG121 ARG121 ARG121 ARG121 ARG121 ARG121 ARG121 ARG121 ARG121 ARG121 ILE122 ILE122 ILE122 ILE122 ILE122 ILE122 ILE122 BGDEZ A B A BGD1D2A BGDE1E2A B G D A BGDEZH1H2A BG1G2 cccccnncccnccccccncccccnncccccncccccncnnnccccc 9.492 17.156  10.434 16.413  8.372 18.392  9.635 18.337  9.738 19.592  9.703 20.841  9.753 22.063  9.404 17.748  10.49 17.663  10.354 16.43  11.331 16  10.469 18.924  9.185 15.81  8.975 14.544  9.351 13.359  9.591 12.26  7.512 14.434  7.153 15.474  5.654 15.48  7.934 15.246  9.563 13.632  10.052 12.633  11.263 11.797  11.09 10.573  10.378 13.373  10.944 12.471  11.394 13.351  11.701 12.857  11.444 14.647  12.388 12.361  13.485 11.486  13.211 10.732  13.761 9.639  14.672 12.386  14.237 13.832  12.764 13.785  12.229 11.159  11.917 10.438  11.02 9.25  11.331 8.136  11.218 11.385  11.741 11.209  11.481 9.812  12.184 9.613  12.714 8.443  13.415 8.346  12.601 7.386  10.18 9.421  9.316 8.332  10.135 7.27  9.975 6.073  8.309 8.918  7.456 9.995  7.422 7.825 101 - (96) 917 1LE122 CD1 6.509 10.636 13.044  918 GLN123 N 11.179 7.724 12.982  919 GLN123 CA 12.088 6.827 12.269  920 GLN123 C 12.914 5.997 13.245  921 GLN123  12.897 4.76 13.156  922 GLN123 C 12.989 7.717 11.423  923 GLN123 CG 13.978 6.941 10.567  924 GLN123 CD 14.72 7.939 9.684  925 GLN123 E1 15.954 7.96 9.626  926 GLN123 NE2 13.946 8.8 9.044  927 GLN124 N 13.295 6.633 14.34  928 GLN124 CA 14.049 5.959 15.394  929 GLN124 C 13.184 4.947 16.144  930 GLN124  13.621 3.798 16.284  931 GLN124 CB 14.544 7.04 16.345  932 GLN124 CG 15.429 6.495 17.455  933 GLN124 CD 15.912 7.668 18.3  934 GLN124 E1 16.786 7.524 19.162  935 GLN124 NE2 15.357 8.832 18.008  936 ILE125 N 11.904 5.252 16.299  937 1LE125 CA 10.961 4.328 16.947  938 ILE125 C 10.716 3.072 16.112  939 ILE125  10.861 1.961 16.642  940 ILE125 CB 9.638 5.064 17.148  941 ILE125 CG1 9.792 6.214 18.13  942 ILE125 CG2 8.543 4.122 17.628  943 ILE125 CD1 8.487 6.984 18.275  944 VAL126 N 10.649 3.229 14.797  945 VAL126 CA 10.44 2.064 13.928  946 VAL126 C 11.693 1.2 13.87  947 VAL126  11.603 -0.021 14.062  948 VAL126 CB 10.119 2.529 12.513  949 VAL126 CG1 9.754 1.334 11.641  950 VAL126 CG2 8.988 3.544 12.503  951 ASP127 N 12.843 1.855 13.909  952 ASP127 CA 14.121 1.141 13.889  953 ASP127 C 14.314 0.34 15.17  954 ASP127  14.537 -0.876 15.088  955 ASP127 CB 15.258 2.153 13.769  956 ASP127 CG 15.158 2.967 12.481  957 ASP127 D1 15.632 4.097 12.49  958 ASP127 D2 14.686 2.426 11.489  959 GLU128 N 13.903 0.919 16.288  960 GLU128 CA 14.048 0.26 17.589  961 GLU128 C 13.094 0.915 17.762  962 GLU128  13.527 -1.952 18.281  963 GLU128 CB 13.764 1.281 18.684  964 GLU128 CG 14.807 2.39 18.707  965 GLU128 CD 14.367 3.489 19.668  966 GLU128 OE1 13.584 4.333 19.247  967 GLU128 E2 14.794 3.452 20.812  968 HIS129 N 11.934 -0.861 17.128  969 HIS129 CA 11.002 -1.985 17.237  970 HIS129 C 11.411 -3.142 16.333 -102- (97) HIS129  11.344 -4.297  HIS129 CB 9.592 -1.533  HIS129 CG 8.963 -0.57  HIS129 ND1 7.942 0.266  HIS129 CD2 9.3 -0.394  HIS129 CE1 7.647 0.969  HIS129 NE2 8.488 0.561  ILE130 N 12.061 -2.848  ILE130 CA 12.564 -3.95  ILE130 C 13.768 -4.577  ILE130  13.69 -5.756  ILE130 CB 12.968 -3.449  ILE130 CG1 11.841 -2.659  ILE130 CG2 13.341 -4.632  ILE130 CD1 12.258 -2.117  ASP131 N 14.65 -3.712  ASP131 CA 15.874 -4.12  ASP131 C 15.604 -5.033  ASP131  15.932 6.226  ASP131 CB 16.565 -2.874  ASP131 CG 17.175 -1.999  ASP131 D1 17.222 -0.791  ASP131 OD2 17.743 -2.564  ALA132 N 14.882 -4.505  ALA132 CA 14.708 -5.198  ALA132 C 13.582 -6.228  ALA132  13.489 -6.983  ALA132 CB 14.465 -4.147  LEU133 N 12.752 -6.286  LEU133 CA 11.712 -7.311  LEU133 C 12.08 -8.444  LEU133  12.567 -9.492  LEU133 CB 10.366 -6.697  LEU133 CG 9.925 -5.686  LEU133 CD1 8.679 -4.93  LEU133 CD2 9.698 -6.366  LEU134 N 11.901 -8.215  LEU134 CA 12.139 -9.288  LEU134 C 12.895 -8.777  LEU134  12.319 -8.632  LEU134 CB 10.808 9.885  LEU134 CG 10.481 -11.234  LEU134 CD1 11.66 -12.193  LEU134 CD2 9.997 -11.115  ALA135 N 14.194 -8.586  ALA135 CA 15.038 -8.142  ALA135 C 15.606 -9.293  ALA135  16.184 9.051  ALA135 CB 16.193 7.323  GLY136 N 15.402 -10.522  GLY136 CA 15.957 -11.679  GLY136 C 14.865 -12.524  GLY136  14.069 -12.022  PR0137 N 14.906 -13.813 103- (98) 200418849 1025 1026 1027 1028 1029 1030 1031 1032 1033 1034 1035 1036 1037 1038 1039 1040 1041 1042 1043 1044 1045 1046 1047 1048 1049 1050 1051 1052 1053 1054 1055 1056 1057 1058 1059 1060 1061 1062 1063 1064 1065 1066 1067 1068 1069 1070 1071 1072 1073 1074 1075 1076 1077 1078 137137137137137137383838383838383838139139139139139139139 ROROROROROROYS1YS1YS1YS1YS1YS1YS1YS1YS1RORORORORORORO ppppppgggglllllppppppp ALA140 ALA140 ALAMO ALA140 ALA140 ASP141 ASP141 ASP141 ASP141 ASP141 ASP141 ASP141 ASP141 LEU142 LEU142 LEU142 LEU142 LEU142 LEU142 LEU142 LEU142 VAL143 VAL143 VAL143 VAL143 VAL143 VAL143 VAL143 GLN144 GLN144 GLN144 GLN144 CA 13.938 -14.772 11.353  C 12.589 -14.709 12.067   12.21 -13.679 12.637  CB 14.573 -16.111 11.568  CG 15.748 -15.953 12.524  CD 15.899 -14.46 12.763  N 11.86 -15.811 11.945  CA 10.575 -16.043 12.639  C 9.412 -15.227 12.069   9.605 -14.143 11.508  CB 10.733 -15.765 14.135  CG 11.795 -16.66 14.765  CD 12.022 -16.303 16.23  CE 13.155 -17.128 16.829  NZ 12.859 -18.567 16.746  N 8.256 -15.868 12.022  CA 6.996 -15.155 11.797  C 6.612 -14.289 12.995   6.167 -14.795 14.031  CB 5.979 -16.232 11.583  CG 6.595 -17.575 11.948  CD 8.04 -17.286 12.322  N 6.749 -12.987 12.824  CA 6.355 -12.036 13.868  C 5.006 -11.411 13.552   4.591 -11.359 12.391  CB 7.395 -10.931 13.953  N 4.297 -10.989 14.582  CA 3.051 -10.264 14.336  C 3.363 -8.779 14.165   3.466 -8.032 15.149  CB 2.073 10.492 15.481  CG 0.741 -9.84 15.132  D1 0.583 -8.673 15.465  D2 0.016 -10.426 14.338  N 3.261 -8.332 12.923  CA 3.692 -6.983 12.541  C 2.753 5.893 13.048   3.232 -4.806 13.4  CB 3.78 -6.96 11.012  CG 4.336 -5.66 10.428  CD1 5.227 5.944 9.225  CD2 3.241 -4.659 10.063  N 1.509 -6.243 13.329  CA 0.585 -5.247 13.868  C 0.992 -4.867 15.287   1.481 -3.748 15.477  CB -0.829 -5.809 13.859  CG1 -1.823 -4.772 14.374  CG2 -1.212 -6.26 12.457  N 1.184 -5.874 16.119  CA 1.491 -5.656 17.535  C 2,945 -5.249 17.797   3.212 -4.566 18.791 -104- (99) 1079 GLN144 CB 1.203 -6.982 18.23  1080 GLN144 CG 1.485 -6.966 19.726  1081 GLN144 CD 1.232 -8.364 20.277  1082 GLN144 E1 1.815 -8.777 21.285  1083 GLN144 NE2 0.374 -9.091 19.582  1084 ALA145 N 3.842 -5.539 16.87  1085 ALA145 CA 5.246 -5.186 17.088  1086 ALA145 C 5.672 -3.884 16.412  1087 ALA145  6.594 -3.211 16.888  1088 ALA145 C 6.108 -6.329 16.561  1089 LEU146 N 4.988 -3.499 15.349  1090 LEU146 CA 5.424 -2.324 14.588  1091 LEU146 C 4.294 -1.342 14.317  1092 LEU146  4.308 -0.207 14.815  1093 LEU146 CB 5.964 -2.825 13.252  1094 LEU146 CG 7.225 -3.659 13.433  1095 LEU146 CD1 7.467 -4.589 12.254  1096 LEU146 CD2 8.432 &amp;gt;2.772 13.705  1097 SER147 N 3.245 -1.868 13.71  1098 SER147 CA 2.169 -1.053 13.134  1099 SER147 C 1.274 -0.369 14.164  1100 SER147  0.727 0.708 13.911  1101 SER147 CB 1.325 -2.001 12.301  1102 SER147 G 0.198 -1.277 11.856  1103 LEU148 N 1.174 -0.97 15.331  1104 LEU148 CA 0.484 -0.358 16.464  1105 LEU148 C 1.433 0.51 17.316  1106 LEU148  1.132 1.707 17.436  1107 LEU148 CB -0.204 1.475 17.259  1108 LEU148 CG -1.13 -0.987 18.372  1109 LEU148 CD1 -2.317 -1.93 18.53  1110 LEU148 CD2 - 0.404 -0.81 19.703  1111 PR0149 N 2.553 0.008 17.848  1112 PR0149 CA 3.299 0.829 18.809  1113 PR0149 C 3.987 2.052 18.203  1114 PR0149  4.043 3.07 18.9  1115 PR0149 CB 4.31 -0.075 19.439  1116 PR0149 CG 4.261 1.43 18.764  1117 PR0149 CD 3.122 -1.352 17.766  1118 VAL150 N 4.316 2.051 16.918  1119 VAL150 CA 4.873 3.276 16.323  1120 VAL150 C 3.867 4.442 16.385  1121 VAL150  4.127 5.334 17.201  1122 VAL150 CB 5.399 3.041 14.905  1123 VAL150 CG1 5.892 4.345 14.284  1124 VAL150 CG2 6.514 2.004 14.891  1125 PR0151 N 2.695 4.396 15.75  1126 PR0151 CA 1.816 5.574 15.799  1127 PR0151 C 1.187 5.843 17.167  1128 PR0151  1.009 7.014 17.532  1129 PR0151 CB 0.742 5.307 14.799  1130 PR0151 CG 0.88 3.895 14.266  1131 PR0151 CD 2.136 3.336 14.896  1132 SER152 N 1.059 4.817 17.993 -105- (100)200418849 1133 SER152 CA 0.504 5.027 19.326  1134 SER152 C 1.506 5.73 20.241  1135 SER152  1.136 6.724 20.879  1136 SER152 C 0.117 3.671 19.898  1137 SER152 G -0.849 3.09 19.031  1138 LEU153 N 2.785 5.428 20.079  1139 LEU153 CA 3.817 6.088 20.883  1140 LEU153 C 4.17 7.458 20.312  1141 LEU153  4.34 8.406 21.091  1142 LEU153 CB 5.06 5.206 20.899  1143 LEU153 CG 6.168 5.789 21.769  1144 LEU153 CD1 5.708 5.934 23.216  1145 LEU153 CD2 7.424 4.928 21.689  1146 VAL154 N 3.995 7.622 19.009  1147 VAL154 CA 4.232 8.925 18.383  1148 VAL154 C 3.185 9.944 18.813  1149 VAL154  3.566 11.036 19.256  1150 VAL154 C 4.205 8.777 16.864  1151 VAL154 CG1 4.148 10.134 16.169  1152 VAL154 CG2 5.402 7.976 16.368  1153 ILE155 N 1.945 9.513 18.977  1154 ILE155 CA 0.935 10.467 19.431  1155 ILE155 C 0.932 10.621 20.956  1156 ILE155  0.6 11.709 21.446  1157 ILE155 CB -0.433 10.065 18.902  1158 ILE155 CG1 -1.405 11.218 19.089  1159 ILE155 CG2 -0.956 8.814 19.593  1160 ILE155 CD1 -0.954 12.46 18.327  1161 CYS156 N 1.569 9.697 21.66  1162 CYS156 CA 1.787 9.891 23.093  1163 CYS156 C 2.835 10.973 23.317  1164 CYS156  2.551 11.93 24.047  1165 CYS156 C 2.261 8.59 23.732  1166 CYS156 SG 1.018 7.295 23.935  1167 GLU157 N 3.838 11.005 22.454  1168 GLU157 CA 4.902 12.011 22.559  1169 GLU157 C 4.512 13.364 21.957  1170 GLU157  5.08 14.393 22.339  1171 GLU157 CB 6.109 11.474 21.801  1172 GLU157 CG 6.57 10.14 22.372  1173 GLU157 CD 7.588 9.499 21.434  1174 GLU157 OE1 8.764 9.522 21.767  1175 GLU157 E2 7.162 8.932 20.437  1176 LEU158 N 3.5 13.37 21.107  1177 LEU158 CA 3.042 14.616 20.492  1178 LEU158 C 2.03 15.308 21.401  1179 LEU158  2.089 16.535 21.586  1180 LEU158 C 2.412 14.232 19.149  1181 LEU158 CG 2.171 15.389 18.175  1182 LEU158 CD1 2.086 14.868 16.745  1183 LEU158 CD2 0.93 16.213 18.504  1184 LEU159 N 1.211 14.518 22.072  1185 LEU159 CA 0.183 15.099 22.929  1186 LEU159 C 0.747 15.411 24.309 106- (101)   1187 LEU159  1188 LEU159  1189 LEU159  1190 LEU159  1191 LEU159  1192 GLY160  1193 GLY160  1194 GLY160  1195 GLY160  1196 VAL161  1197 VAL161  1198 VAL161  1199 VAL161  1200 VAL161  1201 VAL161  1202 VAL161  1203 PR0162  1204 PR0162  1205 PR0162  1206 PR0162  1207 PR0162  1208 PR0162  1209 PR0162  1210 TYR163  1211 TYR163  1212 TYR163  1213 TYR163  1214 TYR163  1215 TYR163  1216 TYR163  1217 TYR163  1218 TYR163  1219 TYR163  1220 TYR163  1221 TYR163  1222 SER164  1223 SER164  1224 SER164  1225 SER164  1226 SER164  1227 SER164  1228 ASP165  1229 ASP165  1230 ASP165  1231 ASP165  1232 ASP165  1233 ASP165  1234 ASP165  1235 ASP165  1236 HIS166  1237 HIS166  1238 HIS166  1239 HIS166  1240 HIS166 BGD1D2A A bg'g:a bgd&amp;gt;  JCCCCNCCONCCCCCNCCccCJii5/wwwwwwy3</span>755 ARG103 CB 756 ARG103 CG 757 ARG103 CD 758 ARG103 NE 759 ARG103 CZ 760 ARG103 NH1 761 ARG103 NH2 762 ARG104 N 763 ARG104 CA 764 ARG104 C 765 ARG104 766 ARG104 C  767 ARG104 CG 768 ARG104 CD 770 ARG104 ARG104 NH1 772 ARG104 NH2 773 ASP105 N 774 ASP105 CA 775 ASP105 C 776 ASP105 777 ASP105 CB 778 ASP105 CG 779 ASP105 D1 780 ASP105 D2 781 VAL106 N 782 VAL106 CA 783 VAL106 C 784 VAL106 785 VAL106 CB 786 786 VAL106 VAL106 CG2 788 VAL107 N 789 VAL107 CA 790 VAL107 C 791 VAL107 792 VAL107 CB 793 VAL107 CG1 794 VAL107 CG2 795 GLY108 N 796 GLY108 CA 797 GLY108 C 798 GLY108 799 GLU109 N 800 GLU109 CA 801 GLU109 C 802 GLU109 804 GLU109 CG 805 GLU109 CD 806 GLU109 E1 807 GLU109 E2 808 PHE110 N -10.347 24.476 11.762 -9.496 23785 10.705 -10.366 23.455 9.496 -9.651 22.682 8.467 -9.807 21.367 8.287 -10.493 20.645 9.175 -9.174 20.755 7.285 -8.811 26.923 13.087 7. 775 27.957 13.006 -6.906 27.966 14.256 -5.675 28.083 14.149 -8.442 29.316 12.84 -9.166 29.412 11.502 -9.828 30.772 11.319 -10.874 30.999 12.329 -11.061 32.171 12.941 -10.231 33.188 12.701 -12.048 32.31 13.829 &amp;gt; 7.5 27.625 15.388 16.493-6.618 27.28 28.083 26.253 16.594 -4.602 26.392 16.716 -7.67 27.427 17.806 -8.198 28.814 18.165  7.389 29.588 18.666 -9.411 28.938 18.257 -6.379 25.125 16.173 -5.636 23.86 16.236 -4.51 23.761 15.214 -3.414 23.348 15.612 -6.611 22.703 16.046 -5.886 21.37 15.89 -7.587 22.637 17.212 -4.641 24.427 14.075 -3.565 24.397 13.071  2.362 25.259 13.474 -1.225 24.918 13.123 -4.142 24.869 11.737 -3.06 25.104 10.687  5.175 23.879 11.213 &amp;gt; 2.576 26.155 14.426 -1.49 26.985 14.953 -0.511 26.183 15.813 0.685 26.491 15.837 -1.006 25.191 16.537 24.388 17.376 0.121 22.976 16.836 1.086 22.311 17.229 -0.677 24.35 18.784 0.577 25.728 19.424 0.886 26.1 19.659 1.612 25.244 20.147 1.22 27.255 19.442 -0.686 22.572 1 5.873 99- (94) 200418849 809 PHE110 CA -0.511 21,269 15.221 810 PHE110 C 0.017 21.468 13.798 811 PHE110 -0.547 20.963 12.819 812 PHE110 C  -1.866 20.568 15.206 813 PHE110 CG -1.834 19.088 14.838 814 PHE110 CD1 -0.842 18.264 15.352 815 PHE110 CD2 -2.808 18.564 13.998 816 PHE110 CE1  0.819 16.915 15.019 817 PHE110 CE2 -2.784 17.216 13.665 818 PHE110 CZ -1.79 16.392 14.176 819 THR111 N 1.103 22.217 13.706 820 THR111 CA 1.68 22.585 12.409 821 THR111 C 2.353 21.411 11.71 822 THR111 . 12.346 823 THR111 CB 2.745 23.65 12.628 824 THR111 G1 3.887 23.019 13.193 825 THR111 CG2 2.27 24.756 13.561 826 VAL112 N 2.564 21.605 10.417 827 VAL112 CA 3.302 20.63 9.605 828 VAL112 C 4.802 20.712 9.887 829 VAL112 5.492 20.689 9.86 830 830 831 VAL112 CG1 3.819 20.01 7.21 832 VAL112 CG2 1.535 20.833 7.832 833 LYS113 N 5.227 21.845 10.425 834 LYS113 CA 6.608 22.003 10.884 835 LYS113 C 6.892 21.096 12.082 836 LYS113 7.864 20.332 12.044 837 LYS113 CB 6.795 23.456 11.298 838 LYS113 CG 8.168 23.696 11.914 839 LYS113 CD 8.232 25.064 12.582 840 LYS113 CE 7.189 25.184 13.692 841 LYS113 NZ 7.407 24.178 14.747 842 ARG114 N 5.945 21.013 13.008 843 ARG114 CA 6.085 20.094 14.094 13.098 ARG114 7.034 17.912 14.09 846 ARG114 CB 4.916 20.313 15.096 847 ARG114 CG 4.939 19.283 16.22 848 ARG114 CD 3.721 19.388 17.131 849 ARG114 NE 3.696 20.666 17.858 850 ARG114 CZ 4.078 20.792 19.131 851 ARG114 NH1 3.903 21.953 19.766 852 ARG114 115 NH2 4.5 4.5 5.28 18.265 12.737 854 MET115 CA 5.23 16.862 12.311 855 MET115 C 6.438 16.467 11.456 856 MET115 6.99 15.378 11.662 857 MET115 C  3.926 16.642 11.555 858 MET115 CG 2.739 16.915 12.474 859 MET115 SD 1.093 16.637 11.78 860 MET115 CE 1.057 17.94 10.532 7.027 861LY 17.445 10.787 862 LYS116 CA 8.25 17.222 10.01 100- (95) LYS116 LYS116 LYS116 LYS116 LYS116 L.YS116 LYS116 ALA117 ALA117 ALA117 ALA117 ALA1 17 LEU118 LEU118 LEU118 LEU118 LEU118 LEU118 LEU118 LEU118 GLN119 GLN119 GLN119 GLN119 GLN119 GLN119 GLN119 GLN119 GLN119 PRO120 PRO120 PRO120 PRO120 PRO120 PRO120 PRO120 ARG121 ARG121 ARG121 ARG121 ARG121 ARG121 ARG121 ARG121 ARG121 ARG121 ARG121 ILE122 ILE122 ILE122 ILE122 ILE122 ILE122 ILE122 BGDEZ ABA BGD1D2A BGDE1E2A BGDA BGDEZH1H2A BG1G2 cccccnncccnccccccncccccnncccccncccccncnnnccccc 9.492 17.156 10.434 16.413 8.372 18.392 9.635 18.337 9.738 19.592 9.703 20.841 9.753 22.063 9.404 17.748 10.49 17.663 10.354 16.43 11.331 16 10.469 18.924 9.185 15.81 8.975 14.544 9.351 13.359 9.591 12.26 14.434 7.153 15.474 5.654 15.48 7.934 15.246 9.563 13.632 10.052 12.633 11.263 11.797 11.09 10.573 10.378 13.373 10.944 12.471 11.394 13.351 11.701 12.857 11.444 14.647 12.388 12.361 13.485 11.486 13.211 10.732 13.761 9.639 14.672 12.386 14.237 13.832 12.764 13.785 11.229 11.159 11.917 10.438 11.02 9.25 11.331 8.136 11.218 11.218 .385 11.741 11.209 11.481 9.812 12.184 9.613 12.714 8.443 13.415 8.346 12.601 7.386 10.18 9.421 9.316 8.332 10.135 7.27 9.975 6.073 8.309 8.918 7.456 9.995 7.422 7.825 101-(96) 917 1LE122 CD1 6.509 10.636 13.044 918 GLN123 N 11.179 7.724 12.982 919 GLN123 CA 12.088 6.827 12.269 920 GLN123 C 12.914 5.997 13.245 921 GLN123 12.897 4.76 13.156 922 GLN123 C  12.989 7.717 11.423 923 GLN123 CG 13.978 6.941 10.567 924 GLN123 CD 14.72 7.939 9.684 925 GLN123 E1 15.954 7.96 9.626 926 GL N123 123 13.22 14.34 928 GLN124 CA 14.049 5.959 15.394 929 GLN124 C 13.184 4.947 16.144 930 GLN124 13.621 3.798 16.284 931 GLN124 CB 14.544 7.04 16.345 932 GLN124 CG 15.429 6.495 17.455 933 GLN124 CD 15.912 7.668 18.3 934 GLN124 E1 16.786 15.832 16.935 7.832 16.935 7.832 16.926 7.832 15. 936 ILE125 N 11.904 5.252 16.299 937 1LE125 CA 10.961 4.328 16.947 938 ILE125 C 10.716 3.072 16.112 939 ILE125 10.861 1.961 16.642 940 ILE 125 CB 9.638 5.064 17.148 941 ILE125 CG1 9.792 6.214 18.13 942 ILE125 CG2 8.543 4.122 17.628 943 ILE125 CD1 8.487 6.984 18.275 944 VAL126 N 10.649 3.229 14.797 945 VAL126 CA 10.44 2.064 13.928 946 VAL126 C 11.693 1.2 13.87 947 VAL126 VAL11 06012.603-12.060.02 CB 10.119 2.529 12.513 949 VAL126 CG1 9.754 1.334 11.641 950 VAL126 CG2 8.988 3.544 12.503 951 ASP127 N 12.843 1.855 13.909 952 ASP127 CA 14.121 1.141 13.889 953 ASP127 C 14.314 0.34 15.17 954 ASP127 14.537 -0.876 15.088 955 ASP127 CB 15.258 2.153 13.769 956 15.158 2.967 12.481 957 ASP127 D1 15.632 4.097 12.49 958 ASP127 D2 14.686 2.426 11.489 959 GLU128 N 13.903 0.919 16.288 960 GLU128 CA 14.048 0.26 17.589 961 GLU128 C 13.094  0.915 17.762 962 GLU128 13.527 -1.952 18.281 963 GLU128 CB 13.764 1. GLU128 CG 14.807 2.39 18.707 965 GLU128 CD 14.367 3.489 19.668 966 GLU128 OE1 13.584 4.333 19.247 967 GLU128 E2 14.794 3.452 20.812 968 HIS129 N 11.934 -0.86 1 17.128 969 HIS129 CA 11.002 -1.985 17.237 970 HIS129 C 11.411 -3.142 16.333 -102- (97) HIS129 11.344 -4.297 HIS129 CB 9.592 -1.533 HIS129 CG 8.963 -0.57 HIS129 ND1 7.942 0.266 HIS129 CD2 9.3 -0.394 HIS129 CE1 7.647 0.969 HIS129 NE2 8.488 0.561 ILE130 N 12.061 -2.848 ILE130 CA 12.564 -3.95 ILE130 C 13.768 -4.577 ILE130 13.69 -5.756 ILE130 CB 12.968 -3.449 ILE130 CG1 11.841 -2.659 ILE130 CG2 13.341 -4.632 ILE130 CD1 12.258 -2.117 ASP131 N 14.65 -3.712 CA 15.874 -4.12 ASP131 C 15.604 -5.033 ASP131 15.932  6.226 ASP131 CB 16.565 -2.874 ASP131 CG 17.175 -1.999 ASP131 D1 17.222 -0.791 ASP131 OD2 17.743 -2.564 ALA132 N 14.882 -4.505 ALA132 CA 14.708 -5.198 ALA132 C 13.582 -6.228 ALA132 13.489 -6.983 ALA132 CB 14.465 -4.147 LEU133 N 12.752 -6.286 LEU133 CA 11.712 -7.311 LEU133 C 12.08 -8.444 LEU133 12.567 -9.492 LEU133 CB 10.366 -6.697 LEU133 CG 9.925 -5.686 LEU133 CD1 8.679 -4.93 LEU133 CD2 9.698 LEU134 N 11.901 -8 .215 LEU134 CA 12.139 -9.288 LEU134 C 12.895 -8.777 LEU134 12.319 -8.632 LEU134 CB 10.808  9.885 LEU134 CG 10.481 -11.234 LEU134 CD1 11.66 -12.193 LEU134 CD2 9.997 -11.115 ALA135 N 14.194 -8.586 ALA135 CA 15.038 -8.142 ALA135 C 15. -9.293 ALA135 16.184  9.051 ALA135 CB 16.193  7.323 GLY136 N 15.402 -10.522 GLY136 CA 15.957 -11.679 GLY136 C 14.865 -12.524 GLY136 14.069 -12.022 PR0137 N 14.906 -13.813 103- (98) 200418849 1025 1026 1027 1028 1029 1030 1030 1032 1033 1034 1035 1036 1037 1038 1039 1040 1041 1042 1043 1044 1045 1046 1047 1048 1049 1050 1051 1052 1053 1054 1055 1056 1057 1058 1059 1060 1061 1062 1063 1064 1065 1066 1067 1068 1069 1070 1071 1072 1073 1074 1075 1076 1077 1078 137137137137137137383838383838383838139139139139139139139 ROROROROROROYS1YS1YS1YS1YS1YS1YS1YS1YS1RORORORORORORO ppppppgggglllllppppppp ALA140 ALA140 ALAMO ALA140 ALA140 ASP141 ASP141 ASP141 ASP141 ASP141 ASP141 ASP141 ASP141 LEU142 LEU142 LEU142 LEU142 LEU142 LEU142 LE U142 LEU142 VAL143 VAL143 VAL143 VAL143 VAL143 VAL143 VAL143 VAL143 GLN144 GLN144 GLN144 GLN144 CA 13.938 -14.772 11.353 C 12.589 -14.709 12.067 12.21 -13.679 12.637 CB 14.573 -16.111 11.568 CG 15.748 -15.953 12.524 CA 11.899 11.945 N 11.899 -14 -16.043 12.639 C 9.412 -15.227 12.069 9.605 -14.143 11.508 CB 10.733 -15.765 14.135 CG 11.795 -16.66 14.765 CD 12.022 -16.303 16.23 CE 13.155 -17.128 16.829 NZ 12.859 -18.567 16.746 N 8.256 -15.868 12.022 CA 6.996 -15.155 11.797 C 6.612 -14.289 12.995 6.167 -14.795 14.031 CB 5.979 -16.232 11.583 CG 6.595 -17.575 11.948 CD 8.04 -17.286 12.322 N 6.749 -12.987 12.824 CA 6.355 -12.036 13.868 C 5.006 -11.411 13.552 4.591 -11.359 12.391 CB 7.395 -10.931 13.953 N 4.297 -10.989 14.582 CA 3.051 -10.264 14.336 C 3.363 -8.779 14.165 3.466 -8.032 15.149 CB 2.073  10.492 15.481 CG 0.741 -9.84 15.132 D1 0.583 -8.673 15.465 D2 0.016 -10.426 14.338 N 3.261 -8.332 12.923 CA 3.69 2 -6.983 12.541 C 2.753  5.893 13.048 3.232 -4.806 13.4 CB 3.78 -6.96 11.012 CG 4.336 -5.66 10.428 CD1 5.227  5.944 9.225 CD2 3.241 -4.659 10.063 N 1.509 -6.243 13.329 CA 0.585 -5.247 13.868 C 0.992 -4.867 15.287  1.481 -3.748 15.477 CB -0.829 -5.809 13.859 CG1 -1.823 -4.772 14.374 CG2 -1.212 -6.26 12.457 N 1.184 -5.874 16.119 CA 1.491 -5.656 17.535 C 2,945 -5.249 17.797 3.212 -4.566 18.791 -104- (99) 1079 GLN144 CB 1.203 -6.982 18.23 1080 GLN144 CG 1.485 -6.966 19.726 1081 GLN144 CD 1.232 -8.364 20.277 1082 GLN144 E1 1.815 -8.777 21.285 1083 GLN144 NE2 0.374 -9.091 19.582 1084 ALA145 N 3.842 -5.539 16.87 1085 ALA145 CA 5.246 -5.186 17.088 1086 C 5.672 -3.884 16.412 1087 ALA145 6.594 -3.211 16.888 1088 ALA145 C  6.108 -6.329 16.561 1089 LEU146 N 4.988 -3.499 15.349 1090 LEU146 CA 5.424 -2.324 14.588 1091 LEU146 C 4.294 -1.342 14.317 1092 LEU146 4.308 -0.207 14.815 1093 LEU146 CB 5.964 -2.825 13.252 1094 LEU146 CG 7.225 -3.659 13.433 1095 LEU146 CD1 7.467 -4.589 12.254 1096 LEU146 CD2 8.432 &amp;gt; 2.772 13.705 1097 SER147 N 3.245 -1.868 13.71 1098 SER147 CA 2.169 -1.053 13.134 1099 SER147 C 1.274 -0.369 14.164 1100 SER147 0.727 0.708 13.911 1101 SER147 CB 1.325 2.001 12.301 1102 SER147 G 0.198 -1.277 11.856 1103 LEU148 N 1.174 -0.97 15.331 1104 LEU148 CA 0.484 -0.358 16.464 1105 LEU148 C 1.433 0.51 17.316 1106 LEU148 1.132 1.707 17.436 1107 LEU148 CB -0.204  1.475 17.259 1108 LEU148 CG -1.13- 0.987 18.372 1109 LEU148 CD1 -2.317 -1.93 18.53 1110 LEU148 CD2-0.404 -0.81 19.703 1111 PR0149 N 2.553 0.008 17.848 1112 PR0149 CA 3.299 0.829 18.809 1113 PR0149 C 3.987 2.052 18.203 1114 PR0149 4.043 3.07 18.9 1115 PR0149 CB 4.31 -0.075 19.439 19.439 PR0149 CG 4.261  1.43 18.764 1117 PR0149 CD 3.122 -1.352 17.766 1118 VAL150 N 4.316 2.051 16.918 1119 VAL150 CA 4.873 3.276 16.323 1120 VAL150 C 3.867 4.442 16.385 1121 VAL150 4.127 5.334 17.201 1122 VAL150 CB 5.399 3 .041 14.905 1123 VAL150 CG1 5.892 4.345 14.284 1124 VAL150 CG2 6.514 2.004 14.891 1125 PR0151 N 2.695 4.396 15.75 1126 PR0151 CA 1.816 5.574 15.799 1127 PR0151 C 1.187 5.843 17.167 1128 PR0151 1.009 7.014 17.532 1129 PR0151 CB 0.742 5.307 14.799 1799 1130 14.266 1131 PR0151 CD 2.136 3.336 14.896 1132 SER152 N 1.059 4.817 17.993 -105- (100) 200418849 1133 SER152 CA 0.504 5.027 19.326 1134 SER152 C 1.506 5.73 20.241 1135 SER152 1.136 6.724 20.879 1136 SER152 C  0.117 3.671 19.898 1137 SER152 G- 0.849 3.09 19.031 1138 LEU153 N 2.785 5.428 20.079 1139 LEU153 CA 3.817 6.088 20.883 1140 LEU153 C 4.17 7.458 20.312 1141 LEU153 4.34 8.406 21.091 1142 LEU153 CB 5.06 5.206 20.899 1143 LEU153 CG 6.168 5.789 21.769 1144 LEU153 CD1 5.708 5.934 23.216 21.689 1146 VAL154 N 3.995 7.622 19.009 1147 VAL154 CA 4.232 8.925 18.383 1148 VAL154 C 3.185 9.944 18.813 1149 VAL154 3.566 11.036 19.256 1150 VAL154 C  4.205 8.777 16.864 1151 VAL154 CG1 4.148 10.134 16.169 1152 VAL154 CG2 5.402 7.976 16.368 1153 ILE155 N 1.945 9.513 18.977 1154 ILE155 CA 0.935 10.467 19.431 1155 ILE155 C 0.932 10.621 20.956 1156 ILE155 0.6 11.709 21.446 1157 ILE155 CB -0.433 10.065 18.902 1158 1158 11.218 19.089 1159 ILE155 CG2 -0.956 8.814 19.593 1160 ILE155 CD1 -0.954 12.46 18.327 1161 CYS156 N 1.569 9.697 21.66 1162 CYS156 CA 1.787 9.891 23.093 1163 CYS156 C 2.835 10.973 23.317 1164 CYS156 2.551 11.93 24.047 1165 CYS59 23.52 2.2 1.018 7.295 23.935 1167 GLU157 N 3.838 11.005 22.454 1168 GLU157 CA 4.902 12.011 22.559 1169 GLU157 C 4.512 13.364 21.957 1170 GLU157 5.08 14.393 22.339 1171 GLU157 CB 6.109 11.474 21.801 1172 GLU157 CG 6.57 10.14 22.372 1173 GLU157 CD 7.588 9.499 21.434 1174 GLU 21.767 1175 GLU157 E2 7.162 8.932 20.437 1176 LEU158 N 3.5 13.37 21.107 1177 LEU158 CA 3.042 14.616 20.492 1178 LEU158 C 2.03 15.308 21.401 1179 LEU158 2.089 16.535 21.586 1180 LEU158 C  2.412 14.232 19.149 1181 LEU158 CG 2.171 15.389 18.175 1182 LEU158 CD1 2.086 14.868 16.745 1183 LEU158 CD2 0.93 16.213 18.504 1184 LEU159 N 1.211 14.518 22.072 1185 LEU159 CA 0.183 15.099 22.929 1186 LE411159 0.71. 106- (101) 1187 LEU159 1188 LEU159 1189 LEU159 1190 LEU159 1191 LEU159 1192 GLY160 1193 GLY160 1194 GLY160 1195 GLY160 1196 VAL161 1197 VAL161 1198 VAL161 1199 VAL161 1200 VAL161 1201 VAL161 1202 VAL161 1203 PR0162 1204 PR0162 1206 1206 1206 1206 2160 PR0162 1210 TYR163 1211 TYR163 1212 TYR163 1213 TYR163 1214 TYR163 1215 TYR163 1216 TYR163 1217 TYR163 1218 TYR163 1219 TYR163 1220 TYR163 1221 TYR163 1222 ASP165 1212 ASP16512 12650 ASP165 1235 ASP165 1236 HIS166 1237 HIS166 1238 HIS166 1239 HIS166 1240 HIS166 BGD1D2A A bg'g: a bgd&gt;  JCCCCNCCONCCCCCNCCccCJii5 / wwwwwwy3</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">1 2 G G</span>1 2 G G</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">A 12 12 bgddeezh</span>A 12 12 bgddeezh</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">A</span>A</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">G A</span> G A</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">1 2 G D D</span>1 2  G D D</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">A</span>A</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">B 0.832 -0.979 -2.24 -2.637 -3.385 1.347 1.874 1.433 1.183 1.366 0.869 2.006 2.715 -0.111 -0.763 -1.175 2.15 3.13 2.838 1.68 3.066 1.972 1.358 3.882 3.793 3.104 2.72 5.192 6.111 6.895 6.181 7.726 7.011 7.78 8.589 2.806 2.007 0.51 -0.319 2.303 1.766 0.173 -1.215 -1.445 -2.542 -1.463 -1.331 -2.358 -0.21 0.343 -0.299 -0.63 -0.515 1.11 16.594 14.12 14.827 15.915 13.84 14.417 14.572 13.413 13.581 12.238 11.037 10.078 9.58 10.348 9.125 11.327 9.806 8.826 7.41 7.021 8.899 9.874 10.384 6.593 5.209 4.257 3.153 4.684 4.367 5.363 3.067 5.066 2.768 3.77 3.478 4.747 3.992 4.158 3.566 4.538 5.854 5.056 5.273 4.619 4.126 6.775 7.38 7.592 7.683 4.475 3.666 2.223 1.86 3.792 24.66 23.044 23.524 22.535 23.707 24.943 26.306 27.2 28.398 26.601 27.284 27.635 26.752 26.338 26.976 25.866 28.922 29.4 28.901 28.689 30.894 31.302 30.009 28.912 28.41 29.393 29.002 28.071 29.254 29.826 29.743 30.898 30.815 31.392 32.467 30.589 31.561 31.301 32 32.952 33.017 30.389 30.002 28.649 28.344 29.944 31.343 31.968 31.732 27.929 26.71 27.09 28.266 26.129 107- (102) HIS166 CG 1.387 3.018  HIS166 ND1 0.916 3.295  HIS166 CD2 2.174 1.896  HIS166 CE1 1.378 2.374  HIS166 NE2 2.156 1.51  GLU167 N -1.239 1.506  GLU167 CA -1.754 0.127  GLU167 C -3.12 0.07  GLU167  -4.04 -0.549  GLU167 CB 0.743 -0.752  GLU167 CG 0.54 -0.919  GLU167 CD 1.593 -1.638  GLU167 E1 1.207 -2.478  GLU167 OE2 2.766 -1.386  PHE168 N -3.329 0.846  PHE168 CA -4.652 0.854  PHE168 C -5.586 1.779  PHE168  -6.692 1.36  PHE168 CB -4.53 1.304  PHE168 CG -5.824 1.165  PHE168 CD1 -6.696 0.117  PHE168 CD2 -6.126 2.078  PHE168 CE1 -7.875 -0.01  PHE168 CE2 -7.305 1.951  PHE168 CZ -8.18 0.908  PHE169 N -5.016 2.851  PHE169 CA -5.773 3.717  PHE169 C -5.944 3.029  PHE169  -7.031 3.098  PHE169 CB -5.003 5.024  PHE169 CG -5.736 6.088  PHE169 CD1 -6.361 7.145  PHE169 CD2 -5.758 6.022  PHE169 CE1 -7.037 8.111  PHE169 CE2 -6.436 6.985  PHE169 CZ -7.082 8.028  GLN170 N -5.019 2.134  GLN170 CA -5.042 1.407  GLN170 C -6.148 0.359  GLN170  -6.903 0.281  GLN170 C -3.705 0.693  GLN170 CG -3.611 -0.121  GLN170 CD -3.411 0.818  GLN170 E1 -2.392 1.516  GLN170 NE2 -4.408 0.885  SER171 N -6.372 -0.281  SER171 CA -7.424 -1.295  SER171 C -8.811 -0.665  SER171  -9.706 -1.107  SER171 C &amp;gt;7.171 -2.209  SER171 G *7.132 -1.427  CYS172 N -8.906 0.511  CYS172 CA -10.204 1.184  CYS172 C -10.569 1.806 108- (103) CYS172  -11.691 1.593  CYS172 CB -10.149 2.266  CYS172 SG -9.817 1.701  SER173 N -9.559 2.253  SER173 CA -9.781 2.826  SER173 C -9.949 1.765  SER173  -10.363 2.096  SER173 CB -8.612 3.741  SER173 G -7.418 2.972  SER174 N -9.687 0.509  SER174 CA -10.001 -0.575  SER174 C -11.395 -1.133  SER174  -12.128 -1.464  SER174 CB -8.949 -1.668  SER174 G -7.691 -1.099  ARG175 N -11.85 -0.974  ARG175 CA -13.224 -1.373  ARG175 C -14.226 -0.342  ARG175  -15.298 -0.72  ARG175 CB -13.371 -1.514  ARG175 CG -12.486 -2.622  ARG175 CD -12.761 -2.871  ARG175 NE -12.544 -1.662  ARG175 CZ -13.06 -1.512  ARG175 NH1 -12.78 -0.421  ARG175 NH2 -13.816 -2.478  MET176 N -13.739 0.876  MET176 CA -14.516 1.951  MET176 C -14.649 1.843  MET176  -15.284 2.71  MET176 CB -13.806 3.259  MET176 CG -13.86 3.657  MET176 SD -13.036 5.228  MET176 CE -13.493 5.972  LEU177 N -14.036 0.846  LEU177 CA -14.184 0.676  LEU177 C -15.219 -0.388  LEU177  -15.593 -0.533  LEU177 CB -12.836 0.29  LEU177 CG -11.824 1.426  LEU177 CD1 -10.568 1.035  LEU177 CD2 -12.4 2.713  SER178 N -15.706 -1.099  SER178 CA -16.693 -2.149  SER178 C -18.117 -1.673  SER178  -18.628 -1.797  SER178 C -16.344 -3.402  SER178 OG 16.323 3.095  ARG179 N -18.765 -1.209  ARG179 CA -20.148 -0.697  ARG179 C -21.17 -1.832  ARG179  -21.743 -2.266  ARG179 C -20.4 0.027  ARG179 CG -19.286 1.01 109- (104)200418849 1349 ARG179 CD -19.098 1.114 13.922  1350 ARG179 NE -18.628 -0.179 13.39  1351 ARG179 CZ -19.337 -0.967 12.576  1352 ARG179 NH1 -18.903 -2.2 12.307  1353 ARG179 NH2 -20.546 -0.583 12.156  1354 GLU180 N -21.375 -2.313 18.405  1355 GLU180 CA -22.261 -3.454 18.627  1356 GLU180 C -23.553 -3.038 19.318  1357 GLU180  -23.676 -1.923 19.833  1358 GLU180 CB -21.517 -4.449 19.508  1359 GLU180 CG -20.175 -4.838 18.899  1360 GLU180 CD -19.442 -5.795 19.828  1361 GLU180 E1 -20.124 -6.592 20.457  1362 GLU180 E2 -18.219 -5.763 19.834  1363 VAL181 N -24.492 -3.969 19.374  1364 VAL181 CA -25.72 -3.744 20.147  1365 VAL181 C -25.463 -4.084 21.616  1366 VAL181  -26.012 -3.452 22.525  1367 VAL181 CB -26.823 -4.627 19.569  1368 VAL181 CG1 -28.119 -4.498 20.362  1369 VAL181 CG2 -27.062 -4.297 18.099  1370 THR182 N -24.438 -4.897 21.822  1371 THR182 CA -23.936 -5.215 23.166  1372 THR182 C -22.767 -4.302 23.547  1373 THR182  -21.827 -4.739 24.222  1374 THR182 CB -23.459 -6.664 23.183  1375 THR182 OG1 -22.348 -6.789 22.302  1376 THR182 CG2 -24.551 -7.622 22.719  1377 ALA183 N -22.882 &amp;gt;3.025 23.214  1378 ALA183 CA -21.777 -2.062 23.352  1379 ALA183 C -21.585 -1.459 24.748  1380 ALA183  -21.031 -0.36 24.855  1381 ALA183 C -21.993 -0.933 22.352  1382 GLU184 N -21.832 -2.224 25.799  1383 GLU184 CA -21.706 -1.688 27.158  1384 GLU184 C -20.242 -1.537 27.568  1385 GLU184  -19.888 -0.536 28.199  1386 GLU184 CB 22.415 -2.636 28.117  1387 GLU184 CG -22.387 -2.118 29.551  1388 GLU184 CD -23.145 -3.086 30.454  1389 GLU184 OE1 -22.924 -3.047 31.655  1390 GLU184 OE2 -23.953 -3.832 29.917  1391 GLU185 N 19.374 2.344 26.978  1392 GLU185 CA 17.936 -2.201 27.244  1393 GLU185 C -17.307 -1.143 26.337  1394 GLU185  -16.294 -0.542 26.708  1395 GLU185 CB -17.216 -3.541 27.073  1396 GLU185 CG -17.189 -4.388 28.351  1397 GLU185 CD -18.561 -4.959 28.705  1398 GLU185 E1 -19.308 -5.237 27.774  1399 GLU185 E2 -18.899 -4.953 29.879  1400 ARG186 N -18.044 -0.74 25.316  1401 ARG186 CA -17.607 0.339 24.434  1402 ARG186 C -18.008 1.676 25.049 110- 4 (105) ARG186  -17.213 2.624  ARG186 CB -18.317 0.135  ARG186 CG -17.936 1.175  ARG186 CD -18.718 0.942  ARG186 NE -18.234 1.82  ARG186 CZ -19.002 2.705  ARG186 NH1 -20.296 2.816  ARG186 NH2 -18.474 3.477  MET187 N -19.086 1.627  MET187 CA -19.545 2.775  MET187 C -18.651 2.978  MET187  -18.251 4.113  MET187 CB -20.958 2.459  MET187 CG -21.569 3.594  MET187 SD -23.104 3.161  MET187 CE -23.995 2.43  THR188 N -18.13 1.874  THR188 CA -17.174 1.925  THR188 C -15.819 2.442  THR188  -15.237 3.307  THR188 CB -17.013 0.516  THR188 G1 -18.274 0.091  THR188 CG2 -16.011 0.473  ALA189 N -15.477 2.139  ALA189 CA -14.243 2.652  ALA189 C -14.333 4.147  ALA189  -13.403 4.884  ALA189 CB -13.996 1.884  PHE190 N -15.518 4.605  PHE190 CA -15.762 6.042  PHE190 C -15.632 6.81  PHE190  -14.815 7.736  PHE190 C -17.194 6.258  PHE190 CG 17.566 5.772  PHE190 CD1 -16.652 5.837  PHE190 CD2 -18.848 5.293  PHE190 CE1 -17.013 5.406  PHE190 CE2 -19.208 4.861  PHE190 CZ -18.291 4.917  GLU191 N -16.281 6.297  GLU191 CA -16.292 6.969  GLU191 C -14.921 6.971  GLU191  -14.432 8.05  GLU191 CB -17.28 6.233  GLU191 CG -17.348 6.853  GLU191 CD -18.28 6.023  GLU191 E1 -18.424 4.843  GLU191 E2 -18.894 6.597  SER192 N -14.202 5.866  SER192 CA -12.877 5.76  SER192 C -11.82 6.545  SER192  -10.892 7.077  SER192 CB -12.467 4.291  SER192 G -12.37 3.801 111 - (106) LEU193 N -12.082 6.827  LEU193 CA &amp;gt;11.152 7.64  LEU193 C -11.441 9.127  LEU193  -10.514 9.943  LEU193 CB -11.243 7.218  LEU193 CG -9.95 6.547  LEU193 CD1 -9.372 5.603  LEU193 CD2 -10.117 5.842  GLU194 N -12.63 9.438  GLU194 CA -12.93 10.811  GLU194 C -12.368 11.096  GLU194  -11.82 12.182  GLU194 CB -14.437 11.017  GLU194 CG -14.97 11.164  GLU194 CD -14.35 12.405  GLU194 E1 &amp;gt;13.524 12.237  GLU194 E2 -14.826 13.487  ASN195 N -12.26 10.058  ASN195 CA -11.645 10.215  ASN195 C -10.126 10.268  ASN195  -9.479 11.112  ASN195 CB -12.076 9.054  ASN195 CG -13.582 9.104  ASN195 D1 -14.272 8.076  ASN195 ND2 14.076 10.3  TYR196 N -9.64 9.613  TYR196 CA -8.236 9.692  TYR196 C -7.88 11.104  TYR196  -6.953 11.707  TYR196 CB -8.065 8.757  TYR196 CG -7.054 7.633  TYR196 CD1 -5.806 7.739  TYR196 CD2 -7.39 6.5  TYR196 CE1 4.884 6.711  TYR196 CE2 -6.466 5.472  TYR196 CZ -5.216 5.581  TYR196 OH 4.301 4.56  LEU197 N 8.76 11.708  LEU197 CA -8.543 13.084  LEU197 C -8.718 14.131  LEU197  -7.966 15.11  LEU197 CB -9.53 13.394  LEU197 CG -9.196 12.63  LEU197 CD1 -10.305 12.816  LEU197 CD2 -7.852 13.072  ASP198 N -9.49 13.83  ASP198 CA -9.6 14.749  ASP198 C -8.257 14.841  ASP198  -7.698 15.938  ASP198 CB -10.627 14.209  ASP198 CG -12.016 14.052  ASP198 D1 -12.428 14.949  ASP198 D2 -12.706 13.123  GLU199 N -7.629 13.685 112- (107)   GLU199 CA -6.334 13.598 34.368  GLU199 C -5.221 14.182 33.506  GLU199  -4.521 15.091 33.965  GLU199 CB 6.053 12.121 34.608  GLU199 CG -7.157 11.486 35.444  GLU199 CD -7.084 9.966 35.341  GLU199 E1 -7.502 9.312 36.287  GLU199 E2 -6.717 9.484 34.277  LEU200 N -5.26 13.885 32.219  LEU200 CA -4.209 14.315 31.289  LEU200 C -4.22 15.821 31.02  LEU200  -3.168 16.467 31.128  LEU200 C -4.459 13.57 29.982  LEU200 CG -3.421 13.888 28.914  LEU200 CD1 -2.036 13.415 29.338  LEU200 CD2 -3.815 13.251 27.587  VAL201 N -5.406 16.402 30.938  VAL201 CA -5.521 17.832 30.642  VAL201 C -5.196 18.69 31.859  VAL201  4.491 19.7 31.709  VAL201 C -6.945 18.094 30.153  VAL201 CG1 -7.324 19.57 30.184  VAL201 CG2 -7.159 17.508 28.761  THR202 N -5.431 18.149 33.045  THR202 CA -5.103 18.884 34.267  THR202 C -3.643 18.677 34.654  THR202  -2.981 19.624 35.101  THR202 CB -6.02 18.391 35.378  THR202 G1 7.359 18.589 34.945  THR202 CG2 -5.814 19.173 36.671  LYS203 N -3.08 17.579 34.182  LYS203 CA -1.672 17.284 34.434  LYS203 C 0.755 18.113 33.539  LYS203  0.305 18.539 34.015  LYS203 C 1.474 15.79 34.209  LYS203 CG - 0.041 15.331 34.439  LYS203 CD 0.025 13.812 34.569  LYS203 CE -0.557 13.099 33.352  LYS203 NZ 0.265 13.325 32.154  LYS204 N -1.256 18.545 32.391  LYS204 CA -0.505 19.493 31.553  LYS204 C -0.788 20.954 31.896  LYS204  -0.031 21.846 31.499  LYS204 C -0.821 19.218 30.092  LYS204 CG -0.076 17.965 29.664  LYS204 CD 1.425 18.213 29.729  LYS204 CE 2.202 16.906 29.69  LYS204 NZ 1.918 16.115 30.896  GLU205 N -1.781 21.179 32.741  GLU205 CA 2.033 22.524 33.264  GLU205 C -1.215 22.777 34.526  GLU205  -1.027 23.931 34.927  GLU205 CB -3.518 22.66 33.57  GLU205 CG -4.309 22.726 32.273</span>B 0.832 -0.979 -2.24 -2.637 -3.385 1.347 1.874 1.433 1.183 1.366 0.869 2.006 2.715 -0.111 -0.763 -1.175 2.15 3.13 2.838 1.68 3.066 1.972 1.358 3.882 3.793 3.104 2.72 5.192 6.111 6.895 6.181 7.726 7.011 7.78 8.589 2.806 2.007 0.51 -0.319 2.303 1.766 0.173 -1.215 -1.445 -2.542 -1.463 -1.331 -2.358 -0.21  0.343 -0.299 -0.63 -0.515 1.11 16.594 14.12 14.827 15.915 13.84 14.417 14.572 13.413 13.581 12.238 11.037 10.078 9.58 10.348 9.125 11.327 9.806 8.826 7.41 7.021 8.899 9.874 10.384 6.593 5.209 4.257 3.153 4.684 4.367 5.363 3.067 5.066 2.768 3.77 3.478 4.747 3.992 4.158 3.566 4.538 5.854 5.056 5.273 4.619 4.126 6.775 7.38 7.592 7.683 4.475 3.666 2.223 1.86 3.792 24.66 23.044 23.524 22.535 23.707 24.943 26.306 27.2 28.398 26.601 27.284 27.635 26.752 26.338 26.976 25.866 28.922 29.4 28.901 28.689 30.894 31.302 30.009 28.912 28.41 29.393 29.002 28.071 29.254 29.826 29.743 30.898 30.815 31.392 32.467 30.589 31.561 31.301 32 32.952 33.017 30.389 30.002 28.649 28.344 29.944 31.343 31.968 31.73 2 27.929 26.71 27.09 28.266 26.129 107- (102) HIS166 CG 1.387 3.018 HIS166 ND1 0.916 3.295 HIS166 CD2 2.174 1.896 HIS166 CE1 1.378 2.374 HIS166 NE2 2.156 1.51 GLU167 N -1.239 1.506 GLU167 CA -1.754 0.127 GLU167 C -3.12 0.07 4.04 GLU167 -0.549 GLU167 CB  0.743 -0.752 GLU167 CG 0.54 -0.919 GLU167 CD 1.593 -1.638 GLU167 E1 1.207 -2.478 GLU167 OE2 2.766 -1.386 PHE168 N -3.329 0.846 PHE168 CA -4.652 0.854 PHE168 C -5.586 1.779 PHE168 -6.692 1.36 PHE168 CB -4.53 1.304 PHE168 CG -5.824 1.165 PHE168 CD1 -6.696 0.117 PHE168 CD2 -6.126 2.078 PHE168 CE1 -7.875 -0.01 PHE168 CE2 -7.305 1.951 PHE168 CZ -8.18 0.908 PHE169 N -5.016 2.851 PHE169 CA -5.773 3.717 PHE169 C -5.944 3.029 PHE169 -7.031 3.098 PHE169 CB -5.003 5.024 PHE169 CG -5.736 6.088 PHE169 CD1 -6.361 7.145 PHE169 CD2 -5.758 6.022 PHE169 CE1 -7.037 8.111 PHE169 CE2 -6.436 6.985 PHE169 CZ -7.082 8.028 GLN170 N -5.019 2.13442 GLN407 CA -5.0 GLN170 C -6.148 0.359 GLN170 -6.90 3 0.281 GLN170 C  -3.705 0.693 GLN170 CG -3.611 -0.121 GLN170 CD -3.411 0.818 GLN170 E1 -2.392 1.516 GLN170 NE2 -4.408 0.885 SER171 N -6.372 -0.281 SER171 CA -7.424 -1.295 SER171 C -8.811 -0.665 SER171  -9.706 -1.107 SER171 C  &amp;gt; 7.171 -2.209 SER171 G * 7.132 -1.427 CYS172 N -8.906 0.511 CYS172 CA -10.204 1.184 CYS172 C -10.569 1.806 108- (103) CYS172 -11.691 1.593 CYS172 CB -10.149 2.266 CYS172 SG -9.817 1.701 SER173 N -9.559 2.253 SER173 CA -9.781 2.826 SER173 C -9.949 1.765 SER173 -10.363 2.096 SER173 CB -8.612 3.741 SER173 G -7.418 2.972 SER174 N -9.687 0.509 SER174 CA -10.001 -0.575 SER174 C -11.395 -1.133 SER174 -12.128 -1.464 SER174 CB -8.949 -1.668 SER174 G -7.691 -1.099 ARG175 N -11.85 -0.974 ARG175 CA -13.224 -1.373 ARG175 C -14.226 -0.342 ARG175 -15.298 -0.72 ARG175 CB -13.371- 1.514 ARG175 CG -12.486 -2.622 ARG175 CD -12.761 -2.871 ARG175 NE -12.544 -1.662 ARG175 CZ -13.06 -1.512 ARG175 NH1 -12.78 -0.4 21 ARG175 NH2 -13.816 -2.478 MET176 N -13.739 0.876 MET176 CA -14.516 1.951 MET176 C -14.649 1.843 MET176 -15.284 2.71 MET176 CB -13.806 3.259 MET176 CG -13.86 3.657 MET176 SD -13.036 5.228 MET176 CE -13.493 5.972 LE 14.036 0.846 LEU177 CA -14.184 0.676 LEU177 C -15.219 -0.388 LEU177 -15.593 -0.533 LEU177 CB -12.836 0.29 LEU177 CG -11.824 1.426 LEU177 CD1 -10.568 1.035 LEU177 CD2 -12.4 2.713 SER178 N -15.706 -1.099 SER178 CA -16.693 2.149 SER178 C -18.117 -1.673 SER178 -18.628 -1.797 SER178 C  -16.344 -3.402 SER178 OG 16.323  3.095 ARG179 N -18.765 -1.209 ARG179 CA -20.148 -0.697 ARG179 C -21.17 -1.832 ARG179 -21.743 -2.266 ARG179 C  -20.4 0.027 ARG179 CG -19.286 1.01 109- (104) 200418849 1349 ARG179 CD -19.098 1.114 13.922 1350 ARG179 NE -18.628 -0.179 13.39 1351 ARG179 CZ -19.337 -0.967 12.576 1352 ARG179 NH1 -18.903 -2.2 12.307 1353 ARG179 NH2 -20.546 -0.583 12.156 1354 GLU180 N -21.375 -2.313 18.405 1355 GLU180 CA -22. 261 -3.454 18.627 1356 GLU180 C -23.553 -3.038 19.318 1357 GLU180 -23.676 -1.923 19.833 1358 GLU180 CB -21.517 -4.449 19.508 1359 GLU180 CG -20.175 -4.838 18.899 1360 GLU180 CD -19.442 -5.795 19.828 1361 GLU180 E1 -20.124 -6.592 20.457 1362 GLU180 E2 -18.219 -5.763 19.834 1363 VAL181 N -24.492 -3.969 19.374 1364 VAL181 CA -25.72 -3.744 20.147 1365 VAL181 C -25.463 -4.084 21.616 1366 VAL181 -26.012 -3.452 22.525 1367 VAL181 CB -26.823- 4.627 19.569 1368 VAL181 CG1 -28.119 -4.498 20.362 1369 VAL181 CG2 -27.062 -4.297 18.099 1370 THR182 N -24.438 -4.897 21.822 1371 THR182 CA -23.936 -5.215 23.166 1372 THR182 C -22.767 -4.302 23.547 1373 THR182 -21.827 -4.739 24 1374 THR182 CB -23.459 -6.664 23.183 1375 THR182 OG1 -22.348 -6.789 22.302 1376 THR182 CG2 -24.551 -7.622 22.719 1377 ALA183 N -22.882 &amp;gt; 3.025 23.214 1378 ALA183 CA -21.777 -2.062 23.352 1379 ALA183 C -21.585 -1.459 24.748 1 ALA183 -21.031 -0.36 24.855 1381 ALA183 C  -21.99 3 -0.933 22.352 1382 GLU184 N -21.832 -2.224 25.799 1383 GLU184 CA -21.706 -1.688 27.158 1384 GLU184 C -20.242 -1.537 27.568 1385 GLU184 -19.888 -0.536 28.199 1386 GLU184 CB  22.415 -2.636 28.117 1387 GLU184 CG -22.387 2.118 29.551 1388 GLU184 CD -23.145 -3.086 30.454 1389 GLU184 OE1 -22.924 -3.047 31.655 1390 GLU184 OE2 -23.953 -3.832 29.917 1391 GLU185 N  19.374  2.344 26.978 1392 GLU185 CA 17.936 -2.201 27.244 1393 GLU185 C-17 1394 GLU185 -16.294 -0.542 26.708 1395 GLU185 CB -17.216 -3.541 27.073 1396 GLU185 CG -17.189 -4.388 28.351 1397 GLU185 CD -18.561 -4.959 28.705 1398 GLU185 E1 -19.308 -5.237 27.774 1399 GLU185 E2 -29.899 1400 ARG186 N -18.044 -0.74 25.316 1401 ARG186 CA -17.607 0.339 24.434 1402 ARG186 C -18.008 1.676 25.049 110- 4 (105) ARG186 -17.213 2.624 ARG186 CB -18.317 0.135 ARG186 CG -17.936 1.175 ARG186 CD -18.718 0.942 ARG -18.234 1.82 ARG186 CZ -19.002 2.705 ARG186 NH1 -20 .296 2.816 ARG186 NH2 -18.474 3.477 MET187 N -19.086 1.627 MET187 CA -19.545 2.775 MET187 C -18.651 2.978 MET187 -18.251 4.113 MET187 CB -20.958 2.459 MET187 CG -21.569 3.594 MET187 SD -23.104 3.161 MET187 CE -23.995 2.943 -18.13 1.874 THR188 CA -17.174 1.925 THR188 C -15.819 2.442 THR188 -15.237 3.307 THR188 CB -17.013 0.516 THR188 G1 -18.274 0.091 THR188 CG2 -16.011 0.473 ALA189 N -15.477 2.139 ALA189 CA -14.243 2.652 ALA189 C -13.403 4.884 ALA189 CB -13.996 1.884 PHE190 N -15.518 4.605 PHE190 CA -15.762 6.042 PHE190 C -15.632 6.81 PHE190 -14.815 7.736 PHE190 C  -17.194 6.258 PHE190 CG  17.566 5.772 PHE190 CD1 -16.652 5.837 PHE190 CD2 -18 PHE190 CE1 -17.013 5.406 PHE190 CE2 -19.208 4.861 PHE190 CZ -18.291 4.917 GLU191 N -16.281 6.297 GLU191 CA -16.292 6.969 GLU191 C -14.921 6.971 GLU191 -14.432 8.05 GLU191 CB -17.28 6.233 GLU191 CG -17.348 6.-18 GLU GLU191 E 1 -18.424 4.843 GLU191 E2 -18.894 6.597 SER192 N -14.202 5.866 SER192 CA -12.877 5.76 SER192 C -11.82 6.545 SER192 -10.892 7.077 SER192 CB -12.467 4.291 SER192 G -12.37 3.801 111-(106) LEU193 N -12.082 6.827 LEU193 CA &amp;gt; 11.152 7.64 LEU193 C -11.441 9.127 LEU193 -10.514 9.943 LEU193 CB -11.243 7.218 LEU193 CG -9.95 6.547 LEU193 CD1 -9.372 5.603 LEU193 CD2 -10.117 5.842 GLU194 N -12.63 9.438 GLU194 CA -12.93 10.811 12.368 11.096 GLU194 -11.82 12.182 GLU194 CB -14.437 11.017 GLU194 CG -14.97 11.164 GLU194 CD -14.35 12.405 GLU194 E1 &amp;gt; 13.524 12.237 GLU194 E2 -14.826 13.487 ASN195 N -12.26 10.058 ASN195 CA -11.645 10.215 ASN195 ASN195 -9.479 11.112 ASN195 CB -12.076 9.054 ASN195 CG -13.582 9.104 ASN195 D1 -14.272 8.076 ASN195 ND2  14.076 10.3 TYR196 N -9.64 9.613 TYR196 CA -8.236 9.692 TYR196 C -7.88 11.104 TYR196 -8.0-5.953 11.65 8.757 TYR196 CG -7.054 7.633 TYR196 CD1 -5.806 7.739 TYR196 CD2 -7.39 6.5 TYR196 CE1  4.884 6.711 TYR196 CE2 -6.466 5.472 TYR196 CZ -5.216 5.581 TYR196 OH  4.301 4.56 LEU197 N  8.76 11.708 LEU197 CA -8.543 13.084 LEU197 C -8.718 14.131 LEU197 -7.966B 15. -9.53 13.394 LEU197 CG -9.196 12.63 LEU197 CD1 -10.305 12.816 LEU197 CD2 -7.852 13.072 ASP198 N -9.49 13.83 ASP198 CA -9.6 14.749 ASP198 C -8.257 14.841 ASP198 -7.698 15.938 ASP198 CB -10.627 14.209 ASP198 CG -12.016 14.052 ASP198  D1 -12.428 14.949 ASP198 D2 -12.706 13.123 GLU199 N -7.629 13.685 112- (107) GLU199 CA -6.334 13.598 34.368 GLU199 C -5.221 14.182 33.506 GLU199 -4.521 15.091 33.965 GLU199 CB  6.053 12.121 34.608 GLU199 CG -7.157 11.486 35.444 GLU199 CD -7.084 9.966 35.341 GLU199 E1 -7.502 9.312 36.287 GLU199 E2 -6.717 9.484 34.277 LEU200 N -5.26 13.885 32.219 LEU200 CA -4.209 14.315 31.289 LEU200 C -4.22 15.821 31.02 LEU200 -3.168 16.467 31.128 LE459200 C 29.982 LEU2 00 CG -3.421 13.888 28.914 LEU200 CD1 -2.036 13.415 29.338 LEU200 CD2 -3.815 13.251 27.587 VAL201 N -5.406 16.402 30.938 VAL201 CA -5.521 17.832 30.642 VAL201 C -5.196 18.69 31.859 VAL201   4.491 19.7 31.709 VAL201 C  -6153 945 18.094 CG1 -7.324 19.57 30.184 VAL201 CG2 -7.159 17.508 28.761 THR202 N -5.431 18.149 33.045 THR202 CA -5.103 18.884 34.267 THR202 C -3.643 18.677 34.654 THR202 -2.981 19.624 35.101 THR202 CB -6.02 18.391 35.378 THR202 18.589 349951CG17.31 -5.814 19.173 36.671 LYS203 N -3.08 17.579 34.182 LYS203 CA -1.672 17.284 34.434 LYS203 C 0.755 18.113 33.539 LYS203 0.305 18.539 34.015 LYS203 C   1.474 15.79 34.209 LYS203 CG-0.041 15.331 34.439 LYS203 CD 0.525 13.099 13.099 LYS203 NZ 0.265 13.325 32.154 LYS204 N -1.256 18.545 32.391 LYS204 CA -0.505 19.493 31.553 LYS204 C -0.788 20.954 31.896 LYS204 -0.031 21.846 31.499 LYS204 C  -0.821 19.218 30.092 LYS204 CG -0.076 17.965 29.664 LYS204 CD 1.425 18.213 29.729 LYS204 CE 2.202 16.906 29.69 LYS204 NZ 1.918 16.115 30.896 GLU205 N -1.781 21.179 32.741 GLU205 CA  2.033 22.524 33.264 GLU205 C -1.215 22.777 34.526 GLU205 -1.027 23.66 34.927 34.927 34.927 GLU205 CG -4.309 22.726 32.273</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">-113- (108)   GLU205 CD -5.788 22.444 32.51  GLU205 E1 -6.547 22.661 31.573  GLU205 E2 -6.093 21.798 33.504  ALA206 N -0.712 21.707 35.12  ALA206 CA 0.208 21.844 36.249  ALA206 C 1.649 21.772 35.756  ALA206  2.464 22.661 36.034  ALA206 C -0.059 20.711 37.233  ASN207 N 1.92 20.765 34.945  ASN207 CA 3.253 20.587 34.366  ASN207 C 3.307 21.204 32.976  ASN207  2.922 20.574 31.982  ASN207 CB 3.568 19.096 34.27  ASN207 CG 3.565 18.441 35.65  ASN207 D1 4.361 18.793 36.527  ASN207 ND2 2.641 17.515 35.835  ALA208 N 3.786 22.435 32.932  ALA208 CA 3.924 23.166 31.668  ALA208 C 5.045 22.601 30.795  ALA208  6.235 22.809 31.052  ALA208 C 4.202 24.631 31.982  THR209 N 4.636 21.874 29.77  THR209 CA 5.581 21.251 28.834  THR209 C 5.499 21.873 27.446  THR209  4.665 22.737 27.168  THR209 CB 5.253 19.774 28.706  THR209 G1 3.938 19.693 28.18  THR209 CG2 5.296 19.059 30.052  GLU210 N 6.34 21.363 26.562  GLU210 CA 6.411 21.861 25.18  GLU210 C 5.542 21.076 24.19  GLU210  5.586 21.361 22.989  GLU210 C 7.861 21.818 24.692  GLU210 CG 8.791 22.752 25.467  GLU210 CD 9.689 21.967 26.424  GLU210 E1 9.258 20.906 26.861  GLU210 E2 10.778 22.442 26.707  ASP211 N 4.786 20.097 24.662  ASP211 CA 3.977 19.273 23.746  ASP211 C 2.732 20.003 23.249  ASP211  2.331 21.027 23.817  ASP211 CB 3.605 17.956 24.418  ASP211 CG 2.942 18.154 25.781  ASP211 D1 2.12 19.054 25.925  ASP211 OD2 3.336 17.427 26.681  ASP212 N 2.034 19.382 22.309  ASP212 CA 0.906 20.044 21.64  ASP212 C -0.365 20.067 22.496  ASP212  -1.223 20.928 22.262  ASP212 CB 0.653 19.331 20.312  ASP212 CG 0.355 20.078 19.435  ASP212 D1 -0.505 21.277 19.623  ASP212 D2 -1.022 19.415 18.653  LEU213 N -0.398 19.318 23.589</span>-113- (108) GLU205 CD -5.788 22.444 32.51 GLU205 E1 -6.547 22.661 31.573 GLU205 E2 -6.093 21.798 33.504 ALA206 N -0.712 21.707 35.12 ALA206 CA 0.208 21.844 36.249 ALA206 C 1.649 21.772 35.756 ALA206 2.464 22.661 36.034 ALA -0.059 20.711 37.233 ASN207 N 1.92 20.765 34.945 ASN207 CA 3.253 20.587 34.366 ASN207 C 3.307 21.204 32.976 ASN207 2.922 20.574 31.982 ASN207 CB 3.568 19.096 34.27 ASN207 CG 3.565 18.441 35.65 ASN207 D1 4.361 18.793 36.527 3.835 ASN207 ND2 ALA208 CA 3.924 23.166 31.668 ALA208 C 5.045 22.601 30.795 ALA208 6.235 22.809 31.052 ALA208 C  4.202 24.631 31.982 THR209 N 4.636 21.874 29.77 THR209 CA 5.581 21.251 28.834 THR209 C 5.499 21.873 27.446 THR209 4.665 22.737 27.168 THR209C HR209C 19.693 28.18 THR209 CG2 5.296 19.059 30.052 GLU210 N 6.34 21.363 26.562 GLU210 CA 6.411 21.861 25.18 GLU210 C 5.542 21.076 24.19 GLU210 5.586 2 1.361 22.989 GLU210 C  7.861 21.818 24.692 GLU210 CG 8.791 22.752 25.467 GLU210 CD 9.689 21.967 26.424 GLU210 E1 9.258 20.906 26.861 GLU210 E2 10.778 22.442 26.707 ASP211 N 4.786 20.097 24.662 ASP211 CA 3.977 19.273 23.746 ASP211 C 2.732 20.2.3 23.249 ASP211 ASP211 CB 3.605 17.956 24.418 ASP211 CG 2.942 18.154 25.781 ASP211 D1 2.12 19.054 25.925 ASP211 OD2 3.336 17.427 26.681 ASP212 N 2.034 19.382 22.309 ASP212 CA 0.906 20.044 21.64 ASP212 C -0.365 20.067 22.496 ASP212 -1.223 20.928 22.262 ASP212 CB 0.653 19.331 20.312  0.355 20.078 19.435 ASP212 D1 -0.505 21.277 19.623 ASP212 D2 -1.022 19.415 18.653 LEU213 N -0.398 19.318 23.589</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">114- (109)   LEU213 CA -1.529 19.451 24.508  LEU213 C -1.474 20.819 25.18  LEU213  -2.372 21.628 24.907  LEU213 CB -1.498 18.346 25.557  LEU213 CG -2.75 18.365 26.427  LEU213 CD1 -4.011 18.334 25.574  LEU213 CD2 -2.751 17.202 27.41  LEU214 N -0.306 21.193 25.691  LEU214 CA -0.187 22.52 26.307  LEU214 C -0.094 23.607 25.241  LEU214  -0.613 24.711 25.445  LEU214 C 1.034 22.608 27.21  LEU214 CG 0.987 23.94 27.954  LEU214 CD1 -0.18 23.972 28.934  LEU214 CD2 2.288 24.25 28.671  GLY215 N 0.35 23.222 24.056  GLY215 CA 0.292 24.093 22.882  GLY215 C -1.121 24.618 22.629  GLY215  -1.327 25.838 22.586  ARG216 N -2.103 23.733 22.582  ARG216 CA -3.473 24.198 22.351  ARG216 C -4.154 24.727 23.616  ARG216  -5.077 25.542 23.506  ARG216 C -4.3 23.068 21.765  ARG216 CG -3.636 22.456 20.539  ARG216 CD -4.555 21.415 19.912  ARG216 NE -5.402 20.812 20.953  ARG216 CZ -5.073 19.756 21.699  ARG216 NH1 -3.937 19.093 21.471  ARG216 NH2 -5.905 19.342 22.652  GLN217 N -3.583 24.445 24.776  GLN217 CA -4.102 25.002 26.031  GLN217 C -3.624 26.435 26.28  GLN217  -4.198 27.131 27.125  GLN217 CB &amp;gt;3.648 24.109 27.181  GLN217 CG -4.235 22.711 27.043  GLN217 CD -3.691 21.772 28.114  GLN217 OE1 -2.544 21.311 28.052  GLN217 NE2 -4.552 21.446 29.059  ILE218 N -2.608 26.875 25.551  ILE218 CA -2.179 28.276 25.625  ILE218 C -2.638 29.086 24.409  ILE218  -2.242 30.25 24.267  ILE218 C -0.66 28.352 25.774  ILE218 CG1 0.061 27.755 24.572  ILE218 CG2 -0.211 27.665 27.059  ILE218 CD1 1.574 27.781 24.754  LEU219 N -3.431 28.478 23.538  LEU219 CA -3.953 29.194 22.365  LEU219 C -4.858 30.352 22.75  LEU219  -5.716 30.229 23.629  LEU219 CB -4.756 28.24 21.493  LEU219 CG -3.859 27.411 20.59  LEU219 CD1 -4.674 26.345 19.873</span>114- (109) LEU213 CA -1.529 19.451 24.508 LEU213 C -1.474 20.819 25.18 LEU213 -2.372 21.628 24.907 LEU213 CB -1.498 18.346 25.557 LEU213 CG -2.75 18.365 26.427 LEU213 CD1 -4.011 18.334 25.574 LEU213 CD2 -2.751214 N 202.41 LEU213 0.306 21.193 25.691 LEU214 CA -0.187 22.52 26.307 LEU214 C -0.094 23.607 25.241 LEU214 -0.613 24.711 25.445 LEU214 C  1.034 22.608 27.21 LEU214 CG 0.987 23.94 27.954 LEU214 CD1 -0.18 23.972 28.934 LEU214 CD2 2.288 24.25 28.215 GYLY. GLY 0.292 24.093 22.882 GLY215 C -1.121 24.618 22.629 GLY215 -1.327 25.838 22.586 ARG216 N -2.103 23.733 22.582 ARG216 CA -3.473 24.198 22.351 ARG216 C -4.154 24.727 23.616 ARG216 -5.077 25.542 23.506 ARG216 C  -4.3 23.068 21.765 ARG216 CG 22.456 20.539 ARG216 CD -4.555 21.415 19.912 ARG216 NE -5.402 20.812 20.953 ARG216 CZ -5.073 19.756 21.699 ARG216 NH1 -3.937 19.093 21.471 ARG216 NH2 -5.905 19.342 22.652 GLN217 N -3.583 24.445 24.776 GLN21 7 CA -4.102 25.002 26.031 GLN217 C -3.624 26.435 26.28 GLN217 -4.198 27.131 27.125 GLN217 CB &amp;gt; 3.648 24.109 27.181 GLN217 CG -4.235 22.711 27.043 GLN217 CD -3.691 21.772 28.114 GLN217 OE1-2.54.521.21. N -2.608 26.875 25.551 ILE218 CA -2.179 28.276 25.625 ILE218 C -2.638 29.086 24.409 ILE218 -2.242 30.25 24.267 ILE218 C  -0.66 28.352 25.774 ILE218 CG1 0.061 27.755 24.572 ILE218 CG2 -0.211 27.665 27.59 1.754 UILE218 CD1 28.478 23.538 LEU219 CA -3.953 29.194 22.365 LEU219 C -4.858 30.352 22.75 LEU219 -5.716 30.229 23.629 LEU219 CB -4.756 28.24 21.493 LEU219 CG -3.859 27.411 20.59 LEU219 CD1 -4.674 26.345 19.873</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">-115- (110) (110)   LEU219 CD2  LYS220 N  LYS220 CA  LYS220 C  LYS220   LYS220 C  LYS220 CG  LYS220 CD  LYS220 CE  LYS220 NZ  GLN221 N  GLN221 CA  GLN221 C  GLN221   GLN221 C  GLN221 CG  GLN221 CD  GLN221 E1  GLN221 NE2  ARG222 N  ARG222 CA  ARG222 C  ARG222   ARG222 CB  ARG222 CG  ARG222 CD  ARG222 NE  ARG222 CZ  ARG222 NH1  ARG222 NH2  GLU223 N  GLU223 CA  GLU223 C  GLU223   GLU223 CB  GLU223 CG  GLU223 CD  GLU223 E1  GLU223 E2  SER224 N  SER224 CA  SER224 C  SER224   SER224 CB  SER224 G  GLY225 N  GLY225 CA  GLY225 C  GLY225   GLU226 N  GLU226 CA  GLU226 C  GLU226   GLU226 C -3.135 -4.667 -5.484 -5.341 -4.556 -5.01 -5.91 -5.389 -6.258 -5.74 6.048 5.987 -6.904 -7.981 -6.389 -5.417 -5.828 -7.019 -4.823 6.553 -7.309 -8.774 -9.669 -6.649 -7.362 -6.787 -7.505 -8.368 -8.626 -8.98 -9.009 -10.382 -10.985 -12.083 -10.395 -9.977 -9.946 -9.041 -10.749 -10.28 -10.803 -11.036 -11.791 -9.789 -9.509 -10.409 -10.436 -9.539 -8.577 -9.863 -9.128 10.067 -11.22 -8.424 28.304 31.454 32.654 33.562 34.519 33.38 34.56 35.289 36.494 37.191 33.217 34.031 35.234 35.112 33.179 32.015 31.088 30.872 30.508 36.345 37.589 37.467 37.72 38.705 40.034 41.134 42.401 42.909 42.255 44.068 36.864 36.702 35.329 35.029 36.959 38.396 38.649 39.35 38.047 34.5 33.162 32.957 32.067 32.13 32.374 33.796 33.613 32.435 32.109 31.807 30.613 29.417 29.553 30.894 19.588 22.047 22.234 21.012 21.016 23.489 23.842 25.074 25.418 26.607 19.951 18.732 18.892 19.48 17.535 17.369 16.227 15.966 15.595 18.268 18.486 18.066 18.877 17.685 17.911 17.026 17.232 16.349 15.213 16.604 16.912 16.426 16.735 16.257 14.926 14.634 13.13 12.701 12.434 17.488 17.775 19.266 19.67 17.308 15.941 20.069 21.517 21.884 21.174 22.998 23.422 23.535 23.963 24.745</span>-115- (110) (110) LEU219 CD2 LYS220 N LYS220 CA LYS220 C LYS220 LYS220 C  LYS220 CG LYS220 CD LYS220 CE LYS220 NZ GLN221 N GLN221 CA GLN221 C GLN221 GLN221 C  GLN221 CG GLN CD ARG222 N ARG222 CA ARG222 C ARG222 ARG222 CB ARG222 CG ARG222 CD ARG222 NE ARG222 CZ ARG222 NH1 ARG222 NH2 GLU223 N GLU223 CA GLU223 C GLU223 GLU223 CB GLU223 CG GLU223 CD GLU223 E1 GSER223 224 SER224 CB SER224 G GLY225 N GLY225 CA GLY225 C GLY225 GLU226 N GLU226 CA GLU226 C GLU226 GLU226 C  -3.135 -4.667 -5.484 -5.341 -4.556 -5.01 -5.91 -5.389 -6.258 -5.74  6.048  5.987 -6.904 -7.981 -6.389 -5.417 -5.828 -7.019 -4.823  6.553 -7.309 -8.774 -9.669 -6.649 -7.362 -6.787 -7.505 -8.368 -8.626 -8.98 -9.009 -10.382 -10.985 -12.083 -10.395 -9.977 -9.946 -9.041 -10.749 -10.28 -10.803 -11.036 -11.791 -9.789 -9.509 -10.409 -10.436 -9.539 -8.577 -9.863 -9.128  1 0.067 -11.22 -8.424 28.304 31.454 32.654 33.562 34.519 33.38 34.56 35.289 36.494 37.191 33.217 34.031 35.234 35.112 33.179 32.015 31.088 30.872 30.508 36.345 37.589 37.467 37.467. 3.3.2.359.359.36.359.359. 33.796 33.613 32.435 32.109 31.807 30.613 29.417 29.553 30.894 19.588 22.047 22.234 21.012 21.016 23.489 23.842 25.074 25.418 26.607 19.951 18.732 18.892 19.48 17.535 17.369 16.227 15.966 15.595 18.268 18.486 18.066 18.877 17.685 17.911 16.026 16.232 16.349 15.213 16.604 16.912 16.426 16.726 16.735 16.735 19.266 19.67 17.308 15.941 20.069 21.517 21.884 21.174 22.998 23.422 23.535 23.963 24.745</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">-116- (111)   GLU226 CG -9.368 31.316 25.86  GLU226 CD -8.548 31.714 27.084  GLU226 E1 -8.364 30.873 27.952  GLU226 E2 -8.166 32.875 27.147  ALA227 N -9.567 28.262 23.127  ALA227 CA -10.372 27.035 23.163  ALA227 C -10.725 26.657 24.598  ALA227  -9.859 26.603 25.477  ALA227 CB -9.587 25.904 22.508  ASP228 N -12.009 26.456 24.833  ASP228 CA -12.471 26.085 26.175  ASP228 C -12.163 24.618 26.455  ASP228  -11.944 23.836 25.52  ASP228 C -13.961 26.399 26.323  ASP228 CG -14.816 25.698 25.268  ASP228 OD1 -14.745 24.475 25.201  ASP228 D2 -15.656 26.364 24.686  HIS229 N 12.329 24.215 27.704  HIS229 CA -11.958 22.854 28.129  HIS229 C -12.84 21.733 27.559  HIS229  -12.334 20.622 27.37  HIS229 C -11.971 22.807 29.658  HIS229 CG -13.265 23.255 30.319  HIS229 ND1 -14.34 22.487 30.584  HIS229 CD2 -13.557 24.519 30.78  HIS229 CE1 -15.293 23.238 31.171  HIS229 NE2 -14.808 24.494 31.292  GLY230 N -14.024 22.066 27.068  GLY230 CA -14.88 21.073 26.412  GLY230 C -14.27 20.684 25.07  GLY230  -13.958 19.508 24.84  GLU231 N -13.866 21.704 24.329  GLU231 CA -13.239 21.508 23.021  GLU231 C -11.809 21.001 23.152  GLU231  -11.365 20.21 22.315  GLU231 CB -13.196 22.856 22.318  GLU231 CG -14.589 23.417 22.086  GLU231 CD -14.471 24.888 21.707  GLU231 OE1 -15.12 25.296 20.756  GLU231 E2 -13.758 25.592 22.418  LEU232 N -11.181 21.28 24.281  LEU232 CA -9.822 20.803 24.524  LEU232 C -9.841 19.292 24.755  LEU232  -9.061 18.578 24.112  LEU232 C -9.297 21.541 25.756  LEU232 CG -7.777 21.685 25.784  LEU232 CD1 -7.057 20.385 26.124  LEU232 CD2 -7.256 22.293 24.486  VAL233 N -10.877 18.804 25.419  VAL233 CA -11.016 17.36 25.627  VAL233 C -11.406 16.646 24.331  VAL233  10.729 15.676 23.963  VAL233 CB -12.066 17.13 26.71  VAL233 CG1 -12.478 15.667 26.802</span>-116- (111) GLU226 CG -9.368 31.316 25.86 GLU226 CD -8.548 31.714 27.084 GLU226 E1 -8.364 30.873 27.952 GLU226 E2 -8.166 32.875 27.147 ALA227 N -9.567 28.262 23.127 ALA227 CA -10.372 27.035 23.163 ALA227 C 657 ALA227 -9.859 26.603 25.477 ALA227 CB -9.587 25.904 22.508 ASP228 N -12.009 26.456 24.833 ASP228 CA -12.471 26.085 26.175 ASP228 C -12.163 24.618 26.455 ASP228 -11.944 23.836 25.52 ASP228 C  -13.961 26.399 26.323 ASP228 CG -25 268 268 OD1 -14.745 24.475 25.201 ASP228 D2 -15.656 26.364 24.686 HIS229 N  12.329 24.215 27.704 HIS229 CA -11.958 22.854 28.129 HIS229 C -12.84 21.733 27.559 HIS229 -12.334 20.622 27.37 HIS229 C  -11.971229 22.255 29IS 658 HIS229 ND1 -14.34 22.487 30.584 HIS229 CD2 -13.557 24.519 30.78 HIS229 CE1 -15.293 23.238 31.171 HIS229 NE2 -14.808 24.494 31.292 GLY230 N -14.024 22.066 27.068 GLY230 CA -14.88 21.073 26.412 GLY230 C -14.27 20.684 25.0 7 GLY230 -13.958 19.508 24.84 GLU231 N -13.866 21.704 24.329 GLU231 CA -13.239 21.508 23.021 GLU231 C -11.809 21.001 23.152 GLU231 -11.365 20.21 22.315 GLU231 CB -13.196 22.856 22.318 GLU231 CG -14.589 23.417 22.086 GLU231 CD 21.-14. OE1 -15.12 25.296 20.756 GLU231 E2 -13.758 25.592 22.418 LEU232 N -11.181 21.28 24.281 LEU232 CA -9.822 20.803 24.524 LEU232 C -9.841 19.292 24.755 LEU232 -9.061 18.578 24.112 LEU232 C  -9.297 21.541 25.756 LEU232 685 -7.7 CD1 -7.057 20.385 26.124 LEU232 CD2 -7.256 22.293 24.486 VAL233 N -10.877 18.804 25.419 VAL233 CA -11.016 17.36 25.627 VAL233 C -11.406 16.646 24.331 VAL233   10.729 15.676 23.963 VAL233 CB -12.066 17.13 26.71802. VAL233 15.CG1 -12</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">-117 (112)</span>-117 (112)</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">VAL233 CG2  GLY234 N  GLY234 CA  GLY234 C  GLY234   LEU235 N  LEU235 CA  LEU235 C  LEU235   LEU235 CB  LEU235 CG  LEU235 CD1  LEU235 CD2  ALA236 N  ALA236 CA  ALA236 C  ALA236   ALA236 CB  PHE237 N  PHE237 CA  PHE237 C  PHE237   PHE237 CB  PHE237 CG  PHE237 CD1  PHE237 CD2  PHE237 CE1  PHE237 CE2  PHE237 CZ  LEU238 N  LEU238 CA  LEU238 C  LEU238   LEU238 C  LEU238 CG  LEU238 CD1  LEU238 CD2  LEU239 N  LEU239 CA  LEU239 C  LEU239   LEU239 CB  LEU239 CG  LEU239 CD1  LEU239 CD2  LEU240 N  LEU240 CA  LEU240 C  LEU240   LEU240 CB  LEU240 CG  LEU240 CD1  LEU240 CD2  ILE241 N -11.571 -12.258 -12.641 -11.443 -11.116 -10.687 -9.532 -8.375 -7.846 -9.04 -10.08 -9.781 -10.2 -8.162 -7.065 -7.38 -6.525 -6.861 -8.66 -9.11 -8.956 -8.27 -10.58 -11.12 10.824 -11.885 -11.305 -12.366 -12.076 -9.329 9.272 -7.845 -7.532 -10.056 -11.539 -12.279 -12.145 -6.947 -5.551 -4.861 -4.202 -4.821 -5.364 -4.722 -5.169 -5.275 4.667 -5.217 -4.445 -4.952 -3.966 -2.53 -4.227 -6.425 17.632 17.278 16.729 16.59 15.477 17.668 17.691 16.809 16.046 19.13 20.034 21.505 19.75 16.713 15.885 14.407 13.633 16.193 14.095 12.723 12.273 11.274 12.657 11.24 10.394 10.786 9.095 9.486 8.644 13.135 12.754 12.681 11.745 13.766 13.75 14.795 12.369 13.467 13.397 12.158 11.444 14.652 15.898 17.162 15.82 11.736 10.56 9.265 8.316 10.634 9.808 10.2 9.976 9.337 28.062 23.536 22.227 21.287 20.849 21.146 20.238 20.702 19.882 20.146 19.495 19.761 18.003 22.006 22.523 22.375 21.922 24.002 22.475 22.306 20.864 20.617 22.682 22.674 23.733 21.608 23.736 21.61 22.676 19.931 18.516 17.984 17.236 17.695 18.042 17.221 17.814 18.554 18.122 18.68 17.913 18.586 17.894 18.454 16.384 19.864 20.485 19.877 19.674 21.981 22.798 22.479 24.289 19.333 118- (113) ILE241 ILE241 ILE241 ILE241 ILE241 ILE241 ILE241 ALA242 ALA242 ALA242 ALA242 ALA242 GLY243 GLY243 GLY243 GLY243 HIS244 HIS244 H1S244 HIS244 HIS244 HIS244 HIS244 HIS244 HIS244 HIS244 GLU245 GLU245 GLU245 GLU245 GLU245 GLU245 GLU245 GLU245 GLU245 THR246 THR246 THR246 THR246 THR246 THR246 THR246 THR247 THR247 THR247 THR247 THR247 THR247 THR247 ALA248 ALA248 ALA248 ALA248 ALA248 12 1 12 12 A BGGDA  A A BGDDEE UCOCCCCNCCOCNCCONCCOCC </span>VAL233 CG2 GLY234 N GLY234 CA GLY234 C GLY234 LEU235 N LEU235 CA LEU235 C LEU235 LEU235 CB LEU235 CG LEU235 CD1 LEU235 CD2 ALA236 N ALA236 CA ALA236 C ALA236 ALA236 CB PHE237 N PHE237 CA PHE237 C PHE237 C PHE237 C PHE237 C PHE237 CD2 PHE237 CE1 PHE237 CE2 PHE237 CZ LEU238 N LEU238 CA LEU238 C LEU238 LEU238 C  LEU238 CG LEU238 CD1 LEU238 CD2 LEU239 N LEU239 CA LEU239 C LEU239 LEU239 CB LEU239 CG LEU239 CD1 LEU239 CDU 240 LEU240 CB LEU240 CG LEU240 CD1 LEU240 CD2 ILE241 N -11.571 -12.258 -12.641 -11.443 -11.116 -10.687 -9.532 -8.375 -7.846 -9.04 -10.08 -9.781 -10.2 -8.162 -7.065 -7.38 -6.525 -6.861 -8.66 -9.11- 8.956 -8.27 -10.58 -11.12  10.824 -11.885 -11.305 -12.366 -12.076 -9.329  9.272 -7.845 -7.532 -10.056 -11.539 -12.279 -12.145 -6.947 -5.551 -4.861 -4.202 -4.821 -5.364 -4.722 -5.169- 5.275  4.667 -5.217 -4.445 -4.952 -3.966 -2.53 -4.227 -6.425 17.632 1 7.278 16.729 16.59 15.477 17.668 17.691 16.809 16.046 19.13 20.034 21.505 19.75 16.713 15.885 14.407 13.633 16.193 14.095 12.723 12.273 11.274 12.657 11.24 10.394 10.786 9.095 9.486 8.644 13.135 12.754 12.681 11.745 13.766 13.75 14.795 12.369 13.467 13.397 12.158 11.444 14.652 15.898 17.162 15.82 11.736 10.56 9.808 8.265 8.316 8.316 10.2 9.976 9.337 28.062 23.536 22.227 21.287 20.849 21.146 20.238 20.702 19.882 20.146 19.495 19.761 18.003 22.006 22.523 22.375 21.922 24.002 22.475 22.306 20.864 20.617 22.682 22.674 23.733 21.608 23.736 21.61 22.676 19.931 18.516 17.984 17.236 17.695 17.695 18.042 17.221 18.814 18.554 18.122 18.68 17.913 18.586 18.894 18.485 19.877 19.674 21.981 22.798 22.479 24.289 19.333 118- (113) ILE241 ILE241 ILE241 ILE241 ILE241 ILE241 ILE241 ALA242 ALA242 ALA242 ALA242 ALA242 GLY243 GLY243 GLY243 GLY243 HIS244 HIS244 HIS244 H245 G245 GLULU245 G245LULU245 GLULU244 HR246 THR246 THR246 THR246 THR246 THR247 THR247 THR247 THR247 THR247 THR247 THR247 ALA248 ALA248 ALA248 ALA248 ALA248 12 1 12 12 A BGGDA  A A BGDDEE UCOCCCCNCCOCNCCONCCOCC </span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">[ [ n EL i/ rt /V</span>[[n EL i / rt / V</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">A BGDE1E2</span>A BGDE1E2</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">A</span>A</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">12 G G</span>12  G G</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">A</span>A</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">1 2 G G -7.027 8.197 18.621  6.509 8.091 17.184  -6.406 6.98 16.64  -8.544 8.391 18.599  -9.122 8.403 20.008  -9.233 7.311 17.771  -8.939 7.06 20.699  -5.959 9.182 16.676  5.353 9.154 15.345  -4.049 8.363 15.361  -3.842 7.511 14.487  -5.09 10.583 14.884  -3.34 8.426 16.473  -2.135 7.612 16.614  -2.461 6.175 17.012  -2.095 5.238 16.289  -3.274 6.021 18.047  -3.556 4.687 18.603  -4.424 3.78 17.729  -4.288 2.555 17.808  -4.271 4.846 19.946  -3.394 5.206 21.132  2.995 6.436 21.502  -2.87 4.327 22.051  -2.232 6.348 22.61  -2.156 5.042 22.95  -5.298 4.33 16.905  -6.157 3.454 16.101  -6.041 3.723 14.61  -5.883 2.788 13.812  -7.616 3.66 16.504  -7.895 3.313 17.963  -7.613 1.839 18.255  -7.721 1.028 17.348  -7.228 1.562 19.381  -5.987 4.996 14.26  -6.101 5.374 12.849  -4.863 4.962 12.066  -4.949 4.04 11.244  -6.314 6.88 12.759  -7.415 7.225 13.59  -6.614 7.344 11.338  -3.702 5.384 12.532  -2.48 5.072 11.792  -2.013 3.634 12.044  -1.437 3.032 11.132  -1.4 6.078 12.171  -1.937 7.39 12.062  -0.192 5.981 11.246  -2.533 3.007 13.09  -2.225 1.596 13.35  -2.915 0.676 12.342  -2.236 -0.089 11.643  -2.678 1.25 14.763 119- (114) ASN249 ASN249 ASN249 ASN249 ASN249 ASN249 ASN249 ASN249 MET250 MET250 MET250 MET250 MET250 MET250 MET250 MET250 ILE251 ILE251 ILE251 ILE251 ILE251 ILE251 ILE251 ILE251 SER252 SER252 SER252 SER252 SER252 SER252 LEU253 LEU253 LEU253 LEU253 LEU253 LEU253 LEU253 LEU253 GLY254 GLY254 GLY254 GLY254 THR255 THR255 THR255 THR255 THR255 THR255 THR255 VAL256 VAL256 VAL256 VAL256 VAL256 zcacocbcgodndncacocbcgsdcencacocbcgcgcdncacocbogncacocbcgwcd: BGD1D2 A bgde a BG1G2D1 2</span>1 2  GG -7.027 8.197 18.621  6.509 8.091 17.184 -6.406 6.98 16.64 -8.544 8.391 18.599 -9.122 8.403 20.008 -9.233 7.311 17.771 -8.939 7.06 20.699 -5.959 9.182 16.676  5.353 9.154 15.345 -4.049 8.363 15.361 -3.842 7.511 14.487 -5.09 10.583 14.884 -3.34 8.426 16.473 -2.135 7.612 16.614 -2.461 6.175 17.012 -2.095 5.238 16.289 -3.274 6.021 18.047 -3.556 4.687 18.603 -4.424 3.78 17.729 -4.288 2.555 17.808 -4.271 4.846 19.946 -3.394 5.206 21.132  2.995 6.436 21.502 -2.87 4.327 22.051 -2.232 6.348 22.61 -2.156 5.042 22.95 -5.298 4.33 16.905 -6.157 3.454 16.101 -6.041 3.723 14.61 -5.883 2.788 13.812 -7.616 3.66 16.504 -7.895 3.313 17.963 -7.613 1.839 18.255 -7.721 1.028 17.348 -7.228 1.562 19.381 -5.987 4.996 14.26 -6.101 5.374 12.849 -4.863 4.962 12.066 -4.949 4.04 11.244 -6.314 6.88 12.759 -7.415 7.225 13.59 -6.614 7.344 11.338 -3.702 5.384 12.532 -2.48 5.072 11.792 -2.013 3.634 12.044 -1.437 3.032 11.132 -1.4 6.078 12.171 -1.937 7.39 12 .062 -0.192 5.981 11.246 -2.533 3.007 13.09 -2.225 1.596 13.35 -2.915 0.676 12.342 -2.236 -0.089 11.643 -2.678 1.25 14.763 119- (114) ASN249 ASN249 ASN249 ASN249 ASN249 ASN249 ASN249 ASN249 MET250 MET250 MET250 MET250 MET250 MET250 MET250 MET250 MET250 MET250 MET250 MET250 MET250 MET250 MET250 MET250 MET250 MET250 MET250 MET250 MET250 MET250 MET250 ILE251 ILE251 ILE251 ILE251 ILE251 ILE251 SER252 SER252 SER252 SER252 SER252 SER252 LEU253 LEU253 LEU253 LEU253 LEU253 LEU253 LEU253 LEU253 GLY254 GLY254 GLY254 GLY254 THR255 THR255 THR255 THR255 THR255 THR255 THR255 VAL256 VAL256 VAL256 VAL256 VAL256 zcacocbcgodndncacocbcgsdcencacocbcgcgcdncacocbogncacocbcgwcd: BGD1D2 A bgde a BG1G2D1 2</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">A</span>A</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">A G1G2 -4.175 0.958 12.049  -4.878 0.157 11.042  -4.453 0.517 9.624  -4.313 -0.39 8.791  -6.377 0.361 11.199  6.902 -0.561 12.291  -6.261 -1.569 12.613  -8.133 -0.32 12.703  -3.945 1.727 9.463  -3.472 2.183 8.159  -2.137 1.529 7.787  -2.018 1.009 6.669  -3.231 3.702 8.255  -3.343 4.33 6.851  -2.234 5.768 6.665  1.954 6.241 8.406  -1.267 1.317 8.764  0.019 0.667 8.477  -0.132 -0.844 8.328  0.449 -1.427 7.402  1.001 0.949 9.613  1.293 2.434 9.757  2.305 0.188 9.404  2.264 2.698 10.902  -1.081 -1.421 9.047  -1.254 -2.875 8.973  -1.947 -3.301 7.679  -1.444 -4.204 6.998  -2.03 -3.36 10.197  -3.28 -2.684 10.279  2.85 -2.465 7.194  -3.558 -2.762 5.948  -2.699 -2.386 4.739  -2.665 -3.129 3.748  -4.857 -1.963 5.994  -5.781 -2.185 4.804  -6.009 -3.665 4.521  -7.109 -1.479 5.058  -1.818 -1.42 4.948  -0.841 -1.031 3.933  0.161 -2.148 3.671  0.267 -2.62 2.531  0.707 -2.706 4.742  1.711 -3.772 4.62  1.123 -5.042 4.017  1.715 -5.592 3.079  2.255 -4.105 6.007  2.837 -2.935 6.563  3.334 -5.18 5.938  0.133 -5.321 4.331  -0.797 -6.505 3.781  -1.11 -6.37 2.291  -0.832 -7.307 1.531  -2.083 6.706 4.571 120- (115) VAL256 CG1 -3.028 -7.693 3.905  VAL256 CG2 -1.774 -7.144 5.993  THR257 N -1.372 -5.158 1.833  THR257 CA -1.675 -4.982 0.413  THR257 C 0.401 -4.902 -0.427  THR257  -0.357 -5.499 -1.512  THR257 C -2.502 -3.717 0.248  THR257 G1 -3.63 -3.818 1.106  THR257 CG2 -3.002 -3.558 -1.183  LEU258 N 0.69 -4.465 0.185  LEU258 CA 1.976 -4.434 -0.526  LEU258 C 2.588 -5.828 -0.61  LEU258  3.147 -6.205 -1.648  LEU258 C 2.936 -3.524 0.233  LEU258 CG 2.45 -2.08 0.263  LEU258 CD1 3.329 -1.229 1.172  LEU258 CD2 2.391 -1.492 -1.141  LEU259 N 2.248 -6.652 0.368  LEU259 CA 2.721 -8.034 0.407  LEU259 C 1.818 -8.965 -0.407  LEU259  2.223 -10.083 -0.747  LEU259 CB 2.742 -8.462 1.869  LEU259 CG 3.979 -9.291 2.183  LEU259 CD1 5.235 -8.559 1.727  LEU259 CD2 4.05 -9.61 3.672  GLU260 N 0.649 -8.473 -0.791  GLU260 CA -0.221 9.215 -1.707  GLU260 C 0.163 -8.932 -3.151  GLU260  0.033 -9.804 -4.019  GLU260 CB -1.67 -8.773 -1.531  GLU260 CG -2.53 -9.85 -0.883  GLU260 CD -2.442 -9.78 0.637  GLU260 OE1 -1.632 -10.498 1.204  GLU260 OE2 -3.287 -9.103 1.202  ASN261 N 0.693 -7.743 -3.382  ASN261 CA 1.158 -7.381 -4.723  ASN261 C 2.669 -7.181 -4.742  ASN261  3.134 -6.032 -4.804  ASN261 CB 0.468 -6.089 5.161  ASN261 CG -0.986 -6.311 -5.586  ASN261 D1 -1.784 -6.942 -4.881  ASN261 ND2 -1.339 -5.701 -6.705  PR0262 N 3.404 -8.259 -4.986  PR0262 CA 4.864 -8.219 -4.846  PR0262 C 5.563 -7.432 -5.958  PR0262  6.612 -6.834 5.706  PR0262 CB 5.298 9.652 -4.871  PR0262 CG 4.108 -10.533 -5.223  PR0262 CD 2.912 -9.601 -5.319  ASP263 N 4.884 -7.222 -7.077  ASP263 CA 5.442 -6.411 -8.166  ASP263 C 5.27 -4.909 7.918  ASP263  6.133 -4.124 -8.327  ASP263 CB 4.783 -6.822 -9.488</span>A G1G2 -4.175 0.958 12.049 -4.878 0.157 11.042 -4.453 0.517 9.624 -4.313 -0.39 8.791 -6.377 0.361 11.199  6.902 -0.561 12.291 -6.261 -1.569 12.613 -8.133 -0.32 12.703 -3.945 1.727 9.463 -3.472 2.183 8.159 -2.137 1.529 7.787 -2.018 1.009 6.669 -3.231 3.702 8.255 -3.343 4.33 6.851 -2.234 5.768 6.665  1.954 6.241 8.406 -1.267 1.317 8.764 0.019 0.667 8.477 -0.132 -0.844 8.328 0.449 -1.427 7.402 1.001 0.949 9.613 1.293 2.434 9.757 2.305 0.188 9.404 2.264 2.698 10.902 -1.081 -1.421 9.047 -1.254 -2.875 8.973 -1.947 -3.301 7.679 -1.444 -4.204 6.998 -2.03 -3.36 10.197 -3.28 -2.684 10.279  2.85 -2.465 7.194 -3.558 -2.762 5.948 -2.699 -2.386 4.739 -2.665 -3.129 3.748 -4.857 -1.963 5.994 -5.781 -2.185 4.804 -6.009 -3.665 4.521 -7.109 -1.479 5.058 -1.818 -1.42 4.948 -0.841 -1.031 3.933 0.161 -2.148 3.671 0.267 -2.62 2.531 0.707 -2.706 4.742 1.711 -3.772 4.62 1.123 -5.042 4.017 1.715- 5.592 3.079 2.255 -4.105 6.007 2.837 -2.935 6.563 3.334 -5.18 5.938  0.133 -5.321 4.331 -0.797 -6.505 3.781 -1.11 -6.37 2.291 -0.832 -7.307 1.531 -2.083  6.706 4.571 120- (115) VAL256 CG1 -3.028 -7.693 3.905 VAL256 CG2 -1.774 -7.144 5.993 THR257 N -1.372- 5.158 1.833 THR257 CA -1.675 -4.982 0.413 THR257 C  0.401 -4.902 -0.427 THR257 -0.357 -5.499 -1.512 THR257 C  -2.502 -3.717 0.248 THR257 G1 -3.63 -3.818 1.106 THR257 CG2 -3.002 -3.558 -1.183 LEU258 N 0.69 -4.465 0.185 LEU258 CA 1.976 -4.434 -0.526 LEU258 C 2.588 -5.828 -0.61 LEU258  3.147 -6.205 -1.648 LEU258 C  2.936 -3.524 0.233 LEU258 CG 2.45 -2.08 0.263 LEU258 CD1 3.329 -1.229 1.172 LEU258 CD2 2.391 -1.492 -1.141 LEU259 N 2.248 -6.652 0.368 LEU259 CA 2.721 -8.034 0.407 LEU259 C 1.818 -8.965 -0.407 LEU259 2.223 -10.083 -0.747 LEU259 CB 2.742 -8.462 1.869 LEU259 CG 3.979 -9.291 2.183 LEU259 CD1 5.235 -8.559 1.727 LEU259 CD2 4.05- 9.61 3.672 GLU260 N 0.649 -8.473 -0.791 GLU260 CA -0.221  9.215 -1.707 GLU260 C 0.163 -8.932 -3.151 GLU260 0.033 -9.804 -4.019 GLU260 CB -1.67 -8.773 -1.531 GLU260 CG -2.53 -9.85 -0.883 GLU260 CD -2.442 -9.78 0.637 GLU260 OE1 -1.632 -10.498 1.204 GLU260 OE2 -3.287 -9.103 1.202 ASN261 N 0.693 -7.743 -3.382 ASN261 CA 1.158 -7.381 -4.723 ASN261 C 2.669 -7.181 -4.742 ASN261 3.134 -6.032 -4.804 ASN261 CB 0.468 -6.089  5.161 ASN261 CG -0.986 -6.311 -5.586 ASN261 D1 -1.784 -6.942 -4.881 ASN261 ND2 -1.339- 5.701 -6.705 PR0262 N 3.404 -8.259 -4.986 PR0262 CA 4.864 -8.219 -4.846 PR0262 C 5.563 -7.432 -5.958 PR0262 6.612 -6.834  5.706 PR0262 CB 5.298  9.652 -4.871 PR0262 CG 4.108 -10.533 -5.223 PR0262 CD 2.912 -9.601 -5.319 ASP263 N 4.884 -7.222 -7.077 ASP263 CA 5.442 -6.411 -8.166 ASP263 C 5.27 -4.909  7.918 ASP263 6.133 -4.124 -8.327 ASP263 CB 4.783 -6.822 -9.488</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">121 - (116) ASP263 CG 3.253 -6.778 -9.42  ASP263 D1 2.69 -5.734 -9.721  ASP263 OD2 2.673 7.766 -8.992  GLN264 N 4.358 -4.558 -7.024  GLN264 CA 4.149 -3.154 -6.675  GLN264 C 5.104 2.803 -5.545  GLN264  5.782 -1.769 -5.59  GLN264 C 2.709 -3.002 -6.206  GLN264 CG 1.723 3.483 -7.265  GLN264 CD 1.637 -2.485 -8.412  GLN264 E1 1.631 -1.274 -8.178  GLN264 NE2 1.592 -2.991 -9.63  LEU265 N 5.398 -3.825 -4.758  LEU265 CA 6.389 -3.701 -3.693  LEU265 C 7.808 -3.677 -4.263  LEU265  8.639 -2.888 -3.798  LEU265 CB 6.218 -4.903 -2.775  LEU265 CG 7.134 -4.809 -1.565  LEU265 CD1 6.869 -3.518 -0.802  LEU265 CD2 6.958 -6.023 -0.662  ALA266 N 7.993 -4.309 -5.411  ALA266 CA 9.286 -4.267 -6.093  ALA266 C 9.528 -2.929 -6.785  ALA266  10.66 -2.434 -6.737  ALA266 CB 9.333 -5.39 -7.123  LYS267 N 8.466 -2.235 -7.167  LYS267 CA 8.641 -0.891 -7.725  LYS267 C 8.887 0.136 -6.625  LYS267  9.706 1.042 -6.818  LYS267 C 7.406 -0.501 -8.523  LYS267 CG 7.223 -1.394 -9.742  LYS267 CD 6.072 -0.894 -10.604  LYS267 CE 4.779 -0.825 -9.803  LYS267 NZ 3.688 -0.267 -10.615  ILE268 N 8.413 0.162 5.427  ILE268 CA 8.717 0.664 -4.254  ILE268 C 10.181 0.531 -3.84  ILE268  10.879 1.544 -3.697  ILE268 CB 7.827 0.171 3.117  ILE268 CG1 6.38 0.576 -3.335  ILE268 CG2 8.311 0.638 -1.75  ILE268 CD1 5.504 0.057 -2.205  LYS269 N 10.693 0.688 -3.902  LYS269 CA 12.068 -0.95 -3.463  LYS269 C 13.116 0.675 -4.542  LYS269  14.306 -0.554 -4.228  LYS269 CB 12.126 -2.405 -3.021  LYS269 CG 11.167 -2.628 -1.858  LYS269 CD 10.997 -4.107 -1.542  LYS269 CE 12.315 -4.756 1.148  LYS269 NZ 12.106 -6.181 -0.856  ALA270 N 12.679 -0.53 -5.782  ALA270 CA 13.585 -0.106 -6.851  ALA270 C 13.478 1.396 -7.101</span>121-(116) ASP263 CG 3.253 -6.778 -9.42 ASP263 D1 2.69 -5.734 -9.721 ASP263 OD2 2.673  7.766 -8.992 GLN264 N 4.358 -4.558 -7.024 GLN264 CA 4.149 -3.154 -6.675 GLN264 C 5.104  2.803 -5.545 GLN264  5.782 -1.769 -5.59 GLN264 C  2.709 -3.002 -6.206 GLN264 CG 1.723  3.483 -7.265 GLN264 CD 1.637 -2.485 -8.412 GLN264 E1 1.631 -1.274 -8.178 GLN264 NE2 1.592 -2.991 -9.63 LEU265 N 5.398 -3.825 -4.758 LEU265 CA 6.389 -3.701 -3.693 LEU265 C 7.808 -3.677 -4.263 LEU265 8.639 -2.888 -3.798 LEU265 CB 6.218 -4.903 -2.775 LEU265 CG 7.134 -4.809 -1.565 LEU265 CD1 6.869 -3.518 -0.802 LEU265 CD2 6.958 -6.023 -0.662 ALA266 N 7.993 -4.309 -5.411 ALA266 CA 9.286 -4.267 -6.093 ALA266 C 9.528 -2.929 -6.785 ALA266  10.66 -2.434 -6.737 ALA266 CB 9.333 -5.39 -7.123 LYS267 N 8.466 -2.235 -7.167 LYS267 CA 8.641 -0.891 -7.725 LYS267 C 8.887 0.136 -6.625 LYS267 9.706 1.042 -6.818 LYS267 C  7.406 -0.501 -8.523 LYS267 CG 7.223 -1.394 -9.742 LYS267 CD 6.072 -0 .894 -10.604 LYS267 CE 4.779 -0.825 -9.803 LYS267 NZ 3.688 -0.267 -10.615 ILE268 N 8.413  0.162  5.427 ILE268 CA 8.717 0.664 -4.254 ILE268 C 10.181 0.531 -3.84 ILE268 10.879 1.544 -3.697 ILE268 CB 7.827 0.171  3.117 ILE268 CG1 6.38 0.576 -3.335 ILE268 CG2 8.311 0.638 -1.75 ILE268 CD1 5.504 0.057 -2.205 LYS269 N 10.693  0.688 -3.902 LYS269 CA 12.068 -0.95 -3.463 LYS269 C 13.116  0.675 -4.542 LYS269 14.306 -0.554 -4.228 LYS269 CB126 12. -3.021 LYS269 CG 11.167 -2.628 -1.858 LYS269 CD 10.997 -4.107 -1.542 LYS269 CE 12.315 -4.756  1.148 LYS269 NZ 12.106 -6.181 -0.856 ALA270 N 12.679 -0.53 -5.782 ALA270 CA 13.585 -0.106 -6.851 ALA270 C 13.478 1.396 -7.101</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">122- (117) ALA270 ALA270 ASP271 ASP271 ASP271 ASP271 ASP271 ASP271 ASP271 ASP271 PR0272 PR0272 PR0272 PR0272 PR0272 PR0272 PR0272 GLY273 GLY273 GLY273 GLY273 LYS274 LYS274 LYS274 LYS274 LYS274 LYS274 LYS274 LYS274 LYS274 THR275 THR275 THR275 THR275 THR275 THR275 THR275 LEU276 LEU276 LEU276 LEU276 LEU276 LEU276 LEU276 LEU276 ALA277 ALA277 ALA277 ALA277 ALA277 ALA278 ALA278 ALA278 ALA278 CBNCAcoCBCGODODNCAcoCBCG8NCACONCAcoCBCG8CENZNCAcoCBOGCGNCAcoCBCGCDCDNCAcoCBNCAco D1D2</span>122- (117) ALA270 ALA270 ASP271 ASP271 ASP271 ASP271 ASP271 ASP271 ASP271 ASP271 PR0272 PR0272 PR0272 PR0272 PR0272 PR0272 PR0272 GLY273 GLY273 GLY273 GLY273 LYS274 LYS274 LYS274 LYS274 LE276 THR276 THR276 THR276 THR276 THR276 THR276 THR276 ALA277 ALA277 ALA277 ALA277 ALA277 ALA278 ALA278 ALA278 ALA278 CBNCAcoCBCGODODNCAcoCBCG8NCACONCAcoCBCG8CENZNCAcoCBOGCGNCAcoCBCGCDCDNCAcoCBNCAco D1D2</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">12 B G I 14.286 1.974 -7.838  13.233 -0.863 -8.125  12.486 2.017 -6.486  12.271 3.453 -6.649  11.505 4.006 -5.448  10.267 4.02 -5.445  11.48 3.653 -7.944  11.354 5.125 -8.337  10.975 5.919 -7.482  11.493 5.405 -9.517  12.238 4.705 -4.592  11.686 5.248 -3.337  10.72 6.438 -3.497  9.973 6.74 -2.558  12.887 5.657 -2.54  14.133 5.544 -3.406  13.673 4.968 -4.734  10.586 6.971 -4.702  9.607 8.03 -4.968  8.207 7.423 -5.037  7.257 7.968 -4.456  8.167 6.171 -5.473  6.916 5.418 5.566  6.39 4.954 -4.21  5.225 4.55 -4.144  7.138 4.184 -6.431  7.528 4.547 -7.856  7.755 3.287 -8.681  8.177 3.615 10.107  8.465 2.383 -10.858  7.12 5.182 -3.127  6.593 4.814 -1.813  5.522 5.804 -1.352  4.516 5.354 -0.798  7.725 4.76 -0.789  8.169 6.074 -0.485  8.912 3.963 -1.305  5.564 7.036 -1.844  4.543 8.012 1.446  3.312 7.905 -2.346  2.175 8.051 -1.876  5.14 9.411 -1.538  4.182 10.462 -0.987  3.836 10.177 0.472  4.77 11.861 1.133  3.53 7.375 -3.539  2.417 7.126 -4.451  1.711 5.836 -4.052  0.475 5.796 -4.026  2.963 7.021 -5.869  2.472 4.947 -3.431  1.909 3.727 -2.859  1.12 4.01 -1.591  0.051 3.416 -1.427 123- (118) ALA278 ILE279 ILE279 ILE279 ILE279 ILE279 ILE279 ILE279 ILE279 GLU280 GLU280 GLU280 GLU280 GLU280 GLU280 GLU280 GLU280 GLU280 GLU281 GLU281 GLU281 GLU281 GLU281 GLU281 GLU281 GLU281 GLU281 LEU282 LEU282 LEU282 LEU282 LEU282 LEU282 LEU282 LEU282 LEU283 LEU283 LEU283 LEU283 LEU283 LEU283 LEU283 LEU283 ARG284 ARG284 ARG284 A  G284 ARG284 ARG284 ARG284 ARG284 ARG284 ARG284 ARG284</span>12 BGI 14.286 1.974 -7.838 13.233 -0.863 -8.125 12.486 2.017 -6.486 12.271 3.453 -6.649 11.505 4.006 -5.448 10.267 4.02 -5.445 11.48 3.653 -7.944 11.354 5.125 -8.337 10.975 5.919 -7.482 11.493 5.405 -9.517 12.238 4.705 -4.592 11.686 5.248- 3.337 10.72 6.438 -3.497 9.973 6.74 -2.558 12.887 5.657 -2.54 14.133 5.544 -3.406 13.673 4.968 -4.734 10.586 6.971 -4.702 9.607 8.03 -4.968 8.207 7.423 -5.037 7.257 7.968 -4.456 8.167 6.171 -5.473 6.916 5.418  5.566 6.39 4.954 -4.21 5.225 4.55 -4.144 7.138 4.184 -6.431 7.528 4.547 -7.856 7.755 3.287 -8.681 8.177 3.615  10.107 8.465 2.383 -10.858 7.12 5.182 -3.127 6.593 4.814 -1.813 5.522 5.804 -1.352 4.516 5.354 -0.798 7.725 4.76 -0.789 8.169 6.074 -0.485 8.912 3.963- 1.305 5.564 7.036 -1.844 4.543 8.012  1.446 3.312 7.905 -2.346 2.175 8.051 -1.876 5.14 9.411 -1.538 4.182 10.462 -0.987 3.836 10.177 0.472 4.77 11.861  1.133 3.53 7.375 -3.539 2.417 7.126 -4.451 1.711 5.836 -4.052 0.475 5.796 -4. 4. 026 2.963 7.021 -5.869 2.472 4.947 -3.431 1.909 3.727 -2.859 1.12 4.01 -1.591 0.051 3.416 -1.427 123- (118) ALA278 ILE279 ILE279 ILE279 ILE279 ILE279 ILE279 ILE279 ILE279 GLU280 GLU280 GLU280 GLU280 GLU280 GLU280 GLU280 GLU280 GLU280 GLU280 GLU280 GLU280 GLU280 GLU280 GLU281 GLU281 GLU281 LEU282 LEU282 LEU282 LEU282 LEU282 LEU282 LEU282 LEU282 LEU283 LEU283 LEU283 LEU283 LEU283 LEU283 LEU283 LEU283 LEU283 ARG284 ARG284 ARG284 A Latch G284 ARG284 ARG284 ARG284 ARG284 ARG284 ARG284 ARG284</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">BA BGGD A  GD E1E2 A CNCCCCCCNCCCCCNCC</span>BA BGGD A  GD E1E2 A CNCCCCCCNCCCCCNCC</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">BGDEE2A BGD1D2A BGD1D2A BGDEZH1H2 CCCNCCCCCCNCCCCCCNCCCCCNCNN 3.048 2.773 -2.523  1.472 5.064 -0.867  0.698 5.447 0.324  -0.692 5.926 -0.078  -1.691 5.368 0.396  1.401 6.588 1.063  2.806 6.209 1.513  0.577 7.041 2.264  2.805 4.984 2.416  -0.748 6.688 -1.159  -2.037 7.217 -1.616  -2.849 6.159 -2.358  -4.075 6.123 -2.217  -1.784 8.404 -2.537  -0.943 9.484 -1.862  -1.616 9.992 -0.587  -2.453 10.875 -0.699  -1.19 9.569 0.479  -2.169 5.173 -2.916  -2.857 4.083 -3.604  -3.402 3.048 -2.616  -4.497 2.514 -2.842  -1.846 3.441 -4.546  2.451 2.329 -5.39  -3.395 2.862 -6.465  -3.893 2.017 -7.199  -3.446 4.068 -6.653  -2.776 2.957 -1.452  -3.272 2.102 -0.369  -4.527 2.703 0.232  -5.563 2.03 0.269  -2.212 2.019 0.727  -1.025 1.151 0.332  0.163 1.385 1.256  -1.415 -0.319 0.305  -4.511 4.018 0.378  -5.666 4.738 0.922  -6.823 4.808 -0.066  -7.989 4.766 0.346  -5.202 6.145 1.257  -4.178 6.103 2.379  -3.386 7.399 2.461  -4.848 5.766 3.706  -6.512 4.723 -1.346  -7.561 4.67 -2.355  -8.306 3.337 -2.3  -9.482 3.316 -1.923  -6.921 4.824 -3.726  -7.99 4.956 -4.798  -7.417 4.674 -6.18  -6.879 3.305 -6.258  -7.603 2.231 -6.583  -8.911 2.35 -6.816  -7.021 1.032 -6.658 -124- ILE285 N -7.588 2.233 -2.423  ILE285 CA -8.284 0.942 -2.519  ILE285 C -8.742 0.397 -1.155  ILE285  -9.703 -0.385 -1.087  ILE285 C -7.36 -0.039 -3.247  ILE285 CG1 -8.033 -1.383 -3.499  ILE285 CG2 -6.045 -0.238 -2.501  ILE285 CD1 -7.119 -2.327 -4.272  PHE286 N -8.188 0.934 -0.082  PHE286 CA -8.543 0.5 1.268  PHE286 C -8.451 1.645 2.27  PHE286  -7.544 1.66 3.115  PHE286 CB -7.575 -0.594 1.708  PHE286 CG -7.737 -1.948 1.025  PHE286 CD1 -6.675 -2.503 0.323  PHE286 CD2 -8.943 -2.632 1.117  PHE286 CE1 -6.822 3.736 -0.298  PHE286 CE2 -9.09 -3.865 0.495  PHE286 CZ -8.03 -4.417 -0.213  THR287 N -9.377 2.584 2.193  THR287 CA -9.413 3.636 3.212  THR287 C -9.931 3.053 4.519  THR287  -10.843 2.216 4.543  THR287 CB -10.294 4.801 2.77  THR287 G1 -10.207 5.833 3.745  THR287 CG2 -11.759 4.423 2.643  ILE288 N -9.305 3.459 5.609  ILE288 CA -9.745 2.966 6.911  ILE288 C -10.966 3.747 7.391  ILE288  -11.856 3.155 8.014  ILE288 CB -8.584 3.044 7.894  ILE288 CG1 -8.04 4.459 8.013  ILE288 CG2 -7.469 2.097 7.461  ILE288 CD1 -6.938 4.507 9.052  ALA289 N -11.116 4.965 6.89  ALA289 CA -12.34 5.743 7.09  ALA289 C -13.315 5.37 5.982  ALA289  -13.433 6.062 4.964  ALA289 CB -12.003 7.228 7.013  GLU290 N 13.969 4.239 6.174  GLU290 CA -14.796 3.652 5.128  GLU290 C -16.223 4.179 5.178  GLU290  16.891 4.234 4.138  GLU290 C -14.766 2.14 5.34  GLU290 CG 15.669 1.375 4.381  GLU290 CD 15.736 -0.091 4.802  GLU290 E1 -16.751 0.475 5.367  GLU290 OE2 -14.809 -0.819 4.472  THR291 N -16.659 4.619 6.347  THR291 CA -17.992 5.216. 6.481  THR291 C -17.983 6.47 7.349  THR291  17.68 6.437 8.55  THR291 CB -18.978 4.219 7.094  THR291 G1 -18.522 3.832 8.382</span>BGDEE2A BGD1D2A BGD1D2A BGDEZH1H2 CCCNCCCCCCNCCCCCCNCCCCCNCNN 3.048 2.773 -2.523 1.472 5.064 -0.867 0.698 5.447 0.324 -0.692 5.926 -0.078-1.691 5.368 0.396 1.401 6.588 2.063 2.806 6.806 6.1.5 6.209 -1.159 -2.037 7.217 -1.616 -2.849 6.159 -2.358 -4.075 6.123 -2.217 -1.784 8.404 -2.537 -0.943 9.484 -1.862 -1.616 9.992 -0.587 -2.453 10.875 -0.699 -1.19 9.569 0.479 -2.169 5.173 -2.916 -2.857 4.083- 3.604 -3.402 3.048 -2.616 -4.497 2.514 -2.842 -1.846 3.441 -4.546  2.451 2.329 -5.39 -3.395 2.862 -6.465 -3.893 2.017 -7.199 -3.446 4.068 -6.653 -2.776 2.957 -1.452 -3.272 2.102 -0.369 -4.527 2.703 0.232 -5.563 2.03 0.269 -2.212 2.019 0.727 -1.025 1.151 0.332 0.163 1.385 1.256 -1.415 -0.319 0.305 -4.511 4.018 0.378 -5.666 4.738 0.922 -6.823 4.808 -0.066 -7.989 4.766 0.346 -5.202 6.145 1.257 -4.178 6.103 2.379 -3.386 7.399 2.461- 4.848 5.766 3.706 -6.512 4.723 -1.346 -7.561 4.67 -2.355 -8. 306 3.337 -2.3 -9.482 3.316 -1.923 -6.921 4.824 -3.726 -7.99 4.956 -4.798 -7.417 4.674 -6.18 -6.879 3.305 -6.258 -7.603 2.231 -6.583 -8.911 2.35 -6.816 -7.021 1.032 -6.658 -124- ILE285 N- 7.588 2.233 -2.423 ILE285 CA -8.284 0.942 -2.519 ILE285 C -8.742 0.397 -1.155 ILE285 -9.703 -0.385 -1.087 ILE285 C  -7.36 -0.039 -3.247 ILE285 CG1 -8.033 -1.383 -3.499 ILE285 CG2 -6.045 -0.238- 2.501 ILE285 CD1 -7.119 -2.327 -4.272 PHE286 N -8.188 0.934 -0.082 PHE286 CA -8.543 0.5 1.268 PHE286 C -8.451 1.645 2.27 PHE286 -7.544 1.66 3.115 PHE286 CB -7.575 -0.594 1.708 PHE286 CG -7.737 -1.948 1.025 PHE286 CD1 -6.675 -2.503 0.323 PHE286 CD2 -8.943 -2.632 1.117 PHE286 CE1 -6.822  3.736 -0.298 PHE286 CE2 -9.09 -3.865 0.495 PHE286 CZ -8.03 -4.417 -0.213 THR287 N -9.377 2.584 2.193 THR287 CA -9.413 3.636 3.212 THR287 C- 9.931 3.053 4.519 THR287 -10.843 2.216 4.543 THR287 CB -10.294 4.801 2.77 THR287 G1 -10.207 5.833 3.745 THR287 CG2 -11.759 4.423 2.643 ILE 288 N -9.305 3.459 5.609 ILE288 CA -9.745 2.966 6.911 ILE288 C -10.966 3.747 7.391 ILE288 -11.856 3.155 8.014 ILE288 CB -8.584 3.044 7.894 ILE288 CG1 -8.04 4.459 8.013 ILE288 CG2 -7.469 2.097 7.461 ILE288 CD1 -6.938 4.507 9.052 A -11.116 4.965 6.89 ALA289 CA -12.34 5.743 7.09 ALA289 C -13.315 5.37 5.982 ALA289 -13.433 6.062 4.964 ALA289 CB -12.003 7.228 7.013 GLU290 N  13.969 4.239 6.174 GLU290 CA -14.796 3.652 5.128 GLU290 C -16.223 4.179 5.16 GLU290 . 4.234 4.138 GLU290 C  -14.766 2.14 5.34 GLU290 CG  15.669 1.375 4.381 GLU290 CD  15.736 -0.091 4.802 GLU290 E1 -16.751  0.475 5.367 GLU290 OE2 -14.809 -0.819 4.472 THR291 N -16.659 4.619 6.347 THR291 CA -17.992 5.216. THR291 C -17.983 6.47 7.349 THR291  17.68 6.437 8.55 THR291 CB -18.978 4.219 7.094 THR291 G1 -18.522 3.832 8.382</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">125- (120) THR291 ALA292 ALA292 ALA292 ALA292 ALA292 THR293 THR293 THR293 THR293 THR293 THR293 THR293 SER294 SER294 SER294 SER294 SER294 SER294 ARG295 ARG295 ARG295 ARG295 ARG295 ARG295 ARG295 ARG295 ARG295 ARG295 ARG295 PHE296 PHE296 PHE296 PHE296 PHE296 PHE296 PHE296 PHE296 PHE296 PHE296 PHE296 ALA297 ALA297 ALA297 ALA297 ALA297 THR298 THR298 THR298 THR298 THR298 THR298 THR298 ALA299 2 12 1 2 12 12 1 2 G A B A bgg a b g a  gdezhh a bgddeez a b a bgg cncccnccccncccncccccncnnncccccccccncccnccccn -19.168 2.961 6.257  -18.397 7.562 6.734  18.676 8.799 7.471  -20.103 8.704 8.001  -21.051 9.169 7.355  -18.552 9.986 6.521  -20.231 8.1 9.172  -21.535 7.709 9.721  -22.161 8.802 10.585  -22.152 8.733 11.82  -21.308 6.447 10.546  -20.564 5.523 9.756  -22.617 5.788 10.962  -22.703 9.805 9.913  -23.293 10.956 10.601  -24.198 11.766 9.675  -25.386 11.443 9.529  -22.167 11.822 11.164  -20.976 11.559 10.426  -23.598 12.789 9.076  -24.233 13.782 8.187  -25.738 13.903 8.366  -26.524 13.3 7.627  -23.905 13.424 6.747  -22.4 13.455 6.518  -21.819 14.841 6.786  -20.362 14.859 6.577  -19.475 14.679 7.56  -19.892 14.437 8.805  -18.167 14.719 7.293  -26.119 14.646 9.387  -27.528 14.825 9.712  -28.176 15.754 8.701  -27.736 16.899 8.529  -27.623 15.428 11.11  -29.052 15.695 11.562  -29.564 16.985 11.533  -29.843 14.643 12.002  -30.871 17.221 11.936  -31.149 14.879 12.406  -31.663 16.167 12.37  -29.2 15.255 8.032  -29.931 16.073 7.065  -30.783 17.111 7.781  -31.769 16.781 8.45  -30.819 15.165 6.224  -30.369 18.361 7.668  -31.129 19.47 8.255  -32.139 20.005 7.246  -33.087 20.712 7.603  -30.17 20.586 8.659  -29.565 21.12 7.488  -29.068 20.082 9.582  -31.926 19.649 5.992 -126- (121) 200418849</span>125- (120) THR291 ALA292 ALA292 ALA292 ALA292 ALA292 THR293 THR293 THR293 THR293 THR293 THR293 THR293 SER294 SER294 SER294 SER294 SER294 SER294 SER294 ARG295 ARG295 ARG295 ARG295LA296 P296 PHE295 PHE PHE PHE PHE PHE PHE PHE PHE ALA297 THR298 THR298 THR298 THR298 THR298 THR298 THR298 ALA299 2 12 1 2 12 12 1 2 GABA bgg abga  gdezhh a bgddeez aba bgg cncccnccccncccncccccncnnncccccccccncccncc96100cn-19-19168 6.257 -18.397 7.562 6.734  18.676 8.799 7.471 -20.103 8.704 8.001 -21.051 9.169 7.355 -18.552 9.986 6.521 -20.231 8.1 9.172 -21.535 7.709 9.721 -22.161 8.802 10.585 -22.152 8.733 11.82 -21.308 6.447 10.546 -20.564 5.523 9.756 -22.617 5.788 10.962- 22.703 9.805 9.913 -23.293 10.956 10.601 -24.198 11.766 9.675 -25.386 11.443 9.529 -22.167 11.822 11.164 -20.976 11.559 10.426 -23.598 12.789 9.076 -24.233 13.782 8.187 -25.738 13.9 03 8.366 -26.524 13.3 7.627 -23.905 13.424 6.747 -22.4 13.455 6.518 -21.819 14.841 6.786 -20.362 14.859 6.577 -19.475 14.679 7.56 -19.892 14.437 8.805 -18.167 14.719 7.293 -26.119 14.646 9.387 -27.528 14.825 9.712 -28.176 15.754 8.701 -27.736 16.899 8.899 -27.623 15.428 11.11 -29.052 15.695 11.562 -29.564 16.985 11.533 -29.843 14.643 12.002 -30.871 17.221 11.936 -31.149 14.879 12.406 -31.663 16.167 12.37 -29.2 15.255 8.032 -29.931 16.073 7.065 -30.783 17.111 7.781 -31.769 16.781 8.45 -30.819 15.369 6.165 6.224 18.361 7.668 -31.129 19.47 8.255 -32.139 20.005 7.246 -33.087 20.712 7.603 -30.17 20.586 8.659 -29.565 21.12 7.488 -29.068 20.082 9.582 -31.926 19.649 5.992 -126- (121) 200418849</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">2267 ALA299 CA  2268 ALA299 C  2269 ALA299   2270 ALA299 C  2271 ASP300 N  2272 ASP300 CA  2273 ASP300 C  2274 ASP300   2275 ASP300 CB  2276 ASP300 CG  2277 ASP300 OD1  2278 ASP300 D2  2279 VAL301 N  2280 VAL301 CA  2281 VAL301 C  2282 VAL301   2283 VAL301 CB  2284 VAL301 CG1  2285 VAL301 CG2  2286 GLU302 N  2287 GLU302 CA  2288 GLU302 C  2289 GLU302 0  2290 GLU302 CB  2291 GLU302 CG  2292 GLU302 CD  2293 GLU302 E1  2294 GLU302 OE2  2295 ILE303 N  2296 ILE303 CA  2297 ILE303 C  2298 ILE303   2299 ILE303 CB  2300 ILE303 CG1  2301 ILE303 CG2  2302 ILE303 CD1  2303 GLY304 N  2304 GLY304 CA  2305 GLY304 C  2306 GLY304   2307 GLY305 N  2308 GLY305 CA  2309 GLY305 C  2310 GLY305   2311 THR306 N  2312 THR306 CA  2313 THR306 C  2314 THR306 0  2315 THR306 CB  2316 THR306 OG1  2317 THR306 CG2  2318 LEU307 N  2319 LEU307 CA  2320 LEU307 C -32.881 19.968 4.932  -33.091 18.734 4.068  -32.427 17.712 4.27  -32.331 21.11 4.086  -34.041 18.817 3.151  -34.256 17.727 2.191  -33.253 17.852 1.048  -33.498 18.545 0.053  -35.679 17.801 1.648  -36.682 17.714 2.793  -36.686 16.69 3.463  -37.289 18.733 3.09  -32.128 17.177 1.199  -31.029 17.337 0.244  -31.065 16.261 -0.833  -30.956 15.062 -0.55  -29.711 17.263 1.007  -28.543 17.673 0.118  -29.756 18.154 2.24  31.242 16.695 -2.067  -31.23 15.75 -3.184  -29.799 15.413 -3.603  -29.141 16.178 -4.318  -31.98 16.355 4.363  -32.053 15.344 5.497  -32.662 15.959 -6.75  -33.484 16.852 -6.606  -32.169 15.628 -7.82  29.365 14.227 -3.222  -28.014 13.766 -3.539  -28.053 12.812 -4.726  -28.452 11.647 -4.6  -27.438 13.076 -2.307  -27.329 14.056 -1.148  -26.066 12.49 -2.608  -26.305 15.146 -1.441  -27.715 13.356 -5.885  -27.705 12.592 7.139  -29.056 11.935 -7.402  -29.187 10.708 -7.311  -30.079 12.757 -7.573  -31.432 12.229 -7.805  -32.226 11.982 -6.517  -33.289 12.584 -6.314  31.694 11.132 -5.653  -32.399 10.717 -4.431  -32.569 11.859 -3.43  31.597 12.497 -3.015  -31.591 9.598 -3.782  -31.461 8.534 -4.716  -32.278 9.051 -2.537  -33.811 12.127 -3.066  -34.093 13.162 -2.068  -33.971 12.6 0.652 127 - (122)</span>2267 ALA299 CA 2268 ALA299 C 2269 ALA299 2270 ALA299 C  2271 ASP300 N 2272 ASP300 CA 2273 ASP300 C 2274 ASP300 2275 ASP300 CB 2276 ASP300 CG 2277 ASP300 OD1 2278 ASP300 D2 2279 VAL301 N 2280 VAL301 CA 2281 VAL301 C 2282 VAL301 2283 VAL301 CB 2284 VAL301 CG1 2285 VAL301 CG2 2286 GLU302 N 2287 GLU302 CA 2288 GLU302 C 2289 GLU302 0 2290 GLU302 CB 2291 GLU302 CG 2292 GLU302 CD 2293 GLU302 E1 2294 GLU302 OE2 2295 ILE303 N 2296 ILE303 2ILE ILE303 2 ILE303 CB 2300 ILE303 CG1 2301 ILE303 CG2 2302 ILE303 CD1 2303 GLY304 N 2304 GLY304 CA 2305 GLY304 C 2306 GLY304 2307 GLY305 N 2308 GLY305 CA 2309 GLY305 C 2310 GLY305 2311 THR306 N 2312 THR306 CA 2313 THR306 C 2316 THR306 OG1 2317 THR306 CG2 2318 LEU307 N 2319 LEU307 CA 2320 LEU307 C -32.881 19.968 4.932 -33.091 18.734 4.068 -32.427 17.712 4.27 -32.331 21.11 4.086 -34.041 18.817 3.151 -34.256 17.727 2.191 -33 .253 17.852 1.048 -33.498 18.545 0.053 -35.679 17.801 1.648 -36.682 17.714 2.793 -36.686 16.69 3.463 -37.289 18.733 3.09 -32.128 17.177 1.199 -31.029 17.337 0.244 -31.065 16.261 -0.833 -30.956 15.062 -0.55 -29.711 17.263 1.007 -28.543 17.673 0.118 -29.756 18.154 2.24  31.242 16.695 -2.067 -31.23 15.75 -3.184 -29.799 15.413 -3.603 -29.141 16.178 -4.318 -31.98 16.355  4.363 -32.053 15.344  5.497 -32.662 15.959 -6.75 -33.484 16.852 -6.606 -32.169 15.628 -7.82  29.365 14.227 -3.222 -28.014 13.766 -3.539 -28.053 12.812 -4.726 -28.452 11.647 -4.6 -27.438 13.076 -2.307 -27.329 14.056 -1.148 -26.066 12.49 -2.608 -26.305 15.146 -1.441 -27.715 13.356 -5.885 -27.705 12.592  7.139- 29.056 11.935 -7.402 -29.187 10.708 -7.311 -30.079 12.757 -7.573 -31.432 12.229 -7.805 -32.226 11.982 -6.517 -33.289 12.584 -6.314  31.694 11.132 -5.653 -32.399 10.717 -4.431 -32.569 11.859 -3.43  31.597 12.497 -3.015- 31.591 9.598 -3.782 -31.461 8.534 -4.716 -32 .278 9.051 -2.537 -33.811 12.127 -3.066 -34.093 13.162 -2.068 -33.971 12.6  0.652 127-(122)</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">(122)   LEU307   LEU307 CB  LEU307 CG  LEU307 CD1  LEU307 CD2  ILE308 N  ILE308 CA  ILE308 C  ILE308   ILE308 C  ILE308 CG1  ILE308 CG2  ILE308 CD1  ARG309 N  ARG309 CA  ARG309 C  ARG309   ARG309 CB  ARG309 CG  ARG309 CD  ARG309 NE  ARG309 CZ  ARG309 NH1  ARG309 NH2  ALA310 N  ALA310 CA  ALA310 C  ALA310   ALA310 CB  GLY311 N  GLY311 CA  GLY311 C  GLY311   GLU312 N  GLU312 CA  GLU312 C  GLU312   GLU312 CB  GLU312 CG  GLU312 CD  GLU312 E1  GLU312 E2  GLY313 N  GLY313 CA  GLY313 C  GLY313   VAL314 N  VAL314 CA  VAL314 C  VAL314   VAL314 CB  VAL314 CG1  VAL314 CG2  VAL315 N -34.803 -35.513 -35.845 -34.946 -37.31 -32.93 -32.738 -33.586 33.481 -31.253 -30.456 -30.976 -28.974 -34.466 -35.386 -34.674 -33.807 &amp;gt;36.447 -37.117 -38.079 -38.721 -40.046 -40.85 -40.566 -35.099 -34.597 -34.952 -36.005 -35.223 -34 -34.172 -33.475 -33.472 -32.982 -32.294 -30.776 -30.236 -32.743 -34.243 -34.639 -34.686 -34.723 -30.107 -28.64 -28.079 -28.674 -26.996 -26.33 -24.93 -24.127 -26.28 -25.639 -27.681 -24.674 11.799 13.67 14.852 16.039 15.248 13.021 12.616 13.486 14.719 12.742 11.77 12.507 11.794 12.834 13.543 14.125 13.486 12.563 11.841 10.778 10.055 9.951 10.528 9.278 15.325 15.999 15.189 14.542 17.386 15.115 14.325 12.968 12.217 12.611 11.322 11.471 12.577 10.651 10.386 9.537 10.083 8.33 10.356 10.335 9.813 8.944 10.429 10.028 9.467 10.02 11.246 10.942 11.804 8.328 -0.209 -2.295 -1.391 -1.71 -1.534 0.045 1.439 2.362 2.356 1.764 0.901 3.245 1.246 3.101 3.993 5.209 5.824 4.474 3.312 3.83 2.721 2.594 3.49 1.566 5.566 6.763 8.003 8.07 6.853 8.913 10.13 10.042 11.023 8.866 8.719 8.737 8.599 7.429 7.453 6.252 5.156 6.419 8.981 8.994 7.673 7.02 7.237 5.994 6.267 7.033 5.073 3.724 4.861 5.649 -128- (123)   VAL315 CA -23.386 7.646 5.773  VAL315 C -22.647 7.666 4.435  VAL315  23.025 6.975 3.479  VAL315 C -23.634 6.207 6.213  VAL315 CG1 -22.325 5.437 6,347  VAL315 CG2 -24.404 6.163 7.528  GLY316 N -21.6 8.467 4.374  GLY316 CA -20.789 8.539 3.152  GLY316 C -19.797 7.382 3.096  GLY316  -18.903 7.28 3.946  LEU317 N -19.978 6.508 2.121  LEU317 CA -19.102 5.343 1.978  LEU317 C -17.841 5.722 1.218  LEU317  -17.758 5.485 0.009  LEU317 C -19.835 4.256 1.199  LEU317 CG -20.312 3.094 2.064  LEU317 CD1 -19.129 2.374 2.697  LEU317 CD2 -21.328 3.519 3.121  SER318 N -16.794 6.056 1.951  SER318 CA -15.563 6.559 1.34  SER318 C -14.789 5.483 0.588  SER318  -14.286 5.777 -0.503  SER318 CB -14.689 7.158 2.434  SER318 G -13.375 7.32 1.915  ASN319 N -14.954 4.222 0.954  ASN319 CA -14.267 3.196 0.162  ASN319 C -15.089 2.774 -1.058  ASN319  -14.498 2.343 -2.052  ASN319 CB -13.899 1.982 1.004  ASN319 CG -12.531 1.507 0.516  ASN319 D1 -11.628 2.333 0.321  ASN319 ND2 -12.378 0.209 0.332  ALA320 N -16.362 3:139 -1.091  ALA320 CA -17.181 2.889 -2.283  ALA320 C -17.001 4.055 -3.248  ALA320  -16.897 3.856 -4.464  ALA320 CB -18.642 2.78 -1.875  GLY321 N -16.68 5.197 -2.666  GLY321 CA -16.233 6.36 -3.426  GLY321 C -14.942 6.045 -4.17  GLY321  -14.924 6.106 5.404  ASN322 N -13.958 5.508 -3.466  ASN322 CA -12.683 5.172 -4.113  ASN322 C -12.765 3.924 -5.006  ASN322  -11.954 3.769 -5.927  ASN322 CB -11.651 4.902 -3.035  ASN322 CG -11.54 6.013 -1.994  ASN322 D1 -11.701 7.211 -2.278  ASN322 ND2 -11.095 5.595 -0.824  HIS323 N -13.785 3.103 -4.807  HIS323 CA -14.068 1.985 -5.716  HIS323 C -15.037 2.337 -6.846  HIS323  15.541 1.435 -7.525  HIS323 CB -14.625 0.804 -4.939</span>(122) LEU307 LEU307 CB LEU307 CG LEU307 CD1 LEU307 CD2 ILE308 N ILE308 CA ILE308 C ILE308 ILE308 C  ILE308 CG1 ILE308 CG2 ILE308 CD1 ARG309 N ARG309 CA ARG309 C ARG309 ARG309 CB ARG309 CG ARG309 CD ARG309 CD ARG309 NH2 ALA310 N ALA310 CA ALA310 C ALA310 ALA310 CB GLY311 N GLY311 CA GLY311 C GLY311 GLU312 N GLU312 CA GLU312 C GLU312 GLU312 CB GLU312 CG GLU312 CD GLU312 E1 GLU312 E2 GLY313 N GLY313 N G313 VAL314 CA VAL314 C VAL314 VAL314 CB VAL314 CG1 VAL314 CG2 VAL315 N -34.803 -35.513 -35.845 -34.946 -37.31 -32.93 -32.738 -33.586  33.481 -31.253 -30.456 -30.976 -28.974 -34.466 -35.386 -34.674. 36.447 -37.117 -38.079 -38.721 -40.046 -40.85 -40.566 -35.099 -34.597 -34.952 -36.005 -35.223 -34 -34.172 -33.475 -33.472 -32.982 -32.294 -30.776 -30.236 -32.743 -34.243 -34.639 -34.686 -34.723 -34.686 30.107 -28.64 -28.079 -28.674 -26.996 -26.33 -24.93 -2 4.127 -26.28 -25.639 -27.681 -24.674 11.799 13.67 14.852 16.039 15.248 13.021 12.616 13.486 14.719 12.742 11.77 12.507 11.794 12.834 13.543 14.125 13.486 12.563 11.841 10.778 10.055 9.951 10.528 9.278 15.325 15.999 15.189 14.542 17.386 15.115 14.325 12.968 12.217 12.611 11.322 11.471 12.577 10.651 10.651 10.386 9.386 8.33 10.356 10.335 9.813 8.944 10.429 10.028 9.467 10.02 11.246 10.942 11.804 8.328 -0.209 -2.295 -1.391 -1.71 -1.534 0.045 1.439 2.362 2.356 1.764 0.901 3.245 1.246 3.101 3.993 5.209 5.824 4.474 3.312 3.83 2.721 2.594 3.49 1.566 5.566 6.763 8.003 8.07 6.853 8.913 10.13 10.042 11.023 8.866 8.719 8.737 8.599 7.429 7.453 6.252 5.156 6.419 8.981 8.994 7.673 7.02 7.237 5.994 6.267 7.033 5.073 3.724 4.861 5.649 -128- (123) VAL315 CA -23.386 7.646 5.773 VAL315 C -22.647 7.666 4.435 VAL315 23.025 6.975 3.479 VAL315 C - 23.634 6.207 6.213 VAL315 CG1 -22.325 5.437 6,347 VAL315 CG2 -24.404 6.163 7.528 GLY316 N -21.6 8.467 4.374 GLY316 CA -20.789 8.539 3.152 GLY316 C -19.797 7.382 3.096 GLY316 -18.903 7.28 3.946 LEU317 N -19.978 6.508 2.121 LEU317 CA -19.102 5.343 1.978 LEU317 C -17.841 5.722 1.218 LEU317 -17.758 5.485 0.009 LEU317 C  -19.835 4.256 1.199 LEU317 CG -20.312 3.094 2.064 LEU317 -19.129 2.374 2.697 LEU317 CD2 -21.328 3.519 3.121 SER318 N -16.794 6.056 1.951 SER318 CA -15.563 6.559 1.34 SER318 C -14.789 5.483 0.588 SER318 -14.286 5.777 -0.503 SER318 CB -14.689 7.158 2.434 SER318 G -13.375 N32 1.915 1.915 -14.954 4.222 0.954 ASN319 CA -14.267 3.196 0.162 ASN319 C -15.089 2.774 -1.058 ASN319 -14.498 2.343 -2.052 ASN319 CB -13.899 1.982 1.004 ASN319 CG -12.531 1.507 0.516 ASN319 D1 -11.628 2.333 0.321 ASN319 0.2ND2 -12.378 N -16.362 3: 139 -1.091 ALA320 CA -17.181 2.889 -2.283 ALA320 C -17.001 4.055 -3.248 ALA320 -16.897 3.856 -4.464 ALA320 CB -18.642 2.78 -1.875 GLY321 N -16.68 5.197 -2.666 GLY321 CA -16.233 6.36 -3.426 GLY321 C -14.942 6.045 -4.17 GLY321 -14.924 6.106  5.404 ASN322 N -13.958 5.508 -3.466 ASN322 CA -12.683 5.172 -4.113 ASN322 C -12.765 3.924 -5.006 ASN322 -11.954 3.769 -5.927 ASN322 CB -11.651 4.902 -3.035 ASN322 CG -11.54 6.013 -1.994 ASN322  D1 -11.701 7.211 -2.278 ASN322 ND2 -11.095 5.595 -0.824 HIS323 N -13.785 3.103 -4.807 HIS323 CA -14.068 1.985 -5.716 HIS323 C -15.037 2.337 -6.846 HIS323  15.541 1.435 -7.525 HIS323 CB -14.625 0.804 -4.939</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">-129- (124) HIS323 HIS323 HIS323 HIS323 HIS323 ASP324 ASP324 ASP324 ASP324 ASP324 ASP324 ASP324 ASP324 PR0325 PR0325 PR0325 PR0325 PR0325 PR0325 PR0325 ASP326 ASP326 ASP326 ASP326 ASP326 ASP326 ASP326 ASP326 GLY327 GLY327 GLY327 GLY327 PHE328 PHE328 PHE328 PHE328 PHE328 PHE328 PHE328 PHE328 PHE328 PHE328 PHE328 GLU329 GLU329 GLU329 GLU329 GLU329 GLU329 GLU329 GLU329 GLU329 ASN330 ASN330</span>-129- (124) HIS323 HIS323 HIS323 HIS323 HIS323 ASP324 ASP324 ASP324 ASP324 ASP324 ASP324 ASP324 ASP324 PR0325 PR0325 PR0325 PR0325 PR0325 PR0325 PR0325 PR0325 ASP326 ASP326 ASP326 ASP326 ASP326 ASP326 ASP326 ASP326 GLY327 PHE328 PHE328 PHE328 PHE328 PHE328 PHE328 PHE328 PHE328 PHE328 PHE328 PHE328 PHE328 PHE328 PHE328 PHE328 PHE328 PHE328 PHE328 GLU329 GLU329 GLU329 GLU329 GLU329 GLU329 GLU329 GLU329 ASN330 ASN330</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">12 12  D D E E N O c N</span>12 12  D D E E N O c N</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">3A</span>3A</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">1 2 1 2 D D A bgd a bgdd -7 r  r-/rw_/r  o y r-/o 3 y D</span>1 2 1 2 D D A bgd a bgdd -7 r t r- / rw_ / r evening o y r- / o 3 y D</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">A</span>A</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">A 12 12 B G D d:e:e:z</span>A 12 12 B G D d: e: e: z</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">A</span>A</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">B G DE1E</span>B G DE1E</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">A -13.581 0.008 -4.19  -13.802 -0.807 -3.144  -12.233 -0.04 -4.46  -12.634 -1.351 -2.748  -11.664 -0.875 -3.563  -15.353 3.608 -7.007  -16.209 4.041 8.109  -15.409 4.076 -9.411  -14.628 5.012 -9.642  -16.72 5.431 -7.737  -17.711 6.002 -8.74  -18.738 6.476 -8.275  -17.28 6.257 -9.859  -15.831 3.237 -10.349  -15.082 2.989 -11.595  -15.201 4.099 -12.648  -14.504 4.065 -13.667  -15.653 1.713 -12.134  -16.9 1.346 -11.345  17.01 2.374 -10.232  16.008 5.11 -12.361  -16.206 6.248 -13.259  -15.146 7.312 -12.983  -14.924 8.216 -13.797  -17.597 6.826 -12.994  -18.675 5.753 -13.155  -19.125 5.571 -14.277  -18.963 5.068 -12.179  -14.468 7.162 -11.857  -13.318 8.005 -11.542  -12.061 7.145 -11.594  -11.166 7.366 -12.418  -12.046 6.12 -10.76  -10.893 5.218 -10.705  -11.119 4.009 -11.602  -11.92 3.114 -11.3  -10.653 4.81 -9.257  -10.261 5.995 -8.376  -9.098 6.702 -8.651  -11.07 6.379 -7.314  -8.737 7.784 -7.858  -10.709 7.459 6.519  -9.542 8.161 -6.791  -10.377 4.006 -12.696  10.511 2.993 -13.752  -10.048 1.63 -13.253  -8.847 1.399 -13.092  -9.673 3.409 14.962  -10.104 4.752 -15.557  -9.106 5.864 -15.215  -8.54 5.804 -14.132  -8.904 6.724 16.059  -10.987 0.698 -13.249  -10.826 -0.597 -12.566 -130- (125)   ASN330 C -10.252 -0.375 -11.171  ASN330  -9.065 -0.629 -10.929  ASN330 C -9.925 -1.547 -13.355  ASN330 CG -9.943 -2.959 -12.75  ASN330 D1 -9.958 -3.151 -11.523  ASN330 ND2 -9.893 -3.939 -13.632  PR0331 N -11.157 -0.201 -10.224  PR0331 CA -10.787 0.129 -8.843  PR0331 C -10.287 -1.061 -8.012  PR0331  -9.994 -0.904 6.822  PR0331 C -12.044 0.673 -8.25  PR0331 CG -13.21 0.356 -9.172  PR0331 CD -12.608 0.26 -10.416  ASP332 N -10.198 -2.236 8.615  ASP332 CA -9.742 -3.417 -7.889  ASP332 C -8.267 -3.667 8.179  ASP332  -7.609 -4.447 7.478  ASP332 CB -10.557 -4.617 -8.358  ASP332 CG -12.048 -4.356 -8.173  ASP332 D1 -12.43 -3.961 -7.08  ASP332 D2 -12.784 -4.566 -9.127  THR333 N -7.744 -2.97 -9.173  THR333 CA -6.339 -3.153 -9.535  THR333 C -5.433 -2.224 -8.735  THR333  -5.283 -1.042 -9.06  THR333 CB -6.189 -2.877 -11.026  THR333 G1 -7.072 -3.748 -11.718  THR333 CG2 -4.769 3.147 -11.514  PHE334 N -4.84 -2.772 -7.688  PHE334 CA -3.868 -2.017 -6.89  PHE334 C -2.589 -1.762 -7.684  PHE334  -1.817 -2.686 -7.979  PHE334 CB -3.543 -2.812 -5.63  PHE334 CG -2.485 -2.162 4.744  PHE334 CD1 -2.784 -1 4.047  PHE334 CD2 -1.221 -2.728 -4.638  PHE334 CE1 -1.82 -0.404 3.246  PHE334 CE2 -0.256 -2.132 -3.837  PHE334 CZ -0.556 -0.969 -3.142  ASP335 N -2.407 -0.509 8.064  ASP335 CA -1.206 -0.106 -8.794  ASP335 C -0.697 1.24 -8.283  ASP335  -1.322 2.277 -8.525  ASP335 CB -1.55 0.019 -10.277  ASP335 CG -0.276 0.2 -11.084  ASP335 D1 0.34 -0.792 -11.448  ASP335 D2 0.15 1.347 -11.159  ILE336 N 0.532 1.249 -7.794  ILE336 CA 1.118 2.426 -7.127  ILE336 C 1.596 3.552 -8.058  ILE336  2.207 4.512 -7.579  ILE336 CB 2.303 1.948 6.301  ILE336 CG1 3.437 1.48 -7.203  ILE336 CG2 1.873 0.817 -5.375</span>A -13.581 0.008 -4.19 -13.802 -0.807 -3.144 -12.233 -0.04 -4.46 -12.634 -1.351 -2.748 -11.664 -0.875 -3.563 -15.353 3.608 -7.007 -16.209 4.041  8.109 -15.409 4.076 -9.411 -14.628 5.012 -9.642 -16.72 5.431 -7.737 -17.711 6.002 -8.74 -18.738 6.476 -8.275 -17.28 6.257 -9.859 -15.831 3.237 -10.349 -15.082 2.989 -11.595 -15.201 4.099 -12.648 -14.504 4.065 -13.667 -15.653 1.713 -12.134 -16.9 1.346 -11.345  17.01 2.374 -10.232  16.008 5.11 -12.361 -16.206 6.248 -13.259 -15.146 7.312 -12.983 -14.924 8.216 -13.797 -17.597 6.826 -12.994 -18.675 5.753 -13.155 -19.125 5.571 -14.277 -18.963 5.068 -12.179 -14.468 7.162 -13.318 8.005 -11.542 -12.061 7.145 -11.594 -11.166 7.366 -12.418 -12.046 6.12 -10.76 -10.893 5.218 -10.705 -11.119 4.009 -11.602 -11.92 3.114 -11.3 -10.653 4.81 -9.257 -10.261 5.995 -8.376 -9.098 6.702 -8.651 -11.07 6.379 -7.314 -8.737 7.784 -7.858 -10.709 7.459  6.519 -9.542 8.161 -6.791 -10.377 4.006 -12.696  10.511 2.99 3 -13.752 -10.048 1.63 -13.253 -8.847 1.399 -13.092 -9.673 3.409  14.962 -10.104 4.752 -15.557 -9.106 5.864 -15.215 -8.54 5.804 -14.132 -8.904 6.724  16.059 -10.987 0.698 -13.249 -10.826 -0.597 -12.566- 130- (125) ASN330 C -10.252 -0.375 -11.171 ASN330 -9.065 -0.629 -10.929 ASN330 C  -9.925 -1.547 -13.355 ASN330 CG -9.943 -2.959 -12.75 ASN330 D1 -9.958 -3.151 -11.523 ASN330 ND2- 9.893 -3.939 -13.632 PR0331 N -11.157 -0.201 -10.224 PR0331 CA -10.787 0.129 -8.843 PR0331 C -10.287 -1.061 -8.012 PR0331 -9.994 -0.904  6.822 PR0331 C  -12.044 0.673 -8.25 PR0331 CG -13.21 0.356- 9.172 PR0331 CD -12.608  0.26 -10.416 ASP332 N -10.198 -2.236  8.615 ASP332 CA -9.742 -3.417 -7.889 ASP332 C -8.267 -3.667  8.179 ASP332 -7.609 -4.447  7.478 ASP332 CB -10.557 -4.617 -8.358 ASP332 CG -12.048 -4.356 -8.173 ASP332 D1 -12.43 -3.961 -7.08 ASP332 D2 -12.784 -4.566 -9.127 THR333 N -7.744 -2.97 -9.173 THR333 CA -6.339 -3.153 -9.535 THR3 33 C -5.433 -2.224 -8.735 THR333 -5.283 -1.042 -9.06 THR333 CB -6.189 -2.877 -11.026 THR333 G1 -7.072 -3.748 -11.718 THR333 CG2 -4.769  3.147 -11.514 PHE334 N -4.84 -2.772 -7.688 PHE334 CA -3.868 -2.017 -6.89 PHE334 C -2.589 -1.762 -7.684 PHE334 -1.817 -2.686 -7.979 PHE334 CB -3.543 -2.812 -5.63 PHE334 CG -2.485 -2.162  4.744 PHE334 CD1 -2.784 -1  4.047 PHE334 CD2- 1.221 -2.728 -4.638 PHE334 CE1 -1.82 -0.404  3.246 PHE334 CE2 -0.256 -2.132 -3.837 PHE334 CZ -0.556 -0.969 -3.142 ASP335 N -2.407 -0.509 8.064 ASP335 CA -1.206 -0.106 -8.794 ASP335 C -0.697 1.24 -8.283 ASP335 -1.322 2.277 -8.525 ASP335 CB -1.55  0.019 -10.277 ASP335 CG -0.276 0.2 -11.084 ASP335 D1 0.34 -0.792 -11.448 ASP335 D2 0.15 1.347 -11.159 ILE336 N 0.532 1.249 -7.794 ILE336 CA 1.118 2.426- 7.127 ILE336 C 1.596 3.552 -8.058 ILE336 2.207 4.512 -7.579 ILE336 CB 2.303 1.948  6.301 ILE336 CG1 3.437 1.48 -7.203 ILE336 CG2 1.873 0.817 -5.375</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">131 - (126)</span>131-(126)</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">(126)   ILE336 CD1  GLU337 N  GLU337 CA  GLU337 C  GLU337   GLU337 C  GLU337 CG  GLU337 CD  GLU337 OE1  GLU337 E2  ARG338 N  ARG338 CA  ARG338 C  ARG338   ARG338 CB  ARG338 CG  ARG338 CD  ARG338 NE  ARG338 CZ  ARG338 NH1  ARG338 NH2  GLY339 N  GLY339 CA  GLY339 C  GLY339   ALA340 N  ALA340 CA  ALA340 C  ALA340   ALA340 CB  ARG341 N  ARG341 CA  ARG341 C  ARG341   ARG341 CB  ARG341 CG  ARG341 CD  ARG341 NE  ARG341 CZ  ARG341 NH1  ARG341 NH2  HIS342 N  HIS342 CA  HIS342 C  HIS342   HIS342 C  HIS342 CG  HIS342 ND1  HIS342 CD2  HIS342 CE1  HIS342 NE2  HIS343 N  HIS343 CA  HIS343 C 1.131 3.407 4.496 5.236 6.246 3.936 3.412 2.871 3.167 2.089 4.719 5.376 6.636 6.625 4.393 4.977 3.961 2.729 I. 847 2.056 0.757 7.735 8.98 8.895 9.045 8.767 8.646 9.982 10.145 7.907 10.917 12.25 12.179 11.307 13.131 13.264 14.135 14.303 15.463 16.534 15.551 13.009 13.12 11.888 11.621 13.367 14.641 15.877 14.758 16.758 16.064 II. 15-10.075 10.661 4.676 1.41 1.712 0.434 0.487 2.393 3.788 4.442 3.934 5.372 0.697 -1.982 -2.095 -1.809 -3.085 -4.484 -5.532 -5.375 -6.357 -7.549 -6.143 -2.423 -2.685 -4.038 -5.086 -3.994 5.202 -5.871 -6.419 -4.817 -5.844 -6.395 -7.904 -8.571 -6.045 -4.535 -4.175 -2.718 -2.142 -2.898 -0.813 -8.378 -9.801 -10.342 -11.547 10.644 -10.32 10.757 9.536 -10.259 -9.504 -9.49 -10.02 -10.572 -6.396 -9.361 -10.291 -10.681 -11.393 -11.533 -11.213 -12.481 -13.552 -12.348 -10.229 -10.464 -9.615 -8.412 -10.082 -10.234 -9.809 -10.591 -10.77 -10.207 -11.509 -10.271 -9.55 -8.852 -9.489 -7.534 -6.691 -6.335 -5.237 -5.414 -7.27 -7.043 -6.87 -7.436 -8.235 -8.387 -9.583 -9.691 -10.016 -10.271 -10.092 5.954 -5.603 -4.88 -4.961 -6.855 -7.606 -7.303 -8.731 8.195 -9.078 -4.184 -3.346 2.053</span>(126) ILE336 CD1 GLU337 N GLU337 CA GLU337 C GLU337 GLU337 C  GLU337 CG GLU337 CD GLU337 OE1 GLU337 E2 ARG338 N ARG338 CA ARG338 C ARG338 ARG338 CB ARG338 CG ARG338 CD ARG338 NE ARG338 CZ ARG338 NH2 NH1 CA GLY339 C GLY339 ALA340 N ALA340 CA ALA340 C ALA340 ALA340 CB ARG341 N ARG341 CA ARG341 C ARG341 ARG341 CB ARG341 CG ARG341 CD ARG341 NE ARG341 CZ ARG341 NH1 ARG341 NH2 HIS342 N HIS342 CA HIS342 C HHI342 C HIS342 ND1 HIS342 CD2 HIS342 CE1 HIS342 NE2 HIS343 N HIS343 CA HIS343 C 1.131 3.407 4.496 5.236 6.246 3.936 3.412 2.871 3.167 2.089 4.719 5.376 6.636 6.625 4.393 4.977 3.961 2.729 I. 847 2.056 0.757 7.735 8.98 8.895 9.045 8.767 8.646 9.982 10.145 7.179 10.907 9.307 9.307 13.131 13.264 14.135 14.303 15.463 16.534 15.551 13.009 13.12 11.888 11.621 13.367 14.641 15.877 14.758 16.758 16.064 II. 15-10.075 10.661 4.676 1.41 1.712 0.434 0.487 2.393 3.788 4. 442 3.934 5.372  0.697 -1.982 -2.095 -1.809 -3.085 -4.484 -5.532 -5.375 -6.357 -7.549 -6.143 -2.423 -2.685 -4.038 -5.086 -3.994  5.202 -5.871 -6.419 -4.817 -5.844 -6.395 -7.904- 8.571 -6.045 -4.535 -4.175 -2.718 -2.142 -2.898 -0.813 -8.378 -9.801 -10.342 -11.547  10.644 -10.32  10.757  9.536 -10.259 -9.504 -9.49 -10.02 -10.572 -6.396 -9.361 -10.291 --10.681- 11.393 -11.533 -11.213 -12.481 -13.552 -12.348 -10.229 -10.464 -9.615 -8.412 -10.082 -10.234 -9.809 -10.591 -10.77 -10.207 -11.509 -10.271 -9.55 -8.852 -9.489 -7.534 -6.691 -6.335 -5.237- 5.414 -7.27 -7.043 -6.87 -7.436 -8.235 -8.387 -9.583 -9.691 -10.016 -10.271 -10.092  5.954 -5.603 -4.88 -4.961 -6.855 -7.606 -7.303 -8.731  8.195 -9.078 -4.184 -3.346  2.053</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">132- (127)</span>132- (127)</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">(127)   HIS343   HIS343 C  HIS343 CG  HIS343 ND1  HIS343 CD2  HIS343 CE1  HIS343 NE2  VAL344 N  VAL344 CA  VAL344 C  VAL344   VAL344 C  VAL344 CG1  VAL344 CG2  ALA345 N  ALA345 CA  ALA345 C  ALA345   ALA345 CB  PHE346 N  PHE346 CA  PHE346 C  PHE346   PHE346 CB  PHE346 CG  PHE346 CD1  PHE346 CD2  PHE346 CE1  PHE346 CE2  PHE346 CZ  GLY347 N  GLY347 CA  GLY347 C  GLY347   PHE348 N  PHE348 CA  PHE348 C  PHE348   PHE348 C  PHE348 CG  PHE348 CD1  PHE348 CD2  PHE348 CE1  PHE348 CE2  PHE348 CZ  GLY349 N  GLY349 CA  GLY349 C  GLY349   VAL350 N  VAL350 CA  VAL350 C  VAL350   VAL350 CB -9.984 -8.988 -7.691 -7.374 -6.597 -6.125 5.643 -11.634 -12.278 -11.732 -12.271 -13.787 -14.412 -14.141 -10.581 9.947 -9.419 -9.242 -8.757 -9.169 -8.763 -9.885 -9.652 7.571 -6.223 -5.295 -5.921 -4.061 -4.687 -3.756 -11.068 -12.177 -12.007 10.882 -13.123 -13.084 -13.98 14.185 -13.497 -12.375 -12.654 -11.075 -11.633 -10.054 -10.332 -14.438 -15.269 -14.484 -13.286 -15.121 -14.426 13.865 -12.964 -15.384 11.572 8.965 9.253 10.313 8.421 10.166 8.993 10.052 10.521 9.82 9.998 10.343 11.457 8.977 9.178 8.411 9.24 8.703 7.686 10.518 11.424 12.363 13.29 12.253 11.583 11.513 11.048 10.916 10.451 10.388 12.169 13.091 14.367 14.764 14.988 16.264 17.307 17.307 16.085 15.613 14.749 16.06 14.322 15.633 14.763 18.227 19.339 20.194 20.433 20.525 21.29 20.408 20.854 22.334 -1.455 -3.085 -2.333 -1.562 2.319 -1.079 -1.549 -1.563 -0.331 0.915 2.015 -0.45 -1.282 -1.019 0.768 1.849 3.02 4.12 1.237 2.787 3.867 4.291 5.083 3.409 3.635 2.605 4.881 2.825 5.101 4.073 3.731 3.979 3.158 2.825 2.822 2.097 2.752 3.976 0.64 -0.285 -1.333 -0.081 -2.172 -0.918 -1.963 1.918 2.376 3.353 3.165 4.458 5.493 6.613 7.327 6.062</span>(127) HIS343 HIS343 C  HIS343 CG HIS343 ND1 HIS343 CD2 HIS343 CE1 HIS343 NE2 VAL344 N VAL344 CA VAL344 C VAL344 VAL344 C  VAL344 CG1 VAL344 CG2 ALA345 N ALA345 CA ALA345 C ALA345 ALA345 CB PHEHE346 N PHE346 CB PHE346 CG PHE346 CD1 PHE346 CD2 PHE346 CE1 PHE346 CE2 PHE346 CZ GLY347 N GLY347 CA GLY347 C GLY347 PHE348 N PHE348 CA PHE348 C PHE348 PHE348 C  PHE348 CE1 PHE348 CD1 PHE348 CD2 PHE348 CE1 P1 GLY349 C GLY349 VAL350 N VAL350 CA VAL350 C VAL350 VAL350 CB -9.984 -8.988 -7.691 -7.374 -6.597 -6.125  5.643 -11.634 -12.278 -11.732 -12.271 -13.787 -14.412 -14.141 -10.581  9.947 -9.419 -9.242 -8.757 -9.169 -8.763 -9.885 -9.652  7.571 -6.223 -5.295 -5.921 -4.061 -4.687 -3.756 -11.068 -12.177 -12.007  10.882 -13.123 -13.084 -13.98  14.185 -13.497 -12.375 -12.654 -11.075 -11.633 -10.054 -10.332 -14.438 -15.269 -14.484 -13.286 -15.12 1 -14.426  13.865 -12.964 -15.384 11.572 8.965 9.253 10.313 8.421 10.166 8.993 10.052 10.521 9.82 9.998 10.343 11.457 8.977 9.178 8.411 9.24 8.703 7.686 10.518 11.424 12.363 13.29 12.253 11.583 11.513 11.048 11.916 10.916 10.451 10.388 12.169 13.091 14.367 14.764 14.988 16.264 17.307 17.307 16.307 16.085 16.06 14.322 15.633 14.763 18.227 19.339 20.194 20.433 20.525 21.29 20.408 20.854 22.334 -1.455 -3.085 -2.333 -1.562  2.319 -1.079 -1.549 -1.563 -0.331 0.915 2.015 -0.45 -1.282 -1.019 0.768 1.849 3.02 4.12 1.237 2.787 3.867 4.291 5.083 3.409 3.635 2.605 4.881 2.825 5.101 4.073 3.731 3.979 3.158 2.825 2.822 2.097 2.752 3.976 0.64 -0.285 -1.333 -0.081 -2.172 -0.918 -1.963 1.918 2.376 3.353 3.165 4.458 5.493 6.613 7.327 6.062</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">-133- (128)</span>-133- (128)</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">VAL350 VAL350 HIS351 HIS351 HIS351 HIS351 HIS351 HIS351 ,HIS351 HIS351 HIS351 HIS351 GLN352 GLN352 GLN352 GLN352 GLN352 GLN352 GLN352 GLN352 GLN352 CYS353 CYS353 CYS353 CYS353 CYS353 CYS353 LEU354 LEU354 LEU354 LEU354 LEU354 LEU354 LEU354 LEU354 GLY355 GLY355 GLY355 GLY355 GLN356 GLN356 GLN356 GLN356 GLN356 GLN356 GLN356 GLN356 GLN356 ASN357 ASN357 ASN357 ASN357 ASN357 ASN357 CG1 -15.675 23.433 5.047  CG2 -16.682 21.709 6.561  N -14.217 19.133 6.631  CA -13.875 18.287 7.784  C -12.75 17.284 7.532   12.711 16.259 8.223  C -15.113 17.514 8.23  CG -16.205 18.349 8.87  ND1 -16.137 18.987 10.054  CD2 -17.46 18.588 8.36  CE1 -17.304 19.619 10.29  NE2 -18.122 19.372 9.242  N -11.876 17.53 6.568  CA -10.852 16.518 6.26  C -9.793 16.385 7.357   -9.716 17.193 8.291  C -10.178 16.785 4.923  CG -9.244 17.98 4.902  CD -8.308 17.803 3.712  E1 -7.316 18.528 3.568  NE2 -8.572 16.766 2.934  N -9.049 15.296 7.271  CA -8.044 14.964 8.287  C -6.883 15.962 8.319   -6.101 16.072 7.366  CB -7.524 13.567 7.961  SG -6.412 12.816 9.17  N 6.724 16.61 9.464  CA -5.607 17.549 9.666  C -4.308 16.852 10.059   -3.224 17.358 9.747  C -5.967 18.561 10.748  CG -6.805 9.7 10.188  CD1 -7.148 20.701 11.284  CD2 -6.054 20.396 9.057  N -4.414 15.6 10.478  CA -3.223 14.791 10.772  C -2.835 13.936 9.566   -2.119 12.936 9.702  N -3.078 14.499 8.393  CA -2.896 13.81 7.119  C -1.414 13.677 6.799   -0.939 12.562 6.544  CB 3.598 14.696 6.094  CG -3.657 14.117 4.691  CD -4.524 15.024 3.818  OE1 -4.609 14.834 2.599  NE2 -5.231 15.939 4.463  N -0.67 14.7 7.191  CA 0.781 14.7 7.005  C 1.476 13.761 7.989   2.328 12.977 7.554  CB 1.258 16.137 7.203  CG 2.78 16.244 7.261</span>VAL350 VAL350 HIS351 HIS351 HIS351 HIS351 HIS351 HIS351, HIS351 HIS351 HIS351 HIS351 GLN352 GLN352 GLN352 GLN352 GLN352 GLN352 GLN352 GLN352 GLN352 CYS353 CYS353 CYS353 CYS353 CYS353 CYS353 LEU354 LEU354 LEU354 LEU354 LEU354 LEU354 LEU354 LEU354 GLY355 GLY355 GLY355 GLY355 GLN356 GLN356 GLN356 GLN356 GLN356 GLN356 GLN356 GLN356 GLN356 ASN357 ASN357 ASN357 ASN357 ASN357 ASN357 CG1 -15.675 23.433 5.047 CG2 -16.682 21.709 6.561 N -14.217 19.133 6.631 CA -13.875 18.287 7.784 C -12.75 17.284 7.532 12.711 16.259 8.223 C  -15.113 17.514 8.23 CG -16.205 18.349 8.137 18.ND1-16 10.054 CD2 -17.46 18.588 8.36 CE1 -17.304 19.619 10.29 NE2 -18.122 19.372 9.242 N -11.876 17.53 6.568 CA -10.852 16.518 6.26 C -9.793 16.385 7.357 -9.716 17.193 8.291 C  -10.178 16.785 4.923 CG -9.244 17.98 4.902 CD -8.308 17.803 3.712 E1 -7.316 18.528 3.568 NE2 -8.572 16.766 2.934 N -9.049 15.296 7.271 CA -8.044 14.964 8.287 C -6.883 15.962 8.319 -6.101 16.072 7.366 CB -7.524 13.567 7.961 SG -6.412 12.816 9.17 N  6.724 16.61 9.464 CA -5.607 17.549 9.666 C -4.308 16.852 10.059 -3.224 17.358 9.747 C  -5.967 18.561 10.748 CG -6.805 9.7 10.188 CD1 -7.148 20.701 11.284 CD2- 6.054 20.396 9.057 N -4.414 15.6 10.478 CA -3.223 14.791 10.772 C -2.835 13.936 9.566 -2.119 12.936 9.702 N -3.078 14.499 8.393 CA -2.896 13.81 7.119 C -1.414 13.677 6.799 -0.939 12.562 6.544 CB  3.598 14.696 6.094 CG- 3.657 14.117 4.691 CD -4.524 15.024 3.818 OE1 -4.609 14.834 2.599 NE2 -5.231 15.939 4.463 N -0.67 14.7 7.191 CA 0.781 14.7 7.005 C 1.476 13.761 7.989 2.328 12.977 7.554 CB 1.258 16.137 7.203 CG 2.78 16.244 7.261</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">-134- (129)</span>-134- (129)</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">(129)   ASN357 D1  ASN357 ND2  LEU358 N  LEU358 CA  LEU358 C  LEU358   LEU358 CB  LEU358 CG  LEU358 CD1  LEU358 CD2  ALA359 N  ALA359 CA  ALA359 C  ALA359   ALA359 CB  ARG360 N  ARG360 CA  ARG360 C  ARG360   ARG360 CB  ARG360 CG  ARG360 CD  ARG360 NE  ARG360 CZ  ARG360 NH1  ARG360 NH2  LEU361 N  LEU361 CA  LEU361 C  LEU361   LEU361 C  LEU361 CG  LEU361 CD1  LEU361 CD2  GLU362 N  GLU362 CA  GLU362 C  GLU362   GLU362 CB  GLU362 CG  GLU362 CD  GLU362 E1  GLU362 E2  LEU363 N  LEU363 CA  LEU363 C  LEU363   LEU363 C  LEU363 CG  LEU363 CD1  LEU363 CD2  GLN364 N  GLN364 CA  GLN364 C 3.324 3.444 0.896 1.515 1.271 2.234 0.908 1.612 3.089 0.931 0.102 -0.235 0.571 1.204 -1.723 0.794 1.585 3.078 3.747 1.354 -0.081 -0.325 -0.144 0.509 0.602 1.045 3.518 4.921 5.238 6.174 5.159 6.612 7.537 6.778 4.306 4.499 4.413 5.31 3.418 3.519 2.341 1.268 2.517 3.541 3.389 4.61 5.174 2.184 0.977 0.35 -0.056 5.151 6.296 7.559 16.808 15.745 13.592 12.727 11.264 10.487 13.034 12.261 12.639 12.493 10.986 9.616 9.151 8.089 9.558 10.045 9.681 9.603 8.677 10.721 10.681 11.752 13.097 14.076 15.283 13.853 10.379 10.356 9.078 8.36 11.566 11.69 11.811 12.888 8.658 7.431 6.2 5.352 7.347 8.493 8.447 8.901 7.891 6.253 5.125 4.946 3.844 5.364 4.517 4.995 4.54 6.047 5.955 5.544 8.217 6.233 9.166 10.168 9.826 9.791 11.53 12.639 12.712 13.982 9.272 8.887 7.679 7.756 8.566 6.728 5.546 5.853 5.374 4.459 3.954 2.898 3.463 2.833 3.393 1.631 6.83 7.239 8.005 7.628 8.133 8.572 7.365 9.498 8.846 9.622 8.73 8.793 10.694 11.693 12.662 12.284 13.736 7.736 6.814 5.927 5.889 5.914 6.298 7.601 5.179 5.432 4.529 5.276</span>(129) ASN357 D1 ASN357 ND2 LEU358 N LEU358 CA LEU358 C LEU358 LEU358 CB LEU358 CG LEU358 CD1 LEU358 CD2 ALA359 N ALA359 CA ALA359 C ALA359 ALA359 CB ARG360 N ARG360 CA ARG360 C ARG360 RG ARG360 CB ARG360 CZ ARG360 NH1 ARG360 NH2 LEU361 N LEU361 CA LEU361 C LEU361 LEU361 C  LEU361 CG LEU361 CD1 LEU361 CD2 GLU362 N GLU362 CA GLU362 C GLU362 GLU362 CB GLU362 CG GLU362 CD GLU362 E1 LEU363 363 LEU363 C  LEU363 CG LEU363 CD1 LEU363 CD2 GLN364 N GLN364 CA GLN364 C 3.324 3.444 0.896 1.515 1.271 2.234 0.908 1.612 3.089 0.931 0.102 -0.235 0.571 1.204 -1.723 0.794 1.585 3.078 3.747 1.354 -0.081 -0.325 -0.144 0.509 0.602 1.921 3.518 6.174 5.159 6.612 7.537 6.778 4.306 4.499 4.413 5.31 3.418 3.519 2.341 1.268 2.517 3.541 3.389 4.61 5.174 2.184 0.977 0.35 -0.056 5.151 6.296 7.559 16.808 15.745 13.592 12.727 11.264 10.487 13.034 12.261 12 .639 12.493 10.986 9.616 9.151 8.089 9.558 10.045 9.681 9.603 8.677 10.721 10.681 11.752 13.097 14.076 15.283 13.853 10.379 10.356 9.078 8.36 11.566 11.69 11.811 12.888 8.658 7.431 6.2 5.352 7.347 8.493 8.493 8.447 8.901 7.891 6.253 5.125 4.946 3.844 5.364 4.517 4.995 4.54 6.047 5.955 5.544 8.217 6.233 9.233 10.168 9.826 9.791 11.53 12.639 12.712 13.982 9.272 8.887 7.679 7.756 8.566 6.728 5.546 5.853 5.374 4.459 3.954 2.898 3.463 2.833 3.833 1.393 6.631 7.83 7.239 8.005 7.628 8.133 8.572 7.365 9.498 8.846 9.622 8.73 8.793 10.694 11.693 12.662 12.284 13.736 7.736 6.814 5.927 5.889 5.914 6.298 7.601 5.179 5.179 5.276</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">135- (130) 200418849 2753 GLN364   2754 GLN364 C  2755 GLN364 CG  2756 GLN364 CD  2757 GLN364 OE1  2758 GLN364 NE2  2759 ILE365 N  2760 ILE365 CA  2761 ILE365 C  2762 ILE365   2763 ILE365 C  2764 ILE365 CG1  2765 ILE365 CG2  2766 ILE365 CD1  2767 VAL366 N  2768 VAL366 CA  2769 VAL366 C  2770 VAL366   2771 VAL366 CB  2772 VAL366 CG1  2773 VAL366 CG2  2774 PHE367 N  2775 PHE367 CA  2776 PHE367 C  2777 PHE367   2778 PHE367 C  2779 PHE367 CG  2780 PHE367 CD1  2781 PHE367 CD2  2782 PHE367 CE1  2783 PHE367 CE2  2784 PHE367 CZ  2785 ASP368 N  2786 ASP368 CA  2787 ASP368 C  2788 ASP368   2789 ASP368 C  2790 ASP368 CG  2791 ASP368 D1  2792 ASP368 D2  2793 THR369 N  2794 THR369 CA  2795 THR369 C  2796 THR369   2797 THR369 C  2798 THR369 G1  2799 THR369 CG2  2800 LEU370 N  2801 LEU370 CA  2802 LEU370 C  2803 LEU370   2804 LEU370 CB  2805 LEU370 CG  2806 LEU370 CD1 8.219 4.588 4.843  6.505 7.308 3.86  7.624 7.232 2.83  7.846 8.592 2.181  7,741 9.637 2.835  8.108 8.565 0.886  7.707 6.019 6.503  8.879 5.644 7.298  8.835 4.171 7.691  9.772 3.438 7.339  8.94 6.511 8.554  9.202 7.973 8.208  10.012 5.996 9.506  10.539 8.151 7.497  7.667 3.695 8.099  7.555 2.301 8.53  7.703 1.335 7.361  8.611 0.497 7.417  6.21 2.076 9.217  6.011 0.606 9.565  6.085 2.923 10.477  7.094 1.641 6.225  7.145 0.701 5.097  8.524 0.663 4.453  9.066 -0.434 4.254  6.125 1.099 4.034  4.662 1.012 4.458  3.726 1.83 3.841  4.258 0.112 5.435  2.391 1.767 4.216  2.924 0.052 5.813  1.99 0-88 5.206  9.187 1.808 4.423  10.522 1.869 3.83  11.52 1.102 4.686  12.115 0.138 4.185  10.964 3.326 3.707  10.101 4.09 2.701  10.208 5.31 2.678  9.508 3.443 1.847  11.438 1.285 5.995  12.381 0.606 6.892  12.07 -0.88 7.076  13.004 -1.658 7.287  12.376 1.294 8.252  11.058 1.242 8.778  12.797 2.756 8.149  10.848 -1.302 6.793  10.523 -2.729 6.862  11.145 -3.486 5.703  12.038 -4.319 5.916  9.012 -2.919 6.78  8.302 -2.463 8.044  6.792 -2.589 7.882 136- (131)   LEU370 CD2 8.79 -3.256 9.248  PHE371 N 10.872 -3.006 4.502  PHE371 CA 11.266 -3.75 3.303  PHE371 C 12.728 -3.529 2.918  PHE371  13.332 -4.398 2.278  PHE371 CB 10.333 -3.34 2.169  PHE371 CG 8.861 -3.634 2.463  PHE371 CD1 7.935 -2.599 2.503  PHE371 CD2 8.447 -4.94 2.692  PHE371 CE1 6.601 -2.867 2.784  PHE371 CE2 7.112 -5.209 2.971  PHE371 CZ 6.19 -4.172 3.019  ARG372 N 13.335 -2.472 3.436  ARG372 CA 14.773 -2.281 3.231  ARG372 C 15.606 -2.901 4.353  ARG372  16.834 -2.982 4.229  ARG372 CB 15.084 -0.795 3.116  ARG372 CG 14.397 -0.176 1.904  ARG372 CD 14.777 1.291 1.76  ARG372 NE 14.497 2.015 3.008  ARG372 CZ 14.919 3.257 3.251  ARG372 NH1 14.646 3.835 4.423  ARG372 NH2 15.631 3.911 2.331  ARG373 N 14.958 -3.347 5.418  ARG373 CA 15.671 -4.087 6.457  ARG373 C 15.659 -5.559 6.09  ARG373  16.694 -6.238 6.097  ARG373 CB 14.937 -3.926 7.783  ARG373 CG 15.657 -4.637 8.922  ARG373 CD 16.912 -3.879 9.335  ARG373 NE 16.548 -2.554 9.862  ARG373 CZ 16.405 -2.304 11.165  ARG373 NH1 16.663 -3.261 12.059  ARG373 NH2 16.046 -1.087 11.576  VAL374 N 14.474 -6.032 5.749  VAL374 CA 14.314 -7.425 5.338  VAL374 C 13.644 -7.499 3.971  VAL374  12.43 -7.307 3.841  VAL374 CB 13.473 -8.164 6.376  VAL374 CG1 13.297 -9.625 5.984  VAL374 CG2 14.086 -8.075 7.77  PR0375 N 14.432 7.873 2.976  PR0375 CA 13.929 -8.036 1.606  PR0375 C 13.088 9.304 1.369  PR0375  12.539 -9.472 0.275  PR0375 CB 15.165 -8.072 0.76  PR0375 CG 16.384 -8.221 1.658  PR0375 CD 15.865 -8.159 3.084  GLY376 N 12.945 -10.158 2.371  GLY376 CA 12.162 -11.386 2.21  GLY376 C 11.071 -11.52 3.271  GLY376  11.012 -12.523 3.992  ILE377 N 10.225 -10.508 3.367  ILE377 CA 9.092 -10.568 4.299</span>135- (130) 200418849 2753 GLN364 2754 GLN364 C  2755 GLN364 CG 2756 GLN364 CD 2757 GLN364 OE1 2758 GLN364 NE2 2759 ILE365 N 2760 ILE365 CA 2761 ILE365 C 2762 ILE365 2763 ILE365 C  2764 ILE 365365 CG1 ILE1 2767 VAL366 N 2768 VAL366 CA 2769 VAL366 C 2770 VAL366 2771 VAL366 CB 2772 VAL366 CG1 2773 VAL366 CG2 2774 PHE367 N 2775 PHE367 CA 2776 PHE367 C 2777 PHE367 2778 PHE367 C  2779 PHE367 CG 2780 PHE367 CD1 2781 2781 PHE367 CE2 2784 PHE367 CZ 2785 ASP368 N 2786 ASP368 CA 2787 ASP368 C 2788 ASP368 2789 ASP368 C  2790 ASP368 CG 2791 ASP368 D1 2792 ASP368 D2 2793 THR369 N 2794 THR369 CA 2795 THR369 C 2796 THR369 2797 THR369 C  2798 T G1 2799 THR369 CG2 2800 LEU370 N 2801 LEU370 CA 2802 LEU370 C 2803 LEU370  2804 LEU370 CB 2805 LEU370 CG 2806 LEU370 CD1 8.219 4.588 4.843 6.505 7.308 3.86 7.624 7.232 2.83 7.846 8.592 2.181 7,741 9.637 2.8 35 8.108 8.565 0.886 7.707 6.019 6.503 8.879 5.644 7.298 8.835 4.171 7.691 9.772 3.438 7.339 8.94 6.511 8.554 9.202 7.973 8.208 10.012 5.996 9.506 10.539 8.151 7.497 7.667 3.695 8.099 7.555 2.301 8.53 7.703 1.335 7.361 8.611 0.497 7.417 6.21 2.076 9.217 6.011 0.606 9.565 6.085 2.923 10.477 1.641 6.225 7.145 0.701 5.097 8.524 0.663 4.453 9.066 -0.434 4.254 6.125 1.099 4.034 4.662 1.012 4.458 3.726 1.83 3.841 4.258 0.112 5.435 2.391 1.767 4.216 2.924 0.052 5.813 1.99 0-88 5.206 9.187 1.808 4.423 10.522 1.869 3.83 11.52 1.102 4.686 12.115 0.138 4.185 10.964 3.326 3.326 3.707 10.101 4.09 2.701 10.208 5.31 2.678 9.508 3.443 1.847 11.438 1.285 5.995 12.381 0.606 6.892 12.07 -0.88 7.076 13.004 -1.658 7.287 12.376 1.294 8.252 11.058 1.242 8.778 12.797 2.756 8.149 10.848 -1.302 6.793 10.523 -2.729 6.862 11.145 -3.486 5.703 12.038 -4.319 5.916 9.012 -012 2.919 6.78 8.302 -2.463 8.044 6.792 -2.589 7.882 136- (131) LEU370 CD2 8.79 -3.256 9.248 PHE371 N 10.872 -3.006 4.502 PHE371 CA 11.266 -3.75 3.303 PHE371 C 12.728 -3.529 2.918 PHE371 13.332 -4.398 2.278 PHE371 CB 10.333 -3.34 2.169 PHE371 CG 8.861 -3.634 2.463 PHE371 CD1 7.935 -2.599 2.503 PHE371 CD2 8.447 -4.94 2.692 PHE371 CE1 -2.867 2.784 PHE371 CE2 7.112 -5.209 2.971 PHE371 CZ 6.19 -4.172 3.019 ARG372 N 13.335 -2.472 3.436 ARG372 CA 14.773 -2.281 3.231 ARG372 C 15.606 -2.901 4.353 ARG372 16.834 -2.982 4.229 ARG372 CB 15.084 -0.795 3.116 ARG372 CG 14.397 -0.1 1.904 ARG372 CD 14.777 1.291 1.76 ARG372 NE 14.497 2.015 3.008 ARG372 CZ 14.919 3.257 3.251 ARG372 NH1 14.646 3.835 4.423 ARG372 NH2 15.631 3.911 2.331 ARG373 N 14.958 -3.347 5.418 ARG373 CA 15.671 -4.087 6.457 ARG373 C 15.659 -5.559 6.09 ARG373 3.8. ARG373 CB 14.937 -3.926 7.783 ARG373 CG 15.657 -4.637 8.922 ARG373 CD 16.912 -3.879 9.335 ARG373 NE 16.548 -2.554 9.862 ARG373 CZ 16.405 -2.304 11.165 ARG373 NH1 16.663 -3.261 12.059 ARG373 NH2 16.046 -1.087 11.576 VAL374 N 14.474 -6.032 5.749 VAL374 CA 14.314 -7.425 5.338 VAL374 C 13.644 -7.499 3.971 VAL374 12.43 -7.307 3.841 VAL374 CB 13.473 -8.164 6.376 VAL374 CG1 13.297 -9.625 5.984 VAL374 CG2 14.086 -8.075 7.77 PR0375 N 14.432  7.87373. CA 13.929 -8.036 1.606 PR0375 C 13.088  9.304 1.369 PR0375 12.539 -9.472 0.275 PR0375 CB 15.165 -8.072 0.76 PR0375 CG 16.384 -8.221 1.658 PR0375 CD 15.865 -8.159 3.084 GLY376 N 12.945 -10.158 2.371 GLY376 CA 12.162 -11.386 2.21 GLY376 C 11.071 -11.52 3.271 GLY376 11.012 -12.523 3.992 ILE377 N 10.225 -10.508 3.367 ILE377 CA 9.092 -10.568 4.299</span>
  </p>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <span class="patent-image-not-available style-scope patent-text"></span>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">137- (132)</span>137- (132)</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">2861 ILE377 C  2862 ILE377   2863 ILE377 CB  2864 ILE377 CG1  2865 ILE377 CG2  2866 ILE377 CD1  2867 ARG378 N  2868 ARG378 CA  2869 ARG378 C  2870 ARG378   2871 ARG378 C  2872 ARG378 CG  2873 ARG378 CD  2874 ARG378 NE  2875 ARG378 CZ  2876 ARG378 NH1  2877 ARG378 NH2  2878 ILE379 N  2879 ILE379 CA  2880 ILE379 C  2881 ILE379   2882 ILE379 CB  2883 ILE379 CG1  2884 ILE379 CG2  2885 ILE379 CD1  2886 ALA380 N  2887 ALA380 CA  2888 ALA380 C  2889 ALA380   2890 ALA380 CB  2891 VAL381 N  2892 VAL381 CA  2893 VAL381 C  2894 VAL381   2895 VAL381 CB  2896 VAL381 CG1  2897 VAL381 CG2  2898 PR0382 N  2899 PR0382 CA  2900 PR0382 C  2901 PR0382   2902 PR0382 CB  2903 PR0382 CG  2904 PR0382 CD  2905 VAL383 N  2906 VAL383 CA  2907 VAL383 C  2908 VAL383   2909 VAL383 CB  2910 VAL383 CG1  2911 VAL383 CG2  2912 ASP384 N  2913 ASP384 CA  2914 ASP384 C 7.921 -11.29b   7.217 -10.731 2.801  8.663 -9.148 4.656  9.836 -8.352 5.203  7.529 -9.169 5.674  9.433 -6.919 5.526  7.743 -12.55 4.009  6.648 -13.342 3.451  5.468 -13.417 4.409  5.629 -13.304 5.627  7.186 -14.734 3.163  8.265 -14.671 2.089  8.975 -16.01 1.929  9.756 -16.33 3.134  9.57 -17.431 3.864  8.587 -18.28 3.556  10.338 -17.659 4.931  4.277 -13.531 3.854  3.096 -13.713 4.703  3.14 -15.13 5.272  3.519 &amp;gt;16.07 4.563  1.841 -13.536 3.855  2.108 -12.589 2.692  0.702 12.984 4.709  0.882 -12.447 1.798  2.872 -15.267 6.56  2.895 -16.598 7.174  1.533 -17.277 7.07  1.435 -18.51 7.097  3.306 -16.471 8.635  0.498 -16.47 6.917  -0.839 -17.005 6.651  -1.231 -16.745 5.2  -0.782 15.768 4.59  -1.847 -16.359 7.599  -1.705 -16.897 9.018  -1.747 -14.839 7.57  -1.999 -17.662 4.635  -2.615 -17.424 3.329  -3.477 -16.166 3.352  -4.045 -15.802 4.391  -3.422 -18.651 3.039  -3.29 -19.627 4.198  -2.414 -18.938 5.231  -3.721 -15.621 2.172  -4.415 -14.327 2.051  -5.892 -14.388 2.452  -6.376 -13.473 3.126  -4.302 -13.886 0.593  -5.05 -12.578 0.343  -2.838 -13.751 0.177  6.478 -15.572 2.355  -7.876 -15.767 2.759  -8.031 -15.962 4.271 -138- (133)200418849 2915 ASP384  -9.156 -16.094 4.761  2916 ASP384 CB -8.42 -17.003 2.048  2917 ASP384 CG -8.293 -16.849 0.534  2918 ASP384 D1 -9.1 -16.128 -0.032  2919 ASP384 D2 -7.312 -17.346 -0.002  2920 GLU385 N -6.926 -15.995 5  2921 GLU385 CA -6.994 -16.177 6.448  2922 GLU385 C -6.674 -14.884 7.194  2923 GLU385  6.638 -14.896 8.429  2924 GLU385 CB -6.01 -17.259 6.874  2925 GLU385 CG -6.219 -18.561 6.111  2926 GLU385 CD -7.651 -19.079 6.248  2927 GLU385 E1 -8.017 -19.462 7.349  2928 GLU385 OE2 -8.266 -19.256 5.205  2929 LEU386 N -6.406 -13.81 6.463  2930 LEU386 CA -6.082 -12.519 7.093  2931 LEU386 C -7.266 -11.953 7.874  2932 LEU386  -8.342 -11.71 7.315  2933 LEU386 CB -5.676 -11.542 5.996  2934 LEU386 CG -4.348 -11.943 5.365  2935 LEU386 CD1 -4.081 -11.153 4.091  2936 LEU386 CD2 -3.204 -11.773 6.357  2937 PR0387 N -7.063 -11.798 9.173  2938 PR0387 CA -8.132 &amp;gt;11.39 10.091  2939 PR0387 C 8.419 -9.89 10.047  2940 PR0387  -7.84 -9.095 10.805  2941 PR0387 CB -7.647 -11.801 11.445  2942 PR0387 CG -6.191 -12.224 11.339  2943 PR0387 CD -5.817 -12.105 9.873  2944 PHE388 N -9.314 9.528 9.143  2945 PHE388 CA -9.775 -8.145 9.012  2946 PHE388 C -10.688 -7.79 10.176  2947 PHE388  -11.522 -8.597 10.603  2948 PHE388 CB -10.558 -7.999 7.709  2949 PHE388 CG -9.785 -8.343 6.437  2950 PHE388 CD1 -8.78 -7.498 5.987  2951 PHE388 CD2 -10.097 -9.492 5.721  2952 PHE388 CE1 -8.076 -7.809 4.831  2953 PHE388 CE2 -9.393 9.804 4.565  2954 PHE388 CZ -8.381 -8.963 4.121  2955 LYS389 N -10.5 -6.599 10.707  2956 LYS389 CA -11.364 -6.141 11.792  2957 LYS389 C -12.626 -5.52 11.203  2958 LYS389  &amp;gt;12.542 -4.581 10.4  2959 LYS389 CB -10.611 -5.115 12.633  2960 LYS389 CG -11.439 -4.708 13.847  2961 LYS389 CD -10.677 -3.763 14.767  2962 LYS389 CE -11.487 -3.466 16.023  2963 LYS389 NZ -10.719 -2.637 16.96  2964 HIS390 N -13.775 -6.068 11.571  2965 HIS390 CA -15.055 -5.523 11.102  2966 HIS390 C -15.385 -4.226 11.836  2967 HIS390  -15.845 -4.213 12.983  2968 HIS390 CB -16.162 -6.548 11.316 139- (134)200418849 2969 HIS390  2970 HIS390  2971 HIS390  2972 HIS390  2973 HIS390  2974 ASP391  2975 ASP391  2976 ASP391  2977 ASP391  2978 ASP391  2979 ASP391  2980 ASP391  2981 ASP391  2982 SER392  2983 SER392  2984 SER392  2985 SER392  2986 SER392  2987 SER392  2988 THR393  2989 THR393  2990 THR393  2991 THR393  2992 THR393  2993 THR393  2994 THR393  2995 ILE394  2996 ILE394  2997 ILE394  2998 ILE394  2999 ILE394  3000 ILE394  3001 ILE394  3002 ILE394  3003 TYR395  3004 TYR395  3005 TYR395  3006 TYR395  3007 TYR395  3008 TYR395  3009 TYR395  3010 TYR395  3011 TYR395  3012 TYR395  3013 TYR395  3014 TYR395  3015 GLY396  3016 GLY396  3017 GLY396  3018 GLY396  3019 LEU397  3020 LEU397  3021 LEU397  3022 LEU397</span>2861 ILE377 C 2862 ILE377 2863 ILE377 CB 2864 ILE377 CG1 2865 ILE377 CG2 2866 ILE377 CD1 2867 ARG378 N 2868 ARG378 CA 2869 ARG378 C 2870 ARG378 2871 ARG378 C  2872 ARG378 CG 2873 ARG378 CD 2874 ARG378 CZ 2875 ANH378 2NH ARG378 NH2 2878 ILE379 N 2879 ILE379 CA 2880 ILE379 C 2881 ILE379 2882 ILE379 CB 2883 ILE379 CG1 2884 ILE379 CG2 2885 ILE379 CD1 2886 ALA380 N 2887 ALA380 CA 2888 ALA380 C 2889 ALA380 2890 ALA380 CB 2891 VAL 891 2381 VAL 2894 VAL381 2895 VAL381 CB 2896 VAL381 CG1 2897 VAL381 CG2 2898 PR0382 N 2899 PR0382 CA 2900 PR0382 C 2901 PR0382 2902 PR0382 CB 2903 PR0382 CG 2904 PR0382 CD 2905 VAL383 N 2906 VAL383 CA 2907 VAL383 C 2908 VAL10 CG1 2911 VAL383 CG2 2912 ASP384 N 2913 ASP384 CA 2914 ASP384 C 7.921 -11.29b 7.217 -10.731 2.801 8.663 -9.148 4.656 9.836 -8.352 5.203 7.529 -9.169 5.674 9.433 -6.919 5.526 7.743 -12.55 4 .009 6.648 -13.342 3.451 5.468 -13.417 4.409 5.629 -13.304 5.627 7.186 -14.734 3.163 8.265 -14.671 2.089 8.975 -16.01 1.929 9.756 -16.33 3.134 9.57 -17.431 3.864 8.587 -18.28 3.556 10.338 -17.659 4.931 4.277 -13.531 3.854 3.096 -13.713 4.703 3.14 -15.13 5.272 3.519 &amp;gt; 16.07 4.563 1.841 -13.536 3.855 2.108 -12.589 2.692 0.702  12.984 4.709 0.882 -12.447 1.798 2.872 -15.267 6.56 2.895 -16.598 7.174 1.533 -17.277 7.07 1.435 -18.51 7.097 3.306 -16.471 8.635 0.498 -16.47 6.98 0.839 -17.005 6.651 -1.231 -16.745 5.2 -0.782  15.768 4.59 -1.847 -16.359 7.599 -1.705 -16.897 9.018 -1.747 -14.839 7.57 -1.999 -17.662 4.635 -2.615 -17.424 3.329 -3.477 -16.166 3.352 -4.045 -15.802 4.391- 3.422 -18.651 3.039 -3.29 -19.627 4.198 -2.414 -18.938 5.231 -3.721 -15.621 2.172 -4.415 -14.327 2.051 -5.892 -14.388 2.452 -6.376 -13.473 3.126 -4.302 -13.886 0.593 -5.05 -12.578 0.343 -2.838 -13.751 0.177  6.478 -15.572 2.355 -7.876 -15.767 2.759 -8.031 -15.96 2 4.271 -138- (133) 200418849 2915 ASP384 -9.156 -16.094 4.761 2916 ASP384 CB -8.42 -17.003 2.048 2917 ASP384 CG -8.293 -16.849 0.534 2918 ASP384 D1 -9.1 -16.128 -0.032 2919 ASP384 D2 -7.312- 17.346 -0.002 2920 GLU385 N -6.926 -15.995 5 2921 GLU385 CA -6.994 -16.177 6.448 2922 GLU385 C -6.674 -14.884 7.194 2923 GLU385 6.638 -14.896 8.429 2924 GLU385 CB -6.01 -17.259 6.874 2925 GLU385 CG -6.219 -18.561 6.111 2926 GLU385 CD -7.651 -19.079 6.248 2927 GLU385 E1 -8.017 -19.462 7.349 2928 GLU385 OE2 -8.266 -19.256 5.205 2929 LEU386 N -6.406 -13.81 6.463 2930 LEU386 CA -6.082 -12.519 7.093 2931 LEU386 C -7.266 -11.953 7.874 2932 LEU386 -8.342 -11.71 7.315 2933 LEU386 CB -5.676 -11.542 5.996 2934 LEU386 CG -4.348 -11.943 5.365 2935 LEU386 CD1 -4.081 -11.153 4.091 2936 LEU386 CD2 -3.204 -11.773 6.357 2937 PR0387 N -7.063 -11.798 9.173 2938 CA -8.132 &amp;gt; 11.39 10.091 2939 PR0387 C  8.419 -9.89 10.047 2940 PR0387 -7.84 -9.095 10.805 29 41 PR0387 CB -7.647 -11.801 11.445 2942 PR0387 CG -6.191 -12.224 11.339 2943 PR0387 CD -5.817 -12.105 9.873 2944 PHE388 N -9.314  9.528 9.143 2945 PHE388 CA -9.775 -8.145 9.012 2946 PHE388 C -10.688 -7.79 10.176 2947 PHE388 -11.522 -8.597 10.603 2948 PHE388 CB -10.558 -7.999 7.709 2949 PHE388 CG -9.785 -8.343 6.437 2950 PHE388 CD1 -8.78 -7.498 5.987 2951 PHE388 CD2 -10.097 -9.492 5.721 2952 PHE388 CE1 -8.076 -7.809 4.831 2953 PHE388 CE2- 9.393  9.804 4.565 2954 PHE388 CZ -8.381 -8.963 4.121 2955 LYS389 N -10.5 -6.599 10.707 2956 LYS389 CA -11.364 -6.141 11.792 2957 LYS389 C -12.626 -5.52 11.203 2958 LYS389  &amp;gt; 12.542 -4.581 10.4 2959 LYS389 CB -5.115 12.633 2960 LYS389 CG -11.439 -4.708 13.847 2961 LYS389 CD -10.677 -3.763 14.767 2962 LYS389 CE -11.487 -3.466 16.023 2963 LYS389 NZ -10.719 -2.637 16.96 2964 HIS390 N -13.775 -6.068 11.571 2965 HIS390 CA -15.055 11.102 2966 HIS390 C -15.385 -4.226 11.836 2967 HIS390 -15.845 -4.213 1 2.983 2968 HIS390 CB -16.162 -6.548 11.316 139- (134) 200418849 2969 HIS390 2970 HIS390 2971 HIS390 2972 HIS390 2973 HIS390 2974 ASP391 2975 ASP391 2976 ASP391 2977 ASP391 2978 ASP391 2979 ASP391 2980 ASP391 2981 ASP391 2982 SER392 2983 SER392 2984 SER392 2987 SER392 2988 THR393 2989 THR393 2990 THR393 2991 THR393 2992 THR393 2993 THR393 2994 THR393 2995 ILE394 2996 ILE394 2997 ILE394 2998 ILE394 2999 ILE394 3000 ILE394 3001 ILER 30010 TYR3953 TYR3955 3005TY 3005TY 3005TY 3005 TYR395 3012 TYR395 3013 TYR395 3014 TYR395 3015 GLY396 3016 GLY396 3017 GLY396 3018 GLY396 3019 LEU397 3020 LEU397 3021 LEU397 3022 LEU397</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">CGND1CD2CE1NE2NCAC OSCGOD1OD2NCA i 2 i 2 1 u 2 i. 2 B G A bgg a bggd a bgddeezh a a CCNCCCCNCCCCCCNCCCCCCCCCNCCNCC -17.525 -17.895 -18.62 -19.181 -19.629 -15.053 -15.269 -15.392 -14.395 -14.068 -14.172 -14.984 -13.241 -16.582 -16.835 -16.261 -15.99 -18.342 -18.876 -15.944 -15.451 -13.931 -13.294 -16.183 -17.557 -15.703 -13.365 -11.899 -11.525 -11.205 -11.37 -11.744 -9.847 -10.977 -11.59 -11.29 9.792 -8.997 -11.747 -11.784 -10.958 -12.648 -10.991 -12.682 -11.852 -11.882 -9.433 8.007 -7.732 -8.601 -6.493 -6.031 -5.334 -4.297 6.094 -5.893 -5.81 -5.487 -5.437 -3.138 -1.789 -0.871 -0.539 -1.414 -0.02 0.771 0.324 -0.335 0.448 1.863 2.431 0.537 1.383 2.392 3.768 3.822 4.722 4.53 4.561 5.972 2.816 2.665 1.204 0.419 3.095 4.533 2:978 5.529 0.854 -0.503 -0.786 0.149 -0.572 -1.967 -2.272 &amp;gt;2.927 -3.543 -4.199 -4.502 -5.763 -2.053 -2.401 -3.871 -4.74 -4.132 -5.497 -5.577 -4.938 10.826 9.545 11.607 9.511 10.786 11.167 11.683 10.48 9.835 12.553 13.18 12.707 13.884 10.283 9.075 9.129 8.07 8.868 9.878 10.297 10.313 10.468 9.909 11.411 11.04 11.528 11.118 11.203 10.92 11.824 12.577 12.921 12.624 12.055 9.649 9.179 9.302 9.447 7.721 7.101 6.026 7.612 5.465 7.052 5.982 5.427 9.421 9.468 9.759 9.609 10.134 10.406 11.751 11.961 -140- (135)200418849 3023 LEU397  3024 LEU397  3025 LEU397  3026 LEU397  3027 HIS398  3028 HIS398  3029 HIS398  3030 HIS398  3031 HIS398  3032 HIS398  3033 HIS398  3034 HIS398  3035 HIS398  3036 HIS398  3037 ALA399  3038 ALA399  3039 ALA399  3040 ALA399  3041 ALA399  3042 LEU400  3043 LEU400  3044 LEU400  3045 LEU400  3046 LEU400  3047 LEU400  3048 LEU400  3049 LEU400  3050 PRO401  3051 PRO401  3052 PRO401  3053 PRO401  3054 PRO401  3055 PRO401  3056 PRO401  3057 VAL402  3058 VAL402  3059 VAL402  3060 VAL402  3061 VAL402  3062 VAL402  3063 VAL402  3064 THR403  3065 THR403  3066 THR403  3067 THR403  3068 THR403  3069 THR403  3070 THR403  3071 TRP404  3072 TRP404  3073 TRP404  3074 TRP404  3075 TRP404  3076 TRP404</span>CGND1CD2CE1NE2NCAC OSCGOD1OD2NCA i 2 i  2 1 u 2 i. 2 BGA bgg a bggd a bgddeezh aa CCNCCCCNCCCCCNCCCCCCCCCNCCNCC-17.525 -17.895 -18.62 -19.181 -19.181-19.629 -15.053 -15.269 -15.392 -14.395 -14.068 -14.172 -14.984 -13.241 -16.582 -16.835 -16.261 -15.99 -18.342 -18.876 -15.944 -15.451 -13.931 -13.294 -16.183 -17.557 -15.703 -13.365 -11.205 -11.899 -11.525 -11.37 -11.744 -9.847 -10.977 -11.59 -11.29  9.792 -8.997 -11.747 -11.784 -10.958 -12.648 -10.991 -12.682 -11.852 -11.882 -9.433  8.007 -7.732 -8.601 -6.493 -6.031 -5.334 -4.297 '6.094 -5.893 -5.81 -5.487 -5.437 -3.138 -1.789 -0.871 -0.539 -1.414 -0.02 0.771 0.324 -0.335 0.448 1.863 2.431 0.537 1.383 2.392 3.768 3.822 4.722 4.53 4.561 5.972 2.816 2.665 1.204 0.419 3.095 4.533 2: 978 5.529 0.854 -0.503 -0.503- 0.786 0.149 -0.572 -1.967 -2.272 &amp;gt; 2.927 -3.543 -4.199 -4.502 -5.763 -2.053 -2.401 -3.871 -4.74 -4.132 -5.497 -5.577 -4.938 10.826 9.545 11.607 9.511 10.786 11.167 11.683 10.48 9.835 12.553 13.18 12. 707 13.884 10.283 9.075 9.129 8.07 8.868 9.878 10.297 10.313 10.468 9.909 11.411 11.04 11.528 11.118 11.203 10.92 11.824 12.577 12.921 12.624 12.055 9.649 9.179 9.302 9.447 7.721 7.101 6.026 7.612 5.465 7.052 5.982 5.427 9.421 9.468 9.759 9.609 10.134 10.406 11.751 11.961 -140- (135) 200418849 3023 LEU397 3024 LEU397 3025 LEU397 3026 LEU397 3027 HIS398 3028 HIS398 3029 HIS398 3030 HIS398 3031 HIS398 3032 HIS398 3033 HIS398 3034 HIS398 3035 HIS398 3036 HIS398 3037 ALA399 30LE LEUUU 3030 3044 LEAUU 3042 ALA399 3040 ALA400 3048 LEU400 3049 LEU400 3050 PRO401 3051 PRO401 3052 PRO401 3053 PRO401 3054 PRO401 3055 PRO401 3056 PRO401 3057 VAL402 3058 VAL402 3059 VAL402 3060 VAL402 3061 VAL402 3062 VAL402 3063 VAL402 3064 THR403 3065 THR403 3066 THR403 3067 THR403 30HR THR403 3068 THR403 30HR 3073 TRP404 3074 TRP404 3075 TRP404 3076 TRP404</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">CBCGCDCDNCACOCBCGND8CENENCACOCBNCACOCBCG88NCACOCBCGCDNCACOCBCGCGNCACOCBOGCGNCACOCBCG -5.051 -5773 -5.037 -5.979 -5.87 -5.274 4.348 -3.651 -6.363 -6.737 -7.052 -6.795 -7.311 -7.152 -4.306 -3.343 -2.7 3.373 -4.014 -1.383 -0.567 0.772 1.712 -0.307 0.615 0.005 0.973 0.815 2.084 2.88 2.413 1.686 0.197 -0.308 4.074 5.008 6.358 7.008 5.194 3.968 5.553 6.772 8.039 9.135 9.102 7.888 6.715 9.077 10.089 11.177 12.136 12.984 11.969 11.163 -5.894 -6.502 -6.225 -7.996 -6.402 6.514 -7.718 -7.848 -6.528 -5.14 -4.804 -3.984 -3.482 -2.975 -8.567 -9.671 -9.903 -10.014 -10.936 -10.004 -10.204 -10.856 -10.165 -8.845 -8.952 -9.826 -7.572 -12.178 -12.9 -12.822 -13.242 -14.312 -14.466 -13.08 -12.267 -12.183 -12.814 -12.485 -10.723 10.185 -9.84 -13.729 -14.428 -13.709 -13.66 -15.853 -16.403 -16.723 -13.156 -12.406 -13.344 -12.835 -11.654 -10.949 9.311 8.12 6.822 8.325 12.634 13.967 14.107 15.12 15.033 15.525 16.79 14.781 16.851 15.607 13.094 13.12 11.756 10.724 13.633 11.794 10.589 10.935 11.35 9.946 8.736 7.644 8.197 10.881 10.974 9.671 8.602 11.269 10.993 10.625 9.78 8.653 8.998 9.998 .8.25 7.523 9.44 8.146 8.342 7.571 6.335 7.827 8.406 8.22 8.298 7.66 6.931 6.21 8.719 9.79 141 - (136) TRP404 TRP404 TRP404 TRP404 TRP404 TRP404 TRP404 TRP404 TRP404 HEM1 HEM1 HEM1 HEM1 HEM1 HEM1 HEM1 HEM1 HEM1 HEM1 HEM1 HEM1 HEM1 HEM1 HEM1 IHEM1 HEM1 HEM1 HEM1 HEM1 HEM1 HEM1 HEM1 HEM1 HEM1 HEM1 HEM1 HEM1 HEM1 HEM1 HEM1 HEM1 HEM1 HEM1 HEM1 HEM1 HEM1 HEM1 HEM1 HEM1 HEM1 HEM1 HEM1</span>CBCGCDCDNCACOCBCBCGND8CENENCACOCBNCACOCBCG88NCACOCBCGCDNCACOCBCGCGNCACOCBOGCGNCACOCBCG -5.051 -5773 -5.037 -5.979 -5.87 -5.274  4.348 -3.651 -6.363 -6.737 -73 -73 -73 -67 -73 0.815 2.084 2.88 2.413 1.686 0.197 -0.308 4.074 5.008 6.358 7.008 5.194 3.968 5.553 6.772 8.039 9.135 9.102 7.888 6.715 9.077 10.089 11.177 12.136 12.984 11.969 11.163 -5.894 -6.502 -6.225 -7.996 -6.402  6.514 -7.718 -7.848 -6.528 -5.14 -4.804 -3.984 -3.482 -2.975 -8.567 -9.671 -9.903 -10.014 -10.936 -10.004 -10.204 -10.856 -10.165 -8.845 -8.952 -9.826 -7.572 -12.178 -12.9 -12.822 -13.242 -14.312 -14.466 -13.08 -12.267 -12.183 -12.814 -12.485 -10.723  10.185 -9.84 -13.729 -14.428 -13.709 -13.66 -15.853 -16.403 -16.723 -13.156 -12.406 -13.344 -12.835 -11.654 -10.949 9.311 8.12 6.822 8.325 12.634 13.967 14.107 15.12 15.033 15.525 16.79 14.781 16.781 16.781 13.094 13.12 11.756 10.724 13.633 11.794 10.589 10.935 11.35 9.9 46 8.736 7.644 8.197 10.881 10.974 9.671 8.602 11.269 10.993 10.625 9.78 8.653 8.998 9.998 .8.25 7.523 9.44 8.146 8.342 7.571 6.335 7.827 8.406 8.22 8.298 7.66 6.931 6.21 8.719 9.79 141-(136) TRP404 TRP404 TRP404 TRP404 HRP1EM1 HEM404 HEM404 HEM404 HEM1 HEM1 HEM1 HEM1 HEM1 HEM1 HEM1 HEM1 HEM1 HEM1 HEM1 HEM1 HEM1 HEM1 HEM1 HEM1 HEM1 HEM1 HEM1 HEM1 HEM1 HEM1 HEM1 HEM1 HEM1 HEM1 HEM1 HEM1</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">12123232T aaaabb BBB1BBBCC DDEEEZZHXEA1234H1B432MABH1C CCNCCCCCOFNCCCCCCNCCCCCCCCN</span>12123232T aaaabb BBB1BBBCC DDEEEZZHXEA1234H1B432MABH1C CCNCCCCCOFNCCCCCCNCCCCCCCCCCN</span>
  </p>
  <p class="style-scope patent-text">
    <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">C4C C3C C2C CMC CAC CBC CHD C1D ND C4D C3D C2D CMD CAD CBD CGD 1D 2D CHA CMA CAA CBA CGA 1A 2A 10.886 -11.444  10.559 -9.637  10.155 10.524  9.943 -9.428  10.506 -8.656  9.278 -8.237  9.838 7.468  9.226 -7.257  12.117 -14.53  -8.08 12.05  -9.653 12.085  -10.7 13.004  -11.687 12.681  -11.292 11.525  -10.019 11.174  -9.224 10.115  -7.931 9.83  -7.308 10.582  -6.086 9.964  -5.946 8.85  -7.068 8.771  -7.416 7.755  -4.833 8.031  -4.44 7.051  -5.212 10.298  -5.439 11.223  -6.519 12.039  -6.227 12.887  -4.926 12.636  -4.491 11.556  -3.265 10.712  -4.462 13.435  -3.452 13.231  -7.061 13.855  -8.237 14.203  -8.777 13.572  -9.915 14.313  -10.045 15.413  -9.006 15.334  -8.71 16.241  -11.178 16.421  -10.91 17.624  -12.079 18.574  -13.198 18.167  -11.889 19.736  -10.849 14.026  -12.005 10.703  -12.907 13.51  -14.087 13.112  -15.442 13.596  -15.522 14.131  -16.439 13.4 142-</span>C4C C3C C2C CMC CAC CBC CHD C1D ND C4D C3D C2D CMD CAD CBD CGD 1D 2D CHA CMA CAA CBA CGA 1A 2A 10.886 -11.444 10.559 -9.637 10.155  10.524 9.943 -9.428 10.506 -8.656 9.278 -8.237 9.838 9.226 -7.257 12.117 -14.53 -8.08 12.05 -9.653 12.085 -10.7 13.004 -11.687 12.681 -11.292 11.525 -10.019 11.174 -9.224 10.115 -7.931 9.83 -7.308 10.582 -6.086 9.964 -5.946 8.85 -7.068 8.771 -7.416 7.755 -4.833 8.031 -4.44 7.051 -5.212 10.298 -5.439 11.223 -6.519 12.039 -6.227 12.887 -4.926 12.636 -4.491 11.556 -3.265 10.712 -4.462 13.435 -3.452 13.231 -7.061 13.855 -8.237 14.203 -8.777 13.572 -9.915 14.313 -10.045 15.413 -9.006 15.334 -8.71 16.241- 11.178 16.421 -10.91 17.624 -12.079 18.574 -13.198 18.167 -11.889 19.736 -10.849 14.026 -12.005 10.703 -12.907 13.51 -14.087 13.112 -15.442 13.596 -15.522 14.131 -16.439 13.4 142-</span>
  </p>
</description>
  </section>
      <div id="margin" class="style-scope patent-text">
        <div id="createHighlight" style="top: 0px;" class="style-scope patent-text" hidden="">
          <iron-icon id="comment" icon="editor:insert-comment" class="style-scope patent-text x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M20 2H4c-1.1 0-2 .9-2 2v12c0 1.1.9 2 2 2h14l4 4V4c0-1.1-.9-2-2-2zm-2 12H6v-2h12v2zm0-3H6V9h12v2zm0-3H6V6h12v2z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
        </div>
      </div>
    </div>
  </patent-text>
              </section>
            </div>
            <div class="flex flex-width style-scope patent-result">
              <section id="claims" class="style-scope patent-result">
                <h3 class="style-scope patent-result">
                  <div class="layout horizontal style-scope patent-result">
                    <div class="flex style-scope patent-result">Claims <span class="style-scope patent-result">(1)</span></div>
                    <div class="collapse style-scope patent-result">
                      Hide Dependent <iron-icon style="height: 15px; width: 15px;" icon="expand-less" class="style-scope patent-result x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M12 8l-6 6 1.41 1.41L12 10.83l4.59 4.58L18 14z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon><template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </div>
                  </div>
                  <div class="layout horizontal style-scope patent-result">
                    <div class="flex style-scope patent-result"></div>
                    <div class="righthead style-scope patent-result">translated from Chinese</div>
                  </div>
                </h3>
                <patent-text name="claims" enable-text-selection="" class="style-scope patent-result" style="padding-top: 0px;">
    
    
    <template is="dom-if" restamp="" class="style-scope patent-text"></template>
    <overlay-tooltip id="tooltip" class="style-scope patent-text" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      
    </div>
  </overlay-tooltip>
    <div class="layout horizontal style-scope patent-text">
      <section id="text" class="flex style-scope patent-text"><claims mxw-id="PCLM106681387" lang="EN" load-source="google" class="style-scope patent-text">
  <claim num="1" class="style-scope patent-text">
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">(1)  (1)200418849    1 . B   2 .i.  SEQ ID NO  13032343637383940 41  42  60  62  64  66  68  72  74   3 .  SEQ ID NO : 1  4.1 B   5 .1   SEQ ID NO : 2 GLU3 1 ARG67ARG88ILE92ALA93VAL106ILE130 ALA140  MET 176  PHE190  GLU23 1  SER294   PHE23 7ILE3 6 5  6 .1   SEQ ID NO : 2 LEU39, GLN43ALA45, MET57LEU58HIS62PHE63SER64, SER65, ASP66, ARG67GLN68, SER69LEU74MET75, VAL76, ALA77ARG78GLN79ILE80ASP84LYS85, PR086PHE87, ARG88PR089, SER90LEU91ILE92, ALA93MET94ASP95HIS99ARG103, PHE110ILE155, PHE169, GLN170, CYS172, SER173, SER174, ARG175,  -143 -  (2)  200418849  MET 1 76, LEU177, SERI 78, ARG1 79, ARG1 86,  LEU193, VAL23 3, GLY23 4, LEU23 5, AL A2 3 6,  LEU23 8, LEU23 9, LEU240, ILE241, AL A242,  HIS244 GLU24 5, THR246, THR247, ALA24 8,  MET2 5 0, LEU28 3, THR2 8 7, ILE2 8 8, AL A2 89,  THR29 1 AL A292, THR2 93, SER294, ARG29 5,  AL A297, THR298, GLU3 12, GLY3 1 3, VAL3 1 4,  GL Y3 1 6, VAL3 44, ALA3 4 5, PHE3 46, GLY3 47,  V AL3 5 0, HIS3 5 1 , GLN3 5 2, C YS 3 5 3 LEU3 54,  GLN3 5 6, LEU3 5 8, ALA3 5 9, GLU3 62, LYS3 8 9,  SER3 92, THR3 9 3, ILE394  TYR3 9 5    7 .1  SEQ ID NO: 434445464748 %    8 . 1   9. 65C 1 6  5 5 C 0.5 X S S C3 0   10. SEQ ID NO : 2  11. SEQ ID NO: 2B  PHE190, PHE23 7, GLY24 3, ASN249, GLU290, PHE296, VAL3 1 5, PHE3 4 8, GLY3 5 5, ASP391   3 49  3X SSC 2      -144,  (3)  (3)200418849  SEQ ID NO : 2 B   12.SEQ ID NO : 2B SEQ ID NO : 2  GLU31ARG67ARG88ILE92ALA93 VAL106  ILE130  ALA 140  MET 1 76  PHE1 90   GLU231SER294PHE237 ILE3 6 5    13SEQ ID NO  2B  SEQ ID NO :  2   LEU39 GLN43 ALA45, MET5 7, LEU58,  HIS62, PHE63, SER64, SER65, ASP66 ARG67, GLN68,  SER69, LEU74, MET75, VAL76, ALA77, ARG78, GLN79,  ILE80 ASP84 LYS85, PR086, PHE87, ARG88 PR089,  SER90, LEU91 , ILE92, ALA93, MET94, ASP95 HIS99,  ARG103, PHE 1 1 0, ILE155, PHE169, GLN1 70 C YS 1 72,  SER1 73 SER1 74, ARG1 75, MET 1 76, LEU1 77, SER1 78,  ARG1 79, ARG186, PHE 1 90, LEU193, VAL 2 3 3, GLY23 4,  LEU23 5, ALA23 6, PHE23 7, LEU23 8, LEU23 9, LEU240,  ILE241 , ALA242 GLY243, HIS244 GLU245, THR246,  THR247, AL A248, ASN249, MET2 5 0, LEU28 3, THR28 7,  ILE2 8 8, ALA2 8 9, GLU290, THR2915 AL A292 THR293 ,  SER294, ARG295, PHE296, ALA297, THR298, GLU3 12,  -145-  (4)  (4)200418849  GLY3 13, V AL3 1 4, VAL3 1 5, GLY3 1 6, VAL3 44, ALA3 4 5,  PHE3 46, GLY3 47, PHE3 4 8, VAL3 5 0, HIS351 GLN3 5 2,  CYS 3 5 3, LEU3 5 4, GLY3 5 5, GLN3 5 6, LEU3 5 8, ALA3 5 9,  GLU3 62, LYS 3 8 9 ASP3 91 SER3 92THR3 9 3, ILE3 94   TYR3 95  14.   SEQ ID NO: 3133356163  6 56 76 97 17 37 5B    15.   SEQ ID NO: 4344454647  4849B  1 6 .   1 7 .1 6  SEQIDNO:3  1 8 . 1 6   19.65C 3X SSC   1 6SEQ ID NO : 3 SEQ ID NO : 3 5 5 C 0.5 X S S C3 0 SEQ ID NO: 3SEQ ID NO : 3  2  . 1   2 1 .2 0  -146-  (5)  (5)200418849    22.20    2 3 . 2 1    2 4 .   202 1   2 5 .    26. 25  SEQ ID NO : : 13032  34  36  37  38  39  40  41  42  60  62  64  66   6 87 27 4  27. I   HO-CH2-(Ai)n-(Q)m-(A2).E (I)   A iA2 Ci-Cs-  Q13  -147-  200418849</span>(1) (1) 200418849 Scope of patent application 1. An isolated nucleic acid sequence encoding epothilone B hydroxylase and its mutants and variants. 2. The isolated nucleic acid sequence according to item i of the patent application scope, which includes SEQ ID NOs: 1, 30, 32, 34, 36, 37, 38, 39, 40, 41, 42, 60, 62, 64 , 66, 68, 72 M 74  3. The isolated nucleic acid sequence according to the scope of the patent application, which includes SEQ ID NO: 1. 4. The isolated nucleic acid sequence according to item 1 of the patent application, which encodes a mutant having at least one amino acid substitution in the active site of the epothilone B hydroxylase enzyme. 5. The isolated nucleic acid sequence according to item 1 of the scope of patent application, which encodes at least one amino acid GLU3 1, SEQ ID NO: 2, ARG67, ARG88, ILE92, ALA93, VAL106, ILE130, ALA140, MET 176 , PHE190, GLU23 1, SER294, PHE23 7, or ILE3 6 5 mutants. 6. The isolated nucleic acid sequence according to item 1 of the scope of patent application, which encodes at least one amino acid LEU39, GLN43, ALA45, MET57, LEU58, HIS62, PHE63, SER64, SER65, ASP66 with SEQ ID NO: 2 , ARG67, GLN68, SER69, LEU74, MET75, VAL76, ALA77, ARG78, GLN79, ILE80, ASP84, LYS85, PR086, PHE87, ARG88, PR089, SER90, LEU91, ILE92, ALA93, MET94, ASP95, HIS99, ARG103, PHE110 , ILE155, PHE169, GLN170, CYS172, SER173, SER174, ARG175, -143-(2) 200418849 MET 1 76, LEU177, SERI 78, ARG1 79, ARG1 86, LEU193, VAL23 3, GLY23 4, LEU23 5, AL A2 3 6, LEU23 8, LEU23 9, LEU240, ILE241, AL A242, HIS244, GLU24 5, THR246, THR247, ALA24 8, MET2 5 0, LEU28 3, THR2 8 7, ILE2 8 8, AL A2 89, THR29 1, AL A292, THR2 93, SER294, ARG29 5, AL A297, THR298, GLU3 12, GLY3 1 3, VAL3 1 4, GL Y3 1 6, VAL3 44, ALA3 4 5, PHE3 46, GLY3 47, V AL3 5 0, HIS3 5 1, GLN3 5 2, C YS 3 5 3, LEU3 54, GLN3 5 6, LEU3 5 8, ALA3 5 9, GLU3 62, LYS3 8 9, SER3 92, THR3 9 3, ILE394 and TYR3 9 5 amine Acid variant. 7. The isolated nucleic acid sequence according to item 1 of the patent application scope includes SEQ ID NO: 43, 44, 45, 46, 47, 48% allosome. 8. A polypeptide encoded by the isolated core of the scope of patent application No. 1. 9. An isolated nucleic acid molecule capable of hybridizing to a patent application range of an acid sequence or a complementary sequence of the nucleic acid sequence under a hybridization condition at 65  C for 16 hours. C 0.5 XSSC to the nucleic acid sequence or the complementary sequence of the nucleic acid sequence under a 30-minute washing condition. 10. An isolated polypeptide comprising SEQ ID NO: 2 11. An epothilone B having SEQ ID NO: 2 PHE190, PHE23 7, GLY24 3, ASN249, GLU290, PHE296, VAL3 1 5, PHE3 4 8, GLY3 5 5, ASP391, the superseded nucleic acid molecule of item 3X SSC 2 of the acid-sequence sequence of its 3 49 can retain hybrid hydroxylase-144, (3) (3) 200418849 isolated mutation A polypeptide comprising an amino acid sequence substituted with at least one amino acid in the active site of an epothilone B hydroxylase enzyme having SEQ ID NO: 2. 12. An isolated mutant polypeptide having epothilone B hydroxylase of SEQ ID NO: 2, which encodes at least one amino acid GLU31, ARG67, ARG88, ILE92, ALA93 in SEQ ID NO: 2 , VAL106, ILE130, ALA 140, MET 1 76, PHE1 90, GLU231, SER294, PHE237, or ILE3 6 5 amino acid substituted amino acid sequence. 13  An isolated mutant polypeptide of epothilone B hydroxylase having SEQ ID NO  2, which encodes at least one amino acid LEU39, GLN43, ALA45, MET5 having SEQ ID NO: 2 , LEU58, HIS62, PHE63, SER64, SER65, ASP66, ARG67, GLN68, SER69, LEU74, MET75, VAL76, ALA77, ARG78, GLN79, ILE80, ASP84, LYS85, PR086, PHE87, ARG88, PR089, SER90, LEU91, LEU91 , ALA93, MET94, ASP95, HIS99, ARG103, PHE 1 1 0, ILE155, PHE169, GLN1 70, C YS 1 72, SER1 73, SER1 74, ARG1 75, MET 1 76, LEU1 77, SER1 78, ARG1 79, ARG186, PHE 1 90, LEU193, VAL 2 3 3, GLY23 4, LEU23 5, ALA23 6, PHE23 7, LEU23 8, LEU23 9, LEU240, ILE241, ALA242, GLY243, HIS244, GLU245, THR246, THR247, AL A248, ASN249, MET2 5 0, LEU28 3, THR28 7, ILE2 8 8, ALA2 8 9, GLU290, THR2915 AL A292, THR293, SER294, ARG295, PHE296, ALA297, THR298, GLU3 12, -145- (4) (4) 200418849 GLY3 13, V AL3 1 4, VAL3 1 5, GLY3 1 6, VAL3 44, ALA3 4 5, PHE3 46, GLY3 47, PHE3 4 8, VAL3 5 0, HIS351, GLN3 5 2, CYS 3 5 3, LEU 3 5 4, GLY3 5 5, GLN3 5 6, LEU3 5 8, ALA3 5 9, GLU3 62, LYS 3 8 9, ASP3 91, SER3 92, THR3 9 3, ILE3 94 and TYR3 95 mutant. 14. An isolated mutant polypeptide comprising epothilone B hydroxylase comprising SEQ ID NO: 31, 33, 35, 61, 63, 6 5, 6, 7, 6, 9, 7, 1, 3 or 75 .  15.  An isolated mutant polypeptide of epothilone B hydroxylase including SEQ ID NO: 43, 44, 45, 46, 47, 48, or 49. 16. An isolated nucleic acid sequence encoding a ferredoxin. 17. The isolated nucleic acid sequence according to item 16 of the patent application, which includes SEQ ID NO: 3. 18. A polypeptide encoded by the isolated nucleic acid sequence in the 16th patent application. 19. An isolated nucleic acid molecule capable of hybridizing to a nucleic acid sequence of SEQ ID NO: 3 or a complementary sequence of the nucleic acid sequence of SEQ ID NO: 3 under hybridization conditions at 65  C 3X SSC for 16 hours, The isolated nucleic acid molecule can maintain hybridization to the nucleic acid sequence of SEQ ID NO: 3 or the complementary sequence of the nucleic acid sequence of SEQ ID NO: 3 under washing conditions at 5  C 0.5 XSSC for 30 minutes. 2..  A vector comprising the isolated nucleic acid sequence of the scope of patent application No. 1. 2 1. The vector of claim 20 in the scope of patent application, further comprising an isolated nucleic acid sequence of -146- (5) (5) 200418849 code iron redox protein. 22. A host cell comprising a vector such as the one claimed in claim 20. 2 3. A host cell comprising a vector such as the item 21 of the patent application. 2 4. A method for manufacturing a recombinant microorganism which can hydroxylate epothilone having a terminal alkyl group to produce epothilone having a terminal hydroxyalkyl group, and the method includes using patent application No. 20 or 2 1 The vectors are used to transfect microorganisms. 25. A recombinantly produced microorganism that can hydroxylate epothilone with a terminal alkyl group to produce epothilone with a terminal hydroxyalkyl group. 26. For example, a microorganism produced by recombination produced in the scope of application for a patent No. 25, wherein the microorganism can express SEQ ID NO :: 1, 30, 32, 34, 36, 37, 38, 39, 40, 41, 42, Nucleic acid sequences of 60, 62, 64, 66, 68, 72 or 74. 27. A method for producing at least one epothilone having the following formula I HO-CH2- (Ai) n- (Q) m- (A2) .E (I) wherein A i and A 2 are independent The ground is selected from the group consisting of Ci-Cs-alkyl and alkenyl, which are optionally substituted; Q is a ring system which is optionally substituted, which includes 1 to 3 rings and -147- 200418849</span>
    </claim-text>
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text">
        <span class="google-src-text style-scope patent-text">
          <span class="patent-image-not-available style-scope patent-text"></span>
        </span>
        <span class="patent-image-not-available style-scope patent-text"></span>
      </span>
    </claim-text>
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">(6)    m1 m1   11 CH3-(Ai)n.(Q)m-(A2)-E (II)  AiQA2Enm  II     2 8 . A</span>(6) at least one carbon-to-carbon double bond in at least one ring; , m and 0 are integers selected from the group consisting of: 0 and 1, and at least one of m or  or  is 1, And E is an epothilone core; the method includes at least one epothilone CH3- (Ai) n. (Q) m- (A2) -E (II) having the following formula 11 , A2, E, n, m and 0 are all defined above; contact with a recombinantly produced microorganism capable of selectively catalyzing the hydroxylation of the formula II, or an enzyme derived therefrom, and promote the hydroxyl group Into. 2 8. Method for preparing epothilone analogue of formula A</span>
    </claim-text>
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text">
        <span class="google-src-text style-scope patent-text">
          <span class="patent-image-not-available style-scope patent-text"></span>
        </span>
        <span class="patent-image-not-available style-scope patent-text"></span>
      </span>
    </claim-text>
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">SEQ ID NO : 3 1 BBa   2 9 . A  -148-  200418849</span>The method includes biologically converting epothilone B to an epothilone analog of formula a via warm dipping with a mutant epothilone B hydroxylase enzyme comprising SEQ ID NO: 31. 2 9. A compound of formula A -148- 200418849</span>
    </claim-text>
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text">
        <span class="google-src-text style-scope patent-text">
          <span class="patent-image-not-available style-scope patent-text"></span>
        </span>
        <span class="patent-image-not-available style-scope patent-text"></span>
      </span>
    </claim-text>
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">  3   B 1 4.0 A</span>Or a pharmaceutically acceptable salt thereof. 3   a homomorphic motif of an epothilone B hydroxylase, which has a conservative residue skeleton of less than about 4.0 A when superimposed on the corresponding skeleton atom described in the structural coordinates listed in Appendix 1. Atomic root mean square deviation.</span>
    </claim-text>
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text">
        <span class="google-src-text style-scope patent-text">
          <span class="patent-image-not-available style-scope patent-text"></span>
        </span>
        <span class="patent-image-not-available style-scope patent-text"></span>
      </span>
    </claim-text>
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">3 1 . B   SEQ ID NO : 2 </span>31. A method for producing a mutant having altered biological properties, function, yield of a desired product, reaction rate, substrate specificity, or activity relative to epothilone B hydroxylase, the method The steps include: identifying an amino acid contained in SEQ ID NO: 2 to be mutated; and mutating the identified amino acid to create a mutant protein.</span>
    </claim-text>
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text">
        <span class="google-src-text style-scope patent-text">
          <span class="patent-image-not-available style-scope patent-text"></span>
        </span>
        <span class="patent-image-not-available style-scope patent-text"></span>
      </span>
    </claim-text>
    <claim-text class="style-scope patent-text">
      <span class="notranslate style-scope patent-text"> <span class="google-src-text style-scope patent-text">3 2.3 1 1 4.0 A B SEQ ID NO : 2  3 3 .3 1   SEQ ID NO :2 LEU39, GLN43, ALA45,  MET5 7, LEU58, HIS62 PHE63, SER64, SER65, ASP66,  ARG67, GLN68 , SER69 LEU74, MET75, VAL76, ALA77,  ARG78, GLN79, ILE80 ASP84, LYS85, PR086, PHE87,  ARG88, PR089, SER90, LEU91 ILE92, ALA93 MET94,   ASP95 HIS99, ARG103 PHE110 ILE155 PHE169,  -149-  (8)  200418849  GLN 1 70, C YS 1 72 SER173 SER1 74, ARG1 75, MET 1 76,  LEU 1 77, SER178, ARG1 79 ARG1 86, PHE 1 90, LEU193,  VAL 2 3 3, GLY23 4, LEU23 5 ALA2 3 6, PHE2 3 7, LEU2 3 8,  LEU23 9, LEU240 ILE241 ALA242, GLY243 HIS244,  GLU24 5, THR246 THR247, ALA248, ASN249, MET2 5 0,  LEU283, THR287 ILE28 8 ALA289, GLU290, THR29 1,  AL A292, THR293, SER294, ARG295, PHE296, ALA297,  THR298, GLU3 12, GLY3 13 VAL3 14, VAL3 15, GLY3 1 6,  VAL344, AL A3 4 5 PHE3 46 GLY3 47, PHE3 4 8, VAL 3 5 0,  HIS3 5 1 GLN3 5 2 CYS 3 5 3 LEU3 54 GLY 3 5 5, GLN3 5 6,  LEU3 5 8, AL A3 5 9 GLU3 62, LYS 3 89, ASP391 SER3 92,  THR3 93, ILE394  TYR3 9 5     3 4 .3 1    SEQ ID | NO : 2  GLU3 1 ARG67 ARG88  ILE92ALA93VAL106ILE130ALA140MET176 PHE190GLU23 1SER294PHE23 7 ILE3 65  3 5 .31 B   3 6 .3 5 F  3 7 .31 B    -150-  200418849  )  3 8 .3 1 B   3 9 .3 8 SER294  4 0 .3 1 B   4 1  1   -151 -</span>3 2. The method according to item 31 of the scope of patent application, wherein the roots of skeleton atoms with conservative residues of less than about 4.0 A are superimposed on the corresponding skeleton atoms described in the structural coordinates listed in Appendix 1. The homogeneity model of epothilone B hydroxylase with a square deviation to identify the amino acid contained in SEQ ID NO: 2 to be mutated. 3 3. The method according to item 31 of the scope of patent application, wherein the identified amino acid is LEU39, GLN43, ALA45, MET5 7, LEU58, HIS62, PHE63, SER64, SER65, ASP66 of SEQ ID NO: 2 , ARG67, GLN68, SER69, LEU74, MET75, VAL76, ALA77, ARG78, GLN79, ILE80, ASP84, LYS85, PR086, PHE87, ARG88, PR089, SER90, LEU91, ILE92, ALA93, MET94, called ASP95, HIS99, ARG PHE110, ILE155, PHE169, -149- (8) 200418849 GLN 1 70, C YS 1 72, SER173, SER1 74, ARG1 75, MET 1 76, LEU 1 77, SER178, ARG1 79, ARG1 86, PHE 1 90, LEU193, VAL 2 3 3, GLY23 4, LEU23 5, ALA2 3 6, PHE2 3 7, LEU2 3 8, LEU23 9, LEU240, ILE241, ALA242, GLY243, HIS244, GLU24 5, THR246, THR247, ALA248, ASN249, MET2 5 0, LEU283, THR287, ILE28 8, ALA289, GLU290, THR29 1, AL A292, THR293, SER294, ARG295, PHE296, ALA297, THR298, GLU3 12, GLY3 13, VAL3 14, VAL3 15, GLY3 1 6, VAL344, AL A3 4 5, PHE3 46, GLY3 47, PHE3 4 8, VAL 3 5 0, HIS3 5 1, GLN3 5 2, CYS 3 5 3, LEU3 54, GLY 3 5 5, GLN3 5 6, LEU3 5 8, AL A3 5 9, GLU3 62, LYS 3 89, ASP391, SER3 92, THR3 93, ILE394 and TYR3 9 5. 3 4. The method according to item 31 of the scope of patent application, wherein the identified amino acid is GLU3 1, SEQ ID | NO: 2, ARG67, ARG88, ILE92, ALA93, VAL106, ILE130, ALA140, MET176, PHE190, GLU23 1, SER294, PHE23 7, or ILE3 65. 35. The method of claim 31, wherein the mutant protein can improve the yield of the desired product relative to the yield of the desired product using epothilone B hydroxylase. 36. The method of claim 35, wherein the desired product is epothilone F. 37. The method according to item 31 of the scope of patent application, wherein the mutant can improve the reaction rate with respect to the reaction rate obtained by using epothilone B hydroxylase 0-150-200418849 0) 3 8 The method of item 31, wherein the mutant exhibits an altered receptor specificity with respect to the receptor specificity of epothilone B hydroxylase. 39. The method according to item 38 of the scope of patent application, wherein the amino group SER294 is mutated. 40. The method of claim 31, wherein the mutant exhibits a biological activity or% energy that is substantially similar to epothilone B hydroxylase. 4 1  A mechanically readable data storage medium that includes data storage data encoded by structural coordinates listed in Appendix 1. -151-</span>
    </claim-text>
  </claim>
</claims>
  </section>
      <div id="margin" class="style-scope patent-text">
        <div id="createHighlight" style="top: 0px;" class="style-scope patent-text" hidden="">
          <iron-icon id="comment" icon="editor:insert-comment" class="style-scope patent-text x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M20 2H4c-1.1 0-2 .9-2 2v12c0 1.1.9 2 2 2h14l4 4V4c0-1.1-.9-2-2-2zm-2 12H6v-2h12v2zm0-3H6V9h12v2zm0-3H6V6h12v2z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
        </div>
      </div>
    </div>
  </patent-text>
              </section>
            </div>
          </div>

          <div class="footer style-scope patent-result">

            <template is="dom-if" class="style-scope patent-result"></template>

            <template is="dom-if" class="style-scope patent-result"></template>

            
            <h3 id="citedBy" class="scroll-target style-scope patent-result">
              Cited By (1)
              <a href="#title" class="style-scope patent-result"><iron-icon icon="arrow-drop-up" class="style-scope patent-result x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M7 14l5-5 5 5z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon></a>
            </h3>
            <div class="responsive-table style-scope patent-result">
              <div class="table style-scope patent-result">
                <div class="thead style-scope patent-result">
                  <div class="tr style-scope patent-result">
                    <span class="th style-scope patent-result">Publication number</span>
                    <span class="th style-scope patent-result">Priority date</span>
                    <span class="th style-scope patent-result">Publication date</span>
                    <span class="th style-scope patent-result">Assignee</span>
                    <span class="th style-scope patent-result">Title</span>
                  </div>
                </div>
                <div class="tbody style-scope patent-result">
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                      <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/CN114480343A/en"><a id="link" href="#" class="style-scope state-modifier">CN114480343A</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2020-10-27</span>
                    <span class="td style-scope patent-result">2022-05-13</span>
                    <span class="td style-scope patent-result"></span>
                    <span class="td style-scope patent-result">PET hydrolase with improved activity 
       </span>
                  </div>
                  <template is="dom-repeat" class="style-scope patent-result"></template>

                  <div class="tr style-scope patent-result">
                    <span class="td famheader style-scope patent-result">Family To Family Citations</span>
                    <span class="td famheader style-scope patent-result"></span>
                    <span class="td famheader style-scope patent-result"></span>
                    <span class="td famheader style-scope patent-result"></span>
                    <span class="td famheader style-scope patent-result"></span>
                  </div>

                  <template is="dom-repeat" class="style-scope patent-result"></template>
                </div>

              </div>
            </div>
            <div style="padding: 10px 12px; font-size: 11px;" class="style-scope patent-result">* Cited by examiner,  Cited by third party,  Family to family citation</div>
            <template is="dom-if" class="style-scope patent-result"></template>

            
            <h3 id="similarDocuments" class="scroll-target style-scope patent-result">
              Similar Documents
              <a href="#title" class="style-scope patent-result"><iron-icon icon="arrow-drop-up" class="style-scope patent-result x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M7 14l5-5 5 5z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon></a>
            </h3>
            <div class="responsive-table style-scope patent-result">
              <div class="table style-scope patent-result">
                <div class="thead style-scope patent-result">
                  <div class="tr style-scope patent-result">
                    <span class="th style-scope patent-result">Publication</span>
                    <span class="th style-scope patent-result">Publication Date</span>
                    <span class="th style-scope patent-result">Title</span>
                  </div>
                </div>
                <div class="tbody style-scope patent-result">
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="scholar/8587822105951080055"><a id="link" href="#" class="style-scope state-modifier">Crawford et al.</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2009</span>
                    <span class="td style-scope patent-result">Structural basis for biosynthetic programming of fungal aromatic polyketide cyclization</span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      
                        <template is="dom-if" class="style-scope patent-result"></template>
                        
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/CA2510079A1/en"><a id="link" href="#" class="style-scope state-modifier">CA2510079A1</a></state-modifier>
                        <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2004-07-22</span>
                    <span class="td style-scope patent-result">Compositions and methods for hydroxilating epothilones 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      
                        <template is="dom-if" class="style-scope patent-result"></template>
                        
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/FI123078B/en"><a id="link" href="#" class="style-scope state-modifier">FI123078B</a></state-modifier>
                        <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2012-10-31</span>
                    <span class="td style-scope patent-result">
  Process for producing a modified G-CSF polypeptide
 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="scholar/1765667907273872356"><a id="link" href="#" class="style-scope state-modifier">Mathews et al.</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2001</span>
                    <span class="td style-scope patent-result">Crystal structure of human uroporphyrinogen III synthase</span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="scholar/12376180974097662589"><a id="link" href="#" class="style-scope state-modifier">Schuller et al.</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">1999</span>
                    <span class="td style-scope patent-result">Crystal structure of human heme oxygenase-1</span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="scholar/1775668287547023050"><a id="link" href="#" class="style-scope state-modifier">Sampathkumar et al.</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2005</span>
                    <span class="td style-scope patent-result">Structure of the Mycobacterium tuberculosis flavin dependent thymidylate synthase (MtbThyX) at 2.0  resolution</span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="scholar/7033649903646586429"><a id="link" href="#" class="style-scope state-modifier">Keller et al.</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2014</span>
                    <span class="td style-scope patent-result">, a new subunit of RNA polymerase found in gram-positive bacteria</span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="scholar/16936713803327824159"><a id="link" href="#" class="style-scope state-modifier">Rebelo et al.</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2000</span>
                    <span class="td style-scope patent-result">Gene sequence and crystal structure of the aldehyde oxidoreductase from Desulfovibrio desulfuricans ATCC 27774</span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="scholar/3886937502396946879"><a id="link" href="#" class="style-scope state-modifier">Maresca et al.</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2004</span>
                    <span class="td style-scope patent-result">The bchU gene of Chlorobium tepidum encodes the C-20 methyltransferase in bacteriochlorophyll c biosynthesis</span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="scholar/12494476445777767715"><a id="link" href="#" class="style-scope state-modifier">Prosise et al.</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2003</span>
                    <span class="td style-scope patent-result">Crystal structures of Tritrichomonas foetus inosine monophosphate dehydrogenase in complex with substrate, cofactor and analogs: a structural basis for the random-in ordered-out kinetic mechanism</span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="scholar/18257130416091149217"><a id="link" href="#" class="style-scope state-modifier">Schlatter et al.</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2005</span>
                    <span class="td style-scope patent-result">Crystal engineering yields crystals of cyclophilin D diffracting to 1.7  resolution</span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="scholar/18327126370599117060"><a id="link" href="#" class="style-scope state-modifier">Walker</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">1995</span>
                    <span class="td style-scope patent-result">Determination of the structures of respiratory enzyme complexes from mammalian mitochondria</span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      
                        <template is="dom-if" class="style-scope patent-result"></template>
                        
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/DE60214869T2/en"><a id="link" href="#" class="style-scope state-modifier">DE60214869T2</a></state-modifier>
                        <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2007-04-12</span>
                    <span class="td style-scope patent-result">
  CYTOCHROME P450 2C9 CRYSTALS, STRUCTURES AND ITS USE
 
     </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="scholar/9119808320419361451"><a id="link" href="#" class="style-scope state-modifier">Capitani et al.</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2001</span>
                    <span class="td style-scope patent-result">Structure of the soluble domain of a membrane-anchored thioredoxin-like protein from Bradyrhizobium japonicum reveals unusual properties</span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      
                        <template is="dom-if" class="style-scope patent-result"></template>
                        
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/WO2004078978A1/en"><a id="link" href="#" class="style-scope state-modifier">WO2004078978A1</a></state-modifier>
                        <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2004-09-16</span>
                    <span class="td style-scope patent-result">Compositions and methods for hydroxylating epothilones 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      
                        <template is="dom-if" class="style-scope patent-result"></template>
                        
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/TW200301697A/en"><a id="link" href="#" class="style-scope state-modifier">TW200301697A</a></state-modifier>
                        <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2003-07-16</span>
                    <span class="td style-scope patent-result">Compositions and methods for hydroxylating epothilones 
     </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="scholar/17540999795658846448"><a id="link" href="#" class="style-scope state-modifier">Xu et al.</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2021</span>
                    <span class="td style-scope patent-result">Identification of a novel class of photolyases as possible ancestors of their family</span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="scholar/10216262557399986210"><a id="link" href="#" class="style-scope state-modifier">Jagdev et al.</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2023</span>
                    <span class="td style-scope patent-result">Crystal structure of the N-terminal domain of MtClpC1 in complex with the anti-mycobacterial natural peptide Lassomycin</span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="scholar/10975332985836425871"><a id="link" href="#" class="style-scope state-modifier">Drake et al.</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2016</span>
                    <span class="td style-scope patent-result">1.2  resolution crystal structure of the periplasmic aminotransferase PvdN from Pseudomonas aeruginosa</span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      
                        <template is="dom-if" class="style-scope patent-result"></template>
                        
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/TW200418849A/en"><a id="link" href="#" class="style-scope state-modifier">TW200418849A</a></state-modifier>
                        <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2004-10-01</span>
                    <span class="td style-scope patent-result">Compositions and methods for hydroxylating epothilones 
     </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="scholar/18127315651054525788"><a id="link" href="#" class="style-scope state-modifier">Arai et al.</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2006</span>
                    <span class="td style-scope patent-result">Structure of human ubiquitin-conjugating enzyme E2 G2 (UBE2G2/UBC7)</span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      
                        <template is="dom-if" class="style-scope patent-result"></template>
                        
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/AU6523100A/en"><a id="link" href="#" class="style-scope state-modifier">AU6523100A</a></state-modifier>
                        <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2001-03-05</span>
                    <span class="td style-scope patent-result">Crystallization and structure determination of staphylococcus aureus udp-n-acetylenolpyruvylglucosamine reductase (s. aureus murb) 
     </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="scholar/17607114704826848597"><a id="link" href="#" class="style-scope state-modifier">Arachea et al.</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2010</span>
                    <span class="td style-scope patent-result">Expansion of the aspartate -semialdehyde dehydrogenase family: the first structure of a fungal ortholog</span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="scholar/7092265086593046443"><a id="link" href="#" class="style-scope state-modifier">He et al.</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2000</span>
                    <span class="td style-scope patent-result">Structure of a new neurotoxin from the scorpion Buthus martensii Karsch at 1.76 </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="scholar/9204121381966057643"><a id="link" href="#" class="style-scope state-modifier">Huang et al.</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2005</span>
                    <span class="td style-scope patent-result">Structure of recombinant human cyclophilin J, a novel member of the cyclophilin family</span>
                  </div>
                  <template is="dom-repeat" class="style-scope patent-result"></template>
                </div>
              </div>
            </div>
            <template is="dom-if" class="style-scope patent-result"></template>

            
            <h3 id="relatedApplications" class="scroll-target style-scope patent-result">Priority And Related Applications</h3>

            <template is="dom-if" class="style-scope patent-result"></template>

            <template is="dom-if" class="style-scope patent-result"></template>

            <template is="dom-if" class="style-scope patent-result"></template>

            
            <h3 id="appsClaimingPriority" class="scroll-target style-scope patent-result">
              Applications Claiming Priority (2)
              <a href="#title" class="style-scope patent-result"><iron-icon icon="arrow-drop-up" class="style-scope patent-result x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M7 14l5-5 5 5z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon></a>
            </h3>
            <div class="responsive-table style-scope patent-result">
              <div class="table style-scope patent-result">
                <div class="thead style-scope patent-result">
                  <div class="tr style-scope patent-result">
                    <span class="th style-scope patent-result">Application</span>
                    <span class="th style-scope patent-result">Filing date</span>
                    <span class="th style-scope patent-result">Title</span>
                  </div>
                </div>
                <div class="tbody style-scope patent-result">
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                        <template is="dom-if" class="style-scope patent-result"></template>
                        US34427101P<template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td nowrap style-scope patent-result">2001-12-26</span>
                    <span class="td style-scope patent-result"></span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US6884608B2/en"><a id="link" href="#" class="style-scope state-modifier">US10/321,188</a></state-modifier><template is="dom-if" class="style-scope patent-result"></template>
                        <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td nowrap style-scope patent-result">2002-12-17</span>
                    <span class="td style-scope patent-result">Compositions and methods for hydroxylating epothilones 
       </span>
                  </div>
                  <template is="dom-repeat" class="style-scope patent-result"></template>
                </div>
              </div>
            </div>
            <template is="dom-if" class="style-scope patent-result"></template>

            <template is="dom-if" class="style-scope patent-result"></template>

            <template is="dom-if" class="style-scope patent-result"></template>

            <template is="dom-if" class="style-scope patent-result"></template>

            <div id="notices" class="vertical layout center style-scope patent-result">
              
                <span class="style-scope patent-result">Data provided by IFI CLAIMS Patent Services</span>
              <template is="dom-if" class="style-scope patent-result"></template>
            </div>
          </div>
          <div class="bottombar style-scope patent-result">
            <bottom-bar class="style-scope patent-result"><template is="dom-if" class="style-scope bottom-bar"></template>
</bottom-bar>
          </div>
        </div>
      </div>
      <template is="dom-if" class="style-scope patent-result"></template>
    </div>
  </patent-result>
    <template is="dom-if" restamp="" class="style-scope result-container"></template>
    <template is="dom-if" restamp="" class="style-scope result-container"></template>
    <template is="dom-if" restamp="" class="style-scope result-container"></template>
  </result-container>
        </div>
      </div>
    
        </div>
      </div>
      <search-footer class="style-scope search-ui">
    
    

    <div id="footer" class="vertical layout center style-scope search-footer">
        <span class="links style-scope search-footer">
          <overlay-tooltip positioning-target="#footerAbout" direction="top" class="style-scope search-footer" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      Learn more about data coverage, search syntax and other features.
    </div>
  </overlay-tooltip>
          <a id="footerAbout" data-proto="ABOUT" href="https://support.google.com/faqs/answer/6390996" target="_blank" class="style-scope search-footer">About</a>
          <overlay-tooltip positioning-target="#footerFeedback" direction="top" class="style-scope search-footer" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      Send feedback about technical issues or feature requests for Google Patents.
    </div>
  </overlay-tooltip>
          <a id="footerFeedback" data-proto="SEND_FEEDBACK" href="#" onclick="sendFeedback();" class="style-scope search-footer">Send Feedback</a>
          <a data-proto="PUBLIC_DATASETS" href="https://console.cloud.google.com/marketplace/product/google_patents_public_datasets/google-patents-public-data" target="_blank" class="style-scope search-footer">Public Datasets</a>
          <a data-proto="TERMS" href="https://www.google.com/policies/terms/" target="_blank" class="style-scope search-footer">Terms</a>
          <a data-proto="PRIVACY_POLICY" href="https://www.google.com/privacy/privacy-policy.html" target="blank" class="style-scope search-footer">Privacy Policy</a>
          <a data-proto="SEARCH_HELP" href="https://support.google.com/websearch/" target="_blank" class="style-scope search-footer">Help</a>
        </span>
    </div>
  </search-footer>
    </div>

  </search-ui>
  </search-result>
    <template is="dom-if" restamp="" class="style-scope search-app"></template>
    <template is="dom-if" restamp="" class="style-scope search-app"></template>
    <template is="dom-if" restamp="" class="style-scope search-app"></template>
    <paper-toast id="toast" duration="3000" class="style-scope search-app x-scope paper-toast-0" aria-hidden="true" style="outline: none; display: none;">
    

    <span id="label" class="style-scope paper-toast"></span>
    
      <div id="noticeContent" class="style-scope search-app"></div>
    
  </paper-toast>
  </search-app>
    <script type="text/javascript" src="//www.gstatic.com/feedback/api.js"></script>
    <script type="text/javascript" src="//www.gstatic.com/feedback/js/help/prod/service/lazy.min.js"></script>
    <script type="text/javascript">
      if (window.help && window.help.service) {
        helpApi = window.help.service.Lazy.create(0, {apiKey: 'AIzaSyDTEI_0tLX4varJ7bwK8aT-eOI5qr3BmyI', locale: 'en-US'});
        window.requestedSurveys = new Set();
        window.requestSurvey = function(triggerId) {
          if (window.requestedSurveys.has(triggerId)) {
            return;
          }
          window.requestedSurveys.add(triggerId);
          helpApi.requestSurvey({
            triggerId: triggerId,
            enableTestingMode: false,
            callback: (requestSurveyCallbackParam) => {
              if (!requestSurveyCallbackParam.surveyData) {
                return;
              }
              helpApi.presentSurvey({
                productData: {
                  productVersion: window.version,
                  customData: {
                    "experiments": "",
                  },
                },
                surveyData: requestSurveyCallbackParam.surveyData,
                colorScheme: 1,
                customZIndex: 10000,
              });
            }
          });
        };

        window.requestSurvey('YXTwAsvoW0kedxbuTdH0RArc9VhT');
      }
    </script>
    <script src="/sw/null_loader.js"></script>
  

<iframe id="apiproxyddf475d63cd8173a637a287407211e3c7614fb050.2406055087" name="apiproxyddf475d63cd8173a637a287407211e3c7614fb050.2406055087" style="width: 1px; height: 1px; position: absolute; top: -100px; display: none;" src="https://scone-pa.clients6.google.com/static/proxy.html?usegapi=1&amp;jsh=m%3B%2F_%2Fscs%2Fabc-static%2F_%2Fjs%2Fk%3Dgapi.lb.en.IKZeRvoAYNY.O%2Fam%3DAAAQ%2Fd%3D1%2Frs%3DAHpOoo-SMWTzMRJrTty6iE5dL_aWGYOnuw%2Fm%3D__features__#parent=https%3A%2F%2Fpatents.google.com&amp;rpctoken=717250989" tabindex="-1" aria-hidden="true"></iframe><iron-a11y-announcer>
    
    <div class="style-scope iron-a11y-announcer" aria-live="polite"></div>
  </iron-a11y-announcer></body></html>