<html lang="en"><head><!-- Shady DOM styles for custom-style --><!-- Shady DOM styles for dom-template --><!-- Shady DOM styles for dom-repeat --><!-- Shady DOM styles for array-selector --><!-- Shady DOM styles for dom-if --><!-- Shady DOM styles for state-modifier --><style scope="state-modifier">a[inherit].state-modifier {
  color: inherit;
}</style><!-- Shady DOM styles for browser-ui --><!-- Shady DOM styles for outlined-text-field --><style scope="outlined-text-field">outlined-text-field {
  display: flex;
  font-family: "Google Sans", Roboto, Arial, sans-serif;
  font-weight: 500;
  flex-direction: column;
}

outlined-text-field span.outlined-text-field {
  display: flex;
  flex-direction: row;
  align-items: center;
}

outlined-text-field input.outlined-text-field {
  height: 40px;
  padding: 0 33px 0 16px;
  
  border-color: #b0b3b6;
  border-width: 1px;
  border-style: solid;
  
  border-radius: 8px;
  caret-color: #3c78e0;
  color: #6a6c71;
  font-size: 14px;
  flex-grow: 1;
}

outlined-text-field[orig] input.outlined-text-field {
  border-color: #e5e5e5;
  
  border-radius: 0;
}

outlined-text-field[dense] input.outlined-text-field {
  height: 32px;
}

outlined-text-field[uppercase] input.outlined-text-field {
  text-transform: uppercase;
}

outlined-text-field:not([wizard-focused]):hover input.outlined-text-field {
  border-color: #737375;
  color: #414447;
}

outlined-text-field:not([wizard-focused]):focus-within input.outlined-text-field,outlined-text-field[wizard-focused] input.outlined-text-field {
  border-color: #3574e0;
  caret-color: #80a5f1;
}

outlined-text-field[wizard-focused] input.outlined-text-field {
  border-width: 2px;
}

outlined-text-field input.outlined-text-field::placeholder {
  color: #65686c;
}

outlined-text-field[uppercase] input.outlined-text-field::placeholder {
  text-transform: none;
}</style><!-- Shady DOM styles for mat-keyword-editor --><!-- Shady DOM styles for mat-chips-input --><!-- Shady DOM styles for mat-input-chip --><!-- Shady DOM styles for pat-dialog --><style scope="pat-dialog">pat-dialog {
  background: white;
  position: absolute;
  border-radius: 8px;
  
  box-shadow: 0 24px 38px 3px rgb(0 0 0 / 14%),
              0 9px 46px 8px rgb(0 0 0 / 12%),
              0 11px 15px -7px rgb(0 0 0 / 20%);
  z-index: 1;
  display: flex;
  flex-direction: column;
}

pat-dialog:not([open]) {
  display: none;
}</style><!-- Shady DOM styles for mat-tabs --><!-- Shady DOM styles for filter-chip --><style scope="filter-chip">filter-chip {
  align-items: center;
  background-color: #fff;
  border-color: #dadce0;
  border-radius: 8px;
  border-style: solid;
  border-width: 1px;
  color: #606368;
  cursor: pointer;
  font-family: "Google Sans", Roboto, Arial, sans-serif;
  font-size: 14px;
  font-weight: 500;
  height: 32px;
  letter-spacing: 0.2px;
  padding: 0 16px;
  user-select: none;
  display: inline-flex;
  white-space: nowrap;
}

filter-chip[disabled] {
  background-color: #e8e8e8;
  border-color: #e8e8e8;
  color: #a9abad;
  cursor: auto;
}

filter-chip svg.filter-chip {
  visibility: collapse;
  font-size: 18px;
  width: 0;
  height: 18px;
}

filter-chip[selected] svg.filter-chip {
  visibility: visible;
  margin: 0 8px;
  width: 18px;
}

filter-chip[selected] svg.filter-chip path.filter-chip {
  fill: #4577d1;
}

filter-chip:not([disabled]):hover {
  background-color: #f7f8f8;
  border-color: #d7d9dc;
  color: #454649;
}

filter-chip:not([disabled]):hover:active {
  background-color: #eff0f0;
  border-color: #d2d5d8;
  color: #222326;
}

filter-chip:not([disabled]):focus {
  background-color: #e7e8e8;
  border-color: #232528;
  color: #202124;
}

filter-chip:not([disabled])[selected] {
  background-color: #e9f0fd;
  border-color: #e9f0fd;
  color: #4577d1;
  padding-left: 0;
}

filter-chip:not([disabled]):focus[selected] {
  background-color: #d5e1fb;
  border-color: #d5e1fb;
  color: #2550a0;
}

filter-chip:not([disabled]):hover[selected] {
  box-shadow: rgba(0, 0, 0, 0.2) 0px 3px 1px -2px,
    rgba(0, 0, 0, 0.14) 0px 2px 2px 0px, rgba(0, 0, 0, 0.12) 0px 1px 5px 0px;
}

filter-chip:not([disabled]):hover:active[selected] {
  background-color: #abc8f2;
  border-color: #abc8f2;
  color: #2a54a3;
}</style><!-- Shady DOM styles for figure-callout --><style scope="figure-callout">figure-callout:not([filenames=""]) a.figure-callout {
  cursor: zoom-in;
  text-decoration: underline;
  text-decoration-color: #ddd;
}</style><!-- Shady DOM styles for mat-button --><style scope="mat-button-0">.mat-button-0 {
  ;
  align-items: center;
  border-radius: 4px;
  color: hsl(217 4% 39%); 
  cursor: pointer;
  font-family: "Google Sans", Roboto, Arial, sans-serif;
  font-size: 14px;
  font-weight: 500;
  height: 36px;
  letter-spacing: 0.2px;
  padding: 0 8px;
  user-select: none;
  display: inline-flex;
  white-space: nowrap;
}

.mat-button-0[filled] {
  background-color: hsl(217 73% 54%); 
  color: white;
}

.mat-button-0[filled]:hover {
  background-color: hsl(217 60% 47%);
}

.mat-button-0[filled]:focus {
  background-color: hsl(217 59% 44%);
}

.mat-button-0[filled]:active {
  background-color: hsl(217 59% 45%);
}

.mat-button-0[filled][disabled] {
  background-color: hsl(217 2% 90%); 
  color: hsl(217 1% 65%);
}</style><!-- Shady DOM styles for search-term-wizard --><style scope="search-term-wizard">search-term-wizard:not([open]) {
  display: none;
}

search-term-wizard .header.search-term-wizard {
  border-bottom: 1px solid #e5e5e5;
  padding: 24px;
  padding-bottom: 0;
}

search-term-wizard .title.search-term-wizard {
  color: #3c4043;
  font-family: Google Sans,Roboto,Arial,sans-serif;
  font-size: 22px;
  user-select: none;
  margin: 0 48px;
}

search-term-wizard #panels.search-term-wizard > *.search-term-wizard:not([selected]) {
  display: none;
}</style><!-- Shady DOM styles for outlined-textarea --><style scope="outlined-textarea">outlined-textarea {
  display: flex;
  font-family: "Google Sans", Roboto, Arial, sans-serif;
  font-weight: 500;
  flex-direction: row;
  
  position: relative;
}

outlined-textarea span.outlined-textarea {
  display: flex;
  flex-direction: column;
  flex-grow: 1;
  justify-content: center;
}

outlined-textarea span.right-button-container.outlined-textarea {
  position: absolute;
  right: 0;
  height: 100%;
}

outlined-textarea textarea.outlined-textarea {
  padding: 16px 16px 0 16px;
  border-color: #b0b3b6;
  border-width: 1px;
  border-style: solid;
  border-radius: 8px;
  caret-color: #3c78e0;
  color: #6a6c71;
  font-size: 14px;
  font-family: "Google Sans", Roboto, Arial, sans-serif;
  font-weight: 500;
  resize: none;
}

outlined-textarea[dense] textarea.outlined-textarea {
  padding: 8px;
}

outlined-textarea[has-icon] textarea.outlined-textarea {
  padding-right: 38px;
}

outlined-textarea[orig] textarea.outlined-textarea {
  border-color: #e5e5e5;
  
  border-radius: 0;
}

outlined-textarea:not([rows]) textarea.outlined-textarea {
  height: 40px;
}

outlined-textarea textarea.resizable.outlined-textarea {
  resize: vertical;
}

outlined-textarea textarea.outlined-textarea:focus,outlined-textarea[wizard-focused] textarea.outlined-textarea,outlined-textarea textarea.outlined-textarea:focus:hover {
  border-color: #3574e0 !important;
  
  outline: none !important;
}

outlined-textarea textarea.outlined-textarea:hover {
  border-color: #202124;
  color: #414447;
}

outlined-textarea.hasValue textarea.outlined-textarea {
  border-color: #888d91;
  color: #45474a;
}

outlined-textarea #helperText.outlined-textarea {
  color: #6a6c71;
  font-size: 12px;
  letter-spacing: 0.3px;
  margin-left: 17px;
  margin-top: 6px;
  user-select: none;
}

outlined-textarea:not([helper-text]) #helperText.outlined-textarea {
  display: none;
}

outlined-textarea[error] textarea.outlined-textarea {
  border-color: #cb4b3d;
  caret-color: #ca493a;
  color: #3d4043;
}

outlined-textarea textarea.outlined-textarea::placeholder {
  font-size: 14px;
  font-weight: 400;
}

outlined-textarea:not([error]):not([disabled]) textarea.outlined-textarea::placeholder {
  color: #6a6c71;
}

outlined-textarea[error] textarea.outlined-textarea::placeholder {
  color: #c84132;
}

outlined-textarea[error] #helperText.outlined-textarea {
  color: #cd5648;
}

outlined-textarea[error] textarea.outlined-textarea:focus {
  border-color: #c84031;
  border-width: 2px;
  outline: none !important;
}

outlined-textarea[error]:not([disabled]) textarea.outlined-textarea:hover {
  border-color: #97231b;
}

outlined-textarea[error]:not([disabled]) textarea.outlined-textarea:hover::placeholder,outlined-textarea[error]:not([disabled]) textarea.outlined-textarea:hover + #helperText.outlined-textarea {
  color: #a94945;
}

outlined-textarea[disabled] textarea.outlined-textarea {
  border-color: #e9e9e9;
  color: #b5b6b8;
}

outlined-textarea[disabled] #helperText.outlined-textarea {
  color: #c4c6c7;
}

outlined-textarea[error][disabled] {
  opacity: 50%;
}

outlined-textarea[error][disabled] textarea.outlined-textarea {
  border-color: #cb4b3d;
  caret-color: #ca493a;
}

outlined-textarea[error][disabled] #helperText.outlined-textarea,outlined-textarea[error][disabled] textarea.outlined-textarea {
  color: #a94945;
}

outlined-textarea[error][disabled] textarea.outlined-textarea::placeholder {
  color: #c84132;
}</style><!-- Shady DOM styles for chemistry-search-term-wizard --><style scope="chemistry-search-term-wizard">chemistry-search-term-wizard {
  display: flex;
  flex-direction: column;
  padding: 24px;
  padding-top: 0;
}

chemistry-search-term-wizard > span.chemistry-search-term-wizard {
  display: flex;
  flex-direction: row;
  align-items: center;
  flex-wrap: wrap;
  margin-top: 16px;
}

chemistry-search-term-wizard .suggestions.chemistry-search-term-wizard {
  margin-top: 4px;
}

chemistry-search-term-wizard > span.chemistry-search-term-wizard svg.chemistry-search-term-wizard {
  margin-right: 24px;
}

chemistry-search-term-wizard path.chemistry-search-term-wizard {
  fill: #5f6368;
}

chemistry-search-term-wizard .help.chemistry-search-term-wizard {
  margin-left: 12px;
  margin-right: 0;
  cursor: pointer;
}

chemistry-search-term-wizard outlined-text-field.chemistry-search-term-wizard {
  flex-grow: 1;
}

chemistry-search-term-wizard mat-button.chemistry-search-term-wizard {
  margin-left: 48px;
}

chemistry-search-term-wizard filter-chip.chemistry-search-term-wizard {
  margin-right: 4px;
}

chemistry-search-term-wizard outlined-textarea.chemistry-search-term-wizard {
  flex-grow: 1;
}

chemistry-search-term-wizard outlined-text-field[batchMode].chemistry-search-term-wizard {
  display: none;
}

chemistry-search-term-wizard outlined-textarea.chemistry-search-term-wizard:not([batchMode]) {
  display: none;
}</style><!-- Shady DOM styles for text-search-term-wizard --><style scope="text-search-term-wizard">text-search-term-wizard {
  display: flex;
  flex-direction: column;
  padding: 24px;
  padding-top: 0;
}

text-search-term-wizard > span.text-search-term-wizard {
  display: flex;
  flex-direction: row;
  align-items: center;
  flex-wrap: wrap;
  margin-top: 16px;
}

text-search-term-wizard > span.text-search-term-wizard svg.text-search-term-wizard {
  margin-right: 24px;
}

text-search-term-wizard path.text-search-term-wizard {
  fill: #5f6368;
}

text-search-term-wizard outlined-text-field.text-search-term-wizard {
  flex-grow: 1;
}

text-search-term-wizard mat-button.text-search-term-wizard {
  margin-left: 48px;
}

text-search-term-wizard filter-chip.text-search-term-wizard {
  margin-right: 4px;
}</style><!-- Shady DOM styles for classification-wizard --><style scope="classification-wizard">classification-wizard {
  display: flex;
  flex-direction: column;
  padding: 24px;
  padding-top: 0;
  max-width: 674px;
}

classification-wizard > span.classification-wizard {
  display: flex;
  flex-direction: row;
  align-items: center;
  flex-wrap: wrap;
  margin-top: 16px;
}

classification-wizard .suggestions.classification-wizard {
  margin-top: 4px;
}

classification-wizard > span.classification-wizard svg.classification-wizard {
  margin-right: 24px;
}

classification-wizard path.classification-wizard {
  fill: #5f6368;
}

classification-wizard outlined-text-field.classification-wizard {
  flex-grow: 1;
}

classification-wizard mat-button.classification-wizard {
  margin-left: 48px;
}

classification-wizard filter-chip.classification-wizard {
  margin-right: 4px;
}

classification-wizard .no-matches.classification-wizard {
  margin-left: 64px;
}

classification-wizard .table-container.classification-wizard {
  max-height: 300px;
  overflow: scroll;
  margin-top: 8px;
}

classification-wizard .table-container.classification-wizard::-webkit-scrollbar {
  -webkit-appearance: none;
  width: 10px;
}

classification-wizard .table-container.classification-wizard::-webkit-scrollbar-thumb {
  border-radius: 5px;
  background-color: #5f636880;
  -webkit-box-shadow: 0 0 1px rgba(255,255,255,.5);
}

classification-wizard table.classification-wizard {
  margin-left: 51px;
  max-width: 575px;
}

classification-wizard tr.classification-wizard {
  margin-top: 4px;
}

classification-wizard tr.classification-wizard suggestive-chip.classification-wizard {
  margin-right: 8px;
}

classification-wizard #spinner.classification-wizard {
  justify-content: space-around;
  margin-left: 48px;
  transform: scale(0.5);
}</style><!-- Shady DOM styles for suggestive-chip --><!-- Shady DOM styles for mat-checkbox --><!-- Shady DOM styles for measure-search-term-wizard --><style scope="measure-search-term-wizard">measure-search-term-wizard {
  display: flex;
  flex-direction: column;
  padding: 24px;
  padding-top: 0;
  max-width: 674px;
}

measure-search-term-wizard > span.measure-search-term-wizard {
  display: flex;
  flex-direction: row;
  align-items: center;
  flex-wrap: wrap;
  margin-top: 16px;
}

measure-search-term-wizard .suggestions.measure-search-term-wizard {
  margin-top: 4px;
}

measure-search-term-wizard > span.measure-search-term-wizard svg.measure-search-term-wizard {
  margin-right: 24px;
}

measure-search-term-wizard path.measure-search-term-wizard {
  fill: #5f6368;
}

measure-search-term-wizard outlined-text-field.measure-search-term-wizard {
  flex-grow: 1;
}

measure-search-term-wizard mat-button.measure-search-term-wizard {
  margin-left: 48px;
}

measure-search-term-wizard filter-chip.measure-search-term-wizard {
  margin-right: 4px;
}</style><!-- Shady DOM styles for numerical-search-term-wizard --><style scope="numerical-search-term-wizard">numerical-search-term-wizard {
  display: flex;
  flex-direction: column;
  padding: 24px;
  padding-top: 0;
  max-width: 674px;
}

numerical-search-term-wizard > span.numerical-search-term-wizard {
  display: flex;
  flex-direction: row;
  align-items: center;
  flex-wrap: wrap;
  margin-top: 16px;
}

numerical-search-term-wizard .suggestions.numerical-search-term-wizard {
  margin-top: 4px;
}

numerical-search-term-wizard > span.numerical-search-term-wizard svg.numerical-search-term-wizard {
  margin-right: 24px;
}

numerical-search-term-wizard path.numerical-search-term-wizard {
  fill: #5f6368;
}

numerical-search-term-wizard outlined-text-field.numerical-search-term-wizard,numerical-search-term-wizard outlined-textarea.numerical-search-term-wizard {
  flex-grow: 1;
}

numerical-search-term-wizard mat-button.numerical-search-term-wizard {
  margin-left: 48px;
}

numerical-search-term-wizard filter-chip.numerical-search-term-wizard {
  margin-right: 4px;
}

numerical-search-term-wizard #spinner.numerical-search-term-wizard {
  justify-content: space-around;
  margin-left: 48px;
  transform: scale(0.5);
}

numerical-search-term-wizard #suggestions.numerical-search-term-wizard {
  margin-left: 58px;
  display: flex;
  flex-direction: column;
  align-items: flex-start;
}

numerical-search-term-wizard .suggestion.numerical-search-term-wizard {
  display: flex;
  flex-direction: row;
  align-items: center;
}

numerical-search-term-wizard .suggestion.numerical-search-term-wizard mat-checkbox.numerical-search-term-wizard {
  margin-right: 8px;
}</style><!-- Shady DOM styles for text-button --><style scope="text-button">text-button {
  align-items: center;
  border-radius: 4px;
  cursor: pointer;
  font-family: "Google Sans", Roboto, Arial, sans-serif;
  font-size: 14px;
  font-weight: 500;
  height: 36px;
  letter-spacing: .25px;
  margin: 6px 0;
  padding: 0 8px;
  user-select: none;
  display: inline-flex;
  white-space: nowrap;
  line-height: 36px;
  text-decoration: none;
  text-transform: none;
  color: #5f6368;
}

text-button:hover {
  background-color: #f8f9fa;
}

text-button:active {
  background-color: #d5e2fa;
}

text-button[selected] {
  background-color: #e8f0fe;
  color: #1967d2;
}

text-button[selected]:active {
  background-color: #d2e3fc;
}</style><!-- Shady DOM styles for mat-spinner --><!-- Shady DOM styles for requery-tile --><style scope="requery-tile">requery-tile {
  display: flex;
  justify-content: space-between;
  align-items: center;
  box-sizing: border-box;
  min-height: 40px;
  padding: 8px 12px 8px 8px;
  gap: 8px;

  border-radius: 16px;
  background: #f7f8f9;
  color: #1f1f1f;
}

requery-tile:hover {
  background: #d7d8d9;
}

requery-tile > [slot="image"] {
  flex-shrink: 0;

  border-radius: 8px;

  background: #fff;
}

.label.requery-tile {
  flex-grow: 999;

  font: 16px/20px "Google Sans", Roboto, Arial, sans-serif;
  font-weight: 500;
}

.search-icon.requery-tile {
  flex-shrink: 0;

  width: 24px;
  height: 24px;
  background-image: url('https://fonts.gstatic.com/s/i/short-term/release/googlesymbols/search/default/24px.svg');
}</style><!-- Shady DOM styles for s-more-inline --><style scope="s-more-inline">s-more-inline {
  display: block;
  margin-top: 24px;
  padding: 16px;
  padding-top: 0;
  position: relative;

  line-height: 18px;
  text-overflow: ellipsis;
  white-space: nowrap;
  cursor: pointer;
}

hr.s-more-inline {
  background-color: #dadce0;
  margin-top: 18px;
  position: absolute;
  border: 0;
  height: 1px;
  left: 0;
  width: 100%;
}

.button.s-more-inline {
  display: flex;
  align-items: center;
  justify-content: center;

  position: relative; 
  margin: 0 auto;

  font: 400 14px Roboto, arial, sans-serif;
  height: 36px;
  width: 300px;
  border-radius: 18px;
  background-color: #f1f3f4;
}

.button.s-more-inline:hover {
  background-color: #d8d7dc;
}

.button.s-more-inline > [slot] {
  padding-right: 8px;
}

.expand.s-more-inline {
  display: inline-block;
  width: 20px;
  height: 20px;
}

.expand.less.s-more-inline {
  background-image: url('https://fonts.gstatic.com/s/i/short-term/release/googlesymbols/expand_less/default/20px.svg');
}

.expand.more.s-more-inline {
  background-image: url('https://fonts.gstatic.com/s/i/short-term/release/googlesymbols/expand_more/default/20px.svg');
}</style><!-- Shady DOM styles for workspace-ui-search --><!-- Shady DOM styles for keyword-wizard --><style scope="keyword-wizard">keyword-wizard {
  display: inline-block;
}

iron-icon.keyword-wizard {
  height: 20px;
    width: 20px;
    color: #999;
    cursor: pointer;
}

iron-icon[active].keyword-wizard {
  color: #4285f4 !important;
}

iron-icon.keyword-wizard:hover {
  color: #ccc;
}

#wizardIcon.keyword-wizard {
  padding-left: 1px;
    height: 16px;
    width: 16px;
    margin: 4px 1px 1px 1px;
}

#wizard.keyword-wizard {
  padding: 0;
    left: 0;
    top: 37px;
    position: absolute;
    background-color: white;
    z-index: 2;
    overflow: hidden;
}

.wizard-header.keyword-wizard {
  display: block;
    width: 100%;
    color: #737373;
    text-transform: uppercase;
    font-size: 13px;
    margin: 16px 0 6px 12px;
}

.filter-chips.keyword-wizard {
  margin-left: 20px;
}

.chip.keyword-wizard {
  background-color: #ebebeb;
    color: #6a6a6a;
    display: inline-block;
    font-size: 13px;
    font-family: inherit;
    border-radius: 12px;
    padding: 6px 12px;
}

.chip.keyword-wizard:hover {
  cursor: pointer;
}

.chip[selected].keyword-wizard {
  background-color: #4285f4;
    color: white;
}

.input-label.keyword-wizard {
  display: inline-block;
    font-size: 13px;
    font-family: inherit;
}

.keyword-input.keyword-wizard {
  display: inline-block;
    border-top: 0;
    border-left: 0;
    border-right: 0;
    border-bottom: 1px solid #dfe1e5;
    padding: 3px;
}

.keyword-input.keyword-wizard::placeholder {
  font-style: italic;
}

.keyword-input.keyword-wizard:hover,.keyword-input.keyword-wizard:focus {
  border-color: #4284f4;
    border-width: 1px;
    outline: none;
}

.cpc-field.keyword-wizard {
  display: inline-block;
    margin: 6px 0 0 24px;
}

#checkboxCPC.keyword-wizard:hover {
  cursor: pointer;
}

.checkbox-label.keyword-wizard {
  font-size: 13px;
    font-style: italic;
    color: #737373;
}

.chemistry-field.keyword-wizard {
  display: block;
}

#inputChem.input-batch.keyword-wizard {
  height: 150px;
    border: 1px solid #dfe1e5;
    padding: 6px;
}

#inputChem.keyword-wizard {
  resize: none;
    height: 15px;
    vertical-align: text-top;
    font-size: 13px;
    font-family: inherit;
    padding: 2px;
}

.chem-type-label.keyword-wizard {
  display: inline;
    font-size: 13px;
    margin-left: 12px;
    margin-right: 4px;
}

#chemType.keyword-wizard {
  border-top: 0;
    border-left: 0;
    border-right: 0;
    border-bottom: 1px solid #dfe1e5;
    margin: 6px;
    padding: 2px;
    outline: none;
    font-family: inherit;
    font-size: 13px;
}

.import.keyword-wizard {
  font-size: 13px;
    font-family: inherit;
}

.import-text.keyword-wizard {
  vertical-align: middle;
    display: inline-block;
}

.import-button.keyword-wizard {
  margin: 6px 0 0 24px;
}

.import-button.keyword-wizard button.keyword-wizard {
  border: 0;
    border-radius: 4px;
    padding: 6px 12px;
    font-family: inherit;
    background-color: #4284f4;
    color: white;
}

.import-button.keyword-wizard button.keyword-wizard:hover {
  cursor: pointer;
    background-color: #4284f4;
    color: white;
}

.wizard-buttons.keyword-wizard {
  float: right;
    padding: 12px;
}

.wizard-buttons.keyword-wizard button.keyword-wizard {
  border: 0;
    border-radius: 4px;
    padding: 6px 12px;
    margin: 2px;
    font-family: inherit;
    background-color: #4284f4;
    color: white;
}

.wizard-buttons.keyword-wizard button.keyword-wizard:hover {
  cursor: pointer;
}

@media screen and (max-width: 700px) {
#wizard.keyword-wizard {
  width: 275px;
}

.chip.keyword-wizard {
  margin: 4px 2px;
}

.keyword-input.keyword-wizard {
  width: 82%;
      margin: 4px 12px 12px 24px;
}

.search-within.keyword-wizard .input-label.keyword-wizard {
  display: none;
}

.cpc-field.keyword-wizard .keyword-input.keyword-wizard {
  width: 225px;
      margin: 12px 0 6px 0;
}

.find-patents.keyword-wizard .input-label.keyword-wizard {
  display: none;
}

.search-by.keyword-wizard .input-label.keyword-wizard {
  width: 55px;
      margin-left: 24px;
}

.search-by.keyword-wizard .keyword-input.keyword-wizard {
  margin-top: 6px;
}

#inputChem.keyword-wizard {
  width: 82%;
      padding: 2px;
      margin: 8px 12px 12px 24px;
}

#inputChem.input-batch.keyword-wizard {
  width: 220px;
      padding: 4px;
}

#chemType.keyword-wizard {
  width: 150px;
}

.import-text.keyword-wizard {
  margin: 6px 12px 6px 24px;
      width: 220px;
}

iron-icon.help-header.keyword-wizard {
  display: none;
}

}

@media screen and (min-width: 700px) {
#wizard.keyword-wizard {
  width: 540px;
}

.chip.keyword-wizard {
  margin: 0 2px;
}

.cpc-field.keyword-wizard .keyword-input.keyword-wizard {
  width: 240px;
      margin: 12px 16px 6px 6px;
}

.cpc-inputs.keyword-wizard {
  display: inline;
}

.find-patents.keyword-wizard .keyword-input.keyword-wizard {
  width: 53%;
      margin: 4px 10px;
}

.cpc-field.keyword-wizard .keyword-input.keyword-wizard::placeholder,#inputAnd.keyword-wizard::placeholder,#inputExact.keyword-wizard::placeholder,#inputOr.keyword-wizard::placeholder,#inputMinus.keyword-wizard::placeholder {
  color: white;
}

.find-patents.keyword-wizard .input-label.keyword-wizard {
  width: 35%;
      margin-left: 24px;
}

.search-by.keyword-wizard .keyword-input.keyword-wizard {
  width: 42%;
      margin: 6px 4px;
}

.search-by.keyword-wizard .input-label.keyword-wizard {
  width: 55px;
      margin-left: 24px;
}

#inputChem.keyword-wizard {
  width: 290px;
      margin: 2px 12px 2px 6px;
}

#inputChem.input-batch.keyword-wizard {
  width: 280px;
      padding: 6px;
}

.import-text.keyword-wizard {
  margin: 6px 12px;
      width: 415px;
}

#chemType.keyword-wizard {
  width: 105px;
}

iron-icon.keyword-wizard:hover.help-header {
  cursor: default;
}

iron-icon.help-header.keyword-wizard {
  height: 16px;
      padding: 0 2px 2px 2px;
      color: #a3a3a3;
}

.help.keyword-wizard p.keyword-wizard {
  margin: 0 0 8px 0
}

.help.keyword-wizard .q.keyword-wizard {
  font-family: monospace;
      color: #0b8043;
      white-space: nowrap;
}

.help.keyword-wizard {
  z-index: 4;
      position: absolute;
      right: 10px;
      padding: 12px;
      border-radius: 2px;
      border: 1px solid #e0e0e0;
      box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.2), 0 6px 10px 0 rgba(0, 0, 0, 0.3);
      background: white;
      font-size: 13px;
      line-height: 20px;
      width: 450px;
      max-height: 400px;
      overflow: auto;
}

#withinHelp.keyword-wizard {
  top: 75px;
}

#chemHelp.keyword-wizard {
  top: 0;
}

#searchBooleanHelp.keyword-wizard {
  top: 140px;
}

}</style><!-- Shady DOM styles for bottom-bar --><style scope="bottom-bar">.fixed.bottom-bar {
  display: block;
  position: fixed;
  height: 46px;
  bottom: 0;
}

.fixed[medium].bottom-bar {
  height: 280px;
}

.fixed[large].bottom-bar {
  height: 480px;
}

.shadow.bottom-bar {
  background: white;
  padding: 16px;
  border-radius: 8px 8px 0px 0px;
  box-shadow: 0px 8px 10px 1px rgba(0,0,0,.14), 0px 3px 14px 2px rgba(0,0,0,.12), 0px 5px 5px -3px rgba(0,0,0,.2);
}

.oneline.bottom-bar {
  white-space: nowrap;
  overflow: hidden;
  text-overflow: ellipsis;
  cursor: pointer;
}

.button.bottom-bar {
  cursor: pointer;
}

.buttonAlign.bottom-bar {
  margin-top: -4px;
}

.header.bottom-bar {
  height: 30px;
  font-family: 'Google Sans';
  font-size: 18px;
  padding-bottom: 8px;
}

.list.bottom-bar {
  overflow-y: auto;
  line-height: 20px;
}

.definition.bottom-bar {
  padding: 0px 0px 4px 0px;
  cursor: pointer;
}

[medium].bottom-bar .content.bottom-bar {
  height: 230px;
}

[large].bottom-bar .content.bottom-bar {
  height: 430px;
}

[medium].bottom-bar .content.bottom-bar .list.bottom-bar {
  height: 200px;
}

[large].bottom-bar .content.bottom-bar .list.bottom-bar {
  height: 400px;
}

[hidden].bottom-bar {
  display: none;
}

.label.bottom-bar {
  font-family: 'Google Sans';
  font-size: 12px;
  line-height: 16px;
  font-weight: 400;
  padding: 6px 0px 4px 0px;
  color: #1f1f1f;
}

[pad].bottom-bar {
  margin-left: 16px;
}

.layout.vertical.bottom-bar {
  display: flex;
  flex-direction: column;
}

.layout.horizontal.bottom-bar {
  display: flex;
  flex-direction: row;
}

.flex.bottom-bar {
  flex: 1;
  flex-basis: 0.000000001px;
}

.layout.start.bottom-bar {
  align-items: flex-start;
}

.layout.center.bottom-bar {
  align-items: center;
}

.layout.end.bottom-bar {
  align-items: flex-end;
}

.flex-2.bottom-bar {
  flex: 2;
}

.flex-3.bottom-bar {
  flex: 3;
}

.flex-4.bottom-bar {
  flex: 4;
}

.flex-5.bottom-bar {
  flex: 5;
}

.flex-6.bottom-bar {
  flex: 6;
}

.flex-7.bottom-bar {
  flex: 7;
}

.flex-8.bottom-bar {
  flex: 8;
}

.flex-9.bottom-bar {
  flex: 9;
}

.flex-10.bottom-bar {
  flex: 10;
}

.flex-11.bottom-bar {
  flex: 11;
}

.flex-12.bottom-bar {
  flex: 12;
}</style><!-- Shady DOM styles for landscape-link --><!-- Shady DOM styles for landscape-summary-box --><!-- Shady DOM styles for search-state --><!-- Shady DOM styles for iron-overlay-backdrop --><!-- Shady DOM styles for opaque-animation --><!-- Shady DOM styles for iron-dropdown --><style scope="iron-dropdown">iron-dropdown {
  position: fixed;
}

#contentWrapper.iron-dropdown  > * {
  overflow: auto;
}

#contentWrapper.animating.iron-dropdown  > * {
  overflow: hidden;
}</style><!-- Shady DOM styles for iron-meta --><!-- Shady DOM styles for iron-icon --><style scope="iron-icon-0">.iron-icon-0 {
  display: -ms-inline-flexbox;
        display: -webkit-inline-flex;
        display: inline-flex;-ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;-ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
        position: relative;

        vertical-align: middle;

        fill: currentcolor;
        stroke: none;

        width: 24px;
        height: 24px;;
}

.iron-icon-0[hidden] {
  display: none;
}</style><!-- Shady DOM styles for iron-iconset-svg --><!-- Shady DOM styles for iron-list --><!-- Shady DOM styles for paper-icon-item --><!-- Shady DOM styles for paper-item-body --><!-- Shady DOM styles for paper-item --><!-- Shady DOM styles for dropdown-menu --><style scope="dropdown-menu">dropdown-menu {
  display: inline-block;
        line-height: 24px;
}

.label.dropdown-menu {
  cursor: pointer;
        color: #666;
        align-items: center;
        margin-right: 8px;
        display: inline-block;
        padding: 2px 0;
}

#selected.dropdown-menu {
  color: #212121;
}

.label[open].dropdown-menu {
  border-color: #e0e0e0;
}

iron-dropdown.dropdown-menu {
  margin-top: 26px;
}

iron-icon.dropdown-menu {
  height: 15px;
        width: 15px;
}

#menu.dropdown-menu {
  max-height: 300px;
        padding: 8px 0;
        background: white;
        display: block;
        border-radius: 2px;
        box-shadow: 0 0 2px rgba(0,0,0,0.12), 0 2px 2px rgba(0,0,0,0.24);
}

.item.dropdown-menu {
  cursor: pointer;
        color: rgba(0,0,0,.87);
        font-size: 13px;
        line-height: 18px;
        padding: 6px 18px;
}

.item.dropdown-menu:hover {
  background-color: #e0e0e0;
}

.item[selected].dropdown-menu {
  background-color: #e0e0e0;
}

[multi].dropdown-menu .item.dropdown-menu {
  padding: 6px 18px 6px 34px;
}

[multi].dropdown-menu .item[selected].dropdown-menu {
  background-image: url('https://www.gstatic.com/images/icons/material/system/2x/done_grey600_24dp.png');
        background-position: 8px center;
        background-size: 16px;
        background-repeat: no-repeat;
}</style><!-- Shady DOM styles for chips-input --><!-- Shady DOM styles for input-chip --><!-- Shady DOM styles for keyword-editor --><!-- Shady DOM styles for paper-ripple --><!-- Shady DOM styles for paper-button --><!-- Shady DOM styles for date-editor --><!-- Shady DOM styles for metadata-editor --><!-- Shady DOM styles for restrict-editor --><!-- Shady DOM styles for overlay-tooltip --><style scope="overlay-tooltip">overlay-tooltip {
  display: none;
        color: #fff;
        background-color: #000;
        padding: 10px 12px;
        border-radius: 3px;
        font-size: 13px;
        font-weight: normal;
        line-height: normal;
        position: fixed;
        z-index: 100;
}

#arrow.overlay-tooltip {
  width: 0;
        height: 0;
        border-style: solid;
        border-width: 0 6px 8px 6px;
        border-color: transparent transparent #000 transparent;
        position: absolute;
        top: -8px;
        left: calc(50% - 6px);
}</style><!-- Shady DOM styles for query-renderer --><!-- Shady DOM styles for result-nav --><style scope="result-nav">result-nav {
  display: block;
        white-space: nowrap;
        padding: 0 30px;
        line-height: 30px;
        height: 30px;
        margin-top: 17px;
        font-weight: 500;
        font-size: 15px;
}

state-modifier.result-nav {
  color: #333;
}

state-modifier[disabled].result-nav {
  pointer-events: none;
        color: #ccc;
}</style><!-- Shady DOM styles for raw-html --><!-- Shady DOM styles for paper-material --><style scope="paper-material-0">.paper-material-0 {
  display: block;
        position: relative;
}

.paper-material-0[elevation="1"] {
  box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.14),
                    0 1px 5px 0 rgba(0, 0, 0, 0.12),
                    0 3px 1px -2px rgba(0, 0, 0, 0.2);
}

.paper-material-0[elevation="2"] {
  box-shadow: 0 4px 5px 0 rgba(0, 0, 0, 0.14),
                    0 1px 10px 0 rgba(0, 0, 0, 0.12),
                    0 2px 4px -1px rgba(0, 0, 0, 0.4);
}

.paper-material-0[elevation="3"] {
  box-shadow: 0 6px 10px 0 rgba(0, 0, 0, 0.14),
                    0 1px 18px 0 rgba(0, 0, 0, 0.12),
                    0 3px 5px -1px rgba(0, 0, 0, 0.4);
}

.paper-material-0[elevation="4"] {
  box-shadow: 0 8px 10px 1px rgba(0, 0, 0, 0.14),
                    0 3px 14px 2px rgba(0, 0, 0, 0.12),
                    0 5px 5px -3px rgba(0, 0, 0, 0.4);
}

.paper-material-0[elevation="5"] {
  box-shadow: 0 16px 24px 2px rgba(0, 0, 0, 0.14),
                    0  6px 30px 5px rgba(0, 0, 0, 0.12),
                    0  8px 10px -5px rgba(0, 0, 0, 0.4);
}

.paper-material-0[animated] {
  transition: box-shadow 0.28s cubic-bezier(0.4, 0, 0.2, 1);
}

.paper-material-0 {
  ;
}</style><!-- Shady DOM styles for search-box --><style scope="search-box">.layout.horizontal.search-box,.layout.vertical.search-box {
  display: -ms-flexbox;
        display: -webkit-flex;
        display: flex;
}

.layout.inline.search-box {
  display: -ms-inline-flexbox;
        display: -webkit-inline-flex;
        display: inline-flex;
}

.layout.horizontal.search-box {
  -ms-flex-direction: row;
        -webkit-flex-direction: row;
        flex-direction: row;
}

.layout.vertical.search-box {
  -ms-flex-direction: column;
        -webkit-flex-direction: column;
        flex-direction: column;
}

.layout.wrap.search-box {
  -ms-flex-wrap: wrap;
        -webkit-flex-wrap: wrap;
        flex-wrap: wrap;
}

.layout.no-wrap.search-box {
  -ms-flex-wrap: nowrap;
        -webkit-flex-wrap: nowrap;
        flex-wrap: nowrap;
}

.layout.center.search-box,.layout.center-center.search-box {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.layout.center-justified.search-box,.layout.center-center.search-box {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.flex.search-box {
  -ms-flex: 1 1 0.000000001px;
        -webkit-flex: 1;
        flex: 1;
        -webkit-flex-basis: 0.000000001px;
        flex-basis: 0.000000001px;
}

.flex-auto.search-box {
  -ms-flex: 1 1 auto;
        -webkit-flex: 1 1 auto;
        flex: 1 1 auto;
}

.flex-none.search-box {
  -ms-flex: none;
        -webkit-flex: none;
        flex: none;
}

.inputbox.search-box {
  border: 1px solid #ccc;
        padding: 0 12px;
        background: white;
}

.inputbox[focused].search-box {
  border: 1px solid #4285f4 !important;
}

.inputbox.search-box:hover {
  border: 1px solid #999;
}

#searchInput.search-box {
  font-family: inherit;
        font-size: 16px;
        padding: 8px 0;
        outline: none;
        border-right: none;
        width: 100%;
        margin-right: 0;
        border: 0;
}

#searchButton.search-box {
  cursor: pointer;
        background-color: #4285f4;
        border: none;
        color: #fff;
        vertical-align: top;
        width: 70px;
        height: 37px;
        outline: none;
}

.suggest-bg.search-box {
  display: inline-block;
        min-width: 0;
        white-space: nowrap;
}

#suggestMenu.search-box {
  position: absolute;
        margin-top: 38px;
        width: 100%;
        background: white;
        z-index: 1;
}

#suggestMenu.search-box .row.search-box {
  font-size: 13px;
        line-height: 28px;
        white-space: nowrap;
        overflow: hidden;
        text-overflow: ellipsis;
        padding: 4px 2px;
        cursor: pointer;
}

.row[selected].search-box {
  background: rgba(0, 0, 0, 0.05);
}

.suggest-icon.search-box {
  height: 20px;
        width: 20px;
        padding: 0 12px 0 6px;
        transition: transform .15s linear;
        transform: scale(1);
}

.suggest-icon[hide].search-box {
  transform: scale(0);
}

#corpora.search-box {
  padding: 6px 0;
        user-select: none;
}

#corpora.search-box iron-icon.search-box {
  height: 20px;
        width: 20px;
        color: #ccc;
        cursor: pointer;
}

#corpora.search-box iron-icon[active].search-box {
  color: #4285f4 !important;
}

#corpora.search-box iron-icon.search-box:hover {
  color: #999;
}</style><!-- Shady DOM styles for iron-media-query --><!-- Shady DOM styles for paper-header-panel --><!-- Shady DOM styles for iron-a11y-announcer --><style scope="iron-a11y-announcer">iron-a11y-announcer {
  display: inline-block;
        position: fixed;
        clip: rect(0px,0px,0px,0px);
}</style><!-- Shady DOM styles for iron-autogrow-textarea --><!-- Shady DOM styles for iron-input --><!-- Shady DOM styles for paper-input-char-counter --><!-- Shady DOM styles for paper-input-container --><!-- Shady DOM styles for paper-input-error --><!-- Shady DOM styles for paper-input --><!-- Shady DOM styles for paper-textarea --><!-- Shady DOM styles for paper-toast --><style scope="paper-toast-0">.paper-toast-0 {
  display: block;
        position: fixed;
        background-color: #323232;
        color: #f1f1f1;
        min-height: 48px;
        min-width: 288px;
        padding: 16px 24px;
        box-sizing: border-box;
        box-shadow: 0 2px 5px 0 rgba(0, 0, 0, 0.26);
        border-radius: 2px;
        margin: 12px;
        font-size: 14px;
        cursor: default;
        -webkit-transition: -webkit-transform 0.3s, opacity 0.3s;
        transition: transform 0.3s, opacity 0.3s;
        opacity: 0;
        -webkit-transform: translateY(100px);
        transform: translateY(100px);font-family: 'Roboto', 'Noto', sans-serif;
        -webkit-font-smoothing: antialiased;
}

.paper-toast-0.capsule {
  border-radius: 24px;
}

.paper-toast-0.fit-bottom {
  width: 100%;
        min-width: 0;
        border-radius: 0;
        margin: 0;
}

.paper-toast-0.paper-toast-open {
  opacity: 1;
        -webkit-transform: translateY(0px);
        transform: translateY(0px);
}</style><!-- Shady DOM styles for paper-toolbar --><!-- Shady DOM styles for search-footer --><style scope="search-footer">#footer.search-footer {
  font-size: 13px;
        margin-top: 10px;
        background: #fafafa;
        padding: 20px 10px;
        align-items: center;
        border-top: 1px solid #ebebeb;
}

#footer.search-footer .links.search-footer > *.search-footer {
  display: inline-block;
        margin: 0 12px;
}

.layout.horizontal.search-footer,.layout.vertical.search-footer {
  display: -ms-flexbox;
        display: -webkit-flex;
        display: flex;
}

.layout.inline.search-footer {
  display: -ms-inline-flexbox;
        display: -webkit-inline-flex;
        display: inline-flex;
}

.layout.horizontal.search-footer {
  -ms-flex-direction: row;
        -webkit-flex-direction: row;
        flex-direction: row;
}

.layout.vertical.search-footer {
  -ms-flex-direction: column;
        -webkit-flex-direction: column;
        flex-direction: column;
}

.layout.wrap.search-footer {
  -ms-flex-wrap: wrap;
        -webkit-flex-wrap: wrap;
        flex-wrap: wrap;
}

.layout.no-wrap.search-footer {
  -ms-flex-wrap: nowrap;
        -webkit-flex-wrap: nowrap;
        flex-wrap: nowrap;
}

.layout.center.search-footer,.layout.center-center.search-footer {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.layout.center-justified.search-footer,.layout.center-center.search-footer {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.flex.search-footer {
  -ms-flex: 1 1 0.000000001px;
        -webkit-flex: 1;
        flex: 1;
        -webkit-flex-basis: 0.000000001px;
        flex-basis: 0.000000001px;
}

.flex-auto.search-footer {
  -ms-flex: 1 1 auto;
        -webkit-flex: 1 1 auto;
        flex: 1 1 auto;
}

.flex-none.search-footer {
  -ms-flex: none;
        -webkit-flex: none;
        flex: none;
}

.layout.start.search-footer {
  -ms-flex-align: start;
        -webkit-align-items: flex-start;
        align-items: flex-start;
}

.layout.center.search-footer,.layout.center-center.search-footer {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.layout.end.search-footer {
  -ms-flex-align: end;
        -webkit-align-items: flex-end;
        align-items: flex-end;
}

.layout.baseline.search-footer {
  -ms-flex-align: baseline;
        -webkit-align-items: baseline;
        align-items: baseline;
}

.layout.start-justified.search-footer {
  -ms-flex-pack: start;
        -webkit-justify-content: flex-start;
        justify-content: flex-start;
}

.layout.center-justified.search-footer,.layout.center-center.search-footer {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.layout.end-justified.search-footer {
  -ms-flex-pack: end;
        -webkit-justify-content: flex-end;
        justify-content: flex-end;
}

.layout.around-justified.search-footer {
  -ms-flex-pack: distribute;
        -webkit-justify-content: space-around;
        justify-content: space-around;
}

.layout.justified.search-footer {
  -ms-flex-pack: justify;
        -webkit-justify-content: space-between;
        justify-content: space-between;
}

.self-start.search-footer {
  -ms-align-self: flex-start;
        -webkit-align-self: flex-start;
        align-self: flex-start;
}

.self-center.search-footer {
  -ms-align-self: center;
        -webkit-align-self: center;
        align-self: center;
}

.self-end.search-footer {
  -ms-align-self: flex-end;
        -webkit-align-self: flex-end;
        align-self: flex-end;
}

.self-stretch.search-footer {
  -ms-align-self: stretch;
        -webkit-align-self: stretch;
        align-self: stretch;
}

.self-baseline.search-footer {
  -ms-align-self: baseline;
        -webkit-align-self: baseline;
        align-self: baseline;
}

.layout.start-aligned.search-footer {
  -ms-flex-line-pack: start;  
        -ms-align-content: flex-start;
        -webkit-align-content: flex-start;
        align-content: flex-start;
}

.layout.end-aligned.search-footer {
  -ms-flex-line-pack: end;  
        -ms-align-content: flex-end;
        -webkit-align-content: flex-end;
        align-content: flex-end;
}

.layout.center-aligned.search-footer {
  -ms-flex-line-pack: center;  
        -ms-align-content: center;
        -webkit-align-content: center;
        align-content: center;
}

.layout.between-aligned.search-footer {
  -ms-flex-line-pack: justify;  
        -ms-align-content: space-between;
        -webkit-align-content: space-between;
        align-content: space-between;
}

.layout.around-aligned.search-footer {
  -ms-flex-line-pack: distribute;  
        -ms-align-content: space-around;
        -webkit-align-content: space-around;
        align-content: space-around;
}</style><!-- Shady DOM styles for search-header --><style scope="search-header">search-header {
  display: block;
}

.lockup-logo.search-header,.lockup-text.search-header {
  display: inline-block;
}

.lockup-logo.search-header {
  width: 74px;
        height: 24px;
        background-image: url('https://www.gstatic.com/images/branding/googlelogo/1x/googlelogo_color_74x24dp.png');
        background-size: 74px 24px;
}

.lockup-text.search-header {
  font-family: "Product Sans", Arial, Helvetica, sans-serif;

        
        font-size: 22.1px;

        
        text-rendering: optimizeLegibility;

        
        position: relative;
        top: -6px;
        left: -5px;

        
        -webkit-font-smoothing: antialiased;
        -moz-osx-font-smoothing: grayscale;
}

.lockup-brand.search-header {
  opacity: 0.54;
}

@media (-webkit-min-device-pixel-ratio: 1.5), (min-resolution: 144dpi) {
.lockup-logo.search-header {
  background-image: url('https://www.gstatic.com/images/branding/googlelogo/2x/googlelogo_color_74x24dp.png');
}

.lockup-text.search-header {
  font-size: 21.3px;
          left: -7px;
}

}

header.search-header {
  background-color: #fafafa;
        border-bottom: 1px solid #ebebeb;
        color: #333;
        height: 60px;
        width: 100%;
        z-index: 1;
        display: block;
}

[latestStyle].search-header header.search-header {
  background-color: transparent;
}

.headerpad.search-header {
  margin-left: 16px;
}

[wide].search-header .headerpad.search-header {
  margin-left: 32px;
}

.leftheader.search-header {
  width: 322px;
}

header.search-header h1.search-header {
  display: none;
}

header.search-header h1.search-header state-modifier.search-header {
  color: #777;
        text-decoration: none;
        display: inline-block;
}

[wide].search-header header.search-header h1.search-header {
  box-sizing: border-box;
        margin: 0;
        padding: 20px 0 10px;
        font-size: 20px;
        font-weight: 400;
        white-space: nowrap;
        display: block;
}

search-box.search-header {
  height: 37px;
        padding: 0 16px;
        width: 900px;
        align-self: center;
}

.layout.horizontal.search-header,.layout.vertical.search-header {
  display: -ms-flexbox;
        display: -webkit-flex;
        display: flex;
}

.layout.inline.search-header {
  display: -ms-inline-flexbox;
        display: -webkit-inline-flex;
        display: inline-flex;
}

.layout.horizontal.search-header {
  -ms-flex-direction: row;
        -webkit-flex-direction: row;
        flex-direction: row;
}

.layout.vertical.search-header {
  -ms-flex-direction: column;
        -webkit-flex-direction: column;
        flex-direction: column;
}

.layout.wrap.search-header {
  -ms-flex-wrap: wrap;
        -webkit-flex-wrap: wrap;
        flex-wrap: wrap;
}

.layout.no-wrap.search-header {
  -ms-flex-wrap: nowrap;
        -webkit-flex-wrap: nowrap;
        flex-wrap: nowrap;
}

.layout.center.search-header,.layout.center-center.search-header {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.layout.center-justified.search-header,.layout.center-center.search-header {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.flex.search-header {
  -ms-flex: 1 1 0.000000001px;
        -webkit-flex: 1;
        flex: 1;
        -webkit-flex-basis: 0.000000001px;
        flex-basis: 0.000000001px;
}

.flex-auto.search-header {
  -ms-flex: 1 1 auto;
        -webkit-flex: 1 1 auto;
        flex: 1 1 auto;
}

.flex-none.search-header {
  -ms-flex: none;
        -webkit-flex: none;
        flex: none;
}

.layout.start.search-header {
  -ms-flex-align: start;
        -webkit-align-items: flex-start;
        align-items: flex-start;
}

.layout.center.search-header,.layout.center-center.search-header {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.layout.end.search-header {
  -ms-flex-align: end;
        -webkit-align-items: flex-end;
        align-items: flex-end;
}

.layout.baseline.search-header {
  -ms-flex-align: baseline;
        -webkit-align-items: baseline;
        align-items: baseline;
}

.layout.start-justified.search-header {
  -ms-flex-pack: start;
        -webkit-justify-content: flex-start;
        justify-content: flex-start;
}

.layout.center-justified.search-header,.layout.center-center.search-header {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.layout.end-justified.search-header {
  -ms-flex-pack: end;
        -webkit-justify-content: flex-end;
        justify-content: flex-end;
}

.layout.around-justified.search-header {
  -ms-flex-pack: distribute;
        -webkit-justify-content: space-around;
        justify-content: space-around;
}

.layout.justified.search-header {
  -ms-flex-pack: justify;
        -webkit-justify-content: space-between;
        justify-content: space-between;
}

.self-start.search-header {
  -ms-align-self: flex-start;
        -webkit-align-self: flex-start;
        align-self: flex-start;
}

.self-center.search-header {
  -ms-align-self: center;
        -webkit-align-self: center;
        align-self: center;
}

.self-end.search-header {
  -ms-align-self: flex-end;
        -webkit-align-self: flex-end;
        align-self: flex-end;
}

.self-stretch.search-header {
  -ms-align-self: stretch;
        -webkit-align-self: stretch;
        align-self: stretch;
}

.self-baseline.search-header {
  -ms-align-self: baseline;
        -webkit-align-self: baseline;
        align-self: baseline;
}

.layout.start-aligned.search-header {
  -ms-flex-line-pack: start;  
        -ms-align-content: flex-start;
        -webkit-align-content: flex-start;
        align-content: flex-start;
}

.layout.end-aligned.search-header {
  -ms-flex-line-pack: end;  
        -ms-align-content: flex-end;
        -webkit-align-content: flex-end;
        align-content: flex-end;
}

.layout.center-aligned.search-header {
  -ms-flex-line-pack: center;  
        -ms-align-content: center;
        -webkit-align-content: center;
        align-content: center;
}

.layout.between-aligned.search-header {
  -ms-flex-line-pack: justify;  
        -ms-align-content: space-between;
        -webkit-align-content: space-between;
        align-content: space-between;
}

.layout.around-aligned.search-header {
  -ms-flex-line-pack: distribute;  
        -ms-align-content: space-around;
        -webkit-align-content: space-around;
        align-content: space-around;
}</style><!-- Shady DOM styles for search-ui --><style scope="search-ui-0">.search-ui-0 paper-button.search-ui {
  display: inline-block;
        text-align: center;
        font-family: inherit;
}

.search-ui-0 .layout.horizontal.search-ui,.search-ui-0 .layout.vertical.search-ui {
  display: -ms-flexbox;
        display: -webkit-flex;
        display: flex;
}

.search-ui-0 .layout.inline.search-ui {
  display: -ms-inline-flexbox;
        display: -webkit-inline-flex;
        display: inline-flex;
}

.search-ui-0 .layout.horizontal.search-ui {
  -ms-flex-direction: row;
        -webkit-flex-direction: row;
        flex-direction: row;
}

.search-ui-0 .layout.vertical.search-ui {
  -ms-flex-direction: column;
        -webkit-flex-direction: column;
        flex-direction: column;
}

.search-ui-0 .layout.wrap.search-ui {
  -ms-flex-wrap: wrap;
        -webkit-flex-wrap: wrap;
        flex-wrap: wrap;
}

.search-ui-0 .layout.no-wrap.search-ui {
  -ms-flex-wrap: nowrap;
        -webkit-flex-wrap: nowrap;
        flex-wrap: nowrap;
}

.search-ui-0 .layout.center.search-ui,.search-ui-0 .layout.center-center.search-ui {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.search-ui-0 .layout.center-justified.search-ui,.search-ui-0 .layout.center-center.search-ui {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.search-ui-0 .flex.search-ui {
  -ms-flex: 1 1 0.000000001px;
        -webkit-flex: 1;
        flex: 1;
        -webkit-flex-basis: 0.000000001px;
        flex-basis: 0.000000001px;
}

.search-ui-0 .flex-auto.search-ui {
  -ms-flex: 1 1 auto;
        -webkit-flex: 1 1 auto;
        flex: 1 1 auto;
}

.search-ui-0 .flex-none.search-ui {
  -ms-flex: none;
        -webkit-flex: none;
        flex: none;
}

.search-ui-0 .layout.start.search-ui {
  -ms-flex-align: start;
        -webkit-align-items: flex-start;
        align-items: flex-start;
}

.search-ui-0 .layout.center.search-ui,.search-ui-0 .layout.center-center.search-ui {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.search-ui-0 .layout.end.search-ui {
  -ms-flex-align: end;
        -webkit-align-items: flex-end;
        align-items: flex-end;
}

.search-ui-0 .layout.baseline.search-ui {
  -ms-flex-align: baseline;
        -webkit-align-items: baseline;
        align-items: baseline;
}

.search-ui-0 .layout.start-justified.search-ui {
  -ms-flex-pack: start;
        -webkit-justify-content: flex-start;
        justify-content: flex-start;
}

.search-ui-0 .layout.center-justified.search-ui,.search-ui-0 .layout.center-center.search-ui {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.search-ui-0 .layout.end-justified.search-ui {
  -ms-flex-pack: end;
        -webkit-justify-content: flex-end;
        justify-content: flex-end;
}

.search-ui-0 .layout.around-justified.search-ui {
  -ms-flex-pack: distribute;
        -webkit-justify-content: space-around;
        justify-content: space-around;
}

.search-ui-0 .layout.justified.search-ui {
  -ms-flex-pack: justify;
        -webkit-justify-content: space-between;
        justify-content: space-between;
}

.search-ui-0 .self-start.search-ui {
  -ms-align-self: flex-start;
        -webkit-align-self: flex-start;
        align-self: flex-start;
}

.search-ui-0 .self-center.search-ui {
  -ms-align-self: center;
        -webkit-align-self: center;
        align-self: center;
}

.search-ui-0 .self-end.search-ui {
  -ms-align-self: flex-end;
        -webkit-align-self: flex-end;
        align-self: flex-end;
}

.search-ui-0 .self-stretch.search-ui {
  -ms-align-self: stretch;
        -webkit-align-self: stretch;
        align-self: stretch;
}

.search-ui-0 .self-baseline.search-ui {
  -ms-align-self: baseline;
        -webkit-align-self: baseline;
        align-self: baseline;
}

.search-ui-0 .layout.start-aligned.search-ui {
  -ms-flex-line-pack: start;  
        -ms-align-content: flex-start;
        -webkit-align-content: flex-start;
        align-content: flex-start;
}

.search-ui-0 .layout.end-aligned.search-ui {
  -ms-flex-line-pack: end;  
        -ms-align-content: flex-end;
        -webkit-align-content: flex-end;
        align-content: flex-end;
}

.search-ui-0 .layout.center-aligned.search-ui {
  -ms-flex-line-pack: center;  
        -ms-align-content: center;
        -webkit-align-content: center;
        align-content: center;
}

.search-ui-0 .layout.between-aligned.search-ui {
  -ms-flex-line-pack: justify;  
        -ms-align-content: space-between;
        -webkit-align-content: space-between;
        align-content: space-between;
}

.search-ui-0 .layout.around-aligned.search-ui {
  -ms-flex-line-pack: distribute;  
        -ms-align-content: space-around;
        -webkit-align-content: space-around;
        align-content: space-around;
}

.search-ui-0 {
  display: block;
        min-width: 350px;
        height: 100%;
}

.search-ui-0 #content.search-ui {
  height: 100%;
}

.search-ui-0 #leftBarOverflow.search-ui {
  padding-right: 16px;
}

.search-ui-0 [wide].search-ui #leftBarOverflow.search-ui {
  overflow: auto;
}

.search-ui-0 #leftBar.search-ui {
  box-sizing: border-box;
        color: #666;
        font-size: 13px;
        padding: 16px 0 16px 16px;
        border-bottom: 1px solid #e5e5e5;
}

.search-ui-0 [wide].search-ui #leftBar.search-ui {
  display: -ms-flexbox;
        display: -webkit-flex;
        display: flex;

        -ms-flex-direction: column;
        -webkit-flex-direction: column;
        flex-direction: column;
        position: relative;
        width: 350px;
        margin-bottom: 0;
        padding-bottom: 0;
        border-bottom: none;
}

@media print {
.search-ui-0 #leftBar.search-ui {
  display: none !important;
}

}

.search-ui-0 #leftBar.search-ui h3.search-ui {
  text-transform: uppercase;
        font-size: 13px;
        font-weight: normal;
        margin-top: 6px;
        margin-bottom: 6px;
        cursor: default;
        display: inline-block;
        color: #a3a3a3;
}

.search-ui-0 #closeButton.search-ui {
  position: absolute;
        right: 0;
}

.search-ui-0 #searchTermsHeader.search-ui {
  position: relative;
}

.search-ui-0 #searchTermsQ.search-ui iron-icon.search-ui {
  height: 16px;
        padding: 0 2px 2px 2px;
        color: #a3a3a3;
}

.search-ui-0 #searchTermsHeader.search-ui:hover iron-icon.search-ui {
  color: #4285f4;
}

.search-ui-0 #closeLeftBar.search-ui {
  height: 16px;
        width: 16px;
        padding: 0 0 8px 0;
        cursor: pointer;
}

.search-ui-0 #compactQuery.search-ui iron-icon.search-ui {
  height: 20px;
}

.search-ui-0 #backToResultsButton.search-ui {
  color: #fff;
        background: #4285f4;
        margin-top: 20px;
        text-align: center;
}

.search-ui-0 #main.search-ui {
  display: block;
        min-height: calc(100vh - 127px);
}

@media print {
.search-ui-0 [wide].search-ui main.search-ui {
  margin-left: 0;
}

}</style><!-- Shady DOM styles for import-list --><!-- Shady DOM styles for iron-request --><!-- Shady DOM styles for iron-ajax --><!-- Shady DOM styles for paper-dialog --><!-- Shady DOM styles for coverage-stats --><!-- Shady DOM styles for iron-image --><!-- Shady DOM styles for iron-selector --><!-- Shady DOM styles for paper-card --><!-- Shady DOM styles for paper-checkbox --><!-- Shady DOM styles for landing-page --><!-- Shady DOM styles for remote-fetch --><!-- Shady DOM styles for paper-spinner --><style scope="paper-spinner-0">.paper-spinner-0 {
  display: inline-block;
        position: relative;
        width: 28px;
        height: 28px;
}

.paper-spinner-0 #spinnerContainer.paper-spinner {
  width: 100%;
        height: 100%;

        
        direction: ltr;
}

.paper-spinner-0 #spinnerContainer.active.paper-spinner {
  -webkit-animation: container-rotate-paper-spinner-0 1568ms linear infinite;
        animation: container-rotate-paper-spinner-0 1568ms linear infinite;
}

@-webkit-keyframes container-rotate-paper-spinner-0 {
to {
  -webkit-transform: rotate(360deg)
}

}

@keyframes container-rotate-paper-spinner-0 {
to {
  transform: rotate(360deg)
}

}

.paper-spinner-0 .spinner-layer.paper-spinner {
  position: absolute;
        width: 100%;
        height: 100%;
        opacity: 0;
        white-space: nowrap;
        color: #4285f4;
}

.paper-spinner-0 .layer-1.paper-spinner {
  color: #4285f4;
}

.paper-spinner-0 .layer-2.paper-spinner {
  color: #db4437;
}

.paper-spinner-0 .layer-3.paper-spinner {
  color: #f4b400;
}

.paper-spinner-0 .layer-4.paper-spinner {
  color: #0f9d58;
}

.paper-spinner-0 .active.paper-spinner .spinner-layer.paper-spinner {
  -webkit-animation-name: fill-unfill-rotate-paper-spinner-0;
        -webkit-animation-duration: 5332ms;
        -webkit-animation-timing-function: cubic-bezier(0.4, 0.0, 0.2, 1);
        -webkit-animation-iteration-count: infinite;
        animation-name: fill-unfill-rotate-paper-spinner-0;
        animation-duration: 5332ms;
        animation-timing-function: cubic-bezier(0.4, 0.0, 0.2, 1);
        animation-iteration-count: infinite;
        opacity: 1;
}

.paper-spinner-0 .active.paper-spinner .spinner-layer.layer-1.paper-spinner {
  -webkit-animation-name: fill-unfill-rotate-paper-spinner-0, layer-1-fade-in-out-paper-spinner-0;
        animation-name: fill-unfill-rotate-paper-spinner-0, layer-1-fade-in-out-paper-spinner-0;
}

.paper-spinner-0 .active.paper-spinner .spinner-layer.layer-2.paper-spinner {
  -webkit-animation-name: fill-unfill-rotate-paper-spinner-0, layer-2-fade-in-out-paper-spinner-0;
        animation-name: fill-unfill-rotate-paper-spinner-0, layer-2-fade-in-out-paper-spinner-0;
}

.paper-spinner-0 .active.paper-spinner .spinner-layer.layer-3.paper-spinner {
  -webkit-animation-name: fill-unfill-rotate-paper-spinner-0, layer-3-fade-in-out-paper-spinner-0;
        animation-name: fill-unfill-rotate-paper-spinner-0, layer-3-fade-in-out-paper-spinner-0;
}

.paper-spinner-0 .active.paper-spinner .spinner-layer.layer-4.paper-spinner {
  -webkit-animation-name: fill-unfill-rotate-paper-spinner-0, layer-4-fade-in-out-paper-spinner-0;
        animation-name: fill-unfill-rotate-paper-spinner-0, layer-4-fade-in-out-paper-spinner-0;
}

@-webkit-keyframes fill-unfill-rotate-paper-spinner-0 {
12.5% {
  -webkit-transform: rotate(135deg)
}

25% {
  -webkit-transform: rotate(270deg)
}

37.5% {
  -webkit-transform: rotate(405deg)
}

50% {
  -webkit-transform: rotate(540deg)
}

62.5% {
  -webkit-transform: rotate(675deg)
}

75% {
  -webkit-transform: rotate(810deg)
}

87.5% {
  -webkit-transform: rotate(945deg)
}

to {
  -webkit-transform: rotate(1080deg)
}

}

@keyframes fill-unfill-rotate-paper-spinner-0 {
12.5% {
  transform: rotate(135deg)
}

25% {
  transform: rotate(270deg)
}

37.5% {
  transform: rotate(405deg)
}

50% {
  transform: rotate(540deg)
}

62.5% {
  transform: rotate(675deg)
}

75% {
  transform: rotate(810deg)
}

87.5% {
  transform: rotate(945deg)
}

to {
  transform: rotate(1080deg)
}

}

@-webkit-keyframes layer-1-fade-in-out-paper-spinner-0 {
0% {
  opacity: 1
}

25% {
  opacity: 1
}

26% {
  opacity: 0
}

89% {
  opacity: 0
}

90% {
  opacity: 1
}

to {
  opacity: 1
}

}

@keyframes layer-1-fade-in-out-paper-spinner-0 {
0% {
  opacity: 1
}

25% {
  opacity: 1
}

26% {
  opacity: 0
}

89% {
  opacity: 0
}

90% {
  opacity: 1
}

to {
  opacity: 1
}

}

@-webkit-keyframes layer-2-fade-in-out-paper-spinner-0 {
0% {
  opacity: 0
}

15% {
  opacity: 0
}

25% {
  opacity: 1
}

50% {
  opacity: 1
}

51% {
  opacity: 0
}

to {
  opacity: 0
}

}

@keyframes layer-2-fade-in-out-paper-spinner-0 {
0% {
  opacity: 0
}

15% {
  opacity: 0
}

25% {
  opacity: 1
}

50% {
  opacity: 1
}

51% {
  opacity: 0
}

to {
  opacity: 0
}

}

@-webkit-keyframes layer-3-fade-in-out-paper-spinner-0 {
0% {
  opacity: 0
}

40% {
  opacity: 0
}

50% {
  opacity: 1
}

75% {
  opacity: 1
}

76% {
  opacity: 0
}

to {
  opacity: 0
}

}

@keyframes layer-3-fade-in-out-paper-spinner-0 {
0% {
  opacity: 0
}

40% {
  opacity: 0
}

50% {
  opacity: 1
}

75% {
  opacity: 1
}

76% {
  opacity: 0
}

to {
  opacity: 0
}

}

@-webkit-keyframes layer-4-fade-in-out-paper-spinner-0 {
0% {
  opacity: 0
}

65% {
  opacity: 0
}

75% {
  opacity: 1
}

90% {
  opacity: 1
}

to {
  opacity: 0
}

}

@keyframes layer-4-fade-in-out-paper-spinner-0 {
0% {
  opacity: 0
}

65% {
  opacity: 0
}

75% {
  opacity: 1
}

90% {
  opacity: 1
}

to {
  opacity: 0
}

}

.paper-spinner-0 .circle-clipper.paper-spinner {
  display: inline-block;
        position: relative;
        width: 50%;
        height: 100%;
        overflow: hidden;
}

.paper-spinner-0 .spinner-layer.paper-spinner::after {
  left: 45%;
        width: 10%;
        border-top-style: solid;
}

.paper-spinner-0 .spinner-layer.paper-spinner::after,.paper-spinner-0 .circle-clipper.paper-spinner::after {
  content: '';
        box-sizing: border-box;
        position: absolute;
        top: 0;
        border-width: 3px;
        border-radius: 50%;
}

.paper-spinner-0 .circle-clipper.paper-spinner::after {
  bottom: 0;
        width: 200%;
        border-style: solid;
        border-bottom-color: transparent !important;
}

.paper-spinner-0 .circle-clipper.left.paper-spinner::after {
  left: 0;
        border-right-color: transparent !important;
        -webkit-transform: rotate(129deg);
        transform: rotate(129deg);
}

.paper-spinner-0 .circle-clipper.right.paper-spinner::after {
  left: -100%;
        border-left-color: transparent !important;
        -webkit-transform: rotate(-129deg);
        transform: rotate(-129deg);
}

.paper-spinner-0 .active.paper-spinner .gap-patch.paper-spinner::after,.paper-spinner-0 .active.paper-spinner .circle-clipper.paper-spinner::after {
  -webkit-animation-duration: 1333ms;
        -webkit-animation-timing-function: cubic-bezier(0.4, 0.0, 0.2, 1);
        -webkit-animation-iteration-count: infinite;
        animation-duration: 1333ms;
        animation-timing-function: cubic-bezier(0.4, 0.0, 0.2, 1);
        animation-iteration-count: infinite;
}

.paper-spinner-0 .active.paper-spinner .circle-clipper.left.paper-spinner::after {
  -webkit-animation-name: left-spin-paper-spinner-0;
        animation-name: left-spin-paper-spinner-0;
}

.paper-spinner-0 .active.paper-spinner .circle-clipper.right.paper-spinner::after {
  -webkit-animation-name: right-spin-paper-spinner-0;
        animation-name: right-spin-paper-spinner-0;
}

@-webkit-keyframes left-spin-paper-spinner-0 {
0% {
  -webkit-transform: rotate(130deg)
}

50% {
  -webkit-transform: rotate(-5deg)
}

to {
  -webkit-transform: rotate(130deg)
}

}

@keyframes left-spin-paper-spinner-0 {
0% {
  transform: rotate(130deg)
}

50% {
  transform: rotate(-5deg)
}

to {
  transform: rotate(130deg)
}

}

@-webkit-keyframes right-spin-paper-spinner-0 {
0% {
  -webkit-transform: rotate(-130deg)
}

50% {
  -webkit-transform: rotate(5deg)
}

to {
  -webkit-transform: rotate(-130deg)
}

}

@keyframes right-spin-paper-spinner-0 {
0% {
  transform: rotate(-130deg)
}

50% {
  transform: rotate(5deg)
}

to {
  transform: rotate(-130deg)
}

}

.paper-spinner-0 #spinnerContainer.cooldown.paper-spinner {
  -webkit-animation: container-rotate-paper-spinner-0 1568ms linear infinite, fade-out-paper-spinner-0 400ms cubic-bezier(0.4, 0.0, 0.2, 1);
        animation: container-rotate-paper-spinner-0 1568ms linear infinite, fade-out-paper-spinner-0 400ms cubic-bezier(0.4, 0.0, 0.2, 1);
}

@-webkit-keyframes fade-out-paper-spinner-0 {
0% {
  opacity: 1
}

to {
  opacity: 0
}

}

@keyframes fade-out-paper-spinner-0 {
0% {
  opacity: 1
}

to {
  opacity: 0
}

}</style><!-- Shady DOM styles for paper-spinner-lite --><!-- Shady DOM styles for concept-card --><!-- Shady DOM styles for concept-mention --><style scope="concept-mention">.block.concept-mention {
  display: block;
}

[hidden].concept-mention {
  display: none !important;
}

.invisible.concept-mention {
  visibility: hidden !important;
}

.relative.concept-mention {
  position: relative;
}

.fit.concept-mention {
  position: absolute;
        top: 0;
        right: 0;
        bottom: 0;
        left: 0;
}

body.fullbleed.concept-mention {
  margin: 0;
        height: 100vh;
}

.scroll.concept-mention {
  -webkit-overflow-scrolling: touch;
        overflow: auto;
}

.fixed-bottom.concept-mention,.fixed-left.concept-mention,.fixed-right.concept-mention,.fixed-top.concept-mention {
  position: fixed;
}

.fixed-top.concept-mention {
  top: 0;
        left: 0;
        right: 0;
}

.fixed-right.concept-mention {
  top: 0;
        right: 0;
        bottom: 0;
}

.fixed-bottom.concept-mention {
  right: 0;
        bottom: 0;
        left: 0;
}

.fixed-left.concept-mention {
  top: 0;
        bottom: 0;
        left: 0;
}

#target.concept-mention:hover {
  background-color: #e8f0fe;
}

#target[noicon].concept-mention {
  padding: 1px;
}

#target.concept-mention:hover[noicon] {
  border: 1px solid #e8f0fe;
        padding: 0;
}

#target.concept-mention {
  display: inline-block;
}

.inline-icon.concept-mention {
  cursor: pointer;
        height: 12px;
        width: 12px;
        padding-left: 4px;
        color: #5e6368;
}

concept-card.concept-mention {
  z-index: 1;
        position: absolute;
}</style><!-- Shady DOM styles for fterm-concept-card --><!-- Shady DOM styles for fterm-concept-mention --><!-- Shady DOM styles for paper-fab --><!-- Shady DOM styles for paper-icon-button --><!-- Shady DOM styles for paper-icon-button-light --><!-- Shady DOM styles for classification-tree --><style scope="classification-tree">.layout.horizontal.classification-tree,.layout.vertical.classification-tree {
  display: -ms-flexbox;
        display: -webkit-flex;
        display: flex;
}

.layout.inline.classification-tree {
  display: -ms-inline-flexbox;
        display: -webkit-inline-flex;
        display: inline-flex;
}

.layout.horizontal.classification-tree {
  -ms-flex-direction: row;
        -webkit-flex-direction: row;
        flex-direction: row;
}

.layout.vertical.classification-tree {
  -ms-flex-direction: column;
        -webkit-flex-direction: column;
        flex-direction: column;
}

.layout.wrap.classification-tree {
  -ms-flex-wrap: wrap;
        -webkit-flex-wrap: wrap;
        flex-wrap: wrap;
}

.layout.no-wrap.classification-tree {
  -ms-flex-wrap: nowrap;
        -webkit-flex-wrap: nowrap;
        flex-wrap: nowrap;
}

.layout.center.classification-tree,.layout.center-center.classification-tree {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.layout.center-justified.classification-tree,.layout.center-center.classification-tree {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.flex.classification-tree {
  -ms-flex: 1 1 0.000000001px;
        -webkit-flex: 1;
        flex: 1;
        -webkit-flex-basis: 0.000000001px;
        flex-basis: 0.000000001px;
}

.flex-auto.classification-tree {
  -ms-flex: 1 1 auto;
        -webkit-flex: 1 1 auto;
        flex: 1 1 auto;
}

.flex-none.classification-tree {
  -ms-flex: none;
        -webkit-flex: none;
        flex: none;
}

classification-tree {
  display: block;
        line-height: 20px;
}

.wrap.classification-tree {
  padding: 6px 12px;
}

.wrap.classification-tree:hover {
  background: #fafafa;
}

.description.classification-tree {
  padding-left: 4px;
}

iron-icon.classification-tree {
  color: #eaeaea;
}

.code[first].classification-tree {
  font-weight: 500;
}

.code[additional].classification-tree {
  font-style: italic;
}</style><!-- Shady DOM styles for classification-viewer --><style scope="classification-viewer">.table.classification-viewer {
  border: 1px solid #eaeaea;
        display: block;
}

.more.classification-viewer {
  padding: 6px 12px;
        cursor: pointer;
        font-style: italic;
        color: #aaa;
}

.more.classification-viewer:hover {
  background: #fafafa;
}

.chip.classification-viewer {
  display: inline-block;
        padding: 4px 6px;
        border-radius: 2px;
}

.chip[additional].classification-viewer {
  font-style: italic;
}</style><!-- Shady DOM styles for image-carousel --><style scope="image-carousel">image-carousel {
  display: block;
}

.thumbnails.image-carousel {
  display: block;
        background-color: transparent;
        padding: 7px;
        white-space: nowrap;
        overflow: auto;
        font-size: 0;
}

.thumbnails.image-carousel img.image-carousel {
  margin-right: 7px;
        cursor: zoom-in;
}

.thumbnails.image-carousel img[selected].image-carousel {
  outline: solid 3px #4285f4;
}

.thumbnails.image-carousel img.image-carousel:last-of-type {
  margin-right: 0;
}</style><!-- Shady DOM styles for image-viewer --><!-- Shady DOM styles for patent-result --><style scope="patent-result">#wrapper.patent-result {
  padding: 20px;
        max-width: 1050px;
        outline: 0;
}

.scroll-target.patent-result:before {
  display: block;
        content: '';
        visibility: hidden;
}

h1.patent-result {
  font-weight: 500;
        font-size: 20px;
        margin: 0 0 20px 0;
}

h3.patent-result {
  font-size: 15px;
        font-weight: 500;
        margin: 28px 0 18px 0;
}

dl.patent-result {
  line-height: 1.5;
        font-size: 0;
}

dt.patent-result,dd.patent-result {
  display: inline;
        font-size: 13px;
        margin: 0;
}

dt.patent-result {
  font-weight: 500;
        color: #000;
}

dt.patent-result::after {
  content: ': ';
}

dd.patent-result {
  color: #aaa;
}

dd.patent-result ~ dd.patent-result::before {
  content: ', ';
}

dt.patent-result + dd.patent-result::before {
  content: '';
}

dd.patent-result ~ dt.patent-result::before {
  display: block;
        content: '';
        margin-top: 8px;
}

dd.patent-result core-icon.patent-result {
  width: 14px;
        height: 14px;
        vertical-align: top;
        color: inherit;
}

.table.patent-result {
  display: table;
        border-collapse: collapse;
        border-spacing: 0;
        border: 1px solid #eaeaea;
        width: 100%;
        box-sizing: border-box;
}

.tr.patent-result {
  display: table-row;
        border-bottom: 1px solid #eaeaea;
}

.th.patent-result,.td.patent-result {
  display: table-cell;
        padding: 10px 12px;
}

.thead.patent-result {
  display: table-header-group;
}

.tbody.patent-result {
  display: table-row-group;
}

.th.patent-result {
  text-align: left;
        white-space: nowrap;
        background-color: #fafafa;
        color: #757575;
}

.td.patent-result {
  vertical-align: top;
}

.td.patent-result > p.patent-result {
  margin-top: 0;
}

.nowrap.patent-result {
  white-space: nowrap;
}

.knowledge-card.patent-result {
  border: 1px solid #e5e5e5;
        border-radius: 2px;
}

.knowledge-card.patent-result > header.patent-result {
  -webkit-font-smoothing: antialiased;
        -moz-osx-font-smoothing: grayscale;
        background: #4285f4;
        color: #fff;
        padding: 20px 12px;
}

.knowledge-card.patent-result > header.patent-result state-modifier.patent-result,.knowledge-card.patent-result > header.patent-result a.patent-result {
  color: inherit;
        margin: 0 12px 0 0;
        display: inline-block;
}

.knowledge-card.patent-result > header.patent-result h2.patent-result,.knowledge-card.patent-result > header.patent-result .tagline.patent-result {
  margin: 0;
        font-weight: normal;
        padding-top: 4px;
}

.knowledge-card.patent-result > header.patent-result h2.patent-result {
  font-size: 20px;
}

.knowledge-card.patent-result .knowledge-card-action-bar.patent-result {
  background: #3367d6;
        padding: 10px 12px;
        margin: 20px -12px -20px;
}

.knowledge-card.patent-result .knowledge-card-action-bar.patent-result core-icon.patent-result {
  margin-right: 12px;
        width: 24px;
        height: 24px;
}

.knowledge-card.patent-result dl.patent-result {
  padding: 16px 12px;
}

.knowledge-card.patent-result hr.patent-result {
  border: none;
        height: 1px;
        background: #e5e5e5;
        margin: 0;
}

.main-link.patent-result {
  margin: 14px;
        font-weight: 500;
}

.footer.patent-result h3.patent-result state-modifier.patent-result,.footer.patent-result h3.patent-result a.patent-result {
  float: right;
        font-weight: 400;
        font-size: 13px;
        line-height: 24px;
}

.footer.patent-result dl.patent-result {
  margin-left: 30px;
}

.footer.patent-result > ul.patent-result > li.patent-result {
  margin-bottom: 8px;
}

h3.patent-result .righthead.patent-result {
  color: #aaa;
        font-weight: 400;
        font-size: 13px;
        cursor: help;
}

.tooltip-hint.patent-result {
  border-bottom: 1px dotted #ccc;
}

[marginRight].patent-result {
  margin-right: 60px;
}

.flex-width.patent-result {
  min-width: 0;
        word-break: break-word;
}

.responsive-table.patent-result {
  width: 100%;
        overflow-x: auto;
        overflow-y: hidden;
}

.layout.horizontal.patent-result,.layout.vertical.patent-result {
  display: -ms-flexbox;
        display: -webkit-flex;
        display: flex;
}

.layout.inline.patent-result {
  display: -ms-inline-flexbox;
        display: -webkit-inline-flex;
        display: inline-flex;
}

.layout.horizontal.patent-result {
  -ms-flex-direction: row;
        -webkit-flex-direction: row;
        flex-direction: row;
}

.layout.vertical.patent-result {
  -ms-flex-direction: column;
        -webkit-flex-direction: column;
        flex-direction: column;
}

.layout.wrap.patent-result {
  -ms-flex-wrap: wrap;
        -webkit-flex-wrap: wrap;
        flex-wrap: wrap;
}

.layout.no-wrap.patent-result {
  -ms-flex-wrap: nowrap;
        -webkit-flex-wrap: nowrap;
        flex-wrap: nowrap;
}

.layout.center.patent-result,.layout.center-center.patent-result {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.layout.center-justified.patent-result,.layout.center-center.patent-result {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.flex.patent-result {
  -ms-flex: 1 1 0.000000001px;
        -webkit-flex: 1;
        flex: 1;
        -webkit-flex-basis: 0.000000001px;
        flex-basis: 0.000000001px;
}

.flex-auto.patent-result {
  -ms-flex: 1 1 auto;
        -webkit-flex: 1 1 auto;
        flex: 1 1 auto;
}

.flex-none.patent-result {
  -ms-flex: none;
        -webkit-flex: none;
        flex: none;
}

.layout.start.patent-result {
  -ms-flex-align: start;
        -webkit-align-items: flex-start;
        align-items: flex-start;
}

.layout.center.patent-result,.layout.center-center.patent-result {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.layout.end.patent-result {
  -ms-flex-align: end;
        -webkit-align-items: flex-end;
        align-items: flex-end;
}

.layout.baseline.patent-result {
  -ms-flex-align: baseline;
        -webkit-align-items: baseline;
        align-items: baseline;
}

.layout.start-justified.patent-result {
  -ms-flex-pack: start;
        -webkit-justify-content: flex-start;
        justify-content: flex-start;
}

.layout.center-justified.patent-result,.layout.center-center.patent-result {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.layout.end-justified.patent-result {
  -ms-flex-pack: end;
        -webkit-justify-content: flex-end;
        justify-content: flex-end;
}

.layout.around-justified.patent-result {
  -ms-flex-pack: distribute;
        -webkit-justify-content: space-around;
        justify-content: space-around;
}

.layout.justified.patent-result {
  -ms-flex-pack: justify;
        -webkit-justify-content: space-between;
        justify-content: space-between;
}

.self-start.patent-result {
  -ms-align-self: flex-start;
        -webkit-align-self: flex-start;
        align-self: flex-start;
}

.self-center.patent-result {
  -ms-align-self: center;
        -webkit-align-self: center;
        align-self: center;
}

.self-end.patent-result {
  -ms-align-self: flex-end;
        -webkit-align-self: flex-end;
        align-self: flex-end;
}

.self-stretch.patent-result {
  -ms-align-self: stretch;
        -webkit-align-self: stretch;
        align-self: stretch;
}

.self-baseline.patent-result {
  -ms-align-self: baseline;
        -webkit-align-self: baseline;
        align-self: baseline;
}

.layout.start-aligned.patent-result {
  -ms-flex-line-pack: start;  
        -ms-align-content: flex-start;
        -webkit-align-content: flex-start;
        align-content: flex-start;
}

.layout.end-aligned.patent-result {
  -ms-flex-line-pack: end;  
        -ms-align-content: flex-end;
        -webkit-align-content: flex-end;
        align-content: flex-end;
}

.layout.center-aligned.patent-result {
  -ms-flex-line-pack: center;  
        -ms-align-content: center;
        -webkit-align-content: center;
        align-content: center;
}

.layout.between-aligned.patent-result {
  -ms-flex-line-pack: justify;  
        -ms-align-content: space-between;
        -webkit-align-content: space-between;
        align-content: space-between;
}

.layout.around-aligned.patent-result {
  -ms-flex-line-pack: distribute;  
        -ms-align-content: space-around;
        -webkit-align-content: space-around;
        align-content: space-around;
}

.flex.patent-result,.flex-1.patent-result {
  -ms-flex: 1 1 0.000000001px;
        -webkit-flex: 1;
        flex: 1;
        -webkit-flex-basis: 0.000000001px;
        flex-basis: 0.000000001px;
}

.flex-2.patent-result {
  -ms-flex: 2;
        -webkit-flex: 2;
        flex: 2;
}

.flex-3.patent-result {
  -ms-flex: 3;
        -webkit-flex: 3;
        flex: 3;
}

.flex-4.patent-result {
  -ms-flex: 4;
        -webkit-flex: 4;
        flex: 4;
}

.flex-5.patent-result {
  -ms-flex: 5;
        -webkit-flex: 5;
        flex: 5;
}

.flex-6.patent-result {
  -ms-flex: 6;
        -webkit-flex: 6;
        flex: 6;
}

.flex-7.patent-result {
  -ms-flex: 7;
        -webkit-flex: 7;
        flex: 7;
}

.flex-8.patent-result {
  -ms-flex: 8;
        -webkit-flex: 8;
        flex: 8;
}

.flex-9.patent-result {
  -ms-flex: 9;
        -webkit-flex: 9;
        flex: 9;
}

.flex-10.patent-result {
  -ms-flex: 10;
        -webkit-flex: 10;
        flex: 10;
}

.flex-11.patent-result {
  -ms-flex: 11;
        -webkit-flex: 11;
        flex: 11;
}

.flex-12.patent-result {
  -ms-flex: 12;
        -webkit-flex: 12;
        flex: 12;
}

@media print {
#text.patent-result {
  flex-direction: column-reverse !important;
}

}

patent-result-ref-list.patent-result {
  width: 100%;
}

patent-result-ref-list.patent-result + patent-result-ref-list.patent-result {
  margin-left: 10px;
}

[narrow].patent-result {
  flex-direction: column-reverse !important;
}

.famheader.patent-result {
  text-align: left;
        white-space: nowrap;
        background-color: #fafafa;
        color: #757575;
}

.collapse.patent-result {
  color: #616161;
        font-size: 13px;
        cursor: pointer;
}

#imageDrag.patent-result {
  position: fixed;
        cursor: col-resize;
}

#imageDrag.patent-result div.patent-result {
  width: 4px;
        height: 100%;
        margin: 0 4px;
        background: #eee;
}

#imageDrag.patent-result:hover {
  background: #eee;
}

#imageDrag.patent-result:hover div.patent-result {
  background: #ccc;
}

.imageWrapper.patent-result {
  position: fixed;
        top: 60px;
        background: white;
        margin-left: 12px;
}

#notices.patent-result {
  padding: 20px;
}

#figures.patent-result {
  background: #efefef;
}

.headerButton.patent-result {
  color: #3c80f6;
        cursor: pointer;
        display: inline-block;
        padding: 0 14px;
        line-height: 27px;
}

.headerButton.patent-result iron-icon.patent-result {
  width: 16px;
        height: 16px;
}

.tr.conceptDomain.patent-result {
  display: table-row;
}

.responsive-grid.patent-result {
  display: grid;
        gap: 8px;
        grid-template-columns: none;
}</style><!-- Shady DOM styles for family-viewer --><!-- Shady DOM styles for app-row --><!-- Shady DOM styles for application-timeline --><style scope="application-timeline">.layout.horizontal.application-timeline,.layout.vertical.application-timeline {
  display: -ms-flexbox;
        display: -webkit-flex;
        display: flex;
}

.layout.inline.application-timeline {
  display: -ms-inline-flexbox;
        display: -webkit-inline-flex;
        display: inline-flex;
}

.layout.horizontal.application-timeline {
  -ms-flex-direction: row;
        -webkit-flex-direction: row;
        flex-direction: row;
}

.layout.vertical.application-timeline {
  -ms-flex-direction: column;
        -webkit-flex-direction: column;
        flex-direction: column;
}

.layout.wrap.application-timeline {
  -ms-flex-wrap: wrap;
        -webkit-flex-wrap: wrap;
        flex-wrap: wrap;
}

.layout.no-wrap.application-timeline {
  -ms-flex-wrap: nowrap;
        -webkit-flex-wrap: nowrap;
        flex-wrap: nowrap;
}

.layout.center.application-timeline,.layout.center-center.application-timeline {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.layout.center-justified.application-timeline,.layout.center-center.application-timeline {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.flex.application-timeline {
  -ms-flex: 1 1 0.000000001px;
        -webkit-flex: 1;
        flex: 1;
        -webkit-flex-basis: 0.000000001px;
        flex-basis: 0.000000001px;
}

.flex-auto.application-timeline {
  -ms-flex: 1 1 auto;
        -webkit-flex: 1 1 auto;
        flex: 1 1 auto;
}

.flex-none.application-timeline {
  -ms-flex: none;
        -webkit-flex: none;
        flex: none;
}

.wrap.application-timeline {
  padding: 16px 12px;
}

.event.application-timeline {
  padding: 3px 0;
}

[date].application-timeline {
  color: #5f6368;
        font-size: 13px;
        padding: 3px 0;
        width: 70px;
        line-height: 16px;
}

[critical].application-timeline [date].application-timeline {
  color: #202124;
        font-weight: 500;
        line-height: 18px;
}

.line.application-timeline {
  margin: 9px 6px;
        border-radius: 4px;
        background: #dadce0;
        width: 4px;
        height: 4px;
}

.dot.application-timeline {
  margin: 3px 0;
        border-radius: 4px;
        background: #dadce0;
        width: 4px;
        height: 4px;
        display: inline-block;
}

[critical].application-timeline .line.application-timeline {
  background: #4285f4;
}

[current].application-timeline .line.application-timeline {
  background: #81c995;
}

[current][critical].application-timeline .line.application-timeline {
  background: #81c995;
}

.title.application-timeline {
  font-size: 13px;
        padding: 3px 0;
        line-height: 16px;
}

.title-text.application-timeline {
  color: #5f6368;
}

.description-item.application-timeline {
  font-size: 12px;
        padding: 6px 0;
        max-width: 300px;
}

#legal.application-timeline {
  height: 14px;
        width: 14px;
        padding: 0 3px 1px 3px;
        vertical-align: text-bottom;
}

#icon.application-timeline {
  height: 12px;
        width: 12px;
        padding: 0 3px 3px 3px;
        vertical-align: text-bottom;
}

.unknown.application-timeline {
  font-style: italic;
}

.not_active.application-timeline {
  text-decoration: line-through;
}

.active.application-timeline {
  font-weight: 500;
}

#cc[this].application-timeline {
  color: #202124 !important;
        text-decoration: underline;
}

#cc[this].not_active.application-timeline {
  text-decoration: line-through underline;
}

hr.application-timeline {
  border: none;
        height: 1px;
        background: #e5e5e5;
        margin: 10px 0 16px 0;
}

.header.application-timeline {
  font-size: 13px;
        line-height: 20px;
        font-weight: 500;
        color: #202124;
        padding-bottom: 8px;
}

.block.application-timeline {
  padding: 0 6px 0 0;
}

.year.application-timeline {
  color: #5f6368;
        font-size: 13px;
        font-weight: 500;
        padding: 3px 1px;
        width: 70px;
}

.expand.application-timeline {
  cursor: pointer;
        padding: 6px 0 3px 0;
}</style><!-- Shady DOM styles for patent-text --><style scope="patent-text">.layout.horizontal.patent-text,.layout.vertical.patent-text {
  display: -ms-flexbox;
        display: -webkit-flex;
        display: flex;
}

.layout.inline.patent-text {
  display: -ms-inline-flexbox;
        display: -webkit-inline-flex;
        display: inline-flex;
}

.layout.horizontal.patent-text {
  -ms-flex-direction: row;
        -webkit-flex-direction: row;
        flex-direction: row;
}

.layout.vertical.patent-text {
  -ms-flex-direction: column;
        -webkit-flex-direction: column;
        flex-direction: column;
}

.layout.wrap.patent-text {
  -ms-flex-wrap: wrap;
        -webkit-flex-wrap: wrap;
        flex-wrap: wrap;
}

.layout.no-wrap.patent-text {
  -ms-flex-wrap: nowrap;
        -webkit-flex-wrap: nowrap;
        flex-wrap: nowrap;
}

.layout.center.patent-text,.layout.center-center.patent-text {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.layout.center-justified.patent-text,.layout.center-center.patent-text {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.flex.patent-text {
  -ms-flex: 1 1 0.000000001px;
        -webkit-flex: 1;
        flex: 1;
        -webkit-flex-basis: 0.000000001px;
        flex-basis: 0.000000001px;
}

.flex-auto.patent-text {
  -ms-flex: 1 1 auto;
        -webkit-flex: 1 1 auto;
        flex: 1 1 auto;
}

.flex-none.patent-text {
  -ms-flex: none;
        -webkit-flex: none;
        flex: none;
}

.layout.start.patent-text {
  -ms-flex-align: start;
        -webkit-align-items: flex-start;
        align-items: flex-start;
}

.layout.center.patent-text,.layout.center-center.patent-text {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.layout.end.patent-text {
  -ms-flex-align: end;
        -webkit-align-items: flex-end;
        align-items: flex-end;
}

.layout.baseline.patent-text {
  -ms-flex-align: baseline;
        -webkit-align-items: baseline;
        align-items: baseline;
}

.layout.start-justified.patent-text {
  -ms-flex-pack: start;
        -webkit-justify-content: flex-start;
        justify-content: flex-start;
}

.layout.center-justified.patent-text,.layout.center-center.patent-text {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.layout.end-justified.patent-text {
  -ms-flex-pack: end;
        -webkit-justify-content: flex-end;
        justify-content: flex-end;
}

.layout.around-justified.patent-text {
  -ms-flex-pack: distribute;
        -webkit-justify-content: space-around;
        justify-content: space-around;
}

.layout.justified.patent-text {
  -ms-flex-pack: justify;
        -webkit-justify-content: space-between;
        justify-content: space-between;
}

.self-start.patent-text {
  -ms-align-self: flex-start;
        -webkit-align-self: flex-start;
        align-self: flex-start;
}

.self-center.patent-text {
  -ms-align-self: center;
        -webkit-align-self: center;
        align-self: center;
}

.self-end.patent-text {
  -ms-align-self: flex-end;
        -webkit-align-self: flex-end;
        align-self: flex-end;
}

.self-stretch.patent-text {
  -ms-align-self: stretch;
        -webkit-align-self: stretch;
        align-self: stretch;
}

.self-baseline.patent-text {
  -ms-align-self: baseline;
        -webkit-align-self: baseline;
        align-self: baseline;
}

.layout.start-aligned.patent-text {
  -ms-flex-line-pack: start;  
        -ms-align-content: flex-start;
        -webkit-align-content: flex-start;
        align-content: flex-start;
}

.layout.end-aligned.patent-text {
  -ms-flex-line-pack: end;  
        -ms-align-content: flex-end;
        -webkit-align-content: flex-end;
        align-content: flex-end;
}

.layout.center-aligned.patent-text {
  -ms-flex-line-pack: center;  
        -ms-align-content: center;
        -webkit-align-content: center;
        align-content: center;
}

.layout.between-aligned.patent-text {
  -ms-flex-line-pack: justify;  
        -ms-align-content: space-between;
        -webkit-align-content: space-between;
        align-content: space-between;
}

.layout.around-aligned.patent-text {
  -ms-flex-line-pack: distribute;  
        -ms-align-content: space-around;
        -webkit-align-content: space-around;
        align-content: space-around;
}

patent-text {
  display: block;
}

#text.patent-text {
  line-height: 20px;
        min-width: 0;
}

#margin.patent-text {
  width: 16px;
}

#createHighlight.patent-text {
  position: relative;
        width: 16px;
        height: 16px;
        cursor: pointer;
}

.notranslate.patent-text > .google-src-text.patent-text {
  display: none;
}

.notranslate.patent-text:hover,mark.patent-text {
  background-color: #fbfcae;
}

.description-paragraph.patent-text {
  margin-block-start: 1em;
        margin-block-end: 1em;
}

.claim-text.patent-text {
  margin: 14px 0 0 20px;
}

.claim-dependent.patent-text {
  margin-left: 40px;
        color: #888;
}

[hide-claims].patent-text .claim-dependent.patent-text .claim-text.patent-text {
  height: 0;
        width: 0;
        visibility: hidden;
}

[hide-claims].patent-text .claim-dependent.patent-text .claim.patent-text:after {
  content: "…"
}

.patent-full-image.patent-text {
  max-width: 100%;
        height: auto;
}

.small-patent-image.patent-text {
  display: inline;
}

ul.patent-text {
  padding: 0 0 0 50px;
}

li.patent-text {
  list-style-type: none;
        position: relative;
}

ol.claims.patent-text li.patent-text {
  list-style-type: decimal;
        font-weight: bold;
}

ol.claims.patent-text li.patent-text > *.patent-text {
  font-weight: normal;
}

li.patent-text > para-num.patent-text {
  position: absolute;
        left: -45px;
        color: #888;
}

mark.patent-text {
  background: #fff59d;
}

mark.searchText.patent-text {
  background: #80deea;
}

.searchTextHeader.patent-text {
  font-weight: 500;
}

pre.patent-text {
  white-space: pre-wrap;
}

heading.patent-text {
  display: block;
        font-weight: 500;
        font-size: 13px;
}

#comment.patent-text {
  width: 16px;
        height: 16px;
}

[has-search][no-results].patent-text {
  display: none;
}

.highlight.patent-text {
  background-color: #fef7e0;
}</style><!-- Shady DOM styles for para-num --><!-- Shady DOM styles for result-container --><style scope="result-container">paper-spinner.result-container {
  position: absolute;
        
        top: calc(50%  + 56px - 16px);
        left: calc(50% + 175px - 16px);
}</style><!-- Shady DOM styles for scholar-result --><!-- Shady DOM styles for search-result --><style scope="search-result">.layout.horizontal.search-result,.layout.vertical.search-result {
  display: -ms-flexbox;
        display: -webkit-flex;
        display: flex;
}

.layout.inline.search-result {
  display: -ms-inline-flexbox;
        display: -webkit-inline-flex;
        display: inline-flex;
}

.layout.horizontal.search-result {
  -ms-flex-direction: row;
        -webkit-flex-direction: row;
        flex-direction: row;
}

.layout.vertical.search-result {
  -ms-flex-direction: column;
        -webkit-flex-direction: column;
        flex-direction: column;
}

.layout.wrap.search-result {
  -ms-flex-wrap: wrap;
        -webkit-flex-wrap: wrap;
        flex-wrap: wrap;
}

.layout.no-wrap.search-result {
  -ms-flex-wrap: nowrap;
        -webkit-flex-wrap: nowrap;
        flex-wrap: nowrap;
}

.layout.center.search-result,.layout.center-center.search-result {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.layout.center-justified.search-result,.layout.center-center.search-result {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.flex.search-result {
  -ms-flex: 1 1 0.000000001px;
        -webkit-flex: 1;
        flex: 1;
        -webkit-flex-basis: 0.000000001px;
        flex-basis: 0.000000001px;
}

.flex-auto.search-result {
  -ms-flex: 1 1 auto;
        -webkit-flex: 1 1 auto;
        flex: 1 1 auto;
}

.flex-none.search-result {
  -ms-flex: none;
        -webkit-flex: none;
        flex: none;
}

.layout.start.search-result {
  -ms-flex-align: start;
        -webkit-align-items: flex-start;
        align-items: flex-start;
}

.layout.center.search-result,.layout.center-center.search-result {
  -ms-flex-align: center;
        -webkit-align-items: center;
        align-items: center;
}

.layout.end.search-result {
  -ms-flex-align: end;
        -webkit-align-items: flex-end;
        align-items: flex-end;
}

.layout.baseline.search-result {
  -ms-flex-align: baseline;
        -webkit-align-items: baseline;
        align-items: baseline;
}

.layout.start-justified.search-result {
  -ms-flex-pack: start;
        -webkit-justify-content: flex-start;
        justify-content: flex-start;
}

.layout.center-justified.search-result,.layout.center-center.search-result {
  -ms-flex-pack: center;
        -webkit-justify-content: center;
        justify-content: center;
}

.layout.end-justified.search-result {
  -ms-flex-pack: end;
        -webkit-justify-content: flex-end;
        justify-content: flex-end;
}

.layout.around-justified.search-result {
  -ms-flex-pack: distribute;
        -webkit-justify-content: space-around;
        justify-content: space-around;
}

.layout.justified.search-result {
  -ms-flex-pack: justify;
        -webkit-justify-content: space-between;
        justify-content: space-between;
}

.self-start.search-result {
  -ms-align-self: flex-start;
        -webkit-align-self: flex-start;
        align-self: flex-start;
}

.self-center.search-result {
  -ms-align-self: center;
        -webkit-align-self: center;
        align-self: center;
}

.self-end.search-result {
  -ms-align-self: flex-end;
        -webkit-align-self: flex-end;
        align-self: flex-end;
}

.self-stretch.search-result {
  -ms-align-self: stretch;
        -webkit-align-self: stretch;
        align-self: stretch;
}

.self-baseline.search-result {
  -ms-align-self: baseline;
        -webkit-align-self: baseline;
        align-self: baseline;
}

.layout.start-aligned.search-result {
  -ms-flex-line-pack: start;  
        -ms-align-content: flex-start;
        -webkit-align-content: flex-start;
        align-content: flex-start;
}

.layout.end-aligned.search-result {
  -ms-flex-line-pack: end;  
        -ms-align-content: flex-end;
        -webkit-align-content: flex-end;
        align-content: flex-end;
}

.layout.center-aligned.search-result {
  -ms-flex-line-pack: center;  
        -ms-align-content: center;
        -webkit-align-content: center;
        align-content: center;
}

.layout.between-aligned.search-result {
  -ms-flex-line-pack: justify;  
        -ms-align-content: space-between;
        -webkit-align-content: space-between;
        align-content: space-between;
}

.layout.around-aligned.search-result {
  -ms-flex-line-pack: distribute;  
        -ms-align-content: space-around;
        -webkit-align-content: space-around;
        align-content: space-around;
}

search-result {
  display: block;
        font-size: 13px;
}

#mainContainer.search-result {
  min-height: calc(100vh - 127px);
        color: #333;
}

#compactQuery.search-result {
  padding: 18px 20px 10px 20px;
        cursor: pointer;
}

#compactQuery.search-result > span.search-result {
  padding: 0 14px 0 0;
}

#compactQuery.search-result iron-icon.search-result {
  padding-right: 6px;
        height: 16px;
        width: 16px;
}</style><!-- Shady DOM styles for webdoc-result --><!-- Shady DOM styles for iron-pages --><!-- Shady DOM styles for paper-tab --><!-- Shady DOM styles for paper-tabs --><!-- Shady DOM styles for debug-info --><!-- Shady DOM styles for search-paging --><!-- Shady DOM styles for search-result-item --><!-- Shady DOM styles for search-result-image --><!-- Shady DOM styles for search-result-entities --><!-- Shady DOM styles for search-result-measures --><!-- Shady DOM styles for search-results --><!-- Shady DOM styles for summary-box --><!-- Shady DOM styles for timeline-page --><!-- Shady DOM styles for iron-localstorage --><!-- Shady DOM styles for search-app --><style scope="search-app">search-app {
  display: block;
        height: 100%;
}

#toast.search-app {
  min-height: 64px;
        min-width: 742px;
}</style><style>body {transition: opacity ease-in 0.2s; } 
body[unresolved] {opacity: 0; display: block; overflow: hidden; position: relative; } 
</style>
    <title>US5874563A - Hepatitis G virus and molecular cloning thereof 
        - Google Patents</title>

    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta charset="UTF-8">
    <meta name="referrer" content="origin-when-crossorigin">
    <link rel="canonical" href="https://patents.google.com/patent/US5874563A/en">
    <meta name="description" content="
     Polypeptide antigens are disclosed which are immunoreactive with sera from individuals having a non-A, non-B, non-C, non-D, non-E Hepatitis, herein designated Hepatitis G Virus (HGV). Corresponding genomic-fragment clones containing polynucleotides encoding the open reading frame sequences for the antigenic polypeptides are taught. The antigens are useful in diagnostic methods for detecting the presence of HGV in test subjects. The antigens are also useful in vaccine and antibody preparations. In addition, the entire coding sequences of two HGV isolates are disclosed. Methods are presented for nucleic acid-based detection of HGV in samples and also methods for the isolation of further genomic sequences corresponding to HGV. 
   
   ">
    <meta name="DC.type" content="patent">
    <meta name="DC.title" content="Hepatitis G virus and molecular cloning thereof 
       ">
    <meta name="DC.date" content="1995-06-05" scheme="dateSubmitted">
    <meta name="DC.description" content="
     Polypeptide antigens are disclosed which are immunoreactive with sera from individuals having a non-A, non-B, non-C, non-D, non-E Hepatitis, herein designated Hepatitis G Virus (HGV). Corresponding genomic-fragment clones containing polynucleotides encoding the open reading frame sequences for the antigenic polypeptides are taught. The antigens are useful in diagnostic methods for detecting the presence of HGV in test subjects. The antigens are also useful in vaccine and antibody preparations. In addition, the entire coding sequences of two HGV isolates are disclosed. Methods are presented for nucleic acid-based detection of HGV in samples and also methods for the isolation of further genomic sequences corresponding to HGV. 
   
   ">
    <meta name="citation_patent_application_number" content="US:08/485,910">
    <meta name="citation_pdf_url" content="https://patentimages.storage.googleapis.com/7d/68/84/324ec6125bbfd8/US5874563.pdf">
    <meta name="citation_patent_number" content="US:5874563">
    <meta name="DC.date" content="1999-02-23" scheme="issue">
    <meta name="DC.contributor" content="Jungsuh P. Kim" scheme="inventor">
    <meta name="DC.contributor" content="Kirk E. Fry" scheme="inventor">
    <meta name="DC.contributor" content="LaVonne Marie Young" scheme="inventor">
    <meta name="DC.contributor" content="Jeffrey M. Linnen" scheme="inventor">
    <meta name="DC.contributor" content="John Wages" scheme="inventor">
    <meta name="DC.contributor" content="Genelabs Technologies Inc" scheme="assignee">
    <meta name="DC.relation" content="US:4870026" scheme="references">
    <meta name="DC.relation" content="US:5218099" scheme="references">
    <meta name="DC.relation" content="US:4810492" scheme="references">
    <meta name="DC.relation" content="US:5032511" scheme="references">
    <meta name="DC.relation" content="EP:0318216:A1" scheme="references">
    <meta name="DC.relation" content="WO:1990000597:A1" scheme="references">
    <meta name="DC.relation" content="EP:0363025:A2" scheme="references">
    <meta name="DC.relation" content="US:5275947" scheme="references">
    <meta name="DC.relation" content="US:5077193" scheme="references">
    <meta name="DC.relation" content="WO:1991006562:A1" scheme="references">
    <meta name="DC.relation" content="WO:1991015603:A1" scheme="references">
    <meta name="DC.relation" content="US:5258284" scheme="references">
    <meta name="DC.relation" content="WO:1994018217:A1" scheme="references">
    <meta name="DC.relation" content="US:5514539" scheme="references">
    <meta name="DC.relation" content="WO:1995021922:A2" scheme="references">
    <meta name="citation_reference" content="Bradley, D.W., et al., &quot;Postransfusion Non-A, Non-B Hepatitis: Physicochemical Properties of Two Distinct Agents,&quot; The Journal of Infectious Diseases, 148(2):254-265 (1983)." scheme="references">
    <meta name="citation_reference" content="Bradley, D.W., et al., Postransfusion Non A, Non B Hepatitis: Physicochemical Properties of Two Distinct Agents, The Journal of Infectious Diseases, 148(2):254 265 (1983)." scheme="references">
    <meta name="citation_reference" content="Buti, M., et al., &quot;Non-A, Non-B, Non-C, Non-E Acute Hepatitis: Does It Really Exist?&quot; Journal of Hepatology, The Journal of the European Association for the Study of the Liver, in Abstracts of The 28th Annual Meeting of the European Association for the Study of the Liver, 1-4 Sep. 1993, Paris, France. 18(Suppl 1):S25 (1993)." scheme="references">
    <meta name="citation_reference" content="Buti, M., et al., Non A, Non B, Non C, Non E Acute Hepatitis: Does It Really Exist Journal of Hepatology, The Journal of the European Association for the Study of the Liver, in Abstracts of The 28th Annual Meeting of the European Association for the Study of the Liver, 1 4 Sep. 1993, Paris, France. 18(Suppl 1):S25 (1993)." scheme="references">
    <meta name="citation_reference" content="Chan, S.W., et al., &quot;Analysis of a new hepatitis C virus type and its phylogenetic relationship to existing variants,&quot; Journal of General Virology, 75(5):1131-1141 (1992)." scheme="references">
    <meta name="citation_reference" content="Chan, S.W., et al., Analysis of a new hepatitis C virus type and its phylogenetic relationship to existing variants, Journal of General Virology, 75(5):1131 1141 (1992)." scheme="references">
    <meta name="citation_reference" content="Choo, Q L., et al., Isolation of a cDNA Clone Derived from a Blood Borne Non A, Non B Viral Hepatitis Genome, Science, 244:359 362 (1989)." scheme="references">
    <meta name="citation_reference" content="Choo, Q-L., et al., &quot;Isolation of a cDNA Clone Derived from a Blood-Borne Non-A, Non-B Viral Hepatitis Genome,&quot; Science, 244:359-362 (1989)." scheme="references">
    <meta name="citation_reference" content="Jiang, X., et al., &quot;Norwalk Virus Genome Cloning and Characterization,&quot; Science, 250:1580-1583 (1990)." scheme="references">
    <meta name="citation_reference" content="Jiang, X., et al., Norwalk Virus Genome Cloning and Characterization, Science, 250:1580 1583 (1990)." scheme="references">
    <meta name="citation_reference" content="Jones, W.F., et al., &quot;The Role of Hepatitis C Virus (HCV) and Hepatitis E Virus (HEV) in Acute Hepatitis: Evidence for a Non-A,B,C,D,E Syndrome,&quot; The American Association for the Study of Liver Diseases, 16(2 Pt. 2):77A (1992)." scheme="references">
    <meta name="citation_reference" content="Jones, W.F., et al., The Role of Hepatitis C Virus (HCV) and Hepatitis E Virus (HEV) in Acute Hepatitis: Evidence for a Non A,B,C,D,E Syndrome, The American Association for the Study of Liver Diseases, 16(2 Pt. 2):77A (1992)." scheme="references">
    <meta name="citation_reference" content="Karayiannis, P., et al., &quot;Studies of GB Hepatitis Agent in Tamarins,&quot; Hepatology, 9(2):186-192 (1989)." scheme="references">
    <meta name="citation_reference" content="Karayiannis, P., et al., Studies of GB Hepatitis Agent in Tamarins, Hepatology, 9(2):186 192 (1989)." scheme="references">
    <meta name="citation_reference" content="Matsui, S.M., et al., &quot;The Isolation and Characterization of a Norwalk Virus-specific cDNA,&quot; J. Clin. Invest., 87:1456-1461 (1991)." scheme="references">
    <meta name="citation_reference" content="Matsui, S.M., et al., The Isolation and Characterization of a Norwalk Virus specific cDNA, J. Clin. Invest., 87:1456 1461 (1991)." scheme="references">
    <meta name="citation_reference" content="Matsuura, Y., et al., &quot;Expression of the S-coded Genes of Lymphocytic Choriomeningitis Arenavirus using a Baculovirus Vector,&quot; J. Gen Virol., 67:1515-1529 (1986)." scheme="references">
    <meta name="citation_reference" content="Matsuura, Y., et al., Expression of the S coded Genes of Lymphocytic Choriomeningitis Arenavirus using a Baculovirus Vector, J. Gen Virol., 67:1515 1529 (1986)." scheme="references">
    <meta name="citation_reference" content="Murphy, F.A., &quot;Virus taxonomy&quot;, pp. 15-57 in Fields Virology, vol. 1, Lippincott-Raven Publishers, Philadelphia, 1996." scheme="references">
    <meta name="citation_reference" content="Murphy, F.A., Virus taxonomy , pp. 15 57 in Fields Virology, vol. 1, Lippincott Raven Publishers, Philadelphia, 1996." scheme="references">
    <meta name="citation_reference" content="Overton, H.A., et al., &quot;Identification of the N and NSs Proteins Coded by the Ambisense S RNA of Punta Toro Phlebovirus Using Monospecific Antisera Raised to Baculovirus Expressed N and NSs Proteins,&quot; Virology, 157:338-350 (1987)." scheme="references">
    <meta name="citation_reference" content="Overton, H.A., et al., Identification of the N and NS s Proteins Coded by the Ambisense S RNA of Punta Toro Phlebovirus Using Monospecific Antisera Raised to Baculovirus Expressed N and NSs Proteins, Virology, 157:338 350 (1987)." scheme="references">
    <meta name="citation_reference" content="Reyes, G.R., &quot;New Strategies for Isolation of Low Abundance Viral and Host cDNAs: Application to Cloning of the Hepatitis E Virus and Analysis of Tissue-Specific Transcription,&quot; Seminars in Liver Disease, 12(3):289-300 (1992)." scheme="references">
    <meta name="citation_reference" content="Reyes, G.R., et al., &quot;Hepatitis E virus (HEV): epitope mapping and detection of strain variation,&quot; in Viral Hepatitus C, D, and E. Proceedings of the International Meeting on Non-A, Non-B Hepatitus, Tokyo, 27-30 Sep. 1989, T. Shikata, et al., eds. Elsevier Science Publishers, Amsterdam, NL. Chapter 43:237-245 (1989)." scheme="references">
    <meta name="citation_reference" content="Reyes, G.R., et al., &quot;Molecular Biology of Non-A, Non-B Hepatitis Agents: Hepatitis C and Hepatitis E Viruses,&quot; Advances in Virus Research, 40:57-103 (1991)." scheme="references">
    <meta name="citation_reference" content="Reyes, G.R., et al., Hepatitis E virus (HEV): epitope mapping and detection of strain variation, in Viral Hepatitus C, D, and E. Proceedings of the International Meeting on Non A, Non B Hepatitus, Tokyo, 27 30 Sep. 1989, T. Shikata, et al., eds. Elsevier Science Publishers, Amsterdam, NL. Chapter 43:237 245 (1989)." scheme="references">
    <meta name="citation_reference" content="Reyes, G.R., et al., Molecular Biology of Non A, Non B Hepatitis Agents: Hepatitis C and Hepatitis E Viruses, Advances in Virus Research, 40:57 103 (1991)." scheme="references">
    <meta name="citation_reference" content="Reyes, G.R., New Strategies for Isolation of Low Abundance Viral and Host cDNAs: Application to Cloning of the Hepatitis E Virus and Analysis of Tissue Specific Transcription, Seminars in Liver Disease, 12(3):289 300 (1992)." scheme="references">
    <meta name="citation_reference" content="U.S. application No. 08/196,030, Simons, et al., filed Feb. 14, 1994." scheme="references">
    <meta name="citation_reference" content="U.S. application No. 08/242,654, Simons, et al., filed May 13, 1994." scheme="references">
    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:400,400italic,500,500italic,700">
    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Product+Sans">
    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Google+Sans:wght@400;500;700">

    <style>
      body { transition: none; }
    </style>

    <script src="https://apis.google.com/_/scs/abc-static/_/js/k=gapi.lb.en.IKZeRvoAYNY.O/m=client/rt=j/sv=1/d=1/ed=1/am=AAAQ/rs=AHpOoo-SMWTzMRJrTty6iE5dL_aWGYOnuw/cb=gapi.loaded_0?le=scs" async=""></script><script>
      window.version = 'patent-search.search_20240531_RC00';

      function sendFeedback() {
        userfeedback.api.startFeedback({
          'productId': '713680',
          'bucket': 'patent-search-web',
          'productVersion': window.version,
        });
      }

      window.experiments = {};
      window.experiments.patentCountries = "ae,ag,al,am,ao,ap,ar,at,au,aw,az,ba,bb,bd,be,bf,bg,bh,bj,bn,bo,br,bw,bx,by,bz,ca,cf,cg,ch,ci,cl,cm,cn,co,cr,cs,cu,cy,cz,dd,de,dj,dk,dm,do,dz,ea,ec,ee,eg,em,ep,es,fi,fr,ga,gb,gc,gd,ge,gh,gm,gn,gq,gr,gt,gw,hk,hn,hr,hu,ib,id,ie,il,in,ir,is,it,jo,jp,ke,kg,kh,km,kn,kp,kr,kw,kz,la,lc,li,lk,lr,ls,lt,lu,lv,ly,ma,mc,md,me,mg,mk,ml,mn,mo,mr,mt,mw,mx,my,mz,na,ne,ng,ni,nl,no,nz,oa,om,pa,pe,pg,ph,pl,pt,py,qa,ro,rs,ru,rw,sa,sc,sd,se,sg,si,sk,sl,sm,sn,st,su,sv,sy,sz,td,tg,th,tj,tm,tn,tr,tt,tw,tz,ua,ug,us,uy,uz,vc,ve,vn,wo,yu,za,zm,zw";
      
      
      window.experiments.keywordWizard = true;
      
      
      
      window.experiments.definitions = true;

      window.Polymer = {
        dom: 'shady',
        lazyRegister: true,
      };
    </script>

    <script src="//www.gstatic.com/patent-search/frontend/patent-search.search_20240531_RC00/scs/compiled_dir/webcomponentsjs/webcomponents-lite.min.js"></script>
    <style>[hidden]:not([style-scope]):not(.style-scope) {
  display: none !important;
}

</style><style></style><style></style><style></style><style></style><link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:400,300,300italic,400italic,500,500italic,700,700italic"><link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto+Mono:400,700"><style></style><style></style><style>
  html, body {
    min-width: 100%;
    width: 100%;
    min-height: 100%;
    height: 100%;
  }

  body {
    background-color: #fff;
    font-family: 'Roboto', sans-serif;
    color: #333;
    margin: 0;
    padding: 0;
    /*
     * Turn off https://github.com/WICG/ScrollAnchoring/blob/master/explainer.md
     * https://memegen.googleplex.com/5230756293246976
     */
    overflow-anchor: none;
  }

  body h1, body h2, body h3 {
    -webkit-font-smoothing: antialiased;
    -moz-osx-font-smoothing: grayscale;
  }

  body a:link {
    color: #3c80f6;
    text-decoration: none;
  }

  body a:visited {
    color: #3c80f6;
  }

  body a[open-result]:visited {
    color: #7e57c2;
  }

  /* For Firefox */
  a:link {
    color: #3c80f6;
    text-decoration: none;
  }

  a:visited {
    color: #3c80f6;
  }

  a[open-result]:visited {
    color: #7e57c2;
  }

  ::selection , body ::selection {
    background-color: #c6dafc;
    color: #333;
  }
</style><link rel="import" href="//www.gstatic.com/patent-search/frontend/patent-search.search_20240531_RC00/scs/compiled_dir/search-app-vulcanized.html">
  <style>[hidden] { display: none !important; }</style><script type="text/javascript" charset="UTF-8" src="https://apis.google.com/js/client.js" gapi_processed="true"></script></head>
  <body>
    <script src="//www.gstatic.com/patent-search/frontend/patent-search.search_20240531_RC00/scs/compiled_dir/search-app-vulcanized.js"></script><span style="font-size: 13px; white-space: pre-wrap; display: inline-block; position: absolute; left: -99999px; top: -99999px; max-width: 217px;"></span><span style="font-size: 13px; white-space: pre-wrap; display: inline-block; position: absolute; left: -99999px; top: -99999px; max-width: 217px;"></span>
    <search-app>
    
    <remote-fetch class="style-scope search-app"></remote-fetch>
    <search-state class="style-scope search-app"></search-state>
    <browser-ui class="style-scope search-app"></browser-ui>
    <template is="dom-if" restamp="" class="style-scope search-app"></template>
    <template is="dom-if" restamp="" class="style-scope search-app"></template>
    
      <search-result id="result" class="style-scope search-app">
    
    
    <search-ui mode="result" class="style-scope search-result x-scope search-ui-0">
    
    
    
    <div id="content" class="style-scope search-ui" wide="">
      <search-header show-search="" class="style-scope search-ui">
    
    
    <div class="style-scope search-header" wide="">
      <header class="style-scope search-header">
        <div class="layout horizontal headerpad style-scope search-header">
          <div class="layout horizontal leftheader style-scope search-header">
            <h1 class="style-scope search-header">
              <state-modifier inherit="" act="{&quot;type&quot;: &quot;HOME&quot;}" class="style-scope search-header"><a id="link" href="#" class="style-scope state-modifier" inherit="">
                <div class="lockup style-scope search-header">
                  <span class="lockup-logo style-scope search-header"></span>
                  <span class="lockup-text style-scope search-header">
                    <span class="lockup-brand style-scope search-header">&nbsp;Patents</span>
                  </span>
                </div>
              </a></state-modifier>
            </h1>
          </div>
          
            <search-box class="style-scope search-header">
    
    <div class="layout horizontal style-scope search-box" style="position: relative;">
      <div class="inputbox flex layout horizontal style-scope search-box">
        <div class="suggest-bg flex style-scope search-box">
          <input name="q" id="searchInput" data-proto="SEARCH_INPUT" autocomplete="off" class="style-scope search-box">
        </div>
        <div id="corpora" class="style-scope search-box">
          <iron-icon icon="icons:lightbulb-outline" id="patentIcon" class="style-scope search-box x-scope iron-icon-0" active=""><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M9 21c0 .55.45 1 1 1h4c.55 0 1-.45 1-1v-1H9v1zm3-19C8.14 2 5 5.14 5 9c0 2.38 1.19 4.47 3 5.74V17c0 .55.45 1 1 1h6c.55 0 1-.45 1-1v-2.26c1.81-1.27 3-3.36 3-5.74 0-3.86-3.14-7-7-7zm2.85 11.1l-.85.6V16h-4v-2.3l-.85-.6C7.8 12.16 7 10.63 7 9c0-2.76 2.24-5 5-5s5 2.24 5 5c0 1.63-.8 3.16-2.15 4.1z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
          <overlay-tooltip positioning-target="#patentIcon" direction="bottom" class="style-scope search-box" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      Include patents
    </div>
  </overlay-tooltip>
          <iron-icon icon="social:school" id="nplIcon" class="style-scope search-box x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M5 13.18v4L12 21l7-3.82v-4L12 17l-7-3.82zM12 3L1 9l11 6 9-4.91V17h2V9L12 3z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
          <overlay-tooltip positioning-target="#nplIcon" direction="bottom" class="style-scope search-box" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      Include non-patent literature
    </div>
  </overlay-tooltip>
          <keyword-wizard id="wizardIcon" class="style-scope search-box">


<div id="wizardContainer" class="style-scope keyword-wizard">
  <iron-icon icon="image:tune" id="wizardIcon" class="style-scope keyword-wizard x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M3 17v2h6v-2H3zM3 5v2h10V5H3zm10 16v-2h8v-2h-8v-2h-2v6h2zM7 9v2H3v2h4v2h2V9H7zm14 4v-2H11v2h10zm-6-4h2V7h4V5h-4V3h-2v6z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
  <paper-material elevation="2" id="wizard" class="style-scope keyword-wizard x-scope paper-material-0" hidden="">
    
    

    
    
    <div class="search-within style-scope keyword-wizard">
      <span class="wizard-header style-scope keyword-wizard">
        Search within
        <iron-icon icon="icons:help-outline" class="help-header style-scope keyword-wizard x-scope iron-icon-0" id="within"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M11 18h2v-2h-2v2zm1-16C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8zm0-14c-2.21 0-4 1.79-4 4h2c0-1.1.9-2 2-2s2 .9 2 2c0 2-3 1.75-3 5h2c0-2.25 3-2.5 3-5 0-2.21-1.79-4-4-4z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      </span>
      
      <div class="help style-scope keyword-wizard" id="withinHelp" hidden="">
        <p class="style-scope keyword-wizard"><b class="style-scope keyword-wizard">Search within the title, abstract, claims, or full patent document</b>: You can
          restrict your search to a specific field using field names.</p>
        <p class="style-scope keyword-wizard">Use <span class="q style-scope keyword-wizard">TI=</span> to search in the title, <span class="q style-scope keyword-wizard">AB=</span> for the
          abstract, <span class="q style-scope keyword-wizard">CL=</span> for the claims, or <span class="q style-scope keyword-wizard">TAC=</span> for
          all three. For example, <span class="q style-scope keyword-wizard">TI=(safety belt)</span>.
        </p>
        <p class="style-scope keyword-wizard"><b class="style-scope keyword-wizard">Search by Cooperative Patent Classifications (CPCs)</b>: These are commonly used to
          represent ideas in place of keywords, and can also be entered in a search term box. If
          you're searching for<span class="q style-scope keyword-wizard">seat belts</span>, you could also search for
          <span class="q style-scope keyword-wizard">B60R22/00</span> to retrieve documents that mention safety belts or body
          harnesses. <span class="q style-scope keyword-wizard">CPC=B60R22</span> will match documents with exactly this CPC,
          <span class="q style-scope keyword-wizard">CPC=B60R22/low</span> matches documents with this CPC or a child
          classification of this CPC.
        </p>
          <a href="https://support.google.com/faqs/answer/7049475" target="_blank" class="style-scope keyword-wizard">Learn More</a>
      </div>
      <div class="filter-chips style-scope keyword-wizard">
        <span class="chip style-scope keyword-wizard" id="title" selected="">Title</span>
        <span class="chip style-scope keyword-wizard" id="abstract" selected="">Abstract</span>
        <span class="chip style-scope keyword-wizard" id="claims" selected="">Claims</span>
        <span class="chip style-scope keyword-wizard" id="fullDoc" selected="">Full Document</span>
      </div>
      
      <div class="cpc-field style-scope keyword-wizard">
        <span class="input-label style-scope keyword-wizard"><label for="inputCPC" class="style-scope keyword-wizard">or CPC</label></span>
        <div class="cpc-inputs style-scope keyword-wizard">
          <input class="keyword-input style-scope keyword-wizard" id="inputCPC" placeholder="Or search with CPC">
          <input type="checkbox" id="childCPC" checked="" class="style-scope keyword-wizard">
          <label class="checkbox-label style-scope keyword-wizard" for="childCPC">
            Include child classifications
          </label>
        </div>
      </div>
    </div>
    
    <div class="find-patents style-scope keyword-wizard">
      <span class="wizard-header style-scope keyword-wizard">
        Find patents
        <iron-icon class="help-header style-scope keyword-wizard x-scope iron-icon-0" icon="icons:help-outline" id="searchBoolean"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M11 18h2v-2h-2v2zm1-16C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8zm0-14c-2.21 0-4 1.79-4 4h2c0-1.1.9-2 2-2s2 .9 2 2c0 2-3 1.75-3 5h2c0-2.25 3-2.5 3-5 0-2.21-1.79-4-4-4z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      </span>
      
      <div class="help style-scope keyword-wizard" id="searchBooleanHelp" hidden="">
        <p class="style-scope keyword-wizard"><b class="style-scope keyword-wizard">Keywords and boolean syntax (USPTO or EPO format)</b>:
          <span class="q style-scope keyword-wizard">seat belt</span> searches these two words, or their plurals and close
          synonyms. <span class="q style-scope keyword-wizard">"seat belt"</span> searches this exact phrase, in order.
          <span class="q style-scope keyword-wizard">-seat -belt</span> searches for documents not containing either word.
        </p>
        <p class="style-scope keyword-wizard"><b class="style-scope keyword-wizard">For searches using boolean logic</b>, the default operator is
          <span class="q style-scope keyword-wizard">AND</span> with left associativity. <i class="style-scope keyword-wizard">Note:</i> this means
          <span class="q style-scope keyword-wizard">safety OR seat belt</span> is searched as <span class="q style-scope keyword-wizard">(safety OR seat)
          AND belt</span>. Each word automatically includes plurals and close synonyms. Adjacent
          words that are implicitly ANDed together, such as <span class="q style-scope keyword-wizard">(safety belt)</span>,
          are treated as a phrase when generating synonyms.
        </p>
        <a href="https://support.google.com/faqs/answer/7049475" target="_blank" class="style-scope keyword-wizard">Learn More</a>
      </div>
      <div class="style-scope keyword-wizard">
        <span class="input-label style-scope keyword-wizard"><label for="inputAnd" class="style-scope keyword-wizard">with ALL of the words</label></span>
        <input class="keyword-input style-scope keyword-wizard" id="inputAnd" placeholder="with ALL of the words">
      </div>
      <div class="style-scope keyword-wizard">
        <span class="input-label style-scope keyword-wizard"><label for="inputExact" class="style-scope keyword-wizard">with the EXACT PHRASE</label></span>
        <input class="keyword-input style-scope keyword-wizard" id="inputExact" placeholder="with the EXACT PHRASE"></div>
      <div class="style-scope keyword-wizard">
        <span class="input-label style-scope keyword-wizard"><label for="inputOr" class="style-scope keyword-wizard">with AT LEAST ONE of the words</label></span>
        <input class="keyword-input style-scope keyword-wizard" id="inputOr" placeholder="with AT LEAST ONE of the words"></div>
      <div class="style-scope keyword-wizard">
        <span class="input-label style-scope keyword-wizard"><label for="inputMinus" class="style-scope keyword-wizard">WITHOUT the words</label></span>
        <input class="keyword-input style-scope keyword-wizard" id="inputMinus" placeholder="WITHOUT the words"></div>
    </div>
    <div class="search-by style-scope keyword-wizard">
      <div class="wizard-header style-scope keyword-wizard">
          Search by
          <iron-icon icon="icons:help-outline" class="help-header style-scope keyword-wizard x-scope iron-icon-0" id="chem"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M11 18h2v-2h-2v2zm1-16C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8zm0-14c-2.21 0-4 1.79-4 4h2c0-1.1.9-2 2-2s2 .9 2 2c0 2-3 1.75-3 5h2c0-2.25 3-2.5 3-5 0-2.21-1.79-4-4-4z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      </div>
      
      <div class="help style-scope keyword-wizard" id="chemHelp" hidden="">
        <p class="style-scope keyword-wizard"><b class="style-scope keyword-wizard">Chemistry searches</b> match terms (trade names, IUPAC names, etc.
          extracted from the entire document, and processed from .MOL files.)
        </p>
        <p class="style-scope keyword-wizard">Substructure (use <span class="q style-scope keyword-wizard">SSS=</span>) and similarity (use
          <span class="q style-scope keyword-wizard">~</span>) searches are limited to one per search at the top-level
          <span class="q style-scope keyword-wizard">AND</span> condition. Exact searches can be used multiple times throughout
          the search query.
        </p>
        <p class="style-scope keyword-wizard">Searching by SMILES or InChi key requires no special syntax. To search by SMARTS, use
          <span class="q style-scope keyword-wizard">SMARTS=</span>.
        </p>
        <p class="style-scope keyword-wizard">To search for multiple molecules, select "Batch" in the "Type" menu. Enter multiple
          molecules separated by whitespace or by comma.
        </p>
        <a href="https://support.google.com/faqs/answer/7049475" target="_blank" class="style-scope keyword-wizard">Learn More</a>
      </div>
      
      <div class="chemistry-field style-scope keyword-wizard">
          <span class="input-label style-scope keyword-wizard"><label for="inputChem" class="style-scope keyword-wizard">Chemistry</label></span>
          <span class="chem-type-label style-scope keyword-wizard"><label for="chemType" hidden="" class="style-scope keyword-wizard">Type</label></span>
          <select id="chemType" class="style-scope keyword-wizard">
            <option value="identical" class="style-scope keyword-wizard">Identical</option>
            <option value="substructure" class="style-scope keyword-wizard">Substructure</option>
            <option value="smarts" class="style-scope keyword-wizard">SMARTS</option>
            <option value="similarity" class="style-scope keyword-wizard">Similarity</option>
            <option value="batch" class="style-scope keyword-wizard">Batch</option>
          </select>
          <textarea class="keyword-input style-scope keyword-wizard" id="inputChem" placeholder="Trade name, SMILES, InChiKey"></textarea>
      </div>
      <div class="import style-scope keyword-wizard">
        <div class="wizard-header style-scope keyword-wizard">
          Patent numbers
        </div>
        <span class="import-button style-scope keyword-wizard">
          <a href="https://patents.google.com/import" class="style-scope keyword-wizard"><button class="style-scope keyword-wizard">Import</button></a>
        </span>
        <p class="import-text style-scope keyword-wizard">
          Search specific patents by importing a CSV or list of patent publication or application numbers.
        </p>
      </div>
    </div>
    
    <div class="wizard-buttons style-scope keyword-wizard">
      <button type="reset" class="style-scope keyword-wizard">Close</button>
      <button type="button" autofocus="" class="style-scope keyword-wizard">Add</button>
    </div>
  
  </paper-material>
</div>
</keyword-wizard>
          <overlay-tooltip positioning-target="#wizardIcon" direction="bottom" class="style-scope search-box" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      Display advanced search options
    </div>
  </overlay-tooltip>
        </div>
      </div>
      <paper-material id="suggestMenu" class="style-scope search-box x-scope paper-material-0" elevation="1" hidden="">
    
    

    
        <template is="dom-repeat" class="style-scope search-box"></template>
        <div class="row style-scope search-box" hidden="">
          <iron-icon class="suggest-icon style-scope search-box x-scope iron-icon-0" icon="error"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm1 15h-2v-2h2v2zm0-4h-2V7h2v6z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon> Sorry, we couldn't find this patent number.
        </div>
        <div class="row style-scope search-box" hidden="">
          <iron-icon class="suggest-icon style-scope search-box x-scope iron-icon-0" icon="error"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm1 15h-2v-2h2v2zm0-4h-2V7h2v6z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon> 
        </div>
      
  </paper-material>
      <button id="searchButton" class="style-scope search-box"><iron-icon class="style-scope search-box x-scope iron-icon-0" icon="search"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon></button>
    </div>
  </search-box>
          <template is="dom-if" class="style-scope search-header"></template>
          <div class="flex style-scope search-header"></div>
          
            <div id="header-container" class="style-scope search-header">
              <result-nav class="style-scope search-header">
    
    <div class="style-scope result-nav" hidden="">
      <state-modifier id="current" act="{&quot;type&quot;: &quot;BACK_TO_RESULTS&quot;}" inherit="" class="style-scope result-nav"><a id="link" href="#" class="style-scope state-modifier" inherit=""><span class="style-scope result-nav"></span> of
      <template is="dom-if" class="style-scope result-nav"></template>
      0<template is="dom-if" class="style-scope result-nav"></template></a></state-modifier>
      <state-modifier id="prevResult" inherit="" act="{&quot;type&quot;: &quot;NAV_RESULT&quot;, &quot;result&quot;: &quot;$id&quot;, &quot;page&quot;: $page}" class="style-scope result-nav" disabled=""><a id="link" href="#" class="style-scope state-modifier" inherit="">
        <iron-icon icon="chevron-left" class="style-scope result-nav x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M15.41 7.41L14 6l-6 6 6 6 1.41-1.41L10.83 12z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      </a></state-modifier><state-modifier id="nextResult" inherit="" act="{&quot;type&quot;: &quot;NAV_RESULT&quot;, &quot;result&quot;: &quot;$id&quot;, &quot;page&quot;: $page}" class="style-scope result-nav" disabled=""><a id="link" href="#" class="style-scope state-modifier" inherit="">
        <iron-icon icon="chevron-right" class="style-scope result-nav x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M10 6L8.59 7.41 13.17 12l-4.58 4.59L10 18l6-6z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      </a></state-modifier>
      <overlay-tooltip positioning-target="#prevResult" direction="bottom" arrow="" class="style-scope result-nav" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip"></div>
    <div class="style-scope overlay-tooltip">
      Previous result
    </div>
  </overlay-tooltip>
      <overlay-tooltip positioning-target="#nextResult" direction="bottom" arrow="" class="style-scope result-nav" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip"></div>
    <div class="style-scope overlay-tooltip">
      Next result
    </div>
  </overlay-tooltip>
    </div>
  </result-nav>
            </div>
          <template is="dom-if" class="style-scope search-header"></template>
        </div>
      </header>
    </div>
  </search-header>

      <div class="layout horizontal style-scope search-ui">
        <template is="dom-if" class="style-scope search-ui"></template>

        <div id="main" class="style-scope search-ui">
          <search-term-wizard class="style-scope search-ui"><pat-dialog open="" class="style-scope search-term-wizard">
  <span class="header style-scope search-term-wizard">
    <span class="title style-scope search-term-wizard">Search tools</span>
    <span id="language" class="style-scope search-term-wizard">
      <text-button id="textButton" selected="" class="style-scope search-term-wizard">Text
</text-button>
      <text-button id="classificationButton" class="style-scope search-term-wizard">Classification
</text-button>
      <text-button id="chemistryButton" class="style-scope search-term-wizard">Chemistry
</text-button>
      <text-button id="measureButton" class="style-scope search-term-wizard">Measure
</text-button>
      <text-button id="numbersButton" class="style-scope search-term-wizard">Numbers
</text-button>
    </span>
  </span>
  <span id="panels" class="style-scope search-term-wizard">
    <text-search-term-wizard id="textPanel" class="style-scope search-term-wizard" selected=""><span class="style-scope text-search-term-wizard">
  <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49" class="style-scope text-search-term-wizard">
    <path d="M3.85 15.275V13h10.3v2.275Zm0-4.275V8.725h16.3V11Z" class="style-scope text-search-term-wizard"></path>
  </svg>
  <outlined-text-field id="textField" placeholder="Type a concept you want to search for here" class="style-scope text-search-term-wizard" value=""><span class="style-scope outlined-text-field">
  <input id="input" spellcheck="false" autocomplete="off" class="style-scope outlined-text-field" aria-disabled="false" placeholder="Type a concept you want to search for here">
  
</span>
</outlined-text-field>
</span>
<span class="style-scope text-search-term-wizard">
  <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49" class="style-scope text-search-term-wizard">
    <path d="M8 18.025h8v-2.1H8Zm0-4h8v-2.1H8ZM6.075 22.2q-.95 0-1.612-.662-.663-.663-.663-1.613V4.075q0-.95.663-1.613.662-.662 1.612-.662h8L20.2 7.925v12q0 .95-.662 1.613-.663.662-1.613.662Zm6.85-13.125v-5h-6.85v15.85h11.85V9.075Zm-6.85-5v5-5 15.85-15.85Z" class="style-scope text-search-term-wizard"></path>
  </svg>
  <filter-chip id="fullDocuments" selected="" class="style-scope text-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Full documents</filter-chip>
  <filter-chip id="title" class="style-scope text-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Title</filter-chip>
  <filter-chip id="abstract" class="style-scope text-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Abstract</filter-chip>
  <filter-chip id="claims" class="style-scope text-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Claims</filter-chip>
</span>
<span class="style-scope text-search-term-wizard">
  <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49" class="style-scope text-search-term-wizard">
    <path d="M16 16.925q2.05 0 3.488-1.437Q20.925 14.05 20.925 12t-1.437-3.488Q18.05 7.075 16 7.075q-.625 0-1.2.15t-1.075.425q.7.925 1.087 2.025.388 1.1.388 2.325t-.388 2.325q-.387 1.1-1.087 2.025.5.275 1.075.425.575.15 1.2.15Zm-4-2.075q.45-.575.688-1.313.237-.737.237-1.537 0-.8-.237-1.538Q12.45 9.725 12 9.15q-.45.575-.688 1.312-.237.738-.237 1.538t.237 1.537q.238.738.688 1.313Zm-4 2.075q.625 0 1.2-.15t1.075-.425q-.7-.925-1.087-2.025Q8.8 13.225 8.8 12t.388-2.325q.387-1.1 1.087-2.025-.5-.275-1.075-.425-.575-.15-1.2-.15-2.05 0-3.487 1.437Q3.075 9.95 3.075 12t1.438 3.488Q5.95 16.925 8 16.925ZM8 19.2q-3 0-5.1-2.1Q.8 15 .8 12q0-3 2.1-5.1Q5 4.8 8 4.8q1.125 0 2.137.312 1.013.313 1.863.913.85-.6 1.863-.913Q14.875 4.8 16 4.8q3 0 5.1 2.1Q23.2 9 23.2 12q0 3-2.1 5.1-2.1 2.1-5.1 2.1-1.125 0-2.137-.313-1.013-.312-1.863-.912-.85.6-1.863.912Q9.125 19.2 8 19.2Z" class="style-scope text-search-term-wizard"></path>
  </svg>
  <filter-chip id="all" selected="" class="style-scope text-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
All</filter-chip>
  <filter-chip id="any" class="style-scope text-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Any</filter-chip>
  <filter-chip id="exact" class="style-scope text-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Exact</filter-chip>
  <filter-chip id="not" class="style-scope text-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Not</filter-chip>
</span>
<span class="style-scope text-search-term-wizard">
  <mat-button id="button" filled="" class="style-scope text-search-term-wizard x-scope mat-button-0" disabled=""><span slot="label" class="style-scope text-search-term-wizard">Add AND condition</span></mat-button></span></text-search-term-wizard>
    <classification-wizard id="classificationPanel" class="style-scope search-term-wizard"><span class="style-scope classification-wizard">
  <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49" class="style-scope classification-wizard">
    <path d="M3.85 15.275V13h10.3v2.275Zm0-4.275V8.725h16.3V11Z" class="style-scope classification-wizard"></path>
  </svg>
  <outlined-text-field id="textField" placeholder="Enter a CPC here" uppercase="" class="style-scope classification-wizard" value=""><span class="style-scope outlined-text-field">
  <input id="input" spellcheck="false" autocomplete="off" class="style-scope outlined-text-field" aria-disabled="false" placeholder="Enter a CPC here">
  
</span>
</outlined-text-field>
</span>
<template is="dom-if" class="style-scope classification-wizard"></template>
<template is="dom-if" class="style-scope classification-wizard"></template>
<template is="dom-if" class="style-scope classification-wizard"></template>
<span class="style-scope classification-wizard">
  <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49" class="style-scope classification-wizard">
    <path d="M16 16.925q2.05 0 3.488-1.437Q20.925 14.05 20.925 12t-1.437-3.488Q18.05 7.075 16 7.075q-.625 0-1.2.15t-1.075.425q.7.925 1.087 2.025.388 1.1.388 2.325t-.388 2.325q-.387 1.1-1.087 2.025.5.275 1.075.425.575.15 1.2.15Zm-4-2.075q.45-.575.688-1.313.237-.737.237-1.537 0-.8-.237-1.538Q12.45 9.725 12 9.15q-.45.575-.688 1.312-.237.738-.237 1.538t.237 1.537q.238.738.688 1.313Zm-4 2.075q.625 0 1.2-.15t1.075-.425q-.7-.925-1.087-2.025Q8.8 13.225 8.8 12t.388-2.325q.387-1.1 1.087-2.025-.5-.275-1.075-.425-.575-.15-1.2-.15-2.05 0-3.487 1.437Q3.075 9.95 3.075 12t1.438 3.488Q5.95 16.925 8 16.925ZM8 19.2q-3 0-5.1-2.1Q.8 15 .8 12q0-3 2.1-5.1Q5 4.8 8 4.8q1.125 0 2.137.312 1.013.313 1.863.913.85-.6 1.863-.913Q14.875 4.8 16 4.8q3 0 5.1 2.1Q23.2 9 23.2 12q0 3-2.1 5.1-2.1 2.1-5.1 2.1-1.125 0-2.137-.313-1.013-.312-1.863-.912-.85.6-1.863.912Q9.125 19.2 8 19.2Z" class="style-scope classification-wizard"></path>
  </svg>
  <filter-chip id="notExact" class="style-scope classification-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>

    These CPCs and their children
  </filter-chip>
  <filter-chip selected="" id="exact" class="style-scope classification-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>

    These exact CPCs
  </filter-chip>
</span>
<span class="style-scope classification-wizard">
  <mat-button id="button" filled="" class="style-scope classification-wizard x-scope mat-button-0" disabled=""><span slot="label" class="style-scope classification-wizard">Add AND condition</span></mat-button>
</span></classification-wizard>
    <chemistry-search-term-wizard id="chemistryPanel" class="style-scope search-term-wizard"><span class="style-scope chemistry-search-term-wizard">
  <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49" class="style-scope chemistry-search-term-wizard">
    <path d="M3.85 15.275V13h10.3v2.275Zm0-4.275V8.725h16.3V11Z" class="style-scope chemistry-search-term-wizard"></path>
  </svg>
  <outlined-text-field id="textField" placeholder="Trade name, SMILES, InChI Key" class="style-scope chemistry-search-term-wizard" value=""><span class="style-scope outlined-text-field">
  <input id="input" spellcheck="false" autocomplete="off" class="style-scope outlined-text-field" aria-disabled="false" placeholder="Trade name, SMILES, InChI Key">
  
</span>
</outlined-text-field>
  <outlined-textarea id="textArea" placeholder="Enter multiple SMILES or InChI Keys, one per line" rows="4" class="style-scope chemistry-search-term-wizard"><span class="style-scope outlined-textarea">
  <textarea id="textarea" autocomplete="off" autocorrect="off" spellcheck="false" tabindex="0" aria-disabled="false" wrap="soft" class="style-scope outlined-textarea" rows="4" placeholder="Enter multiple SMILES or InChI Keys, one per line">  </textarea>
  <div id="helperText" class="style-scope outlined-textarea"> </div>
</span>
<span class="right-button-container style-scope outlined-textarea">
  
</span>
</outlined-textarea>
  <a href="https://support.google.com/faqs/answer/7049475#zippy=%2Csearching-by-chemistry" target="_blank" class="style-scope chemistry-search-term-wizard">
    <svg class="help style-scope chemistry-search-term-wizard" xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49">
      <path d="M11.95 18q.525 0 .888-.363.362-.362.362-.887t-.362-.887q-.363-.363-.888-.363t-.888.363q-.362.362-.362.887t.362.887q.363.363.888.363Zm-.9-3.85h1.85q0-.825.188-1.3.187-.475 1.062-1.3.65-.65 1.025-1.238.375-.587.375-1.412 0-1.4-1.025-2.15T12.1 6q-1.425 0-2.312.75-.888.75-1.238 1.8l1.65.65q.125-.45.563-.975Q11.2 7.7 12.1 7.7q.8 0 1.2.437.4.438.4.963 0 .5-.3.937-.3.438-.75.813-1.1.975-1.35 1.475-.25.5-.25 1.825ZM12 22q-2.075 0-3.9-.788-1.825-.787-3.175-2.137-1.35-1.35-2.137-3.175Q2 14.075 2 12t.788-3.9q.787-1.825 2.137-3.175 1.35-1.35 3.175-2.138Q9.925 2 12 2t3.9.787q1.825.788 3.175 2.138 1.35 1.35 2.137 3.175Q22 9.925 22 12t-.788 3.9q-.787 1.825-2.137 3.175-1.35 1.35-3.175 2.137Q14.075 22 12 22Zm0-2q3.35 0 5.675-2.325Q20 15.35 20 12q0-3.35-2.325-5.675Q15.35 4 12 4 8.65 4 6.325 6.325 4 8.65 4 12q0 3.35 2.325 5.675Q8.65 20 12 20Zm0-8Z" class="style-scope chemistry-search-term-wizard"></path>
    </svg>
  </a>
</span>
<span id="chemQuantifiers" class="style-scope chemistry-search-term-wizard">
  <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49" class="style-scope chemistry-search-term-wizard">
    <path d="M5 21.2q-1.4 0-2-1.25-.6-1.25.3-2.325l5.5-6.7V5.15h-.825q-.475 0-.8-.338Q6.85 4.475 6.85 4q0-.475.325-.813.325-.337.8-.337h8.05q.475 0 .8.337.325.338.325.813 0 .475-.325.812-.325.338-.8.338H15.2v5.775l5.5 6.7q.9 1.075.3 2.325-.6 1.25-2 1.25Zm2.15-3.275h9.7l-3.275-3.85h-3.15Zm-2 1h13.7l-5.925-7.2V5.15h-1.85v6.575Zm6.85-6.9Z" class="style-scope chemistry-search-term-wizard"></path>
  </svg>
  <filter-chip id="exact" selected="" class="style-scope chemistry-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Exact</filter-chip>
  <filter-chip id="exactBatch" class="style-scope chemistry-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Exact Batch</filter-chip>
  <filter-chip id="similar" class="style-scope chemistry-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Similar</filter-chip>
  <filter-chip id="substructure" class="style-scope chemistry-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Substructure</filter-chip>
  <filter-chip id="smarts" class="style-scope chemistry-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Substructure (SMARTS)</filter-chip>
</span>
<span id="location" class="style-scope chemistry-search-term-wizard">
  <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49" class="style-scope chemistry-search-term-wizard">
    <path d="M8 18.025h8v-2.1H8Zm0-4h8v-2.1H8ZM6.075 22.2q-.95 0-1.612-.662-.663-.663-.663-1.613V4.075q0-.95.663-1.613.662-.662 1.612-.662h8L20.2 7.925v12q0 .95-.662 1.613-.663.662-1.613.662Zm6.85-13.125v-5h-6.85v15.85h11.85V9.075Zm-6.85-5v5-5 15.85-15.85Z" class="style-scope chemistry-search-term-wizard"></path>
  </svg>
  <filter-chip id="fullDocuments" selected="" class="style-scope chemistry-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Full documents</filter-chip>
  <filter-chip id="claims" class="style-scope chemistry-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Claims only</filter-chip>
</span>
<span class="style-scope chemistry-search-term-wizard">
  <mat-button id="button" filled="" class="style-scope chemistry-search-term-wizard x-scope mat-button-0" disabled=""><span slot="label" class="style-scope chemistry-search-term-wizard">Add AND condition</span></mat-button>
</span>
</chemistry-search-term-wizard>
    <measure-search-term-wizard id="measurePanel" class="style-scope search-term-wizard"><span class="style-scope measure-search-term-wizard">
  <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49" class="style-scope measure-search-term-wizard">
    <path d="M4.1 18.15q-.95 0-1.612-.663-.663-.662-.663-1.612v-7.75q0-.95.663-1.613Q3.15 5.85 4.1 5.85h15.8q.95 0 1.613.662.662.663.662 1.613v7.75q0 .95-.662 1.612-.663.663-1.613.663Zm0-2.275h15.8v-7.75H17V12h-2V8.125h-2V12h-2V8.125H9V12H7V8.125H4.1v7.75ZM7 12h2Zm4 0h2Zm4 0h2Zm-3 0Z" class="style-scope measure-search-term-wizard"></path>
  </svg>
  <outlined-text-field id="measureExpression" placeholder="1.5 mm, 20 ft, 400-500 fahrenheit, 800 MHz, 0.01-100 mol" class="style-scope measure-search-term-wizard" value=""><span class="style-scope outlined-text-field">
  <input id="input" spellcheck="false" autocomplete="off" class="style-scope outlined-text-field" aria-disabled="false" placeholder="1.5 mm, 20 ft, 400-500 fahrenheit, 800 MHz, 0.01-100 mol">
  
</span>
</outlined-text-field>
</span>
<span class="style-scope measure-search-term-wizard">
  <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49" class="style-scope measure-search-term-wizard">
    <path d="M3.875 20.15V8.85h4.25v11.3Zm6 0V3.85h4.25v16.3Zm6 0v-7.3h4.25v7.3Z" class="style-scope measure-search-term-wizard"></path>
  </svg>
  <outlined-text-field id="context" placeholder="activity, wavelength, embossing depth, absorption" class="style-scope measure-search-term-wizard" value=""><span class="style-scope outlined-text-field">
  <input id="input" spellcheck="false" autocomplete="off" class="style-scope outlined-text-field" aria-disabled="false" placeholder="activity, wavelength, embossing depth, absorption">
  
</span>
</outlined-text-field>
</span>
<span class="style-scope measure-search-term-wizard">
  <mat-button id="button" filled="" class="style-scope measure-search-term-wizard x-scope mat-button-0" disabled=""><span slot="label" class="style-scope measure-search-term-wizard">Add AND condition</span></mat-button>
</span>
</measure-search-term-wizard>
    <numerical-search-term-wizard id="numericalPanel" class="style-scope search-term-wizard"><span class="style-scope numerical-search-term-wizard">
  <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49" class="style-scope numerical-search-term-wizard">
    <path d="M3.85 15.275V13h10.3v2.275Zm0-4.275V8.725h16.3V11Z" class="style-scope numerical-search-term-wizard"></path>
  </svg>
  <outlined-textarea id="batch" placeholder="Enter multiple Patent Application and Publication Numbers, one per line" rows="4" class="style-scope numerical-search-term-wizard"><span class="style-scope outlined-textarea">
  <textarea id="textarea" autocomplete="off" autocorrect="off" spellcheck="false" tabindex="0" aria-disabled="false" wrap="soft" class="style-scope outlined-textarea" rows="4" placeholder="Enter multiple Patent Application and Publication Numbers, one per line">  </textarea>
  <div id="helperText" class="style-scope outlined-textarea"> </div>
</span>
<span class="right-button-container style-scope outlined-textarea">
  
</span>
</outlined-textarea>
</span>
<template is="dom-if" class="style-scope numerical-search-term-wizard"></template>
<template is="dom-if" class="style-scope numerical-search-term-wizard"></template>
<span class="style-scope numerical-search-term-wizard">
  <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49" class="style-scope numerical-search-term-wizard">
    <path d="m10.825 23.45-1.6-1.625L10.9 20.15H8q-.35.9-1.162 1.488-.813.587-1.838.587-1.35 0-2.287-.937-.938-.938-.938-2.288 0-1.025.588-1.837Q2.95 16.35 3.85 16V8q-.9-.35-1.487-1.163Q1.775 6.025 1.775 5q0-1.35.938-2.288Q3.65 1.775 5 1.775q1.025 0 1.838.587Q7.65 2.95 8 3.85h2.9L9.225 2.175l1.6-1.625L15.275 5l-4.45 4.45-1.6-1.625L10.9 6.15H8q-.25.65-.725 1.125T6.15 8v8q.65.25 1.125.725T8 17.85h2.9l-1.675-1.675 1.6-1.625 4.45 4.45ZM19 22.225q-1.35 0-2.287-.937-.938-.938-.938-2.288 0-1.05.588-1.863.587-.812 1.487-1.137V8q-.9-.325-1.487-1.137-.588-.813-.588-1.863 0-1.35.938-2.288.937-.937 2.287-.937 1.35 0 2.288.937.937.938.937 2.288 0 1.05-.587 1.863Q21.05 7.675 20.15 8v8q.9.35 1.488 1.163.587.812.587 1.837 0 1.35-.937 2.288-.938.937-2.288.937ZM5 19.95q.4 0 .675-.275T5.95 19q0-.4-.275-.675T5 18.05q-.4 0-.675.275T4.05 19q0 .4.275.675T5 19.95Zm14 0q.4 0 .675-.275T19.95 19q0-.4-.275-.675T19 18.05q-.4 0-.675.275T18.05 19q0 .4.275.675t.675.275Zm-14-14q.4 0 .675-.275T5.95 5q0-.4-.275-.675T5 4.05q-.4 0-.675.275T4.05 5q0 .4.275.675T5 5.95Zm14 0q.4 0 .675-.275T19.95 5q0-.4-.275-.675T19 4.05q-.4 0-.675.275T18.05 5q0 .4.275.675T19 5.95ZM5 19Zm14 0ZM5 5Zm14 0Z" class="style-scope numerical-search-term-wizard"></path>
  </svg>
  <filter-chip id="applicationNumbers" class="style-scope numerical-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Application Numbers</filter-chip>
  <filter-chip id="publicationNumbers" class="style-scope numerical-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Publication Numbers</filter-chip>
  <filter-chip id="either" selected="" class="style-scope numerical-search-term-wizard"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" preserveAspectRatio="none" class="style-scope filter-chip">
  <path d="M18.9 35.7 7.7 24.5 9.85 22.35 18.9 31.4 38.1 12.2 40.25 14.35Z" class="style-scope filter-chip"></path>
</svg>
Either</filter-chip>
</span>
<span class="style-scope numerical-search-term-wizard">
  <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49" class="style-scope numerical-search-term-wizard">
    <path d="M12 22.2q-2.1 0-3.962-.8-1.863-.8-3.25-2.175Q3.4 17.85 2.6 15.988 1.8 14.125 1.8 12t.8-3.988q.8-1.862 2.188-3.237Q6.175 3.4 8.038 2.6 9.9 1.8 12 1.8q2.125 0 3.988.8 1.862.8 3.237 2.175Q20.6 6.15 21.4 8.012q.8 1.863.8 3.988t-.8 3.988q-.8 1.862-2.175 3.237Q17.85 20.6 15.988 21.4q-1.863.8-3.988.8Zm-1-2.325v-1.95q-.8 0-1.387-.575-.588-.575-.588-1.4v-.975L4.275 10.2q-.075.45-.137.9-.063.45-.063.9 0 3 1.975 5.25T11 19.875Zm6.85-2.525q.5-.55.888-1.175.387-.625.65-1.313.262-.687.4-1.412.137-.725.137-1.45 0-2.425-1.338-4.438-1.337-2.012-3.612-2.887v.375q0 .825-.587 1.4-.588.575-1.388.575h-2v2q0 .4-.287.687-.288.288-.688.288h-2v2h5.95q.425 0 .713.287.287.288.287.688v2.975h1q.65 0 1.163.387.512.388.712 1.013Z" class="style-scope numerical-search-term-wizard"></path>
  </svg>
  <outlined-text-field id="countryFilter" placeholder="Restrict search to these countries" class="style-scope numerical-search-term-wizard" value=""><span class="style-scope outlined-text-field">
  <input id="input" spellcheck="false" autocomplete="off" class="style-scope outlined-text-field" aria-disabled="false" placeholder="Restrict search to these countries">
  
</span>
</outlined-text-field>
</span>
<span class="style-scope numerical-search-term-wizard">
  <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" fill="#041E49" class="style-scope numerical-search-term-wizard">
    <path d="M2.85 18.225V15.95h6.3v2.275Zm0-5.075v-2.3h12.3v2.3Zm0-5.1V5.775h18.3V8.05Z" class="style-scope numerical-search-term-wizard"></path>
  </svg>
  <outlined-text-field id="countryOrder" placeholder="Prefer results from these countries in this order" class="style-scope numerical-search-term-wizard" value="US, EP, WO, JP, CN"><span class="style-scope outlined-text-field">
  <input id="input" spellcheck="false" autocomplete="off" class="style-scope outlined-text-field" aria-disabled="false" placeholder="Prefer results from these countries in this order">
  
</span>
</outlined-text-field>
</span>
<span class="style-scope numerical-search-term-wizard">
  <mat-button id="button" filled="" class="style-scope numerical-search-term-wizard x-scope mat-button-0" disabled=""><span slot="label" class="style-scope numerical-search-term-wizard">Add AND condition</span></mat-button>
</span>
</numerical-search-term-wizard>
  </span>

</pat-dialog>
</search-term-wizard>
          
      <div content="" class="style-scope search-result">
        <div id="mainContainer" class="vertical layout style-scope search-result">
          <template is="dom-if" class="style-scope search-result"></template>
          <result-container class="style-scope search-result">
    

    <paper-spinner class="style-scope result-container x-scope paper-spinner-0" aria-hidden="true" hidden=""><div id="spinnerContainer" class="  style-scope paper-spinner"><div class="spinner-layer layer-1 style-scope paper-spinner"><div class="circle-clipper left style-scope paper-spinner"></div><div class="circle-clipper right style-scope paper-spinner"></div></div><div class="spinner-layer layer-2 style-scope paper-spinner"><div class="circle-clipper left style-scope paper-spinner"></div><div class="circle-clipper right style-scope paper-spinner"></div></div><div class="spinner-layer layer-3 style-scope paper-spinner"><div class="circle-clipper left style-scope paper-spinner"></div><div class="circle-clipper right style-scope paper-spinner"></div></div><div class="spinner-layer layer-4 style-scope paper-spinner"><div class="circle-clipper left style-scope paper-spinner"></div><div class="circle-clipper right style-scope paper-spinner"></div></div></div></paper-spinner>
    
      <patent-result class="style-scope result-container">
    
    
    <div class="layout horizontal style-scope patent-result">
      <div style="width: 1200px; height: auto; overflow: auto;" class="style-scope patent-result">
        <div id="wrapper" tabindex="0" class="style-scope patent-result">
          <result-tags class="style-scope patent-result"></result-tags>
          <h1 id="title" class="scroll-target style-scope patent-result">
            Hepatitis G virus and molecular cloning thereof 
       </h1>
          <div class="horizontal layout style-scope patent-result">
            
            <div class="flex-3 style-scope patent-result" style="margin-top:-20px;overflow:hidden;" marginright="">
              <section id="abstract" class="style-scope patent-result">
                <h3 class="layout horizontal style-scope patent-result">
                  <div class="flex style-scope patent-result">Abstract</div>
                  <div class="righthead style-scope patent-result" hidden="">translated from </div>
                </h3>
                <patent-text name="abstract" class="style-scope patent-result" style="padding-top: 0px;">
    
    
    <template is="dom-if" restamp="" class="style-scope patent-text"></template>
    <overlay-tooltip id="tooltip" class="style-scope patent-text" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      
    </div>
  </overlay-tooltip>
    <div class="layout horizontal style-scope patent-text">
      <section id="text" class="flex style-scope patent-text"><abstract mxw-id="PA37880898" lang="EN" load-source="patent-office" class="style-scope patent-text">
    <div class="abstract style-scope patent-text">Polypeptide antigens are disclosed which are immunoreactive with sera from individuals having a non-A, non-B, non-C, non-D, non-E Hepatitis, herein designated Hepatitis G Virus (HGV). Corresponding genomic-fragment clones containing polynucleotides encoding the open reading frame sequences for the antigenic polypeptides are taught. The antigens are useful in diagnostic methods for detecting the presence of HGV in test subjects. The antigens are also useful in vaccine and antibody preparations. In addition, the entire coding sequences of two HGV isolates are disclosed. Methods are presented for nucleic acid-based detection of HGV in samples and also methods for the isolation of further genomic sequences corresponding to HGV.</div>
  </abstract>
  </section>
      <div id="margin" class="style-scope patent-text">
        <div id="createHighlight" style="top: 0px;" class="style-scope patent-text" hidden="">
          <iron-icon id="comment" icon="editor:insert-comment" class="style-scope patent-text x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M20 2H4c-1.1 0-2 .9-2 2v12c0 1.1.9 2 2 2h14l4 4V4c0-1.1-.9-2-2-2zm-2 12H6v-2h12v2zm0-3H6V9h12v2zm0-3H6V6h12v2z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
        </div>
      </div>
    </div>
  </patent-text>
              </section>
              
              
                <section id="thumbnails" class="style-scope patent-result">
                  <h3 class="style-scope patent-result">Images (<span class="style-scope patent-result">17</span>)</h3>
                  <image-carousel id="figures" class="style-scope patent-result">
    
    
      <div class="thumbnails style-scope image-carousel">
        
          <img class="style-scope image-carousel" src="https://patentimages.storage.googleapis.com/e5/0c/b3/ca0dede671cb06/US5874563-drawings-page-3.png">
        
          <img class="style-scope image-carousel" src="https://patentimages.storage.googleapis.com/1b/3b/43/51aa6d108b8ea7/US5874563-drawings-page-4.png">
        
          <img class="style-scope image-carousel" src="https://patentimages.storage.googleapis.com/fc/9f/80/259a81b563a1df/US5874563-drawings-page-5.png">
        
          <img class="style-scope image-carousel" src="https://patentimages.storage.googleapis.com/3a/dd/ed/567e8db6432a36/US5874563-drawings-page-6.png">
        
          <img class="style-scope image-carousel" src="https://patentimages.storage.googleapis.com/3c/ba/60/62c6e8f99c2ab3/US5874563-drawings-page-7.png">
        
          <img class="style-scope image-carousel" src="https://patentimages.storage.googleapis.com/e7/ea/f3/312d91b6d933f4/US5874563-drawings-page-8.png">
        
          <img class="style-scope image-carousel" src="https://patentimages.storage.googleapis.com/8b/45/68/7ae598a6777a6d/US5874563-drawings-page-9.png">
        
          <img class="style-scope image-carousel" src="https://patentimages.storage.googleapis.com/a9/94/5c/5aa0e734fab248/US5874563-drawings-page-10.png">
        
          <img class="style-scope image-carousel" src="https://patentimages.storage.googleapis.com/84/0f/2d/ff5fc2b2b99e73/US5874563-drawings-page-11.png">
        
          <img class="style-scope image-carousel" src="https://patentimages.storage.googleapis.com/4d/ef/e0/3a3f9cd3dd5930/US5874563-drawings-page-12.png">
        
          <img class="style-scope image-carousel" src="https://patentimages.storage.googleapis.com/9c/10/52/449ef98e7fa9e1/US5874563-drawings-page-13.png">
        
          <img class="style-scope image-carousel" src="https://patentimages.storage.googleapis.com/aa/87/83/8097212f1cd1f5/US5874563-drawings-page-14.png">
        
          <img class="style-scope image-carousel" src="https://patentimages.storage.googleapis.com/04/f5/98/43d07015894225/US5874563-drawings-page-15.png">
        
          <img class="style-scope image-carousel" src="https://patentimages.storage.googleapis.com/f4/88/57/ba8ca9b81d83be/US5874563-drawings-page-16.png">
        
          <img class="style-scope image-carousel" src="https://patentimages.storage.googleapis.com/f0/5a/90/146620e9434f3a/US5874563-drawings-page-17.png">
        
          <img class="style-scope image-carousel" src="https://patentimages.storage.googleapis.com/2d/e9/50/77307b257ff925/US5874563-drawings-page-18.png">
        
          <img class="style-scope image-carousel" src="https://patentimages.storage.googleapis.com/04/cd/f1/1ca912babf2d3e/US5874563-drawings-page-19.png">
        <template is="dom-repeat" as="img" class="style-scope image-carousel"></template>
      </div>
    <template is="dom-if" class="style-scope image-carousel"></template>
  </image-carousel>
                </section>
              <template is="dom-if" class="style-scope patent-result"></template>
              
              <section id="classifications" class="style-scope patent-result">
                <h3 class="layout horizontal style-scope patent-result">
                  <div class="flex style-scope patent-result">Classifications</div>
                  <div id="machineClassifiedWarning" class="righthead tooltip-hint style-scope patent-result" hidden="">machine-classified</div>
                  <div id="machineClassifiedCPCWarning" class="righthead tooltip-hint style-scope patent-result" hidden="">cpc-machine-classified</div>
                  <div id="machineClassifiedFtermWarning" class="righthead tooltip-hint style-scope patent-result" hidden="">fterm-machine-classified</div>
                  <div class="style-scope patent-result" hidden="">&nbsp;</div>
                  <div id="familyClassifiedFtermWarning" class="righthead tooltip-hint style-scope patent-result" hidden="">fterm-family-classified</div>
                </h3>
                <overlay-tooltip positioning-target="#machineClassifiedWarning" direction="bottom" class="style-scope patent-result" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      The classifications are assigned by a computer and are not a legal conclusion. <br class="style-scope patent-result"> Google has not performed a legal analysis and makes no representation as to the accuracy of the classifications listed.
    </div>
  </overlay-tooltip>
                <overlay-tooltip positioning-target="#machineClassifiedCPCWarning" direction="bottom" class="style-scope patent-result" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      The CPC classifications are assigned by a computer and are not a legal conclusion. <br class="style-scope patent-result"> Google has not performed a legal analysis and makes no representation as to the accuracy of the classifications listed.
    </div>
  </overlay-tooltip>
                <overlay-tooltip positioning-target="#familyClassifiedFtermWarning" direction="bottom" class="style-scope patent-result" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      The F-term classifications are assigned based on a patent family member containing these classification codes.
    </div>
  </overlay-tooltip>
                <overlay-tooltip positioning-target="#machineClassifiedFtermWarning" direction="bottom" class="style-scope patent-result" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      The F-term classifications are assigned by a computer and are not a legal conclusion. <br class="style-scope patent-result"> Google has not performed a legal analysis and makes no representation as to the accuracy of the classifications listed.
    </div>
  </overlay-tooltip>
                <classification-viewer class="style-scope patent-result">
    
    <div class="table style-scope classification-viewer">
      
        <classification-tree class="style-scope classification-viewer">
    
    <div class="style-scope classification-tree">
      <div class="layout horizontal wrap style-scope classification-tree">
        <div class="flex style-scope classification-tree">
          
            
              <div class="style-scope classification-tree" hidden="">
                <concept-mention class="style-scope classification-tree">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                  <state-modifier class="code style-scope classification-tree" act="{&quot;type&quot;: &quot;QUERY_ADD_CPC&quot;, &quot;cpc&quot;: &quot;$cpc&quot;}" data-cpc="C"><a id="link" href="#" class="style-scope state-modifier">C</a></state-modifier>
                  <span class="description style-scope classification-tree">CHEMISTRY; METALLURGY</span>
                
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
              </div>
            <template is="dom-if" class="style-scope classification-tree"></template>
            <template is="dom-if" class="style-scope classification-tree"></template>
          
            
              <div class="style-scope classification-tree" hidden="">
                <concept-mention class="style-scope classification-tree">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                  <state-modifier class="code style-scope classification-tree" act="{&quot;type&quot;: &quot;QUERY_ADD_CPC&quot;, &quot;cpc&quot;: &quot;$cpc&quot;}" data-cpc="C07"><a id="link" href="#" class="style-scope state-modifier">C07</a></state-modifier>
                  <span class="description style-scope classification-tree">ORGANIC CHEMISTRY</span>
                
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
              </div>
            <template is="dom-if" class="style-scope classification-tree"></template>
            <template is="dom-if" class="style-scope classification-tree"></template>
          
            
              <div class="style-scope classification-tree" hidden="">
                <concept-mention class="style-scope classification-tree">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                  <state-modifier class="code style-scope classification-tree" act="{&quot;type&quot;: &quot;QUERY_ADD_CPC&quot;, &quot;cpc&quot;: &quot;$cpc&quot;}" data-cpc="C07K"><a id="link" href="#" class="style-scope state-modifier">C07K</a></state-modifier>
                  <span class="description style-scope classification-tree">PEPTIDES</span>
                
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
              </div>
            <template is="dom-if" class="style-scope classification-tree"></template>
            <template is="dom-if" class="style-scope classification-tree"></template>
          
            
              <div class="style-scope classification-tree" hidden="">
                <concept-mention class="style-scope classification-tree">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                  <state-modifier class="code style-scope classification-tree" act="{&quot;type&quot;: &quot;QUERY_ADD_CPC&quot;, &quot;cpc&quot;: &quot;$cpc&quot;}" data-cpc="C07K14/00"><a id="link" href="#" class="style-scope state-modifier">C07K14/00</a></state-modifier>
                  <span class="description style-scope classification-tree">Peptides having more than 20 amino acids; Gastrins; Somatostatins; Melanotropins; Derivatives thereof</span>
                
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
              </div>
            <template is="dom-if" class="style-scope classification-tree"></template>
            <template is="dom-if" class="style-scope classification-tree"></template>
          
            
              <div class="style-scope classification-tree">
                <concept-mention class="style-scope classification-tree">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                  <state-modifier class="code style-scope classification-tree" act="{&quot;type&quot;: &quot;QUERY_ADD_CPC&quot;, &quot;cpc&quot;: &quot;$cpc&quot;}" first="true" data-cpc="C07K14/005"><a id="link" href="#" class="style-scope state-modifier">C07K14/005</a></state-modifier>
                  <span class="description style-scope classification-tree">Peptides having more than 20 amino acids; Gastrins; Somatostatins; Melanotropins; Derivatives thereof from viruses</span>
                
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
              </div>
            <template is="dom-if" class="style-scope classification-tree"></template>
            <template is="dom-if" class="style-scope classification-tree"></template>
          <template is="dom-repeat" class="style-scope classification-tree"></template>
        </div>
      </div>
    </div>
  </classification-tree>
      <template is="dom-repeat" class="style-scope classification-viewer"></template>
      
      <div class="style-scope classification-viewer">
        
          <classification-tree class="style-scope classification-viewer">
    
    <div class="style-scope classification-tree">
      <div class="layout horizontal wrap style-scope classification-tree">
        <div class="flex style-scope classification-tree">
          
            
              <div class="style-scope classification-tree" hidden="">
                <concept-mention class="style-scope classification-tree">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                  <state-modifier class="code style-scope classification-tree" act="{&quot;type&quot;: &quot;QUERY_ADD_CPC&quot;, &quot;cpc&quot;: &quot;$cpc&quot;}" data-cpc="A"><a id="link" href="#" class="style-scope state-modifier">A</a></state-modifier>
                  <span class="description style-scope classification-tree">HUMAN NECESSITIES</span>
                
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
              </div>
            <template is="dom-if" class="style-scope classification-tree"></template>
            <template is="dom-if" class="style-scope classification-tree"></template>
          
            
              <div class="style-scope classification-tree" hidden="">
                <concept-mention class="style-scope classification-tree">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                  <state-modifier class="code style-scope classification-tree" act="{&quot;type&quot;: &quot;QUERY_ADD_CPC&quot;, &quot;cpc&quot;: &quot;$cpc&quot;}" data-cpc="A61"><a id="link" href="#" class="style-scope state-modifier">A61</a></state-modifier>
                  <span class="description style-scope classification-tree">MEDICAL OR VETERINARY SCIENCE; HYGIENE</span>
                
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
              </div>
            <template is="dom-if" class="style-scope classification-tree"></template>
            <template is="dom-if" class="style-scope classification-tree"></template>
          
            
              <div class="style-scope classification-tree" hidden="">
                <concept-mention class="style-scope classification-tree">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                  <state-modifier class="code style-scope classification-tree" act="{&quot;type&quot;: &quot;QUERY_ADD_CPC&quot;, &quot;cpc&quot;: &quot;$cpc&quot;}" data-cpc="A61K"><a id="link" href="#" class="style-scope state-modifier">A61K</a></state-modifier>
                  <span class="description style-scope classification-tree">PREPARATIONS FOR MEDICAL, DENTAL OR TOILETRY PURPOSES</span>
                
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
              </div>
            <template is="dom-if" class="style-scope classification-tree"></template>
            <template is="dom-if" class="style-scope classification-tree"></template>
          
            
              <div class="style-scope classification-tree">
                <concept-mention class="style-scope classification-tree">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                  <state-modifier class="code style-scope classification-tree" act="{&quot;type&quot;: &quot;QUERY_ADD_CPC&quot;, &quot;cpc&quot;: &quot;$cpc&quot;}" additional="true" data-cpc="A61K38/00"><a id="link" href="#" class="style-scope state-modifier">A61K38/00</a></state-modifier>
                  <span class="description style-scope classification-tree">Medicinal preparations containing peptides</span>
                
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
              </div>
            <template is="dom-if" class="style-scope classification-tree"></template>
            <template is="dom-if" class="style-scope classification-tree"></template>
          <template is="dom-repeat" class="style-scope classification-tree"></template>
        </div>
      </div>
    </div>
  </classification-tree>
        
          <classification-tree class="style-scope classification-viewer">
    
    <div class="style-scope classification-tree">
      <div class="layout horizontal wrap style-scope classification-tree">
        <div class="flex style-scope classification-tree">
          
            
              <div class="style-scope classification-tree" hidden="">
                <concept-mention class="style-scope classification-tree">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                  <state-modifier class="code style-scope classification-tree" act="{&quot;type&quot;: &quot;QUERY_ADD_CPC&quot;, &quot;cpc&quot;: &quot;$cpc&quot;}" data-cpc="C"><a id="link" href="#" class="style-scope state-modifier">C</a></state-modifier>
                  <span class="description style-scope classification-tree">CHEMISTRY; METALLURGY</span>
                
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
              </div>
            <template is="dom-if" class="style-scope classification-tree"></template>
            <template is="dom-if" class="style-scope classification-tree"></template>
          
            
              <div class="style-scope classification-tree" hidden="">
                <concept-mention class="style-scope classification-tree">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                  <state-modifier class="code style-scope classification-tree" act="{&quot;type&quot;: &quot;QUERY_ADD_CPC&quot;, &quot;cpc&quot;: &quot;$cpc&quot;}" data-cpc="C12"><a id="link" href="#" class="style-scope state-modifier">C12</a></state-modifier>
                  <span class="description style-scope classification-tree">BIOCHEMISTRY; BEER; SPIRITS; WINE; VINEGAR; MICROBIOLOGY; ENZYMOLOGY; MUTATION OR GENETIC ENGINEERING</span>
                
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
              </div>
            <template is="dom-if" class="style-scope classification-tree"></template>
            <template is="dom-if" class="style-scope classification-tree"></template>
          
            
              <div class="style-scope classification-tree" hidden="">
                <concept-mention class="style-scope classification-tree">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                  <state-modifier class="code style-scope classification-tree" act="{&quot;type&quot;: &quot;QUERY_ADD_CPC&quot;, &quot;cpc&quot;: &quot;$cpc&quot;}" data-cpc="C12N"><a id="link" href="#" class="style-scope state-modifier">C12N</a></state-modifier>
                  <span class="description style-scope classification-tree">MICROORGANISMS OR ENZYMES; COMPOSITIONS THEREOF; PROPAGATING, PRESERVING, OR MAINTAINING MICROORGANISMS; MUTATION OR GENETIC ENGINEERING; CULTURE MEDIA</span>
                
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
              </div>
            <template is="dom-if" class="style-scope classification-tree"></template>
            <template is="dom-if" class="style-scope classification-tree"></template>
          
            
              <div class="style-scope classification-tree" hidden="">
                <concept-mention class="style-scope classification-tree">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                  <state-modifier class="code style-scope classification-tree" act="{&quot;type&quot;: &quot;QUERY_ADD_CPC&quot;, &quot;cpc&quot;: &quot;$cpc&quot;}" data-cpc="C12N2770/00"><a id="link" href="#" class="style-scope state-modifier">C12N2770/00</a></state-modifier>
                  <span class="description style-scope classification-tree">MICROORGANISMS OR ENZYMES; COMPOSITIONS THEREOF; PROPAGATING, PRESERVING, OR MAINTAINING MICROORGANISMS; MUTATION OR GENETIC ENGINEERING; CULTURE MEDIA ssRNA viruses positive-sense</span>
                
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
              </div>
            <template is="dom-if" class="style-scope classification-tree"></template>
            <template is="dom-if" class="style-scope classification-tree"></template>
          
            
              <div class="style-scope classification-tree" hidden="">
                <concept-mention class="style-scope classification-tree">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                  <state-modifier class="code style-scope classification-tree" act="{&quot;type&quot;: &quot;QUERY_ADD_CPC&quot;, &quot;cpc&quot;: &quot;$cpc&quot;}" data-cpc="C12N2770/00011"><a id="link" href="#" class="style-scope state-modifier">C12N2770/00011</a></state-modifier>
                  <span class="description style-scope classification-tree">Details</span>
                
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
              </div>
            <template is="dom-if" class="style-scope classification-tree"></template>
            <template is="dom-if" class="style-scope classification-tree"></template>
          
            
              <div class="style-scope classification-tree" hidden="">
                <concept-mention class="style-scope classification-tree">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                  <state-modifier class="code style-scope classification-tree" act="{&quot;type&quot;: &quot;QUERY_ADD_CPC&quot;, &quot;cpc&quot;: &quot;$cpc&quot;}" data-cpc="C12N2770/24011"><a id="link" href="#" class="style-scope state-modifier">C12N2770/24011</a></state-modifier>
                  <span class="description style-scope classification-tree">Flaviviridae</span>
                
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
              </div>
            <template is="dom-if" class="style-scope classification-tree"></template>
            <template is="dom-if" class="style-scope classification-tree"></template>
          
            
              <div class="style-scope classification-tree" hidden="">
                <concept-mention class="style-scope classification-tree">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                  <state-modifier class="code style-scope classification-tree" act="{&quot;type&quot;: &quot;QUERY_ADD_CPC&quot;, &quot;cpc&quot;: &quot;$cpc&quot;}" data-cpc="C12N2770/24211"><a id="link" href="#" class="style-scope state-modifier">C12N2770/24211</a></state-modifier>
                  <span class="description style-scope classification-tree">Hepacivirus, e.g. hepatitis C virus, hepatitis G virus</span>
                
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
              </div>
            <template is="dom-if" class="style-scope classification-tree"></template>
            <template is="dom-if" class="style-scope classification-tree"></template>
          
            
              <div class="style-scope classification-tree">
                <concept-mention class="style-scope classification-tree">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                  <state-modifier class="code style-scope classification-tree" act="{&quot;type&quot;: &quot;QUERY_ADD_CPC&quot;, &quot;cpc&quot;: &quot;$cpc&quot;}" additional="true" data-cpc="C12N2770/24222"><a id="link" href="#" class="style-scope state-modifier">C12N2770/24222</a></state-modifier>
                  <span class="description style-scope classification-tree">New viral proteins or individual genes, new structural or functional aspects of known viral proteins or genes</span>
                
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
              </div>
            <template is="dom-if" class="style-scope classification-tree"></template>
            <template is="dom-if" class="style-scope classification-tree"></template>
          <template is="dom-repeat" class="style-scope classification-tree"></template>
        </div>
      </div>
    </div>
  </classification-tree>
        <template is="dom-repeat" class="style-scope classification-viewer"></template>
        <div class="more style-scope classification-viewer" style="" hidden="">View 2 more classifications</div>
        <div class="more style-scope classification-viewer">Hide more classifications</div>
      </div>
      <template is="dom-if" class="style-scope classification-viewer"></template>
    </div>
  </classification-viewer>
              </section>
              
              <template is="dom-if" class="style-scope patent-result"></template>
              
              <template is="dom-if" class="style-scope patent-result"></template>
              
              
                <section class="style-scope patent-result">
                  <h3 class="layout horizontal style-scope patent-result">
                    <div class="flex style-scope patent-result">Landscapes</div>
                  </h3>
                  <div class="responsive-grid style-scope patent-result">
                    
                      <landscape-link class="style-scope patent-result"><state-modifier act="{&quot;type&quot;:&quot;QUERY_SET_TERM&quot;,&quot;term&quot;:&quot;$term&quot;,&quot;origin&quot;:&quot;LANDSCAPE_LINK&quot;}" class="style-scope landscape-link" data-term="Chemical &amp; Material Sciences"><a id="link" href="#" class="style-scope state-modifier">
  <requery-tile class="area style-scope landscape-link">
<span class="label style-scope requery-tile">
    Chemical &amp; Material Sciences
  </span>
<span class="search-icon style-scope requery-tile"></span>
</requery-tile>
</a></state-modifier>
</landscape-link>
                    
                      <landscape-link class="style-scope patent-result"><state-modifier act="{&quot;type&quot;:&quot;QUERY_SET_TERM&quot;,&quot;term&quot;:&quot;$term&quot;,&quot;origin&quot;:&quot;LANDSCAPE_LINK&quot;}" class="style-scope landscape-link" data-term="Organic Chemistry"><a id="link" href="#" class="style-scope state-modifier">
  <requery-tile class="area style-scope landscape-link">
<span class="label style-scope requery-tile">
    Organic Chemistry
  </span>
<span class="search-icon style-scope requery-tile"></span>
</requery-tile>
</a></state-modifier>
</landscape-link>
                    <template is="dom-repeat" class="style-scope patent-result"></template>
                  </div>
                  <s-more-inline class="style-scope patent-result"><hr aria-hidden="true" class="style-scope s-more-inline">

<div class="button style-scope s-more-inline">
  <span slot="expand_text" class="style-scope patent-result">Show more</span>
  <span class="expand more style-scope s-more-inline"></span>
</div>
<div class="button style-scope s-more-inline" hidden="">
  <span slot="collapse_text" class="style-scope patent-result">Show less</span>
  <span class="expand less style-scope s-more-inline"></span>
</div>
</s-more-inline>
                </section>
              <template is="dom-if" class="style-scope patent-result"></template>
            </div>
            <div class="flex-2 style-scope patent-result">
              <section class="knowledge-card style-scope patent-result">
                <header class="style-scope patent-result">
                  <h2 id="pubnum" class="style-scope patent-result">US5874563A</h2>
                  <p class="tagline style-scope patent-result">United States</p>
                  <overlay-tooltip positioning-target="#pubnum" direction="bottom" class="style-scope patent-result" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      
                    <div style="max-width: 300px;" class="style-scope patent-result">Patent</div>
                  
    </div>
  </overlay-tooltip>

                  <div class="knowledge-card-action-bar style-scope patent-result">
                    <a target="_blank" class="style-scope patent-result" href="https://patentimages.storage.googleapis.com/7d/68/84/324ec6125bbfd8/US5874563.pdf"><iron-icon icon="description" class="style-scope patent-result x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M14 2H6c-1.1 0-1.99.9-1.99 2L4 20c0 1.1.89 2 1.99 2H18c1.1 0 2-.9 2-2V8l-6-6zm2 16H8v-2h8v2zm0-4H8v-2h8v2zm-3-5V3.5L18.5 9H13z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon> Download PDF</a>
                    <state-modifier inherit="" act="{&quot;type&quot;: &quot;QUERY_FIND_PRIOR_ART&quot;, &quot;keywords&quot;: $keywords, &quot;before&quot;: &quot;$before&quot;}" class="style-scope patent-result" data-keywords="[&quot;hgv&quot;,&quot;polynucleotide&quot;,&quot;seq&quot;,&quot;dna&quot;,&quot;hepatitis&quot;]" data-before="19940520"><a id="link" href="#" class="style-scope state-modifier" inherit="">
                      <iron-icon icon="find-in-page" class="style-scope patent-result x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M20 19.59V8l-6-6H6c-1.1 0-1.99.9-1.99 2L4 20c0 1.1.89 2 1.99 2H18c.45 0 .85-.15 1.19-.4l-4.43-4.43c-.8.52-1.74.83-2.76.83-2.76 0-5-2.24-5-5s2.24-5 5-5 5 2.24 5 5c0 1.02-.31 1.96-.83 2.75L20 19.59zM9 13c0 1.66 1.34 3 3 3s3-1.34 3-3-1.34-3-3-3-3 1.34-3 3z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon> Find Prior Art
                    </a></state-modifier>
                    <state-modifier inherit="" act="{&quot;type&quot;: &quot;QUERY_SIMILAR_DOCUMENTS&quot;, &quot;id&quot;: &quot;$id&quot;}" class="style-scope patent-result" data-id="patent/US5874563A"><a id="link" href="#" class="style-scope state-modifier" inherit="">
                      <iron-icon icon="editor:functions" class="style-scope patent-result x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M18 4H6v2l6.5 6L6 18v2h12v-3h-7l5-5-5-5h7z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon> Similar
                    </a></state-modifier>
                  </div>
                </header>

                <dl class="important-people style-scope patent-result">
                  <template is="dom-if" class="style-scope patent-result"></template>
                  
                  <dt class="style-scope patent-result">Inventor</dt>
                  <dd class="style-scope patent-result"><state-modifier act="{&quot;type&quot;: &quot;QUERY_SEARCH_INVENTOR&quot;, &quot;inventor&quot;: &quot;$inventor&quot;}" class="style-scope patent-result" data-inventor="Jungsuh P. Kim"><a id="link" href="#" class="style-scope state-modifier">Jungsuh P. Kim</a></state-modifier></dd><dd class="style-scope patent-result"><state-modifier act="{&quot;type&quot;: &quot;QUERY_SEARCH_INVENTOR&quot;, &quot;inventor&quot;: &quot;$inventor&quot;}" class="style-scope patent-result" data-inventor="Kirk E. Fry"><a id="link" href="#" class="style-scope state-modifier">Kirk E. Fry</a></state-modifier></dd><dd class="style-scope patent-result"><state-modifier act="{&quot;type&quot;: &quot;QUERY_SEARCH_INVENTOR&quot;, &quot;inventor&quot;: &quot;$inventor&quot;}" class="style-scope patent-result" data-inventor="LaVonne Marie Young"><a id="link" href="#" class="style-scope state-modifier">LaVonne Marie Young</a></state-modifier></dd><dd class="style-scope patent-result"><state-modifier act="{&quot;type&quot;: &quot;QUERY_SEARCH_INVENTOR&quot;, &quot;inventor&quot;: &quot;$inventor&quot;}" class="style-scope patent-result" data-inventor="Jeffrey M. Linnen"><a id="link" href="#" class="style-scope state-modifier">Jeffrey M. Linnen</a></state-modifier></dd><dd class="style-scope patent-result"><state-modifier act="{&quot;type&quot;: &quot;QUERY_SEARCH_INVENTOR&quot;, &quot;inventor&quot;: &quot;$inventor&quot;}" class="style-scope patent-result" data-inventor="John Wages"><a id="link" href="#" class="style-scope state-modifier">John Wages</a></state-modifier></dd><template is="dom-repeat" strip-whitespace="" class="style-scope patent-result"></template>
                  <template is="dom-if" class="style-scope patent-result"></template>

                  
                  <dt class="style-scope patent-result">
                    <span class="tooltip-hint style-scope patent-result" id="assigneeWarning">Current Assignee</span>
                    <overlay-tooltip positioning-target="#assigneeWarning" direction="bottom" class="style-scope patent-result" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      
                      The listed assignees may be inaccurate.<br class="style-scope patent-result">
                      Google has not performed a legal analysis and makes no representation or warranty as to the accuracy of the list.
                    
    </div>
  </overlay-tooltip>
                  </dt>
                  <dd class="style-scope patent-result">
                    
    Genelabs Technologies Inc
  
                  </dd><template is="dom-repeat" strip-whitespace="" class="style-scope patent-result"></template>
                  <template is="dom-if" class="style-scope patent-result"></template>
                </dl>

                <hr class="style-scope patent-result">

                <application-timeline class="style-scope patent-result">
    
    
    <div class="wrap style-scope application-timeline">
      <div class="header style-scope application-timeline">Worldwide applications</div>
      <div class="event style-scope application-timeline">
        
          <span class="block style-scope application-timeline">
            <span class="year style-scope application-timeline">1995</span>
            <span class="dot style-scope application-timeline"></span>
            
              <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope application-timeline" data-result="patent/US5874563A/en"><a id="link" href="#" class="style-scope state-modifier">
                <span id="cc" class="not_active style-scope application-timeline" this="">US</span>
              </a></state-modifier>
              <overlay-tooltip positioning-target="#cc" direction="bottom" class="style-scope application-timeline" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      
                <div class="style-scope application-timeline">Application number: US08/485,910</div>
                <div class="style-scope application-timeline">Filing date: 1995-06-05</div>
                <div class="style-scope application-timeline">Legal status: Expired - Fee Related</div>
              
    </div>
  </overlay-tooltip>
            <template is="dom-repeat" class="style-scope application-timeline"></template>
          </span>
        <template is="dom-repeat" class="style-scope application-timeline"></template>
      </div>
      <hr class="style-scope application-timeline">
      <div class="header style-scope application-timeline">Application US08/485,910 events <iron-icon id="legal" icon="icons:help-outline" class="style-scope application-timeline x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M11 18h2v-2h-2v2zm1-16C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8zm0-14c-2.21 0-4 1.79-4 4h2c0-1.1.9-2 2-2s2 .9 2 2c0 2-3 1.75-3 5h2c0-2.25 3-2.5 3-5 0-2.21-1.79-4-4-4z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
        <overlay-tooltip positioning-target="#legal" direction="right" class="style-scope application-timeline" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      
          <div style="max-width: 300px;" class="style-scope application-timeline">
            A timeline of key events for this patent application, including priority claims, publications, legal status, reassignments, and litigation.<br class="style-scope application-timeline"><br class="style-scope application-timeline">
            Google has not performed a legal analysis and makes no representation as to the accuracy or completeness of the events listed.
          </div>
        
    </div>
  </overlay-tooltip>
      </div>
      <div class="event layout horizontal style-scope application-timeline"><div date="" class="external-priority style-scope application-timeline">1994-12-16</div><div class="line style-scope application-timeline"></div><div class="flex title style-scope application-timeline"><template is="dom-if" class="style-scope application-timeline"></template><state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope application-timeline" data-result="patent/US5546335A/en"><a id="link" href="#" class="style-scope state-modifier"><span class="title-text style-scope application-timeline">Priority claimed from US08/357,009</span></a></state-modifier><template is="dom-if" class="style-scope application-timeline"></template><template is="dom-if" class="style-scope application-timeline"></template><template is="dom-if" class="style-scope application-timeline"></template><template is="dom-if" class="style-scope application-timeline"></template></div></div><div class="event layout horizontal style-scope application-timeline"><div date="" class="external-priority style-scope application-timeline">1995-05-19</div><div class="line style-scope application-timeline"></div><div class="flex title style-scope application-timeline"><template is="dom-if" class="style-scope application-timeline"></template><state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope application-timeline" data-result="patent/US5824507A/en"><a id="link" href="#" class="style-scope state-modifier"><span class="title-text style-scope application-timeline">Priority claimed from US08/444,733</span></a></state-modifier><template is="dom-if" class="style-scope application-timeline"></template><template is="dom-if" class="style-scope application-timeline"></template><template is="dom-if" class="style-scope application-timeline"></template><template is="dom-if" class="style-scope application-timeline"></template></div></div><div class="event layout horizontal style-scope application-timeline" critical=""><div date="" class="filed style-scope application-timeline">1995-06-05</div><div class="line style-scope application-timeline"></div><div class="flex title style-scope application-timeline"><template is="dom-if" class="style-scope application-timeline"></template><template is="dom-if" class="style-scope application-timeline"></template><state-modifier act="{&quot;type&quot;: &quot;QUERY_SEARCH_ASSIGNEE&quot;, &quot;assignee&quot;: &quot;$assignee&quot;}" class="style-scope application-timeline" data-assignee="Genelabs Technologies Inc"><a id="link" href="#" class="style-scope state-modifier"><span class="title-text style-scope application-timeline">Application filed by Genelabs Technologies Inc</span></a></state-modifier><template is="dom-if" class="style-scope application-timeline"></template><template is="dom-if" class="style-scope application-timeline"></template><template is="dom-if" class="style-scope application-timeline"></template></div></div><div class="event layout horizontal style-scope application-timeline" critical=""><div date="" class="priority style-scope application-timeline">1995-06-05</div><div class="line style-scope application-timeline"></div><div class="flex title style-scope application-timeline"><template is="dom-if" class="style-scope application-timeline"></template><state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope application-timeline" data-result="patent/US5874563A/en"><a id="link" href="#" class="style-scope state-modifier"><span class="title-text style-scope application-timeline">Priority to US08/485,910</span></a></state-modifier><template is="dom-if" class="style-scope application-timeline"></template><template is="dom-if" class="style-scope application-timeline"></template><template is="dom-if" class="style-scope application-timeline"></template><template is="dom-if" class="style-scope application-timeline"></template></div></div><div class="event layout horizontal style-scope application-timeline" critical=""><div date="" class="granted style-scope application-timeline">1999-02-23</div><div class="line style-scope application-timeline"></div><div class="flex title style-scope application-timeline"><template is="dom-if" class="style-scope application-timeline"></template><template is="dom-if" class="style-scope application-timeline"></template><template is="dom-if" class="style-scope application-timeline"></template><span class="title-text style-scope application-timeline">Application granted</span><template is="dom-if" class="style-scope application-timeline"></template><template is="dom-if" class="style-scope application-timeline"></template></div></div><div class="event layout horizontal style-scope application-timeline" critical=""><div date="" class="publication style-scope application-timeline">1999-02-23</div><div class="line style-scope application-timeline"></div><div class="flex title style-scope application-timeline"><template is="dom-if" class="style-scope application-timeline"></template><state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope application-timeline" data-result="patent/US5874563A/en"><a id="link" href="#" class="style-scope state-modifier"><span class="title-text style-scope application-timeline">Publication of US5874563A</span></a></state-modifier><template is="dom-if" class="style-scope application-timeline"></template><template is="dom-if" class="style-scope application-timeline"></template><template is="dom-if" class="style-scope application-timeline"></template><template is="dom-if" class="style-scope application-timeline"></template></div></div><div class="event layout horizontal style-scope application-timeline" critical=""><div date="" class="legal-status style-scope application-timeline">2016-02-23</div><div class="line style-scope application-timeline"></div><div class="flex title style-scope application-timeline"><template is="dom-if" class="style-scope application-timeline"></template><template is="dom-if" class="style-scope application-timeline"></template><template is="dom-if" class="style-scope application-timeline"></template><span class="title-text style-scope application-timeline">Anticipated expiration</span><template is="dom-if" class="style-scope application-timeline"></template><template is="dom-if" class="style-scope application-timeline"></template></div></div><div class="event layout horizontal style-scope application-timeline" current="" critical=""><div date="" class="legal-status style-scope application-timeline">Status</div><div class="line style-scope application-timeline"></div><div class="flex title style-scope application-timeline"><template is="dom-if" class="style-scope application-timeline"></template><template is="dom-if" class="style-scope application-timeline"></template><template is="dom-if" class="style-scope application-timeline"></template><span class="title-text style-scope application-timeline">Expired - Fee Related</span><template is="dom-if" class="style-scope application-timeline"></template><template is="dom-if" class="style-scope application-timeline"></template></div></div><template is="dom-repeat" strip-whitespace="" class="style-scope application-timeline"></template>
      <template is="dom-if" class="style-scope application-timeline"></template>
    </div>
  </application-timeline>
                <hr class="style-scope patent-result">

                <dl class="links style-scope patent-result">
                  <dt class="style-scope patent-result">Info</dt>

                  
                  <dd class="style-scope patent-result"><a href="#patentCitations" class="style-scope patent-result">Patent citations (15)</a></dd>
                  <template is="dom-if" restamp="" class="style-scope patent-result"></template>

                  
                  <dd class="style-scope patent-result"><a href="#nplCitations" class="style-scope patent-result">Non-patent citations (30)</a></dd>
                  <template is="dom-if" restamp="" class="style-scope patent-result"></template>

                  
                  <dd class="style-scope patent-result"><a href="#citedBy" class="style-scope patent-result">Cited by (9)</a></dd>
                  <template is="dom-if" restamp="" class="style-scope patent-result"></template>

                  
                  <dd class="style-scope patent-result"><a href="#legalEvents" class="style-scope patent-result">Legal events</a></dd>
                  <template is="dom-if" restamp="" class="style-scope patent-result"></template>

                  
                  <dd class="style-scope patent-result"><a href="#similarDocuments" class="style-scope patent-result">Similar documents</a></dd>
                  <template is="dom-if" restamp="" class="style-scope patent-result"></template>

                  
                  <dd class="style-scope patent-result"><a href="#relatedApplications" class="style-scope patent-result">Priority and Related Applications</a></dd>
                  <template is="dom-if" restamp="" class="style-scope patent-result"></template>

                  <dt class="style-scope patent-result">External links</dt>

                  <dd class="style-scope patent-result"><a target="_blank" class="style-scope patent-result" href="https://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO1&amp;Sect2=HITOFF&amp;p=1&amp;u=/netahtml/PTO/srchnum.html&amp;r=1&amp;f=G&amp;l=50&amp;d=PALL&amp;s1=5874563.PN.">USPTO</a></dd><dd class="style-scope patent-result"><a target="_blank" class="style-scope patent-result" href="https://patentcenter.uspto.gov/applications/08485910">USPTO PatentCenter</a></dd><dd class="style-scope patent-result"><a target="_blank" class="style-scope patent-result" href="https://assignment.uspto.gov/patent/index.html#/patent/search/resultFilter?searchInput=5874563">USPTO Assignment</a></dd><dd class="style-scope patent-result"><a target="_blank" class="style-scope patent-result" href="https://worldwide.espacenet.com/publicationDetails/biblio?CC=US&amp;NR=5874563A&amp;KC=A&amp;FT=D">Espacenet</a></dd><dd class="style-scope patent-result"><a target="_blank" class="style-scope patent-result" href="https://globaldossier.uspto.gov/#/result/patent/US/5874563/1">Global Dossier</a></dd><dd class="style-scope patent-result"><a target="_blank" class="style-scope patent-result" href="https://patents.stackexchange.com/questions/tagged/US5874563">Discuss</a></dd><template is="dom-repeat" strip-whitespace="" class="style-scope patent-result"></template>
                </dl>
              </section>
            </div>
          </div>

          <div id="text" class="horizontal layout style-scope patent-result">
            <div class="flex flex-width style-scope patent-result" marginright="">
              <section id="description" class="style-scope patent-result">
                <h3 class="layout horizontal style-scope patent-result">
                  <div class="flex style-scope patent-result">Description</div>
                  <div class="righthead style-scope patent-result" hidden="">translated from </div>
                </h3>
                <patent-text name="description" id="descriptionText" class="style-scope patent-result" style="padding-top: 0px;">
    
    
    <template is="dom-if" restamp="" class="style-scope patent-text"></template>
    <overlay-tooltip id="tooltip" class="style-scope patent-text" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      
    </div>
  </overlay-tooltip>
    <div class="layout horizontal style-scope patent-text">
      <section id="text" class="flex style-scope patent-text"><div mxw-id="PDES317779073" lang="EN" load-source="patent-office" class="description style-scope patent-text">
    
    <div class="description-paragraph style-scope patent-text">This application is a divisional of U.S. application Ser. No. 08/444,733, filed 19 May 1995, which is a continuation-in-part of U.S. application Ser. No. 08/344,271, filed 23 Nov. 1994, herein incorporated by reference, and is a continuation-in-part of U.S. Pat. application Ser. No. 08/389,886, filed 15 Feb. 1995, herein incorporated by reference, which is a continuation-in-part of 08/357,509, filed 16 Dec. 1994, herein incorporated by reference, which is a continuation-in-part of U.S. patent application Ser. No. 08/329,729, filed 26 Oct. 1994, herein incorporated by reference, which is a continuation-in-part of U.S. patent application Ser. No. 08/285,558, filed 3 Aug. 1994, and of U.S. patent application Ser. No. 08/285,543, filed 3 Aug. 1994, herein incorporated by reference, which are continuation-in-part of U.S. patent application Ser. No. 08/246,985, filed 20 May 1994, herein incorporated by reference.</div>
    
    
    <heading class="style-scope patent-text">FIELD OF INVENTION</heading>
    <div class="description-paragraph style-scope patent-text">This invention relates to nucleic acid, polypeptide, antigen, epitope, vaccine and antibody compositions related to a NonA/NonB/NonC/NonD/NonE (N-(ABCDE)) hepatitis-associated viral agent (HGV). The invention also relates to diagnostic and therapeutic methods.</div>
    <heading class="style-scope patent-text">REFERENCES</heading>
    <div class="description-paragraph style-scope patent-text">Abstracts, The 1992 San Diego Conf.: Genetic Recognition, Clin. Chem. 39(4):705 (1993).</div>
    <div class="description-paragraph style-scope patent-text">Alexander, W. A., et al., J. Virol. 66:2934-2942 (1992).</div>
    <div class="description-paragraph style-scope patent-text">Alter, H. J., et al., New Eng. J. Med. 321:1494-1500 (1989a).</div>
    <div class="description-paragraph style-scope patent-text">Alter. M. J., et al., N. Engl. J. Med. 327:1899 (1989b).</div>
    <div class="description-paragraph style-scope patent-text">Alter, H. J., Abstracts of Int. Symp. on Viral Hepatitis and Liver Dis., p. 47 (1993).</div>
    <div class="description-paragraph style-scope patent-text">Altschul, S., et al., J. Mol. Biol. 215:403-10 (1990).</div>
    <div class="description-paragraph style-scope patent-text">Ascadi, G., et al., Nature 352:815 (1991).</div>
    <div class="description-paragraph style-scope patent-text">Ausubel, F. M., et al., Current Protocols in Molecular Biology, John Wiley and Sons, Inc., Media Pa.</div>
    <div class="description-paragraph style-scope patent-text">Barany, F., PCR Methods Appl. 1:5 (1991).</div>
    <div class="description-paragraph style-scope patent-text">Barham, W. B., et al., J. Med. Virol. 42:129-132 (1994).</div>
    <div class="description-paragraph style-scope patent-text">Baron, S., et al., JAMA 266:1375 (1991).</div>
    <div class="description-paragraph style-scope patent-text">Bazan, J. F., et al., Virology 171:637-639 (1989).</div>
    <div class="description-paragraph style-scope patent-text">Beames, et al., Biotechniques 11:378 (1991).</div>
    <div class="description-paragraph style-scope patent-text">Belyavsky, A., et al., Nuc. Acids Res. 17:2919-2932 (1989).</div>
    <div class="description-paragraph style-scope patent-text">Blackburn, G. F., et al., Clin. Chem. 37:1534-1539 (1991).</div>
    <div class="description-paragraph style-scope patent-text">Bradley, D. W., et al., J. Infec. Dis., 148:2 (1983).</div>
    <div class="description-paragraph style-scope patent-text">Bradley, D. W., et al., J Gen. Virol., 69:1 (1988).</div>
    <div class="description-paragraph style-scope patent-text">Bradley, D. W. et al., Proc. Nat. Acad. Sci., USA, 84:6277 (1987).</div>
    <div class="description-paragraph style-scope patent-text">Briand, J. -P., et al., J. Immunol. Meth. 156:255 (1992).</div>
    <div class="description-paragraph style-scope patent-text">Cahill, P., et al., Clin. Chem. 37:1482 (1991).</div>
    <div class="description-paragraph style-scope patent-text">Carter, J. M., et al., Methods Mol. Biol. 36:207-223 (1994).</div>
    <div class="description-paragraph style-scope patent-text">Chambers, T. J., et al., Ann. Rev. Microbiol. 44:649 (1990a).</div>
    <div class="description-paragraph style-scope patent-text">Chambers, T. J., et al., PNAS 87:8898 (1990b).</div>
    <div class="description-paragraph style-scope patent-text">Chomczynski et al, Anal. Biochem. 162:159 (1987).</div>
    <div class="description-paragraph style-scope patent-text">Christian, R. B., et al., J. Mol. Biol. 227:771 (1992).</div>
    <div class="description-paragraph style-scope patent-text">Commandaeur, et al., Virology 198:282-287 (1994).</div>
    <div class="description-paragraph style-scope patent-text">Crea, R., U.S. Pat. No. 4,888,286, issued Dec. 19, 1989.</div>
    <div class="description-paragraph style-scope patent-text">DeGraaf, M. E., et al., Gene 128:13 (1993).</div>
    <div class="description-paragraph style-scope patent-text">DiBisceglie, A. M., et al., Hepatology 16:649 (1992).</div>
    <div class="description-paragraph style-scope patent-text">DiBisceglie, A. M., et al., NEJM 321:1506 (1989).</div>
    <div class="description-paragraph style-scope patent-text">DiCesare, J., et al., Biotechniques 15:152-157 (1993).</div>
    <div class="description-paragraph style-scope patent-text">Dienstag, J. L., et al, Sem Liver Disease 6:67 (1986).</div>
    <div class="description-paragraph style-scope patent-text">Earl, P. L., et al., "Expression of proteins in mammalian cells using vaccinia" In Current Protocols in Molecular Biology (F. M. Ausubel, et al. Eds.), Greene Publishing Associates &amp; Wiley Interscience, New York (1991).</div>
    <div class="description-paragraph style-scope patent-text">Eaton, M. A. W., et al., U.S. Pat. No. 4,719,180, issued Jan. 12, 1988.</div>
    <div class="description-paragraph style-scope patent-text">Egholm, et al., Nature 365:566 (1993).</div>
    <div class="description-paragraph style-scope patent-text">Elroy-Stein, O., et al., Proc. Natl. Acad. Sci. USA. 86:6126-6130 (1989).</div>
    <div class="description-paragraph style-scope patent-text">EPO patent application 88310922.5, filed Nov. 18, 1988.</div>
    <div class="description-paragraph style-scope patent-text">Falkner, F. G., et al., J. Virol. 62:1849-1854 (1988).</div>
    <div class="description-paragraph style-scope patent-text">Farci, P., et al., NEJM 330:88 (1994).</div>
    <div class="description-paragraph style-scope patent-text">Feigner and Rhodes, Nature 349:251 (1991).</div>
    <div class="description-paragraph style-scope patent-text">Fickett, J. W., Nuc. Acids Res. 10:5303-5318 (1982).</div>
    <div class="description-paragraph style-scope patent-text">Fling, S. P., et al., Analytical Biochem. 155:83-88 (1986).</div>
    <div class="description-paragraph style-scope patent-text">Folgori, A., et al., EMBO J. 13:2236 (1994).</div>
    <div class="description-paragraph style-scope patent-text">Francki, R. I. B., et al., Arch. Virol. Supl2:223 (1991).</div>
    <div class="description-paragraph style-scope patent-text">Frank, R., and Doring, R., Tetrahedron 44:6031-6040 (1988).</div>
    <div class="description-paragraph style-scope patent-text">Frohman, M. A., et al., Proc. Natl. Acad. Sci. USA 85:8998-9002 (1988).</div>
    <div class="description-paragraph style-scope patent-text">Fuerst, T. R., et al., Proc. Natl. Acad. Sci. USA 83:8122-8126 (1986).</div>
    <div class="description-paragraph style-scope patent-text">Gellissen, G., et al., Antonie Van Leeuwenhoek, 62(1-2):79-93 (1992).</div>
    <div class="description-paragraph style-scope patent-text">Geysen, M., et al., Proc. Natl. Acad. Sci. USA 81:3998-4002 (1984).</div>
    <div class="description-paragraph style-scope patent-text">Gingeras, T. R., et al., Ann. Biol. Clin. 48:498 (1990).</div>
    <div class="description-paragraph style-scope patent-text">Gingeras, T. R., et al., J. Inf. Dis. 164:1066 (1991).</div>
    <div class="description-paragraph style-scope patent-text">Goeddel, D. V., Methods in Enzymology 185 (1990).</div>
    <div class="description-paragraph style-scope patent-text">Grakoui, A., et al., J. Virol. 67:2832 (1993).</div>
    <div class="description-paragraph style-scope patent-text">Grakoui, A., et al., J. Virol. 67:1385-1395 (1993).</div>
    <div class="description-paragraph style-scope patent-text">Guatelli, J. C., et al., Proc. Natl. Acad. Sci. USA 87:1874 (1990).</div>
    <div class="description-paragraph style-scope patent-text">Gubler, U., et al, Gene, 25:263 (1983).</div>
    <div class="description-paragraph style-scope patent-text">Guthrie, C., and G. R. Fink, Methods in Enzymology 194 (1991).</div>
    <div class="description-paragraph style-scope patent-text">Gutterman, J. U., PNAS 91:1198 (1994).</div>
    <div class="description-paragraph style-scope patent-text">Harlow, E., et al., Antibodies: A Laboratory Manual, Cold Spring Harbor Laboratory Press (1988).</div>
    <div class="description-paragraph style-scope patent-text">Haynes, J., et al., Nuc. Acid. Res. 11:687-706 (1983).</div>
    <div class="description-paragraph style-scope patent-text">Hieter, P. A., et al., Cell 22:197-207 (1980).</div>
    <div class="description-paragraph style-scope patent-text">Hijikata, M., et al., PNAS 88:5547 (1991).</div>
    <div class="description-paragraph style-scope patent-text">Hochuli, E., in Genetic Engineering. Principals and Practice, Vol. 12 (J. Stelow Ed.) Plenum, N.Y., pp. 87-98 (1990).</div>
    <div class="description-paragraph style-scope patent-text">Holodniy, M., et al., Biotechniques 12:36 (1992).</div>
    <div class="description-paragraph style-scope patent-text">Hopp, T. P., et al., Proc. Natl. Acad. Sci. USA 78:3824-3828 (1981).</div>
    <div class="description-paragraph style-scope patent-text">Horn, T., and Urdea, M. S., Nuc. Acids. Res. 17:6959 (1989).</div>
    <div class="description-paragraph style-scope patent-text">Houghten, R. A., Proc. Natl. Acad. Sci. USA 82:5131 (1985).</div>
    <div class="description-paragraph style-scope patent-text">Hudson, D., J. Org. Chem. 53:617 (1988).</div>
    <div class="description-paragraph style-scope patent-text">Irwin, M. J., et al., J. Virol. 58:5036 (1994).</div>
    <div class="description-paragraph style-scope patent-text">Jacob, J. R. , et al., in The Molecular Biology of HCV, <figure-callout id="4" label="Section" filenames="US5874563-drawings-page-10.png,US5874563-drawings-page-12.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">Section</a></figure-callout> 4, pages 387-392 (1991).</div>
    <div class="description-paragraph style-scope patent-text">Jacob, J. R., et al., Hepatology 10:921-927 (1989).</div>
    <div class="description-paragraph style-scope patent-text">Jacob, J. R., et al., J. Infect. Dis. 161:1121-1127 (1990).</div>
    <div class="description-paragraph style-scope patent-text">Janknecht, R., et al., Proc. Natl. Acad. Sci. USA 88:8972-8976 (1991).</div>
    <div class="description-paragraph style-scope patent-text">Kaufman, R. J., "Selection and coamplification of heterologous genes in mammalian cells," in Methods in Enzymology, vol. 185, pp537-566. Academic Press, Inc., San Diego Calif. (1991).</div>
    <div class="description-paragraph style-scope patent-text">Kakumu, S., et al., Gastroenterol. 105:507 (1993).</div>
    <div class="description-paragraph style-scope patent-text">Katz, E. D., and Dong, M., Biotechniques 8:546 (1990).</div>
    <div class="description-paragraph style-scope patent-text">Kawasaki, E. S., et al., in PCR Technology: Principles and Applications of DNA Amplification (H. A. Erlich, ed.) Stockton Press (1989).</div>
    <div class="description-paragraph style-scope patent-text">King, L. A., et al., The baculovirus expression system. A laboratory guide, Chapman &amp; Hall, London, New York, Tokyo, Melbourne, Madras, 1992.</div>
    <div class="description-paragraph style-scope patent-text">Kyte, J., &amp; Doolittle, R. F., J. Mol. Biol. 157:105-132 (1982).</div>
    <div class="description-paragraph style-scope patent-text">Koonin, E. V., and Dolja, V. V., Critical Reviews in Biochem. &amp; Mol. Biol. 28:375-430 (1993).</div>
    <div class="description-paragraph style-scope patent-text">Krausslich, H. G., et al., Viral Proteinases as Targets for Chemotherapy (Cold Spring Harbor Press, Plainville, N.Y.) (1989).</div>
    <div class="description-paragraph style-scope patent-text">Kumar, R., et al., AIDS Res. Human Retroviruses 5(3) :345-354 (1989).</div>
    <div class="description-paragraph style-scope patent-text">Lanford, R. E., et al., In Vitro Cell. Dev. Biol. 25:174-182 (1989).</div>
    <div class="description-paragraph style-scope patent-text">Larder, B. A., and Kemp, S. D., Science 246:1155 (1989).</div>
    <div class="description-paragraph style-scope patent-text">Lau, Y. F., et al., Mol. Cell. Biol. 4:1469-1475 (1984).</div>
    <div class="description-paragraph style-scope patent-text">Lomell, H., et al., Clin. Chem. 48:492 (1990).</div>
    <div class="description-paragraph style-scope patent-text">Maniatis, T., et al., Molecular Cloning: A Laboratory Manual, Cold Spring Harbor Laboratory (1982).</div>
    <div class="description-paragraph style-scope patent-text">Marshall, W. S., and Caruthers, M. H., Science 259:1564 (1993).</div>
    <div class="description-paragraph style-scope patent-text">Messing, J., Methods in Enzymol. 101:20 (1983).</div>
    <div class="description-paragraph style-scope patent-text">Michelle, et al., International Symposium on Viral Hepatitis.</div>
    <div class="description-paragraph style-scope patent-text">Miller, J. H., Experiments in Molecul Genetics, Cold Spring Harbor Laboratories, Cold Spring Harbor, N.Y. (1972).</div>
    <div class="description-paragraph style-scope patent-text">Morrissey, D. V., et al., Anal. Biochem. 181:345 (1989).</div>
    <div class="description-paragraph style-scope patent-text">Moss, B., et al., Current Protocols in Molecular Biology (Section IV, Unit 16) (1991).</div>
    <div class="description-paragraph style-scope patent-text">Moss, B., et al., U.S. Pat. No. 5,135,855, issued 4 Aug., 1992.</div>
    <div class="description-paragraph style-scope patent-text">Mullis, K. B., U.S. Pat. No. 4,683,202, issued 28 Jul., 1987.</div>
    <div class="description-paragraph style-scope patent-text">Mullis, K. B., et al., U.S. Pat. No. 4,683,195, issued 28 Jul. 1987.</div>
    <div class="description-paragraph style-scope patent-text">Obeid, O. E., et al., Virus Research 32:69-84 (1994).</div>
    <div class="description-paragraph style-scope patent-text">Osikowicz, G., et al., Clin. Chem. 36:1586 (1990).</div>
    <div class="description-paragraph style-scope patent-text">Patterson, J. L., and Fernandez-Larsson, R., Rev. Infect. Dis. 12:1139 (1990).</div>
    <div class="description-paragraph style-scope patent-text">Pearson, W. R. and Lipman, D. J., PNAS 85:2444-2448 (1988).</div>
    <div class="description-paragraph style-scope patent-text">Pearson, W. R., Methods in Enzymology 183:63-98 (1990).</div>
    <div class="description-paragraph style-scope patent-text">Pitha, Biochem Biophys Acta, 204:39 (1970a).</div>
    <div class="description-paragraph style-scope patent-text">Pitha, Biopolymers, 9:965 (1970b).</div>
    <div class="description-paragraph style-scope patent-text">Porath, J., Protein Exp. and Purif. 3:263 (1992).</div>
    <div class="description-paragraph style-scope patent-text">Pritchard, C. G., and Stefano, J. E., Ann. Biol. Chem. 48:492 (1990).</div>
    <div class="description-paragraph style-scope patent-text">Reichard, O., et al., Lancet 337:1058 (1991).</div>
    <div class="description-paragraph style-scope patent-text">Reilly, P. R., et al., Baculovirus Expressint Vectors: A Laboratory Manual (1992).</div>
    <div class="description-paragraph style-scope patent-text">Reyes, G., et al, Science, 247:1335 (1990).</div>
    <div class="description-paragraph style-scope patent-text">Reyes, G., et al., Molecular and Cellular Probes 5:473-481 (1991).</div>
    <div class="description-paragraph style-scope patent-text">Rice, C. M., et al., New Biol. 1:285-296 (1989).</div>
    <div class="description-paragraph style-scope patent-text">Roberts, N. A., et al., Science 248:358 (1990).</div>
    <div class="description-paragraph style-scope patent-text">Romanos, M. A., et al., Yeast 8(6):423-488 (1992).</div>
    <div class="description-paragraph style-scope patent-text">Sanger, et al., Proc. Natl. Acad. Sci. 74:5463 (1977).</div>
    <div class="description-paragraph style-scope patent-text">Sambrook, J., et al., In Molecular Cloning: A Laboratory Manual, Cold Spring Harbor Laboratory Press, Vol. 2 (1989).</div>
    <div class="description-paragraph style-scope patent-text">Saiki, R. K., et al., Science 239:487-491 (1988).</div>
    <div class="description-paragraph style-scope patent-text">Schagger, H., et al., Anal. Biochem. 166:368-379 (1987).</div>
    <div class="description-paragraph style-scope patent-text">Scharf, S. J., et al., Science 233:1076 (1986).</div>
    <div class="description-paragraph style-scope patent-text">Schuler, G. D., et al., Proteins: Struc., Func. and Genet. 9:180 (1989).</div>
    <div class="description-paragraph style-scope patent-text">Scott, J. K., and Smith, G. P., Science 249:386 (1990).</div>
    <div class="description-paragraph style-scope patent-text">Scott, J. K., et al., Proc. Natl. Acad. Sci. USA 89:5398 (1992).</div>
    <div class="description-paragraph style-scope patent-text">Smith, D. B., et al., Gene 67:31 (1988).</div>
    <div class="description-paragraph style-scope patent-text">Smith, J. P., Curr. Opin. Biotechnol. 2:668 (1991).</div>
    <div class="description-paragraph style-scope patent-text">Sreenivasan, M. A., et al., J. Gen. Virol. 65:1005 (1984).</div>
    <div class="description-paragraph style-scope patent-text">Sumiyoshi, H., et al., J. Virol. 66:5425-5431 (1992).</div>
    <div class="description-paragraph style-scope patent-text">Summerton, J., et al., U.S. Pat. No. 5,142,047, issued Aug. 25, 1992.</div>
    <div class="description-paragraph style-scope patent-text">Summerton, J., et al., U.S. Pat. No. 5,185,444 issued Feb. 9, 1993.</div>
    <div class="description-paragraph style-scope patent-text">Tam, A., et al., Virology 185:120 (1991).</div>
    <div class="description-paragraph style-scope patent-text">Tam, J. P., Proc. Natl. Acad. Sci. USA 85:5409 (1988).</div>
    <div class="description-paragraph style-scope patent-text">Tessier, D. C., Gene 98:177-183 (1991).</div>
    <div class="description-paragraph style-scope patent-text">Tonkinson, J. L., and Stein, C. A., Antiviral Chem. and Chemother. 4(4):193-200 (1993).</div>
    <div class="description-paragraph style-scope patent-text">Ulmer, et al., Science 259:1745 (1993).</div>
    <div class="description-paragraph style-scope patent-text">Urdea, M., Clin. Chem. 39:725 (1993).</div>
    <div class="description-paragraph style-scope patent-text">Urdea, M., et al., AIDS 7:Sll (1993).</div>
    <div class="description-paragraph style-scope patent-text">Wages, J. M., et al., Amplifications 10:1-6 (1993).</div>
    <div class="description-paragraph style-scope patent-text">Walker, G. T., PCR Methods Appl. 3:1-6 (1993).</div>
    <div class="description-paragraph style-scope patent-text">Wang, A. M., et al. in PCR Protocols: A Guide to Methods and Applications (M. A. Innis, et al., eds.) Academic Press (1990).</div>
    <div class="description-paragraph style-scope patent-text">Wang, B., et al., Proc. Natl. Acad. Sci. USA 90:4156 (1993).</div>
    <div class="description-paragraph style-scope patent-text">Whetsell, A. J., et al., J. Clin. Micro. 30:845 (1992).</div>
    <div class="description-paragraph style-scope patent-text">Wolf, J. A., et al., Nature 247:1465 (1990).</div>
    <div class="description-paragraph style-scope patent-text">Vacca, J. P., et al., PNAS 91:4096 (1994).</div>
    <div class="description-paragraph style-scope patent-text">VanGemen, B., et al., J. Virol. Methods 43:177 (1993).</div>
    <div class="description-paragraph style-scope patent-text">Valenzuela, P., et al., Nature 298:344 (1982).</div>
    <div class="description-paragraph style-scope patent-text">Valenzuela, P., et al., in Hepatitis B, eds. I. Millman, et al., Plenum Press, pages 225-236 (1984).</div>
    <div class="description-paragraph style-scope patent-text">Yarbrough, et al., J. Virol. 65:5790 (1991).</div>
    <div class="description-paragraph style-scope patent-text">Yoo, B. J., et al., J. Virol. 69:32-38 (1995).</div>
    <div class="description-paragraph style-scope patent-text">Yoshio, T., et al., U.S. Pat. No. 4,849,350, issued Jul. 18, 1989.</div>
    <div class="description-paragraph style-scope patent-text">Zhang, Y., et al., J. Virol. 65:6101-6110 (1991).</div>
    <heading class="style-scope patent-text">BACKGROUND OF THE INVENTION</heading>
    <div class="description-paragraph style-scope patent-text">Viral hepatitis resulting from a virus other than hepatitis A virus (HAV) and hepatitis B virus (HBV) has been referred to as non-A, non-B hepatitis (NANBH). NANBH can be further defined based on the mode of transmission of an individual type, for example, enteric versus parenteral.</div>
    <div class="description-paragraph style-scope patent-text">One form of NANBH, known as enterically transmitted NANBH or ET-NANBH, is contracted predominantly in poor-sanitation areas where food and drinking water have been contaminated by fecal matter. The molecular cloning of the causative agent, referred to as the hepatitis E virus (HEV), has recently been described (Reyes et al., 1990; Tam et al.).</div>
    <div class="description-paragraph style-scope patent-text">A second form of NANB, known as parenterally transmitted NANBH, or PT-NANBH, is transmitted by parenteral routes, typically by exposure to blood or blood products. The rate of this hepatitis varied by (i) locale, (ii) whether ALT testing was done in blood banks, and (iii) elimination of high-risk patients for AIDS. Appoximately 10% of transfusions caused PT-NANBH infection and about half of those went on to a chronic disease state (Dienstag). After implementation of anti-HCV testing, HCV seroconversion per unit transfused was decreased to less than 1% among heart surgery patients (Alter).</div>
    <div class="description-paragraph style-scope patent-text">Human plasma samples documented as having produced post-transfusion NANBH in human recipients have been used successfully to produce PT-NANBH infection in chimpanzees (Bradley). RNA isolated from infected chimpanzee plasma has been used to construct cDNA libraries in an expression vector for immunoscreening with serum from human subjects with chronic PT-NANBH infection. This procedure identified a PT-NANBH specific cDNA clone and the viral sequence was then used as a probe to identify a set of overlapping fragments making up 7,300 contiguous basepairs of a PT-NANBH viral agent. The sequenced viral agent has been named the hepatitis C virus (HCV) (for example, the sequence of HCV is presented in EPO patent application 88310922.5, filed Nov. 18, 1988). The full-length sequence (˜9,500 nt) of HCV is now available.</div>
    <div class="description-paragraph style-scope patent-text">Primate transmission studies conducted at the Centers for Disease Control (CDC; Phoenix, Ariz., 1973-1975; 1978-1983) originally provided substantial evidence for the existence of multiple agents of non-A, non-B hepatitis (NANBH): the primary agents associated with the majority of cases of NANBH are now recognized to be HCV and HEV (see above), for PT-NANBH and ET-NANBH, respectively. Later epidemiologic studies conducted at the CDC (Atlanta, Ga., 1989-present) using both research (prototype) and commercial tests for anti-HCV antibody showed that approximately 20% of all community-acquired NANBH was also non-C. Further testing of these samples for the presence of HEV (Reyes, et al., WO A 9115603 (Genelabs Inc.) 17 Oct. 1991) have indicated that these cases of community-acquired non-A, non-B, non-C hepatitis were also non-E.</div>
    <div class="description-paragraph style-scope patent-text">Liver biopsy specimens, sera and plasma of Sentinel County patients (study of Drs. Miriam Alter and Kris Krawczynski) also showed that many bona fide cases of NANBH were also non-C hepatitis (serologically and by Reverse Transcriptase-Polymerase Chain Reaction (RT-PCR; Kawasaki, et al.; Wang, et al., 1990) negative for all markers of HCV infection) developed subsequently into chronic hepatitis with presentation of chronic persistent hepatitis (CPH) or chronic active hepatitis (CAH) consistent with a viral infection.</div>
    <heading class="style-scope patent-text">SUMMARY OF THE INVENTION</heading>
    <div class="description-paragraph style-scope patent-text">The invention pertains to the characterization and isolation of a newly discovered NonA/NonB/NonC/NonD/NonE (N-(ABCDE)) hepatitis-associated viral agent, herein designated Hepatitis G Virus (HGV). Disclosed here is a family of cDNA replicas of portions of the HGV genome. Also disclosed are methods for the isolation and characterization of further HGV sequences and sequences of HGV variants.</div>
    <div class="description-paragraph style-scope patent-text">The present invention includes HGV genomic polynucleotides, cDNAs thereto and complements thereof. With respect to polynucleotides, some aspects of the invention include: a purified Hepatitis G Virus genomic polynucleotide; HGV derived RNA and DNA polynucleotides; recombinant HGV polynucleotides; a recombinant polynucleotide making up a sequence derived from HGV or HGV variant CDNA or complementary sequences thereof; a recombinant polynucleotide encoding an epitope of HGV; a recombinant vector including any of the above recombinant polynucleotides, and a host cell transformed with any of these vectors. Another aspect of the invention is a polynucleotide probe for HGV and/or its variants.</div>
    <div class="description-paragraph style-scope patent-text">Current studies on the nature of the genome of HGV, utilizing sequence information to compare HGV to other viral sequences, suggest that HGV is a member of the Flaviviridae family of viruses.</div>
    <div class="description-paragraph style-scope patent-text">Portions of the HGV-derived cDNA sequences are effective as probes to isolate variants of the virus which occur naturally, or to determine the presence of virus in samples. These cDNAs also make available HGV-encoded polypeptide sequences, including HGV-specific polypeptide antigens. These coding sequences allow the production of polypeptides which are useful as reagents in diagnostic tests and/or as components of vaccines, or as standards. Further, it is possible to isolate and sequence other portions of the HGV genome by utilizing probes derived from these cDNAs, therefore giving rise to additional probes and polypeptides useful in the prophylactic, therapeutic and diagnosis applications.</div>
    <div class="description-paragraph style-scope patent-text">Other aspects of the invention include: a recombinant expression system which incorporates an open reading frame (ORF) derived from HGV cDNA or complements thereof, wherein the ORF is linked operably to a control sequence which is compatible with a desired host, a cell transformed with the recombinant expression system, and a polypeptide produced by the transformed cell.</div>
    <div class="description-paragraph style-scope patent-text">Yet another aspect of the invention are purified HGV particles; a preparation of polypeptides from the purified HGV; a purified HGV polypeptide; a purified HGV peptide; and a purified polypeptide which comprises an epitope immunologically identifiable with an epitope contained in HGV or an HGV variant.</div>
    <div class="description-paragraph style-scope patent-text">Included aspects of the invention are an HGV polypeptide; a recombinant polypeptide consisting of a sequence derived from a HGV genome, HGV cDNA or complements thereof; a recombinant polypeptide made of an HGV epitope; and a fusion polypeptide comprised of an HGV polypeptide.</div>
    <div class="description-paragraph style-scope patent-text">Both polyclonal and monoclonal antibodies directed against HGV epitopes contained within the polypeptide sequences are also useful as therapeutic agents, for diagnostic tests, for the isolation of the HGV agent from which these cDNAs derive, and for screening of antiviral agents.</div>
    <div class="description-paragraph style-scope patent-text">Also included in the invention are a purified preparation of polyclonal antibodies directed against an HGV epitope; and monoclonal antibodies directed against HGV epitopes.</div>
    <div class="description-paragraph style-scope patent-text">Some aspects of the invention pertaining to kits are those for: investigating samples for the presence of polynucleotides derived from HGV which comprise a polynucleotide probe including a nucleotide sequence from HGV of approximately 8 or more nucleotides, in an appropriate container; analyzing samples for the presence of antibodies directed against an HGV antigen made up of a polypeptide which contains an HGV epitope present in the HGV antigen, in a suitable container; and analyzing samples for the presence of HGV antigens made up of an anti-HGV antibody, in a suitable container.</div>
    <div class="description-paragraph style-scope patent-text">Still other aspects of the invention include a polypeptide comprised of an HGV epitope, which is attached to a solid substrate; and an antibody to an HGV epitope, which is attached to a solid substrate.</div>
    <div class="description-paragraph style-scope patent-text">Other aspects of the invention are: a technique for the production of an HGV polypeptide, which includes incubating host cells which are transformed with an expression vector, containing a sequence encoding an HGV polypeptide, under conditions which allow expression of said polypeptide; and a polypeptide which has been produced by this method (containing, for example, an HGV epitope).</div>
    <div class="description-paragraph style-scope patent-text">Also included in the invention are a method for the detection of HGV nucleic acids in samples comprising reacting nucleic acids of the sample with a probe for an HGV polynucleotide, under conditions allowing the creation of a polynucleotide duplex between the probe and the HGV nucleic acid from the sample; as well as detecting a polynucleotide duplex containing the probe. The invention includes the following hybridization based detection methods: reporter labeling; polymerase chain reaction; self-sustained sequence replication; ligase chain reaction; and strand displacement amplification. Further, detection methods include signal amplification (e.g., branch-chained DNA probes and the Q-beta replicase method).</div>
    <div class="description-paragraph style-scope patent-text">The invention also includes immunoassays, including an immunoassay for detecting HGV, comprising the incubation of a sample (which is suspected of being infected with HGV) with a probe antibody directed against an antigen/epitope of HGV, to be detected under conditions allowing the formation of an antigen-antibody complex; and detecting the antigen-antibody complex which contains the probe antibody. An immunoassay for the detection of antibodies which are directed against an HGV antigen comprising the incubation of a sample suspected of containing HGV with a probe polypeptide including an epitope of HGV, under conditions that allow the formation of an antibody-antigen complex; and distinguishing the antibody-antigen complex which contains the probe antigen.</div>
    <div class="description-paragraph style-scope patent-text">Also forming part of the invention are HGV vaccines, for the treatment and/or prevention of HGV infection, comprising an immunogenic peptide containing an HGV epitope, or an inactivated preparation of HGV, or a reduced preparation of HGV.</div>
    <div class="description-paragraph style-scope patent-text">In still another aspect, the invention includes a tissue culture grown cell, infected with HGV. In one embodiment, the tissue culture grown cells are primate liver cells.</div>
    <div class="description-paragraph style-scope patent-text">Another aspect of the invention is a method for producing antibodies to HGV, comprising administering to a test subject an immunogenic polypeptide containing HGV epitopes in an adequate amount to elicit an immune response.</div>
    <div class="description-paragraph style-scope patent-text">The present invention also includes an HGV mosaic polypeptide, where the mosaic polypeptide contains at least two epitopes of HGV, and, where the polypeptide substantially lacks amino acids normally intervening between the epitopes in the native HGV coding sequence. Such mosaic polypeptides are useful in the applications and methods discussed above.</div>
    <div class="description-paragraph style-scope patent-text">The present invention further includes a random peptide epitope (mimitope) that mimics a natural HGV antigenic epitope during epitope presentation. Such mimitopes are useful in the applications and methods discussed above. Also included in the present invention is a method of identifying a random peptide HGV epitope. In the method, a library of random peptide epitopes is generated or selected. The library is contacted with an anti-HGV antibody. Mimitopes are identified that are specifically immunoreactive with the antibody. Sera (containing anti-HGV antibodies) or antibodies generated by the methods of the present invention can be used. Random peptide libraries can, for example, be displayed on phage or generated as combinatorial libraries.</div>
    <div class="description-paragraph style-scope patent-text">In another aspect, the present invention includes therapeutic compounds and methods for the prevention and/or treatment of HGV infection.</div>
    
    
    <div class="description-paragraph style-scope patent-text">These and other objects and features of the invention will be more fully appreciated when the following detailed description of the invention is read in conjunction with the accompanying drawings.</div>
    <heading class="style-scope patent-text">BRIEF DESCRIPTION OF THE FIGURES</heading>
    <div class="description-paragraph style-scope patent-text">FIG. 1: the relationship of the SEQ ID NO:14 open reading frame to the 470-20-1 clone.</div>
    <div class="description-paragraph style-scope patent-text">FIG. 2: shows an exemplary protein profile from gradient fractions eluted from a glutathione affinity column.</div>
    <div class="description-paragraph style-scope patent-text">FIG. 3: shows an exemplary Sodium dodecyl sulfate polyacrylamide gel electrophoresis analysis of fraction samples from FIG. 2.</div>
    <div class="description-paragraph style-scope patent-text">FIG. 4A: shows an exemplary protein profile from gradient fractions eluted from an anion exchange column.</div>
    <div class="description-paragraph style-scope patent-text">FIGS. 4B and 4C: show exemplary Sodium dodecyl sulfate polyacrylamide gel electrophoresis analysis of fraction samples from FIG. 4A.</div>
    <div class="description-paragraph style-scope patent-text">FIGS. 5A and 5B: amino acid alignments of HGV with two other members of Flaviviridae family--Hog Cholera Virus and Hepatitis C Virus.</div>
    <div class="description-paragraph style-scope patent-text">FIG. 6 shows a map of a portion of the vector pGEX-Hisb-GE3-2, a bacterial expression plasmid carrying an HGV epitope.</div>
    <div class="description-paragraph style-scope patent-text">FIGS. 7A to 7D show the results of Western blot analysis of the purified HGV GE3-2 protein.</div>
    <div class="description-paragraph style-scope patent-text">FIGS. 8A to 8D show the results of Western blot analysis of the purified HGV Y5-l0 antigen.</div>
    <div class="description-paragraph style-scope patent-text">FIGS. 9A to 9D show the results of Western blot analysis of the following antigens: Y5-5, GE3-2 and Y5-10.</div>
    <div class="description-paragraph style-scope patent-text">FIGS. 10A to 10F show the results of Western blot analysis of antigens GE-NS2b and GE-NS5a.</div>
    <div class="description-paragraph style-scope patent-text">FIG. 11 presents a Kyte-Doolittle hydrophobicity plot of the coding sequence of HGV.</div>
    <div class="description-paragraph style-scope patent-text">FIG. 12 shows the results of Western blot analysis of HGV pET clones with anti-T7.Tag monoclonal antibody.</div>
    <div class="description-paragraph style-scope patent-text">FIGS. 13A to 13D show the results of Western blot analysis of HGV pET clone GE-NS5b. FIG. 13E shows a corresponding coomassie stained gel.</div>
    <div class="description-paragraph style-scope patent-text">FIGS. 14A to 14C show the results of Western blot analysis of HGV pET clone GE-E2. FIG. 14D shows a corresponding coomassie stained gel.</div>
    <div class="description-paragraph style-scope patent-text">FIGS. 15A to 15C show the results of Western blot analysis of HGV pET clone GE-NS5b. FIG. 15D shows a corresponding coomassie stained gel.</div>
    <div class="description-paragraph style-scope patent-text">FIG. 16 shows a schematic representation of the coding regions of HGV.</div>
    
    
    <heading class="style-scope patent-text">DETAILED DESCRIPTION OF THE INVENTION</heading>
    <heading class="style-scope patent-text">I. Definitions</heading>
    <div class="description-paragraph style-scope patent-text">The terms defined below have the following meaning herein:</div>
    <div class="description-paragraph style-scope patent-text">1. "nonA/nonB/nonC/nonD/nonE hepatitis viral agent {N-(ABCDE)}," herein provisionally designated HGV, means a virus, virus type, or virus class which (i) is transmissible in some primates, including, mystax, chimpanzees or humans as characterized by elevated serum alanine amino transferase levels in an infected primate, (ii) is serologically distinct from hepatitis A virus (HAV), hepatitis B virus (HBV), hepatitis C virus (HCV), hepatitis D virus, and hepatitis E (HEV) (although HGV may co-infect a subject with these viruses), and (iii) is a member of the virus family Flaviviridae.</div>
    <div class="description-paragraph style-scope patent-text">2. "HGV variants" are defined as viral isolates that have at least about 40%, preferably 55% or 65%, or more preferably 80% global sequence homology, that is, sequence identity over a length of the viral genome polynucleotide sequence, to the HGV polynucleotide sequences disclosed herein (e.g., SEQ ID NO:14).</div>
    <div class="description-paragraph style-scope patent-text">"Sequence homology" is determined essentially as follows. Two polynucleotide sequences of similar length (preferably, the entire viral genome) are considered to be homologous to one another, if, when they are aligned using the ALIGN program, over 40%, preferably 55% or 65%, or more preferably 80% of the nucleic acids in the highest scoring alignment are identically aligned using a ktup of 1, the default parameters and the default PAM matrix.</div>
    <div class="description-paragraph style-scope patent-text">The ALIGN program is found in the FASTA version 1.7 suite of sequence comparison programs (Pearson, et al., 1988; Pearson, 1990; program available from William R. Pearson, Department of Biological Chemistry, Box 440, Jordan Hall, Charlottesville, Va.).</div>
    <div class="description-paragraph style-scope patent-text">In determining whether two viruses are "highly homologous" to each other, the complete sequence of all the viral proteins (or the polyprotein) for one virus are optimally, globally aligned with the viral proteins or polyprotein of the other virus using the ALIGN program of the above suite using a ktup of 1, the default parameters and the default PAM matrix. Regions of dissimilarity or similarity are not excluded from the analysis. Differences in lengths between the two sequences are considered as mismatches. Alternatively, viral structural protein regions are typically used to determine relatedness between viral isolates. Highly homologous viruses have over 40%, or preferably 55% or 65%, or more preferably 80% global polypeptide sequence identity.</div>
    <div class="description-paragraph style-scope patent-text">3. Two nucleic acid fragments are considered to be "selectively hybridizable" to an HGV polynucleotide, if they are capable of specifically hybridizing to HGV or a variant thereof (e.g., a probe that hybridizes to HGV nucleic acid but not to polynucleotides from other members of the virus family Flaviviridae) or specifically priming a polymerase chain reaction: (i) under typical hybridization and wash conditions, as described, for example, in Maniatis, et al., pages 320-328, and 382-389, (ii) using reduced stringency wash conditions that allow at most about 25-30% basepair mismatches, for example: 2×SSC, 0.1% SDS, room temperature twice, 30 minutes each; then 2×SSC, 0.1% SDS, 37° C. once, 30 minutes; then 2×SSC room temperature twice, 10 minutes each, or (iii) selecting primers for use in typical polymerase chain reactions (PCR) under standard conditions (for example, in Saiki, R. K, et al.), which result in specific amplification of sequences of HGV or its variants.</div>
    <div class="description-paragraph style-scope patent-text">Preferably, highly homologous nucleic acid strands contain less than 20-30% basepair mismatches, even more preferably less than 5-20% basepair mismatches. These degrees of homology can be selected by using wash conditions of appropriate stringency for identification of clones from gene libraries (or other sources of genetic material), as is well known in the art.</div>
    <div class="description-paragraph style-scope patent-text">4. An "HGV polynucleotide," as used herein, is defined as follows. For polynucleotides greater than about 100 nucleotides, HGV polynucleotides encompass polynucleotide sequences encoded by HGV variants and homologous sequences as defined in "2" above. For polynucleotides less than about 100 nucleotides in length, HGV polynucleotide encompasses sequences that selectively hybridizes to sequences of HGV or its variants. Further, HGV polynucleotides include polynucleotides encoding HGV polypeptides (see below).</div>
    <div class="description-paragraph style-scope patent-text">The term "polynucleotide" as used herein refers to a polymeric molecule having a backbone that supports bases capable of hydrogen bonding to typical nucleic acids, where the polymer backbone presents the bases in a manner to permit such hydrogen bonding in a sequence specific fashion between the polymeric molecule and a typically nucleic acid (e.g., single-stranded DNA). Such bases are typically inosine, adenosine, guanosine, cytosine, uracil and thymidine. Numerous polynucleotide modifications are known in the art, for example, labels, methylation, and substitution of one or more of the naturally occurring nucleotides with an analog.</div>
    <div class="description-paragraph style-scope patent-text">Polymeric molecules include double and single stranded RNA and DNA, and backbone modifications thereof, for example, methylphosphonate linkages. Further, such polymeric molecules include alternative polymer backbone structures such as, but not limited to, polyvinyl backbones (Pitha, 1970a/b), morpholino backbones (Summerton, et al., 1992, 1993). A variety of other charged and uncharged polynucleotide analogs have been reported. Numerous backbone modifications are known in the art, including, but not limited to, uncharged linkages (e.g., methyl phosphonates, phosphotriesters, phosphoamidates, and carbamates) and charged linkages (e.g., phosphorothioates and phosphorodithioates). In addition linkages may contain the following exemplary modifications: pendant moieties, such as, proteins (including, for example, nucleases, toxins, antibodies, signal peptides and poly-L-lysine); intercalators (e.g., acridine and psoralen), chelators (e.g., metals, radioactive metals, boron and oxidative metals), alkylators, and other modified linkages (e.g., alpha anomeric nucleic acids).</div>
    <div class="description-paragraph style-scope patent-text">5. An "HGV polypeptide" is defined herein as any polypeptide homologous to an HGV polypeptide. "Homology," as used herein, is defined as follows. In one embodiment, a polypeptide is homologous to an HGV polypeptide if it is encoded by nucleic acid that selectively hybridizes to sequences of HGV or its variants.</div>
    <div class="description-paragraph style-scope patent-text">In another embodiment, a polypeptide is homologous to an HGV polypeptide if it is encoded by HGV or its variants, as defined above, polypeptides of this group are typically larger than 15, preferable 25, or more preferable 35, contiguous amino acids. Further, for polypeptides longer than about 60 amino acids, sequence comparisons for the purpose of determining "polypeptide homology" are performed using the local alignment program LALIGN. The polypeptide sequence, is compared against the HGV amino acid sequence or any of its variants, as defined above, using the LALIGN program with a ktup of 1, default parameters and the default PAM.</div>
    <div class="description-paragraph style-scope patent-text">Any polypeptide (typically a polypeptide not specifically immunoreactive with HGV antibodies) with an optimal alignment longer than 60 amino acids and greater than 60%, preferably 70%, or more preferably 80% of identically aligned amino acids is considered to be a "homologous polypeptide." The LALIGN program is found in the FASTA version 1.7 suite of sequence comparison programs (Pearson, et al., 1988; Pearson, 1990; program available from William R. Pearson, Department of Biological Chemistry, Box 440, Jordan Hall, Charlottesville, Va.).</div>
    <div class="description-paragraph style-scope patent-text">6. A polynucleotide is "derived from" HGV if it has the same or substantially the same basepair sequence as a region of an HGV genome, CDNA of HGV or complements thereof, or if it displays homology as noted under "2", "3" or "4" above.</div>
    <div class="description-paragraph style-scope patent-text">A polypeptide or polypeptide "fragment" is "derived from" HGV if it is (i) encoded by an open reading frame of an HGV polynucleotide, or (ii) displays homology to HGV polypeptides as noted under "2" and "5" above, or (iii) is specifically immunoreactive with HGV positive sera.</div>
    <div class="description-paragraph style-scope patent-text">7. "Substantially isolated" and "purified" are used in several contexts and typically refer to at least partial purification of an HGV virus particle, component (e.g., polynucleotide or polypeptide), or related compound (e.g., anti-HGV antibodies) away from unrelated or contaminating components (e.g., serum cells, proteins, non-HGV polynucleotides and non-anti-HGV antibodies). Methods and procedures for the isolation or purification of compounds or components of interest are described below (e.g., affinity purification of fusion proteins and recombinant production of HGV polypeptides).</div>
    <div class="description-paragraph style-scope patent-text">8. In the context of the present invention, the phrase "nucleic acid sequences," when referring to sequences which encode a protein, polypeptide, or peptide, is meant to include degenerative nucleic acid sequences which encode homologous protein, polypeptide or peptide sequences as well as the disclosed sequence.</div>
    <div class="description-paragraph style-scope patent-text">9. An "epitope" is the antigenic determinant defined as the specific portion of an antigen with which the antigen binding portion of a specific antibody interacts.</div>
    <div class="description-paragraph style-scope patent-text">10. An antigen or epitope is "specifically immunoreactive" with HGV positive sera when the epitope/antigen binds to antibodies present in the HGV infected sera but does not bind to antibodies present in the majority (greater than about 90%, preferably greater than 95%) of sera from individuals who are not or have not been infected with HGV. "Specifically immunoreactive" antigens or epitopes may also be immunoreactive with monoclonal or polyclonal antibodies generated against specific HGV epitopes or antigens.</div>
    <div class="description-paragraph style-scope patent-text">An antibody or antibody composition (e.g., polyclonal antibodies) is "specifically immunoreactive" with HGV when the antibody or antibody composition is immunoreactive with an HGV antigen but not with HAV, HBV, HCV, HDV or HEV antigens. Further, "specifically immunoreactive antibodies" are not immunoreactive with antigens typically present in normal sera obtained from subjects not infected with or exposed to HGV, HAV, HBV, HCV, HDV or HEV.</div>
    <heading class="style-scope patent-text">II. N-(ABCDE) Sera</heading>
    <div class="description-paragraph style-scope patent-text">Availability of a serologic test for anti-HCV and the development of an RT-PCR assay for HCV-RNA (Kawasaki, et al.; Wang, et al., 1990) allowed the identification of several cases of both post-transfusion and community acquired non-HCV hepatitis. The human hepatitis case, PNF 2161, was originally identified as having NANB hepatitis (NANBH) through the Sentinel Counties Study of community acquired hepatitis, sponsored by the Centers for Disease Control and Prevention (Alter, et al., 1989b). PNF 2161 was a sample obtained from an elderly Caucasian male patient who developed acute hepatitis approximately 8 weeks following a blood transfusion, with a peak serum ALT level of 1141 IU (normal, ≦45 IU). Following resolution of the episode of acute hepatitis, he had fluctuating, but persistently elevated ALT levels over the next seven years, consistent with chronic hepatitis, although histopathologic confirmation of this diagnosis was not obtained.</div>
    <div class="description-paragraph style-scope patent-text">The plasma specimen used to clone HGV (as described herein) was obtained in June 1989, approximately 4<sup class="style-scope patent-text">1/2</sup>  years following the episode of acute hepatitis, and cryo-preserved. Patient PNF 2161 was initially believed not to be infected with HCV, based on consistently negative results with a first generation immunoassay test (Ortho HCV ELISA Test System; Ortho Diagnostics, Raritan, N.J.). However, subsequent testing using a second generation HCV immunoassay (Ortho) and PCR with HCV 5'-non-coding region primers demonstrated that the patient was infected with HCV.</div>
    <heading class="style-scope patent-text">III. Isolation of HGV Associated Sequences.</heading>
    <div class="description-paragraph style-scope patent-text">As one approach toward identifying clones containing HGV sequences, a CDNA library was prepared from infected-HGV sera in the expression vector lambda gt11 (Example 1). Polynucleotide sequences were then selected for the expression of peptides which are immunoreactive with serum PNF 2161. First round screening was typically performed using the PNF 2161 serum (used to generate the phage library). It is also possible to screen with other suspected N-(ABCDE) sera.</div>
    <div class="description-paragraph style-scope patent-text">Recombinant proteins identified by this approach provide candidates for peptides which can serve as substrates in diagnostic tests. Further, the nucleic acid coding sequences identified by this approach serve as useful hybridization probes for the identification of additional HGV coding sequences.</div>
    <div class="description-paragraph style-scope patent-text">The sera described above were used to generate cDNA libraries in lambda gt11 (Example 1). In the method illustrated in Example 1, infected serum was precipitated in 8% PEG without dilution, and the libra- ries were generated from the resulting pelleted virus. Sera from infected human sources were treated in the same fashion.</div>
    <div class="description-paragraph style-scope patent-text">As an advantageous alternative to PEG precipitation, ultracentrifugation can be used to pellet particulate agents from infected sera or other biological specimens. To isolate viral particles from which nucleic acids could be extracted, serum, ranging up to 2 ml, is diluted to approximately 10 ml with PBS, spun at 3K for 10 minutes, and the supernatant is centrifuged for a minimum of 2 hours at 40,000 rpm (approximately 110,000×g) in a Ti70.1 rotor (Beckman Instruments, Fullerton, Calif.) at 4° C. The supernatant is then aspirated and the pellet extracted by standard nucleic acid extraction techniques.</div>
    <div class="description-paragraph style-scope patent-text">CDNA libraries were generated using random primers in reverse transcription reactions with RNA extracted from pelleted sera as starting material. The resulting molecules were ligated to Sequence Independent Single Primer Amplification (SISPA; Reyes, et al., 1991) linker primers and expanded in a non-selective manner, and then cloned into a suitable vector, for example, lambda gt11, for expression and screening of peptide antigens. Alternatively, the lambda gt10 vector may also be used.</div>
    <div class="description-paragraph style-scope patent-text">Lambda gt11 is a particularly useful expression vector which contains a unique EcoRI insertion site 53 base pairs upstream of the translation termination codon of the β-galactosidase gene. Thus, an inserted sequence is expressed as a β-galactosidase fusion protein which contains the N-terminal portion of the β-galactosidase gene product, the heterologous peptide, and optionally the C-terminal region of the β-galactosidase peptide (the C-terminal portion being expressed when the heterologous peptide coding sequence does not contain a translation termination codon).</div>
    <div class="description-paragraph style-scope patent-text">This vector also produces a temperature-sensitive repressor (cI857) which causes viral lysogeny at permissive temperatures, e.g., 32° C., and leads to viral lysis at elevated temperatures, e.g., 42° C. Advantages of this vector include: (1) highly efficient recombinant clone generation, (2) ability to select lysogenized host cells on the basis of host-cell growth at permissive, but not non-permissive, temperatures, and (3) production of recombinant fusion protein. Further, since phage containing a heterologous insert produces an inactive β-galactosidase enzyme, phage with inserts are typically identified using a calorimetric substrate conversion reaction employing β-galactosidase.</div>
    <div class="description-paragraph style-scope patent-text">Example 1 describes the preparation of a cDNA library for the N-(ABCDE) hepatitis sera PNF 2161. The library was immunoscreened using PNF 2161 (Example 3). A number of lambda gt11 clones were identified which were immunoreactive. Immunopositive clones were plaque-purified and their immunoreactivity retested. Also, the immunoreactivity of the clones with normal human sera was also tested.</div>
    <div class="description-paragraph style-scope patent-text">These clones were also examined for the "exogenous" nature of the cloned insert sequence. This basic test establishes that the cloned fragment does not represent a portion of human or other potentially contaminating nucleic acids (e.g., E. coli, S. cerevisiea and mitochondrial). The clone inserts were isolated by EcoRI digestion following polymerase chain reaction amplification. The inserts were purified then radiolabelled and used as hybridization probes against membrane bound normal human DNA, normal mystax DNA and bacterial DNA (control DNAs) (Example 4A).</div>
    <div class="description-paragraph style-scope patent-text">Clone 470-20-1 (PNF2161 CDNA source) was one of the clones isolated by immunoscreening with the PNF 2161 serum. The clone was not reactive with normal human sera. The clone has a large open reading frame (203 base pairs; SEQ ID NO:3), in-frame with the β-galactosidase gene of the lambda gt11 vector. The clone is exogenous by genomic DNA hybridization analysis and genomic PCR analysis, using human, yeast and E. coli genomic DNAs (Example 4B).</div>
    <div class="description-paragraph style-scope patent-text">The sequence was present in PNF2161 serum as determined by RT-PCR (Example 4C). RT-PCR of serially diluted PNF 2161 RNA suggested at least about 10<sup class="style-scope patent-text">5</sup> copies of 470-20-1 specific sequence per ml. The sequence was also detected in sucrose density gradient fractions at densities consistent with the sequence banding in association with a virus-like particle (Example 5).</div>
    <div class="description-paragraph style-scope patent-text">Bacterial lysates of E. coli expressing a second clone, clone 470-expl, (SEQ ID NO:37) were also shown to be specifically immunoreactive with PNF 2161 serum at comparable levels to clone 470-20-1. The coding sequence of 470-exp1 was flanked by termination codons (based on sequence comparisons to SEQ ID NO:14, also see FIG. 1) and had an internal methionine.</div>
    <div class="description-paragraph style-scope patent-text">Further sequences contained in SEQ ID NO:14, adjacent to clone 470-20-1, were obtained by anchor polymerase chain reaction (Anchor PCR) using primers from clone 470-20-1 (Example 6). In this case a PNF 2161 2-cDNA source library was used as template, where the cDNA/complement double-stranded DNA products were ligated to lambda arms, but the mixture was not packaged. 470-20-1 specific primers were used in amplification reactions with SISPA-amplified PNF 2161 cDNA as a template (Example 4). The identity of the amplified DNA fragments were confirmed by (i) size and (ii) hybridization with a 470-20-1 specific oligonucleotide probe (SEQ ID NO:16). The 470-20-1 specific signal was detected in cDNA amplified by PCR from SISPA-amplified PNF 2161, demonstrating the presence of the 470-20-1 sequences in the source material.</div>
    <div class="description-paragraph style-scope patent-text">The 470-20-1 specific primers were also used in amplification reactions with the following RNA sources as substrate: normal mystax liver RNA, normal tamarin (Sanguins laboriatis) liver RNA, and MY131 liver RNA (Example 4). The results from these experiments demonstrate the 470-20-1 sequences are present in the parent serum sample (PNF 2161) and in an RNA liver sample from an animal challenged with the PNF 2161 sample (MY131). Both normal control RNAs were negative for the presence of 470-20-1 sequences.</div>
    <div class="description-paragraph style-scope patent-text">Further, PNF 2161 serum and other cloning source or related source materials were directly tested by PCR using primers from selected cloned sequences. Specific amplification products were detected by hybridization to a specific oligonucleotide probe 470-20-1-152F (SEQ ID NO:16). A specific signal was reproducibly detected in multiple extracts of PNF 2161, with the 470-20-1 specific primers.</div>
    <div class="description-paragraph style-scope patent-text">The disease association between HGV and liver disease is further supported by the data presented in Example 4F. Sera from hepatitis patients and from blood donors with abnormal liver function were assessed for the presence of HGV by RT-PCR screening, using HGV specific primers. HGV specific sequence were detected in 6/152 of these sera samples. No HGV positives were detected among the control samples (n=11).</div>
    <div class="description-paragraph style-scope patent-text">The results presented above indicate the isolation of a viral agent associated with N-(ABCDE) viral infection of liver (i.e., hepatitis) and/or infection, and resulting disease, of other tissue and cell types.</div>
    <heading class="style-scope patent-text">IV. Further Characterization of HGV Recombinant Antigens</heading>
    <div class="description-paragraph style-scope patent-text">A. Screening Recombinant Libraries.</div>
    <div class="description-paragraph style-scope patent-text">Further candidate HGV antigens can be obtained from the libraries of the present invention using the screening methods described above. The cDNA library described above has been deposited with the American Type Culture Collection, 12301 Parklawn Dr., Rockville, Md., 20852, and has been assigned the following designation: PNF 2161 CDNA source, ATCC 75268. The deposit was accepted by the International Depository Authority on Jul. 16, 1992.</div>
    <div class="description-paragraph style-scope patent-text">A second PNF 2161 CDNA library has been generated essentially as described for the first PNF 2161 CDNA library, except that second PNF 2161 cDNA source library was ligated to lambda gt11 arms but was not packaged. This non-packaged library was used to obtain the extension clones described below. A packaged version of this second library (PNF 2161 2-cDNA source library) has been deposited with the American Type Culture Collection, 12301 Parklawn Drive, Rockville, Md., 20852, and has been assigned the following designation: PNF 2161 2-cDNA source, ATCC 75837 . The deposit was accepted by the International Depository Authority on Jul. 22, 1994.</div>
    <div class="description-paragraph style-scope patent-text">In addition to the recombinant libraries generated above, other recombinant libraries from N-(ABCDE) hepatitis sera can likewise be generated and screened as described herein.</div>
    <div class="description-paragraph style-scope patent-text">B. Epitope Mapping, Cross Hybridization and Isolation of Genomic Sequences.</div>
    <div class="description-paragraph style-scope patent-text">Antigen encoding DNA fragments can be identified by (i) immunoscreening, as described above, or (ii) computer analysis of coding sequences (e.g., SEQ ID NO:14) using an algorithm (such as, "ANTIGEN," Intelligenetics, Mountain View, Calif.) to identify potential antigenic regions. An antigen-encoding DNA fragment can be subcloned. The subcloned insert can then be fragmented by partial DNase I digestion to generate random fragments or by specific restriction endonuclease digestion to produce specific subfragments. The resulting DNA fragments can be inserted into the lambda gt11 vector and subjected to immunoscreening in order to provide an epitope map of the cloned insert.</div>
    <div class="description-paragraph style-scope patent-text">In addition, the DNA fragments can be employed as probes in hybridization experiments to identify overlapping HGV sequences, and these in turn can be further used as probes to identify a set of contiguous clones. The generation of sets of contiguous clones allows the elucidation of the sequence of the HGV's genome.</div>
    <div class="description-paragraph style-scope patent-text">Any of the above-described clone sequences (e.g., derived from SEQ ID NO:14 or clone 470-20-1) can be used to probe the cDNA and DNA libraries, generated in a vector such as lambda gtlo or "LAMBDA ZAP II" (Stratagene, San Diego, Calif.). Specific subfragments of known sequence may be isolated by polymerase chain reaction or after restriction endonuclease cleavage of vectors carrying such sequences. The resulting DNA fragments can be used as radiolabelled probes against any selected library. In particular, the 5' and 3' terminal sequences of the clone inserts are useful as probes to identify additional clones.</div>
    <div class="description-paragraph style-scope patent-text">Further, the sequences provided by the 5' end of cloned inserts are useful as sequence specific primers in first-strand cDNA or DNA synthesis reactions (Maniatis et al.; Scharf et al.). For example, specifically primed PNF 2161 cDNA and DNA libraries can be prepared by using specific primers derived from SEQ ID NO:14 on PNF 2161 nucleic acids as a template. The second-strand of the new cDNA is synthesized using RNase H and DNA polymerase I. The above procedures identify or produce DNA/cDNA molecules corresponding to nucleic acid regions that are 5' adjacent to the known clone insert sequences. These newly isolated sequences can in turn be used to identify further flanking sequences, and so on, to identify the sequences composing the entire genome for HGV. As described above, after new HGV sequences are isolated, the polynucleotides can be cloned and immunoscreened to identify specific sequences encoding HGV antigens.</div>
    <div class="description-paragraph style-scope patent-text">Extension clone sequences (SEQ ID NO:14), containing further sequences of interest, have been obtained for clone PNF 470-20-1 (SEQ ID NO:3) using the "Anchor PCR" method described in Example 6. Briefly, the strategy consists of ligating PNF 2161 SISPA cDNA to lambda gt11 arms and amplifying the ligation reaction with a gt11-specific primer and one of two 470-20-1 specific primers.</div>
    <div class="description-paragraph style-scope patent-text">The amplification products are electrophoretically separated, transferred to filters and the DNA bound to the filters is probed with a 470-20-1 specific probe. Bands corresponding to hybridization positive band signals were gel purified, cloned and sequenced.</div>
    <div class="description-paragraph style-scope patent-text">C. Preparation of Antigenic Polypeptides and Antibodies.</div>
    <div class="description-paragraph style-scope patent-text">The recombinant peptides of the present invention can be purified by standard protein purification procedures which may include differential precipitation, molecular sieve chromatography, ion-exchange chromatography, isoelectric focusing, gel electrophoresis and affinity chromatography.</div>
    <div class="description-paragraph style-scope patent-text">In one embodiment of the present invention, the polynucleotide sequences of the antigens of the present invention have been cloned in the plasmid p-GEX (Example 7A) or various derivatives thereof (pGEX-GLI). The plasmid PGEX (Smith, et al., 1988) and its derivatives express the polypeptide sequences of a cloned insert fused in-frame to the protein glutathione-S-transferase (sj26). In one vector construction, plasmid pGEX-hisB, an amino acid sequence of 6 histidines is introduced at the carboxy terminus of the fusion protein.</div>
    <div class="description-paragraph style-scope patent-text">The various recombinant PGEX plasmids can be transformed into appropriate strains of E. coli and fusion protein production can be induced by the addition of IPTG (isopropyl-thio galactopyranoside) as described in Example 7A. Solubilized recombinant fusion protein can then be purified from cell lysates of the induced cultures using glutathione agarose affinity chromatography (Example 7A).</div>
    <div class="description-paragraph style-scope patent-text">Insoluble fusion protein expressed by the plasmid pGEX-hisB can be purified by means of immobilized metal ion affinity chromatography (Porath) in buffers containing 6M Urea or 6M guanidinium isothiocyanate, both of which are useful for the solubilization of proteins. Alternatively insoluble proteins expressed in pGEX-GLI or derivatives thereof can be purified using combinations of centrifugation to remove soluble proteins followed by solubilization of insoluble proteins and standard chromatographic methodologies, such as ion exchange or size exclusion chromatography, and other such methods are known in the art.</div>
    <div class="description-paragraph style-scope patent-text">In the case of β-galactosidase fusion proteins (such as those produced by lambda gt11 clones) the fused protein can be isolated readily by affinity chromatography, by passing cell lysis material over a solid support having surface-bound anti-β-galactosidase antibody. For example, purification of a β-galactosidase/fusion protein, derived from 470-20-1 coding sequences, by affinity chromatography is described in Example 7B.</div>
    <div class="description-paragraph style-scope patent-text">Also included in the invention is an expression vector, such as the lambda gt11 or pGEX vectors described above, containing HGV coding sequences and expression control elements which allow expression of the coding regions in a suitable host. The control elements generally include a promoter, translation initiation codon, and translation and transcription termination sequences, and an insertion site for introducing the insert into the vector.</div>
    <div class="description-paragraph style-scope patent-text">The DNA encoding the desired antigenic polypeptide can be cloned into any number of commercially available vectors to generate expression of the polypeptide in the appropriate host system. These systems include, but are not limited to, the following: baculovirus expression (Reilly, et al.; Beames, et al.; Pharmingen; Clontech, Palo Alto, Calif.), vaccinia expression (Earl, 1991; Moss, et al.), expression in bacteria (Ausubel, et al.; Clontech), expression in yeast (Gellissen, 1992; Romanos, 1992; Goeddel; Guthrie and Fink), expression in mammalian cells (Clontech; Gibco-BRL, Ground Island, N.Y.), e.g., Chinese hamster ovary (CHO) cell lines (Haynes, 1983, Lau, 1984, Kaufman, 1990). These recombinant polypeptide antigens can be expressed directly or as fusion proteins. A number of features can be engineered into the expression vectors, such as leader sequences which promote the secretion of the expressed sequences into culture medium.</div>
    <div class="description-paragraph style-scope patent-text">Expression of large HGV polypeptides using several of these systems is described in Example 16.</div>
    <div class="description-paragraph style-scope patent-text">Expression in yeast systems has the advantage of commercial production. Recombinant protein production by vaccinia and CHO cell line have the advantage of being mammalian expression systems. Further, vaccinia virus expression has several advantages including the following: (i) its wide host range; (ii) faithful post-transcriptional modification, processing, folding, transport, secretion, and assembly of recombinant proteins; (iii) high level expression of relatively soluble recombinant proteins; and (iv) a large capacity to accommodate foreign DNA.</div>
    <div class="description-paragraph style-scope patent-text">The recombinant expressed polypeptide produced HGV polypeptide antigens are typically isolated from lysed cells or culture media. Purification can be carried out by methods known in the art including salt fractionation, ion exchange chromatography, and affinity chromatography. Immunoaffinity chromatography can be employed using antibodies generated based on the HGV antigens identified by the methods of the present invention.</div>
    <div class="description-paragraph style-scope patent-text">HGV polypeptide antigens may also be isolated from HGV particles (see below).</div>
    <div class="description-paragraph style-scope patent-text">Continuous antigenic determinants of polypeptides are generally relatively small, typically 6 to 10 amino acids in length. Smaller fragments have been identified as antigenic regions, for example, in conformational epitopes. HGV polypeptide antigens are identified as described above. The resulting DNA coding regions of either strand can be expressed recombinantly either as fusion proteins or isolated polypeptides. In addition, amino acid sequences can be conveniently chemically synthesized using commercially available synthesizer (Applied Biosystems, Foster City, Calif.) or "PIN" technology (Applied Biosytems).</div>
    <div class="description-paragraph style-scope patent-text">In another embodiment, the present invention includes mosaic proteins that are composed of multiple epitopes. An HGV mosaic polypeptide typically contains at least two epitopes of HGV, where the polypeptide substantially lacks amino acids normally intervening between the epitopes in the native HGV coding sequence. Synthetic genes (Crea; Yoshio et al.; Eaton et al.) encoding multiple, tandem epitopes can be constructed that will produce mosaic proteins using standard recombinant DNA technology using polypeptide expression vector/host system described above.</div>
    <div class="description-paragraph style-scope patent-text">Further, multiple antigen peptides can be synthesized chemically by methods described previously (Tam, J. P., 1988; Briand et al.). For example, a small immunologically inert core matrix of lysine residues with α- and e- amino groups can be used to anchor multiple copies of the same or different synthetic peptides (typically 6-15 residues long) representing epitopes of interest. Mosaic proteins or multiple antigen peptide antigens give higher sensitivity and specificity in immunoassays due to the signal amplification resulting from distribution of multiple epitopes.</div>
    <div class="description-paragraph style-scope patent-text">Antigens obtained by any of these methods can be used for antibody generation, diagnostic tests and vaccine development.</div>
    <div class="description-paragraph style-scope patent-text">In another aspect, the invention includes specific antibodies directed against the polypeptide antigens of the present invention. Antigens obtained by any of these methods may be directly used for the generation of antibodies or they may be coupled to appropriate carrier molecules. Many such carriers are known in the art and are commercially available (e.g., Pierce, Rockford Ill.). Typically, to prepare antibodies, a host animal, such as a rabbit, is immunized with the purified antigen or fused protein antigen. Hybrid, or fused, proteins may be generated using a variety of coding sequence derived from other proteins, such as glutathione-S-transferase or β-galactosidase. The host serum or plasma is collected following an appropriate time interval, and this serum is tested for antibodies specific against the antigen. Example 8 describes the production of rabbit serum antibodies which are specific against the 470-20-1 antigen in the Sj26/470-20-1 hybrid protein. These techniques are equally applicable to all immunogenic sequences derived from HGV, including, but not limited to, those derived from the coding sequence presented as SEQ ID NO:14.</div>
    <div class="description-paragraph style-scope patent-text">The gamma globulin fraction or the IgG antibodies of immunized animals can be obtained, for example, by use of saturated ammonium sulfate precipitation or DEAE Sephadex chromatography, affinity chromatography, or other techniques known to those skilled in the art for producing polyclonal antibodies.</div>
    <div class="description-paragraph style-scope patent-text">Alternatively, purified antigen or fused antigen protein may be used for producing monoclonal antibodies. Here the spleen or lymphocytes from an immunized animal are removed and immortalized or used to prepare hybridomas by methods known to those skilled in the art. To produce a human-derived hybridoma, a human lymphocyte donor is selected. A donor known to be infected with a HGV may serve as a suitable lymphocyte donor. Lymphocytes can be isolated from a peripheral blood sample. Epstein-Barr virus (EBV) can be used to immortalize human lymphocytes or a suitable fusion partner can be used to produce human-derived hybridomas. Primary in vitro sensitization with viral specific polypeptides can also be used in the generation of human monoclonal antibodies.</div>
    <div class="description-paragraph style-scope patent-text">Antibodies secreted by the immortalized cells are screened to determine the clones that secrete antibodies of the desired specificity, for example, by using the ELISA or Western blot method (Example 10; Ausubel et al.).</div>
    <div class="description-paragraph style-scope patent-text">Using HGV-positive serum or plasma, or the antibodies of the present invention, other antigenic peptides and epitopes can be isolated. For example, a number of different techniques have been developed for the simultaneous synthesis of many peptides (Geysen, et al.; Houghten; Frank and Doring; Hudson). The method developed by Geysen, et al., is especially useful because of the relative simplicity with which large numbers of different peptide sequences can be generated and tested for antigenicity. In the Geysen method (also referred to as MULTI-PIN peptide synthesis), the peptides are synthesized on polyacrylamide acid grafted polyethylene rods attached to a micro-titer plate. The MULTI-PIN strategy allows large numbers of syntheses (96 peptides per plate) to be immunologically screened using the polyclonal or monoclonal antibodies of the present invention and commercially available reagents and instrumentation. Immunoreactive peptides are identified and characterized.</div>
    <div class="description-paragraph style-scope patent-text">It has been reported that up to 6,000 oligopeptides can be synthesized in a two week period, thus making it practical (by synthesizing all of the possible overlapping amino acid sequences of a particular antigen) to screen viral antigen sequences for epitopes to the resolution of a single amino acid (Geysen, et al.).</div>
    <div class="description-paragraph style-scope patent-text">An alternative method of scanning for immunodominate peptides is to synthesize longer peptides (e.g., 10 to 30 amino acids) corresponding to HGV coding sequences using conventional automated peptide synthesis (Carter, et al., 1994; Obeid, et al., 1994; Commandaeur, et al., 1994). This method has the advantage that the longer peptides can fold into shapes that mimic conformational epitopes.</div>
    <div class="description-paragraph style-scope patent-text">Also, HGV antibodies, in particular, monoclonals, can be used to identify random polypeptides that mimic their virus-encoded target polypeptides (Scott and Smith, 1990; Smith, 1991). For example, random peptide libraries displayed on phage (RPL) (Scott and Smith, 1990) can be used as a source of peptide ligands for antibody generation or for vaccine development. RPL approach allows the expression of peptide-ligand containing fusion proteins on the phage surface and enrichment of these ligand encoding phages by affinity selection using antibodies (Smith, J. P., 1991; Christian, et al.; Scott, et al., 1992; Folgori, et al.). These random peptide epitopes detected by specific antibodies mimic the natural antigenic epitopes (mimotopes) during epitope presentation. HGV antigenic mimics (mimotopes) can be isolated from RPL. Hexa- to decapeptide phagotope (mimotope displayed on phage) expressing RPL can be made by published methods (Scott and Smith; Smith, J. P, 1991; Christian, et al.; Scott, et al.; DeGraaf, et al.; Folgori, et al.) and screened by HGV-associated human sera or the antibodies of the present invention.</div>
    <div class="description-paragraph style-scope patent-text">One example of the use of RPL for isolation of 470-20-1 mimotopes is as follows. The random decapeptide-pIII fusion phage display library is constructed according to the methods described previously (DeGraaf, et al., 1993). Briefly, a chemically synthesized single-stranded degenerate insert is annealed to shorter oligonucleotides which generate SfiI restriction overhangs. Annealed DNA is ligated into SfiI-cut fUSE-5 vector DNA.</div>
    <div class="description-paragraph style-scope patent-text">E. coll MC1061 is transformed with the ligated DNA. The library is amplified through approximately ten population doublings in LB medium with 20 mg/ml tetracycline. This library is affinity selected using one or more of 470-20-1 immunoreactive sera (or antibodies of the present invention). Polystyrene beads (Precision Plastic Ball Company, Chicago. Ill.) are coated with ammonium sulfate fractionated positive serum (e.g., PNF 2161) in 50 mM NaHCO3, pH 9.6 overnight at 4° C. Antibody coated beads are thoroughly washed with PBS and blocked with BSA.</div>
    <div class="description-paragraph style-scope patent-text">These serum coated, blocked beads are pre-incubated with an excess of M13K07-UV killed phage for 4 hours at 4° C. Library phage are then added to the above pre-incubation mixture and incubated for 12 hours at 4° C. Unbound phage are removed and the beads are washed extensively with TTB (50 mM Tris, pH 7.5, 150 mM NaCl, 0.5% "<figure-callout id="20" label="TWEEN" filenames="US5874563-drawings-page-4.png,US5874563-drawings-page-5.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">TWEEN</a></figure-callout> 20" (v/v), 1 mg/ml BSA) buffer. Bound phage are eluted with elution buffer (0.1M HCl adjusted to pH 2.2 with 2M Tris-HCl, pH 9.0). Eluted, enriched phage are screened with a second positive serum (e.g., Mys 136 sera) by plaque immunoscreening.</div>
    <div class="description-paragraph style-scope patent-text">Further screening of the selected phagotopes can be carried out using large panels of positive and negative sera or specific HGV monoclonal antibodies. Selected phagotopes can be used directly in ELISA assay or antibody generation. Alternatively, the sequences of the phagotope encoding nucleotides can be determined and expressed in conventional vector/host system and used as antigen.</div>
    <div class="description-paragraph style-scope patent-text">Mimic polypeptides identified as described above can in turn can serve as antigens in detection assays or can be used for the generation of antigen-specific antibodies.</div>
    <div class="description-paragraph style-scope patent-text">D. Elisa and Protein Blot Screening.</div>
    <div class="description-paragraph style-scope patent-text">When HGV antigens are identified, typically through plaque immunoscreening as described above, the antigens can be expressed and purified. The antigens can then be screened rapidly against a large number of suspected HGV hepatitis sera using alternative immunoassays, such as, ELISAs or Protein Blot Assays (Western blots) employing the isolated antigen peptide. The antigen polypeptides fusion can be isolated as described above, usually by affinity chromatography to the fusion partner such as β-galactosidase or glutathione-S-transferase. Alternatively, the antigen itself can be purified using antibodies generated against it (see below).</div>
    <div class="description-paragraph style-scope patent-text">A general ELISA assay format is presented in Example 10. Harlow, et al., describe a number of useful techniques for immunoassays and antibody/antigen screening.</div>
    <div class="description-paragraph style-scope patent-text">The purified antigen polypeptide or fusion polypeptide containing the antigen of interest, is attached to a solid support, for example, a multiwell polystyrene plate. Sera to be tested are diluted and added to the wells. After a period of time sufficient for the binding of antibodies to the bound antigens, the sera are washed out of the wells. A labelled reporter antibody is added to each well along with an appropriate substrate: wells containing antibodies bound to the purified antigen polypeptide or fusion polypeptide containing the antigen are detected by a positive signal.</div>
    <div class="description-paragraph style-scope patent-text">A typical format for protein blot analysis using the polypeptide antigens of the present invention is presented in Example 10. General protein blotting methods are described by Ausubel, et al. In Example 10, the 470-20-1/sj26 fusion protein was used to screen a number of sera samples. The results presented in Example 10 demonstrate that several different source N-(ABCDE) hepatitis sera are immunoreactive with the polypeptide antigen.</div>
    <div class="description-paragraph style-scope patent-text">The results presented above demonstrate that the polypeptide antigens of the present invention can, by these methods, be rapidly screened against panels of suspected HGV infected serum samples for the detection of HGV.</div>
    <div class="description-paragraph style-scope patent-text">E. Cell Culture Systems, Animal Models and Isolation of HGV.</div>
    <div class="description-paragraph style-scope patent-text">HGV infectivity studies have been carried out in chimpanzees, cynomolgus monkey and four mystax subjects (Example 4H). These studies have yielded further information about HGV infectivity in these animal models. The HGV described in the present specification have the advantage of being capable of infecting tamarins, cynomologous monkeys and chimpanzees.</div>
    <div class="description-paragraph style-scope patent-text">Alternatively, primary hepatocytes obtained from infected animals (chimpanzees, baboons, monkeys, or humans) can be cultured in vitro. A serum-free medium, supplemented with growth factors and hormones, has been described which permits the long-term maintenance of differentiated primate hepatocytes (Lanford, et al.; Jacob, et al., 1989, 1990, 1991). In addition to primary hepatocyte cultures, immortalized cultures of infected cells may also be generated. For example, primary liver cultures may be fused to a variety of cells (like HepG2) to provide stable immortalized cell lines. Primary hepatocyte cell cultures may also be immortalized by introduction of oncogenes or genes causing a transformed phenotype. Such oncogenes or genes can be derived from a number of sources known in the art including SV40, human cellular oncogenes and Epstein Barr Virus.</div>
    <div class="description-paragraph style-scope patent-text">Further, the un-infected hepatocytes (e.g., primary or continuous hepatoma cell lines) may be infected by exposing the cells in culture to the HGV either as partially purified particle preparations (prepared, for example, from infected sera by differential centrifugation and/or molecular sieving) or in infectious sera. These infected cells can then be propagated and the virus passaged by methods known in the art. In addition, other cell types, such as lymphoid cell lines, may be useful for the propagation of HGV.</div>
    <div class="description-paragraph style-scope patent-text">Protein similarity studies of HGV have detected amino acid regions similar to other viruses in the family Flaviviridae. It is known that members of this family of viruses can be propagated in a variety of tissue culture systems (ATCC-Viruses catalogue, 1990). By analogy it is likely that HGV can be propagated in one or more of the following tissue culture systems: Hela cells, primary hamster kidney cells, monkey kidney cells, vero cells, LLC-MK2 (rhesus monkey kidney cells), KB cells(human oral epidermoid carcinoma cells), duck embryo cells, primary sheep leptomeningeal cells, primary sheep choroid plexus cells, pig kidney cells, bovine embryonic kidney cells, bovine turbinate cells, chick embryo cells, primary rabbit kidney cells, BHD-21 cells, or PK-13 cells.</div>
    <div class="description-paragraph style-scope patent-text">In addition to expression of HGV, regions of HGV polynucleotide sequences, cDNA or in vitro transcribed RNA can be introduced by recombinant means into tissue culture cells. Such recombinant manipulations allow the individual expression of individual components of the HGV.</div>
    <div class="description-paragraph style-scope patent-text">RNA samples can be prepared from infected tissue or, in particular, from infected cell cultures. The RNA samples can be fractionated on gels and transferred to membranes for hybridization analysis using probes derived from the cloned HGV sequences.</div>
    <div class="description-paragraph style-scope patent-text">HGV particles may be isolated from infected sera, infected tissue, the above-described cell culture media, or the cultured infected cells by methods known in the art. Such methods include techniques based on size fractionation (i.e., ultrafiltration, precipitation, sedimentation), using anionic and/or cationic exchange materials, separation on the basis of density, hydrophilic properties, and affinity chromatography. During the isolation procedure the HGV can be identified (i) using the anti-HGV hepatitis associated agent antibodies of the present invention, (ii) by using hybridization probes based on identified HGV nucleic acid sequences (e.g., Example 5) or (iii) by RT-PCR.</div>
    <div class="description-paragraph style-scope patent-text">Antibodies directed against HGV can be used in purification of HGV particles through immunoaffinity chromatography (Harlow, et al.; Pierce). Antibodies directed against HGV polypeptides or fusion polypeptides (such as 470-20-1) are fixed to solid supports in such a manner that the antibodies maintain their immunoselectivity. To accomplish such attachment of antibodies to solid support bifunctional coupling agents (Pierce; Pharmacia, Piscataway, N.J.) containing spacer groups are frequently used to retain accessibility of the antigen binding site of the antibody.</div>
    <div class="description-paragraph style-scope patent-text">HGV particles can be further characterized by standard procedures including, but not limited to, immunofluorescence microscopy, electron microscopy, Western blot analysis of proteins composing the particles, infection studies in animal and/or cell systems utilizing the partially purified particles, and sedimentation characteristics. The results presented in Example 5 suggest that the viral particle of the present invention is more similar to an enveloped viral particle than to a non-enveloped viral particle.</div>
    <div class="description-paragraph style-scope patent-text">HGV particles can be disrupted to obtain HGV genomes. Disruption of the particles can be achieved by, for example, treatment with detergents in the presence of chelating agents. The genomic nucleic acid can then be further characterized. Characterization may include analysis of DNase and RNase sensitivity. The strandedness (Example 41) and conformation (e.g., circular) of the genome can be determined by techniques known in the art, including visualization by electron microscopy and sedimentation characteristics.</div>
    <div class="description-paragraph style-scope patent-text">The isolated genomes also make it possible to sequence the entire genome whether it is segmented or not, and whether it is an RNA or DNA genome (using, for example RT-PCR, chromosome walking techniques, or PCR which utilizes primers from adjacent cloned sequences). Determination of the entire sequence of HGV allows genomic organization studies and the comparison of the HGV sequences to the coding and regulatory sequences of known viral agents.</div>
    <div class="description-paragraph style-scope patent-text">F. Screening for Agents Having Anti-HGV Hepatitis Activity.</div>
    <div class="description-paragraph style-scope patent-text">The use of cell culture and animal model systems for propagation of HGV provides the ability to screen for anti-hepatitis agents which inhibit the production of infectious HGV: in particular, drugs that inhibit the replication of HGV. Cell culture and animal models allow the evaluation of the effect of such anti-hepatitis drugs on normal cellular functions and viability. Potential anti-viral agents (including natural products or synthetic compounds; for example, small molecules, complex mixtures such as fungal extracts, and anti-sense oligonucleotides) are typically screened for anti-viral activity over a range of concentrations. The effect on HGV replication and/or antigen production is then evaluated, typically by monitering viral macromolecular synthesis or accumulation of macromolecules (e.g., DNA, RNA or protein). This evaluation is often made relative to the effect of the anti-viral agent on normal cellular function (DNA replication, RNA transcription, general protein translation, etc.).</div>
    <div class="description-paragraph style-scope patent-text">The detection of the HGV can be accomplished by many methods including those described in the present specification. For example, antibodies can be generated against the antigens of the present invention and these antibodies used in antibody-based assays (Harlow, et al.) to identify and quantitate HGV antigens in cell culture. HGV antigens can be quantitated in culture using competition assays: polypeptides encoded by the cloned HGV sequences can be used in such assays. Typically, a recombinantly produced HGV antigenic polypeptide is produced and used to generate a monoclonal or polyclonal antibody. The recombinant HGV polypeptide is labelled using a reporter molecule. The inhibition of binding of this labelled polypeptide to its cognate antibody is then evaluated in the presence of samples (e.g., cell culture media or sera) that contain HGV antigens. The level of HGV antigens in the sample is determined by comparison of levels of inhibition to a standard curve generated using unlabelled recombinant proteins at known concentrations.</div>
    <div class="description-paragraph style-scope patent-text">The HGV sequences of the present invention are particularly useful for the generation of polynucleotide probes/primers that may be used to quantitate the amount of HGV nucleic acid sequences produced in a cell culture system. Such quantification can be accomplished in a number of ways. For example, probes labelled with reporter molecules can be used in standard dot-blot hybridizations or competition assays of labelled probes with infected cell nucleic acids. Further, there are a number of methods using the polymerase chain reaction to quantitate target nucleic acid levels in a sample (Osikowicz, et al.).</div>
    <div class="description-paragraph style-scope patent-text">Protective antibodies can also be identified using the cell culture and animal model systems described above. For example, polyclonal or monoclonal antibodies are generated against the antigens of the present invention. These antibodies are then used to pre-treat an infectious HGV-containing inoculum (e.g., serum) before infection of cell cultures or animals. The ability of a single antibody or mixtures of antibodies to protect the cell culture or animal from infection is evaluated. For example, in cell culture and animals the absence of viral antigen and/or nucleic acid production serves as a screen. Further in animals, the absence of HGV hepatitis disease symptoms, e.g., elevated ALT values, is also indicative of the presence of protective antibodies.</div>
    <div class="description-paragraph style-scope patent-text">Alternatively, convalescent sera can be screened for the presence of protective antibodies and then these sera used to identify HGV hepatitis associated agent antigens that bind with the antibodies. The identified HGV antigen is then recombinantly or synthetically produced. The ability of the antigen to generate protective antibodies is tested as above.</div>
    <div class="description-paragraph style-scope patent-text">After initial screening, the antigen or antigens identified as capable of generating protective antibodies, either singly or in combination, can be used as a vaccine to inoculate test animals. The animals are then challenged with infectious HGV. Protection from infection indicates the ability of the animals to generate antibodies that protect them from infection. Further, use of the animal models allows identification of antigens that activate cellular immunity.</div>
    <div class="description-paragraph style-scope patent-text">In animal model studies, a protective immune response in response to challenge by a viral preparation (e.g., infected serum) (i) protects the animal from infection or (ii) prevents manifestation of disease.</div>
    <div class="description-paragraph style-scope patent-text">G. Vaccines and the Generation of Protective Immunity.</div>
    <div class="description-paragraph style-scope patent-text">Vaccines can be prepared from one or more of the immunogenic polypeptides identified by the method of the present invention. Genomic organization similarities between the isolated sequences from HGV and other known viral proteins may provide information concerning the polypeptides that are likely to be candidates for effective vaccines. In addition, a number of computer programs can be used for to identify likely regions of isolated sequences that encode protein antigenic determinant regions (for example, Hopp, et al.; "ANTIGEN," Intelligenetics, Mountain View Calif.).</div>
    <div class="description-paragraph style-scope patent-text">Vaccines containing immunogenic polypeptides as active ingredients are typically prepared as injectables either as solutions or suspensions. Further, the immunogenic polypeptides may be prepared in a solid or lyophilized state that is suitable for resuspension, prior to injection, in an aqueous form. The immunogenic polypeptides may also be emulsified or encapsulated in liposomes. The polypeptides are frequently mixed with pharmaceutically acceptable excipients that are compatible with the polypeptides. Such excipients include, but are not limited to, the following and combinations of the following: saline, water, sugars (such as dextrose and sorbitol), glycerol, alcohols (such as ethanol  EtOH!), and others known in the art. Further, vaccine preparations may contain minor amounts of other auxiliary substances such as wetting agents, emulsifying agents (e.g., detergents), and pH buffering agents. In addition, a number of adjuvants are available which may enhance the effectiveness of vaccine preparations. Examples of such adjuvants include, but are not limited to, the following: the group of related compounds including N-acetyl-muranyl-L-threonyl-D-isoglutamine and N-acetyl-nor-muranyl-L-alanyl-D-isoglutamine, and aluminum hydroxide.</div>
    <div class="description-paragraph style-scope patent-text">The immunogenic polypeptides used in the vaccines of the present invention may be recombinant, synthetic or isolated from, for example, attenuated HGV particles. The polypeptides are commonly formulated into vaccines in neutral or salt forms. Pharmaceutically acceptable organic and inorganic salts are well known in the art.</div>
    <div class="description-paragraph style-scope patent-text">HGV hepatitis associated agent vaccines are parenterally administered, typically by subcutaneous or intramuscular injection. Other possible formulations include oral and suppository formulations. Oral formulations commonly employ excipients (e.g., pharmaceutical grade sugars, saccharine, cellulose, and the like) and usually contain within 10-98% immunogenic polypeptide. Oral compositions take the form of pills, capsules, tablets, solutions, suspensions, powders, etc., and may be formulated to allow sustained or long-term release. Suppository formulations use traditional binders and carriers and typically contain between 0.1% and 10% of the immunogenic polypeptide.</div>
    <div class="description-paragraph style-scope patent-text">In view of the above information, multivalent vaccines against HGV hepatitis associated agents can be generated which are composed of one or more structural or non-structural viral-agent polypeptide(s). These vaccines can contain, for example, recombinant expressed HGV polypeptides, polypeptides isolated from HGV virions, synthetic polypeptides or assembled epitopes in the form of mosaic polypeptides. In addition, it may be possible to prepare vaccines, which confer protection against HGV hepatitis infection through the use of inactivated HGV. Such inactivation might be achieved by preparation of viral lysates followed by treatment of the lysates with appropriate organic solvents, detergents or formalin.</div>
    <div class="description-paragraph style-scope patent-text">Vaccines may also be prepared from attenuated HGV strains. Such attenuated HGV may be obtained utilizing the above described cell culture and/or animal model systems. Typically, attenuated strains are isolated after multiple passages in vitro or in vivo. Detection of attenuated strains is accomplished by methods known in the art. One method for detecting attenuated HGV is the use of antibody probes against HGV antigens, sequence-specific hybridization probes, or amplification with sequence-specific primers for infected animals or assay of HGV-infected in vitro cultures.</div>
    <div class="description-paragraph style-scope patent-text">Alternatively, or in addition to the above methods, attenuated HGV strains may be constructed based on the genomic information that can be obtained from the information presented in the present specification. Typically, a region of the infectious agent genome that encodes, for example, a polypeptide that is related to viral pathogenesis can be deleted. The deletion should not interfere with viral replication. Further, the recombinant attenuated HGV construct allows the expression of an epitope or epitopes that are capable of giving rise to protective immune responses against the HGV. The desired immune response may include both humeral and cellular immunity.The genome of the attenuated HGV is then used to transform cells and the cells grown under conditions that allow viral replication. Such attenuated strains are useful not only as vaccines, but also as production sources of viral antigens and/or HGV particles.</div>
    <div class="description-paragraph style-scope patent-text">Hybrid particle immunogens that contain HGV epitopes can also be generated. The immunogenicity of HGV epitopes may be enhanced by expressing the epitope in eucaryotic systems (e.g., mammalian or yeast systems) where the epitope is fused or assembled with known particle forming proteins. One such protein is the hepatitis B surface antigen. Recombinant constructs where the HGV epitope is directly linked to coding sequence for the particle forming protein will produce hybrid proteins that are immunogenic with respect to the HGV epitope and the particle forming protein. Alternatively, selected portions of the particle-forming protein coding sequence, which are not involved in particle formation, may be replaced with coding sequences corresponding to HGV epitopes. For example, regions of specific immunoreactivity to the particle-forming protein can be replaced by HGV epitope sequences.</div>
    <div class="description-paragraph style-scope patent-text">The hepatitis B surface antigen has been shown to be expressed and assembled into particles in the yeast Saccharomyces cerevisiea and in mammalian cells (Valenzuela, et al., 1982 and 1984; Michelle, et al.). These particles have been shown to have enhanced immunoreactivity. Formation of these particles using hybrid proteins, i.e., recombinant constructs with heterologous viral sequences, has been previously disclosed (EPO 175,261, published 26 Mar. 1986). Such hybrid particles containing HGV epitopes may also be useful in vaccine applications.</div>
    <div class="description-paragraph style-scope patent-text">The vaccines of the present invention are administered in dosages compatible with the method of formulation, and in such amounts that will be pharmacologically effective for prophylactic or therapeutic treatments. The quantity of immunogen administered depends on the subject being treated, the capacity of the treatment subject's immune system for generation of protective immune response, and the desired level of protection.</div>
    <div class="description-paragraph style-scope patent-text">HGV vaccines of the present invention can be administered in single or multiple doses. Dosage regimens are also determined relative to the treatment subject's needs and tolerances. In addition to the HGV immunogenic polypeptides, vaccine formulations may be administered in conjunction with other immunoregulatory agents.</div>
    <div class="description-paragraph style-scope patent-text">In an additional approach to HGV vaccination, DNA constructs encoding HGV proteins under appropriate regulatory control are introduced directly into mammalian tissue, in vivo. Introduction of such constructs produces "genetic immunization". Similar DNA constructs have been shown to be taken up by cells and the encoded proteins expressed (Wolf, et al.; Ascadi, et al.). Injected DNA does not appear to integrate into host cells chromatin or replicate. This expression gives rise to substantial humoral and cellular immune responses, including protection from in vivo viral challenge in animal systems (Wang, et al., 1993; Ulmer, et al.). In one embodiment, the DNA construct is injected into skeletal muscle following pre-treatment with local anesthetics, such as, bupivicaine hydrochloride with methylparaben in isotonic saline, to facilitate cellular DNA uptake. The injected DNA constructs are taken up by muscle cells and the encoded proteins expressed.</div>
    <div class="description-paragraph style-scope patent-text">Compared to vaccination with soluble viral subunit proteins, genetic immunization has the advantage of authentic in vivo expression of the viral proteins. These viral proteins are expressed in association with host cell histocompatibility antigens, and other proteins, as would occur with natural viral infection. This type of immunization is capable of inducing both humoral and cellular immune responses, in contrast to many soluble subunit protein vaccines. Accordingly, this type of immunization retains many of the beneficial features of live attenuated vaccines, without the use of infectious agents for vaccination and attendant safety concerns.</div>
    <div class="description-paragraph style-scope patent-text">Direct injection of plasmid or other DNA constructs encoding the desired vaccine antigens into in vivo tissues is one delivery means. Other means of delivery of the DNA constructs can be employed as well. These include a variety of lipid-based approaches in which the DNA is packaged using liposomes, cationic lipid reagents or cytofectins (such as, lipofectin). These approaches facilitate in vivo uptake and expression, as summarized by Felgner and Rhodes (1991). Various modifications to these basic approaches include the following: incorporation of peptides, or other moieties, to facilitate (i) targeting to particular cells, (ii) the intracellular disposition of the DNA construct following uptake, or (iii) to facilitate expression. Alternatively, the sequences encoding the desired vaccine antigens may be inserted into a suitable retroviral vector. The resulting recombinant retroviral vector inoculated into the subject for in vivo expression of the vaccine antigen. The antigen then induces the immune responses. As noted above, this approach has been shown to induce both humoral and cellular immunity to viral antigens (Irwin, et al.).</div>
    <div class="description-paragraph style-scope patent-text">Further, the HGV vaccines of the present invention may be administered in combination with other vaccine agents, for example, with other hepatitis vaccines.</div>
    <div class="description-paragraph style-scope patent-text">H. Synthetic Peptides.</div>
    <div class="description-paragraph style-scope patent-text">Using the coding sequences of HGV polypeptide, synthetic peptides can be generated which correspond to these polypeptides. Synthetic peptides can be commercially synthesized or prepared using standard methods and apparatus in the art (Applied Biosystems, Foster City Calif.).</div>
    <div class="description-paragraph style-scope patent-text">Alternatively, oligonucleotide sequences encoding peptides can be either synthesized directly by standard methods of oligonucleotide synthesis, or, in the case of large coding sequences, synthesized by a series of cloning steps involving a tandem array of multiple oligonucleotide fragments corresponding to the coding sequence (Crea; Yoshio et al.; Eaton et al.). oligonucleotide coding sequences can be expressed by standard recombinant procedures (Maniatis et al.; Ausubel et al.).</div>
    <heading class="style-scope patent-text">V. Charcterization of the Viral Genome</heading>
    <div class="description-paragraph style-scope patent-text">As shown in Example 4, the HGV genome appears to be an RNA molecule and has the closest sequence similarity to viral sequences that are catagorized in the Flaviviridae family of viruses. This family includes the Flaviviruses, Pestiviruses and an unclassified Genus made up of one member, Hepatitis C virus. The HGV virus does not have significant global (i.e., over the length of the virus) sequence identity with other recognized members of the Flaviviridae--with the exception of the protein motifs discussed below.</div>
    <div class="description-paragraph style-scope patent-text">In general members of the Flaviviridae are enveloped viruses that have densities in sucrose gradients between 1.1 and 1.23 g/ml and are sensitive to heat, organic solvents and detergents. As shown in Example 5, HGV has density characteristics similar to an enveloped Flaviviridae virus (HCV). The integrity of the HGV virion also appears to be sensitive to organic solvents (Example 5).</div>
    <div class="description-paragraph style-scope patent-text">Flaviviridae virions contain a single molecule of linear single-stranded (ss) RNA which also serves as the only mRNA that codes for the viral proteins. The ssRNA molecule is typically between the size of 9 and 12 kilobases long.</div>
    <div class="description-paragraph style-scope patent-text">Viral proteins are derived from one polyprotein precursor that is subsequently processed to the mature viral proteins. Most members of the Flaviviridae do not contain poly(A) tails at their 3' ends. Virions are about 15-20% lipid by weight.</div>
    <div class="description-paragraph style-scope patent-text">Members in the Flaviviridae family have a core protein and two or three membrane-associated proteins. The analogous structural proteins of members in the three genera Flavivirus family show little similarity to one another at the sequence level. The nonstructural proteins contain conserved motifs for RNA dependent RNA polymerase (RDRP), helicase, and a serine protease. These short blocks of conserved amino acids or motifs can be detected using computer algorithms known in the art such as "MACAW" (Schuler, et al.). These motifs are presumably related to constraints imposed by substrates processed by these proteins (Koonin and Dolja). The order of these motifs is conserved in all members of the Flaviviridae family. The genome of HGV contains protein motifs found in members of the Flaviviridae family, for example, (i) the helicase gene, (ii) the serine-like protease domain, and (iii) the RNA dependent RNA polymerase (RDRP) of (see FIG. 5, "GDD" sequence);</div>
    <div class="description-paragraph style-scope patent-text">Sequence information is disclosed herein on several different strains/isolates of HGV. This information can be used by one skilled in the art to isolate new stains/isolates using the techniques of hybridization, primer extension, and RT-PCR as described herein (e.g., using degenerate primers based on the disclosed HGV variant sequences).</div>
    <div class="description-paragraph style-scope patent-text">In the present case, HGV is an new isolate believed to be a member of the family Flaviviridae. Within this virus family, examination of the structural proteins encoded by a virus allows the most definitive determination of whether a viral isolate is a member of a distinct species of virus. Non-structural proteins are most conserved between different species of viruses within a family of virus species. This is believed to be the result of the necessity for preserving enzymatic functions, such as, the following: the proteolytic cleavage of a viral polyprotein, and replication of the RNA genome by viral helicase and RNA dependent RNA polymerase of the virus.</div>
    <div class="description-paragraph style-scope patent-text">Examination of several species within any genus of the Flaviviridae family, e.g., the flavivirus genus, demonstrates that the genes for these conserved functions are more highly conserved between species than the structural proteins. Accordingly, one of the major determining factors of whether a virus isolate represents a new species, versus a "variant isolate" of a known species, is a determination of global homology of the structural proteins between known viral species and the new virus isolate.</div>
    <div class="description-paragraph style-scope patent-text">Local homologies found within regions about 200 amino acids or less which are found in non-structural proteins are indeterminant indicators of whether an isolate is a variant or a new species. Typically, virus isolates having global structural protein homologies of less than or about 40% are classified as either different species (viruses) or different genuses. The structural regions of HGV each have homologies lower than 40% compared with any virus described in "GENBANK" (comparisons carried out by methods standard in the art). Accordingly, HGV is considered to be a new species and possibly a new genus of positive strand RNA virus.</div>
    <div class="description-paragraph style-scope patent-text">Another important region that is examined in determining the phylogenetic placement of a viral isolate is the 5' and 3' untranslated regions (UTRs). These regions are compared between viral isolates. For example, all the members HCV, an unclassified genus of Flaviviridae, have 5' untranslated regions that are greater than about 90% conserved with all other members in the genus. Further, the members of the <figure-callout id="3'" label="HCV share" filenames="US5874563-drawings-page-3.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">HCV share</a></figure-callout> 3' untranslated regions between about 24 and about 50 nucleotides long.</div>
    <div class="description-paragraph style-scope patent-text">No significant alignments are found with any virus in "GENBANK" (Ver. 86) when the 5'-untranslated region is used as a query sequence with FASTA on BLASTN. Further, HGV contains a 3' untranslated region that is at least about 250 nucleotides long that also contains little homology to any other known virus.</div>
    <div class="description-paragraph style-scope patent-text">Members of the Flaviviridae family are known to replicate in a wide variety of animals ranging from (i) hematophagous arthropod vectors (ticks and mosquitoes), where they do not cause disease, to (ii) a large range of vertebrate hosts (humans, primates, other mammals, marsupials, and birds). Over 30 members of the Flaviviridae family cause diseases in man, ranging from febrile illness, or rash, to potentially fatal diseases such as hemorrhagic fever, encephalitis, or hepatitis. At least 10 members of the Flaviviridae family cause severe and economically important diseases in domestic animals.</div>
    <heading class="style-scope patent-text">VI. Utility</heading>
    <div class="description-paragraph style-scope patent-text">A. The Invention.</div>
    <div class="description-paragraph style-scope patent-text">In one aspect, the invention pertains to polynucleotides derived from a Hepatitis G Virus (HGV) polynucleotide in substantially isolated form. In one embodiment the HGV polynucleotide is characterized by (i) transmission in primates, (ii) serologically distinguishable from hepatitis A virus (HAV), hepatitis B virus (HBV), hepatitis C virus (HCV), hepatitis D virus, and hepatitis E virus (HEV), and (iii) membership of the virus family Flaviviridae. Polynucleotides of the invention may be comprised of DNA or RNA (or analogs or variants thereof) and may be produced recombinantly, isolated, or synthesized according to methods known in the art.</div>
    <div class="description-paragraph style-scope patent-text">Generally, HGV polynucleotides of the invention will be at least 10 nucleotides in length. In an alternative embodiment, the HGV polynucleotide will be at least 15 nucleotides in length. In still a further alternative embodiment, the HGV polynucleotide will be at least 20 nucleotides in length.</div>
    <div class="description-paragraph style-scope patent-text">In a more specific embodiment, polynucleotides of the invention include cDNA or CDNA complements of the HGV genome. In a more specific embodiment, such a CDNA or CDNA complement will have at least a 40% sequence homology to a polynucleotide selected from the group consisting of SEQ ID NO:14, SEQ ID NO:37, and SEQ ID NO:19, or complements thereof. In yet another embodiment such cDNA's will exhibit at least 55% sequence homology to a polynucleotide selected from the group consisting of SEQ ID NO:14, SEQ ID NO:37, and SEQ ID NO:19, or complements thereof. In more specific embodiments, CDNA or cDNA complement polynucleotides of the invention will have sequences derived from sequences selected from the group consisting of SEQ ID NO:14, SEQ ID NO:37, and SEQ ID NO:19, or complements thereof.</div>
    <div class="description-paragraph style-scope patent-text">In another general embodiment, polynucleotides of the invention are polynucleotide probes that specifically hybridize with HGV. In yet another general embodiment, polynucleotides of the invention will encode an epitope of HGV. More specifically, such epitope encoding polynucleotides may include sequences derived from SEQ ID NO:14, SEQ ID NO:19 or SEQ ID NO:37.</div>
    <div class="description-paragraph style-scope patent-text">In another general embodiment, the polynucleotide of the invention includes a contiguous sequence of nucleotides that is capable of selectively hybridizing to an HGV polynucleotide. In this regard, HGV is characterized as a genome comprising an open reading frame (ORF) encoding an amino acid sequence having at least 40% sequence homology to one of the following amino acid sequences: the 2873 amino acid sequence of SEQ ID NO:15, the 190 amino acid sequence of SEQ ID NO:38, or the 67 amino acid sequence of SEQ ID NO:20. More particularly, the polynucleotide probe will specifically hybridize with HGV. Such a polynucleotide probe may carry detection labels or other modifications or be fixed to a solid support.</div>
    <div class="description-paragraph style-scope patent-text">DNA polynucleotides as described above may also encode an HGV specifically immunoreactive antigenic determinants. In this regard, HGV is characterized as having a genome, cDNA or complements thereof comprising an open reading frame (ORF) encoding an amino acid sequence. Such, an amino acid sequence having at least 40% sequence homology to one of the following amino acid sequences: the 2873 amino acid sequence of SEQ ID NO:15, the 190 amino acid sequence of SEQ ID NO:38, or the 67 amino acid sequence of SEQ ID NO:20.</div>
    <div class="description-paragraph style-scope patent-text">In another specific embodiment, an HGV-encoding DNA polynucleotide that is specifically reactive with an HGV antigenic determinant will, in accordance with the invention, include an amino acid sequence having at least 55% sequence homology to the 2873 amino acid sequence of SEQ ID NO:15 or to the 190 amino acid sequence of SEQ ID NO:38 or to the 67 amino acid sequence of SEQ ID NO:20.</div>
    <div class="description-paragraph style-scope patent-text">In yet another specific embodiment, the DNA polynucleotide may exhibit at least 40% sequence homology to a polynucleotide selected from the group consisting of SEQ ID NO:14, SEQ ID NO:37, and SEQ ID NO:19, or complements thereof.</div>
    <div class="description-paragraph style-scope patent-text">In still a further embodiment, the invention includes a DNA polynucleotide that encodes an HGV-derived polypeptide. More particularly, the polypeptide encoded by the polynucleotide will include a contiguous sequence of at least 15-60 amino acids having 55% sequence homology to a contiguous sequence of at least 15-60 amino acids encoded by an HGV genome, CDNA or complements thereof.</div>
    <div class="description-paragraph style-scope patent-text">In a specific embodiment, HGV-polypeptide encoding polynucleotides may be encoded within the PNF 2161 cDNA source lambda gt11 library. In yet another specific embodiment, the DNA polynucleotide may encode an epitope of HGV. In still a further embodiment, the polynucleotide may be a probe that specifically hybridizes with HGV.</div>
    <div class="description-paragraph style-scope patent-text">In a related aspect, the invention includes a recombinant vector that contains a DNA polynucleotide that encodes an HGV polypeptide. In another related aspect, the invention includes a cell transformed with such a vector.</div>
    <div class="description-paragraph style-scope patent-text">In still another related aspect, the invention includes a polynucleotide probe that specifically hybridizes with an HGV hepatitis virus genome, cDNA or complements thereof. In a more specific embodiment, the polynucleotide probe sequence has at least 40% homology to a sequence derived from SEQ ID NO:19, SEQ ID NO:37, or SEQ ID NO:14, or complements thereof. In another specific embodiment, the polynucleotide probe is derived from SEQ ID NO:19, SEQ ID NO:37, or SEQ ID NO:14, or complements thereof.</div>
    <div class="description-paragraph style-scope patent-text">In another related aspect, the invention includes a method of detecting an HGV hepatitis virus nucleic acid in a test subject. According to the method a nucleic acid-containing sample is obtained from the subject. The sample is then combined with and at least one polynucleotide probe that specifically hybridizes with the HGV hepatitis viral genome. HGV nucleic acid/probe complexes, formed by hybridization of the HGV nucleic acid with probe, are then detected. Such detecting may be accomplished by hybridization of a probe containing at least one reporter moiety to the HGV nucleic acid.</div>
    <div class="description-paragraph style-scope patent-text">In a more specific embodiment, the above-described method includes the use of HGV nucleic acid specific probes where the two probes (primers) define an internal region of the HGV nucleic acid. In this embodiment, each probe has one strand containing a 3'-end internal to the HGV nucleic acid internal region. The nucleic acid/probe hybridization complexes are then converted to double-strand probe containing fragments by primer extension reactions. Probe-containing fragments are amplified by successively repeating the steps of (i) denaturing the double-strand fragments to produce single-strand fragments, (ii) hybridizing the single strands with the probes to form strand/probe complexes, (iii) generating double-strand fragments from the strand/probe complexes in the presence of DNA polymerase and all four deoxyribonucleotides, and (iv) repeating steps (i) to (iii) until a desired degree of amplification has been achieved. Amplification products are then identified according to established procedures. The method of the invention may further include a third polynucleotide probe capable of selectively hybridizing to the internal region described above but not to the specific probe/primer sequences used for amplification.</div>
    <div class="description-paragraph style-scope patent-text">In another specific embodiment, detection of HGV nucleic acid/probe complexes is accomplished by a target amplification method, such as by self-sustained sequence replication, ligase chain reaction, or strand displacement amplification. In a further specific embodiment detection is accomplished employing a signal amplification technique such as branch-chained DNA probes or the Q-beta replicase method.</div>
    <div class="description-paragraph style-scope patent-text">In still another related aspect, the invention includes a kit for analyzing samples for the presence of polynucleotides derived HGV hepatitis virus. In a general embodiment, the kit includes at least one polynucleotide probe containing a nucleotide sequence that will specifically hybridize with an HGV polynucleotide and a suitable container. In a specific embodiment, the kit includes two polynucleotide probes defining an internal region of the HGV polynucleotide, where each probe has one strand containing a 3'-end internal to the region. In a further embodiment, the probes may be useful as primers for polymerase chain reaction amplification.</div>
    <div class="description-paragraph style-scope patent-text">In still a further related aspect, the invention includes the HGV hepatitis virus particle in substantially isolated form.</div>
    <div class="description-paragraph style-scope patent-text">The invention also includes a polypeptide or a preparation of polypeptides from the HGV hepatitis virus in substantially isolated form. In this regard, the HGV virus is characterized as follows: (i) it is transmissible in primates; (ii) it is serologically distinct from hepatitis A virus (HAV), hepatitis B virus (HBV), hepatitis C virus (HCV), hepatitis D virus, and hepatitis E virus (HEV); and (iii) it is a member of the virus family Flaviviridae. HGV polypeptides, as defined above, may be prepared by conventional means, including chemical synthesis and recombinant DNA expression. Such polypeptides may also be fixed to a solid phase.</div>
    <div class="description-paragraph style-scope patent-text">In a specific embodiment the polypeptide is specifically immunoreactive with at least one anti-HGV antibody. In still a further specific embodiment, the polypeptide comprises an antigenic determinant specifically immunoreactive with HGV. In this context, HGV is characterized by having a genome comprising an open reading frame (ORF) encoding an amino acid sequence having at least 40% sequence homology to the 2873 amino acid sequence of SEQ ID NO:15 or to the 190 amino acid sequence of SEQ ID NO:38 or to the 67 amino acid sequence of SEQ ID NO:20. In a more specific embodiment, the ORF encodes amino acid sequence has at least 55% sequence homology to one of the aforementioned amino acid sequences. In still a further embodiment, the polypeptide sequence is derived from the 2873 amino acid sequence of SEQ ID NO:15, or fragments thereof, the 190 amino acid sequence of SEQ ID NO:38, or fragments thereof, or the 67 amino acid sequence of SEQ ID NO:20, or fragments thereof.</div>
    <div class="description-paragraph style-scope patent-text">In another specific embodiment, the polypeptide from the HGV hepatitis virus includes a contiguous sequence of at least about 60 amino acids encoded by an HGV genome, cDNA or complements thereof. More specifically, such peptide sequence may be encoded by the PNF 2161 cDNA source lambda gt11 library.</div>
    <div class="description-paragraph style-scope patent-text">Recombinantly expressed HGV polypeptides may, in a more specific embodiment, include a polypeptide sequence derived from SEQ ID NO:20, SEQ ID NO:38, or SEQ ID NO:15. In another embodiment such a polypeptide may be encoded by a sequence derived from SEQ ID NO:14, or from the complement of SEQ ID NO:14.</div>
    <div class="description-paragraph style-scope patent-text">In a further related embodiment, in accordance with the invention, an HGV hepatitis virus polypeptide may be a fusion polypeptide comprising an HGV polypeptide and a second polypeptide. More specifically, such a fusion polypeptide may include, as a second polypeptide signal sequences, β-galactosidase or glutathione-S-transferase protein sequences. Alternatively, the second polypeptide may comprise a particle forming protein.</div>
    <div class="description-paragraph style-scope patent-text">The above-described polypeptides may be derived from structural or non-structural viral proteins.</div>
    <div class="description-paragraph style-scope patent-text">In still a further related aspect, the invention includes a cloning vector capable of expressing, under suitable conditions, an open reading frame (ORF) of cDNA derived from HGV hepatitis virus genome, cDNA or complements thereof. In this aspect of the invention, the ORF is operably linked to a control sequence compatible with a desired host. In a related aspect, the invention includes a cell transformed with such a vector. In a more specific embodiment of the vector, the ORF may be derived from SEQ ID NO:14 or its complement. In yet further specific embodiments, the ORF may be derived from SEQ ID NO:37 or SEQ ID NO:19.</div>
    <div class="description-paragraph style-scope patent-text">In a related aspect, the invention includes a method of producing an HGV hepatitis virus polypeptide. The method includes culturing cells containing the above-described vectors under conditions suitable to achieve expression of the open reading frame (ORF) sequence. In a more specific embodiment, the ORF sequence encodes a polypeptide sequence selected from the group of polypeptide sequences, or fragments thereof, consisting of SEQ ID NO:15, SEQ ID NO:38 and SEQ ID NO:20. Further, the ORF sequences may be derived from an HGV cDNA, or complement thereof. In yet another specific embodiment, the vector is a lambda gt11 phage vector expressed in Escherichia coil cells.</div>
    <div class="description-paragraph style-scope patent-text">In a further related aspect, the invention includes a diagnostic kit for use in screening serum containing antibodies specific against HGV hepatitis virus infection. Such a kit may include a substantially isolated HGV polypeptide antigen comprising an epitope which is specifically immunoreactive with at least one anti-HGV antibody. Such a kit also includes means for detecting the binding of said antibody to the antigen. In regard to such a kit, HGV is characterized by having a genome, cDNA or complements thereof comprising an open reading frame (ORF) encoding an amino acid sequence. Such an amino acid sequence typically having at least 40% sequence homology to the 2873 amino acid sequence of SEQ ID NO:15 or to the 190 amino acid sequence of SEQ ID NO:38 or to the 67 amino acid sequence of SEQ ID NO:20. In specific embodiments, the kit may include a recombinantly produced or chemically synthesized polypeptide antigen. The polypeptide antigen of the kit may also be attached to a solid support.</div>
    <div class="description-paragraph style-scope patent-text">In a more specific embodiment, the detecting means of the above-described kit includes a solid support to which said polypeptide antigen is attached. Such a kit may also include a non-attached reporter-labelled anti-human antibody. In this embodiment, binding of the antibody to the HGV polypeptide antigen can be detected by binding of the reporter-labelled antibody the antibody.</div>
    <div class="description-paragraph style-scope patent-text">In a related aspect, the invention includes a method of detecting HGV hepatitis virus infection in a test subject. This detection method includes reacting serum from an HGV test subject with a substantially isolated HGV polypeptide antigen, and examining the antigen for the presence of bound antibody. In a specific embodiment, the method includes a polypeptide antigen attached to a solid support, and the serum is reacted with the support. Subsequently, the support is reacted with a reporter-labelled anti-human antibody. The solid support is then examined for the presence of reporter-labelled antibody.</div>
    <div class="description-paragraph style-scope patent-text">In a further aspect, the invention includes an HGV hepatitis virus vaccine composition. The composition includes a substantially isolated HGV polypeptide antigen, where the antigen includes an epitope which is specifically immunoreactive with at least one anti-HGV antibody. The peptide antigen may be produced according to methods known in the art, including recombinant expression or chemical synthesis. The peptide antigen is preferably present in a pharmacologically effective dose in a pharmaceutically acceptable carrier.</div>
    <div class="description-paragraph style-scope patent-text">In still a further related aspect, the invention includes a monoclonal antibody that is specifically immunoreactive with the HGV hepatitis virus epitope. In another related aspect, the invention includes a substantially isolated preparation of polyclonal antibodies specifically immunoreactive with HGV. In a more specific embodiment, such polyclonal antibodies are prepared by affinity chromatography.</div>
    <div class="description-paragraph style-scope patent-text">In a related aspect, the invention includes a method for producing antibodies to HGV. The method includes administering to a test subject a substantially isolated HGV polypeptide antigen, where the antigen includes an epitope which is specifically immunoreactive with at least one anti-HGV antibody. The antigen is administered in an amount sufficient to produce an immune response in the subject.</div>
    <div class="description-paragraph style-scope patent-text">In yet another related aspect, the invention includes a diagnostic kit for use in screening serum containing HGV antigens. The diagnostic kit includes a substantially isolated antibody specifically immunoreactive with an HGV polypeptide antigen, and means for detecting the binding of the polypeptide antigen to the antibody. In one embodiment, the antibody is attached to a solid support. In a specific embodiment, the antibody may be a monoclonal antibody. The detecting means of the kit may include a second, labelled monoclonal antibody. Alternatively, or in addition, the detecting means may include a labelled, competing antigen.</div>
    <div class="description-paragraph style-scope patent-text">In another, related aspect, the invention includes a method of detecting HGV infection in a test subject. According to this aspect of the invention, serum from a test subject is reacted with a substantially isolated HGV specific antibody of the kit described above. The HGV specific antibody is then examined for the presence of bound antigen.</div>
    <div class="description-paragraph style-scope patent-text">In still a further related aspect, the invention includes an in vitro grown cell infected with HGV. In a specific embodiment, the cell is a hepatocyte grown in tissue culture. More specifically, the tissue culture cell may be an immortalized hepatocyte, or it may be a from a cell line derived from liver of an HGV infected primate.</div>
    <div class="description-paragraph style-scope patent-text">In a related aspect, the invention includes a method of propagating HGV. The method includes culturing in vitro grown, HGV-infected cells, as described above, under conditions effective to promote the propagation of HGV. In another related aspect, the invention includes HGV particles produced by such a propagation method.</div>
    <div class="description-paragraph style-scope patent-text">In still a further aspect, the invention includes a mosaic polypeptide. Such a polypeptide may include at least two epitopes of HGV, where the polypeptide substantially lacks amino acids normally intervening between the epitopes in the native HGV coding sequence. In a more specific embodiment, the mosaic polypeptide is attached to a solid support. In still a further related aspect, the invention includes a nucleic acid that encodes the above-described mosaic polypeptide.</div>
    <div class="description-paragraph style-scope patent-text">In another related aspect, the invention includes a method of detecting HGV infection in a test subject. The method includes contacting an antibody-containing sample from the subject with a mosaic polypeptide, as described above, and examining the antigen for the presence of bound antibody.</div>
    <div class="description-paragraph style-scope patent-text">In still a further related aspect, the invention includes an HGV vaccine composition. The vaccine composition includes mosaic polypeptide that includes more than one HGV epitope. The mosaic polypeptide is present in a pharmacologically effective dose in a pharmaceutically acceptable carrier.</div>
    <div class="description-paragraph style-scope patent-text">B. Immunoassays for HGV.</div>
    <div class="description-paragraph style-scope patent-text">One utility for the antigens obtained by the methods of the present invention is their use as diagnostic reagents for the detection of antibodies present in the sera of test subjects infected with HGV hepatitis virus, thereby indicating infection in the subject; for example, 470-20-1 antigen, antigens encoded by SEQ ID NO:14 or its complement, and antigens encoded by portions of either strand of the complete viral sequence. The antigens of the present invention can be used singly, or in combination with each other, in order to detect HGV. The antigens of the present invention may also be coupled with diagnostic assays for other hepatitis agents such as HAV, HBV, HCV, and HEV.</div>
    <div class="description-paragraph style-scope patent-text">In one diagnostic configuration, test serum is reacted with a solid phase reagent having a surface-bound antigen obtained by the methods of the present invention, e.g., the 470-20-1 antigen. After binding with anti-HGV antibody to the reagent and removing unbound serum components by washing, the reagent is reacted with reporter-labelled anti-human antibody to bind reporter to the reagent in proportion to the amount of bound anti-HGV antibody on the solid support. The reagent is again washed to remove unbound labelled antibody, and the amount of reporter associated with the reagent is determined. Typically, the reporter is an enzyme which is detected by incubating the solid phase in the presence of a suitable fluorometric or calorimetric substrate (Sigma, St. Louis, Mo.).</div>
    <div class="description-paragraph style-scope patent-text">The solid surface reagent in the above assay is prepared by known techniques for attaching protein material to solid support material, such as polymeric beads, dip sticks, 96-well plate or filter material. These attachment methods generally include non-specific adsorption of the protein to the support or covalent attachment of the protein, typically through a free amine group, to a chemically reactive group on the solid support, such as an activated carboxyl, hydroxyl, or aldehyde group. Alternatively, streptavidin coated plates can be used in conjunction with biotinylated antigen(s).</div>
    <div class="description-paragraph style-scope patent-text">Also forming part of the invention is an assay system or kit for carrying out this diagnostic method. The kit generally includes a support with surface-bound recombinant HGV antigen (e.g., the 470-20-1 antigen, as above), and a reporter-labelled anti-human antibody for detecting surface-bound anti-HGV antigen antibody.</div>
    <div class="description-paragraph style-scope patent-text">In a second diagnostic configuration, known as a homogeneous assay, antibody binding to a solid support produces some change in the reaction medium which can be directly detected in the medium. Known general types of homogeneous assays proposed heretofore include (a) spin-labelled reporters, where antibody binding to the antigen is detected by a change in reported mobility (broadening of the spin splitting peaks), (b) fluorescent reporters, where binding is detected by a change in fluorescence efficiency or polarization, (c) enzyme reporters, where antibody binding causes enzyme/substrate interactions, and (d) liposome-bound reporters, where binding leads to liposome lysis and release of encapsulated reporter. The adaptation of these methods to the protein antigen of the present invention follows conventional methods for preparing homogeneous assay reagents.</div>
    <div class="description-paragraph style-scope patent-text">In each of the assays described above, the assay method involves reacting the serum from a test individual with the protein antigen and examining the antigen for the presence of bound antibody. The examining may involve attaching a labelled anti-human antibody to the antibody being examined (for example from acute, chronic or convalescent phase) and measuring the amount of reporter bound to the solid support, as in the first method, or may involve observing the effect of antibody binding on a homogeneous assay reagent, as in the second method.</div>
    <div class="description-paragraph style-scope patent-text">A third diagnostic configuration involves use of HGV antibodies capable of detecting HGV-specific antigens. The HGV antigens may be detected, for example, using an antigen capture assay where HGV antigens present in candidate serum samples are reacted with a HGV specific monoclonal or polyclonal antibody. The antibody is bound to a solid substrate and the antigen is then detected by a second, different labelled anti-HGV antibody. Antibodies can be prepared, utilizing the peptides of the present invention, by standard methods. Further, substantially isolated antibodies (essentially free of serum proteins which may affect reactivity) can be generated (e.g., affinity purification (Harlow et al.)).</div>
    <div class="description-paragraph style-scope patent-text">C. Hybridization Assays for HGV.</div>
    <div class="description-paragraph style-scope patent-text">One utility for the nucleic acid sequences obtained by the methods of the present invention is their use as diagnostic agents for HGV sequences present in sera, thereby indicating infection in the individual. Primers and/or probes derived from the coding sequences of the present invention, in particular, Clone 470-20-1 and SEQ ID NO:14, can be used singly, or in combination with each other, in order to detect HGV.</div>
    <div class="description-paragraph style-scope patent-text">In one diagnostic configuration, test serum is reacted under PCR or RT-PCR conditions using primers derived from, for example, 470-20-1 sequences. The presence of HGV, in the serum used in the amplification reaction, can be detected by specific amplification of the sequences targeted by the primers. Example 4 describes the use of polymerase chain amplification reactions, employing primers derived from the clones of the present invention, to screen different source material. The results of these amplification reactions demonstrate the ability of primers derived from the clones of the present invention (for example, 470-20-1), to detect homologous sequences by amplification reactions employing a variety of different source templates. The amplification reactions in Example 4 included use of nucleic acids obtained directly from sera as template material.</div>
    <div class="description-paragraph style-scope patent-text">Alternatively, probes can be derived from the HGV sequences of the present invention. These probes can then be labelled and used as hybridization probes against nucleic acids obtained from test serum or tissue samples. The probes can be labelled using a variety of reporter molecules and detected accordingly: for example, radioactive isotopic labelling and chemiluminescent detection reporter systems (Tropix, Bedford, Mass.).</div>
    <div class="description-paragraph style-scope patent-text">Target amplification methods, embodied by the polymerase chain reaction, the self-sustained sequence replication technique  "3SR," (Guatelli, et al.; Gingeras, et al., 1990) also known as "NASBA" (VanGemen, et al.)!, the ligase chain reaction (Barany), strand-displacement amplification  "SDA," (Walker)!, and other techniques, multiply the number of copies of the target sequence. Signal amplification techniques, exemplified by branched-chain DNA probes (Horn and Urdea; Urdea; Urdea, et al.) and the Q-beta replicase method (Cahill, et al.; Lomell, et al.), first bind a specific molecular probe, then replicate all of or part of this probe or in some other manner amplify the probe signal.</div>
    <div class="description-paragraph style-scope patent-text">For the detection of the specific nucleic acid sequences disclosed in the present invention or contiguous sequences in the same or a similar (related) viral genome, amplification and detection methodologies may be employed, as alternatives to amplification by the PCR. A number of such techniques are known to the field of nucleic acid diagnostics (The 1992 San Diego Conference: Genetic Recognition, Clin. Chem. 39(4):705 (1993)).</div>
    <div class="description-paragraph style-scope patent-text">1. Self-sustained Sequence Replication.</div>
    <div class="description-paragraph style-scope patent-text">The Self-Sustained Sequence Replication (3SR) technique results in amplification to a similar magnitude as PCR, but isothermally. Rather than thermal cycle-driven PCR, the 3SR operates as a concerted three-enzyme reaction of a) cDNA synthesis by reverse transcriptase, b) RNA strand degradation by RNase H, and c) RNA transcription by T7 RNA polymerase.</div>
    <div class="description-paragraph style-scope patent-text">As the entire reaction sequence occurs isothermally (typically at 42° C.), expensive temperature-cycling instrumentation is not required. In the absence of duplex denaturation via heating, organic solvents, or other mechanism, only single-stranded templates (i.e., predominantly RNA) are amplified.</div>
    <div class="description-paragraph style-scope patent-text">Suitable primers for use in 3SR amplification can be selected from the viral sequences of the present invention by those having ordinary skill in the art. For example, for isothermal amplification of viral sequences by the 3SR technique, primer 470-20-1-77F (SEQ ID NO:9) is modified by the addition of the T7 promoter sequence and a preferred T7 transcription initiation site to the 5'-end of the oligonucleotide. This modification results in a suitable 3SR primer T7-470-20-1-77F (SEQ ID NO:9). Primer 470-20-1-211R (SEQ ID NO:10) can be used in these reactions either without modification or T7 promoter.</div>
    <div class="description-paragraph style-scope patent-text">RNA extracted from PNF 2161 is incubated with AMV reverse transcriptase (30 U), RNase H (3 U), T7 RNA polymerase (100 U), in 100 ul reactions containing 20 mM Tris-HCl, pH 8.1 (at room temperature), 15 mM MgCl<sub class="style-scope patent-text">2</sub>, 10 mM KCl, 2 mM spermidine HCl, 5 mM dithiothreitol (DTT), 1 mM each of dATP, dCTP, dGTP, and TTP, 7 mM each of ATP, CTP, GTP, and UTP, and 0.15 uM each primer. Amplification takes place during incubation at 42° C. for 1-2 h.</div>
    <div class="description-paragraph style-scope patent-text">Initially, primer T7-470-20-1-77F anneals to the target RNA, and is extended by AMV reverse transcriptase to form cDNA complementary to the starting RNA strand. Following degradation of the RNA strand by RNase H, reverse transcriptase catalyzes the synthesis of the second strand DNA, resulting in a double-stranded template containing the (double-stranded) T7 promoter sequence. RNA transcription results in production of single-stranded RNA. This RNA then serves to re-enter the cycle for additional rounds of amplification, finally resulting in a pool of high-concentration product RNA. The product is predominantly single-stranded RNA of the same strand as the primer containing the T7 promoter (T7-470-20-1-77F), with much smaller amounts of cDNA.</div>
    <div class="description-paragraph style-scope patent-text">Alternatively, the other primer (470-20-1-211R) may contain the T7 promoter, or both primers may contain the promoter, resulting in production of both strands of RNA as products of the reaction. Products of the 3SR reaction may be detected, characterized, or quantitated by standard techniques for the analysis of RNA (e.g., Northern blots, RNA slot or dot blots, direct gel electrophoresis with RNA-staining dyes). Further, the products may be detected by methods making use of biotin-avidin affinity interactions or specific hybridizations of nucleic acid probes.</div>
    <div class="description-paragraph style-scope patent-text">In one technique for rapid and specific analysis of 3SR products, solution hybridization of the product to radiolabelled oligonucleotide 470-20-1-152R (SEQ ID NO:21) is followed by non-denaturing polyacrylamide gel electrophoresis. This assay (a gel mobility shift-type assay) results in the detection of specific probe-product hybrid as a slower-moving band than the band corresponding to unhybridized oligonucleotide.</div>
    <div class="description-paragraph style-scope patent-text">2. Ligase Chain Reaction (LCR)</div>
    <div class="description-paragraph style-scope patent-text">As another example of a detection system, the HGV sequence may form the basis for design of ligase chain reaction (LCR) primers. LCR makes use of the nick-closing activity of DNA ligase to join two immediately adjacent oligonucleotides possessing adjacent 5'-phosphate ("donor" oligo) and 3'-hydroxyl ("acceptor" oligo) terminii. The property of DNA ligase to join only fully complementary ends in a template-dependent way, leads to a high degree of specificity, in that ligation. will not occur unless the terminii to be linked are perfectly matched in sequence to the target strand.</div>
    <div class="description-paragraph style-scope patent-text">As an alternative to PCR, with some advantages in terms of specificity for discrimination of single base mismatches between primer and target nucleic acid, the LCR may be used to detect or "type" strains of virus possessing homology to HGV sequences. These techniques are suitable for assessing the presence of specific mutations when such base changes are known to confer drug resistance (e.g., Larder and Kemp; Gingeras, et al., 1991).</div>
    <div class="description-paragraph style-scope patent-text">In the presence of template-complementary donor and acceptor oligonucleotides and oligonucleotides complementary to the donor and acceptor, exponential amplification by LCR is possible. In this embodiment, each round of ligation generates additional template for subsequent rounds, in a cyclic reaction.</div>
    <div class="description-paragraph style-scope patent-text">For example, primer 470-20-1-211R (SEQ ID NO:10), an adjacent oligonucleotide (B, SEQ ID NO:22) and cognate oligos (211R', SEQ ID NO:23, and B', SEQ ID NO:24), can be used to perform LCR amplification of the sequence of this invention. Reverse transcription is first performed by standard methods to generate cDNA, which is then amplified in reactions containing 0.1-1 μM each of the four LCR primers, 20 mM Tris-HCl, pH 8.3 (room temperature), 25 mM KCl, 10 mM MgCl<sub class="style-scope patent-text">2</sub>, 10 mM dithiothreitol (DTT), 0.5 mM NAD+, 0.01% Triton X-100, and 5 Units of DNA ligase (Ampligase, Epicentre Technologies, Madison, Wis., or other commercial supplier of thermostable DNA ligase), in 25 ul reactions.</div>
    <div class="description-paragraph style-scope patent-text">Thermal cycling is performed at 94° C. for 1 min. 30 s; 94° C. for 1 min., 65° C. for 2 min., repeated for 25-40 cycles. Specificity of product synthesis depends on primer-template match at the 3'-terminal position. Products are detected by polyacrylamide gel electrophoresis, followed by ethidium bromide staining; alternatively, one of the acceptor oligos (211R' or B) is 5'-radiolabelled for visualization by autoradiography following gel electrophoresis.</div>
    <div class="description-paragraph style-scope patent-text">Alternatively, a donor oligo is 3'-end-labelled with a specific bindable moiety (e.g., biotin), and the acceptor is 5'-labelled with a specific detectable group (e.g., a fluorescent dye), for solid phase capture and detection.</div>
    <div class="description-paragraph style-scope patent-text">3. Methods for Analysis of Amplified DNA</div>
    <div class="description-paragraph style-scope patent-text">Numerous techniques have been described for the analysis of amplified DNA. Several such techniques are advantageous for high-throughput applications, where gel electrophoresis is impractical, for example, rapid and high-resolution HPLC techniques (Katz and Dong). However, in general, methods for infectious disease organism screening using nucleic acid probes involve a separate post-amplification hybridization step in order to assure requisite specificity for pathogen detection.</div>
    <div class="description-paragraph style-scope patent-text">One such detection embodiment is an affinity-based hybrid capture technique (Holodniy, et al.). In this embodiment the PCR is conducted with one biotinylated primer. Following amplification, the double-stranded product is denatured then hybridized to a peroxidase-labelled probe complementary to the strand having incorporated the biotinylated primer. The hybridized product is then incubated in a buffer which is in contact with an avidin (or streptavidin) coated surface (e.g., membrane filter, microwell, latex or paramagnetic beads).</div>
    <div class="description-paragraph style-scope patent-text">The mass of coated solid phase which contacts the volume of PCR product to be analyzed by this method must contain sufficient biotin-binding sites to capture essentially all of the free biotinylated primer, as well as the much lower concentration of biotinylated PCR product. Following three to four washes of the solid phase, bound hybridized product is detected by incubation with o-phenylenediamine in citrate buffer containing hydrogen peroxide.</div>
    <div class="description-paragraph style-scope patent-text">Alternatively, capture may be mediated by probe-coated surfaces, followed by affinity-based detection via the biotinylated primer and an avidin-reporter enzyme conjugate (Whetsell, et al.).</div>
    <div class="description-paragraph style-scope patent-text">4. Additional Methods</div>
    <div class="description-paragraph style-scope patent-text">Viral sequences of the present invention may also form the basis for a signal amplification approach to detection, using branched-chain DNA probes. Branched-chain probes (Horn and Urdea; Urdea) have been described for detection and quantification of rare RNA and DNA sequences (Urdea, et al.). In this method, an oligonucleotide probe (RNA, DNA, or nucleic acid analogue) is synthesized with a sequence complementary to the target RNA or DNA. The probe also contains a unique branching sequence or sequences not complementary to the target RNA or DNA.</div>
    <div class="description-paragraph style-scope patent-text">This unique sequence constitutes a target for hybridization of branched secondary detector probes, each of which contains one or more other unique sequences, serving as targets for tertiary probes. At each branch point in the signal amplification pathway, a different unique sequence directs hybridization of secondary, tertiary, etc., detection probes. The last probe in the series typically is linked to an enzyme useful for detection (e.g., alkaline phosphatase). The sequential hybridization of primers eventually results in the buildup of a highly-branched structure, the arms of which terminate in enzyme-linked probes.</div>
    <div class="description-paragraph style-scope patent-text">Enzymatic turnover provides a final amplification, and the choice of highly sensitive chemiluminescent substrates (e.g., LumiPhos, Lumigen, Detroit, Mich., as a substrate for alkaline phosphatase labels) results in exquisite sensitivity, on the order of 10,000 molecules or less of original target sequence per assay. In such a detection method, amplification depends only on molecular hybridization, rather than enzymatic mechanisms, and is thus far less susceptible to inhibitory substances in clinical specimens than, for example, PCR. Thus, this detection method allows the use of crude techniques for nucleic acid release in test samples, without extensive purification before assay.</div>
    <div class="description-paragraph style-scope patent-text">Amplification for sensitive detection of the viral sequences of the present invention may also be accomplished by the Q-β replicase technique (Cahill, et al.; Lomell, et al.; Pritchard, et al.). In this method, a specific probe is designed to be complementary to the target sequence. This probe is then inserted by standard molecular cloning techniques into the sequence of the replicatable RNA from Q-β phage. Insertion into a specific region of the replicon does not prevent replication by Q-β replicase.</div>
    <div class="description-paragraph style-scope patent-text">Following molecular hybridization, and several cycles of washing, the replicase is added and amplification of the probe RNA ensues. "Reversible target capture" is one known technique for reducing the potential background from replication of unhybridized probes (Morrissey, et al.). Amplified replicons are detectable by standard molecular hybridization techniques employing DNA, RNA or nucleic acid analogue probes.</div>
    <div class="description-paragraph style-scope patent-text">Additional methods for amplification and detection of rare DNA or RNA sequences are known in the literature and preferred to the PCR for some applications in the field of molecular diagnostics. These alternative techniques may form the basis for detection, characterization (e.g., sequence diversity existing as multiple related strains of the sequence described herein, genotypic changes characteristic of drug resistance), or quantification of the sequence disclosed in the present invention.</div>
    <div class="description-paragraph style-scope patent-text">Also forming part of the invention are assay systems or kits for carrying out the amplification/hybridization assay methods just described. Such kits generally include either specific primers for use in amplification reactions or hybridization probes.</div>
    <div class="description-paragraph style-scope patent-text">D. Therapeutic Uses.</div>
    <div class="description-paragraph style-scope patent-text">As discussed above, the HGV antigens of the present invention can be used in vaccine preparation.</div>
    <div class="description-paragraph style-scope patent-text">Further, antibodies generated against the polypeptide antigens of the present invention can be used for passive immunotherapy or passive immunoprophylaxis. The antibodies can be administered in amounts similar to those used for other therapeutic administrations of antibody. For example, pooled gamma globulin is administered at 0.02-0.1 ml/lb body weight during the early incubation of other viral diseases such as rabies, measles and hepatitis B to interfere with establishment of infection. Thus, antibodies reactive with the HGV antigens can be passively administered alone or in conjunction with another anti-viral agent to a host infected with HGV to enhance the ability of the host to deal with the infection.</div>
    <div class="description-paragraph style-scope patent-text">The HGV sequences disclosed herein identify HGV as a member of the Flaviviridae family (see above). The Flaviviridae are classified into 3 genera, flaviviruses, petstiviruses, and the hepatitis C virus genera (Francki, et al.). All Flaviviridae possess a positive strand RNA genome of 9.0-12 kb in length which encodes a single long polypeptide of 3000-4000 amino acids. This polypeptide is proteolytically cleaved into approximately 10 proteins, including, a viral capsid protein, viral envelope protein(s), and a minimum of 5 non-structural proteins (NS). The non-structural proteins include a chymotrypsin like serine protease, RNA helicase (NS3), and an RNA-dependent RNA polymerase (NS5). The NS3 protein of Flaviviridae is required for proteolytic cleavage of the viral polypeptide. The NS5 protein is required for replication of the viral genome (Chambers, et al., 1990a).</div>
    <div class="description-paragraph style-scope patent-text">Additionally, several cellular proteins have been identified as being involved in the replication of the Flaviviridae. For example, cellular signal peptidase enzyme may be required to cleave the viral polypeptide at several cleavage sites, to allow for expression of the viral protease (Hijikata, et al.).</div>
    <div class="description-paragraph style-scope patent-text">Inhibitors which prevent these proteins from carrying out their required functions in flavivirus replication may also have therapeutic value at treating infection with HGV. Finally cytokines or other polypeptides which are known to have antiviral activity and/or modulate the human immune system may be efficacious at treating HGV infection.</div>
    <div class="description-paragraph style-scope patent-text">One compound known to inhibit Flaviviridae RNA dependent RNA polymerases, which by analogy may be expected to inhibit the activity of the NS5 protein of HGV, is the nucleotide analogue 1-B-D-ribofuranosyl-1-2,4-triazole, 3-carboxamide, also known as ribavirin (Patterson, et al.). The method of action of ribavirin is thought to involve depletion of intercellular guanine pools and interference with the capping of viral RNAs (Patterson et al.).</div>
    <div class="description-paragraph style-scope patent-text">In individuals infected with HCV, significant reductions in viral titer and in serum levels of alanine aminotransferase (ALT--an indicator enzyme for liver dysfunction) were observed while ribavirin was administered (Reichard, et al.; Di Bisceglie, et al., 1992). Ribavirin appears to have broad efficacy for treating Flaviviridae infections, accordingly, beneficial results are expected after administration of ribavirin to individuals suffering from HGV derived liver disease.</div>
    <div class="description-paragraph style-scope patent-text">Another class of compounds known to be efficacious for treating Flaviviridae infections include the cytokines interferon α, interferon β, and interferon γ (Baron, et al.; Gutterman). Interferons are thought to act as antivirals by both (i) inducing the expression of cellular proteins that interfere with the replication and translation of viral RNAs, and (ii) by the activation of components of the human cellular immune system (Baron, et al.) . The interferons have broad applicability to the treatment of viral infections including infection with HBV, HDV, and HCV (Gutterman; Farci, et al.). In particular, multiple studies have indicated that the interferons, either alone or in combination with other antiviral therapies, are effective at treating infection with hepatitis C virus (Di Bisceglie, et al., 1989; Kakumu, et al.). Due to both the apparent hepatotropic nature of HGV and its classification in the family Flaviviridae, HGV infection may be expected to respond to similar interferon therapy.</div>
    <div class="description-paragraph style-scope patent-text">Still another class of compounds with potent anti-viral activity are inhibitors of viral proteases (Krausslich, et al.). All Flaviviridae encode a chymotrypsin-like serine protease which is required to cleave multiple sites of the genome polypeptide at multiple sites in the non-structural region. The amino acid residues that make up the catalytic site of this protease are well described and include a Histidine, an Aspartic acid, and a Serine residue (Grakoui, et al.). Furthermore studies of the flavivirus, Yellow Fever Virus have indicated that mutation of the Serine residue of the active site inhibits viral replication (Chambers, et al., 1990b).</div>
    <div class="description-paragraph style-scope patent-text">Inhibitors of the HGV NS3 protein can be designed to mimic the transition state of enzymatic cleavage. Alternatively, such inhibitors may be isolated by mass screening of previously synthesized compounds. The activity of putative HGV NS3 proteinase inhibitors can be determined through the use of in vitro transcription/translation systems, which are widely used in Flaviviridae research (Hijikata, et al.; Grakoui, et al.).</div>
    <div class="description-paragraph style-scope patent-text">Alternatively, the HGV genome can be cloned into a suitable vector for eukaryotic protein expression, such a bacculovirus or vaccinia, and the efficacy of the compounds can be determined in tissue culture systems (Grakoui, et al.). Similar approaches have been employed successfully to obtain potent inhibitors of the HIV protease (Vacca, et al.; Roberts, et al.).</div>
    <div class="description-paragraph style-scope patent-text">Another approach to treating disease caused by infection with the HGV relies on the synthesis of antisense oligonucleotides (Tonkinson and Stein) or oligonucleotide analogs which encode portions of the sequences of HGV disclosed in the present invention. As is true for all Flaviviridae, it would be expected that the genome of HGV is a positive strand RNA molecule of 9-12 kb in size. The single stranded nature of the viral genome should make HGV exquisitely sensitive to antisense oligonucleotides. Possible target sequences which might be employed to inhibit viral replication include the 5' untranslated region of HGV, the ribosome binding site of HGV or other sequences which would interfere with the translation of the HGV genome.</div>
    <div class="description-paragraph style-scope patent-text">Antisense oligonucleotides can be synthesized using commercially available synthesizers. Preferably the oligonucleotides are synthesized using phosphorodithioate backbones which have the advantage of being resistant to nuclease cleavage (Marshall &amp; Caruthers). Additionally other oligonucleotide analogues, such as those having a uncharged or amide type backbone (Egholm, et al.) may be employed. These oligonucleotides are commercially available (Biosearch, Millipore, Bedford, Mass.) and advantageous in that their lack of charge allows them to cross biological membranes, which are typically resist the passage of charged macromolecules.</div>
    <div class="description-paragraph style-scope patent-text">Oligonucleotides (or analogs thereof) for antisense applications are typically greater than 8 nucleotides in length to facilitate hybridization to a target sequence within the HGV genome. Upon hybridization of, for example, DNA oligomers to viral RNA target sequences, the hybridization complex can be degraded by a cellular enzyme such as RNAse H. The reduction in HGV templates then lessens the severity of HGV associated disease.</div>
    <div class="description-paragraph style-scope patent-text">The usefulness and efficacy of the above described therapeutic methods can be evaluated in vitro, using the cell systems described above, and in vivo, using the animal model systems described above.</div>
    <div class="description-paragraph style-scope patent-text">The following examples illustrate, but in no way are intended to limit the present invention.</div>
    <div class="description-paragraph style-scope patent-text">Materials and Methods</div>
    <div class="description-paragraph style-scope patent-text">Synthetic oligonucleotide linkers and primers were prepared using commercially available automated oligo-nucleotide synthesizers. Alternatively, custom designed synthetic oligonucleotides may be purchased from commercial suppliers.</div>
    <div class="description-paragraph style-scope patent-text">Standard molecular biology and cloning techniques were performed essentially as previously described in Ausubel, et al., Sambrook, et al., and Maniatis, et al.</div>
    <div class="description-paragraph style-scope patent-text">Common manipulations relevant to employing antisera and/or antibodies for screening and detection of immunoreactive protein antigens were performed essentially as described (Harlow, et al.). Similarly ELISA and Western blot assays for the detection of anti viral antibodies were performed either as described by their manufacturer (Abbott, N. Chicago, Ill., Genelabs Diagnostics, Singapore) or using standard techniques known in the art (Harlow, et al).</div>
    <heading class="style-scope patent-text">EXAMPLES</heading>
    <heading class="style-scope patent-text">Example 1</heading>
    <heading class="style-scope patent-text">Construction of PNF2161 CDNA Libraries</heading>
    <div class="description-paragraph style-scope patent-text">A. Isolation of RNA from Sera.</div>
    <div class="description-paragraph style-scope patent-text">One milliliter of undiluted PNF 2161 serum was precipitated by the addition of PEG (MW 6,000) to 8% and centrifugation at 12K, for 15 minutes in a microfuge, at 4° C. RNA was extracted from the resulting serum pellet essentially as described by Chomczynski.</div>
    <div class="description-paragraph style-scope patent-text">The pellet was treated with a solution containing 4M guanidinium isothiocyanate, 0.18% 2-mercaptoethanol, and 0.5% sarcosyl. The treated pellet was extracted several times with acidic phenol-chloroform, and the RNA was precipitated with ethanol. This solution was held at -70° C. for approximately 10 minutes and then spun in a microfuge at 4° C. for 10 minutes. The resulting pellet was resuspended in 100 μl of DEPC-treated (diethyl pyrocarbonate) water, and 10 μl of 3M NaOAc, pH=5.2, two volumes of 100% ethanol and one volume of 100% isopropanol were added to the solution. The solution was held at -70° C. for at least 10 minutes. The RNA pellet was recovered by centrifugation in a microfuge at 12,000×g for 15 minutes at 5° C. The pellet was washed in 70% ethanol and dried under vacuum.</div>
    <div class="description-paragraph style-scope patent-text">B. Synthesis of CDNA</div>
    <div class="description-paragraph style-scope patent-text">(i) First Strand Synthesis</div>
    <div class="description-paragraph style-scope patent-text">The synthesis of CDNA molecules was accomplished as follows. The above described RNA preparations were transcribed into cDNA, according to the method of Gubler et al. using random nucleotide hexamer primers (CDNA Synthesis Kit, BMB, Indianapolis, Ind. or GIBCO/BRL).</div>
    <div class="description-paragraph style-scope patent-text">After the second-strand cDNA synthesis, T4 DNA polymerase was added to the mixture to maximize the number of blunt-ends of cDNA molecules. The reaction mixture was incubated at room temperature for 10 minutes. The reaction mixture was extracted with phenol/chloroform and chloroform isoamyl alcohol.</div>
    <div class="description-paragraph style-scope patent-text">The cDNA was precipitated by the addition of two volumes of 100% ethanol and chilling at -70° C. for 15 minutes. The cDNA was collected by centrifugation, the pellet washed with 70% ethanol and dried under vacuum.</div>
    <div class="description-paragraph style-scope patent-text">C. Amplification of the Double Stranded CDNA Molecules.</div>
    <div class="description-paragraph style-scope patent-text">The CDNA pellet was resuspended in 12 μl distilled water. To the resuspended cDNA molecules the following components were added: 5 μl phosphorylated linkers (Linker AB, a double strand linker comprised of SEQ ID NO:1 and SEQ ID NO:2, where SEQ ID NO:2 is in a 3' to 5' orientation relative to SEQ ID NO:1--as a partially complementary sequence to SEQ ID NO:1), 2 <figure-callout id="10" label="μl" filenames="US5874563-drawings-page-12.png,US5874563-drawings-page-19.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">μl</a></figure-callout> 10× ligation buffer (0.66M Tris.Cl pH=7.6, 50 mM MgCl<sub class="style-scope patent-text">2</sub>, 50 mM DTT, 10 mM ATP) and 1 μl T4 DNA ligase (0.3 to 0.6 Weiss Units). Typically, the cDNA and linker were mixed at a 1:100 ratio. The reaction was incubated at 14° C. overnight. The following morning the reaction was incubated at 70° C. for three minutes to inactivate the ligase.</div>
    <div class="description-paragraph style-scope patent-text">To 100 μl of 10 mM Tris-Cl buffer, pH 8.3, containing 1.5 mM MgCl<sub class="style-scope patent-text">2</sub> and 50 mM KCl (Buffer A) was added about 1 μl of the linker-ligated CDNA preparation, 2 μM of a primer having the sequence shown as SEQ ID NO:1, 200 μM each of DATP, dCTP, dGTP, and dTTP, and 2.5 units of Thermus aquaticus DNA polymerase (Taq polymerase). The reaction mixture was heated to 94° C. for 30 sec for denaturation, allowed to cool to 50° C. for 30 sec for primer annealing, and then heated to 72° C. for 0.5-3 minutes to allow for primer extension by Tag polymerase. The amplification reaction, involving successive heating, cooling, and polymerase reaction, was repeated an additional 25-40 times with the aid of a Perkin-Elmer Cetus DNA thermal cycler (Mullis; Mullis, et al.; Reyes, et al., 1991; Perkin-Elmer Cetus, Norwalk, Conn.).</div>
    <div class="description-paragraph style-scope patent-text">After the amplification reactions, the solution was then phenol/chloroform, chloroform/isoamyl alcohol extracted and precipitated with two volumes of ethanol. The resulting amplified CDNA pellets were resuspended in 20 μl TE (pH=7.5).</div>
    <div class="description-paragraph style-scope patent-text">D. Cloning of the CDNA into Lambda Vectors.</div>
    <div class="description-paragraph style-scope patent-text">The linkers used in the construction of the cDNAs contained an EcoRI site which allowed for direct insertion of the amplified cDNAs into lambda gt11 vectors (Promega, Madison Wis. or Stratagene, La Jolla, Calif.). Lambda vectors were purchased from the manufacturer (Promega) which were already digested with EcoRI and treated with alkaline phosphatase, to remove the 5' phosphate and prevent self-ligation of the vector. The EcoRI-digested cDNA preparations were ligated into lambda gt11 (Promega). The conditions of the ligation reactions were as follows: 1 μl vector DNA (Promega, 0.5 mg/ml); 0.5 or 3 μl of the PCR amplified insert cDNA; 0.5 μl 10× ligation buffer (0.5M Tris-HCl , pH=7.8; 0.1M MgCl<sub class="style-scope patent-text">2</sub> ; 0.2M DTT; 10 mM ATP; 0.5 mg/ml bovine serum albumin (BSA)), 0.5 μl T4 DNA ligase (0.3 to 0.6 Weiss units) and distilled water to a final reaction volume of 5 μl.</div>
    <div class="description-paragraph style-scope patent-text">The ligation reactions were incubated at 14° C. overnight (12-18 hours). The ligated cDNA was packaged by standard procedures using a lambda DNA packaging system ("GIGAPAK", Stratagene, LaJolla, Calif.), and then plated at various dilutions to determine the titer. A standard X-gal blue/white assay was used to determine recombinant frequency of the libraries (Miller; Maniatis et al.).</div>
    <div class="description-paragraph style-scope patent-text">Percent recombination in each library was also determined as follows. A number of random clones were selected and corresponding phage DNA isolated. Polymerase chain reaction (Mullis; Mullis, et al.) was then performed using isolated phage DNA as template and lambda DNA sequences, derived from lambda sequences flanking the EcoRI insert site for the cDNA molecules, as primers. The presence or absence of insert was evident from gel analysis of the polymerase chain reaction products.</div>
    <div class="description-paragraph style-scope patent-text">The cDNA-insert phage libraries generated from serum sample PNF 2161 was deposited with the American Type Culture Collection, 12301 Parklawn Dr., Rockville Md. 20852, and has been assigned the deposit designation ATCC 75268 (PNF 2161 cDNA source).</div>
    <heading class="style-scope patent-text">Example 2</heading>
    <heading class="style-scope patent-text">Immunoscreening of Recombinant Libraries.</heading>
    <div class="description-paragraph style-scope patent-text">The lambda gt11 libraries generated in Example 1 were immunoscreened for the production of antigens recognizable by the PNF 2161 serum from which the libraries were generated. The phage were plated for plaque formation using the Escherichia coli bacterial plating strain E. coli KM392. Alternatively, E. coli Y1090R (Promega, Madison Wis.) may be used.</div>
    <div class="description-paragraph style-scope patent-text">The fusion proteins expressed by the lambda gt11 clones were screened with serum antibodies essentially as described by Ausubel, et al.</div>
    <div class="description-paragraph style-scope patent-text">Each library was plated at approximately 2×10<sup class="style-scope patent-text">4</sup> phages per 150 mm plate. Plates were overlaid with nitrocellulose filters overnight. Filters were washed with TBS (10 mM, Tris pH 7.5; 150 mM NaCl), blocked with AIB (TBS buffer with 1% gelatin) and incubated with a primary antibody diluted 100 times in AIB.</div>
    <div class="description-paragraph style-scope patent-text">After washing with TBS, filters were incubated with a second antibody, goat-anti-human IgG conjugated to alkaline phosphatase (Promega). Reactive plaques were developed with a substrate (for example, BCIP, 5-bromo-4-chloro-3-indolyl-phosphate), with NBT (nitro blue tetrazolium salt (Sigma)). Positive areas from the primary screening were replated and immunoscreened until pure plaques were obtained.</div>
    <heading class="style-scope patent-text">Example 3</heading>
    <heading class="style-scope patent-text">Screening of the PNF 2161 Library</heading>
    <div class="description-paragraph style-scope patent-text">The cDNA library of PNF 2161 in lambda gt11 was screened, as described in Example 2, with PNF 2161 sera. The results of the screening are presented in Table 1.</div>
    <div class="description-paragraph style-scope patent-text">
      </div> <pre xml:space="preserve" listing-type="tabular" class="style-scope patent-text">              TABLE 1
______________________________________
PNF2161 Libraries
                                     # Clones
                                     Plaque-
Library.sup.1
       % Recomb..sup.2
                   Antibody.sup.3
                             # Screened
                                     Purified
______________________________________
PNF/RNA
       85          PNF       5.5 × 10.sup.5
                                     4
PNF/<figure-callout id="90" label="RNA" filenames="US5874563-drawings-page-7.png,US5874563-drawings-page-8.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">RNA</a></figure-callout>
       90          <figure-callout id="8" label="PNF" filenames="US5874563-drawings-page-19.png,US5874563-drawings-page-6.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">PNF</a></figure-callout>         8 × 10.sup.4
                                     7
TOTALS:                              11
______________________________________
 .sup.1 cDNA library constructed from the indicated human source.
 .sup.2 Percent recombinant clones in the indicated λgt11 library a
 determined by blue/white plaque assay and confirmed by PCR amplification
 of randomly selected clones.
 .sup.3 Antisera source used for the immunoscreening of each indicated
 library.
</pre>
    
    <div class="description-paragraph style-scope patent-text">One of the clones isolated by the above screen (PNF 2161 clone 470-20-1, SEQ ID NO:3; β-galactosidase in-frame fusion translated sequence, SEQ ID NO:4), was used to generate extension clones, as described in Example 6. Clone 470-20-1 nucleic acid sequence is presented as SEQ ID NO:3 (protein sequence SEQ ID NO:4). The isolated nucleic acid sequence without the SISPA cloning linkers is presented as SEQ ID NO:19 (protein SEQ ID NO:20).</div>
    <heading class="style-scope patent-text">Example 4</heading>
    <heading class="style-scope patent-text">Characterization of the Immunoreactive 470-20-1 Clone</heading>
    <div class="description-paragraph style-scope patent-text">A. Southern Blot Analysis of Immunoreactive Clones.</div>
    <div class="description-paragraph style-scope patent-text">The inserts of immunoreactive clones were screened for their ability to hybridize to the following control DNA sources: normal human peripheral blood lymphocyte (purchased from Stanford University Blood Bank, Stanford, Calif.) DNA, and Escherichia coli KM392 genomic DNA (Ausubel, et al.; Maniatis, et al.; Sambrook, et al.). Ten micrograms of human lymphocyte DNA and 2 micrograms of E. coli genomic DNA were digested with EcoRI and HindIII. The restriction digestion products were electrophoretically fractionated on an agarose gel (Ausubel, et al.) and transferred to nylon or nitrocellulose membranes (Schleicher and Schuell, Keene, N.H.) as per the manufacturer's instructions.</div>
    <div class="description-paragraph style-scope patent-text">Probes from the immunoreactive clones were prepared as follows. Each clone was amplified using primers corresponding to lambda gt11 sequences that flank the EcoRI cloning site of the gt11 vector. Amplification was carried out by polymetase chain reactions utilizing each immunoreactive clone as template. The resulting amplification products were digested with EcoRI, the amplified fragments gel purified and eluted from the gel (Ausubel, et al.). The resulting amplified fragments, derived from the immunoreactive clones, were then random prime labelled using a commercially available kit (BMB) employing <sup class="style-scope patent-text">32</sup> P-dNTPs.</div>
    <div class="description-paragraph style-scope patent-text">The random primed probes were then hybridized to the above-prepared nylon membrane to test for hybridization of the insert sequences to the control DNAs. The 470-20-1 insert did not hybridize with any of the control DNAs.</div>
    <div class="description-paragraph style-scope patent-text">As positive hybridization controls, a probe derivative from a human C-kappa gene fragment (Hieter) was used as single gene copy control for human DNA and a E. coli polymerase gene fragment was similarly used for E. coli DNA.</div>
    <div class="description-paragraph style-scope patent-text">B. Genomic PCR.</div>
    <div class="description-paragraph style-scope patent-text">PCR detection was developed first to verify exogenicity with respect to several genomic DNAs which could have been inadvertently cloned during library construction, then to test for the presence of the cloned sequence in the cloning source and related specimen materials. Several different types of specimens, including SISPA-amplified nucleic acids and nucleic acids extracted from the primary source, and nucleic acids extracted from related source materials (e.g., from animal passage studies), were tested.</div>
    <div class="description-paragraph style-scope patent-text">The term "genomic PCR" refers to testing for the presence of specific sequences in genomic DNA from relevant organisms. For example, a genomic PCR for a Mystax-derived clone would include genomic DNAs as follows:</div>
    <div class="description-paragraph style-scope patent-text">1. human DNA (1 μg/rxn.)</div>
    <div class="description-paragraph style-scope patent-text">2. Mystax DNA (0.1-1 μg/rxn.)</div>
    <div class="description-paragraph style-scope patent-text">3. E. coli (10-100 ng/rxn.)</div>
    <div class="description-paragraph style-scope patent-text">4. yeast (10-100 ng/rxn.)</div>
    <div class="description-paragraph style-scope patent-text">Human and Mystax DNAs are tested, as the immediate and ultimate source for the agent. E. coli genomic DNA, as a frequent contaminant of commercial enzyme preparations, is tested. Yeast is also tested, as a ubiquitous organism, whose DNA can contaminate reagents and thus, be cloned.</div>
    <div class="description-paragraph style-scope patent-text">In addition, a negative control (i.e., buffer or water only), and positive controls to include approximately 10<sup class="style-scope patent-text">5</sup> c/rxn., are also amplified.</div>
    <div class="description-paragraph style-scope patent-text">Amplification conditions vary, as may be determined for individual sequences, but follow closely the following standard PCR protocol: PCR was performed in reactions containing 10 mM Tris, pH 8.3, 50 mM KCl, 1.75 mM MgCl<sub class="style-scope patent-text">2</sub>, 1.0 uM each primer, 200 uM each dATP, dCTP, and dGTP, and 300 μM dUTP, 2.5 units Taq DNA polymerase, and 0.2 units uracil-N-glycosylase per 100 ul reaction. Cycling was for at least 1 minute at 94° C., followed by 30 to 40 repetitions of denaturation (92°-94° C. for 15 seconds), annealing (55°-56° C. for 30 seconds), and extension (72° C. for 30 seconds). PCR reagents were assembled, and amplification reactions were constituted, in a specially-designated laboratory maintained free of amplified DNA.</div>
    <div class="description-paragraph style-scope patent-text">As a further barrier to contamination by amplified sequences and thus compromise of the test by "false positives," the PCR was performed with dUTP replacing TTP, in order to render the amplified sequences biochemically distinguishable from native DNA. To enzymatically render unamplifiable any contaminating PCR product, the enzyme uracil-N-glycosylase was included in all genomic PCR reactions. Upon conclusion of thermal cycling, the reactions were held at 72° C. to prevent renaturation of uracil-N-glycosylase and possible degradation of amplified U-containing sequences.</div>
    <div class="description-paragraph style-scope patent-text">A "HOT START PCR" was performed, using standard techniques ("AMPLIWAX", Perkin-Elmer Biotechnology; alternatively, manual techniques were used), in order to make the above general protocol more robust for amplification of diverse sequences, which ideally require different amplification conditions for maximal sensitivity and specificity.</div>
    <div class="description-paragraph style-scope patent-text">Detection of amplified DNA was performed by hybridization to specific oligonucleotide probes located internal to the two PCR primer sequences and having no or minimal overlap with the primers. In some cases, direct visualization of electrophoresed PCR products was performed, using ethidium bromide fluorescence, but probe hybridization was in each case also performed, to help ensure discrimination between specific and non-specific amplification products. Hybridization to radiolabelled probes in solution was followed by electrophoresis in 8-15% polyacrylamide gels (as appropriate to the size of the amplified sequence) and autoradiography.</div>
    <div class="description-paragraph style-scope patent-text">Clone 470-20-1 was tested by genomic PCR, against human, E. coli, and yeast DNAs. No specific sequence was detected in negative control reactions, nor in any genomic DNA which was tested, and 10<sup class="style-scope patent-text">5</sup> copies of DNA/reaction resulted in a readily-detectable signal. This sensitivity (i.e., 10<sup class="style-scope patent-text">5</sup> /reaction) is adequate for detection of single-copy human sequences in reactions containing 1 ug total DNA, representing the DNA from approximately 1.5×10<sup class="style-scope patent-text">5</sup> cells.</div>
    <div class="description-paragraph style-scope patent-text">C. Direct Serum PCR</div>
    <div class="description-paragraph style-scope patent-text">Serum or other cloning source or related source materials were directly tested by PCR using primers from selected cloned sequences. In these experiments, HGV viral particles were directly precipitated from sera with polyethylene glycol (PEG), or, in the case of PNF and certain other sera, were pelleted by ultracentrifugation. For purification of RNA, the pelleted materials were dissolved in guanidinium thiocyanate and extracted by the acid guanidinium phenol technique (Chomczynski, et al.).</div>
    <div class="description-paragraph style-scope patent-text">Alternatively, a modification of this method afforded through and implemented by the use of commercially available reagents, e.g., "TRIREAGENT" (Molecular Research Center, Cincinnati, Ohio) or "TRIZOL" (Life Technologies, Gaithersburg, Md.), and associated protocols was used to isolate RNA. In addition, RNA suitable for PCR analysis was isolated directly from serum or other fluids containing virus, without prior concentration or pelleting of virus particles, through the use of "PURESCRIPT" reagents and protocols (Gentra Systems, Minneapolis, Minn.).</div>
    <div class="description-paragraph style-scope patent-text">Isolated DNA was used directly as a template for the PCR. RNA was reverse transcribed using reverse transcriptase (Gibco/BRL), and the cDNA product was then used as a template for subsequent PCR amplification.</div>
    <div class="description-paragraph style-scope patent-text">In the case of 470-20-1, nucleic acid from the equivalent of 20-50 ul of PNF serum was used as the input template into each RT-PCR or PCR reaction. Primers were designed based on the 470-20-1 sequence, as follows: 470-20-1-77F (SEQ ID NO:9) and 470-20-1-211R (SEQ ID NO:10). Reverse transcription was performed using MMLV-RT (Gibco/BRL) and random hexamers (Promega) by incubation at room temperature for approximately 10 minutes, 42° C. for 15 minutes, and 99° C. for 5 minutes, with rapid cooling to 4° C. The synthesized cDNA was amplified directly, without purification, by PCR, in reactions containing 1.75 mM MgCl<sub class="style-scope patent-text">2</sub>, 0.2-1 μM each primer, 200 uM each DATP, dCTP, dGTP, and dTTP, and 2.5-5.0 units Taq DNA polymerase ("AMPLITAQ", Perkin-Elmer) per 100 ul reaction.</div>
    <div class="description-paragraph style-scope patent-text">Cycling was for at least one minute at 94° C., followed by 40-45 repetitions of denaturation (94° C. for 15 for 10 cycles; 92° C. or 94° C. for 15 seconds for the succeeding cycles), annealing (55° C. for 30 seconds), and extension (72° C. for 30 seconds), in the "GENEAMP SYSTEM 9600" thermal cycler (Perkin-Elmer) or comparable cycling conditions in other thermal cyclers (Perkin-Elmer; MJ Research, Watertown, Mass.).</div>
    <div class="description-paragraph style-scope patent-text">Positive controls consisted of (i) previously amplified PCR product whose concentration was estimated using the Hoechst 33258 fluroescence assay, (ii) purified plasmid DNA containing the DNA sequence of interest, or (iii) purified RNA transcripts derived from plasmid clones in which the DNA sequence of interest is disposed under the transcriptional control of phage RNA promoters such as T7, T3, or SP6 and RNA prepared through the use of commercially available in vitro transcription kits. In addition, an aliquot of positive control DNA corresponding to approximately 10-100 copies/rxn. can be spiked into reactions containing nucleic acids extracted from the cloning source specimen, as a control for the presence of inhibitors of DNA amplification reactions. Each separate extract was tested with at least one positive control.</div>
    <div class="description-paragraph style-scope patent-text">Specific products were detected by hybridization to a specific oligonucleotide probe 470-20-1-152F (SEQ ID NO:16), for confirmation of specificity. Hybridization of 10 ul of PCR product was performed in solution in 20 ul reactions containing approximately 1×10<sup class="style-scope patent-text">6</sup> cpm of <sup class="style-scope patent-text">32</sup> P-labelled 470-20-1-152F. Specific hybrids were detected following electrophoretic separation from unhybridized oligo in polyacrylamide gels, and autoradiography.</div>
    <div class="description-paragraph style-scope patent-text">In addition to PNF, extracted nucleic acids from normal serum was also reverse transcribed and amplified, using the "serum PCR" protocol sequence. No signal was detected in normal human serum. The specific signal in PNF serum was reproducibly detected in multiple extracts, with the 470-20-1 specific primers.</div>
    <div class="description-paragraph style-scope patent-text">D. Amplification from SISPA Uncloned Nucleic Acids</div>
    <div class="description-paragraph style-scope patent-text">SISPA (Sequence-Independent Single Primer Amplification) amplified cDNA was used as templates (Example 1). Sequence-specific primers designed from selected cloned sequences were used to amplify DNA fragments of interest from the templates. Typically, the templates were the SISPA-amplified samples used in the cloning manipulations. For example, amplification primers 470-20-1-77F (SEQ ID NO:9) and 470-20-1-211R (SEQ ID NO:10) were selected from the clone 470-20-1 sequence (SEQ ID NO:3). These primers were used in amplification reactions with the SISPA-amplified PNF2161 cDNA as a template.</div>
    <div class="description-paragraph style-scope patent-text">The identity of the amplified DNA fragments were confirmed by (i) hybridization with the specific oligonucleotide probe 470-20-1-152F (SEQ ID NO:16), designed based on the 470-20-1 sequence (SEQ ID NO:3) and/or (ii) size. The probe used for DNA blot detection was labelled with digoxygenin using terminal transferase according to the manufacturer's recommendations (BMB). Hybridization to the amplified DNA was then performed using either Southern blot or liquid hybridization (Kumar, et al., 1989) analyses.</div>
    <div class="description-paragraph style-scope patent-text">Positive control DNA used in the amplification reactions was previously amplified PCR product whose concentration was estimated by the Hoechst 33258 fluorescence assay, or, alternatively, purified plasmid DNA containing the cloned inserts of interest.</div>
    <div class="description-paragraph style-scope patent-text">The 470-20-1 specific signal was detected in cDNA amplified by PCR from SISPA-amplified PNF2161. Negative control reactions were nonreactive, and positive control DNA templates were detected.</div>
    <div class="description-paragraph style-scope patent-text">E. Amplification from Liver RNA Samples.</div>
    <div class="description-paragraph style-scope patent-text">RNA was prepared from liver biopsy material following the methods of Cathal, et al., wherein tissue was extracted in 5M guanidine thiocyanate followed by direct precipitation of RNA by 4M LiCl. After washing of the RNA pellet with 2M LiCl, residual contaminating protein was removed by extraction with phenol:chloroform and the RNA recovered by ethanol precipitation.</div>
    <div class="description-paragraph style-scope patent-text">The 470-20-1 specific primers were also used in amplification reactions with the following RNA sources as substrate: normal mystax liver RNA, normal tamarin (Sanguinus labiatus) liver RNA, and MY131 liver RNA. MY131 is a mystax that was inoculated intravenously with 1 ml of PNF 2161 plasma. There were obvious elevations of a liver enzyme (SCID) and histological evidence of an apparent viral infection. The histological correlation was most obvious in the liver of MY131, whose liver was obtained at or near the peak of SCID activity. <figure-callout id="131" label="Mystax" filenames="US5874563-drawings-page-9.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">Mystax</a></figure-callout> 131 liver RNA did not give amplified products with the non-coding primers (SEQ ID NO:7 and SEQ ID NO:8) of HCV.</div>
    <div class="description-paragraph style-scope patent-text">The amplification reactions were carried out in duplicate for two experiments. The results of these amplification reactions are presented in Table 2.</div>
    <div class="description-paragraph style-scope patent-text">
      </div> <pre xml:space="preserve" listing-type="tabular" class="style-scope patent-text">              TABLE 2
______________________________________
PCR with 470-20-1 Primers
                  Exp. 1          Exp. 2
                  A     B         A   B
______________________________________
Normal My liver RNA
                  -     -         -   -
Normal tamarin liver RNA
                  -     -         -   -
My131 liver RNA   +     +         +   +
PNF 2161          ++    ++        ++  ++
______________________________________
</pre>
    
    <div class="description-paragraph style-scope patent-text">These results demonstrate the 470-20-1 sequences are present in the parent serum sample (PNF 2161) and in a liver RNA sample from a passage animal of the PNF 2161 sample (MY131). However, both control RNAs were negative for the presence of 470-20-1 sequences.</div>
    <div class="description-paragraph style-scope patent-text">F. Screening of a Serum Panel for HGV Sequences by Polymerase Chain Reaction Using RNA Templates.</div>
    <div class="description-paragraph style-scope patent-text">1. High-ALT Donors</div>
    <div class="description-paragraph style-scope patent-text">The disease association between HGV and liver disease was assessed by polymerase chain reaction screening, using HGV specific primers, of sera from hepatitis patients and from blood donors with abnormal liver function. The latter consisted of serum from blood donations with serum ALT levels greater than 45 International Units per ml.</div>
    <div class="description-paragraph style-scope patent-text">A serum panel consisting of 152 total sera was selected. The following sera were selected for the serum panel: 104 high-ALT sera from screened blood donations at the Stanford University Blood Bank (SUBB); 34 N-(ABCDE) hepatitis sera from northern California, Egypt, and Peru; and 14 sera from other donors suspected of having liver disease and/or hepatitis virus infection. The negative controls for the panel were as follows: 9 highly-screened blood donors (SUBB) notable for the absence of risk factors for viral infections ("supernormal" sera, e.g., O-negative, Rh-negative; negative for HIV, known hepatitis agents, and CMV; whose multiple previous blood donations had been transfused without causing disease); and 2 random blood donors. These sera were assayed for the presence of HGV specific sequences by RT-PCR using the 470-20-1 primers 77F (SEQ ID NO:9) and 211R (SEQ ID NO:10).</div>
    <div class="description-paragraph style-scope patent-text">RNA extraction and RT-PCR were performed essentially as described in Example 4C, except that the primer 470-20-1-211R was 5'-biotinylated to facilitate rapid screening of amplified products by a method involving hybridization in solution, followed by affinity capture of hybridized probe using streptavidin-coated paramagnetic beads. Methods for the analysis of nucleic acids by hybridization to specific labelled probes with capture of the hybridized sequences through affinity interactions are well known in the art of nucleic acid analysis.</div>
    <div class="description-paragraph style-scope patent-text">Depending on the amount of serum available for testing, RNA from 30 to 50 μl of serum was used per RT/PCR reaction. Each serum was tested in duplicate, with positive controls corresponding to 10, 100, or 1000 copies of RNA transcript per reaction and with appropriate negative (buffer) controls. No negative controls were reactive, and at least 10 copies per reaction were detectable in each PCR run. Indeterminate results were defined as specific hybridizing signal being present in only one of two duplicate reactions.</div>
    <div class="description-paragraph style-scope patent-text">Efficient, highly sensitive analysis of the products from the amplification analysis of this serum panel was performed using an instrument specifically designed for affinity-based hybrid capture using electrochemiluminscent oligonucleotide probes (QPCR System 5000™, Perkin-Elmer). Assays utilizing the QPCR 5000™ have been described (DiCesare, et al; Wages, et al).</div>
    <div class="description-paragraph style-scope patent-text">The products of each reaction were assayed by hybridization to probe 470-20-1-152F (5'-end-labelled with an electrochemiluminescent ruthenium chelate), and measurement using the "QPCR 5000. " Based on a cutoff of the sum of the mean and three times the standard deviation of negative controls in a given amplification run, a total of 34 possible positives were selected for confirmatory testing.</div>
    <div class="description-paragraph style-scope patent-text">The 34 samples were analyzed by solution hybridization and electrophoresis (Example 4C). Out of these 34 samples, 6 sera (i.e., 6/152) were shown to have specific hybridizing sequences in duplicate reactions. Of these six samples, three were strongly reactive by comparison with positive controls: one High-ALT serum from SUBB, and two N-(ABCDE) sera from Egypt.</div>
    <div class="description-paragraph style-scope patent-text">A second blood sample was obtained from the highly positive SUBB serum donor one year after the initial sample was taken. The second serum sample was confirmed to be HGV positive by the PCR methods described above. This result confirms persistant infection by HGV in a human. The serum was designated "JC." Further, the serum donor was HCV negative (determined by seroreactivity tests and PCR) and antibody negative for HAV and HBV.</div>
    <div class="description-paragraph style-scope patent-text">In addition, a third N-(ABCDE) serum from Egypt, a northern California blood donor with N-(ABCDE) hepatitis, and a N-(ABCDE) hepatitis serum, were also shown to be weakly positive by this method. Two other sera gave indeterminate results, defined as the presence of specific sequences in one of two amplification reactions.</div>
    <div class="description-paragraph style-scope patent-text">Subsequent PCR analysis of replicate serum aliquots from these HGV-positive and indeterminate sera resulted in HGV-positive results in 6 of 8 sera tested and indeterminate results in the remaining 2 sera.</div>
    <div class="description-paragraph style-scope patent-text">A second primer set was used for the confirmation of HGV positive samples. This primer set (GV57-4512MF, SEQ ID NO:121, and GV57-4657MR, SEQ ID NO:122) for diagnostic amplification, was selected from a conserved region of HGV derived from the putative NS5 coding region. An approximately 2.2 kb fragment was amplified from each of 5 separate HGV isolates. The primers used for the amplification reactions were 470EXT4-2189R (SEQ ID NO:119) and 470EXT4-29F (SEQ ID NO:120). The amplified DNA fragments were sequenced and the sequences aligned. Highly conserved regions were identified from the alignment and optimal primer sequences were designed incorporating mixed base synthesis at those positions that remained divergent throughout the five sequences. The resulting NS5 primers were as follows: GV57-4512MF, SEQ ID NO:121, and GV57-4657MR, SEQ ID NO:122. These primers were used to amplify a diagnostic fragment of 165 bp from test samples.</div>
    <div class="description-paragraph style-scope patent-text">An internal probe sequence, GV22dc-89MF (SEQ ID NO:123) was derived from another highly conserved region for detection of the specifically amplified product. The probe is also of sufficient length to allow for detection of minimally divergent HGV sequences under lowered stringency conditions.</div>
    <div class="description-paragraph style-scope patent-text">Analysis of specimens for the presence of the diagnostic NS5 sequence followed the same conditions for sample preparation, amplification, and liquid hybridization as described for the 470-20-1 primers (Example 4C). The concordance of results for sera samples analyzed by PCR using both the 470-20-1 and NS5 primer pairs are shown in Table 3.</div>
    <div class="description-paragraph style-scope patent-text">
      </div> <pre xml:space="preserve" listing-type="tabular" class="style-scope patent-text">              TABLE 3
______________________________________
            470-20-1 Primer Pair
                                  Indeter-
                    +         -   minant
______________________________________
NS5-Region  +       71        0   1
Primer      -       6         13  2
Pair        Indeter-
                    2         1   0
(GV57)      minant
______________________________________
</pre>
    
    <div class="description-paragraph style-scope patent-text">Further PCR analyses of additional aliquots obtained from the 8 sera identified above as being HGV-positive were carried out using the 470-20-1 primer set (SEQ ID NO:9 and SEQ ID NO:10) and the NS5 primer set. In these assays, the HGV PCR analyses gave consistently positive results in 5 of the 8 sera. These results are presented in Table 4.</div>
    <div class="description-paragraph style-scope patent-text">In contrast, none of the two random donors or nine highly-screened "supernormal" sera was positive in either set of PCR analysis.</div>
    <div class="description-paragraph style-scope patent-text">These results reinforce the disease association between HGV and liver disease.</div>
    <div class="description-paragraph style-scope patent-text">
      </div> <pre xml:space="preserve" listing-type="tabular" class="style-scope patent-text">              TABLE 4
______________________________________
                  Number  Number
Specimen Group    Tested  Positive
______________________________________
High-ALT Donor    104     1
Non-ABCDE, other  48      4
<figure-callout id="2" label="Normal Donor" filenames="US5874563-drawings-page-10.png,US5874563-drawings-page-11.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">Normal Donor</a></figure-callout>      2       0
"Supernormal"     9       0
Totals            163     5
______________________________________
</pre>
    
    <div class="description-paragraph style-scope patent-text">Further testing of sera from High-ALT donors has yielded the following results. A total of 495 sera have been tested, in addition to the initial panel of 104 sera described above. Of these 495 specimens, 6 were identified as HGV positive using the primer pair 470-20-1-77F (SEQ ID NO:9) and 470-20-1-211R (SEQ ID NO:10). These six sera have the following HCV profiles: R25342, HCV negative; R17749, HCV positive; J53171, HCV positive, HBV positive; J54406, HCV negative; R08074, HCV negative; and X31049, HCV negative. Positive scores are based on repeated reactivity in at least 2 separate reactions. R25342 was tested and confirmed positive by PCR using the NS5 primer pair. Accordingly, a detection rate of approximately 1.2% has been observed (7 of 599 tested).</div>
    <div class="description-paragraph style-scope patent-text">Freshly-obtained plasma samples from blood donors with elevated ALT were also obtained from SUBB, the Peninsula Blood Bank (Burlingame, Calif.), and the New York Blood Center (New York, N.Y.), for testing for HGV RNA by PCR (470-20-1 primer pair). Of 214 total donations which were tested, a total of 5 (approximately 2.3%) were HGV RNA positive. These five sera have the following HCV profiles: T55806, HCV positive; T55875, HCV negative; T56633, HCV negative; R38730, HCV negative; and 3831781, HCV negative. Subsequent donations from two of these donors, T55806 and T55875, were also HGV RNA positive. T55806, T55875 and T56633 were tested and confirmed positive by PCR using the NS5 primer pair.</div>
    <div class="description-paragraph style-scope patent-text">2. Screening of Accepted Blood Donors</div>
    <div class="description-paragraph style-scope patent-text">To assess the prevalence of HGV in the normal blood donor population, serum was collected from screened blood donors for transfusion at SUBB. A total of 968 specimens, representing 769 unique donors, was tested for HGV RNA. The samples were screened by PCR using the 470-20-1 primer pair.</div>
    <div class="description-paragraph style-scope patent-text">A total of 16 sera were identified as having detectable HGV RNA. Of these, 6 represent duplicates from 3 donors, such that a total of 13 unique donors of 769 tested were HGV positive by RNA PCR. All positive samples were tested and confirmed positive by PCR using the NS5 primer pair. These donors were characterized by normal ALT levels, as well as otherwise normal serology. Accordingly, approximately 1.7% of the sera tested in the normal blood donor population are HGV positive. Therefore, the presence of HGV was detected in both accepted and rejected blood donors.</div>
    <div class="description-paragraph style-scope patent-text">3. Specimens from Various Geographic Locales.</div>
    <div class="description-paragraph style-scope patent-text">The presence of HGV infection in populations of hepatitis patients from geographically widespread sources was assessed by PCR. The PCR reactions were carried out essentially as described in Example 4C using the 470-20-1 PCR primer pairs. Serum samples from Egypt, Greece, Australia (see Example 4F-4), Peru, England, Italy, Germany, South Korea, United States and Japan were tested. HGV RNA was detectable in subsets of all populations tested.</div>
    <div class="description-paragraph style-scope patent-text">4. Post-transfusion Associated HGV Infection and Parenteral Transmission.</div>
    <div class="description-paragraph style-scope patent-text">HGV RNA was detected in several post-transfusion hepatitis cases (those of Japanese and European origin were included in Example 4F-3). For 4 total cases, one from Japan, two from the U.S. and one from Australia, multiple time-points were assayed for the presence of HGV RNA. For 3 of these cases, (i) pre-transfusion samples were available to estabish previous HGV status of the patient, and (ii) samples were available from individual blood donors to those three cases, to establish donor HGV status.</div>
    <div class="description-paragraph style-scope patent-text">The first case was a Japanese patient transfused on Dec. 2, 1980. Following the transfusion the patient developed Non-B Non-C hepatitis. A total of 5 sera from this patient were tested for HGV RNA by PCR using the 470-20-1 primer pair. HGV RNA was detectable from about 2 weeks to about 8 months following transfusion. A sample taken greater than 1 year post-transfusion was indeterminate (i.e., positive in one duplicate reaction only). No pre-transfusion sample was available for testing.</div>
    <div class="description-paragraph style-scope patent-text">Cases BIZ and STO (Tables 5 and 6, respectively) were from a prospectively-followed heart surgery study (Alter, et al., 1989) conducted at the NIH. For each of these patients, pre-transfusion sera were available and were determined to be negative for HGV RNA by PCR using the 470-20-1 primer pair. BIZ tested positive for HGV RNA from day one post-transfusion to week 198 post-transfusion. Of 9 total blood donors to BIZ, 2 out of 8 tested were found to be HGV positive. STO tested positive for HGV RNA from <figure-callout id="5" label="week" filenames="US5874563-drawings-page-19.png,US5874563-drawings-page-3.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">week</a></figure-callout> 5 post-transfusion through week 92 post-transfusion.</div>
    <div class="description-paragraph style-scope patent-text">
      </div> <pre xml:space="preserve" listing-type="tabular" class="style-scope patent-text">              TABLE 5
______________________________________
Transfusion-Associated Transmission
of HGV: Case BIZ
Draw                    ALT in  470 PCR
Date       Time         IU/L    Result
______________________________________
10/30/78   -4 days      23      -
11/01/78   -1 day       31      -
11/03/78   +1 day       29      +
11/17/78   +2 <figure-callout id="51" label="weeks" filenames="US5874563-drawings-page-6.png,US5874563-drawings-page-9.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">weeks</a></figure-callout>     51      +
03/22/79   +20 weeks    135     +
06/28/79   +34 weeks    133     +
04/06/81   +127 <figure-callout id="141" label="weeks" filenames="US5874563-drawings-page-9.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">weeks</a></figure-callout>   141     +
08/20/82   +198 <figure-callout id="39" label="weeks" filenames="US5874563-drawings-page-6.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">weeks</a></figure-callout>   39      +
______________________________________
</pre>
    
    <div class="description-paragraph style-scope patent-text">
      </div> <pre xml:space="preserve" listing-type="tabular" class="style-scope patent-text">              TABLE 6
______________________________________
Transfusion-Associated Transmission
of HGV: Case STO
Draw                   ALT in  470 PCR
Date        Time       IU/L    Result
______________________________________
06/15/83    -1 day     23      -
07/18/83    +5 <figure-callout id="80" label="weeks" filenames="US5874563-drawings-page-7.png,US5874563-drawings-page-8.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">weeks</a></figure-callout>   80      +
10/31/83    +20 weeks  75      +
12/31/83    +28 <figure-callout id="30" label="weeks" filenames="US5874563-drawings-page-13.png,US5874563-drawings-page-15.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">weeks</a></figure-callout>  30      +
01/02/85    +81 weeks  90      -
03/20/85    +92 weeks  23      +
______________________________________
</pre>
    
    <div class="description-paragraph style-scope patent-text">The fourth case, also prospectively-defined, was a cardiac surgery patient who participated in a post-transfusion hepatitis study conducted in Sydney, Australia. The patient (PA-124), having no other identifiable risk factors, received 14 units of blood during surgery (4 units packed red cells, 10 units of platelets). of these 14 units one was HGV positive; the other 13 were HGV negative. HBV and HCV serologies of the 14 blood donors were negative with the exception of a reactive HCV EIA (first generation test). No other HCV test confirmed the positive finding.</div>
    <div class="description-paragraph style-scope patent-text">In patient PA-124 (Table 7), serum ALT was elevated beginning with a sample taken two weeks post-operation, and was observed to be at least 10 times the pre-operation level for a period of 14 weeks. PCR results for HCV performed on pre-transfusion, 4 week, and 8 week sera from PA-124, were all negative. Serum from this patient was tested for HGV RNA using the 470-20-1 PCR primers. A pre-transfusion sample was negative for HGV RNA. Positive results were demonstrated following transfusion, coinciding with and succeeding the ALT elevation. The presence of HGV RNA was detected out to one year post-transfusion. These data support the conclusion that HGV may be parenterally transmitted.</div>
    <div class="description-paragraph style-scope patent-text">
      </div> <pre xml:space="preserve" listing-type="tabular" class="style-scope patent-text">              TABLE 7
______________________________________
Transfusion-Associated Transmission
of HGV: Case PA-124
Weeks           ALT in  470 PCR
Post-Operation  IU/L    Result
______________________________________
pre-transfusion 7       -
2               74      +
4               86      +
8               135     +
12              179     +
14              78      +
18              9       +
24              6       +
36              11      +
52              11      +
64              23      -
84              10      -
______________________________________
</pre>
    
    <div class="description-paragraph style-scope patent-text">In addition to prospectively-defined post-transfusion transmission cases, additional cases of HGV infection were identified in risk groups defined by multiple transfusions and intravenous drug use (IVDU) (Table 8).</div>
    <div class="description-paragraph style-scope patent-text">
      </div> <pre xml:space="preserve" listing-type="tabular" class="style-scope patent-text">              TABLE 8
______________________________________
HGV RT-PCR Testing of Coded Sera:
Selected Hepatitis and Parenteral Risk Groups
Group           Number Tested
                           Number Positive
______________________________________
<figure-callout id="10" label="Autoimmune Hepatitis" filenames="US5874563-drawings-page-12.png,US5874563-drawings-page-19.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">Autoimmune Hepatitis</a></figure-callout>
                10         0
<figure-callout id="20" label="Primary Biliary Cirrhosis" filenames="US5874563-drawings-page-4.png,US5874563-drawings-page-5.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">Primary Biliary Cirrhosis</a></figure-callout>
                20         0
Suspected Acute NonA-E
                24         2
Hepatitis
Chronic Hepatitis (NonA-C)
                34         3
(confirmed by liver
biopsy)
<figure-callout id="20" label="Hepatocellular Carcinoma" filenames="US5874563-drawings-page-4.png,US5874563-drawings-page-5.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">Hepatocellular Carcinoma</a></figure-callout>
                20         2
<figure-callout id="20" label="Chronic HBV" filenames="US5874563-drawings-page-4.png,US5874563-drawings-page-5.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">Chronic HBV</a></figure-callout>     20         2
<figure-callout id="50" label="Chronic HCV" filenames="US5874563-drawings-page-10.png,US5874563-drawings-page-11.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">Chronic HCV</a></figure-callout>     50         6
Hemophilia      49         9
IVDU            54         15
Multiply Transfused <figure-callout id="100" label="Anemia" filenames="US5874563-drawings-page-7.png,US5874563-drawings-page-8.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">Anemia</a></figure-callout>
                100        19
______________________________________
</pre>
    
    <div class="description-paragraph style-scope patent-text">Among 100 multiply-transfused sickle cell anemia and thalassemia patients, 19 (19%) were found to have detectable serum HGV RNA. Similarly, 9 of 49 hemophilia patients (18%) were HGV positive with 470-20-1 and NS5 primers. Significantly, 15 of 54 (28%) IVDU were found to be PCR positive for HGV RNA. Infection rates in these parenteral risk groups (18-28%) appear to be higher than rates in blood donors with elevated ALT (1-2%). These results reinforce the significance of the parenteral route for HGV transmission.</div>
    <div class="description-paragraph style-scope patent-text">5. PCR Screening of Selected Hepatitis Desease Groups</div>
    <div class="description-paragraph style-scope patent-text">Sera from patients with acute and chronic hepatitis, hepatocellular carcinoma, HBV infection or HCV infection were tested for the presence of HGV using polymerase chain reaction (data presented in Table 8). In each of sets of specimens from patients with liver disease, HGV positive specimens were demonstrated (with the exception of specimens from patients with autoimmune hepatitis and primary biliary cirrhosis, both conditions not thought to be exclusively associated with an infectious agent).</div>
    <div class="description-paragraph style-scope patent-text">As shown in the collections of sera from post-transfusion hepatitis patients (Example 4F-4), HGV infection is established during acute hepatitis, but circulating viral RNA continues to be detected during chronic infection for periods of time measured in months to years.</div>
    <div class="description-paragraph style-scope patent-text">Approximately 10-20% co-infection rates were observed in patients with HBV and HCV infection. HGV infection is thus shown to be associated with hepatitis with or without co-infection with other hepatitis viruses. Co-infection may reflect similar risk factors and routes of transmission for these hepatitis viruses. As noted above, there is a higher prevalence of HGV in parenteral risk groups, such as hemophiliacs, IVDU's, and multiply transfused anemia patients (compared with other hepatitis risk groups).</div>
    <div class="description-paragraph style-scope patent-text">6. Persistent Infection by HGV in Humans</div>
    <div class="description-paragraph style-scope patent-text">Post-transfusion hepatitis cases BIZ, STO, and PA-124 were show to have PCR-detectable viral RNA up to 3.8, 1.8, and 1.0 years, respectively, following transfusion and acute infection. Additional serum samples were obtained from donor JC (Example 4F-1), one year and 1.5 years following the initial positive sample. These follow-up serum samples were also HGV positive. Additional sera from other high-ALT donors (T55806, T55875, R25342), obtained several months following the serum sample in which HGV infection was originally detected, were also positive. Similarly, when HGV infection was established in an experimental primate (CH1356, Example 4H), HGV RNA was detected over 1.5 years following innoculation. These data establish persistent HGV viremia in humans and experimental primates.</div>
    <div class="description-paragraph style-scope patent-text">G. Amplification of Long Fragments from Patient RNA for Sequencing.</div>
    <div class="description-paragraph style-scope patent-text">PCR primers were designed to amplify several informative regions of the HGV genome in order to obtain sequence information on varied HGV isolates. The primers 470EXT4-2189R (SEQ ID NO:119) and 470EXT4-29F (SEQ ID NO:120) were designed to amplify a 2.2 kb fragment that contained the original 470-20-1 sequence. RNA from samples was reverse-transcribed using "SUPERSCRIPT II" reverse transcriptase (Gibco/BRL, Gaithersburg, MD). The resulting cDNA was amplified using reagents for efficient long-range PCR ("XL PCR BUFFERS" and "rTth-XL", Perkin Elmer/Applied Biosystems Div., Foster City, Calif.).</div>
    <div class="description-paragraph style-scope patent-text">The amplification reaction was considered to be positive if a band of the correct size on agarose gel electrophoresis was detected. The sample was confirmed as positive by preliminary DNA sequencing of the amplification product. The following sera samples tested positive for HGV RNA by this amplification method: PNF2161; R10291 (JC); and specimens from each of the North American, Egyptian, and Japanese groups. However, no positive samples were detected from the Peruvian sera.</div>
    <div class="description-paragraph style-scope patent-text">Successful amplification from a variety of HGV-positive specimens provides confirmation of the results obtained by PCR amplification using the 470-20-1 primer pair discussed above. Failure to obtain amplification, however, may reflect poor RNA quality or low copy number or local sequence differences among isolates such that the selected primer sets would not function universally.</div>
    <div class="description-paragraph style-scope patent-text">In order to obtain sequence information from the putative 5'-untranslated region of the HGV genome, primers were designed to amplify fragments from the 5'-untranslated region (based on the HGV PNF 2161-variant). The two fragments were defined by the following primer sets: FV94-22F (SEQ ID NO:124) and FV94-724R (SEQ ID NO:125), yielding a 728 base pair fragment; and FV94-94F (SEQ ID NO:126) and FV94-912R (SEQ ID NO:127), yielding an 847 base pair fragment.</div>
    <div class="description-paragraph style-scope patent-text">The conditions just described to promote efficient long-range PCR were used. Products were obtained from most of the samples tested, providing additional confirmation of the presence of HGV RNA in the samples.</div>
    <div class="description-paragraph style-scope patent-text">H. Inrectivety of HGV in Primates.</div>
    <div class="description-paragraph style-scope patent-text">Two chimpanzees (designated CH1323 and CH1356), six cynomolgus monkeys (CY143, CY8904, CY8908, CY8912, CY8917, and CH8918), and six Mystax (MY29, MY131, MY98, MY187, MY229, MY254) subjects were inoculated with PNF 2161. Pre-inoculation and post-inoculation sera were monitored for ALT and for the presence of HGV RNA sequences (as determined by PCR screening--described above).</div>
    <div class="description-paragraph style-scope patent-text">One cynomologous monkey (CY8904) showed a positive RNA PCR result (39 days post-inoculation) and one indeterminant result from a total of 17 seperate blood draws. In one chimpanzee, designated CH1356, was sustained viremia observed by RT-PCR. As shown in Table 9, no significant ALT elevation was observed, and circulating virus was detected only at time points considerably after inoculation. Viremia was observed at and following 118 days post-inoculation. Suggestive reactivity was also observed in the first post-inoculation time-point (8 days), which may indicate residual inoculum.</div>
    <div class="description-paragraph style-scope patent-text">
      </div> <pre xml:space="preserve" listing-type="tabular" class="style-scope patent-text">              TABLE 9
______________________________________
ALT and PCR Results from CH1356 Following
Inoculation with PNF 2161
Days Post-
Inoculation     ALT*     HGV PCR
______________________________________
0               59       -
8               65       ±
15              85       -
22              89       -
29              89       -
36              86       -
39              31       -
47              74       -
54              40       -
61              57       -
84              65       ±
89              63       +
98              64       -
118             84       +
125             73       +
134             74       +
159             80       +
610             (ALT not +
                available)
______________________________________
 *average ALT baseline before inoculation was 50.
</pre>
    
    <div class="description-paragraph style-scope patent-text">The data presented above indicate that HGV infection was persistent up to 1.7 years in an experimental primate.</div>
    <div class="description-paragraph style-scope patent-text">I. CHARACTERIZATION OF THE VIRAL GENOME.</div>
    <div class="description-paragraph style-scope patent-text">The isolation of 470-20-1 from a cDNA library (Example 1) suggests that the viral genome detected in PNF 2161 is RNA. Further experiments to confirm the identity of the HGV viral genome as RNA include the following.</div>
    <div class="description-paragraph style-scope patent-text">Selective degradation of either RNA or DNA (e.g., by DNase-free RNase or RNase-free DNase) in the original cloning source followed by amplification with HGV specific primers and detection of the amplification products serves to distinguish RNA from DNA templates.</div>
    <div class="description-paragraph style-scope patent-text">An alternative method makes use of amplification reactions (nucleic acids from the original cloning source as template and HGV specific primers) that employ (i) a DNA-dependent DNA polymerase, in the absence of any RNA-dependent DNA polymerase (i.e., reverse transcripase) in the reactions, and (ii) a DNA-dependent DNA polymerase and an RNA-dependent DNA polymerase in the reactions. In this method, if the HGV genome is DNA or has a DNA intermediate, then amplified product is detected in both types of amplification reactions. If the HGV genome is only RNA, the amplified product is detected in only the reverse transcriptase-containing reactions.</div>
    <div class="description-paragraph style-scope patent-text">Total nucleic acid (i.e., DNA or RNA) was extracted from PNF 2161, using proteinase K and SDS followed by phenol extraction, as described in Example 4C. The purified nucleic acid was then amplified using polymerase chain reaction (PCR) where either (i) the PCR was preceded by a reverse transcription step, or (ii) the reverse transcription step was omitted. Amplification was reproducibly obtained only when the PCR reactions were preceded by reverse transcription. As a control, DNA templates were successfully amplified in separate reactions. These results demonstrate that the nature of the HGV viral genome is RNA.</div>
    <div class="description-paragraph style-scope patent-text">The strand of the cloned, double-stranded DNA sequence that was originally present in PNF 2161 may be deduced by various means, including the following. Northern or dot blotting of the unamplified genomic RNA from an infected source serum can be performed, followed by hybridization of duplicate blots to probes corresponding to each strand of the cloned sequence. Alternatively, single-stranded cDNA probes isolated from M13 vectors (Messing), or multiple strand-specific oligonucleotide probes are used for added sensitivity. If the source serum contains single-stranded RNA, only one probe (i.e., sequences from one strand of the 470-20-1 clones) yield a signal, under appropriate conditions of hybridization stringency. If the source serum contains double-stranded RNA, both strand-probes will yeild a signal.</div>
    <div class="description-paragraph style-scope patent-text">The polymerase chain reaction, prefaced by reverse transcription using one or the other specific primer, represents a much more sensitive alternative to Northern blotting. Genomic RNA extracted from purified virions present in PNF 2161 serum is used as the input template into each RT/PCR. Rather than cDNA synthesis with random hexamers, HGV sequence-specific primers were used. One cDNA synthesis reaction was performed with a primer complementary to one strand of the cloned sequence (e.g., 470-20-1-77F); a second cDNA synthesis reaction was also performed using a primer derived from the opposite strand (e.g., 470-20-1-211R).</div>
    <div class="description-paragraph style-scope patent-text">The resulting first strand cDNA was amplified in using two HGV specific primers. Controls were included for successful amplification by PCR (e.g., DNA controls). RNA transcripts from each strand of the cloned sequence was also used, to control also for the reverse transcription efficiency obtained when using the specific primers which are described.</div>
    <div class="description-paragraph style-scope patent-text">Specific products were detected by agarose gel electrophoresis with ethidium bromide staining. DNA controls (i.e., double-stranded DNA controls for the PCR amplifcation) were successfully amplified regardless of the primer used for reverse transcription. Single-stranded RNA transcripts (i.e., controls for reverse transcription efficiency and strand specificity) were amplified only when the opposite-strand primer was used for cDNA synthesis.</div>
    <div class="description-paragraph style-scope patent-text">The PNF-derived HGV polynucleotide gave rise to a specific amplified product only when the primer 470-20-1-211R was used for reverse transcription, thus indicating that the original HGV polynucleotide sequence present in the serum is complementary to 470-20-1-211R and is likely a single-strand RNA.</div>
    <heading class="style-scope patent-text">Example 5</heading>
    <heading class="style-scope patent-text">Sucrose Densiry Gradient Separation of PNF2161</heading>
    <div class="description-paragraph style-scope patent-text">A. Banding of PNF-2161 Agent.</div>
    <div class="description-paragraph style-scope patent-text">A continuous gradient of 10-60% sucrose ("ULTRAPURE", Gibco/BRL) in TNE (50 mM Tris-Cl, pH 7.5, 100 mM NaCl, 1 mM EDTA) was prepared using a gradient maker from Hoefer Scientific (San Francisco, Calif.). Approximately 12.5 ml of the gradient was overlaid with 0.4 ml of PNF serum which had been stored at -70° C., rapidly thawed at 37° C., then diluted in TNE.</div>
    <div class="description-paragraph style-scope patent-text">The gradient was then centrifuged in the SW40 rotor (Beckman Instruments) at 40,000 rpm (approximately 200,000×g at r<sub class="style-scope patent-text">av</sub>) at 4° C. for approximately 18 hours. Fractions of volume approximately 0.6 ml were collected from the bottom of the tube, and 0.5 ml was weighed directly into the. ultracentrifuge tube, for calculation of density.</div>
    <div class="description-paragraph style-scope patent-text">
      </div> <pre xml:space="preserve" listing-type="tabular" class="style-scope patent-text">              TABLE 10
______________________________________
Measured Densities of PNF Fractions
and Presence of 470-20-1
Fraction     Density 470-20-1 Detected*
______________________________________
1            1.274   -
2            1.274   -
3            1.266   -
4            1.266   -
5            1.260   -
6            1.254   -
7            1.248   +
8            1.206   +
9            1.146   +
10           1.126   +++
11           1.098   +++++
12           1.068   +++
13           1.050   +
14           1.034   +
15           1.036   +
16           1.018   -
17           1.008   +
18           1.020   +
______________________________________
 *"+" and "-" scores were initially based on 40cycle PCR. In order to
 distinguish "+", "++", "+++", and "++++", fraction giving initial positiv
 scores (7-18) were amplified with 30 cycles of PCR.
</pre>
    
    <div class="description-paragraph style-scope patent-text">The putative viral particles were then pelleted by centrifugation at 40,000 rpm in the Ti70.1 rotor (approximately 110,000×g) at 4° C. for 2 hours, and RNA was extracted using the acid guanidinium phenol technique ("TRI REAGENT", Molecular Research Center, Cincinnati, Ohio), and alcohol-precipitated using glycogen as a carrier to improve recovery. The purified nucleic acid was dissolved in an RNase-free buffer containing 2 mM DTT and 1 U/μl recombinant RNasin.</div>
    <div class="description-paragraph style-scope patent-text">Analysis of the gradient fractions by RNA PCR (Example 4C) showed a distinct peak in the 470-20-1 specific signal, localized in fractions of density ranging from 1.126 to 1.068 g/ml (Table 10). The 470-20-1 signal was thus shown, under these conditions, to form a discrete band, consistent with the expected behavior of a viral particle in a sucrose gradient.</div>
    <div class="description-paragraph style-scope patent-text">B. Relative Viral Particle Densities.</div>
    <div class="description-paragraph style-scope patent-text">PNF 2161 has been demonstrated to be co-infected with HCV (see above). In order to compare the properties of the 470-20-1 viral particle to other known hepatitis viral particles, the serum PNF 2161 and a sample of purified Hepatitis A Virus were layered on a sucrose gradient (as described above). Fractions (0.6 ml) were collected, pelleted and the RNA extracted. The isolated RNA from each fraction was subjected to amplification reactions (PCR) using HAV (SEQ ID NO:5; SEQ ID NO:6), HCV (SEQ ID NO:7; SEQ ID NO:8) and 470-20-1 (SEQ ID NO:9, SEQ ID NO:10) specific primers.</div>
    <div class="description-paragraph style-scope patent-text">Product bands were identified by electrophoretic separation of the amplification reactions on agarose gels followed by ethidium bromide staining. The results of this analysis are presented in Table 11.</div>
    <div class="description-paragraph style-scope patent-text">
      </div> <pre xml:space="preserve" listing-type="tabular" class="style-scope patent-text">              TABLE 11
______________________________________
Average Density
            HAV         HCV    470-20-1
______________________________________
1.269       -           -      -
1.263       +           -      -
1.260       +           -      -
1.246       ++          -      -
1.238       ++          -      -
1.240       +           -      -
1.207       +           -      -
1.193       +           -      -
1.172       +           ±   -
1.150       +           ±   ±
1.134       +           +      ±
1.118       +           +      +
1.103       +           +      +
1.118       +           +      +
1.103       +           +      +
1.088       ±        +      +
1.084       -           +      +
1.080       -           +      +
1.070       -           +      +
1.057       -           +      ±
1.035       -           ±   -
1.017       -           -      -
1.009       -           -      -
______________________________________
</pre>
    
    <div class="description-paragraph style-scope patent-text">These results suggest that 470-20-1 particles are more similar to HCV particles than to HAV.</div>
    <div class="description-paragraph style-scope patent-text">Further, serum PNF 2161 and HAV particles were treated with chloroform before sucrose gradient centrifugation. The results of these experiments suggest that 470-20-1 agent may be an enveloped virus since it has more similar properties to an enveloped Flaviviridae member (HCV) than a non-enveloped virus (HAV).</div>
    <heading class="style-scope patent-text">Example 6</heading>
    <heading class="style-scope patent-text">Genetation of 470-20-1 Extension Clones</heading>
    <div class="description-paragraph style-scope patent-text">A. Anchoe PCR.</div>
    <div class="description-paragraph style-scope patent-text">RNA was extracted directly from PNF2161 serum as described in Example 1. The RNA was passed through a "CHROMA SPIN" 100 gel filtration column (Clontech) to remove small molecular weight impurities. CDNA was synthesized using a BMB CDNA synthesis kit. After CDNA synthesis, the PNF cDNA was ligated to a 50 to 100 fold excess of KL-1/KL-2 SISPA or JML-A/JML-B linkers (SEQ ID NO:11/SEQ ID NO:12, and SEQ ID NO:17/SEQ ID NO:18, respectively) and amplified for 35 cycles using either the primer KL-1 or the primer JML-A.</div>
    <div class="description-paragraph style-scope patent-text">The 470 extension clones were generated by anchored PCR of a 1 μl aliquot from a 10 μl ligation reaction containing EcoRI digested (dephosphorylated) lambda gt11 arms (1 μg) and EcoRI digested PNF cDNA (0.2 μg). PCR amplification (40 cycles) of the ligation reaction was carried out using the lambda gt11 reverse primer (SEQ ID NO:13) in combination with either 470-20-77F (SEQ ID NO:9) or 470-20-1-211R (SEQ ID NO:10). All primer concentrations for PCR were 0.2 μM.</div>
    <div class="description-paragraph style-scope patent-text">The amplification products (9 μl/100 μl) were separated on a 1.5% agarose gel, blotted to "NYTRAN" (Schleicher and Schuell, Keene, N.H.), and probed with a digoxygenin labelled oligonucleotide probe specific for 470-20-1. The digoxygenin labeling was performed according to the manufacturer's recommendations using terminal transferase (BMB). Bands that hybridized were gel-purified, cloned into the "TA CLONING VECTOR pCR II" (Invitrogen), and sequenced.</div>
    <div class="description-paragraph style-scope patent-text">Numerous clones having both 5' and 3' extensions to 470-20-1 were identified. All sequences are based on a consensus sequence from the sequencing of at least two independent isolates. This Anchor PCR approach was repeated in a similar manner to obtain further 5' and 3' extension sequences. These PCR amplification reactions were carried out using the lambda gt11 reverse primer (SEQ ID NO:13) in combination with HGV specific primers derived from sequences obtained from previous extension clones. The substrate for these reactions was unpackaged PNF 2161 2-cDNA source DNA.</div>
    <div class="description-paragraph style-scope patent-text">Sequencing was carried out using "DYEDEOXY TERMINATOR CYCLE SEQUENCING" (a modification of the procedure of Sanger, et al.) on an Applied Biosystems model 373A DNA sequencing system according to the manufacturer's recommendations (Applied Biosystems, Foster City, Calif.). Sequence data is presented in the Sequence Listing. Sequences were compared with "GEN-BANK", EMBL database and dbEST (National Library of Medicine) sequences at both nucleic acid and amino acid levels. Search programs FASTA, BLASTP, BLASTN and BLASTX (Altschul, et al.) indicated that these sequences were novel as both nucleic acid and amino acid sequences.</div>
    <div class="description-paragraph style-scope patent-text">Individual clones obtained using a selected primer pair were aligned to yield a consensus sequence. The series of consensus sequences used to construct the sequence for the HGV-PNF 2161 variant was as follows: 4E3, SEQ ID NO:26; 3E3, SEQ ID NO:27; 2E5, SEQ ID NO:28; 1E5, SEQ ID NO:29; 4E5, SEQ ID NO:30; 3E5, SEQ ID NO:31; 2E3, SEQ ID NO:32; 1E3, SEQ ID NO:33; 4E5-20, SEQ ID NO:34; 5E3, SEQ ID NO:39; 6E3, SEQ ID NO:40; 7E3, SEQ ID NO:42; 5E5, SEQ ID NO:43; 6E5(44F), SEQ ID NO:44; 8E3, SEQ ID NO:98; 9E3, SEQ ID NO:109; 10E3, SEQ ID NO:110; 11E3, SEQ ID NO:116; 12E3, SEQ ID NO:118; 5'-end, SEQ ID NO:175; and 3'-END, SEQ ID NO:167.</div>
    <div class="description-paragraph style-scope patent-text">The individual consensus sequences were aligned, overlapping sequences identified and a consensus sequence for the HGV-PNF 2161 variant was determined. This consensus sequence was compared with the sequences obtained for four other HGV variants: JC (SEQ ID NO:182), BG34 (SEQ ID NO:176), T55806 (SEQ ID NO:178), and EB20-2 (SEQ ID NO:180).</div>
    <div class="description-paragraph style-scope patent-text">The consensus sequence of the HGV-PNF 2161 variant consists of 9391 base pairs presented as SEQ ID NO:14. This sequence represents a continuous open reading frame (SEQ ID NO:15). A Kyte-Doolittle hydrophobicity plot of the polyprotein is presented as FIG. 11.</div>
    <div class="description-paragraph style-scope patent-text">The relationship between the original 470-20-1 clone and the sequences obtained by extension is shown schematically in FIG. 1. As seen in the figure, the DNA strand having opposite polarity to the protein coding sequence of 470-20-1 comprising a long continuous open reading frame.</div>
    <div class="description-paragraph style-scope patent-text">The amino acid sequence of HGV was compared against the sequences of all viral sequence in the PIR database (IntelliGenetics, Inc., Mountain View, Calif.) of protein sequences. The comparison was carried out using the "SSEARCH" program of the "FASTA" suite of programs version 1.7 (Pearson, et al.). Regions of local sequence similarities were found between the HGV sequences and two viruses in the Flaviviridae family of viruses. The similarity alignments are presented in FIGS. 5A and 5B.</div>
    <div class="description-paragraph style-scope patent-text">Present in these alignments are motifs for the RNA dependent RNA polymerase (RDRP) of these viruses. Conserved RDRP amino acid motifs are indicated in FIGS. 5A and 5B by stars and uppercase, bold letters (Koonin and Dolja). These alignments demonstrate that this portion of the HGV coding sequence correspond to RDRP. This alignment data combined with the data concerning the RNA genome of HGV supports the placement of HGV as a member of the Flaviviridae family.</div>
    <div class="description-paragraph style-scope patent-text">The global amino acid sequence identities of the HGV polyprotein (SEQ ID NO:15) with HoCV (Hog Cholera Virus) and HCV are 17.1% and 25.5%, respectively. Such levels of global sequence identity demonstrates that HGV is a separate viral entity from both HOCV and HCV. To illustrate, in two members of the Flaviviridae family of viruses BVDV (Bovine Diarrhea Virus) and HCV, 16.2% of the amino acids can be globally aligned with HGV.</div>
    <div class="description-paragraph style-scope patent-text">Members within a genus generally show high homology when aligned globally, for example, BVDV vs. HOCV show 71.2% identity. Various members (variants) of the un-named genus of which HCV is a member are between 65% and 100% identical when globally aligned.</div>
    <div class="description-paragraph style-scope patent-text">B. RACE PCR: 5' End Cloning.</div>
    <div class="description-paragraph style-scope patent-text">Clones representing the 5'-end of the HGV genome were obtained by a modified Anchor PCR approach that utilized RACE (Rapid Amplification of cDNA Ends) technology. The RACE method was originally described by Frohman, et al., (1988) and Belyausky, et al., (1989). Briefly, the 5'-end clones of HGV were obtained as follows.</div>
    <div class="description-paragraph style-scope patent-text">First-strand cDNA synthesis was primed using random hexamers and synthesis was carried out using either "SUPERSCRIPT II" or "rTth" reverse transcriptase (GIBCO/BRL). After first-strand synthesis, the RNA template was degraded by base hydrolysis (NaOH). The cDNA sample was neutralized by the addition of acetic acid and purified by absorption to a glass matrix support ("GENO-BIND," C.lontech, Palo Alto, Calif.). Following purification, the cDNA was concentrated by ethanol precipitation and washed twice with 80% ethanol.</div>
    <div class="description-paragraph style-scope patent-text">The originally described RACE method was modified as follows. A single-stranded oligonucleotide anchor (SEQ ID NO:174) (Clontech) was ligated to the 3' end of the first-strand cDNA using T4 RNA ligase in the presence of cobalt chloride. The oligonucleotide anchor was obtained from the manufacturer with two modifications: (i) the 3'-end of the anchor was modified with an amino group which prevents concatamer formation, and (ii) the 5'-end contains a phosphate group which allows ligation to the first-strand cDNA.</div>
    <div class="description-paragraph style-scope patent-text">After ligation of the anchor, the cDNA was used as a template for PCR amplification using several HGV-specific primers in combination with a primer complementary to the anchor sequence (AP primer, SEQ ID NO:134). The resulting amplification products were separated by agarose gel electrophoresis, transferred to filters and hybridized with a nested, HGV-specific oligonucleotide probe. Bands that hybridized to the HGV-probe were isolated, cloned into "pCR-II" (Invitrogen, San Diego, Calif.) and sequenced.</div>
    <div class="description-paragraph style-scope patent-text"> <figure-callout id="3'" label="C. HGV" filenames="US5874563-drawings-page-3.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">C. HGV</a></figure-callout> 3' End Cloning.</div>
    <div class="description-paragraph style-scope patent-text">Clones representing the 3'-end of the HGV genome were obtained by a modified anchored RT-PCR method. Briefly, poly A polymerase (GIBCO/BRL, Gaithersburg, Md.) was used to catalyze the addition of a poly(A) tail to PNF 2161 RNA prior to CDNA synthesis. The poly(A) addition was performed according to the manufacturer's recommendations. Following purification of the poly(A) modified RNA, reverse transcription with "SUPERSCRIPT II" (GIBCO/BRL) was carried out using primer GV-5446IRT (SEQ ID NO:184). The resulting cDNA was amplified by PCR using the following primer set: GV59-5446F (SEQ ID NO:171) and GV-5446IR (SEQ ID NO:172).</div>
    <div class="description-paragraph style-scope patent-text">After amplification, the products were separated by agarose gel electrophoresis, transferred to filters and hybridized with a digoxigenin-labelled oligonucleotide probe (E5-7-PRB, SEQ ID NO:173). Products that hybridized with the oligonucleotide were isolated, purified, cloned into "pCR-II" and sequenced. The two clones isolated by this method were MP3-3 (SEQ ID NO:168) and MP3-7 (SEQ ID NO:169).</div>
    <heading class="style-scope patent-text">Example 7</heading>
    <heading class="style-scope patent-text">Isolation of 470-20-1 Fusion Protein</heading>
    <div class="description-paragraph style-scope patent-text">A. Expression and Purification of 470-20-1/Glutathione-S-Transferase Fusion Pritein</div>
    <div class="description-paragraph style-scope patent-text">Expression of a glutathione-S-transferase (sj26) fused protein containing the 470-20-1 peptide was achieved as follows. A 237 base pair insert (containing 17 nucleotides of SISPA linkers on both sides) corresponding to the original lambda gt11 470-20-1 clone was isolated from the lambda gt11 470-20-1 clone by polymerase chain reaction using primers gt11 F(SEQ ID NO:25) and gt11 R(SEQ ID NO:13) followed by Eco RI digestion.</div>
    <div class="description-paragraph style-scope patent-text">The insert was cloned into a modified PGEX vector, PGEX MOV. PGEX MOV encodes sj26 protein fused with six histidines at the carboxy terminal end (sj26his). The 470-20-1 polypeptide coding sequences were introduced into the vector at a cloning site located downstream of sj26his coding sequence in the vector. Thus, the 470-20-1 polypeptide is expressed as sj26his/470-20-1 fusion protein. The sj26 protein and six histidine region of the fusion protein allow the affinity purification of the fusion protein by dual chromatographic methods employing glutathione-conjugated conjugated beads (Smith, D.B., et al.) and immobilized metal ion beads (Hochula; Porath).</div>
    <div class="description-paragraph style-scope patent-text">E. coli strain W3110 (ATCC catalogue number 27352) was transformed with PGEX MOV and PGEX MOV containing 470-20-1 insert. Sj26his protein and 470-20-1 fusion protein were induced by the addition of 2 mM isopropyl-β-thiogalactopyranoside (IPTG). The fusion proteins were purified either by glutathione-affinity chromatography or by immobilized metal ion chromatography (IMAC) according to the published methods (Smith, D.B., et al.; Porath) in conjunction with conventional ion-exchange chromatography.</div>
    <div class="description-paragraph style-scope patent-text">The purified 470-20-1 fusion protein was immunoreactive with PNF 2161. However, purified sj26his protein was not immunoreactive with PNF 2161, indicating the presence of specific immunoreaction between the 470-20-1 peptide and PNF 2161.</div>
    <div class="description-paragraph style-scope patent-text">B. Isolation of 470-20-1/b-Galacrosidase Fusion Protein</div>
    <div class="description-paragraph style-scope patent-text">KM392 lysogens infected either with lambda phage gt11 or with gt11/470-20-1 are incubated in 32° C. until the culture reaches to an O.D. of 0.4. Then the culture is incubated in a 43° C. water bath for 15 minutes to induce gt11 peptide synthesis, and further incubated at 37° C. for 1 hour. Bacterial cells are pelleted and lysed in lysis buffer (10 mM Tris, pH 7.4, 2% "TRITON X-100" and 1% aprotinin). Bacterial lysates are clarified by centrifugation (10K, for 10 minutes, Sorvall JA20 rotor) and the clarified lysates are incubated with <figure-callout id="4B" label="Sepharose" filenames="US5874563-drawings-page-19.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">Sepharose</a></figure-callout> 4B beads conjugated with anti-β-galactosidase (Promega).</div>
    <div class="description-paragraph style-scope patent-text">Binding and elution of β-galactosidase fusion proteins are performed according to the manufacturer's instruction. Typically binding of the proteins and washing of the column are done with lysis buffer. Bound proteins are eluted with 0.1M carbonate/bicarbonate buffer, <figure-callout id="10" label="pH" filenames="US5874563-drawings-page-12.png,US5874563-drawings-page-19.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">pH</a></figure-callout> 10. The purified 470-20-1/b-galactosidase protein is immunoreactive with both PNF2161 and anti-b-galactosidase antibody. However, β-galactosidase, expressed by gt11 lysogen and purified, is not immunoreactive with PNF2161 but immunoreactive with anti-β-galactosidase antibody.</div>
    <heading class="style-scope patent-text">Example 8</heading>
    <heading class="style-scope patent-text">Purification of the 470-20-1 Fusion Protein and Preparation of Anti-470-20-1 Antibody</heading>
    <div class="description-paragraph style-scope patent-text">A. Glutathione Affinity Purification</div>
    <div class="description-paragraph style-scope patent-text">Materials included 50 ml glutathione affinity matrix reduced form (Sigma), XK 26/30 Pharmacia column, 2.5×10 cm Bio-Rad "ECONO-COLUMN" (Richmond, Calif.), Gilson (Middleton, WI) HPLC, DTT (Sigma), glutathione reduced form (Sigma), urea, and sodium phosphate dibasic.</div>
    <div class="description-paragraph style-scope patent-text">The following solutions were used in purification of the fusion protein:</div>
    <div class="description-paragraph style-scope patent-text">Buffer A: phosphate buffer saline, pH 7.4, and</div>
    <div class="description-paragraph style-scope patent-text">Buffer B: 50 mM Tris Ph 8.5, 8 mM glutathione, (reduced form glutathione)</div>
    <div class="description-paragraph style-scope patent-text">Strip buffer: 8M urea, 100 mM Tris pH 8.8, 10 mM glutathione, 1.5 NaCl.</div>
    <div class="description-paragraph style-scope patent-text">E. coli carrying the plasmid pGEX MOV containing 470-20-1 insert, were grown in a fermentor (20 liters). The bacteria were collected and lysed in phosphate buffered saline (PBS) containing 2 mM phenylmethyl sulfonyl fluoride (PMSF) using a micro-fluidizer. Unless otherwise noted, all of the following procedures were carried out at 4° C.</div>
    <div class="description-paragraph style-scope patent-text">The crude lysate was prepared for loading by placing lysed bacteria into "OAKRIDGE" tubes and spinning at 20K rpms (40k×g) in a Beckman model JA-20 rotor. The supernatant was filtered through a 0.4 μm filter and then through a 0.2 μm filter.</div>
    <div class="description-paragraph style-scope patent-text">The 2.5×10 cm "ECONO-COLUMN" was packed with the glutathione affinity matrix that was swelled in PBS for two hours at room temperature. The column was brought into equilibrium by washing with 4 bed volumes of PBS.</div>
    <div class="description-paragraph style-scope patent-text">The column was loaded with the crude lysate at a flow rate of 8 ml per minute. Subsequently, the column was washed with 5 column volumes of PBS at the same flow rate.</div>
    <div class="description-paragraph style-scope patent-text">The column was eluted by setting the flow rate to 0.75-1 ml/min. and introducing Buffer B. Buffer B was pumped through the column for 5 column volumes and two-minute fractions were collected. An exemplary elution profile is shown in FIG. 2. The content and purity of the proteins present in the fractions were assessed by standard SDS PAGE (FIG. 3). The 470-20-1/sj26his fusion protein was identified based on its predicted molecular weight and its immunoreactivity to PNF 2161 serum. For further manipulations, the protein can be isolated from fractions containing the fusion protein or from the gel by extraction of gel regions containing the fusion protein.</div>
    <div class="description-paragraph style-scope patent-text">B. Purification of Clone 470-20-1 Fusion Protein by Anion Exchange.</div>
    <div class="description-paragraph style-scope patent-text">Solutions include the following:</div>
    <div class="description-paragraph style-scope patent-text">Buffer A (10 mM sodium phosphate pH 8.0, 4M urea, 10 mM DTT);</div>
    <div class="description-paragraph style-scope patent-text">Buffer B (10 mM sodium phosphate pH 8.0, 4M urea, 10 mM DTT, 2.0M NaCl); and</div>
    <div class="description-paragraph style-scope patent-text">Strip Buffer (8M urea, 100 mM Tris pH 8.8, 10 mM glutathione, 1.5 NaCl).</div>
    <div class="description-paragraph style-scope patent-text">Crude lysate (or other protein source, such as pooled fractions from above) was loaded onto "HIGH-Q-50" (Biorad, Richmond, Calif.) column at a flow rate of 4.0 ml/min. The column was then washed with Buffer A for 5 column volumes at a flow rate of 4.0 ml/min.</div>
    <div class="description-paragraph style-scope patent-text">After these washes, a gradient was started and ran from Buffer A to Buffer B in 15 column volumes. The gradient then stepped to 100% Buffer B for one column volume. An exemplary gradient is shown in FIG. 4A. Fractions were collected every 10 minutes. Purity of the 470-20-1/sj26his fusion protein was assessed by standard SDS-PAGE (FIGS. 4B and 4C) and relevant fractions were pooled (approximately fractions 34 through 37, FIG. 4C).</div>
    <div class="description-paragraph style-scope patent-text">C. Prepation of Anti-470-20-1 Antibody</div>
    <div class="description-paragraph style-scope patent-text">The purified 470-20-1/sj26his fusion protein is injected subcutaneously in Freund's adjuvant in a rabbit. Approximately 1 mg of fusion protein is injected at <figure-callout id="21" label="days" filenames="US5874563-drawings-page-9.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">days</a></figure-callout> 0 and 21, and rabbit serum is typically collected at 6 and 8 weeks.</div>
    <div class="description-paragraph style-scope patent-text">A second rabbit is similarly immunized with purified sj26his protein.</div>
    <div class="description-paragraph style-scope patent-text">Minilysates are prepared from bacteria expressing the 470-20-1/sj26his fusion protein, sj26his protein, and β-galactosidase/470-20-1 fusion protein. The lysates are fractionated on a gel and transfered to a membrane. Separate Western blots are performed using the sera from the two rabbits.</div>
    <div class="description-paragraph style-scope patent-text">Serum from the animal immunized with 470-20-1 fusion protein is immunoreactive with all sj26his fusion protein in minilysates of IPTG induced E. coli W3110 that are transformed either with pGEX MOV or with pGEX MOV containing 470-20-1 insert. This serum is also immunoreactive with the fusion protein in the minilysate from the 470-20-1 lambda gt11 construct.</div>
    <div class="description-paragraph style-scope patent-text">The second rabbit serum is immunoreactive with both sj26his and 470-20-1/sj26his fusion proteins in the minilysates. This serum is not expected to immunoreactive with 470-20-1/β-galactosidase fusion protein in the minilysate from the 470-20-1 lambda gt11 construct. None of the sera are expected to be immunoreactive with β-galactosidase.</div>
    <div class="description-paragraph style-scope patent-text">Anti-470-20-1 antibody present in the sera from the animal immunized with the fusion protein is purified by affinity chromatography (using the 470-20-1 ligand).</div>
    <div class="description-paragraph style-scope patent-text">Alternatively, the fusion protein can be cleaved to provide the 470-20-1 antigen free of the sj-26 protein sequences. The 470-20-1 antigen alone is then used to generate antibodies as described above.</div>
    <heading class="style-scope patent-text">Example 9</heading>
    <heading class="style-scope patent-text">Rabbit Anti-Peptide Sera</heading>
    <div class="description-paragraph style-scope patent-text">Peptides were designed to cover the entire HGV sequence, in particular, to cover each of the functional groups in the non-structural and structural genes. Peptides were synthesized commercially by conventional techniques. Representative peptides are presented in Table 12.</div>
    <div class="description-paragraph style-scope patent-text">
      </div> <pre xml:space="preserve" listing-type="tabular" class="style-scope patent-text">              TABLE 12
______________________________________
              Size of
              Peptide End Points Relative
Desgination   (aa)    to SEQ ID NO: 14
______________________________________
PEP1/<figure-callout id="30" label="NS2a" filenames="US5874563-drawings-page-13.png,US5874563-drawings-page-15.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">NS2a</a></figure-callout>     30      2674/2763
PEP2/E1       16      733/780
PEP3/E2       18      1219/1272
PEP4/NS2B     18      3061/3114
PEP5/<figure-callout id="21" label="NS3" filenames="US5874563-drawings-page-9.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">NS3</a></figure-callout>      21      3571/3633
PEP6/NS3**    18      4909/4959
PEP7/NS4A     18      5275/5328
PEP8/NS4B     16      6097/6144
PEP9/NS5A     16      7033/7080
PEP10/NS5B    18      7783/7836
______________________________________
 **The NS3 peptide has an extraneous Cysteine on the C terminal end that i
 not in the HGVPNF 2161 variant polypeptide sequence; the actual sequence
 was a Q.
</pre>
    
    <div class="description-paragraph style-scope patent-text">The peptides were coupled to KLH. Using rabbits as host, the conjugated peptides were injected subcutaneously at multiple sites. Anti-peptide rabbit serum were generated by a commercial facility. A two-week immunization protocol was used with bleeds taken at alternate weeks.</div>
    <div class="description-paragraph style-scope patent-text">Rabbit anti-peptide sera were shown to be peptide specific and to have high titer. Rabbit anti-peptide sera also recognize corresponding recombinant proteins expressed in E. coi and baculovirus. Antibody endpoint titers range from 1:50,000 dilution to 1:625,000 dilution. Rabbit anti-peptide 7 (NS4a) had low end point titers of only 1:1,000. Accordingly, rabbit anti-serum to the NS4a protein expressed in, for example, the baculovirus system may be a more useful reagent.</div>
    <div class="description-paragraph style-scope patent-text">Rabbit anti-peptide sera are useful for immunoprecipitating corresponding HGV proteins expressed, for example, in baculovirus and vaccinia. Rabbit anti-peptide sera are also useful as capture antibody in EIAs to detect HGV antigen. Rabbit anti-peptide sera are further useful in the characterization of the HGV proteins.</div>
    <heading class="style-scope patent-text">Example 10</heading>
    <heading class="style-scope patent-text">SEROLOGY</heading>
    <div class="description-paragraph style-scope patent-text">A. Western Blot Analysis of Sera Panels</div>
    <div class="description-paragraph style-scope patent-text">The 470-20-1 fusion antigen (described above) was used to screen panels of sera. Many of the panels were of human sera derived both from individuals suffering from hepatitis and uninfected controls.</div>
    <div class="description-paragraph style-scope patent-text">Affinity purified 470-20-1 fusion antigen (Example 8) was loaded onto a 12% SDS-PAGE at 2 μg/cm. The gel was run for two hours at 200 V. The antigen was transfered from the gel to a nitrocellulose filter.</div>
    <div class="description-paragraph style-scope patent-text">The membrane was then blocked for 2 hours using a solution of 1% bovine serum albumin, 3% normal goat serum, 0.25% gelatin, 100 mM NaPO<sub class="style-scope patent-text">4</sub>, 100 mM NaCl, and 1% nonfat dry milk. The membrane was then dried and cut into 1-2 mm strips; each strip contained the 470-20-1 fusion antigen. The strip was typically rehydrated with TBS (150 mM NaCl; 20 mM Tris HCl, pH 7.5) and incubated in panel sera (1:100) overnight with rocking at room temperature.</div>
    <div class="description-paragraph style-scope patent-text">The strips were washed twice for five minutes each time in TBS plus "<figure-callout id="20" label="TWEEN" filenames="US5874563-drawings-page-4.png,US5874563-drawings-page-5.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">TWEEN</a></figure-callout> 20" (0.05%), and then washed twice for five minutes each time in TBS. The strips were then incubated in secondary antibody (Promega anti-human IgG-Alkaline Phosphatase conjugate, 1:7500), for 1 hour with rocking at room temperature. The strips were then washed twice×5 minutes in TBS+"<figure-callout id="20" label="TWEEN" filenames="US5874563-drawings-page-4.png,US5874563-drawings-page-5.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">TWEEN</a></figure-callout> 20", then twice×5 minutes in TBS.</div>
    <div class="description-paragraph style-scope patent-text">Bound antibody was detected by incubating the strips in a substrate solution containing BCIP (Example 2) and NBT (Example 2) in pH 9.5 buffer (100 mM Tris, 100 mM NaCl, 5 mM MgCl<sub class="style-scope patent-text">2</sub>). Color development was allowed to proceed for approximately 15 minutes at which point color development was halted by 3 washes in distilled H<sub class="style-scope patent-text">2</sub> O.</div>
    <div class="description-paragraph style-scope patent-text">Test sera were derived from the following groups of individuals: (i) blood donors, negative for HBV Ab, surface Ag, negative for HCV, HIV, HTLV-1 Abs; (ii) HBV, sera from individuals who are infected with Hepatitis B virus; (iii) HCV, sera from individuals infected with Hepatitis C virus by virtue of being reactive in a second-generation HCV ELISA assay; and (iv) HXV, individuals serologically negative for HAV, HBV, HCV, or HEV.</div>
    <div class="description-paragraph style-scope patent-text">The results of these screens are presented in Table 13.</div>
    <div class="description-paragraph style-scope patent-text">
      </div> <pre xml:space="preserve" listing-type="tabular" class="style-scope patent-text">              TABLE 13
______________________________________
470-20-1 Sera Panelling Result Summary
        No. Human*
Sample  Sera Tested
                   +         IND*    -
<figure-callout id="30" label="______________________________________ blood donor" filenames="US5874563-drawings-page-13.png,US5874563-drawings-page-15.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">______________________________________
blood donor</a></figure-callout>
        30         1 (3.3%)  2 (6.7%)
                                     27 (90.0%)
<figure-callout id="40" label="HBV" filenames="US5874563-drawings-page-5.png,US5874563-drawings-page-6.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">HBV</a></figure-callout>     40         7 (17.5%) 4 (10.0%)
                                     29 (72.5%)
<figure-callout id="38" label="HCV" filenames="US5874563-drawings-page-6.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">HCV</a></figure-callout>     38         11 (28.95%)
                             11 (28.95%)
                                     16 (42.1%)
HXV     122        20 (16.4%)
                             12 (9.8%)
                                     90 (73.8%)
______________________________________
 *Indeterminate, weak reactivity
</pre>
    
    <div class="description-paragraph style-scope patent-text">These results suggest the presence of the 470-20-1 antigen in a number of different sera samples. The antigen is not immunoreactive with normal human sera.</div>
    <div class="description-paragraph style-scope patent-text">B. General Elisa Protocal for Detection of Antibodies</div>
    <div class="description-paragraph style-scope patent-text">Polystyrene 96 well plates ("IMMULON II" (PGC)) are coated with 5 μg/ml (100 μL per well) antigen in 0.1M sodium bicarbonate buffer, pH 9.5. Plates are sealed with "PARAFILM" and stored at 4° C. overnight.</div>
    <div class="description-paragraph style-scope patent-text">Plates are aspirated and blocked with 300 <figure-callout id="10" label="uL" filenames="US5874563-drawings-page-12.png,US5874563-drawings-page-19.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">uL</a></figure-callout> 10% normal goat serum and incubated at 37° C. for 1 hr.</div>
    <div class="description-paragraph style-scope patent-text">Plates are washed 5 times with PBS 0.5% "TWEEN-20".</div>
    <div class="description-paragraph style-scope patent-text">Antisera is diluted in 1×PBS, pH 7.2. The desired dilution(s) of antisera (0.1 mL) are added to each well and the plate incubated 1 hour at 37° C. The plates are then washed 5 times with PBS 0.5% "TWEEN-20".</div>
    <div class="description-paragraph style-scope patent-text">Horseradish peroxidase (HRP) conjugated goat anti-human antiserum (Cappel) is diluted 1/5,000 in PBS. 0.1 mL of this solution is added to each well. The plate is incubated 30 min at 37° C., then washed 5 times with PBS.</div>
    <div class="description-paragraph style-scope patent-text">Sigma ABTS (substrate) is prepared just prior to addition to the plate.</div>
    <div class="description-paragraph style-scope patent-text">The reagent consists of 50 ml 0.05M citric acid, pH 4.2, 0.078 <figure-callout id="30" label="ml" filenames="US5874563-drawings-page-13.png,US5874563-drawings-page-15.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">ml</a></figure-callout> 30% hydrogen peroxide solution and 15 mg ABTS. 0.1 ml of the substrate is added to each well, then incubated for 30 min at room temperature. The reaction is stopped with the addition of 0.050 <figure-callout id="5" label="mL" filenames="US5874563-drawings-page-19.png,US5874563-drawings-page-3.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">mL</a></figure-callout> 5% SDS (w/v). The relative absorbance is determined at 410 nm.</div>
    <heading class="style-scope patent-text">Example 11</heading>
    <heading class="style-scope patent-text">Expression of Selected HGV Antigens</heading>
    <div class="description-paragraph style-scope patent-text">The entire coding sequence of HGV was subcloned into greater than 50 distinct overlapping cDNA fragments. The length of most cDNA fragments ranged from about 200 bp to about 500 bp. The cDNA fragments were cloned separately into the expression vector, pGEX-HisB. This vector is similar to pGEX-MOV, described above.</div>
    <div class="description-paragraph style-scope patent-text">pGEX-hisB is a modification of pGEX-2T (Genbank accession number A01438; a commercially available expression vector). The vector pGEX-2T has been modified by insertion of a NcoI site directly downstream from the thrombin cleavage site. This site is followed by a BamHI site, which is followed by a poly-histidine (six histidines) encoding sequence, followed by the EcoRI site found in pGEX-2T. Coding sequences of interest are typically inserted between the NcoI site and the BamHI site. In FIG. 6 (SEQ ID NO:115), the inserted sequence encodes the GE3-2 antigen. The rest of the vector sequence is identical to pGEX-2T. Expression of fusion protein is carried out essentially as described above with other pGEX-derived expression vectors.</div>
    <div class="description-paragraph style-scope patent-text">Cloning of all 50 fragments was carried out essentially as described below, where specific primers were selected for each of the 50 coding regions. Each HGV insert DNA is PCR amplified from RNA extracted from PNF 2161 or other HGV(+) sera using a specific set of primers as described in Example 4C. Typically, the 5' primer contained a NcoI restriction site and the 3' primer contained a BamHI restriction site. The NcoI primers in the amplified fragments allowed in-frame fusion of amplified coding sequences to the GST-Sj26 coding sequence in the expression vectors pGEX-Hisb or pGEX MOV.</div>
    <div class="description-paragraph style-scope patent-text">Amplified HGV insert DNA is digested with restriction enzymes NcoI and Bam HI. Digested insert DNA is gel purified and ligated with NcoI and BamHI digested PGEX hisB or pGEX MOV. E. coil strain W3110 (ATCC #27325, American Type Culture Collection, Rockville, Md.) was transformed with the ligation product. Ampicillin resistant colonies were selected. Presence of the insert was confirmed by the PCR amplification of the insert from the ampicillin resistant colony using primers homologous to pGEX vector sequences flanking the inserted molecules (primers GLI F (SEQ ID NO:235) and GLI R (SEQ ID NO:236).</div>
    <div class="description-paragraph style-scope patent-text">The size of the PCR amplification product is the insert size plus approximately 160 bp derived from vector. Transformants with appropriate inserts were selected and subjected to protein induction by IPTG as described in Example 7. Expressed recombinant proteins were analyzed for specific immunoreactivity against putative HGV-infected human sera by Western blot.</div>
    <div class="description-paragraph style-scope patent-text">Eight fragments designated GE3, GE9, GE15, GE17, GE4, EXP3, GE1-N and GE-57 encoded antigens that gave a clear immunogenic response when reacted with putative HGV-infected human sera.</div>
    <div class="description-paragraph style-scope patent-text">A. Cloning of GE3, GE9, GE15, GE17, GE4, EXP3, GEI-N and GE57.</div>
    <div class="description-paragraph style-scope patent-text">The coding sequence inserts for clones GE3, GE9, GE15, GE17, GE4, EXP3, GE1-N and GE57 were generated by polymerase chain reaction from SISPA-amplified double-stranded cDNA or RNA obtained from PNF 2161 or T55806 using PCR primers specific for each fragment. Following Table 14 lists the coordinates of each clone relative to SEQ ID NO:14 and the primer sets used for generation of each clone insert.</div>
    <div class="description-paragraph style-scope patent-text">
      </div> <pre xml:space="preserve" listing-type="tabular" class="style-scope patent-text">              TABLE 14
______________________________________
              Coordinate
      Serum   on SEQ    F Primer  R Primer
Clone Source  ID NO: 14 (SEQ ID NO:)
                                  (SEQ ID NO:)
______________________________________
GE3   PNF     6615-6977 GE-3F     GE-3R
      2161              (SEQ ID NO: 46)
                                  (SEQ ID NO: 47)
GE9   PNF     8154-8441 GE-9F     GE-9R
      2161              (SEQ ID NO: 48)
                                  (SEQ ID NO: 49)
GE15  PNF     3615-3935 GE-15F    GE-15R
      2161              (SEQ ID NO:
                                  (SEQ ID NO: 112)
                        111)
GE17  PNF     3168-3305 GE-17F    GE-17R
      2161              (SEQ ID NO:
                                  (SEQ ID NO: 114)
                        113)
GE4   PNF     6825-7226 GE4F      GE4R
      2161              (SEQ ID NO:
                                  (SEQ ID NO: 150)
                        149)
EXP3  PNF     6648-7658 470EXP3F  470EXP3R
      2161              (SEQ ID NO:
                                  (SEQ ID NO: 152)
                        151)
GE1-N PNF     5850-6239 GE1-NF    GE1-NR
      2161              (SEQ ID NO:
                                  (SEQ ID NO: 238)
                        237)
<figure-callout id="271" label="GE57 T55806" filenames="US5874563-drawings-page-9.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">GE57  T55806</a></figure-callout>  271*-456* GE57F     GE57R
                        (SEQ ID NO:
                                  (SEQ ID NO: 240)
                        239)
______________________________________
 *These sequences are given relative to SEQ ID NO: 178.
</pre>
    
    <div class="description-paragraph style-scope patent-text">The amino acid sequence of GE57 is presented as SEQ ID NO:241.</div>
    <div class="description-paragraph style-scope patent-text">In the GE3-5' primer (GE-3F, SEQ ID NO:46) a silent point mutation was introduced to modify a natural NcoI restriction site. Using the above-described primers, PCR amplification products were generated. The amplification products were gel purified, digested with Ncol and BamHI, and gel purified again. The purified NcoI/BamHI GE3, GE9, GE15, GE17, GE4, GE1-N and GE57 fragments were independently ligated into dephosphorylated, NcoI/BamIH cut pGEX-HisB vectors. The purified NcoI/BamHI EXP3 fragment was ligated into dephosphorylated, NcoI/BamHI cut pGEX-MOV vector.</div>
    <div class="description-paragraph style-scope patent-text">Each ligation mixture was transformed into E. coli W3110 strain and ampicillin resistant colonies were selected. The ampicillin resistant colonies were resuspended in a Tris/EDTA buffer and analyzed by PCR, using primers GLI F (SEQ ID NO:235) and GLI R (SEQ ID NO:236) to confirm the presence of insert sequences. Eight candidate clones were designated GE3-2, GE9-2, GE15-1, GE17-2, GE4-8, EXP3-7, GE1-N and GE57, respectively.</div>
    <div class="description-paragraph style-scope patent-text">B. Expression of the GE3-2, GE9-2, GE15-1, GE17-2, GE4-8, EXP3-7, GE1-N and GE57 Fusion Proteins.</div>
    <div class="description-paragraph style-scope patent-text">Colonies of ampicillin resistant bacteria carrying GE3-2, GE9-2, GE15-1, and GE17-2, GE4-8, EXP3-7, GE1-N and GE57 containing-vectors were individually inoculated into LB medium containing ampicillin. The cultures were grown to OD of 0.8 to 0.9 at which time IPTG (isopropylthio-beta-galactoside; Gibco-BRL) was added to a final concentration of 0.3 to 1 mM, for the induction of protein expression. Incubation in the presence of IPTG was continued for 3 to 4 hours.</div>
    <div class="description-paragraph style-scope patent-text">Bacterial cells were harvested by centrifugation and resuspended in SDS sample buffer (0.0625M Tris, pH 6.8, 10% glycerol, 5% mercaptoethanol, 2.3% SDS). The resuspended pellet was boiled for 5 min. and then cleared of insoluble cellular debris by centrifugation. The supernatants obtained from IPTG-induced cultures of GE3-2, GE9-2, GE15-1, GE17-2, GE4-8, EXP3-7, GE1-N and GE57 were analyzed by SDS-polyacrylamide gel electrophoresis (PAGE) together with uninduced lysates. The proteins from these gels were then transferred to nitrocellulose filters (i.e., by Western blotting).</div>
    <div class="description-paragraph style-scope patent-text">The filters were first incubated with rabbit polyclonal antibody or mouse monoclonal antibody (RM001 from Sierra Biosource, Calif.) directed to GST protein to detect the expression of appropriate size GST-fusion protein expression. Expected protein sizes of above clones are 40, 38, 39, 32, 42, 64, 42 and 33 KDa, respectively. Immunoreactivity of RM001 with bands at the appropriate molecular weight for the fusion proteins demonstrated the successful expression of the fusion proteins of above clones by the bacterial cells. Expression of the clone proteins were also monitored by the appearance of over-expressed proteins of appropriate sizes upon IPTG induction on the Coomassie brilliant blue stained gel.</div>
    <div class="description-paragraph style-scope patent-text">C. Western Blot Analysis of HGV Proteins.</div>
    <div class="description-paragraph style-scope patent-text">Once the expression of the HGV clone protein was confirmed by Western blot analysis with anti-GST antibody a second set of filters, prepared as above, were then exposed to several HGV(+) and HGV(-) human sera. Human sera used for Western blot analyses of whole cell lysates were pre-absorbed with the lambda-gt11-nitrocellulose filters. Lambda-gt11-nitrocellulose filters were prepared as follows. Briefly, an overnight culture of KM392 culture was prepared in LB. The culture was diluted 10 fold in fresh LB containing 0.2% maltose and incubated for 1 hour at 37° C. with shaking.</div>
    <div class="description-paragraph style-scope patent-text">After 1 hour the culture was mixed with an equal volume of MgCa solution (0.01M MgCl<sub class="style-scope patent-text">2</sub> and 0.01M CaCl<sub class="style-scope patent-text">2</sub>). To this mixture lambda gt11 was added to a titer of 2×10<sup class="style-scope patent-text">4</sup> PFU/ml and incubated for 30 min without shaking. After 30 minutes (per each ml of this phage/E.coli mixture) 15 ml of molten (55° C.) LB top agar (LB with 0.8% agar) was added: 8 ml of this mixture was spread onto each 15 cm LB agar plate. After the top agar solidified the plate was incubated at 37° C. for 3-5 hr.</div>
    <div class="description-paragraph style-scope patent-text">After plaques developed, a nitrocellulose filter was placed on the plate and the plate further incubated at 37° C. overnight. The nitrocellulose filter was removed and washed thoroughly with TBS (50 mM Tris-HCl, pH 7.5, 150 mM NaCl) plus 0.05 % "<figure-callout id="20" label="TWEEN" filenames="US5874563-drawings-page-4.png,US5874563-drawings-page-5.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">TWEEN</a></figure-callout> 20." The washed filter was then blocked with 1% gelatin in TBS overnight. The filter was washed three times (5 minutes each wash) with TBS.</div>
    <div class="description-paragraph style-scope patent-text">For the pre-absorption of human sera each serum was diluted 100 fold in blocking solution (described in Example 10). Ten mls. of diluted serum was then incubated overnight with two lambda gt11 filters prepared as above. Lambda gt11 filters were removed and the pre-absorbed serum used for Western blot analysis.</div>
    <div class="description-paragraph style-scope patent-text">Western blot analyses demonstrated that clones GE3-2, GE9-2, GE15-1, GE17-2, GE4-8, EXP3-7, GE1-N and GE57 showed specific immunoreactivity toward HGV(+) sera. The GE-4-8 protein was immunoreactive with J21689 serum. J21689 is HGV (+) serum as determined by HGV PCR (Example 4) and HCV (+) as determined by HCV PCR and serological analyses. The EXP3-7 protein was immunoreactive with JC and T55806. JC is the HGV-positive serum identified in Example 4F that was rejected by the blood bank for being high ALT. A second JC sample, taken one year after the initial serum sample, was also positive for HGV by PCR analysis. T55806 is also the HGV-positive serum identified in Example 4F that was rejected by the blood bank for being High ALT. This serum is co-positive with HCV.</div>
    <div class="description-paragraph style-scope patent-text">Further, GE15-1 and GE-17 showed weak but specific immunoreactivity toward PNF 2161 and T55806. GE1-N was immunoreactive with PNF2161, JC, T55806, T56633, T27034 and R0001. T56633, T27034 and R0001 are HGV (+) sera identified in Example 4F. GE57 was immunoreactive with E57963 and R0001. E57963 is HGV and HCV co-positive serum. GE3-2 and GE9-2 were also immunoreactive with HGV sera specifically. However, none of the eight antigens were immunoreactive with HGV negative sera T43608 and R05072.</div>
    <div class="description-paragraph style-scope patent-text">The GE3-2 and GE9-2 fusion proteins were purified from bacterial cell lysates essentially as in Example 7 using dual chromatographic methods employing glutathione-conjugated beads (Smith, D.B., et al.) and immobilized metal ion beads (Hochuli; Porath). The purified proteins were subjected to Western blot analysis as follows.</div>
    <div class="description-paragraph style-scope patent-text">Various amounts of the purified HGV proteins (e.g., GE3-2 and GE9-2 proteins) were loaded on 12% acrylamide gels. Following PAGE, proteins were transferred from the gels to nitrocellulose membranes, using standard procedures. Individual membranes were incubated with one of a number of human or mouse sera. Excess sera were removed by washing the membranes.</div>
    <div class="description-paragraph style-scope patent-text">These membranes were incubated with alkaline phosphatase-conjugated goat anti-human antibody (Promega) or alkaline phosphatase-conjugated goat anti-mouse antibodies (Sigma), depending on the serum being used for screening. The membranes were washed again, to remove excess goat anti-human IgG antibody, and exposed to NBT/BCIP. Photographs of exemplary stained membranes having the GE3 fusion protein are shown in FIGS. 7A to 7D.</div>
    <div class="description-paragraph style-scope patent-text">The Figures show the results of Western blot analysis of the purified GE3-2 protein using the following sera: N-(ABCDE) human (JC) serum (FIG. 7A), N-(ABDE) human (PNF 2161) serum (FIG. 7B), a super normal (SN2) serum (FIG. 7C), and mouse monoclonal antibody (RM001) directed against GST-Sj26 protein (FIG. 7D).</div>
    <div class="description-paragraph style-scope patent-text">In each of the figures, <figure-callout id="1" label="lane" filenames="US5874563-drawings-page-10.png,US5874563-drawings-page-14.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">lane</a></figure-callout> 1 contains pre-stained molecular weight standards(Bio-Rad), and lanes 2-5 contain, respectively, the following amounts of the GE3-2 fusion protein: 4 μg, 2 μg, 1 μg, and 0.5 μg. Numbers represent loading amounts in micrograms per 0.6 centimeter of gel (well size). Dilutions of the human JC, PNF 2161 and <figure-callout id="2" label="Super Normal" filenames="US5874563-drawings-page-10.png,US5874563-drawings-page-11.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">Super Normal</a></figure-callout> 2 sera were 1:100. The anti-sj26 dilution was 1:1000. The band seen at about 97K in the JC blot is reactivity against a minor contaminant in the GE3.2 fusion protein preparation. Protein marker sizes are 142.9, 97.2, 50, 35.1, 29.7 and 21.9 KD.</div>
    <div class="description-paragraph style-scope patent-text">As shown in FIGS. 7A to 7D, GE3-2 showed specific immunoreactivity with JC serum. GE3-2 reacted weakly with PNF 2161 serum and would be scored as an indeterminant or negative.</div>
    <div class="description-paragraph style-scope patent-text">In parallel experiments, GE9-2 showed weak but specific immunoreactivity toward PNF 2161 serum.</div>
    <heading class="style-scope patent-text">Example 12</heading>
    <heading class="style-scope patent-text">Construction of Exemplary Epitope Libraries</heading>
    <div class="description-paragraph style-scope patent-text">A. The Y5 Liberty.</div>
    <div class="description-paragraph style-scope patent-text">Polymerase Chain Reactions were employed to amplify 3 overlapping DNA fragments from PNF 2161 SISPA-amplified cDNA. The PNF 2161 SISPA-amplified CDNA was prepared using the JML-A/B linkers (SEQ ID NO:54 and SEQ ID NO:55). One microliter of this material was re-amplified for 30 cycles (1 minute at 94° C., 1.5 minutes at 55° C. and 2 minutes at 72° C.) using 1 μM of the JML-A primers. The total reaction volume was 100 μl. The products from 3 of these amplifications were combined and separated from excess PCR primers by a single pass through a "WIZARD PCR COLUMN" (Promega) following the manufacturer's instructions. The "WIZARD PCR COLUMN" is a silica based resin that binds DNA in high ionic strength buffers and will release DNA in low ionic strength buffers. The amplified DNA was eluted from the column with 100 μl distilled H20.</div>
    <div class="description-paragraph style-scope patent-text">The eluted DNA was fractionated on a 1.5% Agarose TBE gel (Maniatis, et al.) and visualized with UV light following ethidium bromide staining. A strong smear of DNA fragments between 150 and 1000 bp was observed. One microliter of the re-amplified cDNA was used as for template in PCR reactions with each primer pair presented in Table 15.</div>
    <div class="description-paragraph style-scope patent-text">
      </div> <pre xml:space="preserve" listing-type="tabular" class="style-scope patent-text">              TABLE 15
______________________________________
Primers   SEQ ID NO:  Size of Amplified Fragment
______________________________________
470ep-F1  SEQ ID NO: 56
                      810
470ep-R1  SEQ ID NO: 57
470ep-F2  SEQ ID NO: 58
                      750
470ep-R3  SEQ ID NO: 59
470ep-F4  SEQ ID NO: 60
                      669
470ep-R4  SEQ ID NO: 61
______________________________________
</pre>
    
    <div class="description-paragraph style-scope patent-text">The primers were designed to result in the amplification of HGV specific DNA fragments of the sizes indicated in Table 15. In the amplification reactions, the primer pairs were used at a concentration of 1 μM. Amplifications were for 30 cycles of 1 minute at 94, 1.5 minutes at 54° C. and 3 minutes at 72° C. in a total reaction volume of 100 μl. Each of the three different primer pair PCR reactions resulted in the specific amplification of products having the expected sizes. For each primer pair reaction, amplification products from 3 independent PCR reactions were combined and purified using a "WIZARD PCR COLUMN" as described above. The purified products were eluted in 50 μl dH20.</div>
    <div class="description-paragraph style-scope patent-text">Samples from each purified product (14 μl, containing approximately 1-2 μg of each primer-pair amplified DNA fragment) were combined. The combined sample of all three different amplified fragments was added to 5 μl of 10× DNAse Digestion buffer (500 mM Tris PH 7.5, 100 mM MnCl<sub class="style-scope patent-text">2</sub>) and 2 μl of dH20. From this digestion mixture, a 10 μl sample was removed and placed in a tube containing 5 μl of Stop solution (100 mM EDTA, pH 8.0). This sample was the 0 "minutes of digestion" time point. The rest of the digestion reaction was placed at 25° C. To the <figure-callout id="1" label="digestion mixture" filenames="US5874563-drawings-page-10.png,US5874563-drawings-page-14.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">digestion mixture</a></figure-callout> 1 μl of 1/25 diluted RNase-free DNAse I (Stratagene) was added. At <figure-callout id="10" label="various time points" filenames="US5874563-drawings-page-12.png,US5874563-drawings-page-19.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">various time points</a></figure-callout> 10 μl aliquots were withdrawn and mixed with 5 μl of Stop solution. The DNAse I digested DNA products were analyzed on a 1.5% Agarose TBE gel.</div>
    <div class="description-paragraph style-scope patent-text">The results of several digestion experiments showed that 40 minutes of digestion provided a good distribution of DNA fragments in the size range of 100-300 bp. A DNAse I digestion was then repeated with the entire digestion being left for 40 minutes at room temperature. The digestion was stopped by the addition of 18 μl of Stop Buffer and the digested DNA products were purified using a "WIZARD PCR COLUMN." The "WIZARD-PCR COLUMN" was eluted with 50 μl of dH20 and the eluted DNA added to the following reaction mixture: 7 μl of Restriction Enzyme Buffer C (Promega, 10 mM MgCl<sub class="style-scope patent-text">2</sub>, 1 mM DTT, 50 mM NaCl, 10 mM Tris, pH 7.9, 1× concentration); 11 μl of 1.25 mM dNTPs; and 2 μl T4 DNA Polymerase (Boehringer-Mannhiem). This reaction mixture was held at 37° C. for 30 minutes, at which <figure-callout id="70" label="point" filenames="US5874563-drawings-page-7.png,US5874563-drawings-page-8.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">point</a></figure-callout> 70 μl of pH 8.0 phenol/CHCl<sub class="style-scope patent-text">3</sub> was added and mixed. The phenol/CHCl<sub class="style-scope patent-text">3</sub> was removed and extracted once to yield a total aqueous volume of 150 μl containing the DNA sample. The DNA was ethanol precipitated using 2 volumes of absolute ethanol and 0.5 volume of 7.5M NH<sub class="style-scope patent-text">4</sub> -acetate. The DNA was pelleted by centrifugation for 15 minutes at 14,000 rpm in an "EPPENDORF MICROFUGE", dried for 5 minutes at 42° C. and resuspended in 25 μl of dH20.</div>
    <div class="description-paragraph style-scope patent-text">The DNA was ligated to 5' phosphorylated SISPA linkers KL1 (SEQ ID NO:62) and KL2 (SEQ ID NO:63). Several different concentrations of SISPA linkers and DNA was tested. The highest level of ligation (assessed as described below) occurred under the following ligation reaction conditions: 6 μl of DNA, 2 μl of 5.0×10-12M KL1/KL2 linkers, 1 μl of 10× ligase buffer (New England Biolabs), and 1 μl of 400 Units/μl T4 DNA Ligase (New England Biolabs) in a total reaction volume of 10 μl. Ligations were carried out overnight at 16° C.</div>
    <div class="description-paragraph style-scope patent-text">Two reactions were run in parallel as follows. A 2 μl sample of the ligated material was amplified using the KL1 SISPA primer in a total reaction volume of 100 μl (25 cycles of 1 minute at 94° C., 1.5 minutes at 55° C. and 2 minutes at 72° C.). The degree of ligation was assessed by separating 1/5 of the PCR reaction amplified products by electrophoresis using a 1.5% agarose TBE gel. The gel was stained with ethidium bromide and the bands visualized with UV light.</div>
    <div class="description-paragraph style-scope patent-text">The amplification products from the duplicate reactions were purified using "WIZARD PCR COLUMNS" and the purified DNA eluted in 50 μl of dH20. A twenty-five microliter aliquot of the PCR KL1/KL2 amplified DNA was digested with 36 Units of EcoRI (Promega) in a total volume of 30 μl. The reaction was carried out overnight. at 37° C. The Digested DNA was purified using a "SEPHADEX G25" spin column.</div>
    <div class="description-paragraph style-scope patent-text">The EcoRI digested DNA was ligated in overnight reactions to λgt11 arms that were pre-digested with EcoRI and treated with calf intestinal alkaline phosphatase (Stratagene, La Jolla, Calif.). The ligation mixture was packaged using a "GIGAPACK GOLD PACKAGING EXTRACT" (Stratagene) following manufacturer's instructions. Titration of the amount of recombinant phage obtained was performed by plating a 1/10 dilution of the packaged phage on a lawn of KM-392, where the plate contained 20 μl of a 100 mg/ml solution of x-gal (5-Bromo-4-chloro-3-indolyl-β-D-galactoside; Sigma) and 20 μl of a 0.1M solution of IPTG (Isopropyl-1-thio-β-D-galactoside; Sigma). A titer was obtained of 1.2×10<sup class="style-scope patent-text">6</sup> phage/ml containing over 75% recombinant phage.</div>
    <div class="description-paragraph style-scope patent-text">The percentage of recombinant plaques was confirmed by PCR analysis of 8 randomly picked plaques using primers 11F (SEQ ID NO:25) and 11R (SEQ ID NO:13). This packaged library containing the DNA fragments derived from the digestion of the amplified DNAs F1/R1, F2/R3, and F4/R4 amplified DNAs and was designated library Y5.</div>
    <div class="description-paragraph style-scope patent-text">B. The ENV Liberty.</div>
    <div class="description-paragraph style-scope patent-text">An expression library, designated the ENV library, was generated as follows. One microliter of PNF 2161 SISPA amplified DNA was used as the template in polymerase chain amplification reactions utilizing the following primer pairs: GEP-F15 (SEQ ID NO:128) and GEP-R15 (SEQ ID NO:129), which generate a 525 nucleotide HGV fragment; and GEP-F17 (SEQ ID NO:130) and GEP-R16 (SEQ ID NO:131), which generate a 765 nucleotide HGV fragment.</div>
    <div class="description-paragraph style-scope patent-text">PCR amplification was for 35 cycles of 94° C. for 1 min, 52° C. for 1.5 minutes, and 72° C. for 3 minutes. The amplified products were purified and digested with DNAse I. Ligation of KL1 and KL2 linkers to cDNA, amplification of DNA fragments and construction of libraries in lambda gt11 were performed essentially as described in Example 12A. The recombinant frequency of the library was greater than 70%. Analysis of the inserts by polymerase chain reaction using primers derived from the flanking regions of lambda gt11 confirmed the recombinant frequency and indicated that the insert size range was 150-500 nucleotides.</div>
    <div class="description-paragraph style-scope patent-text">C. The NS3 Library.</div>
    <div class="description-paragraph style-scope patent-text">An expression library designated NS3 was constructed as follows. A first fragment was amplified by polymerase chain reaction using the primers 470ep-F9 (SEQ ID NO:132) and 470ep-R9 (SEQ ID NO:133) and, as template, PNF 2161 SISPA amplified nucleic acids. The predicted product of this amplification reaction was 777 base pairs. The amplified fragment was gel purified by separation on a TAE gel. The fragment was further purified using "GENECLEAN" (<figure-callout id="101" label="Bio" filenames="US5874563-drawings-page-9.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">Bio</a></figure-callout> 101, La Jolla, Calif.).</div>
    <div class="description-paragraph style-scope patent-text">Fragment F9/R9 was also amplified using the extension clone GE3L-11 (SEQ ID NO:41) as source material. Approximately 25 ng of GE3L-11 was used as template with the F9 and R9 primers in amplification reactions.</div>
    <div class="description-paragraph style-scope patent-text">Both of the F9/R9 amplifications were for 30 cycles of 94° C., for 1 minute, 52° C. for two minutes, and 72° C. for 3 minutes, using "TAQ START" (Clonetech, Palo Alto, Calif.). The amplification products from both reactions were combined. The products were digested with DNAse I (10 μl GE3L product and 25 ul of PNF SISPA product). The GE3L-based amplification product represented the majority of the amplification product starting material. Ligation of KL1 and KL2 linkers to cDNA, amplification of DNA fragments and construction of libraries in lambda gt11 were performed essentially as described in Example 12A.</div>
    <div class="description-paragraph style-scope patent-text">The titer obtained was 2.5×10<sup class="style-scope patent-text">6</sup> phage/ml and the percent recombinant phage was determined to be greater than 99%. Polymerase chain reaction analysis of the insert sizes confirmed the recombinant frequency and indicated an insert size range of 150 to 550 nucleotides.</div>
    <div class="description-paragraph style-scope patent-text">In addition, a second fragment was also amplified using the GEP-F10/GEP-R10 primers (SEQ ID NO:135 and SEQ ID NO:136, respectively). One microliter of PNF 2161 SISPA amplified nucleic acids was used as template. The predicted fragment size of 570 nucleotides was obtained. The resulting amplification products were manipulated as just described for the F9/R9 amplifications. The titer obtained for this fragment when inserted in lambda gt11 was 1.47×10<sup class="style-scope patent-text">6</sup> phage/ml, with a recombinant frequency of 90%.</div>
    <div class="description-paragraph style-scope patent-text">D. The NS2 Library.</div>
    <div class="description-paragraph style-scope patent-text">The NS2 epitope library was constructed using the methodologies described in Example 12A. Four DNA fragments containing all or part of the HGV proteins NS2, NS3, and NS5b were amplified from 1 ul of PNF 2161 SISPA DNA (prepared essentially as described in Example 12A). The library was generated using the primers given in Table 16 and SISPA amplified PNF 2162 DNA as template.</div>
    <div class="description-paragraph style-scope patent-text">
      </div> <pre xml:space="preserve" listing-type="tabular" class="style-scope patent-text">              TABLE 16
______________________________________
Fragments         nt
______________________________________
9E3-REV (SEQ ID NO: 264)
                  592     aa 358 (of 389) of
E394-R (SEQ ID NO: 265)   E2 to aa 166 of NS-2
GEP-F12 (SEQ ID NO: 266)
                  663     aa 144 (of 313) of
GEP-R12 (SEQ ID NO: 267)  NS-2 to aa 51 of
                          NS-3
GEP-F14 (SEQ ID NO: 268)
                  715     aa 357 - 594 of NS-3
GEP-R13 (SEQ ID NO: 269)
470epF8 (SEQ ID NO: 270)
                  648     aa 716 - 847 of NS-5
GEP-R14 (SEQ ID NO: 271)  (716 to end)
______________________________________
</pre>
    
    <div class="description-paragraph style-scope patent-text">All amplifications were for 35 cycles of 94° C./1 minute, 48° C./2 minutes, and 73° C./3 minutes. All amplifications yielded at least a fragment of the expected size. The amplified products were mixed and in an approximately 1:1:1:1 ratio and partially digested with DNase I. As above, the digestion products were ligated to KL1 SISPA linkers, amplified and EcoRI digested. The digested fragments were ligated into lambda gt11. The ligation reactions were packaged.</div>
    <div class="description-paragraph style-scope patent-text">The packaged ligation products were plated. The resulting library was determined to contain ˜70% recombinant phage with an observed insert size of 150 to 500 nucleotides.</div>
    <div class="description-paragraph style-scope patent-text">E. The VNS5a Library.</div>
    <div class="description-paragraph style-scope patent-text">Primers 470EXT4-2189R (SEQ ID NO:119) and 470EXT4-29F (SEQ ID NO:120) were used to isolate a 2.1 kb DNA fragment that contains the entire coding sequences for the HGV proteins NS4b and NS5a, as well as the 3' end of NS4a and the 5' end of NS5b. PCR amplifications using these primers were performed as described in Example 4G. Successful amplification was observed with multiple HGV-infected sera including the following: T56633 was from a blood donor whose donation was rejected due to an ALT value above the cutoff; samples E21-A and E20 were derived from Egyptian individuals suffering from hepatitis; and sample AH0591 is derived from an Australian individual who developed fulminant hepatitis.</div>
    <div class="description-paragraph style-scope patent-text">The amplified products of E21-A and E20 were cloned into the T overhang site of the vector T/A (obtained from InVitrogen, San Diego, Calif.) essentially as described in Example 6. The 2.1 kb HGV inserts from these 2 plasmids were then isolated by the digestion of approximately 20 ug of plasmid DNA with approximately 150 units of the restriction enzyme EcoRI. After incubation overnight at 37° C., the products of the digestion were separated by TAE agarose gel electrophoresis. The products were excised from the section of the agarose gel containing the fragment of interest. The agarose was melted and extraction of the liberated DNA was carried out using the "GENECLEAN II" kit according to the manufacturers instructions (<figure-callout id="101" label="Bio" filenames="US5874563-drawings-page-9.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">Bio</a></figure-callout> 101, La Jolla, Calif.).</div>
    <div class="description-paragraph style-scope patent-text">The purified 2.1 kb fragments derived from the E21-A and E20 samples, as well as the DNA fragments obtained from PCR amplification of samples T56633 and AH0591, were digested separately with DNAse I as described in Example 12A. For all 4 samples digestion conditions were determined that resulted in the isolation of fragments of between 100 to 1000 nts in size. After purification and trimming (Example 12A) the fragments derived from each of the 4 HGV infected samples were ligated separately to different sets of SISPA linkers. After ligation the DNAs were SISPA amplified.</div>
    <div class="description-paragraph style-scope patent-text">The amplified DNAs were separately digested overnight at 37° C. with approximately 100 units of EcoRI. The digested DNAs were then purified by spin column chromatography using G25 resin (5'3' Inc, Boulder, Colo.). Digested DNA from the samples T56633, AH0591, and E21-A were combined at a ratio of 1:1:1 and the mixture of DNAs was ligated into the EcoRI site of λgt11 as described in Example 12A. After packaging using the "GIGAPACK III XL" extract (Stratagene, LaJolla, Calif.), the resulting library was plated in the presence of IPTG and XGAL and determined to have a titer of approximately 1.0×10<sup class="style-scope patent-text">6</sup> phage/ml and a recombinant frequency of approximately 70%.</div>
    <heading class="style-scope patent-text">Example 13</heading>
    <heading class="style-scope patent-text">Immunoscreening of the Epitope Libraries</heading>
    <div class="description-paragraph style-scope patent-text">A. Isolation of Immunoreactive Y5 Clones.</div>
    <div class="description-paragraph style-scope patent-text">Two HGV positive sera, PNF2161 and JC, were used for immunoscreening of the Y5 library, essentially as described in Example 2. The Y5 phage library was plated onto 20 plates at approximately 15,000 phage per plate. The plates were incubated for approximately 5 hours and were overlaid with nitrocellulose filters (Schleicher and Schuell) overnight. The filters were blocked by incubation in AIB (1% gelatin plus 0.02% Na azide) for approximately 6 hours. The blocked filters were washed once with TBS.</div>
    <div class="description-paragraph style-scope patent-text">Ten Y5 library filters were incubated overnight, with agitation, with PNF2161 serum and ten filters with JC serum. Both sera were diluted 1:10 in AIB. In order to reduce non-specific antibody binding, the diluted sera had been pre-treated by incubation overnight with nitrocellulose filters to which wild type λgt11 were adsorbed.</div>
    <div class="description-paragraph style-scope patent-text">The filters were removed from the sera, washed 3 times with TBS and incubated with goat anti-human alkaline phosphatase-conjugated secondary antibody (Promega; diluted 1/7500 in AIB) for one hour. The filters were washed 4 times with TBS. Bound secondary antibody was detected by incubation of the filters in AP buffer (100 mM NaCl, 5 mM MgCl<sub class="style-scope patent-text">2</sub>, 100 mM Tris pH 9.5) containing NBT and BCIP.</div>
    <div class="description-paragraph style-scope patent-text">Plaques that tested positive in the initial screen were picked and eluted in 500 μl of PDB (100 mM NaCl, 8.1 mM MgSO<sub class="style-scope patent-text">4</sub>, 50 mM Tris pH 7.5, 0.02% Gelatin). The immunoreactive phage were purified by replating the eluted phage at a total density of 100-500 plaques per 100 mm plate. The plates were re-immunoscreened with the appropriate HGV-pdsitive sera, essentially as described above. After color development several isolated, positive plaques were picked and put into 500 μl of PDB. After 1 hour of incubation, 2 μl of the re-purified phage PDB solution was used as template in a PCR reaction containing the 11F (SEQ ID NO:25) and 11R (SEQ ID NO:13) PCR primers. These primers are homologous to sequences located 70 nucleotides (nt) 5' and 90 nt 3' of the EcoRI site of λgt11. The PCR reactions were amplified through 30 cycles of 94° C. for 1 minute, 55° C. for 1.5 minutes and 72° C. for 2 minutes.</div>
    <div class="description-paragraph style-scope patent-text">The PCR amplification reactions were size-fractionated on agarose gels. PCR amplification of purified plaques resulted in a single band for each single-plaque amplification reaction, where the amplified fragment contained the DNA insert plus approximately 140 bp of 5' and 3' phage flanking sequences. The amplified products, from PCR reactions resulting in single bands, were purified using a "S-300 HR" spin column (Pharmacia), following manufacturers instructions. The DNA was quantitated and DNA sequenced employing an Applied Biosystems automated sequencer 373A and appropriate protocols.</div>
    <div class="description-paragraph style-scope patent-text">The above-described screening of the Y5 library with JC sera resulted in the purification and DNA sequencing of the positive-strand clones presented in Table 17. Positive-strand clones correspond to the <figure-callout id="3'" label="5'to" filenames="US5874563-drawings-page-3.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">5'to</a></figure-callout> 3' translation of the HGV sequence presented in SEQ ID NO:14--the polyprotein reading frame.</div>
    <div class="description-paragraph style-scope patent-text">
      </div> <pre xml:space="preserve" listing-type="tabular" class="style-scope patent-text">              TABLE 17
______________________________________
                Insert   Insert Size
                                Nucleic
                                       Encoded
       Screening
                Size (base
                         (amino Acid SEQ
                                       Protein SEQ
Clone  Sera     pairs)   acids) ID NO. ID NO.
______________________________________
Y5-10  JC       210      62     64     65
Y5-12  JC       333      94     66     67
Y5-26  JC       303      93     68     69
Y5-5   JC       153      36     70     71
Y5-3   JC       162      44     72     73
Y5-27  JC       288      86     74     75
Y5-25  JC       165      36     76     77
Y5-20  JC       165       19.sup.1
                                78     79
Y5-16  JC       234      56     80     81
______________________________________
 .sup.1 the clone contained a double insert, nt 69 to 126 of the clone
 insert correspond to HGV sequences.
</pre>
    
    <div class="description-paragraph style-scope patent-text">These clones delineated 2 immunogenic regions within the putative NS5 protein of HGV. These two region, relative to the sequence presented as SEQ ID NO:14 are positions 6636 to 6821 and 7278 to 7385.</div>
    <div class="description-paragraph style-scope patent-text">Further, screening of the Y5 library with PNF 2161 sera resulted in the purification and DNA sequencing of the following negative-strand clones presented in Table 18. Negative-strand clones correspond to the 5' to 3' translation of the sequence complementary to the HGV sequence presented in SEQ ID NO:14.</div>
    <div class="description-paragraph style-scope patent-text">
      </div> <pre xml:space="preserve" listing-type="tabular" class="style-scope patent-text">              TABLE 18
______________________________________
                Insert   Insert Size
                                Nucleic
                                       Encoded
       Screening
                Size (Base
                         (amino Acid SEQ
                                       Protein
Clone  Sera     pairs)   acids) ID NO. SEQ ID NO.
______________________________________
Y5-50  PNF 2161 349      104    82     63
Y5-52  PNF 2161 119      20.sup.1
                                84     85
Y5-53  PNF 2161 250      33.sup.2
                                86     87
Y5-55  PNF 2161 143      20.sup.3
                                68     89
Y5-56  PNF 2161 366      110    90     91
Y5-57  PNF 2161 231      65     92     93
Y5-60  PNF 2161 151      38     94     95
Y5-63  PNF 2161  125.sup.4
                         25     96     97
______________________________________
 .sup.1 the clone contained a double insert, nt 46 to 105 of the clone
 insert correspond to HGV sequences.
 .sup.2 the clone contained a double insert, nt 19 to 118 of the clone
 insert correspond to HGV sequences.
 .sup.3 the clone contained a double insert, nt 70 to 126 of the clone
 insert correspond to HGV sequences.
 .sup.4 the insert contains an extra, nonHGV sequence between <figure-callout id="35" label="nucleotides" filenames="US5874563-drawings-page-15.png,US5874563-drawings-page-16.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">nucleotides</a></figure-callout>
 19 and 35.
</pre>
    
    <div class="description-paragraph style-scope patent-text">All of these sequences contain portions of the original HGV clone 470-20-1 isolated using the PNF 2161 serum.</div>
    <div class="description-paragraph style-scope patent-text">Additional epitope clones from the Y5 library were isolated as follows. The Y5 library was screened with the HGV infected sera J21689 and T56633 using the methods described in Example 13. Greater than 400 positive plaques were obtained, indicating the presence of a strongly immunogenic sequence recognized by both of these HGV infected sera. Ten of these positive plaques were purified and DNA sequenced. The results obtained from the DNA sequencing are delineated in Table 19.</div>
    <div class="description-paragraph style-scope patent-text">
      </div> <pre xml:space="preserve" listing-type="tabular" class="style-scope patent-text">              TABLE 19
______________________________________
CLONE      HGV VAR   SERA      START* STOP
______________________________________
Y5-114-1A  PNF       J21689    6636   6827
Y5-114-2B  PNF       J21689    6678   6935
Y5-121-19A PNF       T56633    6678   7063
Y5-121-11A PNF       T56633    6636   6917
Y5-121-12A PNF       T56633    6636   6959
Y5-121-15A PNF       T56633    6636   6917
Y5-121-16A PNF       T56633    6636   6989
Y5-121-17A PNF       T56633    6636   7082
Y5-121-20A PNF       T56633    6636   6929
Y5-121-18A PNF       T56633    6636   6896
______________________________________
 *start/stop locations are given relative to SEQ ID NO: 14.
</pre>
    
    <div class="description-paragraph style-scope patent-text">Comparison of these sequences with those obtained previously from screening this library indicated that these clones all contained the same epitope(s) that are contained in the previously isolated epitope clone Y5-10. Two of the clones, Y5-114-2B and Y5-121-19A are distinguished by the fact that their 5' ends are located 14 amino acids closer to the carboxy terminal of NS5a than the previously observed start of clones Y5-10, Y5-12, and Y5-26. None of the above clones has its 3' end interior to that observed in the clone Y5-10. Thus a minimal sequence of this epitope is contained within amino acid sequence (SEQ ID NO:272).</div>
    <div class="description-paragraph style-scope patent-text">B. Antigenic Clones from the ENV Library.</div>
    <div class="description-paragraph style-scope patent-text">The ENV library was screened with HGV serum J21094. This serum (J21094) was identified as HCV positive based on the first generation (c-100) HCV test. Subsequent testing of the initial J21094 serum sample, and of subsequently obtained J21094 samples, by PCR and with other HCV antigens confirmed that the source individual for the serum was HCV infected. Evidence for the presence of HGV nucleic acid was obtained via PCR analysis using the 470-20-1 and NS5 primer sets.</div>
    <div class="description-paragraph style-scope patent-text">A number of phage clones were identified as immunoreactive with J21094 serum. The phage were plaque purified and sequenced. Seven of the clones (Q7-12-1, Q7-16-2-2, Q7-15-2, Q7-17-2-1, Q7-19-1, and Q7-19-2-1) contained the same insert. The nucleotide sequence for Q7-12-1 is presented as SEQ ID NO:143 (polypeptide sequence, SEQ ID NO:144).</div>
    <div class="description-paragraph style-scope patent-text">One additional clone, Q7-16-1, obtained by the method just described, has the same 5' end as Q7-12-1, but is 26 amino acids shorter at the 3' end.</div>
    <div class="description-paragraph style-scope patent-text">C. Antigenic Clones from the NS3 Library.</div>
    <div class="description-paragraph style-scope patent-text">A one to one mixture of the F9/R9 phage and F10/R10 phage were screened using the following sera: PNF 2161, J21689 and E57963. Both J21689 and E57963 are sera that test co-positive for HCV and HGV by PCR (using multiple primers). Each immunoscreening was of 10 plates or approximately 150,000 phage. Some of the immunopositive clones identified in these screens are as follows.</div>
    <div class="description-paragraph style-scope patent-text">Clone Y12-10-3 (polynucleotide sequence, SEQ ID NO:145; polypeptide sequence, SEQ ID NO:146) was identified by its immunoreactivity with J21689 serum. The clone expresses an 88 amino acid insert from HGV NS3.</div>
    <div class="description-paragraph style-scope patent-text">Clone Y12-15-1 (polynucleotide sequence, SEQ ID NO:147; polypeptide sequence, SEQ ID NO:148) was identified by its immunoreactivity with E57963 serum. The clone expresses a 64 amino acid insert from the NS3 protein of HGV. This sequence is located approximately 70 amino acids 5' to clone Y12-10-3.</div>
    <div class="description-paragraph style-scope patent-text">D. Antigenic Clones from the NS2 Library.</div>
    <div class="description-paragraph style-scope patent-text">Multiple positive plaques were isolated by screening the NS2 library with HGV-positive serum T56633. Eleven of these plaques were subsequently purified and DNA sequenced. The locations of the inserts contained within these plaques (relative to SEQ ID NO:14) are delineated in Table 20.</div>
    <div class="description-paragraph style-scope patent-text">
      </div> <pre xml:space="preserve" listing-type="tabular" class="style-scope patent-text">              TABLE 20
______________________________________
CLONE    HGV VAR   SERA      START* STOP
______________________________________
Q9-18-5  PNF       T56633    3071   2778
Q9-18-3  PNF       T56633    2951   2745
Q9-20-4  PNF       T56633    3002   2745
Q9-18-2  PNF       T56633    2990   2745
Q9-20-8  PNF       T56633    3062   2745
Q9-20-5  PNF       T56633    2972   2787
Q9-17-1  PNF       T56633    2990   2745
Q9-19-3  PNF       T56633    2982   2745
Q9-19-1  PNF       T56633    2982   2745
Q9-19-5  PNF       T56633    2984   2745
Q9-20-2  PNF       T56633    3027   2745
______________________________________
 *in this table the locations are given with respect to SEQ ID NO: 14. The
 actual sequence of the clones are the complement of the indicated
 fragment.
</pre>
    
    <div class="description-paragraph style-scope patent-text">All of the immunoclones express portions of the same open reading frame (ORF). This reading frame is encoded by the HGV polynucleotide strand that is complementary to the sequence encoding the polyprotein. This ORF extends between nts 6322 and 6865 of the sequence complementary to SEQ ID NO:14. There is a Methionine that could serve as a site of translation initiation located at nt 6388 of the complementary strand that would allow for the production of a 159 amino acid protein.</div>
    <div class="description-paragraph style-scope patent-text">The smallest amino acid sequence common to all of the 11 sequenced clones is located between nts 6342 to 6606 (relative to the complementary strand of SEQ ID NO:14). The amino acid sequence encoded by this region of the negative strand of HGV-PNF 2161 is presented as SEQ ID NO:273.</div>
    <div class="description-paragraph style-scope patent-text">The subcloning and subsequent Western blot analysis of immunoreactive negative strand regions is described below.</div>
    <div class="description-paragraph style-scope patent-text">E. Antigenic Clones from the VNS5a Library.</div>
    <div class="description-paragraph style-scope patent-text">Approximately 1.5×10<sup class="style-scope patent-text">5</sup> phage from the VNS5a library was plated out and subsequently screened with the HGV-positive serum J29374 using the procedures described in Example 13. Immunoscreening of the VNS5a library with J29374 resulted in the isolation of multiple positive plaques. Six of these plaques were purified and subsequently DNA sequenced. The original strain of the DNA sequence obtained could be determined by which of the SISPA linker sequences was present at the 5' and 3' ends of the clones. The locations of the starts and stops of the obtained clones (relative to SEQ ID NO:14) and their source sera are summarized in Table 21.</div>
    <div class="description-paragraph style-scope patent-text">
      </div> <pre xml:space="preserve" listing-type="tabular" class="style-scope patent-text">              TABLE 21
______________________________________
          HGV Variant
Clone     Source     Sera       Start*
                                     Stop
______________________________________
Q11-14-2  AH0591     J29374     6525 6749
Q11-16-1  E21-A      J29374     6432 6935
Q11-10-2  T56633     J29374     6579 6710
Q11-18-2  T56633     J29374     6579 6758
Q11-22-1  T56633     J29374     6576 6680
Q11-9-1   T56633     J29374     6531 6851
______________________________________
</pre>
    
    <div class="description-paragraph style-scope patent-text">All of these clones contain the sequence of the clone Q11-22-1 in common (SEQ ID NO:274). This amino acid sequence is located immediately 5' to the minimal sequence of the Y5-10 epitope. Thus it defines an additional unique epitope in HGV NS5a (along with Y5-10 and Y5-5). Comparison of the observed amino acid sequence of these 3 HGV variants with the sequence of the PNF-2161 and JC isolates reveals few amino acid substitutions.</div>
    <heading class="style-scope patent-text">Example 14</heading>
    <heading class="style-scope patent-text">Further Characterization of Immunoreactive Clones</heading>
    <div class="description-paragraph style-scope patent-text">A. Subcloning.</div>
    <div class="description-paragraph style-scope patent-text">1. Y5 Clones.</div>
    <div class="description-paragraph style-scope patent-text">Clones Y5-10, Y5-16, and Y5-5 were selected for subcloning into the expression vector pGEX-HisB. PCR primers were designed which removed the extraneous linker sequences at the end of these clones. These primers also introduced (i) a NcoI site at the 5' end (relative to the coding sequence) of each insert, and (ii) a BamHI site at the 3' end of each insert. Using these primers (see Table 22), the DNA fragments were amplified from 2 μl of the plaque pure stocks.</div>
    <div class="description-paragraph style-scope patent-text">
      </div> <pre xml:space="preserve" listing-type="tabular" class="style-scope patent-text">              TABLE 22
______________________________________
Clone        Primer Set
______________________________________
Y5-10        Y5-10-F1    SEQ ID NO: 99
             Y5-10-R1    SEQ ID NO: 100
Y5-16        Y5-16F1     SEQ ID NO: 101
             470ep-R3    SEQ ID NO: 102
Y5-5         Y5-5-F1     SEQ ID NO: 103
             470ep-R3    SEQ ID NO: 102
______________________________________
</pre>
    
    <div class="description-paragraph style-scope patent-text">Amplifications were performed as follows: 30 cycles of 94° C. for 1 minute, 50° C. for 1.5 minutes, and 72° C. for 2 minutes. After amplification the resulting DNAs were purified using "WIZARD PCR," spin columns, the samples eluted in 50 μl, and digested overnight with NcoI and BamNHI. A minimum of 30 units of each enzyme was used in the restriction endonuclease digestions (NcoI, Boehringer Mannhiem; BamNHI, Promega).</div>
    <div class="description-paragraph style-scope patent-text">The digested PCR fragments were ligated overnight to expression vector pGEX-HisB that had been digested with NcoI and BamHI. Each set of ligated plasmids was independently used to transform E. coli strain W3110, using a heat shock protocol (Ausubel, et al.; Maniatis, et al.). Transformants were selected on LB plates containing 100 μg/ml ampicillin and resistant colonies were used to inoculate 2 mls of LB containing 100 μg/ml ampicillin. Cultures expressing non-recombinant sj26/his protein were also prepared.</div>
    <div class="description-paragraph style-scope patent-text">After incubation overnight at 37° C. the cultures were diluted 1/10 into 2 mls of fresh LB plus ampicillin and grown for an additional 1 hour at 37° C. IPTG was added to a final concentration of 0.2 mM and the cultures were grown for an additional 3 hours at 37° C. The bacteria were pelleted by centrifugation and the bacterial pellet was resuspended in 100 μl PBS. To the pellet, 100 μl of 2× SDS sample buffer (0.125M Tris, pH 6.8, 10% glycine, 5% β-mercaptoethanol, 2.3% SDS) was added. The resulting lysates were vortexed and heated to 100° C. for 5 minutes. Aliquots (15 μl) of each lysate were loaded onto a 12% acrylamide SDS-PAGE gel.</div>
    <div class="description-paragraph style-scope patent-text">The expressed proteins were size-fractionated by electrophoresis. The separated proteins were transferred from the gel to nitrocellulose filters using standard techniques (Harlow, et al.). An additional gel containing the expressed proteins was stained using coomasie blue protein stain.</div>
    <div class="description-paragraph style-scope patent-text">Transformants carrying plasmids Y5-10, Y5-5 and Y5-16 expressed significant amounts of correctly sized recombinant fusion proteins. The identity of the recombinant fusions were confirmed by incubating a Western blot (prepared above) with a murine monoclonal antibody that is specifically immunoreactive with sj26 (Sierra BioSource, Gilroy, Calif.).</div>
    <div class="description-paragraph style-scope patent-text">Additional confirmation that the picked colonies contained the appropriate insert was obtained as follows. A phage solution for each colony was prepared by inoculating 40 μl of TE solution with a toothpick containing a small amount of bacteria putatively expressing a recombinant clone had been inoculated. A 5 μl sample was taken from each solution and separately PCR amplified.</div>
    <div class="description-paragraph style-scope patent-text">The amplifications employed the appropriate forward primer, (e.g., Y5-10 F for a colony putatively expressing Y5-10) and a reverse primer (SEQ ID NO:104) homologous to a sequence located 3' to the cloning sites of the plasmid pGEX-HisB. The PCR amplifications were for 25 cycles as follows: 94° C. for 1 minute, 50° C. for 1.5 minutes and 72° C. for 2 minutes. All of the colonies selected for further analysis produced a correctly sized DNA band with no other obvious bands under these conditions.</div>
    <div class="description-paragraph style-scope patent-text">The immunoreactivity of the antigens expressed from the Y5-10, Y5-16, &amp; Y5-5 inserts (expressed as sj26-his fusion proteins) was determined as follows. Aliquots (15 μl) of the crude lysates prepared above were size-fractionated by SDS-PAGE using a 12% acrylamide gel. The proteins were electro-blotted ("NOVEX MINICELL MINIBLOT II," San Diego, Calif.) onto nitrocellulose filters. The filters were then individually incubated with one of the following sera: JC, PNF 2161, and super normal serum 4 (SN4) (R05072) as a negative control. In addition, one filter was incubated with anti-sj26 monoclonal antibodies (RM001; Sierra BioSource).</div>
    <div class="description-paragraph style-scope patent-text">As expected, the recombinant protein produced by the bacteria expressing the antigens encoded by the Y5-10, Y5-5, and Y5-16 inserts all reacted with JC sera. No reactivity was observed with either PNF 2161 or SN4 sera. All proteins appeared to be expressed at similar levels as determined by their reactivity to the anti-sj26 monoclonal antibody. The Y5-5 and Y5-10 encoded proteins were selected for further purification.</div>
    <div class="description-paragraph style-scope patent-text">E. coli carrying Y5-5- and Y5-10- containing pGEX-HisB vectors were cultured and expression of the fusion protein induced as described above. The cells were lysed in PBS, containing 2 mM PMSF, using a French Press at 1500 psi. The crude lysate was spun to remove cellular debris. The supernatant was loaded onto the glutathione affinity column at a high flow rate and the column was washed with 10 column volumes of PBS. The Y5-5 and Y5-10 fusion proteins were eluted with 10 mM Tris pH 8.8 containing 10 mM glutathione.</div>
    <div class="description-paragraph style-scope patent-text">Each of the fusion protein samples was diluted 1/10 with Buffer A (10 mM Tris pH 8.8, containing 8M urea) and loaded onto a nickel charged-chelating "SEPHAROSE" fast flow column. Each column was repeatedly washed with Buffer A until no further contaminants were eluted. The fusion proteins were eluted using a gradient of imidazole in buffer A. An imidazole gradient was run from 0 to 0.5M imidazole in 20 column volumes. Fractions were collected.</div>
    <div class="description-paragraph style-scope patent-text">Each set of fractions was analyzed by standard SDS-PAGE using 12% polyacrylamide gels. Pools of the Y5-5 and Y5-10 fusion protein-containing fractions were separately made.</div>
    <div class="description-paragraph style-scope patent-text">FIGS. 8A to 8D show the results of Western blot analysis of the following samples (μg/lane): <figure-callout id="1" label="lane" filenames="US5874563-drawings-page-10.png,US5874563-drawings-page-14.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">lane</a></figure-callout> 1, Y5-10 antigen 1.6 μg; <figure-callout id="2" label="lane" filenames="US5874563-drawings-page-10.png,US5874563-drawings-page-11.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">lane</a></figure-callout> 2, Y5-10 antigen 0.8 μg; <figure-callout id="3" label="lane" filenames="US5874563-drawings-page-19.png,US5874563-drawings-page-3.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">lane</a></figure-callout> 3, Y5-10 antigen 0.4 μg; and <figure-callout id="4" label="lane" filenames="US5874563-drawings-page-10.png,US5874563-drawings-page-12.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">lane</a></figure-callout> 4, Y5-10 antigen 0.2 μg. Human serum JC (FIG. 8A) and Super-Normal 2 serum (FIG. 8B) were diluted 1:100. The anti-GST mouse monoclonal antibody RM001 (FIG. 8C) was diluted 1:1000. FIG. 8D shows the Y5-10 antigen resolved by SDS-PAGE, transferred onto the nitrocellulose membrane and stained with Ponceau S protein stain (Kodak, Rochester, N.Y.; Sigma). Arrow indicates the location of Y5.10 antigen. These results demonstrate that Y5-10 is specifically immunoreactive with N-(ABCDE) human serum JC.</div>
    <div class="description-paragraph style-scope patent-text">FIGS. 9A to 9D show the results of Western blot analysis of the following samples: <figure-callout id="1" label="lane" filenames="US5874563-drawings-page-10.png,US5874563-drawings-page-14.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">lane</a></figure-callout> 1, Y5-5 antigen 3.2 μg; <figure-callout id="2" label="lane" filenames="US5874563-drawings-page-10.png,US5874563-drawings-page-11.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">lane</a></figure-callout> 2, Y5-5 antigen 1.6 μg; <figure-callout id="3" label="lane" filenames="US5874563-drawings-page-19.png,US5874563-drawings-page-3.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">lane</a></figure-callout> 3, Y5-5 antigen 0.8 μg; <figure-callout id="4" label="lane" filenames="US5874563-drawings-page-10.png,US5874563-drawings-page-12.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">lane</a></figure-callout> 4, Y5-5 antigen 0.4 μg; <figure-callout id="5" label="lane" filenames="US5874563-drawings-page-19.png,US5874563-drawings-page-3.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">lane</a></figure-callout> 5, Y5-5 antigen 0.2 μg; <figure-callout id="6" label="lane" filenames="US5874563-drawings-page-19.png,US5874563-drawings-page-4.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">lane</a></figure-callout> 6, GE3-2 antigen 0.4 μg; and <figure-callout id="7" label="lane" filenames="US5874563-drawings-page-4.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">lane</a></figure-callout> 7, Y5-10 antigen 0.4 μg. Human serum JC (FIG. 9A), T55806 (FIG. 9B), and <figure-callout id="2" label="Super Normal" filenames="US5874563-drawings-page-10.png,US5874563-drawings-page-11.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">Super Normal</a></figure-callout> 2 serum (FIG. 9C) were diluted 1:100. RM001, the anti-GST mouse monoclonal antibody, (FIG. 9D) was diluted 1:1000. Arrows indicate the locations of antigens Y5.5, GE3.2 and Y5.10. These results show specific immunoreactivity of the Y5-5 antigen with the JC serum. Further, the antigens GE3-2 and Y5-10 were reactive with T55806. However, the Y5-5 antigen was not reactive with the HGV-positive sera T55806.</div>
    <div class="description-paragraph style-scope patent-text">The Y5-10 antigen was also size-fractionated by SDS polyacrylamide gel electrophoresis. The gel was stained using coomasie blue protein stain. The gel was scanned for purity with a laser densitometer. The purity of the Y5-10 fusion protein was approximately 95%.</div>
    <div class="description-paragraph style-scope patent-text">2. Env Clones.</div>
    <div class="description-paragraph style-scope patent-text">The immunoclone Q7-12-1 was originally isolated by screening the ENV epitope library with the HGV positive sera J21094. Sequence specific primers were employed to isolate the HGV insert contained within the Q7-12-1 λgt11 clone. The Q7-12-1 insert was excised and cloned into pGEX-Nde. The sequence of the insert was confirmed by the DNA sequencing (SEQ ID NO:275).</div>
    <div class="description-paragraph style-scope patent-text">3. NS3 Clones.</div>
    <div class="description-paragraph style-scope patent-text">The immunoclone Y12-15-1 was originally isolated by screening the NS3 epitope library with the HGV positive sera E57963. Sequence specific primers were employed to isolate the HGV insert contained within the Y12-15-1 λgt11 clone. The Y12-15-1 insert was excised and cloned into pGEX-Nde. The sequence of the insert was confirmed by the DNA sequencing (SEQ ID NO:276).</div>
    <div class="description-paragraph style-scope patent-text">The immunoclone Y12-10-3 was originally isolated by screening the NS3 epitope library with the HGV positive sera J21689. Sequence specific primers were employed to isolate the HGV insert contained within the Y12-10-3 λgt11 clone. The Y12-10-3 insert was excised and cloned into pGEX-Nde. Production of fusion proteins by selected clone was evaluated by Western blot analysis. The sequence of the insert was confirmed by the DNA sequencing (SEQ ID NO:277).</div>
    <div class="description-paragraph style-scope patent-text">4 . NS2 Clones.</div>
    <div class="description-paragraph style-scope patent-text">Multiple negative strand immunoclones derived from sequences complementary to the sequences of the NS2 region of SEQ ID NO:14 were isolated. There are at least 2 significant ORFs encoded by the negative strand of HGV. The first of these ORFs, represented by the Q9 series of clones was described above. The second of these ORFs is located between nts 6723 and 7259 of the complement of SEQ ID NO:14 and also possess a 5' methionine at nt 6774. The second ORF encodes a 162 amino acid protein.</div>
    <div class="description-paragraph style-scope patent-text">Selected portions of the sequences of both of these negative strand ORFs were cloned into the expression vector pGEX-Nde. All of these subclones were obtained by the PCR amplification of PNF 2161 SISPA material using appropriate oligonucleotide primers, thus they contain the sequence of the HGV-PNF 2161 variant. Table 23 indicates the names, size of the ORF and locations relative to the complement of SEQ ID NO:14.</div>
    <div class="description-paragraph style-scope patent-text">
      </div> <pre xml:space="preserve" listing-type="tabular" class="style-scope patent-text">              TABLE 23
______________________________________
NAME/ORF   ORF       FROM NT (ATG)
                                  TO NT
______________________________________
5' NEG ORF 159 AA    6388         6865
3' NEG ORF 162 AA    6722         7258
NORF-F1/<figure-callout id="3'" label="R1" filenames="US5874563-drawings-page-3.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">R1</a></figure-callout> 3'        7107         7259
NORF-F4/<figure-callout id="3'" label="R1" filenames="US5874563-drawings-page-3.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">R1</a></figure-callout> 3'        6900         7259
NORF-F4/<figure-callout id="3'" label="KR2" filenames="US5874563-drawings-page-3.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">KR2</a></figure-callout>
           3'        6901         7172
NORF-F2/<figure-callout id="3'" label="R1" filenames="US5874563-drawings-page-3.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">R1</a></figure-callout> 3'        6744         7259
NORF-KF2/R4
           5'        6684         6865
NORF-KF1/R2
           5'        6881         6742
NORF-F3/R2 5'        6389         6742
NORF-F2/<figure-callout id="3'" label="R3" filenames="US5874563-drawings-page-3.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">R3</a></figure-callout> 3'        6744         6899
K3P-KF2/KR1
           5'        6684         6772
           3'        6744         6791
______________________________________
</pre>
    
    <div class="description-paragraph style-scope patent-text">The first 2 lines of this table identify the locations of the <figure-callout id="3'" label="NS2 region" filenames="US5874563-drawings-page-3.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">NS2 region</a></figure-callout> 5' and 3' negative strand ORFs relative to the complement of SEQ ID NO:14. The remaining lines indicate the specific nucleotide sequences expressed by all of the 9 clones. Note that several of the clones express amino acids located 5' to the hypothetical HGV initiating methionine of the ORF. Also note that the last clone listed, K3p-KF2/KR1, is a chimera expressing the indicated portions of the 5' ORF followed by the indicated portions of the 3' ORF.</div>
    <div class="description-paragraph style-scope patent-text">All of the DNA fragments were subsequently cloned into pGEX-Nde. Insert containing clones were also identified and confirmed.</div>
    <div class="description-paragraph style-scope patent-text">5. NS5a Clones.</div>
    <div class="description-paragraph style-scope patent-text">Table 24 lists a number of NS5a clones and the regions of SEQ ID NO:14 to which they correspond.</div>
    <div class="description-paragraph style-scope patent-text">
      </div> <pre xml:space="preserve" listing-type="tabular" class="style-scope patent-text">              TABLE 24
______________________________________
           HGV
Name       Source        Start  Stop
______________________________________
EXY10-F2   PNF           6416   6827
EXY10-F3   PNF           6537   6827
Q11-F1-R1  T56633        6537   6680
Q11-F1-R2  T56633        6537   6827
Q11-F2-R1  T56633        6576   6680
Q11-F2-R2  T56633        6576   6827
Y5-12      PNF           6633   6917
EXY12      PNF           6918   6977
EXY10F14   PNF           6822   6977
______________________________________
</pre>
    
    <div class="description-paragraph style-scope patent-text">These sequences were cloned into the vector pGEX-Nde for expression of the encoded protein antigens.</div>
    <div class="description-paragraph style-scope patent-text">B. Western Blot Analysis of Selected HGV Subclones.</div>
    <div class="description-paragraph style-scope patent-text">To determine the reactivity of both the negative and positive strand constructs described above whole cell lysates from bacteria expressing the various HGV subclones were prepared essentially as described in Example 13B. Aliquots of the expressed proteins were then fractionated by SDS-PAGE, the proteins transferred to nitrocellulose filters, and the filters probed with HGV-positive or control sera (e.g., anti-SJ26 MAB RM01). The blots were incubated with an appropriate reporter antibody.</div>
    <div class="description-paragraph style-scope patent-text">With respect to the HGV proteins tested, clear immunoreactivity to the protein NORF-F3/R2 was detected with the HGV sera J21689 and T56633. The NORF-F3/R2 subclone expresses the amino acid sequences that were also encoded by the Q9 series of negative strand epitope clones. The observed strong reactivity with HGV sera T56633 confirms the immunoreactivity of this region of the negative strand of HGV. Reactivity to the NORF-F3/R2 protein was not observed with the sera from the HGV negative individual R04316 or any of 5 other HGV negative supernormal sera tested.</div>
    <div class="description-paragraph style-scope patent-text">Additional blots indicated that the other major 5' ORF clone NORF KF2-R4, which expresses amino acids of the carboxy terminal half of the 5' negative strand ORF located does not react with the HGV-positive sera T56633. This observation in conjunction with the locations of the Q9 epitope clones described above suggest that the immunogenic epitope of this portion of the negative strand is contained within the 55 amino acid delineated above (SEQ ID NO:273). The fact that this sequence is recognized by other HGV antisera, including J21689, indicates that immunoreactivity towards this sequence is relatively widespread among HGV infected individuals.</div>
    <div class="description-paragraph style-scope patent-text">Further, clear immunoreactivity with the Y12-10-3 protein was observed with the HGV-infected sera J21689, J29374, and E57963. The specificity of this reactivity is additionally supported by the failure to observe immunoreactivity with the HGV antisera J29374 or E57963 in the absence of the induction of Y12-10-3 protein expression by IPTG. No reactivity to Y12-10-3 was observed with any of 7 supernormal sera tested.</div>
    <heading class="style-scope patent-text">Example 15</heading>
    <heading class="style-scope patent-text">A Multi-Antigen HGV Diagnostic Assay</heading>
    <div class="description-paragraph style-scope patent-text">Although the epitope clones described above do not appear to be reactive with all HGV PCR-positive sera, many of these clones react with a substantial fraction of the HGV infected sera they have been tested against. Additionally these proteins have not exhibited substantial cross reactivity with HGV-negative sera. It is therefore possible to construct a diagnostic assay in which several of these proteins are combined so that the individual reactivities of the protein are summed. Such an assay is expected to have a relatively high sensitivity for the detection of HGV-positive sera and a relatively low background reactivity with HGV-negative sera.</div>
    <div class="description-paragraph style-scope patent-text">Exemplary epitopes/antigens useful in such an assay include, but are not limited to, NORF-F3/R2 (NS2-Neg strand), Y12-10-3 (NS3), Q11-F2-R1 (NS5a), Y5-10 (NS5a), Y5-5 (NS5a), Q11-F2-R2 (combines 2 epitopes of NS5a).</div>
    <div class="description-paragraph style-scope patent-text">For this assay, individual antigens are typically selected that contain different unique epitopes that recognized different subset of HGV-positive sera. Further, such antigens typically do not significantly react with HGV-negative sera. Following the guidance of the present invention, additional useful immunogenic clones can be isolated.</div>
    <div class="description-paragraph style-scope patent-text">A multi-antigen diagnostic assay can take many formats. In one embodiment, the assay might entail immobilizing each of, et al., 5 HGV proteins and control proteins at separate locations on a nitrocellulose strip or other convenient solid phase format. Alternatively the non-viral portions of, for example, an HGV-fusion protein could be modified, either by insertions or deletions such that they would naturally migrate to easily distinguishable locations upon SDS PAGE and subsequent Western blot analysis. Strips are then incubated in test sera. After detection of bound antibody, a serum may then be scored based on (i) the number of antigens with which it is immunoreactive, and (ii) the strength of the immunological reactions. Reactivity to a non-HGV control protein would render a serum un-typeable. Reactivity with no HGV protein would classify a serum as HGV-negative.</div>
    <div class="description-paragraph style-scope patent-text">ELISA-based screening assay can be formed by combining purified antigen proteins in a single reaction zone or by creating protein constructs that express 2 or more of the reactive epitopes as a single protein (e.g., a HGV mosaic polypeptide). The methods to construct mosaic polypeptides is described herein. Q11-F2-R2 construct described above, in fact, represents a "matrix protein" that encodes 2 individual epitopes in a single polypeptide chain. Western blot assays may serve as a confirmatory assay for such an ELISA screening test.</div>
    <div class="description-paragraph style-scope patent-text">Alternatively or in addition, full length HGV proteins, such as E2, NS5a and NS3 might be placed in a single reaction zone. Sera reactive with such proteins may also be confirmed as HGV positive by Western blot assay.</div>
    <heading class="style-scope patent-text">Example 16</heading>
    <heading class="style-scope patent-text">Expression of Large HGV Polvpeptides</heading>
    <div class="description-paragraph style-scope patent-text">A. Exnression of Larger HGV Antigens in E. coli</div>
    <div class="description-paragraph style-scope patent-text">1. Cloning and Expression.</div>
    <div class="description-paragraph style-scope patent-text">To identify conformational HGV epitopes (not covered by small overlapping HGV constructs or by phage library screening) larger HGV protein constructs were generated in the pET-21a(+) vector (Novagen, Wisc.) based on the prediction of cleavage sites (Bazan, et al., 1989; Chambers, et al., 1990b; Grakoui, et al., 1993; Kyte and Doolittle, 1982). Individual HGV protein constructs were generated in a similar fashion to HGV sequences cloned into pGEX vectors.</div>
    <div class="description-paragraph style-scope patent-text">Briefly, selected HGV sequences were RT-PCR amplified from a HGV(+) human sera source using HGV sequence specific primers. The primers were engineered to contain appropriate restriction sites for cloning manipulations in the pET vector. Coding sequences of interest were typically inserted between the EcoRI site and the HindIII sites in the vector to produce 5' inframe fusions with T7.Tag leader sequence and 3' inframe fusion with a hexamer histidine sequence. T7.Tag (an 11 amino acid sequence) allows the detection of the fusion proteins using an anti-T7.Tag monoclonal antibody (Novagen, Wis.). The histidine hexamer at the carboxyl end of the fusion protein allows the purification of the protein using immobilized metal ion affinity chromatography.</div>
    <div class="description-paragraph style-scope patent-text">HGV fragments were ligated into appropriately digested pET-21a(+) vectors. Ligated products were transformed into competent E.coli (HMS174; Novagen, Wis.). Plasmid DNA from transformed HMS174 was analyzed for the presence of HGV sequences by PCR, using primers T7F(SEQ ID NO:157) and T7R(SEQ ID NO:158), which are homologous to pET-21a(+) vector sequences flanking the inserted molecule. The size of the PCR product was the insert size plus approximately 260 bp derived from the vector.</div>
    <div class="description-paragraph style-scope patent-text">For each construct the PCR results confirmed the presence of the insert sequences. Transformants with appropriate inserts were selected, plasmid DNAs with HGV inserts prepared and introduced into HMS174(DE3) competent E.coli (Novagen, Wis.) for the expression of HGV proteins.</div>
    <div class="description-paragraph style-scope patent-text">Expression of HGV proteins was induced with 1 mM IPTG. Expression of the T7.Tag fusion proteins was monitored by the appearance of the predicted size proteins on the Coomassie blue stained gel. Expression of the fusion proteins was confirmed by Western blot analysis using anti-T7.Tag antibody (Novagen, WI). HGV proteins expressed in pET-21a(+) vector are shown in the Table 25. The start and end points of the expressed sequences are given relative to SEQ ID NO:14. The amino acid sequence of GE-Cap is shown in SEQ ID NO:185.</div>
    <div class="description-paragraph style-scope patent-text">
      </div> <pre xml:space="preserve" listing-type="tabular" class="style-scope patent-text">              TABLE 25
______________________________________
               Serum
Name   Domain  Source  Start End   HGV aa
                                         Size (KDa)
______________________________________
GE-<figure-callout id="271" label="Cap capsid T55806" filenames="US5874563-drawings-page-9.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">Cap capsid  T55806</a></figure-callout>   271*  480* 70    11
GE-E1a E1      PNF      594  1148  185   24
GE-E2  E2/NS1  PNF     1149  2183  345   41
GE-NS2b
       NS2b    PNF     2904  3254  117   16
GE-<figure-callout id="3255" label="NS3 NS3 PNF" filenames="US5874563-drawings-page-19.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">NS3 NS3     PNF</a></figure-callout>     3255  5081  609   70
GE-NS4a
       NS4a    PNF     5082  6083  334   40
GE-NS4b
       NS4b    PNF     6084  6536  151   20
GE-NS4 NS4     PNF     5082  6536  485   57
GE-NS5a
       NS5a    PNF     6537  7529  331   39
GE-<figure-callout id="7530" label="NS5b NS5b PNF" filenames="US5874563-drawings-page-19.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">NS5b
       NS5b    PNF</a></figure-callout>     7530  9044  505   59
______________________________________
 *These sequences are given relative to SEQ ID NO: 178
</pre>
    
    <div class="description-paragraph style-scope patent-text">FIG. 12 shows the expression of each HGV proteins demonstrated by Western blot analysis with T7.Tag monoclonal antibody. The lanes in FIG. 12 are as follows: <figure-callout id="1" label="Lane" filenames="US5874563-drawings-page-10.png,US5874563-drawings-page-14.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">Lane</a></figure-callout> 1, pre-stained molecular weight marker (Bio-Rad); <figure-callout id="2" label="Lane" filenames="US5874563-drawings-page-10.png,US5874563-drawings-page-11.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">Lane</a></figure-callout> 2, uninduced GE-Cap lysate; Lanes 3-11, IPTG induced lysates of GE-Cap, E1a, E2, NS2b, NS3, NS4a, NS4b, NS4, and NS5b lysate, respectively. Lane 12 contained 1 μg of purified NS5a. Locations of each antigen are marked with arrow heads. As shown in FIG. 12 all the HGV proteins were expressed in E.coli.</div>
    <div class="description-paragraph style-scope patent-text">2. Western Blot Analyses of HGV proteins expressed in DET vector</div>
    <div class="description-paragraph style-scope patent-text">Western blot analyses of the HGV protein expressed in pET vector were performed as described in Example 11C using E. coli whole cell lysates and pre-absorbed sera. The results of these analyses demonstrated that several of pET HGV proteins are specifically immunoreactive with HGV-positive human sera but not with HGV-negative human sera. GE-NS2b-1 protein was immunoreactive with J21689 serum. The GE-NS5a-3 protein was immunoreactivity with several HGV (+) sera on Western blot analysis, including JC, T55806, T56633, J21689, E57963 and R0001. Among these sera T55806, J21689 and E57963 are HCV co-positive (by the PCR analysis). Neither GE-NS2b-1 nor GE-NS5a-3 were immunoreactive with several HGV negative sera tested.</div>
    <div class="description-paragraph style-scope patent-text">FIGS. 10A to 10F show the exemplary results of a series of Western blot experiments examining the reactivity of antigens GE-NS2b and GE-NS5a3. The lanes in each blot of FIGS. 10A to 10F are as follows: <figure-callout id="1" label="Lane" filenames="US5874563-drawings-page-10.png,US5874563-drawings-page-14.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">Lane</a></figure-callout> 1, uninduced GE-NS2b lysate; <figure-callout id="2" label="Lane" filenames="US5874563-drawings-page-10.png,US5874563-drawings-page-11.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">Lane</a></figure-callout> 2, IPTG induced GE-NS2b lysate; <figure-callout id="3" label="Lane" filenames="US5874563-drawings-page-19.png,US5874563-drawings-page-3.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">Lane</a></figure-callout> 3, uninduced GE-NS5a lysate; and <figure-callout id="4" label="Lane" filenames="US5874563-drawings-page-10.png,US5874563-drawings-page-12.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">Lane</a></figure-callout> 4, IPTG induced GE-NS5a lysate. Each blot was incubated with a human serum or mouse monoclonal antibody: FIG. 10A, J29374; FIG. 10B, J21689; FIG. 10C, T56633; FIG. 10D, T43608 (super normal serum); FIG. 10E, Anti-T7.Tag; and FIG. 10F, coomassie stained gel. The serum or monoclonal antibody that was used is indicated above each blot. Human sera were diluted 1:100 and anti-T7.Tag mouse monoclonal antibody was diluted 1:1000.</div>
    <div class="description-paragraph style-scope patent-text">In addition to the sera listed above, additional HGV-PCR positive sera have been screened using GE-NS5a. The results of all these analyses have demonstrated the reactivity of the GE-NS5a antigen with multiple HGV-infected sera. GE-NS5b was immunoreactive with HGV(+) sera JC and T55806 but was not immunoreactive with HGV(-) negative sera tested. FIGS. 13A to 13E show the results of a series of Western blot experiments examining the reactivity of antigen GE-NS5b. The lanes in each blot the figures are as follows: <figure-callout id="1" label="Lane" filenames="US5874563-drawings-page-10.png,US5874563-drawings-page-14.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">Lane</a></figure-callout> 1, pre-stained molecular weight marker (Bio-Rad); <figure-callout id="2" label="Lane" filenames="US5874563-drawings-page-10.png,US5874563-drawings-page-11.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">Lane</a></figure-callout> 2, uninduced GE-NS5b lysate; <figure-callout id="3" label="Lane" filenames="US5874563-drawings-page-19.png,US5874563-drawings-page-3.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">Lane</a></figure-callout> 3, IPTG induced GE-NS5b lysate.</div>
    <div class="description-paragraph style-scope patent-text">Each blot was incubated with a human serum or mouse monoclonal antibody: FIG. 13A, anti-T7.Tag monoclonal antibody; FIG. 13B, JC; FIG. 13C, T55806; and FIG. 13D, T43608 (super normal serum). FIG. 13E is a Coomassie Stain.</div>
    <div class="description-paragraph style-scope patent-text">FIGS. 14A to 14D show the results of a series of Western blot experiments examining the reactivity of antigen GE-E2. The lanes in each of FIGS. 14A to 14D are as follows: <figure-callout id="1" label="Lane" filenames="US5874563-drawings-page-10.png,US5874563-drawings-page-14.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">Lane</a></figure-callout> 1, pre-stained molecular weight marker (Bio-Rad); <figure-callout id="2" label="Lane" filenames="US5874563-drawings-page-10.png,US5874563-drawings-page-11.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">Lane</a></figure-callout> 2, uninduced GE-E2 lysate; <figure-callout id="3" label="Lane" filenames="US5874563-drawings-page-19.png,US5874563-drawings-page-3.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">Lane</a></figure-callout> 3, IPTG induced GE-E2 lysate. Each blot was incubated with a human serum or mouse monoclonal antibody: FIG. 14A, anti-T7.Tag monoclonal antibody; FIG. 14B, 3831781; and FIG. 14C, T43608 (super normal serum). FIG. 14D is Coomassie Stain. The serum or monoclonal antibody that was used is indicated above each blot. GE-E2 protein was immunoreactive with HGV-<figure-callout id="3831781" label="positive serum" filenames="US5874563-drawings-page-17.png,US5874563-drawings-page-18.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">positive serum</a></figure-callout> 3831781 but was not immunoreactive with supernormal serum T43608 (FIGS. 14B and 14C, respectively).</div>
    <div class="description-paragraph style-scope patent-text">Antigens GE-Cap and GE-NS4a were also specifically immunoreactive with HGV(+) serum J21689.</div>
    <div class="description-paragraph style-scope patent-text">B. Expression larger HGV Antigens in Insect Cells.</div>
    <div class="description-paragraph style-scope patent-text">Expression of proteins using recombinant baculoviruses offers the following advantages (i) a high level of recombinant protein expression, and (ii) the benefits of a higher eucaryotic system, including efficient protein translocation and modification. This. system is particularly useful for expression of translocated proteins, e.g., HGV E1, E2 and NS2a.</div>
    <div class="description-paragraph style-scope patent-text">1. Cloning and Expression.</div>
    <div class="description-paragraph style-scope patent-text">Spodoptera frugiperda insect cell culture Sf21 and a derivative of Autografa californica nuclear polyhedrosis virus "BACULOGOLD" (Pharmingen, San Diego, Calif.) were used for expression of HGV polypeptides. Established protocols were used for insect cell cultivation and for generation of recombinant baculoviruses by co-transfection of baculovirus plasmid transfer vectors with linearized baculovirus DNA (King, 1992). Conventional techniques were used for construction of baculovirus plasmid transfer vectors (Maniatis, et al.; Sambrook, et al.).</div>
    <div class="description-paragraph style-scope patent-text">The baculovirus transfer vector pAcYM1 (King, et al., 1992) was modified by ligating a double-stranded oligonucleotide coding for a Histidine hexamer into the vector's BamHI cloning site (vector designated pAcYMIH). A stop codon (TAA) was placed after the Histidine hexamer sequence. This provides a histidine hexamer on the carboxy-termini of expressed proteins. The BamHI cloning site of the pAcYMI parent vector remained intact in the pAcYMIH and could be used for cloning various genes in-frame with the Histidine hexamer. The histidine hexamer provides a method of rapid and efficient purification of the expressed protein (Janknecht, et al., 1991).</div>
    <div class="description-paragraph style-scope patent-text">A second baculovirus transfer vector, pVT-Bac, was also modified in a similar manner to provide a histidine hexamer on the carboxy-termini of expressed proteins. pVT-Bac like the pAcYMI vector contains a strong late polyhedrin promoter. In addition, pVT-Bac also provides a strong insect translocation signal sequence to ensure efficient translocation of the expressed proteins (Tessier, et al., 1991). The pVT-Bac vector was modified by ligating a double-stranded oligonucleotide coding for a histidine hexamer into the vector's BamHlI cloning site (yielding the pVT-BacH vector). The BamHI cloning site of the pVT-Bac parent vector remains intact in the obtained pVT-BacH vector and can be used for cloning genes in-frame with the insect leader sequence and the histidine hexamer sequence.</div>
    <div class="description-paragraph style-scope patent-text">DNA fragments coding for various HGV genes were obtained by reverse transcription PCR. Regions of the HGV genome were selected according to predicted cleavage sites (Bazan, et al., 1989; Chambers, et al., 1990b; Grakoui, et al., 1993; Kyte and Doolittle, 1982). The following primer pairs were used in RT-PCR amplification reactions using PNF 2161 source nucleic acid: E1, SEQ ID NO:242, SEQ ID NO:243; E2B (HGV signal sequence), SEQ ID NO:246, SEQ ID NO:247; E2C (insect signal sequence), SEQ ID NO:244, SEQ ID NO:245; NS2a, SEQ ID NO:248, SEQ ID NO:249; N82b, SEQ ID NO:250, SEQ ID NO:251; NS3, SEQ ID.NO:252, SEQ ID NO:253; NS4a, SEQ ID NO:254, SEQ ID NO:255; NS4b, SEQ ID NO:256, SEQ ID NO:257; NS5a, SEQ ID NO:258, SEQ ID NO:259; NS5b, SEQ ID NO:260, SEQ ID NO:261; and E1-E2-NS2a, SEQ ID NO:262, SEQ ID NO:263.</div>
    <div class="description-paragraph style-scope patent-text">Amplified DNA fragments were digested with BamHI or BglII endonucleases and cloned into BamHI cut pAcYMI, pAcYMIH, pVT-Bac or pVT-BacH vectors.</div>
    <div class="description-paragraph style-scope patent-text">Sequences coding for the E1 and E2 carboxy-terminal anchors as well as a hydrophobic sequence at the carboxy-terminus of NS5b were deleted in order to facilitate subsequent protein purification.</div>
    <div class="description-paragraph style-scope patent-text">The recombinant baculovirus plasmid transfer vectors containing HGV sequences were co-transfected with linearized baculovirus DNA and the recombinant viruses were selected as white foci in presence of X-gal (King, et al., 1992). Recombinant viruses were twice plaque-purified and propagated. Monolayers of Sf21 cells were infected with the recombinant baculoviruses at the <figure-callout id="5" label="multiplicity" filenames="US5874563-drawings-page-19.png,US5874563-drawings-page-3.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">multiplicity</a></figure-callout> 5 p.f.u. per cell and incubated at 27° C. for 60 h. The cells were washed with PBS and lysed in TNN buffer (50 mM Tris-HCl pH 8.0, 150 mM NaCl, 0.5% "NONIDET-P40"). Inclusion bodies were isolated by spinning the cell samples at 14k for 5 minutes. The inclusion bodies were resuspended in protein dissociation buffer (10% 2-mercaptoethanol, 10% SDS, 25% glycerol, 10 mM Tris-HCl pH 6.8, 0.02% Bromphenol blue) and incubated at 100° C. for 10 minutes.</div>
    <div class="description-paragraph style-scope patent-text">The protein expression patterns analyzed by SDS-PAGE. Proteins were separated by 0.1% SDS-18% PAGE and stained with Coomassie brilliant blue. The majority of the HGV proteins were expressed to a high level and could be easily detected on the Coomassie blue stained gels. NS5a and NS2a polypeptides were detected by <sup class="style-scope patent-text">35</sup> S methionine protein labeling (King, et al., 1992).</div>
    <div class="description-paragraph style-scope patent-text">HGV E2 protein glycosylation was examined as follows. Sf21 cells were infected with recombinant baculoviruses and processed as described above. Proteins were separated by 0.1% SDS-12% PAGE, electroblotted onto an "IMMOBILON-P" (Millipore, Bedford MA) membrane and reacted with Galanthus nivalis agglutinin (Boehringer Mannheim DIG Glycan differentiation kit) which is specific for mannose residues. The HGV E2 protein that was expressed with its own signal sequence was extensively glycosylated, indicating that the predicted E2 signal sequence can function as such.</div>
    <div class="description-paragraph style-scope patent-text">2. Immunofluorescence Assay Analysis.</div>
    <div class="description-paragraph style-scope patent-text">SF21 insect cells were infected with the baculovirus-HGV constructs described above. Cells were harvested, spun at 1.5K rpm for 3 minutes, washed in 1× PBS, and spun again.</div>
    <div class="description-paragraph style-scope patent-text">For Immunofluorescence Assay (IFA) (King, et al., 1992) the cells were resuspended in PBS and layered into the wells of glass slides such that the cells formed a sub-confluent layer in the wells of the slides. The slides were air-dried. The cells fixed with pre-chilled -70° C. acetone for 10 minutes and rehydrated with PBS for 5 minutes. The excess PBS was removed by blotting. The fixed cells were treated for one hour with the following "Blocking" buffer: 40 mM Tris HCl pH 7.5, 3% goat serum, 1% BSA, 1% nonfat milk and 0.1% gelatin.</div>
    <div class="description-paragraph style-scope patent-text">Primary antibody was then added to the fixed cells. Primary antibodies included a series of human HGV-positive sera and a positive control monoclonal antibody. Before use, the sera were pre-absorbed for non-specific proteins using insect cell lysate. Pre-absorption was carried out overnight at 40° C. Uninfected SF21 cells were used as a negative control. After addition of a selected primary antibody (sera), the slides were incubated for 2 hours then washed several times with PBS and excess buffer removed. A secondary antibody conjugated with fluorescein (0.5 μg/ml conc.) was then added to the samples on the slides. The incubation time and temperature for the secondary antibody was the same as for the primary antibody. After incubation, slides were washed in PBS and mounted with a cover slip. The fluorescence of the cells was then determined using a fluorescence microscope.</div>
    <div class="description-paragraph style-scope patent-text">The results of this analysis were as follows. Cells expressing HGV antigen E1-E2-NS2a were immunoreactive with 4/10 HGV-positive sera and weakly immunoreactive with an additional 2/10 sera. Cells expressing E1 were weakly immunoreactive with 1/10 sera. Cells expressing E2 were immunoreactive with 3/10 sera and weakly immunoreactive with 1/10 sera. None of the cells carrying HGV antigens were immunoreactive with supernormal control sera.</div>
    <div class="description-paragraph style-scope patent-text">3. Western Blot Analyses of HGV proteins expressed in baculo vector</div>
    <div class="description-paragraph style-scope patent-text">Western blot analyses of the HGV proteins expressed in recombinant baculo virus infected Sf21 insect cells were also performed. Inclusion bodies were prepared as described above and subjected to Western blot analysis. Western blot analysis was performed using pre-absorbed sera. The results of the analyses demonstrated that E2 proteins (one variant having the endogenous HGV signal sequence, E2B, and a second variant carrying an insect signal sequence, E2C) were specifically immunoreactive with HGV(+) <figure-callout id="3831781" label="serum" filenames="US5874563-drawings-page-17.png,US5874563-drawings-page-18.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">serum</a></figure-callout> 3831781.</div>
    <div class="description-paragraph style-scope patent-text">FIGS. 15A to 15D show the results of a series of Western blot experiments examining the reactivity of baculo antigens E2B and E2C. The lanes in each blot of FIGS. 15A to 15D were as follows: <figure-callout id="1" label="Lane" filenames="US5874563-drawings-page-10.png,US5874563-drawings-page-14.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">Lane</a></figure-callout> 1, pre-stained molecular weight marker (Bio-Rad); <figure-callout id="2" label="Lane" filenames="US5874563-drawings-page-10.png,US5874563-drawings-page-11.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">Lane</a></figure-callout> 2, E2B lysate; <figure-callout id="3" label="Lane" filenames="US5874563-drawings-page-19.png,US5874563-drawings-page-3.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">Lane</a></figure-callout> 3, E2C. lysate; <figure-callout id="4" label="Lane" filenames="US5874563-drawings-page-10.png,US5874563-drawings-page-12.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">Lane</a></figure-callout> 4, β-galactosidase lysate. Each blot was incubated with a human or rabbit serum : FIG. 15A, rabbit anti-E2 antibody; FIG. 15B, 3831781 (an HGV-PCR-positive serum); FIG. 15C, 3838857 (an HGV-negative serum). FIG. 15D a Coomassie Stain. The serum or rabbit antibody that was used is indicated above each blot. Human sera were diluted 1:100 and rabbit serum was diluted 1:1000.</div>
    <div class="description-paragraph style-scope patent-text">Further, HGV antigen NS2b protein expressed in insect cells was immunoreactive with J21689. These results are consistent with the results obtained with pET expressed HGV proteins.</div>
    <div class="description-paragraph style-scope patent-text">C. Exoression of Larger Antigens in Vaccinia.</div>
    <div class="description-paragraph style-scope patent-text">1. Cloning and Expression.</div>
    <div class="description-paragraph style-scope patent-text">Various regions of HGV genome were integrated into vaccinia virus genome for expression. An exemplary HGV polypeptide expression strategy is given in FIG. 16. HGV (PNF 2161 variant) proteins expressed in vaccinia virus are schematically illustrated in FIG. 16. Full length polyprotein is drawn (not to scale) by an open box indicating regions of predicted proteins: C=highly basic protein, 4A=NS4A, 4B=NS4B, 5a=NS5A, 5b=NS5B. The individual boxes with nucleotide locations (below the polyprotein) represents exemplary regions of HGV for expression in vaccinia virus. The number in the box stands for recombinant virus nomenclature. <figure-callout id="1" label="Virus #" filenames="US5874563-drawings-page-10.png,US5874563-drawings-page-14.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">Virus #</a></figure-callout>1 was derived from the highly basic protein region of HGV Stain T55806 (SEQ ID NO:185).</div>
    <div class="description-paragraph style-scope patent-text">Two sets of recombinant viruses were generated. The first set contained HGV sequences that correspond to individual protein domains based on sequence analysis of HGV cDNA (FIG. 16, fragments #1 to #9). The second set contained HGV sequences that spanned multiple protein domains, up to full length of HGV genome (FIG. 16, #10, #11, #14).</div>
    <div class="description-paragraph style-scope patent-text">The various regions of the HGV genome were cloned into the multicloning site of the vaccinia expression vector. A recombinant vaccinia virus expression system was used that included bacterial phage T7 system and E. coli lac repressor for high level inducible expression (Fuerst, 1986; Elroy-Stein, 1989; Alexander, 1992; Moss, et al.). Therefore, recombinant protein is expressed only in the presence of an inducer, such as isopropyl beta-D-thiogalactoside (IPTG). Both direct cloning and PCR were used for plasmid construction. In the latter, restriction endonuclease sites suitable for cloning into the vaccinia vector were incorporated into primers used to amplify individual DNA fragment.</div>
    <div class="description-paragraph style-scope patent-text">A polyhistidine tag was also incorporated into every clone covering individual domains of HGV for use in purifying the expressed proteins. HGV-PCR amplification products were digested with the appropriate restriction enzymes and ligated into the vaccinia vector. Target HGV cDNA fragments were integrated into vaccinia virus genome through homologous recombination and drug (mycophenolic acid) selection (Falkner, 1988, Earl, 1991). Recombinant virus were plaque purified 4 times before a viral stock was generated.</div>
    <div class="description-paragraph style-scope patent-text">The length of each clone in nucleotides is indicated in FIG. 16. The group of smaller clones (#1 to #9) are useful for HGV epitope mapping. The larger clones (e.g., #10, #11 and #14) are also useful for mapping the HGV polyprotein cleavage sites experimentally. In addition to the clones shown in FIG. 16, additional recombinant viruses covering multiple domains from NS3 to NS5b can be constructed.</div>
    <div class="description-paragraph style-scope patent-text">Expression plasmids were transfected into mammalian cells which had been infected with a parent vaccinia virus. CV-1 and BS-C-1 cells were maintained in Minimum Essential Medium (MEM) supplemented with 10% fetal bovine serum. The cells were used for transfection (CV-1) and recombinant virus selection and propagation (BS-C-1).</div>
    <div class="description-paragraph style-scope patent-text">2. Evaluation of recombinant protein expression.</div>
    <div class="description-paragraph style-scope patent-text">BS-C-1 cells were infected with recombinant virus in the presence or absence of IPTG for 7 hours after which cells were labeled with <sup class="style-scope patent-text">35</sup> S-methionine for another one hour (Zhang, 1991). Briefly, 1×10<sup class="style-scope patent-text">6</sup> BS-C-1 cells were infected with recombinant virus at a multiplicity of infection (MOI) of 10 plaque forming unit (PFU) per cell for 1 h and then supplemented with medium in the presence or absence of 5 mM IPTG for another 6 h. Cells were pulse-labeled with 600 ul Methionine-free medium supplemented with 2.5% dialyzed fetal bovine serum plus 60 uCi 35S-methionine ("TRAN <sup class="style-scope patent-text">35</sup> S-LABEL", ICN, Costa Mesa, Calif.) in the presence or absence of 5 mM IPTG for another 60 min. Labeled cells were then lysed on ice for 10 min in the presence of 100 mM Tris pH8.0, 150 mM NaCl, and 1% "TRITON X-100." Nuclei were spun down and supernatant was collected for analysis.</div>
    <div class="description-paragraph style-scope patent-text">Cell lysate was analyzed by SDS-polyacrylamide gel electrophoresis (Fling, 1986; Schagger, 1987). Gels were fixed with 50% methanol and 10% acetic acid before they were treated with a fluorograph solution "AMPLIFY" (Amersham, Arlington Heights, Ill.). Gels were dried and exposed to X-ray film.</div>
    <div class="description-paragraph style-scope patent-text">Using this method, expression of HGV polypeptides by viruses containing <figure-callout id="4" label="inserts #" filenames="US5874563-drawings-page-10.png,US5874563-drawings-page-12.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">inserts #</a></figure-callout>4 to #11, and #14 (FIG. 16) has been confirmed. Expression of polypeptides corresponding to other regions is confirmed in a similar manner. For example, in a NS5a construct, upon induction by IPTG, a unique polypeptide was produced that migrated just below a 46 KDa protein standard. This protein was not seen in the infection in the absence of IPTG induction, establishing the identity of the protein as NS5a recombinant protein.</div>
    <div class="description-paragraph style-scope patent-text">Further, limited immunoprecipitations using HGV region-specific antisera (for example, rabbit anti-sera raised against an isolated HGV polypeptide from the region of interest) against <sup class="style-scope patent-text">35</sup> S-Met labeled cell lysate from individual virus infections was carried out to evaluate the protein expression from recombinant viruses. For example, expression of NS2, NS3, NS4B, NS5A and NS5B has been confirmed. An alternative method, to evaluate recombinant protein expression is to perform western blot analysis with HGV-region-specific antisera.</div>
    <div class="description-paragraph style-scope patent-text">When the full length HGV polyprotein was expressed in #14 virus (FIG. 16), processed products of NS2, NS3 and NS5A were detected using immunoprecipitation with HGV region-specific antisera, demonstrating the usefulness of the full length HGV clone to evaluate polyprotein processing.</div>
    <div class="description-paragraph style-scope patent-text">Using an expression strategy similar to that shown in FIG. 16, candidate HGV proteins/antigens can be expressed in yeast or CHO cells. Yeast offers high level of expression, economical operation, and ease of scaling up for commercial production. CHO cell lines allow secretion of the recombinant proteins into growth media for large scale protein production and purification useful, for example, for vaccine development.</div>
    <heading class="style-scope patent-text">Example 17</heading>
    <heading class="style-scope patent-text">HGV Encoded Highly Basic Proteins</heading>
    <div class="description-paragraph style-scope patent-text">A. Determination of the Methionine used for Initiation in the Translation of HGV from PNF and T55806.</div>
    <div class="description-paragraph style-scope patent-text">The methionine located at nucleotide (nt) 459 (relative to SEQ ID NO:14) in the HGV-PNF 2161 variant is in-frame with the polyprotein. The "capsid" region appears to be 32 amino acid long. In other HGV isolates, such as T55806, this region is longer (e.g., about 83 amino acids). The methionine located at nt 349 (relative to SEQ ID NO:14) in HGV-PNF 2161 variant is not in-frame with the polyprotein sequence, but a methionine at the same position in HGV-T55806 variant is in frame with the polyprotein. To see if there is a read-through or a ribosomal frame shift at this position in HGV-PNF 2161, the following experiments were carried out.</div>
    <div class="description-paragraph style-scope patent-text">Constructs were made containing (i) HGV genomic sequences having all the MET codons upstream of the HGV E1 region (e.g., in HGV-PNF 2161 there are six such METs and five such in T55806), (ii) two different 3' ends for each construct to allow determination of whether a ribosome shift of read-through occurs. For a given genomic DNA, if both translated products are the same size, that suggests they are terminated prematurely at the stop codon. On the other hand, if read-through or frameshift occurs two products that differ by 55 amino acids are expected.</div>
    <div class="description-paragraph style-scope patent-text">A total of 21 constructs containing sequences from variants HGV-PNF 2161 and HGV-T55806 were subcloned in a pGEX vector and corresponding proteins expressed in E. coli. Sizes of the resulting translation products were determined by both Coomassie stained gels and Westerns that were blotted with monoclonal anti-GST antibody. Induced and un-induced samples were prepared for each construct.</div>
    <div class="description-paragraph style-scope patent-text">The results demonstrated that the size of the protein products corresponded to that expected by translation initiating at the first MET in-frame with the polyprotein. There was no evidence of frame-shifting or read-through.</div>
    <div class="description-paragraph style-scope patent-text">B. Alternative Encoded Highly Basic Proteins.</div>
    <div class="description-paragraph style-scope patent-text">The method of Fickett (1982) was used to scan the genomic sequences HGV-PNF 2161 and HGV-JC for sequences that potentially encode proteins (i) alternative to the previously described polyprotein, (ii) showing conservation between HGV-PNF 2161 and HGV-JC, and (iii) having predicted isoelectric points in excess of <figure-callout id="10" label="pH" filenames="US5874563-drawings-page-12.png,US5874563-drawings-page-19.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">pH</a></figure-callout> 10. Two such potential proteins were identified.</div>
    <div class="description-paragraph style-scope patent-text">The first protein is encoded by residues 628 through 882 (relative to SEQ ID NO:14) in HGV-PNF 2161 and by residues 556 through 810 (relative to SEQ ID NO:182) in HGV-JC. This protein is 85 amino acids long, is greater than 75% homologous between HFV94-1 and JC9B, and has a predicted pI of 11.6-12.3.</div>
    <div class="description-paragraph style-scope patent-text">The second protein is encoded by residues 6844 through 7125 in HGV-PNF 2161 (relative to SEQ ID NO:14) and by 6772 to 7053 in HGV-JC (relative to SEQ ID NO:182). This protein is 94 amino acids long, is greater than 88% homologous between HGV-PNF 2161 and HGV-JC, and has a predicted pI of 12.4-12.7.</div>
    <div class="description-paragraph style-scope patent-text">These exemplary two proteins represent potentially expressed highly basic proteins of HGV.</div>
    <heading class="style-scope patent-text">Example 18</heading>
    <heading class="style-scope patent-text">Cloning Further HGV Isolates and Design of Diagnostic Primers</heading>
    <div class="description-paragraph style-scope patent-text">A. Construction of a cDNA Clone of HGV-PNF 2161.</div>
    <div class="description-paragraph style-scope patent-text">A CDNA clone of the nearly full-length HGV genome from PNF 2161 was constructed by cloning three overlapping PCR products into the plasmid vector pGEM3Z (Promega, Madison, Wis.). The PCR products used in this construction were obtained by reverse transcription with "SUPERSCRIPT II" (Gibco/BRL, Gaithersburg, Md.) followed by PCR using reaction conditions that allowed for the amplification of long target sequences ("rTth-XL" polymerase and "XL PCR BUFFERS", Applied Biosystems, Foster City, Calif.). The rTth enzyme used for these "long-range" PCR reactions has proof-reading activity (i.e. 3' to 5' exonuclease activity) that corrects mis-incorporated nucleotides, thus providing for high fidelity PCR.</div>
    <div class="description-paragraph style-scope patent-text">The three products used to construct the HGV genome included (i) an internal 6.7 kb product (nt 2101 to 8834 of SEQ ID NO:14) amplified using the primers GV75-36FE (SEQ ID NO:228) and GV75-7064RLE (SEQ ID NO:229), (ii) a 2.8 kb 5'-end product (nt 38 to 2899 of SEQ ID NO:14) amplified using 28F (SEQ ID NO:230) and FV94-2864R (SEQ ID NO:231), and (iii) a 2.9 <figure-callout id="3'" label="kb" filenames="US5874563-drawings-page-3.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">kb</a></figure-callout> 3'-end product (nt 6449 to 9366 of SEQ ID NO:14) amplified using FV94-6439F (SEQ ID NO:232) and FV94-9331R (SEQ ID NO:233).</div>
    <div class="description-paragraph style-scope patent-text">Initially, the 6.7 kb internal fragment was cloned into the "TA-vector" pCRII to create the clone HGV7. Subsequently, a 6.1 kb KpnI/EcoRI fragment was removed from HGV7 and combined with the KpnI/XbaI digested 2.8 kb 5'-end product (the primer 28F contains an artificial XbaI site) and cloned into XbaI/EcoRI digested pGEM3Z. This 8.8 kb clone, which lacks about 0.6 kb of the 3' portion of the HGV genome, was designated HGV-KEX-2. To construct the nearly full-length HGV genome, the 3'-end HGV product was digested with NheI and EcoRI (the primer FV94-9331R contains an artificial EcoRI site) and cloned into NheI/EcoRI digested HGV-KEX-2 plasmid creating a cloned HGV-PNF2161 sequence of 9329 nt (nt 38 to 9366 of SEQ ID NO:14) that is designated 3Z-HGV94-6. The complete sequence of 3Z-HGV94-6 is presented as SEQ ID NO:234.</div>
    <div class="description-paragraph style-scope patent-text">The clone 3Z-HGV94-6 may be used to generate in vitro-transcribed full-length HGV RNA or portions thereof (e.g., using SP6 polymerase). The RNA molecules can be used to transfect human cell lines. This approach could be used to map the various regions of the viral genome, study its replication, and understand the mechanisms of HGV pathogenicity in human cells (Rice, et al., 1989; Sumiyoshi, et al., 1992; Yoo, et al., 1995).</div>
    <div class="description-paragraph style-scope patent-text">B. Cloning the JC Variant.</div>
    <div class="description-paragraph style-scope patent-text">One milliliter of JC serum was spun at 40,000 rpms (Beckman, Spinco Rotor 70.1Ti) for 2 hours. The resulting pellet was extracted using "TRIREAGENT" (MRC, Cincinnati, Ohio), resulting in the formation of 3 phases. The upper phase contained RNA only. This phase was taken and RNA recovered by ethanol precipitation.</div>
    <div class="description-paragraph style-scope patent-text">HGV cDNA molecules were generated from the JC sample by two methods. The first method was amplification (RT-PCR) of the JC nucleic acid sample using specific and nested primers. The primer sequences were based on the HGV sequence obtained from PNF 2161 serum. The criteria used to select the primers were (i) regions having a high G/C content, and (ii) no repetitious sequences.</div>
    <div class="description-paragraph style-scope patent-text">The second method used to generate HGV cDNA molecules was amplification using HGV (PNF 2161) specific primers followed by identification of HGV specific sequences with <sup class="style-scope patent-text">32</sup> P-labelled oligonucleotide probes. Such DNA hybridizations were carried out essentially as described by Sambrook, et al. (1989). The PCR derived clones were either (i) cloned into the "TA" vector (Invitrogen, San Diego, Calif.) and sequenced with vector primers (TAR and TAF), or (ii) sequenced directly after PCR amplification. Both the probe and primer sequences were based on the HGV variant obtained from the PNF 2161 serum.</div>
    <div class="description-paragraph style-scope patent-text">These two approaches yielded multiply-overlapping HGV fragments from the JC serum. Each of these fragments were cloned and sequenced. The sequences were aligned to obtain the HGV (JC-variant) consensus sequence presented as SEQ ID NO:182 (polypeptide sequence, SEQ ID NO:183). The sequence of each region of the HGV (JC-variant) virus was based on a consensus from at least three different, overlapping, independent clones.</div>
    <div class="description-paragraph style-scope patent-text">C. Other HGV Variants.</div>
    <div class="description-paragraph style-scope patent-text">In addition to the HGV PNF 2161-variant and JC-variant sequences, three partial HGV isolates have been obtained from the sera BG34, T55806 and EB20 by methods similar to those described above. The partial sequences of these isolates are presented as SEQ ID NO:176 (BG34 nucleic acid), SEQ ID NO:177 (BG34 polypeptide), SEQ ID NO:178 (T55806 nucleic acid), SEQ ID NO:179 (T55806 polypeptide), SEQ ID NO:180 (EB20-2 nucleic acid) and SEQ ID NO:181 (EB20-2 polypeptide).</div>
    <div class="description-paragraph style-scope patent-text">D. Alternative Primers for Diagnostic PCR.</div>
    <div class="description-paragraph style-scope patent-text">PCR primers and corresponding assay development may be derived from regions of the HGV genome(s) typically based on the analysis of conserved regions. Based on comparisons of the HGV-JC variant and the HGV-PNF 2161 variant, the 5' untranslated region of HGV was selected as one such region for development of a further PCR-based diagnostic test for the detection of HGV isolates. Two exemplary primers are FV-94-22F (SEQ ID NO:124) and FV94-724R (SEQ ID NO:125). These primers amplify an approximately 728 bp fragment of the HGV genome.</div>
    <div class="description-paragraph style-scope patent-text">Sequence analysis was performed on amplification products from reactions employing these two primers for 36 isolates of HGV (including PNF 2161 and JC, see Table 26). An approximately 400 bp region (nt 69 to 469 of SEQ ID NO:14) of the approximately 728 bp amplification product was used for multiple sequence alignments (Table 26) and further determination of conserved regions (see below).</div>
    <div class="description-paragraph style-scope patent-text">
      </div> <pre xml:space="preserve" listing-type="tabular" class="style-scope patent-text">              TABLE 26
______________________________________
SEQ ID    Serum
NO:       Code      Country  % ID PNF 2161
______________________________________
186       S59       England  96.8
187       S368      England  98.8
188       S309      England  95.5
189       FZ        Australia
                             96
190       G21       Greece   97.8
191       G23       Greece   94.3
192       G59       Greece   93.6
193       E36       Egypt    94
194       R38730    USA      94.8
195       G281      Greece   97.8
196       G157      Greece   94.3
197       G154      Greece   96
198       G213      Greece   94.8
199       G204      Greece   98.3
200       G191      Greece   94.8
201       G299      Greece   94.8
202       T56957    USA      95.3
203       C01698    USA      98.8
204       T27034    USA      93.5
205       E57963    USA      98.5
206       R37166    USA      97.5
207       B5        Germany  95.5
208       B33       Germany  95.5
209       FH010     Australia
                             95
210       <figure-callout id="100" label="PNF2161 USA" filenames="US5874563-drawings-page-7.png,US5874563-drawings-page-8.png" class="style-scope patent-text"><a title="Show image containing callout" class="style-scope figure-callout">PNF2161   USA</a></figure-callout>      100
211       JC        USA      96.3
212       7155      Peru     89.8
213       7244      Peru     89
214       K27       Korea    89.5
215       K30       Korea    89.5
216       T55875    USA      97.3
217       T56633    USA      93.5
218       EB20      Egypt    94.1
219       T55806    USA      95.6
220       BG34      Greece   94.8
221       BE12      Egypt    95
______________________________________
</pre>
    
    <div class="description-paragraph style-scope patent-text">The development of an amplification-based (e.g., PCR) or probe-based method/assay for the detection of HGV isolates in samples involves the selection of appropriate primer/probe sequences. Two criteria for such an assay are low copy sensitivity and specificity for HGV sequences. Alignments of sequences (such as just described) can help guide primer/probe selection and design.</div>
    <div class="description-paragraph style-scope patent-text">Several criteria for selecting primers are as follows: (i) forward and reverse primers of a pair should not be significantly complementary in sequence, and (ii) primers should not have significant self complementarity or the potential to form secondary structures. These precautions minimize the potential for generation of primer dimers or oligomers.</div>
    <div class="description-paragraph style-scope patent-text">Primers may optimally be designed from sequence regions showing no variation among different isolates but may also be designed from regions of less homology by incorporating mixed base synthesis or neutral bases, such as inosine, at those positions to account for known isolate divergence. The following two groups of primers are examples of primers may be employed in development of a PCR-based assay for detection of HGV genomes: forward primers SEQ ID NO:222, SEQ ID NO:223 and SEQ ID NO:224; and reverse primers SEQ ID NO:225, SEQ ID NO:226 and SEQ ID NO:227.</div>
    <div class="description-paragraph style-scope patent-text">Various combinations of primers, may be employed in development of an HGV diagnostic assay. Optimal combinations of primers are experimentally determined and typically address considerations for assay sensitivity and specificity. Such considerations include the following: (i) a PCR product length of 100-300 bp for efficient amplification and ease of product detection; (ii) an ability to reproducibly detect at least 10 copies of target HGV, and (iii) an ability to reproducibly detect a majority of HGV variants.</div>
    <div class="description-paragraph style-scope patent-text">In addition, probe sequences may be similarly designed with mixed base or neutral base syntheses and/or may be used at reduced stringency so as to detect a majority of HGV variants.</div>
    <div class="description-paragraph style-scope patent-text">While the invention has been described with reference to specific methods and embodiments, it will be appreciated that various modifications and changes may be made without departing from the invention.</div>
    <div class="description-paragraph style-scope patent-text">
      </div> <pre xml:space="preserve" listing-type="tabular" class="style-scope patent-text">__________________________________________________________________________
SEQUENCE LISTING
(1) GENERAL INFORMATION:
(iii) NUMBER OF SEQUENCES: 277
(2) INFORMATION FOR SEQ ID NO:1:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 18 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: SISPA primer, top strand Linker AB
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:1:
GGAATTCGCGGCCGCTCG18
(2) INFORMATION FOR SEQ ID NO:2:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 20 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Linker AB, bottom strand
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:2:
CGAGCGGCCGCGAATTCCTT20
(2) INFORMATION FOR SEQ ID NO:3:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 237 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: double
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: PNF 2161 CLONE 470-20-1
(ix) FEATURE:
(A) NAME/KEY: CDS
(B) LOCATION: 1..237
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:3:
GAATTCGCGGCCGCTCGGGCTGTCTCGGACTCTTGGATGACCTCGAAT48
GluPheAlaAlaAlaArgAlaValSerAspSerTrpMetThrSerAsn
151015
GAGTCAGAGGACGGGGTATCCTCCTGCGAGGAGGACACCGGCGGGGTC96
GluSerGluAspGlyValSerSerCysGluGluAspThrGlyGlyVal
202530
TTCTCATCTGAGCTGCTCTCAGTAACCGAGATAAGTGCTGGCGATGGA144
PheSerSerGluLeuLeuSerValThrGluIleSerAlaGlyAspGly
354045
GTACGGGGGATGTCTTCTCCCCATACAGGCATCTCTCGGCTACTACCA192
ValArgGlyMetSerSerProHisThrGlyIleSerArgLeuLeuPro
505560
CAAAGAGAGGGTGTACTGCAGTCCTCCACGAGCGGCCGCGAATTC237
GlnArgGluGlyValLeuGlnSerSerThrSerGlyArgGluPhe
657075
(2) INFORMATION FOR SEQ ID NO:4:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 79 amino acids
(B) TYPE: amino acid
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: protein
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:4:
GluPheAlaAlaAlaArgAlaValSerAspSerTrpMetThrSerAsn
151015
GluSerGluAspGlyValSerSerCysGluGluAspThrGlyGlyVal
202530
PheSerSerGluLeuLeuSerValThrGluIleSerAlaGlyAspGly
354045
ValArgGlyMetSerSerProHisThrGlyIleSerArgLeuLeuPro
505560
GlnArgGluGlyValLeuGlnSerSerThrSerGlyArgGluPhe
657075
(2) INFORMATION FOR SEQ ID NO:5:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 22 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: HAV-R1
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:5:
GTTGACCAACTGAGTCTGAAGC22
(2) INFORMATION FOR SEQ ID NO:6:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 22 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: HAV-F1
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:6:
GATTGGAAATCTGATCCGTCCC22
(2) INFORMATION FOR SEQ ID NO:7:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 19 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: HCV- LANR
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:7:
TCGCGACCCAACACTACTC19
(2) INFORMATION FOR SEQ ID NO:8:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 18 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: HCV 1532
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:8:
GGGGGCGACACTCCACCA18
(2) INFORMATION FOR SEQ ID NO:9:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 25 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer 470- 20-1-77F
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:9:
CTCTTTGTGGTAGTAGCCGAGAGAT25
(2) INFORMATION FOR SEQ ID NO:10:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 24 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer 470- 20-1-211R
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:10:
CGAATGAGTCAGAGGACGGGGTAT24
(2) INFORMATION FOR SEQ ID NO:11:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 27 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer KL- 1
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:11:
GCAGGATCCGAATTCGCATCTAGAGAT27
(2) INFORMATION FOR SEQ ID NO:12:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 29 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer KL- 2
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:12:
ATCTCTAGATGCGAATTCGGATCCTGCGA29
(2) INFORMATION FOR SEQ ID NO:13:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 20 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: LAMBDA GT11, REVERSE PRIMER
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:13:
GGCAGACATGGCCTGCCCGG20
(2) INFORMATION FOR SEQ ID NO:14:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 9392 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: double
(D) TOPOLOGY: unknown
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: HGV-PNF 2161 Variant
(ix) FEATURE:
(A) NAME/KEY: CDS
(B) LOCATION: 459..9077
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:14:
ACGTGGGGGAGTTGATCCCCCCCCCCCGGCACTGGGTGCAAGCCCCAGAAACCGACGCCT60
ATCTAAGTAGACGCAATGACTCGGCGCCGACTCGGCGACCGGCCAAAAGGTGGTGGATGG120
GTGATGACAGGGTTGGTAGGTCGTAAATCCCGGTCACCTTGGTAGCCACTATAGGTGGGT180
CTTAAGAGAAGGTTAAGATTCCTCTTGTGCCTGCGGCGAGACCGCGCACGGTCCACAGGT240
GTTGGCCCTACCGGTGGGAATAAGGGCCCGACGTCAGGCTCGTCGTTAAACCGAGCCCGT300
TACCCACCTGGGCAAACGACGCCCACGTACGGTCCACGTCGCCCTTCAATGTCTCTCTTG360
ACCAATAGGCGTAGCCGGCGAGTTGACAAGGACCAGTGGGGGCCGGGGGCTTGGAGAGGG420
ACTCCAAGTCCCGCCCTTCCCGGTGGGCCGGGAAATGCATGGGGCCACCCAGC473
MetGlyProProSer
15
TCCGCGGCGGCCTGCAGCCGGGGTAGCCCAAGAATCCTTCGGGTGAGG521
SerAlaAlaAlaCysSerArgGlySerProArgIleLeuArgValArg
101520
GCGGGTGGCATTTCCTTTTTCTATACCATCATGGCAGTCCTTCTGCTC569
AlaGlyGlyIleSerPhePheTyrThrIleMetAlaValLeuLeuLeu
253035
CTTCTCGTGGTTGAGGCCGGGGCCATTCTGGCCCCGGCCACCCACGCT617
LeuLeuValValGluAlaGlyAlaIleLeuAlaProAlaThrHisAla
404550
TGTCGAGCGAATGGGCAATATTTCCTCACAAATTGTTGTGCCCCGGAG665
CysArgAlaAsnGlyGlnTyrPheLeuThrAsnCysCysAlaProGlu
556065
GACATCGGGTTCTGCCTGGAGGGTGGATGCCTGGTGGCCCTGGGGTGC713
AspIleGlyPheCysLeuGluGlyGlyCysLeuValAlaLeuGlyCys
70758085
ACGATTTGCACTGACCAATGCTGGCCACTGTATCAGGCGGGTTTGGCT761
ThrIleCysThrAspGlnCysTrpProLeuTyrGlnAlaGlyLeuAla
9095100
GTGCGGCCTGGCAAGTCCGCGGCCCAACTGGTGGGGGAGCTGGGTAGC809
ValArgProGlyLysSerAlaAlaGlnLeuValGlyGluLeuGlySer
105110115
CTATACGGGCCCCTGTCGGTCTCGGCCTATGTGGCTGGGATCCTGGGC857
LeuTyrGlyProLeuSerValSerAlaTyrValAlaGlyIleLeuGly
120125130
CTGGGTGAGGTGTACTCGGGTGTCCTAACGGTGGGAGTCGCGTTGACG905
LeuGlyGluValTyrSerGlyValLeuThrValGlyValAlaLeuThr
135140145
CGCCGGGTCTACCCGGTGCCTAACCTGACGTGTGCAGTCGCGTGTGAG953
ArgArgValTyrProValProAsnLeuThrCysAlaValAlaCysGlu
150155160165
CTAAAGTGGGAAAGTGAGTTTTGGAGATGGACTGAACAGCTGGCCTCC1001
LeuLysTrpGluSerGluPheTrpArgTrpThrGluGlnLeuAlaSer
170175180
AACTACTGGATTCTGGAATACCTCTGGAAGGTCCCATTTGATTTCTGG1049
AsnTyrTrpIleLeuGluTyrLeuTrpLysValProPheAspPheTrp
185190195
AGAGGCGTGATAAGCCTGACCCCCTTGTTGGTTTGCGTGGCCGCATTG1097
ArgGlyValIleSerLeuThrProLeuLeuValCysValAlaAlaLeu
200205210
CTGCTGCTTGAGCAACGGATTGTCATGGTCTTCCTGTTGGTGACGATG1145
LeuLeuLeuGluGlnArgIleValMetValPheLeuLeuValThrMet
215220225
GCCGGGATGTCGCAAGGCGCCCCTGCCTCCGTTTTGGGGTCACGCCCC1193
AlaGlyMetSerGlnGlyAlaProAlaSerValLeuGlySerArgPro
230235240245
TTTGACTACGGGTTGACTTGGCAGACCTGCTCTTGCAGGGCCAACGGT1241
PheAspTyrGlyLeuThrTrpGlnThrCysSerCysArgAlaAsnGly
250255260
TCGCGTTTTTCGACTGGGGAGAAGGTGTGGGACCGTGGGAACGTTACG1289
SerArgPheSerThrGlyGluLysValTrpAspArgGlyAsnValThr
265270275
CTTCAGTGTGACTGCCCTAACGGCCCCTGGGTGTGGTTGCCAGCCTTT1337
LeuGlnCysAspCysProAsnGlyProTrpValTrpLeuProAlaPhe
280285290
TGCCAAGCAATCGGCTGGGGTGACCCCATCACTTATTGGAGCCACGGG1385
CysGlnAlaIleGlyTrpGlyAspProIleThrTyrTrpSerHisGly
295300305
CAAAATCAGTGGCCCCTTTCATGCCCCCAGTATGTCTATGGGTCTGCT1433
GlnAsnGlnTrpProLeuSerCysProGlnTyrValTyrGlySerAla
310315320325
ACAGTCACTTGCGTGTGGGGTTCCGCTTCTTGGTTTGCCTCCACCAGT1481
ThrValThrCysValTrpGlySerAlaSerTrpPheAlaSerThrSer
330335340
GGTCGCGACTCGAAGATAGATGTGTGGAGTTTAGTGCCAGTTGGCTCT1529
GlyArgAspSerLysIleAspValTrpSerLeuValProValGlySer
345350355
GCCACCTGCACCATAGCCGCACTTGGATCATCGGATCGCGACACGGTG1577
AlaThrCysThrIleAlaAlaLeuGlySerSerAspArgAspThrVal
360365370
CCTGGGCTCTCCGAGTGGGGAATCCCGTGCGTGACGTGTGTTCTGGAC1625
ProGlyLeuSerGluTrpGlyIleProCysValThrCysValLeuAsp
375380385
CGTCGGCCTGCCTCCTGCGGCACCTGTGTGAGGGACTGCTGGCCCGAG1673
ArgArgProAlaSerCysGlyThrCysValArgAspCysTrpProGlu
390395400405
ACCGGGTCGGTTAGGTTCCCATTCCATCGGTGCGGCGTGGGGCCTCGG1721
ThrGlySerValArgPheProPheHisArgCysGlyValGlyProArg
410415420
CTGACAAAGGACTTGGAAGCTGTGCCCTTCGTCAACAGGACAACTCCC1769
LeuThrLysAspLeuGluAlaValProPheValAsnArgThrThrPro
425430435
TTCACCATTAGGGGGCCCCTGGGCAACCAGGGCCGAGGCAACCCGGTG1817
PheThrIleArgGlyProLeuGlyAsnGlnGlyArgGlyAsnProVal
440445450
CGGTCGCCCTTGGGTTTTGGGTCCTACGCCATGACCAGGATCCGAGAT1865
ArgSerProLeuGlyPheGlySerTyrAlaMetThrArgIleArgAsp
455460465
ACCCTACATCTGGTGGAGTGTCCCACACCAGCCATTGAGCCTCCCACC1913
ThrLeuHisLeuValGluCysProThrProAlaIleGluProProThr
470475480485
GGGACGTTTGGGTTCTTCCCCGGGACGCCGCCTCTCAACAACTGCATG1961
GlyThrPheGlyPhePheProGlyThrProProLeuAsnAsnCysMet
490495500
CTCTTGGGCACGGAAGTGTCCGAGGCACTTGGGGGGGCTGGCCTCACG2009
LeuLeuGlyThrGluValSerGluAlaLeuGlyGlyAlaGlyLeuThr
505510515
GGGGGGTTCTATGAACCCCTGGTGCGCAGGTGTTCGAAGCTGATGGGA2057
GlyGlyPheTyrGluProLeuValArgArgCysSerLysLeuMetGly
520525530
AGCCGAAATCCGGTTTGTCCGGGGTTTGCATGGCTCTCTTCGGGCAGG2105
SerArgAsnProValCysProGlyPheAlaTrpLeuSerSerGlyArg
535540545
CCTGATGGGTTTATACATGTCCAGGGTCACTTGCAGGAGGTGGATGCA2153
ProAspGlyPheIleHisValGlnGlyHisLeuGlnGluValAspAla
550555560565
GGCAACTTCATCCCGCCCCCGCGCTGGTTGCTCTTGGACTTTGTATTT2201
GlyAsnPheIleProProProArgTrpLeuLeuLeuAspPheValPhe
570575580
GTCCTGTTATACCTGATGAAGCTGGCTGAGGCACGGTTGGTCCCGCTG2249
ValLeuLeuTyrLeuMetLysLeuAlaGluAlaArgLeuValProLeu
585590595
ATCTTGCTGCTGCTATGGTGGTGGGTGAACCAGCTGGCAGTCCTAGGG2297
IleLeuLeuLeuLeuTrpTrpTrpValAsnGlnLeuAlaValLeuGly
600605610
CTGCCGGCTGTGGAAGCCGCCGTGGCAGGTGAGGTCTTCGCGGGCCCT2345
LeuProAlaValGluAlaAlaValAlaGlyGluValPheAlaGlyPro
615620625
GCCCTGTCCTGGTGTCTGGGACTCCCGGTCGTCAGTATGATATTGGGT2393
AlaLeuSerTrpCysLeuGlyLeuProValValSerMetIleLeuGly
630635640645
TTGGCAAACCTGGTGCTGTACTTTAGATGGTTGGGACCCCAACGCCTG2441
LeuAlaAsnLeuValLeuTyrPheArgTrpLeuGlyProGlnArgLeu
650655660
ATGTTCCTCGTGTTGTGGAAGCTTGCTCGGGGAGCTTTCCCGCTGGCC2489
MetPheLeuValLeuTrpLysLeuAlaArgGlyAlaPheProLeuAla
665670675
CTCTTGATGGGGATTTCGGCGACCCGCGGGCGCACCTCAGTGCTCGGG2537
LeuLeuMetGlyIleSerAlaThrArgGlyArgThrSerValLeuGly
680685690
GCCGAGTTCTGCTTCGATGCTACATTCGAGGTGGACACTTCGGTGTTG2585
AlaGluPheCysPheAspAlaThrPheGluValAspThrSerValLeu
695700705
GGCTGGGTGGTGGCCAGTGTGGTAGCTTGGGCCATTGCGCTCCTGAGC2633
GlyTrpValValAlaSerValValAlaTrpAlaIleAlaLeuLeuSer
710715720725
TCGATGAGCGCAGGGGGGTGGAGGCACAAAGCCGTGATCTATAGGACG2681
SerMetSerAlaGlyGlyTrpArgHisLysAlaValIleTyrArgThr
730735740
TGGTGTAAGGGGTACCAGGCAATCCGTCAAAGGGTGGTGAGGAGCCCC2729
TrpCysLysGlyTyrGlnAlaIleArgGlnArgValValArgSerPro
745750755
CTCGGGGAGGGGCGGCCTGCCAAACCCCTGACCTTTGCCTGGTGCTTG2777
LeuGlyGluGlyArgProAlaLysProLeuThrPheAlaTrpCysLeu
760765770
GCCTCGTACATCTGGCCAGATGCTGTGATGATGGTGGTGGTTGCCTTG2825
AlaSerTyrIleTrpProAspAlaValMetMetValValValAlaLeu
775780785
GTCCTTCTCTTTGGCCTGTTCGACGCGTTGGATTGGGCCTTGGAGGAG2873
ValLeuLeuPheGlyLeuPheAspAlaLeuAspTrpAlaLeuGluGlu
790795800805
ATCTTGGTGTCCCGGCCCTCGTTGCGGCGTTTGGCTCGGGTGGTTGAG2921
IleLeuValSerArgProSerLeuArgArgLeuAlaArgValValGlu
810815820
TGCTGTGTGATGGCGGGTGAGAAGGCCACAACCGTCCGGCTGGTCTCC2969
CysCysValMetAlaGlyGluLysAlaThrThrValArgLeuValSer
825830835
AAGATGTGTGCGAGAGGAGCTTATTTGTTCGATCATATGGGCTCTTTT3017
LysMetCysAlaArgGlyAlaTyrLeuPheAspHisMetGlySerPhe
840845850
TCGCGTGCTGTCAAGGAGCGCCTGTTGGAATGGGACGCAGCTCTTGAA3065
SerArgAlaValLysGluArgLeuLeuGluTrpAspAlaAlaLeuGlu
855860865
CCTCTGTCATTCACTAGGACGGACTGTCGCATCATACGGGATGCCGCG3113
ProLeuSerPheThrArgThrAspCysArgIleIleArgAspAlaAla
870875880885
AGGACTTTGTCCTGCGGGCAGTGCGTCATGGGTTTACCCGTGGTTGCG3161
ArgThrLeuSerCysGlyGlnCysValMetGlyLeuProValValAla
890895900
CGCCGTGGTGATGAGGTTCTCATCGGCGTCTTCCAGGATGTGAATCAT3209
ArgArgGlyAspGluValLeuIleGlyValPheGlnAspValAsnHis
905910915
TTGCCTCCCGGGTTTGTTCCGACCGCGCCTGTTGTCATCCGACGGTGC3257
LeuProProGlyPheValProThrAlaProValValIleArgArgCys
920925930
GGAAAGGGCTTCTTGGGGGTCACAAAGGCTGCCTTGACAGGTCGGGAT3305
GlyLysGlyPheLeuGlyValThrLysAlaAlaLeuThrGlyArgAsp
935940945
CCTGACTTACATCCAGGGAACGTCATGGTGTTGGGGACGGCTACGTCG3353
ProAspLeuHisProGlyAsnValMetValLeuGlyThrAlaThrSer
950955960965
CGAAGCATGGGAACATGCTTGAACGGCCTGCTGTTCACGACCTTCCAT3401
ArgSerMetGlyThrCysLeuAsnGlyLeuLeuPheThrThrPheHis
970975980
GGGGCTTCATCCCGAACCATCGCCACACCCGTGGGGGCCCTTAATCCC3449
GlyAlaSerSerArgThrIleAlaThrProValGlyAlaLeuAsnPro
985990995
AGATGGTGGTCAGCCAGTGATGATGTCACGGTGTATCCACTCCCGGAT3497
ArgTrpTrpSerAlaSerAspAspValThrValTyrProLeuProAsp
100010051010
GGGGCTACTTCGTTAACACCTTGTACTTGCCAGGCTGAGTCCTGTTGG3545
GlyAlaThrSerLeuThrProCysThrCysGlnAlaGluSerCysTrp
101510201025
GTCATCAGATCCGACGGGGCCCTATGCCATGGCTTGAGCAAGGGGGAC3593
ValIleArgSerAspGlyAlaLeuCysHisGlyLeuSerLysGlyAsp
1030103510401045
AAGGTGGAGCTGGATGTGGCCATGGAGGTCTCTGACTTCCGTGGCTCG3641
LysValGluLeuAspValAlaMetGluValSerAspPheArgGlySer
105010551060
TCTGGCTCACCGGTCCTATGTGACGAAGGGCACGCAGTAGGAATGCTC3689
SerGlySerProValLeuCysAspGluGlyHisAlaValGlyMetLeu
106510701075
GTGTCTGTGCTTCACTCCGGTGGTAGGGTCACCGCGGCACGGTTCACT3737
ValSerValLeuHisSerGlyGlyArgValThrAlaAlaArgPheThr
108010851090
AGGCCGTGGACCCAAGTGCCAACAGATGCCAAAACCACTACTGAACCC3785
ArgProTrpThrGlnValProThrAspAlaLysThrThrThrGluPro
109511001105
CCTCCGGTGCCGGCCAAAGGAGTTTTCAAAGAGGCCCCGTTGTTTATG3833
ProProValProAlaLysGlyValPheLysGluAlaProLeuPheMet
1110111511201125
CCTACGGGAGCGGGAAAGAGCACTCGCGTCCCGTTGGAGTACGATAAC3881
ProThrGlyAlaGlyLysSerThrArgValProLeuGluTyrAspAsn
113011351140
ATGGGGCACAAGGTCTTAATCTTGAACCCCTCAGTGGCCACTGTGCGG3929
MetGlyHisLysValLeuIleLeuAsnProSerValAlaThrValArg
114511501155
GCCATGGGCCCGTACATGGAGCGGCTGGCGGGTAAACATCCAAGTATA3977
AlaMetGlyProTyrMetGluArgLeuAlaGlyLysHisProSerIle
116011651170
TACTGTGGGCATGATACAACTGCTTTCACAAGGATCACTGACTCCCCC4025
TyrCysGlyHisAspThrThrAlaPheThrArgIleThrAspSerPro
117511801185
CTGACGTATTCAACCTATGGGAGGTTTTTGGCCAACCCTAGGCAGATG4073
LeuThrTyrSerThrTyrGlyArgPheLeuAlaAsnProArgGlnMet
1190119512001205
CTACGGGGCGTTTCGGTGGTCATTTGTGATGAGTGCCACAGTCATGAC4121
LeuArgGlyValSerValValIleCysAspGluCysHisSerHisAsp
121012151220
TCAACCGTGCTGTTAGGCATTGGGAGAGTCCGGGAGCTGGCGCGTGGG4169
SerThrValLeuLeuGlyIleGlyArgValArgGluLeuAlaArgGly
122512301235
TGCGGGGTGCAACTAGTGCTCTACGCCACCGCTACACCTCCCGGATCC4217
CysGlyValGlnLeuValLeuTyrAlaThrAlaThrProProGlySer
124012451250
CCTATGACGCAGCACCCTTCCATAATTGAGACAAAATTGGACGTGGGC4265
ProMetThrGlnHisProSerIleIleGluThrLysLeuAspValGly
125512601265
GAGATTCCCTTTTATGGGCATGGAATACCCCTCGAGCGGATGCGAACC4313
GluIleProPheTyrGlyHisGlyIleProLeuGluArgMetArgThr
1270127512801285
GGAAGGCACCTCGTGTTCTGCCATTCTAAGGCTGAGTGCGAGCGCCTT4361
GlyArgHisLeuValPheCysHisSerLysAlaGluCysGluArgLeu
129012951300
GCTGGCCAGTTCTCCGCTAGGGGGGTCAATGCCATTGCCTATTATAGG4409
AlaGlyGlnPheSerAlaArgGlyValAsnAlaIleAlaTyrTyrArg
130513101315
GGTAAAGACAGTTCTATCATCAAGGATGGGGACCTGGTGGTCTGTGCT4457
GlyLysAspSerSerIleIleLysAspGlyAspLeuValValCysAla
132013251330
ACAGACGCGCTTTCCACTGGGTACACTGGAAATTTCGACTCCGTCACC4505
ThrAspAlaLeuSerThrGlyTyrThrGlyAsnPheAspSerValThr
133513401345
GACTGTGGATTAGTGGTGGAGGAGGTCGTTGAGGTGACCCTTGATCCC4553
AspCysGlyLeuValValGluGluValValGluValThrLeuAspPro
1350135513601365
ACCATTACCATCTCCCTGCGGACAGTGCCTGCGTCGGCTGAACTGTCG4601
ThrIleThrIleSerLeuArgThrValProAlaSerAlaGluLeuSer
137013751380
ATGCAAAGACGAGGACGCACGGGTAGGGGCAGGTCTGGACGCTACTAC4649
MetGlnArgArgGlyArgThrGlyArgGlyArgSerGlyArgTyrTyr
138513901395
TACGCGGGGGTGGGCAAAGCCCCTGCGGGTGTGGTGCGCTCAGGTCCT4697
TyrAlaGlyValGlyLysAlaProAlaGlyValValArgSerGlyPro
140014051410
GTCTGGTCGGCGGTGGAAGCTGGAGTGACCTGGTACGGAATGGAACCT4745
ValTrpSerAlaValGluAlaGlyValThrTrpTyrGlyMetGluPro
141514201425
GACTTGACAGCTAACCTACTGAGACTTTACGACGACTGCCCTTACACC4793
AspLeuThrAlaAsnLeuLeuArgLeuTyrAspAspCysProTyrThr
1430143514401445
GCAGCCGTCGCGGCTGATATCGGAGAAGCCGCGGTGTTCTTCTCTGGG4841
AlaAlaValAlaAlaAspIleGlyGluAlaAlaValPhePheSerGly
145014551460
CTCGCCCCATTGAGGATGCACCCTGATGTCAGCTGGGCAAAAGTTCGC4889
LeuAlaProLeuArgMetHisProAspValSerTrpAlaLysValArg
146514701475
GGCGTCAACTGGCCCCTCTTGGTGGGTGTTCAGCGGACCATGTGTCGG4937
GlyValAsnTrpProLeuLeuValGlyValGlnArgThrMetCysArg
148014851490
GAAACACTGTCTCCCGGCCCATCGGATGACCCCCAATGGGCAGGTCTG4985
GluThrLeuSerProGlyProSerAspAspProGlnTrpAlaGlyLeu
149515001505
AAGGGCCCAAATCCTGTCCCACTCCTGCTGAGGTGGGGCAATGATTTA5033
LysGlyProAsnProValProLeuLeuLeuArgTrpGlyAsnAspLeu
1510151515201525
CCATCTAAAGTGGCCGGCCACCACATAGTGGACGACCTGGTCCGGAGA5081
ProSerLysValAlaGlyHisHisIleValAspAspLeuValArgArg
153015351540
CTCGGTGTGGCGGAGGGTTACGTCCGCTGCGACGCTGGGCCGATCTTG5129
LeuGlyValAlaGluGlyTyrValArgCysAspAlaGlyProIleLeu
154515501555
ATGATCGGTCTAGCTATCGCGGGGGGAATGATCTACGCGTCATACACC5177
MetIleGlyLeuAlaIleAlaGlyGlyMetIleTyrAlaSerTyrThr
156015651570
GGGTCGCTAGTGGTGGTGACAGACTGGGATGTGAAGGGGGGTGGCGCC5225
GlySerLeuValValValThrAspTrpAspValLysGlyGlyGlyAla
157515801585
CCCCTTTATCGGCATGGAGACCAGGCCACGCCTCAGCCGGTGGTGCAG5273
ProLeuTyrArgHisGlyAspGlnAlaThrProGlnProValValGln
1590159516001605
GTTCCTCCGGTAGACCATCGGCCGGGGGGTGAATCAGCACCATCGGAT5321
ValProProValAspHisArgProGlyGlyGluSerAlaProSerAsp
161016151620
GCCAAGACAGTGACAGATGCGGTGGCAGCCATCCAGGTGGACTGCGAT5369
AlaLysThrValThrAspAlaValAlaAlaIleGlnValAspCysAsp
162516301635
TGGACTATCATGACTCTGTCGATCGGAGAAGTGTTGTCCTTGGCTCAG5417
TrpThrIleMetThrLeuSerIleGlyGluValLeuSerLeuAlaGln
164016451650
GCTAAGACGGCCGAGGCCTACACAGCAACCGCCAAGTGGCTCGCTGGC5465
AlaLysThrAlaGluAlaTyrThrAlaThrAlaLysTrpLeuAlaGly
165516601665
TGCTATACGGGGACGCGGGCCGTTCCCACTGTATCCATTGTTGACAAG5513
CysTyrThrGlyThrArgAlaValProThrValSerIleValAspLys
1670167516801685
CTCTTCGCCGGAGGGTGGGCGGCTGTGGTGGGCCATTGCCACAGCGTG5561
LeuPheAlaGlyGlyTrpAlaAlaValValGlyHisCysHisSerVal
169016951700
ATTGCTGCGGCGGTGGCGGCCTACGGGGCTTCAAGGAGCCCGCCGTTG5609
IleAlaAlaAlaValAlaAlaTyrGlyAlaSerArgSerProProLeu
170517101715
GCAGCCGCGGCTTCCTACCTGATGGGGTTGGGCGTTGGAGGCAACGCT5657
AlaAlaAlaAlaSerTyrLeuMetGlyLeuGlyValGlyGlyAsnAla
172017251730
CAGACGCGCCTGGCGTCTGCCCTCCTATTGGGGGCTGCTGGAACCGCC5705
GlnThrArgLeuAlaSerAlaLeuLeuLeuGlyAlaAlaGlyThrAla
173517401745
TTGGGCACTCCTGTCGTGGGCTTGACCATGGCAGGTGCGTTCATGGGG5753
LeuGlyThrProValValGlyLeuThrMetAlaGlyAlaPheMetGly
1750175517601765
GGGGCCAGTGTCTCCCCCTCCTTGGTCACCATTTTATTGGGGGCCGTC5801
GlyAlaSerValSerProSerLeuValThrIleLeuLeuGlyAlaVal
177017751780
GGAGGTTGGGAGGGTGTTGTCAACGCGGCGAGCCTAGTCTTTGACTTC5849
GlyGlyTrpGluGlyValValAsnAlaAlaSerLeuValPheAspPhe
178517901795
ATGGCGGGGAAACTTTCATCAGAAGATCTGTGGTATGCCATCCCGGTA5897
MetAlaGlyLysLeuSerSerGluAspLeuTrpTyrAlaIleProVal
180018051810
CTGACCAGCCCGGGGGCGGGCCTTGCGGGGATCGCTCTCGGGTTGGTT5945
LeuThrSerProGlyAlaGlyLeuAlaGlyIleAlaLeuGlyLeuVal
181518201825
TTGTATTCAGCTAACAACTCTGGCACTACCACTTGGTTGAACCGTCTG5993
LeuTyrSerAlaAsnAsnSerGlyThrThrThrTrpLeuAsnArgLeu
1830183518401845
CTGACTACGTTACCAAGGTCTTCATGTATCCCGGACAGTTACTTTCAG6041
LeuThrThrLeuProArgSerSerCysIleProAspSerTyrPheGln
185018551860
CAAGTTGACTATTGCGACAAGGTCTCAGCCGTGCTCCGGCGCCTGAGC6089
GlnValAspTyrCysAspLysValSerAlaValLeuArgArgLeuSer
186518701875
CTCACCCGCACAGTGGTTGCCCTGGTCAACAGGGAGCCTAAGGTGGAT6137
LeuThrArgThrValValAlaLeuValAsnArgGluProLysValAsp
188018851890
GAGGTACAGGTGGGGTATGTCTGGGACCTGTGGGAGTGGATCATGCGC6185
GluValGlnValGlyTyrValTrpAspLeuTrpGluTrpIleMetArg
189519001905
CAAGTGCGCGTGGTCATGGCCAGACTCAGGGCCCTCTGCCCCGTGGTG6233
GlnValArgValValMetAlaArgLeuArgAlaLeuCysProValVal
1910191519201925
TCACTACCCTTGTGGCATTGCGGGGAGGGGTGGTCCGGGGAATGGTTG6281
SerLeuProLeuTrpHisCysGlyGluGlyTrpSerGlyGluTrpLeu
193019351940
CTTGACGGTCATGTTGAGAGTCGCTGCCTCTGTGGCTGCGTGATCACT6329
LeuAspGlyHisValGluSerArgCysLeuCysGlyCysValIleThr
194519501955
GGTGACGTTCTGAATGGGCAACTCAAAGAACCAGTTTACTCTACCAAG6377
GlyAspValLeuAsnGlyGlnLeuLysGluProValTyrSerThrLys
196019651970
CTGTGCCGGCACTATTGGATGGGGACTGTCCCTGTGAACATGCTGGGT6425
LeuCysArgHisTyrTrpMetGlyThrValProValAsnMetLeuGly
197519801985
TACGGTGAAACGTCGCCTCTCCTGGCCTCCGACACCCCGAAGGTTGTG6473
TyrGlyGluThrSerProLeuLeuAlaSerAspThrProLysValVal
1990199520002005
CCCTTCGGGACGTCTGGCTGGGCTGAGGTGGTGGTGACCACTACCCAC6521
ProPheGlyThrSerGlyTrpAlaGluValValValThrThrThrHis
201020152020
GTGGTAATCAGGAGGACCTCCGCCTATAAGCTGCTGCGCCAGCAAATC6569
ValValIleArgArgThrSerAlaTyrLysLeuLeuArgGlnGlnIle
202520302035
CTATCGGCTGCTGTAGCTGAGCCCTACTACGTCGACGGCATTCCGGTC6617
LeuSerAlaAlaValAlaGluProTyrTyrValAspGlyIleProVal
204020452050
TCATGGGACGCGGACGCTCGTGCGCCCGCCATGGTCTATGGCCCTGGG6665
SerTrpAspAlaAspAlaArgAlaProAlaMetValTyrGlyProGly
205520602065
CAAAGTGTTACCATTGACGGGGAGCGCTACACCTTGCCTCATCAACTG6713
GlnSerValThrIleAspGlyGluArgTyrThrLeuProHisGlnLeu
2070207520802085
AGGCTCAGGAATGTGGCACCCTCTGAGGTTTCATCCGAGGTGTCCATT6761
ArgLeuArgAsnValAlaProSerGluValSerSerGluValSerIle
209020952100
GACATTGGGACGGAGACTGAAGACTCAGAACTGACTGAGGCCGATCTG6809
AspIleGlyThrGluThrGluAspSerGluLeuThrGluAlaAspLeu
210521102115
CCGCCGGCGGCTGCTGCTCTCCAAGCGATCGAGAATGCTGCGAGGATT6857
ProProAlaAlaAlaAlaLeuGlnAlaIleGluAsnAlaAlaArgIle
212021252130
CTTGAACCGCACATTGATGTCATCATGGAGGACTGCAGTACACCCTCT6905
LeuGluProHisIleAspValIleMetGluAspCysSerThrProSer
213521402145
CTTTGTGGTAGTAGCCGAGAGATGCCTGTATGGGGAGAAGACATCCCC6953
LeuCysGlySerSerArgGluMetProValTrpGlyGluAspIlePro
2150215521602165
CGTACTCCATCGCCAGCACTTATCTCGGTTACTGAGAGCAGCTCAGAT7001
ArgThrProSerProAlaLeuIleSerValThrGluSerSerSerAsp
217021752180
GAGAAGACCCCGTCGGTGTCCTCCTCGCAGGAGGATACCCCGTCCTCT7049
GluLysThrProSerValSerSerSerGlnGluAspThrProSerSer
218521902195
GACTCATTCGAGGTCATCCAAGAGTCCGAGACAGCCGAAGGGGAGGAA7097
AspSerPheGluValIleGlnGluSerGluThrAlaGluGlyGluGlu
220022052210
AGTGTCTTCAACGTGGCTCTTTCCGTATTAAAAGCCTTATTTCCACAG7145
SerValPheAsnValAlaLeuSerValLeuLysAlaLeuPheProGln
221522202225
AGCGACGCGACCAGGAAGCTTACCGTCAAGATGTCGTGCTGCGTTGAA7193
SerAspAlaThrArgLysLeuThrValLysMetSerCysCysValGlu
2230223522402245
AAGAGCGTCACGCGCTTTTTCTCATTGGGGTTGACGGTGGCTGATGTT7241
LysSerValThrArgPhePheSerLeuGlyLeuThrValAlaAspVal
225022552260
GCTAGCCTGTGTGAGATGGAAATCCAGAACCATACAGCCTATTGTGAC7289
AlaSerLeuCysGluMetGluIleGlnAsnHisThrAlaTyrCysAsp
226522702275
CAGGTGCGCACTCCGCTTGAATTGCAGGTTGGGTGCTTGGTGGGCAAT7337
GlnValArgThrProLeuGluLeuGlnValGlyCysLeuValGlyAsn
228022852290
GAACTTACCTTTGAATGTGACAAGTGTGAGGCTAGGCAAGAAACCTTG7385
GluLeuThrPheGluCysAspLysCysGluAlaArgGlnGluThrLeu
229523002305
GCCTCCTTCTCTTACATTTGGTCTGGAGTGCCGCTGACTAGGGCCACG7433
AlaSerPheSerTyrIleTrpSerGlyValProLeuThrArgAlaThr
2310231523202325
CCGGCCAAGCCTCCCGTGGTGAGGCCGGTTGGCTCTTTGTTAGTGGCC7481
ProAlaLysProProValValArgProValGlySerLeuLeuValAla
233023352340
GACACTACTAAGGTGTATGTTACCAATCCAGACAATGTGGGACGGAGG7529
AspThrThrLysValTyrValThrAsnProAspAsnValGlyArgArg
234523502355
GTGGACAAGGTGACCTTCTGGCGTGCTCCTAGGGTTCATGATAAGTAC7577
ValAspLysValThrPheTrpArgAlaProArgValHisAspLysTyr
236023652370
CTCGTGGACTCTATTGAGCGCGCTAAGAGGGCCGCTCAAGCCTGCCTA7625
LeuValAspSerIleGluArgAlaLysArgAlaAlaGlnAlaCysLeu
237523802385
AGCATGGGTTACACTTATGAGGAAGCAATAAGGACTGTAAGGCCACAT7673
SerMetGlyTyrThrTyrGluGluAlaIleArgThrValArgProHis
2390239524002405
GCTGCCATGGGCTGGGGATCTAAGGTGTCGGTTAAGGACTTAGCCACC7721
AlaAlaMetGlyTrpGlySerLysValSerValLysAspLeuAlaThr
241024152420
CCCGCGGGGAAGATGGCCGTCCATGACCGGCTTCAGGAGATACTTGAA7769
ProAlaGlyLysMetAlaValHisAspArgLeuGlnGluIleLeuGlu
242524302435
GGGACTCCGGTCCCCTTTACTCTTACTGTGAAAAAGGAGGTGTTCTTC7817
GlyThrProValProPheThrLeuThrValLysLysGluValPhePhe
244024452450
AAAGACCGGAAGGAGGAGAAGGCCCCCCGCCTCATTGTGTTCCCCCCC7865
LysAspArgLysGluGluLysAlaProArgLeuIleValPheProPro
245524602465
CTGGACTTCCGGATAGCTGAAAAGCTCATCTTGGGAGACCCAGGCCGG7913
LeuAspPheArgIleAlaGluLysLeuIleLeuGlyAspProGlyArg
2470247524802485
GTAGCCAAGGCGGTGTTGGGGGGGGCCTACGCCTTCCAGTACACCCCA7961
ValAlaLysAlaValLeuGlyGlyAlaTyrAlaPheGlnTyrThrPro
249024952500
AATCAGCGAGTTAAGGAGATGCTCAAGCTATGGGAGTCTAAGAAGACC8009
AsnGlnArgValLysGluMetLeuLysLeuTrpGluSerLysLysThr
250525102515
CCTTGCGCCATCTGTGTGGACGCCACCTGCTTCGACAGTAGCATAACT8057
ProCysAlaIleCysValAspAlaThrCysPheAspSerSerIleThr
252025252530
GAAGAGGACGTGGCTTTGGAGACAGAGCTATACGCTCTGGCCTCTGAC8105
GluGluAspValAlaLeuGluThrGluLeuTyrAlaLeuAlaSerAsp
253525402545
CATCCAGAATGGGTGCGGGCACTTGGGAAATACTATGCCTCAGGCACC8153
HisProGluTrpValArgAlaLeuGlyLysTyrTyrAlaSerGlyThr
2550255525602565
ATGGTCACCCCGGAAGGGGTGCCCGTCGGTGAGAGGTATTGCAGATCC8201
MetValThrProGluGlyValProValGlyGluArgTyrCysArgSer
257025752580
TCGGGTGTCCTAACAACTAGCGCGAGCAACTGCTTGACCTGCTACATC8249
SerGlyValLeuThrThrSerAlaSerAsnCysLeuThrCysTyrIle
258525902595
AAGGTGAAAGCTGCCTGTGAGAGAGTGGGGCTGAAAAATGTCTCTCTT8297
LysValLysAlaAlaCysGluArgValGlyLeuLysAsnValSerLeu
260026052610
CTCATAGCCGGCGATGACTGCTTGATCATATGTGAGCGGCCAGTGTGC8345
LeuIleAlaGlyAspAspCysLeuIleIleCysGluArgProValCys
261526202625
GACCCAAGCGACGCTTTGGGCAGAGCCCTAGCGAGCTATGGGTACGCG8393
AspProSerAspAlaLeuGlyArgAlaLeuAlaSerTyrGlyTyrAla
2630263526402645
TGCGAGCCCTCATATCATGCATCATTGGACACGGCCCCCTTCTGCTCC8441
CysGluProSerTyrHisAlaSerLeuAspThrAlaProPheCysSer
265026552660
ACTTGGCTTGCTGAGTGCAATGCAGATGGGAAGCGCCATTTCTTCCTG8489
ThrTrpLeuAlaGluCysAsnAlaAspGlyLysArgHisPhePheLeu
266526702675
ACCACGGACTTCCGGAGGCCGCTCGCTCGCATGTCGAGTGAGTATAGT8537
ThrThrAspPheArgArgProLeuAlaArgMetSerSerGluTyrSer
268026852690
GACCCGATGGCTTCGGCGATCGGTTACATCCTCCTTTATCCTTGGCAC8585
AspProMetAlaSerAlaIleGlyTyrIleLeuLeuTyrProTrpHis
269527002705
CCCATCACACGGTGGGTCATCATCCCTCATGTGCTAACGTGCGCATTC8633
ProIleThrArgTrpValIleIleProHisValLeuThrCysAlaPhe
2710271527202725
AGGGGTGGAGGCACACCGTCTGATCCGGTTTGGTGCCAGGTGCATGGT8681
ArgGlyGlyGlyThrProSerAspProValTrpCysGlnValHisGly
273027352740
AACTACTACAAGTTTCCACTGGACAAACTGCCTAACATCATCGTGGCC8729
AsnTyrTyrLysPheProLeuAspLysLeuProAsnIleIleValAla
274527502755
CTCCACGGACCAGCAGCGTTGAGGGTTACCGCAGACACAACTAAAACA8777
LeuHisGlyProAlaAlaLeuArgValThrAlaAspThrThrLysThr
276027652770
AAGATGGAGGCTGGTAAGGTTCTGAGCGACCTCAAGCTCCCTGGCTTA8825
LysMetGluAlaGlyLysValLeuSerAspLeuLysLeuProGlyLeu
277527802785
GCAGTCCACCGAAAGAAGGCCGGGGCGTTGCGAACACGCATGCTCCGC8873
AlaValHisArgLysLysAlaGlyAlaLeuArgThrArgMetLeuArg
2790279528002805
TCGCGCGGTTGGGCTGAGTTGGCTAGGGGCTTGTTGTGGCATCCAGGC8921
SerArgGlyTrpAlaGluLeuAlaArgGlyLeuLeuTrpHisProGly
281028152820
CTACGGCTTCCTCCCCCTGAGATTGCTGGTATCCCGGGGGGTTTCCCT8969
LeuArgLeuProProProGluIleAlaGlyIleProGlyGlyPhePro
282528302835
CTCTCCCCCCCCTATATGGGGGTGGTACATCAATTGGATTTCACAAGC9017
LeuSerProProTyrMetGlyValValHisGlnLeuAspPheThrSer
284028452850
CAGAGGAGTCGCTGGCGGTGGTTGGGGTTCTTAGCCCTGCTCATCGTA9065
GlnArgSerArgTrpArgTrpLeuGlyPheLeuAlaLeuLeuIleVal
285528602865
GCCCTCTTCGGGTGAACTAAATTCATCTGTTGCGGCAAGGTCTGGTGACTGA9117
AlaLeuPheGly
2870
TCATCACCGGAGGAGGTTCCCGCCCTCCCCGCCCCAGGGGTCTCCCCGCTGGGTAAAAAG9177
GGCCCGGCCTTGGGAGGCATGGTGGTTACTAACCCCCTGGCAGGGTCAAAGCCTGATGGT9237
GCTAATGCACTGCCACTTCGGTGGCGGGTCGCTACCTTATAGCGTAATCCGTGACTACGG9297
GCTGCTCGCAGAGCCCTCCCCGGATGGGGCACAGTGCACTGTGATCTGAAGGGGTGCACC9357
CCGGGAAGAGCTCGGCCCGAAGGCCGGSTTCTACT9392
(2) INFORMATION FOR SEQ ID NO:15:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 2873 amino acids
(B) TYPE: amino acid
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: protein
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:15:
MetGlyProProSerSerAlaAlaAlaCysSerArgGlySerProArg
151015
IleLeuArgValArgAlaGlyGlyIleSerPhePheTyrThrIleMet
202530
AlaValLeuLeuLeuLeuLeuValValGluAlaGlyAlaIleLeuAla
354045
ProAlaThrHisAlaCysArgAlaAsnGlyGlnTyrPheLeuThrAsn
505560
CysCysAlaProGluAspIleGlyPheCysLeuGluGlyGlyCysLeu
65707580
ValAlaLeuGlyCysThrIleCysThrAspGlnCysTrpProLeuTyr
859095
GlnAlaGlyLeuAlaValArgProGlyLysSerAlaAlaGlnLeuVal
100105110
GlyGluLeuGlySerLeuTyrGlyProLeuSerValSerAlaTyrVal
115120125
AlaGlyIleLeuGlyLeuGlyGluValTyrSerGlyValLeuThrVal
130135140
GlyValAlaLeuThrArgArgValTyrProValProAsnLeuThrCys
145150155160
AlaValAlaCysGluLeuLysTrpGluSerGluPheTrpArgTrpThr
165170175
GluGlnLeuAlaSerAsnTyrTrpIleLeuGluTyrLeuTrpLysVal
180185190
ProPheAspPheTrpArgGlyValIleSerLeuThrProLeuLeuVal
195200205
CysValAlaAlaLeuLeuLeuLeuGluGlnArgIleValMetValPhe
210215220
LeuLeuValThrMetAlaGlyMetSerGlnGlyAlaProAlaSerVal
225230235240
LeuGlySerArgProPheAspTyrGlyLeuThrTrpGlnThrCysSer
245250255
CysArgAlaAsnGlySerArgPheSerThrGlyGluLysValTrpAsp
260265270
ArgGlyAsnValThrLeuGlnCysAspCysProAsnGlyProTrpVal
275280285
TrpLeuProAlaPheCysGlnAlaIleGlyTrpGlyAspProIleThr
290295300
TyrTrpSerHisGlyGlnAsnGlnTrpProLeuSerCysProGlnTyr
305310315320
ValTyrGlySerAlaThrValThrCysValTrpGlySerAlaSerTrp
325330335
PheAlaSerThrSerGlyArgAspSerLysIleAspValTrpSerLeu
340345350
ValProValGlySerAlaThrCysThrIleAlaAlaLeuGlySerSer
355360365
AspArgAspThrValProGlyLeuSerGluTrpGlyIleProCysVal
370375380
ThrCysValLeuAspArgArgProAlaSerCysGlyThrCysValArg
385390395400
AspCysTrpProGluThrGlySerValArgPheProPheHisArgCys
405410415
GlyValGlyProArgLeuThrLysAspLeuGluAlaValProPheVal
420425430
AsnArgThrThrProPheThrIleArgGlyProLeuGlyAsnGlnGly
435440445
ArgGlyAsnProValArgSerProLeuGlyPheGlySerTyrAlaMet
450455460
ThrArgIleArgAspThrLeuHisLeuValGluCysProThrProAla
465470475480
IleGluProProThrGlyThrPheGlyPhePheProGlyThrProPro
485490495
LeuAsnAsnCysMetLeuLeuGlyThrGluValSerGluAlaLeuGly
500505510
GlyAlaGlyLeuThrGlyGlyPheTyrGluProLeuValArgArgCys
515520525
SerLysLeuMetGlySerArgAsnProValCysProGlyPheAlaTrp
530535540
LeuSerSerGlyArgProAspGlyPheIleHisValGlnGlyHisLeu
545550555560
GlnGluValAspAlaGlyAsnPheIleProProProArgTrpLeuLeu
565570575
LeuAspPheValPheValLeuLeuTyrLeuMetLysLeuAlaGluAla
580585590
ArgLeuValProLeuIleLeuLeuLeuLeuTrpTrpTrpValAsnGln
595600605
LeuAlaValLeuGlyLeuProAlaValGluAlaAlaValAlaGlyGlu
610615620
ValPheAlaGlyProAlaLeuSerTrpCysLeuGlyLeuProValVal
625630635640
SerMetIleLeuGlyLeuAlaAsnLeuValLeuTyrPheArgTrpLeu
645650655
GlyProGlnArgLeuMetPheLeuValLeuTrpLysLeuAlaArgGly
660665670
AlaPheProLeuAlaLeuLeuMetGlyIleSerAlaThrArgGlyArg
675680685
ThrSerValLeuGlyAlaGluPheCysPheAspAlaThrPheGluVal
690695700
AspThrSerValLeuGlyTrpValValAlaSerValValAlaTrpAla
705710715720
IleAlaLeuLeuSerSerMetSerAlaGlyGlyTrpArgHisLysAla
725730735
ValIleTyrArgThrTrpCysLysGlyTyrGlnAlaIleArgGlnArg
740745750
ValValArgSerProLeuGlyGluGlyArgProAlaLysProLeuThr
755760765
PheAlaTrpCysLeuAlaSerTyrIleTrpProAspAlaValMetMet
770775780
ValValValAlaLeuValLeuLeuPheGlyLeuPheAspAlaLeuAsp
785790795800
TrpAlaLeuGluGluIleLeuValSerArgProSerLeuArgArgLeu
805810815
AlaArgValValGluCysCysValMetAlaGlyGluLysAlaThrThr
820825830
ValArgLeuValSerLysMetCysAlaArgGlyAlaTyrLeuPheAsp
835840845
HisMetGlySerPheSerArgAlaValLysGluArgLeuLeuGluTrp
850855860
AspAlaAlaLeuGluProLeuSerPheThrArgThrAspCysArgIle
865870875880
IleArgAspAlaAlaArgThrLeuSerCysGlyGlnCysValMetGly
885890895
LeuProValValAlaArgArgGlyAspGluValLeuIleGlyValPhe
900905910
GlnAspValAsnHisLeuProProGlyPheValProThrAlaProVal
915920925
ValIleArgArgCysGlyLysGlyPheLeuGlyValThrLysAlaAla
930935940
LeuThrGlyArgAspProAspLeuHisProGlyAsnValMetValLeu
945950955960
GlyThrAlaThrSerArgSerMetGlyThrCysLeuAsnGlyLeuLeu
965970975
PheThrThrPheHisGlyAlaSerSerArgThrIleAlaThrProVal
980985990
GlyAlaLeuAsnProArgTrpTrpSerAlaSerAspAspValThrVal
99510001005
TyrProLeuProAspGlyAlaThrSerLeuThrProCysThrCysGln
101010151020
AlaGluSerCysTrpValIleArgSerAspGlyAlaLeuCysHisGly
1025103010351040
LeuSerLysGlyAspLysValGluLeuAspValAlaMetGluValSer
104510501055
AspPheArgGlySerSerGlySerProValLeuCysAspGluGlyHis
106010651070
AlaValGlyMetLeuValSerValLeuHisSerGlyGlyArgValThr
107510801085
AlaAlaArgPheThrArgProTrpThrGlnValProThrAspAlaLys
109010951100
ThrThrThrGluProProProValProAlaLysGlyValPheLysGlu
1105111011151120
AlaProLeuPheMetProThrGlyAlaGlyLysSerThrArgValPro
112511301135
LeuGluTyrAspAsnMetGlyHisLysValLeuIleLeuAsnProSer
114011451150
ValAlaThrValArgAlaMetGlyProTyrMetGluArgLeuAlaGly
115511601165
LysHisProSerIleTyrCysGlyHisAspThrThrAlaPheThrArg
117011751180
IleThrAspSerProLeuThrTyrSerThrTyrGlyArgPheLeuAla
1185119011951200
AsnProArgGlnMetLeuArgGlyValSerValValIleCysAspGlu
120512101215
CysHisSerHisAspSerThrValLeuLeuGlyIleGlyArgValArg
122012251230
GluLeuAlaArgGlyCysGlyValGlnLeuValLeuTyrAlaThrAla
123512401245
ThrProProGlySerProMetThrGlnHisProSerIleIleGluThr
125012551260
LysLeuAspValGlyGluIleProPheTyrGlyHisGlyIleProLeu
1265127012751280
GluArgMetArgThrGlyArgHisLeuValPheCysHisSerLysAla
128512901295
GluCysGluArgLeuAlaGlyGlnPheSerAlaArgGlyValAsnAla
130013051310
IleAlaTyrTyrArgGlyLysAspSerSerIleIleLysAspGlyAsp
131513201325
LeuValValCysAlaThrAspAlaLeuSerThrGlyTyrThrGlyAsn
133013351340
PheAspSerValThrAspCysGlyLeuValValGluGluValValGlu
1345135013551360
ValThrLeuAspProThrIleThrIleSerLeuArgThrValProAla
136513701375
SerAlaGluLeuSerMetGlnArgArgGlyArgThrGlyArgGlyArg
138013851390
SerGlyArgTyrTyrTyrAlaGlyValGlyLysAlaProAlaGlyVal
139514001405
ValArgSerGlyProValTrpSerAlaValGluAlaGlyValThrTrp
141014151420
TyrGlyMetGluProAspLeuThrAlaAsnLeuLeuArgLeuTyrAsp
1425143014351440
AspCysProTyrThrAlaAlaValAlaAlaAspIleGlyGluAlaAla
144514501455
ValPhePheSerGlyLeuAlaProLeuArgMetHisProAspValSer
146014651470
TrpAlaLysValArgGlyValAsnTrpProLeuLeuValGlyValGln
147514801485
ArgThrMetCysArgGluThrLeuSerProGlyProSerAspAspPro
149014951500
GlnTrpAlaGlyLeuLysGlyProAsnProValProLeuLeuLeuArg
1505151015151520
TrpGlyAsnAspLeuProSerLysValAlaGlyHisHisIleValAsp
152515301535
AspLeuValArgArgLeuGlyValAlaGluGlyTyrValArgCysAsp
154015451550
AlaGlyProIleLeuMetIleGlyLeuAlaIleAlaGlyGlyMetIle
155515601565
TyrAlaSerTyrThrGlySerLeuValValValThrAspTrpAspVal
157015751580
LysGlyGlyGlyAlaProLeuTyrArgHisGlyAspGlnAlaThrPro
1585159015951600
GlnProValValGlnValProProValAspHisArgProGlyGlyGlu
160516101615
SerAlaProSerAspAlaLysThrValThrAspAlaValAlaAlaIle
162016251630
GlnValAspCysAspTrpThrIleMetThrLeuSerIleGlyGluVal
163516401645
LeuSerLeuAlaGlnAlaLysThrAlaGluAlaTyrThrAlaThrAla
165016551660
LysTrpLeuAlaGlyCysTyrThrGlyThrArgAlaValProThrVal
1665167016751680
SerIleValAspLysLeuPheAlaGlyGlyTrpAlaAlaValValGly
168516901695
HisCysHisSerValIleAlaAlaAlaValAlaAlaTyrGlyAlaSer
170017051710
ArgSerProProLeuAlaAlaAlaAlaSerTyrLeuMetGlyLeuGly
171517201725
ValGlyGlyAsnAlaGlnThrArgLeuAlaSerAlaLeuLeuLeuGly
173017351740
AlaAlaGlyThrAlaLeuGlyThrProValValGlyLeuThrMetAla
1745175017551760
GlyAlaPheMetGlyGlyAlaSerValSerProSerLeuValThrIle
176517701775
LeuLeuGlyAlaValGlyGlyTrpGluGlyValValAsnAlaAlaSer
178017851790
LeuValPheAspPheMetAlaGlyLysLeuSerSerGluAspLeuTrp
179518001805
TyrAlaIleProValLeuThrSerProGlyAlaGlyLeuAlaGlyIle
181018151820
AlaLeuGlyLeuValLeuTyrSerAlaAsnAsnSerGlyThrThrThr
1825183018351840
TrpLeuAsnArgLeuLeuThrThrLeuProArgSerSerCysIlePro
184518501855
AspSerTyrPheGlnGlnValAspTyrCysAspLysValSerAlaVal
186018651870
LeuArgArgLeuSerLeuThrArgThrValValAlaLeuValAsnArg
187518801885
GluProLysValAspGluValGlnValGlyTyrValTrpAspLeuTrp
189018951900
GluTrpIleMetArgGlnValArgValValMetAlaArgLeuArgAla
1905191019151920
LeuCysProValValSerLeuProLeuTrpHisCysGlyGluGlyTrp
192519301935
SerGlyGluTrpLeuLeuAspGlyHisValGluSerArgCysLeuCys
194019451950
GlyCysValIleThrGlyAspValLeuAsnGlyGlnLeuLysGluPro
195519601965
ValTyrSerThrLysLeuCysArgHisTyrTrpMetGlyThrValPro
197019751980
ValAsnMetLeuGlyTyrGlyGluThrSerProLeuLeuAlaSerAsp
1985199019952000
ThrProLysValValProPheGlyThrSerGlyTrpAlaGluValVal
200520102015
ValThrThrThrHisValValIleArgArgThrSerAlaTyrLysLeu
202020252030
LeuArgGlnGlnIleLeuSerAlaAlaValAlaGluProTyrTyrVal
203520402045
AspGlyIleProValSerTrpAspAlaAspAlaArgAlaProAlaMet
205020552060
ValTyrGlyProGlyGlnSerValThrIleAspGlyGluArgTyrThr
2065207020752080
LeuProHisGlnLeuArgLeuArgAsnValAlaProSerGluValSer
208520902095
SerGluValSerIleAspIleGlyThrGluThrGluAspSerGluLeu
210021052110
ThrGluAlaAspLeuProProAlaAlaAlaAlaLeuGlnAlaIleGlu
211521202125
AsnAlaAlaArgIleLeuGluProHisIleAspValIleMetGluAsp
213021352140
CysSerThrProSerLeuCysGlySerSerArgGluMetProValTrp
2145215021552160
GlyGluAspIleProArgThrProSerProAlaLeuIleSerValThr
216521702175
GluSerSerSerAspGluLysThrProSerValSerSerSerGlnGlu
218021852190
AspThrProSerSerAspSerPheGluValIleGlnGluSerGluThr
219522002205
AlaGluGlyGluGluSerValPheAsnValAlaLeuSerValLeuLys
221022152220
AlaLeuPheProGlnSerAspAlaThrArgLysLeuThrValLysMet
2225223022352240
SerCysCysValGluLysSerValThrArgPhePheSerLeuGlyLeu
224522502255
ThrValAlaAspValAlaSerLeuCysGluMetGluIleGlnAsnHis
226022652270
ThrAlaTyrCysAspGlnValArgThrProLeuGluLeuGlnValGly
227522802285
CysLeuValGlyAsnGluLeuThrPheGluCysAspLysCysGluAla
229022952300
ArgGlnGluThrLeuAlaSerPheSerTyrIleTrpSerGlyValPro
2305231023152320
LeuThrArgAlaThrProAlaLysProProValValArgProValGly
232523302335
SerLeuLeuValAlaAspThrThrLysValTyrValThrAsnProAsp
234023452350
AsnValGlyArgArgValAspLysValThrPheTrpArgAlaProArg
235523602365
ValHisAspLysTyrLeuValAspSerIleGluArgAlaLysArgAla
237023752380
AlaGlnAlaCysLeuSerMetGlyTyrThrTyrGluGluAlaIleArg
2385239023952400
ThrValArgProHisAlaAlaMetGlyTrpGlySerLysValSerVal
240524102415
LysAspLeuAlaThrProAlaGlyLysMetAlaValHisAspArgLeu
242024252430
GlnGluIleLeuGluGlyThrProValProPheThrLeuThrValLys
243524402445
LysGluValPhePheLysAspArgLysGluGluLysAlaProArgLeu
245024552460
IleValPheProProLeuAspPheArgIleAlaGluLysLeuIleLeu
2465247024752480
GlyAspProGlyArgValAlaLysAlaValLeuGlyGlyAlaTyrAla
248524902495
PheGlnTyrThrProAsnGlnArgValLysGluMetLeuLysLeuTrp
250025052510
GluSerLysLysThrProCysAlaIleCysValAspAlaThrCysPhe
251525202525
AspSerSerIleThrGluGluAspValAlaLeuGluThrGluLeuTyr
253025352540
AlaLeuAlaSerAspHisProGluTrpValArgAlaLeuGlyLysTyr
2545255025552560
TyrAlaSerGlyThrMetValThrProGluGlyValProValGlyGlu
256525702575
ArgTyrCysArgSerSerGlyValLeuThrThrSerAlaSerAsnCys
258025852590
LeuThrCysTyrIleLysValLysAlaAlaCysGluArgValGlyLeu
259526002605
LysAsnValSerLeuLeuIleAlaGlyAspAspCysLeuIleIleCys
261026152620
GluArgProValCysAspProSerAspAlaLeuGlyArgAlaLeuAla
2625263026352640
SerTyrGlyTyrAlaCysGluProSerTyrHisAlaSerLeuAspThr
264526502655
AlaProPheCysSerThrTrpLeuAlaGluCysAsnAlaAspGlyLys
266026652670
ArgHisPhePheLeuThrThrAspPheArgArgProLeuAlaArgMet
267526802685
SerSerGluTyrSerAspProMetAlaSerAlaIleGlyTyrIleLeu
269026952700
LeuTyrProTrpHisProIleThrArgTrpValIleIleProHisVal
2705271027152720
LeuThrCysAlaPheArgGlyGlyGlyThrProSerAspProValTrp
272527302735
CysGlnValHisGlyAsnTyrTyrLysPheProLeuAspLysLeuPro
274027452750
AsnIleIleValAlaLeuHisGlyProAlaAlaLeuArgValThrAla
275527602765
AspThrThrLysThrLysMetGluAlaGlyLysValLeuSerAspLeu
277027752780
LysLeuProGlyLeuAlaValHisArgLysLysAlaGlyAlaLeuArg
2785279027952800
ThrArgMetLeuArgSerArgGlyTrpAlaGluLeuAlaArgGlyLeu
280528102815
LeuTrpHisProGlyLeuArgLeuProProProGluIleAlaGlyIle
282028252830
ProGlyGlyPheProLeuSerProProTyrMetGlyValValHisGln
283528402845
LeuAspPheThrSerGlnArgSerArgTrpArgTrpLeuGlyPheLeu
285028552860
AlaLeuLeuIleValAlaLeuPheGly
28652870
(2) INFORMATION FOR SEQ ID NO:16:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 27 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: PROBE 470-20- 1-152F
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:16:
TCGGTTACTGAGAGCAGCTCAGATGAG27
(2) INFORMATION FOR SEQ ID NO:17:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 20 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: JML-A, PRIMER
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:17:
AGGAATTCAGCGGCCGCGAG20
(2) INFORMATION FOR SEQ ID NO:18:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 22 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: JML-B, PRIMER
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:18:
CTCGCGGCCGCTGAATTCCTTT22
(2) INFORMATION FOR SEQ ID NO:19:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 203 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: double
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: 470-20-1 CLONE, WITHOUT SISPA
LINKERS
(ix) FEATURE:
(A) NAME/KEY: CDS
(B) LOCATION: 2..203
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:19:
GGCTGTCTCGGACTCTTGGATGACCTCGAATGAGTCAGAGGACGGG46
AlaValSerAspSerTrpMetThrSerAsnGluSerGluAspGly
151015
GTATCCTCCTGCGAGGAGGACACCGGCGGGGTCTTCTCATCTGAGCTG94
ValSerSerCysGluGluAspThrGlyGlyValPheSerSerGluLeu
202530
CTCTCAGTAACCGAGATAAGTGCTGGCGATGGAGTACGGGGGATGTCT142
LeuSerValThrGluIleSerAlaGlyAspGlyValArgGlyMetSer
354045
TCTCCCCATACAGGCATCTCTCGGCTACTACCACAAAGAGAGGGTGTA190
SerProHisThrGlyIleSerArgLeuLeuProGlnArgGluGlyVal
505560
CTGCAGTCCTCCA203
LeuGlnSerSer
65
(2) INFORMATION FOR SEQ ID NO:20:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 67 amino acids
(B) TYPE: amino acid
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: protein
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:20:
AlaValSerAspSerTrpMetThrSerAsnGluSerGluAspGlyVal
151015
SerSerCysGluGluAspThrGlyGlyValPheSerSerGluLeuLeu
202530
SerValThrGluIleSerAlaGlyAspGlyValArgGlyMetSerSer
354045
ProHisThrGlyIleSerArgLeuLeuProGlnArgGluGlyValLeu
505560
GlnSerSer
65
(2) INFORMATION FOR SEQ ID NO:21:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 27 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: 470-20-1- 152R
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:21:
CTCATCTGAGCTGCTCTCAGTAACCGA27
(2) INFORMATION FOR SEQ ID NO:22:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 24 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: OLIGONUCLEOTIDE B
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:22:
CTGTCTCGGACTCTTGGATGACCT24
(2) INFORMATION FOR SEQ ID NO:23:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 24 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: COGNATE OLIGONUCLEOTIDE 211R'
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:23:
ATACCCCGTCCTCTGACTCATTCG24
(2) INFORMATION FOR SEQ ID NO:24:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 24 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: COGNATE OLIGONUCLEOTIDE B'
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:24:
AGGTCATCCAAGAGTCCGAGACAG24
(2) INFORMATION FOR SEQ ID NO:25:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 20 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: LAMBDA GT 11 FORWARD PRIMER, 20mer
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:25:
CACATGGCTGAATATCGACG20
(2) INFORMATION FOR SEQ ID NO:26:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 180 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Consensus Sequence 4E3
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:26:
GCGAGCCTAGTCTTTGACTTCATGGCGGGGAAACTTTCATCAGAAGATCTGTGGTATGCC60
ATCCCGGTACTGACCAGCCCGGGGGCGGGCCTTGCGGGGATCGCTCTCGGGTTGGTTTTG120
TATTCAGCTAACAACTCTGGCACTACCACTTGGTTGAACCGTCTGCTGACTACGTTACCA180
(2) INFORMATION FOR SEQ ID NO:27:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 430 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Consensus Sequence 3E3
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:27:
GGCACTACCACTTGGTTGAACCGTCTGCTGACTACGTTACCAAGGTCTTCATGTATCCCG60
GACAGTTACTTTCAGCAAGTTGACTATTGCGACAAGGTCTCAGCCGTGCTCCGGCGCCTG120
AGCCTCACCCGCACAGTGGTTGCCCTGGTCAACAGGGAGCCTAAGGTGGATGAGGTACAG180
GTGGGGTATGTCTGGGACCTGTGGGAGTGGATCATGCGCCAAGTGCGCGTGGTCATGGCC240
AGACTCAGGGCCCTCTGCCCCGTGGTGTCACTACCCTTGTGGCATTGCGGGGAGGGGTGG300
TCCGGGGAATGGTTGCTTGACGGTCATGTTGAGAGTCGCTGCCTCTGTGGCTGCGTGATC360
ACTGGTGACGTTCTGAATGGGCAACTCAAAGAACCAGTTTACTCTACCAAGCTGTGCCGG420
CACTATTGGA430
(2) INFORMATION FOR SEQ ID NO:28:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 180 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Consensus Sequence 2E5
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:28:
CTTACCGTCAAGATGTCGTGCTGCGTTGAAAAGAGCGTCACGCGCTTTTTCTCATTGGGG60
TTGACGGTGGCTGATGTTGCTAGCCTGTGTGAGATGGAAATCCAGAACCATACAGCCTAT120
TGTGACCAGGTGCGCACTCCGCTTGAATTGCAGGTTGGGTGCTTGGTGGGCAATGAACTT180
(2) INFORMATION FOR SEQ ID NO:29:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 344 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Consensus Sequence 1E5
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:29:
CTTCTCTTTGTGGTAGTAGCCGAGAGATGCCTGTATGGGGAGAAGACATCCCCCGTACTC60
CATCGCCAGCACTTATCTCGGTTACTGAGAGCAGCTCAGATGAGAAGACCCCGTCGGTGT120
CCTCCTCGCAGGAGGATACCCCGTCCTCTGACTCATTCGAGGTCATCCAAGAGTCCGAGA180
CAGCCGAAGGGGAGGAAAGTGTCTTCAACGTGGCTCTTTCCGTATTAAAAGCCTTATTTC240
CACAGAGCGACGCGACCAGGAAGCTTACCGTCAAGATGTCGTGCTGCGTTGAAAAGAGCG300
TCACGCGCTTTTTCTCATTGGGGTTGACGGTGGCTGATGTTGCT344
(2) INFORMATION FOR SEQ ID NO:30:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 423 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Consensus Sequence 4E5
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:30:
GTAAGGCCACATGCTGCCATGGGCTGGGGATCTAAGGTGTCGGTTAAGGACTTAGCCACC60
CCCGCGGGGAAGATGGCCGTCCATGACCGGCTTCAGGAGATACTTGAAGGGACTCCGGTC120
CCCTTTACTCTTACTGTGAAAAAGGAGGTGTTCTTCAAAGACCGGAAGGAGGAGAAGGCC180
CCCCGCCTCATTGTGTTCCCCCCCCTGGACTTCCGGATAGCTGAAAAGCTCATCTTGGGA240
GACCCAGGCCGGGTAGCCAAGGCGGTGTTGGGGGGGGCCTACGCCTTCCAGTACACCCCA300
AATCAGCGAGTTAAGGAGATGCTCAAGCTATGGGAGTCTAAGAAGACCCCTTGCGCCATC360
TGTGTGGACGCCACCTGCTTCGACAGTAGCATAACTGAAGAGGACGTGGCTTTGGAGACA420
GAG423
(2) INFORMATION FOR SEQ ID NO:31:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 516 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Consensus Sequence 3E5
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:31:
TACAGCCTATTGTGACCAGGTGCGCACTCCGCTTGAATTGCAGGTTGGGTGCTTGGTGGG60
CAATGAACTTACCTTTGAATGTGACAAGTGTGAGGCTAGGCAAGAAACCTTGGCCTCCTT120
CTCTTACATTTGGTCTGGAGTGCCGCTGACTAGGGCCACGCCGGCCAAGCCTCCCGTGGT180
GAGGCCGGTTGGCTCTTTGTTAGTGGCCGACACTACTAAGGTGTATGTTACCAATCCAGA240
CAATGTGGGACGGAGGGTGGACAAGGTGACCTTCTGGCGTGCTCCTAGGGTTCATGATAA300
GTACCTCGTGGACTCTATTGAGCGCGCTAAGAGGGCCGCTCAAGCCTGCCTAAGCATGGG360
TTACACTTATGAGGAAGCAATAAGGACTGTAAGGCCACATGCTGCCATGGGCTGGGGATC420
TAAGGTGTCGGTTAAGGACTTAGCCACCCCCGCGGGGAAGATGGCCGTCCATGACCGGCT480
TCAGGAGATACTTGAAGGGACTCCGGTCCCCTTTAC516
(2) INFORMATION FOR SEQ ID NO:32:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 518 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Consensus Sequence 2E3
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:32:
GAATGGGCAACTCAAAGAACCAGTTTACTCTACCAAGCTGTGCCGGCACTATTGGATGGG60
GACTGTCCCTGTGAACATGCTGGGTTACGGTGAAACGTCGCCTCTCCTGGCCTCCGACAC120
CCCGAAGGTTGTGCCCTTCGGGACGTCTGGCTGGGCTGAGGTGGTGGTGACCACTACCCA180
CGTGGTAATCAGGAGGACCTCCGCCTATAAGCTGCTGCGCCAGCAAATCCTATCGGCTGC240
TGTAGCTGAGCCCTACTACGTCGACGGCATTCCGGTCTCATGGGACGCGGACGCTCGTGC300
GCCCGCCATGGTCTATGGCCCTGGGCAAAGTGTTACCATTGACGGGGAGCGCTACACCTT360
GCCTCATCAACTGAGGCTCAGGAATGTGGCACCCTCTGAGGTTTCATCCGAGGTGTCCAT420
TGACATTGGGACGGAGACTGAAGACTCAGAACTGACTGAGGCCGATCTGCCGCCGGCGGC480
TGCTGCTCTCCAAGCGATCGAGAATGCTGCGAGGATTC518
(2) INFORMATION FOR SEQ ID NO:33:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 268 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Consensus Sequence 1E3
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:33:
CTTACTGAGGCCGATCTGCCGCCGGCGGCTGCTGCTCTCCAAGCGATCGAGAATGCTGCG60
AGGATTCTTGAACCGCACATTGATGTCATCATGGAGGACTGCAGTACACCCTCTCTTTGT120
GGTAGTAGCCGAGAGATGCCTGTATGGGGAGAAGACATCCCCCGTACTCCATCGCCAGCA180
CTTATCTCGGTTACTGAGAGCAGCTCAGATGAGAAGACCCCGTCGGTGTCCTCCTCGCAG240
GAGGATACCCCGTCCTCTGACTCATTCG268
(2) INFORMATION FOR SEQ ID NO:34:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 781 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: INDIVIDUAL CLONE 4E5-20
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:34:
GTAAGGCCACATGCTGCCATGGGCTGGGGATCTAAGGTGTCGGTTAAGGACTTAGCCACC60
CCCGCGGGGAAGATGGCCGTCCATGACCGGCTTCAGGAGATACTTGAAGGGACTCCGGTC120
CCCTTTACTCTTACTGTGAAAAAGGAGGTGTTCTTCAAAGACCGGAAGGAGGAGAAGGCC180
CCCCGCCTCATTGTGTTCCCCCCCCTGGACTTCCGGATAGCTGAAAAGCTCATCTTGGGA240
GACCCAGGCCGGGTAGCCAAGGCGGTGTTGGGGGGGGCCTACGCCTTCCAGTACACCCCA300
AATCAGCGAGTTAAGGAGATGCTCAAGCTATGGGAGTCTAAGAAGACCCCTTGCGCCATC360
TGTGTGGACGCCACCTGCTTCGACAGTAGCATAACTGAAGAGGACGTGGCTTTGGAGACA420
GAGTTATACGCTCTGGCCTCTGACCATCCAGAATGGGTGCGGGCACCTGGGAAATACTAT480
GCCTCAGGCACCATGGTCACCCCGGAAGGGGTGCCCGTCGGTGAGAGGTATTGCAGATCC540
TCGGGTGTCCTAACAACTAGCGCGAGCAACTGCCTGACCTGCTACATCAAGGTGAAAGCT600
GCCTGTGAGAGAGTGGGGCTGAAAAATGTCTCTCTTCTCATAGCCGGCGATGACTGCTTG660
ATCATATGTGAGCGGCCAGTGTGCGACCCAAGCGACGCTTTGGGCAGAGCCCTAGCGAGC720
TATGGGTACGCGTGCGAGCCCTCATATCATGCATCATTGGACACGGCCCCCTTCTGCTCC780
A781
(2) INFORMATION FOR SEQ ID NO:35:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 27 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: PROBE 470- 201-1-142R
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:35:
TCGGTTACTGAGAGCAGCTCAGATGAG27
(2) INFORMATION FOR SEQ ID NO:36:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 27 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: PROBE 470-20- 1-152F
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:36:
TCGGTTACTGAGAGCAGCTCAGATGAG27
(2) INFORMATION FOR SEQ ID NO:37:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 570 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: double
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Clone 470EXP1
(ix) FEATURE:
(A) NAME/KEY: CDS
(B) LOCATION: 1..570
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:37:
GCTGTATGGTTCTGGATTTCCATCTCACACAGGCTAGCAACATCAGCC48
AlaValTrpPheTrpIleSerIleSerHisArgLeuAlaThrSerAla
151015
ACCGTCAACCCCAATGAGAAAAAGCGCGTGACGCTCTTTTCAACGCAG96
ThrValAsnProAsnGluLysLysArgValThrLeuPheSerThrGln
202530
CACGACATCTTGACGGTAAGCTTCCTGGTCGCGTCGCTCTGTGGAAAT144
HisAspIleLeuThrValSerPheLeuValAlaSerLeuCysGlyAsn
354045
AAGGCTTTTAATACGGAAAGAGCCACGTTGAAGACACTTTCCTCCCCT192
LysAlaPheAsnThrGluArgAlaThrLeuLysThrLeuSerSerPro
505560
TCGGCTGTCTCGGACTCTTGGATGACCTCGAATGAGTCAGAGGACGGG240
SerAlaValSerAspSerTrpMetThrSerAsnGluSerGluAspGly
65707580
GTATCCTCCTGCGAGGAGGACACCGACGGGGTCTTCTCATCTGAGCTG288
ValSerSerCysGluGluAspThrAspGlyValPheSerSerGluLeu
859095
CTCTCAGTAACCGAGATAAGTGCTGGCGATGGAGTACGGGGGATGTCT336
LeuSerValThrGluIleSerAlaGlyAspGlyValArgGlyMetSer
100105110
TCTCCCCATACAGGCATCTCTCGGCTACTACCACAAAGAGAGGGTGTA384
SerProHisThrGlyIleSerArgLeuLeuProGlnArgGluGlyVal
115120125
CTGCAGTCCTCCATGATGACATCAATGTGCGGTTCAAGAATCCTCGCA432
LeuGlnSerSerMetMetThrSerMetCysGlySerArgIleLeuAla
130135140
GCATTCTCGATCGCTTGGAGAGCAGCAGCCGCCGGCGGCAGATCGGCC480
AlaPheSerIleAlaTrpArgAlaAlaAlaAlaGlyGlyArgSerAla
145150155160
TCAGTCAGTTCTGAGTCTTCAGTCTCCGTCCCAATGTCAATGGACACC528
SerValSerSerGluSerSerValSerValProMetSerMetAspThr
165170175
TCGGATGAAACCTCAGAGGGTGCCACATTCCTGAGCCTCAGT570
SerAspGluThrSerGluGlyAlaThrPheLeuSerLeuSer
180185190
(2) INFORMATION FOR SEQ ID NO:38:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 190 amino acids
(B) TYPE: amino acid
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: protein
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:38:
AlaValTrpPheTrpIleSerIleSerHisArgLeuAlaThrSerAla
151015
ThrValAsnProAsnGluLysLysArgValThrLeuPheSerThrGln
202530
HisAspIleLeuThrValSerPheLeuValAlaSerLeuCysGlyAsn
354045
LysAlaPheAsnThrGluArgAlaThrLeuLysThrLeuSerSerPro
505560
SerAlaValSerAspSerTrpMetThrSerAsnGluSerGluAspGly
65707580
ValSerSerCysGluGluAspThrAspGlyValPheSerSerGluLeu
859095
LeuSerValThrGluIleSerAlaGlyAspGlyValArgGlyMetSer
100105110
SerProHisThrGlyIleSerArgLeuLeuProGlnArgGluGlyVal
115120125
LeuGlnSerSerMetMetThrSerMetCysGlySerArgIleLeuAla
130135140
AlaPheSerIleAlaTrpArgAlaAlaAlaAlaGlyGlyArgSerAla
145150155160
SerValSerSerGluSerSerValSerValProMetSerMetAspThr
165170175
SerAspGluThrSerGluGlyAlaThrPheLeuSerLeuSer
180185190
(2) INFORMATION FOR SEQ ID NO:39:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 1288 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Consensus Sequence 5E3
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:39:
ACGGGTAGGGGCAGGTCTGGACGCTACTACTACGCGGGGGTGGGCAAAGCCCCTGCGGGT60
GTGGTGCGCTCAGGTCCTGTCTGGTCGGCGGTGGAAGCTGGAGTGACCTGGTACGGAATG120
GAACCTGACTTGACAGCTAACCTACTGAGACTTTACGACGACTGCCCTTACACCGCAGCC180
GTCGCGGCTGATATCGGAGAAGCCGCGGTGTTCTTCTCTGGGCTCGCCCCATTGAGGATG240
CACCCTGATGTCAGCTGGGCAAAAGTTCGCGGCGTCAACTGGCCCCTCTTGGTGGGTGTT300
CAGCGGACCATGTGTCGGGAAACACTGTCTCCCGGCCCATCGGATGACCCCCAATGGGCA360
GGTCTGAAGGGCCCAAATCCTGTCCCACTCCTGCTGAGGTGGGGCAATGATTTACCATCT420
AAAGTGGCCGGCCACCACATAGTGGACGACCTGGTCCGGAGACTCGGTGTGGCGGAGGGT480
TACGTCCGCTGCGACGCTGGGCCGATCTTGATGATCGGTCTAGCTATCGCGGGGGGAATG540
ATCTACGCGTCATACACCGGGTCGCTAGTGGTGGTGACAGACTGGGATGTGAAGGGGGGT600
GGCGCCCCCCTTTATCGGCATGGAGACCAGGCCACGCCTCAGCCGGTGGTGCAGGTTCCT660
CCGGTAGACCATCGGCCGGGGGGTGAATCAGCACCATCGGATGCCAAGACAGTGACAGAT720
GCGGTGGCAGCCATCCAGGTGGACTGCGATTGGACTATCATGACTCTGTCGATCGGAGAA780
GTGTTGTCCTTGGCTCAGGCTAAGACGGCCGAGGCCTACACAGCAACCGCCAAGTGGCTC840
GCTGGCTGCTATACGGGGACGCGGGCCGTTCCCACTGTATCCATTGTTGACAAGCTCTTC900
GCCGGAGGGTGGGCGGCTGTGGTGGGCCATTGCCACAGCGTGATTGCTGCGGCGGTGGCG960
GCCTACGGGGCTTCAAGGAGCCCGCCGTTGGCAGCCGCGGCTTCCTACCTGATGGGGTTG1020
GGCGTTGGAGGCAACGCTCAGACGCGCCTGGCGTCTGCCCTCCTATTGGGGGCTGCTGGA1080
ACCGCCTTGGGCACTCCTGTCGTGGGCTTGACCATGGCAGGTGCGTTCATGGGGGGGGCC1140
AGTGTCTCCCCCTCCTTGGTCACCATTTTATTGGGGGCCGTCGGAGGTTGGGAGGGTGTT1200
GTCAACGCGGCGAGCCTAGTCTTTGACTTCATGGCGGGGAAACTTTCATCAGAAGATCTG1260
TGGTATGCCATCCCGGTACTGACCAGCC1288
(2) INFORMATION FOR SEQ ID NO:40:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 862 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Consensus Sequence 6E3
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:40:
ACGGCAACATGGGGCACAAGGTCTTAATCTTGAACCCCTCAGTGGCCACTGTGCGGGCCA60
TGGGCCCGTACATGGAGCGGCTGGCGGGTAAACATCCAAGTATATACTGTGGGCATGATA120
CAACTGCTTTCACAAGGATCACTGACTCCCCCCTGACGTATTCAACCTATGGGAGGTTTT180
TGGCCAACCCTAGGCAGATGCTACGGGGCGTTTCGGTGGTCATTTGTGATGAGTGCCACA240
GTCATGACTCAACCGTGCTGTTAGGCATTGGGAGAGTTCGGGAGCTGGCGCGTGGGTGCG300
GAGTGCAACTAGTGCTCTACGCCACCGCTACACCTCCCGGATCCCCTATGACGCAGCACC360
CTTCCATAATTGAGACAAAATTGGACGTGGGCGAGATTCCCTTTTATGGGCATGGAATAC420
CCCTCGAGCGGATGCGAACCGGAAGGCACCTCGTGTTCTGCCATTCTAAGGCTGAGTGCG480
AGCGCCTTGCTGGCCAGTTCTCCGCTAGGGGGGTCAATGCCATTGCCTATTATAGGGGTA540
AAGACAGTTCTATCATCAAGGATGGGGACCTGGTGGTCTGTGCTACAGACGCGCTTTCCA600
CTGGGTACACTGGAAATTTCGACTCCGTCACCGACTGTGGATTAGTGGTGGAGGAGGTCG660
TTGAGGTGACCCTTGATCCCACCATTACCATCTCCCTGCGGACAGTGCCTGCGTCGGCTG720
AACTGTCGATGCAAAGACGAGGACGCACGGGTAGGGGCAGGTCTGGACGCTACTACTACG780
CGGGGGTGGGCAAAGCCCCTGCGGGTGTGGTGCGCTCAGGTCCTGTCTGGTCGGCGGTGG840
AAGCTGGAGTGACCTCGTACGG862
(2) INFORMATION FOR SEQ ID NO:41:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 865 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Individual Clone GE3L-11
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:41:
AGTACGGCAACATGGGGCACAAGGTCTTAATCTTGAACCCCTCAGTGGCCACTGTGCGGG60
CCATGGGCCCGTACATGGAGCGGCTGGCGGGTAAACATCCAAGTATATACTGTGGGCATG120
ATACAACTGCTTTCACAAGGATCACTGACTCCCCCCTGACGTATTCAACCTATGGGAGGT180
TTTTGGCCAACCCTAGGCAGATGCTACGGGGCGTTTCGGTGGTCATTTGTGATGAGTGCC240
ACAGTCATGACTCAACCGTGCTGTTAGGCATTGGGAGAGTCCGGGAGCTGGCGCGTGGGT300
GCGGGGTGCAACTAGTGCTCTACGCCACCGCTACACCTCCCGGATCCCCTATGACGCAGC360
ACCCTTCCATAATTGAGACAAAATTGGACGTGGGCGAGATTCCCTTTTATGGACATGGAA420
TACCCCTCGAGCGGATGCGAACCGGAAGGCACCTCGTGTTCTGCCATTCTAAGGCTGAGT480
GCGAGCGCCTTGCTGGCCAGTTCTCCGCTAGGGGGGTCAATGCCATTGCCTATTATAGGG540
GTAAAGACAGTTCTATCATCAAGGATGGGGACCTGGTGGTCTGTGCTACAGACGCGCTTT600
CCACTGGGTACACTGGAAATTTCGACTCCGTCACCGACTGTGGATTAGTGGTGGAGGAGG660
TCGTTGAGGTGACCCTTGATCCCACCATTACCATCTCCCTGCGGACAGTGCCTGCGTCGG720
CTGAACTGTCGATGCAAAGACGAGGACGCACGGGTAGGGGCAGGTCTGGACGCTACTACT780
ACGCGGGGGTGGGCAAAGCCCCTGCGGGTGTGGTGCGCTCAGGTCCTGTCTGGTCGGCGG840
TGGAAGCTGGAGTGACCTCGTACGG865
(2) INFORMATION FOR SEQ ID NO:42:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 596 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Consensus Sequence 7E3
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:42:
AGCATGGGAACATGCTTGAACGGCCTGCTGTTCACGACCTTCCATGGGGCTTCATCCCGA60
ACCATCGCCACACCCGTGGGGGCCCTTAATCCCAGATGGTGGTCAGCCAGTGATGATGTC120
ACGGTGTATCCACTCCCGGATGGGGCTACTTCGTTAACACCTTGTACTTGCCAGGCTGAG180
TCCTGTTGGGTCATCAGATCCGACGGGGCCCTATGCCATGGCTTGAGCAAGGGGGACAAG240
GTGGAGCTGGATGTGGCCATGGAGGTCTCTGACTTCCGTGGCTCGTCTGGCTCACCGGTC300
CTATGTGACGAAGGGCACGCAGTAGGAATGCTCGTGTCTGTGCTTCACTCCGGTGGTAGG360
GTCACCGCGGCACGGTTCACTAGGCCGTGGACCCAAGTGCCAACAGATGCCAAAACCACT420
ACTGAACCCCCTCCGGTGCCGGCCAAAGGAGTTTTCAAAGAGGCCCCGTTGTTTATGCCT480
ACGGGAGCGGGAAAGAGCACTCGCGTCCCGTTGGAGTACGATAACATGGGGCACAAGGTC540
TTAATCTTGAACCCCTCAGTGGCCACTGTGCGGGCCATGGGCCCGTACATGGAGCG596
(2) INFORMATION FOR SEQ ID NO:43:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 586 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Consensus Sequence 5E5
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:43:
GAGCTATGGGTACGCGTGCGAGCCCTCATATCATGCATCATTGGACACGGCCCCCTTCTG60
CTCCACTTGGCTTGCTGAGTGCAATGCAGATGGGAAGCGCCATTTCTTCCTGACCACGGA120
CTTCCGGAGGCCGCTCGCTCGCATGTCGAGTGAGTATAGTGACCCGATGGCTTCGGCGAT180
CGGTTACATCCTCCTTTATCCTTGGCACCCCATCACACGGTGGGTCATCATCCCTCATGT240
GCTAACGTGCGCATTCAGGGGTGGAGGCACACCGTCTGATCCGGTTTGGTGCCAGGTGCA300
TGGTAACTACTACAAGTTTCCACTGGACAAACTGCCTAACATCATCGTGGCCCTCCACGG360
ACCAGCAGCGTTGAGGGTTACCGCAGACACAACTAAAACAAAGATGGAGGCTGGTAAGGT420
TCTGAGCGACCTCAAGCTCCCTGGCTTAGCAGTCCACCGAAAGAAGGCCGGGGCGTTGCG480
AACACGCATGCTCCGCTCGCGCGGTTGGGCTGAGTTGGCTAGGGGCTTGTTGTGGCATCC540
AGGCCTACGGCTTCCTCCCCCTGAGATTGCTGGTATCCCGGGGGGT586
(2) INFORMATION FOR SEQ ID NO:44:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 242 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Consensus Sequence 6E5 (44F)
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:44:
CGAACGCGCATGCTCCGCTCGCGCGGTTGGGCTGAGTTGGCTAGGGGCTTGTTGTGGCAT60
CCAGGCCTACGGCTTCCTCCCCCTGAGATTGCTGGTATCCCGGGGGGTTTCCCTCTCTCC120
CCCCCCTATATGGGGGTGGTACACCAATTGGATTTCACAAGCCAGAGGAGTCGCTGGCGG180
TGGTTGGGGTTCTTAGCCCTGCTCATCGTAGCCCTCTTCGGGTGAACTAAATTCATCTGT240
TG242
(2) INFORMATION FOR SEQ ID NO:45:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 27 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer Gt11 rev-JL
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:45:
TGGTAATGGTAGCGACCGGCGCTCAGC27
(2) INFORMATION FOR SEQ ID NO:46:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 45 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer GE- 3F
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:46:
GCCGCCATGGTCTCATGGGACGCGGACGCTCGTGCGCCCGCGATG45
(2) INFORMATION FOR SEQ ID NO:47:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 34 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer GE- 3R
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:47:
GCGCGGATCCGATAAGTGCTGGCGATGGAGTACG34
(2) INFORMATION FOR SEQ ID NO:48:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 22 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer GE- 9F
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:48:
GGCACCATGGTCACCCCGGAAG22
(2) INFORMATION FOR SEQ ID NO:49:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 28 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer GE- 9R
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:49:
GCTCGGATCCGGAGCAGAAGGGGGCCGT28
(2) INFORMATION FOR SEQ ID NO:50:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 364 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: GE3-2
(ix) FEATURE:
(A) NAME/KEY: CDS
(B) LOCATION: 2..364
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:50:
GGTCTCATGGGACGCGGACGCTCGTGCGCCCGCGATGGTCTATGGC46
ValSerTrpAspAlaAspAlaArgAlaProAlaMetValTyrGly
151015
CCTGGGCAAAGTGTTACCATTGACGGGGAGCGCTACACCTTGCCTCAT94
ProGlyGlnSerValThrIleAspGlyGluArgTyrThrLeuProHis
202530
CAACTGAGGCTCAGGAATGTGGCACCCTCTGAGGTTTCATCCGAGGTG142
GlnLeuArgLeuArgAsnValAlaProSerGluValSerSerGluVal
354045
TCCATTGACATTGGGACGGAGACTGAAGACTCAGAACTGACTGAGGCC190
SerIleAspIleGlyThrGluThrGluAspSerGluLeuThrGluAla
505560
GATCTGCCGCCGGCGGCTGCTGCTCTCCAAGCGATCGAGAATGCTGCG238
AspLeuProProAlaAlaAlaAlaLeuGlnAlaIleGluAsnAlaAla
657075
AGGATTCTTGAACCGCACATTGATGTCATCATGGAGGACTGCAGTACA286
ArgIleLeuGluProHisIleAspValIleMetGluAspCysSerThr
80859095
CCCTCTCTTTGTGGTAGTAGCCGAGAGATGCCTGTATGGGGAGAAGAC334
ProSerLeuCysGlySerSerArgGluMetProValTrpGlyGluAsp
100105110
ATCCCCCGTACTCCATCGCCAGCACTTATC364
IleProArgThrProSerProAlaLeuIle
115120
(2) INFORMATION FOR SEQ ID NO:51:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 121 amino acids
(B) TYPE: amino acid
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: protein
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:51:
ValSerTrpAspAlaAspAlaArgAlaProAlaMetValTyrGlyPro
151015
GlyGlnSerValThrIleAspGlyGluArgTyrThrLeuProHisGln
202530
LeuArgLeuArgAsnValAlaProSerGluValSerSerGluValSer
354045
IleAspIleGlyThrGluThrGluAspSerGluLeuThrGluAlaAsp
505560
LeuProProAlaAlaAlaAlaLeuGlnAlaIleGluAsnAlaAlaArg
65707580
IleLeuGluProHisIleAspValIleMetGluAspCysSerThrPro
859095
SerLeuCysGlySerSerArgGluMetProValTrpGlyGluAspIle
100105110
ProArgThrProSerProAlaLeuIle
115120
(2) INFORMATION FOR SEQ ID NO:52:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 290 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Clone GE9- 2
(ix) FEATURE:
(A) NAME/KEY: CDS
(B) LOCATION: 3..290
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:52:
CCATGGTCACCCCGGAAGGGGTGCCCGTTGGTGAGAGGTATTGCAGA47
MetValThrProGluGlyValProValGlyGluArgTyrCysArg
151015
TCCTCGGGTGTCCTAACAACTAGCGCGAGCAACTGCTTGACCTGCTAC95
SerSerGlyValLeuThrThrSerAlaSerAsnCysLeuThrCysTyr
202530
ATCAAGGTGAAAGCCGCCTGTGAGAGGGTGGGGCTGAAAAATGTCTCT143
IleLysValLysAlaAlaCysGluArgValGlyLeuLysAsnValSer
354045
CTTCTCATAGCCGGCGATGACTGCTTGATCATATGTGAGCGGCCAGTG191
LeuLeuIleAlaGlyAspAspCysLeuIleIleCysGluArgProVal
505560
TGCGACCCAAGCGACGCTTTGGGCAGAGCCCTAGCGAGCTATGGGTAC239
CysAspProSerAspAlaLeuGlyArgAlaLeuAlaSerTyrGlyTyr
657075
GCGTGCGAGCCCTCATATTATGCATGCTCGGACACGGCCCCCTTCTGC287
AlaCysGluProSerTyrTyrAlaCysSerAspThrAlaProPheCys
80859095
TCC290
Ser
(2) INFORMATION FOR SEQ ID NO:53:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 96 amino acids
(B) TYPE: amino acid
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: protein
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:53:
MetValThrProGluGlyValProValGlyGluArgTyrCysArgSer
151015
SerGlyValLeuThrThrSerAlaSerAsnCysLeuThrCysTyrIle
202530
LysValLysAlaAlaCysGluArgValGlyLeuLysAsnValSerLeu
354045
LeuIleAlaGlyAspAspCysLeuIleIleCysGluArgProValCys
505560
AspProSerAspAlaLeuGlyArgAlaLeuAlaSerTyrGlyTyrAla
65707580
CysGluProSerTyrTyrAlaCysSerAspThrAlaProPheCysSer
859095
(2) INFORMATION FOR SEQ ID NO:54:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 20 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: JML-A SISPA Primer
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:54:
AGGAATTCAGCGGCCGCGAG20
(2) INFORMATION FOR SEQ ID NO:55:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 22 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: JML-B SISPA Primer
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:55:
CTCGCGGCCGCTGAATTCCTTT22
(2) INFORMATION FOR SEQ ID NO:56:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 32 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: 470ep-f1 Primer
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:56:
GCGAATTCGCCATGGCGGGGAGACTTTCATCA32
(2) INFORMATION FOR SEQ ID NO:57:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 35 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: 470ep-R1 Primer
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:57:
GCGAATTCGGATCCAGGGCCATAGACCATCGCGGG35
(2) INFORMATION FOR SEQ ID NO:58:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 26 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: 470ep-f2 Primer
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:58:
GCGAATTCCGTGCGCCCGCCATGGTC26
(2) INFORMATION FOR SEQ ID NO:59:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 32 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: 470ep-R3 Primer
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:59:
GCGAATTCGGATCCCAAGGTTTCTTGCCTAGC32
(2) INFORMATION FOR SEQ ID NO:60:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 26 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: 470ep-f4 Primer
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:60:
GCGAATTCAAGTGTGAGGCTAGGCAA26
(2) INFORMATION FOR SEQ ID NO:61:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 35 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: 470ep-R4 Primer
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:61:
GCGAATTCGGATCCCCACACAGATGGCGCAAGGGG35
(2) INFORMATION FOR SEQ ID NO:62:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 27 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: KL-1 SISPA Primer
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:62:
GCAGGATCCGAATTCGCATCTAGAGAT27
(2) INFORMATION FOR SEQ ID NO:63:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 29 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: KL-2 SISPA Primer
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:63:
ATCTCTAGATGCGAATTCGGATCCTGCGA29
(2) INFORMATION FOR SEQ ID NO:64:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 186 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Clone Y5- 10
(ix) FEATURE:
(A) NAME/KEY: CDS
(B) LOCATION: 1..186
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:64:
CGTGCGCCCGCCATGGTCTATGGCCCTGGGCAAAGTGTTGCCATTGAC48
ArgAlaProAlaMetValTyrGlyProGlyGlnSerValAlaIleAsp
151015
GGGGAGCGCTACACCTTGCCTCATCAACTGAGGCTCAGGAATGTGGCA96
GlyGluArgTyrThrLeuProHisGlnLeuArgLeuArgAsnValAla
202530
CCCTCTGAGGTTTCATCCGAGGTGTCCATTGACATTGGGACGGAGGCT144
ProSerGluValSerSerGluValSerIleAspIleGlyThrGluAla
354045
GAAAACTCAGAACTGACTGAGGCCGATCTGCCGCCGGCGGCT186
GluAsnSerGluLeuThrGluAlaAspLeuProProAlaAla
505560
(2) INFORMATION FOR SEQ ID NO:65:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 62 amino acids
(B) TYPE: amino acid
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: protein
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:65:
ArgAlaProAlaMetValTyrGlyProGlyGlnSerValAlaIleAsp
151015
GlyGluArgTyrThrLeuProHisGlnLeuArgLeuArgAsnValAla
202530
ProSerGluValSerSerGluValSerIleAspIleGlyThrGluAla
354045
GluAsnSerGluLeuThrGluAlaAspLeuProProAlaAla
505560
(2) INFORMATION FOR SEQ ID NO:66:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 282 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Clone Y5- 12
(ix) FEATURE:
(A) NAME/KEY: CDS
(B) LOCATION: 1..282
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:66:
CGTGCGCCCGCCATGGTCTATGGCCCTGGGCAAAGTGTTACCATTGAC48
ArgAlaProAlaMetValTyrGlyProGlyGlnSerValThrIleAsp
151015
GGGGAGCGCTACACCTTGCCTCATCAACTGAGGCTCAGGAATGTGGCA96
GlyGluArgTyrThrLeuProHisGlnLeuArgLeuArgAsnValAla
202530
CCCTCTGAGGTTTCATCCGAGGTGTCCATTGACATTGGGACGGAGACT144
ProSerGluValSerSerGluValSerIleAspIleGlyThrGluThr
354045
GAAGACTCAGAACTGACTGAGGCCGATCTGCCGCCGGCGGCTGCTGCT192
GluAspSerGluLeuThrGluAlaAspLeuProProAlaAlaAlaAla
505560
CTCCAAGCGATCGAGAATGCTGCGAGGATTCTTGAACCGCACATTGAT240
LeuGlnAlaIleGluAsnAlaAlaArgIleLeuGluProHisIleAsp
65707580
GTCATCATGGAGGACTGCAGTACACCCTCTCTTTGTGGTAGT282
ValIleMetGluAspCysSerThrProSerLeuCysGlySer
8590
(2) INFORMATION FOR SEQ ID NO:67:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 94 amino acids
(B) TYPE: amino acid
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: protein
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:67:
ArgAlaProAlaMetValTyrGlyProGlyGlnSerValThrIleAsp
151015
GlyGluArgTyrThrLeuProHisGlnLeuArgLeuArgAsnValAla
202530
ProSerGluValSerSerGluValSerIleAspIleGlyThrGluThr
354045
GluAspSerGluLeuThrGluAlaAspLeuProProAlaAlaAlaAla
505560
LeuGlnAlaIleGluAsnAlaAlaArgIleLeuGluProHisIleAsp
65707580
ValIleMetGluAspCysSerThrProSerLeuCysGlySer
8590
(2) INFORMATION FOR SEQ ID NO:68:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 279 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Clone Y5- 26
(ix) FEATURE:
(A) NAME/KEY: CDS
(B) LOCATION: 1..279
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:68:
CGTGCGCCCGCCATGGTCTATGGCCCTGGGCAAAGTGTTTCCATTGAC48
ArgAlaProAlaMetValTyrGlyProGlyGlnSerValSerIleAsp
151015
GGGGAGCGCTACACCTTGCCTCATCAACTGAGGCTCAGGAATGTGGCA96
GlyGluArgTyrThrLeuProHisGlnLeuArgLeuArgAsnValAla
202530
CCCTCTGAGGTTTCATCCGAGGTGTCCATTGACATTGGGACGGAGACT144
ProSerGluValSerSerGluValSerIleAspIleGlyThrGluThr
354045
GAAGACTCAGAACTGACTGAGGCCGACCTGCCGCCGGCGGCTGCTGCT192
GluAspSerGluLeuThrGluAlaAspLeuProProAlaAlaAlaAla
505560
CTCCAAGCGATCGAGAATGCTGCGAGGATTCTTGAACCGCACATCGAT240
LeuGlnAlaIleGluAsnAlaAlaArgIleLeuGluProHisIleAsp
65707580
GTCATCATGGAGGACTGCAGTACACCCTCTCTTTGTGGT279
ValIleMetGluAspCysSerThrProSerLeuCysGly
8590
(2) INFORMATION FOR SEQ ID NO:69:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 93 amino acids
(B) TYPE: amino acid
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: protein
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:69:
ArgAlaProAlaMetValTyrGlyProGlyGlnSerValSerIleAsp
151015
GlyGluArgTyrThrLeuProHisGlnLeuArgLeuArgAsnValAla
202530
ProSerGluValSerSerGluValSerIleAspIleGlyThrGluThr
354045
GluAspSerGluLeuThrGluAlaAspLeuProProAlaAlaAlaAla
505560
LeuGlnAlaIleGluAsnAlaAlaArgIleLeuGluProHisIleAsp
65707580
ValIleMetGluAspCysSerThrProSerLeuCysGly
8590
(2) INFORMATION FOR SEQ ID NO:70:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 108 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Clone Y5- 5
(ix) FEATURE:
(A) NAME/KEY: CDS
(B) LOCATION: 1..108
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:70:
GCCTATTGTGACAAGGTGCGCACTCCGCTTGAATTGCAGGTTGGGTGC48
AlaTyrCysAspLysValArgThrProLeuGluLeuGlnValGlyCys
151015
TTGGTGGGCAATGAACTTACCTTTGAATGTGACAAGTGTGAGGCTAGG96
LeuValGlyAsnGluLeuThrPheGluCysAspLysCysGluAlaArg
202530
CAAGAAACCTTG108
GlnGluThrLeu
35
(2) INFORMATION FOR SEQ ID NO:71:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 36 amino acids
(B) TYPE: amino acid
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: protein
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:71:
AlaTyrCysAspLysValArgThrProLeuGluLeuGlnValGlyCys
151015
LeuValGlyAsnGluLeuThrPheGluCysAspLysCysGluAlaArg
202530
GlnGluThrLeu
35
(2) INFORMATION FOR SEQ ID NO:72:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 132 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Clone Y5- 3
(ix) FEATURE:
(A) NAME/KEY: CDS
(B) LOCATION: 1..132
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:72:
GAGATGGAAATCCAGAACCATACAGCCTATTGTGACAAGGTGCGCACT48
GluMetGluIleGlnAsnHisThrAlaTyrCysAspLysValArgThr
151015
CCGCTTGAATTGCAGGTTGGGTGCTTGGTGGGCAATGAACTTACCTTT96
ProLeuGluLeuGlnValGlyCysLeuValGlyAsnGluLeuThrPhe
202530
GAATGTGACAAGTGTGAGGCTAGGCAAGAAACCTTG132
GluCysAspLysCysGluAlaArgGlnGluThrLeu
3540
(2) INFORMATION FOR SEQ ID NO:73:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 44 amino acids
(B) TYPE: amino acid
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: protein
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:73:
GluMetGluIleGlnAsnHisThrAlaTyrCysAspLysValArgThr
151015
ProLeuGluLeuGlnValGlyCysLeuValGlyAsnGluLeuThrPhe
202530
GluCysAspLysCysGluAlaArgGlnGluThrLeu
3540
(2) INFORMATION FOR SEQ ID NO:74:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 258 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Clone Y5- 27
(ix) FEATURE:
(A) NAME/KEY: CDS
(B) LOCATION: 1..258
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:74:
AAAGCCTTATTTCCACAGAGCGACGCGACCAGGAAGCTTACCGTCAAG48
LysAlaLeuPheProGlnSerAspAlaThrArgLysLeuThrValLys
151015
ATGTCATGCTGCGTTGAAAAGAGCGTCACGCGCTTTTTCTCATTGGGG96
MetSerCysCysValGluLysSerValThrArgPhePheSerLeuGly
202530
TTGACGGTGGCTGATGTTGCTAGCCTGTGTGAGATGGAAATCCAGAAC144
LeuThrValAlaAspValAlaSerLeuCysGluMetGluIleGlnAsn
354045
CATATAGCCTATTGTGACAAGGTGCGCACTCCGCTTGAATTGCAGGTT192
HisIleAlaTyrCysAspLysValArgThrProLeuGluLeuGlnVal
505560
GGGTGCTTGGTGGGCAATGAACTCACCTTTGAATGTGACAAGTGTGAG240
GlyCysLeuValGlyAsnGluLeuThrPheGluCysAspLysCysGlu
65707580
GCTAGGCAAGAAACCTTG258
AlaArgGlnGluThrLeu
85
(2) INFORMATION FOR SEQ ID NO:75:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 86 amino acids
(B) TYPE: amino acid
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: protein
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:75:
LysAlaLeuPheProGlnSerAspAlaThrArgLysLeuThrValLys
151015
MetSerCysCysValGluLysSerValThrArgPhePheSerLeuGly
202530
LeuThrValAlaAspValAlaSerLeuCysGluMetGluIleGlnAsn
354045
HisIleAlaTyrCysAspLysValArgThrProLeuGluLeuGlnVal
505560
GlyCysLeuValGlyAsnGluLeuThrPheGluCysAspLysCysGlu
65707580
AlaArgGlnGluThrLeu
85
(2) INFORMATION FOR SEQ ID NO:76:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 108 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Clone Y5- 25
(ix) FEATURE:
(A) NAME/KEY: CDS
(B) LOCATION: 1..108
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:76:
ACCTATTGTGACAAGGTGCGCACTCCGCTTGAATTGCAGGTTGGGTGC48
ThrTyrCysAspLysValArgThrProLeuGluLeuGlnValGlyCys
151015
TTGGTGGGCAATGAACTTACCTTTGAATGTGACAAGTGTGAGGCTAGG96
LeuValGlyAsnGluLeuThrPheGluCysAspLysCysGluAlaArg
202530
CAAGAAACCTTG108
GlnGluThrLeu
35
(2) INFORMATION FOR SEQ ID NO:77:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 36 amino acids
(B) TYPE: amino acid
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: protein
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:77:
ThrTyrCysAspLysValArgThrProLeuGluLeuGlnValGlyCys
151015
LeuValGlyAsnGluLeuThrPheGluCysAspLysCysGluAlaArg
202530
GlnGluThrLeu
35
(2) INFORMATION FOR SEQ ID NO:78:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 108 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Clone Y5- 20
(ix) FEATURE:
(A) NAME/KEY: CDS
(B) LOCATION: 52..108
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:78:
GCCGACACTACTAAGGTGTATGTTACCAATCCAGACAATGTGGGACGAAGGGTGGGC57
ValGly
AATGAACTTACCTTTGAATGTGACAAGTGTGAGGCTAGGCAAGAAACC105
AsnGluLeuThrPheGluCysAspLysCysGluAlaArgGlnGluThr
51015
TTG108
Leu
(2) INFORMATION FOR SEQ ID NO:79:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 19 amino acids
(B) TYPE: amino acid
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: protein
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:79:
ValGlyAsnGluLeuThrPheGluCysAspLysCysGluAlaArgGln
151015
GluThrLeu
(2) INFORMATION FOR SEQ ID NO:80:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 168 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Clone Y5- 16
(ix) FEATURE:
(A) NAME/KEY: CDS
(B) LOCATION: 1..168
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:80:
TTGGGGTTGACGGTGGCTGATGTTGCTAGCCTGTGTGAGATGGAAATC48
LeuGlyLeuThrValAlaAspValAlaSerLeuCysGluMetGluIle
151015
CAGAACCATACAGCCTATTGTGACAAGGTGCGCACTCCGCTTGAATTG96
GlnAsnHisThrAlaTyrCysAspLysValArgThrProLeuGluLeu
202530
CAGGTTGGGTGCTTGGTGGGCAATGAACTTACCTTTGAATGTGACAAG144
GlnValGlyCysLeuValGlyAsnGluLeuThrPheGluCysAspLys
354045
TGTGAGGCTAGGCAAGAAACCTTG168
CysGluAlaArgGlnGluThrLeu
5055
(2) INFORMATION FOR SEQ ID NO:81:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 56 amino acids
(B) TYPE: amino acid
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: protein
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:81:
LeuGlyLeuThrValAlaAspValAlaSerLeuCysGluMetGluIle
151015
GlnAsnHisThrAlaTyrCysAspLysValArgThrProLeuGluLeu
202530
GlnValGlyCysLeuValGlyAsnGluLeuThrPheGluCysAspLys
354045
CysGluAlaArgGlnGluThrLeu
5055
(2) INFORMATION FOR SEQ ID NO:82:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 313 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Clone Y5- 50
(ix) FEATURE:
(A) NAME/KEY: CDS
(B) LOCATION: 1..313
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:82:
ATCACCGTCAACCCCAATGAGAAAAAGCGCGTGACGCTCTTTTCAACG48
IleThrValAsnProAsnGluLysLysArgValThrLeuPheSerThr
151015
CAGCACGACATCTTGACGGTAAGCTTCCTGGTCGCGTCGCTCTGTGGA96
GlnHisAspIleLeuThrValSerPheLeuValAlaSerLeuCysGly
202530
AATAAGGCTTTTAATACGGAAAGAGCCACGTTGAAGACACTTTCCTCC144
AsnLysAlaPheAsnThrGluArgAlaThrLeuLysThrLeuSerSer
354045
CCTTCGGCTGTCTCGGACTCTTGGATGACCTCGAATGAGTCAGAGGAC192
ProSerAlaValSerAspSerTrpMetThrSerAsnGluSerGluAsp
505560
GGGGTATCCTCCTGCGAGGAGGACACCGACGGGGTCTTCTCATCTGAG240
GlyValSerSerCysGluGluAspThrAspGlyValPheSerSerGlu
65707580
CTGCTCTCAGTAACCGAGATAAGTGCTGGCGATGGAGTACGGGGGATG288
LeuLeuSerValThrGluIleSerAlaGlyAspGlyValArgGlyMet
859095
TCTTCTCCCCATACAGGCATCTCTC313
SerSerProHisThrGlyIleSer
100
(2) INFORMATION FOR SEQ ID NO:83:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 104 amino acids
(B) TYPE: amino acid
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: protein
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:83:
IleThrValAsnProAsnGluLysLysArgValThrLeuPheSerThr
151015
GlnHisAspIleLeuThrValSerPheLeuValAlaSerLeuCysGly
202530
AsnLysAlaPheAsnThrGluArgAlaThrLeuLysThrLeuSerSer
354045
ProSerAlaValSerAspSerTrpMetThrSerAsnGluSerGluAsp
505560
GlyValSerSerCysGluGluAspThrAspGlyValPheSerSerGlu
65707580
LeuLeuSerValThrGluIleSerAlaGlyAspGlyValArgGlyMet
859095
SerSerProHisThrGlyIleSer
100
(2) INFORMATION FOR SEQ ID NO:84:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 89 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Clone Y5- 52
(ix) FEATURE:
(A) NAME/KEY: CDS
(B) LOCATION: 28..87
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:84:
ACTGAGAGCAGCTCAGATGAGAAGACCCCTTCGGCTGTCTCGGACTCTTGG51
ProSerAlaValSerAspSerTrp
15
ATGACCTCGAATGAGTCAGAGGACGGGGTATCCTCGCA89
MetThrSerAsnGluSerGluAspGlyValSerSer
101520
(2) INFORMATION FOR SEQ ID NO:85:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 20 amino acids
(B) TYPE: amino acid
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: protein
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:85:
ProSerAlaValSerAspSerTrpMetThrSerAsnGluSerGluAsp
151015
GlyValSerSer
20
(2) INFORMATION FOR SEQ ID NO:86:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 214 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Clone Y5- 53
(ix) FEATURE:
(A) NAME/KEY: CDS
(B) LOCATION: 1..100
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:86:
AATAAGGCTTTTAATACGGAAAGAGCCACGTTGAAGACACTTTCCTCC48
AsnLysAlaPheAsnThrGluArgAlaThrLeuLysThrLeuSerSer
151015
CCTTCGGCTGTCTCGGACTCTTGGATGACCTCGAATGAGTCAGAGGAC96
ProSerAlaValSerAspSerTrpMetThrSerAsnGluSerGluAsp
202530
GGGGATCTCTAGATGCGAATTCAAGTGTGAGGCTAGGCAAGAAACCTTGGCCTC150
Gly
CTTCTCTTACATTTGGTCTGGAGTGCCGCTGACTAGGGCCACGCCGGCCAAGCCTCCCGT210
GGTG214
(2) INFORMATION FOR SEQ ID NO:87:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 33 amino acids
(B) TYPE: amino acid
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: protein
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:87:
AsnLysAlaPheAsnThrGluArgAlaThrLeuLysThrLeuSerSer
151015
ProSerAlaValSerAspSerTrpMetThrSerAsnGluSerGluAsp
202530
Gly
(2) INFORMATION FOR SEQ ID NO:88:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 113 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Clone Y5- 55
(ix) FEATURE:
(A) NAME/KEY: CDS
(B) LOCATION: 52..113
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:88:
CCATCGCCAGCACTTATCTCGGTTACTGAGAGCAGCTCAGATCAGAAGACCCCTTCG57
ProSer
1
GCTGTCTCGGACTCTTGGATGACCTCGAATGAGTCAGAGGACGGGGTA105
AlaValSerAspSerTrpMetThrSerAsnGluSerGluAspGlyVal
51015
TCCTCGCA113
SerSer
20
(2) INFORMATION FOR SEQ ID NO:89:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 20 amino acids
(B) TYPE: amino acid
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: protein
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:89:
ProSerAlaValSerAspSerTrpMetThrSerAsnGluSerGluAsp
151015
GlyValSerSer
20
(2) INFORMATION FOR SEQ ID NO:90:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 330 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Clone Y5- 56
(ix) FEATURE:
(A) NAME/KEY: CDS
(B) LOCATION: 1..330
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:90:
ACGTTGAAGACACTTTCCTCCCCTTCGGCTGTCTCGGACTCTTGGATG48
ThrLeuLysThrLeuSerSerProSerAlaValSerAspSerTrpMet
151015
ACCTCGAATGAGTCAGAGGACGGGGTATCCTCCTGCGAGGAGGACACC96
ThrSerAsnGluSerGluAspGlyValSerSerCysGluGluAspThr
202530
GACGGGGTCTTCTCATCTGAGCTGCTCTCAGTAACCGAGATAAGTGCT144
AspGlyValPheSerSerGluLeuLeuSerValThrGluIleSerAla
354045
GGCGATGGAGTACGGGGGATGTCTTCTCCCCATACAGGCATCTCTCGG192
GlyAspGlyValArgGlyMetSerSerProHisThrGlyIleSerArg
505560
CTACTACCACAAAGAGAGGGTGTACTGCAGTCCTCCATGATGACATCA240
LeuLeuProGlnArgGluGlyValLeuGlnSerSerMetMetThrSer
65707580
ATGTGCGGTTCAAGAATCCTCGCAGCATTCTCGATCGCTTGGAGAGCA288
MetCysGlySerArgIleLeuAlaAlaPheSerIleAlaTrpArgAla
859095
GCAGCCGCCGGCGGCAGATCGGCCTCAGTCAGTTCTGAGTCT330
AlaAlaAlaGlyGlyArgSerAlaSerValSerSerGluSer
100105110
(2) INFORMATION FOR SEQ ID NO:91:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 110 amino acids
(B) TYPE: amino acid
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: protein
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:91:
ThrLeuLysThrLeuSerSerProSerAlaValSerAspSerTrpMet
151015
ThrSerAsnGluSerGluAspGlyValSerSerCysGluGluAspThr
202530
AspGlyValPheSerSerGluLeuLeuSerValThrGluIleSerAla
354045
GlyAspGlyValArgGlyMetSerSerProHisThrGlyIleSerArg
505560
LeuLeuProGlnArgGluGlyValLeuGlnSerSerMetMetThrSer
65707580
MetCysGlySerArgIleLeuAlaAlaPheSerIleAlaTrpArgAla
859095
AlaAlaAlaGlyGlyArgSerAlaSerValSerSerGluSer
100105110
(2) INFORMATION FOR SEQ ID NO:92:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 195 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Clone Y5- 57
(ix) FEATURE:
(A) NAME/KEY: CDS
(B) LOCATION: 1..195
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:92:
ACGGAAAGAGCCACGTTGAAGACACTTTCCTCCCCTTCGGCTGCCTCG48
ThrGluArgAlaThrLeuLysThrLeuSerSerProSerAlaAlaSer
151015
GACTCTTGGATGACCTCGAATGAGTCGGAGGACGGGGTATCCTCCTGC96
AspSerTrpMetThrSerAsnGluSerGluAspGlyValSerSerCys
202530
GAAGAGGACACCGACGGGGTCTTCTCATCTGAGCTGCTCTCAGTAACC144
GluGluAspThrAspGlyValPheSerSerGluLeuLeuSerValThr
354045
GAGATAAGTGCTGGCGGTGGAGTACGGGGGATGTCTTCTCCCCATACG192
GluIleSerAlaGlyGlyGlyValArgGlyMetSerSerProHisThr
505560
GGC195
Gly
65
(2) INFORMATION FOR SEQ ID NO:93:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 65 amino acids
(B) TYPE: amino acid
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: protein
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:93:
ThrGluArgAlaThrLeuLysThrLeuSerSerProSerAlaAlaSer
151015
AspSerTrpMetThrSerAsnGluSerGluAspGlyValSerSerCys
202530
GluGluAspThrAspGlyValPheSerSerGluLeuLeuSerValThr
354045
GluIleSerAlaGlyGlyGlyValArgGlyMetSerSerProHisThr
505560
Gly
65
(2) INFORMATION FOR SEQ ID NO:94:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 115 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Clone Y5- 60
(ix) FEATURE:
(A) NAME/KEY: CDS
(B) LOCATION: 1..115
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:94:
AAGACACTTTCCTCCCCTTCGGCTGTCTCGGACTCTTGGATGACCTCG48
LysThrLeuSerSerProSerAlaValSerAspSerTrpMetThrSer
151015
AATGAGTCAGAGGACGGGGTATCCTCCTGCGAGGAGGACACCGACTGG96
AsnGluSerGluAspGlyValSerSerCysGluGluAspThrAspTrp
202530
GTCTTCTCATCTGAGCTGC115
ValPheSerSerGluLeu
35
(2) INFORMATION FOR SEQ ID NO:95:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 38 amino acids
(B) TYPE: amino acid
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: protein
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:95:
LysThrLeuSerSerProSerAlaValSerAspSerTrpMetThrSer
151015
AsnGluSerGluAspGlyValSerSerCysGluGluAspThrAspTrp
202530
ValPheSerSerGluLeu
35
(2) INFORMATION FOR SEQ ID NO:96:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 93 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Clone Y5- 63
(ix) FEATURE:
(A) NAME/KEY: CDS
(B) LOCATION: 19..93
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:96:
GAGAGCAGCTCAGATGAGAAGACACTTTCCTCCCCTTCGGCTGTCTCGGAC51
LysThrLeuSerSerProSerAlaValSerAsp
1510
TCTTGGATGACCTCGAATGAGTCAGAGGACGGGGTATCCTCG93
SerTrpMetThrSerAsnGluSerGluAspGlyValSerSer
152025
(2) INFORMATION FOR SEQ ID NO:97:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 25 amino acids
(B) TYPE: amino acid
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: protein
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:97:
LysThrLeuSerSerProSerAlaValSerAspSerTrpMetThrSer
151015
AsnGluSerGluAspGlyValSerSer
2025
(2) INFORMATION FOR SEQ ID NO:98:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 1181 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Consensus Sequence 8E3
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:98:
GCTGGCTGAGGCACGGTTGGTCCCGCTGATCTTGCTGCTGCTATGGTGGTGGGTGAACCA60
GCTGGCAGTCCTAGGGCTGCCGGCTGTGGAAGCCGCCGTGGCAGGTGAGGTCTTCGCGGG120
CCCTGCCCTGTCCTGGTGTCTGGGACTCCCGGTCGTCAGTATGATATTGGGTTTGGCAAA180
CCTGGTGCTGTACTTTAGATGGTTGGGACCCCAACGCCTGATGTTCCTCGTGTTGTGGAA240
GCTTGCTCGGGGAGCTTTCCCGCTGGCCCTCTTGATGGGGATTTCGGCGACCCGCGGGCG300
CACCTCAGTGCTCGGGGCCGAGTTCTGCTTCGATGCTACATTCGAGGTGGACACTTCGGT360
GTTGGGCTGGGTGGTGGCCAGTGTGGTAGCTTGGGCCATTGCGCTCCTGAGCTCGATGAG420
CGCAGGGGGGTGGAGGCACAAAGCCGTGATCTATAGGACGTGGTGTAAGGGGTACCAGGC480
AATCCGTCAAAGGGTGGTGAGGAGCCCCCTCGGGGAGGGGCGGCCTGCCAAACCCCTGAC540
CTTTGCCTGGTGCTTGGCCTCGTACATCTGGCCAGATGCTGTGATGATGGTGGTGGTTGC600
CTTGGTCCTTCTCTTTGGCCTGTTCGACGCGTTGGATTGGGCCTTGGAGGAGATCTTGGT660
GTCCCGGCCCTCGTTGCGGCGTTTGGCTCGGGTGGTTGAGTGCTGTGTGATGGCGGGTGA720
GAAGGCCACAACCGTCCGGCTGGTCTCCAAGATGTGTGCGAGAGGAGCTTATTTGTTCGA780
TCATATGGGCTCTTTTTCGCGTGCTGTCAAGGAGCGCCTGTTGGAATGGGACGCAGCTCT840
TGAACCTCTGTCATTCACTAGGACGGACTGTCGCATCATACGGGATGCCGCGAGGACTTT900
GTCCTGCGGGCAGTGCGTCATGGGTTTACCCGTGGTTGCGCGCCGTGGTGATGAGGTTCT960
CATCGGCGTCTTCCAGGATGTGAATCATTTGCCTCCCGGGTTTGTTCCGACCGCGCCTGT1020
TGTCATCCGACGGTGCGGAAAGGGCTTCTTGGGGGTCACAAAGGCTGCCTTGACAGGTCG1080
GGATCCTGACTTACATCCAGGGAACGTCATGGTGTTGGGGACGGCTACGTCGCGAAGCAT1140
GGGAACATGCTTGAACGGCCTGCTGTTCACGACCTTCCATG1181
(2) INFORMATION FOR SEQ ID NO:99:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 30 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer Y5-10- F1
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:99:
TCAGCCATGGCTCGTGCGCCCGCGATGGTC30
(2) INFORMATION FOR SEQ ID NO:100:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 27 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer Y5-10- R1
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:100:
CGAGGATCCAGCCGCCGGCGGCAGATC27
(2) INFORMATION FOR SEQ ID NO:101:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 32 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer Y5- 16F1
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:101:
GATTCCATGGGTTTGGGGTTGACGGTGGCTGA32
(2) INFORMATION FOR SEQ ID NO:102:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 32 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer 470EP- R3
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:102:
GCGAATTCGGATCCCAAGGTTTCTTGCCTAGC32
(2) INFORMATION FOR SEQ ID NO:103:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 27 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer Y5-5- F1
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:103:
GAGGCCATGGCCTATTGTGACAAGGTG27
(2) INFORMATION FOR SEQ ID NO:104:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 17 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer PGEX- R
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:104:
GACCGTCTCCGGGAGCT17
(2) INFORMATION FOR SEQ ID NO:105:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 326 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Clone GE15
(ix) FEATURE:
(A) NAME/KEY: CDS
(B) LOCATION: 3..326
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:105:
CCATGGAGGTCTCTGACTTCCGTGGCTCGTCTGGCTCACCGGTCCTA47
MetGluValSerAspPheArgGlySerSerGlySerProValLeu
151015
TGTGACGAAGGGCACGCAGTAGGAATGCTCGTGTCTGTGCTTCACTCC95
CysAspGluGlyHisAlaValGlyMetLeuValSerValLeuHisSer
202530
GGTGGTAGGGTCACCGCGGCACGGTTCACTAGGCCGTGGACCCAAGTG143
GlyGlyArgValThrAlaAlaArgPheThrArgProTrpThrGlnVal
354045
CCAACAGATGCCAAAACCACCACTGAACCCCCTCCGGTGCCGGCCAAA191
ProThrAspAlaLysThrThrThrGluProProProValProAlaLys
505560
GGAGTTTTCAAAGAGGCCCCGTTGTTTATGCCTACGGGAGCGGGAAAG239
GlyValPheLysGluAlaProLeuPheMetProThrGlyAlaGlyLys
657075
AGCACTCGCGTCCCGTTGGAGTACGGCAACATGGGGCACAAGGTCTTA287
SerThrArgValProLeuGluTyrGlyAsnMetGlyHisLysValLeu
80859095
ATCTTGAACCCCTCAGTGGCCACTGTGCGGGCGATGGGC326
IleLeuAsnProSerValAlaThrValArgAlaMetGly
100105
(2) INFORMATION FOR SEQ ID NO:106:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 108 amino acids
(B) TYPE: amino acid
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: protein
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:106:
MetGluValSerAspPheArgGlySerSerGlySerProValLeuCys
151015
AspGluGlyHisAlaValGlyMetLeuValSerValLeuHisSerGly
202530
GlyArgValThrAlaAlaArgPheThrArgProTrpThrGlnValPro
354045
ThrAspAlaLysThrThrThrGluProProProValProAlaLysGly
505560
ValPheLysGluAlaProLeuPheMetProThrGlyAlaGlyLysSer
65707580
ThrArgValProLeuGluTyrGlyAsnMetGlyHisLysValLeuIle
859095
LeuAsnProSerValAlaThrValArgAlaMetGly
100105
(2) INFORMATION FOR SEQ ID NO:107:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 138 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Clone GE17
(ix) FEATURE:
(A) NAME/KEY: CDS
(B) LOCATION: 1..138
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:107:
GGTGATGAGGTTCTCATCGGCGTCTTCCAGGATGTGAATCATTTGCCT48
GlyAspGluValLeuIleGlyValPheGlnAspValAsnHisLeuPro
151015
CCCGGGTTTGTTCCGACCGCGCCTGTTGTCATCCGACGGTGCGGAAAG96
ProGlyPheValProThrAlaProValValIleArgArgCysGlyLys
202530
GGCTTCTTGGGGGTCACAAAGGCTGCCTTGACAGGTCGGGAT138
GlyPheLeuGlyValThrLysAlaAlaLeuThrGlyArgAsp
354045
(2) INFORMATION FOR SEQ ID NO:108:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 46 amino acids
(B) TYPE: amino acid
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: protein
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:108:
GlyAspGluValLeuIleGlyValPheGlnAspValAsnHisLeuPro
151015
ProGlyPheValProThrAlaProValValIleArgArgCysGlyLys
202530
GlyPheLeuGlyValThrLysAlaAlaLeuThrGlyArgAsp
354045
(2) INFORMATION FOR SEQ ID NO:109:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 395 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Consensus Sequence 9E3
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:109:
TGTATTTGTCCTGTTATACCTGATGAAGCTGGCTGAGGCACGGTTGGTCCCGCTGATCTT60
GCTGCTGCTATGGTGGTGGGTGAACCAGCTGGCAGTCCTAGGGCTGCCGGCTGTGGAAGC120
CGCCGTGGCAGGTGAGGTCTTCGCGGGCCCTGCCCTGTCCTGGTGTCTGGGACTCCCGGT180
CGTCAGTATGATATTGGGTTTGGCAAACCTAGTGCTGTACTTTAGATGGTTGGGACCCCA240
ACGCCTGATGTTCCTCGTGTTGTGGAAGCTTGCTCGGGGAGCTTTCCCGCTGGCCCTCTT300
GATGGGGATTTCGGCGACCCGCGGGCGCACCTCAGTGCTCGGGGCCGAGTTCTGCTTCGA360
TGCTACATTCGAGGTGGACACTTCGGTGTTGGGCT395
(2) INFORMATION FOR SEQ ID NO:110:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 460 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Consensus Sequence 10E3
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:110:
GCCCCTGGGCAACCAGGGCCGAGGCAACCCGGTGCGGTCGCCCTTGGGTTTTGGGTCCTA60
CGCCATGACCAGGATCCGAGATACCCTACATCTGGTGGAGTGTCCCACACCAGCCATTGA120
GCCTCCCACCGGGACGTTTGGGTTCTTCCCCGGGACGCCGCCTCTCAACAACTGCATGCT180
CTTGGGCACGGAAGTGTCCGAGGCACTTGGGGGGGCTGGCCTCACGGGGGGGTTCTATGA240
ACCCCTGGTGCGCAGGTGTTCGAAGCTGATGGGAAGCCGAAATCCGGTTTGTCCGGGGTT300
TGCATGGCTCTCTTCGGGCAGGCCTGATGGGTTTATACATGTCCAGGGTCACTTGCAGGA360
GGTGGATGCAGGCAACTTCATCCCGCCCCCGCGCTGGTTGCTCTTGGACTTTGTATTTGT420
CCTGTTATACCTGATGAAGCTGGCTGAGGCACGGTTGGTC460
(2) INFORMATION FOR SEQ ID NO:111:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 28 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer GE15F
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:111:
GCCGCCATGGAGGTCTCTGACTTCCGTG28
(2) INFORMATION FOR SEQ ID NO:112:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 31 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer GE15R
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:112:
GCGCGGATCCGCCCATCGCCCGCACAGTGGC31
(2) INFORMATION FOR SEQ ID NO:113:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 31 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer GE17F
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:113:
CGCTCCATGGGTGATGAGGTTCTCATCGGCG31
(2) INFORMATION FOR SEQ ID NO:114:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 28 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer GE17R
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:114:
GTAAGTCAGGATCCCGACCTGTCAAGGC28
(2) INFORMATION FOR SEQ ID NO:115:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 452 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: double
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: NcoI/EcoRI- containing fragment of
pGEX-HISb- GE3-s HGV plasmid
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:115:
CAAAATCGGATCTGGTTCCGCGTGGTTCCATGGTCTCATGGGACGCGGACGCTCGTGCGC60
CCGCGATGGTCTATGGCCCTGGGCAAAGTGTTACCATTGACGGGGAGCGCTACACCTTGC120
CTCATCAACTGAGGCTCAGGAATGTGGCACCCTCTGAGGTTTCATCCGAGGTGTCCATTG180
ACATTGGGACGGAGACTGAAGACTCAGAACTGACTGAGGCCGATCTGCCGCCGGCGGCTG240
CTGCTCTCCAAGCGATCGAGAATGCTGCGAGGATTCTTGAACCGCACATTGATGTCATCA300
TGGAGGACTGCAGTACACCCTCTCTTTGTGGTAGTAGCCGAGAGATGCCTGTATGGGGAG360
AAGACATCCCCCGTACTCCATCGCCAGCACTTATCGGATCCCACCATCACCATCACCATT420
AGAATTCATCGTGACTGACTGACGATCTACCT452
(2) INFORMATION FOR SEQ ID NO:116:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 590 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Consensus Sequence 11E3
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:116:
AGCAATCGGCTGGGGTGACCCCATCACTTATTGGAGCCACGGGCAAAATCAGTGGCCCCT60
TTCATGCCCCCAGTATGTCTATGGGTCTGCTACAGTCACTTGCGTGTGGGGTTCCGCTTC120
TTGGTTTGCCTCCACCAGTGGTCGCGACTCGAAGATAGATGTGTGGAGTTTAGTGCCAGT180
TGGCTCTGCCACCTGCACCATAGCCGCACTTGGATCATCGGATCGCGACACGGTGCCTGG240
GCTCTCCGAGTGGGGAATCCCGTGCGTGACGTGTGTTCTGGACCGTCGGCCTGCCTCCTG300
CGGCACCTGTGTGAGGGACTGCTGGCCCGAGACCGGGTCGGTTAGGTTCCCATTCCATCG360
GTGCGGCGTGGGGCCTCGGCTGACAAAGGACTTGGAAGCTGTGCCCTTCGTCAACAGGAC420
AACTCCCTTCACCATTAGGGGGCCCCTGGGCAACCAGGGCCGAGGCAACCCGGTGCGGTC480
GCCCTTGGGTTTTGGGTCCTACGCCATGACCAGGATCCGAGATACCCTACATCTGGTGGA540
GTGTCCCACACCAGCCATCGAGCCTCCCACCGGGACGTTTGGGTTCTTCC590
(2) INFORMATION FOR SEQ ID NO:117:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 29 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Probe E3- 111PROB
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:117:
TGGTGAAGGGAGTTGTCCTATTGACGAAG29
(2) INFORMATION FOR SEQ ID NO:118:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 735 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Consensus Sequence 12E3
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:118:
ATTGTTGTGCCCCGGAGGACATCGGGTTCTGCCTGGAGGGTGGATGCCTGGTGGCCCTGG60
GGTGCACGATTTGCACTGACCAATGCTGGCCACTGTATCAGGCGGGTTTGGCTGTGCGGC120
CTGGCAAGTCCGCGGCCCAACTGGTGGGGGAGCTGGGTAGCCTATACGGGCCCCTGTCGG180
TCTCGGCCTATGTGGCTGGGATCCTGGGCCTGGGTGAGGTGTACTCGGGTGTCCTAACGG240
TGGGAGTCGCGTTGACGCGCCGGGTCTACCCGGTGCCTAACCTGACGTGTGCAGTCGCGT300
GTGAGCTAAAGTGGGAAAGTGAGTTTTGGAGATGGACTGAACAGCTGGCCTCCAACTACT360
GGATTCTGGAATACCTCTGGAAGGTCCCATTTGATTTCTGGAGAGGCGTGATAAGCCTGA420
CCCCCTTGTTGGTTTGCGTGGCCGCATTGCTGCTGCTTGAGCAACGGATTGTCATGGTCT480
TCCTGTTGGTGACGATGGCCGGGATGTCGCAAGGCGCCCCTGCCTCCGTTTTGGGGTCAC540
GCCCCTTTGACTACGGGTTGACTTGGCAGACCTGCTCTTGCAGGGCCAACGGTTCGCGTT600
TTTCGACTGGGGAGAAGGTGTGGGACCGTGGGAACGTTACGCTTCAGTGTGACTGCCCTA660
ACGGCCCCTGGGTGTGGTTGCCAGCCTTTTGCCAAGCAATCGGCTGGGGTGACCCCATCA720
CTTATTGGAGCCACG735
(2) INFORMATION FOR SEQ ID NO:119:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 22 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer 470EXT4-2189R
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:119:
ATCTGTGGTATGCCATCCCGGT22
(2) INFORMATION FOR SEQ ID NO:120:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 23 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer 470EXT4-29F
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:120:
GTTATGCTACTGTCGAAGCAGGT23
(2) INFORMATION FOR SEQ ID NO:121:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 24 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: NS5 Primer GV57-4512 MF
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:121:
GGACTTCCGGATAGCTGARAAGCT24
(2) INFORMATION FOR SEQ ID NO:122:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 20 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: NS5 Primer GV57-4657 MR
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:122:
GCRTCCACACAGATGGCGCA20
(2) INFORMATION FOR SEQ ID NO:123:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 28 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: NS5 Probe GV22dc-89 MF
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:123:
CYCGCTGRTTTGGGGTGTACTGGAAGGC28
(2) INFORMATION FOR SEQ ID NO:124:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 31 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: 5'-UTR Primer FV94-22F
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:124:
GAAAGCCCCAGAAACCGACGCCTATCTAAGT31
(2) INFORMATION FOR SEQ ID NO:125:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 26 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: 5'UTR Primer FV94-724R
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:125:
GCACAGCCAAACCCGCCTGATACAGT26
(2) INFORMATION FOR SEQ ID NO:126:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 28 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: 5'-UTR Primer FV94-94F
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:126:
GTGGTGGATGGGTGATGACAGGGTTGGT28
(2) INFORMATION FOR SEQ ID NO:127:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 29 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: 5'-UTR Primer FV94-912R
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:127:
TAACTCACACGCGACTGCACACGTCAGGT29
(2) INFORMATION FOR SEQ ID NO:128:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 30 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: ENV Library Primer GEP-F15
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:128:
GCGGCCATGGTGCCCTTCGTCAATAGGACA30
(2) INFORMATION FOR SEQ ID NO:129:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 29 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: ENV Library Primer GEP-R15
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:129:
CTTGCCATGGCCAGCTGGTTCACCCACCA29
(2) INFORMATION FOR SEQ ID NO:130:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 30 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer GEP- F17
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:130:
GCAGGATCCCCTCTGGAAGGTCCCATTTGA30
(2) INFORMATION FOR SEQ ID NO:131:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 27 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer GEP- R16
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:131:
TGCGAATCCTCGGCCCTGGTTGCCCAG27
(2) INFORMATION FOR SEQ ID NO:132:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 30 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer 470ep- F9
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:132:
GCTAGATCTGGCAACATGGGGCACAAGGTC30
(2) INFORMATION FOR SEQ ID NO:133:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 30 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer 470ep- R9
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:133:
CACAGATCTCGCGTAGTAGTAGCGTCCAGA30
(2) INFORMATION FOR SEQ ID NO:134:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 38 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: AP Primer for Race PCR
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:134:
CTGGTTCGGCCCACCTCTGAAGGTTCCAGAATCGATAG38
(2) INFORMATION FOR SEQ ID NO:135:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 30 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer GEP- F10
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:135:
GCTGGATCCAGCATGGGAACATGCTTGAAC30
(2) INFORMATION FOR SEQ ID NO:136:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 30 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer GEP- R10
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:136:
CGCGGATCCCACAGTGGCCACTGAGGGGTT30
(2) INFORMATION FOR SEQ ID NO:137:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 27 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer EXY10- F1
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:137:
GCCCATATGGTGATCACTGGTGACGTT27
(2) INFORMATION FOR SEQ ID NO:138:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 24 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer EXY10- F2
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:138:
GCCCATATGCTGGGTTACGGTGAA24
(2) INFORMATION FOR SEQ ID NO:139:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 27 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer EXY10- F3
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:139:
GCCCATATGACCTCCGCCTATAAGCTG27
(2) INFORMATION FOR SEQ ID NO:140:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 27 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer EXY10- R1
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:140:
GCCCATATGAGCCGCCGGCGGCAGATC27
(2) INFORMATION FOR SEQ ID NO:141:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 24 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer EXY5- R1
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:141:
TGCGGATCCCACATTGTCTGGATT24
(2) INFORMATION FOR SEQ ID NO:142:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 27 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer Y5-5- F1
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:142:
TCGGCCATGGCCTATTGTGACAAGGTG27
(2) INFORMATION FOR SEQ ID NO:143:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 219 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Antigen Clone Q7-12-1
(ix) FEATURE:
(A) NAME/KEY: CDS
(B) LOCATION: 1..219
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:143:
GTGCCCTTCGTCAATAGGACAACTCTCTTCACCATTAGGGGGCCCCTG48
ValProPheValAsnArgThrThrLeuPheThrIleArgGlyProLeu
151015
GGCAACCAGGGCCGAGGCAACCCGGTGCGGTCGCCCTTGGGTTTTGGG96
GlyAsnGlnGlyArgGlyAsnProValArgSerProLeuGlyPheGly
202530
TCCTACGCCATGACCAGGATCCGAGATACCCTACATCTGGTGGAGTGT144
SerTyrAlaMetThrArgIleArgAspThrLeuHisLeuValGluCys
354045
CCCACACCAGCCATCGAGCCTCCCACCGGGACGTCTGGGTTCTTCCCC192
ProThrProAlaIleGluProProThrGlyThrSerGlyPhePhePro
505560
GGGACGCCGCCTCTCAACAGCTGCATG219
GlyThrProProLeuAsnSerCysMet
6570
(2) INFORMATION FOR SEQ ID NO:144:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 73 amino acids
(B) TYPE: amino acid
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: protein
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:144:
ValProPheValAsnArgThrThrLeuPheThrIleArgGlyProLeu
151015
GlyAsnGlnGlyArgGlyAsnProValArgSerProLeuGlyPheGly
202530
SerTyrAlaMetThrArgIleArgAspThrLeuHisLeuValGluCys
354045
ProThrProAlaIleGluProProThrGlyThrSerGlyPhePhePro
505560
GlyThrProProLeuAsnSerCysMet
6570
(2) INFORMATION FOR SEQ ID NO:145:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 264 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Antigen Clone Y12-10-3
(ix) FEATURE:
(A) NAME/KEY: CDS
(B) LOCATION: 1..264
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:145:
CCCCTCGAGCGGATGCGAACCGGAAGGCACCTCGTGTTCTGCCATTCT48
ProLeuGluArgMetArgThrGlyArgHisLeuValPheCysHisSer
151015
AAGGCTGAGTGCGAGCGCCTTGCTGGCCAGTTCTCCGCTAGGGGGGTC96
LysAlaGluCysGluArgLeuAlaGlyGlnPheSerAlaArgGlyVal
202530
AATGCCATTGCCTATTATAGGGGTAAAGACAGCTCTATCATCAAGGAT144
AsnAlaIleAlaTyrTyrArgGlyLysAspSerSerIleIleLysAsp
354045
GGGGACCTGGTGGTCTGTGCTACAGACGCGCTTTCCACTGGGTACACT192
GlyAspLeuValValCysAlaThrAspAlaLeuSerThrGlyTyrThr
505560
GGAAATTTCGACTCCGTCACCGACTGTGGATTAGTGGTGGAGGAGGTC240
GlyAsnPheAspSerValThrAspCysGlyLeuValValGluGluVal
65707580
GTTGAGGTGACCCTTGATCCCACC264
ValGluValThrLeuAspProThr
85
(2) INFORMATION FOR SEQ ID NO:146:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 88 amino acids
(B) TYPE: amino acid
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: protein
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:146:
ProLeuGluArgMetArgThrGlyArgHisLeuValPheCysHisSer
151015
LysAlaGluCysGluArgLeuAlaGlyGlnPheSerAlaArgGlyVal
202530
AsnAlaIleAlaTyrTyrArgGlyLysAspSerSerIleIleLysAsp
354045
GlyAspLeuValValCysAlaThrAspAlaLeuSerThrGlyTyrThr
505560
GlyAsnPheAspSerValThrAspCysGlyLeuValValGluGluVal
65707580
ValGluValThrLeuAspProThr
85
(2) INFORMATION FOR SEQ ID NO:147:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 205 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Antigen Clone Y12-15-1
(ix) FEATURE:
(A) NAME/KEY: CDS
(B) LOCATION: 1..205
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:147:
GCTAGATCTGGCAACATGGGGCACAAGGTCTTAATCTTGAACCCCTCA48
AlaArgSerGlyAsnMetGlyHisLysValLeuIleLeuAsnProSer
151015
GTGGCCACTGTGCGGGCCATGGGCCCGTACATGGAGCGGCTGGCGGGT96
ValAlaThrValArgAlaMetGlyProTyrMetGluArgLeuAlaGly
202530
AAACATCCAAGTATATACTGTGGGCATGATACAACTGCTTTCACAAGG144
LysHisProSerIleTyrCysGlyHisAspThrThrAlaPheThrArg
354045
ATCACTGACTCCCCCCTGACGTATTCAACCTATGGGAGGTTTTTGGCC192
IleThrAspSerProLeuThrTyrSerThrTyrGlyArgPheLeuAla
505560
AACCCTAGGCAGA205
AsnProArgGln
65
(2) INFORMATION FOR SEQ ID NO:148:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 68 amino acids
(B) TYPE: amino acid
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: protein
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:148:
AlaArgSerGlyAsnMetGlyHisLysValLeuIleLeuAsnProSer
151015
ValAlaThrValArgAlaMetGlyProTyrMetGluArgLeuAlaGly
202530
LysHisProSerIleTyrCysGlyHisAspThrThrAlaPheThrArg
354045
IleThrAspSerProLeuThrTyrSerThrTyrGlyArgPheLeuAla
505560
AsnProArgGln
65
(2) INFORMATION FOR SEQ ID NO:149:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 32 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer GE4F
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:149:
GCCGCCATGGCTCTCCAAGCGATCGAGAATGC32
(2) INFORMATION FOR SEQ ID NO:150:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 31 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer GE4R
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:150:
GCGCGGATCCCAACCCCAATGAGAAAAAGCG31
(2) INFORMATION FOR SEQ ID NO:151:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 23 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer 470EXP3F
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:151:
CCGCCATGGGACGCGGACGCTCG23
(2) INFORMATION FOR SEQ ID NO:152:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 28 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer 470EXP3R
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:152:
CGCGGATCCTTACTGTCTTATTGCTTCC28
(2) INFORMATION FOR SEQ ID NO:153:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 34 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer FV94- 2888F
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:153:
GCGGAATTCTTGGCTCGGGTGGTTGAGTGCTGTG34
(2) INFORMATION FOR SEQ ID NO:154:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 32 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer FV94- 3216R
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:154:
GCGAAGCTTCCGTCGGATGACAACAGGCGCGG32
(2) INFORMATION FOR SEQ ID NO:155:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 36 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer FV94- 6521F
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:155:
GCGGAATTCACCTCCGCCTATAAGCTGCTGCGCCAG36
(2) INFORMATION FOR SEQ ID NO:156:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 42 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer FV94- 7483R
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:156:
GCTGCGGCCGCCCTCCGTCCCACATTGTCTGGATTGGTAACA42
(2) INFORMATION FOR SEQ ID NO:157:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 22 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer T7F
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:157:
ATTAATACGACTCACTATAGGG22
(2) INFORMATION FOR SEQ ID NO:158:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 22 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer T7R
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:158:
CAAGGGGTTATGCTAGTTATTG22
(2) INFORMATION FOR SEQ ID NO:159:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 402 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Antigen Clone GE4-8
(ix) FEATURE:
(A) NAME/KEY: CDS
(B) LOCATION: 1..402
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:159:
GCTCTCCAAGCGATCGAGAATGCTGCGAGGATTCTTGAACCGCACATT48
AlaLeuGlnAlaIleGluAsnAlaAlaArgIleLeuGluProHisIle
151015
GATGTCATCATGGAGGACTGCAGTACACCCTCTCTTTGTGGTAGTAGC96
AspValIleMetGluAspCysSerThrProSerLeuCysGlySerSer
202530
CGAGAGATGCCTGTATGGGGAGAAGACATCCCCCGTACTCCATCGCCA144
ArgGluMetProValTrpGlyGluAspIleProArgThrProSerPro
354045
GCACTTATCTCGGTTACTGAGAGCAGCTCAGATGAGAAGACCCCGTCG192
AlaLeuIleSerValThrGluSerSerSerAspGluLysThrProSer
505560
GTGTCCTCCTCGCAGGAGGATACCCCGTCCTCTGACTCATTCGAGGTC240
ValSerSerSerGlnGluAspThrProSerSerAspSerPheGluVal
65707580
ATCCAAGAGTCCGAGACAGCCGAAGGGGAGGAAAGTGTCTTCAACGTG288
IleGlnGluSerGluThrAlaGluGlyGluGluSerValPheAsnVal
859095
GCTCTTTCCGTATTAAAAGCCTTATTTCCACAGAGCGACGCGACCAGG336
AlaLeuSerValLeuLysAlaLeuPheProGlnSerAspAlaThrArg
100105110
AAGCTTACCGTCAAGATGTCGTGCTGCGTTGAAAAGAGCGTCACGCGC384
LysLeuThrValLysMetSerCysCysValGluLysSerValThrArg
115120125
TTTTTCTCATTGGGGTTG402
PhePheSerLeuGlyLeu
130
(2) INFORMATION FOR SEQ ID NO:160:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 134 amino acids
(B) TYPE: amino acid
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: protein
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:160:
AlaLeuGlnAlaIleGluAsnAlaAlaArgIleLeuGluProHisIle
151015
AspValIleMetGluAspCysSerThrProSerLeuCysGlySerSer
202530
ArgGluMetProValTrpGlyGluAspIleProArgThrProSerPro
354045
AlaLeuIleSerValThrGluSerSerSerAspGluLysThrProSer
505560
ValSerSerSerGlnGluAspThrProSerSerAspSerPheGluVal
65707580
IleGlnGluSerGluThrAlaGluGlyGluGluSerValPheAsnVal
859095
AlaLeuSerValLeuLysAlaLeuPheProGlnSerAspAlaThrArg
100105110
LysLeuThrValLysMetSerCysCysValGluLysSerValThrArg
115120125
PhePheSerLeuGlyLeu
130
(2) INFORMATION FOR SEQ ID NO:161:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 1011 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Antigen Clone EXP3-7
(ix) FEATURE:
(A) NAME/KEY: CDS
(B) LOCATION: 1..1011
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:161:
ATGGTCTATGGCCCTGGGCAAAGTGTTACCATTGACGGGGAGCGCTAC48
MetValTyrGlyProGlyGlnSerValThrIleAspGlyGluArgTyr
151015
ACCTTGCCTCATCAACTGAGGCTCAGGAATGTGGCACCCTCTGAGGTT96
ThrLeuProHisGlnLeuArgLeuArgAsnValAlaProSerGluVal
202530
TCATCCGAGGTGTCCATTGACATTGGGACGGAGACTGAAGACTCAGAA144
SerSerGluValSerIleAspIleGlyThrGluThrGluAspSerGlu
354045
CTGACTGAGGCCGATCTGCCGCCGGCGGCTGCTGCTCTCCAAGCGATC192
LeuThrGluAlaAspLeuProProAlaAlaAlaAlaLeuGlnAlaIle
505560
GAGAATGCTGCGAGGATTCTTGAACCGCACATTGATGTCATCATGGAG240
GluAsnAlaAlaArgIleLeuGluProHisIleAspValIleMetGlu
65707580
GACTGCAGTACACCCTCTCTTTGTGGTAGTAGCCGAGAGATGCCTGTA288
AspCysSerThrProSerLeuCysGlySerSerArgGluMetProVal
859095
TGGGGAGAAGACATCCCCCGTACTCCATCGCCAGCACTTATCTCGGTT336
TrpGlyGluAspIleProArgThrProSerProAlaLeuIleSerVal
100105110
ACTGAGAGCAGCTCAGATGAGAAGACCCCGTCGGTGTCCTCCTCGCAG384
ThrGluSerSerSerAspGluLysThrProSerValSerSerSerGln
115120125
GAGGATACCCCGTCCTCTGACTCATTCGAGGTCATCCAAGAGTCCGAG432
GluAspThrProSerSerAspSerPheGluValIleGlnGluSerGlu
130135140
ACAGCCGAAGGGGAGGAAAGTGTCTTCAACGTGGCTCTTTCCGTATTA480
ThrAlaGluGlyGluGluSerValPheAsnValAlaLeuSerValLeu
145150155160
AAAGCCTTATTTCCACAGAGCGACGCGACCAGGAAGCTTACCGTCAAG528
LysAlaLeuPheProGlnSerAspAlaThrArgLysLeuThrValLys
165170175
ATGTCGTGCTGCGTTGAAAAGAGCGTCACGCGCTTTTTCTCATTGGGG576
MetSerCysCysValGluLysSerValThrArgPhePheSerLeuGly
180185190
TTGACGGTGGCTGATGTTGCTAGCCTGTGTGAGATGGAAATCCAGAAC624
LeuThrValAlaAspValAlaSerLeuCysGluMetGluIleGlnAsn
195200205
CATACAGCCTATTGTGACCAGGTGCGCACTCCGCTTGAATTGCAGGTT672
HisThrAlaTyrCysAspGlnValArgThrProLeuGluLeuGlnVal
210215220
GGGTGCTTGGTGGGCAATGAACTTACCTTTGAATGTGACAAGTGTGAG720
GlyCysLeuValGlyAsnGluLeuThrPheGluCysAspLysCysGlu
225230235240
GCTAGGCAAGAAACCTTGGCCTCCTTCTCTTACATTTGGTCTGGAGTG768
AlaArgGlnGluThrLeuAlaSerPheSerTyrIleTrpSerGlyVal
245250255
CCGCTGACTAGGGCCACGCCGGCCAAGCCTCCCGTGGTGAGGCCGGTT816
ProLeuThrArgAlaThrProAlaLysProProValValArgProVal
260265270
GGCTCTTTGTTAGTGGCCGACACTACTAAGGTGTATGTTACCAATCCA864
GlySerLeuLeuValAlaAspThrThrLysValTyrValThrAsnPro
275280285
GACAATGTGGGACGGAGGGTGGACAAGGTGACCTTCTGGCGTGCTCCT912
AspAsnValGlyArgArgValAspLysValThrPheTrpArgAlaPro
290295300
AGGGTTCATGATAAGTACCTCGTGGACTCTATTGAGCGCGCTAAGAGG960
ArgValHisAspLysTyrLeuValAspSerIleGluArgAlaLysArg
305310315320
GCCGCTCAAGCCTGCCTAAGCATGGGTTACACTTATGAGGAAGCAATA1008
AlaAlaGlnAlaCysLeuSerMetGlyTyrThrTyrGluGluAlaIle
325330335
AGG1011
Arg
(2) INFORMATION FOR SEQ ID NO:162:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 337 amino acids
(B) TYPE: amino acid
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: protein
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:162:
MetValTyrGlyProGlyGlnSerValThrIleAspGlyGluArgTyr
151015
ThrLeuProHisGlnLeuArgLeuArgAsnValAlaProSerGluVal
202530
SerSerGluValSerIleAspIleGlyThrGluThrGluAspSerGlu
354045
LeuThrGluAlaAspLeuProProAlaAlaAlaAlaLeuGlnAlaIle
505560
GluAsnAlaAlaArgIleLeuGluProHisIleAspValIleMetGlu
65707580
AspCysSerThrProSerLeuCysGlySerSerArgGluMetProVal
859095
TrpGlyGluAspIleProArgThrProSerProAlaLeuIleSerVal
100105110
ThrGluSerSerSerAspGluLysThrProSerValSerSerSerGln
115120125
GluAspThrProSerSerAspSerPheGluValIleGlnGluSerGlu
130135140
ThrAlaGluGlyGluGluSerValPheAsnValAlaLeuSerValLeu
145150155160
LysAlaLeuPheProGlnSerAspAlaThrArgLysLeuThrValLys
165170175
MetSerCysCysValGluLysSerValThrArgPhePheSerLeuGly
180185190
LeuThrValAlaAspValAlaSerLeuCysGluMetGluIleGlnAsn
195200205
HisThrAlaTyrCysAspGlnValArgThrProLeuGluLeuGlnVal
210215220
GlyCysLeuValGlyAsnGluLeuThrPheGluCysAspLysCysGlu
225230235240
AlaArgGlnGluThrLeuAlaSerPheSerTyrIleTrpSerGlyVal
245250255
ProLeuThrArgAlaThrProAlaLysProProValValArgProVal
260265270
GlySerLeuLeuValAlaAspThrThrLysValTyrValThrAsnPro
275280285
AspAsnValGlyArgArgValAspLysValThrPheTrpArgAlaPro
290295300
ArgValHisAspLysTyrLeuValAspSerIleGluArgAlaLysArg
305310315320
AlaAlaGlnAlaCysLeuSerMetGlyTyrThrTyrGluGluAlaIle
325330335
Arg
(2) INFORMATION FOR SEQ ID NO:163:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 351 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Antigen Clone GENS2b-1
(ix) FEATURE:
(A) NAME/KEY: CDS
(B) LOCATION: 1..351
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:163:
TTGGCTCGGGTGGTTGAGTGCTGTGTGATGGCGGGTGAGAAGGCCACA48
LeuAlaArgValValGluCysCysValMetAlaGlyGluLysAlaThr
151015
ACCGTCCGGCTGGTCTCCAAGATGTGTGCGAGAGGAGCTTATTTGTTC96
ThrValArgLeuValSerLysMetCysAlaArgGlyAlaTyrLeuPhe
202530
GATCATATGGGCTCTTTTTCGCGTGCTGTCAAGGAGCGCCTGTTGGAA144
AspHisMetGlySerPheSerArgAlaValLysGluArgLeuLeuGlu
354045
TGGGACGCAGCTCTTGAACCTCTGTCATTCACTAGGACGGACTGTCGC192
TrpAspAlaAlaLeuGluProLeuSerPheThrArgThrAspCysArg
505560
ATCATACGGGATGCCGCGAGGACTTTGTCCTGCGGGCAGTGCGTCATG240
IleIleArgAspAlaAlaArgThrLeuSerCysGlyGlnCysValMet
65707580
GGTTTACCCGTGGTTGCGCGCCGTGGTGATGAGGTTCTCATCGGCGTC288
GlyLeuProValValAlaArgArgGlyAspGluValLeuIleGlyVal
859095
TTCCAGGATGTGAATCATTTGCCTCCCGGGTTTGTTCCGACCGCGCCT336
PheGlnAspValAsnHisLeuProProGlyPheValProThrAlaPro
100105110
GTTGTCATCCGACGG351
ValValIleArgArg
115
(2) INFORMATION FOR SEQ ID NO:164:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 117 amino acids
(B) TYPE: amino acid
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: protein
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:164:
LeuAlaArgValValGluCysCysValMetAlaGlyGluLysAlaThr
151015
ThrValArgLeuValSerLysMetCysAlaArgGlyAlaTyrLeuPhe
202530
AspHisMetGlySerPheSerArgAlaValLysGluArgLeuLeuGlu
354045
TrpAspAlaAlaLeuGluProLeuSerPheThrArgThrAspCysArg
505560
IleIleArgAspAlaAlaArgThrLeuSerCysGlyGlnCysValMet
65707580
GlyLeuProValValAlaArgArgGlyAspGluValLeuIleGlyVal
859095
PheGlnAspValAsnHisLeuProProGlyPheValProThrAlaPro
100105110
ValValIleArgArg
115
(2) INFORMATION FOR SEQ ID NO:165:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 993 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Antigen Clone GENS5a-3
(ix) FEATURE:
(A) NAME/KEY: CDS
(B) LOCATION: 1..993
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:165:
ACCTCCGCCTATAAGCTGCTGCGCCAGCAAATCCTATCGGCTGCTGTA48
ThrSerAlaTyrLysLeuLeuArgGlnGlnIleLeuSerAlaAlaVal
151015
GCTGAGCCCTACTACGTCGACGGCATTCCGGTCTCATGGGACGCGGAC96
AlaGluProTyrTyrValAspGlyIleProValSerTrpAspAlaAsp
202530
GCTCGTGCGCCCGCCATGGTCTATGGCCCTGGGCAAAGTGTTACCATT144
AlaArgAlaProAlaMetValTyrGlyProGlyGlnSerValThrIle
354045
GACGGGGAGCGCTACACCTTGCCTCATCAACTGAGGCTCAGGAATGTG192
AspGlyGluArgTyrThrLeuProHisGlnLeuArgLeuArgAsnVal
505560
GCACCCTCTGAGGTTTCATCCGAGGTGTCCATTGACATTGGGACGGAG240
AlaProSerGluValSerSerGluValSerIleAspIleGlyThrGlu
65707580
ACTGAAGACTCAGAACTGACTGAGGCCGATCTGCCGCCGGCGGCTGCT288
ThrGluAspSerGluLeuThrGluAlaAspLeuProProAlaAlaAla
859095
GCTCTCCAAGCGATCGAGAATGCTGCGAGGATTCTTGAACCGCACATT336
AlaLeuGlnAlaIleGluAsnAlaAlaArgIleLeuGluProHisIle
100105110
GATGTCATCATGGAGGACTGCAGTACACCCTCTCTTTGTGGTAGTAGC384
AspValIleMetGluAspCysSerThrProSerLeuCysGlySerSer
115120125
CGAGAGATGCCTGTATGGGGAGAAGACATCCCCCGTACTCCATCGCCA432
ArgGluMetProValTrpGlyGluAspIleProArgThrProSerPro
130135140
GCACTTATCTCGGTTACTGAGAGCAGCTCAGATGAGAAGACCCCGTCG480
AlaLeuIleSerValThrGluSerSerSerAspGluLysThrProSer
145150155160
GTGTCCTCCTCGCAGGAGGATACCCCGTCCTCTGACTCATTCGAGGTC528
ValSerSerSerGlnGluAspThrProSerSerAspSerPheGluVal
165170175
ATCCAAGAGTCCGAGACAGCCGAAGGGGAGGAAAGTGTCTTCAACGTG576
IleGlnGluSerGluThrAlaGluGlyGluGluSerValPheAsnVal
180185190
GCTCTTTCCGTATTAAAAGCCTTATTTCCACAGAGCGACGCGACCAGG624
AlaLeuSerValLeuLysAlaLeuPheProGlnSerAspAlaThrArg
195200205
AAGCTTACCGTCAAGATGTCGTGCTGCGTTGAAAAGAGCGTCACGCGC672
LysLeuThrValLysMetSerCysCysValGluLysSerValThrArg
210215220
TTTTTCTCATTGGGGTTGACGGTGGCTGATGTTGCTAGCCTGTGTGAG720
PhePheSerLeuGlyLeuThrValAlaAspValAlaSerLeuCysGlu
225230235240
ATGGAAATCCAGAACCATACAGCCTATTGTGACCAGGTGCGCACTCCG768
MetGluIleGlnAsnHisThrAlaTyrCysAspGlnValArgThrPro
245250255
CTTGAATTGCAGGTTGGGTGCTTGGTGGGCAATGAACTTACCTTTGAA816
LeuGluLeuGlnValGlyCysLeuValGlyAsnGluLeuThrPheGlu
260265270
TGTGACAAGTGTGAGGCTAGGCAAGAAACCTTGGCCTCCTTCTCTTAC864
CysAspLysCysGluAlaArgGlnGluThrLeuAlaSerPheSerTyr
275280285
ATTTGGTCTGGAGTGCCGCTGACTAGGGCCACGCCGGCCAAGCCTCCC912
IleTrpSerGlyValProLeuThrArgAlaThrProAlaLysProPro
290295300
GTGGTGAGGCCGGTTGGCTCTTTGTTAGTGGCCGACACTACTAAGGTG960
ValValArgProValGlySerLeuLeuValAlaAspThrThrLysVal
305310315320
TATGTTACCAATCCAGACAATGTGGGACGGAGG993
TyrValThrAsnProAspAsnValGlyArgArg
325330
(2) INFORMATION FOR SEQ ID NO:166:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 331 amino acids
(B) TYPE: amino acid
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: protein
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:166:
ThrSerAlaTyrLysLeuLeuArgGlnGlnIleLeuSerAlaAlaVal
151015
AlaGluProTyrTyrValAspGlyIleProValSerTrpAspAlaAsp
202530
AlaArgAlaProAlaMetValTyrGlyProGlyGlnSerValThrIle
354045
AspGlyGluArgTyrThrLeuProHisGlnLeuArgLeuArgAsnVal
505560
AlaProSerGluValSerSerGluValSerIleAspIleGlyThrGlu
65707580
ThrGluAspSerGluLeuThrGluAlaAspLeuProProAlaAlaAla
859095
AlaLeuGlnAlaIleGluAsnAlaAlaArgIleLeuGluProHisIle
100105110
AspValIleMetGluAspCysSerThrProSerLeuCysGlySerSer
115120125
ArgGluMetProValTrpGlyGluAspIleProArgThrProSerPro
130135140
AlaLeuIleSerValThrGluSerSerSerAspGluLysThrProSer
145150155160
ValSerSerSerGlnGluAspThrProSerSerAspSerPheGluVal
165170175
IleGlnGluSerGluThrAlaGluGlyGluGluSerValPheAsnVal
180185190
AlaLeuSerValLeuLysAlaLeuPheProGlnSerAspAlaThrArg
195200205
LysLeuThrValLysMetSerCysCysValGluLysSerValThrArg
210215220
PhePheSerLeuGlyLeuThrValAlaAspValAlaSerLeuCysGlu
225230235240
MetGluIleGlnAsnHisThrAlaTyrCysAspGlnValArgThrPro
245250255
LeuGluLeuGlnValGlyCysLeuValGlyAsnGluLeuThrPheGlu
260265270
CysAspLysCysGluAlaArgGlnGluThrLeuAlaSerPheSerTyr
275280285
IleTrpSerGlyValProLeuThrArgAlaThrProAlaLysProPro
290295300
ValValArgProValGlySerLeuLeuValAlaAspThrThrLysVal
305310315320
TyrValThrAsnProAspAsnValGlyArgArg
325330
(2) INFORMATION FOR SEQ ID NO:167:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 536 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Consensus Sequence 3'-end
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:167:
CTGAGCGACCTCAAGCTCCCTGGCTTAGCAGTCCACCGAAAGAAGGCCGGGGCGTTGCGA60
ACACGCATGCTCCGCTCGCGCGGTTGGGCTGAGTTGGCTAGGGGCTTGTTGTGGCATCCA120
GGCCTACGGCTTCCTCCCCCTGAGATTGCTGGTATCCCGGGGGGTTTCCCTCTCTCCCCC180
CCCTATATGGGGGTGGTACATCAATTGGATTTCACAAGCCAGAGGAGTCGCTGGCGGTGG240
TTGGGGTTCTTAGCCCTGCTCATCGTAGCCCTCTTCGGGTGAACTAAATTCATCTGTTGC300
GGCAAGGTCTGGTGACTGATCATCACCGGAGGAGGTTCCCGCCCTCCCCGCCCCAGGGGT360
CTCCCCGCTGGGTAAAAAGGGCCCGGCCTTGGGAGGCATGGTGGTTACTAACCCCCTGGC420
AGGGTCAAAGCCTGATGGTGCTAATGCACTGCCACTTCGGTGGCGGGTCGCTACCTTATA480
GCGTAATCCGTGACTACGGGCTGCTCGCAGAGCCCTCCCCGGATGGGGCACAGTGC536
(2) INFORMATION FOR SEQ ID NO:168:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 594 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Individual Clone MP3-3
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:168:
CTGAGCGACCTCAAGCTCCCTGGCTTAGCAGTCCACCGAAAGAAGGCCGGGGCGTTGCGA60
ACACGCATGCTCCGCTCGCGCGGTTGGGCTGAGTTGGCTAGGGGCTTGTTGTGGCATCCA120
GGCCTACGGCTTCCTCCCCCTGAGATTGCTGGTATCCCGGGGGGTTTCCCTCTCTCCCCC180
CCCTATATGGGGGTGGTACACCAATTGGATTTCACAAGCCAGAGGAGTCGCTGGCGGTGG240
TTGGGGTTCTTAGCCCTGCTCATCGTAGCCCTCTTCGGGTGAACTAAATTCATCTGTTGC300
GGCAAGGTCTGGTGACTGATCATCACCGGAGGAGGTTCCCGCCCTCCCCGCCCCAGGGGT360
CTCCCCGCTGGGTAAAAAGGGCCCGGCCTTGGGAGGCATGGTGGTTACTAACCCCCTGGC420
AGGGTCAAAGCCTGATGGTGCTAATGCACTGCCACTTCGGTGGCGGGTCGCTACCTTATA480
GCGTAATCCGTGACTACGGGCTGCTCGCAGAGCCCTCCCCGGATGGGGCACAGTGCACTG540
TGATCTGAAGGGGTGCACCCCGGGAAGAGCTCGGCCCGAAGGCCGGCTTCTACT594
(2) INFORMATION FOR SEQ ID NO:169:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 594 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Individual Clone MP3-7
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:169:
CTGAGCGACCTCAAGCTCCCTGGCTTAGCAGTCCACCGAAAGAAGGCCGGGGCGTTGCGA60
ACACGCATGCTCCGCTCGCGCGGTTGGGCTGAGTTGGCTAGGGGCTTGTTGTGGCATCCA120
GGCCTACGGCTTCCTCCCCCTGAGATTGCTGGTGTCCCGGGGGGTTTCCCTCTCTCCCCC180
CCCTATATGGGGGTGGTACACCAATTGGATTTCACAAGCCAGAGGAGTCGCTGGCGGTGG240
TTGGGGTTCTTAGCCCTGCTCATCGTAGCCCTCTTCGGGTGAACTAAATTCATCTGTTGC300
GGCAAGGTCTGGTGACTGATCATCACCGGAGGAGGTTCCCGCCCTCCCCGCCCCAGGGGT360
CTCCCCGCTGGGTAAAAAGGGCCCGGCCTTGGGAGGCATGGTGGTTACTAACCCCCTGGC420
AGGGTCAAAGCCTGATGGTGCTAATGCACTGCCACTTCGGTGGCGGGTCGCTACCTTATA480
GCGTAATCCGTGACTACGGGCTGCTCGCAGAGCCCTCCCCGGATGGGGCACAGTGCACTG540
TGATCTGAAGGGGTGCACCCCGGTAAGAGCTCGGCCCGAAGGCCGGGTTCTACT594
(2) INFORMATION FOR SEQ ID NO:170:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 39 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer GV5446IRT
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:170:
CGGTCCCTCGAACTCCAGCGAGTCTTTTTTTTTTTTTTT39
(2) INFORMATION FOR SEQ ID NO:171:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 24 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer GV59- 5446F
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:171:
CTGAGCGACCTCAAGCTCCCTGGC24
(2) INFORMATION FOR SEQ ID NO:172:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 24 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer GV- 5446IR
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:172:
CGGTCCCTCGAACTCCAGCGAGTC24
(2) INFORMATION FOR SEQ ID NO:173:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 23 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Probe E5-7- PRB
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:173:
CGTAGCCCTCGGGTGAACTAAAT23
(2) INFORMATION FOR SEQ ID NO:174:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 35 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Race Anchor Sequence
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:174:
CACGAATTCACTATCGATTCTGGAACCTTCAGAGG35
(2) INFORMATION FOR SEQ ID NO:175:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 736 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Consensus Sequence 5'-end
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:175:
ACGTGGGGGAGTTGATCCCCCCCCCCCGGCACTGGGTGCAAGCCCCAGAAACCGACGCCT60
ATCTAAGTAGACGCAATGACTCGGCGCCGACTCGGCGACCGGCCAAAAGGTGGTGGATGG120
GTGATGACAGGGTTGGTAGGTCGTAAATCCCGGTCACCTTGGTAGCCACTATAGGTGGGT180
CTTAAGAGAAGGTTAAGATTCCTCTTGTGCCTGCGGCGAGACCGCGCACGGTCCACAGGT240
GTTGGCCCTACCGGTGGGAATAAGGGCCCGACGTCAGGCTCGTCGTTAAACCGAGCCCGT300
TACCCACCTGGGCAAACGACGCCCACGTACGGTCCACGTCGCCCTTCAATGTCTCTCTTG360
ACCAATAGGCGTAGCCGGCGAGTTGACAAGGACCAGTGGGGGCCGGGGGCTTGGAGAGGG420
ACTCCAAGTCCCGCCCTTCCCGGTGGGCCGGGAAATGCATGGGGCCACCCAGCTCCGCGG480
CGGCCTGCAGCCGGGGTAGCCCAAGAATCCTTCGGGTGAGGGCGGGTGGCATTTCCTTTT540
TCTATACCATCATGGCAGTCCTTCTGCTCCTTCTCGTGGTTGAGGCCGGGGCCATTCTGG600
CCCCGGCCACCCACGCTTGTCGAGCGAATGGGCAATATTTCCTCACAAATTGTTGTGCCC660
CGGAGGACATCGGGTTCTGCCTGGAGGGTGGATGCCTGGTGGCCCTGGGGTGCACGATTT720
GCACTGACCAATGCTG736
(2) INFORMATION FOR SEQ ID NO:176:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 688 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: HGV Variant BG34
(ix) FEATURE:
(A) NAME/KEY: CDS
(B) LOCATION: 272..688
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:176:
GACTCGGCGCCGACTCGGCGACCGGCCAAAAGGTGGTGGATGGGTGATGACAGGGTTGGT60
AGGTCGTAAATCCCGGTCACCTTGGTAGCCACTATAGGTGGGTCTTAAGAGAAGGTTAAG120
ATTCCTCTTGTGCCTGCGGCGAGACCGCGCACGGTCCACAGGTGTTGGCCCTACCGGTGT180
GAATAAGGGCCCGACGTCAGGCTCGTCGTTAAACCGAGCCCGTCACCCACCTGGGCAAAC240
GACGCCCACGTACGGTCCACGTCGCCCTTCAATGCCTCTCTTGGCCAATAGG292
MetProLeuLeuAlaAsnArg
15
AGTATCCGGCGAGTTGACAAGGACCAGTGGGGGCCGGGAGTCACGGGG340
SerIleArgArgValAspLysAspGlnTrpGlyProGlyValThrGly
101520
ATGGACCCCGGGCTCTGCCCTTCCCGGTGGAACGGGAAACGCATGGGG388
MetAspProGlyLeuCysProSerArgTrpAsnGlyLysArgMetGly
253035
CCACCCAGCTCCGCGGCGGCCTGCAGCCGGGGTAGCCCAAGAACCCTT436
ProProSerSerAlaAlaAlaCysSerArgGlySerProArgThrLeu
40455055
CGGGTGAGGGCGGGTGGCATTTCTCTTTTCTGTATCATCATGGCAGTC484
ArgValArgAlaGlyGlyIleSerLeuPheCysIleIleMetAlaVal
606570
CTCCTGCTCCTTCTCGTGGTTGAGGCCGGGGCCATTCTGGCCCCGGCC532
LeuLeuLeuLeuLeuValValGluAlaGlyAlaIleLeuAlaProAla
758085
ACCCACGCTTGTCGAGCGAATGGACAATATTTCCTCACAAACTGTTGC580
ThrHisAlaCysArgAlaAsnGlyGlnTyrPheLeuThrAsnCysCys
9095100
GCCCTCGAGGACATCGGGTTCTGCCTGGAAGGCGGGTGCCTGGTGGCC628
AlaLeuGluAspIleGlyPheCysLeuGluGlyGlyCysLeuValAla
105110115
TTAGGGTGCACCATTTGCACTGACCGTTGCTGGCCACTGTATCAGGCG676
LeuGlyCysThrIleCysThrAspArgCysTrpProLeuTyrGlnAla
120125130135
GGTTTGGCTGTG688
GlyLeuAlaVal
(2) INFORMATION FOR SEQ ID NO:177:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 139 amino acids
(B) TYPE: amino acid
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: protein
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:177:
MetProLeuLeuAlaAsnArgSerIleArgArgValAspLysAspGln
151015
TrpGlyProGlyValThrGlyMetAspProGlyLeuCysProSerArg
202530
TrpAsnGlyLysArgMetGlyProProSerSerAlaAlaAlaCysSer
354045
ArgGlySerProArgThrLeuArgValArgAlaGlyGlyIleSerLeu
505560
PheCysIleIleMetAlaValLeuLeuLeuLeuLeuValValGluAla
65707580
GlyAlaIleLeuAlaProAlaThrHisAlaCysArgAlaAsnGlyGln
859095
TyrPheLeuThrAsnCysCysAlaLeuGluAspIleGlyPheCysLeu
100105110
GluGlyGlyCysLeuValAlaLeuGlyCysThrIleCysThrAspArg
115120125
CysTrpProLeuTyrGlnAlaGlyLeuAlaVal
130135
(2) INFORMATION FOR SEQ ID NO:178:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 663 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: HGV Variant T55806
(ix) FEATURE:
(A) NAME/KEY: CDS
(B) LOCATION: 271..663
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:178:
GACTCGGCGCCGACTCGGCGACCGGCCAAAAGGTGGTGGATGGGTGATGCCAGGGTTGGT60
AGGTCGTAAATCCCGGTCATCTTGGTAGCCACTATAGGTGGGTCTTAAGAGAAGGTTAAG120
ATTCCTCTTGTGCCTGCGGCGAGACCGCGCACGGTCCACAGGTGTTGGCCCTACCGGTGG180
AATAAGGGCCCGACGTCAGGCTCGTCGTTAAACCGAGCCCGTCACCCACCTGGGCAAACG240
ACGCTCACGTACGGTCCACGTCGCCCTTCAATGTCTCTCTTGACCAATAGGTTT294
MetSerLeuLeuThrAsnArgPhe
15
ATCCGGCGAGTTGACAAGGACCAGTGGGGGCCGGGGGTTACGGGGACG342
IleArgArgValAspLysAspGlnTrpGlyProGlyValThrGlyThr
101520
GACCCCGAACCCTGCCCTTCCCGGTGGGCCGGGAAATGCATGGGGCCA390
AspProGluProCysProSerArgTrpAlaGlyLysCysMetGlyPro
25303540
CCCAGCTCCGCGGCGGCCTGCAGCCGGGGTAGCCCAAGAATCCTTCGG438
ProSerSerAlaAlaAlaCysSerArgGlySerProArgIleLeuArg
455055
GTGAGGGCGGGTGGCATTTCTCTTTTCTATACCATCATGGCAGTCCTT486
ValArgAlaGlyGlyIleSerLeuPheTyrThrIleMetAlaValLeu
606570
CTGCTCTTCTTCGTGGTTGAGGCCGGGGCGATTCTCGCCCCGGCCACC534
LeuLeuPhePheValValGluAlaGlyAlaIleLeuAlaProAlaThr
758085
CACGCTTGTCGGGCGAATGGGCAATATTTCCTCACAAATTGTTGCGCC582
HisAlaCysArgAlaAsnGlyGlnTyrPheLeuThrAsnCysCysAla
9095100
CCAGAGGATGTTGGGTTCTGCCTGGAGGGCGGATGCCTGGTGGCTCTG630
ProGluAspValGlyPheCysLeuGluGlyGlyCysLeuValAlaLeu
105110115120
GGGTGTACGATTTGCACTGACCGTTGCTGGCCA663
GlyCysThrIleCysThrAspArgCysTrpPro
125130
(2) INFORMATION FOR SEQ ID NO:179:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 131 amino acids
(B) TYPE: amino acid
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: protein
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:179:
MetSerLeuLeuThrAsnArgPheIleArgArgValAspLysAspGln
151015
TrpGlyProGlyValThrGlyThrAspProGluProCysProSerArg
202530
TrpAlaGlyLysCysMetGlyProProSerSerAlaAlaAlaCysSer
354045
ArgGlySerProArgIleLeuArgValArgAlaGlyGlyIleSerLeu
505560
PheTyrThrIleMetAlaValLeuLeuLeuPhePheValValGluAla
65707580
GlyAlaIleLeuAlaProAlaThrHisAlaCysArgAlaAsnGlyGln
859095
TyrPheLeuThrAsnCysCysAlaProGluAspValGlyPheCysLeu
100105110
GluGlyGlyCysLeuValAlaLeuGlyCysThrIleCysThrAspArg
115120125
CysTrpPro
130
(2) INFORMATION FOR SEQ ID NO:180:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 632 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: HGV Variant EB20-2
(ix) FEATURE:
(A) NAME/KEY: CDS
(B) LOCATION: 271..632
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:180:
GACTCGGCGCCGACTCGGCGACCGGCCAAAAGGTGGTGGATGGGTGATGCCAGGGTTGGT60
AGGTCGTAAATCCCGGTCATCTTGGTAGCCACTATAGGTGGGTCTTAAGAGAAGGTTAAG120
ATTCCTCTTGTGCCTGCGGCGAGACCGCGCACGGTCCACAGGTGTTGGCCCTACCGGTGT180
AATAAGGGCCCGACGTCAGGCTCGTCGTTAAACCGAGCCCGTCACCCACCTGGGCAAACG240
ACGCCCACGTACGGTCCACGTCGCCCTTCAATGCCTCTCTTGGCCAATAGGAGT294
MetProLeuLeuAlaAsnArgSer
15
TATCTCCGGCGAGTTGGCAAGGACCAGTGGGGGCCGGGGGTTACGGGG342
TyrLeuArgArgValGlyLysAspGlnTrpGlyProGlyValThrGly
101520
AAGGACCCCGAACCCTGCCCTTCCCGGTGGGCCGGGAAATGCATGGGG390
LysAspProGluProCysProSerArgTrpAlaGlyLysCysMetGly
25303540
CCACCCAGCTCCGCGGCGGCCTGCAGCCGGGGTAGCCCAAAAAACCTT438
ProProSerSerAlaAlaAlaCysSerArgGlySerProLysAsnLeu
455055
CGGGTGAGGGCGGGTGGCATTTTCTTTTCCTATACCATCATGGCAGTC486
ArgValArgAlaGlyGlyIlePhePheSerTyrThrIleMetAlaVal
606570
CTTCTGCTCCTTCTCGTGGTTGAGGCCGGGGCCATTTTGGCCCCGGCC534
LeuLeuLeuLeuLeuValValGluAlaGlyAlaIleLeuAlaProAla
758085
ACCCACGCTTGCAGAGCTAATGGGCAATATTTCCTCACAAACTGTTGT582
ThrHisAlaCysArgAlaAsnGlyGlnTyrPheLeuThrAsnCysCys
9095100
GCCTTGGAGGACATCGGGTTCTGCCTGGAAGGCGGATGCTTGGTGGCGCT632
AlaLeuGluAspIleGlyPheCysLeuGluGlyGlyCysLeuValAla
105110115120
(2) INFORMATION FOR SEQ ID NO:181:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 120 amino acids
(B) TYPE: amino acid
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: protein
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:181:
MetProLeuLeuAlaAsnArgSerTyrLeuArgArgValGlyLysAsp
151015
GlnTrpGlyProGlyValThrGlyLysAspProGluProCysProSer
202530
ArgTrpAlaGlyLysCysMetGlyProProSerSerAlaAlaAlaCys
354045
SerArgGlySerProLysAsnLeuArgValArgAlaGlyGlyIlePhe
505560
PheSerTyrThrIleMetAlaValLeuLeuLeuLeuLeuValValGlu
65707580
AlaGlyAlaIleLeuAlaProAlaThrHisAlaCysArgAlaAsnGly
859095
GlnTyrPheLeuThrAsnCysCysAlaLeuGluAspIleGlyPheCys
100105110
LeuGluGlyGlyCysLeuValAla
115120
(2) INFORMATION FOR SEQ ID NO:182:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 9103 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: HGV-JC Variant
(ix) FEATURE:
(A) NAME/KEY: CDS
(B) LOCATION: 276..9005
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:182:
CAATGACTCGGCGCCGACTCGGCGACCGGCCAAAAGGTGGTGGATGGGTGATGACAGGGT60
TGGTAGGTCGTAAATCCCGGTCACCTTGGTAGCCACTATAGGTGGGTCTTAAGAGAAGGT120
TAAGATTCCTCTTGTGCCTGCGGCGAGACCGCGCACGGTCCACAGGTGTTGGCCCTACCG180
GTGGGAATAAGGGCCCGACGTCAGGCTCGTCGTTAAACCGAGCCCGTAACCCGCCTGGGC240
AAACGACGCCCACGTACGGTCCACGTCGCCCTTCAATGTCGCTCTTGACCAAT293
MetSerLeuLeuThrAsn
15
AGGCTTAGCCGGCGAGTTGACAAGGACCAGTGGGGGCCGGGGTTTATG341
ArgLeuSerArgArgValAspLysAspGlnTrpGlyProGlyPheMet
101520
GGGAAGGACCCCAAACCCTGCCCTTCCCGGCGGACCGGGAAATGCATG389
GlyLysAspProLysProCysProSerArgArgThrGlyLysCysMet
253035
GGGCCACCCAGCTCCGCGGCGGCCTGCAGCCGGGGTAGCCCAAGAATC437
GlyProProSerSerAlaAlaAlaCysSerArgGlySerProArgIle
404550
CTTCGGGTGAGGGCGGGTGGCATTTCTCTTCCTTATACCATCATGGAA485
LeuArgValArgAlaGlyGlyIleSerLeuProTyrThrIleMetGlu
55606570
GCCCTCCTGTTCCTCCTCGGGGTGGAGGCCGGGGCCATTCTGGCCCCG533
AlaLeuLeuPheLeuLeuGlyValGluAlaGlyAlaIleLeuAlaPro
758085
GCCACCCACGCTTGTCGAGCGAATGGGCAATATTTCCTCACAAACTGT581
AlaThrHisAlaCysArgAlaAsnGlyGlnTyrPheLeuThrAsnCys
9095100
TGTGCTCCAGAGGACATTGGGTTCTGCCTCGAAGGCGGTTGCCTTGTG629
CysAlaProGluAspIleGlyPheCysLeuGluGlyGlyCysLeuVal
105110115
GCCCTGGGGTGCACAGTTTGCACTGACCGATGCTGGCCGCTGTATCAG677
AlaLeuGlyCysThrValCysThrAspArgCysTrpProLeuTyrGln
120125130
GCGGGCTTGGCTGTGCGGCCTGGCAAGTCCGCAGCCCAGCTGGTGGGG725
AlaGlyLeuAlaValArgProGlyLysSerAlaAlaGlnLeuValGly
135140145150
CAACTGGGTGGCCTCTACGGGCCCTTGTCGGTGTCGGCCTACGTGGCC773
GlnLeuGlyGlyLeuTyrGlyProLeuSerValSerAlaTyrValAla
155160165
GGCATCCTGGGCCTGGGTGAGGTGTACTCGGGTGTCCTAACAGTTGGT821
GlyIleLeuGlyLeuGlyGluValTyrSerGlyValLeuThrValGly
170175180
GTTGCGTTGACGCGCCGGGTCTACCCGATGCCCAACCTGACGTGTGCA869
ValAlaLeuThrArgArgValTyrProMetProAsnLeuThrCysAla
185190195
GTAGAGTGTGAGCTTAAGTGGGAAAGTGAGTTTTGGAGATGGACTGAG917
ValGluCysGluLeuLysTrpGluSerGluPheTrpArgTrpThrGlu
200205210
CAGCTGGCCTCCAATTACTGGATTCTGGAATACCTTTGGAAGGTCCCG965
GlnLeuAlaSerAsnTyrTrpIleLeuGluTyrLeuTrpLysValPro
215220225230
TTTGACTTCTGGAGAGGCGTGCTAAGCCTGACTCCCTTGCTGGTTTGC1013
PheAspPheTrpArgGlyValLeuSerLeuThrProLeuLeuValCys
235240245
GTGGCCGCGTTGCTGCTGCTGGAGCAACGGATTGTCATGGTCTTCCTG1061
ValAlaAlaLeuLeuLeuLeuGluGlnArgIleValMetValPheLeu
250255260
TTGGTGACGATGGCCGGGATGTCGCAAGGCGCTCCGGCCTCCGTTTTG1109
LeuValThrMetAlaGlyMetSerGlnGlyAlaProAlaSerValLeu
265270275
GGGTCTCGCCCCTTTGACTACGGGTTGACATGGCAGTCTTGTTCCTGC1157
GlySerArgProPheAspTyrGlyLeuThrTrpGlnSerCysSerCys
280285290
AGGGCTAATGGGTCGCGCTATACTACTGGGGAGAAGGTGTGGGACCGT1205
ArgAlaAsnGlySerArgTyrThrThrGlyGluLysValTrpAspArg
295300305310
GGGAACGTCACGCTCCTGTGTGACTGCCCCAACGGCCCCTGGGTGTGG1253
GlyAsnValThrLeuLeuCysAspCysProAsnGlyProTrpValTrp
315320325
TTGCCGGCCTTTTGCCAAGCAATCGGCTGGGGCGATCCCATCACTCAT1301
LeuProAlaPheCysGlnAlaIleGlyTrpGlyAspProIleThrHis
330335340
TGGAGCCACGGCCAAAATCGGTGGCCCCTCTCATGCCCCCAGTATGTC1349
TrpSerHisGlyGlnAsnArgTrpProLeuSerCysProGlnTyrVal
345350355
TATGGGTCTGTTTCAGTCACTTGCGTGTGGGGTTCCGTCTCTTGGTTT1397
TyrGlySerValSerValThrCysValTrpGlySerValSerTrpPhe
360365370
GCCTCGACTGGCGGTCGCGACTCGAAGATCGATGTGTGGAGTCTGGTG1445
AlaSerThrGlyGlyArgAspSerLysIleAspValTrpSerLeuVal
375380385390
CCGGTTGGTTCCGCCAGCTGCACCATAGCCGCTCTTGGATCGTCGGAT1493
ProValGlySerAlaSerCysThrIleAlaAlaLeuGlySerSerAsp
395400405
CGGGACACGGTAGTTGAGCTCTCCGAGTGGGGAGTCCCGTGCGCAACG1541
ArgAspThrValValGluLeuSerGluTrpGlyValProCysAlaThr
410415420
TGCATTCTGGATCGTCGGCCGGCCTCGTGCGGCACCTGTGTGAGAGAC1589
CysIleLeuAspArgArgProAlaSerCysGlyThrCysValArgAsp
425430435
TGCTGGCCCGAAACCGGGTCGGTTAGGTTTCCATTCCATCGGTGCGGC1637
CysTrpProGluThrGlySerValArgPheProPheHisArgCysGly
440445450
GCGGGGCCTAAGCTGACAAAGGACTTGGAAGCTGTGCCCTTCGTCAAT1685
AlaGlyProLysLeuThrLysAspLeuGluAlaValProPheValAsn
455460465470
AGGACAACTCCCTTCACCATAAGGGGCCCCCTGGGCAACCAGGGGAGA1733
ArgThrThrProPheThrIleArgGlyProLeuGlyAsnGlnGlyArg
475480485
GGCAACCCGGTGCGGTCGCCCTTGGGTTTTGGGTCCTACGCCATGACC1781
GlyAsnProValArgSerProLeuGlyPheGlySerTyrAlaMetThr
490495500
AAGATCCGAGACTCCTTACATTTGGTGAAATGTCCCACACCAGCCATT1829
LysIleArgAspSerLeuHisLeuValLysCysProThrProAlaIle
505510515
GAGCCTCCCACCGGGACGTTTGGGTTCTTCCCCGGAGTGCCGCCTCTT1877
GluProProThrGlyThrPheGlyPhePheProGlyValProProLeu
520525530
AACAACTGCCTGCTGTTGGGCACGGAAGTGTCCGAAGCGCTGGGCGGG1925
AsnAsnCysLeuLeuLeuGlyThrGluValSerGluAlaLeuGlyGly
535540545550
GCCGGCCTCACGGGGGGGTTCTATGAACCCCTGGTGCGCAGGCGTTCG1973
AlaGlyLeuThrGlyGlyPheTyrGluProLeuValArgArgArgSer
555560565
GAGCTGATGGGGCGCCGAAATCCGGTTTGCCCGGGGTTTGCATGGCTG2021
GluLeuMetGlyArgArgAsnProValCysProGlyPheAlaTrpLeu
570575580
TCCTCGGGTCGACCTGACGGGTTTATACACGTCCAGGGCCACTTGCAG2069
SerSerGlyArgProAspGlyPheIleHisValGlnGlyHisLeuGln
585590595
GAGGTCGATGCTGGCAACTTCATCCCTCCACCTCGCTGGTTGCTCTTG2117
GluValAspAlaGlyAsnPheIleProProProArgTrpLeuLeuLeu
600605610
GACTTTGTGTTTGTCCTGTTATACCTGATGAAGCTGGCTGAGGCACGG2165
AspPheValPheValLeuLeuTyrLeuMetLysLeuAlaGluAlaArg
615620625630
CTGGTCCCGTTGATCTTGCTTCTGCTGTGGTGGTGGGTGAACCAGTTG2213
LeuValProLeuIleLeuLeuLeuLeuTrpTrpTrpValAsnGlnLeu
635640645
GCAGTCCTTGGACTGCCGGCTGTGGACGCCGCCGTGGCTGGTGAGGTC2261
AlaValLeuGlyLeuProAlaValAspAlaAlaValAlaGlyGluVal
650655660
TTCGCGGGCCCGGCCCTGTCGTGGTGTCTGGGCCTCCCCACCGTTAGT2309
PheAlaGlyProAlaLeuSerTrpCysLeuGlyLeuProThrValSer
665670675
ATGATCCTGGGCTTAGCAAACCTGGTGTTGTATTTCCGGTGGATGGGT2357
MetIleLeuGlyLeuAlaAsnLeuValLeuTyrPheArgTrpMetGly
680685690
CCCCAACGCCTCATGTTCCTCGTGTTGTGGAAGCTCGCTCGGGGAGCC2405
ProGlnArgLeuMetPheLeuValLeuTrpLysLeuAlaArgGlyAla
695700705710
TTCCCGCTGGCACTTCTGATGGGGATCTCGGCAACCCGCGGGCGCACC2453
PheProLeuAlaLeuLeuMetGlyIleSerAlaThrArgGlyArgThr
715720725
TCGGTGCTCGGGGCCGAGTTCTGCTTCGATGTCACATTCGAGGTGGAC2501
SerValLeuGlyAlaGluPheCysPheAspValThrPheGluValAsp
730735740
ACGTCGGTTTTGGGCTGGGTGGTGGCCAGTGTGGTAGCCTGGGCCATT2549
ThrSerValLeuGlyTrpValValAlaSerValValAlaTrpAlaIle
745750755
GCGCTCCTGAGCTCGATGAGCGCGGGAGGGTGGAGGCACAAGGCCGTG2597
AlaLeuLeuSerSerMetSerAlaGlyGlyTrpArgHisLysAlaVal
760765770
ATCTATAGGACGTGGTGTAAGGGGTACCAGGCAATACGCCAACGGGTG2645
IleTyrArgThrTrpCysLysGlyTyrGlnAlaIleArgGlnArgVal
775780785790
GTGCGGAGCCCCCTCGGGGAGGGGCGGCCCACCAAACCCTTGACGTTT2693
ValArgSerProLeuGlyGluGlyArgProThrLysProLeuThrPhe
795800805
GCTTGGTGCTTGGCCTCATACATCTGGCCGGATGCTGTGATGATGGTG2741
AlaTrpCysLeuAlaSerTyrIleTrpProAspAlaValMetMetVal
810815820
GTGGTAGCCTTGGTGCTCCTCTTTGGCCTGTTCGACGCGTTGGACTGG2789
ValValAlaLeuValLeuLeuPheGlyLeuPheAspAlaLeuAspTrp
825830835
GCTTTGGAGGAGCTCTTGGTGTCCCGGCCCTCGTTACGGCGTCTGGCC2837
AlaLeuGluGluLeuLeuValSerArgProSerLeuArgArgLeuAla
840845850
CGGGTGGTTGAGTGCTGTGTGATGGCGGGAGAGAAGGCCACAACCGTC2885
ArgValValGluCysCysValMetAlaGlyGluLysAlaThrThrVal
855860865870
CGGCTGGTCTCCAAGATGTGCGCGAGAGGGGCCTATTTGTTTGACCAT2933
ArgLeuValSerLysMetCysAlaArgGlyAlaTyrLeuPheAspHis
875880885
ATGGGCTCTTTTTCGCGCGCTGTCAAGGAGCGCCTGCTGGAGTGGGAC2981
MetGlySerPheSerArgAlaValLysGluArgLeuLeuGluTrpAsp
890895900
GCGGCTTTGGAACCCCTGTCATTCACTAGGACGGACTGTCGCATCATT3029
AlaAlaLeuGluProLeuSerPheThrArgThrAspCysArgIleIle
905910915
AGAGATGCTGCGAGGACCTTGGCCTGCGGGCAGTGCGTCATGGGCTTG3077
ArgAspAlaAlaArgThrLeuAlaCysGlyGlnCysValMetGlyLeu
920925930
CCTGTGGTAGCGCGCCGTGGTGACGAGGTTCTTATCGGTGTCTTTCAG3125
ProValValAlaArgArgGlyAspGluValLeuIleGlyValPheGln
935940945950
GATGTGAACCATTTGCCTCCCGGATTCGTCCCGACCGCACCCGTTGTC3173
AspValAsnHisLeuProProGlyPheValProThrAlaProValVal
955960965
ATCCGGCGGTGCGGGAAGGGGTTTCTGGGGGTCACTAAGGCTGCCTTG3221
IleArgArgCysGlyLysGlyPheLeuGlyValThrLysAlaAlaLeu
970975980
ACTGGTCGGGATCCTGACTTACATCCAGGGAACGTCATGGTGTTGGGG3269
ThrGlyArgAspProAspLeuHisProGlyAsnValMetValLeuGly
985990995
ACGGCTACGTCGCGAAGCATGGGGACATGCCTGAACGGCCTGCTGTTC3317
ThrAlaThrSerArgSerMetGlyThrCysLeuAsnGlyLeuLeuPhe
100010051010
ACGACTTTCCATGGGGCTTCATCCCGAACCATCGCCACGCCCGTGGGG3365
ThrThrPheHisGlyAlaSerSerArgThrIleAlaThrProValGly
1015102010251030
GCCCTTAATCCCAGGTGGTGGTCCGCCAGTGATGACGTCACGGTGTAC3413
AlaLeuAsnProArgTrpTrpSerAlaSerAspAspValThrValTyr
103510401045
CCGCTCCCGGATGGGGCAACCTCGTTGACGCCCTGCACTTGCCAGGCT3461
ProLeuProAspGlyAlaThrSerLeuThrProCysThrCysGlnAla
105010551060
GAGTCCTGTTGGGTCATACGGTCCGACGGGGCTTTGTGCCATGGCTTG3509
GluSerCysTrpValIleArgSerAspGlyAlaLeuCysHisGlyLeu
106510701075
AGTAAGGGAGACAAGGTGGAGCTAGATGTGGCCATGGAGGTCTCAGAT3557
SerLysGlyAspLysValGluLeuAspValAlaMetGluValSerAsp
108010851090
TTCCGTGGCTCGTCCGGCTCACCTGTCCTGTGCGACGAGGGGCACGCA3605
PheArgGlySerSerGlySerProValLeuCysAspGluGlyHisAla
1095110011051110
GTAGGAATGCTCGTGTCGGTGCTCCACTCGGGTGGTCGGGTCACCGCG3653
ValGlyMetLeuValSerValLeuHisSerGlyGlyArgValThrAla
111511201125
GCTCGATTCACCAGGCCGTGGACCCAGGTCCCAACAGATGCTAAGACC3701
AlaArgPheThrArgProTrpThrGlnValProThrAspAlaLysThr
113011351140
ACCACTGAACCCCCTCCGGTGCCGGCAAAGGGAGTTTTCAAGGAAGCC3749
ThrThrGluProProProValProAlaLysGlyValPheLysGluAla
114511501155
CCACTGTTTATGCCCACGGGCGCAGGAAAGAGCACGCGCGTCCCGTTG3797
ProLeuPheMetProThrGlyAlaGlyLysSerThrArgValProLeu
116011651170
GAGTATGGCAACATGGGGCACAAGGTCCTGATTTTGAACCCCTCGGTG3845
GluTyrGlyAsnMetGlyHisLysValLeuIleLeuAsnProSerVal
1175118011851190
GCGACAGTGAGGGCCATGGGCCCTTACATGGAGCGACTGGCGGGAAAA3893
AlaThrValArgAlaMetGlyProTyrMetGluArgLeuAlaGlyLys
119512001205
CATCCAAGTATCTACTGTGGCCATGACACCACTGCCTTCACAAGGATC3941
HisProSerIleTyrCysGlyHisAspThrThrAlaPheThrArgIle
121012151220
ACTGATTCCCCCTTAACGTACTCTACCTATGGGAGGTTTCTGGCCAAC3989
ThrAspSerProLeuThrTyrSerThrTyrGlyArgPheLeuAlaAsn
122512301235
CCTAGGCAGATGCTGCGAGGTGTGTCGGTGGTCATTTGCGATGAATGC4037
ProArgGlnMetLeuArgGlyValSerValValIleCysAspGluCys
124012451250
CACAGTCATGATTCCACTGTGTTGTTGGGGATTGGACGGGTCCGGGAG4085
HisSerHisAspSerThrValLeuLeuGlyIleGlyArgValArgGlu
1255126012651270
CTGGCACGAGAGTGTGGGGTGCAGCTTGTGCTCTACGCCACTGCCACG4133
LeuAlaArgGluCysGlyValGlnLeuValLeuTyrAlaThrAlaThr
127512801285
CCTCCTGGGTCCCCCATGACTCAGCATCCGTCAATCATTGAGACCAAA4181
ProProGlySerProMetThrGlnHisProSerIleIleGluThrLys
129012951300
TTGGATGTGGGTGAGATTCCCTTCTATGGGCATGGCATACCCCTCGAG4229
LeuAspValGlyGluIleProPheTyrGlyHisGlyIleProLeuGlu
130513101315
CGGATGCGGACCGGTAGGCACCTCGTATTCTGCTACTCTAAGGCAGAG4277
ArgMetArgThrGlyArgHisLeuValPheCysTyrSerLysAlaGlu
132013251330
TGTGAGCGGCTAGCCGGTCAGTTTTCTGCTAGGGGAGTTAACGCCATA4325
CysGluArgLeuAlaGlyGlnPheSerAlaArgGlyValAsnAlaIle
1335134013451350
GCCTATTACAGGGGAAAAGACAGTTCTATCATCAAGGACGGAGATCTG4373
AlaTyrTyrArgGlyLysAspSerSerIleIleLysAspGlyAspLeu
135513601365
GTGGTGTGCGCGACCGACGCGCTATCCACTGGATACACTGGGAACTTC4421
ValValCysAlaThrAspAlaLeuSerThrGlyTyrThrGlyAsnPhe
137013751380
GATTCTGTCACCGACTGTGGGTTAGTGGTGGAGGAGGTCGTCGAGGTG4469
AspSerValThrAspCysGlyLeuValValGluGluValValGluVal
138513901395
ACCCTTGATCCCACCATTACCATCTCCCTGCGGACAGTGCCCGCGTCG4517
ThrLeuAspProThrIleThrIleSerLeuArgThrValProAlaSer
140014051410
GCAGAACTGTCGATGCAGAGACGAGGACGCACGGGTAGAGGCAGGTCT4565
AlaGluLeuSerMetGlnArgArgGlyArgThrGlyArgGlyArgSer
1415142014251430
GGGCGCTACTACTACGCCGGGGTCGGAAAGGCCCCCGCGGGTGTGGTG4613
GlyArgTyrTyrTyrAlaGlyValGlyLysAlaProAlaGlyValVal
143514401445
CGCTCGGGTCCTGTCTGGTCGGCGGTGGAGGCCGGAGTGACCTGGTAT4661
ArgSerGlyProValTrpSerAlaValGluAlaGlyValThrTrpTyr
145014551460
GGAATGGAACCTGACTTGACAGCTAACCTATTGAGACTTTACGACGAC4709
GlyMetGluProAspLeuThrAlaAsnLeuLeuArgLeuTyrAspAsp
146514701475
TGCCCTTACACCGCAGCCGTCGCAGCTGACATCGGTGAAGCCGCGGTG4757
CysProTyrThrAlaAlaValAlaAlaAspIleGlyGluAlaAlaVal
148014851490
TTTTTCTCCGGGCTAGCCCCGTTGAGGATGCATCCCGATGTTAGCTGG4805
PhePheSerGlyLeuAlaProLeuArgMetHisProAspValSerTrp
1495150015051510
GCAAAAGTGCGCGGCGTCAACTGGCCCCTCTTGGTGGGTGTTCAGCGG4853
AlaLysValArgGlyValAsnTrpProLeuLeuValGlyValGlnArg
151515201525
ACCATGTGCCGGGAAACACTGTCTCCCGGACCATCGGACGACCCCCAA4901
ThrMetCysArgGluThrLeuSerProGlyProSerAspAspProGln
153015351540
TGGGCAGGTCTGAAGGGCCCGAATCCTGTTCCACTACTGCTGAGGTGG4949
TrpAlaGlyLeuLysGlyProAsnProValProLeuLeuLeuArgTrp
154515501555
GGCAATGATTTACCATCAAAAGTGGCCGGCCACCACATTGTTGACGAC4997
GlyAsnAspLeuProSerLysValAlaGlyHisHisIleValAspAsp
156015651570
CTGGTTCGTAGGCTTGGTGTGGCGGAGGGTTATGTCCGCTGCGATGCG5045
LeuValArgArgLeuGlyValAlaGluGlyTyrValArgCysAspAla
1575158015851590
GGGCCGATCTTAATGGTCGGCCTCGCTATCGCGGGGGGGATGATCTAC5093
GlyProIleLeuMetValGlyLeuAlaIleAlaGlyGlyMetIleTyr
159516001605
GCATCTTACACCGGGTCTTTAGTGGTGGTGACAGACTGGGATGTAAAG5141
AlaSerTyrThrGlySerLeuValValValThrAspTrpAspValLys
161016151620
GGGGGTGGCAGCCCTCTTTATCGGCATGGAGACCAGGCCACGCCACAG5189
GlyGlyGlySerProLeuTyrArgHisGlyAspGlnAlaThrProGln
162516301635
CCGGTTGTGCAGGTCCCCCCGGTAGACCATCGGCCGGGGGGGGAGTCT5237
ProValValGlnValProProValAspHisArgProGlyGlyGluSer
164016451650
GCGCCTTCGGATGCCAAGACAGTGACAGATGCGGTGGCGGCCATCCAG5285
AlaProSerAspAlaLysThrValThrAspAlaValAlaAlaIleGln
1655166016651670
GTGGATTGCGATTGGTCAGTCATGACCCTGTCGATCGGGGAAGTGCTG5333
ValAspCysAspTrpSerValMetThrLeuSerIleGlyGluValLeu
167516801685
TCCTTGGCTCAGGCTAAAACAGCTGAGGCCTACACGGCAACCGCCAAG5381
SerLeuAlaGlnAlaLysThrAlaGluAlaTyrThrAlaThrAlaLys
169016951700
TGGCTCGCTGGCTGCTACACGGGGACGCGGGCCGTTCCCACTGTTTCA5429
TrpLeuAlaGlyCysTyrThrGlyThrArgAlaValProThrValSer
170517101715
ATTGTTGACAAGCTCTTTGCCGGAGGGTGGGCGGCTGTGGTTGGCCAC5477
IleValAspLysLeuPheAlaGlyGlyTrpAlaAlaValValGlyHis
172017251730
TGTCACAGCGTCATAGCTGCGGCGGTGGCTGCCTACGGGGCTTCCAGG5525
CysHisSerValIleAlaAlaAlaValAlaAlaTyrGlyAlaSerArg
1735174017451750
AGTCCGCCGTTGGCAGCCGCGGCTTCCTACCTGATGGGACTGGGCGTC5573
SerProProLeuAlaAlaAlaAlaSerTyrLeuMetGlyLeuGlyVal
175517601765
GGAGGCAACGCTCAGACGCGTTTGGCGTCTGCCCTCCTGTTGGGGGCC5621
GlyGlyAsnAlaGlnThrArgLeuAlaSerAlaLeuLeuLeuGlyAla
177017751780
GCTGGCACCGCCCTGGGCACTCCCGTCGTGGGTTTAACCATGGCGGGG5669
AlaGlyThrAlaLeuGlyThrProValValGlyLeuThrMetAlaGly
178517901795
GCGTTCATGGGGGGTGCTAGCGTCTCTCCCTCCTTGGTCACCATCTTG5717
AlaPheMetGlyGlyAlaSerValSerProSerLeuValThrIleLeu
180018051810
TTGGGGGCCGTGGGAGGCTGGGAGGGCGTCGTCAACGCTGCTAGCCTT5765
LeuGlyAlaValGlyGlyTrpGluGlyValValAsnAlaAlaSerLeu
1815182018251830
GTCTTTGACTTCATGGCGGGGAAACTATCGTCAGAAGATCTGTGGTAC5813
ValPheAspPheMetAlaGlyLysLeuSerSerGluAspLeuTrpTyr
183518401845
GCCATCCCAGTGCTCACCAGCCCGGGGGCGGGCCTTGCGGGGATCGCC5861
AlaIleProValLeuThrSerProGlyAlaGlyLeuAlaGlyIleAla
185018551860
CTTGGGTTGGTGCTGTACTCAGCTAACAACTCTGGTACTACCACTTGG5909
LeuGlyLeuValLeuTyrSerAlaAsnAsnSerGlyThrThrThrTrp
186518701875
TTGAACCGTCTGCTGACTACGTTACCTAGGTCTTCTTGCATCCCTGAC5957
LeuAsnArgLeuLeuThrThrLeuProArgSerSerCysIleProAsp
188018851890
AGCTATTTCCAACAGGCCGATTACTGTGACAAGGTCTCGGCCGTGCTT6005
SerTyrPheGlnGlnAlaAspTyrCysAspLysValSerAlaValLeu
1895190019051910
CGCCGACTGAGCCTCACCCGCACTGTGGTGGCCCTAGTCAATAGGGAA6053
ArgArgLeuSerLeuThrArgThrValValAlaLeuValAsnArgGlu
191519201925
CCCAAGGTGGACGAGGTACAGGTGGGGTACGTCTGGGATCTCTGGGAG6101
ProLysValAspGluValGlnValGlyTyrValTrpAspLeuTrpGlu
193019351940
TGGATCATGCGTCAAGTGCGCATGGTCATGGCCAGGCTCCGGGCTCTC6149
TrpIleMetArgGlnValArgMetValMetAlaArgLeuArgAlaLeu
194519501955
TGCCCCGTGGTGTCACTGCCTTTGTGGCACTGCGGGGAGGGGTGGTCC6197
CysProValValSerLeuProLeuTrpHisCysGlyGluGlyTrpSer
196019651970
GGAGAGTGGTTGTTGGACGGCCATGTGGAGAGTCGCTGTCTTTGCGGG6245
GlyGluTrpLeuLeuAspGlyHisValGluSerArgCysLeuCysGly
1975198019851990
TGCGTGATCACCGGCGATGTTTTCAATGGGCAACTCAAAGAGCCAGTT6293
CysValIleThrGlyAspValPheAsnGlyGlnLeuLysGluProVal
199520002005
TACTCTACAAAGTTGTGCCGGCACTATTGGATGGGGACCGTTCCTGTG6341
TyrSerThrLysLeuCysArgHisTyrTrpMetGlyThrValProVal
201020152020
AACATGCTGGGTTACGGCGAAACATCACCCCTCTTGGCCTCTGACACC6389
AsnMetLeuGlyTyrGlyGluThrSerProLeuLeuAlaSerAspThr
202520302035
CCGAAGGTGGTGCCTTTTGGGACGTCGGGCTGGGCTGAGGTGGTGGTG6437
ProLysValValProPheGlyThrSerGlyTrpAlaGluValValVal
204020452050
ACCCCTACCCACGTGGTGATCAGGAGAACCTCTCCCTACGAGTTGCTG6485
ThrProThrHisValValIleArgArgThrSerProTyrGluLeuLeu
2055206020652070
CGCCAACAAATCCTATCAGCTGCAGTTGCTGAGCCCTATTATGTCGAC6533
ArgGlnGlnIleLeuSerAlaAlaValAlaGluProTyrTyrValAsp
207520802085
GGCATACCGGTCTCATGGGACGCGGACGCTCGTGCGCCTGCTATGGTT6581
GlyIleProValSerTrpAspAlaAspAlaArgAlaProAlaMetVal
209020952100
TATGGCCCTGGGCAAAGTGTTACCATTGACGGGGAGCGCTACACCCTG6629
TyrGlyProGlyGlnSerValThrIleAspGlyGluArgTyrThrLeu
210521102115
CCGCATCAACTGCGGCTCAGGAATGTAGCGCCCTCTGAGGTTTCATCC6677
ProHisGlnLeuArgLeuArgAsnValAlaProSerGluValSerSer
212021252130
GAGGTGTCCATAGACATTGGGACGGAGACTGAAGACTCAGAACTGACT6725
GluValSerIleAspIleGlyThrGluThrGluAspSerGluLeuThr
2135214021452150
GAGGCCGACCTGCCGCCGGCAGCTGCAGCCCTCCAGGCTATCGAGAAT6773
GluAlaAspLeuProProAlaAlaAlaAlaLeuGlnAlaIleGluAsn
215521602165
GCTGCGAGGATTCTTGAGCCTCATATTGATGTCATCATGGAGGATTGC6821
AlaAlaArgIleLeuGluProHisIleAspValIleMetGluAspCys
217021752180
AGTACACCCTCTCTTTGTGGTAGTAGCCGAGAGATGCCTGTGTGGGGA6869
SerThrProSerLeuCysGlySerSerArgGluMetProValTrpGly
218521902195
GAAGACATCCCCCGCACTCCATCGCCAGCACTTATCTCGGTTACCGAG6917
GluAspIleProArgThrProSerProAlaLeuIleSerValThrGlu
220022052210
AGCAGCTCAGATGAGAAGACCCCGTCGGTGTCCTCCTCGCAGGAGGAT6965
SerSerSerAspGluLysThrProSerValSerSerSerGlnGluAsp
2215222022252230
ACCCCGTCCTCTGACTCATTCGAAGTCATCCAAGAGTCTGAGACAGCT7013
ThrProSerSerAspSerPheGluValIleGlnGluSerGluThrAla
223522402245
GAAGGAGAGGAAAGTGTCTTCAACGTGGCTCTTTCCGTACTAGAAGCC7061
GluGlyGluGluSerValPheAsnValAlaLeuSerValLeuGluAla
225022552260
TTGTTTCCACAGAGTGATGCCACTAGAAAGCTTACCGTCAGGATGAAT7109
LeuPheProGlnSerAspAlaThrArgLysLeuThrValArgMetAsn
226522702275
TGCTGCGTTGAGAAGAGCGTCACGCGCTTCTTTTCTTTGGGGCTGACG7157
CysCysValGluLysSerValThrArgPhePheSerLeuGlyLeuThr
228022852290
GTGGCTGATGTGGCCAGTCTGTGTGAGATGGAGATCCAGAACCATACA7205
ValAlaAspValAlaSerLeuCysGluMetGluIleGlnAsnHisThr
2295230023052310
GCCTATTGTGACAAGGTGCGCACTCCGCTCGAATTGCAAGTTGGGTGC7253
AlaTyrCysAspLysValArgThrProLeuGluLeuGlnValGlyCys
231523202325
TTGGTGGGCAATGAACTTACCTTTGAATGTGATAAGTGTGAGGCTAGG7301
LeuValGlyAsnGluLeuThrPheGluCysAspLysCysGluAlaArg
233023352340
CAAGAGACTTTGGCCTCCTTCTCCTATATTTGGTCTGGGGTGCCATTG7349
GlnGluThrLeuAlaSerPheSerTyrIleTrpSerGlyValProLeu
234523502355
ACTAGGGCCACACCGGCTAAACCACCTGTGGTGAGGCCGGTGGGGTCC7397
ThrArgAlaThrProAlaLysProProValValArgProValGlySer
236023652370
TTGTTGGTGGCTGACACCACGAAAGTGTATGTCACAAACCCGGACAAT7445
LeuLeuValAlaAspThrThrLysValTyrValThrAsnProAspAsn
2375238023852390
GTTGGGAGAAGAGTGGACAAGGTGACCTTCTGGCGCGCCCCCAGGGTC7493
ValGlyArgArgValAspLysValThrPheTrpArgAlaProArgVal
239524002405
CATGACAAATATCTCGTGGACTCCATCGAGCGTGCCAGGAGGGCGGCT7541
HisAspLysTyrLeuValAspSerIleGluArgAlaArgArgAlaAla
241024152420
CAAGCCTGCCAAAGCATGGGTTACACTTATGAGGAAGCAATAAGGACT7589
GlnAlaCysGlnSerMetGlyTyrThrTyrGluGluAlaIleArgThr
242524302435
GTTAGGCCACATGCTGCCATGGGCTGGGGATCTAAGGTGTCGGTCAAG7637
ValArgProHisAlaAlaMetGlyTrpGlySerLysValSerValLys
244024452450
GACTTGGCCACCCCTGCGGGGAAGATGGCCGTCCACGACCGACTTCAG7685
AspLeuAlaThrProAlaGlyLysMetAlaValHisAspArgLeuGln
2455246024652470
GAGATACTTGAGGGGACTCCGGTCCCTTTTACTCTTACTGTGAAAAAG7733
GluIleLeuGluGlyThrProValProPheThrLeuThrValLysLys
247524802485
GAGGTGTTCTTCAAAGACCGTAAGGAGGAGAAGGCCCCCCGCCTCATT7781
GluValPhePheLysAspArgLysGluGluLysAlaProArgLeuIle
249024952500
GTGTTCCCCCCCCTGGACTTCCGGATAGCTGAGAAGCTTATCCTGGGA7829
ValPheProProLeuAspPheArgIleAlaGluLysLeuIleLeuGly
250525102515
GACCCGGGGCGGGTGGCCAAGGCGGTGTTGGGGGGGGCTTACGCCTTC7877
AspProGlyArgValAlaLysAlaValLeuGlyGlyAlaTyrAlaPhe
252025252530
CAGTACACCCCAAATCAGCGAGTTAAGGAGATGCTCAAACTGTGGGAG7925
GlnTyrThrProAsnGlnArgValLysGluMetLeuLysLeuTrpGlu
2535254025452550
TCAAAGAAAACACCTTGCGCCATCTGTGTGGACGCCACTTGCTTCGAC7973
SerLysLysThrProCysAlaIleCysValAspAlaThrCysPheAsp
255525602565
AGTAGCATTACTGAAGAGGACGTGGCGCTGGAGACAGAGCTGTACGCT8021
SerSerIleThrGluGluAspValAlaLeuGluThrGluLeuTyrAla
257025752580
CTGGCCTCTGACCATCCAGAGTGGGTGCGAGCTTTGGGGAAGTACTAT8069
LeuAlaSerAspHisProGluTrpValArgAlaLeuGlyLysTyrTyr
258525902595
GCCTCAGGAACCATGGTCACCCCTGAGGGGGTTCCCGTAGGTGAGAGG8117
AlaSerGlyThrMetValThrProGluGlyValProValGlyGluArg
260026052610
TATTGTAGATCCTCAGGCGTTTTGACTACCAGCGCGAGTAACTGCCTG8165
TyrCysArgSerSerGlyValLeuThrThrSerAlaSerAsnCysLeu
2615262026252630
ACCTGCTACATCAAGGTGAAAGCCGCTTGTGAGAGAGTGGGGCTGAAA8213
ThrCysTyrIleLysValLysAlaAlaCysGluArgValGlyLeuLys
263526402645
AATGTCTCGCTTCTCATAGCCGGCGATGACTGTTTGATCATATGCGAA8261
AsnValSerLeuLeuIleAlaGlyAspAspCysLeuIleIleCysGlu
265026552660
CGGCCAGTGTGCGACCCTTGTGACGCCTTGGGCAGAGCCCTGGCGAGC8309
ArgProValCysAspProCysAspAlaLeuGlyArgAlaLeuAlaSer
266526702675
TATGGGTATGCTTGCGAGCCTTCGTATCATGCATCACTGGACACGGCC8357
TyrGlyTyrAlaCysGluProSerTyrHisAlaSerLeuAspThrAla
268026852690
CCCTTCTGCTCCACTTGGCTCGCTGAGTGCAACGCAGATGGGAAACGC8405
ProPheCysSerThrTrpLeuAlaGluCysAsnAlaAspGlyLysArg
2695270027052710
CATTTCTTCCTGACCACGGACTTTCGGAGGCCGCTTGCTCGCATGTCG8453
HisPhePheLeuThrThrAspPheArgArgProLeuAlaArgMetSer
271527202725
AGCGAGTATAGTGACCCAATGGCTTCGGCCATAGGTTACATCCTCCTG8501
SerGluTyrSerAspProMetAlaSerAlaIleGlyTyrIleLeuLeu
273027352740
TATCCCTGGCATCCCATCACACGGTGGGTCATCATCCCTCATGTGCTA8549
TyrProTrpHisProIleThrArgTrpValIleIleProHisValLeu
274527502755
ACGTGCGCATTCAGGGGTGGTGGTACACCGTCTGATCCGGTTTGGTGT8597
ThrCysAlaPheArgGlyGlyGlyThrProSerAspProValTrpCys
276027652770
CAGGTGCATGGTAACTACTACAAGTTTCCACTGGACAAACTGCCTAAC8645
GlnValHisGlyAsnTyrTyrLysPheProLeuAspLysLeuProAsn
2775278027852790
ATCATCGTGGCCCTCCACGGACCAGCAGCGTTGAGGGTTACCGCAGAC8693
IleIleValAlaLeuHisGlyProAlaAlaLeuArgValThrAlaAsp
279528002805
ACAACTAAGACAAAAATGGAAGCTGGGAAGGTGCTGAGTGACCTCAAG8741
ThrThrLysThrLysMetGluAlaGlyLysValLeuSerAspLeuLys
281028152820
CTCCCTGGCCTAGCGGTCCACCGAAAGAAGGCCGGAGCACTGCGAACA8789
LeuProGlyLeuAlaValHisArgLysLysAlaGlyAlaLeuArgThr
282528302835
CGCATGCTTCGGTCGCGCGGTTGGGCCGAGTTGGCGAGGGGCCTGTTG8837
ArgMetLeuArgSerArgGlyTrpAlaGluLeuAlaArgGlyLeuLeu
284028452850
TGGCATCCAGGCCTCCGGCTCCCTCCCCCTGAGATTGCTGGTATCCCG8885
TrpHisProGlyLeuArgLeuProProProGluIleAlaGlyIlePro
2855286028652870
GGGGGTTTCCCCCTCTCCCCCCCCTACATGGGGGTGGTGCATCAATTG8933
GlyGlyPheProLeuSerProProTyrMetGlyValValHisGlnLeu
287528802885
GATTTTACAAGCCAGAGGAGTCGCTGGCGGTGGCTGGGGTTCTTAGCC8981
AspPheThrSerGlnArgSerArgTrpArgTrpLeuGlyPheLeuAla
289028952900
CTGCTCATCGTAGCCCTCTTCGGGTGAACTAAATTCATCTGTTGCGGCAAGGTC9035
LeuLeuIleValAlaLeuPheGly
29052910
CAGTGACTGATCATCACTGGAGGAGGTTCCCGCCCTCCCCGCCCCAGGGGTCTCCCCGCT9095
GGGTAAAA9103
(2) INFORMATION FOR SEQ ID NO:183:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 2910 amino acids
(B) TYPE: amino acid
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: protein
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:183:
MetSerLeuLeuThrAsnArgLeuSerArgArgValAspLysAspGln
151015
TrpGlyProGlyPheMetGlyLysAspProLysProCysProSerArg
202530
ArgThrGlyLysCysMetGlyProProSerSerAlaAlaAlaCysSer
354045
ArgGlySerProArgIleLeuArgValArgAlaGlyGlyIleSerLeu
505560
ProTyrThrIleMetGluAlaLeuLeuPheLeuLeuGlyValGluAla
65707580
GlyAlaIleLeuAlaProAlaThrHisAlaCysArgAlaAsnGlyGln
859095
TyrPheLeuThrAsnCysCysAlaProGluAspIleGlyPheCysLeu
100105110
GluGlyGlyCysLeuValAlaLeuGlyCysThrValCysThrAspArg
115120125
CysTrpProLeuTyrGlnAlaGlyLeuAlaValArgProGlyLysSer
130135140
AlaAlaGlnLeuValGlyGlnLeuGlyGlyLeuTyrGlyProLeuSer
145150155160
ValSerAlaTyrValAlaGlyIleLeuGlyLeuGlyGluValTyrSer
165170175
GlyValLeuThrValGlyValAlaLeuThrArgArgValTyrProMet
180185190
ProAsnLeuThrCysAlaValGluCysGluLeuLysTrpGluSerGlu
195200205
PheTrpArgTrpThrGluGlnLeuAlaSerAsnTyrTrpIleLeuGlu
210215220
TyrLeuTrpLysValProPheAspPheTrpArgGlyValLeuSerLeu
225230235240
ThrProLeuLeuValCysValAlaAlaLeuLeuLeuLeuGluGlnArg
245250255
IleValMetValPheLeuLeuValThrMetAlaGlyMetSerGlnGly
260265270
AlaProAlaSerValLeuGlySerArgProPheAspTyrGlyLeuThr
275280285
TrpGlnSerCysSerCysArgAlaAsnGlySerArgTyrThrThrGly
290295300
GluLysValTrpAspArgGlyAsnValThrLeuLeuCysAspCysPro
305310315320
AsnGlyProTrpValTrpLeuProAlaPheCysGlnAlaIleGlyTrp
325330335
GlyAspProIleThrHisTrpSerHisGlyGlnAsnArgTrpProLeu
340345350
SerCysProGlnTyrValTyrGlySerValSerValThrCysValTrp
355360365
GlySerValSerTrpPheAlaSerThrGlyGlyArgAspSerLysIle
370375380
AspValTrpSerLeuValProValGlySerAlaSerCysThrIleAla
385390395400
AlaLeuGlySerSerAspArgAspThrValValGluLeuSerGluTrp
405410415
GlyValProCysAlaThrCysIleLeuAspArgArgProAlaSerCys
420425430
GlyThrCysValArgAspCysTrpProGluThrGlySerValArgPhe
435440445
ProPheHisArgCysGlyAlaGlyProLysLeuThrLysAspLeuGlu
450455460
AlaValProPheValAsnArgThrThrProPheThrIleArgGlyPro
465470475480
LeuGlyAsnGlnGlyArgGlyAsnProValArgSerProLeuGlyPhe
485490495
GlySerTyrAlaMetThrLysIleArgAspSerLeuHisLeuValLys
500505510
CysProThrProAlaIleGluProProThrGlyThrPheGlyPhePhe
515520525
ProGlyValProProLeuAsnAsnCysLeuLeuLeuGlyThrGluVal
530535540
SerGluAlaLeuGlyGlyAlaGlyLeuThrGlyGlyPheTyrGluPro
545550555560
LeuValArgArgArgSerGluLeuMetGlyArgArgAsnProValCys
565570575
ProGlyPheAlaTrpLeuSerSerGlyArgProAspGlyPheIleHis
580585590
ValGlnGlyHisLeuGlnGluValAspAlaGlyAsnPheIleProPro
595600605
ProArgTrpLeuLeuLeuAspPheValPheValLeuLeuTyrLeuMet
610615620
LysLeuAlaGluAlaArgLeuValProLeuIleLeuLeuLeuLeuTrp
625630635640
TrpTrpValAsnGlnLeuAlaValLeuGlyLeuProAlaValAspAla
645650655
AlaValAlaGlyGluValPheAlaGlyProAlaLeuSerTrpCysLeu
660665670
GlyLeuProThrValSerMetIleLeuGlyLeuAlaAsnLeuValLeu
675680685
TyrPheArgTrpMetGlyProGlnArgLeuMetPheLeuValLeuTrp
690695700
LysLeuAlaArgGlyAlaPheProLeuAlaLeuLeuMetGlyIleSer
705710715720
AlaThrArgGlyArgThrSerValLeuGlyAlaGluPheCysPheAsp
725730735
ValThrPheGluValAspThrSerValLeuGlyTrpValValAlaSer
740745750
ValValAlaTrpAlaIleAlaLeuLeuSerSerMetSerAlaGlyGly
755760765
TrpArgHisLysAlaValIleTyrArgThrTrpCysLysGlyTyrGln
770775780
AlaIleArgGlnArgValValArgSerProLeuGlyGluGlyArgPro
785790795800
ThrLysProLeuThrPheAlaTrpCysLeuAlaSerTyrIleTrpPro
805810815
AspAlaValMetMetValValValAlaLeuValLeuLeuPheGlyLeu
820825830
PheAspAlaLeuAspTrpAlaLeuGluGluLeuLeuValSerArgPro
835840845
SerLeuArgArgLeuAlaArgValValGluCysCysValMetAlaGly
850855860
GluLysAlaThrThrValArgLeuValSerLysMetCysAlaArgGly
865870875880
AlaTyrLeuPheAspHisMetGlySerPheSerArgAlaValLysGlu
885890895
ArgLeuLeuGluTrpAspAlaAlaLeuGluProLeuSerPheThrArg
900905910
ThrAspCysArgIleIleArgAspAlaAlaArgThrLeuAlaCysGly
915920925
GlnCysValMetGlyLeuProValValAlaArgArgGlyAspGluVal
930935940
LeuIleGlyValPheGlnAspValAsnHisLeuProProGlyPheVal
945950955960
ProThrAlaProValValIleArgArgCysGlyLysGlyPheLeuGly
965970975
ValThrLysAlaAlaLeuThrGlyArgAspProAspLeuHisProGly
980985990
AsnValMetValLeuGlyThrAlaThrSerArgSerMetGlyThrCys
99510001005
LeuAsnGlyLeuLeuPheThrThrPheHisGlyAlaSerSerArgThr
101010151020
IleAlaThrProValGlyAlaLeuAsnProArgTrpTrpSerAlaSer
1025103010351040
AspAspValThrValTyrProLeuProAspGlyAlaThrSerLeuThr
104510501055
ProCysThrCysGlnAlaGluSerCysTrpValIleArgSerAspGly
106010651070
AlaLeuCysHisGlyLeuSerLysGlyAspLysValGluLeuAspVal
107510801085
AlaMetGluValSerAspPheArgGlySerSerGlySerProValLeu
109010951100
CysAspGluGlyHisAlaValGlyMetLeuValSerValLeuHisSer
1105111011151120
GlyGlyArgValThrAlaAlaArgPheThrArgProTrpThrGlnVal
112511301135
ProThrAspAlaLysThrThrThrGluProProProValProAlaLys
114011451150
GlyValPheLysGluAlaProLeuPheMetProThrGlyAlaGlyLys
115511601165
SerThrArgValProLeuGluTyrGlyAsnMetGlyHisLysValLeu
117011751180
IleLeuAsnProSerValAlaThrValArgAlaMetGlyProTyrMet
1185119011951200
GluArgLeuAlaGlyLysHisProSerIleTyrCysGlyHisAspThr
120512101215
ThrAlaPheThrArgIleThrAspSerProLeuThrTyrSerThrTyr
122012251230
GlyArgPheLeuAlaAsnProArgGlnMetLeuArgGlyValSerVal
123512401245
ValIleCysAspGluCysHisSerHisAspSerThrValLeuLeuGly
125012551260
IleGlyArgValArgGluLeuAlaArgGluCysGlyValGlnLeuVal
1265127012751280
LeuTyrAlaThrAlaThrProProGlySerProMetThrGlnHisPro
128512901295
SerIleIleGluThrLysLeuAspValGlyGluIleProPheTyrGly
130013051310
HisGlyIleProLeuGluArgMetArgThrGlyArgHisLeuValPhe
131513201325
CysTyrSerLysAlaGluCysGluArgLeuAlaGlyGlnPheSerAla
133013351340
ArgGlyValAsnAlaIleAlaTyrTyrArgGlyLysAspSerSerIle
1345135013551360
IleLysAspGlyAspLeuValValCysAlaThrAspAlaLeuSerThr
136513701375
GlyTyrThrGlyAsnPheAspSerValThrAspCysGlyLeuValVal
138013851390
GluGluValValGluValThrLeuAspProThrIleThrIleSerLeu
139514001405
ArgThrValProAlaSerAlaGluLeuSerMetGlnArgArgGlyArg
141014151420
ThrGlyArgGlyArgSerGlyArgTyrTyrTyrAlaGlyValGlyLys
1425143014351440
AlaProAlaGlyValValArgSerGlyProValTrpSerAlaValGlu
144514501455
AlaGlyValThrTrpTyrGlyMetGluProAspLeuThrAlaAsnLeu
146014651470
LeuArgLeuTyrAspAspCysProTyrThrAlaAlaValAlaAlaAsp
147514801485
IleGlyGluAlaAlaValPhePheSerGlyLeuAlaProLeuArgMet
149014951500
HisProAspValSerTrpAlaLysValArgGlyValAsnTrpProLeu
1505151015151520
LeuValGlyValGlnArgThrMetCysArgGluThrLeuSerProGly
152515301535
ProSerAspAspProGlnTrpAlaGlyLeuLysGlyProAsnProVal
154015451550
ProLeuLeuLeuArgTrpGlyAsnAspLeuProSerLysValAlaGly
155515601565
HisHisIleValAspAspLeuValArgArgLeuGlyValAlaGluGly
157015751580
TyrValArgCysAspAlaGlyProIleLeuMetValGlyLeuAlaIle
1585159015951600
AlaGlyGlyMetIleTyrAlaSerTyrThrGlySerLeuValValVal
160516101615
ThrAspTrpAspValLysGlyGlyGlySerProLeuTyrArgHisGly
162016251630
AspGlnAlaThrProGlnProValValGlnValProProValAspHis
163516401645
ArgProGlyGlyGluSerAlaProSerAspAlaLysThrValThrAsp
165016551660
AlaValAlaAlaIleGlnValAspCysAspTrpSerValMetThrLeu
1665167016751680
SerIleGlyGluValLeuSerLeuAlaGlnAlaLysThrAlaGluAla
168516901695
TyrThrAlaThrAlaLysTrpLeuAlaGlyCysTyrThrGlyThrArg
170017051710
AlaValProThrValSerIleValAspLysLeuPheAlaGlyGlyTrp
171517201725
AlaAlaValValGlyHisCysHisSerValIleAlaAlaAlaValAla
173017351740
AlaTyrGlyAlaSerArgSerProProLeuAlaAlaAlaAlaSerTyr
1745175017551760
LeuMetGlyLeuGlyValGlyGlyAsnAlaGlnThrArgLeuAlaSer
176517701775
AlaLeuLeuLeuGlyAlaAlaGlyThrAlaLeuGlyThrProValVal
178017851790
GlyLeuThrMetAlaGlyAlaPheMetGlyGlyAlaSerValSerPro
179518001805
SerLeuValThrIleLeuLeuGlyAlaValGlyGlyTrpGluGlyVal
181018151820
ValAsnAlaAlaSerLeuValPheAspPheMetAlaGlyLysLeuSer
1825183018351840
SerGluAspLeuTrpTyrAlaIleProValLeuThrSerProGlyAla
184518501855
GlyLeuAlaGlyIleAlaLeuGlyLeuValLeuTyrSerAlaAsnAsn
186018651870
SerGlyThrThrThrTrpLeuAsnArgLeuLeuThrThrLeuProArg
187518801885
SerSerCysIleProAspSerTyrPheGlnGlnAlaAspTyrCysAsp
189018951900
LysValSerAlaValLeuArgArgLeuSerLeuThrArgThrValVal
1905191019151920
AlaLeuValAsnArgGluProLysValAspGluValGlnValGlyTyr
192519301935
ValTrpAspLeuTrpGluTrpIleMetArgGlnValArgMetValMet
194019451950
AlaArgLeuArgAlaLeuCysProValValSerLeuProLeuTrpHis
195519601965
CysGlyGluGlyTrpSerGlyGluTrpLeuLeuAspGlyHisValGlu
197019751980
SerArgCysLeuCysGlyCysValIleThrGlyAspValPheAsnGly
1985199019952000
GlnLeuLysGluProValTyrSerThrLysLeuCysArgHisTyrTrp
200520102015
MetGlyThrValProValAsnMetLeuGlyTyrGlyGluThrSerPro
202020252030
LeuLeuAlaSerAspThrProLysValValProPheGlyThrSerGly
203520402045
TrpAlaGluValValValThrProThrHisValValIleArgArgThr
205020552060
SerProTyrGluLeuLeuArgGlnGlnIleLeuSerAlaAlaValAla
2065207020752080
GluProTyrTyrValAspGlyIleProValSerTrpAspAlaAspAla
208520902095
ArgAlaProAlaMetValTyrGlyProGlyGlnSerValThrIleAsp
210021052110
GlyGluArgTyrThrLeuProHisGlnLeuArgLeuArgAsnValAla
211521202125
ProSerGluValSerSerGluValSerIleAspIleGlyThrGluThr
213021352140
GluAspSerGluLeuThrGluAlaAspLeuProProAlaAlaAlaAla
2145215021552160
LeuGlnAlaIleGluAsnAlaAlaArgIleLeuGluProHisIleAsp
216521702175
ValIleMetGluAspCysSerThrProSerLeuCysGlySerSerArg
218021852190
GluMetProValTrpGlyGluAspIleProArgThrProSerProAla
219522002205
LeuIleSerValThrGluSerSerSerAspGluLysThrProSerVal
221022152220
SerSerSerGlnGluAspThrProSerSerAspSerPheGluValIle
2225223022352240
GlnGluSerGluThrAlaGluGlyGluGluSerValPheAsnValAla
224522502255
LeuSerValLeuGluAlaLeuPheProGlnSerAspAlaThrArgLys
226022652270
LeuThrValArgMetAsnCysCysValGluLysSerValThrArgPhe
227522802285
PheSerLeuGlyLeuThrValAlaAspValAlaSerLeuCysGluMet
229022952300
GluIleGlnAsnHisThrAlaTyrCysAspLysValArgThrProLeu
2305231023152320
GluLeuGlnValGlyCysLeuValGlyAsnGluLeuThrPheGluCys
232523302335
AspLysCysGluAlaArgGlnGluThrLeuAlaSerPheSerTyrIle
234023452350
TrpSerGlyValProLeuThrArgAlaThrProAlaLysProProVal
235523602365
ValArgProValGlySerLeuLeuValAlaAspThrThrLysValTyr
237023752380
ValThrAsnProAspAsnValGlyArgArgValAspLysValThrPhe
2385239023952400
TrpArgAlaProArgValHisAspLysTyrLeuValAspSerIleGlu
240524102415
ArgAlaArgArgAlaAlaGlnAlaCysGlnSerMetGlyTyrThrTyr
242024252430
GluGluAlaIleArgThrValArgProHisAlaAlaMetGlyTrpGly
243524402445
SerLysValSerValLysAspLeuAlaThrProAlaGlyLysMetAla
245024552460
ValHisAspArgLeuGlnGluIleLeuGluGlyThrProValProPhe
2465247024752480
ThrLeuThrValLysLysGluValPhePheLysAspArgLysGluGlu
248524902495
LysAlaProArgLeuIleValPheProProLeuAspPheArgIleAla
250025052510
GluLysLeuIleLeuGlyAspProGlyArgValAlaLysAlaValLeu
251525202525
GlyGlyAlaTyrAlaPheGlnTyrThrProAsnGlnArgValLysGlu
253025352540
MetLeuLysLeuTrpGluSerLysLysThrProCysAlaIleCysVal
2545255025552560
AspAlaThrCysPheAspSerSerIleThrGluGluAspValAlaLeu
256525702575
GluThrGluLeuTyrAlaLeuAlaSerAspHisProGluTrpValArg
258025852590
AlaLeuGlyLysTyrTyrAlaSerGlyThrMetValThrProGluGly
259526002605
ValProValGlyGluArgTyrCysArgSerSerGlyValLeuThrThr
261026152620
SerAlaSerAsnCysLeuThrCysTyrIleLysValLysAlaAlaCys
2625263026352640
GluArgValGlyLeuLysAsnValSerLeuLeuIleAlaGlyAspAsp
264526502655
CysLeuIleIleCysGluArgProValCysAspProCysAspAlaLeu
266026652670
GlyArgAlaLeuAlaSerTyrGlyTyrAlaCysGluProSerTyrHis
267526802685
AlaSerLeuAspThrAlaProPheCysSerThrTrpLeuAlaGluCys
269026952700
AsnAlaAspGlyLysArgHisPhePheLeuThrThrAspPheArgArg
2705271027152720
ProLeuAlaArgMetSerSerGluTyrSerAspProMetAlaSerAla
272527302735
IleGlyTyrIleLeuLeuTyrProTrpHisProIleThrArgTrpVal
274027452750
IleIleProHisValLeuThrCysAlaPheArgGlyGlyGlyThrPro
275527602765
SerAspProValTrpCysGlnValHisGlyAsnTyrTyrLysPhePro
277027752780
LeuAspLysLeuProAsnIleIleValAlaLeuHisGlyProAlaAla
2785279027952800
LeuArgValThrAlaAspThrThrLysThrLysMetGluAlaGlyLys
280528102815
ValLeuSerAspLeuLysLeuProGlyLeuAlaValHisArgLysLys
282028252830
AlaGlyAlaLeuArgThrArgMetLeuArgSerArgGlyTrpAlaGlu
283528402845
LeuAlaArgGlyLeuLeuTrpHisProGlyLeuArgLeuProProPro
285028552860
GluIleAlaGlyIleProGlyGlyPheProLeuSerProProTyrMet
2865287028752880
GlyValValHisGlnLeuAspPheThrSerGlnArgSerArgTrpArg
288528902895
TrpLeuGlyPheLeuAlaLeuLeuIleValAlaLeuPheGly
290029052910
(2) INFORMATION FOR SEQ ID NO:184:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 39 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer GV5446IRT
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:184:
CGGTCCCTCGAACTCCAGCGAGTCTTTTTTTTTTTTTTT39
(2) INFORMATION FOR SEQ ID NO:185:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 70 amino acids
(B) TYPE: amino acid
(C) STRANDEDNESS: double
(D) TOPOLOGY: unknown
(ii) MOLECULE TYPE: protein
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: GE-CAP from T55806
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:185:
MetSerLeuLeuThrAsnArgPheIleArgArgValAspLysAspGln
151015
TrpGlyProGlyValThrGlyThrAspProGluProCysProSerArg
202530
TrpAlaGlyLysCysMetGlyProProSerSerAlaAlaAlaCysSer
354045
ArgGlySerProArgIleLeuArgValArgAlaGlyGlyIleSerLeu
505560
PheTyrThrIleMetAla
6570
(2) INFORMATION FOR SEQ ID NO:186:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 401 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: double
(D) TOPOLOGY: unknown
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: HGV-S59 Variant
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:186:
AGACGCAATGACTCGGCGCCGACTCGGCGACCGGCCAAAAGGTGGTGGATGGGTGATGAC60
AGGGTTGGTAGGTCGTAAATCCCGGTCACCTTGGTAGCCACTATAGGTGGGTCTTAAGAG120
AAGGTTAAGATTCCTCTTGTGCCTGCGGCGAGACCGCGCACGGTCCACAGGTGTTGGCCC180
TACCGGTGTGAATAAGGGCCCGACGTCAGGCTCGTCGTTAAACCGAGCCCGTTACCCACC240
TGGGCAAACGACGCCCACGTACGGTCCACGTCGCCCTTCAATGCCTCTCTTGGCCAATAG300
GTTTATCCGGCGAGTTGACAAGGACCAGTGGGGGCCGGGGGCTTGGGGAAGGACCTCAAG360
CCCTGCCCTTCCCGGTGGGGCGGGAAATGCATGGGGCCACC401
(2) INFORMATION FOR SEQ ID NO:187:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 401 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: double
(D) TOPOLOGY: unknown
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: HGV-S368 Variant
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:187:
AGACGCAATGACTCGGCGCCAACTCGGCGACCGGCCAAAAGGTGGTGGATGGGTGATGAC60
AGGGTTGGTAGGTCGTAAATCCCGGTCACCTTGGTAGCCACTATAGGTGGGTCTTAAGAG120
AAGGTTAAGATTCCTCTTGTGCCTGCGGCGAGACCGCGCACGGTCCACAGGTGTTGGCCC180
TACCGGTGTGAATAAGGGCCCGACGTCAGGCTCGTCGTTAAACCGAGCCCGTTACCCACC240
CGGGCAAACGACGCCCACGTACGGTCCACGTCGCCCTTCAATGTCTCTCTTGACCAATAG300
GCTTAGCCGGCGAGTTGACAAGGACCAGTGGGGGCCGGGGGCTTGGAGAGGGACTCCAAG360
TCCTGCCCTTCCCGGTGGGCCGGGAAATGCATGGGGCCACC401
(2) INFORMATION FOR SEQ ID NO:188:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 402 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: double
(D) TOPOLOGY: unknown
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: HGV-S309 VARIANT
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:188:
AGACGCAATGACTCGGCGCCGACTCGGCGACCGGCCAAAAGGTGGTGGATGGGTGATGAC60
AGGGTTGGTAGGTCGTAAATCCCGGTCATCCTGGTAGCCACTATAGGTGGGTCTTAAGAG120
AAGGTTAAGATTCCTCTTGTGCATGCGGCGAGAACGCGCACGGTCCACAGGTGTTGGCCC180
TACCGGTGTGAATAAGGGCCCGACGTCAGGCTCGTCGTTAAACCGAGCCCGTTACCCACC240
TGGGCAAACGACGCCCACGTACGGTCCACGTCGCCCTTCAATGTCTCTCTTGACCAATAG300
GTTTATCCGGCGAGTTGACAAGGACCAGTGGGGGCCGGGGGTCACGGGGAAGGACCCCGG360
ATCCTGCCCTTCCCGGTGGGCCGGGAAATGCATGGGGCCACC402
(2) INFORMATION FOR SEQ ID NO:189:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 402 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: double
(D) TOPOLOGY: unknown
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: HGV-FZ VARIANT
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:189:
AGACGCAATGACTCGGCGCCGACTCGGCGACCGGCCAAAAGGTGGTGGATGGGTGATGAC60
AGGGTTGGTAGGTCGTAAATCCCGGTCACCTTGGTAGCCACTATAGGTGGGTCTTAAGAG120
AAGGTTAAGATTCCTCTTGTGCCTGCGGCGAGACCGCGCACGGTCCACAGGTGTTGGCCC180
TACCGGTGTGAATAAGGGCCCGACGTCAGGCTCGTCGTTAAACCGAGCCCGCTACCCACC240
TGGGCAAACGACGCCCATGTACGGTCCACGTCGCCCTTCAATGTCTCTCTTGACCAATAG300
GATTCGTCCGGCGAGTTGACAAGGACCAGTGGGGGCCGGGGGCCTGGGGAAGGACCCCAG360
ACCCTGCCCTTCCCGGTGGGACGGGAAATGCATGGGGCCACC402
(2) INFORMATION FOR SEQ ID NO:190:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 401 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: double
(D) TOPOLOGY: unknown
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: HGV-G21 VARIANT
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:190:
AGACGCAATGACTCGGCGCCGACTCGGCGACCGGCCAAAAGGTGGTGGATGGGTGATGAC60
AGGGTTGGTAGGTCGTAAATCCCGGTCACCTTGGTAGCCACTATAGGTGGGTCTTAAGAG120
AAGGTTAAGATTCCTCTTGTGCCTGCGGCGAGACCGCGCACGGTCCACAGGTGTTGGTCC180
TACCGGTGTGAATAAGGACCCGACGTCAGGCTCGTCGTTAAACCGAGCCCGTTACCCACC240
TGGGCAAACGACGCCCACGTACGGTCCACGTCGCCCTTCAATGTCTCTCTTGACCAATAG300
GCTTAGCCGGCGAGTTGACAAGGACCAGTGGGGGCCGGGGGCTTGGGGAAGGACCCCAAG360
CCCTGCCCTTCCCGGTGGGCCGGGAAATGCATGGGGCCACC401
(2) INFORMATION FOR SEQ ID NO:191:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 402 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: double
(D) TOPOLOGY: unknown
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: HGV-G23 VARIANT
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:191:
AGACGCAATGACTCGGCGCCAACTCGGCGACCGGCCAAAAGGTGGTGGATGGGTGATGAC60
AGGGTTGGTAGGTCGTAAATCCCGGTCACCTTGGTAGCCACTATAGGTGGGTCTTAAGAG120
AAGGTTAAGATTCCTCTTGTGCCTGCGGCGAGACCGCGCACGGTCCGCAGGTGTTGGCCC180
TACCGGTGTGAATAAGGGCCCGACATCAGGCATGTCGTTAAACCGAGCCCGTTACCCGCC240
TGGGCTAACGACGCCCACGTACGGTCCACGTCGCCCTTCAATGTCTCTCTTGACCAATAG300
GTTTATCCGGCGAGTTGACAAGGACCAGTGGGGGCCGGGGGTTACGGGGAAGGACCCCGA360
ACCCTGCCCTTCCCGGCGGACCGGGAAATGCATGGGGCCACC402
(2) INFORMATION FOR SEQ ID NO:192:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 405 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: double
(D) TOPOLOGY: unknown
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: HGV-G59 VARIANT
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:192:
AGACGCAATGACTCGGCGCCGACTCGGCGACCGGCCAAAAGGTGGTGGATGGGTGATGCC60
AGGGTTGGTAGGTCGTAAATCCCGGTCATCTTGGTAGCCACTATAGGTGGGTCTTAAGAG120
AAGGTTAAGATTCCTCTTGTGCCTGCGGCGAGACCGCGCACGGTCCACAGGTGTTGGCCC180
TACCGGTGGGAATAAGGGCCCGACGTCAGGCTCGTCGTTAAACTGAGCCCGTAACCCACC240
TGGGCAAACGACGCCCACGTACGGTCCACGTCGCCCTTCAATGCCTCTCTTGGCCAATAG300
GGATTATTCCCGGCGAGTTGGCAAGGACCAGTGGGGGCCGGGAGCTACAGAGAAGGACTC360
TGAGCTCTGCCCTTCCCGGTGGAACGGGAAATGCATGGGGCCACC405
(2) INFORMATION FOR SEQ ID NO:193:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 402 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: double
(D) TOPOLOGY: unknown
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: HGV-E36 VARIANT
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:193:
AGACGCAATGACTCGGCGCCGACTCGGCGACCGGCCAAAAGGTGGTGGATGGGTGATGAC60
AGGGTTGGTAGGTCGTAAATCCCGGTCACCTTGGTAGCCACTATAGGTGGGTCTTAAGAG120
AAGGTTAAGATTCCTCTTGTGCCTGCGGCGAGACCGCGCACGGCCCACAGGTGTTGGCCC180
TACCGGTGTGAATAAGGGCCCGACGTCAGGCTCGTCGTTAAACCGAGCCCACTACCCACC240
TGGGCAAACGACGCCCACGTACGGTCTACGTCGCCCTTCAATGTCTCTCTTGACCAATAG300
GCTAAGCCGGCGAGTTGACAAAGACCAGTGGGGGCCGGGGGTCACAGGGATGGACCCTGG360
ACCCTGCCCTTCCCGGTGGAGTGGGAAATGCATGGGGCCACC402
(2) INFORMATION FOR SEQ ID NO:194:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 402 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: double
(D) TOPOLOGY: unknown
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: HGV-R38730 VARIANT
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:194:
AGACGCAATGACTCGGCGCCGACTCGGCGACCGGCCAAAAGGTGGTGGATGGGTGATGAC60
AGGGTTGGTAGGTCGTAAATCCCGGTCACCTTGGTAGCCACTATAGGTGGGTCTTAAGAG120
AAGGTTAAGGATCCTCTTGTGCCTGCGGCGAGACCGCGCACGGTCCACAGGTGTTGGCCC180
TACCGGTGTGAATAAGGGCCCGACGTCAGGCTCGTCGTTAAACCGAGCCCGTATCCCACC240
TGGGCAAACGACGCCCACGTACGGTCCACGTCGCCCTTCAATGTCTCTCTTGACCAATAG300
GTTCGTCCGGCGAGTTGACAAGGACCAGTGGGGGCCGGGGGTTGCGGGGAAGGACCCCGA360
ACTCTGCCCTTCCCGGTGGGCCGGGAAATGCATGGGGCCACC402
(2) INFORMATION FOR SEQ ID NO:195:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 401 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: double
(D) TOPOLOGY: unknown
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: HGV-G281 VARIANT
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:195:
AGACGCAATGACTCGGCGCCGACTCGGCGACCGGCCAAAAGGTGGTGGATGGGTGATGAC60
AGGGTTGGTAGGTCGTAAATCCCGGTCACCTTGGTAGCCACTATAGGTGGGTCTTAAGAG120
AAGGTTAAGATTCCTCTTGTGCCTGCGGCGAGACCGCGCACGGTCCACAGGTGTTGGTCC180
TACCGGTGTGAATAAGGACCCGACGTCAGGCTCGTCGTTAAACCGAGCCCGTTACCCACC240
TGGGCAAACGACGCCCACGTACGGTCCACGTCGCCCTTCAATGTCTCTCTTGACCAATAG300
GCTTAGCCGGCGAGTTGACAAGGACCAGTGGGGGCCGGGGGCTTGGGGAAGGACCCCAAG360
CCCTGCCCTTCCCGGTGGGCCGGGAAATGCATGGGGCCACC401
(2) INFORMATION FOR SEQ ID NO:196:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 402 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: double
(D) TOPOLOGY: unknown
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: HGV-G157 VARIANT
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:196:
AGACGCAATGACTCGGCGCCGACCCGGCGACCGGCCAAAAGGTGGTGGATGGGTGATGCC60
AGGGTTGGTAGGTCGTAAATCCCGGTCATCTTGGTAGCCACTATAGGTGGGTCTTAAGGG120
AAGGTTAAGATTCCTCTTGTGCCTGTGGCGAGACAGCGCACGGTCCACAGGTGTTGGCCC180
TACCGGTGTGAATAAGGGCCCGACGTCAGGCTCGTCGTTAAACCGAGACCGACACCCACC240
TGGGCAAACGACGCCCACGTACGGTCCACGTCGCCCTTCAATGTCTCTCTTGACCAATAG300
GCTTTGCCGGCGAGTTGACAAGGACCAGTGGGGGCCGGGTGCTGGGGGAAGGACCCCCTT360
GCACCGCCCTTCCCGGTGGGACGGGAAATGCATGGGGCCACC402
(2) INFORMATION FOR SEQ ID NO:197:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 401 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: double
(D) TOPOLOGY: unknown
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: HGV-G154 VARIANT
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:197:
AGACGCAATGACTCGGCGCCGACTCGGCGACCGGCCAAAAGGTGGTGGATGGGTGATGAC60
AGGGTTGGTAGGTCGTAAATCCCGGTCATCCTGGTAGCCACTATAGGTGGGTCTTAAGAG120
AAGGTTAAGATTCCTCTTACGCCTGCGGCGAGACCGCGCACGGTCCACAGGTGCTGGCCT180
TACCGGTGTGAATAAAGGCCCGACGTCAGGCTCGTCGTTAAACCGAGCCCGTCACCCACC240
TGGGCAAACGACGCCCACGTACGGTCCACGTCGCCCTTCAATGTCTCTCTTGACCAGTAG300
GTTTAACCGGCGAGTTGACAAGGACCAGTGGGGGCCGGGGCCTTGGAGATGGACTCCAAG360
TCCTGCCCTTCCCGGTGGGCCGGGAAATGCATGGGGCCACC401
(2) INFORMATION FOR SEQ ID NO:198:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 401 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: double
(D) TOPOLOGY: unknown
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: HGV-G213 VARIANT
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:198:
AGACGCAATGACTCGGCGCCAACTCGGCGACCGGCCAAAAGGTGGTGGATGGGTGATGAC60
AGGGTTGGTAGGTCGTAAATCCCGGTCACCTTGGTAGCCACTATAGGTGGGTCTTAAGAG120
AAGGTTAAGATTCCTCTTGTGCCTGCGGCGAGACCGCGCACGGTCCACAGGTGTTGGTCC180
TACCGGTGGGAATAAGGGCCCGACGTCAGGCTCGTCGTTAAACCGAGCCCGTCACCCACC240
TGGGCAAACGACGCCCACGTATGGTCCACGTCGCCCTTCAATGCCTCTCTTGGCCAATAG300
GTTTATCCGGCGAGTTGACAAGGACCAGTGGGGGCCGGGGGTTCGGGGAAGGACCCCGTA360
CCCTGCCCTTCCCGGTGGAACGGGAAATGCATGGGGCCACC401
(2) INFORMATION FOR SEQ ID NO:199:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 401 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: double
(D) TOPOLOGY: unknown
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: HGV-G204 VARIANT
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:199:
AGACGCAATGACTCGGCGCCGACTCGGCGACCGGCCAAAAGGTGGTGGATGGGTGATGAC60
AGGGTTGGTAGGTCGTAAATCCCGGTCACCTTGGTAGCCACTATAGGTGGGTCTTAAGAG120
AAGGTTAAGATTCCTCTTGTGCCTGCGGCGAGACCGCGCACGGTCCACAGGTGTTGGCCC180
TACCGGTGTTAATAAGGGCCCGACGTCAGGCTCGTCGTTAAACCGAGCCCGTCACCCACC240
TGGGCAAACGACGCCCACGTACGGTCCACGTCGCCCTTCAATGTCTCTCTTGACCAATAG300
GCTTAGCCGGCGAGTTGACAAGGACCAGTGGGGGCCGGGGGCCTGGAGAGGGACTCCAGG360
TCCTGCCCTTCCCGGTGGGCCGGGAAATGCATGGGGCCACC401
(2) INFORMATION FOR SEQ ID NO:200:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 402 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: double
(D) TOPOLOGY: unknown
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: HGV-G191 VARIANT
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:200:
AGACGCAATGACTCGGCGCCGACTCGGCGACCGGCCAAAAGGTGGTGGATGGGTGATGAC60
AGGGTTGGTAGGTCGTAAATCCCGGTCATCCTGGTAGCCACTATAGGTGGGTCTTAAGAG120
AAGGTTAAGGATCCTCTTGTGCCTGCGGCGAGACCGCGCACGGTCCACAGGTGTTGGCCC180
TACCGGTGTGAATAAGGGCCCGACGTCAGGCTCGTCGCTAAACCGAGCCCGTATCCCACC240
TGGGCAAACGACGCCCACGTACGGTCCACGTCGCCCTTCAATGTCTCTCTTGACCAATAG300
GTTTATCCGGCGAGTTGACAAGGACCAGTGGGGGCCGGAGGTTACGGGGAAGGACCCCGA360
GCCTCGCCCTTCCCGGTGGGCCGGGAAATGCATGGGGCCACC402
(2) INFORMATION FOR SEQ ID NO:201:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 402 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: double
(D) TOPOLOGY: unknown
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: HGV-G299 VARIANT
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:201:
AGACGCAATGACTCGGCGCCGACTCGGCGACCGGCCAAAAGGTGGTGGATGGGTGATGAC60
AGGGTTGGTAGGTCGTAAATCCCGGTCACCTTGGTAGCCACTATAGGTGGGTCTTAAGAG120
AAGGTTAAGATTCCTCTTGTGCCTGCGGCGAGACCGCGCACGGTCCACAGGTGTTGGCCC180
TACCGGTGTGAATAAGGGCCCGACGTCAGGCTCGTCGTTAAACCGAGCCCGTCACCCACC240
TGGGCAAACGACGCCCACGCACGGTCCACGTCGCCCTTCAATGCCTCTCTTGGCCAATAG300
GAGTATCCGGCGAGTTGACAAGGACCAGTGGGGGCCGGGAGTCACGGGGATGGACCCCGG360
GCTCTGCCCTTCCCGGTGGAACGGGAAATGCATGGGGCCACC402
(2) INFORMATION FOR SEQ ID NO:202:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 402 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: double
(D) TOPOLOGY: unknown
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: HGV-T56957 VARIANT
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:202:
AGACGCAATGACTCGGCGCCGACTCGGCGACCGGCCAAAAGGTGGTGGATGGGTGATGAC60
AGGGTTGGTAGGTCGTAAATCCCGGTCACCTTGGTAGCCACTATAGGTGGGTCTTAAGAG120
AAGGTTAAGATTCCTCTTGTGCCTGCGGCGAGACCGCGCACGGTCCACAGGTGTTGGCCC180
TACCGGTGTGAATAAGGGCCCGACGTCAGGCTCGTCGTTAAACCGAGCCCATCACCCACC240
TGGGCAAACGACGCCCACGTACGGTCCACGTCGCCCTACAATGTCTCTCTTGACCAATAG300
GCTTAGCCGGCGAGTTGACAAGGACCAGTGGGGGCCGGGGGTCACAGGGATGGACCCTGG360
GCCCTGCCCTTCCCGGTGGGGTGGGAAATGCATGGGGCCACC402
(2) INFORMATION FOR SEQ ID NO:203:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 401 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: double
(D) TOPOLOGY: unknown
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: HGV-C01698 VARIANT
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:203:
AGACGCAATGACTCGGCGCCGACTCGGCGACCGGCCAAAAGGTGGTGGATGGGTGATGAC60
AGGGTTGGTAGGTCGTAAATCCCGGTCACCTTGGTAGCCACTATAGGTGGGTCTTAAGAG120
AAGGTTAAGATTCCTCTTGTGCCTGCGGCGAGACCGCGCACGGTCCACAGGTGTTGGCCC180
TACCGGTGTGAATAAGGGCCCGACGTCAGGCTCGTCGTTAAACCGAGCCCGTCACCCACC240
TGGGCAAACGACGCCCACGTACGGTCCACGTCGCCCTTCAATGTCTCTCTTGACCAATAG300
GCTTAGCCGGCGAGTTGACAAGGACCAGTGGGGGCCGGGGGCTTGGAGATGGACTCCAAG360
TCCTGCCCTTCCCGGTGGGCCGGGAAATGCATGGGGCCACC401
(2) INFORMATION FOR SEQ ID NO:204:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 402 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: double
(D) TOPOLOGY: unknown
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: HGV-T27034 VARIANT
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:204:
AGACGCAATGACTCGGCGCCGACTCGGCGACCGGCCAAAAGGTGGTGGATGGGTGATGAC60
AGGGTTGGTAGGTCGTAAATCCCGGTCACCTTGGTAGCCACTATAGGTGGGTCTTAAGAG120
AAGGTTAAGATTCCTCTTGTGCCTGCGGCGAGACCGCGCACGGTCCACAGGTGTTGGCCC180
TACCGGTGTGAATAAGGGCCCGACGTCAGGCTCGTCGTTAAACCGAGCCCATTTCCCGCC240
TGGGCTAACGACGCCCACGTACGGTCCACGTCGCCCTTCAATGTCTCTCTTGACCAATAG300
GTTTATCCGGCGAGTTGACAAGGACCAGTGGGGGCCGGGAGTCACTGGGATGGACCCAGG360
GCTCTGCCCTTCCCGGCGGGGTGGGAAAAGCATGGGGCCACC402
(2) INFORMATION FOR SEQ ID NO:205:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 401 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: double
(D) TOPOLOGY: unknown
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: HGV-E57963 VARIANT
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:205:
AGACGCAATGACTCGGCGCCGACTCGGCGACCGGCCAAAAGGTGGTGGATGGGTGATGAC60
AGGGTTGGTAGGTCGTAAATCCCGGTCACCTTGGTAGCCACTATAGGTGGGTCTTAAGAG120
AAGGTTAAGATTCCTCTTGTGCCTGCGGCGAGACCGCGCACGGTCCGCAGGTGTTGGCCC180
TACCGGTGTGAATAAGGGCCCGACGTCAGGCTCGTCGTTAAACCGAGCCCGTCACCCACC240
TGGGCAAACGACGCCCACGTACGGTCCACGTCGCCCTTCAATGTCTCTCTTGACCAATAG300
GCTTAGCCGGCGAGTTGACAAGGACCAGTGGGGGCCGGGGGCTTGGAGAAGGACTCCAAG360
TCCTGCCCTTCCCGGTGGGCCGGGAAATGCATGGGGCCACC401
(2) INFORMATION FOR SEQ ID NO:206:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 401 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: double
(D) TOPOLOGY: unknown
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: HGV-R37166 VARIANT
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:206:
AGACGCAATGACTCGGCGCCGACTCGGCGACCGGCCAAAAGGTGGTGGATGGGTGATGAC60
AGGGTTGGTAGGTCGTAAATCCCGGTCACCTTGGTAGCCACTATAGGTGGGTCTTAAGAG120
AAGGTTAAGATTCCTCTTGTGCCTGCGGCGAGACCGCGCACGGTCCACAGGTGTTGGCCC180
TACCGGTGTGAATAAGGGCCCGACGTCAGGCTCGTCGTTAAACCGAGCCCGTAACCCGCC240
TGGGCAAACGACGCCCACGTACGGTCCACGTCGCCCTTCAATGTCTCTCTTGACCAATAG300
GTTTAACCGGCGAGTTGACAAGGACCAGTGGGGGCCGGGGCCTTGGAGATGGACTCCAAG360
TCCTGCCCTTCCCGGCGGGCCGGGAAATGCATGGGGCCACC401
(2) INFORMATION FOR SEQ ID NO:207:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 404 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: double
(D) TOPOLOGY: unknown
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: HGV-B5 VARIANT
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:207:
AGACGCAATGACTCGGCGCCGACTCGGCGACCGGCCAAAAGGTGGTGGATGGGTGATGAC60
AGGGTTGGTAGGTCGTAAATCCCGGTCATCCTGGTAGCCACTATAGGTGGGTCTTAAGGG120
AAGGTTAAGATTCCTCTTGTGCCTGCGGCGAGACCGCGCACGGTCCACAGGTGTTGGCCC180
TACCGGTGTGAATAAGGGCCCGACGTCAGGCTCGTCGTTAAACCGAGCCCGTCACCCACC240
TGGGCTAACGACGCCCACGTACGGTCCACGTCGCCCTTCAATGTCTCTCTTGACCAATAG300
GCTTTTTGCCGGCGAGTTGACAAGGACCAGTGGGGGCCGGGGGTTATGGGGAAGGACCCC360
AAACCCTGCCCTTCCCGGTGGGCCGGGAAATGCATGGGGCCACC404
(2) INFORMATION FOR SEQ ID NO:208:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 402 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: double
(D) TOPOLOGY: unknown
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: HGV-B33 VARIANT
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:208:
AGACGCAATGACTCGGCGCCGACTCGGCGACCGGCCAAAAGGTGGTGGATGGGTGATGAC60
AGGGTTGGTAGGTCGTAAATCCCGGTCATCCTGGTAGCCACTATAGGTGGGTCTTAAGAG120
AAGGTTAAGATTCCTCTTGTGCCTGCGGCGAGACCGCGCACGGTCCACAGGTGTTGGCCC180
TACCGGTGTGAATAAGGGCCCGACGTCAGGCTCGTCGTTAAACCGAGCCCGTTCCCCGCC240
TGGGCAAACGACGCCCACGTACGGTCCACGTCGCCCTTCAATGTCTCTCTTGACCAATAG300
GTTTATCCGGCGAGTTGACAAGGACCAGTGGGGGCCGGGGATCATGGGGAAGGACCCCAG360
ATCCTGCCCTTCCCGGCGGGCCGGGAAATGCATGGGGCCACC402
(2) INFORMATION FOR SEQ ID NO:209:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 401 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: double
(D) TOPOLOGY: unknown
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: HGV-FH010 VARIANT
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:209:
AGACGCAATGACTCGGCGCCGACCCGGCGACCGGCCAAAAGGTGGTGGATGGGTGATGCC60
AGGGTTGGTAGGTCGTAAATCCCGGTCATCTTGGTAGCCACTATAGGTGGGTCTTAAGGG120
AAGGTTAAGATTCCTCTTGTGCCTGTGGCGAGACAGCGCACGGTCCACAGGTGTTGGCCC180
TACCGGTGTGAATAAGGGCCCGACGTCAGGCTCGTCGTTAAACTGAGACCGACACCCACC240
TGGGCAAACGACGCCCACGTACGGTCCACGTCGCCCTTCAATGTCTCTCTTGACCAATAG300
GCTTTGCCGGCGAGTTGACAAGGACCAGTGGGGGCCGGGGGCTGGGGGAAGGACCCCCAG360
TCCTGCCCTTCCCGGTGGGACGGGAAATGCATGGGGCCACC401
(2) INFORMATION FOR SEQ ID NO:210:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 401 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: double
(D) TOPOLOGY: unknown
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: HGV-PNF2161 VARIANT
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:210:
AGACGCAATGACTCGGCGCCGACTCGGCGACCGGCCAAAAGGTGGTGGATGGGTGATGAC60
AGGGTTGGTAGGTCGTAAATCCCGGTCACCTTGGTAGCCACTATAGGTGGGTCTTAAGAG120
AAGGTTAAGATTCCTCTTGTGCCTGCGGCGAGACCGCGCACGGTCCACAGGTGTTGGCCC180
TACCGGTGGGAATAAGGGCCCGACGTCAGGCTCGTCGTTAAACCGAGCCCGTTACCCACC240
TGGGCAAACGACGCCCACGTACGGTCCACGTCGCCCTTCAATGTCTCTCTTGACCAATAG300
GCGTAGCCGGCGAGTTGACAAGGACCAGTGGGGGCCGGGGGCTTGGAGAGGGACTCCAAG360
TCCCGCCCTTCCCGGTGGGCCGGGAAATGCATGGGGCCACC401
(2) INFORMATION FOR SEQ ID NO:211:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 402 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: double
(D) TOPOLOGY: unknown
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: HGV-JC VARIANT
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:211:
AGACGCAATGACTCGGCGCCGACTCGGCGACCGGCCAAAAGGTGGTGGATGGGTGATGAC60
AGGGTTGGTAGGTCGTAAATCCCGGTCACCTTGGTAGCCACTATAGGTGGGTCTTAAGAG120
AAGGTTAAGATTCCTCTTGTGCCTGCGGCGAGACCGCGCACGGTCCACAGGTGTTGGCCC180
TACCGGTGGGAATAAGGGCCCGACGTCAGGCTCGTCGTTAAACCGAGCCCGTAACCCGCC240
TGGGCAAACGACGCCCACGTACGGTCCACGTCGCCCTTCAATGTCGCTCTTGACCAATAG300
GCTTAGCCGGCGAGTTGACAAGGACCAGTGGGGGCCGGGGTTTATGGGGAAGGACCCCAA360
ACCCTGCCCTTCCCGGCGGACCGGGAAATGCATGGGGCCACC402
(2) INFORMATION FOR SEQ ID NO:212:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 401 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: double
(D) TOPOLOGY: unknown
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: HGV-7155 VARIANT
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:212:
AGACGTTATGAACCGGCGCCGCCCCGGCGACCGGCCAAAAGGTGGTGGATGGGTGATGCC60
AGGGTTGGTAGGTCGTAAATCCCGGTCATCTTGGTAGCCACTATAGGTGGGTCTTAAGGG120
GTGGTCAAGGTCCCTCTAGCGCTTGTGGCGAGAAAGCGCACGGTCCACAGGTGTTGGCCC180
TACCGGTGTGAATAAGGGCCCGACGTCAGGCTCGTCGTTAAACCGAGCCCATTATCCTCC240
TGGGCAAACGACGCCCACGTACGGTCCACGTCGCCCTTCAATGTCTCTCTTGACCAATAG300
GCTTTGCCGGCGAGTTGACAAGGACCAGTGGGGGCCGGGTGCCGGGGGAAGGACCCCCGG360
TACTGCCCCTCCCGGAGGAGTGGGAAATGCATGGGGCCACC401
(2) INFORMATION FOR SEQ ID NO:213:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 401 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: double
(D) TOPOLOGY: unknown
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: HGV-7244 VARIANT
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:213:
AGACGTTAAGAACCGGCGCCGCCCCGGCGACCGGCCAAAAGGTGGTGGATGGGTGATGCC60
AGGGTTGGTAGGTCGTAAATCCCGGTCATCTTGGTAGCCACTATAGGTGGGTCTTAAGGG120
GTGGTCAAGGTCCCTCTGGCGCTTGTGGCGAGAAAGCGCACGGTCCACAGGTGTTGGCCC180
TACCGGTGTGAATAAGGGCCCGACGTCAGGCTCGTCGTTAAACCGAGCCCATTACCCTCC240
TGGGCAAACGACGCCCATGTACGGTCCACGTCGCCCTTCAATGTCTCTCTTGACCAATAG300
GCTTTGCCGGCGAGTTGACAAGGACCAGTGGGGGCCGGGTGGCGGGGGAAGGACCCCCGT360
CACTGCCCTTCCCGGAGGGGTGGGAAATGCATGGGGCCACC401
(2) INFORMATION FOR SEQ ID NO:214:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 401 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: double
(D) TOPOLOGY: unknown
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: HGV-K27 VARIANT
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:214:
AGACGTTAAGTACCGGCGCCGACCCGGCGACCGGCCAAAAGGTGGTGGATGGGTGATGCC60
AGGGTTGGTAGGTCGTAAATCCCGGTCATCTTGGTAGCCACTATAGGTGGGTCTTAAGGG120
TTGGTCAAGGTCCCTCTGGCGCTTGTGGCGAGAAAGCGCACGGTCCACAGGTGTTGGCCC180
TACCGGTGTGAATAAGGGCCCGACGTCAGGCTCGTCGTTAAACCGAGCCCATTACCCACC240
TGGGCAAACAACGCCCACGTACGGTCCACGTCGCCCTACAATGTCTCTCTTGACCAATAG300
GCTTTGCCGGCGAGTTGACAAGGACCAGTGGGGGCTGGGCGGCGAGGGAAGGACCCTCGT360
CGCTGCCCTTCCCGGCGGGGTGGGGAATGCATGGGGCCACC401
(2) INFORMATION FOR SEQ ID NO:215:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 401 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: double
(D) TOPOLOGY: unknown
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: HGV-K30 VARIANT
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:215:
AGACGTTAAGAACCGGCGCCTTCCCGGCGACCGGCCAAAAGGTGGTGGATGGGTGATGCC60
AGGGTTGGTAGGTCGTAAGTCCCGGTCATCTTGGTAGCCACTATAGGTGGGTCTTAAGGG120
AGGGTTAAGGTCCCTCTGGCGCTTGTGGCGAGAAAGCGCACGGTCCACAGGTGTTGGCCC180
TACCGGTGTGAATAAGGGCCCGACGTCAGGCTCGTCGTTAAACCGAGCCCATTACCCACC240
TGGGCAAACAACGCCCACGTACGGTCCACGTCGCCCTTCAATGTCTCTCTTGACCAATAG300
GCTTTGCCGGCGAGTTGACAAGGACCAGTGGGGGCTGGGCGGTAGGGGAAGGACCCTTGC360
CGCTGCCCTTCCCGGTGGGGTGGGAAATGCATGGGGCCACC401
(2) INFORMATION FOR SEQ ID NO:216:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 401 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: double
(D) TOPOLOGY: unknown
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: HGV-T55875 VARIANT
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:216:
AGACGCAATGACTCGGCGCCGACTCGGCGACCGGCCAAAAGGTGGTGGATGGGTGATGAC60
AGGGTTGGTAGGTCGTAAATCCCGGTCACCTTGGTAGCCACTATAGGTGGGTCTTAAGAG120
AAGGTTAAGATTCCTCTTGTGCCTGCGACGAGACCGCGCACGGTCCGCAGGTGTTGGCCC180
TACCGGTGTGAATAAGGGCCCGACGTCAGGCTCGTCGTTAAACCGAGCCCGTCACCCACC240
TGGGCAAACGACGCCCACGTACGGTCCACGTCGCCCTTCAATGCCTCTCTTGGCCAATAG300
GTTTAACCGGCGAGTTGGCAAGGACCAGTGGGGGCCGGGGGCTTGGAGAGGGACTCCAAG360
TCCTGCCCTTCCCGGTGGGCCGGGAAATGCATGGGGCCACC401
(2) INFORMATION FOR SEQ ID NO:217:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 402 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: double
(D) TOPOLOGY: unknown
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: HGV-T56633 VARIANT
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:217:
AGACGCAATGACTCGGCGCCGACTCGGCGACCGGCCAAAAGGTGGTGGATGGGTGATGAC60
AGGGTTGGTAGGTCGTAAATCCCGGTCACCTTGGTAGCCACTATAGGTGGGTCTTAAGAG120
AAGGTTAAGATTCCTCTTGTGCCTGCGGCGAGACCGCGCACGGTCCACAGGTGTTGGCCC180
TACCGGTGTGAATAAGGGCCCGACGTCAGGCTCGTCGTTAAACCGAGCCCACTACCCACC240
TGGGCTAACGACGCCCACGTACGGTCCACGTCGCCCTTCAATGTCTCTCTTGACCAATAG300
GCTAGTCCGGCGAGTTGACAAGGACCAGTGGGGGCCGGAGGTCACAGGGATGGACCCTGG360
GCCTTGCCCTTCCCGGTGGAGTGGGAAAAGCATGGGGCCACC402
(2) INFORMATION FOR SEQ ID NO:218:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 404 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: double
(D) TOPOLOGY: unknown
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: HGV-EB20 VARIANT
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:218:
AGACGCAATGACTCGGCGCCGACTCGGCGACCGGCCAAAAGGTGGTGGATGGGTGATGCC60
AGGGTTGGTAGGTCGTAAATCCCGGTCATCTTGGTAGCCACTATAGGTGGGTCTTAAGAG120
AAGGTTAAGATTCCTCTTGTGCCTGCGGCGAGACCGCGCACGGTCCACAGGTGTTGGCCC180
TACCGGTGTAATAAGGGCCCGACGTCAGGCTCGTCGTTAAACCGAGCCCGTCACCCACCT240
GGGCAAACGACGCCCACGTACGGTCCACGTCGCCCTTCAATGCCTCTCTTGGCCAATAGG300
AGTTATCTCCGGCGAGTTGGCAAGGACCAGTGGGGGCCGGGGGTTACGGGGAAGGACCCC360
GAACCCTGCCCTTCCCGGTGGGCCGGGAAATGCATGGGGCCACC404
(2) INFORMATION FOR SEQ ID NO:219:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 401 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: double
(D) TOPOLOGY: unknown
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: HGV-T55806 VARIANT
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:219:
AGACGCAATGACTCGGCGCCGACTCGGCGACCGGCCAAAAGGTGGTGGATGGGTGATGCC60
AGGGTTGGTAGGTCGTAAATCCCGGTCATCTTGGTAGCCACTATAGGTGGGTCTTAAGAG120
AAGGTTAAGATTCCTCTTGTGCCTGCGGCGAGACCGCGCACGGTCCACAGGTGTTGGCCC180
TACCGGTGGAATAAGGGCCCGACGTCAGGCTCGTCGTTAAACCGAGCCCGTCACCCACCT240
GGGCAAACGACGCTCACGTACGGTCCACGTCGCCCTTCAATGTCTCTCTTGACCAATAGG300
TTTATCCGGCGAGTTGACAAGGACCAGTGGGGGCCGGGGGTTACGGGGACGGACCCCGAA360
CCCTGCCCTTCCCGGTGGGCCGGGAAATGCATGGGGCCACC401
(2) INFORMATION FOR SEQ ID NO:220:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 402 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: double
(D) TOPOLOGY: unknown
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: HGV-BG34 VARIANT
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:220:
AGACGCAATGACTCGGCGCCGACTCGGCGACCGGCCAAAAGGTGGTGGATGGGTGATGAC60
AGGGTTGGTAGGTCGTAAATCCCGGTCACCTTGGTAGCCACTATAGGTGGGTCTTAAGAG120
AAGGTTAAGATTCCTCTTGTGCCTGCGGCGAGACCGCGCACGGTCCACAGGTGTTGGCCC180
TACCGGTGTGAATAAGGGCCCGACGTCAGGCTCGTCGTTAAACCGAGCCCGTCACCCACC240
TGGGCAAACGACGCCCACGTACGGTCCACGTCGCCCTTCAATGCCTCTCTTGGCCAATAG300
GAGTATCCGGCGAGTTGACAAGGACCAGTGGGGGCCGGGAGTCACGGGGATGGACCCCGG360
GCTCTGCCCTTCCCGGTGGAACGGGAAACGCATGGGGCCACC402
(2) INFORMATION FOR SEQ ID NO:221:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 402 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: double
(D) TOPOLOGY: unknown
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: HGV-BE12 VARIANT
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:221:
AGACGCAATGACTCGGCGCCGACTCGGCGACCGGCCAAAAGGTGGTGGATGGGTGATGAC60
AGGGTTGGTAGGTCGTAAATCCCGGTCACCTTGGTAGCCACTATAGGTGGGTCTTAAGAG120
AAGGTTAAGATTCCTCTTGTGCCTGCGGCGAGACCGCGCACGGTCCGCAGGTGTTGGTCC180
TACCGGTGTGAATAAGGACCCGACGTCAGGCTCGTCGTTAAACCGAGCCCGCCACCCACC240
TGGGCAAACGACGCCCACGTACGGTCCACGTCGCCCTTCAATGCCTCTCTTGGCCAATAG300
GTTTATCCGGCGAGTTGACAAGGACCAGTGGGGGCCGGGGGCTCCGGGGAAGAACCCCGA360
GCCCCGCCCTTCCCGGTGGGACGGGAAATGCATGGGGCCACC402
(2) INFORMATION FOR SEQ ID NO:222:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 24 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: unknown
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: HGV-FORWARD PRIMER
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:222:
CCAAAAGGTGGTGGATGGGTGATG24
(2) INFORMATION FOR SEQ ID NO:223:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 24 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: unknown
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: HGV-FORWARD PRIMER
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:223:
GTGATGMCAGGGTTGGTAGGTCGT24
(2) INFORMATION FOR SEQ ID NO:224:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 26 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: unknown
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: HGV-FORWARD PRIMER
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:224:
GGTAGCCACTATAGGTGGGTCTTAAG26
(2) INFORMATION FOR SEQ ID NO:225:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 25 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: unknown
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: HGV-REVERSE PRIMER
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:225:
GAGMGRCATTGWAGGGCGACGTRGA25
(2) INFORMATION FOR SEQ ID NO:226:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 21 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: unknown
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: HGV-REVERSE PRIMER
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:226:
GRCATTGWAGGGCGACGTRGA21
(2) INFORMATION FOR SEQ ID NO:227:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 22 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: unknown
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: HGV-REVERSE PRIMER
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:227:
CCCCACTGGTCYTTGYCAACTC22
(2) INFORMATION FOR SEQ ID NO:228:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 29 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: unknown
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: PRIMER GV75- 36FE
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:228:
GCGAGATCTAAAATGCAGGCCTGATGGGT29
(2) INFORMATION FOR SEQ ID NO:229:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 29 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: unknown
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: PRIMER GV75- 7064RLE
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:229:
GCGAGATCTAAAATGTGGACTGCTAAGCC29
(2) INFORMATION FOR SEQ ID NO:230:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 46 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: unknown
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: PRIMER FV94- 28F
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:230:
GCGAGATCTAAAATGGCAAGCCCCAGAAACCGACGCCTATCTAAGT46
(2) INFORMATION FOR SEQ ID NO:231:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 39 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: unknown
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: PRIMER FV94- 2864R
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:231:
GGCATGATGAATTCGCAACGAGGGCCGGGACACCAAGAT39
(2) INFORMATION FOR SEQ ID NO:232:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 39 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: unknown
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: PRIMER FV94- 6439F
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:232:
GCGAGATCTAAAATGGGCCTCCGACACCCCGAAGGTTGT39
(2) INFORMATION FOR SEQ ID NO:233:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 39 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: unknown
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: PRIMER FV94- 9331R
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:233:
GCGAGATCTGAATTCTTCCCGGGGTGCACCCCTTCAGAT39
(2) INFORMATION FOR SEQ ID NO:234:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 9327 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: double
(D) TOPOLOGY: unknown
(ii) MOLECULE TYPE: cDNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: 3ZHGV-6, HGV FROM PNF2161
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:234:
GCAAGCCCCAGAAACCGACGCCTATCTAAGTAGACGCAATGACTCGGCGCCGACTCGGCG60
ACCGGCCAAAAGGTGGTGGATGGGTGATGACAGGGTTGGTAGGTCGTAAATCCCGGTCAC120
CTTGGTAGCCACTATAGGTGGGTCTTAAGAGAAGGTTAAGATTCCTCTTGTGCCTGCGGC180
GAGACCGCGCACGGTCCACAGGTGTTGGCCCTACCGGTGGGAATAAGGGCCCGACGTCAG240
GCTCGTCGTTAAACCGAGCCCGTTACCCACCTGGGCAAACGACGCCCACGTACGGTCCAC300
GTCGCCCTTCAATGTCTCTCTTGACCAATAGGCGTAGCCGGCGAGTTGACAAGGACCAGT360
GGGGGCCGGGGGCTTGGAGAGGGACTCCAAGTCCCGCCCTTCCCGGTGGGCCGGGAAATG420
CATGGGGCCACCCAGCTCCGCGGCGGCCTGCAGCCGGGGTAGCCCAAGAATCCTTCGGGT480
GAGGGCGGGTGGCATTTCCTTTTTCTATACCATCATGGCAGTCCTTCTGCTCCTTCTCGT540
GGTTGAGGCCGGGGCCATTCTGGCCCCGGCCACCCACGCTTGTCGAGCGAATGGGCAATA600
TTTCCTCACAAATTGTTGTGCCCCGGAGGACATCGGGTTCTGCCTGGAGGGTGGATGCCT660
GGTGGCCCTGGGGTGCACGATTTGCACTGACCAATGCTGGCCACTGTATCAGGCGGGTTT720
GGCTGTGCGGCCTGGCAAGTCCGCGGCCCAACTGGTGGGGGAGCTGGGTAGCCTATACGG780
GCCCCTGTCGGTCTCGGCCTATGTGGCTGGGATCCTGGGCCTGGGTGAGGTGTACTCGGG840
TGTCCTAACGGTGGGAGTCGCGTTGACGCGCCGGATCTACCCGGTGCCTAACCTGACGTG900
TGCAGTCGCGTGTGAGTTAAAGTGGGAAAGTGAGTTTTGGAGATGGACTGAACAGCTGGC960
CTCCAACTACTGGATTCTGGAATACCTCTGGAAGGTCCCATTTGATTTCTGGAGAGGCGT1020
GATAAGCCTGACCCCCTTGTTGGTTTGCGTGGCCGCATTGCTGCTGCTTGAGCAACGGGT1080
TGTCATGGTCTTCCTGTTGGTGACGATGGCCGGGATGTCGCAAGGCGCCCCTGCCTCCGT1140
TTTGGGGTCACGCCCCTTTGACTACGGGTTGACTTGGCAGACCTGCTCTTGCAGGGCCAA1200
CGGTTCGCGTTTTTCGACTGGGGAGAAGGTGTGGGACCGTGGGAACGTTACGCTTCAGTG1260
TGACTGCCCTAACGGCCCCTGGGTGTGGTTGCCAGCCTTTTGCCAAGCAATCGGCTGGGG1320
TGACCCCATCACTTATTGGAGCCACGGGCAAAATCAGTGGCCCCTTTCATGCCCCCAGTA1380
TGTCTATGGGTCTGCTACAGTCACTTGCGTGTGGGGTTCCGCTTCTTGGTATGCCTCCAC1440
CAGTGGTCGCGACTCGAAGATAGATGTGTGGAGTTTAGTGCCAGTTGGCTCTGCCACCTG1500
CACCATAGCCGCACTTGGATCATCGGATCGCGACACGGTGCCTGGGCTCTCCGAGTGGGG1560
AATCCCGTGCGTGACGTGTGTTCTGGACCGTCGGCCTGCTTCATGCGGCACCTGTGTGAG1620
GGACTGCTGGCCCGAGACCGGGTCGGTTAGGTTCCCATTCCATCGGTGCGGCGTGGGGCC1680
TCGGCTGACAAAGGACTTGGAAGCTGTGCCCTTCGTCAATAGGACAACTCCCTTCACCAT1740
TAGGGGGCCCCTGGGCAACCAGGGCCGAGGCAACCCGGTGCGGTCGCCCTTGGGTTTTGG1800
GTCCTACGCCATGACCAGGATCCGAGATACCCTACATCTGGTGGAGTGTCCCACACCAGC1860
CATCGAGCCTCCCACCGGGACGTTTGGGTTCTTCCCCGGGACGCCGCCTCTCAACAACTG1920
CATGCTCTTGGGCACGGAAGTGTCCGAGGCACTTGGGGGGGCTGGCCTCACGGGGGGGTT1980
CTATGAACCCCTGGTGCGCAGGTGTTCGGAGCTGATGGGAAGCCGAAATCCGGTTTGTCC2040
GGGGTTTGCATGGCTCTCTTCGGGCAGGCCTGATGGGTTTATACATGTCCAGGGTCACTT2100
GCAGGAGGTGGATGCAGGCAACTTCATCCCGCCCCCGCGCTGGTTGCTCTTGGACTTTGT2160
ATTTGTCCTGTTATACCTGATGAAGCTGGCTGAGGCACGGTTGGTCCCGCTGATCTTGCT2220
GCTGCTATGGTGGTGGGTGAACCAGCTGGCAGTCCTAGGGCTGCCGGCTGTGGAAGCCGC2280
CGTGGCAGGTGAGGTCTTCGCGGGCCCTGCCCTGTCCTGGTGTCTGGGACTCCCGGTCGT2340
CAGTATGATATTGGGTTTGGCAAACCTGGTGCTGTACTTTAGATGGTTGGGACCCCAACG2400
CCTGATGTTCCTCGTGTTGTGGAAGCTTGCTCGGGGAGCTTTCCCGCTGGCCCTCTTGAT2460
GGGGATTTCGGCGACCCGCGGGCGCACCTCAGTGCTCGGGGCCGAGTTCTGCTTCGATGC2520
TACATTCGAGGTGGACACTTCGGTGTTGGGCTGGGTGGTGGCCAATGTGGTAGCTTGGGC2580
CATTGCGCTCCTGAGCTCGATGAGCGCAGGGGGGTGGAGGCACAAAGCCGTGATCTATAG2640
GACGTGGTGTAAGGGGTACCAGGCAATCCGTCAAAGGGTGGTGAGGAGCCCCCTCGGGGA2700
GGGGCGGCCTGCCAAACCCCTGACCTTTGCCTGGTGCTTGGCCTCGTACATCTGGCCAGA2760
TGCTGTGATGATGGTGGTGGTTGCCTTGGTTCTTCTCTTTGGCCTGTTCGACGCGTTGGA2820
TTGGGCCTTGGAGGAGATCTTGGTGTCCCGGCCCTCGCTGCGGCGTTTGGCTCGGGTGGT2880
TGAGTGCTGTGTGATGGCGGGTGAGAAGGCCACAACCGTCCGGCTGGTCTCCAAGATGTG2940
TGCGAGAGGAGCTTATTTGTTCGATCATATGGGCTCATTTTCGCGTGCTGTCAAGGAGCG3000
CCTGTTGGAATGGGACGCGGCTCTTGAACCTCTGTCATTCACTAGGACGGACTGTCGCAT3060
CATACGGGATGCCGCGAGGACTTTGTCCTGCGGGCAATGCGTCATGGGTTTACCCGTGGT3120
TGCGCGCCGTGGTGATGAGGTTCTCATCGGCGTCTTCCAGGATGTGAATCATTTGCCTCC3180
CGGGTTTGTTCCGACCGCGCCTGTTGTCATCCGACGGTGCGGAAAGGGCTTCTTGGGGGT3240
CACAAAGGCTGCCTTGACAGGTCGGGATCCTGACTTACATCCAGGGAACGTCATGGTGTT3300
GGGGACGGCTACGTCGCGAAGCATGGGAACATGCTTGAACGGCCTGCTGTTCACGACCTT3360
CCATGGGGCTTCATCCCGAACCATCGCCACACCCGTGGGGGCCCTTAATCCCAGATGGTG3420
GTCAGCCAGTGATGATGTCACGGTGTATCCACTCCCGGATGGGGCTACTTCGTTAACGCC3480
TTGTACTTGCCAGGCTGAGTCCTGTTGGGTCATCAGATCCGACGGGGCCCTATGCCATGG3540
CTTGAGCAAGGGGGACAAGGTGGAGCTGGATGTGGCCATGGAGGTCCCTGATTTCCGTGG3600
CTCGTCTGGCTCACCGGTCCTATGTGACGAGGGGCACGCAGTAGGAATGCTCGTGTCTGT3660
GCTTCACTCCGGTGGTAGGGTCACCGCGGCACGGTTCACTAGGCCGTGGACCCAAGTGCC3720
AACAGATGCCAAAACCACCACTGAACCCCCTCCGGTGCCGGCCAAAGGAGTTTTCAAAGA3780
GGCCCCGTTGTTTATGCCTACGGGAGCGGGAAAGAGCACTCGCGTCCCGTTGGAGTACGG3840
CAACATGGGGCACAAGGTCTTAGTCTTGAACCCCTCAGTGGCCACTGTGCGGGCCATGGG3900
CCCGTACATGGAGCGGCTGGCGGGTAAACATCCAAGTATATACTGTGGGCATGATACAAC3960
TGCTTTCACAAGGATCACTGACTCCCCCCTGACGTATTCAACCTATGGGAGGTTTTTGGC4020
CAACCCTAGGCAGATGCTACGGGGCGTTTCGGTGGTCATTTGTGATGAGTGCCACAGTTA4080
TGACTCAACCGTGCTGTTAGGCATTGGGAGGGTTCGGGAGCTGGCGCGTGGGTGCGGAGT4140
GCAACTAGTGCTCTACGCCACCGCTACGCCTCCCGGATCCCCTATGACGCAGCACCCTTC4200
CATAATTGAGACAAAATTGGACGTGGGCGAGATTCCCTTTTATGGGCACGGAATACCCCT4260
CGAGCGGATGCGAACCGGAAGGCACCTCGTGTTCTGCCATTCTAAGGCTGAGTGCGAGCG4320
CCTTGCTGGCCAGTTCTCCGCTAGGGGGGTCAATGCCATTGCCTATTATAGGGGTAAAGA4380
CAGTTCTATCATCAAGGATGGGGACCTGGTGGTCTGTGCCACAGACGCGCTTTCCACTGG4440
GTACACTGGAAATTTCGACTCCGTCACCGACTGTGGATTAGTGGTGGAGGAGGTCGTTGA4500
GGTGACCCTTGATCCTACCATTACCATCTCCCTGCGGACAGTGCCTGCGTCGGCTGAACT4560
GTCGATGCAAAGACGAGGACGCACGGGTAGGGGCAGGTCTGGACGCTACTACTACGCGGG4620
GGTGGGCAAAGCCCCTGCGGGTGTGGTGCGCTCAGGTCCTGTCTGGTCGGCGGTGGAAGC4680
TGGAGTGACCTGGTACGGAATGGAACCTGACTTGACAGCTAACCTACTGAGACTTTACGA4740
CGACTGCCCTTACACCGCAGCCGTCGCGGCTGATATCGGAGAAGCCGCGGTGTTCTTCTC4800
TGGGCTCGCCCCATTGAGGATGCACCCTGATGTCAGCTGGGCAAAAGTTCGCGGCGTCAA4860
CTGGCCCCTCTTGGTGGGTGTTCAGCGGACCATGTGTCGGGAAACACTGTCTCCCGGCCC4920
ATCGGATGACCCCCAATGGGCAGGTCTGAAGGGCCCAAATCCTGTCCCACTCCTGCTGAG4980
GTGGGGCAATGATTTACCATCTAAAGTGGCCGGCCACCACATAGTGGACGACCTGGTCCG5040
GAGACTCGGTGTGGCGGAGGGTTACGCCCGCTGCGACGCTGGGCCGATCTTGATGATCGG5100
TCTAGCTATCGCGGGGGGAATGATCTACGCGTCGTACACCGGGTCGCTAGTGGTGGTGAC5160
AGACTGGGATGTGAAGGGGGGTGGCGCCCCCCTTTATCGGCATGGAGACCAGGCCACGCC5220
TCAGCCGGTGGTGCAGGTTCCTCCGGTAGACCATCGGCCGGGGGGTGAATCAGCACCATC5280
GGATGCCAAGACAGTGACAGATGCGGTGGCAGCGATCCAGGTGGACTGCGATTGGACTAT5340
CATGACTCTGTCGATCGGAGAAGTGTTGTCCTTGGCTCAGGCTAAGACGGCCGAGGCCTA5400
CACAGCAGCCACCAAGTGGCTCGCTGGCTGCTATACGGGGACGCGGGCCGTTCCCACTGT5460
ATCCATTGTTGACAAGCTCTTCGCCGGAGGGTGGGCGGCTGTGGTGGGCCATTGCCACAA5520
CGTGATTGCTGCGGCGGTGGCGGCCTACGGGGCTTCAAAGAGCCCGCCGTTGGCAGCCGC5580
GGCTTCCTACCTGATGGGGTTGGGCGTTGGAGGCAACGCTCAGACGCGTCTGGCATCTGC5640
CCTCCTATTGGGGGCTGCTGGAACCGCCTTGGGCACTCCTGTCGTGGGCTTGACCATGGC5700
AGGTGCGTTCATGGGGGGCGCCAGTGTCTCCCCCTCCTTGGTCACCATTTTATTGGGGGC5760
CGTCGGAGGTTGGGAGGGTGTTGTCAACGCGGCGAGCCTAGTCTTTGACTTCATGGCGGG5820
GAAACTTTCATCAGAAGATCTGTGGTATGCCATCCCGGTACTGACCAGCCCGGGGGCGGG5880
CCTTGCGGGGATCGCTCTCGGGTTGGTTTTGTATTCAGCTAACAACTCTGGCACTACCAC5940
TTGGTTGAACCGTCTGCTGACTACGTTACCAAGGTCTTCATGTATCCCGGACAGTTACTT6000
TCAGCAAGTTGACTATTGCGACAAGGTCTCAGCCGTGCTCCGGCGCCTGAGCCTCACCCG6060
CACAGTGGTTGCCCTGGTCAACAGGGAGCCTAAGGTGGATGAGGTACAGGTGGGGTATGT6120
CTGGGACCTGTGGGAGTGGATCATGCGCCAAGTGCGCGTGGTCATGGCCAGACTCAGGGC6180
CCTCTGCCCCGTGGTGTCATTACCCTTGTGGCACTGCGGGGAGGGGTGGTCCGGGGAATG6240
GTTGCTTGACGGTCATGTTGAGAGTCGCTGCCTCTGTGGCTGCGCGATCACTGGTGACGT6300
TCTGAATGGGCAACTCAAAGAACCAGTTTACTCTACCAAGCTGTGCCGGCACTATTGGAT6360
GGGGACTGTCCCTGTGAACATGCTGGGTTACGGTGAAACGTCGCCTCTCCTGGCCTCCGA6420
CACCCCGAAGGTTGTGCCCTTCGGGACGTCTGGCTGGGCTGAGGTGGTGGTGACCACTAC6480
CCACGTGGTAATCAGGAGAACCTCCGCCTATAAGCTGCTGCGCCAGCAAATCCTATCGGC6540
TGCTGTAGCTGAGCCCTACTACGTCGACGGCATTCCGGTCTCATGGGACGCGGACGCTCG6600
TGCGCCCGCCATGGTCTATGGCCCTGGGCAAAGTGTTACCATTGACGGGGAGCGCTACAC6660
CCTGCCTCATCAACTGAGGCTCAGGAATGTGGCGCCCTCTGAGGTTTCATCCGAGGTGTC6720
CATTGACATTGGGACGGAGACTGGAGACTCAGAACTGACTGAGGCCGATCTGCCGCCGGC6780
GGCTGCTGCTCTCCAAGCGATCGAGAATGCTGCGAGGATTCTTGAACCGCACATTGATGC6840
CATCATGGAGGACTGCAGTACACCCTCTCTTTGTGGTAGTAGCCGAGAGATGCCTGTATG6900
GGGAGAAGACATCCCCCGTACTCCATCGCCAGCACTTATCTCGGTTACTGAGAGCAGCTC6960
AGATGAGAAGACCCCGTCGGTGTCCTCCTCGCAGGAGGATACCCCGTCCTCTGACTCATT7020
CGAGGTCATCCAAGAGTCCGAGACAGCCGAAGGGGAGGAAAGCGTCTTCAACGTGGCTCT7080
TTCCGTATTAGAAGCCTCATTTCCACAGAGCGACGCGACCAGGAAGCTTACCGTCAAGAT7140
GTCGTGCTGCGTTGAAAAGAGCGTCACGCGCTTTTTCTCATTGGGGTTGACGGTGGCTGA7200
TGTTGCTAGCCTGTGTGAGATGGAAATCCAGAACCATACAGCCTATTGTGACAAGGTGCG7260
CACTCCGCTTGAATTGCAGGTTGGGTGCTTGGTGGGCAATGAACTTACCTTTGAATGTGA7320
CAAGTGTGAGGCTAGGCAAGAAACCTTGGCCTCCTTCTCTTACATTTGGTCTGGAGTGCC7380
GCTGACTAGGGCCACGCCGGCCAAGCCTCCCGTGGTGAGGCCGGTTGGCTCTTTATTAGT7440
GGCCGACACTACTAAGGTGTATGTTACCAATCCAGACAATGTGGGACGGAGGGTGGACAA7500
GGTGACCTTCTGGCGTGCTCCTAGGGTTCATGATAAGTACCTCGTGGACTCTATTGAGCG7560
CGCTAAGAGGGCCGCTCAAGCCTGCCTAAGCATGGGTTACACTTATGAGGAAGCAATAAG7620
GACTGTAAGGCCACATGCTGCCATGGGCTGGGGATCTAAGGTGTCGGTTAAGGACTTAGC7680
CACCCCCGCGGGGAAGATGGCCGTCCATGACCGGCTCCAGGAGATACTTGAAGGGACTCC7740
GGTCCCCTTTACTCTTACTGTGAAAAAGGAGGTGTTCTTCAAAGACCGGAAGGAGGAGGA7800
GGCCCCCCGCCTCATTGTGTTCCCCCCCCTGGACTTCCGGATAGCTGAAAAGCTCATCTT7860
GGGAGACCCAGACCGGGTAGCCAAGGCGGTGTTGGGGGGGGCCTACGCCTTCCAGTACAC7920
CCCAAATCAGCGAGTTAAGGAGATGCTCAAGCTATGGGAGTCTAAGAAGACCCCTTGCGC7980
CATCTGTGTGGACGCCACCTGCTTCGACAGTAGCATAACTGAAGAGGACGTGGCTTTGGA8040
GACAGAGCTGTACGCTCTGGCCTCTGACCATCCAGAATGGGTGCGGGCACTTGGGAAATA8100
CTATGCCTCAGGCACCATGGTCACCCCGGAAGGGGTGCCCGTCGGTGAGAGGTATTGCAG8160
ATCCTCGGGTGTCCTAACAACTAGCGCGAGCAACTGCTTGACCTGCTACATCAAGGTGAA8220
AGCCGCCTGTGAGAGGGTGGGGCTGAAGAATGTCTCTCTTCTCATAGCCGGCGATGACTG8280
CTTGATCATATGTGAGCGGCCAGTGTGCGACCCAAGCGACGCTTTGGGCAGAGCCCTAGC8340
GAGCTATGGGTACGCGTGCGAGCCCTCATATCATGCATCCTTGGACACGGCCCCCTTCTG8400
CTCCACTTGGCTTGCTGAGTGCAATGCAGATGGGAAGCGCCATTTCTTCCTGACCACGGA8460
CTTCCGGAGGCCGCTCGCTCGCATGTCGAGTGAGTATAGTGACCCGATGGCTTCGGCGAT8520
CGGTTACATCCTCCTTTATCCTTGGCACCCCATCACACGGTGGGTCATCATCCCTCATGT8580
GCTAACGTGCGCATTCAGGGGTGGAGGCACACCGTCTGATCCGGTTTGGTGCCAGGTACA8640
TGGTAACTACTACAAGTTTCCACTGGACAAACTGCCTAACATCATCGTGGCCCTCCACGG8700
ACCAGCAGCGTTGAGGGTTACCGCAGACACAACTAAAACAAAGATGGAGGCTGGTAAGGT8760
TCTGAGCGACCTCAAGCTCCCTGGCTTAGCAGTCCACCGAAAGAAGGCCGGGGCGTTGCG8820
AACACGCATGCTCCGCTCGCGCGGTTGGGCTGAGTTGGCTAGGGGCTTGTTGTGGCATCC8880
AGGCCTACGGCTTCCTCCCCCTGAGATTGCTGGTATCCCGGGGGGTTTCCCTCTCTCCCC8940
CCCCTATATGGGGGTGGTACACCAATTGGATTTTACAAGCCAGAGGAGTCGCTGGCGGTG9000
GTTGGGGTTCTTAGCCCTGCTCATCGTAGCCCTCTTCGGGTGAACTAAATTCATCTGTTG9060
CGGCGAGGTCTGGTGACTGATCGTCACCGGAGGAGGTTCCCGCCCTCCCCGCCCCAGGGG9120
TCTCCCCGCTGGGTAAAAAGGGCCCGGCCTTGGGAGGCATGGTGGTTACTAACCCCCTGG9180
CAGGGTTAAAGCCTGATGGTGCTAATGCACTGCCACTTCGGTGGCGGGTCGCTACCTTAT9240
AGCGTAATCCGTGACTACGGGCTGCTCGCAGAGCCCTCCCCGGATGGGGCACAGTGCACT9300
GAGATCTGAAGGGGTGCACCCCGGGAA9327
(2) INFORMATION FOR SEQ ID NO:235:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 22 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer GLI- F
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:235:
TAGCATGGCCTTTGCAGGGCTG22
(2) INFORMATION FOR SEQ ID NO:236:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 18 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer GLI- R
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:236:
AAGCTGTGACCGTCTCCG18
(2) INFORMATION FOR SEQ ID NO:237:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 31 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer GE1- NF
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:237:
GCCGCCATGGCGGGGAAACTTTCATCAGAAG31
(2) INFORMATION FOR SEQ ID NO:238:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 32 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer GE1- NR
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:238:
GCGCGGATCCTAGTGACACCACGGGGCAGAGG32
(2) INFORMATION FOR SEQ ID NO:239:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 33 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer GE57F
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:239:
GCCGCCATGGCTCTCTTGACCAATAGGTTTATC33
(2) INFORMATION FOR SEQ ID NO:240:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 31 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer GE57R
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:240:
GCGCGGATCCAGAAATGCCACCCGCCCTCAC31
(2) INFORMATION FOR SEQ ID NO:241:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 61 amino acids
(B) TYPE: amino acid
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: protein
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: GE57 amino acid sequence
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:241:
MetSerLeuLeuThrAsnArgPheIleArgArgValAspLysAspGln
151015
TrpGlyProGlyValThrGlyThrAspProGluProCysProSerArg
202530
TrpAlaGlyLysCysMetGlyProProSerSerAlaAlaAlaCysSer
354045
ArgGlySerProArgIleLeuArgValArgAlaGlyGly
505560
(2) INFORMATION FOR SEQ ID NO:242:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 52 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Forward Primer for E1
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:242:
GCGCAGATCTAAAATGAGCCGTGGTGGCATTTCCTTTTTCTATACCATCATG52
(2) INFORMATION FOR SEQ ID NO:243:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 38 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Reverse Primer for E1
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:243:
GCGCAGATCTCCAGAAATCAAATGGGACCTTCCAGAGG38
(2) INFORMATION FOR SEQ ID NO:244:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 26 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Forward Primer for E2 with insect
signal sequence
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:244:
CGCGAGATCTGTCGCAAGGCGCCCCT26
(2) INFORMATION FOR SEQ ID NO:245:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 28 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Reverse Primer for E2 with insect
signal sequence
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:245:
GCGCAGATCTAGTTGCCTGCATCCACCT28
(2) INFORMATION FOR SEQ ID NO:246:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 42 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Forward Primer for E2 with HGV
signal sequence
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:246:
CGCGAGATCTAAAATGAAACTGCTTGTCATGGTCTTCCTGTT42
(2) INFORMATION FOR SEQ ID NO:247:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 28 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Reverse Primer for E2 with HGV
signal sequence
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:247:
GCGCAGATCTAGTTGCCTGCATCCACCT28
(2) INFORMATION FOR SEQ ID NO:248:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 34 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Forward Primer for NS2a
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:248:
GCGCAGATCTGGCCGTGGCAGGTGAGGTCTTCGC34
(2) INFORMATION FOR SEQ ID NO:249:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 31 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Reverse Primer for NS2a
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:249:
GCGCAGATCTTAACGCCGCAACGAGGGCCGG31
(2) INFORMATION FOR SEQ ID NO:250:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 46 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Forward Primer for NS2b
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:250:
GCGCGGATCCAAAATGATCGCTCGGGTGGTTGAGTGCTGTGTGATG46
(2) INFORMATION FOR SEQ ID NO:251:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 32 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Reverse Primer for NS2b
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:251:
GCGCGGATCCAGGCGCGGTCGGAACAAACCCG32
(2) INFORMATION FOR SEQ ID NO:252:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 39 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Forward Primer NS3
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:252:
GCGAGATCTAAAATGTGCGGAAAGGGCTTCTTGGGGGTC39
(2) INFORMATION FOR SEQ ID NO:253:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 39 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Reverse Primer NS3
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:253:
GCGAGATCTCATCTCCGGACCAGGTCGTCCACTATGTGG39
(2) INFORMATION FOR SEQ ID NO:254:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 31 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Forward Primer NS4a
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:254:
GGCGGATCCAAAATGATCGGTGTGGCGGAGG31
(2) INFORMATION FOR SEQ ID NO:255:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 26 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Reverse Primer NS4a
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:255:
GGCGGGATCCATGCGCCGGAGCACGG26
(2) INFORMATION FOR SEQ ID NO:256:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 34 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Forward Primer NS4b
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:256:
GCGGGATCCAAAATGATCAGCCTCACCCGCACAG34
(2) INFORMATION FOR SEQ ID NO:257:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 29 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Reverse Primer NS5a
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:257:
GGCGGGATCCTACCTCCTGATTACCACGT29
(2) INFORMATION FOR SEQ ID NO:258:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 42 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Forward Primer NS5a
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:258:
GCGAGATCTAAAATGACCTCCGCCTATAAGCTGCTGCGCCAG42
(2) INFORMATION FOR SEQ ID NO:259:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 40 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Reverse Primer NS5a
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:259:
GGCAGATCTACCTCCGTCCCACATTGTCTGGATTGGTAAC40
(2) INFORMATION FOR SEQ ID NO:260:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 43 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Forward Primer NS5b
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:260:
GCGAGATCTAAAATGGTGGACAAGGTGACCTTCTGGCGTGCTC43
(2) INFORMATION FOR SEQ ID NO:261:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 36 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Reverse Primer NS5b
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:261:
GCGAGATCTCACCCGAAGAGGGCTACGATGAGCAGG36
(2) INFORMATION FOR SEQ ID NO:262:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 52 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Forward Primer E1-E2-NS2a
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:262:
GCGCAGATCTAAAATGAGCCGTGGTGGCATTTCCTTTTTCTATACCATCATG52
(2) INFORMATION FOR SEQ ID NO:263:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 31 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Reverse Primer E1-E2-NS2a
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:263:
GCGCAGATCTTAACGCCGCAACGAGGGCCGG31
(2) INFORMATION FOR SEQ ID NO:264:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 22 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer 9E3- REV
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:264:
GCTGGCTGAGGCACGGTTGGTC22
(2) INFORMATION FOR SEQ ID NO:265:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 22 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer E39- 94PR
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:265:
CACCATCATCACAGCATCTGGC22
(2) INFORMATION FOR SEQ ID NO:266:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 32 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer GEP- F12
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:266:
GCAACCATGGAACCTGCCAAACCCCTGACCTT32
(2) INFORMATION FOR SEQ ID NO:267:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 21 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer GEP- R12
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:267:
AGCCCCATGGAAGGTCGTGAA21
(2) INFORMATION FOR SEQ ID NO:268:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 30 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer GEP- F14
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:268:
TTGGGATCCCTCGTGTTCCGCCATTCTAAG30
(2) INFORMATION FOR SEQ ID NO:269:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 30 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer GEP- R13
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:269:
TATGGATCCTGGTAAATCATTGCCCCACCT30
(2) INFORMATION FOR SEQ ID NO:270:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 39 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer 470EP- F8
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:270:
GCTGAATTCGCCATGGCGACGTGCGCATTCAGGGGTGGA39
(2) INFORMATION FOR SEQ ID NO:271:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 27 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: single
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Primer GEP- R14
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:271:
GGAGGATCCGCGACCCGCCACCGAAGT27
(2) INFORMATION FOR SEQ ID NO:272:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 48 amino acids
(B) TYPE: amino acid
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: protein
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Y5 epitope
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:272:
IleAspGlyGluArgTyrThrLeuProHisGlnLeuArgLeuArgAsn
151015
ValAlaProSerGluValSerSerGluValSerIleAspIleGlyThr
202530
GluAlaGluAsnSerGluLeuThrGluAlaAspLeuProProAlaAla
354045
(2) INFORMATION FOR SEQ ID NO:273:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 55 amino acids
(B) TYPE: amino acid
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: protein
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Q9 Epitope
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:273:
CysGlyLeuLeuThrArgHisHisThrAlaLeuAsnHisProSerGln
151015
ThrProGlnArgGlyProGlyHisGlnAspLeuLeuGlnGlyProIle
202530
GlnArgValGluGlnAlaLysGluLysAspGlnGlyAsnHisHisHis
354045
HisHisSerIleTrpProAsp
5055
(2) INFORMATION FOR SEQ ID NO:274:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 35 amino acids
(B) TYPE: amino acid
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: protein
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Q11 Epitope
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:274:
AlaAlaValAlaGluProTyrTyrValAspGlyIleProValSerTrp
151015
AspAlaAspAlaArgAlaProAlaMetValTyrGlyProGlyGlnSer
202530
ValThrIle
35
(2) INFORMATION FOR SEQ ID NO:275:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 225 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Q7-12-1 env clone
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:275:
GTGCCCTTCGTCAACAGGACAACTCTCTTCACCATTAGGGGGCCCCTGGGCAACCAGGGC60
CGAGGCAACCCGGTGCGGTCGCCCTTGGGTTTTGGGTCCTACGCCATGACCAGGATCCGA120
GATACCCTACATCTGGTGGAGTGTCCCACACCAGCCATCGAGCCTCCCACCGGGACGTCT180
GGGTTCTTCCCCGGGACGCCGCCTCTCAACAACTGCATGCATATG225
(2) INFORMATION FOR SEQ ID NO:276:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 192 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Y12-15-1 NS3 clone DNA
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:276:
AACATGGGGCACAAGGTCTTAATCTTGAACCCCTCAGTGGCCACTGTGCGGGCCATGGGC60
CCGTACATGGAGCGGCTGGCGGGTAAACATCCAAGTATATACTGTGGGCATGATACAACT120
GCTTTCACAAGGATCACTGACTCCCCCCTGACGTATTCAACCTATGGGAGGTTTTTGGCC180
AACCCTAGGCAA192
(2) INFORMATION FOR SEQ ID NO:277:
(i) SEQUENCE CHARACTERISTICS:
(A) LENGTH: 264 base pairs
(B) TYPE: nucleic acid
(C) STRANDEDNESS: both
(D) TOPOLOGY: linear
(ii) MOLECULE TYPE: DNA
(iii) HYPOTHETICAL: NO
(iv) ANTI-SENSE: NO
(vi) ORIGINAL SOURCE:
(C) INDIVIDUAL ISOLATE: Y12-10-2 NS3 clone
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:277:
CCCCTCGAGCGGATGCGAACCGGAAGGCACCTCGTGTTCTGCCATTCTAAGGCTGAGTGC60
GAGCGCCTTGCTGGCCAGTTCTCCGCTAGGGGGGTCAATGCCATTGCCTATTATAGGGGT120
AAAGACAGCTCTATCATCAAGGATGGGGACCTGGTGGTCTGTGCTACAGACGCGCTTTCC180
ACTGGGTACACTGGAAATTTCGACTCCGTCACCGACTGTGGATTAGTGGTGGAGGAGGTC240
GTTGAGGTGACCCTTGATCCCACC264
__________________________________________________________________________
</pre>
    
    
  </div>
  </section>
      <div id="margin" class="style-scope patent-text">
        <div id="createHighlight" style="top: 0px;" class="style-scope patent-text" hidden="">
          <iron-icon id="comment" icon="editor:insert-comment" class="style-scope patent-text x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M20 2H4c-1.1 0-2 .9-2 2v12c0 1.1.9 2 2 2h14l4 4V4c0-1.1-.9-2-2-2zm-2 12H6v-2h12v2zm0-3H6V9h12v2zm0-3H6V6h12v2z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
        </div>
      </div>
    </div>
  </patent-text>
              </section>
            </div>
            <div class="flex flex-width style-scope patent-result">
              <section id="claims" class="style-scope patent-result">
                <h3 class="style-scope patent-result">
                  <div class="layout horizontal style-scope patent-result">
                    <div class="flex style-scope patent-result">Claims <span class="style-scope patent-result">(20)</span></div>
                    <div class="collapse style-scope patent-result">
                      Hide Dependent <iron-icon style="height: 15px; width: 15px;" icon="expand-less" class="style-scope patent-result x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M12 8l-6 6 1.41 1.41L12 10.83l4.59 4.58L18 14z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon><template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </div>
                  </div>
                  <div class="layout horizontal style-scope patent-result">
                    <div class="flex style-scope patent-result"></div>
                    <div class="righthead style-scope patent-result" hidden="">translated from </div>
                  </div>
                </h3>
                <patent-text name="claims" enable-text-selection="" class="style-scope patent-result" style="padding-top: 0px;">
    
    
    <template is="dom-if" restamp="" class="style-scope patent-text"></template>
    <overlay-tooltip id="tooltip" class="style-scope patent-text" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      
    </div>
  </overlay-tooltip>
    <div class="layout horizontal style-scope patent-text">
      <section id="text" class="flex style-scope patent-text"><div mxw-id="PCLM5356787" lang="EN" load-source="patent-office" class="claims style-scope patent-text">
    <claim-statement class="style-scope patent-text">It is claimed:</claim-statement>
    <div class="claim style-scope patent-text"> <div num="1" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">1. A purified Non-A Non-B Non-C Non-D Non-E Hepatitis Virus (HGV) polynucleotide having at least 70% sequence identity to a polynucleotide selected from the group consisting of SEO ID NO:14, its complement, and contiguous fragments thereof of at least 30 nucleotides in length,<div class="claim-text style-scope patent-text">wherein HGV is characterized by: (i) production of elevated serum alanine aminotransferase levels in an infected primate, (ii) its serological distinction from hepatitis A virus (HAV), hepatitis B virus (HBV), hepatitis C virus (HCV), hepatitis D virus, and hepatitis E virus (HEV), (iii) membership in the virus family Flaviviridae and (iv) a viral genome comprising a polynucleotide region that is selectively hybridizable with SEQ ID NO:19.</div> </div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="2" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">2. A polynucleotide of claim 1, wherein said polynucleotide is a DNA polynucleotide.</div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="3" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">3. A polynucleotide of claim 1, wherein said polynucleotide is a RNA polynucleotide.</div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="4" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">4. A polynucleotide of claim 1, wherein said polynucleotide is a recombinant polynucleotide.</div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="5" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">5. A polynucleotide of claim 1, wherein said polynucleotide encodes an HGV polypeptide antigen.</div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="6" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">6. A polynucleotide of claim 1, wherein said polynucleotide is included within the PNF 2161 cDNA source lambda gt11 library identified by ATCC designation number 75268 or ATCC 75837.</div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="7" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">7. A DNA polynucleotide of claim 1 encoding a Non-A Non-B Non-C Non-D Non-E Hepatitis Virus (HGV) polypeptide.</div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="8" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">8. A polynucleotide of claim 1 encoding an E2 polypeptide.</div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="9" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">9. A polynucleotide of claim 8 comprising a series of bases included within the region extending from base 1149 to base 2183 of SEQ ID NO:14.</div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="10" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">10. A polynucleotide of claim 1, wherein said polynucleotide is obtained from HGV viral particles.</div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="11" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">11. A DNA polynucleotide of claim 7, wherein said polypeptide comprises a contiguous sequence of at least 60 amino acids having at least 70% sequence identity to a contiguous region of at least 60 amino acids contained in SEQ ID NO:15.</div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="12" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">12. A polynucleotide of claim 1, comprising a probe that specifically hybridizes with an Non-A Non-B Non-C Non-D Non-E Hepatitis Virus (HGV) genome, cDNA or complements thereof.</div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="13" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">13. A kit for analyzing a sample for the presence of a Non-A Non-B Non-C Non-D Non-E Hepatitis Virus (HGV) polynucleotide, comprising<div class="claim-text style-scope patent-text">at least one polynucleotide of claim 1 comprising a probe that will specifically hybridize with an HGV polynucleotide, and</div> <div class="claim-text style-scope patent-text">reagents for detecting the presence of HGV polynucleotide/probe complexes formed by hybridization of the HGV polynucleotide with said probe.</div> </div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="14" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">14. A kit of claim 13, wherein said kit contains two polynucleotide probes defining an internal region of the HGV polynucleotide and each probe has one strand containing a 3'-end internal to the region.</div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="15" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">15. A kit of claim 14, wherein said probes are useful as primers for polymerase chain reaction amplification.</div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="16" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">16. A method of detecting a Non-A Non-B Non-C Non-D Non-E Hepatitis Virus (HGV) nucleic acid in a test subject, comprising<div class="claim-text style-scope patent-text">obtaining a nucleic acid-containing sample from the subject,</div> <div class="claim-text style-scope patent-text">combining the sample and at least one polynucleotide probe of claim 12, and</div> <div class="claim-text style-scope patent-text">detecting the presence of HGV nucleic acid/probe complexes formed by hybridization of the HGV nucleic acid with said probe.</div> </div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="17" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">17. A method of claim 16, wherein said probe contains at least one reporter moiety.</div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="18" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">18. A method of claim 16, wherein said detecting further comprises:<div class="claim-text style-scope patent-text">using two HGV nucleic acid specific probes where the two probes define an internal region of the HGV nucleic acid and each probe has one strand containing a 3'-end internal to the region,</div> <div class="claim-text style-scope patent-text">converting the nucleic acid/probe hybridization complexes to double-stranded probe-containing fragments by primer extension reactions,</div> <div class="claim-text style-scope patent-text">amplifying the probe-containing fragments by successively repeating the steps of (i) denaturing the double-stranded fragments to produce single-stranded fragments, (ii) hybridizing the single strands with the probes to form strand/probe complexes, (iii) generating double-stranded fragments from the strand/probe complexes in the presence of DNA polymerase and all four deoxyribonucleotides, and (iv) repeating steps (i) to (iii) until a desired degree of amplification has been achieved,</div> <div class="claim-text style-scope patent-text">identifying the amplification products.</div> </div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="19" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">19. A method of claim 16, wherein said detecting is accomplished by a target amplification method selected from the group consisting of self-sustained sequence replication, ligase chain reaction, and strand displacement amplification.</div>
    </div>
    </div> <div class="claim-dependent style-scope patent-text"> <div num="20" class="claim style-scope patent-text">
      <div class="claim-text style-scope patent-text">20. A method of claim 16, wherein said detecting is accomplished employing a signal amplification technique selected from the group consisting of branch-chained DNA probes and the Q-beta replicase method.</div>
    </div>
  </div> </div>
  </section>
      <div id="margin" class="style-scope patent-text">
        <div id="createHighlight" style="top: 0px;" class="style-scope patent-text" hidden="">
          <iron-icon id="comment" icon="editor:insert-comment" class="style-scope patent-text x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M20 2H4c-1.1 0-2 .9-2 2v12c0 1.1.9 2 2 2h14l4 4V4c0-1.1-.9-2-2-2zm-2 12H6v-2h12v2zm0-3H6V9h12v2zm0-3H6V6h12v2z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
        </div>
      </div>
    </div>
  </patent-text>
              </section>
            </div>
          </div>

          <div class="footer style-scope patent-result">

            
            <h3 id="patentCitations" class="scroll-target style-scope patent-result">
              Patent Citations (15)
              <a href="#title" class="style-scope patent-result"><iron-icon icon="arrow-drop-up" class="style-scope patent-result x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M7 14l5-5 5 5z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon></a>
            </h3>
            <div class="responsive-table style-scope patent-result">
              <div class="table style-scope patent-result">
                <div class="thead style-scope patent-result">
                  <div class="tr style-scope patent-result">
                    <span class="th style-scope patent-result">Publication number</span>
                    <span class="th style-scope patent-result">Priority date</span>
                    <span class="th style-scope patent-result">Publication date</span>
                    <span class="th style-scope patent-result">Assignee</span>
                    <span class="th style-scope patent-result">Title</span>
                  </div>
                </div>
                <div class="tbody style-scope patent-result">
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                      <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US4810492A/en"><a id="link" href="#" class="style-scope state-modifier">US4810492A</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">1986-06-05</span>
                    <span class="td style-scope patent-result">1989-03-07</span>
                    <span class="td style-scope patent-result">The Research Foundation For Microbial Diseases Of Osaka University</span>
                    <span class="td style-scope patent-result">Flavivirus antigen 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                      <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/EP0318216A1/en"><a id="link" href="#" class="style-scope state-modifier">EP0318216A1</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">1987-11-18</span>
                    <span class="td style-scope patent-result">1989-05-31</span>
                    <span class="td style-scope patent-result">Chiron Corporation</span>
                    <span class="td style-scope patent-result">NANBV diagnostics and vaccines 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                      <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US4870026A/en"><a id="link" href="#" class="style-scope state-modifier">US4870026A</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">1982-09-16</span>
                    <span class="td style-scope patent-result">1989-09-26</span>
                    <span class="td style-scope patent-result">The General Hospital Corporation</span>
                    <span class="td style-scope patent-result">Non-A, non-B. hepatitis, virus, methods of identification purification, characterization, diagnosis and immunization 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                      <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/WO1990000597A1/en"><a id="link" href="#" class="style-scope state-modifier">WO1990000597A1</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">1988-07-06</span>
                    <span class="td style-scope patent-result">1990-01-25</span>
                    <span class="td style-scope patent-result">Genelabs Incorporated</span>
                    <span class="td style-scope patent-result">Post-transfusion, non-a, non-b hepatitis virus and antigens 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                      <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/EP0363025A2/en"><a id="link" href="#" class="style-scope state-modifier">EP0363025A2</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">1988-09-30</span>
                    <span class="td style-scope patent-result">1990-04-11</span>
                    <span class="td style-scope patent-result">The Research Foundation for Microbial Diseases of Osaka University</span>
                    <span class="td style-scope patent-result">Non-A, non-B hepatitis virus antigen peptide 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                      <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/WO1991006562A1/en"><a id="link" href="#" class="style-scope state-modifier">WO1991006562A1</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">1989-10-24</span>
                    <span class="td style-scope patent-result">1991-05-16</span>
                    <span class="td style-scope patent-result">University Of North Carolina At Chapel Hill</span>
                    <span class="td style-scope patent-result">Immunological domains of hepatitis delta virus antigen 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                      <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US5032511A/en"><a id="link" href="#" class="style-scope state-modifier">US5032511A</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">1987-03-31</span>
                    <span class="td style-scope patent-result">1991-07-16</span>
                    <span class="td style-scope patent-result">Mitsubishi Kasei Corporation</span>
                    <span class="td style-scope patent-result">DNA fragments coding for antigens specific to non-A non-B hepatitis, expression vectors containing said DNA fragments, transformants and process for producing said antigens 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                      <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/WO1991015603A1/en"><a id="link" href="#" class="style-scope state-modifier">WO1991015603A1</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">1990-04-05</span>
                    <span class="td style-scope patent-result">1991-10-17</span>
                    <span class="td style-scope patent-result">Genelabs Incorporated</span>
                    <span class="td style-scope patent-result">Enterically transmitted non-a/non-b hepatitis viral agent and characteristic epitopes thereof 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                      <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US5077193A/en"><a id="link" href="#" class="style-scope state-modifier">US5077193A</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">1988-12-20</span>
                    <span class="td style-scope patent-result">1991-12-31</span>
                    <span class="td style-scope patent-result">Immuno Japan Inc.</span>
                    <span class="td style-scope patent-result">Non-a, non-b hepatitis virus genome rna, cdna and virus antigen protein 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                      <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US5218099A/en"><a id="link" href="#" class="style-scope state-modifier">US5218099A</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">1986-04-01</span>
                    <span class="td style-scope patent-result">1993-06-08</span>
                    <span class="td style-scope patent-result">The United States Of America As Represented By The Department Of Health And Human Services</span>
                    <span class="td style-scope patent-result">Post-transfusion, non-A, non-B hepatitis virus polynucleotides 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                      <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US5258284A/en"><a id="link" href="#" class="style-scope state-modifier">US5258284A</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">1991-01-22</span>
                    <span class="td style-scope patent-result">1993-11-02</span>
                    <span class="td style-scope patent-result">University Of Maryland, School Of Medicine</span>
                    <span class="td style-scope patent-result">Nucleic acid probes specific for pathogenic strains of vibrio vulnificus and method employing the same 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                      <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US5275947A/en"><a id="link" href="#" class="style-scope state-modifier">US5275947A</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">1988-09-30</span>
                    <span class="td style-scope patent-result">1994-01-04</span>
                    <span class="td style-scope patent-result">The Research Foundation For Microbial Disease Of Osaka University</span>
                    <span class="td style-scope patent-result">Non-A, non-B hepatitis virus antigen peptide 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                      <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/WO1994018217A1/en"><a id="link" href="#" class="style-scope state-modifier">WO1994018217A1</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">1993-02-03</span>
                    <span class="td style-scope patent-result">1994-08-18</span>
                    <span class="td style-scope patent-result">Abbott Laboratories</span>
                    <span class="td style-scope patent-result">Non-a, non-b, non-c, non-d, non-e hepatitis reagents and methods for their use 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                      <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/WO1995021922A2/en"><a id="link" href="#" class="style-scope state-modifier">WO1995021922A2</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">1994-02-14</span>
                    <span class="td style-scope patent-result">1995-08-17</span>
                    <span class="td style-scope patent-result">Abbott Laboratories</span>
                    <span class="td style-scope patent-result">Non-a, non-b, non-c, non-d, non-e hepatitis reagents and methods for their use 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                      <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US5514539A/en"><a id="link" href="#" class="style-scope state-modifier">US5514539A</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">1993-06-29</span>
                    <span class="td style-scope patent-result">1996-05-07</span>
                    <span class="td style-scope patent-result">The United States Of America As Represented By The Department Of Health And Human Services</span>
                    <span class="td style-scope patent-result">Nucleotide and deduced amino acid sequences of the envelope 1 gene of 51 isolates of hepatitis C virus and the use of reagents derived from these sequences in diagnostic methods and vaccines 
       </span>
                  </div>
                  <template is="dom-repeat" class="style-scope patent-result"></template>

                  <div class="tr style-scope patent-result">
                    <span class="td famheader style-scope patent-result">Family To Family Citations</span>
                    <span class="td famheader style-scope patent-result"></span>
                    <span class="td famheader style-scope patent-result"></span>
                    <span class="td famheader style-scope patent-result"></span>
                    <span class="td famheader style-scope patent-result"></span>
                  </div>

                  <template is="dom-repeat" class="style-scope patent-result"></template>

                </div>
              </div>
            </div>
            <div style="padding: 10px 12px; font-size: 11px;" class="style-scope patent-result">* Cited by examiner, † Cited by third party</div>
            <template is="dom-if" class="style-scope patent-result"></template>

            
            <h3 id="nplCitations" class="scroll-target style-scope patent-result">
              Non-Patent Citations (30)
              <a href="#title" class="style-scope patent-result"><iron-icon icon="arrow-drop-up" class="style-scope patent-result x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M7 14l5-5 5 5z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon></a>
            </h3>
            <div class="responsive-table style-scope patent-result">
              <div class="table style-scope patent-result">
                <div class="thead style-scope patent-result">
                  <div class="tr style-scope patent-result">
                    <span class="th style-scope patent-result">Title</span>
                  </div>
                </div>
                <div class="tbody style-scope patent-result">
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      Bradley, D.W., et al., "Postransfusion Non-A, Non-B Hepatitis: Physicochemical Properties of Two Distinct Agents," The Journal of Infectious Diseases, 148(2):254-265 (1983).
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      Bradley, D.W., et al., Postransfusion Non A, Non B Hepatitis: Physicochemical Properties of Two Distinct Agents, The Journal of Infectious Diseases, 148(2):254 265 (1983).
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      Buti, M., et al., "Non-A, Non-B, Non-C, Non-E Acute Hepatitis: Does It Really Exist?" Journal of Hepatology, The Journal of the European Association for the Study of the Liver, in Abstracts of The 28th Annual Meeting of the European Association for the Study of the Liver, 1-4 Sep. 1993, Paris, France. 18(Suppl 1):S25 (1993).
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      Buti, M., et al., Non A, Non B, Non C, Non E Acute Hepatitis: Does It Really Exist Journal of Hepatology, The Journal of the European Association for the Study of the Liver, in Abstracts of The 28th Annual Meeting of the European Association for the Study of the Liver, 1 4 Sep. 1993, Paris, France. 18(Suppl 1):S25 (1993).
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      Chan, S.W., et al., "Analysis of a new hepatitis C virus type and its phylogenetic relationship to existing variants," Journal of General Virology, 75(5):1131-1141 (1992).
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      Chan, S.W., et al., Analysis of a new hepatitis C virus type and its phylogenetic relationship to existing variants, Journal of General Virology, 75(5):1131 1141 (1992).
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      Choo, Q L., et al., Isolation of a cDNA Clone Derived from a Blood Borne Non A, Non B Viral Hepatitis Genome, Science, 244:359 362 (1989).
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      Choo, Q-L., et al., "Isolation of a cDNA Clone Derived from a Blood-Borne Non-A, Non-B Viral Hepatitis Genome," Science, 244:359-362 (1989).
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      Jiang, X., et al., "Norwalk Virus Genome Cloning and Characterization," Science, 250:1580-1583 (1990).
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      Jiang, X., et al., Norwalk Virus Genome Cloning and Characterization, Science, 250:1580 1583 (1990).
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      Jones, W.F., et al., "The Role of Hepatitis C Virus (HCV) and Hepatitis E Virus (HEV) in Acute Hepatitis: Evidence for a Non-A,B,C,D,E Syndrome," The American Association for the Study of Liver Diseases, 16(2 Pt. 2):77A (1992).
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      Jones, W.F., et al., The Role of Hepatitis C Virus (HCV) and Hepatitis E Virus (HEV) in Acute Hepatitis: Evidence for a Non A,B,C,D,E Syndrome, The American Association for the Study of Liver Diseases, 16(2 Pt. 2):77A (1992).
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      Karayiannis, P., et al., "Studies of GB Hepatitis Agent in Tamarins," Hepatology, 9(2):186-192 (1989).
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      Karayiannis, P., et al., Studies of GB Hepatitis Agent in Tamarins, Hepatology, 9(2):186 192 (1989).
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      Matsui, S.M., et al., "The Isolation and Characterization of a Norwalk Virus-specific cDNA," J. Clin. Invest., 87:1456-1461 (1991).
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      Matsui, S.M., et al., The Isolation and Characterization of a Norwalk Virus specific cDNA, J. Clin. Invest., 87:1456 1461 (1991).
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      Matsuura, Y., et al., "Expression of the S-coded Genes of Lymphocytic Choriomeningitis Arenavirus using a Baculovirus Vector," J. Gen Virol., 67:1515-1529 (1986).
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      Matsuura, Y., et al., Expression of the S coded Genes of Lymphocytic Choriomeningitis Arenavirus using a Baculovirus Vector, J. Gen Virol., 67:1515 1529 (1986).
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      Murphy, F.A., "Virus taxonomy", pp. 15-57 in Fields Virology, vol. 1, Lippincott-Raven Publishers, Philadelphia, 1996.
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      Murphy, F.A., Virus taxonomy , pp. 15 57 in Fields Virology, vol. 1, Lippincott Raven Publishers, Philadelphia, 1996.
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      Overton, H.A., et al., "Identification of the N and NSs Proteins Coded by the Ambisense S RNA of Punta Toro Phlebovirus Using Monospecific Antisera Raised to Baculovirus Expressed N and NSs Proteins," Virology, 157:338-350 (1987).
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      Overton, H.A., et al., Identification of the N and NS s Proteins Coded by the Ambisense S RNA of Punta Toro Phlebovirus Using Monospecific Antisera Raised to Baculovirus Expressed N and NSs Proteins, Virology, 157:338 350 (1987).
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      Reyes, G.R., "New Strategies for Isolation of Low Abundance Viral and Host cDNAs: Application to Cloning of the Hepatitis E Virus and Analysis of Tissue-Specific Transcription," Seminars in Liver Disease, 12(3):289-300 (1992).
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      Reyes, G.R., et al., "Hepatitis E virus (HEV): epitope mapping and detection of strain variation," in Viral Hepatitus C, D, and E. Proceedings of the International Meeting on Non-A, Non-B Hepatitus, Tokyo, 27-30 Sep. 1989, T. Shikata, et al., eds. Elsevier Science Publishers, Amsterdam, NL. Chapter 43:237-245 (1989).
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      Reyes, G.R., et al., "Molecular Biology of Non-A, Non-B Hepatitis Agents: Hepatitis C and Hepatitis E Viruses," Advances in Virus Research, 40:57-103 (1991).
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      Reyes, G.R., et al., Hepatitis E virus (HEV): epitope mapping and detection of strain variation, in Viral Hepatitus C, D, and E. Proceedings of the International Meeting on Non A, Non B Hepatitus, Tokyo, 27 30 Sep. 1989, T. Shikata, et al., eds. Elsevier Science Publishers, Amsterdam, NL. Chapter 43:237 245 (1989).
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      Reyes, G.R., et al., Molecular Biology of Non A, Non B Hepatitis Agents: Hepatitis C and Hepatitis E Viruses, Advances in Virus Research, 40:57 103 (1991).
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      Reyes, G.R., New Strategies for Isolation of Low Abundance Viral and Host cDNAs: Application to Cloning of the Hepatitis E Virus and Analysis of Tissue Specific Transcription, Seminars in Liver Disease, 12(3):289 300 (1992).
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      U.S. application No. 08/196,030, Simons, et al., filed Feb. 14, 1994.
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      U.S. application No. 08/242,654, Simons, et al., filed May 13, 1994.
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                  </div>
                  <template is="dom-repeat" class="style-scope patent-result"></template>
                </div>
              </div>
            </div>
            <div style="padding: 10px 12px; font-size: 11px;" class="style-scope patent-result">* Cited by examiner, † Cited by third party</div>
            <template is="dom-if" class="style-scope patent-result"></template>

            
            <h3 id="citedBy" class="scroll-target style-scope patent-result">
              Cited By (9)
              <a href="#title" class="style-scope patent-result"><iron-icon icon="arrow-drop-up" class="style-scope patent-result x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M7 14l5-5 5 5z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon></a>
            </h3>
            <div class="responsive-table style-scope patent-result">
              <div class="table style-scope patent-result">
                <div class="thead style-scope patent-result">
                  <div class="tr style-scope patent-result">
                    <span class="th style-scope patent-result">Publication number</span>
                    <span class="th style-scope patent-result">Priority date</span>
                    <span class="th style-scope patent-result">Publication date</span>
                    <span class="th style-scope patent-result">Assignee</span>
                    <span class="th style-scope patent-result">Title</span>
                  </div>
                </div>
                <div class="tbody style-scope patent-result">
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                      <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US20040261897A1/en"><a id="link" href="#" class="style-scope state-modifier">US20040261897A1</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2003-06-12</span>
                    <span class="td style-scope patent-result">2004-12-30</span>
                    <span class="td style-scope patent-result">Symyx Technologies, Inc.</span>
                    <span class="td style-scope patent-result">Methods and apparatus for mixing powdered samples 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                      <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US6870043B2/en"><a id="link" href="#" class="style-scope state-modifier">US6870043B2</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2000-04-06</span>
                    <span class="td style-scope patent-result">2005-03-22</span>
                    <span class="td style-scope patent-result">The University Of Iowa Research Foundation</span>
                    <span class="td style-scope patent-result">Full-length GB virus C (hepatitis G virus) RNA transcripts are infectious in primary CD4 positive T cells 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                      <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US20050119472A1/en"><a id="link" href="#" class="style-scope state-modifier">US20050119472A1</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2002-10-24</span>
                    <span class="td style-scope patent-result">2005-06-02</span>
                    <span class="td style-scope patent-result">The University Of Iowa Research Foundation</span>
                    <span class="td style-scope patent-result">GB virus C and methods of treating viral infections 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                      <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US20060210966A1/en"><a id="link" href="#" class="style-scope state-modifier">US20060210966A1</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2005-03-18</span>
                    <span class="td style-scope patent-result">2006-09-21</span>
                    <span class="td style-scope patent-result">Asiagen Corporation</span>
                    <span class="td style-scope patent-result">Kits and methods for detecting bovine ephemeral fever virus 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                      <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US20070036825A1/en"><a id="link" href="#" class="style-scope state-modifier">US20070036825A1</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2005-02-17</span>
                    <span class="td style-scope patent-result">2007-02-15</span>
                    <span class="td style-scope patent-result">University Of Iowa Research Foundation</span>
                    <span class="td style-scope patent-result">Flavivirus NS5A proteins for the treatment of HIV 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                      <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US20090010932A1/en"><a id="link" href="#" class="style-scope state-modifier">US20090010932A1</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2003-06-05</span>
                    <span class="td style-scope patent-result">2009-01-08</span>
                    <span class="td style-scope patent-result">Stapleton Jack T</span>
                    <span class="td style-scope patent-result">Gb virus c (hepatitis g virus) for the treatment of hiv 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                      <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US20100127022A1/en"><a id="link" href="#" class="style-scope state-modifier">US20100127022A1</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2008-11-21</span>
                    <span class="td style-scope patent-result">2010-05-27</span>
                    <span class="td style-scope patent-result">Symyx Technologies, Inc.</span>
                    <span class="td style-scope patent-result">Dispensing valve 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                      <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US8032487B1/en"><a id="link" href="#" class="style-scope state-modifier">US8032487B1</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2003-10-29</span>
                    <span class="td style-scope patent-result">2011-10-04</span>
                    <span class="td style-scope patent-result">At&amp;T Intellectual Property I, L.P.</span>
                    <span class="td style-scope patent-result">System and method for synchronizing data in a networked system 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                      <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/CN105063018A/en"><a id="link" href="#" class="style-scope state-modifier">CN105063018A</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      *<template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2015-08-31</span>
                    <span class="td style-scope patent-result">2015-11-18</span>
                    <span class="td style-scope patent-result">山西大学</span>
                    <span class="td style-scope patent-result">Method for extracting total RNA from mature grape fruits 
       </span>
                  </div>
                  <template is="dom-repeat" class="style-scope patent-result"></template>

                  <div class="tr style-scope patent-result">
                    <span class="td famheader style-scope patent-result">Family To Family Citations</span>
                    <span class="td famheader style-scope patent-result"></span>
                    <span class="td famheader style-scope patent-result"></span>
                    <span class="td famheader style-scope patent-result"></span>
                    <span class="td famheader style-scope patent-result"></span>
                  </div>

                  <template is="dom-repeat" class="style-scope patent-result"></template>
                </div>

              </div>
            </div>
            <div style="padding: 10px 12px; font-size: 11px;" class="style-scope patent-result">* Cited by examiner, † Cited by third party, ‡ Family to family citation</div>
            <template is="dom-if" class="style-scope patent-result"></template>

            
            <h3 id="similarDocuments" class="scroll-target style-scope patent-result">
              Similar Documents
              <a href="#title" class="style-scope patent-result"><iron-icon icon="arrow-drop-up" class="style-scope patent-result x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M7 14l5-5 5 5z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon></a>
            </h3>
            <div class="responsive-table style-scope patent-result">
              <div class="table style-scope patent-result">
                <div class="thead style-scope patent-result">
                  <div class="tr style-scope patent-result">
                    <span class="th style-scope patent-result">Publication</span>
                    <span class="th style-scope patent-result">Publication Date</span>
                    <span class="th style-scope patent-result">Title</span>
                  </div>
                </div>
                <div class="tbody style-scope patent-result">
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      
                        <template is="dom-if" class="style-scope patent-result"></template>
                        
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US5856134A/en"><a id="link" href="#" class="style-scope state-modifier">US5856134A</a></state-modifier>
                        <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">1999-01-05</span>
                    <span class="td style-scope patent-result">Hepatitis G virus and molecular cloning thereof 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      
                        <template is="dom-if" class="style-scope patent-result"></template>
                        
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/JP4296174B2/en"><a id="link" href="#" class="style-scope state-modifier">JP4296174B2</a></state-modifier>
                        <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2009-07-15</span>
                    <span class="td style-scope patent-result">
  Hepatitis G virus and its molecular cloning
 
     </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      
                        <template is="dom-if" class="style-scope patent-result"></template>
                        
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US5871903A/en"><a id="link" href="#" class="style-scope state-modifier">US5871903A</a></state-modifier>
                        <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">1999-02-16</span>
                    <span class="td style-scope patent-result">HCV isolates 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      
                        <template is="dom-if" class="style-scope patent-result"></template>
                        
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/CA2065287C/en"><a id="link" href="#" class="style-scope state-modifier">CA2065287C</a></state-modifier>
                        <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">1999-12-21</span>
                    <span class="td style-scope patent-result">New hcv isolates 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      
                        <template is="dom-if" class="style-scope patent-result"></template>
                        
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/JP2662358B2/en"><a id="link" href="#" class="style-scope state-modifier">JP2662358B2</a></state-modifier>
                        <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">1997-10-08</span>
                    <span class="td style-scope patent-result">
  NANBV diagnostics
 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      
                        <template is="dom-if" class="style-scope patent-result"></template>
                        
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/AU666576B2/en"><a id="link" href="#" class="style-scope state-modifier">AU666576B2</a></state-modifier>
                        <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">1996-02-15</span>
                    <span class="td style-scope patent-result">NANBV diagnostics and vaccines 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      
                        <template is="dom-if" class="style-scope patent-result"></template>
                        
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US7201911B1/en"><a id="link" href="#" class="style-scope state-modifier">US7201911B1</a></state-modifier>
                        <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2007-04-10</span>
                    <span class="td style-scope patent-result">Cloned genomes of infectious hepatitis C viruses and uses thereof 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      
                        <template is="dom-if" class="style-scope patent-result"></template>
                        
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/JP2008133301A/en"><a id="link" href="#" class="style-scope state-modifier">JP2008133301A</a></state-modifier>
                        <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2008-06-12</span>
                    <span class="td style-scope patent-result">Immunoreactive polypeptide composition of hepatitis c virus 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      
                        <template is="dom-if" class="style-scope patent-result"></template>
                        
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/WO1995021922A2/en"><a id="link" href="#" class="style-scope state-modifier">WO1995021922A2</a></state-modifier>
                        <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">1995-08-17</span>
                    <span class="td style-scope patent-result">Non-a, non-b, non-c, non-d, non-e hepatitis reagents and methods for their use 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      
                        <template is="dom-if" class="style-scope patent-result"></template>
                        
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/WO1995032292A2/en"><a id="link" href="#" class="style-scope state-modifier">WO1995032292A2</a></state-modifier>
                        <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">1995-11-30</span>
                    <span class="td style-scope patent-result">Detection of viral antigens coded by reverse-reading frames 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      
                        <template is="dom-if" class="style-scope patent-result"></template>
                        
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US5874563A/en"><a id="link" href="#" class="style-scope state-modifier">US5874563A</a></state-modifier>
                        <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">1999-02-23</span>
                    <span class="td style-scope patent-result">Hepatitis G virus and molecular cloning thereof 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      
                        <template is="dom-if" class="style-scope patent-result"></template>
                        
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/EP0747482A2/en"><a id="link" href="#" class="style-scope state-modifier">EP0747482A2</a></state-modifier>
                        <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">1996-12-11</span>
                    <span class="td style-scope patent-result">Hepatitis GB virus recombinant proteins and uses thereof 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      
                        <template is="dom-if" class="style-scope patent-result"></template>
                        
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US5859230A/en"><a id="link" href="#" class="style-scope state-modifier">US5859230A</a></state-modifier>
                        <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">1999-01-12</span>
                    <span class="td style-scope patent-result">Non-A/non-B/non-C/non-D/non-E hepatitis agents and molecular cloning thereof 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      
                        <template is="dom-if" class="style-scope patent-result"></template>
                        
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/WO1995032290A2/en"><a id="link" href="#" class="style-scope state-modifier">WO1995032290A2</a></state-modifier>
                        <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">1995-11-30</span>
                    <span class="td style-scope patent-result">Non-a/non-b/non-c/non-d/non-e hepatitis agents and molecular cloning thereof 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      
                        <template is="dom-if" class="style-scope patent-result"></template>
                        
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US6720166B2/en"><a id="link" href="#" class="style-scope state-modifier">US6720166B2</a></state-modifier>
                        <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2004-04-13</span>
                    <span class="td style-scope patent-result">Non-a, non-b, non-c, non-c, non-d, non-e hepatitis reagents and methods for their use 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      
                        <template is="dom-if" class="style-scope patent-result"></template>
                        
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/DK175975B1/en"><a id="link" href="#" class="style-scope state-modifier">DK175975B1</a></state-modifier>
                        <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2005-10-10</span>
                    <span class="td style-scope patent-result">
  NANBV diagnostics and vaccines
 
     </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      
                        <template is="dom-if" class="style-scope patent-result"></template>
                        
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/AU684177C/en"><a id="link" href="#" class="style-scope state-modifier">AU684177C</a></state-modifier>
                        <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">1999-07-15</span>
                    <span class="td style-scope patent-result">Hepatitis G virus and molecular cloning thereof 
     </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      
                        <template is="dom-if" class="style-scope patent-result"></template>
                        
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US6586568B1/en"><a id="link" href="#" class="style-scope state-modifier">US6586568B1</a></state-modifier>
                        <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2003-07-01</span>
                    <span class="td style-scope patent-result">Non-A, non-B, non-C, non-D, non-E hepatitis reagents and methods for their use 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      
                        <template is="dom-if" class="style-scope patent-result"></template>
                        
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US6558898B1/en"><a id="link" href="#" class="style-scope state-modifier">US6558898B1</a></state-modifier>
                        <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2003-05-06</span>
                    <span class="td style-scope patent-result">Non-A, non-B, non-C, non-D, non-E hepatitis reagents and methods for their use 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      
                        <template is="dom-if" class="style-scope patent-result"></template>
                        
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/JPH08504421A/en"><a id="link" href="#" class="style-scope state-modifier">JPH08504421A</a></state-modifier>
                        <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">1996-05-14</span>
                    <span class="td style-scope patent-result">
  Peptide derived from C33 region of HCV, antibody against the peptide, and method for detecting HCV
 
     </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      
                        <template is="dom-if" class="style-scope patent-result"></template>
                        
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/EP0745129B1/en"><a id="link" href="#" class="style-scope state-modifier">EP0745129B1</a></state-modifier>
                        <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">2004-11-24</span>
                    <span class="td style-scope patent-result">Non-a, non-b, non-c, non-d, non-e hepatitis reagents and methods for their use 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td style-scope patent-result">
                      <template is="dom-if" class="style-scope patent-result"></template>
                      
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="scholar/17876299678964398653"><a id="link" href="#" class="style-scope state-modifier">Long</a></state-modifier>
                      <template is="dom-if" class="style-scope patent-result"></template>
                      <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">1994</span>
                    <span class="td style-scope patent-result">2 C. BRÉCHOT</span>
                  </div>
                  <template is="dom-repeat" class="style-scope patent-result"></template>
                </div>
              </div>
            </div>
            <template is="dom-if" class="style-scope patent-result"></template>

            
            <h3 id="relatedApplications" class="scroll-target style-scope patent-result">Priority And Related Applications</h3>

            
            <h3 id="parentApplications" class="scroll-target style-scope patent-result">
              Parent Applications (1)
              <a href="#title" class="style-scope patent-result"><iron-icon icon="arrow-drop-up" class="style-scope patent-result x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M7 14l5-5 5 5z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon></a>
            </h3>
            <div class="responsive-table style-scope patent-result">
              <div class="table style-scope patent-result">
                <div class="thead style-scope patent-result">
                  <div class="tr style-scope patent-result">
                    <span class="th style-scope patent-result">Application</span>
                    <span class="th style-scope patent-result">Priority date</span>
                    <span class="th style-scope patent-result">Filing date</span>
                    <span class="th style-scope patent-result">Relation</span>
                    <span class="th style-scope patent-result">Title</span>
                  </div>
                </div>
                <div class="tbody style-scope patent-result">
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US5824507A/en"><a id="link" href="#" class="style-scope state-modifier">US08/444,733</a></state-modifier><template is="dom-if" class="style-scope patent-result"></template>
                        <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td nowrap style-scope patent-result">1994-05-20</span>
                    <span class="td nowrap style-scope patent-result">1995-05-19</span>
                    <span class="td nowrap style-scope patent-result">
                        Division<template is="dom-if" class="style-scope patent-result"></template>
                        <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td style-scope patent-result">Hepatitis G virus and molecular cloning thereof 
       </span>
                  </div>
                  <template is="dom-repeat" class="style-scope patent-result"></template>
                </div>
              </div>
            </div>
            <template is="dom-if" class="style-scope patent-result"></template>

            <template is="dom-if" class="style-scope patent-result"></template>

            
            <h3 id="applicationPriorityApps" class="scroll-target style-scope patent-result">
              Priority Applications (1)
              <a href="#title" class="style-scope patent-result"><iron-icon icon="arrow-drop-up" class="style-scope patent-result x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M7 14l5-5 5 5z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon></a>
            </h3>
            <div class="responsive-table style-scope patent-result">
              <div class="table style-scope patent-result">
                <div class="thead style-scope patent-result">
                  <div class="tr style-scope patent-result">
                    <span class="th style-scope patent-result">Application</span>
                    <span class="th style-scope patent-result">Priority date</span>
                    <span class="th style-scope patent-result">Filing date</span>
                    <span class="th style-scope patent-result">Title</span>
                  </div>
                </div>
                <div class="tbody style-scope patent-result">
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US5874563A/en"><a id="link" href="#" class="style-scope state-modifier">US08/485,910</a></state-modifier><template is="dom-if" class="style-scope patent-result"></template>
                        <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td nowrap style-scope patent-result">1994-05-20</span>
                    <span class="td nowrap style-scope patent-result">1995-06-05</span>
                    <span class="td style-scope patent-result">
                        <template is="dom-if" class="style-scope patent-result"></template>
                        Hepatitis G virus and molecular cloning thereof 
       <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                  </div>
                  <template is="dom-repeat" class="style-scope patent-result"></template>
                </div>
              </div>
            </div>
            <template is="dom-if" class="style-scope patent-result"></template>

            
            <h3 id="appsClaimingPriority" class="scroll-target style-scope patent-result">
              Applications Claiming Priority (9)
              <a href="#title" class="style-scope patent-result"><iron-icon icon="arrow-drop-up" class="style-scope patent-result x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M7 14l5-5 5 5z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon></a>
            </h3>
            <div class="responsive-table style-scope patent-result">
              <div class="table style-scope patent-result">
                <div class="thead style-scope patent-result">
                  <div class="tr style-scope patent-result">
                    <span class="th style-scope patent-result">Application</span>
                    <span class="th style-scope patent-result">Filing date</span>
                    <span class="th style-scope patent-result">Title</span>
                  </div>
                </div>
                <div class="tbody style-scope patent-result">
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                        <template is="dom-if" class="style-scope patent-result"></template>
                        US24698594A<template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td nowrap style-scope patent-result">1994-05-20</span>
                    <span class="td style-scope patent-result"></span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                        <template is="dom-if" class="style-scope patent-result"></template>
                        US28554394A<template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td nowrap style-scope patent-result">1994-08-03</span>
                    <span class="td style-scope patent-result"></span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                        <template is="dom-if" class="style-scope patent-result"></template>
                        US28555894A<template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td nowrap style-scope patent-result">1994-08-03</span>
                    <span class="td style-scope patent-result"></span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                        <template is="dom-if" class="style-scope patent-result"></template>
                        US32972994A<template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td nowrap style-scope patent-result">1994-10-26</span>
                    <span class="td style-scope patent-result"></span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                        <template is="dom-if" class="style-scope patent-result"></template>
                        US34427194A<template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td nowrap style-scope patent-result">1994-11-23</span>
                    <span class="td style-scope patent-result"></span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US5546335A/en"><a id="link" href="#" class="style-scope state-modifier">US08/357,009</a></state-modifier><template is="dom-if" class="style-scope patent-result"></template>
                        <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td nowrap style-scope patent-result">1994-12-16</span>
                    <span class="td style-scope patent-result">Absolute value calculation method and circuit 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                        <template is="dom-if" class="style-scope patent-result"></template>
                        US38988695A<template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td nowrap style-scope patent-result">1995-02-15</span>
                    <span class="td style-scope patent-result"></span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US5824507A/en"><a id="link" href="#" class="style-scope state-modifier">US08/444,733</a></state-modifier><template is="dom-if" class="style-scope patent-result"></template>
                        <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td nowrap style-scope patent-result">1995-05-19</span>
                    <span class="td style-scope patent-result">Hepatitis G virus and molecular cloning thereof 
       </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">
                        <state-modifier act="{&quot;type&quot;: &quot;OPEN_RESULT&quot;, &quot;result&quot;: &quot;$result&quot;}" class="style-scope patent-result" data-result="patent/US5874563A/en"><a id="link" href="#" class="style-scope state-modifier">US08/485,910</a></state-modifier><template is="dom-if" class="style-scope patent-result"></template>
                        <template is="dom-if" class="style-scope patent-result"></template>
                    </span>
                    <span class="td nowrap style-scope patent-result">1995-06-05</span>
                    <span class="td style-scope patent-result">Hepatitis G virus and molecular cloning thereof 
       </span>
                  </div>
                  <template is="dom-repeat" class="style-scope patent-result"></template>
                </div>
              </div>
            </div>
            <template is="dom-if" class="style-scope patent-result"></template>

            <template is="dom-if" class="style-scope patent-result"></template>

            
            <h3 id="legalEvents" class="scroll-target style-scope patent-result">
              Legal Events
              <a href="#title" class="style-scope patent-result"><iron-icon icon="arrow-drop-up" class="style-scope patent-result x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M7 14l5-5 5 5z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon></a>
            </h3>
            <div class="responsive-table style-scope patent-result">
              <div class="table style-scope patent-result">
                <div class="thead style-scope patent-result">
                  <div class="tr style-scope patent-result">
                    <span class="th style-scope patent-result">Date</span>
                    <span class="th style-scope patent-result">Code</span>
                    <span class="th style-scope patent-result">Title</span>
                    <span class="th style-scope patent-result">Description</span>
                  </div>
                </div>
                <div class="tbody style-scope patent-result">
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">2002-07-12</span>
                    <span class="td nowrap style-scope patent-result">FPAY</span>
                    <span class="td nowrap style-scope patent-result">Fee payment</span>
                    <span class="td style-scope patent-result">
                      
                      <p class="style-scope patent-result"><strong class="style-scope patent-result">Year of fee payment</strong>: 4</p>
                      <template is="dom-repeat" as="attr" class="style-scope patent-result"></template>
                    </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">2006-07-26</span>
                    <span class="td nowrap style-scope patent-result">FPAY</span>
                    <span class="td nowrap style-scope patent-result">Fee payment</span>
                    <span class="td style-scope patent-result">
                      
                      <p class="style-scope patent-result"><strong class="style-scope patent-result">Year of fee payment</strong>: 8</p>
                      <template is="dom-repeat" as="attr" class="style-scope patent-result"></template>
                    </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">2010-09-27</span>
                    <span class="td nowrap style-scope patent-result">REMI</span>
                    <span class="td nowrap style-scope patent-result">Maintenance fee reminder mailed</span>
                    <span class="td style-scope patent-result">
                      <template is="dom-repeat" as="attr" class="style-scope patent-result"></template>
                    </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">2011-02-23</span>
                    <span class="td nowrap style-scope patent-result">LAPS</span>
                    <span class="td nowrap style-scope patent-result">Lapse for failure to pay maintenance fees</span>
                    <span class="td style-scope patent-result">
                      <template is="dom-repeat" as="attr" class="style-scope patent-result"></template>
                    </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">2011-03-21</span>
                    <span class="td nowrap style-scope patent-result">STCH</span>
                    <span class="td nowrap style-scope patent-result">Information on status: patent discontinuation</span>
                    <span class="td style-scope patent-result">
                      
                      <p class="style-scope patent-result"><strong class="style-scope patent-result">Free format text</strong>: PATENT EXPIRED DUE TO NONPAYMENT OF MAINTENANCE FEES UNDER 37 CFR 1.362</p>
                      <template is="dom-repeat" as="attr" class="style-scope patent-result"></template>
                    </span>
                  </div>
                  
                  <div class="tr style-scope patent-result">
                    <span class="td nowrap style-scope patent-result">2011-04-12</span>
                    <span class="td nowrap style-scope patent-result">FP</span>
                    <span class="td nowrap style-scope patent-result">Lapsed due to failure to pay maintenance fee</span>
                    <span class="td style-scope patent-result">
                      
                      <p class="style-scope patent-result"><strong class="style-scope patent-result">Effective date</strong>: 20110223</p>
                      <template is="dom-repeat" as="attr" class="style-scope patent-result"></template>
                    </span>
                  </div>
                  <template is="dom-repeat" class="style-scope patent-result"></template>
                </div>
              </div>
            </div>
            <template is="dom-if" class="style-scope patent-result"></template>

            
              <h3 id="concepts" class="scroll-target layout horizontal style-scope patent-result">
                <div class="flex style-scope patent-result">Concepts</div>
                <a href="#title" class="style-scope patent-result"><iron-icon icon="arrow-drop-up" class="style-scope patent-result x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M7 14l5-5 5 5z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon></a>
              </h3>
              <div class="layout horizontal style-scope patent-result">
                <div class="flex style-scope patent-result">
                  <span id="conceptsWarning" class="righthead tooltip-hint style-scope patent-result">machine-extracted</span>
                  <overlay-tooltip positioning-target="#conceptsWarning" direction="bottom" class="style-scope patent-result" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      The concepts are extracted by a computer and may be incomplete or incorrect.
    </div>
  </overlay-tooltip>
                </div>
                <span class="headerButton style-scope patent-result"><a href="#" target="_blank" style="display: inline-block;" class="style-scope patent-result"><iron-icon icon="icons:file-download" class="style-scope patent-result x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M19 9h-4V3H9v6H5l7 7 7-7zM5 18v2h14v-2H5z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon> Download</a></span>
                <span class="style-scope patent-result">
                  <dropdown-menu multi="true" label="Filter table" show-label="true" change-action="RESULT_CONCEPT_MENU" class="style-scope patent-result">
    
    <span class="label style-scope dropdown-menu">
      <span class="style-scope dropdown-menu">Filter table</span>
      <span id="selected" class="style-scope dropdown-menu" hidden="">
        <span style="margin: 0 4px 0 2px" class="style-scope dropdown-menu">·</span>
        <span class="style-scope dropdown-menu"></span>
      </span>
      <iron-icon icon="icons:arrow-drop-down" class="style-scope dropdown-menu x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M7 10l5 5 5-5z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon><iron-icon icon="icons:arrow-drop-up" class="style-scope dropdown-menu x-scope iron-icon-0" hidden=""><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M7 14l5-5 5 5z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
    </span>
    <iron-dropdown allow-outside-scroll="" horizontal-align="left" vertical-align="top" class="style-scope dropdown-menu" aria-disabled="false" aria-hidden="true" style="outline: none; display: none;">
    

    <div id="contentWrapper" class="style-scope iron-dropdown">
      <div class="dropdown-content style-scope dropdown-menu" slot="dropdown-content" id="menu" multi="">
        <div class="style-scope dropdown-menu">
          
            <div class="item style-scope dropdown-menu">Anatomy</div>
          
            <div class="item style-scope dropdown-menu">Chemical class</div>
          
            <div class="item style-scope dropdown-menu">Chemical compound</div>
          
            <div class="item style-scope dropdown-menu">Chemical group</div>
          
            <div class="item style-scope dropdown-menu">Diseases</div>
          
            <div class="item style-scope dropdown-menu">Drugs</div>
          
            <div class="item style-scope dropdown-menu">Effects</div>
          
            <div class="item style-scope dropdown-menu">Human genes</div>
          
            <div class="item style-scope dropdown-menu">Inorganic materials</div>
          
            <div class="item style-scope dropdown-menu">Methods</div>
          
            <div class="item style-scope dropdown-menu">Natural products</div>
          
            <div class="item style-scope dropdown-menu">Nutrition</div>
          
            <div class="item style-scope dropdown-menu">Polymers</div>
          
            <div class="item style-scope dropdown-menu">Proteins</div>
          
            <div class="item style-scope dropdown-menu">Species</div>
          
            <div class="item style-scope dropdown-menu">Substances</div>
          
            <div class="item style-scope dropdown-menu">Toxicity</div>
          <template is="dom-repeat" class="style-scope dropdown-menu"></template>
        </div>
      </div>
    </div>
  </iron-dropdown>
  </dropdown-menu>
                </span>
              </div>
              <div class="responsive-table style-scope patent-result">
                <div class="table style-scope patent-result">
                  <div class="thead style-scope patent-result">
                    <div class="tr style-scope patent-result">
                      <span class="th style-scope patent-result">Name</span>
                      <span class="th style-scope patent-result">Image</span>
                      <span class="th style-scope patent-result">Sections</span>
                      <span class="th style-scope patent-result">Count</span>
                      <span class="th style-scope patent-result">Query match</span>
                    </div>
                  </div>
                  <div class="tbody style-scope patent-result">
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Species style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">GB virus C</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">title,abstract,description</span>
                        <span class="td nowrap style-scope patent-result">598</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Methods style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">molecular cloning</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">title,description</span>
                        <span class="td nowrap style-scope patent-result">3</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Proteins style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">processed proteins &amp; peptides</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,abstract,description</span>
                        <span class="td nowrap style-scope patent-result">218</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Human genes style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">processed proteins &amp; peptides</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,abstract,description</span>
                        <span class="td nowrap style-scope patent-result">195</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Proteins style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">antigens</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,abstract,description</span>
                        <span class="td nowrap style-scope patent-result">193</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Human genes style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">antigens</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,abstract,description</span>
                        <span class="td nowrap style-scope patent-result">192</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Substances style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">antigen</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,abstract,description</span>
                        <span class="td nowrap style-scope patent-result">189</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Polymers style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">polypeptide</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,abstract,description</span>
                        <span class="td nowrap style-scope patent-result">172</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Methods style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">method</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,abstract,description</span>
                        <span class="td nowrap style-scope patent-result">152</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Human genes style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">polynucleotide</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,abstract,description</span>
                        <span class="td nowrap style-scope patent-result">104</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Proteins style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">polynucleotide</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,abstract,description</span>
                        <span class="td nowrap style-scope patent-result">104</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Substances style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">polynucleotide</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,abstract,description</span>
                        <span class="td nowrap style-scope patent-result">104</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Substances style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">fragment</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,abstract,description</span>
                        <span class="td nowrap style-scope patent-result">89</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Diseases style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">hepatitis</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,abstract,description</span>
                        <span class="td nowrap style-scope patent-result">75</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Chemical class style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">nucleic acids</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,abstract,description</span>
                        <span class="td nowrap style-scope patent-result">75</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Human genes style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">nucleic acids</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,abstract,description</span>
                        <span class="td nowrap style-scope patent-result">66</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Proteins style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">nucleic acids</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,abstract,description</span>
                        <span class="td nowrap style-scope patent-result">66</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Toxicity style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">hepatitis</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,abstract,description</span>
                        <span class="td nowrap style-scope patent-result">62</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Methods style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">testing method</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,abstract,description</span>
                        <span class="td nowrap style-scope patent-result">44</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Substances style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">sample</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">158</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Anatomy style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">serum</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">149</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Methods style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">nucleic acid amplification method</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">119</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Effects style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">amplification</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">118</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Methods style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">chemical reaction</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">107</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Substances style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">complementary DNA</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">80</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Species style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">Hepacivirus C</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">79</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Species style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">Viruses</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">68</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Effects style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">virological effect</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">64</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Methods style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">hybridization</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">54</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Substances style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">particle</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">41</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Species style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">Flaviviridae</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">36</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Nutrition style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">amino acid</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">36</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Chemical class style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">amino acids</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">34</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Effects style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">complement effect</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">30</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Chemical group style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">nucleotide group</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">27</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Substances style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">nucleotide</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">26</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Substances style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">chemical reaction reagent</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">21</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Species style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">Hepatovirus A</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">19</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Methods style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">PCR amplification</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">19</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Species style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">Hepatitis B virus</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">18</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Methods style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">manufacturing process</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">18</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Species style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">Hepatitis E virus</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">15</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Effects style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">replication</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">15</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Species style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">Primates</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">14</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Methods style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">ligase chain reaction</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">13</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Proteins style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">DNA-directed DNA polymerase</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">7</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Human genes style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">DNA-directed DNA polymerase</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">7</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Diseases style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">Hepatitis delta</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">7</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Species style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">Hepatitis delta virus</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">7</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Proteins style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">Q beta Replicase</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">6</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Substances style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">DNA probe</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">5</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Methods style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">displacement reaction</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">4</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Human genes style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">Alanine aminotransferase 1</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">3</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Proteins style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">Alanine transaminase</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">3</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Substances style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">deoxyribonucleotide</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">2</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Chemical group style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">deoxyribonucleotide group</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">2</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Effects style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">serological effect</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">claims,description</span>
                        <span class="td nowrap style-scope patent-result">2</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Methods style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">detection method</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">abstract,description</span>
                        <span class="td nowrap style-scope patent-result">45</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Proteins style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">Open Reading Frames</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">abstract,description</span>
                        <span class="td nowrap style-scope patent-result">41</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Drugs style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">vaccine</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">abstract,description</span>
                        <span class="td nowrap style-scope patent-result">33</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Methods style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">preparation method</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">abstract,description</span>
                        <span class="td nowrap style-scope patent-result">21</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Effects style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">antigenic effect</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">abstract,description</span>
                        <span class="td nowrap style-scope patent-result">20</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Methods style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">isolation</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">abstract,description</span>
                        <span class="td nowrap style-scope patent-result">18</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    
                      <div style="background-color: rgba(129, 201, 144, 0.000)" class="tr layout horizontal conceptDomain Methods style-scope patent-result">
                        <span class="td flex style-scope patent-result">
                          <concept-mention class="style-scope patent-result">
    
    <span id="target" tabindex="0" aria-label="Details of concept" role="link" class="style-scope concept-mention">
      
        <iron-icon class="inline-icon style-scope concept-mention x-scope iron-icon-0" icon="icons:label"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope iron-icon"><g class="style-scope iron-icon"><path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z" class="style-scope iron-icon"></path></g></svg>
    
  </iron-icon>
      <template is="dom-if" class="style-scope concept-mention"></template>
      
                           <span class="style-scope patent-result">diagnostic procedure</span>
                          
      <template is="dom-if" restamp="" class="style-scope concept-mention"></template>
    </span>
  </concept-mention>
                        </span>
                        <span class="td nowrap style-scope patent-result">
                          <template is="dom-if" class="style-scope patent-result"></template>
                        </span>
                        <span class="td nowrap style-scope patent-result">abstract,description</span>
                        <span class="td nowrap style-scope patent-result">7</span>
                        <span class="td nowrap style-scope patent-result">0.000</span>
                      </div>
                    <template is="dom-repeat" initial-count="25" class="style-scope patent-result"></template>
                    
                      <div class$="tr layout horizontal conceptDomain" class="style-scope patent-result">
                        <a href="#" class="td flex style-scope patent-result">
                          Show all concepts from the description section
                        </a>
                      </div>
                    <template is="dom-if" class="style-scope patent-result"></template>
                  </div>
                </div>
              </div>
            <template is="dom-if" class="style-scope patent-result"></template>

            <div id="notices" class="vertical layout center style-scope patent-result">
              
                <span class="style-scope patent-result">Data provided by IFI CLAIMS Patent Services</span>
              <template is="dom-if" class="style-scope patent-result"></template>
            </div>
          </div>
          <div class="bottombar style-scope patent-result">
            <bottom-bar class="style-scope patent-result"><template is="dom-if" class="style-scope bottom-bar"></template>
</bottom-bar>
          </div>
        </div>
      </div>
      <template is="dom-if" class="style-scope patent-result"></template>
    </div>
  </patent-result>
    <template is="dom-if" restamp="" class="style-scope result-container"></template>
    <template is="dom-if" restamp="" class="style-scope result-container"></template>
    <template is="dom-if" restamp="" class="style-scope result-container"></template>
  </result-container>
        </div>
      </div>
    
        </div>
      </div>
      <search-footer class="style-scope search-ui">
    
    

    <div id="footer" class="vertical layout center style-scope search-footer">
        <span class="links style-scope search-footer">
          <overlay-tooltip positioning-target="#footerAbout" direction="top" class="style-scope search-footer" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      Learn more about data coverage, search syntax and other features.
    </div>
  </overlay-tooltip>
          <a id="footerAbout" data-proto="ABOUT" href="https://support.google.com/faqs/answer/6390996" target="_blank" class="style-scope search-footer">About</a>
          <overlay-tooltip positioning-target="#footerFeedback" direction="top" class="style-scope search-footer" style="display: none;">
    
    <div id="arrow" class="style-scope overlay-tooltip" hidden=""></div>
    <div class="style-scope overlay-tooltip">
      Send feedback about technical issues or feature requests for Google Patents.
    </div>
  </overlay-tooltip>
          <a id="footerFeedback" data-proto="SEND_FEEDBACK" href="#" onclick="sendFeedback();" class="style-scope search-footer">Send Feedback</a>
          <a data-proto="PUBLIC_DATASETS" href="https://console.cloud.google.com/marketplace/product/google_patents_public_datasets/google-patents-public-data" target="_blank" class="style-scope search-footer">Public Datasets</a>
          <a data-proto="TERMS" href="https://www.google.com/policies/terms/" target="_blank" class="style-scope search-footer">Terms</a>
          <a data-proto="PRIVACY_POLICY" href="https://www.google.com/privacy/privacy-policy.html" target="blank" class="style-scope search-footer">Privacy Policy</a>
          <a data-proto="SEARCH_HELP" href="https://support.google.com/websearch/" target="_blank" class="style-scope search-footer">Help</a>
        </span>
    </div>
  </search-footer>
    </div>

  </search-ui>
  </search-result>
    <template is="dom-if" restamp="" class="style-scope search-app"></template>
    <template is="dom-if" restamp="" class="style-scope search-app"></template>
    <template is="dom-if" restamp="" class="style-scope search-app"></template>
    <paper-toast id="toast" duration="3000" class="style-scope search-app x-scope paper-toast-0" aria-hidden="true" style="outline: none; display: none;">
    

    <span id="label" class="style-scope paper-toast"></span>
    
      <div id="noticeContent" class="style-scope search-app"></div>
    
  </paper-toast>
  </search-app>
    <script type="text/javascript" src="//www.gstatic.com/feedback/api.js"></script>
    <script type="text/javascript" src="//www.gstatic.com/feedback/js/help/prod/service/lazy.min.js"></script>
    <script type="text/javascript">
      if (window.help && window.help.service) {
        helpApi = window.help.service.Lazy.create(0, {apiKey: 'AIzaSyDTEI_0tLX4varJ7bwK8aT-eOI5qr3BmyI', locale: 'en-US'});
        window.requestedSurveys = new Set();
        window.requestSurvey = function(triggerId) {
          if (window.requestedSurveys.has(triggerId)) {
            return;
          }
          window.requestedSurveys.add(triggerId);
          helpApi.requestSurvey({
            triggerId: triggerId,
            enableTestingMode: false,
            callback: (requestSurveyCallbackParam) => {
              if (!requestSurveyCallbackParam.surveyData) {
                return;
              }
              helpApi.presentSurvey({
                productData: {
                  productVersion: window.version,
                  customData: {
                    "experiments": "",
                  },
                },
                surveyData: requestSurveyCallbackParam.surveyData,
                colorScheme: 1,
                customZIndex: 10000,
              });
            }
          });
        };

        window.requestSurvey('YXTwAsvoW0kedxbuTdH0RArc9VhT');
      }
    </script>
    <script src="/sw/null_loader.js"></script>
  

<iframe id="apiproxyddf475d63cd8173a637a287407211e3c7614fb050.3247846756" name="apiproxyddf475d63cd8173a637a287407211e3c7614fb050.3247846756" style="width: 1px; height: 1px; position: absolute; top: -100px; display: none;" src="https://scone-pa.clients6.google.com/static/proxy.html?usegapi=1&amp;jsh=m%3B%2F_%2Fscs%2Fabc-static%2F_%2Fjs%2Fk%3Dgapi.lb.en.IKZeRvoAYNY.O%2Fam%3DAAAQ%2Fd%3D1%2Frs%3DAHpOoo-SMWTzMRJrTty6iE5dL_aWGYOnuw%2Fm%3D__features__#parent=https%3A%2F%2Fpatents.google.com&amp;rpctoken=511142070" tabindex="-1" aria-hidden="true"></iframe><iron-a11y-announcer>
    
    <div class="style-scope iron-a11y-announcer" aria-live="polite"></div>
  </iron-a11y-announcer></body></html>